<!DOCTYPE html>
<!-- saved from url=(0083)https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/d11094.htm -->
<html class="" data-reactroot=""><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><script src="./1094.癌症中的DNA甲基化 - 搜狗科学百科_files/analytics.js.download" type="text/javascript"></script>
<script type="text/javascript">window.addEventListener('DOMContentLoaded',function(){var v=archive_analytics.values;v.service='wb';v.server_name='wwwb-app222.us.archive.org';v.server_ms=295;archive_analytics.send_pageview({});});</script>
<script type="text/javascript" src="./1094.癌症中的DNA甲基化 - 搜狗科学百科_files/bundle-playback.js.download" charset="utf-8"></script>
<script type="text/javascript" src="./1094.癌症中的DNA甲基化 - 搜狗科学百科_files/wombat.js.download" charset="utf-8"></script>
<script type="text/javascript">
  __wm.init("https://web.archive.org/web");
  __wm.wombat("https://baike.sogou.com/kexue/d11094.htm","20221025130020","https://web.archive.org/","web","/_static/",
	      "1666702820");
</script>
<link rel="stylesheet" type="text/css" href="./1094.癌症中的DNA甲基化 - 搜狗科学百科_files/banner-styles.css">
<link rel="stylesheet" type="text/css" href="./1094.癌症中的DNA甲基化 - 搜狗科学百科_files/iconochive.css">
<!-- End Wayback Rewrite JS Include -->
<meta name="save" content="history"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="baidu-site-verification" content="VWGb6TyYx8"><meta content="癌症中的DNA甲基化 - 搜狗科学百科" name="keywords"><meta content="搜狗科学百科是一部有着平等、协作、分享、自由理念的网络科学全书，为每一个互联网用户创造一个涵盖所有领域知识、服务的中文知识性平台。" name="description"><meta http-equiv="x-dns-prefetch-control" content="on"><meta name="server" baike="235" ip="210" env="online"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025130020/https://cache.soso.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025130020/https://hhy.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025130020/https://pic.baike.soso.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025130020/https://ugc.qpic.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025130020/https://xui.ptlogin2.qq.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025130020/https://q1.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025130020/https://q2.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025130020/https://q3.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025130020/https://q4.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025130020/https://q.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025130020/https://img01.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025130020/https://img02.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025130020/https://img03.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025130020/https://img04.sogoucdn.com/"><link rel="Shortcut Icon" href="https://web.archive.org/web/20221025130020im_/https://www.sogou.com/images/logo/new/favicon.ico?v=4"><link rel="Bookmark" href="https://www.sogou.com/images/logo/new/favicon.ico?v=4"><link href="./1094.癌症中的DNA甲基化 - 搜狗科学百科_files/base_b849887.css" rel="stylesheet"><link href="./1094.癌症中的DNA甲基化 - 搜狗科学百科_files/detail_378aed5.css" rel="stylesheet"><link href="./1094.癌症中的DNA甲基化 - 搜狗科学百科_files/inviteAudit_7894507.css" rel="stylesheet"><link rel="stylesheet" href="./1094.癌症中的DNA甲基化 - 搜狗科学百科_files/highlight.min.css"><title>癌症中的DNA甲基化 - 搜狗科学百科</title><style>.onekey-close {
	position: absolute;
	top: 16px;
	right: 16px;
	width: 24px;
	height: 24px;
	background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	text-indent: -999em;
	background-size: 84px;
	background-position: -63px 0;
}

.onekey-login {
	position: absolute;
	top: 16.4%;
	left: 0;
	right: 0;
	width: 100%;
}

/* .onekey-login-img {
    width: 75px;
    height: 75px;
    background: url("https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/images/sprite_wap_baike.png") no-repeat;
    background-size: 100px 91px;
    background-position: 0 0;
    background-repeat: no-repeat;
    margin: 0 auto;
} */

.onekey-login-title {
	text-align: center;
	padding-bottom: 3px;
	font-size: 21px;
	font-weight: bold;
	line-height: 30px;
	color: #000;
}

.onekey-login-txt {
	text-align: center;
	font-family: PingFangSC;
	font-size: 14px;
	line-height: 20px;
	color: #8f8f8f;
}

.onekey-login-qq,
.onekey-login-wx,
.onekey-login-phone {
	display: block;
	width: 245px;
	height: 54px;
	border-radius: 45px;
	text-align: center;

	margin: 0 auto;
	font-size: 17px;
	line-height: 24px;
	color: #000;
	/* padding: 16px 77px; */
	border-radius: 12px;
	border: solid 1px #e0e0e0;
}
.onekey-qq-content,
.onekey-vx-content,
.onekey-phone-content {
	display: inline-block;
	margin-top: 16px;
}
.onekey-qq-content {
	padding: 0 5px;
}

.onekey-login-qq {
	margin-top: 48px;
	margin-bottom: 24px;
}

.onekey-login-qq:before {
	display: inline-block;
	content: "";
	width: 20px;
	height: 20px;
	background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	background-size: 80px;
	background-position: -20px 0;
	vertical-align: top;
	margin: 17px 8px 0 0;
}

.onekey-login-wx {
	margin-bottom: 24px;
}

.onekey-login-wx:before {
	display: inline-block;
	content: "";
	width: 21px;
	height: 21px;
	background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	background-size: 84px;
	background-position: 0 0;
	vertical-align: top;
	margin: 17px 10px 0 0;
}

.onekey-login-phone {
}

.onekey-login-phone:before {
	display: inline-block;
	content: "";
	width: 21px;
	height: 21px;
	background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	background-size: 84px;
	background-position: -42px 0;
	vertical-align: top;
	margin: 17px 10px 0 0;
}

.onekey-fixed {
	z-index: 100;
	position: fixed;
	top: 0;
	bottom: 0;
	left: 0;
	right: 0;
	background: #fff;
	width: 100%;
	height: 100%;
}

.onekey-fixed.forbid {
	z-index: 100;
	position: fixed;
	top: auto;
	bottom: 68px;
	left: 9%;
	right: 0;
	background: rgba(0, 0, 0, 0.7);
	width: 82%;
	height: 43px;
	border-radius: 25px;
	color: #ffffff;
}
.onekey-login-title.forbid {
	text-align: center;
	padding-bottom: 3px;
	font-size: 14px;
	font-weight: normal;
	line-height: 30px;
	color: white;
}
</style><style>#login_mask {
  background: #000;
  opacity: 0.5;
  filter: alpha(opacity=50);
  position: fixed;
  /*fixed好像在哪个IE上有BUG，先用用*/
  left: 0;
  top: 0;
  z-index: 999;
  height: 100%;
}

#login_iframe_container {
  position: fixed;
  width: 550px;
  height: 360px;
  z-index: 1020;
  background-color: #ffffff;
}

@media screen and (max-width: 828px) {
  #login_iframe_container {
    top: 50% !important;
    left: 50% !important;
    transform: translate(-50%, -50%);
  }
}

#login_iframe_container.new-login {
  width: 550px;
  height: 360px;
  background-image: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/background_2a4a8a6.png);
}

#login_iframe_container.new-login.no-bg {
  background: #fff;
}

#login_iframe_container.new-login .login-title {
  width: 100%;
  height: 42px;
  line-height: 42px;
  text-align: center;
  font-size: 30px;
  letter-spacing: 0.19px;
  color: #ffffff;
  margin-top: 62px;
}
#login_iframe_container.new-login .forbid-title {
  width: 100%;
  height: 42px;
  line-height: 42px;
  text-align: center;
  font-size: 24px;
  letter-spacing: 0.19px;
  color: #333333;
  margin-top: 150px;
}

#login_iframe_container.new-login.no-bg .login-title {
  color: #333333;
}

#login_iframe_container.new-login .login-subtitle {
  width: 100%;
  height: 18px;
  line-height: 18px;
  font-size: 13px;
  letter-spacing: 0.08px;
  color: #ffffff;
  text-align: center;
  margin-top: 9px;
  margin-bottom: 43px;
}

#login_iframe_container.new-login.no-bg .login-subtitle {
  color: #999999;
}

#login_iframe_container.new-login .login-subtitle::before {
  content: '';
  display: inline-block;
  width: 10px;
  height: 1px;
  background-color: #ffffff;
  position: relative;
  top: -4px;
  left: -5px;
}

#login_iframe_container.new-login .login-subtitle::after {
  content: '';
  display: inline-block;
  width: 10px;
  height: 1px;
  background-color: #ffffff;
  position: relative;
  top: -4px;
  left: 5px;
}

#login_iframe_container.new-login.no-bg .login-subtitle::before {
  background-color: #999999;
}

#login_iframe_container.new-login.no-bg .login-subtitle::after {
  background-color: #999999;
}

#login_iframe_container.new-login .close-btn {
  position: absolute;
  top: 20px;
  right: 20px;
  width: 12px;
  height: 12px;
  background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/login-sprites_e3853e5.png) -59px -10px;
  background-size: 81px 91px;
  cursor: pointer;
}

#login_iframe_container.new-login .login-btn {
  width: 220px;
  height: 47px;
  border-radius: 24px;
  border: solid 1px #dddddd;
  background-color: #ffffff;
  margin: 0 auto;
  margin-top: 28px;
  position: relative;
  display: block;
}

#login_iframe_container.new-login .login-btn .login-icon {
  position: absolute;
}

#login_iframe_container.new-login .login-btn .login-text {
  width: 61px;
  height: 47px;
  line-height: 47px;
  vertical-align: middle;
  font-size: 15px;
  letter-spacing: 0.1px;
  color: #666666;
  position: absolute;
  right: 62px;
}

#login_iframe_container.new-login .login-btn.qq-btn .login-icon {
  width: 22px;
  height: 27px;
  top: 10px;
  left: 67px;
  background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/login-sprites_e3853e5.png) -10px -54px;
  background-size: 81px 91px;
}

#login_iframe_container.new-login .login-btn.qq-btn .login-text {
  right: 59px;
}

#login_iframe_container.new-login .login-btn.wechat-btn .login-icon {
  width: 29px;
  height: 24px;
  top: 12px;
  left: 62px;
  background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/login-sprites_e3853e5.png) -10px -10px;
  background-size: 81px 91px;
}</style><style>/* -- container -- */
.rodal,
.rodal-mask {
    top: 0;
    left: 0;
    width: 100%;
    height: 100%;
    z-index: 100;
}

.rodal {
    position: fixed;
}

/* -- mask -- */
.rodal-mask {
    position: fixed;
    background: rgba(0, 0, 0, .5);
}

/* -- dialog -- */
.rodal-dialog {
    position: absolute;
    z-index: 101;
    background: #fff;
    border-radius: 3px;
    box-shadow: 0 1px 3px rgba(0, 0, 0, .2);
}

.rodal-center {
    top: 50%;
    transform: translateY(-50%);
    left: 0;
    right: 0;
    margin: 0 auto;
}

.rodal-bottom {
    left: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

.rodal-top {
    left: 0;
    right: 0;
    top: 0;
    margin: auto;
}

.rodal-left {
    top: 0;
    left: 0;
    bottom: 0;
    margin: auto;
}

.rodal-right {
    top: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

/* -- close button -- */
.rodal-close {
    position: absolute;
    cursor: pointer;
    top: 16px;
    right: 16px;
    width: 16px;
    height: 16px;
}

.rodal-close:before,
.rodal-close:after {
    position: absolute;
    content: '';
    height: 2px;
    width: 100%;
    top: 50%;
    left: 0;
    margin-top: -1px;
    background: #999;
    border-radius: 100%;
    -webkit-transition: background .2s;
    transition: background .2s;
}

.rodal-close:before {
    -webkit-transform: rotate(45deg);
    transform: rotate(45deg);
}

.rodal-close:after {
    -webkit-transform: rotate(-45deg);
    transform: rotate(-45deg);
}

.rodal-close:hover:before,
.rodal-close:hover:after {
    background: #333;
}

/* -- fade -- */
/* @-webkit-keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

@keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

.rodal-fade-enter {
    -webkit-animation: rodal-fade-enter both ease-in;
    animation: rodal-fade-enter both ease-in;
} */

@-webkit-keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

@keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

.rodal-fade-leave {
    -webkit-animation: rodal-fade-leave both ease-out;
    animation: rodal-fade-leave both ease-out;
}

/* -- zoom -- */
@-webkit-keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-enter {
    -webkit-animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-leave {
    -webkit-animation: rodal-zoom-leave both;
    animation: rodal-zoom-leave both;
}

/* -- slideDown -- */
@-webkit-keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-enter {
    -webkit-animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-leave {
    -webkit-animation: rodal-slideDown-leave both;
    animation: rodal-slideDown-leave both;
}

/* -- slideLeft -- */
@-webkit-keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-enter {
    -webkit-animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-leave {
    -webkit-animation: rodal-slideLeft-leave both;
    animation: rodal-slideLeft-leave both;
}

/* -- slideRight -- */
@-webkit-keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-enter {
    -webkit-animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-leave {
    -webkit-animation: rodal-slideRight-leave both;
    animation: rodal-slideRight-leave both;
}

/* -- slideUp -- */
@-webkit-keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-enter {
    -webkit-animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
    animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
}

@-webkit-keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-leave {
    -webkit-animation: rodal-slideUp-leave both;
    animation: rodal-slideUp-leave both;
}

/* -- flip -- */
@-webkit-keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

@keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

.rodal-flip-enter {
    -webkit-animation: rodal-flip-enter both ease-in;
    animation: rodal-flip-enter both ease-in;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

@-webkit-keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

@keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

.rodal-flip-leave {
    -webkit-animation: rodal-flip-leave both;
    animation: rodal-flip-leave both;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

/* -- rotate -- */
@-webkit-keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-enter {
    -webkit-animation: rodal-rotate-enter both;
    animation: rodal-rotate-enter both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

@-webkit-keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-leave {
    -webkit-animation: rodal-rotate-leave both;
    animation: rodal-rotate-leave both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

/* -- door -- */
@-webkit-keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

@keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

.rodal-door-enter {
    -webkit-animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

@keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

.rodal-door-leave {
    -webkit-animation: rodal-door-leave both;
    animation: rodal-door-leave both;
}</style><style>/* -- container -- */
.rodal,
.rodal-mask {
    top: 0;
    left: 0;
    width: 100%;
    height: 100%;
    z-index: 100;
}

.rodal {
    position: fixed;
}

/* -- mask -- */
.rodal-mask {
    position: fixed;
    background: rgba(0, 0, 0, .5);
}

/* -- dialog -- */
.rodal-dialog {
    position: absolute;
    z-index: 101;
    background: #fff;
    border-radius: 3px;
    box-shadow: 0 1px 3px rgba(0, 0, 0, .2);
}

.rodal-center {
    top: 50%;
    transform: translateY(-50%);
    left: 0;
    right: 0;
    margin: 0 auto;
}

.rodal-bottom {
    left: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

.rodal-top {
    left: 0;
    right: 0;
    top: 0;
    margin: auto;
}

.rodal-left {
    top: 0;
    left: 0;
    bottom: 0;
    margin: auto;
}

.rodal-right {
    top: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

/* -- close button -- */
.rodal-close {
    position: absolute;
    cursor: pointer;
    top: 16px;
    right: 16px;
    width: 16px;
    height: 16px;
}

.rodal-close:before,
.rodal-close:after {
    position: absolute;
    content: '';
    height: 2px;
    width: 100%;
    top: 50%;
    left: 0;
    margin-top: -1px;
    background: #999;
    border-radius: 100%;
    -webkit-transition: background .2s;
    transition: background .2s;
}

.rodal-close:before {
    -webkit-transform: rotate(45deg);
    transform: rotate(45deg);
}

.rodal-close:after {
    -webkit-transform: rotate(-45deg);
    transform: rotate(-45deg);
}

.rodal-close:hover:before,
.rodal-close:hover:after {
    background: #333;
}

/* -- fade -- */
/* @-webkit-keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

@keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

.rodal-fade-enter {
    -webkit-animation: rodal-fade-enter both ease-in;
    animation: rodal-fade-enter both ease-in;
} */

@-webkit-keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

@keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

.rodal-fade-leave {
    -webkit-animation: rodal-fade-leave both ease-out;
    animation: rodal-fade-leave both ease-out;
}

/* -- zoom -- */
@-webkit-keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-enter {
    -webkit-animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-leave {
    -webkit-animation: rodal-zoom-leave both;
    animation: rodal-zoom-leave both;
}

/* -- slideDown -- */
@-webkit-keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-enter {
    -webkit-animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-leave {
    -webkit-animation: rodal-slideDown-leave both;
    animation: rodal-slideDown-leave both;
}

/* -- slideLeft -- */
@-webkit-keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-enter {
    -webkit-animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-leave {
    -webkit-animation: rodal-slideLeft-leave both;
    animation: rodal-slideLeft-leave both;
}

/* -- slideRight -- */
@-webkit-keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-enter {
    -webkit-animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-leave {
    -webkit-animation: rodal-slideRight-leave both;
    animation: rodal-slideRight-leave both;
}

/* -- slideUp -- */
@-webkit-keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-enter {
    -webkit-animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
    animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
}

@-webkit-keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-leave {
    -webkit-animation: rodal-slideUp-leave both;
    animation: rodal-slideUp-leave both;
}

/* -- flip -- */
@-webkit-keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

@keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

.rodal-flip-enter {
    -webkit-animation: rodal-flip-enter both ease-in;
    animation: rodal-flip-enter both ease-in;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

@-webkit-keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

@keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

.rodal-flip-leave {
    -webkit-animation: rodal-flip-leave both;
    animation: rodal-flip-leave both;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

/* -- rotate -- */
@-webkit-keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-enter {
    -webkit-animation: rodal-rotate-enter both;
    animation: rodal-rotate-enter both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

@-webkit-keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-leave {
    -webkit-animation: rodal-rotate-leave both;
    animation: rodal-rotate-leave both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

/* -- door -- */
@-webkit-keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

@keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

.rodal-door-enter {
    -webkit-animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

@keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

.rodal-door-leave {
    -webkit-animation: rodal-door-leave both;
    animation: rodal-door-leave both;
}</style></head><body class=""><!-- BEGIN WAYBACK TOOLBAR INSERT -->
<style type="text/css">
body {
  margin-top:0 !important;
  padding-top:0 !important;
  /*min-width:800px !important;*/
}
</style>
<script>__wm.rw(0);</script>
<div id="wm-ipp-base" lang="en" style="display: block; direction: ltr;">
</div><div id="wm-ipp-print">The Wayback Machine - https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/d11094.htm</div>
<script type="text/javascript">//<![CDATA[
__wm.bt(675,27,25,2,"web","https://baike.sogou.com/kexue/d11094.htm","20221025130020",1996,"/_static/",["/_static/css/banner-styles.css?v=S1zqJCYt","/_static/css/iconochive.css?v=qtvMKcIJ"], false);
  __wm.rw(1);
//]]></script>
<!-- END WAYBACK TOOLBAR INSERT --><script>window._gtag=window._gtag||{};window._gtag.shouldGrayed = false;if ('7773fe33fdc045d89685a09fae70e004') window._gtag.traceId = '7773fe33fdc045d89685a09fae70e004';if ({"illegality":true}) window.userInfo = {"illegality":true};</script><div class="topnavbox"><ul class="topnav"><li><a href="https://web.archive.org/web/20221025130020/https://www.sogou.com/web?query=">网页</a></li><li><a href="https://web.archive.org/web/20221025130020/https://weixin.sogou.com/weixin?p=75351201">微信</a></li><li><a href="https://web.archive.org/web/20221025130020/https://zhihu.sogou.com/zhihu?p=75351218">知乎</a></li><li><a href="https://web.archive.org/web/20221025130020/https://pic.sogou.com/pics?query=">图片</a></li><li><a href="https://web.archive.org/web/20221025130020/https://v.sogou.com/v?query=">视频</a></li><li><a href="https://web.archive.org/web/20221025130020/https://mingyi.sogou.com/">医疗</a></li><li class="cur"><strong>科学</strong></li><li><a href="https://web.archive.org/web/20221025130020/https://hanyu.sogou.com/">汉语</a></li><li><a href="https://web.archive.org/web/20221025130020/https://wenwen.sogou.com/">问问</a></li><li><a href="https://web.archive.org/web/20221025130020/https://www.sogou.com/docs/more.htm">更多<span class="topraquo">»</span></a></li></ul></div><div id="header"><div class="header-wrap"><a class="header-logo" href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue"></a><div class="header-search"><div class="querybox" id="suggBox"><form><input id="searchInput" class="query" type="text" placeholder="搜科学领域专业百科词条" name="query" autocomplete="off" value=""><a href="javascript:;" class="query-search"></a></form></div></div><div class="header-rgt"><span class="btn-header-rgt btn-edit" id="editLemma">创建</span><div class="header-user no-login"></div></div></div></div><div class="fixed-placeholder" style="visibility:none"></div><div id="container" class=""><div class="content lemma-level1"><div class="detail-title" id="abstract-title"><h1>癌症中的DNA甲基化</h1><a href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/d11094.htm#!" class="detail-edit">编辑</a></div><div class="section_content" data-id="14996317944152328"><div><p>DNA甲基化在癌症中起着多种作用，它能够促使将基因表达调控的健康模式转化为疾病模式。 </p>
<p>所有来自同一受精卵的哺乳动物细胞都有一个共同的DNA序列(除了某些谱系的新突变)。然而，在不同组织的发育和形成过程中，表观遗传因素发生变化。这些变化包括组蛋白修饰、CpG岛甲基化和染色质重组，它们可以导致特定基因的稳定沉默或激活。一旦分化组织形成，CpG岛甲基化一般通过DNA甲基化维持机体从一个细胞分裂稳定地遗传到下一个细胞分裂。<sup><a href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/d11094.htm#quote_1" class="kx_ref">[1]</a></sup> </p>
<p>在癌症中，蛋白质编码基因上有大量的突变。结直肠癌通常有3-6个驱动突变以及33-66个搭便车突变（或乘客突变），这些突变促使那些受影响基因的蛋白质表达沉默。<sup><a href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/d11094.htm#quote_2" class="kx_ref">[2]</a></sup> 然而，在导致癌症的基因沉默方面，转录沉默可能比基因突变更加重要。<sup><a href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/d11094.htm#quote_3" class="kx_ref">[3]</a></sup> 在结直肠癌中，与相邻的正常组织相比，大约600-800个基因由于CpG岛甲基化被转录沉默。癌症中的转录抑制也可以通过其他表观遗传机制发生，如微小RNA（核糖核酸）的表达改变。<sup><a href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/d11094.htm#quote_4" class="kx_ref">[4]</a></sup> </p></div></div><div id="catalog"><h2 class="title2">目录<a href="javascript:" class="detail-edit">编辑</a></h2><div class="catalog_wrap" style=""><ul class="catalog_list col2"><li><span class="order">1</span><a href="javascript:" data-level="1" data-id="14996317944152329">CpG岛是常见的控制元件</a></li><li><span class="order">2</span><a href="javascript:" data-level="1" data-id="14996317944152330">启动子中CpG岛的甲基化稳定沉默基因</a></li><li><span class="order">3</span><a href="javascript:" data-level="1" data-id="14996317960929550">癌症中启动子CpG高/低甲基化</a></li><li class="secondary_catalog"><span>3.1 </span><a href="javascript:" data-id="14996317960929550">癌症中的DNA修复基因沉默</a></li><li class="secondary_catalog"><span>3.2 </span><a href="javascript:" data-id="14996317960929550">癌症中DNA修复基因的高甲基化频率</a></li></ul><ul class="catalog_list col2"><li><span class="order">4</span><a href="javascript:" data-level="1" data-id="14996317960929551">DNA修复基因高度甲基化在癌症中的可能作用</a></li><li><span class="order">5</span><a href="javascript:" data-level="1" data-id="14996317960929552">癌症中微小RNA的甲基化</a></li><li><span class="order">6</span><a href="javascript:" data-level="1" data-id="14996317977706766">癌症中控制DNA甲基转移酶基因的微小RNA</a></li><li><span class="order">7</span><a href="javascript:" data-level="1" data-id="references">参考文献</a></li></ul></div></div><div id="paragraphs"><div><div id="par_14996317944152329"><h2 class="title">1 CpG岛是常见的控制元件<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>CpG岛通常有200-2000个碱基对的长度，C:G碱基对含量&gt; 50%，并且频繁重复5' → 3' CpG序列。人类位于基因转录起始位点附近的启动子大约70%包含CpG岛。<sup><a href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/d11094.htm#quote_5" class="kx_ref">[5]</a></sup><sup><a href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/d11094.htm#quote_6" class="kx_ref">[6]</a></sup> </p>
<p>位于基因转录起始位点远处的启动子也经常包含CpG岛。例如，DNA修复基因ERCC1的启动子被定位于其编码区上游约5400个核苷酸处。<sup><a href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/d11094.htm#quote_7" class="kx_ref">[7]</a></sup> CpG岛也经常出现在功能性非编码RNA的启动子中，如微小RNA和长链非编码RNA。 </p></div></div><div id="par_14996317944152330"><h2 class="title">2 启动子中CpG岛的甲基化稳定沉默基因<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>由于启动子CpG岛中CpG位点的大量甲基化，基因会被沉默。<sup><a href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/d11094.htm#quote_8" class="kx_ref">[8]</a></sup> 即使一个基因的沉默是由另一种机制引起的，也通常会伴随着启动子CpG岛中CpG位点的甲基化，以促使该基因的稳定沉默。<sup><a href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/d11094.htm#quote_8" class="kx_ref">[8]</a></sup> 另一方面，启动子中CpG岛的低甲基化会导致基因过表达。 </p></div></div><div id="par_14996317960929550"><h2 class="title">3 癌症中启动子CpG高/低甲基化<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>在癌症中，启动子CpG岛的高甲基化导致的基因表达缺失的频率比突变导致的基因表达缺失高10倍。例如，在结肠肿瘤中（与相邻正常出现的结肠粘膜相比），肿瘤基因的启动子中出现约600-800个高度甲基化的CpG岛，而这些CpG岛在相邻粘膜中没有发生甲基化。<sup><a href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/d11094.htm#quote_9" class="kx_ref">[9]</a></sup><sup><a href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/d11094.htm#quote_10" class="kx_ref">[10]</a></sup><sup><a href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/d11094.htm#quote_11" class="kx_ref">[11]</a></sup> 相比之下，正如沃格尔斯坦等（Vogelstein et al.）人指出的那样，<sup><a href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/d11094.htm#quote_2" class="kx_ref">[2]</a></sup> 结直肠癌通常只有大约3-6个驱动突变和33-66个搭便车突变（或乘客突变）。 </p> 
<h3>3.1 <span>癌症中的DNA修复基因沉默</span></h3> 
<p>在散发性癌症中，偶尔会发现由于DNA修复基因的突变而导致DNA修复缺陷。然而，更常见的是，癌症中DNA修复基因表达的减少或缺失是由于其启动子的甲基化。例如，在被检查的113例结直肠癌中，只有4例在修复基因MGMT中有错义突变，而大多数是由于MGMT启动子区甲基化而降低了MGMT表达。<sup><a href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/d11094.htm#quote_12" class="kx_ref">[12]</a></sup> 同样，在缺乏修复基因PMS2表达的119例错配修复缺陷型结直肠癌中，6例PMS2基因发生突变，而103例是因为PMS2的配对基因MLH1启动子甲基化进而导致PMS2表达受到抑制(缺少MLH1时PMS2蛋白不稳定)。<sup><a href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/d11094.htm#quote_13" class="kx_ref">[13]</a></sup> 在剩余10例病例中，PMS2表达的缺失可能是由于微小RNA(miR-155，能够下调MLH1)的表观遗传过表达所致。<sup><a href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/d11094.htm#quote_14" class="kx_ref">[14]</a></sup> </p> 
<h3>3.2 <span>癌症中DNA修复基因的高甲基化频率</span></h3> 
<p>在两篇综述文章中所列出的17种癌症中，发现有22 种DNA修复基因，具有高甲基化启动子，并且表达减少或缺失。<sup><a href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/d11094.htm#quote_15" class="kx_ref">[15]</a></sup><sup><a href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/d11094.htm#quote_16" class="kx_ref">[16]</a></sup> 其中一篇综述中提到，<sup><a href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/d11094.htm#quote_15" class="kx_ref">[15]</a></sup> MGMT启动子高度甲基化经常发生在许多癌症中，包括93%的膀胱癌、88%的胃癌、74%的甲状腺癌、40%-90%的结直肠癌和50%的脑癌。该综述还表明，在头颈癌、非小细胞肺癌或非小细胞肺癌鳞状细胞癌等一种或多种癌中，LIG4、NEIL1、ATM、MLH1或FANCB的启动子高甲基化发生的频率在33%-82%之间。关于沃纳综合征三磷酸腺苷（ATP）依赖性解旋酶的研究文献指出，DNA修复基因WRN有一个启动子，该启动子在许多癌症中经常高甲基化，大约有11%-38%的结直肠癌、头颈癌、胃癌、前列腺癌、乳腺癌、甲状腺癌、非霍奇金淋巴瘤、软骨肉瘤和骨肉瘤中有高甲基化的发生 。 </p></div></div><div id="par_14996317960929551"><h2 class="title">4 DNA修复基因高度甲基化在癌症中的可能作用<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>正如金和罗伯森（Jin and Roberston）在他们的综述中所讨论的那样，<sup><a href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/d11094.htm#quote_16" class="kx_ref">[16]</a></sup> 通过高甲基化沉默一个DNA修复基因可能是癌变的一个非常早期的步骤。这种沉默被认为与DNA修复基因的种系突变作用相似，并使细胞及其后代易于发展成癌症。另一篇综述中<sup><a href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/d11094.htm#quote_17" class="kx_ref">[17]</a></sup> 也指出了癌症中DNA修复基因高度甲基化的早期作用。如果一个DNA修复需要的基因高甲基化—导致修复基因的缺陷——那么基因损伤就会累积。增加的DNA损伤会导致在DNA合成过程中引起更多的错误，导致出现一些会引发癌症的突变。 </p>
<p>如果一个DNA修复基因的高甲基化是癌变的早期步骤，那么它也可能发生在肿瘤周围的正常出现的组织中(区域缺陷)。见下表： </p> 
<table class="wikitable sortable"> 
 <caption>
  散发性癌症和邻近区域缺陷中DNA修复基因中高甲基化启动子的频率 
 </caption> 
 <tbody>
  <tr> 
   <th>癌症</th> 
   <th>基因</th> 
   <th>癌症中的频率</th> 
   <th>区域缺陷频率</th> 
   <th>Ref. </th>
  </tr> 
  <tr> 
   <th>结直肠癌 </th> 
   <td>MGMT</td> 
   <td>55%</td> 
   <td>54%</td> 
   <td><sup><a href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/d11094.htm#quote_18" class="kx_ref">[18]</a></sup> </td>
  </tr> 
  <tr> 
   <th>结直肠癌 </th> 
   <td>MSH2</td> 
   <td>13%</td> 
   <td>5%</td> 
   <td><sup><a href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/d11094.htm#quote_19" class="kx_ref">[19]</a></sup> </td>
  </tr> 
  <tr> 
   <th>结直肠癌 </th> 
   <td>WRN</td> 
   <td>29%</td> 
   <td>13%</td> 
   <td><sup><a href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/d11094.htm#quote_20" class="kx_ref">[20]</a></sup> </td>
  </tr> 
  <tr> 
   <th>头颈癌 </th> 
   <td>MGMT</td> 
   <td>54%</td> 
   <td>38%</td> 
   <td><sup><a href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/d11094.htm#quote_21" class="kx_ref">[21]</a></sup> </td>
  </tr> 
  <tr> 
   <th>头颈癌 </th> 
   <td>MLH1</td> 
   <td>33%</td> 
   <td>25%</td> 
   <td><sup><a href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/d11094.htm#quote_22" class="kx_ref">[22]</a></sup> </td>
  </tr> 
  <tr> 
   <th>非小细胞肺癌 </th> 
   <td>ATM</td> 
   <td>69%</td> 
   <td>59%</td> 
   <td><sup><a href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/d11094.htm#quote_23" class="kx_ref">[23]</a></sup> </td>
  </tr> 
  <tr> 
   <th>非小细胞肺癌 </th> 
   <td>MLH1</td> 
   <td>69%</td> 
   <td>72%</td> 
   <td><sup><a href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/d11094.htm#quote_23" class="kx_ref">[23]</a></sup> </td>
  </tr> 
  <tr> 
   <th>胃癌 </th> 
   <td>MGMT</td> 
   <td>88%</td> 
   <td>78%</td> 
   <td><sup><a href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/d11094.htm#quote_24" class="kx_ref">[24]</a></sup> </td>
  </tr> 
  <tr> 
   <th>胃癌 </th> 
   <td>MLH1</td> 
   <td>73%</td> 
   <td>20%</td> 
   <td><sup><a href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/d11094.htm#quote_25" class="kx_ref">[25]</a></sup> </td>
  </tr> 
  <tr> 
   <th>食管癌 </th> 
   <td>MLH1</td> 
   <td>77%-100%</td> 
   <td>23%-79%</td> 
   <td><sup><a href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/d11094.htm#quote_26" class="kx_ref">[26]</a></sup> </td>
  </tr>
 </tbody>
</table> 
<p>DNA损伤可能会容易导致跨损伤合成的错误进而引起突变，同时DNA损伤也可能在错误的DNA修复过程中引起表观遗传改变。<sup><a href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/d11094.htm#quote_27" class="kx_ref">[27]</a></sup><sup><a href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/d11094.htm#quote_28" class="kx_ref">[28]</a></sup><sup><a href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/d11094.htm#quote_29" class="kx_ref">[29]</a></sup><sup><a href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/d11094.htm#quote_30" class="kx_ref">[30]</a></sup> 由于DNA修复基因启动子的高甲基化而积累的DNA损伤可能是癌症中许多基因表观遗传改变增加的原因之一。 </p>
<p>在一项早期的研究中，费尔南德斯等（Fernandez et al）人着眼于一组有限的转录启动子，<sup><a href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/d11094.htm#quote_31" class="kx_ref">[31]</a></sup> 研究了855例原发性肿瘤的DNA甲基化谱。将每种肿瘤类型与其相应的正常组织进行比较，729个CpG岛位点(评估的1322个CpG岛位点的55%)显示不同的DNA甲基化。在这些位点中，496个位点高甲基化（被抑制)，233个位点低甲基化(被激活)。因此，在肿瘤中存在高水平的启动子甲基化改变，其中一些改变可能会导致癌变。 </p></div></div><div id="par_14996317960929552"><h2 class="title">5 癌症中微小RNA的甲基化<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>在哺乳动物中，微小RNA(miRNAs)调控大约60%蛋白质编码基因的转录活性。<sup><a href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/d11094.htm#quote_32" class="kx_ref">[32]</a></sup> 每个独立的微小RNA平均可以靶向抑制大约200个蛋白质编码基因的信使RNA的转录。<sup><a href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/d11094.htm#quote_33" class="kx_ref">[33]</a></sup> 在 Vrba等人评估的乳腺组织的167个微小RNA中，约三分之一的启动子在乳腺癌和正常乳腺组织中存在高或低甲基化差异。<sup><a href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/d11094.htm#quote_34" class="kx_ref">[34]</a></sup> 最近的一项研究指出，Vrba等人评估的167个微小 RNA仅占乳腺组织中已发现表达的微小RNA的10%。<sup><a href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/d11094.htm#quote_35" class="kx_ref">[35]</a></sup> 这项新的研究发现，乳腺组织中58%的微小RNA在其乳腺癌启动子中具有不同的甲基化区域，包括278个高甲基化的微小RNA和802个低甲基化的微小RNA。 </p>
<p>miR-182是乳腺癌中过表达约100倍的微小RNA之一。<sup><a href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/d11094.htm#quote_36" class="kx_ref">[36]</a></sup> MiR-182靶向调控BRCA1信使RNA，因此，miR-182过表达可能是许多乳腺癌中BRCA1蛋白表达减少的主要原因。<sup><a href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/d11094.htm#quote_37" class="kx_ref">[37]</a></sup> </p></div></div><div id="par_14996317977706766"><h2 class="title">6 癌症中控制DNA甲基转移酶基因的微小RNA<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>一些微小RNA靶向调控DNA甲基转移酶基因<i>DNMT1</i>, <i>DNMT3A</i> 和 <i>DNMT3B</i>的信使RNA，它们的基因产物是启动和稳定启动子甲基化所必需的。正如以上三篇文献中所总结的那样，<sup><a href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/d11094.htm#quote_38" class="kx_ref">[38]</a></sup><sup><a href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/d11094.htm#quote_39" class="kx_ref">[39]</a></sup><sup><a href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/d11094.htm#quote_40" class="kx_ref">[40]</a></sup> 微小RNA miR-29a、miR-29b和miR-29c靶向定位DNMT3A和DNMT3B；miR-148a和miR-148b靶向定位DNMT3B；miR-152和miR-301靶向定位DNMT1。此外，miR-34b靶向定位DNMT1，并且miR-34b自身的启动子在大多数前列腺癌中高甲基化和低表达。<sup><a href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/d11094.htm#quote_41" class="kx_ref">[41]</a></sup> 当这些微小RNA的表达改变时，它们也可能就成为癌症中蛋白质编码基因启动子高/低甲基化的诱因。 </p></div></div></div></div><div id="references"><h2 class="title" id="par_references">参考文献</h2><ul class="references"><li id="quote_1"><span class="references-num">[1]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Seisenberger S, Peat JR, Hore TA, Santos F, Dean W, Reik W (2013). "Reprogramming DNA methylation in the mammalian life cycle: building and breaking epigenetic barriers". Philos. Trans. R. Soc. Lond. B Biol. Sci. 368 (1609): 20110330. doi:10.1098/rstb.2011.0330. PMC 3539359. PMID 23166394..</span></p></li><li id="quote_2"><span class="references-num">[2]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Vogelstein B, Papadopoulos N, Velculescu VE, Zhou S, Diaz LA, Kinzler KW (2013). "Cancer genome landscapes". Science. 339 (6127): 1546–58. doi:10.1126/science.1235122. PMC 3749880. PMID 23539594..</span></p></li><li id="quote_3"><span class="references-num">[3]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Wang YP, Lei QY (2018). "Metabolic recoding of epigenetics in cancer". Cancer Commun (Lond). 38 (1): 25. doi:10.1186/s40880-018-0302-3. PMC 5993135. PMID 29784032..</span></p></li><li id="quote_4"><span class="references-num">[4]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Tessitore A, Cicciarelli G, Del Vecchio F, Gaggiano A, Verzella D, Fischietti M, Vecchiotti D, Capece D, Zazzeroni F, Alesse E (2014). "MicroRNAs in the DNA Damage/Repair Network and Cancer". Int J Genom. 2014: 1–10. doi:10.1155/2014/820248. PMC 3926391. PMID 24616890..</span></p></li><li id="quote_5"><span class="references-num">[5]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Saxonov S, Berg P, Brutlag DL (2006). "A genome-wide analysis of CpG dinucleotides in the human genome distinguishes two distinct classes of promoters". Proc. Natl. Acad. Sci. U.S.A. 103 (5): 1412–7. doi:10.1073/pnas.0510310103. PMC 1345710. PMID 16432200..</span></p></li><li id="quote_6"><span class="references-num">[6]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Deaton AM, Bird A (2011). "CpG islands and the regulation of transcription". Genes Dev. 25 (10): 1010–22. doi:10.1101/gad.2037511. PMC 3093116. PMID 21576262..</span></p></li><li id="quote_7"><span class="references-num">[7]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Chen HY, Shao CJ, Chen FR, Kwan AL, Chen ZP (2010). "Role of ERCC1 promoter hypermethylation in drug resistance to cisplatin in human gliomas". Int. J. Cancer. 126 (8): 1944–54. doi:10.1002/ijc.24772. PMID 19626585..</span></p></li><li id="quote_8"><span class="references-num">[8]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Bird A (2002). "DNA methylation patterns and epigenetic memory". Genes Dev. 16 (1): 6–21. doi:10.1101/gad.947102. PMID 11782440..</span></p></li><li id="quote_9"><span class="references-num">[9]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Illingworth RS, Gruenewald-Schneider U, Webb S, Kerr AR, James KD, Turner DJ, Smith C, Harrison DJ, Andrews R, Bird AP (2010). "Orphan CpG islands identify numerous conserved promoters in the mammalian genome". PLoS Genet. 6 (9): e1001134. doi:10.1371/journal.pgen.1001134. PMC 2944787. PMID 20885785..</span></p></li><li id="quote_10"><span class="references-num">[10]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Wei J, Li G, Dang S, Zhou Y, Zeng K, Liu M (2016). "Discovery and Validation of Hypermethylated Markers for Colorectal Cancer". Dis. Markers. 2016: 1–7. doi:10.1155/2016/2192853. PMC 4963574. PMID 27493446..</span></p></li><li id="quote_11"><span class="references-num">[11]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Beggs AD, Jones A, El-Bahrawy M, El-Bahwary M, Abulafi M, Hodgson SV, Tomlinson IP (2013). "Whole-genome methylation analysis of benign and malignant colorectal tumours". J. Pathol. 229 (5): 697–704. doi:10.1002/path.4132. PMC 3619233. PMID 23096130..</span></p></li><li id="quote_12"><span class="references-num">[12]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Halford S, Rowan A, Sawyer E, Talbot I, Tomlinson I (June 2005). "O(6)-methylguanine methyltransferase in colorectal cancers: detection of mutations, loss of expression, and weak association with G:C&gt;A:T transitions". Gut. 54 (6): 797–802. doi:10.1136/gut.2004.059535. PMC 1774551. PMID 15888787..</span></p></li><li id="quote_13"><span class="references-num">[13]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Truninger K, Menigatti M, Luz J, et al. (May 2005). "Immunohistochemical analysis reveals high frequency of PMS2 defects in colorectal cancer". Gastroenterology. 128 (5): 1160–71. doi:10.1053/j.gastro.2005.01.056. PMID 15887099..</span></p></li><li id="quote_14"><span class="references-num">[14]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Valeri N, Gasparini P, Fabbri M, et al. (April 2010). "Modulation of mismatch repair and genomic stability by miR-155". Proceedings of the National Academy of Sciences of the United States of America. 107 (15): 6982–7. Bibcode:2010PNAS..107.6982V. doi:10.1073/pnas.1002472107. JSTOR 25665289. PMC 2872463. PMID 20351277..</span></p></li><li id="quote_15"><span class="references-num">[15]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Carol Bernstein and Harris Bernstein (2015). Epigenetic Reduction of DNA Repair in Progression to Cancer, Advances in DNA Repair, Prof. Clark Chen (Ed.), ISBN 978-953-51-2209-8, InTech, Available from: http://www.intechopen.com/books/advances-in-dna-repair/epigenetic-reduction-of-dna-repair-in-progression-to-cancer.</span></p></li><li id="quote_16"><span class="references-num">[16]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Jin B, Robertson KD (2013). DNA methyltransferases, DNA damage repair, and cancer. Adv. Exp. Med. Biol. Advances in Experimental Medicine and Biology. 754. pp. 3–29. doi:10.1007/978-1-4419-9967-2_1. ISBN 978-1-4419-9966-5. PMC 3707278. PMID 22956494..</span></p></li><li id="quote_17"><span class="references-num">[17]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Bernstein C, Nfonsam V, Prasad AR, Bernstein H (2013). "Epigenetic field defects in progression to cancer". World J Gastrointest Oncol. 5 (3): 43–9. doi:10.4251/wjgo.v5.i3.43. PMC 3648662. PMID 23671730..</span></p></li><li id="quote_18"><span class="references-num">[18]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Svrcek M, Buhard O, Colas C, Coulet F, Dumont S, Massaoudi I, Lamri A, Hamelin R, Cosnes J, Oliveira C, Seruca R, Gaub MP, Legrain M, Collura A, Lascols O, Tiret E, Fléjou JF, Duval A (November 2010). "Methylation tolerance due to an O6-methylguanine DNA methyltransferase (MGMT) field defect in the colonic mucosa: an initiating step in the development of mismatch repair-deficient colorectal cancers". Gut. 59 (11): 1516–26. doi:10.1136/gut.2009.194787. PMID 20947886..</span></p></li><li id="quote_19"><span class="references-num">[19]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Lee KH, Lee JS, Nam JH, Choi C, Lee MC, Park CS, Juhng SW, Lee JH (2011). "Promoter methylation status of hMLH1, hMSH2, and MGMT genes in colorectal cancer associated with adenoma-carcinoma sequence". Langenbecks Arch Surg. 396 (7): 1017–26. doi:10.1007/s00423-011-0812-9. PMID 21706233..</span></p></li><li id="quote_20"><span class="references-num">[20]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Kawasaki T, Ohnishi M, Suemoto Y, Kirkner GJ, Liu Z, Yamamoto H, Loda M, Fuchs CS, Ogino S (2008). "WRN promoter methylation possibly connects mucinous differentiation, microsatellite instability and CpG island methylator phenotype in colorectal cancer". Mod. Pathol. 21 (2): 150–8. doi:10.1038/modpathol.3800996. PMID 18084250..</span></p></li><li id="quote_21"><span class="references-num">[21]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Paluszczak J, Misiak P, Wierzbicka M, Woźniak A, Baer-Dubowska W (February 2011). "Frequent hypermethylation of DAPK, RARbeta, MGMT, RASSF1A and FHIT in laryngeal squamous cell carcinomas and adjacent normal mucosa". Oral Oncol. 47 (2): 104–7. doi:10.1016/j.oraloncology.2010.11.006. PMID 21147548..</span></p></li><li id="quote_22"><span class="references-num">[22]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Zuo C, Zhang H, Spencer HJ, Vural E, Suen JY, Schichman SA, Smoller BR, Kokoska MS, Fan CY (October 2009). "Increased microsatellite instability and epigenetic inactivation of the hMLH1 gene in head and neck squamous cell carcinoma". Otolaryngol Head Neck Surg. 141 (4): 484–90. doi:10.1016/j.otohns.2009.07.007. PMID 19786217..</span></p></li><li id="quote_23"><span class="references-num">[23]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Safar AM, Spencer H, Su X, Coffey M, Cooney CA, Ratnasinghe LD, Hutchins LF, Fan CY (2005). "Methylation profiling of archived non-small cell lung cancer: a promising prognostic system". Clin. Cancer Res. 11 (12): 4400–5. doi:10.1158/1078-0432.CCR-04-2378. PMID 15958624..</span></p></li><li id="quote_24"><span class="references-num">[24]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Zou XP, Zhang B, Zhang XQ, Chen M, Cao J, Liu WJ (November 2009). "Promoter hypermethylation of multiple genes in early gastric adenocarcinoma and precancerous lesions". Hum. Pathol. 40 (11): 1534–42. doi:10.1016/j.humpath.2009.01.029. PMID 19695681..</span></p></li><li id="quote_25"><span class="references-num">[25]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Wani M, Afroze D, Makhdoomi M, Hamid I, Wani B, Bhat G, Wani R, Wani K (2012). "Promoter methylation status of DNA repair gene (hMLH1) in gastric carcinoma patients of the Kashmir valley". Asian Pac. J. Cancer Prev. 13 (8): 4177–81. doi:10.7314/APJCP.2012.13.8.4177. PMID 23098428..</span></p></li><li id="quote_26"><span class="references-num">[26]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Agarwal A, Polineni R, Hussein Z, Vigoda I, Bhagat TD, Bhattacharyya S, Maitra A, Verma A (2012). "Role of epigenetic alterations in the pathogenesis of Barrett's esophagus and esophageal adenocarcinoma". Int J Clin Exp Pathol. 5 (5): 382–96. PMC 3396065. PMID 22808291..</span></p></li><li id="quote_27"><span class="references-num">[27]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">O'Hagan HM, Mohammad HP, Baylin SB (2008). "Double strand breaks can initiate gene silencing and SIRT1-dependent onset of DNA methylation in an exogenous promoter CpG island". PLoS Genetics. 4 (8): e1000155. doi:10.1371/journal.pgen.1000155. PMC 2491723. PMID 18704159..</span></p></li><li id="quote_28"><span class="references-num">[28]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Cuozzo C, Porcellini A, Angrisano T, et al. (July 2007). "DNA damage, homology-directed repair, and DNA methylation". PLoS Genetics. 3 (7): e110. doi:10.1371/journal.pgen.0030110. PMC 1913100. PMID 17616978..</span></p></li><li id="quote_29"><span class="references-num">[29]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Shanbhag NM, Rafalska-Metcalf IU, Balane-Bolivar C, Janicki SM, Greenberg RA (June 2010). "ATM-dependent chromatin changes silence transcription in cis to DNA double-strand breaks". Cell. 141 (6): 970–81. doi:10.1016/j.cell.2010.04.038. PMC 2920610. PMID 20550933..</span></p></li><li id="quote_30"><span class="references-num">[30]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Morano A, Angrisano T, Russo G, Landi R, Pezone A, Bartollino S, Zuchegna C, Babbio F, Bonapace IM, Allen B, Muller MT, Chiariotti L, Gottesman ME, Porcellini A, Avvedimento EV (January 2014). "Targeted DNA methylation by homology-directed repair in mammalian cells. Transcription reshapes methylation on the repaired gene". Nucleic Acids Res. 42 (2): 804–21. doi:10.1093/nar/gkt920. PMC 3902918. PMID 24137009..</span></p></li><li id="quote_31"><span class="references-num">[31]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Fernandez AF, Assenov Y, Martin-Subero JI, Balint B, Siebert R, Taniguchi H, Yamamoto H, Hidalgo M, Tan AC, Galm O, Ferrer I, Sanchez-Cespedes M, Villanueva A, Carmona J, Sanchez-Mut JV, Berdasco M, Moreno V, Capella G, Monk D, Ballestar E, Ropero S, Martinez R, Sanchez-Carbayo M, Prosper F, Agirre X, Fraga MF, Graña O, Perez-Jurado L, Mora J, Puig S, Prat J, Badimon L, Puca AA, Meltzer SJ, Lengauer T, Bridgewater J, Bock C, Esteller M (2012). "A DNA methylation fingerprint of 1628 human samples". Genome Res. 22 (2): 407–19. doi:10.1101/gr.119867.110. PMC 3266047. PMID 21613409..</span></p></li><li id="quote_32"><span class="references-num">[32]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Friedman, RC; Farh, KK; Burge, CB; Bartel, DP (January 2009). "Most mammalian mRNAs are conserved targets of microRNAs". Genome Res. 19 (1): 92–105. doi:10.1101/gr.082701.108. PMC 2612969. PMID 18955434..</span></p></li><li id="quote_33"><span class="references-num">[33]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Krek A, Grün D, Poy MN, Wolf R, Rosenberg L, Epstein EJ, MacMenamin P, da Piedade I, Gunsalus KC, Stoffel M, Rajewsky N (2005). "Combinatorial microRNA target predictions". Nat. Genet. 37 (5): 495–500. doi:10.1038/ng1536. PMID 15806104..</span></p></li><li id="quote_34"><span class="references-num">[34]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Vrba L, Muñoz-Rodríguez JL, Stampfer MR, Futscher BW (2013). "miRNA gene promoters are frequent targets of aberrant DNA methylation in human breast cancer". PLoS ONE. 8 (1): e54398. doi:10.1371/journal.pone.0054398. PMC 3547033. PMID 23342147..</span></p></li><li id="quote_35"><span class="references-num">[35]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Li Y, Zhang Y, Li S, Lu J, Chen J, Wang Y, Li Y, Xu J, Li X (2015). "Genome-wide DNA methylome analysis reveals epigenetically dysregulated non-coding RNAs in human breast cancer". Sci Rep. 5: 8790. doi:10.1038/srep08790. PMC 4350105. PMID 25739977..</span></p></li><li id="quote_36"><span class="references-num">[36]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Krishnan K, Steptoe AL, Martin HC, Wani S, Nones K, Waddell N, Mariasegaram M, Simpson PT, Lakhani SR, Gabrielli B, Vlassov A, Cloonan N, Grimmond SM (2013). "MicroRNA-182-5p targets a network of genes involved in DNA repair". RNA. 19 (2): 230–42. doi:10.1261/rna.034926.112. PMC 3543090. PMID 23249749..</span></p></li><li id="quote_37"><span class="references-num">[37]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Moskwa P, Buffa FM, Pan Y, Panchakshari R, Gottipati P, Muschel RJ, Beech J, Kulshrestha R, Abdelmohsen K, Weinstock DM, Gorospe M, Harris AL, Helleday T, Chowdhury D (2011). "miR-182-mediated downregulation of BRCA1 impacts DNA repair and sensitivity to PARP inhibitors". Mol. Cell. 41 (2): 210–20. doi:10.1016/j.molcel.2010.12.005. PMC 3249932. PMID 21195000..</span></p></li><li id="quote_38"><span class="references-num">[38]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Suzuki H, Maruyama R, Yamamoto E, Kai M (2012). "DNA methylation and microRNA dysregulation in cancer". Mol Oncol. 6 (6): 567–78. doi:10.1016/j.molonc.2012.07.007. PMC 5528344. PMID 22902148..</span></p></li><li id="quote_39"><span class="references-num">[39]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Suzuki H, Maruyama R, Yamamoto E, Kai M (2013). "Epigenetic alteration and microRNA dysregulation in cancer". Front Genet. 4: 258. doi:10.3389/fgene.2013.00258. PMC 3847369. PMID 24348513..</span></p></li><li id="quote_40"><span class="references-num">[40]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Kaur S, Lotsari-Salomaa JE, Seppänen-Kaijansinkko R, Peltomäki P (2016). MicroRNA Methylation in Colorectal Cancer. Adv. Exp. Med. Biol. Advances in Experimental Medicine and Biology. 937. pp. 109–22. doi:10.1007/978-3-319-42059-2_6. ISBN 978-3-319-42057-8. PMID 27573897..</span></p></li><li id="quote_41"><span class="references-num">[41]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Majid S, Dar AA, Saini S, Shahryari V, Arora S, Zaman MS, Chang I, Yamamura S, Tanaka Y, Chiyomaru T, Deng G, Dahiya R (2013). "miRNA-34b inhibits prostate cancer through demethylation, active chromatin modifications, and AKT pathways". Clin. Cancer Res. 19 (1): 73–84. doi:10.1158/1078-0432.CCR-12-2952. PMC 3910324. PMID 23147995..</span></p></li></ul></div><div class="read-num">阅读 <!-- -->143</div></div><div class="right-side" id="rightSide"><div class="side" id="lemma-side"><div class="side-title">版本记录</div><ul class="side-lst"><li><p class="side-lst-txt">暂无</p></li></ul><div class="user-card userCard"></div></div><div class="side"><div class="side-event"></div></div></div></div><div class="footer-box"><div id="footer"><div class="footer-logo-wrap"><div class="footer-logo"></div><div class="footer-logo-text">知识·传播·科普</div></div><div class="footer-info">本网站内容采用<a target="_blank" href="https://web.archive.org/web/20221025130020/https://creativecommons.org/licenses/by-sa/3.0/deed.zh?tdsourcetag=s_pctim_aiomsg">CC-BY-SA 3.0</a>授权</div><div class="footer-btn-wrap"><a target="_blank" href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/help/#user_protocol">用户协议</a><a target="_blank" href="https://web.archive.org/web/20221025130020/http://www.sogou.com/docs/terms.htm?v=1">免责声明</a><a target="_blank" href="https://web.archive.org/web/20221025130020/http://corp.sogou.com/private.html">隐私政策</a><a target="_blank" href="https://web.archive.org/web/20221025130020/https://baike.sogou.com/kexue/intro.htm">关于我们</a></div></div></div><script>window.lemmaInfo ={"lemmaId":"11094","versionId":"14996317944152320","title":"癌症中的DNA甲基化","subtitle":"","abstracts":{"paragraphId":"14996317944152328","title":"简介","versionId":"14996317944152321","lemmaId":11094,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":10145103,"name":"科学百科编辑组","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576234501,"comment":null,"dependVersionId":0,"contentType":2,"content":"<p>DNA甲基化在癌症中起着多种作用，它能够促使将基因表达调控的健康模式转化为疾病模式。 </p>\n<p>所有来自同一受精卵的哺乳动物细胞都有一个共同的DNA序列(除了某些谱系的新突变)。然而，在不同组织的发育和形成过程中，表观遗传因素发生变化。这些变化包括组蛋白修饰、CpG岛甲基化和染色质重组，它们可以导致特定基因的稳定沉默或激活。一旦分化组织形成，CpG岛甲基化一般通过DNA甲基化维持机体从一个细胞分裂稳定地遗传到下一个细胞分裂。<sup><a href=\"#quote_1\" class=\"kx_ref\">[1]</a></sup> </p>\n<p>在癌症中，蛋白质编码基因上有大量的突变。结直肠癌通常有3-6个驱动突变以及33-66个搭便车突变（或乘客突变），这些突变促使那些受影响基因的蛋白质表达沉默。<sup><a href=\"#quote_2\" class=\"kx_ref\">[2]</a></sup> 然而，在导致癌症的基因沉默方面，转录沉默可能比基因突变更加重要。<sup><a href=\"#quote_3\" class=\"kx_ref\">[3]</a></sup> 在结直肠癌中，与相邻的正常组织相比，大约600-800个基因由于CpG岛甲基化被转录沉默。癌症中的转录抑制也可以通过其他表观遗传机制发生，如微小RNA（核糖核酸）的表达改变。<sup><a href=\"#quote_4\" class=\"kx_ref\">[4]</a></sup> </p>","pics":null,"card":null,"references":[],"versionCount":0},"card":{"paragraphId":"0","title":null,"versionId":"0","lemmaId":0,"createType":0,"creator":null,"createTime":0,"versionEditor":null,"editTime":0,"comment":null,"dependVersionId":0,"contentType":0,"content":null,"pics":null,"card":null,"references":null,"versionCount":0},"categories":[{"id":13,"name":"药学","parents":[]}],"creator":{"uid":10145103,"name":"柚子otto","pic":"https://web.archive.org/web/20221025130020/https://img02.sogoucdn.com/app/a/200698/1152_1152_1864088_20200427232849-1518843971.png","introduction":"","educations":[{"schoolName":"中国地质大学（北京）","major":"","degree":"本科","universityId":22,"universityLogo":"https://web.archive.org/web/20221025130020/https://img01.sogoucdn.com/app/a/200943/d3465c1c-6011-11e9-b353-fc4dd4f70029","majorLevel1":"理学","majorLevel2":"地理学","majorLevel3":"地图学与地理信息系统","majorLevel1Id":1,"majorLevel2Id":103,"majorLevel3Id":109,"state":"毕业","lab":"","researchField":""}],"jobs":[{"company":"搜狗","title":"产品经理"}],"works":null,"educationBrief":"中国地质大学（北京）","jobBrief":"产品经理","role":0,"roleName":null,"title":"中国地质大学（北京） · 地理学本科","professionalTitle":null,"phoneNo":null,"editable":true,"partnerId":139,"partnerIdCreateTime":1595844881,"partnerIdPoped":true},"createTime":1571038563,"editor":{"uid":10145103,"name":"柚子otto","pic":"https://web.archive.org/web/20221025130020/https://img02.sogoucdn.com/app/a/200698/1152_1152_1864088_20200427232849-1518843971.png","introduction":"","educations":[{"schoolName":"中国地质大学（北京）","major":"","degree":"本科","universityId":22,"universityLogo":"https://web.archive.org/web/20221025130020/https://img01.sogoucdn.com/app/a/200943/d3465c1c-6011-11e9-b353-fc4dd4f70029","majorLevel1":"理学","majorLevel2":"地理学","majorLevel3":"地图学与地理信息系统","majorLevel1Id":1,"majorLevel2Id":103,"majorLevel3Id":109,"state":"毕业","lab":"","researchField":""}],"jobs":[{"company":"搜狗","title":"产品经理"}],"works":null,"educationBrief":"中国地质大学（北京）","jobBrief":"产品经理","role":0,"roleName":null,"title":"中国地质大学（北京） · 地理学本科","professionalTitle":null,"phoneNo":null,"editable":true,"partnerId":139,"partnerIdCreateTime":1595844881,"partnerIdPoped":true},"editTime":1576234501,"state":1,"versionCount":1,"upNum":0,"downNum":0,"pics":[],"catalogs":[{"level":1,"title":"CpG岛是常见的控制元件","paragraphId":"14996317944152329","subCatalogs":null},{"level":1,"title":"启动子中CpG岛的甲基化稳定沉默基因","paragraphId":"14996317944152330","subCatalogs":null},{"level":1,"title":"癌症中启动子CpG高/低甲基化","paragraphId":"14996317960929550","subCatalogs":[{"level":2,"title":"癌症中的DNA修复基因沉默","paragraphId":"14996317960929550","subCatalogs":null},{"level":2,"title":"癌症中DNA修复基因的高甲基化频率","paragraphId":"14996317960929550","subCatalogs":null}]},{"level":1,"title":"DNA修复基因高度甲基化在癌症中的可能作用","paragraphId":"14996317960929551","subCatalogs":null},{"level":1,"title":"癌症中微小RNA的甲基化","paragraphId":"14996317960929552","subCatalogs":null},{"level":1,"title":"癌症中控制DNA甲基转移酶基因的微小RNA","paragraphId":"14996317977706766","subCatalogs":null},{"level":1,"title":"参考文献","paragraphId":"-1","subCatalogs":null}],"paragraphs":[{"paragraphId":"14996317944152329","title":"CpG岛是常见的控制元件","versionId":"14996317944152322","lemmaId":11094,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":10145103,"name":"科学百科编辑组","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576234501,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>CpG岛通常有200-2000个碱基对的长度，C:G碱基对含量&gt; 50%，并且频繁重复5' → 3' CpG序列。人类位于基因转录起始位点附近的启动子大约70%包含CpG岛。<sup><a href=\"#quote_5\" class=\"kx_ref\">[5]</a></sup><sup><a href=\"#quote_6\" class=\"kx_ref\">[6]</a></sup> </p>\n<p>位于基因转录起始位点远处的启动子也经常包含CpG岛。例如，DNA修复基因ERCC1的启动子被定位于其编码区上游约5400个核苷酸处。<sup><a href=\"#quote_7\" class=\"kx_ref\">[7]</a></sup> CpG岛也经常出现在功能性非编码RNA的启动子中，如微小RNA和长链非编码RNA。 </p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14996317944152330","title":"启动子中CpG岛的甲基化稳定沉默基因","versionId":"14996317944152323","lemmaId":11094,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":10145103,"name":"科学百科编辑组","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576234501,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>由于启动子CpG岛中CpG位点的大量甲基化，基因会被沉默。<sup><a href=\"#quote_8\" class=\"kx_ref\">[8]</a></sup> 即使一个基因的沉默是由另一种机制引起的，也通常会伴随着启动子CpG岛中CpG位点的甲基化，以促使该基因的稳定沉默。<sup><a href=\"#quote_8\" class=\"kx_ref\">[8]</a></sup> 另一方面，启动子中CpG岛的低甲基化会导致基因过表达。 </p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14996317960929550","title":"癌症中启动子CpG高/低甲基化","versionId":"14996317944152324","lemmaId":11094,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":10145103,"name":"科学百科编辑组","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576234501,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>在癌症中，启动子CpG岛的高甲基化导致的基因表达缺失的频率比突变导致的基因表达缺失高10倍。例如，在结肠肿瘤中（与相邻正常出现的结肠粘膜相比），肿瘤基因的启动子中出现约600-800个高度甲基化的CpG岛，而这些CpG岛在相邻粘膜中没有发生甲基化。<sup><a href=\"#quote_9\" class=\"kx_ref\">[9]</a></sup><sup><a href=\"#quote_10\" class=\"kx_ref\">[10]</a></sup><sup><a href=\"#quote_11\" class=\"kx_ref\">[11]</a></sup> 相比之下，正如沃格尔斯坦等（Vogelstein et al.）人指出的那样，<sup><a href=\"#quote_2\" class=\"kx_ref\">[2]</a></sup> 结直肠癌通常只有大约3-6个驱动突变和33-66个搭便车突变（或乘客突变）。 </p> \n<h3>癌症中的DNA修复基因沉默</h3> \n<p>在散发性癌症中，偶尔会发现由于DNA修复基因的突变而导致DNA修复缺陷。然而，更常见的是，癌症中DNA修复基因表达的减少或缺失是由于其启动子的甲基化。例如，在被检查的113例结直肠癌中，只有4例在修复基因MGMT中有错义突变，而大多数是由于MGMT启动子区甲基化而降低了MGMT表达。<sup><a href=\"#quote_12\" class=\"kx_ref\">[12]</a></sup> 同样，在缺乏修复基因PMS2表达的119例错配修复缺陷型结直肠癌中，6例PMS2基因发生突变，而103例是因为PMS2的配对基因MLH1启动子甲基化进而导致PMS2表达受到抑制(缺少MLH1时PMS2蛋白不稳定)。<sup><a href=\"#quote_13\" class=\"kx_ref\">[13]</a></sup> 在剩余10例病例中，PMS2表达的缺失可能是由于微小RNA(miR-155，能够下调MLH1)的表观遗传过表达所致。<sup><a href=\"#quote_14\" class=\"kx_ref\">[14]</a></sup> </p> \n<h3>癌症中DNA修复基因的高甲基化频率</h3> \n<p>在两篇综述文章中所列出的17种癌症中，发现有22 种DNA修复基因，具有高甲基化启动子，并且表达减少或缺失。<sup><a href=\"#quote_15\" class=\"kx_ref\">[15]</a></sup><sup><a href=\"#quote_16\" class=\"kx_ref\">[16]</a></sup> 其中一篇综述中提到，<sup><a href=\"#quote_15\" class=\"kx_ref\">[15]</a></sup> MGMT启动子高度甲基化经常发生在许多癌症中，包括93%的膀胱癌、88%的胃癌、74%的甲状腺癌、40%-90%的结直肠癌和50%的脑癌。该综述还表明，在头颈癌、非小细胞肺癌或非小细胞肺癌鳞状细胞癌等一种或多种癌中，LIG4、NEIL1、ATM、MLH1或FANCB的启动子高甲基化发生的频率在33%-82%之间。关于沃纳综合征三磷酸腺苷（ATP）依赖性解旋酶的研究文献指出，DNA修复基因WRN有一个启动子，该启动子在许多癌症中经常高甲基化，大约有11%-38%的结直肠癌、头颈癌、胃癌、前列腺癌、乳腺癌、甲状腺癌、非霍奇金淋巴瘤、软骨肉瘤和骨肉瘤中有高甲基化的发生 。 </p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14996317960929551","title":"DNA修复基因高度甲基化在癌症中的可能作用","versionId":"14996317944152325","lemmaId":11094,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":10145103,"name":"科学百科编辑组","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576234501,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>正如金和罗伯森（Jin and Roberston）在他们的综述中所讨论的那样，<sup><a href=\"#quote_16\" class=\"kx_ref\">[16]</a></sup> 通过高甲基化沉默一个DNA修复基因可能是癌变的一个非常早期的步骤。这种沉默被认为与DNA修复基因的种系突变作用相似，并使细胞及其后代易于发展成癌症。另一篇综述中<sup><a href=\"#quote_17\" class=\"kx_ref\">[17]</a></sup> 也指出了癌症中DNA修复基因高度甲基化的早期作用。如果一个DNA修复需要的基因高甲基化—导致修复基因的缺陷——那么基因损伤就会累积。增加的DNA损伤会导致在DNA合成过程中引起更多的错误，导致出现一些会引发癌症的突变。 </p>\n<p>如果一个DNA修复基因的高甲基化是癌变的早期步骤，那么它也可能发生在肿瘤周围的正常出现的组织中(区域缺陷)。见下表： </p> \n<table class=\"wikitable sortable\"> \n <caption>\n  散发性癌症和邻近区域缺陷中DNA修复基因中高甲基化启动子的频率 \n </caption> \n <tbody>\n  <tr> \n   <th>癌症</th> \n   <th>基因</th> \n   <th>癌症中的频率</th> \n   <th>区域缺陷频率</th> \n   <th>Ref. </th>\n  </tr> \n  <tr> \n   <th>结直肠癌 </th> \n   <td>MGMT</td> \n   <td>55%</td> \n   <td>54%</td> \n   <td><sup><a href=\"#quote_18\" class=\"kx_ref\">[18]</a></sup> </td>\n  </tr> \n  <tr> \n   <th>结直肠癌 </th> \n   <td>MSH2</td> \n   <td>13%</td> \n   <td>5%</td> \n   <td><sup><a href=\"#quote_19\" class=\"kx_ref\">[19]</a></sup> </td>\n  </tr> \n  <tr> \n   <th>结直肠癌 </th> \n   <td>WRN</td> \n   <td>29%</td> \n   <td>13%</td> \n   <td><sup><a href=\"#quote_20\" class=\"kx_ref\">[20]</a></sup> </td>\n  </tr> \n  <tr> \n   <th>头颈癌 </th> \n   <td>MGMT</td> \n   <td>54%</td> \n   <td>38%</td> \n   <td><sup><a href=\"#quote_21\" class=\"kx_ref\">[21]</a></sup> </td>\n  </tr> \n  <tr> \n   <th>头颈癌 </th> \n   <td>MLH1</td> \n   <td>33%</td> \n   <td>25%</td> \n   <td><sup><a href=\"#quote_22\" class=\"kx_ref\">[22]</a></sup> </td>\n  </tr> \n  <tr> \n   <th>非小细胞肺癌 </th> \n   <td>ATM</td> \n   <td>69%</td> \n   <td>59%</td> \n   <td><sup><a href=\"#quote_23\" class=\"kx_ref\">[23]</a></sup> </td>\n  </tr> \n  <tr> \n   <th>非小细胞肺癌 </th> \n   <td>MLH1</td> \n   <td>69%</td> \n   <td>72%</td> \n   <td><sup><a href=\"#quote_23\" class=\"kx_ref\">[23]</a></sup> </td>\n  </tr> \n  <tr> \n   <th>胃癌 </th> \n   <td>MGMT</td> \n   <td>88%</td> \n   <td>78%</td> \n   <td><sup><a href=\"#quote_24\" class=\"kx_ref\">[24]</a></sup> </td>\n  </tr> \n  <tr> \n   <th>胃癌 </th> \n   <td>MLH1</td> \n   <td>73%</td> \n   <td>20%</td> \n   <td><sup><a href=\"#quote_25\" class=\"kx_ref\">[25]</a></sup> </td>\n  </tr> \n  <tr> \n   <th>食管癌 </th> \n   <td>MLH1</td> \n   <td>77%-100%</td> \n   <td>23%-79%</td> \n   <td><sup><a href=\"#quote_26\" class=\"kx_ref\">[26]</a></sup> </td>\n  </tr>\n </tbody>\n</table> \n<p>DNA损伤可能会容易导致跨损伤合成的错误进而引起突变，同时DNA损伤也可能在错误的DNA修复过程中引起表观遗传改变。<sup><a href=\"#quote_27\" class=\"kx_ref\">[27]</a></sup><sup><a href=\"#quote_28\" class=\"kx_ref\">[28]</a></sup><sup><a href=\"#quote_29\" class=\"kx_ref\">[29]</a></sup><sup><a href=\"#quote_30\" class=\"kx_ref\">[30]</a></sup> 由于DNA修复基因启动子的高甲基化而积累的DNA损伤可能是癌症中许多基因表观遗传改变增加的原因之一。 </p>\n<p>在一项早期的研究中，费尔南德斯等（Fernandez et al）人着眼于一组有限的转录启动子，<sup><a href=\"#quote_31\" class=\"kx_ref\">[31]</a></sup> 研究了855例原发性肿瘤的DNA甲基化谱。将每种肿瘤类型与其相应的正常组织进行比较，729个CpG岛位点(评估的1322个CpG岛位点的55%)显示不同的DNA甲基化。在这些位点中，496个位点高甲基化（被抑制)，233个位点低甲基化(被激活)。因此，在肿瘤中存在高水平的启动子甲基化改变，其中一些改变可能会导致癌变。 </p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14996317960929552","title":"癌症中微小RNA的甲基化","versionId":"14996317944152326","lemmaId":11094,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":10145103,"name":"科学百科编辑组","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576234501,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>在哺乳动物中，微小RNA(miRNAs)调控大约60%蛋白质编码基因的转录活性。<sup><a href=\"#quote_32\" class=\"kx_ref\">[32]</a></sup> 每个独立的微小RNA平均可以靶向抑制大约200个蛋白质编码基因的信使RNA的转录。<sup><a href=\"#quote_33\" class=\"kx_ref\">[33]</a></sup> 在 Vrba等人评估的乳腺组织的167个微小RNA中，约三分之一的启动子在乳腺癌和正常乳腺组织中存在高或低甲基化差异。<sup><a href=\"#quote_34\" class=\"kx_ref\">[34]</a></sup> 最近的一项研究指出，Vrba等人评估的167个微小 RNA仅占乳腺组织中已发现表达的微小RNA的10%。<sup><a href=\"#quote_35\" class=\"kx_ref\">[35]</a></sup> 这项新的研究发现，乳腺组织中58%的微小RNA在其乳腺癌启动子中具有不同的甲基化区域，包括278个高甲基化的微小RNA和802个低甲基化的微小RNA。 </p>\n<p>miR-182是乳腺癌中过表达约100倍的微小RNA之一。<sup><a href=\"#quote_36\" class=\"kx_ref\">[36]</a></sup> MiR-182靶向调控BRCA1信使RNA，因此，miR-182过表达可能是许多乳腺癌中BRCA1蛋白表达减少的主要原因。<sup><a href=\"#quote_37\" class=\"kx_ref\">[37]</a></sup> </p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14996317977706766","title":"癌症中控制DNA甲基转移酶基因的微小RNA","versionId":"14996317944152327","lemmaId":11094,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":10145103,"name":"科学百科编辑组","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576234501,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>一些微小RNA靶向调控DNA甲基转移酶基因<i>DNMT1</i>, <i>DNMT3A</i> 和 <i>DNMT3B</i>的信使RNA，它们的基因产物是启动和稳定启动子甲基化所必需的。正如以上三篇文献中所总结的那样，<sup><a href=\"#quote_38\" class=\"kx_ref\">[38]</a></sup><sup><a href=\"#quote_39\" class=\"kx_ref\">[39]</a></sup><sup><a href=\"#quote_40\" class=\"kx_ref\">[40]</a></sup> 微小RNA miR-29a、miR-29b和miR-29c靶向定位DNMT3A和DNMT3B；miR-148a和miR-148b靶向定位DNMT3B；miR-152和miR-301靶向定位DNMT1。此外，miR-34b靶向定位DNMT1，并且miR-34b自身的启动子在大多数前列腺癌中高甲基化和低表达。<sup><a href=\"#quote_41\" class=\"kx_ref\">[41]</a></sup> 当这些微小RNA的表达改变时，它们也可能就成为癌症中蛋白质编码基因启动子高/低甲基化的诱因。 </p>","pics":null,"card":null,"references":[],"versionCount":0}],"references":[{"id":1,"type":"book","title":"Seisenberger S, Peat JR, Hore TA, Santos F, Dean W, Reik W (2013). \"Reprogramming DNA methylation in the mammalian life cycle: building and breaking epigenetic barriers\". Philos. Trans. R. Soc. Lond. B Biol. Sci. 368 (1609): 20110330. doi:10.1098/rstb.2011.0330. PMC 3539359. PMID 23166394.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":2,"type":"book","title":"Vogelstein B, Papadopoulos N, Velculescu VE, Zhou S, Diaz LA, Kinzler KW (2013). \"Cancer genome landscapes\". Science. 339 (6127): 1546–58. doi:10.1126/science.1235122. PMC 3749880. PMID 23539594.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":3,"type":"book","title":"Wang YP, Lei QY (2018). \"Metabolic recoding of epigenetics in cancer\". Cancer Commun (Lond). 38 (1): 25. doi:10.1186/s40880-018-0302-3. PMC 5993135. PMID 29784032.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":4,"type":"book","title":"Tessitore A, Cicciarelli G, Del Vecchio F, Gaggiano A, Verzella D, Fischietti M, Vecchiotti D, Capece D, Zazzeroni F, Alesse E (2014). \"MicroRNAs in the DNA Damage/Repair Network and Cancer\". Int J Genom. 2014: 1–10. doi:10.1155/2014/820248. PMC 3926391. PMID 24616890.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":5,"type":"book","title":"Saxonov S, Berg P, Brutlag DL (2006). \"A genome-wide analysis of CpG dinucleotides in the human genome distinguishes two distinct classes of promoters\". Proc. Natl. Acad. Sci. U.S.A. 103 (5): 1412–7. doi:10.1073/pnas.0510310103. PMC 1345710. PMID 16432200.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":6,"type":"book","title":"Deaton AM, Bird A (2011). \"CpG islands and the regulation of transcription\". Genes Dev. 25 (10): 1010–22. doi:10.1101/gad.2037511. PMC 3093116. PMID 21576262.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":7,"type":"book","title":"Chen HY, Shao CJ, Chen FR, Kwan AL, Chen ZP (2010). \"Role of ERCC1 promoter hypermethylation in drug resistance to cisplatin in human gliomas\". Int. J. Cancer. 126 (8): 1944–54. doi:10.1002/ijc.24772. PMID 19626585.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":8,"type":"book","title":"Bird A (2002). \"DNA methylation patterns and epigenetic memory\". Genes Dev. 16 (1): 6–21. doi:10.1101/gad.947102. PMID 11782440.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":9,"type":"book","title":"Illingworth RS, Gruenewald-Schneider U, Webb S, Kerr AR, James KD, Turner DJ, Smith C, Harrison DJ, Andrews R, Bird AP (2010). \"Orphan CpG islands identify numerous conserved promoters in the mammalian genome\". PLoS Genet. 6 (9): e1001134. doi:10.1371/journal.pgen.1001134. PMC 2944787. PMID 20885785.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":10,"type":"book","title":"Wei J, Li G, Dang S, Zhou Y, Zeng K, Liu M (2016). \"Discovery and Validation of Hypermethylated Markers for Colorectal Cancer\". Dis. Markers. 2016: 1–7. doi:10.1155/2016/2192853. PMC 4963574. PMID 27493446.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":11,"type":"book","title":"Beggs AD, Jones A, El-Bahrawy M, El-Bahwary M, Abulafi M, Hodgson SV, Tomlinson IP (2013). \"Whole-genome methylation analysis of benign and malignant colorectal tumours\". J. Pathol. 229 (5): 697–704. doi:10.1002/path.4132. PMC 3619233. PMID 23096130.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":12,"type":"book","title":"Halford S, Rowan A, Sawyer E, Talbot I, Tomlinson I (June 2005). \"O(6)-methylguanine methyltransferase in colorectal cancers: detection of mutations, loss of expression, and weak association with G:C>A:T transitions\". Gut. 54 (6): 797–802. doi:10.1136/gut.2004.059535. PMC 1774551. PMID 15888787.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":13,"type":"book","title":"Truninger K, Menigatti M, Luz J, et al. (May 2005). \"Immunohistochemical analysis reveals high frequency of PMS2 defects in colorectal cancer\". Gastroenterology. 128 (5): 1160–71. doi:10.1053/j.gastro.2005.01.056. PMID 15887099.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":14,"type":"book","title":"Valeri N, Gasparini P, Fabbri M, et al. (April 2010). \"Modulation of mismatch repair and genomic stability by miR-155\". Proceedings of the National Academy of Sciences of the United States of America. 107 (15): 6982–7. Bibcode:2010PNAS..107.6982V. doi:10.1073/pnas.1002472107. JSTOR 25665289. PMC 2872463. PMID 20351277.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":15,"type":"book","title":"Carol Bernstein and Harris Bernstein (2015). Epigenetic Reduction of DNA Repair in Progression to Cancer, Advances in DNA Repair, Prof. Clark Chen (Ed.), ISBN 978-953-51-2209-8, InTech, Available from: http://www.intechopen.com/books/advances-in-dna-repair/epigenetic-reduction-of-dna-repair-in-progression-to-cancer","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":16,"type":"book","title":"Jin B, Robertson KD (2013). DNA methyltransferases, DNA damage repair, and cancer. Adv. Exp. Med. Biol. Advances in Experimental Medicine and Biology. 754. pp. 3–29. doi:10.1007/978-1-4419-9967-2_1. ISBN 978-1-4419-9966-5. PMC 3707278. PMID 22956494.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":17,"type":"book","title":"Bernstein C, Nfonsam V, Prasad AR, Bernstein H (2013). \"Epigenetic field defects in progression to cancer\". World J Gastrointest Oncol. 5 (3): 43–9. doi:10.4251/wjgo.v5.i3.43. PMC 3648662. PMID 23671730.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":18,"type":"book","title":"Svrcek M, Buhard O, Colas C, Coulet F, Dumont S, Massaoudi I, Lamri A, Hamelin R, Cosnes J, Oliveira C, Seruca R, Gaub MP, Legrain M, Collura A, Lascols O, Tiret E, Fléjou JF, Duval A (November 2010). \"Methylation tolerance due to an O6-methylguanine DNA methyltransferase (MGMT) field defect in the colonic mucosa: an initiating step in the development of mismatch repair-deficient colorectal cancers\". Gut. 59 (11): 1516–26. doi:10.1136/gut.2009.194787. PMID 20947886.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":19,"type":"book","title":"Lee KH, Lee JS, Nam JH, Choi C, Lee MC, Park CS, Juhng SW, Lee JH (2011). \"Promoter methylation status of hMLH1, hMSH2, and MGMT genes in colorectal cancer associated with adenoma-carcinoma sequence\". Langenbecks Arch Surg. 396 (7): 1017–26. doi:10.1007/s00423-011-0812-9. PMID 21706233.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":20,"type":"book","title":"Kawasaki T, Ohnishi M, Suemoto Y, Kirkner GJ, Liu Z, Yamamoto H, Loda M, Fuchs CS, Ogino S (2008). \"WRN promoter methylation possibly connects mucinous differentiation, microsatellite instability and CpG island methylator phenotype in colorectal cancer\". Mod. Pathol. 21 (2): 150–8. doi:10.1038/modpathol.3800996. PMID 18084250.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":21,"type":"book","title":"Paluszczak J, Misiak P, Wierzbicka M, Woźniak A, Baer-Dubowska W (February 2011). \"Frequent hypermethylation of DAPK, RARbeta, MGMT, RASSF1A and FHIT in laryngeal squamous cell carcinomas and adjacent normal mucosa\". Oral Oncol. 47 (2): 104–7. doi:10.1016/j.oraloncology.2010.11.006. PMID 21147548.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":22,"type":"book","title":"Zuo C, Zhang H, Spencer HJ, Vural E, Suen JY, Schichman SA, Smoller BR, Kokoska MS, Fan CY (October 2009). \"Increased microsatellite instability and epigenetic inactivation of the hMLH1 gene in head and neck squamous cell carcinoma\". Otolaryngol Head Neck Surg. 141 (4): 484–90. doi:10.1016/j.otohns.2009.07.007. PMID 19786217.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":23,"type":"book","title":"Safar AM, Spencer H, Su X, Coffey M, Cooney CA, Ratnasinghe LD, Hutchins LF, Fan CY (2005). \"Methylation profiling of archived non-small cell lung cancer: a promising prognostic system\". Clin. Cancer Res. 11 (12): 4400–5. doi:10.1158/1078-0432.CCR-04-2378. PMID 15958624.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":24,"type":"book","title":"Zou XP, Zhang B, Zhang XQ, Chen M, Cao J, Liu WJ (November 2009). \"Promoter hypermethylation of multiple genes in early gastric adenocarcinoma and precancerous lesions\". Hum. Pathol. 40 (11): 1534–42. doi:10.1016/j.humpath.2009.01.029. PMID 19695681.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":25,"type":"book","title":"Wani M, Afroze D, Makhdoomi M, Hamid I, Wani B, Bhat G, Wani R, Wani K (2012). \"Promoter methylation status of DNA repair gene (hMLH1) in gastric carcinoma patients of the Kashmir valley\". Asian Pac. J. Cancer Prev. 13 (8): 4177–81. doi:10.7314/APJCP.2012.13.8.4177. PMID 23098428.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":26,"type":"book","title":"Agarwal A, Polineni R, Hussein Z, Vigoda I, Bhagat TD, Bhattacharyya S, Maitra A, Verma A (2012). \"Role of epigenetic alterations in the pathogenesis of Barrett's esophagus and esophageal adenocarcinoma\". Int J Clin Exp Pathol. 5 (5): 382–96. PMC 3396065. PMID 22808291.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":27,"type":"book","title":"O'Hagan HM, Mohammad HP, Baylin SB (2008). \"Double strand breaks can initiate gene silencing and SIRT1-dependent onset of DNA methylation in an exogenous promoter CpG island\". PLoS Genetics. 4 (8): e1000155. doi:10.1371/journal.pgen.1000155. PMC 2491723. PMID 18704159.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":28,"type":"book","title":"Cuozzo C, Porcellini A, Angrisano T, et al. (July 2007). \"DNA damage, homology-directed repair, and DNA methylation\". PLoS Genetics. 3 (7): e110. doi:10.1371/journal.pgen.0030110. PMC 1913100. PMID 17616978.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":29,"type":"book","title":"Shanbhag NM, Rafalska-Metcalf IU, Balane-Bolivar C, Janicki SM, Greenberg RA (June 2010). \"ATM-dependent chromatin changes silence transcription in cis to DNA double-strand breaks\". Cell. 141 (6): 970–81. doi:10.1016/j.cell.2010.04.038. PMC 2920610. PMID 20550933.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":30,"type":"book","title":"Morano A, Angrisano T, Russo G, Landi R, Pezone A, Bartollino S, Zuchegna C, Babbio F, Bonapace IM, Allen B, Muller MT, Chiariotti L, Gottesman ME, Porcellini A, Avvedimento EV (January 2014). \"Targeted DNA methylation by homology-directed repair in mammalian cells. Transcription reshapes methylation on the repaired gene\". Nucleic Acids Res. 42 (2): 804–21. doi:10.1093/nar/gkt920. PMC 3902918. PMID 24137009.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":31,"type":"book","title":"Fernandez AF, Assenov Y, Martin-Subero JI, Balint B, Siebert R, Taniguchi H, Yamamoto H, Hidalgo M, Tan AC, Galm O, Ferrer I, Sanchez-Cespedes M, Villanueva A, Carmona J, Sanchez-Mut JV, Berdasco M, Moreno V, Capella G, Monk D, Ballestar E, Ropero S, Martinez R, Sanchez-Carbayo M, Prosper F, Agirre X, Fraga MF, Graña O, Perez-Jurado L, Mora J, Puig S, Prat J, Badimon L, Puca AA, Meltzer SJ, Lengauer T, Bridgewater J, Bock C, Esteller M (2012). \"A DNA methylation fingerprint of 1628 human samples\". Genome Res. 22 (2): 407–19. doi:10.1101/gr.119867.110. PMC 3266047. PMID 21613409.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":32,"type":"book","title":"Friedman, RC; Farh, KK; Burge, CB; Bartel, DP (January 2009). \"Most mammalian mRNAs are conserved targets of microRNAs\". Genome Res. 19 (1): 92–105. doi:10.1101/gr.082701.108. PMC 2612969. PMID 18955434.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":33,"type":"book","title":"Krek A, Grün D, Poy MN, Wolf R, Rosenberg L, Epstein EJ, MacMenamin P, da Piedade I, Gunsalus KC, Stoffel M, Rajewsky N (2005). \"Combinatorial microRNA target predictions\". Nat. Genet. 37 (5): 495–500. doi:10.1038/ng1536. PMID 15806104.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":34,"type":"book","title":"Vrba L, Muñoz-Rodríguez JL, Stampfer MR, Futscher BW (2013). \"miRNA gene promoters are frequent targets of aberrant DNA methylation in human breast cancer\". PLoS ONE. 8 (1): e54398. doi:10.1371/journal.pone.0054398. PMC 3547033. PMID 23342147.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":35,"type":"book","title":"Li Y, Zhang Y, Li S, Lu J, Chen J, Wang Y, Li Y, Xu J, Li X (2015). \"Genome-wide DNA methylome analysis reveals epigenetically dysregulated non-coding RNAs in human breast cancer\". Sci Rep. 5: 8790. doi:10.1038/srep08790. PMC 4350105. PMID 25739977.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":36,"type":"book","title":"Krishnan K, Steptoe AL, Martin HC, Wani S, Nones K, Waddell N, Mariasegaram M, Simpson PT, Lakhani SR, Gabrielli B, Vlassov A, Cloonan N, Grimmond SM (2013). \"MicroRNA-182-5p targets a network of genes involved in DNA repair\". RNA. 19 (2): 230–42. doi:10.1261/rna.034926.112. PMC 3543090. PMID 23249749.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":37,"type":"book","title":"Moskwa P, Buffa FM, Pan Y, Panchakshari R, Gottipati P, Muschel RJ, Beech J, Kulshrestha R, Abdelmohsen K, Weinstock DM, Gorospe M, Harris AL, Helleday T, Chowdhury D (2011). \"miR-182-mediated downregulation of BRCA1 impacts DNA repair and sensitivity to PARP inhibitors\". Mol. Cell. 41 (2): 210–20. doi:10.1016/j.molcel.2010.12.005. PMC 3249932. PMID 21195000.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":38,"type":"book","title":"Suzuki H, Maruyama R, Yamamoto E, Kai M (2012). \"DNA methylation and microRNA dysregulation in cancer\". Mol Oncol. 6 (6): 567–78. doi:10.1016/j.molonc.2012.07.007. PMC 5528344. PMID 22902148.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":39,"type":"book","title":"Suzuki H, Maruyama R, Yamamoto E, Kai M (2013). \"Epigenetic alteration and microRNA dysregulation in cancer\". Front Genet. 4: 258. doi:10.3389/fgene.2013.00258. PMC 3847369. PMID 24348513.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":40,"type":"book","title":"Kaur S, Lotsari-Salomaa JE, Seppänen-Kaijansinkko R, Peltomäki P (2016). MicroRNA Methylation in Colorectal Cancer. Adv. Exp. Med. Biol. Advances in Experimental Medicine and Biology. 937. pp. 109–22. doi:10.1007/978-3-319-42059-2_6. ISBN 978-3-319-42057-8. PMID 27573897.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":41,"type":"book","title":"Majid S, Dar AA, Saini S, Shahryari V, Arora S, Zaman MS, Chang I, Yamamura S, Tanaka Y, Chiyomaru T, Deng G, Dahiya R (2013). \"miRNA-34b inhibits prostate cancer through demethylation, active chromatin modifications, and AKT pathways\". Clin. Cancer Res. 19 (1): 73–84. doi:10.1158/1078-0432.CCR-12-2952. PMC 3910324. PMID 23147995.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false}],"recommendReferences":null,"auditState":2,"lemmaLevel":1,"origin":0,"originEnTitle":null,"originZhTitle":null,"pv":143,"auditType":0,"synonyms":null,"showEditTime":"2019.12.13 18:55","auditors":[{"uid":0,"name":"Ki.κe","pic":"https://web.archive.org/web/20221025130020/https://wx.qlogo.cn/mmopen/vi_32/y67kfr32Doib4wg71Jiau7jVWvharic3nRKgdRRQSl6koeQJCo0GQs2Krw0vwdFRsOWnHIQOwAZsSg5lIkIrFCOcQ/132","introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":"","jobBrief":"","role":0,"roleName":null,"title":"","professionalTitle":null,"phoneNo":null,"editable":true,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false}],"hasZhishiNav":false,"auditInfos":{},"isHistory":false};</script><script crossorigin="anonymous" src="./1094.癌症中的DNA甲基化 - 搜狗科学百科_files/aegis.min.js.download"></script><script crossorigin="anonymous" src="./1094.癌症中的DNA甲基化 - 搜狗科学百科_files/main_2020092401.js.download"></script><script crossorigin="anonymous" src="./1094.癌症中的DNA甲基化 - 搜狗科学百科_files/react.production.min.js.download"></script><script crossorigin="anonymous" src="./1094.癌症中的DNA甲基化 - 搜狗科学百科_files/react-dom.production.min.js.download"></script><script crossorigin="anonymous" src="./1094.癌症中的DNA甲基化 - 搜狗科学百科_files/jquery-1.11.1.min.js.download"></script><script crossorigin="anonymous" src="./1094.癌症中的DNA甲基化 - 搜狗科学百科_files/main_2022062701.js.download"></script><script crossorigin="anonymous" src="./1094.癌症中的DNA甲基化 - 搜狗科学百科_files/main_66bbe21.js.download"></script><script crossorigin="anonymous" src="./1094.癌症中的DNA甲基化 - 搜狗科学百科_files/react.production.min.js.download"></script><script crossorigin="anonymous" src="./1094.癌症中的DNA甲基化 - 搜狗科学百科_files/react-dom.production.min.js.download"></script><script crossorigin="anonymous" src="./1094.癌症中的DNA甲基化 - 搜狗科学百科_files/main_edf0f08.js.download"></script>
</body></html>