<!DOCTYPE html>
<!-- saved from url=(0083)https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/d10275.htm -->
<html class="" data-reactroot=""><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><script src="./275.单核苷酸多态性 - 搜狗科学百科_files/analytics.js.download" type="text/javascript"></script>
<script type="text/javascript">window.addEventListener('DOMContentLoaded',function(){var v=archive_analytics.values;v.service='wb';v.server_name='wwwb-app226.us.archive.org';v.server_ms=623;archive_analytics.send_pageview({});});</script>
<script type="text/javascript" src="./275.单核苷酸多态性 - 搜狗科学百科_files/bundle-playback.js.download" charset="utf-8"></script>
<script type="text/javascript" src="./275.单核苷酸多态性 - 搜狗科学百科_files/wombat.js.download" charset="utf-8"></script>
<script type="text/javascript">
  __wm.init("https://web.archive.org/web");
  __wm.wombat("https://baike.sogou.com/kexue/d10275.htm","20221028214411","https://web.archive.org/","web","/_static/",
	      "1666993451");
</script>
<link rel="stylesheet" type="text/css" href="./275.单核苷酸多态性 - 搜狗科学百科_files/banner-styles.css">
<link rel="stylesheet" type="text/css" href="./275.单核苷酸多态性 - 搜狗科学百科_files/iconochive.css">
<!-- End Wayback Rewrite JS Include -->
<meta name="save" content="history"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="baidu-site-verification" content="VWGb6TyYx8"><meta content="单核苷酸多态性 - 搜狗科学百科" name="keywords"><meta content="搜狗科学百科是一部有着平等、协作、分享、自由理念的网络科学全书，为每一个互联网用户创造一个涵盖所有领域知识、服务的中文知识性平台。" name="description"><meta http-equiv="x-dns-prefetch-control" content="on"><meta name="server" baike="235" ip="210" env="online"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028214411/https://cache.soso.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028214411/https://hhy.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028214411/https://pic.baike.soso.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028214411/https://ugc.qpic.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028214411/https://xui.ptlogin2.qq.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028214411/https://q1.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028214411/https://q2.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028214411/https://q3.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028214411/https://q4.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028214411/https://q.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028214411/https://img01.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028214411/https://img02.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028214411/https://img03.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028214411/https://img04.sogoucdn.com/"><link rel="Shortcut Icon" href="https://web.archive.org/web/20221028214411im_/https://www.sogou.com/images/logo/new/favicon.ico?v=4"><link rel="Bookmark" href="https://www.sogou.com/images/logo/new/favicon.ico?v=4"><link href="./275.单核苷酸多态性 - 搜狗科学百科_files/base_b849887.css" rel="stylesheet"><link href="./275.单核苷酸多态性 - 搜狗科学百科_files/detail_378aed5.css" rel="stylesheet"><link href="./275.单核苷酸多态性 - 搜狗科学百科_files/inviteAudit_7894507.css" rel="stylesheet"><link rel="stylesheet" href="./275.单核苷酸多态性 - 搜狗科学百科_files/highlight.min.css"><title>单核苷酸多态性 - 搜狗科学百科</title><style>.onekey-close {
	position: absolute;
	top: 16px;
	right: 16px;
	width: 24px;
	height: 24px;
	background: url(//web.archive.org/web/20221028205211/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	text-indent: -999em;
	background-size: 84px;
	background-position: -63px 0;
}

.onekey-login {
	position: absolute;
	top: 16.4%;
	left: 0;
	right: 0;
	width: 100%;
}

/* .onekey-login-img {
    width: 75px;
    height: 75px;
    background: url("https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/images/sprite_wap_baike.png") no-repeat;
    background-size: 100px 91px;
    background-position: 0 0;
    background-repeat: no-repeat;
    margin: 0 auto;
} */

.onekey-login-title {
	text-align: center;
	padding-bottom: 3px;
	font-size: 21px;
	font-weight: bold;
	line-height: 30px;
	color: #000;
}

.onekey-login-txt {
	text-align: center;
	font-family: PingFangSC;
	font-size: 14px;
	line-height: 20px;
	color: #8f8f8f;
}

.onekey-login-qq,
.onekey-login-wx,
.onekey-login-phone {
	display: block;
	width: 245px;
	height: 54px;
	border-radius: 45px;
	text-align: center;

	margin: 0 auto;
	font-size: 17px;
	line-height: 24px;
	color: #000;
	/* padding: 16px 77px; */
	border-radius: 12px;
	border: solid 1px #e0e0e0;
}
.onekey-qq-content,
.onekey-vx-content,
.onekey-phone-content {
	display: inline-block;
	margin-top: 16px;
}
.onekey-qq-content {
	padding: 0 5px;
}

.onekey-login-qq {
	margin-top: 48px;
	margin-bottom: 24px;
}

.onekey-login-qq:before {
	display: inline-block;
	content: "";
	width: 20px;
	height: 20px;
	background: url(//web.archive.org/web/20221028205211/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	background-size: 80px;
	background-position: -20px 0;
	vertical-align: top;
	margin: 17px 8px 0 0;
}

.onekey-login-wx {
	margin-bottom: 24px;
}

.onekey-login-wx:before {
	display: inline-block;
	content: "";
	width: 21px;
	height: 21px;
	background: url(//web.archive.org/web/20221028205211/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	background-size: 84px;
	background-position: 0 0;
	vertical-align: top;
	margin: 17px 10px 0 0;
}

.onekey-login-phone {
}

.onekey-login-phone:before {
	display: inline-block;
	content: "";
	width: 21px;
	height: 21px;
	background: url(//web.archive.org/web/20221028205211/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	background-size: 84px;
	background-position: -42px 0;
	vertical-align: top;
	margin: 17px 10px 0 0;
}

.onekey-fixed {
	z-index: 100;
	position: fixed;
	top: 0;
	bottom: 0;
	left: 0;
	right: 0;
	background: #fff;
	width: 100%;
	height: 100%;
}

.onekey-fixed.forbid {
	z-index: 100;
	position: fixed;
	top: auto;
	bottom: 68px;
	left: 9%;
	right: 0;
	background: rgba(0, 0, 0, 0.7);
	width: 82%;
	height: 43px;
	border-radius: 25px;
	color: #ffffff;
}
.onekey-login-title.forbid {
	text-align: center;
	padding-bottom: 3px;
	font-size: 14px;
	font-weight: normal;
	line-height: 30px;
	color: white;
}
</style><style>#login_mask {
  background: #000;
  opacity: 0.5;
  filter: alpha(opacity=50);
  position: fixed;
  /*fixed好像在哪个IE上有BUG，先用用*/
  left: 0;
  top: 0;
  z-index: 999;
  height: 100%;
}

#login_iframe_container {
  position: fixed;
  width: 550px;
  height: 360px;
  z-index: 1020;
  background-color: #ffffff;
}

@media screen and (max-width: 828px) {
  #login_iframe_container {
    top: 50% !important;
    left: 50% !important;
    transform: translate(-50%, -50%);
  }
}

#login_iframe_container.new-login {
  width: 550px;
  height: 360px;
  background-image: url(//web.archive.org/web/20221028205211/https://hhy.sogoucdn.com/js/common/hhy/background_2a4a8a6.png);
}

#login_iframe_container.new-login.no-bg {
  background: #fff;
}

#login_iframe_container.new-login .login-title {
  width: 100%;
  height: 42px;
  line-height: 42px;
  text-align: center;
  font-size: 30px;
  letter-spacing: 0.19px;
  color: #ffffff;
  margin-top: 62px;
}
#login_iframe_container.new-login .forbid-title {
  width: 100%;
  height: 42px;
  line-height: 42px;
  text-align: center;
  font-size: 24px;
  letter-spacing: 0.19px;
  color: #333333;
  margin-top: 150px;
}

#login_iframe_container.new-login.no-bg .login-title {
  color: #333333;
}

#login_iframe_container.new-login .login-subtitle {
  width: 100%;
  height: 18px;
  line-height: 18px;
  font-size: 13px;
  letter-spacing: 0.08px;
  color: #ffffff;
  text-align: center;
  margin-top: 9px;
  margin-bottom: 43px;
}

#login_iframe_container.new-login.no-bg .login-subtitle {
  color: #999999;
}

#login_iframe_container.new-login .login-subtitle::before {
  content: '';
  display: inline-block;
  width: 10px;
  height: 1px;
  background-color: #ffffff;
  position: relative;
  top: -4px;
  left: -5px;
}

#login_iframe_container.new-login .login-subtitle::after {
  content: '';
  display: inline-block;
  width: 10px;
  height: 1px;
  background-color: #ffffff;
  position: relative;
  top: -4px;
  left: 5px;
}

#login_iframe_container.new-login.no-bg .login-subtitle::before {
  background-color: #999999;
}

#login_iframe_container.new-login.no-bg .login-subtitle::after {
  background-color: #999999;
}

#login_iframe_container.new-login .close-btn {
  position: absolute;
  top: 20px;
  right: 20px;
  width: 12px;
  height: 12px;
  background: url(//web.archive.org/web/20221028205211/https://hhy.sogoucdn.com/js/common/hhy/login-sprites_e3853e5.png) -59px -10px;
  background-size: 81px 91px;
  cursor: pointer;
}

#login_iframe_container.new-login .login-btn {
  width: 220px;
  height: 47px;
  border-radius: 24px;
  border: solid 1px #dddddd;
  background-color: #ffffff;
  margin: 0 auto;
  margin-top: 28px;
  position: relative;
  display: block;
}

#login_iframe_container.new-login .login-btn .login-icon {
  position: absolute;
}

#login_iframe_container.new-login .login-btn .login-text {
  width: 61px;
  height: 47px;
  line-height: 47px;
  vertical-align: middle;
  font-size: 15px;
  letter-spacing: 0.1px;
  color: #666666;
  position: absolute;
  right: 62px;
}

#login_iframe_container.new-login .login-btn.qq-btn .login-icon {
  width: 22px;
  height: 27px;
  top: 10px;
  left: 67px;
  background: url(//web.archive.org/web/20221028205211/https://hhy.sogoucdn.com/js/common/hhy/login-sprites_e3853e5.png) -10px -54px;
  background-size: 81px 91px;
}

#login_iframe_container.new-login .login-btn.qq-btn .login-text {
  right: 59px;
}

#login_iframe_container.new-login .login-btn.wechat-btn .login-icon {
  width: 29px;
  height: 24px;
  top: 12px;
  left: 62px;
  background: url(//web.archive.org/web/20221028205211/https://hhy.sogoucdn.com/js/common/hhy/login-sprites_e3853e5.png) -10px -10px;
  background-size: 81px 91px;
}</style><style>/* -- container -- */
.rodal,
.rodal-mask {
    top: 0;
    left: 0;
    width: 100%;
    height: 100%;
    z-index: 100;
}

.rodal {
    position: fixed;
}

/* -- mask -- */
.rodal-mask {
    position: fixed;
    background: rgba(0, 0, 0, .5);
}

/* -- dialog -- */
.rodal-dialog {
    position: absolute;
    z-index: 101;
    background: #fff;
    border-radius: 3px;
    box-shadow: 0 1px 3px rgba(0, 0, 0, .2);
}

.rodal-center {
    top: 50%;
    transform: translateY(-50%);
    left: 0;
    right: 0;
    margin: 0 auto;
}

.rodal-bottom {
    left: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

.rodal-top {
    left: 0;
    right: 0;
    top: 0;
    margin: auto;
}

.rodal-left {
    top: 0;
    left: 0;
    bottom: 0;
    margin: auto;
}

.rodal-right {
    top: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

/* -- close button -- */
.rodal-close {
    position: absolute;
    cursor: pointer;
    top: 16px;
    right: 16px;
    width: 16px;
    height: 16px;
}

.rodal-close:before,
.rodal-close:after {
    position: absolute;
    content: '';
    height: 2px;
    width: 100%;
    top: 50%;
    left: 0;
    margin-top: -1px;
    background: #999;
    border-radius: 100%;
    -webkit-transition: background .2s;
    transition: background .2s;
}

.rodal-close:before {
    -webkit-transform: rotate(45deg);
    transform: rotate(45deg);
}

.rodal-close:after {
    -webkit-transform: rotate(-45deg);
    transform: rotate(-45deg);
}

.rodal-close:hover:before,
.rodal-close:hover:after {
    background: #333;
}

/* -- fade -- */
/* @-webkit-keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

@keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

.rodal-fade-enter {
    -webkit-animation: rodal-fade-enter both ease-in;
    animation: rodal-fade-enter both ease-in;
} */

@-webkit-keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

@keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

.rodal-fade-leave {
    -webkit-animation: rodal-fade-leave both ease-out;
    animation: rodal-fade-leave both ease-out;
}

/* -- zoom -- */
@-webkit-keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-enter {
    -webkit-animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-leave {
    -webkit-animation: rodal-zoom-leave both;
    animation: rodal-zoom-leave both;
}

/* -- slideDown -- */
@-webkit-keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-enter {
    -webkit-animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-leave {
    -webkit-animation: rodal-slideDown-leave both;
    animation: rodal-slideDown-leave both;
}

/* -- slideLeft -- */
@-webkit-keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-enter {
    -webkit-animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-leave {
    -webkit-animation: rodal-slideLeft-leave both;
    animation: rodal-slideLeft-leave both;
}

/* -- slideRight -- */
@-webkit-keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-enter {
    -webkit-animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-leave {
    -webkit-animation: rodal-slideRight-leave both;
    animation: rodal-slideRight-leave both;
}

/* -- slideUp -- */
@-webkit-keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-enter {
    -webkit-animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
    animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
}

@-webkit-keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-leave {
    -webkit-animation: rodal-slideUp-leave both;
    animation: rodal-slideUp-leave both;
}

/* -- flip -- */
@-webkit-keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

@keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

.rodal-flip-enter {
    -webkit-animation: rodal-flip-enter both ease-in;
    animation: rodal-flip-enter both ease-in;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

@-webkit-keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

@keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

.rodal-flip-leave {
    -webkit-animation: rodal-flip-leave both;
    animation: rodal-flip-leave both;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

/* -- rotate -- */
@-webkit-keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-enter {
    -webkit-animation: rodal-rotate-enter both;
    animation: rodal-rotate-enter both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

@-webkit-keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-leave {
    -webkit-animation: rodal-rotate-leave both;
    animation: rodal-rotate-leave both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

/* -- door -- */
@-webkit-keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

@keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

.rodal-door-enter {
    -webkit-animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

@keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

.rodal-door-leave {
    -webkit-animation: rodal-door-leave both;
    animation: rodal-door-leave both;
}</style><style>/* -- container -- */
.rodal,
.rodal-mask {
    top: 0;
    left: 0;
    width: 100%;
    height: 100%;
    z-index: 100;
}

.rodal {
    position: fixed;
}

/* -- mask -- */
.rodal-mask {
    position: fixed;
    background: rgba(0, 0, 0, .5);
}

/* -- dialog -- */
.rodal-dialog {
    position: absolute;
    z-index: 101;
    background: #fff;
    border-radius: 3px;
    box-shadow: 0 1px 3px rgba(0, 0, 0, .2);
}

.rodal-center {
    top: 50%;
    transform: translateY(-50%);
    left: 0;
    right: 0;
    margin: 0 auto;
}

.rodal-bottom {
    left: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

.rodal-top {
    left: 0;
    right: 0;
    top: 0;
    margin: auto;
}

.rodal-left {
    top: 0;
    left: 0;
    bottom: 0;
    margin: auto;
}

.rodal-right {
    top: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

/* -- close button -- */
.rodal-close {
    position: absolute;
    cursor: pointer;
    top: 16px;
    right: 16px;
    width: 16px;
    height: 16px;
}

.rodal-close:before,
.rodal-close:after {
    position: absolute;
    content: '';
    height: 2px;
    width: 100%;
    top: 50%;
    left: 0;
    margin-top: -1px;
    background: #999;
    border-radius: 100%;
    -webkit-transition: background .2s;
    transition: background .2s;
}

.rodal-close:before {
    -webkit-transform: rotate(45deg);
    transform: rotate(45deg);
}

.rodal-close:after {
    -webkit-transform: rotate(-45deg);
    transform: rotate(-45deg);
}

.rodal-close:hover:before,
.rodal-close:hover:after {
    background: #333;
}

/* -- fade -- */
/* @-webkit-keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

@keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

.rodal-fade-enter {
    -webkit-animation: rodal-fade-enter both ease-in;
    animation: rodal-fade-enter both ease-in;
} */

@-webkit-keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

@keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

.rodal-fade-leave {
    -webkit-animation: rodal-fade-leave both ease-out;
    animation: rodal-fade-leave both ease-out;
}

/* -- zoom -- */
@-webkit-keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-enter {
    -webkit-animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-leave {
    -webkit-animation: rodal-zoom-leave both;
    animation: rodal-zoom-leave both;
}

/* -- slideDown -- */
@-webkit-keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-enter {
    -webkit-animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-leave {
    -webkit-animation: rodal-slideDown-leave both;
    animation: rodal-slideDown-leave both;
}

/* -- slideLeft -- */
@-webkit-keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-enter {
    -webkit-animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-leave {
    -webkit-animation: rodal-slideLeft-leave both;
    animation: rodal-slideLeft-leave both;
}

/* -- slideRight -- */
@-webkit-keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-enter {
    -webkit-animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-leave {
    -webkit-animation: rodal-slideRight-leave both;
    animation: rodal-slideRight-leave both;
}

/* -- slideUp -- */
@-webkit-keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-enter {
    -webkit-animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
    animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
}

@-webkit-keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-leave {
    -webkit-animation: rodal-slideUp-leave both;
    animation: rodal-slideUp-leave both;
}

/* -- flip -- */
@-webkit-keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

@keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

.rodal-flip-enter {
    -webkit-animation: rodal-flip-enter both ease-in;
    animation: rodal-flip-enter both ease-in;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

@-webkit-keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

@keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

.rodal-flip-leave {
    -webkit-animation: rodal-flip-leave both;
    animation: rodal-flip-leave both;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

/* -- rotate -- */
@-webkit-keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-enter {
    -webkit-animation: rodal-rotate-enter both;
    animation: rodal-rotate-enter both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

@-webkit-keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-leave {
    -webkit-animation: rodal-rotate-leave both;
    animation: rodal-rotate-leave both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

/* -- door -- */
@-webkit-keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

@keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

.rodal-door-enter {
    -webkit-animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

@keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

.rodal-door-leave {
    -webkit-animation: rodal-door-leave both;
    animation: rodal-door-leave both;
}</style></head><body class=""><!-- BEGIN WAYBACK TOOLBAR INSERT -->
<style type="text/css">
body {
  margin-top:0 !important;
  padding-top:0 !important;
  /*min-width:800px !important;*/
}
</style>
<script>__wm.rw(0);</script>
<div id="wm-ipp-base" lang="en" style="display: block; direction: ltr;">
</div><div id="wm-ipp-print">The Wayback Machine - https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/d10275.htm</div>
<script type="text/javascript">//<![CDATA[
__wm.bt(675,27,25,2,"web","https://baike.sogou.com/kexue/d10275.htm","20221028214411",1996,"/_static/",["/_static/css/banner-styles.css?v=S1zqJCYt","/_static/css/iconochive.css?v=qtvMKcIJ"], false);
  __wm.rw(1);
//]]></script>
<!-- END WAYBACK TOOLBAR INSERT --><script>window._gtag=window._gtag||{};window._gtag.shouldGrayed = false;if ('9d7693345b1b4c5bb16b927cdbb8187e') window._gtag.traceId = '9d7693345b1b4c5bb16b927cdbb8187e';if ({"illegality":true}) window.userInfo = {"illegality":true};</script><div class="topnavbox"><ul class="topnav"><li><a href="https://web.archive.org/web/20221028214411/https://www.sogou.com/web?query=">网页</a></li><li><a href="https://web.archive.org/web/20221028214411/https://weixin.sogou.com/weixin?p=75351201">微信</a></li><li><a href="https://web.archive.org/web/20221028214411/https://zhihu.sogou.com/zhihu?p=75351218">知乎</a></li><li><a href="https://web.archive.org/web/20221028214411/https://pic.sogou.com/pics?query=">图片</a></li><li><a href="https://web.archive.org/web/20221028214411/https://v.sogou.com/v?query=">视频</a></li><li><a href="https://web.archive.org/web/20221028214411/https://mingyi.sogou.com/">医疗</a></li><li class="cur"><strong>科学</strong></li><li><a href="https://web.archive.org/web/20221028214411/https://hanyu.sogou.com/">汉语</a></li><li><a href="https://web.archive.org/web/20221028214411/https://wenwen.sogou.com/">问问</a></li><li><a href="https://web.archive.org/web/20221028214411/https://www.sogou.com/docs/more.htm">更多<span class="topraquo">»</span></a></li></ul></div><div id="header"><div class="header-wrap"><a class="header-logo" href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue"></a><div class="header-search"><div class="querybox" id="suggBox"><form><input id="searchInput" class="query" type="text" placeholder="搜科学领域专业百科词条" name="query" autocomplete="off" value=""><a href="javascript:;" class="query-search"></a></form></div></div><div class="header-rgt"><span class="btn-header-rgt btn-edit" id="editLemma">创建</span><div class="header-user no-login"></div></div></div></div><div class="fixed-placeholder" style="visibility:none"></div><div id="container" class=""><div class="content lemma-level1"><div class="detail-title" id="abstract-title"><h1>单核苷酸多态性</h1><a href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/d10275.htm#!" class="detail-edit">编辑</a></div><div class="section_content" data-id="54127453290947095"><div class="text_img ed_imgfloat_right"><a class="ed_image_link" data-src="https://img01.sogoucdn.com/app/a/200698/sogou_science_3230" data-bigsrc="https://img01.sogoucdn.com/app/a/200698/sogou_science_3230?width=231&amp;height=174&amp;titlename=%E4%B8%8A%E9%83%A8%E7%9A%84DNA%E5%88%86%E5%AD%90%E5%92%8C%E4%B8%8B%E9%83%A8%E7%9A%84DNA%E5%88%86%E5%AD%90%E5%9C%A8%E5%8D%95%E7%A2%B1%E5%9F%BA%E5%AF%B9%EF%BC%88%E4%B8%80%E7%A7%8DC/A%E7%9A%84%E5%A4%9A%E6%80%81%E6%80%A7%EF%BC%89%E7%9A%84%E4%BD%8D%E7%BD%AE%E5%85%B7%E6%9C%89%E5%B7%AE%E5%BC%82&amp;w=306&amp;h=200" title="点击查看大图" data-w="306" data-h="200" style="background-image:url(https://web.archive.org/web/20221028214411im_/https://img01.sogoucdn.com/app/a/200698/sogou_science_3230)" href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/d10275.htm#!"></a><div class="text_img_title">上部的DNA分子和下部的DNA分子在单碱基对（一种C/A的多态性）的位置具有差异</div></div><div><p>单核苷酸多态性，通常缩写为SNP(/snɪp/；复数/snɪps/)，是发生在基因组特殊位置的单个核苷酸的替换，其中每个变异在群体中存在一定程度(例如&gt; 1%)。<sup><a href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/d10275.htm#quote_1" class="kx_ref">[1]</a></sup> </p>
<p>在人类基因组的特定碱基位置上，C核苷酸可能出现在大多数个体中，但是在少数个体中，该位置被A占据。这意味着在该特定位置上存在单核苷酸多态性，并且两种可能的核苷酸变异——C或A——被称为该位置的等位基因。</p>
<p>单核苷酸多态性预示了我们对多种疾病(如镰状细胞性贫血、β地中海贫血和单核苷酸多态性导致的囊性纤维化)的易感性差异。<sup><a href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/d10275.htm#quote_2" class="kx_ref">[2]</a></sup><sup><a href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/d10275.htm#quote_3" class="kx_ref">[3]</a></sup><sup><a href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/d10275.htm#quote_4" class="kx_ref">[4]</a></sup> 疾病的严重性和身体对治疗的反应也是遗传变异的表现。例如，APOE(载脂蛋白E)基因的单碱基突变与阿尔茨海默病的风险较低相关。<sup><a href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/d10275.htm#quote_5" class="kx_ref">[5]</a></sup> </p>
<p>单核苷酸变体(SNV)是单核苷酸的变体，没有任何频率限制，可能出现在体细胞中。体细胞单核苷酸变异(比如，由癌症引起)也可以称为单核苷酸变异。</p></div></div><div id="catalog"><h2 class="title2">目录<a href="javascript:" class="detail-edit">编辑</a></h2><div class="catalog_wrap" style=""><ul class="catalog_list col3"><li><span class="order">1</span><a href="javascript:" data-level="1" data-id="14995084281905671">类型</a></li><li><span class="order">2</span><a href="javascript:" data-level="1" data-id="14995084281905672">应用</a></li><li><span class="order">3</span><a href="javascript:" data-level="1" data-id="14995084281905673">频率</a></li><li class="secondary_catalog"><span>3.1 </span><a href="javascript:" data-id="14995084281905673">在基因组中</a></li><li class="secondary_catalog"><span>3.2 </span><a href="javascript:" data-id="14995084281905673">在人群中</a></li><li><span class="order">4</span><a href="javascript:" data-level="1" data-id="14995084281905674">重要性</a></li></ul><ul class="catalog_list col3"><li class="secondary_catalog"><span>4.1 </span><a href="javascript:" data-id="14995084281905674">临床研究</a></li><li class="secondary_catalog"><span>4.2 </span><a href="javascript:" data-id="14995084281905674">法医学</a></li><li class="secondary_catalog"><span>4.3 </span><a href="javascript:" data-id="14995084281905674">遗传药理学</a></li><li class="secondary_catalog"><span>4.4 </span><a href="javascript:" data-id="14995084281905674">疾病</a></li><li><span class="order">5</span><a href="javascript:" data-level="1" data-id="14995084281905675">例子</a></li><li><span class="order">6</span><a href="javascript:" data-level="1" data-id="14995084298682891">数据库</a></li></ul><ul class="catalog_list col3"><li><span class="order">7</span><a href="javascript:" data-level="1" data-id="14995084298682892">命名法</a></li><li><span class="order">8</span><a href="javascript:" data-level="1" data-id="14995084298682893">单核苷酸多态性分析</a></li><li><span class="order">9</span><a href="javascript:" data-level="1" data-id="14995084298682894">预测单核苷酸多态性效应的程序</a></li><li><span class="order">10</span><a href="javascript:" data-level="1" data-id="references">参考文献</a></li></ul></div></div><div id="paragraphs"><div><div id="par_14995084281905671"><h2 class="title">1 类型<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>
 </p><table class="infobox">
  <tbody>
   <tr>
    <th>Types of SNPs</th>
   </tr>
   <tr>
    <td>
     <ul>
      <li>Non-coding region</li>
      <li>Coding region
       <ul>
        <li>Synonymous</li>
        <li>Nonsynonymous
         <ul>
          <li>Missense</li>
          <li>Nonsense</li>
         </ul></li>
       </ul></li>
     </ul></td>
   </tr>
  </tbody>
 </table><p></p><p></p><div class="text_img ed_imgfloat_right">
            <a class="ed_image_link lazyLoad" data-src="https://img01.sogoucdn.com/app/a/200698/sogou_science_3231" data-bigsrc="" title="点击查看大图" href="javascript:" data-observer="true"></a>
            <div class="text_img_title" style="text-align: center;">单核苷酸多态性(SNPs)的种类</div>   
        </div> <p></p><p>单核苷酸多态性可能属于基因编码序列、基因非编码区或基因间区(基因间区)。由于遗传密码的简并性，编码序列中的单核苷酸多态性不一定改变产生的蛋白质的氨基酸序列。</p><p>编码区的SNPs有两种类型:同义和非同义SNPs。同义SNPs不影响蛋白质序列，而非同义SNPs改变蛋白质的氨基酸序列。非同义SNPs有两种类型:错义和无意义。</p><p>不在蛋白质编码区的单核苷酸多态性仍可能影响基因剪接、转录因子结合、信使核糖核酸降解或非编码核糖核酸序列。受这种类型的单核苷酸多态性影响的基因表达被称为eSNP(表达单核苷酸多态性)，可以位于基因的上游或下游。</p></div></div><div id="par_14995084281905672"><h2 class="title">2 应用<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>
 </p><ul>
  <li>关联研究可以确定基因变异是否与疾病或性状相关。<sup><a href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/d10275.htm#quote_6" class="kx_ref">[6]</a></sup></li>
  <li>标签单核苷酸多态性(tag SNP)是基因组中一个具有高度连锁不平衡(两个或多个位点等位基因的非随机关联)的区域的代表性单核苷酸多态性。标签单核苷酸多态性在全基因组单核苷酸多态性关联研究中是有用的，在全基因组单核苷酸多态性关联研究中，数十万个单核苷酸多态性是基因分型的。</li>
  <li>单体型映射:等位基因或脱氧核糖核酸序列的集合可以被聚集，这样一个单核苷酸多态性可以识别许多连接的单核苷酸多态性。</li>
  <li>连锁不平衡是群体遗传学中的一个术语，指等位基因在两个或多个位点的非随机关联，不一定在同一条染色体上。它是指基因组中紧密相连的单核苷酸多态性等位基因或脱氧核糖核酸序列倾向于一起遗传的现象。LD受两个参数的影响:1)单核苷酸多态性之间的距离。2)重组率。<sup><a href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/d10275.htm#quote_7" class="kx_ref">[7]</a></sup></li>
 </ul><p></p></div></div><div id="par_14995084281905673"><h2 class="title">3 频率<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>在不同人群中发现了超过8400万个单核苷酸多态性。一个典型的基因组在400万到500万个位点上不同于参考人类基因组，其中大部分(超过99.9%)由单核苷酸多态性和短序列组成。<sup><a href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/d10275.htm#quote_8" class="kx_ref">[8]</a></sup> </p><p></p><h3>3.1 <span>在基因组中</span></h3><p></p><p>单核苷酸多态性的基因组分布不均匀；单核苷酸多态性出现在非编码区的频率高于编码区，或者一般来说，在自然选择起作用并“固定”构成最有利遗传适应的单核苷酸多态性等位基因(消除其他变体)的地方。<sup><a href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/d10275.htm#quote_9" class="kx_ref">[9]</a></sup> 其他因素，如基因重组和突变率，也可以决定单核苷酸多态性密度。<sup><a href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/d10275.htm#quote_10" class="kx_ref">[10]</a></sup> </p><p>单核苷酸多态性密度可以通过微卫星的存在来预测:特别是腺苷酸微卫星是单核苷酸多态性密度的有效预测因子，长的（AT）重复片段通常被发现存在在单核苷酸多态性频率显著较低和GC含量较低的区域。<sup><a href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/d10275.htm#quote_11" class="kx_ref">[11]</a></sup> </p><p></p><h3>3.2 <span>在人群中</span></h3><p></p><p>人类群体之间存在差异，因此在一个地理或种族群体中常见的单核苷酸多态性等位基因在另一个群体中可能要罕见得多。在一个群体中，单核苷酸多态性可以被赋予一个较低的等位基因频率——在特定群体中观察到的某一位点的最低等位基因频率。这只是单核苷酸多态性的两个等位基因频率中较小的一个。</p></div></div><div id="par_14995084281905674"><h2 class="title">4 重要性<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>人类基因序列的变化会影响人类疾病的发生以及对病原体、化学物质、药物、疫苗和其他药剂的反应。单核苷酸多态性对个性化医学也至关重要。<sup><a href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/d10275.htm#quote_12" class="kx_ref">[12]</a></sup> 例子包括生物医学研究、法医学、药物遗传学和疾病成因，如下所述。</p><p></p><h3>4.1 <span>临床研究</span></h3><p></p><p>单核苷酸多态性在临床研究中最重要的是在全基因组关联研究中比较不同队列(如有疾病和无疾病的匹配队列)之间的基因组区域。单核苷酸多态性已经在全基因组关联研究中用作与疾病或正常性状相关的基因定位的高分辨率标记。对表型没有明显影响的单核苷酸多态性(所谓的沉默突变)在全基因组关联研究中仍然是有用的遗传标记，因为它们的数量和世代间的稳定遗传。</p><p></p><h3>4.2 <span>法医学</span></h3><p></p><p>单核苷酸多态性最初用于将法医DNA样本与嫌疑人进行匹配，但随着基于串联重复序列的脱氧核糖核酸指纹技术的发展，它已经被淘汰。当前的新一代测序(NGS)技术可能允许在法医应用中更好地使用单核苷酸多态性基因分型，只要避免有问题的基因座。<sup><a href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/d10275.htm#quote_13" class="kx_ref">[13]</a></sup> 在未来，SNPs可能被用于一些表型线索的取证，如眼睛颜色、头发颜色、种族等。Kidd等人已经证明，一个由19个SNPs组成的小组可以在所研究的40个人群中识别出匹配概率很高的种族群体(Pm = 10^-7)。<sup><a href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/d10275.htm#quote_14" class="kx_ref">[14]</a></sup> 这可能有潜在用途的一个例子是对未知个体骨骼化遗骸可能的预解剖外观进行艺术重建。虽然严格地基于人类学特征，面部重建可以相当精确，但是其他包括眼睛颜色、皮肤颜色、头发颜色等数据可能会让重建更精确。</p><p>在法医样本或降解样本数量较少的情况下，由于潜在标记物的丰富性、对自动化的适应性以及所需片段长度可能减少到仅60-80 bp，单核苷酸多态性方法可能是STR方法的良好替代方法。在脱氧核糖核酸图谱数据库中没有字符串匹配的情况下；不同的单核苷酸多态性可以用来获得关于种族、表现型、谱系甚至身份的线索。</p><p></p><h3>4.3 <span>遗传药理学</span></h3><p></p><p>一些单核苷酸多态性与不同药物的代谢有关。<sup><a href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/d10275.htm#quote_15" class="kx_ref">[15]</a></sup><sup><a href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/d10275.htm#quote_16" class="kx_ref">[16]</a></sup><sup><a href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/d10275.htm#quote_17" class="kx_ref">[17]</a></sup> 癌症、传染病(艾滋病、麻风病、肝炎等)等多种人类疾病的关联。)具有不同SNPs的自身免疫性、神经精神性和许多其他疾病可以作为药物治疗的相关药物基因组靶标。<sup><a href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/d10275.htm#quote_18" class="kx_ref">[18]</a></sup> </p><p></p><h3>4.4 <span>疾病</span></h3><p></p><p>单个单核苷酸多态性可能导致孟德尔病，尽管对于复杂的疾病，单核苷酸多态性通常不单独发挥作用，相反，它们与其他单核苷酸多态性协同工作，表现出骨质疏松症的疾病状态。<sup><a href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/d10275.htm#quote_19" class="kx_ref">[19]</a></sup> 该领域最早的成功之一是在APOC3(载脂蛋白C3基因)的非编码区发现了一个单碱基突变，该突变与高甘油三酯血症和动脉粥样硬化的高风险相关。<sup><a href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/d10275.htm#quote_20" class="kx_ref">[20]</a></sup> </p><p>所有类型的单核苷酸多态性都可能具有可观察到的表型或导致疾病:</p><p>
 </p><ul>
  <li>非编码区的单核苷酸多态性可能表现出更高的癌症风险，<sup><a href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/d10275.htm#quote_21" class="kx_ref">[21]</a></sup> 并可能影响基因结构和疾病易感性。<sup><a href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/d10275.htm#quote_22" class="kx_ref">[22]</a></sup> 非编码SNPs也可以改变基因的表达水平，如eQTL(表达数量性状基因座)。</li>
  <li>编码区的单核苷酸多态性:
   <ul>
    <li>顾名思义，同义替换不会导致蛋白质中氨基酸的变化，但仍能以其他方式影响其功能。一个例子是多药耐药基因1 (MDR1)中看似沉默的突变，其编码从细胞中排出药物的细胞膜泵，可以减缓翻译并允许肽链折叠成不寻常的构象，导致突变泵功能减弱(在MDR1蛋白中，例如，C1236T多态性将多肽氨基酸位置412的GGC密码子改变为GGT(均编码甘氨酸)，C3435T多态性将ATC改变为位置1145的ATT(均编码异亮氨酸))。<sup><a href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/d10275.htm#quote_23" class="kx_ref">[23]</a></sup></li>
    <li>非同步替换:
     <ul>
      <li>错义——单个碱基的变化导致蛋白质氨基酸的变化及其导致疾病的功能障碍(例如，LMNA基因中的1580G&gt;T单核苷酸多态性——脱氧核糖核酸序列中的1580位(CGT密码子)导致鸟嘌呤被胸腺嘧啶取代，在脱氧核糖核酸序列中产生CTT密码子，在蛋白质水平上导致527位的亮氨酸取代精氨酸，<sup><a href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/d10275.htm#quote_24" class="kx_ref">[24]</a></sup> 在表型水平上这表现为重叠的下颌骨发育不良和早衰综合征)</li>
      <li>无意义——在脱氧核糖核酸序列中的点突变，导致转录的脱氧核糖核酸中的过早终止密码子或无意义密码子，以及截短的、不完整的、通常无功能的蛋白质产物(如囊性纤维化跨膜传导调节基因G542X突变引起的囊性纤维化)。<sup><a href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/d10275.htm#quote_25" class="kx_ref">[25]</a></sup></li>
     </ul></li>
   </ul></li>
 </ul><p></p></div></div><div id="par_14995084281905675"><h2 class="title">5 例子<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>
 </p><ul>
  <li>rs6311和rs6313是人类染色体13上血清素5-HT2A受体基因的单核苷酸多态性。<sup><a href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/d10275.htm#quote_26" class="kx_ref">[26]</a></sup></li>
  <li>F5基因中的单核苷酸多态性导致莱顿因子血栓形成倾向。<sup><a href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/d10275.htm#quote_27" class="kx_ref">[27]</a></sup></li>
  <li>rs3091244是人类染色体1上CRP基因中三链单核苷酸多态性的一个例子。<sup><a href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/d10275.htm#quote_28" class="kx_ref">[28]</a></sup></li>
  <li>TAS2R38编码PTC品尝能力，并包含6个注释的SNPs。<sup><a href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/d10275.htm#quote_29" class="kx_ref">[29]</a></sup></li>
  <li>编码无花果蛋白酶的FCN1基因中的rs148649884和rs138055828削弱了重组无花果蛋白酶的配体结合能力。<sup><a href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/d10275.htm#quote_30" class="kx_ref">[30]</a></sup></li>
  <li>脱氧核糖核酸错配修复基因PMS2 (rs1059060，Ser775Asn)的内含子单核苷酸多态性与精子脱氧核糖核酸损伤增加和男性不育风险有关。<sup><a href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/d10275.htm#quote_31" class="kx_ref">[31]</a></sup></li>
 </ul><p></p></div></div><div id="par_14995084298682891"><h2 class="title">6 数据库<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>正如基因一样，SNPs也有生物信息学数据库。</p><p>
 </p><ul>
  <li><i>dbSNP</i> 数据库是国家生物技术信息中心(NCBI)的SNP数据库。截至2015年6月8日，数据库单核苷酸多态性列出了149，735，377个人类单核苷酸多态性。<sup><a href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/d10275.htm#quote_32" class="kx_ref">[32]</a></sup><sup><a href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/d10275.htm#quote_33" class="kx_ref">[33]</a></sup></li>
  <li><i>Kaviar</i><sup><a href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/d10275.htm#quote_34" class="kx_ref">[34]</a></sup> 是来自多个数据源的SNP概要，包括dbSNP。</li>
  <li><i>SNPedia</i> 是一个维基风格的数据库，支持个人基因组注释、解释和分析。</li>
  <li><i>OMIM</i> 数据库描述多态性和疾病之间的关联(例如，以文本形式给出疾病)。</li>
  <li>dbSap——用于蛋白质变异检测的单一氨基酸多态性数据库。<sup><a href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/d10275.htm#quote_35" class="kx_ref">[35]</a></sup></li>
  <li>人类基因突变数据库提供导致或与人类遗传疾病和功能性相关的 &nbsp; 单核苷酸多态性相关的基因突变。</li>
  <li>国际单倍体项目，研究人员正在鉴定标签单核苷酸多态性，以便能够确定每个受试者中存在的单倍体集合。</li>
  <li>GWAS中心允许用户在一个或多个全基因组关联研究中直观地询问实际的汇总级关联数据。</li>
 </ul><p></p><p>国际单核苷酸多态性图谱工作组通过与基因库中插入大克隆片段的基因组序列比对，绘制了每个单核苷酸多态性侧翼的序列。这些比对被转换成染色体坐标，如表1所示。这个列表已经大大增加了，例如，卡维亚数据库现在列出了1.62亿个单核苷酸变异体。</p><p>
 </p><table class="wikitable">
  <tbody>
   <tr>
    <th>Chromosome</th>
    <th>Length(bp)</th>
    <th>All SNPs</th>
    <th></th>
    <th>TSC SNPs</th>
    <th></th>
   </tr>
   <tr>
    <td></td>
    <td></td>
    <td>Total SNPs</td>
    <td>kb per SNP</td>
    <td>Total SNPs</td>
    <td>kb per SNP</td>
   </tr>
   <tr>
    <td>1</td>
    <td>214,066,000</td>
    <td>129,931</td>
    <td>1.65</td>
    <td>75,166</td>
    <td>2.85</td>
   </tr>
   <tr>
    <td>2</td>
    <td>222,889,000</td>
    <td>103,664</td>
    <td>2.15</td>
    <td>76,985</td>
    <td>2.90</td>
   </tr>
   <tr>
    <td>3</td>
    <td>186,938,000</td>
    <td>93,140</td>
    <td>2.01</td>
    <td>63,669</td>
    <td>2.94</td>
   </tr>
   <tr>
    <td>4</td>
    <td>169,035,000</td>
    <td>84,426</td>
    <td>2.00</td>
    <td>65,719</td>
    <td>2.57</td>
   </tr>
   <tr>
    <td>5</td>
    <td>170,954,000</td>
    <td>117,882</td>
    <td>1.45</td>
    <td>63,545</td>
    <td>2.69</td>
   </tr>
   <tr>
    <td>6</td>
    <td>165,022,000</td>
    <td>96,317</td>
    <td>1.71</td>
    <td>53,797</td>
    <td>3.07</td>
   </tr>
   <tr>
    <td>7</td>
    <td>149,414,000</td>
    <td>71,752</td>
    <td>2.08</td>
    <td>42,327</td>
    <td>3.53</td>
   </tr>
   <tr>
    <td>8</td>
    <td>125,148,000</td>
    <td>57,834</td>
    <td>2.16</td>
    <td>42,653</td>
    <td>2.93</td>
   </tr>
   <tr>
    <td>9</td>
    <td>107,440,000</td>
    <td>62,013</td>
    <td>1.73</td>
    <td>43,020</td>
    <td>2.50</td>
   </tr>
   <tr>
    <td>10</td>
    <td>127,894,000</td>
    <td>61,298</td>
    <td>2.09</td>
    <td>42,466</td>
    <td>3.01</td>
   </tr>
   <tr>
    <td>11</td>
    <td>129,193,000</td>
    <td>84,663</td>
    <td>1.53</td>
    <td>47,621</td>
    <td>2.71</td>
   </tr>
   <tr>
    <td>12</td>
    <td>125,198,000</td>
    <td>59,245</td>
    <td>2.11</td>
    <td>38,136</td>
    <td>3.28</td>
   </tr>
   <tr>
    <td>13</td>
    <td>93,711,000</td>
    <td>53,093</td>
    <td>1.77</td>
    <td>35,745</td>
    <td>2.62</td>
   </tr>
   <tr>
    <td>14</td>
    <td>89,344,000</td>
    <td>44,112</td>
    <td>2.03</td>
    <td>29,746</td>
    <td>3.00</td>
   </tr>
   <tr>
    <td>15</td>
    <td>73,467,000</td>
    <td>37,814</td>
    <td>1.94</td>
    <td>26,524</td>
    <td>2.77</td>
   </tr>
   <tr>
    <td>16</td>
    <td>74,037,000</td>
    <td>38,735</td>
    <td>1.91</td>
    <td>23,328</td>
    <td>3.17</td>
   </tr>
   <tr>
    <td>17</td>
    <td>73,367,000</td>
    <td>34,621</td>
    <td>2.12</td>
    <td>19,396</td>
    <td>3.78</td>
   </tr>
   <tr>
    <td>18</td>
    <td>73,078,000</td>
    <td>45,135</td>
    <td>1.62</td>
    <td>27,028</td>
    <td>2.70</td>
   </tr>
   <tr>
    <td>19</td>
    <td>56,044,000</td>
    <td>25,676</td>
    <td>2.18</td>
    <td>11,185</td>
    <td>5.01</td>
   </tr>
   <tr>
    <td>20</td>
    <td>63,317,000</td>
    <td>29,478</td>
    <td>2.15</td>
    <td>17,051</td>
    <td>3.71</td>
   </tr>
   <tr>
    <td>21</td>
    <td>33,824,000</td>
    <td>20,916</td>
    <td>1.62</td>
    <td>9,103</td>
    <td>3.72</td>
   </tr>
   <tr>
    <td>22</td>
    <td>33,786,000</td>
    <td>28,410</td>
    <td>1.19</td>
    <td>11,056</td>
    <td>3.06</td>
   </tr>
   <tr>
    <td>X</td>
    <td>131,245,000</td>
    <td>34,842</td>
    <td>3.77</td>
    <td>20,400</td>
    <td>6.43</td>
   </tr>
   <tr>
    <td>Y</td>
    <td>21,753,000</td>
    <td>4,193</td>
    <td>5.19</td>
    <td>1,784</td>
    <td>12.19</td>
   </tr>
   <tr>
    <td>RefSeq</td>
    <td>15,696,674</td>
    <td>14,534</td>
    <td>1.08</td>
   </tr>
   <tr>
    <td>Totals</td>
    <td>2,710,164,000</td>
    <td>1,419,190</td>
    <td>1.91</td>
    <td>887,450</td>
    <td>3.05</td>
   </tr>
  </tbody>
 </table><p></p></div></div><div id="par_14995084298682892"><h2 class="title">7 命名法<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>单核苷酸多态性的命名可能令人困惑:单个单核苷酸多态性可能存在几种变异，但尚未达成共识。一种表示单核苷酸多态性的方法是写带有前缀、句点和“大于”符号，显示野生型和改变的核苷酸或氨基酸；例如，c.76A&gt;T.<sup><a href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/d10275.htm#quote_36" class="kx_ref">[36]</a></sup><sup><a href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/d10275.htm#quote_37" class="kx_ref">[37]</a></sup><sup><a href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/d10275.htm#quote_38" class="kx_ref">[38]</a></sup> SNPs经常用它们的dbSNP rs数来表示。</p></div></div><div id="par_14995084298682893"><h2 class="title">8 单核苷酸多态性分析<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>单核苷酸多态性通常是双等位基因，因此很容易分析。<sup><a href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/d10275.htm#quote_39" class="kx_ref">[39]</a></sup> 发现新的单核苷酸多态性和检测已知单核苷酸多态性的分析方法包括:</p><p>
 </p><ul>
  <li>脱氧核糖核酸测序；<sup><a href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/d10275.htm#quote_40" class="kx_ref">[40]</a></sup></li>
  <li>毛细管电泳；<sup><a href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/d10275.htm#quote_41" class="kx_ref">[41]</a></sup></li>
  <li>质谱分析；<sup><a href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/d10275.htm#quote_42" class="kx_ref">[42]</a></sup></li>
  <li>单链构象多态性(SSCP)；<sup><a href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/d10275.htm#quote_43" class="kx_ref">[43]</a></sup></li>
  <li>单碱基延伸；</li>
  <li>电化学分析；</li>
  <li>变性高效液相色谱和凝胶电泳；</li>
  <li>限制性片段长度多态性；</li>
  <li>杂交分析；</li>
 </ul><p></p></div></div><div id="par_14995084298682894"><h2 class="title">9 预测单核苷酸多态性效应的程序<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>单核苷酸多态性中比较重要的一类是那些与导致蛋白质水平氨基酸变化的错义突变相对应的多态性。特定残基的点突变对蛋白质功能有不同的影响(从无影响到完全破坏其功能)。通常，具有相似大小和物理化学性质的氨基酸改变(例如亮氨酸被缬氨酸取代)具有温和的效果，反之亦然。类似地，如果单核苷酸多态性破坏二级结构元件(例如α螺旋区脯氨酸被取代)，这种突变通常会影响整个蛋白质的结构和功能。利用这些简单的和许多其他机器学习导出的规则，开发了一组预测单核苷酸多态性效应的程序:</p><p>
 </p><ul>
  <li>SIFT 这个项目基于氨基酸的物理性质和序列同源性，提供了实验室诱导的错义或非同源突变如何影响蛋白质功能的见解。</li>
  <li>LIST<sup><a href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/d10275.htm#quote_44" class="kx_ref">[44]</a></sup> (局部同一性和共享分类群) 估计突变所导致的由蛋白质功能的改变的所产生的潜在有害。这是基于这样一个假设，即与人类密切相关的物种中观察到的变异在评估潜在危害时，比那些远亲物种中的变异更为重要。</li>
  <li>SNAP2</li>
  <li>SuSPect</li>
  <li>PolyPhen-2</li>
  <li>PredictSNP</li>
  <li>突变网站:官方网站 official website</li>
  <li>Variant Effect Predictor 恩塞布勒项目中的变量效应预测器</li>
  <li>SNPViz<sup><a href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/d10275.htm#quote_45" class="kx_ref">[45]</a></sup> 该程序提供了受影响蛋白质的三维表示，突出了氨基酸的变化，这样医生就可以确定突变蛋白质的致病性。</li>
 </ul><p></p></div></div></div></div><div id="references"><h2 class="title" id="par_references">参考文献</h2><ul class="references"><li id="quote_1"><span class="references-num">[1]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"single-nucleotide polymorphism / SNP | Learn Science at Scitable". www.nature.com. Archived from the original on 2015-11-10. Retrieved 2015-11-13..</span></p></li><li id="quote_2"><span class="references-num">[2]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Ingram VM (October 1956). "A specific chemical difference between the globins of normal human and sickle-cell anaemia haemoglobin". Nature. 178 (4537): 792–4. doi:10.1038/178792a0. PMID 13369537..</span></p></li><li id="quote_3"><span class="references-num">[3]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Chang JC, Kan YW (June 1979). "beta 0 thalassemia, a nonsense mutation in man". Proceedings of the National Academy of Sciences of the United States of America. 76 (6): 2886–9. doi:10.1073/pnas.76.6.2886. PMC 383714. PMID 88735..</span></p></li><li id="quote_4"><span class="references-num">[4]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Hamosh A, King TM, Rosenstein BJ, Corey M, Levison H, Durie P, Tsui LC, McIntosh I, Keston M, Brock DJ (August 1992). "Cystic fibrosis patients bearing both the common missense mutation Gly----Asp at codon 551 and the delta F508 mutation are clinically indistinguishable from delta F508 homozygotes, except for decreased risk of meconium ileus". American Journal of Human Genetics. 51 (2): 245–50. PMC 1682672. PMID 1379413..</span></p></li><li id="quote_5"><span class="references-num">[5]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Wolf AB, Caselli RJ, Reiman EM, Valla J (April 2013). "APOE and neuroenergetics: an emerging paradigm in Alzheimer's disease". Neurobiology of Aging. 34 (4): 1007–17. doi:10.1016/j.neurobiolaging.2012.10.011. PMC 3545040. PMID 23159550..</span></p></li><li id="quote_6"><span class="references-num">[6]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Zhang K, Qin ZS, Liu JS, Chen T, Waterman MS, Sun F (May 2004). "Haplotype block partitioning and tag SNP selection using genotype data and their applications to association studies". Genome Research. 14 (5): 908–16. doi:10.1101/gr.1837404. PMC 479119. PMID 15078859. Archived from the original on 8 May 2018..</span></p></li><li id="quote_7"><span class="references-num">[7]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Gupta PK, Roy JK, Prasad M (25 February 2001). "Single nucleotide polymorphisms: a new paradigm for molecular marker technology and DNA polymorphism detection with emphasis on their use in plants". Current Science. 80 (4): 524–535. Archived from the original on 13 February 2017..</span></p></li><li id="quote_8"><span class="references-num">[8]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Auton A, Brooks LD, Durbin RM, Garrison EP, Kang HM, Korbel JO, Marchini JL, McCarthy S, McVean GA, Abecasis GR (October 2015). "A global reference for human genetic variation". Nature. 526 (7571): 68–74. doi:10.1038/nature15393. PMC 4750478. PMID 26432245..</span></p></li><li id="quote_9"><span class="references-num">[9]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Barreiro LB, Laval G, Quach H, Patin E, Quintana-Murci L (March 2008). "Natural selection has driven population differentiation in modern humans". Nature Genetics. 40 (3): 340–5. doi:10.1038/ng.78. PMID 18246066..</span></p></li><li id="quote_10"><span class="references-num">[10]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Nachman MW (September 2001). "Single nucleotide polymorphisms and recombination rate in humans". Trends in Genetics. 17 (9): 481–5. doi:10.1016/S0168-9525(01)02409-X. PMID 11525814..</span></p></li><li id="quote_11"><span class="references-num">[11]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Varela MA, Amos W (March 2010). "Heterogeneous distribution of SNPs in the human genome: microsatellites as predictors of nucleotide diversity and divergence". Genomics. 95 (3): 151–9. doi:10.1016/j.ygeno.2009.12.003. PMID 20026267..</span></p></li><li id="quote_12"><span class="references-num">[12]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Carlson, Bruce (15 June 2008). "SNPs — A Shortcut to Personalized Medicine". Genetic Engineering &amp; Biotechnology News. Mary Ann Liebert, Inc. 28 (12). Archived from the original on 26 December 2010. Retrieved 2008-07-06. (subtitle) Medical applications are where the market's growth is expected.</span></p></li><li id="quote_13"><span class="references-num">[13]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Cornelis S, Gansemans Y, Deleye L, Deforce D, Van Nieuwerburgh F (February 2017). "Forensic SNP Genotyping using Nanopore MinION Sequencing". Scientific Reports. 7: 41759. doi:10.1038/srep41759. PMC 5290523. PMID 28155888..</span></p></li><li id="quote_14"><span class="references-num">[14]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Kidd KK, Pakstis AJ, Speed WC, Grigorenko EL, Kajuna SL, Karoma NJ, Kungulilo S, Kim JJ, Lu RB, Odunsi A, Okonofua F, Parnas J, Schulz LO, Zhukova OV, Kidd JR (December 2006). "Developing a SNP panel for forensic identification of individuals". Forensic Science International. 164 (1): 20–32. doi:10.1016/j.forsciint.2005.11.017. PMID 16360294..</span></p></li><li id="quote_15"><span class="references-num">[15]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Goldstein JA (October 2001). "Clinical relevance of genetic polymorphisms in the human CYP2C subfamily". British Journal of Clinical Pharmacology. 52 (4): 349–55. doi:10.1046/j.0306-5251.2001.01499.x. PMC 2014584. PMID 11678778..</span></p></li><li id="quote_16"><span class="references-num">[16]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Lee CR (July–August 2004). "CYP2C9 genotype as a predictor of drug disposition in humans". Methods and Findings in Experimental and Clinical Pharmacology. 26 (6): 463–72. PMID 15349140..</span></p></li><li id="quote_17"><span class="references-num">[17]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Yanase K, Tsukahara S, Mitsuhashi J, Sugimoto Y (March 2006). "Functional SNPs of the breast cancer resistance protein-therapeutic effects and inhibitor development". Cancer Letters. 234 (1): 73–80. doi:10.1016/j.canlet.2005.04.039. PMID 16303243..</span></p></li><li id="quote_18"><span class="references-num">[18]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Fareed M, Afzal M (April 2013). "Single-nucleotide polymorphism in genome-wide association of human population: A tool for broad spectrum service". Egyptian Journal of Medical Human Genetics. 14 (2): 123–134. doi:10.1016/j.ejmhg.2012.08.001..</span></p></li><li id="quote_19"><span class="references-num">[19]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Singh M, Singh P, Juneja PK, Singh S, Kaur T (March 2011). "SNP-SNP interactions within APOE gene influence plasma lipids in postmenopausal osteoporosis". Rheumatology International. 31 (3): 421–3. doi:10.1007/s00296-010-1449-7. PMID 20340021..</span></p></li><li id="quote_20"><span class="references-num">[20]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Rees A, Shoulders CC, Stocks J, Galton DJ, Baralle FE (February 1983). "DNA polymorphism adjacent to human apoprotein A-1 gene: relation to hypertriglyceridaemia". Lancet. 1 (8322): 444–6. doi:10.1016/S0140-6736(83)91440-X. PMID 6131168..</span></p></li><li id="quote_21"><span class="references-num">[21]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Li G, Pan T, Guo D, Li LC (2014). "Regulatory Variants and Disease: The E-Cadherin -160C/A SNP as an Example". Molecular Biology International. 2014: 967565. doi:10.1155/2014/967565. PMC 4167656. PMID 25276428..</span></p></li><li id="quote_22"><span class="references-num">[22]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Lu YF, Mauger DM, Goldstein DB, Urban TJ, Weeks KM, Bradrick SS (November 2015). "IFNL3 mRNA structure is remodeled by a functional non-coding polymorphism associated with hepatitis C virus clearance". Scientific Reports. 5: 16037. doi:10.1038/srep16037. PMC 4631997. PMID 26531896..</span></p></li><li id="quote_23"><span class="references-num">[23]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Kimchi-Sarfaty C, Oh JM, Kim IW, Sauna ZE, Calcagno AM, Ambudkar SV, Gottesman MM (January 2007). "A "silent" polymorphism in the MDR1 gene changes substrate specificity". Science. 315 (5811): 525–8. doi:10.1126/science.1135308. PMID 17185560..</span></p></li><li id="quote_24"><span class="references-num">[24]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Al-Haggar M, Madej-Pilarczyk A, Kozlowski L, Bujnicki JM, Yahia S, Abdel-Hadi D, Shams A, Ahmad N, Hamed S, Puzianowska-Kuznicka M (November 2012). "A novel homozygous p.Arg527Leu LMNA mutation in two unrelated Egyptian families causes overlapping mandibuloacral dysplasia and progeria syndrome". European Journal of Human Genetics. 20 (11): 1134–40. doi:10.1038/ejhg.2012.77. PMC 3476705. PMID 22549407. Archived from the original on 2015-11-16..</span></p></li><li id="quote_25"><span class="references-num">[25]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Cordovado SK, Hendrix M, Greene CN, Mochal S, Earley MC, Farrell PM, Kharrazi M, Hannon WH, Mueller PW (February 2012). "CFTR mutation analysis and haplotype associations in CF patients". Molecular Genetics and Metabolism. 105 (2): 249–54. doi:10.1016/j.ymgme.2011.10.013. PMC 3551260. PMID 22137130..</span></p></li><li id="quote_26"><span class="references-num">[26]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Giegling I, Hartmann AM, Möller HJ, Rujescu D (November 2006). "Anger- and aggression-related traits are associated with polymorphisms in the 5-HT-2A gene". Journal of Affective Disorders. 96 (1–2): 75–81. doi:10.1016/j.jad.2006.05.016. PMID 16814396..</span></p></li><li id="quote_27"><span class="references-num">[27]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Kujovich JL (January 2011). "Factor V Leiden thrombophilia". Genetics in Medicine. 13 (1): 1–16. doi:10.1097/GIM.0b013e3181faa0f2. PMID 21116184..</span></p></li><li id="quote_28"><span class="references-num">[28]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Morita A, Nakayama T, Doba N, Hinohara S, Mizutani T, Soma M (June 2007). "Genotyping of triallelic SNPs using TaqMan PCR". Molecular and Cellular Probes. 21 (3): 171–6. doi:10.1016/j.mcp.2006.10.005. PMID 17161935..</span></p></li><li id="quote_29"><span class="references-num">[29]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Prodi DA, Drayna D, Forabosco P, Palmas MA, Maestrale GB, Piras D, Pirastu M, Angius A (October 2004). "Bitter taste study in a sardinian genetic isolate supports the association of phenylthiocarbamide sensitivity to the TAS2R38 bitter receptor gene". Chemical Senses. 29 (8): 697–702. doi:10.1093/chemse/bjh074. PMID 15466815..</span></p></li><li id="quote_30"><span class="references-num">[30]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Ammitzbøll CG, Kjær TR, Steffensen R, Stengaard-Pedersen K, Nielsen HJ, Thiel S, Bøgsted M, Jensenius JC (28 November 2012). "Non-synonymous polymorphisms in the FCN1 gene determine ligand-binding ability and serum levels of M-ficolin". PLOS ONE. 7 (11): e50585. doi:10.1371/journal.pone.0050585. PMC 3509001. PMID 23209787. Archived from the original on 7 June 2015..</span></p></li><li id="quote_31"><span class="references-num">[31]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Ji G, Long Y, Zhou Y, Huang C, Gu A, Wang X (May 2012). "Common variants in mismatch repair genes associated with increased risk of sperm DNA damage and male infertility". BMC Medicine. 10: 49. doi:10.1186/1741-7015-10-49. PMC 3378460. PMID 22594646..</span></p></li><li id="quote_32"><span class="references-num">[32]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">National Center for Biotechnology Information, United States National Library of Medicine. 2014. NCBI dbSNP build 142 for human. "Archived copy". Archived from the original on 2017-09-10. Retrieved 2017-09-11.CS1 maint: Archived copy as title (link).</span></p></li><li id="quote_33"><span class="references-num">[33]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">National Center for Biotechnology Information, United States National Library of Medicine. 2015. NCBI dbSNP build 144 for human. Summary Page. "Archived copy". Archived from the original on 2017-09-10. Retrieved 2017-09-11.CS1 maint: Archived copy as title (link).</span></p></li><li id="quote_34"><span class="references-num">[34]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Glusman G, Caballero J, Mauldin DE, Hood L, Roach JC (November 2011). "Kaviar: an accessible system for testing SNV novelty". Bioinformatics. 27 (22): 3216–7. doi:10.1093/bioinformatics/btr540. PMC 3208392. PMID 21965822..</span></p></li><li id="quote_35"><span class="references-num">[35]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Cao R, Shi Y, Chen S, Ma Y, Chen J, Yang J, Chen G, Shi T (January 2017). "dbSAP: single amino-acid polymorphism database for protein variation detection". Nucleic Acids Research. 45 (D1): D827–D832. doi:10.1093/nar/gkw1096. PMC 5210569. PMID 27903894..</span></p></li><li id="quote_36"><span class="references-num">[36]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">J.T. Den Dunnen (2008-02-20). "Recommendations for the description of sequence variants". Human Genome Variation Society. Archived from the original on 2008-09-14. Retrieved 2008-09-05..</span></p></li><li id="quote_37"><span class="references-num">[37]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">den Dunnen JT, Antonarakis SE (2000). "Mutation nomenclature extensions and suggestions to describe complex mutations: a discussion". Human Mutation. 15 (1): 7–12. doi:10.1002/(SICI)1098-1004(200001)15:1&lt;7::AID-HUMU4&gt;3.0.CO;2-N. PMID 10612815..</span></p></li><li id="quote_38"><span class="references-num">[38]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Ogino S, Gulley ML, den Dunnen JT, Wilson RB (February 2007). "Standard mutation nomenclature in molecular diagnostics: practical and educational challenges". The Journal of Molecular Diagnostics. 9 (1): 1–6. doi:10.2353/jmoldx.2007.060081. PMC 1867422. PMID 17251329..</span></p></li><li id="quote_39"><span class="references-num">[39]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Sachidanandam R, Weissman D, Schmidt SC, Kakol JM, Stein LD, Marth G, Sherry S, Mullikin JC, Mortimore BJ, Willey DL, Hunt SE, Cole CG, Coggill PC, Rice CM, Ning Z, Rogers J, Bentley DR, Kwok PY, Mardis ER, Yeh RT, Schultz B, Cook L, Davenport R, Dante M, Fulton L, Hillier L, Waterston RH, McPherson JD, Gilman B, Schaffner S, Van Etten WJ, Reich D, Higgins J, Daly MJ, Blumenstiel B, Baldwin J, Stange-Thomann N, Zody MC, Linton L, Lander ES, Altshuler D (February 2001). "A map of human genome sequence variation containing 1.42 million single nucleotide polymorphisms". Nature. 409 (6822): 928–33. doi:10.1038/35057149. PMID 11237013..</span></p></li><li id="quote_40"><span class="references-num">[40]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Altshuler D, Pollara VJ, Cowles CR, Van Etten WJ, Baldwin J, Linton L, Lander ES (September 2000). "An SNP map of the human genome generated by reduced representation shotgun sequencing". Nature. 407 (6803): 513–6. doi:10.1038/35035083. PMID 11029002..</span></p></li><li id="quote_41"><span class="references-num">[41]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Drabovich AP, Krylov SN (March 2006). "Identification of base pairs in single-nucleotide polymorphisms by MutS protein-mediated capillary electrophoresis". Analytical Chemistry. 78 (6): 2035–8. doi:10.1021/ac0520386. PMID 16536443..</span></p></li><li id="quote_42"><span class="references-num">[42]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Griffin TJ, Smith LM (July 2000). "Genetic identification by mass spectrometric analysis of single-nucleotide polymorphisms: ternary encoding of genotypes". Analytical Chemistry. 72 (14): 3298–302. doi:10.1021/ac991390e. PMID 10939403..</span></p></li><li id="quote_43"><span class="references-num">[43]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Tahira T, Kukita Y, Higasa K, Okazaki Y, Yoshinaga A, Hayashi K (2009). "Estimation of SNP allele frequencies by SSCP analysis of pooled DNA". Methods in Molecular Biology. Methods in Molecular Biology. 578: 193–207. doi:10.1007/978-1-60327-411-1_12. ISBN 978-1-60327-410-4. PMID 19768595..</span></p></li><li id="quote_44"><span class="references-num">[44]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Malhis N, Jones SJ, Gsponer J (April 2019). "Improved measures for evolutionary conservation that exploit taxonomy distances". Nature Communications. 10 (1): 1556. doi:10.1038/s41467-019-09583-2. PMID 30952844..</span></p></li><li id="quote_45"><span class="references-num">[45]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"View of SNPViz - Visualization of SNPs in proteins". genomicscomputbiol.org (in 英语). Retrieved 2018-10-20..</span></p></li></ul></div><div class="read-num">阅读 <!-- -->3089</div></div><div class="right-side" id="rightSide"><div class="side" id="lemma-side"><div class="side-title">版本记录</div><ul class="side-lst"><li><p class="side-lst-txt">暂无</p></li></ul><div class="user-card userCard"></div></div><div class="side"><div class="side-event"></div></div></div></div><div class="footer-box"><div id="footer"><div class="footer-logo-wrap"><div class="footer-logo"></div><div class="footer-logo-text">知识·传播·科普</div></div><div class="footer-info">本网站内容采用<a target="_blank" href="https://web.archive.org/web/20221028214411/https://creativecommons.org/licenses/by-sa/3.0/deed.zh?tdsourcetag=s_pctim_aiomsg">CC-BY-SA 3.0</a>授权</div><div class="footer-btn-wrap"><a target="_blank" href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/help/#user_protocol">用户协议</a><a target="_blank" href="https://web.archive.org/web/20221028214411/http://www.sogou.com/docs/terms.htm?v=1">免责声明</a><a target="_blank" href="https://web.archive.org/web/20221028214411/http://corp.sogou.com/private.html">隐私政策</a><a target="_blank" href="https://web.archive.org/web/20221028214411/https://baike.sogou.com/kexue/intro.htm">关于我们</a></div></div></div><script>window.lemmaInfo ={"lemmaId":"10275","versionId":"54127453290947083","title":"单核苷酸多态性","subtitle":"","abstracts":{"paragraphId":"54127453290947095","title":"摘要","versionId":"54127453290947084","lemmaId":10275,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82812762,"name":"雷克斯班纳","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599558476,"comment":null,"dependVersionId":0,"contentType":2,"content":"<p>单核苷酸多态性，通常缩写为SNP(/snɪp/；复数/snɪps/)，是发生在基因组特殊位置的单个核苷酸的替换，其中每个变异在群体中存在一定程度(例如&gt; 1%)。<sup><a href=\"#quote_1\" class=\"kx_ref\">[1]</a></sup> </p>\n<p>在人类基因组的特定碱基位置上，C核苷酸可能出现在大多数个体中，但是在少数个体中，该位置被A占据。这意味着在该特定位置上存在单核苷酸多态性，并且两种可能的核苷酸变异——C或A——被称为该位置的等位基因。</p>\n<p>单核苷酸多态性预示了我们对多种疾病(如镰状细胞性贫血、β地中海贫血和单核苷酸多态性导致的囊性纤维化)的易感性差异。<sup><a href=\"#quote_2\" class=\"kx_ref\">[2]</a></sup><sup><a href=\"#quote_3\" class=\"kx_ref\">[3]</a></sup><sup><a href=\"#quote_4\" class=\"kx_ref\">[4]</a></sup> 疾病的严重性和身体对治疗的反应也是遗传变异的表现。例如，APOE(载脂蛋白E)基因的单碱基突变与阿尔茨海默病的风险较低相关。<sup><a href=\"#quote_5\" class=\"kx_ref\">[5]</a></sup> </p>\n<p>单核苷酸变体(SNV)是单核苷酸的变体，没有任何频率限制，可能出现在体细胞中。体细胞单核苷酸变异(比如，由癌症引起)也可以称为单核苷酸变异。</p>","pics":[{"originalUrl":"https://web.archive.org/web/20221028214411/https://img01.sogoucdn.com/app/a/200698/sogou_science_3230?width=231&height=174&titlename=%E4%B8%8A%E9%83%A8%E7%9A%84DNA%E5%88%86%E5%AD%90%E5%92%8C%E4%B8%8B%E9%83%A8%E7%9A%84DNA%E5%88%86%E5%AD%90%E5%9C%A8%E5%8D%95%E7%A2%B1%E5%9F%BA%E5%AF%B9%EF%BC%88%E4%B8%80%E7%A7%8DC/A%E7%9A%84%E5%A4%9A%E6%80%81%E6%80%A7%EF%BC%89%E7%9A%84%E4%BD%8D%E7%BD%AE%E5%85%B7%E6%9C%89%E5%B7%AE%E5%BC%82&w=306&h=200","url":"https://web.archive.org/web/20221028214411/https://img01.sogoucdn.com/app/a/200698/sogou_science_3230","rw":306,"rh":200,"title":"上部的DNA分子和下部的DNA分子在单碱基对（一种C/A的多态性）的位置具有差异","alt":null,"width":231,"height":174}],"card":null,"references":[],"versionCount":0},"card":{"paragraphId":"54127453290947096","title":"基本信息","versionId":"54127453290947085","lemmaId":10275,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82812762,"name":"雷克斯班纳","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599558476,"comment":null,"dependVersionId":0,"contentType":3,"content":"","pics":null,"card":{"cardItems":[{"key":"外文名","value":"single nucleotide polymorphism"},{"key":"性质","value":"DNA序列多态性"},{"key":"引发原因","value":"由碱基的插入或缺失所致"}]},"references":[],"versionCount":0},"categories":[{"id":4,"name":"生物学","parents":[]}],"creator":{"uid":76031615,"name":"郑逸舒","pic":"https://web.archive.org/web/20221028214411/https://cache.soso.com/qlogo/g?b=oidb&k=tNeDnMw0iaNLWnqt7O8Iv3w&s=100&t=1557391867","introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":"","jobBrief":"","role":0,"roleName":null,"title":"","professionalTitle":"学生","phoneNo":null,"editable":true,"partnerId":154,"partnerIdCreateTime":1595844883,"partnerIdPoped":false},"createTime":1568626289,"editor":{"uid":76031615,"name":"郑逸舒","pic":"https://web.archive.org/web/20221028214411/https://cache.soso.com/qlogo/g?b=oidb&k=tNeDnMw0iaNLWnqt7O8Iv3w&s=100&t=1557391867","introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":"","jobBrief":"","role":0,"roleName":null,"title":"","professionalTitle":"学生","phoneNo":null,"editable":true,"partnerId":154,"partnerIdCreateTime":1595844883,"partnerIdPoped":false},"editTime":1576233878,"state":1,"versionCount":2,"upNum":6,"downNum":0,"pics":[{"originalUrl":"https://web.archive.org/web/20221028214411/https://img01.sogoucdn.com/app/a/200698/sogou_science_3230?width=231&height=174&titlename=%E4%B8%8A%E9%83%A8%E7%9A%84DNA%E5%88%86%E5%AD%90%E5%92%8C%E4%B8%8B%E9%83%A8%E7%9A%84DNA%E5%88%86%E5%AD%90%E5%9C%A8%E5%8D%95%E7%A2%B1%E5%9F%BA%E5%AF%B9%EF%BC%88%E4%B8%80%E7%A7%8DC/A%E7%9A%84%E5%A4%9A%E6%80%81%E6%80%A7%EF%BC%89%E7%9A%84%E4%BD%8D%E7%BD%AE%E5%85%B7%E6%9C%89%E5%B7%AE%E5%BC%82&w=306&h=200","url":"https://web.archive.org/web/20221028214411/https://img01.sogoucdn.com/app/a/200698/sogou_science_3230","rw":306,"rh":200,"title":"上部的DNA分子和下部的DNA分子在单碱基对（一种C/A的多态性）的位置具有差异","alt":null,"width":231,"height":174},{"originalUrl":"https://web.archive.org/web/20221028214411/https://img01.sogoucdn.com/app/a/200698/sogou_science_3231?w=400&h=235&titlename=%E5%8D%95%E6%A0%B8%E8%8B%B7%E9%85%B8%E5%A4%9A%E6%80%81%E6%80%A7(SNPs)%E7%9A%84%E7%A7%8D%E7%B1%BB","url":"https://web.archive.org/web/20221028214411/https://img01.sogoucdn.com/app/a/200698/sogou_science_3231","rw":400,"rh":235,"title":"单核苷酸多态性(SNPs)的种类","alt":null,"width":0,"height":0}],"catalogs":[{"level":1,"title":"类型","paragraphId":"14995084281905671","subCatalogs":null},{"level":1,"title":"应用","paragraphId":"14995084281905672","subCatalogs":null},{"level":1,"title":"频率","paragraphId":"14995084281905673","subCatalogs":[{"level":2,"title":"在基因组中","paragraphId":"14995084281905673","subCatalogs":null},{"level":2,"title":"在人群中","paragraphId":"14995084281905673","subCatalogs":null}]},{"level":1,"title":"重要性","paragraphId":"14995084281905674","subCatalogs":[{"level":2,"title":"临床研究","paragraphId":"14995084281905674","subCatalogs":null},{"level":2,"title":"法医学","paragraphId":"14995084281905674","subCatalogs":null},{"level":2,"title":"遗传药理学","paragraphId":"14995084281905674","subCatalogs":null},{"level":2,"title":"疾病","paragraphId":"14995084281905674","subCatalogs":null}]},{"level":1,"title":"例子","paragraphId":"14995084281905675","subCatalogs":null},{"level":1,"title":"数据库","paragraphId":"14995084298682891","subCatalogs":null},{"level":1,"title":"命名法","paragraphId":"14995084298682892","subCatalogs":null},{"level":1,"title":"单核苷酸多态性分析","paragraphId":"14995084298682893","subCatalogs":null},{"level":1,"title":"预测单核苷酸多态性效应的程序","paragraphId":"14995084298682894","subCatalogs":null},{"level":1,"title":"参考文献","paragraphId":"-1","subCatalogs":null}],"paragraphs":[{"paragraphId":"14995084281905671","title":"类型","versionId":"54127453290947086","lemmaId":10275,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82812762,"name":"雷克斯班纳","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599558476,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>\n </p><table class=\"infobox\">\n  <tbody>\n   <tr>\n    <th>Types of SNPs</th>\n   </tr>\n   <tr>\n    <td>\n     <ul>\n      <li>Non-coding region</li>\n      <li>Coding region\n       <ul>\n        <li>Synonymous</li>\n        <li>Nonsynonymous\n         <ul>\n          <li>Missense</li>\n          <li>Nonsense</li>\n         </ul></li>\n       </ul></li>\n     </ul></td>\n   </tr>\n  </tbody>\n </table><p></p><p><img alt=\"\" class=\"fileimage kx_img ed_imgfloat_right\" img_height=\"235\" img_width=\"400\" titlename=\"单核苷酸多态性(SNPs)的种类\" data-src=\"https://img01.sogoucdn.com/app/a/200698/sogou_science_3231\"> </p><p>单核苷酸多态性可能属于基因编码序列、基因非编码区或基因间区(基因间区)。由于遗传密码的简并性，编码序列中的单核苷酸多态性不一定改变产生的蛋白质的氨基酸序列。</p><p>编码区的SNPs有两种类型:同义和非同义SNPs。同义SNPs不影响蛋白质序列，而非同义SNPs改变蛋白质的氨基酸序列。非同义SNPs有两种类型:错义和无意义。</p><p>不在蛋白质编码区的单核苷酸多态性仍可能影响基因剪接、转录因子结合、信使核糖核酸降解或非编码核糖核酸序列。受这种类型的单核苷酸多态性影响的基因表达被称为eSNP(表达单核苷酸多态性)，可以位于基因的上游或下游。</p>","pics":[{"originalUrl":"https://web.archive.org/web/20221028214411/https://img01.sogoucdn.com/app/a/200698/sogou_science_3231?w=400&h=235&titlename=%E5%8D%95%E6%A0%B8%E8%8B%B7%E9%85%B8%E5%A4%9A%E6%80%81%E6%80%A7(SNPs)%E7%9A%84%E7%A7%8D%E7%B1%BB","url":"https://web.archive.org/web/20221028214411/https://img01.sogoucdn.com/app/a/200698/sogou_science_3231","rw":400,"rh":235,"title":"单核苷酸多态性(SNPs)的种类","alt":null,"width":0,"height":0}],"card":null,"references":[],"versionCount":0},{"paragraphId":"14995084281905672","title":"应用","versionId":"54127453290947087","lemmaId":10275,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82812762,"name":"雷克斯班纳","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599558476,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>\n <ul>\n  <li>关联研究可以确定基因变异是否与疾病或性状相关。<sup><a href=\"#quote_6\" class=\"kx_ref\">[6]</a></sup></li>\n  <li>标签单核苷酸多态性(tag SNP)是基因组中一个具有高度连锁不平衡(两个或多个位点等位基因的非随机关联)的区域的代表性单核苷酸多态性。标签单核苷酸多态性在全基因组单核苷酸多态性关联研究中是有用的，在全基因组单核苷酸多态性关联研究中，数十万个单核苷酸多态性是基因分型的。</li>\n  <li>单体型映射:等位基因或脱氧核糖核酸序列的集合可以被聚集，这样一个单核苷酸多态性可以识别许多连接的单核苷酸多态性。</li>\n  <li>连锁不平衡是群体遗传学中的一个术语，指等位基因在两个或多个位点的非随机关联，不一定在同一条染色体上。它是指基因组中紧密相连的单核苷酸多态性等位基因或脱氧核糖核酸序列倾向于一起遗传的现象。LD受两个参数的影响:1)单核苷酸多态性之间的距离。2)重组率。<sup><a href=\"#quote_7\" class=\"kx_ref\">[7]</a></sup></li>\n </ul></p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14995084281905673","title":"频率","versionId":"54127453290947088","lemmaId":10275,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82812762,"name":"雷克斯班纳","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599558476,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>在不同人群中发现了超过8400万个单核苷酸多态性。一个典型的基因组在400万到500万个位点上不同于参考人类基因组，其中大部分(超过99.9%)由单核苷酸多态性和短序列组成。<sup><a href=\"#quote_8\" class=\"kx_ref\">[8]</a></sup> </p><p><h3>在基因组中</h3></p><p>单核苷酸多态性的基因组分布不均匀；单核苷酸多态性出现在非编码区的频率高于编码区，或者一般来说，在自然选择起作用并“固定”构成最有利遗传适应的单核苷酸多态性等位基因(消除其他变体)的地方。<sup><a href=\"#quote_9\" class=\"kx_ref\">[9]</a></sup> 其他因素，如基因重组和突变率，也可以决定单核苷酸多态性密度。<sup><a href=\"#quote_10\" class=\"kx_ref\">[10]</a></sup> </p><p>单核苷酸多态性密度可以通过微卫星的存在来预测:特别是腺苷酸微卫星是单核苷酸多态性密度的有效预测因子，长的（AT）重复片段通常被发现存在在单核苷酸多态性频率显著较低和GC含量较低的区域。<sup><a href=\"#quote_11\" class=\"kx_ref\">[11]</a></sup> </p><p><h3>在人群中</h3></p><p>人类群体之间存在差异，因此在一个地理或种族群体中常见的单核苷酸多态性等位基因在另一个群体中可能要罕见得多。在一个群体中，单核苷酸多态性可以被赋予一个较低的等位基因频率——在特定群体中观察到的某一位点的最低等位基因频率。这只是单核苷酸多态性的两个等位基因频率中较小的一个。</p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14995084281905674","title":"重要性","versionId":"54127453290947089","lemmaId":10275,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82812762,"name":"雷克斯班纳","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599558476,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>人类基因序列的变化会影响人类疾病的发生以及对病原体、化学物质、药物、疫苗和其他药剂的反应。单核苷酸多态性对个性化医学也至关重要。<sup><a href=\"#quote_12\" class=\"kx_ref\">[12]</a></sup> 例子包括生物医学研究、法医学、药物遗传学和疾病成因，如下所述。</p><p><h3>临床研究</h3></p><p>单核苷酸多态性在临床研究中最重要的是在全基因组关联研究中比较不同队列(如有疾病和无疾病的匹配队列)之间的基因组区域。单核苷酸多态性已经在全基因组关联研究中用作与疾病或正常性状相关的基因定位的高分辨率标记。对表型没有明显影响的单核苷酸多态性(所谓的沉默突变)在全基因组关联研究中仍然是有用的遗传标记，因为它们的数量和世代间的稳定遗传。</p><p><h3>法医学</h3></p><p>单核苷酸多态性最初用于将法医DNA样本与嫌疑人进行匹配，但随着基于串联重复序列的脱氧核糖核酸指纹技术的发展，它已经被淘汰。当前的新一代测序(NGS)技术可能允许在法医应用中更好地使用单核苷酸多态性基因分型，只要避免有问题的基因座。<sup><a href=\"#quote_13\" class=\"kx_ref\">[13]</a></sup> 在未来，SNPs可能被用于一些表型线索的取证，如眼睛颜色、头发颜色、种族等。Kidd等人已经证明，一个由19个SNPs组成的小组可以在所研究的40个人群中识别出匹配概率很高的种族群体(Pm = 10^-7)。<sup><a href=\"#quote_14\" class=\"kx_ref\">[14]</a></sup> 这可能有潜在用途的一个例子是对未知个体骨骼化遗骸可能的预解剖外观进行艺术重建。虽然严格地基于人类学特征，面部重建可以相当精确，但是其他包括眼睛颜色、皮肤颜色、头发颜色等数据可能会让重建更精确。</p><p>在法医样本或降解样本数量较少的情况下，由于潜在标记物的丰富性、对自动化的适应性以及所需片段长度可能减少到仅60-80 bp，单核苷酸多态性方法可能是STR方法的良好替代方法。在脱氧核糖核酸图谱数据库中没有字符串匹配的情况下；不同的单核苷酸多态性可以用来获得关于种族、表现型、谱系甚至身份的线索。</p><p><h3>遗传药理学</h3></p><p>一些单核苷酸多态性与不同药物的代谢有关。<sup><a href=\"#quote_15\" class=\"kx_ref\">[15]</a></sup><sup><a href=\"#quote_16\" class=\"kx_ref\">[16]</a></sup><sup><a href=\"#quote_17\" class=\"kx_ref\">[17]</a></sup> 癌症、传染病(艾滋病、麻风病、肝炎等)等多种人类疾病的关联。)具有不同SNPs的自身免疫性、神经精神性和许多其他疾病可以作为药物治疗的相关药物基因组靶标。<sup><a href=\"#quote_18\" class=\"kx_ref\">[18]</a></sup> </p><p><h3>疾病</h3></p><p>单个单核苷酸多态性可能导致孟德尔病，尽管对于复杂的疾病，单核苷酸多态性通常不单独发挥作用，相反，它们与其他单核苷酸多态性协同工作，表现出骨质疏松症的疾病状态。<sup><a href=\"#quote_19\" class=\"kx_ref\">[19]</a></sup> 该领域最早的成功之一是在APOC3(载脂蛋白C3基因)的非编码区发现了一个单碱基突变，该突变与高甘油三酯血症和动脉粥样硬化的高风险相关。<sup><a href=\"#quote_20\" class=\"kx_ref\">[20]</a></sup> </p><p>所有类型的单核苷酸多态性都可能具有可观察到的表型或导致疾病:</p><p>\n <ul>\n  <li>非编码区的单核苷酸多态性可能表现出更高的癌症风险，<sup><a href=\"#quote_21\" class=\"kx_ref\">[21]</a></sup> 并可能影响基因结构和疾病易感性。<sup><a href=\"#quote_22\" class=\"kx_ref\">[22]</a></sup> 非编码SNPs也可以改变基因的表达水平，如eQTL(表达数量性状基因座)。</li>\n  <li>编码区的单核苷酸多态性:\n   <ul>\n    <li>顾名思义，同义替换不会导致蛋白质中氨基酸的变化，但仍能以其他方式影响其功能。一个例子是多药耐药基因1 (MDR1)中看似沉默的突变，其编码从细胞中排出药物的细胞膜泵，可以减缓翻译并允许肽链折叠成不寻常的构象，导致突变泵功能减弱(在MDR1蛋白中，例如，C1236T多态性将多肽氨基酸位置412的GGC密码子改变为GGT(均编码甘氨酸)，C3435T多态性将ATC改变为位置1145的ATT(均编码异亮氨酸))。<sup><a href=\"#quote_23\" class=\"kx_ref\">[23]</a></sup></li>\n    <li>非同步替换:\n     <ul>\n      <li>错义——单个碱基的变化导致蛋白质氨基酸的变化及其导致疾病的功能障碍(例如，LMNA基因中的1580G&gt;T单核苷酸多态性——脱氧核糖核酸序列中的1580位(CGT密码子)导致鸟嘌呤被胸腺嘧啶取代，在脱氧核糖核酸序列中产生CTT密码子，在蛋白质水平上导致527位的亮氨酸取代精氨酸，<sup><a href=\"#quote_24\" class=\"kx_ref\">[24]</a></sup> 在表型水平上这表现为重叠的下颌骨发育不良和早衰综合征)</li>\n      <li>无意义——在脱氧核糖核酸序列中的点突变，导致转录的脱氧核糖核酸中的过早终止密码子或无意义密码子，以及截短的、不完整的、通常无功能的蛋白质产物(如囊性纤维化跨膜传导调节基因G542X突变引起的囊性纤维化)。<sup><a href=\"#quote_25\" class=\"kx_ref\">[25]</a></sup></li>\n     </ul></li>\n   </ul></li>\n </ul></p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14995084281905675","title":"例子","versionId":"54127453290947090","lemmaId":10275,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82812762,"name":"雷克斯班纳","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599558476,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>\n <ul>\n  <li>rs6311和rs6313是人类染色体13上血清素5-HT2A受体基因的单核苷酸多态性。<sup><a href=\"#quote_26\" class=\"kx_ref\">[26]</a></sup></li>\n  <li>F5基因中的单核苷酸多态性导致莱顿因子血栓形成倾向。<sup><a href=\"#quote_27\" class=\"kx_ref\">[27]</a></sup></li>\n  <li>rs3091244是人类染色体1上CRP基因中三链单核苷酸多态性的一个例子。<sup><a href=\"#quote_28\" class=\"kx_ref\">[28]</a></sup></li>\n  <li>TAS2R38编码PTC品尝能力，并包含6个注释的SNPs。<sup><a href=\"#quote_29\" class=\"kx_ref\">[29]</a></sup></li>\n  <li>编码无花果蛋白酶的FCN1基因中的rs148649884和rs138055828削弱了重组无花果蛋白酶的配体结合能力。<sup><a href=\"#quote_30\" class=\"kx_ref\">[30]</a></sup></li>\n  <li>脱氧核糖核酸错配修复基因PMS2 (rs1059060，Ser775Asn)的内含子单核苷酸多态性与精子脱氧核糖核酸损伤增加和男性不育风险有关。<sup><a href=\"#quote_31\" class=\"kx_ref\">[31]</a></sup></li>\n </ul></p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14995084298682891","title":"数据库","versionId":"54127453290947091","lemmaId":10275,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82812762,"name":"雷克斯班纳","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599558476,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>正如基因一样，SNPs也有生物信息学数据库。</p><p>\n <ul>\n  <li><i>dbSNP</i> 数据库是国家生物技术信息中心(NCBI)的SNP数据库。截至2015年6月8日，数据库单核苷酸多态性列出了149，735，377个人类单核苷酸多态性。<sup><a href=\"#quote_32\" class=\"kx_ref\">[32]</a></sup><sup><a href=\"#quote_33\" class=\"kx_ref\">[33]</a></sup></li>\n  <li><i>Kaviar</i><sup><a href=\"#quote_34\" class=\"kx_ref\">[34]</a></sup> 是来自多个数据源的SNP概要，包括dbSNP。</li>\n  <li><i>SNPedia</i> 是一个维基风格的数据库，支持个人基因组注释、解释和分析。</li>\n  <li><i>OMIM</i> 数据库描述多态性和疾病之间的关联(例如，以文本形式给出疾病)。</li>\n  <li>dbSap——用于蛋白质变异检测的单一氨基酸多态性数据库。<sup><a href=\"#quote_35\" class=\"kx_ref\">[35]</a></sup></li>\n  <li>人类基因突变数据库提供导致或与人类遗传疾病和功能性相关的 &nbsp; 单核苷酸多态性相关的基因突变。</li>\n  <li>国际单倍体项目，研究人员正在鉴定标签单核苷酸多态性，以便能够确定每个受试者中存在的单倍体集合。</li>\n  <li>GWAS中心允许用户在一个或多个全基因组关联研究中直观地询问实际的汇总级关联数据。</li>\n </ul></p><p>国际单核苷酸多态性图谱工作组通过与基因库中插入大克隆片段的基因组序列比对，绘制了每个单核苷酸多态性侧翼的序列。这些比对被转换成染色体坐标，如表1所示。这个列表已经大大增加了，例如，卡维亚数据库现在列出了1.62亿个单核苷酸变异体。</p><p>\n <table class=\"wikitable\">\n  <tbody>\n   <tr>\n    <th>Chromosome</th>\n    <th>Length(bp)</th>\n    <th>All SNPs</th>\n    <th></th>\n    <th>TSC SNPs</th>\n    <th></th>\n   </tr>\n   <tr>\n    <td></td>\n    <td></td>\n    <td>Total SNPs</td>\n    <td>kb per SNP</td>\n    <td>Total SNPs</td>\n    <td>kb per SNP</td>\n   </tr>\n   <tr>\n    <td>1</td>\n    <td>214,066,000</td>\n    <td>129,931</td>\n    <td>1.65</td>\n    <td>75,166</td>\n    <td>2.85</td>\n   </tr>\n   <tr>\n    <td>2</td>\n    <td>222,889,000</td>\n    <td>103,664</td>\n    <td>2.15</td>\n    <td>76,985</td>\n    <td>2.90</td>\n   </tr>\n   <tr>\n    <td>3</td>\n    <td>186,938,000</td>\n    <td>93,140</td>\n    <td>2.01</td>\n    <td>63,669</td>\n    <td>2.94</td>\n   </tr>\n   <tr>\n    <td>4</td>\n    <td>169,035,000</td>\n    <td>84,426</td>\n    <td>2.00</td>\n    <td>65,719</td>\n    <td>2.57</td>\n   </tr>\n   <tr>\n    <td>5</td>\n    <td>170,954,000</td>\n    <td>117,882</td>\n    <td>1.45</td>\n    <td>63,545</td>\n    <td>2.69</td>\n   </tr>\n   <tr>\n    <td>6</td>\n    <td>165,022,000</td>\n    <td>96,317</td>\n    <td>1.71</td>\n    <td>53,797</td>\n    <td>3.07</td>\n   </tr>\n   <tr>\n    <td>7</td>\n    <td>149,414,000</td>\n    <td>71,752</td>\n    <td>2.08</td>\n    <td>42,327</td>\n    <td>3.53</td>\n   </tr>\n   <tr>\n    <td>8</td>\n    <td>125,148,000</td>\n    <td>57,834</td>\n    <td>2.16</td>\n    <td>42,653</td>\n    <td>2.93</td>\n   </tr>\n   <tr>\n    <td>9</td>\n    <td>107,440,000</td>\n    <td>62,013</td>\n    <td>1.73</td>\n    <td>43,020</td>\n    <td>2.50</td>\n   </tr>\n   <tr>\n    <td>10</td>\n    <td>127,894,000</td>\n    <td>61,298</td>\n    <td>2.09</td>\n    <td>42,466</td>\n    <td>3.01</td>\n   </tr>\n   <tr>\n    <td>11</td>\n    <td>129,193,000</td>\n    <td>84,663</td>\n    <td>1.53</td>\n    <td>47,621</td>\n    <td>2.71</td>\n   </tr>\n   <tr>\n    <td>12</td>\n    <td>125,198,000</td>\n    <td>59,245</td>\n    <td>2.11</td>\n    <td>38,136</td>\n    <td>3.28</td>\n   </tr>\n   <tr>\n    <td>13</td>\n    <td>93,711,000</td>\n    <td>53,093</td>\n    <td>1.77</td>\n    <td>35,745</td>\n    <td>2.62</td>\n   </tr>\n   <tr>\n    <td>14</td>\n    <td>89,344,000</td>\n    <td>44,112</td>\n    <td>2.03</td>\n    <td>29,746</td>\n    <td>3.00</td>\n   </tr>\n   <tr>\n    <td>15</td>\n    <td>73,467,000</td>\n    <td>37,814</td>\n    <td>1.94</td>\n    <td>26,524</td>\n    <td>2.77</td>\n   </tr>\n   <tr>\n    <td>16</td>\n    <td>74,037,000</td>\n    <td>38,735</td>\n    <td>1.91</td>\n    <td>23,328</td>\n    <td>3.17</td>\n   </tr>\n   <tr>\n    <td>17</td>\n    <td>73,367,000</td>\n    <td>34,621</td>\n    <td>2.12</td>\n    <td>19,396</td>\n    <td>3.78</td>\n   </tr>\n   <tr>\n    <td>18</td>\n    <td>73,078,000</td>\n    <td>45,135</td>\n    <td>1.62</td>\n    <td>27,028</td>\n    <td>2.70</td>\n   </tr>\n   <tr>\n    <td>19</td>\n    <td>56,044,000</td>\n    <td>25,676</td>\n    <td>2.18</td>\n    <td>11,185</td>\n    <td>5.01</td>\n   </tr>\n   <tr>\n    <td>20</td>\n    <td>63,317,000</td>\n    <td>29,478</td>\n    <td>2.15</td>\n    <td>17,051</td>\n    <td>3.71</td>\n   </tr>\n   <tr>\n    <td>21</td>\n    <td>33,824,000</td>\n    <td>20,916</td>\n    <td>1.62</td>\n    <td>9,103</td>\n    <td>3.72</td>\n   </tr>\n   <tr>\n    <td>22</td>\n    <td>33,786,000</td>\n    <td>28,410</td>\n    <td>1.19</td>\n    <td>11,056</td>\n    <td>3.06</td>\n   </tr>\n   <tr>\n    <td>X</td>\n    <td>131,245,000</td>\n    <td>34,842</td>\n    <td>3.77</td>\n    <td>20,400</td>\n    <td>6.43</td>\n   </tr>\n   <tr>\n    <td>Y</td>\n    <td>21,753,000</td>\n    <td>4,193</td>\n    <td>5.19</td>\n    <td>1,784</td>\n    <td>12.19</td>\n   </tr>\n   <tr>\n    <td>RefSeq</td>\n    <td>15,696,674</td>\n    <td>14,534</td>\n    <td>1.08</td>\n   </tr>\n   <tr>\n    <td>Totals</td>\n    <td>2,710,164,000</td>\n    <td>1,419,190</td>\n    <td>1.91</td>\n    <td>887,450</td>\n    <td>3.05</td>\n   </tr>\n  </tbody>\n </table></p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14995084298682892","title":"命名法","versionId":"54127453290947092","lemmaId":10275,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82812762,"name":"雷克斯班纳","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599558476,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>单核苷酸多态性的命名可能令人困惑:单个单核苷酸多态性可能存在几种变异，但尚未达成共识。一种表示单核苷酸多态性的方法是写带有前缀、句点和“大于”符号，显示野生型和改变的核苷酸或氨基酸；例如，c.76A&gt;T.<sup><a href=\"#quote_36\" class=\"kx_ref\">[36]</a></sup><sup><a href=\"#quote_37\" class=\"kx_ref\">[37]</a></sup><sup><a href=\"#quote_38\" class=\"kx_ref\">[38]</a></sup> SNPs经常用它们的dbSNP rs数来表示。</p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14995084298682893","title":"单核苷酸多态性分析","versionId":"54127453290947093","lemmaId":10275,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82812762,"name":"雷克斯班纳","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599558476,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>单核苷酸多态性通常是双等位基因，因此很容易分析。<sup><a href=\"#quote_39\" class=\"kx_ref\">[39]</a></sup> 发现新的单核苷酸多态性和检测已知单核苷酸多态性的分析方法包括:</p><p>\n <ul>\n  <li>脱氧核糖核酸测序；<sup><a href=\"#quote_40\" class=\"kx_ref\">[40]</a></sup></li>\n  <li>毛细管电泳；<sup><a href=\"#quote_41\" class=\"kx_ref\">[41]</a></sup></li>\n  <li>质谱分析；<sup><a href=\"#quote_42\" class=\"kx_ref\">[42]</a></sup></li>\n  <li>单链构象多态性(SSCP)；<sup><a href=\"#quote_43\" class=\"kx_ref\">[43]</a></sup></li>\n  <li>单碱基延伸；</li>\n  <li>电化学分析；</li>\n  <li>变性高效液相色谱和凝胶电泳；</li>\n  <li>限制性片段长度多态性；</li>\n  <li>杂交分析；</li>\n </ul></p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14995084298682894","title":"预测单核苷酸多态性效应的程序","versionId":"54127453290947094","lemmaId":10275,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82812762,"name":"雷克斯班纳","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599558476,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>单核苷酸多态性中比较重要的一类是那些与导致蛋白质水平氨基酸变化的错义突变相对应的多态性。特定残基的点突变对蛋白质功能有不同的影响(从无影响到完全破坏其功能)。通常，具有相似大小和物理化学性质的氨基酸改变(例如亮氨酸被缬氨酸取代)具有温和的效果，反之亦然。类似地，如果单核苷酸多态性破坏二级结构元件(例如α螺旋区脯氨酸被取代)，这种突变通常会影响整个蛋白质的结构和功能。利用这些简单的和许多其他机器学习导出的规则，开发了一组预测单核苷酸多态性效应的程序:</p><p>\n <ul>\n  <li>SIFT 这个项目基于氨基酸的物理性质和序列同源性，提供了实验室诱导的错义或非同源突变如何影响蛋白质功能的见解。</li>\n  <li>LIST<sup><a href=\"#quote_44\" class=\"kx_ref\">[44]</a></sup> (局部同一性和共享分类群) 估计突变所导致的由蛋白质功能的改变的所产生的潜在有害。这是基于这样一个假设，即与人类密切相关的物种中观察到的变异在评估潜在危害时，比那些远亲物种中的变异更为重要。</li>\n  <li>SNAP2</li>\n  <li>SuSPect</li>\n  <li>PolyPhen-2</li>\n  <li>PredictSNP</li>\n  <li>突变网站:官方网站 official website</li>\n  <li>Variant Effect Predictor 恩塞布勒项目中的变量效应预测器</li>\n  <li>SNPViz<sup><a href=\"#quote_45\" class=\"kx_ref\">[45]</a></sup> 该程序提供了受影响蛋白质的三维表示，突出了氨基酸的变化，这样医生就可以确定突变蛋白质的致病性。</li>\n </ul></p>","pics":null,"card":null,"references":[],"versionCount":0}],"references":[{"id":1,"type":"book","title":"\"single-nucleotide polymorphism / SNP | Learn Science at Scitable\". www.nature.com. Archived from the original on 2015-11-10. Retrieved 2015-11-13.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":2,"type":"book","title":"Ingram VM (October 1956). \"A specific chemical difference between the globins of normal human and sickle-cell anaemia haemoglobin\". Nature. 178 (4537): 792–4. doi:10.1038/178792a0. PMID 13369537.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":3,"type":"book","title":"Chang JC, Kan YW (June 1979). \"beta 0 thalassemia, a nonsense mutation in man\". Proceedings of the National Academy of Sciences of the United States of America. 76 (6): 2886–9. doi:10.1073/pnas.76.6.2886. PMC 383714. PMID 88735.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":4,"type":"book","title":"Hamosh A, King TM, Rosenstein BJ, Corey M, Levison H, Durie P, Tsui LC, McIntosh I, Keston M, Brock DJ (August 1992). \"Cystic fibrosis patients bearing both the common missense mutation Gly----Asp at codon 551 and the delta F508 mutation are clinically indistinguishable from delta F508 homozygotes, except for decreased risk of meconium ileus\". American Journal of Human Genetics. 51 (2): 245–50. PMC 1682672. PMID 1379413.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":5,"type":"book","title":"Wolf AB, Caselli RJ, Reiman EM, Valla J (April 2013). \"APOE and neuroenergetics: an emerging paradigm in Alzheimer's disease\". Neurobiology of Aging. 34 (4): 1007–17. doi:10.1016/j.neurobiolaging.2012.10.011. PMC 3545040. PMID 23159550.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":6,"type":"book","title":"Zhang K, Qin ZS, Liu JS, Chen T, Waterman MS, Sun F (May 2004). \"Haplotype block partitioning and tag SNP selection using genotype data and their applications to association studies\". Genome Research. 14 (5): 908–16. doi:10.1101/gr.1837404. PMC 479119. PMID 15078859. Archived from the original on 8 May 2018.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":7,"type":"book","title":"Gupta PK, Roy JK, Prasad M (25 February 2001). \"Single nucleotide polymorphisms: a new paradigm for molecular marker technology and DNA polymorphism detection with emphasis on their use in plants\". Current Science. 80 (4): 524–535. Archived from the original on 13 February 2017.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":8,"type":"book","title":"Auton A, Brooks LD, Durbin RM, Garrison EP, Kang HM, Korbel JO, Marchini JL, McCarthy S, McVean GA, Abecasis GR (October 2015). \"A global reference for human genetic variation\". Nature. 526 (7571): 68–74. doi:10.1038/nature15393. PMC 4750478. PMID 26432245.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":9,"type":"book","title":"Barreiro LB, Laval G, Quach H, Patin E, Quintana-Murci L (March 2008). \"Natural selection has driven population differentiation in modern humans\". Nature Genetics. 40 (3): 340–5. doi:10.1038/ng.78. PMID 18246066.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":10,"type":"book","title":"Nachman MW (September 2001). \"Single nucleotide polymorphisms and recombination rate in humans\". Trends in Genetics. 17 (9): 481–5. doi:10.1016/S0168-9525(01)02409-X. PMID 11525814.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":11,"type":"book","title":"Varela MA, Amos W (March 2010). \"Heterogeneous distribution of SNPs in the human genome: microsatellites as predictors of nucleotide diversity and divergence\". Genomics. 95 (3): 151–9. doi:10.1016/j.ygeno.2009.12.003. PMID 20026267.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":12,"type":"book","title":"Carlson, Bruce (15 June 2008). \"SNPs — A Shortcut to Personalized Medicine\". Genetic Engineering & Biotechnology News. Mary Ann Liebert, Inc. 28 (12). Archived from the original on 26 December 2010. Retrieved 2008-07-06. (subtitle) Medical applications are where the market's growth is expected","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":13,"type":"book","title":"Cornelis S, Gansemans Y, Deleye L, Deforce D, Van Nieuwerburgh F (February 2017). \"Forensic SNP Genotyping using Nanopore MinION Sequencing\". Scientific Reports. 7: 41759. doi:10.1038/srep41759. PMC 5290523. PMID 28155888.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":14,"type":"book","title":"Kidd KK, Pakstis AJ, Speed WC, Grigorenko EL, Kajuna SL, Karoma NJ, Kungulilo S, Kim JJ, Lu RB, Odunsi A, Okonofua F, Parnas J, Schulz LO, Zhukova OV, Kidd JR (December 2006). \"Developing a SNP panel for forensic identification of individuals\". Forensic Science International. 164 (1): 20–32. doi:10.1016/j.forsciint.2005.11.017. PMID 16360294.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":15,"type":"book","title":"Goldstein JA (October 2001). \"Clinical relevance of genetic polymorphisms in the human CYP2C subfamily\". British Journal of Clinical Pharmacology. 52 (4): 349–55. doi:10.1046/j.0306-5251.2001.01499.x. PMC 2014584. PMID 11678778.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":16,"type":"book","title":"Lee CR (July–August 2004). \"CYP2C9 genotype as a predictor of drug disposition in humans\". Methods and Findings in Experimental and Clinical Pharmacology. 26 (6): 463–72. PMID 15349140.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":17,"type":"book","title":"Yanase K, Tsukahara S, Mitsuhashi J, Sugimoto Y (March 2006). \"Functional SNPs of the breast cancer resistance protein-therapeutic effects and inhibitor development\". Cancer Letters. 234 (1): 73–80. doi:10.1016/j.canlet.2005.04.039. PMID 16303243.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":18,"type":"book","title":"Fareed M, Afzal M (April 2013). \"Single-nucleotide polymorphism in genome-wide association of human population: A tool for broad spectrum service\". Egyptian Journal of Medical Human Genetics. 14 (2): 123–134. doi:10.1016/j.ejmhg.2012.08.001.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":19,"type":"book","title":"Singh M, Singh P, Juneja PK, Singh S, Kaur T (March 2011). \"SNP-SNP interactions within APOE gene influence plasma lipids in postmenopausal osteoporosis\". Rheumatology International. 31 (3): 421–3. doi:10.1007/s00296-010-1449-7. PMID 20340021.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":20,"type":"book","title":"Rees A, Shoulders CC, Stocks J, Galton DJ, Baralle FE (February 1983). \"DNA polymorphism adjacent to human apoprotein A-1 gene: relation to hypertriglyceridaemia\". Lancet. 1 (8322): 444–6. doi:10.1016/S0140-6736(83)91440-X. PMID 6131168.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":21,"type":"book","title":"Li G, Pan T, Guo D, Li LC (2014). \"Regulatory Variants and Disease: The E-Cadherin -160C/A SNP as an Example\". Molecular Biology International. 2014: 967565. doi:10.1155/2014/967565. PMC 4167656. PMID 25276428.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":22,"type":"book","title":"Lu YF, Mauger DM, Goldstein DB, Urban TJ, Weeks KM, Bradrick SS (November 2015). \"IFNL3 mRNA structure is remodeled by a functional non-coding polymorphism associated with hepatitis C virus clearance\". Scientific Reports. 5: 16037. doi:10.1038/srep16037. PMC 4631997. PMID 26531896.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":23,"type":"book","title":"Kimchi-Sarfaty C, Oh JM, Kim IW, Sauna ZE, Calcagno AM, Ambudkar SV, Gottesman MM (January 2007). \"A \"silent\" polymorphism in the MDR1 gene changes substrate specificity\". Science. 315 (5811): 525–8. doi:10.1126/science.1135308. PMID 17185560.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":24,"type":"book","title":"Al-Haggar M, Madej-Pilarczyk A, Kozlowski L, Bujnicki JM, Yahia S, Abdel-Hadi D, Shams A, Ahmad N, Hamed S, Puzianowska-Kuznicka M (November 2012). \"A novel homozygous p.Arg527Leu LMNA mutation in two unrelated Egyptian families causes overlapping mandibuloacral dysplasia and progeria syndrome\". European Journal of Human Genetics. 20 (11): 1134–40. doi:10.1038/ejhg.2012.77. PMC 3476705. PMID 22549407. Archived from the original on 2015-11-16.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":25,"type":"book","title":"Cordovado SK, Hendrix M, Greene CN, Mochal S, Earley MC, Farrell PM, Kharrazi M, Hannon WH, Mueller PW (February 2012). \"CFTR mutation analysis and haplotype associations in CF patients\". Molecular Genetics and Metabolism. 105 (2): 249–54. doi:10.1016/j.ymgme.2011.10.013. PMC 3551260. PMID 22137130.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":26,"type":"book","title":"Giegling I, Hartmann AM, Möller HJ, Rujescu D (November 2006). \"Anger- and aggression-related traits are associated with polymorphisms in the 5-HT-2A gene\". Journal of Affective Disorders. 96 (1–2): 75–81. doi:10.1016/j.jad.2006.05.016. PMID 16814396.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":27,"type":"book","title":"Kujovich JL (January 2011). \"Factor V Leiden thrombophilia\". Genetics in Medicine. 13 (1): 1–16. doi:10.1097/GIM.0b013e3181faa0f2. PMID 21116184.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":28,"type":"book","title":"Morita A, Nakayama T, Doba N, Hinohara S, Mizutani T, Soma M (June 2007). \"Genotyping of triallelic SNPs using TaqMan PCR\". Molecular and Cellular Probes. 21 (3): 171–6. doi:10.1016/j.mcp.2006.10.005. PMID 17161935.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":29,"type":"book","title":"Prodi DA, Drayna D, Forabosco P, Palmas MA, Maestrale GB, Piras D, Pirastu M, Angius A (October 2004). \"Bitter taste study in a sardinian genetic isolate supports the association of phenylthiocarbamide sensitivity to the TAS2R38 bitter receptor gene\". Chemical Senses. 29 (8): 697–702. doi:10.1093/chemse/bjh074. PMID 15466815.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":30,"type":"book","title":"Ammitzbøll CG, Kjær TR, Steffensen R, Stengaard-Pedersen K, Nielsen HJ, Thiel S, Bøgsted M, Jensenius JC (28 November 2012). \"Non-synonymous polymorphisms in the FCN1 gene determine ligand-binding ability and serum levels of M-ficolin\". PLOS ONE. 7 (11): e50585. doi:10.1371/journal.pone.0050585. PMC 3509001. PMID 23209787. Archived from the original on 7 June 2015.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":31,"type":"book","title":"Ji G, Long Y, Zhou Y, Huang C, Gu A, Wang X (May 2012). \"Common variants in mismatch repair genes associated with increased risk of sperm DNA damage and male infertility\". BMC Medicine. 10: 49. doi:10.1186/1741-7015-10-49. PMC 3378460. PMID 22594646.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":32,"type":"book","title":"National Center for Biotechnology Information, United States National Library of Medicine. 2014. NCBI dbSNP build 142 for human. \"Archived copy\". Archived from the original on 2017-09-10. Retrieved 2017-09-11.CS1 maint: Archived copy as title (link)","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":33,"type":"book","title":"National Center for Biotechnology Information, United States National Library of Medicine. 2015. NCBI dbSNP build 144 for human. Summary Page. \"Archived copy\". Archived from the original on 2017-09-10. Retrieved 2017-09-11.CS1 maint: Archived copy as title (link)","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":34,"type":"book","title":"Glusman G, Caballero J, Mauldin DE, Hood L, Roach JC (November 2011). \"Kaviar: an accessible system for testing SNV novelty\". Bioinformatics. 27 (22): 3216–7. doi:10.1093/bioinformatics/btr540. PMC 3208392. PMID 21965822.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":35,"type":"book","title":"Cao R, Shi Y, Chen S, Ma Y, Chen J, Yang J, Chen G, Shi T (January 2017). \"dbSAP: single amino-acid polymorphism database for protein variation detection\". Nucleic Acids Research. 45 (D1): D827–D832. doi:10.1093/nar/gkw1096. PMC 5210569. PMID 27903894.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":36,"type":"book","title":"J.T. Den Dunnen (2008-02-20). \"Recommendations for the description of sequence variants\". Human Genome Variation Society. Archived from the original on 2008-09-14. Retrieved 2008-09-05.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":37,"type":"book","title":"den Dunnen JT, Antonarakis SE (2000). \"Mutation nomenclature extensions and suggestions to describe complex mutations: a discussion\". Human Mutation. 15 (1): 7–12. doi:10.1002/(SICI)1098-1004(200001)15:1<7::AID-HUMU4>3.0.CO;2-N. PMID 10612815.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":38,"type":"book","title":"Ogino S, Gulley ML, den Dunnen JT, Wilson RB (February 2007). \"Standard mutation nomenclature in molecular diagnostics: practical and educational challenges\". The Journal of Molecular Diagnostics. 9 (1): 1–6. doi:10.2353/jmoldx.2007.060081. PMC 1867422. PMID 17251329.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":39,"type":"book","title":"Sachidanandam R, Weissman D, Schmidt SC, Kakol JM, Stein LD, Marth G, Sherry S, Mullikin JC, Mortimore BJ, Willey DL, Hunt SE, Cole CG, Coggill PC, Rice CM, Ning Z, Rogers J, Bentley DR, Kwok PY, Mardis ER, Yeh RT, Schultz B, Cook L, Davenport R, Dante M, Fulton L, Hillier L, Waterston RH, McPherson JD, Gilman B, Schaffner S, Van Etten WJ, Reich D, Higgins J, Daly MJ, Blumenstiel B, Baldwin J, Stange-Thomann N, Zody MC, Linton L, Lander ES, Altshuler D (February 2001). \"A map of human genome sequence variation containing 1.42 million single nucleotide polymorphisms\". Nature. 409 (6822): 928–33. doi:10.1038/35057149. PMID 11237013.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":40,"type":"book","title":"Altshuler D, Pollara VJ, Cowles CR, Van Etten WJ, Baldwin J, Linton L, Lander ES (September 2000). \"An SNP map of the human genome generated by reduced representation shotgun sequencing\". Nature. 407 (6803): 513–6. doi:10.1038/35035083. PMID 11029002.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":41,"type":"book","title":"Drabovich AP, Krylov SN (March 2006). \"Identification of base pairs in single-nucleotide polymorphisms by MutS protein-mediated capillary electrophoresis\". Analytical Chemistry. 78 (6): 2035–8. doi:10.1021/ac0520386. PMID 16536443.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":42,"type":"book","title":"Griffin TJ, Smith LM (July 2000). \"Genetic identification by mass spectrometric analysis of single-nucleotide polymorphisms: ternary encoding of genotypes\". Analytical Chemistry. 72 (14): 3298–302. doi:10.1021/ac991390e. PMID 10939403.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":43,"type":"book","title":"Tahira T, Kukita Y, Higasa K, Okazaki Y, Yoshinaga A, Hayashi K (2009). \"Estimation of SNP allele frequencies by SSCP analysis of pooled DNA\". Methods in Molecular Biology. Methods in Molecular Biology. 578: 193–207. doi:10.1007/978-1-60327-411-1_12. ISBN 978-1-60327-410-4. PMID 19768595.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":44,"type":"book","title":"Malhis N, Jones SJ, Gsponer J (April 2019). \"Improved measures for evolutionary conservation that exploit taxonomy distances\". Nature Communications. 10 (1): 1556. doi:10.1038/s41467-019-09583-2. PMID 30952844.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":45,"type":"book","title":"\"View of SNPViz - Visualization of SNPs in proteins\". genomicscomputbiol.org (in 英语). Retrieved 2018-10-20.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false}],"recommendReferences":null,"auditState":2,"lemmaLevel":1,"origin":0,"originEnTitle":null,"originZhTitle":null,"pv":3089,"auditType":0,"synonyms":null,"showEditTime":"2019.12.13 18:44","auditors":[{"uid":0,"name":"Ki.κe","pic":"https://web.archive.org/web/20221028214411/https://wx.qlogo.cn/mmopen/vi_32/y67kfr32Doib4wg71Jiau7jVWvharic3nRKgdRRQSl6koeQJCo0GQs2Krw0vwdFRsOWnHIQOwAZsSg5lIkIrFCOcQ/132","introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":"","jobBrief":"","role":0,"roleName":null,"title":"","professionalTitle":null,"phoneNo":null,"editable":true,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false}],"hasZhishiNav":false,"auditInfos":{},"isHistory":false};</script><script crossorigin="anonymous" src="./275.单核苷酸多态性 - 搜狗科学百科_files/aegis.min.js.download"></script><script crossorigin="anonymous" src="./275.单核苷酸多态性 - 搜狗科学百科_files/main_2020092401.js.download"></script><script crossorigin="anonymous" src="./275.单核苷酸多态性 - 搜狗科学百科_files/react.production.min.js.download"></script><script crossorigin="anonymous" src="./275.单核苷酸多态性 - 搜狗科学百科_files/react-dom.production.min.js.download"></script><script crossorigin="anonymous" src="./275.单核苷酸多态性 - 搜狗科学百科_files/jquery-1.11.1.min.js.download"></script><script crossorigin="anonymous" src="./275.单核苷酸多态性 - 搜狗科学百科_files/main_2022062701.js.download"></script><script crossorigin="anonymous" src="./275.单核苷酸多态性 - 搜狗科学百科_files/main_66bbe21.js.download"></script><script crossorigin="anonymous" src="./275.单核苷酸多态性 - 搜狗科学百科_files/react.production.min.js.download"></script><script crossorigin="anonymous" src="./275.单核苷酸多态性 - 搜狗科学百科_files/react-dom.production.min.js.download"></script><script crossorigin="anonymous" src="./275.单核苷酸多态性 - 搜狗科学百科_files/main_edf0f08.js.download"></script>
</body></html>