<!DOCTYPE html>
<!-- saved from url=(0083)https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm -->
<html class="" data-reactroot=""><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><script src="./402.基因突变 - 搜狗科学百科_files/analytics.js.download" type="text/javascript"></script>
<script type="text/javascript">window.addEventListener('DOMContentLoaded',function(){var v=archive_analytics.values;v.service='wb';v.server_name='wwwb-app224.us.archive.org';v.server_ms=290;archive_analytics.send_pageview({});});</script>
<script type="text/javascript" src="./402.基因突变 - 搜狗科学百科_files/bundle-playback.js.download" charset="utf-8"></script>
<script type="text/javascript" src="./402.基因突变 - 搜狗科学百科_files/wombat.js.download" charset="utf-8"></script>
<script type="text/javascript">
  __wm.init("https://web.archive.org/web");
  __wm.wombat("https://baike.sogou.com/kexue/d10402.htm","20221025113755","https://web.archive.org/","web","/_static/",
	      "1666697875");
</script>
<link rel="stylesheet" type="text/css" href="./402.基因突变 - 搜狗科学百科_files/banner-styles.css">
<link rel="stylesheet" type="text/css" href="./402.基因突变 - 搜狗科学百科_files/iconochive.css">
<!-- End Wayback Rewrite JS Include -->
<meta name="save" content="history"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="baidu-site-verification" content="VWGb6TyYx8"><meta content="基因突变 - 搜狗科学百科" name="keywords"><meta content="搜狗科学百科是一部有着平等、协作、分享、自由理念的网络科学全书，为每一个互联网用户创造一个涵盖所有领域知识、服务的中文知识性平台。" name="description"><meta http-equiv="x-dns-prefetch-control" content="on"><meta name="server" baike="235" ip="210" env="online"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025113755/https://cache.soso.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025113755/https://hhy.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025113755/https://pic.baike.soso.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025113755/https://ugc.qpic.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025113755/https://xui.ptlogin2.qq.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025113755/https://q1.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025113755/https://q2.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025113755/https://q3.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025113755/https://q4.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025113755/https://q.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025113755/https://img01.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025113755/https://img02.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025113755/https://img03.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025113755/https://img04.sogoucdn.com/"><link rel="Shortcut Icon" href="https://web.archive.org/web/20221025113755im_/https://www.sogou.com/images/logo/new/favicon.ico?v=4"><link rel="Bookmark" href="https://www.sogou.com/images/logo/new/favicon.ico?v=4"><link href="./402.基因突变 - 搜狗科学百科_files/base_b849887.css" rel="stylesheet"><link href="./402.基因突变 - 搜狗科学百科_files/detail_378aed5.css" rel="stylesheet"><link href="./402.基因突变 - 搜狗科学百科_files/inviteAudit_7894507.css" rel="stylesheet"><link rel="stylesheet" href="./402.基因突变 - 搜狗科学百科_files/highlight.min.css"><title>基因突变 - 搜狗科学百科</title><style>.onekey-close {
	position: absolute;
	top: 16px;
	right: 16px;
	width: 24px;
	height: 24px;
	background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	text-indent: -999em;
	background-size: 84px;
	background-position: -63px 0;
}

.onekey-login {
	position: absolute;
	top: 16.4%;
	left: 0;
	right: 0;
	width: 100%;
}

/* .onekey-login-img {
    width: 75px;
    height: 75px;
    background: url("https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/images/sprite_wap_baike.png") no-repeat;
    background-size: 100px 91px;
    background-position: 0 0;
    background-repeat: no-repeat;
    margin: 0 auto;
} */

.onekey-login-title {
	text-align: center;
	padding-bottom: 3px;
	font-size: 21px;
	font-weight: bold;
	line-height: 30px;
	color: #000;
}

.onekey-login-txt {
	text-align: center;
	font-family: PingFangSC;
	font-size: 14px;
	line-height: 20px;
	color: #8f8f8f;
}

.onekey-login-qq,
.onekey-login-wx,
.onekey-login-phone {
	display: block;
	width: 245px;
	height: 54px;
	border-radius: 45px;
	text-align: center;

	margin: 0 auto;
	font-size: 17px;
	line-height: 24px;
	color: #000;
	/* padding: 16px 77px; */
	border-radius: 12px;
	border: solid 1px #e0e0e0;
}
.onekey-qq-content,
.onekey-vx-content,
.onekey-phone-content {
	display: inline-block;
	margin-top: 16px;
}
.onekey-qq-content {
	padding: 0 5px;
}

.onekey-login-qq {
	margin-top: 48px;
	margin-bottom: 24px;
}

.onekey-login-qq:before {
	display: inline-block;
	content: "";
	width: 20px;
	height: 20px;
	background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	background-size: 80px;
	background-position: -20px 0;
	vertical-align: top;
	margin: 17px 8px 0 0;
}

.onekey-login-wx {
	margin-bottom: 24px;
}

.onekey-login-wx:before {
	display: inline-block;
	content: "";
	width: 21px;
	height: 21px;
	background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	background-size: 84px;
	background-position: 0 0;
	vertical-align: top;
	margin: 17px 10px 0 0;
}

.onekey-login-phone {
}

.onekey-login-phone:before {
	display: inline-block;
	content: "";
	width: 21px;
	height: 21px;
	background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	background-size: 84px;
	background-position: -42px 0;
	vertical-align: top;
	margin: 17px 10px 0 0;
}

.onekey-fixed {
	z-index: 100;
	position: fixed;
	top: 0;
	bottom: 0;
	left: 0;
	right: 0;
	background: #fff;
	width: 100%;
	height: 100%;
}

.onekey-fixed.forbid {
	z-index: 100;
	position: fixed;
	top: auto;
	bottom: 68px;
	left: 9%;
	right: 0;
	background: rgba(0, 0, 0, 0.7);
	width: 82%;
	height: 43px;
	border-radius: 25px;
	color: #ffffff;
}
.onekey-login-title.forbid {
	text-align: center;
	padding-bottom: 3px;
	font-size: 14px;
	font-weight: normal;
	line-height: 30px;
	color: white;
}
</style><style>#login_mask {
  background: #000;
  opacity: 0.5;
  filter: alpha(opacity=50);
  position: fixed;
  /*fixed好像在哪个IE上有BUG，先用用*/
  left: 0;
  top: 0;
  z-index: 999;
  height: 100%;
}

#login_iframe_container {
  position: fixed;
  width: 550px;
  height: 360px;
  z-index: 1020;
  background-color: #ffffff;
}

@media screen and (max-width: 828px) {
  #login_iframe_container {
    top: 50% !important;
    left: 50% !important;
    transform: translate(-50%, -50%);
  }
}

#login_iframe_container.new-login {
  width: 550px;
  height: 360px;
  background-image: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/background_2a4a8a6.png);
}

#login_iframe_container.new-login.no-bg {
  background: #fff;
}

#login_iframe_container.new-login .login-title {
  width: 100%;
  height: 42px;
  line-height: 42px;
  text-align: center;
  font-size: 30px;
  letter-spacing: 0.19px;
  color: #ffffff;
  margin-top: 62px;
}
#login_iframe_container.new-login .forbid-title {
  width: 100%;
  height: 42px;
  line-height: 42px;
  text-align: center;
  font-size: 24px;
  letter-spacing: 0.19px;
  color: #333333;
  margin-top: 150px;
}

#login_iframe_container.new-login.no-bg .login-title {
  color: #333333;
}

#login_iframe_container.new-login .login-subtitle {
  width: 100%;
  height: 18px;
  line-height: 18px;
  font-size: 13px;
  letter-spacing: 0.08px;
  color: #ffffff;
  text-align: center;
  margin-top: 9px;
  margin-bottom: 43px;
}

#login_iframe_container.new-login.no-bg .login-subtitle {
  color: #999999;
}

#login_iframe_container.new-login .login-subtitle::before {
  content: '';
  display: inline-block;
  width: 10px;
  height: 1px;
  background-color: #ffffff;
  position: relative;
  top: -4px;
  left: -5px;
}

#login_iframe_container.new-login .login-subtitle::after {
  content: '';
  display: inline-block;
  width: 10px;
  height: 1px;
  background-color: #ffffff;
  position: relative;
  top: -4px;
  left: 5px;
}

#login_iframe_container.new-login.no-bg .login-subtitle::before {
  background-color: #999999;
}

#login_iframe_container.new-login.no-bg .login-subtitle::after {
  background-color: #999999;
}

#login_iframe_container.new-login .close-btn {
  position: absolute;
  top: 20px;
  right: 20px;
  width: 12px;
  height: 12px;
  background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/login-sprites_e3853e5.png) -59px -10px;
  background-size: 81px 91px;
  cursor: pointer;
}

#login_iframe_container.new-login .login-btn {
  width: 220px;
  height: 47px;
  border-radius: 24px;
  border: solid 1px #dddddd;
  background-color: #ffffff;
  margin: 0 auto;
  margin-top: 28px;
  position: relative;
  display: block;
}

#login_iframe_container.new-login .login-btn .login-icon {
  position: absolute;
}

#login_iframe_container.new-login .login-btn .login-text {
  width: 61px;
  height: 47px;
  line-height: 47px;
  vertical-align: middle;
  font-size: 15px;
  letter-spacing: 0.1px;
  color: #666666;
  position: absolute;
  right: 62px;
}

#login_iframe_container.new-login .login-btn.qq-btn .login-icon {
  width: 22px;
  height: 27px;
  top: 10px;
  left: 67px;
  background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/login-sprites_e3853e5.png) -10px -54px;
  background-size: 81px 91px;
}

#login_iframe_container.new-login .login-btn.qq-btn .login-text {
  right: 59px;
}

#login_iframe_container.new-login .login-btn.wechat-btn .login-icon {
  width: 29px;
  height: 24px;
  top: 12px;
  left: 62px;
  background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/login-sprites_e3853e5.png) -10px -10px;
  background-size: 81px 91px;
}</style><style>/* -- container -- */
.rodal,
.rodal-mask {
    top: 0;
    left: 0;
    width: 100%;
    height: 100%;
    z-index: 100;
}

.rodal {
    position: fixed;
}

/* -- mask -- */
.rodal-mask {
    position: fixed;
    background: rgba(0, 0, 0, .5);
}

/* -- dialog -- */
.rodal-dialog {
    position: absolute;
    z-index: 101;
    background: #fff;
    border-radius: 3px;
    box-shadow: 0 1px 3px rgba(0, 0, 0, .2);
}

.rodal-center {
    top: 50%;
    transform: translateY(-50%);
    left: 0;
    right: 0;
    margin: 0 auto;
}

.rodal-bottom {
    left: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

.rodal-top {
    left: 0;
    right: 0;
    top: 0;
    margin: auto;
}

.rodal-left {
    top: 0;
    left: 0;
    bottom: 0;
    margin: auto;
}

.rodal-right {
    top: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

/* -- close button -- */
.rodal-close {
    position: absolute;
    cursor: pointer;
    top: 16px;
    right: 16px;
    width: 16px;
    height: 16px;
}

.rodal-close:before,
.rodal-close:after {
    position: absolute;
    content: '';
    height: 2px;
    width: 100%;
    top: 50%;
    left: 0;
    margin-top: -1px;
    background: #999;
    border-radius: 100%;
    -webkit-transition: background .2s;
    transition: background .2s;
}

.rodal-close:before {
    -webkit-transform: rotate(45deg);
    transform: rotate(45deg);
}

.rodal-close:after {
    -webkit-transform: rotate(-45deg);
    transform: rotate(-45deg);
}

.rodal-close:hover:before,
.rodal-close:hover:after {
    background: #333;
}

/* -- fade -- */
/* @-webkit-keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

@keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

.rodal-fade-enter {
    -webkit-animation: rodal-fade-enter both ease-in;
    animation: rodal-fade-enter both ease-in;
} */

@-webkit-keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

@keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

.rodal-fade-leave {
    -webkit-animation: rodal-fade-leave both ease-out;
    animation: rodal-fade-leave both ease-out;
}

/* -- zoom -- */
@-webkit-keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-enter {
    -webkit-animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-leave {
    -webkit-animation: rodal-zoom-leave both;
    animation: rodal-zoom-leave both;
}

/* -- slideDown -- */
@-webkit-keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-enter {
    -webkit-animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-leave {
    -webkit-animation: rodal-slideDown-leave both;
    animation: rodal-slideDown-leave both;
}

/* -- slideLeft -- */
@-webkit-keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-enter {
    -webkit-animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-leave {
    -webkit-animation: rodal-slideLeft-leave both;
    animation: rodal-slideLeft-leave both;
}

/* -- slideRight -- */
@-webkit-keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-enter {
    -webkit-animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-leave {
    -webkit-animation: rodal-slideRight-leave both;
    animation: rodal-slideRight-leave both;
}

/* -- slideUp -- */
@-webkit-keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-enter {
    -webkit-animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
    animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
}

@-webkit-keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-leave {
    -webkit-animation: rodal-slideUp-leave both;
    animation: rodal-slideUp-leave both;
}

/* -- flip -- */
@-webkit-keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

@keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

.rodal-flip-enter {
    -webkit-animation: rodal-flip-enter both ease-in;
    animation: rodal-flip-enter both ease-in;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

@-webkit-keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

@keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

.rodal-flip-leave {
    -webkit-animation: rodal-flip-leave both;
    animation: rodal-flip-leave both;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

/* -- rotate -- */
@-webkit-keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-enter {
    -webkit-animation: rodal-rotate-enter both;
    animation: rodal-rotate-enter both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

@-webkit-keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-leave {
    -webkit-animation: rodal-rotate-leave both;
    animation: rodal-rotate-leave both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

/* -- door -- */
@-webkit-keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

@keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

.rodal-door-enter {
    -webkit-animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

@keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

.rodal-door-leave {
    -webkit-animation: rodal-door-leave both;
    animation: rodal-door-leave both;
}</style><style>/* -- container -- */
.rodal,
.rodal-mask {
    top: 0;
    left: 0;
    width: 100%;
    height: 100%;
    z-index: 100;
}

.rodal {
    position: fixed;
}

/* -- mask -- */
.rodal-mask {
    position: fixed;
    background: rgba(0, 0, 0, .5);
}

/* -- dialog -- */
.rodal-dialog {
    position: absolute;
    z-index: 101;
    background: #fff;
    border-radius: 3px;
    box-shadow: 0 1px 3px rgba(0, 0, 0, .2);
}

.rodal-center {
    top: 50%;
    transform: translateY(-50%);
    left: 0;
    right: 0;
    margin: 0 auto;
}

.rodal-bottom {
    left: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

.rodal-top {
    left: 0;
    right: 0;
    top: 0;
    margin: auto;
}

.rodal-left {
    top: 0;
    left: 0;
    bottom: 0;
    margin: auto;
}

.rodal-right {
    top: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

/* -- close button -- */
.rodal-close {
    position: absolute;
    cursor: pointer;
    top: 16px;
    right: 16px;
    width: 16px;
    height: 16px;
}

.rodal-close:before,
.rodal-close:after {
    position: absolute;
    content: '';
    height: 2px;
    width: 100%;
    top: 50%;
    left: 0;
    margin-top: -1px;
    background: #999;
    border-radius: 100%;
    -webkit-transition: background .2s;
    transition: background .2s;
}

.rodal-close:before {
    -webkit-transform: rotate(45deg);
    transform: rotate(45deg);
}

.rodal-close:after {
    -webkit-transform: rotate(-45deg);
    transform: rotate(-45deg);
}

.rodal-close:hover:before,
.rodal-close:hover:after {
    background: #333;
}

/* -- fade -- */
/* @-webkit-keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

@keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

.rodal-fade-enter {
    -webkit-animation: rodal-fade-enter both ease-in;
    animation: rodal-fade-enter both ease-in;
} */

@-webkit-keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

@keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

.rodal-fade-leave {
    -webkit-animation: rodal-fade-leave both ease-out;
    animation: rodal-fade-leave both ease-out;
}

/* -- zoom -- */
@-webkit-keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-enter {
    -webkit-animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-leave {
    -webkit-animation: rodal-zoom-leave both;
    animation: rodal-zoom-leave both;
}

/* -- slideDown -- */
@-webkit-keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-enter {
    -webkit-animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-leave {
    -webkit-animation: rodal-slideDown-leave both;
    animation: rodal-slideDown-leave both;
}

/* -- slideLeft -- */
@-webkit-keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-enter {
    -webkit-animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-leave {
    -webkit-animation: rodal-slideLeft-leave both;
    animation: rodal-slideLeft-leave both;
}

/* -- slideRight -- */
@-webkit-keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-enter {
    -webkit-animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-leave {
    -webkit-animation: rodal-slideRight-leave both;
    animation: rodal-slideRight-leave both;
}

/* -- slideUp -- */
@-webkit-keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-enter {
    -webkit-animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
    animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
}

@-webkit-keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-leave {
    -webkit-animation: rodal-slideUp-leave both;
    animation: rodal-slideUp-leave both;
}

/* -- flip -- */
@-webkit-keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

@keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

.rodal-flip-enter {
    -webkit-animation: rodal-flip-enter both ease-in;
    animation: rodal-flip-enter both ease-in;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

@-webkit-keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

@keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

.rodal-flip-leave {
    -webkit-animation: rodal-flip-leave both;
    animation: rodal-flip-leave both;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

/* -- rotate -- */
@-webkit-keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-enter {
    -webkit-animation: rodal-rotate-enter both;
    animation: rodal-rotate-enter both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

@-webkit-keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-leave {
    -webkit-animation: rodal-rotate-leave both;
    animation: rodal-rotate-leave both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

/* -- door -- */
@-webkit-keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

@keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

.rodal-door-enter {
    -webkit-animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

@keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

.rodal-door-leave {
    -webkit-animation: rodal-door-leave both;
    animation: rodal-door-leave both;
}</style></head><body class=""><!-- BEGIN WAYBACK TOOLBAR INSERT -->
<style type="text/css">
body {
  margin-top:0 !important;
  padding-top:0 !important;
  /*min-width:800px !important;*/
}
</style>
<script>__wm.rw(0);</script>
<div id="wm-ipp-base" lang="en" style="display: block; direction: ltr;">
</div><div id="wm-ipp-print">The Wayback Machine - https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm</div>
<script type="text/javascript">//<![CDATA[
__wm.bt(675,27,25,2,"web","https://baike.sogou.com/kexue/d10402.htm","20221025113755",1996,"/_static/",["/_static/css/banner-styles.css?v=S1zqJCYt","/_static/css/iconochive.css?v=qtvMKcIJ"], false);
  __wm.rw(1);
//]]></script>
<!-- END WAYBACK TOOLBAR INSERT --><script>window._gtag=window._gtag||{};window._gtag.shouldGrayed = false;if ('ed7fc25d87254961977c8cae0a0627d2') window._gtag.traceId = 'ed7fc25d87254961977c8cae0a0627d2';if ({"illegality":true}) window.userInfo = {"illegality":true};</script><div class="topnavbox"><ul class="topnav"><li><a href="https://web.archive.org/web/20221025113755/https://www.sogou.com/web?query=">网页</a></li><li><a href="https://web.archive.org/web/20221025113755/https://weixin.sogou.com/weixin?p=75351201">微信</a></li><li><a href="https://web.archive.org/web/20221025113755/https://zhihu.sogou.com/zhihu?p=75351218">知乎</a></li><li><a href="https://web.archive.org/web/20221025113755/https://pic.sogou.com/pics?query=">图片</a></li><li><a href="https://web.archive.org/web/20221025113755/https://v.sogou.com/v?query=">视频</a></li><li><a href="https://web.archive.org/web/20221025113755/https://mingyi.sogou.com/">医疗</a></li><li class="cur"><strong>科学</strong></li><li><a href="https://web.archive.org/web/20221025113755/https://hanyu.sogou.com/">汉语</a></li><li><a href="https://web.archive.org/web/20221025113755/https://wenwen.sogou.com/">问问</a></li><li><a href="https://web.archive.org/web/20221025113755/https://www.sogou.com/docs/more.htm">更多<span class="topraquo">»</span></a></li></ul></div><div id="header"><div class="header-wrap"><a class="header-logo" href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue"></a><div class="header-search"><div class="querybox" id="suggBox"><form><input id="searchInput" class="query" type="text" placeholder="搜科学领域专业百科词条" name="query" autocomplete="off" value=""><a href="javascript:;" class="query-search"></a></form></div></div><div class="header-rgt"><span class="btn-header-rgt btn-edit" id="editLemma">创建</span><div class="header-user no-login"></div></div></div></div><div class="fixed-placeholder" style="visibility:none"></div><div id="container" class=""><div class="content lemma-level1"><div class="detail-title" id="abstract-title"><h1>基因突变</h1><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#!" class="detail-edit">编辑</a></div><div class="section_content" data-id="54231093351604751"><div class="text_img ed_imgfloat_right"><a class="ed_image_link" data-src="https://img03.sogoucdn.com/app/a/200698/sogou_science_11618" data-bigsrc="https://img03.sogoucdn.com/app/a/200698/sogou_science_11618?width=231&amp;height=174&amp;titlename=%E9%83%81%E9%87%91%E9%A6%99%E8%A1%A8%E7%8E%B0%E5%87%BA%E9%83%A8%E5%88%86%E9%BB%84%E8%89%B2%E8%8A%B1%E7%93%A3%E6%98%AF%E7%94%B1%E4%BA%8E%E5%85%B6%E5%9F%BA%E5%9B%A0%E5%8F%91%E7%94%9F%E4%BA%86%E7%AA%81%E5%8F%98%E3%80%82&amp;w=300&amp;h=200" title="点击查看大图" data-w="300" data-h="200" style="background-image:url(https://web.archive.org/web/20221025113755im_/https://img03.sogoucdn.com/app/a/200698/sogou_science_11618)" href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#!"></a><div class="text_img_title">郁金香表现出部分黄色花瓣是由于其基因发生了突变。</div></div><div><p>在生物学中，<b>突变</b>是指生物体、病毒或染色体外DNA的基因组中的核苷酸序列发生了改变。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_1" class="kx_ref">[1]</a></sup>基因突变包括单个碱基改变所引起的点突变，或多个碱基的缺失、重复和插入。引起突变的原因可以是细胞分裂时遗传基因的复制发生错误、或受化学物质、基因毒性、辐射或病毒的影响。</p>
<p>突变通常会导致细胞运作不正常、或细胞死亡，甚至可以在较高等生物中引发癌症。但同时，突变也被视为演化的“推动力”：不理想的突变会经天择过程被淘汰，而对物种有利的突变则会被累积下去。中性突变对物种没有影响而逐渐累积，会导致间断平衡。</p></div></div><div id="catalog"><h2 class="title2">目录<a href="javascript:" class="detail-edit">编辑</a></h2><div class="catalog_wrap" style=""><ul class="catalog_list col3"><li><span class="order">1</span><a href="javascript:" data-level="1" data-id="14995100639691029">描述</a></li><li><span class="order">2</span><a href="javascript:" data-level="1" data-id="14995100656468239">历史</a></li><li><span class="order">3</span><a href="javascript:" data-level="1" data-id="14995100656468240">原因</a></li><li class="secondary_catalog"><span>3.1 </span><a href="javascript:" data-id="14995100656468240">自发突变</a></li><li class="secondary_catalog"><span>3.2 </span><a href="javascript:" data-id="14995100656468240">易错的复制旁路</a></li><li class="secondary_catalog"><span>3.3 </span><a href="javascript:" data-id="14995100656468240">DNA修复过程中引入的错误</a></li><li class="secondary_catalog"><span>3.4 </span><a href="javascript:" data-id="14995100656468240">诱发突变</a></li><li><span class="order">4</span><a href="javascript:" data-level="1" data-id="14995100656468241">类型分类</a></li></ul><ul class="catalog_list col3"><li class="secondary_catalog"><span>4.1 </span><a href="javascript:" data-id="14995100656468241">根据对结构的影响</a></li><li class="secondary_catalog"><span>4.2 </span><a href="javascript:" data-id="14995100656468241">根据对功能的影响</a></li><li class="secondary_catalog"><span>4.3 </span><a href="javascript:" data-id="14995100656468241">根据对适应性的影响</a></li><li class="secondary_catalog"><span>4.4 </span><a href="javascript:" data-id="14995100656468241">根据对蛋白质序列的影响</a></li><li class="secondary_catalog"><span>4.5 </span><a href="javascript:" data-id="14995100656468241">根据遗传性</a></li><li class="secondary_catalog"><span>4.6 </span><a href="javascript:" data-id="14995100656468241">特殊类型</a></li><li class="secondary_catalog"><span>4.7 </span><a href="javascript:" data-id="14995100656468241">系统命名法</a></li><li><span class="order">5</span><a href="javascript:" data-level="1" data-id="14995100656468242">突变率</a></li></ul><ul class="catalog_list col3"><li><span class="order">6</span><a href="javascript:" data-level="1" data-id="14995100656468243">有害突变</a></li><li><span class="order">7</span><a href="javascript:" data-level="1" data-id="14995100673245449">有益突变</a></li><li><span class="order">8</span><a href="javascript:" data-level="1" data-id="14995100673245450">朊病毒突变</a></li><li><span class="order">9</span><a href="javascript:" data-level="1" data-id="14995100673245451">体细胞突变</a></li><li><span class="order">10</span><a href="javascript:" data-level="1" data-id="14995100673245452">无效突变</a></li><li><span class="order">11</span><a href="javascript:" data-level="1" data-id="14995100673245453">亚效和超效突变</a></li><li><span class="order">12</span><a href="javascript:" data-level="1" data-id="references">参考文献</a></li></ul></div></div><div id="paragraphs"><div><div id="par_14995100639691029"><h2 class="title">1 描述<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>突变可能涉及大部分DNA的复制，通常是通过基因重组。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_2" class="kx_ref">[2]</a></sup> 这些复制是进化新基因的主要原料来源，每百万年动物基因组中就有几十到几百个基因被复制。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_3" class="kx_ref">[3]</a></sup> 大多数基因属于共享祖先的更大基因家族，可通过它们的序列同源性来检测。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_4" class="kx_ref">[4]</a></sup> 新基因通过几种方法产生，通常是通过祖先基因的复制和突变，或者通过重组不同基因的部分以形成具有新功能的新组合。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_5" class="kx_ref">[5]</a></sup><sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_6" class="kx_ref">[6]</a></sup> </p><p>在这里，蛋白质结构域充当模块，每个模块具有特定且独立的功能，可以混合在一起产生编码具有新特性的新蛋白质的基因。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_7" class="kx_ref">[7]</a></sup> 例如，人眼利用四个基因来组成感知光的结构：三个用于视锥细胞或色觉，一个用于视杆细胞或夜视；这四个基因都来自同一个祖先基因。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_8" class="kx_ref">[8]</a></sup> 复制一个基因(甚至整个基因组)的另一个优点是增加了工程冗余；从而允许这对基因中的其中一个获得新的功能，而另一个拷贝执行原始功能。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_9" class="kx_ref">[9]</a></sup><sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_10" class="kx_ref">[10]</a></sup> 其他类型的突变偶尔会从以前的非编码DNA产生新的基因。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_11" class="kx_ref">[11]</a></sup><sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_12" class="kx_ref">[12]</a></sup> </p><p>染色体数目的变化可能涉及更大的突变，染色体内的DNA片段断裂然后重新排列。例如，在人亚科，两条染色体融合产生了人类第2号染色体；这种融合并没有发生在其他猿类的谱系中，它们保留了这些独立的染色体。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_13" class="kx_ref">[13]</a></sup> 在进化中，这种染色体重排最重要的作用可能是通过使种群不太可能杂交来加速种群向新物种的分化，从而保持这些种群之间的遗传差异。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_14" class="kx_ref">[14]</a></sup> </p><p>可以在基因组中移动的DNA序列，例如转座子，构成了植物和动物遗传物质的主要部分，并且在基因组的进化中可能起了重要作用。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_15" class="kx_ref">[15]</a></sup> 例如，人类基因组中存在100多万份Alu序列，这些序列现在已经被招募来执行诸如调节基因表达的功能。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_16" class="kx_ref">[16]</a></sup> 这些可移动的DNA序列的另一个作用是，当它们在基因组中移动时，它们可以突变或删除现有的基因，从而产生遗传多样性。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_17" class="kx_ref">[17]</a></sup> </p><p>非致死突变在基因库中积累，并增加遗传变异的数量。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_17" class="kx_ref">[17]</a></sup> 基因库中一些基因变化的丰度可以通过自然选择而减少，而其他“更有利”的突变可能积累并导致适应性变化。</p><p></p><div class="text_img ed_imgfloat_right">
            <a class="ed_image_link lazyLoad" data-src="https://img02.sogoucdn.com/app/a/200698/sogou_science_11620" data-bigsrc="" title="点击查看大图" href="javascript:" data-observer="true"></a>
            <div class="text_img_title">Prodryas persephone 是一种晚始新世蝴蝶品种。</div>   
        </div> <p></p><p>例如，蝴蝶可能会产生带有新突变的后代。这些突变中的大多数都没有效果；但是人们可能会改变蝴蝶后代的颜色，使捕食者更难(或更容易)看到。如果这种颜色的变化是有利的，这种蝴蝶存活和繁殖后代的机率会稍微高一点，随着时间的推移，这种突变的蝴蝶数量可能会占总数量的更大比例。</p><p>中性突变被定义为其效应不影响个体适应性的突变。由于遗传漂变，这些频率会随着时间的推移而增加。据信绝大多数突变对生物体的适应性没有显著影响。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_18" class="kx_ref">[18]</a></sup> 此外，DNA修复机制能够在大部分变化成为永久突变之前修复它们，并且许多生物体具有消除体细胞非永久突变的机制。</p><p>有益的突变可以提高生殖成功率。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_19" class="kx_ref">[19]</a></sup><sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_20" class="kx_ref">[20]</a></sup> </p></div></div><div id="par_14995100656468239"><h2 class="title">2 历史<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p></p><div class="text_img ed_imgfloat_right">
            <a class="ed_image_link lazyLoad" data-src="https://img01.sogoucdn.com/app/a/200698/sogou_science_11621" data-bigsrc="" title="点击查看大图" href="javascript:" data-observer="true"></a>
            <div class="text_img_title">雨果·德·弗里斯画了一幅月见草的画，这种植物显然是在他的实验中发生了大量突变而产生了新的形态。 由Thérèse Schwartze于1918所作。</div>   
        </div> <p></p><p>突变论是查尔斯·达尔文1859年著作《<i>物种起源》</i>出版前后存在的几种关于自然选择进化的理论之一， 。在理论上，突变是新事物的来源，创造新的形式和新的物种，可能是瞬间的，<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_21" class="kx_ref">[21]</a></sup> 跳跃性的。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_22" class="kx_ref">[22]</a></sup> 突变被认为推动进化，进化受突变的限制。</p><p>在达尔文之前，生物学家通常相信跳跃论，即大规模进化跳跃的可能性，包括直接的物种形成。例如，1822年，艾蒂安·若弗鲁瓦·圣伊莱尔认为物种可以通过突然的转变，或者后来被称为大分子突变来形成。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_23" class="kx_ref">[23]</a></sup> 达尔文反对跳跃，坚持进化论和地质学中的渐进主义。1864年，阿尔伯特·冯·科立克复活了杰弗里的理论。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_24" class="kx_ref">[24]</a></sup> 1901年，遗传学家雨果·德·弗里斯给在月见草（<i>Oenothera lamarckiana）</i>上的实验中突然出现的看似新的形式命名为“突变”。在20世纪的第一个十年，突变论，或德弗里斯称之为 <i>突变理论</i>,<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_25" class="kx_ref">[25]</a></sup><sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_21" class="kx_ref">[21]</a></sup> 成为在一段时间内受到包括威廉·贝特森<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_26" class="kx_ref">[26]</a></sup> ，托马斯·亨特·摩尔根和雷金纳德·普内特。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_27" class="kx_ref">[27]</a></sup><sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_21" class="kx_ref">[21]</a></sup>在内的遗传学家所支持的达尔文主义的竞争对手。</p><p>20世纪中期，现代综合论的支持者将早期突变学家描绘成达尔文进化论的反对者和生物测定学派（biometrics school）的反对者，认为选择依赖于连续的变异，这给对突变论的理解蒙上了阴影。在这幅描绘中，突变论被遗传学和自然选择的综合论所击败，据说这种综合论始于后来，大约1918年，数学家罗纳德·费雪的工作。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_28" class="kx_ref">[28]</a></sup><sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_29" class="kx_ref">[29]</a></sup><sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_30" class="kx_ref">[30]</a></sup><sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_31" class="kx_ref">[31]</a></sup> 然而，孟德尔遗传学和自然选择的一致性早在1902年就开始了，<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_32" class="kx_ref">[32]</a></sup> 并在欧洲和美国建立了理论和实验工作。尽管存在争议，早期的突变学家到1918年已经接受了自然选择，并解释了由于多个基因作用于同一特征(如身高)而导致的连续变异。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_29" class="kx_ref">[29]</a></sup><sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_30" class="kx_ref">[30]</a></sup> </p><p>突变论，以及其他达尔文主义理论，如拉马克主义和直生论，被大多数生物学家所抛弃，因为他们发现孟德尔遗传学和自然选择可以相互兼容；突变成为自然选择所必需的遗传变异的来源。 然而，突变论并没有完全消失。1940年，理查德·戈尔德施米特(Richard Goldschmidt)再次主张通过大分子突变进行单步物种形成，将由此产生的生物体描述为“充满希望的怪物”，然而受到广泛嘲笑。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_33" class="kx_ref">[33]</a></sup><sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_34" class="kx_ref">[34]</a></sup> 1987年，根井正利争论说进化通常是有限的突变。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_35" class="kx_ref">[35]</a></sup> 现代生物学家道格拉斯·富图伊玛(Douglas J. Futuyma)认为，基本上所有关于由大突变驱动的进化的主张都可以用达尔文进化论来解释。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_36" class="kx_ref">[36]</a></sup> </p></div></div><div id="par_14995100656468240"><h2 class="title">3 原因<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>四类突变是(1)自发突变(分子衰变)，(2)由于自然发生的DNA损伤的易错复制旁路引起的突变(也称为易错的跨损伤DNA复制)，(3)在DNA修复过程中引入的错误，和(4)由诱变剂引起的突变。为了科学实验的目的，科学家也可以通过DNA操作故意引入突变序列。</p><p>2017年的一项研究声称，66%的致癌突变是随机的，29%是由环境造成的(研究人群跨越69个国家)，5%是遗传的。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_37" class="kx_ref">[37]</a></sup> </p><p>人类平均给他们的孩子传递60个新的突变，但是父亲根据他们的年龄传递更多的突变，每年给一个孩子增加两个新的突变。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_38" class="kx_ref">[38]</a></sup> </p><p></p><h3>3.1 <span>自发突变</span></h3><p></p><p><i>自发突变</i> 即使给定一个健康的、未受污染的细胞，也以非零概率发生。它们可以用特定的变化来描述:<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_39" class="kx_ref">[39]</a></sup> </p><p>
 </p><ul>
  <li>互变异构现象——氢原子的重新定位改变了碱基，改变了碱基的氢键模式，导致复制过程中碱基配对不正确。</li>
  <li>脱嘌呤——嘌呤碱基(A或G)的缺失形成一个无嘌呤位点(AP位点)。</li>
  <li>脱氨基——水解将正常碱基变为含有酮基取代原始胺基的非典型碱基。例子包括C → U和A → HX(次黄嘌呤)，这可以通过DNA修复机制来纠正；和5MeC(5-甲基胞嘧啶)→ T，这是不太可能被检测为突变，因为胸腺嘧啶是正常的DNA碱基。</li>
  <li>滑动链错配——在复制过程中，新链从模板变性，然后在不同的位点复性(“滑动”)。这可能导致插入或删除。</li>
  <li>复制滑移</li>
 </ul><p></p><p></p><h3>3.2 <span>易错的复制旁路</span></h3><p></p><p>越来越多的证据表明，大多数自发产生的突变是由于模板链过去的DNA损伤导致的易错复制(跨损伤DNA复制)。自然发生的氧化DNA损伤在人类中每天每细胞至少发生10，000次，在大鼠中每天每细胞至少发生50，000次或更多次。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_40" class="kx_ref">[40]</a></sup> 在小鼠中，大多数突变是由跨损伤DNA复制引起的。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_41" class="kx_ref">[41]</a></sup> 同样，在酵母中，Kunz等人<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_42" class="kx_ref">[42]</a></sup> 发现超过60%的自发单碱基对替换和缺失是由跨损伤DNA复制引起的。</p><p></p><h3>3.3 <span>DNA修复过程中引入的错误</span></h3><p></p><p>虽然天然存在的双链断裂在DNA中发生的频率相对较低，但它们的修复通常会导致突变。非同源性末端接合(NHEJ)是修复双链断裂的主要途径。NHEJ涉及去除几个核苷酸，以允许重新连接的两端稍微不准确的对齐，然后添加核苷酸来填补空白。因此，NHEJ经常引入突变。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_43" class="kx_ref">[43]</a></sup> </p><p></p><div class="text_img ed_imgfloat_right">
            <a class="ed_image_link lazyLoad" data-src="https://img02.sogoucdn.com/app/a/200698/sogou_science_11628" data-bigsrc="" title="点击查看大图" href="javascript:" data-observer="true"></a>
            <div class="text_img_title">苯并[a]芘代谢物与DNA的共价化合物，苯并[a]芘是烟草烟雾中的主要诱变剂[8]</div>   
        </div> <p></p><p></p><h3>3.4 <span>诱发突变</span></h3><p></p><p>诱导突变是基因在与诱变剂和环境因素接触后的改变。</p><p><i>诱发突变</i> 在分子水平上可以由以下原因引起:</p><p>
 </p><ul>
  <li>化学药品
   <ul>
    <li>羟胺</li>
    <li>碱基类似物(例如，溴化去氧尿苷(BrdU))</li>
    <li>烷化剂(例如， <i>N</i>-乙基-<i>N</i>-亚硝脲(ENU))。这些物质可以突变复制中和非复制中的DNA。相比之下，碱基类似物只有与复制中的DNA接触时才能使其突变。这些化学诱变剂中的每一种都有一定的作用，然后导致碱基转换、碱基颠换或缺失。</li>
    <li>形成DNA加合物的试剂(如赭曲霉毒素A)<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_44" class="kx_ref">[44]</a></sup></li>
    <li>DNA嵌入剂(例如溴化乙锭)</li>
    <li>DNA交联剂</li>
    <li>氧化损伤</li>
    <li>亚硝酸将A和C上的胺基转化为重氮基，改变它们的氢键模式，导致复制过程中不正确的碱基配对。</li>
   </ul></li>
  <li>辐射
   <ul>
    <li>紫外线（UV）(非电离辐射)。DNA中的两个核苷酸碱基——胞嘧啶和胸腺嘧啶——最容易受到能改变其性质的辐射。紫外光可以诱导DNA链中相邻的嘧啶碱基共价结合成嘧啶二聚体。紫外线辐射，尤其是长波紫外线，也能对DNA造成氧化损伤。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_45" class="kx_ref">[45]</a></sup></li>
    <li>电离辐射。暴露于电离辐射，如伽马辐射，会导致突变，可能导致癌症或死亡。</li>
   </ul></li>
 </ul><p></p></div></div><div id="par_14995100656468241"><h2 class="title">4 类型分类<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p></p><h3>4.1 <span>根据对结构的影响</span></h3><p></p><p></p><div class="text_img ed_imgfloat_right">
            <a class="ed_image_link lazyLoad" data-src="https://img01.sogoucdn.com/app/a/200698/sogou_science_11634" data-bigsrc="" title="点击查看大图" href="javascript:" data-observer="true"></a>
            <div class="text_img_title" style="text-align: center;">染色体的5种突变类型。</div>   
        </div> <p></p><p></p><div class="text_img ed_imgfloat_right">
            <a class="ed_image_link lazyLoad" data-src="https://img02.sogoucdn.com/app/a/200698/sogou_science_11636" data-bigsrc="" title="点击查看大图" href="javascript:" data-observer="true"></a>
            <div class="text_img_title">氨基酸遗传密码标准表中致病突变的选择。[1]</div>   
        </div> <p></p><p>基因的序列可以通过多种方式改变。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_46" class="kx_ref">[46]</a></sup> 基因突变对适应性有不同的影响，这取决于它们发生的位置以及它们是否改变了必需蛋白质的功能。基因结构的突变可以分为几种类型。</p><p><strong>小规模突变</strong></p><p>小规模突变影响一个基因的一个或几个核苷酸。(如果只有一个核苷酸受到影响，它们被称为点突变。)小规模突变包括:</p><p>
 </p><ul>
  <li>插入将一个或多个额外的核苷酸添加到DNA中。它们通常是由转座因子或重复因子复制过程中的错误引起的。基因编码区的插入可以改变mRNA的剪接(剪接位点突变)，或者引起阅读框的改变(移码)，这两者都可以显著改变基因产物。插入可以通过切除转位因子来逆转。</li>
  <li>缺失会从DNA中移除一个或多个核苷酸。像插入一样，这些突变可以改变基因的阅读框。一般来说，它们是不可逆的:虽然理论上完全相同的序列可以通过插入来恢复，但是转座因子能够在<i>任何</i>位置恢复非常短的缺失(比如1-2个碱基) 要么极不可能存在，要么根本不存在。</li>
  <li>替换突变，通常由化学物质或DNA复制错误引起，用一个核苷酸替换另一个核苷酸。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_47" class="kx_ref">[47]</a></sup> 这些变化被分类为碱基置换或碱基颠换。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_48" class="kx_ref">[48]</a></sup> 最常见的是嘌呤与嘌呤(A↔G)或嘧啶与嘧啶(C↔T)的转换。碱基置换可以由亚硝酸、碱基错配或诱变碱基类似物如BrdU引起。不太常见的是碱基颠换，它将嘌呤交换为嘧啶或嘧啶交换为嘌呤(C/T ↔ A/G)。碱基颠换的一个例子是腺嘌呤(A)转化为胞嘧啶(C)。点突变是基因中单个碱基对或其他小碱基对的修饰。点突变可以通过另一个点突变来逆转，其中核苷酸被改变回其原始状态(真正的逆转)或通过第二位点逆转(在其他地方导致恢复基因功能的互补突变)。如下所述，发生在基因蛋白质编码区的点突变可分为同义或非同义替换，后者又可分为错义或无义突变。</li>
 </ul><p></p><p><strong>大规模突变</strong></p><p>染色体结构的大规模突变包括:</p><p>
 </p><ul>
  <li>扩增(或基因复制)导致所有染色体区域的多个拷贝，增加了位于其中的基因的剂量。</li>
  <li>大染色体区域的缺失，导致这些区域内的基因丢失。</li>
  <li>其作用是将先前分离的DNA片段并置，潜在地将分离的基因聚集在一起形成功能不同的融合基因(例如bcr-abl)。</li>
  <li>被称为染色体重排的染色体结构的大规模变化会导致适应性下降，但也会导致分离的近交群体中的物种形成。这些包括:
   <ul>
    <li>染色体易位:非同源染色体遗传部分的交换。</li>
    <li>染色体反转:反转染色体片段的方向。</li>
    <li>非同源染色体互换。</li>
    <li>间隙缺失:染色体内的缺失，从单个染色体上除去一段DNA，从而与先前较远的基因并置。例如，从人类星形细胞瘤(一种脑肿瘤)分离的细胞被发现缺失了胶质母细胞瘤(FIG)基因和受体酪氨酸激酶(ROS)之间的一段序列，并产生融合蛋白(FIG-ROS)。异常的FIG-ROS融合蛋白具有持续激活的激酶活性，可引起致癌转化(从正常细胞转化为癌细胞)。</li>
   </ul></li>
  <li>杂合性丧失:在以前有两个不同等位基因的生物体中，由于缺失或基因重组事件而丧失一个等位基因。</li>
 </ul><p></p><p></p><h3>4.2 <span>根据对功能的影响</span></h3><p></p><p>
 </p><ul>
  <li>功能丧失突变，也称为失活突变，导致基因产物具有较少或没有功能(部分或全部失活)。当等位基因完全丧失功能(零等位基因)时，在穆勒的突变模式中，它通常被称为无效或无义突变。与这种突变相关的表型通常是隐性的。例外情况是当生物体是单倍体时，或者当正常基因产物的减少剂量不足以形成正常表型时(这被称为单倍剂量不足)。</li>
  <li>功能获得突变，也称为激活突变，改变基因产物，使其效果变得更强(增强激活)，甚至被不同的异常功能所取代。当新的等位基因产生时，含有新产生的等位基因以及原始等位基因的杂合子表达新的等位基因；遗传上，这将突变定义为显性表型。穆勒的几种突变中对应于功能增益，包括超效等位基因和新效等位基因。2017年12月，美国政府解除了2014年实施的一项临时禁令，该禁令禁止联邦资助任何新的“功能增强”实验，以增强病原体“如流感禽流感病毒、非典病毒和中东呼吸系统综合症病毒或MER病毒”<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_49" class="kx_ref">[49]</a></sup></li>
  <li>显性负突变(也称为反突变)的基因产物发生改变，与野生型等位基因产生对抗作用。这些突变通常导致分子功能改变(通常不活跃)，并以显性或半显性表型为特征。在人类中，显性负突变与癌症有关(例如，基因p53的突变，<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_50" class="kx_ref">[50]</a></sup> ATM，<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_51" class="kx_ref">[51]</a></sup> CEBPA<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_52" class="kx_ref">[52]</a></sup> 和PPARgamma<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_53" class="kx_ref">[53]</a></sup>)中。马方综合征是由 <i>FBN1</i> 基因，位于染色体15上，编码原纤维蛋白-1，细胞外基质的糖蛋白成分。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_54" class="kx_ref">[54]</a></sup> 马方综合征也是显性负突变和单倍剂量不足的例子。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_55" class="kx_ref">[55]</a></sup><sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_56" class="kx_ref">[56]</a></sup></li>
  <li>穆勒模式分类中，亚效突变的特征是基因产物发生改变，与野生型等位基因相比，基因表达降低。</li>
  <li>新效突变的特征是控制新蛋白质产物的合成。</li>
  <li>致死突变是导致携带突变的生物体死亡的突变。</li>
  <li>反向突变或逆转是恢复原始序列并因此恢复原始表型的点突变。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_57" class="kx_ref">[57]</a></sup></li>
 </ul><p></p><p></p><h3>4.3 <span>根据对适应性的影响</span></h3><p></p><p>在应用遗传学中，通常说突变是有害的还是有益的。</p><p>
 </p><ul>
  <li>有害或有害的突变会降低有机体的适应性。</li>
  <li>有益或有利的突变增加了有机体的适应性。</li>
  <li>中性突变对生物体没有有害或有益的影响。这种突变以稳定的速度发生，形成了分子钟的基础。在分子进化的中性学说，中性突变提供了遗传漂变作为分子水平上大多数变异的基础。</li>
  <li>近中性突变是一种可能稍微有害或有利的突变，尽管大多数近中性突变是轻微有害的。</li>
 </ul><p></p><p><strong>适应性效应的分布</strong></p><p>已经尝试使用诱变实验和应用于分子序列数据的理论模型来推断适应性效应（DFE）的分布。DFE，用于确定不同类型突变(即，强烈有害、接近中性或有利)的相对丰度，与许多进化问题相关，例如遗传变异的维持，<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_58" class="kx_ref">[58]</a></sup> 基因组衰退的速度，<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_59" class="kx_ref">[59]</a></sup> 维持异交有性生殖，而不是近亲繁殖<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_60" class="kx_ref">[60]</a></sup> 和基因重组的进化。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_61" class="kx_ref">[61]</a></sup> 总之，DFE在预测进化动态中起着重要作用。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_62" class="kx_ref">[62]</a></sup><sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_63" class="kx_ref">[63]</a></sup> 已经使用了各种方法来研究DFE，包括理论、实验和分析方法。</p><p>
 </p><ul>
  <li>诱变实验:研究DFE的直接方法是诱导突变，然后测量突变适应性效应，这已经应用于病毒、细菌、酵母和 <i>果蝇</i>。例如，对病毒中DFE的大多数研究使用定点突变来产生点突变并测量每个突变的相对适应性。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_64" class="kx_ref">[64]</a></sup><sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_65" class="kx_ref">[65]</a></sup><sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_66" class="kx_ref">[66]</a></sup><sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_67" class="kx_ref">[67]</a></sup> 在<i>大肠杆菌</i>中的一项研究，使用转座子诱变直接测量随机插入Tn10衍生物的适应度。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_68" class="kx_ref">[68]</a></sup> 在酵母中，已经开发了一种组合诱变和深度测序方法来产生高质量的系统突变体库，并以高通量测量其适应性。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_69" class="kx_ref">[69]</a></sup> 然而，鉴于许多突变的影响太小而无法检测<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_70" class="kx_ref">[70]</a></sup> 诱变实验只能检测到中等效果的突变；DNA序列数据分析可以提供关于这些突变的有价值的信息。</li>
 </ul><p></p><p></p><div class="text_img ed_imgfloat_right">
            <a class="ed_image_link lazyLoad" data-src="https://img02.sogoucdn.com/app/a/200698/sogou_science_11639" data-bigsrc="" title="点击查看大图" href="javascript:" data-observer="true"></a>
            <div class="text_img_title">水疱性口炎病毒突变的适应性效应(DFE)分布。在本实验中，通过定点诱变将随机突变引入病毒，并将每个突变体的适应性与祖先类型进行比较。适应度分别小于1和1以上，表明突变是致命的、有害的、中性的和有利的。[20]</div>   
        </div> <p></p><p>
 </p><ul>
  <li>分子序列分析:随着DNA测序技术的快速发展，可以获得大量的DNA序列数据，将来还会有更多的数据。已经开发了各种方法来从DNA序列数据推断DFE。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_71" class="kx_ref">[71]</a></sup><sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_72" class="kx_ref">[72]</a></sup><sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_73" class="kx_ref">[73]</a></sup><sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_74" class="kx_ref">[74]</a></sup> 通过检测物种内部和物种之间的DNA序列差异，我们能够推断中性、有害和有利突变的DFE的各种特征。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_75" class="kx_ref">[75]</a></sup> 具体来说，DNA序列分析方法允许我们估计突变的影响，而这种影响非常小，很难通过诱变实验检测出来。</li>
 </ul><p></p><p>适应性效应分布的最早理论研究之一是由木村资生完成的，他是一位有影响力的理论群体遗传学家。他的分子进化中立原理提出，大多数新突变都是非常有害的，一小部分是中性的。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_75" class="kx_ref">[75]</a></sup><sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_76" class="kx_ref">[76]</a></sup> 藤原皓明石最近提出了一个双峰DFE模型，双峰集中在高度有害和中性突变。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_76" class="kx_ref">[76]</a></sup> 两种理论都认为绝大多数新突变是中性的或有害的，有利的突变很少，这得到了实验结果的支持。一个例子是对水泡性口炎病毒随机突变的DFE的研究。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_64" class="kx_ref">[64]</a></sup> 在所有突变中，39.6%是致命的，31.2%是非致命有害的，27.1%是中性的。另一个例子来自酵母的高通量诱变实验。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_69" class="kx_ref">[69]</a></sup> 在这个实验中，显示了整个DFE是双峰的，带有一簇中性突变和广泛分布的有害突变。</p><p>尽管相对较少的突变是有利的，但它们在进化变化中起重要作用。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_77" class="kx_ref">[77]</a></sup> 像中性突变一样，弱选择的有利突变可能由于随机遗传漂变而丢失，但是强选择的有利突变更有可能被固定。了解有利突变的DFE可能会提高预测进化动态的能力。约翰·吉莱斯皮<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_78" class="kx_ref">[78]</a></sup>和艾伦·奥尔<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_79" class="kx_ref">[79]</a></sup>对有利突变的DFE进行了理论研究。 他们提出有利突变的分布在广泛的条件下应该是指数的，这通常得到实验研究的支持，至少对于强选择的有利突变是如此。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_80" class="kx_ref">[80]</a></sup><sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_81" class="kx_ref">[81]</a></sup><sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_82" class="kx_ref">[82]</a></sup> </p><p>一般来说，大多数突变是中性或有害的，有利的突变很少；然而，突变类型的比例因物种而异。这表明了两点:第一，由于对有效群体大小的依赖，有效中性突变的比例可能因物种而异；第二，有害突变的平均效应在不同物种间差异很大。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_75" class="kx_ref">[75]</a></sup> 此外，编码区和非编码区的DFE也不同，非编码DNA的DFE包含更弱选择的突变。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_75" class="kx_ref">[75]</a></sup> </p><p></p><h3>4.4 <span>根据对蛋白质序列的影响</span></h3><p></p><p>
 </p><ul>
  <li>移码突变是一种由插入或缺失一些核苷酸引起的突变，这些核苷酸不是3的整数倍。由于密码子基因表达的三重性，插入或缺失会破坏阅读框或密码子的分组，导致与原始翻译完全不同的翻译。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_83" class="kx_ref">[83]</a></sup> 序列中缺失或插入发生得越早，产生的蛋白质变化就越大。(例如，密码子CCU GAC UAC CUA编码脯氨酸、天冬氨酸、酪氨酸和亮氨酸。 如果CCU中的U被删除，得到的序列将是CCG ACU ACC UAx，它将编码脯氨酸、苏氨酸、苏氨酸和另一个氨基酸的一部分，或者可能是终止密码子(其中x代表下一个核苷酸)。)相比之下，任何可被3整除的插入或删除都称为<i>整码突变</i>。</li>
  <li>点替换突变导致单个核苷酸的变化，可以是同义的，也可以是非同义的。
   <ul>
    <li>同义替换是用编码相同氨基酸的另一个密码子替换原密码子，从而产生的氨基酸序列没有被修改。同义突变是由于遗传密码的简并性而发生的。如果这种突变没有导致任何表型效应，那么它被称为沉默，但不是所有同义替换都是沉默的。(编码区之外的核苷酸，如内含子，也可能有沉默突变，因为准确的核苷酸序列并不像编码区那样重要，但它们不被认为是同义替换。)</li>
    <li>非同义替换是用编码不同氨基酸的另一个密码子替换原密码子，从而改变产生的氨基酸序列。非同义替换可分为无义或错义突变:
     <ul>
      <li>错义突变改变核苷酸，导致不同氨基酸的替换。这反过来会导致产生的蛋白质不起作用。这种突变是导致大疱性表皮松解症、镰状细胞病和SOD1介导的肌萎缩侧索硬化症等疾病的原因。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_84" class="kx_ref">[84]</a></sup> 另一方面，如果错义突变出现在氨基酸密码子中，导致使用不同但化学性质相似的氨基酸，那么有时蛋白质中很少或没有变化。例如，从AAA到AGA的变化将编码精氨酸，一种与预期赖氨酸化学相似的分子。在后一种情况下，突变对表型影响很小或没有影响，因此是中性的。</li>
      <li>无义突变是导致DNA序列中突变为提前终止密码子的点突变，或<i>无义密码子</i>在转录的mRNA中，可能是截短的，通常是无功能的蛋白质产物。这种突变与不同的突变有关，如先天性肾上腺增生。(见终止密码子。)</li>
     </ul></li>
   </ul></li>
 </ul><p></p><p></p><h3>4.5 <span>根据遗传性</span></h3><p></p><p></p><div class="text_img ed_imgfloat_right">
            <a class="ed_image_link lazyLoad" data-src="https://img04.sogoucdn.com/app/a/200698/sogou_science_11642" data-bigsrc="" title="点击查看大图" href="javascript:" data-observer="true"></a>
            <div class="text_img_title">一种突变导致这种苔藓玫瑰植物产生不同颜色的花。这是一种体细胞突变，也可能在生殖细胞中传播。</div>   
        </div> <p></p><p>在具有专门生殖细胞的多细胞生物中，突变可细分为种系突变和体细胞突变(也称为获得性突变)。种系突变可通过生殖细胞传递给后代。 体细胞突变涉及专用生殖组之外的细胞，并且通常不传播给后代。</p><p>种系突变在后代中产生<i>天生突变</i>，即存在于每个细胞中的突变。一个天生突变也可能在受精后很快发生，或者从父母先前的天生突变延续下来。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_85" class="kx_ref">[85]</a></sup> </p><p>种系突变和体细胞突变之间的区别对于有专门种系产生生殖细胞的动物来说很重要。然而，这对于理解植物突变的效应没有什么价值，因为植物缺乏专门的生殖系。在那些通过出芽等机制无性繁殖的动物中，这种区别也很模糊，因为产生子代生物体的细胞也会产生该生物体的生殖系。一个不是从双亲遗传的新种系突变称为 <i>de novo</i> 突变。</p><p>二倍体生物(如人类)包含每个基因的两个拷贝——一个父系等位基因和一个母系等位基因。根据每条染色体上突变的发生情况，我们可以将突变分为三种类型。</p><p>
 </p><ul>
  <li>杂合子突变是只有一个等位基因发生突变。</li>
  <li>纯合突变是父系和母系等位基因发生了相同的突变。</li>
  <li>复合杂合突变或遗传复合由来自两种不同的父系和母系等位基因的突变组成。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_86" class="kx_ref">[86]</a></sup></li>
 </ul><p></p><p>野生型或纯合非突变生物是指两个等位基因都没有发生突变的生物。</p><p></p><h3>4.6 <span>特殊类型</span></h3><p></p><p>
 </p><ul>
  <li>条件突变是指在某些“许可”环境条件下具有野生型(或不太严重)表型，而在某些“限制性”条件下具有突变表型的突变。例如，温度敏感突变可在高温(限制性条件)下导致细胞死亡，但在低温(许可条件)下可能没有有害后果。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_87" class="kx_ref">[87]</a></sup> 这些突变是非自主的，因为它们的表现取决于某些条件的存在，与其他自主出现的突变不同。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_88" class="kx_ref">[88]</a></sup> 许可条件可以是温度，<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_89" class="kx_ref">[89]</a></sup> 某些化学物质，<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_90" class="kx_ref">[90]</a></sup> 光线<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_90" class="kx_ref">[90]</a></sup> 或者基因组其他部分的突变。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_88" class="kx_ref">[88]</a></sup> <i>体内</i>机制如转录开关等可以产生条件突变。例如，类固醇结合结构域的结合可以创建转录开关，该转录开关基于类固醇配体的存在来改变基因的表达。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_91" class="kx_ref">[91]</a></sup> 条件突变在研究中已经有应用，因为它们允许控制基因的表达。这对于研究成年人的疾病特别有用，因为它允许在一定的生长时期后进行表达，从而消除了在模式生物发育阶段看到的基因表达的有害影响。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_90" class="kx_ref">[90]</a></sup> DNA重组酶系统，如Cre-Lox重组，与在特定条件下激活的启动子结合使用，可以产生条件突变。双重组酶技术可用于诱导多种条件突变，以研究多种基因同时突变导致的疾病。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_90" class="kx_ref">[90]</a></sup> 已经鉴定出某些内含物仅在特定许可温度下剪接，导致蛋白质合成不当，从而在其他温度下丧失功能突变。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_92" class="kx_ref">[92]</a></sup> 条件突变也可以用于与衰老相关的基因研究，因为在生物体生命周期的某个时间段后，其表达会发生变化。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_89" class="kx_ref">[89]</a></sup></li>
  <li>复制时间数量性状位点影响DNA的复制。</li>
 </ul><p></p><p></p><h3>4.7 <span>系统命名法</span></h3><p></p><p>为了对突变进行分类，必须从“正常”或“健康”生物体(相对于“突变”或“病态”生物体)的DNA中获得“正常”序列，并对其进行鉴定和报道；理想的情况是，它应该可以公共获取以方便进行明确的核苷酸与核苷酸之间的比较，并得到科学界或一组遗传学家和生物学家等专家的公认，他们有责任建立<i>标准</i>或者所谓的“共识”序列。这一步需要巨大的科学努力。一旦知道了公认序列，基因组中的突变就可以被精确定位、描述和分类。人类基因组变异学会委员会（HGVS）已经开发了标准的人类序列变异命名法，<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_93" class="kx_ref">[93]</a></sup> 这应该被研究者和DNA诊断中心用来产生明确的突变描述。原则上，这个术语也可以用来描述其他生物体的突变。命名法规定了突变的类型和碱基或氨基酸的变化。</p><p>
 </p><ul>
  <li>核苷酸替换(例如，76A &gt; T)-数字是核苷酸从5’端开始的位置；第一个字母代表野生型核苷酸，第二个字母代表替换野生型核苷酸的核苷酸。在给定的例子中，第76位的腺嘌呤被胸腺嘧啶替换。
   <ul>
    <li>如果有必要区分基因组DNA、线粒体DNA和RNA的突变，可以使用简单的约定。例如，如果核苷酸序列的第100个碱基从G突变为C，那么如果突变发生在基因组DNA中，则记为G . 100G &gt; C；如果突变发生在线粒体DNA中，则记为m . 100G &gt; C；如果突变发生在RNA中，则记为r.100g&gt;c。注意，对于RNA的突变，核苷酸是用小写字母写的。</li>
   </ul></li>
  <li>氨基酸替换(例如，D111E)-第一个字母是野生型氨基酸的单字母代码，数字是来自N端氨基酸的位置，第二个字母是突变中存在的氨基酸的单字母代码。无义突变第二个氨基酸用X表示(例如D111X)。</li>
  <li>氨基酸缺失(如，ΔF508)——希腊字母Δ(delta)表示缺失。字母指的是野生型氨基酸，数字是野生型氨基酸在N端的位置。</li>
 </ul><p></p></div></div><div id="par_14995100656468242"><h2 class="title">5 突变率<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>不同物种的突变率差异很大，通常决定突变的进化力是正在研究中的主题</p></div></div><div id="par_14995100656468243"><h2 class="title">6 有害突变<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>由突变引起的DNA变化会导致蛋白质序列的错误，产生部分或完全不起作用的蛋白质。每个细胞，为了正常运作，依赖成千上万的蛋白质在正确的时间和正确的地点起作用。当突变改变了在身体中起关键作用的蛋白质时，可能会导致医学状况。有些突变会改变基因的DNA碱基序列，但不会改变基因所产生的蛋白质的功能。一项在<i>果蝇</i>不同种间的基因比较研究表明，如果突变确实改变了蛋白质，这可能是有害的，估计70%的氨基酸多态性具有破坏性影响，其余的是中性或弱有益的。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_94" class="kx_ref">[94]</a></sup> 研究表明，酵母非编码DNA中只有7%的点突变是有害的，编码DNA中只有12%是有害的。其余的突变要么是中性的，要么是轻微有益的。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_94" class="kx_ref">[94]</a></sup> </p><p>如果一个突变存在于生殖细胞中，它可以产生在其所有细胞中携带该突变的后代。遗传疾病就是这种情况。特别是，如果生殖细胞内的DNA修复基因发生突变，携带这种种系突变的人患癌症的风险可能会增加。34个这种种系突变的列表在文章《DNA修复缺陷疾病》中给出。一个例子是白化病，一种发生在OCA1或OCA2基因中的突变。患有这种疾病的人更容易患多种癌症和其他疾病，并且视力受损。另一方面，突变可能发生在生物体的体细胞中。这种突变将存在于同一生物体中该细胞的所有后代中，某些突变会导致细胞变得恶性，从而导致癌症。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_95" class="kx_ref">[95]</a></sup> </p><p>当DNA被复制时，DNA的损坏会导致错误，而这种复制错误会导致基因突变，进而导致遗传疾病。DNA损伤由细胞的DNA修复系统修复。每个细胞都有许多途径，酶通过这些途径识别和修复DNA损伤。因为DNA可以在许多方面被破坏，DNA修复的过程是身体保护自己免于疾病的重要途径。一旦DNA损伤导致突变，突变即无法修复。</p></div></div><div id="par_14995100673245449"><h2 class="title">7 有益突变<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>虽然引起蛋白质序列变化的突变可能对生物体有害，但有时在特定环境中这种影响可能是积极的。在这种情况下，突变可以使突变生物比野生型生物更好地承受特定的环境压力，或者更快地繁殖。在这些情况下，通过自然选择，突变在群体中会变得更加普遍。例子包括:</p><p>艾滋病毒抗性:人类<i>CCR5</i>基因中特异性32碱基对缺失(CCR5-Δ32)赋予纯合子艾滋病毒抗性，并延迟杂合子艾滋病发病。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_96" class="kx_ref">[96]</a></sup> 对CCR5-Δ32在欧洲人群中相对较高频率的病因学的一个可能解释是，它在14世纪中叶赋予了欧洲对腺鼠疫的抵抗力。具有这种突变的人更有可能在感染中存活；因此它在人口中的频率增加了。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_97" class="kx_ref">[97]</a></sup> 这个理论可以解释为什么在南部非洲没有发现这种突变，因为那里没有受到淋巴腺鼠疫的影响。一个较新的理论表明，CCR5-Δ32突变上的选择压力是由天花而不是腺鼠疫引起的。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_98" class="kx_ref">[98]</a></sup> </p><p>疟疾抗性:有害突变的一个例子是镰状细胞病，一种血液疾病，身体在红细胞中产生一种异常类型的携氧物质血红蛋白。撒哈拉以南非洲三分之一的土著居民携带该等位基因，因为在疟疾常见的地区，携带单一镰状细胞等位基因(镰状细胞性状)的人具有更高的生存率。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_99" class="kx_ref">[99]</a></sup> 只有镰状细胞病两个等位基因中其中一个的人对疟疾更有抵抗力，因为疟疾（<i>疟原虫）</i>的侵扰被镰状细胞抑制。</p><p>抗生素抗性:几乎所有细菌在接触抗生素时都会产生抗生素耐药性。事实上，细菌群体已经有了这样的突变，可以通过抗生素的选择来筛选。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_100" class="kx_ref">[100]</a></sup> 显然，这种突变只对细菌有益，而对受感染的人无益。</p><p>“乳糖酶耐受性”<i>。</i>一种突变允许人类在自然断奶后表达乳糖酶，从而允许成年人消化乳糖，这可能是最近人类进化中最有益的突变之一。<i><sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_101" class="kx_ref">[101]</a></sup></i> </p></div></div><div id="par_14995100673245450"><h2 class="title">8 朊病毒突变<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>朊病毒是蛋白质，不含遗传物质。然而，朊病毒复制已经显示出像其他形式的复制一样易受突变和自然选择的影响。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_102" class="kx_ref">[102]</a></sup> 人类基因PRNP编码主要朊病毒蛋白PrP，并且易发生突变，从而导致致病朊病毒。</p></div></div><div id="par_14995100673245451"><h2 class="title">9 体细胞突变<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>遗传结构的变化不是从父母那里遗传的，也不会传给后代，这种变化被称为体细胞突变<i>。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_103" class="kx_ref">[103]</a></sup> 体细胞突变不是遗传的，因为它们不影响种系。这些类型的突变通常是由环境原因引起的，例如紫外线辐射或任何暴露于某些有害化学物质的情况，并可能导致包括癌症在内的疾病。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_103" class="kx_ref">[103]</a></sup></i> </p><p>对于植物，一些体细胞突变可以在不需要种子产生的情况下繁殖，例如，通过嫁接和茎插条。这些类型的突变产生了新的水果类型，如“美味”苹果和“华盛顿”脐橙。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_104" class="kx_ref">[104]</a></sup> </p><p>人类和小鼠体细胞的突变率比两个物种的种系突变率高十倍以上；小鼠每个细胞分裂的体细胞和生殖细胞的突变率都要高于人类。种系和体细胞组织之间突变率的差异可能反映了种系中基因组的维护比体细胞中基因组的维护更重要。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_105" class="kx_ref">[105]</a></sup> </p></div></div><div id="par_14995100673245452"><h2 class="title">10 无效突变<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>穆勒在1932年使用的术语“amorph”是一种突变的等位基因，它失去了亲本(无论是野生型还是任何其他类型)等位基因编码任何功能蛋白的能力。无效突变可能是由使酶失活的氨基酸替换或产生酶的基因部分缺失引起的。</p><p>具有杂合突变(基因含一个正常拷贝和一个突变拷贝)的细胞可以在没有突变拷贝的情况下正常工作，直到正常拷贝在身体上自发突变。这种突变在生物体中一直存在，但是很难检测其发生率。检测这个比率对于预测人们患癌症的概率很重要。<sup><a href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/d10402.htm#quote_106" class="kx_ref">[106]</a></sup> </p><p>点突变可能源于DNA复制过程中发生的自发突变。诱变剂可以提高突变率。诱变剂可以是物理的，例如来自紫外线、X射线或极热的辐射，或化学的(使碱基对错位或破坏DNA双螺旋结构的分子)。经常研究与癌症相关的诱变剂从而了解癌症及其预防。</p></div></div><div id="par_14995100673245453"><h2 class="title">11 亚效和超效突变<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>亚效突变是导致基因表达量降低的突变。通常，亚效突变是隐性的，但是单倍剂量不足导致一些等位基因是显性的。</p><p>超效突变导致基因表达量增加。</p></div></div></div></div><div id="references"><h2 class="title" id="par_references">参考文献</h2><ul class="references"><li id="quote_1"><span class="references-num">[1]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"mutation | Learn Science at Scitable". www.nature.com (in 英语). Retrieved 2018-09-24..</span></p></li><li id="quote_2"><span class="references-num">[2]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Hastings PJ, Lupski JR, Rosenberg SM, Ira G (August 2009). "Mechanisms of change in gene copy number". Nature Reviews Genetics. 10 (8): 551–64. doi:10.1038/nrg2593. PMC 2864001. PMID 19597530..</span></p></li><li id="quote_3"><span class="references-num">[3]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Carroll SB, Grenier JK, Weatherbee SD (2005). From DNA to Diversity: Molecular Genetics and the Evolution of Animal Design (2nd ed.). Malden, MA: Blackwell Publishing. ISBN 978-1-4051-1950-4. LCCN 2003027991. OCLC 53972564..</span></p></li><li id="quote_4"><span class="references-num">[4]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Harrison PM, Gerstein M (May 2002). "Studying genomes through the aeons: protein families, pseudogenes and proteome evolution". Journal of Molecular Biology. 318 (5): 1155–74. doi:10.1016/S0022-2836(02)00109-2. PMID 12083509..</span></p></li><li id="quote_5"><span class="references-num">[5]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Orengo CA, Thornton JM (July 2005). "Protein families and their evolution-a structural perspective". Annual Review of Biochemistry. 74: 867–900. doi:10.1146/annurev.biochem.74.082803.133029. PMID 15954844..</span></p></li><li id="quote_6"><span class="references-num">[6]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Long M, Betrán E, Thornton K, Wang W (November 2003). "The origin of new genes: glimpses from the young and old". Nature Reviews Genetics. 4 (11): 865–75. doi:10.1038/nrg1204. PMID 14634634..</span></p></li><li id="quote_7"><span class="references-num">[7]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Wang M, Caetano-Anollés G (January 2009). "The evolutionary mechanics of domain organization in proteomes and the rise of modularity in the protein world". Structure. 17 (1): 66–78. doi:10.1016/j.str.2008.11.008. PMID 19141283..</span></p></li><li id="quote_8"><span class="references-num">[8]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Bowmaker JK (May 1998). "Evolution of colour vision in vertebrates". Eye. 12 (Pt 3b): 541–7. doi:10.1038/eye.1998.143. PMID 9775215..</span></p></li><li id="quote_9"><span class="references-num">[9]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Gregory TR, Hebert PD (April 1999). "The modulation of DNA content: proximate causes and ultimate consequences". Genome Research. 9 (4): 317–24. doi:10.1101/gr.9.4.317 (inactive 2019-03-11). PMID 10207154..</span></p></li><li id="quote_10"><span class="references-num">[10]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Hurles M (July 2004). "Gene duplication: the genomic trade in spare parts". PLoS Biology. 2 (7): E206. doi:10.1371/journal.pbio.0020206. PMC 449868. PMID 15252449..</span></p></li><li id="quote_11"><span class="references-num">[11]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Liu N, Okamura K, Tyler DM, Phillips MD, Chung WJ, Lai EC (October 2008). "The evolution and functional diversification of animal microRNA genes". Cell Research. 18 (10): 985–96. doi:10.1038/cr.2008.278. PMC 2712117. PMID 18711447..</span></p></li><li id="quote_12"><span class="references-num">[12]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Siepel A (October 2009). "Darwinian alchemy: Human genes from noncoding DNA". Genome Research. 19 (10): 1693–5. doi:10.1101/gr.098376.109. PMC 2765273. PMID 19797681..</span></p></li><li id="quote_13"><span class="references-num">[13]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Zhang J, Wang X, Podlaha O (May 2004). "Testing the chromosomal speciation hypothesis for humans and chimpanzees". Genome Research. 14 (5): 845–51. doi:10.1101/gr.1891104. PMC 479111. PMID 15123584..</span></p></li><li id="quote_14"><span class="references-num">[14]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Ayala FJ, Coluzzi M (May 2005). "Chromosome speciation: humans, Drosophila, and mosquitoes". Proceedings of the National Academy of Sciences of the United States of America. 102 Suppl 1 (Suppl 1): 6535–42. Bibcode:2005PNAS..102.6535A. doi:10.1073/pnas.0501847102. PMC 1131864. PMID 15851677..</span></p></li><li id="quote_15"><span class="references-num">[15]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Hurst GD, Werren JH (August 2001). "The role of selfish genetic elements in eukaryotic evolution". Nature Reviews Genetics. 2 (8): 597–606. doi:10.1038/35084545. PMID 11483984..</span></p></li><li id="quote_16"><span class="references-num">[16]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Häsler J, Strub K (November 2006). "Alu elements as regulators of gene expression". Nucleic Acids Research. 34 (19): 5491–7. doi:10.1093/nar/gkl706. PMC 1636486. PMID 17020921..</span></p></li><li id="quote_17"><span class="references-num">[17]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Aminetzach YT, Macpherson JM, Petrov DA (July 2005). "Pesticide resistance via transposition-mediated adaptive gene truncation in Drosophila". Science. 309 (5735): 764–7. Bibcode:2005Sci...309..764A. doi:10.1126/science.1112699. PMID 16051794..</span></p></li><li id="quote_18"><span class="references-num">[18]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Bohidar HB (January 2015). Fundamentals of Polymer Physics and Molecular Biophysics. Cambridge University Press. ISBN 978-1-316-09302-3..</span></p></li><li id="quote_19"><span class="references-num">[19]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Dover, Gabriel A.; Darwin, Charles (2000). Dear Mr. Darwin: Letters on the Evolution of Life and Human Nature (in 英语). University of California Press. ISBN 9780520227903..</span></p></li><li id="quote_20"><span class="references-num">[20]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Tibayrenc, Michel (2017-01-12). Genetics and Evolution of Infectious Diseases (in 英语). Elsevier. ISBN 9780128001530..</span></p></li><li id="quote_21"><span class="references-num">[21]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Bowler, Peter J. (1992) [1983]. The Eclipse of Darwinism. p. 198..</span></p></li><li id="quote_22"><span class="references-num">[22]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Smocovitis, Vassiliki Betty (1996). Unifying Biology: The Evolutionary Synthesis and Evolutionary Biology. Princeton, NJ: Princeton University Press. p. 56. ISBN 978-0-691-03343-3. LCCN 96005605. OCLC 34411399..</span></p></li><li id="quote_23"><span class="references-num">[23]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Hallgrímsson, Benedikt; Hall, Brian K. (2011). Variation: A Central Concept in Biology. Academic Press. p. 18..</span></p></li><li id="quote_24"><span class="references-num">[24]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Sewall Wright. (1984). Evolution and the Genetics of Populations: Genetics and Biometric Foundations Volume 1. University of Chicago Press. p. 10.</span></p></li><li id="quote_25"><span class="references-num">[25]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">De Vries, Hugo (1905). Species and Varieties: Their Origin by Mutation..</span></p></li><li id="quote_26"><span class="references-num">[26]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Bateson, William (1894). Materials for the Study of Variation, Treated with Especial Regard to Discontinuity in the Origin of Species..</span></p></li><li id="quote_27"><span class="references-num">[27]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Punnett, Reginald C. (1915). Mimicry in Butterflies. Cambridge University Press..</span></p></li><li id="quote_28"><span class="references-num">[28]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Mayr, Ernst (2007). What Makes Biology Unique?: Considerations on the Autonomy of a Scientific Discipline. Cambridge University Press..</span></p></li><li id="quote_29"><span class="references-num">[29]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Provine, W. B. (2001). The Origins of Theoretical Population Genetics, with a new afterword. University of Chicago Press, Chicago. pp. 56–107..</span></p></li><li id="quote_30"><span class="references-num">[30]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Stoltzfus, A.; Cable, K. (2014). "Mendelian-Mutationism: The Forgotten Evolutionary Synthesis". J Hist Biol. 47 (4): 501–546. doi:10.1007/s10739-014-9383-2. PMID 24811736..</span></p></li><li id="quote_31"><span class="references-num">[31]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Hull, D. L. (1985). "Darwinism as an historical entity: A historiographic proposal". In Kohn, D. The Darwinian Heritage. Princeton University Press. pp. 773–812..</span></p></li><li id="quote_32"><span class="references-num">[32]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Yule, G. Udny (1902). "Mendel's Laws and their probable relations to inter-racial heredity". New Phytologist. 1 (10): 226–227. doi:10.1111/j.1469-8137.1902.tb07336.x..</span></p></li><li id="quote_33"><span class="references-num">[33]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Gould, S. J. (1982). "The uses of heresey; an introduction to Richard Goldschmidt's The Material Basis of Evolution." pp. xiii-xlii. Yale University Press..</span></p></li><li id="quote_34"><span class="references-num">[34]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Ruse, Michael (1996). Monad to man: the Concept of Progress in Evolutionary Biology. Harvard University Press. pp. 412–413. ISBN 978-0-674-03248-4..</span></p></li><li id="quote_35"><span class="references-num">[35]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">A. Stoltzfus (2014). "In search of mutation-driven evolution". Evolution &amp; Development. 16: 57–59. doi:10.1111/ede.12062..</span></p></li><li id="quote_36"><span class="references-num">[36]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Futuyma, Douglas J. (2015). Serrelli, E.; Gontier, N., eds. Can Modern Evolutionary Theory Explain Macroevolution? (PDF). Macroevolution. Springer. pp. 29–85..</span></p></li><li id="quote_37"><span class="references-num">[37]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Cancer Is Partly Caused By Bad Luck, Study Finds". Archived from the original on 13 July 2017..</span></p></li><li id="quote_38"><span class="references-num">[38]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Jha, Alok (22 August 2012). "Older fathers pass on more genetic mutations, study shows". the Guardian..</span></p></li><li id="quote_39"><span class="references-num">[39]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Montelone BA (1998). "Mutation, Mutagens, and DNA Repair". www-personal.ksu.edu. Archived from the original on 26 September 2015. Retrieved 2 October 2015..</span></p></li><li id="quote_40"><span class="references-num">[40]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Bernstein C, Prasad AR, Nfonsam V, Bernstein H (2013). DNA Damage, DNA Repair and Cancer, New Research Directions in DNA Repair, Prof. Clark Chen (Ed.). intechopen.com. ISBN 978-953-51-1114-6. Archived from the original on 24 September 2015..</span></p></li><li id="quote_41"><span class="references-num">[41]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Stuart GR, Oda Y, de Boer JG, Glickman BW (March 2000). "Mutation frequency and specificity with age in liver, bladder and brain of lacI transgenic mice". Genetics. 154 (3): 1291–300. PMC 1460990. PMID 10757770..</span></p></li><li id="quote_42"><span class="references-num">[42]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Kunz BA, Ramachandran K, Vonarx EJ (April 1998). "DNA sequence analysis of spontaneous mutagenesis in Saccharomyces cerevisiae". Genetics. 148 (4): 1491–505. PMC 1460101. PMID 9560369..</span></p></li><li id="quote_43"><span class="references-num">[43]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Lieber MR (July 2010). "The mechanism of double-strand DNA break repair by the nonhomologous DNA end-joining pathway". Annual Review of Biochemistry. 79: 181–211. doi:10.1146/annurev.biochem.052308.093131. PMC 3079308. PMID 20192759..</span></p></li><li id="quote_44"><span class="references-num">[44]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Pfohl-Leszkowicz A, Manderville RA (January 2007). "Ochratoxin A: An overview on toxicity and carcinogenicity in animals and humans". Molecular Nutrition &amp; Food Research. 51 (1): 61–99. doi:10.1002/mnfr.200600137. PMID 17195275..</span></p></li><li id="quote_45"><span class="references-num">[45]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Kozmin S, Slezak G, Reynaud-Angelin A, Elie C, de Rycke Y, Boiteux S, Sage E (September 2005). "UVA radiation is highly mutagenic in cells that are unable to repair 7,8-dihydro-8-oxoguanine in Saccharomyces cerevisiae". Proceedings of the National Academy of Sciences of the United States of America. 102 (38): 13538–43. Bibcode:2005PNAS..10213538K. doi:10.1073/pnas.0504497102. PMC 1224634. PMID 16157879..</span></p></li><li id="quote_46"><span class="references-num">[46]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Rahman, Nazneen. "The clinical impact of DNA sequence changes". Transforming Genetic Medicine Initiative. Archived from the original on 4 August 2017. Retrieved 27 June 2017..</span></p></li><li id="quote_47"><span class="references-num">[47]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Freese E (April 1959). "THE DIFFERENCE BETWEEN SPONTANEOUS AND BASE-ANALOGUE INDUCED MUTATIONS OF PHAGE T4". Proceedings of the National Academy of Sciences of the United States of America. 45 (4): 622–33. Bibcode:1959PNAS...45..622F. doi:10.1073/pnas.45.4.622. PMC 222607. PMID 16590424..</span></p></li><li id="quote_48"><span class="references-num">[48]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Freese E (June 1959). "The specific mutagenic effect of base analogues on Phage T4". Journal of Molecular Biology. 1 (2): 87–105. doi:10.1016/S0022-2836(59)80038-3..</span></p></li><li id="quote_49"><span class="references-num">[49]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Steenhuysen, Julie (2017-12-19). "U.S. Lifts Funding Ban on Studies That Enhance Dangerous Germs". U.S. News &amp; World Report. Retrieved 2018-01-15..</span></p></li><li id="quote_50"><span class="references-num">[50]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Goh AM, Coffill CR, Lane DP (January 2011). "The role of mutant p53 in human cancer". The Journal of Pathology. 223 (2): 116–26. doi:10.1002/path.2784. PMID 21125670..</span></p></li><li id="quote_51"><span class="references-num">[51]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Chenevix-Trench G, Spurdle AB, Gatei M, Kelly H, Marsh A, Chen X, Donn K, Cummings M, Nyholt D, Jenkins MA, Scott C, Pupo GM, Dörk T, Bendix R, Kirk J, Tucker K, McCredie MR, Hopper JL, Sambrook J, Mann GJ, Khanna KK (February 2002). "Dominant negative ATM mutations in breast cancer families". Journal of the National Cancer Institute. 94 (3): 205–15. CiteSeerX 10.1.1.557.6394. doi:10.1093/jnci/94.3.205. PMID 11830610..</span></p></li><li id="quote_52"><span class="references-num">[52]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Paz-Priel I, Friedman A (2011). "C/EBPα dysregulation in AML and ALL". Critical Reviews in Oncogenesis. 16 (1–2): 93–102. doi:10.1615/critrevoncog.v16.i1-2.90. PMC 3243939. PMID 22150310..</span></p></li><li id="quote_53"><span class="references-num">[53]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Capaccio D, Ciccodicola A, Sabatino L, Casamassimi A, Pancione M, Fucci A, Febbraro A, Merlino A, Graziano G, Colantuoni V (June 2010). "A novel germline mutation in peroxisome proliferator-activated receptor gamma gene associated with large intestine polyp formation and dyslipidemia". Biochimica et Biophysica Acta. 1802 (6): 572–81. doi:10.1016/j.bbadis.2010.01.012. PMID 20123124..</span></p></li><li id="quote_54"><span class="references-num">[54]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">McKusick VA (July 1991). "The defect in Marfan syndrome". Nature. 352 (6333): 279–81. Bibcode:1991Natur.352..279M. doi:10.1038/352279a0. PMID 1852198..</span></p></li><li id="quote_55"><span class="references-num">[55]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Judge DP, Biery NJ, Keene DR, Geubtner J, Myers L, Huso DL, Sakai LY, Dietz HC (July 2004). "Evidence for a critical contribution of haploinsufficiency in the complex pathogenesis of Marfan syndrome". The Journal of Clinical Investigation. 114 (2): 172–81. doi:10.1172/JCI20641. PMC 449744. PMID 15254584..</span></p></li><li id="quote_56"><span class="references-num">[56]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Judge DP, Dietz HC (December 2005). "Marfan's syndrome". Lancet. 366 (9501): 1965–76. doi:10.1016/S0140-6736(05)67789-6. PMC 1513064. PMID 16325700..</span></p></li><li id="quote_57"><span class="references-num">[57]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Ellis NA, Ciocci S, German J (February 2001). "Back mutation can produce phenotype reversion in Bloom syndrome somatic cells". Human Genetics. 108 (2): 167–73. doi:10.1007/s004390000447. PMID 11281456..</span></p></li><li id="quote_58"><span class="references-num">[58]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Charlesworth D, Charlesworth B, Morgan MT (December 1995). "The pattern of neutral molecular variation under the background selection model". Genetics. 141 (4): 1619–32. PMC 1206892. PMID 8601499..</span></p></li><li id="quote_59"><span class="references-num">[59]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Loewe L (April 2006). "Quantifying the genomic decay paradox due to Muller's ratchet in human mitochondrial DNA". Genetical Research. 87 (2): 133–59. doi:10.1017/S0016672306008123. PMID 16709275..</span></p></li><li id="quote_60"><span class="references-num">[60]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Bernstein H, Hopf FA, Michod RE (1987). The molecular basis of the evolution of sex. Advances in Genetics. 24. pp. 323–70. doi:10.1016/s0065-2660(08)60012-7. ISBN 9780120176243. PMID 3324702..</span></p></li><li id="quote_61"><span class="references-num">[61]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Peck JR, Barreau G, Heath SC (April 1997). "Imperfect genes, Fisherian mutation and the evolution of sex". Genetics. 145 (4): 1171–99. PMC 1207886. PMID 9093868..</span></p></li><li id="quote_62"><span class="references-num">[62]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Keightley PD, Lynch M (March 2003). "Toward a realistic model of mutations affecting fitness". Evolution; International Journal of Organic Evolution. 57 (3): 683–5, discussion 686–9. doi:10.1554/0014-3820(2003)057[0683:tarmom]2.0.co;2. JSTOR 3094781. PMID 12703958..</span></p></li><li id="quote_63"><span class="references-num">[63]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Barton NH, Keightley PD (January 2002). "Understanding quantitative genetic variation". Nature Reviews Genetics. 3 (1): 11–21. doi:10.1038/nrg700. PMID 11823787..</span></p></li><li id="quote_64"><span class="references-num">[64]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Sanjuán R, Moya A, Elena SF (June 2004). "The distribution of fitness effects caused by single-nucleotide substitutions in an RNA virus". Proceedings of the National Academy of Sciences of the United States of America. 101 (22): 8396–401. Bibcode:2004PNAS..101.8396S. doi:10.1073/pnas.0400146101. PMC 420405. PMID 15159545..</span></p></li><li id="quote_65"><span class="references-num">[65]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Carrasco P, de la Iglesia F, Elena SF (December 2007). "Distribution of fitness and virulence effects caused by single-nucleotide substitutions in Tobacco Etch virus". Journal of Virology. 81 (23): 12979–84. doi:10.1128/JVI.00524-07. PMC 2169111. PMID 17898073..</span></p></li><li id="quote_66"><span class="references-num">[66]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Sanjuán R (June 2010). "Mutational fitness effects in RNA and single-stranded DNA viruses: common patterns revealed by site-directed mutagenesis studies". Philosophical Transactions of the Royal Society of London. Series B, Biological Sciences. 365 (1548): 1975–82. doi:10.1098/rstb.2010.0063. PMC 2880115. PMID 20478892..</span></p></li><li id="quote_67"><span class="references-num">[67]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Peris JB, Davis P, Cuevas JM, Nebot MR, Sanjuán R (June 2010). "Distribution of fitness effects caused by single-nucleotide substitutions in bacteriophage f1". Genetics. 185 (2): 603–9. doi:10.1534/genetics.110.115162. PMC 2881140. PMID 20382832..</span></p></li><li id="quote_68"><span class="references-num">[68]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Elena SF, Ekunwe L, Hajela N, Oden SA, Lenski RE (March 1998). "Distribution of fitness effects caused by random insertion mutations in Escherichia coli". Genetica. 102–103 (1–6): 349–58. doi:10.1023/A:1017031008316. PMID 9720287..</span></p></li><li id="quote_69"><span class="references-num">[69]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Hietpas RT, Jensen JD, Bolon DN (May 2011). "Experimental illumination of a fitness landscape". Proceedings of the National Academy of Sciences of the United States of America. 108 (19): 7896–901. Bibcode:2011PNAS..108.7896H. doi:10.1073/pnas.1016024108. PMC 3093508. PMID 21464309..</span></p></li><li id="quote_70"><span class="references-num">[70]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Davies EK, Peters AD, Keightley PD (September 1999). "High frequency of cryptic deleterious mutations in Caenorhabditis elegans". Science. 285 (5434): 1748–51. doi:10.1126/science.285.5434.1748. PMID 10481013..</span></p></li><li id="quote_71"><span class="references-num">[71]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Loewe L, Charlesworth B (September 2006). "Inferring the distribution of mutational effects on fitness in Drosophila". Biology Letters. 2 (3): 426–30. doi:10.1098/rsbl.2006.0481. PMC 1686194. PMID 17148422..</span></p></li><li id="quote_72"><span class="references-num">[72]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Eyre-Walker A, Woolfit M, Phelps T (June 2006). "The distribution of fitness effects of new deleterious amino acid mutations in humans". Genetics. 173 (2): 891–900. doi:10.1534/genetics.106.057570. PMC 1526495. PMID 16547091..</span></p></li><li id="quote_73"><span class="references-num">[73]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Sawyer SA, Kulathinal RJ, Bustamante CD, Hartl DL (August 2003). "Bayesian analysis suggests that most amino acid replacements in Drosophila are driven by positive selection". Journal of Molecular Evolution. 57 Suppl 1 (1): S154–64. Bibcode:2003JMolE..57S.154S. CiteSeerX 10.1.1.78.65. doi:10.1007/s00239-003-0022-3. PMID 15008412..</span></p></li><li id="quote_74"><span class="references-num">[74]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Piganeau G, Eyre-Walker A (September 2003). "Estimating the distribution of fitness effects from DNA sequence data: implications for the molecular clock". Proceedings of the National Academy of Sciences of the United States of America. 100 (18): 10335–40. Bibcode:2003PNAS..10010335P. doi:10.1073/pnas.1833064100. PMC 193562. PMID 12925735..</span></p></li><li id="quote_75"><span class="references-num">[75]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Eyre-Walker A, Keightley PD (August 2007). "The distribution of fitness effects of new mutations" (PDF). Nature Reviews Genetics. 8 (8): 610–8. doi:10.1038/nrg2146. PMID 17637733. Archived (PDF) from the original on 4 March 2016..</span></p></li><li id="quote_76"><span class="references-num">[76]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Kimura M (1983). The Neutral Theory of Molecular Evolution. Cambridge, UK; New York: Cambridge University Press. ISBN 978-0-521-23109-1. LCCN 82022225. OCLC 9081989..</span></p></li><li id="quote_77"><span class="references-num">[77]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Eyre-Walker A (October 2006). "The genomic rate of adaptive evolution". Trends in Ecology &amp; Evolution. 21 (10): 569–75. doi:10.1016/j.tree.2006.06.015. PMID 16820244..</span></p></li><li id="quote_78"><span class="references-num">[78]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Orr HA (April 2003). "The distribution of fitness effects among beneficial mutations". Genetics. 163 (4): 1519–26. PMC 1462510. PMID 12702694..</span></p></li><li id="quote_79"><span class="references-num">[79]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Gillespie JH (September 1984). "Molecular Evolution Over the Mutational Landscape". Evolution. 38 (5): 1116–1129. doi:10.2307/2408444. JSTOR 2408444. PMID 28555784..</span></p></li><li id="quote_80"><span class="references-num">[80]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Kassen R, Bataillon T (April 2006). "Distribution of fitness effects among beneficial mutations before selection in experimental populations of bacteria". Nature Genetics. 38 (4): 484–8. doi:10.1038/ng1751. PMID 16550173..</span></p></li><li id="quote_81"><span class="references-num">[81]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Rokyta DR, Joyce P, Caudle SB, Wichman HA (April 2005). "An empirical test of the mutational landscape model of adaptation using a single-stranded DNA virus". Nature Genetics. 37 (4): 441–4. doi:10.1038/ng1535. PMID 15778707..</span></p></li><li id="quote_82"><span class="references-num">[82]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Imhof M, Schlotterer C (January 2001). "Fitness effects of advantageous mutations in evolving Escherichia coli populations". Proceedings of the National Academy of Sciences of the United States of America. 98 (3): 1113–7. Bibcode:2001PNAS...98.1113I. doi:10.1073/pnas.98.3.1113. PMC 14717. PMID 11158603..</span></p></li><li id="quote_83"><span class="references-num">[83]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Hogan, C. Michael (12 October 2010). "Mutation". In Monosson, Emily. Encyclopedia of Earth. Washington, D.C.: Environmental Information Coalition, National Council for Science and the Environment. OCLC 72808636. Archived from the original on 14 November 2015. Retrieved 8 October 2015..</span></p></li><li id="quote_84"><span class="references-num">[84]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Boillée S, Vande Velde C, Cleveland DW (October 2006). "ALS: a disease of motor neurons and their nonneuronal neighbors". Neuron. 52 (1): 39–59. CiteSeerX 10.1.1.325.7514. doi:10.1016/j.neuron.2006.09.018. PMID 17015226..</span></p></li><li id="quote_85"><span class="references-num">[85]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"RB1 Genetics". Daisy's Eye Cancer Fund. Oxford, UK. Archived from the original on 2011-11-26. Retrieved 2015-10-09..</span></p></li><li id="quote_86"><span class="references-num">[86]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Compound heterozygote". MedTerms. New York: WebMD. 14 June 2012. Archived from the original on 4 March 2016. Retrieved 9 October 2015..</span></p></li><li id="quote_87"><span class="references-num">[87]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Alberts (2014). Molecular Biology of the Cell (6 ed.). Garland Science. p. 487. ISBN 9780815344322..</span></p></li><li id="quote_88"><span class="references-num">[88]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Chadov BF, Fedorova NB, Chadova EV (2015-07-01). "Conditional mutations in Drosophila melanogaster: On the occasion of the 150th anniversary of G. Mendel's report in Brünn". Mutation Research/Reviews in Mutation Research. 765: 40–55. doi:10.1016/j.mrrev.2015.06.001. PMID 26281767..</span></p></li><li id="quote_89"><span class="references-num">[89]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Landis G, Bhole D, Lu L, Tower J (July 2001). "High-frequency generation of conditional mutations affecting Drosophila melanogaster development and life span". Genetics. 158 (3): 1167–76. PMC 1461716. PMID 11454765. Archived from the original on 22 March 2017..</span></p></li><li id="quote_90"><span class="references-num">[90]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Gierut JJ, Jacks TE, Haigis KM (April 2014). "Strategies to achieve conditional gene mutation in mice". Cold Spring Harbor Protocols. 2014 (4): 339–49. doi:10.1101/pdb.top069807. PMC 4142476. PMID 24692485..</span></p></li><li id="quote_91"><span class="references-num">[91]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Spencer DM (May 1996). "Creating conditional mutations in mammals". Trends in Genetics. 12 (5): 181–7. doi:10.1016/0168-9525(96)10013-5. PMID 8984733..</span></p></li><li id="quote_92"><span class="references-num">[92]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Tan G, Chen M, Foote C, Tan C (September 2009). "Temperature-sensitive mutations made easy: generating conditional mutations by using temperature-sensitive inteins that function within different temperature ranges". Genetics. 183 (1): 13–22. doi:10.1534/genetics.109.104794. PMC 2746138. PMID 19596904..</span></p></li><li id="quote_93"><span class="references-num">[93]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">den Dunnen JT, Antonarakis SE (January 2000). "Mutation nomenclature extensions and suggestions to describe complex mutations: a discussion". Human Mutation. 15 (1): 7–12. doi:10.1002/(SICI)1098-1004(200001)15:1&lt;7::AID-HUMU4&gt;3.0.CO;2-N. PMID 10612815..</span></p></li><li id="quote_94"><span class="references-num">[94]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Sawyer SA, Parsch J, Zhang Z, Hartl DL (April 2007). "Prevalence of positive selection among nearly neutral amino acid replacements in Drosophila". Proceedings of the National Academy of Sciences of the United States of America. 104 (16): 6504–10. Bibcode:2007PNAS..104.6504S. doi:10.1073/pnas.0701572104. PMC 1871816. PMID 17409186..</span></p></li><li id="quote_95"><span class="references-num">[95]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Ionov Y, Peinado MA, Malkhosyan S, Shibata D, Perucho M (June 1993). "Ubiquitous somatic mutations in simple repeated sequences reveal a new mechanism for colonic carcinogenesis". Nature. 363 (6429): 558–61. Bibcode:1993Natur.363..558I. doi:10.1038/363558a0. PMID 8505985..</span></p></li><li id="quote_96"><span class="references-num">[96]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Sullivan AD, Wigginton J, Kirschner D (August 2001). "The coreceptor mutation CCR5Delta32 influences the dynamics of HIV epidemics and is selected for by HIV". Proceedings of the National Academy of Sciences of the United States of America. 98 (18): 10214–9. Bibcode:2001PNAS...9810214S. doi:10.1073/pnas.181325198. PMC 56941. PMID 11517319..</span></p></li><li id="quote_97"><span class="references-num">[97]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Mystery of the Black Death". Secrets of the Dead. Season 3. Episode 2. 30 October 2002. PBS. Archived from the original on 12 October 2015. Retrieved 10 October 2015. Episode background..</span></p></li><li id="quote_98"><span class="references-num">[98]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Galvani AP, Slatkin M (December 2003). "Evaluating plague and smallpox as historical selective pressures for the CCR5-Delta 32 HIV-resistance allele". Proceedings of the National Academy of Sciences of the United States of America. 100 (25): 15276–9. Bibcode:2003PNAS..10015276G. doi:10.1073/pnas.2435085100. PMC 299980. PMID 14645720..</span></p></li><li id="quote_99"><span class="references-num">[99]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Konotey-Ahulu, Felix. "Frequently Asked Questions [FAQ's]". sicklecell.md. Archived from the original on 30 April 2011..</span></p></li><li id="quote_100"><span class="references-num">[100]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Hughes, Diarmaid; Andersson, Dan I. (09 08, 2017). "Evolutionary Trajectories to Antibiotic Resistance". Annual Review of Microbiology. 71: 579–596. doi:10.1146/annurev-micro-090816-093813. ISSN 1545-3251. PMID 28697667. Check date values in: |date= (help).</span></p></li><li id="quote_101"><span class="references-num">[101]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Ségurel, Laure; Bon, Céline (08 31, 2017). "On the Evolution of Lactase Persistence in Humans". Annual Review of Genomics and Human Genetics. 18: 297–319. doi:10.1146/annurev-genom-091416-035340. ISSN 1545-293X. PMID 28426286. Check date values in: |date= (help).</span></p></li><li id="quote_102"><span class="references-num">[102]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"'Lifeless' prion proteins are 'capable of evolution'". Health. BBC News Online. London. 1 January 2010. Archived from the original on 25 September 2015. Retrieved 10 October 2015..</span></p></li><li id="quote_103"><span class="references-num">[103]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Somatic cell genetic mutation". Genome Dictionary. Athens, Greece: Information Technology Associates. 30 June 2007. Archived from the original on 24 February 2010. Retrieved 6 June 2010..</span></p></li><li id="quote_104"><span class="references-num">[104]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Hartl, Jones, Daniel L., Elizabeth W. (1998). Genetics Principles and Analysis. Sudbury, Massachusetts: Jones and Bartlett Publishers. p. 556. ISBN 978-0-7637-0489-6..</span></p></li><li id="quote_105"><span class="references-num">[105]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Milholland B, Dong X, Zhang L, Hao X, Suh Y, Vijg J (2017). "Differences between germline and somatic mutation rates in humans and mice". Nat Commun. 8: 15183. Bibcode:2017NatCo...815183M. doi:10.1038/ncomms15183. PMC 5436103. PMID 28485371..</span></p></li><li id="quote_106"><span class="references-num">[106]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Araten DJ, Golde DW, Zhang RH, Thaler HT, Gargiulo L, Notaro R, Luzzatto L (September 2005). "A quantitative measurement of the human somatic mutation rate". Cancer Research. 65 (18): 8111–7. doi:10.1158/0008-5472.CAN-04-1198. PMID 16166284..</span></p></li></ul></div><div class="read-num">阅读 <!-- -->2.8<!-- -->w</div></div><div class="right-side" id="rightSide"><div class="side" id="lemma-side"><div class="side-title">版本记录</div><ul class="side-lst"><li><p class="side-lst-txt">暂无</p></li></ul><div class="user-card userCard"></div></div><div class="side"><div class="side-event"></div></div></div></div><div class="footer-box"><div id="footer"><div class="footer-logo-wrap"><div class="footer-logo"></div><div class="footer-logo-text">知识·传播·科普</div></div><div class="footer-info">本网站内容采用<a target="_blank" href="https://web.archive.org/web/20221025113755/https://creativecommons.org/licenses/by-sa/3.0/deed.zh?tdsourcetag=s_pctim_aiomsg">CC-BY-SA 3.0</a>授权</div><div class="footer-btn-wrap"><a target="_blank" href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/help/#user_protocol">用户协议</a><a target="_blank" href="https://web.archive.org/web/20221025113755/http://www.sogou.com/docs/terms.htm?v=1">免责声明</a><a target="_blank" href="https://web.archive.org/web/20221025113755/http://corp.sogou.com/private.html">隐私政策</a><a target="_blank" href="https://web.archive.org/web/20221025113755/https://baike.sogou.com/kexue/intro.htm">关于我们</a></div></div></div><script>window.lemmaInfo ={"lemmaId":"10402","versionId":"54406546892841476","title":"基因突变","subtitle":"","abstracts":{"paragraphId":"54231093351604751","title":"摘要","versionId":"54406546892841477","lemmaId":10402,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":51990732,"name":"creeper","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599724829,"comment":null,"dependVersionId":0,"contentType":2,"content":"<p>在生物学中，<b>突变</b>是指生物体、病毒或染色体外DNA的基因组中的核苷酸序列发生了改变。<sup><a href=\"#quote_1\" class=\"kx_ref\">[1]</a></sup>基因突变包括单个碱基改变所引起的点突变，或多个碱基的缺失、重复和插入。引起突变的原因可以是细胞分裂时遗传基因的复制发生错误、或受化学物质、基因毒性、辐射或病毒的影响。</p>\n<p>突变通常会导致细胞运作不正常、或细胞死亡，甚至可以在较高等生物中引发癌症。但同时，突变也被视为演化的“推动力”：不理想的突变会经天择过程被淘汰，而对物种有利的突变则会被累积下去。中性突变对物种没有影响而逐渐累积，会导致间断平衡。</p>","pics":[{"originalUrl":"https://web.archive.org/web/20221025113755/https://img03.sogoucdn.com/app/a/200698/sogou_science_11618?width=231&height=174&titlename=%E9%83%81%E9%87%91%E9%A6%99%E8%A1%A8%E7%8E%B0%E5%87%BA%E9%83%A8%E5%88%86%E9%BB%84%E8%89%B2%E8%8A%B1%E7%93%A3%E6%98%AF%E7%94%B1%E4%BA%8E%E5%85%B6%E5%9F%BA%E5%9B%A0%E5%8F%91%E7%94%9F%E4%BA%86%E7%AA%81%E5%8F%98%E3%80%82&w=300&h=200","url":"https://web.archive.org/web/20221025113755/https://img03.sogoucdn.com/app/a/200698/sogou_science_11618","rw":300,"rh":200,"title":"郁金香表现出部分黄色花瓣是由于其基因发生了突变。","alt":null,"width":231,"height":174}],"card":null,"references":[],"versionCount":0},"card":{"paragraphId":"54231093318050306","title":"基本信息","versionId":"54231093301273092","lemmaId":10402,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82812762,"name":"雷克斯班纳","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599620250,"comment":null,"dependVersionId":0,"contentType":3,"content":"","pics":null,"card":{"cardItems":[{"key":"外文名","value":"genic mutation"},{"key":"提出者","value":"E.佛里兹"},{"key":"提出时间","value":"1959年"},{"key":"首例","value":"果蝇"},{"key":"应用学科","value":"生物"},{"key":"本质","value":"遗传变异"}]},"references":[],"versionCount":0},"categories":[{"id":4,"name":"生物学","parents":[]}],"creator":{"uid":76033348,"name":"廖志刚","pic":"https://web.archive.org/web/20221025113755/https://cache.soso.com/qlogo/g?b=oidb&k=VK4KcKvSwAcZWWvibMYwrqw&s=100&t=1483307853","introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":"","jobBrief":"","role":0,"roleName":null,"title":"","professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":164,"partnerIdCreateTime":1595844887,"partnerIdPoped":false},"createTime":1568622584,"editor":{"uid":51990732,"name":"沙粒","pic":"https://web.archive.org/web/20221025113755/https://img04.sogoucdn.com/app/a/200698/590_590_510152_20200731172509-912306427.png","introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":"","jobBrief":"","role":0,"roleName":null,"title":"","professionalTitle":"","phoneNo":null,"editable":true,"partnerId":232,"partnerIdCreateTime":1595845100,"partnerIdPoped":true},"editTime":1599724829,"state":1,"versionCount":3,"upNum":10,"downNum":0,"pics":[{"originalUrl":"https://web.archive.org/web/20221025113755/https://img03.sogoucdn.com/app/a/200698/sogou_science_11618?width=231&height=174&titlename=%E9%83%81%E9%87%91%E9%A6%99%E8%A1%A8%E7%8E%B0%E5%87%BA%E9%83%A8%E5%88%86%E9%BB%84%E8%89%B2%E8%8A%B1%E7%93%A3%E6%98%AF%E7%94%B1%E4%BA%8E%E5%85%B6%E5%9F%BA%E5%9B%A0%E5%8F%91%E7%94%9F%E4%BA%86%E7%AA%81%E5%8F%98%E3%80%82&w=300&h=200","url":"https://web.archive.org/web/20221025113755/https://img03.sogoucdn.com/app/a/200698/sogou_science_11618","rw":300,"rh":200,"title":"郁金香表现出部分黄色花瓣是由于其基因发生了突变。","alt":null,"width":231,"height":174},{"originalUrl":"https://web.archive.org/web/20221025113755/https://img02.sogoucdn.com/app/a/200698/sogou_science_11620?w=199&h=153&titlename=Prodryas%20persephone%20%E6%98%AF%E4%B8%80%E7%A7%8D%E6%99%9A%E5%A7%8B%E6%96%B0%E4%B8%96%E8%9D%B4%E8%9D%B6%E5%93%81%E7%A7%8D%E3%80%82","url":"https://web.archive.org/web/20221025113755/https://img02.sogoucdn.com/app/a/200698/sogou_science_11620","rw":199,"rh":153,"title":"Prodryas persephone 是一种晚始新世蝴蝶品种。","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221025113755/https://img01.sogoucdn.com/app/a/200698/sogou_science_11621?w=300&h=261&titlename=%E9%9B%A8%E6%9E%9C%C2%B7%E5%BE%B7%C2%B7%E5%BC%97%E9%87%8C%E6%96%AF%E7%94%BB%E4%BA%86%E4%B8%80%E5%B9%85%E6%9C%88%E8%A7%81%E8%8D%89%E7%9A%84%E7%94%BB%EF%BC%8C%E8%BF%99%E7%A7%8D%E6%A4%8D%E7%89%A9%E6%98%BE%E7%84%B6%E6%98%AF%E5%9C%A8%E4%BB%96%E7%9A%84%E5%AE%9E%E9%AA%8C%E4%B8%AD%E5%8F%91%E7%94%9F%E4%BA%86%E5%A4%A7%E9%87%8F%E7%AA%81%E5%8F%98%E8%80%8C%E4%BA%A7%E7%94%9F%E4%BA%86%E6%96%B0%E7%9A%84%E5%BD%A2%E6%80%81%E3%80%82%20%E7%94%B1Th%C3%A9r%C3%A8se%20Schwartze%E4%BA%8E1918%E6%89%80%E4%BD%9C%E3%80%82","url":"https://web.archive.org/web/20221025113755/https://img01.sogoucdn.com/app/a/200698/sogou_science_11621","rw":300,"rh":261,"title":"雨果·德·弗里斯画了一幅月见草的画，这种植物显然是在他的实验中发生了大量突变而产生了新的形态。 由Thérèse Schwartze于1918所作。","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221025113755/https://img02.sogoucdn.com/app/a/200698/sogou_science_11628?w=250&h=346&titlename=%E8%8B%AF%E5%B9%B6%5Ba%5D%E8%8A%98%E4%BB%A3%E8%B0%A2%E7%89%A9%E4%B8%8EDNA%E7%9A%84%E5%85%B1%E4%BB%B7%E5%8C%96%E5%90%88%E7%89%A9%EF%BC%8C%E8%8B%AF%E5%B9%B6%5Ba%5D%E8%8A%98%E6%98%AF%E7%83%9F%E8%8D%89%E7%83%9F%E9%9B%BE%E4%B8%AD%E7%9A%84%E4%B8%BB%E8%A6%81%E8%AF%B1%E5%8F%98%E5%89%82%5B8%5D","url":"https://web.archive.org/web/20221025113755/https://img02.sogoucdn.com/app/a/200698/sogou_science_11628","rw":250,"rh":346,"title":"苯并[a]芘代谢物与DNA的共价化合物，苯并[a]芘是烟草烟雾中的主要诱变剂[8]","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221025113755/https://img01.sogoucdn.com/app/a/200698/sogou_science_11634?w=301&h=595&titlename=%E6%9F%93%E8%89%B2%E4%BD%93%E7%9A%845%E7%A7%8D%E7%AA%81%E5%8F%98%E7%B1%BB%E5%9E%8B%E3%80%82","url":"https://web.archive.org/web/20221025113755/https://img01.sogoucdn.com/app/a/200698/sogou_science_11634","rw":301,"rh":595,"title":"染色体的5种突变类型。","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221025113755/https://img02.sogoucdn.com/app/a/200698/sogou_science_11636?w=301&h=178&titlename=%E6%B0%A8%E5%9F%BA%E9%85%B8%E9%81%97%E4%BC%A0%E5%AF%86%E7%A0%81%E6%A0%87%E5%87%86%E8%A1%A8%E4%B8%AD%E8%87%B4%E7%97%85%E7%AA%81%E5%8F%98%E7%9A%84%E9%80%89%E6%8B%A9%E3%80%82%5B1%5D","url":"https://web.archive.org/web/20221025113755/https://img02.sogoucdn.com/app/a/200698/sogou_science_11636","rw":301,"rh":178,"title":"氨基酸遗传密码标准表中致病突变的选择。[1]","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221025113755/https://img02.sogoucdn.com/app/a/200698/sogou_science_11639?w=360&h=210&titlename=%E6%B0%B4%E7%96%B1%E6%80%A7%E5%8F%A3%E7%82%8E%E7%97%85%E6%AF%92%E7%AA%81%E5%8F%98%E7%9A%84%E9%80%82%E5%BA%94%E6%80%A7%E6%95%88%E5%BA%94(DFE)%E5%88%86%E5%B8%83%E3%80%82%E5%9C%A8%E6%9C%AC%E5%AE%9E%E9%AA%8C%E4%B8%AD%EF%BC%8C%E9%80%9A%E8%BF%87%E5%AE%9A%E7%82%B9%E8%AF%B1%E5%8F%98%E5%B0%86%E9%9A%8F%E6%9C%BA%E7%AA%81%E5%8F%98%E5%BC%95%E5%85%A5%E7%97%85%E6%AF%92%EF%BC%8C%E5%B9%B6%E5%B0%86%E6%AF%8F%E4%B8%AA%E7%AA%81%E5%8F%98%E4%BD%93%E7%9A%84%E9%80%82%E5%BA%94%E6%80%A7%E4%B8%8E%E7%A5%96%E5%85%88%E7%B1%BB%E5%9E%8B%E8%BF%9B%E8%A1%8C%E6%AF%94%E8%BE%83%E3%80%82%E9%80%82%E5%BA%94%E5%BA%A6%E5%88%86%E5%88%AB%E5%B0%8F%E4%BA%8E1%E5%92%8C1%E4%BB%A5%E4%B8%8A%EF%BC%8C%E8%A1%A8%E6%98%8E%E7%AA%81%E5%8F%98%E6%98%AF%E8%87%B4%E5%91%BD%E7%9A%84%E3%80%81%E6%9C%89%E5%AE%B3%E7%9A%84%E3%80%81%E4%B8%AD%E6%80%A7%E7%9A%84%E5%92%8C%E6%9C%89%E5%88%A9%E7%9A%84%E3%80%82%5B20%5D","url":"https://web.archive.org/web/20221025113755/https://img02.sogoucdn.com/app/a/200698/sogou_science_11639","rw":360,"rh":210,"title":"水疱性口炎病毒突变的适应性效应(DFE)分布。在本实验中，通过定点诱变将随机突变引入病毒，并将每个突变体的适应性与祖先类型进行比较。适应度分别小于1和1以上，表明突变是致命的、有害的、中性的和有利的。[20]","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221025113755/https://img04.sogoucdn.com/app/a/200698/sogou_science_11642?w=300&h=346&titlename=%E4%B8%80%E7%A7%8D%E7%AA%81%E5%8F%98%E5%AF%BC%E8%87%B4%E8%BF%99%E7%A7%8D%E8%8B%94%E8%97%93%E7%8E%AB%E7%91%B0%E6%A4%8D%E7%89%A9%E4%BA%A7%E7%94%9F%E4%B8%8D%E5%90%8C%E9%A2%9C%E8%89%B2%E7%9A%84%E8%8A%B1%E3%80%82%E8%BF%99%E6%98%AF%E4%B8%80%E7%A7%8D%E4%BD%93%E7%BB%86%E8%83%9E%E7%AA%81%E5%8F%98%EF%BC%8C%E4%B9%9F%E5%8F%AF%E8%83%BD%E5%9C%A8%E7%94%9F%E6%AE%96%E7%BB%86%E8%83%9E%E4%B8%AD%E4%BC%A0%E6%92%AD%E3%80%82","url":"https://web.archive.org/web/20221025113755/https://img04.sogoucdn.com/app/a/200698/sogou_science_11642","rw":300,"rh":346,"title":"一种突变导致这种苔藓玫瑰植物产生不同颜色的花。这是一种体细胞突变，也可能在生殖细胞中传播。","alt":null,"width":0,"height":0}],"catalogs":[{"level":1,"title":"描述","paragraphId":"14995100639691029","subCatalogs":null},{"level":1,"title":"历史","paragraphId":"14995100656468239","subCatalogs":null},{"level":1,"title":"原因","paragraphId":"14995100656468240","subCatalogs":[{"level":2,"title":"自发突变","paragraphId":"14995100656468240","subCatalogs":null},{"level":2,"title":"易错的复制旁路","paragraphId":"14995100656468240","subCatalogs":null},{"level":2,"title":"DNA修复过程中引入的错误","paragraphId":"14995100656468240","subCatalogs":null},{"level":2,"title":"诱发突变","paragraphId":"14995100656468240","subCatalogs":null}]},{"level":1,"title":"类型分类","paragraphId":"14995100656468241","subCatalogs":[{"level":2,"title":"根据对结构的影响","paragraphId":"14995100656468241","subCatalogs":null},{"level":2,"title":"根据对功能的影响","paragraphId":"14995100656468241","subCatalogs":null},{"level":2,"title":"根据对适应性的影响","paragraphId":"14995100656468241","subCatalogs":null},{"level":2,"title":"根据对蛋白质序列的影响","paragraphId":"14995100656468241","subCatalogs":null},{"level":2,"title":"根据遗传性","paragraphId":"14995100656468241","subCatalogs":null},{"level":2,"title":"特殊类型","paragraphId":"14995100656468241","subCatalogs":null},{"level":2,"title":"系统命名法","paragraphId":"14995100656468241","subCatalogs":null}]},{"level":1,"title":"突变率","paragraphId":"14995100656468242","subCatalogs":null},{"level":1,"title":"有害突变","paragraphId":"14995100656468243","subCatalogs":null},{"level":1,"title":"有益突变","paragraphId":"14995100673245449","subCatalogs":null},{"level":1,"title":"朊病毒突变","paragraphId":"14995100673245450","subCatalogs":null},{"level":1,"title":"体细胞突变","paragraphId":"14995100673245451","subCatalogs":null},{"level":1,"title":"无效突变","paragraphId":"14995100673245452","subCatalogs":null},{"level":1,"title":"亚效和超效突变","paragraphId":"14995100673245453","subCatalogs":null},{"level":1,"title":"参考文献","paragraphId":"-1","subCatalogs":null}],"paragraphs":[{"paragraphId":"14995100639691029","title":"描述","versionId":"54406546892841478","lemmaId":10402,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":51990732,"name":"creeper","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599724829,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>突变可能涉及大部分DNA的复制，通常是通过基因重组。<sup><a href=\"#quote_2\" class=\"kx_ref\">[2]</a></sup> 这些复制是进化新基因的主要原料来源，每百万年动物基因组中就有几十到几百个基因被复制。<sup><a href=\"#quote_3\" class=\"kx_ref\">[3]</a></sup> 大多数基因属于共享祖先的更大基因家族，可通过它们的序列同源性来检测。<sup><a href=\"#quote_4\" class=\"kx_ref\">[4]</a></sup> 新基因通过几种方法产生，通常是通过祖先基因的复制和突变，或者通过重组不同基因的部分以形成具有新功能的新组合。<sup><a href=\"#quote_5\" class=\"kx_ref\">[5]</a></sup><sup><a href=\"#quote_6\" class=\"kx_ref\">[6]</a></sup> </p><p>在这里，蛋白质结构域充当模块，每个模块具有特定且独立的功能，可以混合在一起产生编码具有新特性的新蛋白质的基因。<sup><a href=\"#quote_7\" class=\"kx_ref\">[7]</a></sup> 例如，人眼利用四个基因来组成感知光的结构：三个用于视锥细胞或色觉，一个用于视杆细胞或夜视；这四个基因都来自同一个祖先基因。<sup><a href=\"#quote_8\" class=\"kx_ref\">[8]</a></sup> 复制一个基因(甚至整个基因组)的另一个优点是增加了工程冗余；从而允许这对基因中的其中一个获得新的功能，而另一个拷贝执行原始功能。<sup><a href=\"#quote_9\" class=\"kx_ref\">[9]</a></sup><sup><a href=\"#quote_10\" class=\"kx_ref\">[10]</a></sup> 其他类型的突变偶尔会从以前的非编码DNA产生新的基因。<sup><a href=\"#quote_11\" class=\"kx_ref\">[11]</a></sup><sup><a href=\"#quote_12\" class=\"kx_ref\">[12]</a></sup> </p><p>染色体数目的变化可能涉及更大的突变，染色体内的DNA片段断裂然后重新排列。例如，在人亚科，两条染色体融合产生了人类第2号染色体；这种融合并没有发生在其他猿类的谱系中，它们保留了这些独立的染色体。<sup><a href=\"#quote_13\" class=\"kx_ref\">[13]</a></sup> 在进化中，这种染色体重排最重要的作用可能是通过使种群不太可能杂交来加速种群向新物种的分化，从而保持这些种群之间的遗传差异。<sup><a href=\"#quote_14\" class=\"kx_ref\">[14]</a></sup> </p><p>可以在基因组中移动的DNA序列，例如转座子，构成了植物和动物遗传物质的主要部分，并且在基因组的进化中可能起了重要作用。<sup><a href=\"#quote_15\" class=\"kx_ref\">[15]</a></sup> 例如，人类基因组中存在100多万份Alu序列，这些序列现在已经被招募来执行诸如调节基因表达的功能。<sup><a href=\"#quote_16\" class=\"kx_ref\">[16]</a></sup> 这些可移动的DNA序列的另一个作用是，当它们在基因组中移动时，它们可以突变或删除现有的基因，从而产生遗传多样性。<sup><a href=\"#quote_17\" class=\"kx_ref\">[17]</a></sup> </p><p>非致死突变在基因库中积累，并增加遗传变异的数量。<sup><a href=\"#quote_17\" class=\"kx_ref\">[17]</a></sup> 基因库中一些基因变化的丰度可以通过自然选择而减少，而其他“更有利”的突变可能积累并导致适应性变化。</p><p><img alt=\"\" class=\"fileimage kx_img ed_imgfloat_right\" img_height=\"153\" img_width=\"199\" titlename=\"Prodryas persephone 是一种晚始新世蝴蝶品种。\" data-src=\"https://img02.sogoucdn.com/app/a/200698/sogou_science_11620\"> </p><p>例如，蝴蝶可能会产生带有新突变的后代。这些突变中的大多数都没有效果；但是人们可能会改变蝴蝶后代的颜色，使捕食者更难(或更容易)看到。如果这种颜色的变化是有利的，这种蝴蝶存活和繁殖后代的机率会稍微高一点，随着时间的推移，这种突变的蝴蝶数量可能会占总数量的更大比例。</p><p>中性突变被定义为其效应不影响个体适应性的突变。由于遗传漂变，这些频率会随着时间的推移而增加。据信绝大多数突变对生物体的适应性没有显著影响。<sup><a href=\"#quote_18\" class=\"kx_ref\">[18]</a></sup> 此外，DNA修复机制能够在大部分变化成为永久突变之前修复它们，并且许多生物体具有消除体细胞非永久突变的机制。</p><p>有益的突变可以提高生殖成功率。<sup><a href=\"#quote_19\" class=\"kx_ref\">[19]</a></sup><sup><a href=\"#quote_20\" class=\"kx_ref\">[20]</a></sup> </p>","pics":[{"originalUrl":"https://web.archive.org/web/20221025113755/https://img02.sogoucdn.com/app/a/200698/sogou_science_11620?w=199&h=153&titlename=Prodryas%20persephone%20%E6%98%AF%E4%B8%80%E7%A7%8D%E6%99%9A%E5%A7%8B%E6%96%B0%E4%B8%96%E8%9D%B4%E8%9D%B6%E5%93%81%E7%A7%8D%E3%80%82","url":"https://web.archive.org/web/20221025113755/https://img02.sogoucdn.com/app/a/200698/sogou_science_11620","rw":199,"rh":153,"title":"Prodryas persephone 是一种晚始新世蝴蝶品种。","alt":null,"width":0,"height":0}],"card":null,"references":[],"versionCount":0},{"paragraphId":"14995100656468239","title":"历史","versionId":"54406546892841479","lemmaId":10402,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":51990732,"name":"creeper","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599724829,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p><img alt=\"\" class=\"fileimage kx_img ed_imgfloat_right currentPic\" img_height=\"261\" img_width=\"300\" titlename=\"雨果·德·弗里斯画了一幅月见草的画，这种植物显然是在他的实验中发生了大量突变而产生了新的形态。 由Thérèse Schwartze于1918所作。\" data-src=\"https://img01.sogoucdn.com/app/a/200698/sogou_science_11621\"> </p><p>突变论是查尔斯·达尔文1859年著作《<i>物种起源》</i>出版前后存在的几种关于自然选择进化的理论之一， 。在理论上，突变是新事物的来源，创造新的形式和新的物种，可能是瞬间的，<sup><a href=\"#quote_21\" class=\"kx_ref\">[21]</a></sup> 跳跃性的。<sup><a href=\"#quote_22\" class=\"kx_ref\">[22]</a></sup> 突变被认为推动进化，进化受突变的限制。</p><p>在达尔文之前，生物学家通常相信跳跃论，即大规模进化跳跃的可能性，包括直接的物种形成。例如，1822年，艾蒂安·若弗鲁瓦·圣伊莱尔认为物种可以通过突然的转变，或者后来被称为大分子突变来形成。<sup><a href=\"#quote_23\" class=\"kx_ref\">[23]</a></sup> 达尔文反对跳跃，坚持进化论和地质学中的渐进主义。1864年，阿尔伯特·冯·科立克复活了杰弗里的理论。<sup><a href=\"#quote_24\" class=\"kx_ref\">[24]</a></sup> 1901年，遗传学家雨果·德·弗里斯给在月见草（<i>Oenothera lamarckiana）</i>上的实验中突然出现的看似新的形式命名为“突变”。在20世纪的第一个十年，突变论，或德弗里斯称之为 <i>突变理论</i>,<sup><a href=\"#quote_25\" class=\"kx_ref\">[25]</a></sup><sup><a href=\"#quote_21\" class=\"kx_ref\">[21]</a></sup> 成为在一段时间内受到包括威廉·贝特森<sup><a href=\"#quote_26\" class=\"kx_ref\">[26]</a></sup> ，托马斯·亨特·摩尔根和雷金纳德·普内特。<sup><a href=\"#quote_27\" class=\"kx_ref\">[27]</a></sup><sup><a href=\"#quote_21\" class=\"kx_ref\">[21]</a></sup>在内的遗传学家所支持的达尔文主义的竞争对手。</p><p>20世纪中期，现代综合论的支持者将早期突变学家描绘成达尔文进化论的反对者和生物测定学派（biometrics school）的反对者，认为选择依赖于连续的变异，这给对突变论的理解蒙上了阴影。在这幅描绘中，突变论被遗传学和自然选择的综合论所击败，据说这种综合论始于后来，大约1918年，数学家罗纳德·费雪的工作。<sup><a href=\"#quote_28\" class=\"kx_ref\">[28]</a></sup><sup><a href=\"#quote_29\" class=\"kx_ref\">[29]</a></sup><sup><a href=\"#quote_30\" class=\"kx_ref\">[30]</a></sup><sup><a href=\"#quote_31\" class=\"kx_ref\">[31]</a></sup> 然而，孟德尔遗传学和自然选择的一致性早在1902年就开始了，<sup><a href=\"#quote_32\" class=\"kx_ref\">[32]</a></sup> 并在欧洲和美国建立了理论和实验工作。尽管存在争议，早期的突变学家到1918年已经接受了自然选择，并解释了由于多个基因作用于同一特征(如身高)而导致的连续变异。<sup><a href=\"#quote_29\" class=\"kx_ref\">[29]</a></sup><sup><a href=\"#quote_30\" class=\"kx_ref\">[30]</a></sup> </p><p>突变论，以及其他达尔文主义理论，如拉马克主义和直生论，被大多数生物学家所抛弃，因为他们发现孟德尔遗传学和自然选择可以相互兼容；突变成为自然选择所必需的遗传变异的来源。 然而，突变论并没有完全消失。1940年，理查德·戈尔德施米特(Richard Goldschmidt)再次主张通过大分子突变进行单步物种形成，将由此产生的生物体描述为“充满希望的怪物”，然而受到广泛嘲笑。<sup><a href=\"#quote_33\" class=\"kx_ref\">[33]</a></sup><sup><a href=\"#quote_34\" class=\"kx_ref\">[34]</a></sup> 1987年，根井正利争论说进化通常是有限的突变。<sup><a href=\"#quote_35\" class=\"kx_ref\">[35]</a></sup> 现代生物学家道格拉斯·富图伊玛(Douglas J. Futuyma)认为，基本上所有关于由大突变驱动的进化的主张都可以用达尔文进化论来解释。<sup><a href=\"#quote_36\" class=\"kx_ref\">[36]</a></sup> </p>","pics":[{"originalUrl":"https://web.archive.org/web/20221025113755/https://img01.sogoucdn.com/app/a/200698/sogou_science_11621?w=300&h=261&titlename=%E9%9B%A8%E6%9E%9C%C2%B7%E5%BE%B7%C2%B7%E5%BC%97%E9%87%8C%E6%96%AF%E7%94%BB%E4%BA%86%E4%B8%80%E5%B9%85%E6%9C%88%E8%A7%81%E8%8D%89%E7%9A%84%E7%94%BB%EF%BC%8C%E8%BF%99%E7%A7%8D%E6%A4%8D%E7%89%A9%E6%98%BE%E7%84%B6%E6%98%AF%E5%9C%A8%E4%BB%96%E7%9A%84%E5%AE%9E%E9%AA%8C%E4%B8%AD%E5%8F%91%E7%94%9F%E4%BA%86%E5%A4%A7%E9%87%8F%E7%AA%81%E5%8F%98%E8%80%8C%E4%BA%A7%E7%94%9F%E4%BA%86%E6%96%B0%E7%9A%84%E5%BD%A2%E6%80%81%E3%80%82%20%E7%94%B1Th%C3%A9r%C3%A8se%20Schwartze%E4%BA%8E1918%E6%89%80%E4%BD%9C%E3%80%82","url":"https://web.archive.org/web/20221025113755/https://img01.sogoucdn.com/app/a/200698/sogou_science_11621","rw":300,"rh":261,"title":"雨果·德·弗里斯画了一幅月见草的画，这种植物显然是在他的实验中发生了大量突变而产生了新的形态。 由Thérèse Schwartze于1918所作。","alt":null,"width":0,"height":0}],"card":null,"references":[],"versionCount":0},{"paragraphId":"14995100656468240","title":"原因","versionId":"54231093301273095","lemmaId":10402,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82812762,"name":"雷克斯班纳","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599620250,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>四类突变是(1)自发突变(分子衰变)，(2)由于自然发生的DNA损伤的易错复制旁路引起的突变(也称为易错的跨损伤DNA复制)，(3)在DNA修复过程中引入的错误，和(4)由诱变剂引起的突变。为了科学实验的目的，科学家也可以通过DNA操作故意引入突变序列。</p><p>2017年的一项研究声称，66%的致癌突变是随机的，29%是由环境造成的(研究人群跨越69个国家)，5%是遗传的。<sup><a href=\"#quote_37\" class=\"kx_ref\">[37]</a></sup> </p><p>人类平均给他们的孩子传递60个新的突变，但是父亲根据他们的年龄传递更多的突变，每年给一个孩子增加两个新的突变。<sup><a href=\"#quote_38\" class=\"kx_ref\">[38]</a></sup> </p><p></p><h3>自发突变</h3><p></p><p><i>自发突变</i> 即使给定一个健康的、未受污染的细胞，也以非零概率发生。它们可以用特定的变化来描述:<sup><a href=\"#quote_39\" class=\"kx_ref\">[39]</a></sup> </p><p>\n </p><ul>\n  <li>互变异构现象——氢原子的重新定位改变了碱基，改变了碱基的氢键模式，导致复制过程中碱基配对不正确。</li>\n  <li>脱嘌呤——嘌呤碱基(A或G)的缺失形成一个无嘌呤位点(AP位点)。</li>\n  <li>脱氨基——水解将正常碱基变为含有酮基取代原始胺基的非典型碱基。例子包括C → U和A → HX(次黄嘌呤)，这可以通过DNA修复机制来纠正；和5MeC(5-甲基胞嘧啶)→ T，这是不太可能被检测为突变，因为胸腺嘧啶是正常的DNA碱基。</li>\n  <li>滑动链错配——在复制过程中，新链从模板变性，然后在不同的位点复性(“滑动”)。这可能导致插入或删除。</li>\n  <li>复制滑移</li>\n </ul><p></p><p></p><h3>易错的复制旁路</h3><p></p><p>越来越多的证据表明，大多数自发产生的突变是由于模板链过去的DNA损伤导致的易错复制(跨损伤DNA复制)。自然发生的氧化DNA损伤在人类中每天每细胞至少发生10，000次，在大鼠中每天每细胞至少发生50，000次或更多次。<sup><a href=\"#quote_40\" class=\"kx_ref\">[40]</a></sup> 在小鼠中，大多数突变是由跨损伤DNA复制引起的。<sup><a href=\"#quote_41\" class=\"kx_ref\">[41]</a></sup> 同样，在酵母中，Kunz等人<sup><a href=\"#quote_42\" class=\"kx_ref\">[42]</a></sup> 发现超过60%的自发单碱基对替换和缺失是由跨损伤DNA复制引起的。</p><p></p><h3>DNA修复过程中引入的错误</h3><p></p><p>虽然天然存在的双链断裂在DNA中发生的频率相对较低，但它们的修复通常会导致突变。非同源性末端接合(NHEJ)是修复双链断裂的主要途径。NHEJ涉及去除几个核苷酸，以允许重新连接的两端稍微不准确的对齐，然后添加核苷酸来填补空白。因此，NHEJ经常引入突变。<sup><a href=\"#quote_43\" class=\"kx_ref\">[43]</a></sup> </p><p><img alt=\"\" class=\"fileimage kx_img ed_imgfloat_right\" img_height=\"346\" img_width=\"250\" titlename=\"苯并[a]芘代谢物与DNA的共价化合物，苯并[a]芘是烟草烟雾中的主要诱变剂[8]\" data-src=\"https://img02.sogoucdn.com/app/a/200698/sogou_science_11628\"> </p><p></p><h3>诱发突变</h3><p></p><p>诱导突变是基因在与诱变剂和环境因素接触后的改变。</p><p><i>诱发突变</i> 在分子水平上可以由以下原因引起:</p><p>\n </p><ul>\n  <li>化学药品\n   <ul>\n    <li>羟胺</li>\n    <li>碱基类似物(例如，溴化去氧尿苷(BrdU))</li>\n    <li>烷化剂(例如， <i>N</i>-乙基-<i>N</i>-亚硝脲(ENU))。这些物质可以突变复制中和非复制中的DNA。相比之下，碱基类似物只有与复制中的DNA接触时才能使其突变。这些化学诱变剂中的每一种都有一定的作用，然后导致碱基转换、碱基颠换或缺失。</li>\n    <li>形成DNA加合物的试剂(如赭曲霉毒素A)<sup><a href=\"#quote_44\" class=\"kx_ref\">[44]</a></sup></li>\n    <li>DNA嵌入剂(例如溴化乙锭)</li>\n    <li>DNA交联剂</li>\n    <li>氧化损伤</li>\n    <li>亚硝酸将A和C上的胺基转化为重氮基，改变它们的氢键模式，导致复制过程中不正确的碱基配对。</li>\n   </ul></li>\n  <li>辐射\n   <ul>\n    <li>紫外线（UV）(非电离辐射)。DNA中的两个核苷酸碱基——胞嘧啶和胸腺嘧啶——最容易受到能改变其性质的辐射。紫外光可以诱导DNA链中相邻的嘧啶碱基共价结合成嘧啶二聚体。紫外线辐射，尤其是长波紫外线，也能对DNA造成氧化损伤。<sup><a href=\"#quote_45\" class=\"kx_ref\">[45]</a></sup></li>\n    <li>电离辐射。暴露于电离辐射，如伽马辐射，会导致突变，可能导致癌症或死亡。</li>\n   </ul></li>\n </ul><p></p>","pics":[{"originalUrl":"https://web.archive.org/web/20221025113755/https://img02.sogoucdn.com/app/a/200698/sogou_science_11628?w=250&h=346&titlename=%E8%8B%AF%E5%B9%B6%5Ba%5D%E8%8A%98%E4%BB%A3%E8%B0%A2%E7%89%A9%E4%B8%8EDNA%E7%9A%84%E5%85%B1%E4%BB%B7%E5%8C%96%E5%90%88%E7%89%A9%EF%BC%8C%E8%8B%AF%E5%B9%B6%5Ba%5D%E8%8A%98%E6%98%AF%E7%83%9F%E8%8D%89%E7%83%9F%E9%9B%BE%E4%B8%AD%E7%9A%84%E4%B8%BB%E8%A6%81%E8%AF%B1%E5%8F%98%E5%89%82%5B8%5D","url":"https://web.archive.org/web/20221025113755/https://img02.sogoucdn.com/app/a/200698/sogou_science_11628","rw":250,"rh":346,"title":"苯并[a]芘代谢物与DNA的共价化合物，苯并[a]芘是烟草烟雾中的主要诱变剂[8]","alt":null,"width":0,"height":0}],"card":null,"references":[],"versionCount":0},{"paragraphId":"14995100656468241","title":"类型分类","versionId":"54231093301273096","lemmaId":10402,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82812762,"name":"雷克斯班纳","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599620250,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p></p><h3>根据对结构的影响</h3><p></p><p><img alt=\"\" class=\"fileimage kx_img ed_imgfloat_right\" img_height=\"595\" img_width=\"301\" titlename=\"染色体的5种突变类型。\" data-src=\"https://img01.sogoucdn.com/app/a/200698/sogou_science_11634\"> </p><p><img alt=\"\" class=\"fileimage kx_img ed_imgfloat_right\" img_height=\"178\" img_width=\"301\" titlename=\"氨基酸遗传密码标准表中致病突变的选择。[1]\" data-src=\"https://img02.sogoucdn.com/app/a/200698/sogou_science_11636\"> </p><p>基因的序列可以通过多种方式改变。<sup><a href=\"#quote_46\" class=\"kx_ref\">[46]</a></sup> 基因突变对适应性有不同的影响，这取决于它们发生的位置以及它们是否改变了必需蛋白质的功能。基因结构的突变可以分为几种类型。</p><p><strong>小规模突变</strong></p><p>小规模突变影响一个基因的一个或几个核苷酸。(如果只有一个核苷酸受到影响，它们被称为点突变。)小规模突变包括:</p><p>\n </p><ul>\n  <li>插入将一个或多个额外的核苷酸添加到DNA中。它们通常是由转座因子或重复因子复制过程中的错误引起的。基因编码区的插入可以改变mRNA的剪接(剪接位点突变)，或者引起阅读框的改变(移码)，这两者都可以显著改变基因产物。插入可以通过切除转位因子来逆转。</li>\n  <li>缺失会从DNA中移除一个或多个核苷酸。像插入一样，这些突变可以改变基因的阅读框。一般来说，它们是不可逆的:虽然理论上完全相同的序列可以通过插入来恢复，但是转座因子能够在<i>任何</i>位置恢复非常短的缺失(比如1-2个碱基) 要么极不可能存在，要么根本不存在。</li>\n  <li>替换突变，通常由化学物质或DNA复制错误引起，用一个核苷酸替换另一个核苷酸。<sup><a href=\"#quote_47\" class=\"kx_ref\">[47]</a></sup> 这些变化被分类为碱基置换或碱基颠换。<sup><a href=\"#quote_48\" class=\"kx_ref\">[48]</a></sup> 最常见的是嘌呤与嘌呤(A↔G)或嘧啶与嘧啶(C↔T)的转换。碱基置换可以由亚硝酸、碱基错配或诱变碱基类似物如BrdU引起。不太常见的是碱基颠换，它将嘌呤交换为嘧啶或嘧啶交换为嘌呤(C/T ↔ A/G)。碱基颠换的一个例子是腺嘌呤(A)转化为胞嘧啶(C)。点突变是基因中单个碱基对或其他小碱基对的修饰。点突变可以通过另一个点突变来逆转，其中核苷酸被改变回其原始状态(真正的逆转)或通过第二位点逆转(在其他地方导致恢复基因功能的互补突变)。如下所述，发生在基因蛋白质编码区的点突变可分为同义或非同义替换，后者又可分为错义或无义突变。</li>\n </ul><p></p><p><strong>大规模突变</strong></p><p>染色体结构的大规模突变包括:</p><p>\n </p><ul>\n  <li>扩增(或基因复制)导致所有染色体区域的多个拷贝，增加了位于其中的基因的剂量。</li>\n  <li>大染色体区域的缺失，导致这些区域内的基因丢失。</li>\n  <li>其作用是将先前分离的DNA片段并置，潜在地将分离的基因聚集在一起形成功能不同的融合基因(例如bcr-abl)。</li>\n  <li>被称为染色体重排的染色体结构的大规模变化会导致适应性下降，但也会导致分离的近交群体中的物种形成。这些包括:\n   <ul>\n    <li>染色体易位:非同源染色体遗传部分的交换。</li>\n    <li>染色体反转:反转染色体片段的方向。</li>\n    <li>非同源染色体互换。</li>\n    <li>间隙缺失:染色体内的缺失，从单个染色体上除去一段DNA，从而与先前较远的基因并置。例如，从人类星形细胞瘤(一种脑肿瘤)分离的细胞被发现缺失了胶质母细胞瘤(FIG)基因和受体酪氨酸激酶(ROS)之间的一段序列，并产生融合蛋白(FIG-ROS)。异常的FIG-ROS融合蛋白具有持续激活的激酶活性，可引起致癌转化(从正常细胞转化为癌细胞)。</li>\n   </ul></li>\n  <li>杂合性丧失:在以前有两个不同等位基因的生物体中，由于缺失或基因重组事件而丧失一个等位基因。</li>\n </ul><p></p><p></p><h3>根据对功能的影响</h3><p></p><p>\n </p><ul>\n  <li>功能丧失突变，也称为失活突变，导致基因产物具有较少或没有功能(部分或全部失活)。当等位基因完全丧失功能(零等位基因)时，在穆勒的突变模式中，它通常被称为无效或无义突变。与这种突变相关的表型通常是隐性的。例外情况是当生物体是单倍体时，或者当正常基因产物的减少剂量不足以形成正常表型时(这被称为单倍剂量不足)。</li>\n  <li>功能获得突变，也称为激活突变，改变基因产物，使其效果变得更强(增强激活)，甚至被不同的异常功能所取代。当新的等位基因产生时，含有新产生的等位基因以及原始等位基因的杂合子表达新的等位基因；遗传上，这将突变定义为显性表型。穆勒的几种突变中对应于功能增益，包括超效等位基因和新效等位基因。2017年12月，美国政府解除了2014年实施的一项临时禁令，该禁令禁止联邦资助任何新的“功能增强”实验，以增强病原体“如流感禽流感病毒、非典病毒和中东呼吸系统综合症病毒或MER病毒”<sup><a href=\"#quote_49\" class=\"kx_ref\">[49]</a></sup></li>\n  <li>显性负突变(也称为反突变)的基因产物发生改变，与野生型等位基因产生对抗作用。这些突变通常导致分子功能改变(通常不活跃)，并以显性或半显性表型为特征。在人类中，显性负突变与癌症有关(例如，基因p53的突变，<sup><a href=\"#quote_50\" class=\"kx_ref\">[50]</a></sup> ATM，<sup><a href=\"#quote_51\" class=\"kx_ref\">[51]</a></sup> CEBPA<sup><a href=\"#quote_52\" class=\"kx_ref\">[52]</a></sup> 和PPARgamma<sup><a href=\"#quote_53\" class=\"kx_ref\">[53]</a></sup>)中。马方综合征是由 <i>FBN1</i> 基因，位于染色体15上，编码原纤维蛋白-1，细胞外基质的糖蛋白成分。<sup><a href=\"#quote_54\" class=\"kx_ref\">[54]</a></sup> 马方综合征也是显性负突变和单倍剂量不足的例子。<sup><a href=\"#quote_55\" class=\"kx_ref\">[55]</a></sup><sup><a href=\"#quote_56\" class=\"kx_ref\">[56]</a></sup></li>\n  <li>穆勒模式分类中，亚效突变的特征是基因产物发生改变，与野生型等位基因相比，基因表达降低。</li>\n  <li>新效突变的特征是控制新蛋白质产物的合成。</li>\n  <li>致死突变是导致携带突变的生物体死亡的突变。</li>\n  <li>反向突变或逆转是恢复原始序列并因此恢复原始表型的点突变。<sup><a href=\"#quote_57\" class=\"kx_ref\">[57]</a></sup></li>\n </ul><p></p><p></p><h3>根据对适应性的影响</h3><p></p><p>在应用遗传学中，通常说突变是有害的还是有益的。</p><p>\n </p><ul>\n  <li>有害或有害的突变会降低有机体的适应性。</li>\n  <li>有益或有利的突变增加了有机体的适应性。</li>\n  <li>中性突变对生物体没有有害或有益的影响。这种突变以稳定的速度发生，形成了分子钟的基础。在分子进化的中性学说，中性突变提供了遗传漂变作为分子水平上大多数变异的基础。</li>\n  <li>近中性突变是一种可能稍微有害或有利的突变，尽管大多数近中性突变是轻微有害的。</li>\n </ul><p></p><p><strong>适应性效应的分布</strong></p><p>已经尝试使用诱变实验和应用于分子序列数据的理论模型来推断适应性效应（DFE）的分布。DFE，用于确定不同类型突变(即，强烈有害、接近中性或有利)的相对丰度，与许多进化问题相关，例如遗传变异的维持，<sup><a href=\"#quote_58\" class=\"kx_ref\">[58]</a></sup> 基因组衰退的速度，<sup><a href=\"#quote_59\" class=\"kx_ref\">[59]</a></sup> 维持异交有性生殖，而不是近亲繁殖<sup><a href=\"#quote_60\" class=\"kx_ref\">[60]</a></sup> 和基因重组的进化。<sup><a href=\"#quote_61\" class=\"kx_ref\">[61]</a></sup> 总之，DFE在预测进化动态中起着重要作用。<sup><a href=\"#quote_62\" class=\"kx_ref\">[62]</a></sup><sup><a href=\"#quote_63\" class=\"kx_ref\">[63]</a></sup> 已经使用了各种方法来研究DFE，包括理论、实验和分析方法。</p><p>\n </p><ul>\n  <li>诱变实验:研究DFE的直接方法是诱导突变，然后测量突变适应性效应，这已经应用于病毒、细菌、酵母和 <i>果蝇</i>。例如，对病毒中DFE的大多数研究使用定点突变来产生点突变并测量每个突变的相对适应性。<sup><a href=\"#quote_64\" class=\"kx_ref\">[64]</a></sup><sup><a href=\"#quote_65\" class=\"kx_ref\">[65]</a></sup><sup><a href=\"#quote_66\" class=\"kx_ref\">[66]</a></sup><sup><a href=\"#quote_67\" class=\"kx_ref\">[67]</a></sup> 在<i>大肠杆菌</i>中的一项研究，使用转座子诱变直接测量随机插入Tn10衍生物的适应度。<sup><a href=\"#quote_68\" class=\"kx_ref\">[68]</a></sup> 在酵母中，已经开发了一种组合诱变和深度测序方法来产生高质量的系统突变体库，并以高通量测量其适应性。<sup><a href=\"#quote_69\" class=\"kx_ref\">[69]</a></sup> 然而，鉴于许多突变的影响太小而无法检测<sup><a href=\"#quote_70\" class=\"kx_ref\">[70]</a></sup> 诱变实验只能检测到中等效果的突变；DNA序列数据分析可以提供关于这些突变的有价值的信息。</li>\n </ul><p></p><p><img alt=\"\" class=\"fileimage kx_img ed_imgfloat_right\" img_height=\"210\" img_width=\"360\" titlename=\"水疱性口炎病毒突变的适应性效应(DFE)分布。在本实验中，通过定点诱变将随机突变引入病毒，并将每个突变体的适应性与祖先类型进行比较。适应度分别小于1和1以上，表明突变是致命的、有害的、中性的和有利的。[20]\" data-src=\"https://img02.sogoucdn.com/app/a/200698/sogou_science_11639\"> </p><p>\n </p><ul>\n  <li>分子序列分析:随着DNA测序技术的快速发展，可以获得大量的DNA序列数据，将来还会有更多的数据。已经开发了各种方法来从DNA序列数据推断DFE。<sup><a href=\"#quote_71\" class=\"kx_ref\">[71]</a></sup><sup><a href=\"#quote_72\" class=\"kx_ref\">[72]</a></sup><sup><a href=\"#quote_73\" class=\"kx_ref\">[73]</a></sup><sup><a href=\"#quote_74\" class=\"kx_ref\">[74]</a></sup> 通过检测物种内部和物种之间的DNA序列差异，我们能够推断中性、有害和有利突变的DFE的各种特征。<sup><a href=\"#quote_75\" class=\"kx_ref\">[75]</a></sup> 具体来说，DNA序列分析方法允许我们估计突变的影响，而这种影响非常小，很难通过诱变实验检测出来。</li>\n </ul><p></p><p>适应性效应分布的最早理论研究之一是由木村资生完成的，他是一位有影响力的理论群体遗传学家。他的分子进化中立原理提出，大多数新突变都是非常有害的，一小部分是中性的。<sup><a href=\"#quote_75\" class=\"kx_ref\">[75]</a></sup><sup><a href=\"#quote_76\" class=\"kx_ref\">[76]</a></sup> 藤原皓明石最近提出了一个双峰DFE模型，双峰集中在高度有害和中性突变。<sup><a href=\"#quote_76\" class=\"kx_ref\">[76]</a></sup> 两种理论都认为绝大多数新突变是中性的或有害的，有利的突变很少，这得到了实验结果的支持。一个例子是对水泡性口炎病毒随机突变的DFE的研究。<sup><a href=\"#quote_64\" class=\"kx_ref\">[64]</a></sup> 在所有突变中，39.6%是致命的，31.2%是非致命有害的，27.1%是中性的。另一个例子来自酵母的高通量诱变实验。<sup><a href=\"#quote_69\" class=\"kx_ref\">[69]</a></sup> 在这个实验中，显示了整个DFE是双峰的，带有一簇中性突变和广泛分布的有害突变。</p><p>尽管相对较少的突变是有利的，但它们在进化变化中起重要作用。<sup><a href=\"#quote_77\" class=\"kx_ref\">[77]</a></sup> 像中性突变一样，弱选择的有利突变可能由于随机遗传漂变而丢失，但是强选择的有利突变更有可能被固定。了解有利突变的DFE可能会提高预测进化动态的能力。约翰·吉莱斯皮<sup><a href=\"#quote_78\" class=\"kx_ref\">[78]</a></sup>和艾伦·奥尔<sup><a href=\"#quote_79\" class=\"kx_ref\">[79]</a></sup>对有利突变的DFE进行了理论研究。 他们提出有利突变的分布在广泛的条件下应该是指数的，这通常得到实验研究的支持，至少对于强选择的有利突变是如此。<sup><a href=\"#quote_80\" class=\"kx_ref\">[80]</a></sup><sup><a href=\"#quote_81\" class=\"kx_ref\">[81]</a></sup><sup><a href=\"#quote_82\" class=\"kx_ref\">[82]</a></sup> </p><p>一般来说，大多数突变是中性或有害的，有利的突变很少；然而，突变类型的比例因物种而异。这表明了两点:第一，由于对有效群体大小的依赖，有效中性突变的比例可能因物种而异；第二，有害突变的平均效应在不同物种间差异很大。<sup><a href=\"#quote_75\" class=\"kx_ref\">[75]</a></sup> 此外，编码区和非编码区的DFE也不同，非编码DNA的DFE包含更弱选择的突变。<sup><a href=\"#quote_75\" class=\"kx_ref\">[75]</a></sup> </p><p></p><h3>根据对蛋白质序列的影响</h3><p></p><p>\n </p><ul>\n  <li>移码突变是一种由插入或缺失一些核苷酸引起的突变，这些核苷酸不是3的整数倍。由于密码子基因表达的三重性，插入或缺失会破坏阅读框或密码子的分组，导致与原始翻译完全不同的翻译。<sup><a href=\"#quote_83\" class=\"kx_ref\">[83]</a></sup> 序列中缺失或插入发生得越早，产生的蛋白质变化就越大。(例如，密码子CCU GAC UAC CUA编码脯氨酸、天冬氨酸、酪氨酸和亮氨酸。 如果CCU中的U被删除，得到的序列将是CCG ACU ACC UAx，它将编码脯氨酸、苏氨酸、苏氨酸和另一个氨基酸的一部分，或者可能是终止密码子(其中x代表下一个核苷酸)。)相比之下，任何可被3整除的插入或删除都称为<i>整码突变</i>。</li>\n  <li>点替换突变导致单个核苷酸的变化，可以是同义的，也可以是非同义的。\n   <ul>\n    <li>同义替换是用编码相同氨基酸的另一个密码子替换原密码子，从而产生的氨基酸序列没有被修改。同义突变是由于遗传密码的简并性而发生的。如果这种突变没有导致任何表型效应，那么它被称为沉默，但不是所有同义替换都是沉默的。(编码区之外的核苷酸，如内含子，也可能有沉默突变，因为准确的核苷酸序列并不像编码区那样重要，但它们不被认为是同义替换。)</li>\n    <li>非同义替换是用编码不同氨基酸的另一个密码子替换原密码子，从而改变产生的氨基酸序列。非同义替换可分为无义或错义突变:\n     <ul>\n      <li>错义突变改变核苷酸，导致不同氨基酸的替换。这反过来会导致产生的蛋白质不起作用。这种突变是导致大疱性表皮松解症、镰状细胞病和SOD1介导的肌萎缩侧索硬化症等疾病的原因。<sup><a href=\"#quote_84\" class=\"kx_ref\">[84]</a></sup> 另一方面，如果错义突变出现在氨基酸密码子中，导致使用不同但化学性质相似的氨基酸，那么有时蛋白质中很少或没有变化。例如，从AAA到AGA的变化将编码精氨酸，一种与预期赖氨酸化学相似的分子。在后一种情况下，突变对表型影响很小或没有影响，因此是中性的。</li>\n      <li>无义突变是导致DNA序列中突变为提前终止密码子的点突变，或<i>无义密码子</i>在转录的mRNA中，可能是截短的，通常是无功能的蛋白质产物。这种突变与不同的突变有关，如先天性肾上腺增生。(见终止密码子。)</li>\n     </ul></li>\n   </ul></li>\n </ul><p></p><p></p><h3>根据遗传性</h3><p></p><p><img alt=\"\" class=\"fileimage kx_img ed_imgfloat_right\" img_height=\"346\" img_width=\"300\" titlename=\"一种突变导致这种苔藓玫瑰植物产生不同颜色的花。这是一种体细胞突变，也可能在生殖细胞中传播。\" data-src=\"https://img04.sogoucdn.com/app/a/200698/sogou_science_11642\"> </p><p>在具有专门生殖细胞的多细胞生物中，突变可细分为种系突变和体细胞突变(也称为获得性突变)。种系突变可通过生殖细胞传递给后代。 体细胞突变涉及专用生殖组之外的细胞，并且通常不传播给后代。</p><p>种系突变在后代中产生<i>天生突变</i>，即存在于每个细胞中的突变。一个天生突变也可能在受精后很快发生，或者从父母先前的天生突变延续下来。<sup><a href=\"#quote_85\" class=\"kx_ref\">[85]</a></sup> </p><p>种系突变和体细胞突变之间的区别对于有专门种系产生生殖细胞的动物来说很重要。然而，这对于理解植物突变的效应没有什么价值，因为植物缺乏专门的生殖系。在那些通过出芽等机制无性繁殖的动物中，这种区别也很模糊，因为产生子代生物体的细胞也会产生该生物体的生殖系。一个不是从双亲遗传的新种系突变称为 <i>de novo</i> 突变。</p><p>二倍体生物(如人类)包含每个基因的两个拷贝——一个父系等位基因和一个母系等位基因。根据每条染色体上突变的发生情况，我们可以将突变分为三种类型。</p><p>\n </p><ul>\n  <li>杂合子突变是只有一个等位基因发生突变。</li>\n  <li>纯合突变是父系和母系等位基因发生了相同的突变。</li>\n  <li>复合杂合突变或遗传复合由来自两种不同的父系和母系等位基因的突变组成。<sup><a href=\"#quote_86\" class=\"kx_ref\">[86]</a></sup></li>\n </ul><p></p><p>野生型或纯合非突变生物是指两个等位基因都没有发生突变的生物。</p><p></p><h3>特殊类型</h3><p></p><p>\n </p><ul>\n  <li>条件突变是指在某些“许可”环境条件下具有野生型(或不太严重)表型，而在某些“限制性”条件下具有突变表型的突变。例如，温度敏感突变可在高温(限制性条件)下导致细胞死亡，但在低温(许可条件)下可能没有有害后果。<sup><a href=\"#quote_87\" class=\"kx_ref\">[87]</a></sup> 这些突变是非自主的，因为它们的表现取决于某些条件的存在，与其他自主出现的突变不同。<sup><a href=\"#quote_88\" class=\"kx_ref\">[88]</a></sup> 许可条件可以是温度，<sup><a href=\"#quote_89\" class=\"kx_ref\">[89]</a></sup> 某些化学物质，<sup><a href=\"#quote_90\" class=\"kx_ref\">[90]</a></sup> 光线<sup><a href=\"#quote_90\" class=\"kx_ref\">[90]</a></sup> 或者基因组其他部分的突变。<sup><a href=\"#quote_88\" class=\"kx_ref\">[88]</a></sup> <i>体内</i>机制如转录开关等可以产生条件突变。例如，类固醇结合结构域的结合可以创建转录开关，该转录开关基于类固醇配体的存在来改变基因的表达。<sup><a href=\"#quote_91\" class=\"kx_ref\">[91]</a></sup> 条件突变在研究中已经有应用，因为它们允许控制基因的表达。这对于研究成年人的疾病特别有用，因为它允许在一定的生长时期后进行表达，从而消除了在模式生物发育阶段看到的基因表达的有害影响。<sup><a href=\"#quote_90\" class=\"kx_ref\">[90]</a></sup> DNA重组酶系统，如Cre-Lox重组，与在特定条件下激活的启动子结合使用，可以产生条件突变。双重组酶技术可用于诱导多种条件突变，以研究多种基因同时突变导致的疾病。<sup><a href=\"#quote_90\" class=\"kx_ref\">[90]</a></sup> 已经鉴定出某些内含物仅在特定许可温度下剪接，导致蛋白质合成不当，从而在其他温度下丧失功能突变。<sup><a href=\"#quote_92\" class=\"kx_ref\">[92]</a></sup> 条件突变也可以用于与衰老相关的基因研究，因为在生物体生命周期的某个时间段后，其表达会发生变化。<sup><a href=\"#quote_89\" class=\"kx_ref\">[89]</a></sup></li>\n  <li>复制时间数量性状位点影响DNA的复制。</li>\n </ul><p></p><p></p><h3>系统命名法</h3><p></p><p>为了对突变进行分类，必须从“正常”或“健康”生物体(相对于“突变”或“病态”生物体)的DNA中获得“正常”序列，并对其进行鉴定和报道；理想的情况是，它应该可以公共获取以方便进行明确的核苷酸与核苷酸之间的比较，并得到科学界或一组遗传学家和生物学家等专家的公认，他们有责任建立<i>标准</i>或者所谓的“共识”序列。这一步需要巨大的科学努力。一旦知道了公认序列，基因组中的突变就可以被精确定位、描述和分类。人类基因组变异学会委员会（HGVS）已经开发了标准的人类序列变异命名法，<sup><a href=\"#quote_93\" class=\"kx_ref\">[93]</a></sup> 这应该被研究者和DNA诊断中心用来产生明确的突变描述。原则上，这个术语也可以用来描述其他生物体的突变。命名法规定了突变的类型和碱基或氨基酸的变化。</p><p>\n </p><ul>\n  <li>核苷酸替换(例如，76A &gt; T)-数字是核苷酸从5’端开始的位置；第一个字母代表野生型核苷酸，第二个字母代表替换野生型核苷酸的核苷酸。在给定的例子中，第76位的腺嘌呤被胸腺嘧啶替换。\n   <ul>\n    <li>如果有必要区分基因组DNA、线粒体DNA和RNA的突变，可以使用简单的约定。例如，如果核苷酸序列的第100个碱基从G突变为C，那么如果突变发生在基因组DNA中，则记为G . 100G &gt; C；如果突变发生在线粒体DNA中，则记为m . 100G &gt; C；如果突变发生在RNA中，则记为r.100g&gt;c。注意，对于RNA的突变，核苷酸是用小写字母写的。</li>\n   </ul></li>\n  <li>氨基酸替换(例如，D111E)-第一个字母是野生型氨基酸的单字母代码，数字是来自N端氨基酸的位置，第二个字母是突变中存在的氨基酸的单字母代码。无义突变第二个氨基酸用X表示(例如D111X)。</li>\n  <li>氨基酸缺失(如，ΔF508)——希腊字母Δ(delta)表示缺失。字母指的是野生型氨基酸，数字是野生型氨基酸在N端的位置。</li>\n </ul><p></p>","pics":[{"originalUrl":"https://web.archive.org/web/20221025113755/https://img01.sogoucdn.com/app/a/200698/sogou_science_11634?w=301&h=595&titlename=%E6%9F%93%E8%89%B2%E4%BD%93%E7%9A%845%E7%A7%8D%E7%AA%81%E5%8F%98%E7%B1%BB%E5%9E%8B%E3%80%82","url":"https://web.archive.org/web/20221025113755/https://img01.sogoucdn.com/app/a/200698/sogou_science_11634","rw":301,"rh":595,"title":"染色体的5种突变类型。","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221025113755/https://img02.sogoucdn.com/app/a/200698/sogou_science_11636?w=301&h=178&titlename=%E6%B0%A8%E5%9F%BA%E9%85%B8%E9%81%97%E4%BC%A0%E5%AF%86%E7%A0%81%E6%A0%87%E5%87%86%E8%A1%A8%E4%B8%AD%E8%87%B4%E7%97%85%E7%AA%81%E5%8F%98%E7%9A%84%E9%80%89%E6%8B%A9%E3%80%82%5B1%5D","url":"https://web.archive.org/web/20221025113755/https://img02.sogoucdn.com/app/a/200698/sogou_science_11636","rw":301,"rh":178,"title":"氨基酸遗传密码标准表中致病突变的选择。[1]","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221025113755/https://img02.sogoucdn.com/app/a/200698/sogou_science_11639?w=360&h=210&titlename=%E6%B0%B4%E7%96%B1%E6%80%A7%E5%8F%A3%E7%82%8E%E7%97%85%E6%AF%92%E7%AA%81%E5%8F%98%E7%9A%84%E9%80%82%E5%BA%94%E6%80%A7%E6%95%88%E5%BA%94(DFE)%E5%88%86%E5%B8%83%E3%80%82%E5%9C%A8%E6%9C%AC%E5%AE%9E%E9%AA%8C%E4%B8%AD%EF%BC%8C%E9%80%9A%E8%BF%87%E5%AE%9A%E7%82%B9%E8%AF%B1%E5%8F%98%E5%B0%86%E9%9A%8F%E6%9C%BA%E7%AA%81%E5%8F%98%E5%BC%95%E5%85%A5%E7%97%85%E6%AF%92%EF%BC%8C%E5%B9%B6%E5%B0%86%E6%AF%8F%E4%B8%AA%E7%AA%81%E5%8F%98%E4%BD%93%E7%9A%84%E9%80%82%E5%BA%94%E6%80%A7%E4%B8%8E%E7%A5%96%E5%85%88%E7%B1%BB%E5%9E%8B%E8%BF%9B%E8%A1%8C%E6%AF%94%E8%BE%83%E3%80%82%E9%80%82%E5%BA%94%E5%BA%A6%E5%88%86%E5%88%AB%E5%B0%8F%E4%BA%8E1%E5%92%8C1%E4%BB%A5%E4%B8%8A%EF%BC%8C%E8%A1%A8%E6%98%8E%E7%AA%81%E5%8F%98%E6%98%AF%E8%87%B4%E5%91%BD%E7%9A%84%E3%80%81%E6%9C%89%E5%AE%B3%E7%9A%84%E3%80%81%E4%B8%AD%E6%80%A7%E7%9A%84%E5%92%8C%E6%9C%89%E5%88%A9%E7%9A%84%E3%80%82%5B20%5D","url":"https://web.archive.org/web/20221025113755/https://img02.sogoucdn.com/app/a/200698/sogou_science_11639","rw":360,"rh":210,"title":"水疱性口炎病毒突变的适应性效应(DFE)分布。在本实验中，通过定点诱变将随机突变引入病毒，并将每个突变体的适应性与祖先类型进行比较。适应度分别小于1和1以上，表明突变是致命的、有害的、中性的和有利的。[20]","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221025113755/https://img04.sogoucdn.com/app/a/200698/sogou_science_11642?w=300&h=346&titlename=%E4%B8%80%E7%A7%8D%E7%AA%81%E5%8F%98%E5%AF%BC%E8%87%B4%E8%BF%99%E7%A7%8D%E8%8B%94%E8%97%93%E7%8E%AB%E7%91%B0%E6%A4%8D%E7%89%A9%E4%BA%A7%E7%94%9F%E4%B8%8D%E5%90%8C%E9%A2%9C%E8%89%B2%E7%9A%84%E8%8A%B1%E3%80%82%E8%BF%99%E6%98%AF%E4%B8%80%E7%A7%8D%E4%BD%93%E7%BB%86%E8%83%9E%E7%AA%81%E5%8F%98%EF%BC%8C%E4%B9%9F%E5%8F%AF%E8%83%BD%E5%9C%A8%E7%94%9F%E6%AE%96%E7%BB%86%E8%83%9E%E4%B8%AD%E4%BC%A0%E6%92%AD%E3%80%82","url":"https://web.archive.org/web/20221025113755/https://img04.sogoucdn.com/app/a/200698/sogou_science_11642","rw":300,"rh":346,"title":"一种突变导致这种苔藓玫瑰植物产生不同颜色的花。这是一种体细胞突变，也可能在生殖细胞中传播。","alt":null,"width":0,"height":0}],"card":null,"references":[],"versionCount":0},{"paragraphId":"14995100656468242","title":"突变率","versionId":"54406546892841480","lemmaId":10402,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":51990732,"name":"creeper","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599724829,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>不同物种的突变率差异很大，通常决定突变的进化力是正在研究中的主题</p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14995100656468243","title":"有害突变","versionId":"54406546892841481","lemmaId":10402,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":51990732,"name":"creeper","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599724829,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>由突变引起的DNA变化会导致蛋白质序列的错误，产生部分或完全不起作用的蛋白质。每个细胞，为了正常运作，依赖成千上万的蛋白质在正确的时间和正确的地点起作用。当突变改变了在身体中起关键作用的蛋白质时，可能会导致医学状况。有些突变会改变基因的DNA碱基序列，但不会改变基因所产生的蛋白质的功能。一项在<i>果蝇</i>不同种间的基因比较研究表明，如果突变确实改变了蛋白质，这可能是有害的，估计70%的氨基酸多态性具有破坏性影响，其余的是中性或弱有益的。<sup><a href=\"#quote_94\" class=\"kx_ref\">[94]</a></sup> 研究表明，酵母非编码DNA中只有7%的点突变是有害的，编码DNA中只有12%是有害的。其余的突变要么是中性的，要么是轻微有益的。<sup><a href=\"#quote_94\" class=\"kx_ref\">[94]</a></sup> </p><p>如果一个突变存在于生殖细胞中，它可以产生在其所有细胞中携带该突变的后代。遗传疾病就是这种情况。特别是，如果生殖细胞内的DNA修复基因发生突变，携带这种种系突变的人患癌症的风险可能会增加。34个这种种系突变的列表在文章《DNA修复缺陷疾病》中给出。一个例子是白化病，一种发生在OCA1或OCA2基因中的突变。患有这种疾病的人更容易患多种癌症和其他疾病，并且视力受损。另一方面，突变可能发生在生物体的体细胞中。这种突变将存在于同一生物体中该细胞的所有后代中，某些突变会导致细胞变得恶性，从而导致癌症。<sup><a href=\"#quote_95\" class=\"kx_ref\">[95]</a></sup> </p><p>当DNA被复制时，DNA的损坏会导致错误，而这种复制错误会导致基因突变，进而导致遗传疾病。DNA损伤由细胞的DNA修复系统修复。每个细胞都有许多途径，酶通过这些途径识别和修复DNA损伤。因为DNA可以在许多方面被破坏，DNA修复的过程是身体保护自己免于疾病的重要途径。一旦DNA损伤导致突变，突变即无法修复。</p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14995100673245449","title":"有益突变","versionId":"54406546892841482","lemmaId":10402,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":51990732,"name":"creeper","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599724829,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>虽然引起蛋白质序列变化的突变可能对生物体有害，但有时在特定环境中这种影响可能是积极的。在这种情况下，突变可以使突变生物比野生型生物更好地承受特定的环境压力，或者更快地繁殖。在这些情况下，通过自然选择，突变在群体中会变得更加普遍。例子包括:</p><p>艾滋病毒抗性:人类<i>CCR5</i>基因中特异性32碱基对缺失(CCR5-Δ32)赋予纯合子艾滋病毒抗性，并延迟杂合子艾滋病发病。<sup><a href=\"#quote_96\" class=\"kx_ref\">[96]</a></sup> 对CCR5-Δ32在欧洲人群中相对较高频率的病因学的一个可能解释是，它在14世纪中叶赋予了欧洲对腺鼠疫的抵抗力。具有这种突变的人更有可能在感染中存活；因此它在人口中的频率增加了。<sup><a href=\"#quote_97\" class=\"kx_ref\">[97]</a></sup> 这个理论可以解释为什么在南部非洲没有发现这种突变，因为那里没有受到淋巴腺鼠疫的影响。一个较新的理论表明，CCR5-Δ32突变上的选择压力是由天花而不是腺鼠疫引起的。<sup><a href=\"#quote_98\" class=\"kx_ref\">[98]</a></sup> </p><p>疟疾抗性:有害突变的一个例子是镰状细胞病，一种血液疾病，身体在红细胞中产生一种异常类型的携氧物质血红蛋白。撒哈拉以南非洲三分之一的土著居民携带该等位基因，因为在疟疾常见的地区，携带单一镰状细胞等位基因(镰状细胞性状)的人具有更高的生存率。<sup><a href=\"#quote_99\" class=\"kx_ref\">[99]</a></sup> 只有镰状细胞病两个等位基因中其中一个的人对疟疾更有抵抗力，因为疟疾（<i>疟原虫）</i>的侵扰被镰状细胞抑制。</p><p>抗生素抗性:几乎所有细菌在接触抗生素时都会产生抗生素耐药性。事实上，细菌群体已经有了这样的突变，可以通过抗生素的选择来筛选。<sup><a href=\"#quote_100\" class=\"kx_ref\">[100]</a></sup> 显然，这种突变只对细菌有益，而对受感染的人无益。</p><p>“乳糖酶耐受性”<i>。</i>一种突变允许人类在自然断奶后表达乳糖酶，从而允许成年人消化乳糖，这可能是最近人类进化中最有益的突变之一。<i><sup><a href=\"#quote_101\" class=\"kx_ref\">[101]</a></sup></i> </p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14995100673245450","title":"朊病毒突变","versionId":"54406546892841483","lemmaId":10402,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":51990732,"name":"creeper","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599724829,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>朊病毒是蛋白质，不含遗传物质。然而，朊病毒复制已经显示出像其他形式的复制一样易受突变和自然选择的影响。<sup><a href=\"#quote_102\" class=\"kx_ref\">[102]</a></sup> 人类基因PRNP编码主要朊病毒蛋白PrP，并且易发生突变，从而导致致病朊病毒。</p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14995100673245451","title":"体细胞突变","versionId":"54406546892841484","lemmaId":10402,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":51990732,"name":"creeper","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599724829,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>遗传结构的变化不是从父母那里遗传的，也不会传给后代，这种变化被称为体细胞突变<i>。<sup><a href=\"#quote_103\" class=\"kx_ref\">[103]</a></sup> 体细胞突变不是遗传的，因为它们不影响种系。这些类型的突变通常是由环境原因引起的，例如紫外线辐射或任何暴露于某些有害化学物质的情况，并可能导致包括癌症在内的疾病。<sup><a href=\"#quote_103\" class=\"kx_ref\">[103]</a></sup></i> </p><p>对于植物，一些体细胞突变可以在不需要种子产生的情况下繁殖，例如，通过嫁接和茎插条。这些类型的突变产生了新的水果类型，如“美味”苹果和“华盛顿”脐橙。<sup><a href=\"#quote_104\" class=\"kx_ref\">[104]</a></sup> </p><p>人类和小鼠体细胞的突变率比两个物种的种系突变率高十倍以上；小鼠每个细胞分裂的体细胞和生殖细胞的突变率都要高于人类。种系和体细胞组织之间突变率的差异可能反映了种系中基因组的维护比体细胞中基因组的维护更重要。<sup><a href=\"#quote_105\" class=\"kx_ref\">[105]</a></sup> </p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14995100673245452","title":"无效突变","versionId":"54406546892841485","lemmaId":10402,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":51990732,"name":"creeper","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599724829,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>穆勒在1932年使用的术语“amorph”是一种突变的等位基因，它失去了亲本(无论是野生型还是任何其他类型)等位基因编码任何功能蛋白的能力。无效突变可能是由使酶失活的氨基酸替换或产生酶的基因部分缺失引起的。</p><p>具有杂合突变(基因含一个正常拷贝和一个突变拷贝)的细胞可以在没有突变拷贝的情况下正常工作，直到正常拷贝在身体上自发突变。这种突变在生物体中一直存在，但是很难检测其发生率。检测这个比率对于预测人们患癌症的概率很重要。<sup><a href=\"#quote_106\" class=\"kx_ref\">[106]</a></sup> </p><p>点突变可能源于DNA复制过程中发生的自发突变。诱变剂可以提高突变率。诱变剂可以是物理的，例如来自紫外线、X射线或极热的辐射，或化学的(使碱基对错位或破坏DNA双螺旋结构的分子)。经常研究与癌症相关的诱变剂从而了解癌症及其预防。</p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14995100673245453","title":"亚效和超效突变","versionId":"54406546892841486","lemmaId":10402,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":51990732,"name":"creeper","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599724829,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>亚效突变是导致基因表达量降低的突变。通常，亚效突变是隐性的，但是单倍剂量不足导致一些等位基因是显性的。</p><p>超效突变导致基因表达量增加。</p>","pics":null,"card":null,"references":[],"versionCount":0}],"references":[{"id":1,"type":"book","title":"\"mutation | Learn Science at Scitable\". www.nature.com (in 英语). Retrieved 2018-09-24.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":2,"type":"book","title":"Hastings PJ, Lupski JR, Rosenberg SM, Ira G (August 2009). \"Mechanisms of change in gene copy number\". Nature Reviews Genetics. 10 (8): 551–64. doi:10.1038/nrg2593. PMC 2864001. PMID 19597530.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":3,"type":"book","title":"Carroll SB, Grenier JK, Weatherbee SD (2005). From DNA to Diversity: Molecular Genetics and the Evolution of Animal Design (2nd ed.). Malden, MA: Blackwell Publishing. ISBN 978-1-4051-1950-4. LCCN 2003027991. OCLC 53972564.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":4,"type":"book","title":"Harrison PM, Gerstein M (May 2002). \"Studying genomes through the aeons: protein families, pseudogenes and proteome evolution\". Journal of Molecular Biology. 318 (5): 1155–74. doi:10.1016/S0022-2836(02)00109-2. PMID 12083509.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":5,"type":"book","title":"Orengo CA, Thornton JM (July 2005). \"Protein families and their evolution-a structural perspective\". Annual Review of Biochemistry. 74: 867–900. doi:10.1146/annurev.biochem.74.082803.133029. PMID 15954844.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":6,"type":"book","title":"Long M, Betrán E, Thornton K, Wang W (November 2003). \"The origin of new genes: glimpses from the young and old\". Nature Reviews Genetics. 4 (11): 865–75. doi:10.1038/nrg1204. PMID 14634634.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":7,"type":"book","title":"Wang M, Caetano-Anollés G (January 2009). \"The evolutionary mechanics of domain organization in proteomes and the rise of modularity in the protein world\". Structure. 17 (1): 66–78. doi:10.1016/j.str.2008.11.008. PMID 19141283.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":8,"type":"book","title":"Bowmaker JK (May 1998). \"Evolution of colour vision in vertebrates\". Eye. 12 (Pt 3b): 541–7. doi:10.1038/eye.1998.143. PMID 9775215.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":9,"type":"book","title":"Gregory TR, Hebert PD (April 1999). \"The modulation of DNA content: proximate causes and ultimate consequences\". Genome Research. 9 (4): 317–24. doi:10.1101/gr.9.4.317 (inactive 2019-03-11). PMID 10207154.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":10,"type":"book","title":"Hurles M (July 2004). \"Gene duplication: the genomic trade in spare parts\". PLoS Biology. 2 (7): E206. doi:10.1371/journal.pbio.0020206. PMC 449868. PMID 15252449.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":11,"type":"book","title":"Liu N, Okamura K, Tyler DM, Phillips MD, Chung WJ, Lai EC (October 2008). \"The evolution and functional diversification of animal microRNA genes\". Cell Research. 18 (10): 985–96. doi:10.1038/cr.2008.278. PMC 2712117. PMID 18711447.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":12,"type":"book","title":"Siepel A (October 2009). \"Darwinian alchemy: Human genes from noncoding DNA\". Genome Research. 19 (10): 1693–5. doi:10.1101/gr.098376.109. PMC 2765273. PMID 19797681.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":13,"type":"book","title":"Zhang J, Wang X, Podlaha O (May 2004). \"Testing the chromosomal speciation hypothesis for humans and chimpanzees\". Genome Research. 14 (5): 845–51. doi:10.1101/gr.1891104. PMC 479111. PMID 15123584.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":14,"type":"book","title":"Ayala FJ, Coluzzi M (May 2005). \"Chromosome speciation: humans, Drosophila, and mosquitoes\". Proceedings of the National Academy of Sciences of the United States of America. 102 Suppl 1 (Suppl 1): 6535–42. Bibcode:2005PNAS..102.6535A. doi:10.1073/pnas.0501847102. PMC 1131864. PMID 15851677.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":15,"type":"book","title":"Hurst GD, Werren JH (August 2001). \"The role of selfish genetic elements in eukaryotic evolution\". Nature Reviews Genetics. 2 (8): 597–606. doi:10.1038/35084545. PMID 11483984.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":16,"type":"book","title":"Häsler J, Strub K (November 2006). \"Alu elements as regulators of gene expression\". Nucleic Acids Research. 34 (19): 5491–7. doi:10.1093/nar/gkl706. PMC 1636486. PMID 17020921.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":17,"type":"book","title":"Aminetzach YT, Macpherson JM, Petrov DA (July 2005). \"Pesticide resistance via transposition-mediated adaptive gene truncation in Drosophila\". Science. 309 (5735): 764–7. Bibcode:2005Sci...309..764A. doi:10.1126/science.1112699. PMID 16051794.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":18,"type":"book","title":"Bohidar HB (January 2015). Fundamentals of Polymer Physics and Molecular Biophysics. Cambridge University Press. ISBN 978-1-316-09302-3.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":19,"type":"book","title":"Dover, Gabriel A.; Darwin, Charles (2000). Dear Mr. Darwin: Letters on the Evolution of Life and Human Nature (in 英语). University of California Press. ISBN 9780520227903.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":20,"type":"book","title":"Tibayrenc, Michel (2017-01-12). Genetics and Evolution of Infectious Diseases (in 英语). Elsevier. ISBN 9780128001530.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":21,"type":"book","title":"Bowler, Peter J. (1992) [1983]. The Eclipse of Darwinism. p. 198.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":22,"type":"book","title":"Smocovitis, Vassiliki Betty (1996). Unifying Biology: The Evolutionary Synthesis and Evolutionary Biology. Princeton, NJ: Princeton University Press. p. 56. ISBN 978-0-691-03343-3. LCCN 96005605. OCLC 34411399.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":23,"type":"book","title":"Hallgrímsson, Benedikt; Hall, Brian K. (2011). Variation: A Central Concept in Biology. Academic Press. p. 18.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":24,"type":"book","title":"Sewall Wright. (1984). Evolution and the Genetics of Populations: Genetics and Biometric Foundations Volume 1. University of Chicago Press. p. 10","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":25,"type":"book","title":"De Vries, Hugo (1905). Species and Varieties: Their Origin by Mutation.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":26,"type":"book","title":"Bateson, William (1894). Materials for the Study of Variation, Treated with Especial Regard to Discontinuity in the Origin of Species.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":27,"type":"book","title":"Punnett, Reginald C. (1915). Mimicry in Butterflies. Cambridge University Press.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":28,"type":"book","title":"Mayr, Ernst (2007). What Makes Biology Unique?: Considerations on the Autonomy of a Scientific Discipline. Cambridge University Press.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":29,"type":"book","title":"Provine, W. B. (2001). The Origins of Theoretical Population Genetics, with a new afterword. University of Chicago Press, Chicago. pp. 56–107.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":30,"type":"book","title":"Stoltzfus, A.; Cable, K. (2014). \"Mendelian-Mutationism: The Forgotten Evolutionary Synthesis\". J Hist Biol. 47 (4): 501–546. doi:10.1007/s10739-014-9383-2. PMID 24811736.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":31,"type":"book","title":"Hull, D. L. (1985). \"Darwinism as an historical entity: A historiographic proposal\". In Kohn, D. The Darwinian Heritage. Princeton University Press. pp. 773–812.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":32,"type":"book","title":"Yule, G. Udny (1902). \"Mendel's Laws and their probable relations to inter-racial heredity\". New Phytologist. 1 (10): 226–227. doi:10.1111/j.1469-8137.1902.tb07336.x.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":33,"type":"book","title":"Gould, S. J. (1982). \"The uses of heresey; an introduction to Richard Goldschmidt's The Material Basis of Evolution.\" pp. xiii-xlii. Yale University Press.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":34,"type":"book","title":"Ruse, Michael (1996). Monad to man: the Concept of Progress in Evolutionary Biology. Harvard University Press. pp. 412–413. ISBN 978-0-674-03248-4.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":35,"type":"book","title":"A. Stoltzfus (2014). \"In search of mutation-driven evolution\". Evolution & Development. 16: 57–59. doi:10.1111/ede.12062.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":36,"type":"book","title":"Futuyma, Douglas J. (2015). Serrelli, E.; Gontier, N., eds. Can Modern Evolutionary Theory Explain Macroevolution? (PDF). Macroevolution. Springer. pp. 29–85.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":37,"type":"book","title":"\"Cancer Is Partly Caused By Bad Luck, Study Finds\". Archived from the original on 13 July 2017.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":38,"type":"book","title":"Jha, Alok (22 August 2012). \"Older fathers pass on more genetic mutations, study shows\". the Guardian.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":39,"type":"book","title":"Montelone BA (1998). \"Mutation, Mutagens, and DNA Repair\". www-personal.ksu.edu. Archived from the original on 26 September 2015. Retrieved 2 October 2015.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":40,"type":"book","title":"Bernstein C, Prasad AR, Nfonsam V, Bernstein H (2013). DNA Damage, DNA Repair and Cancer, New Research Directions in DNA Repair, Prof. Clark Chen (Ed.). intechopen.com. ISBN 978-953-51-1114-6. Archived from the original on 24 September 2015.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":41,"type":"book","title":"Stuart GR, Oda Y, de Boer JG, Glickman BW (March 2000). \"Mutation frequency and specificity with age in liver, bladder and brain of lacI transgenic mice\". Genetics. 154 (3): 1291–300. PMC 1460990. PMID 10757770.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":42,"type":"book","title":"Kunz BA, Ramachandran K, Vonarx EJ (April 1998). \"DNA sequence analysis of spontaneous mutagenesis in Saccharomyces cerevisiae\". Genetics. 148 (4): 1491–505. PMC 1460101. PMID 9560369.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":43,"type":"book","title":"Lieber MR (July 2010). \"The mechanism of double-strand DNA break repair by the nonhomologous DNA end-joining pathway\". Annual Review of Biochemistry. 79: 181–211. doi:10.1146/annurev.biochem.052308.093131. PMC 3079308. PMID 20192759.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":44,"type":"book","title":"Pfohl-Leszkowicz A, Manderville RA (January 2007). \"Ochratoxin A: An overview on toxicity and carcinogenicity in animals and humans\". Molecular Nutrition & Food Research. 51 (1): 61–99. doi:10.1002/mnfr.200600137. PMID 17195275.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":45,"type":"book","title":"Kozmin S, Slezak G, Reynaud-Angelin A, Elie C, de Rycke Y, Boiteux S, Sage E (September 2005). \"UVA radiation is highly mutagenic in cells that are unable to repair 7,8-dihydro-8-oxoguanine in Saccharomyces cerevisiae\". Proceedings of the National Academy of Sciences of the United States of America. 102 (38): 13538–43. Bibcode:2005PNAS..10213538K. doi:10.1073/pnas.0504497102. PMC 1224634. PMID 16157879.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":46,"type":"book","title":"Rahman, Nazneen. \"The clinical impact of DNA sequence changes\". Transforming Genetic Medicine Initiative. Archived from the original on 4 August 2017. Retrieved 27 June 2017.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":47,"type":"book","title":"Freese E (April 1959). \"THE DIFFERENCE BETWEEN SPONTANEOUS AND BASE-ANALOGUE INDUCED MUTATIONS OF PHAGE T4\". Proceedings of the National Academy of Sciences of the United States of America. 45 (4): 622–33. Bibcode:1959PNAS...45..622F. doi:10.1073/pnas.45.4.622. PMC 222607. PMID 16590424.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":48,"type":"book","title":"Freese E (June 1959). \"The specific mutagenic effect of base analogues on Phage T4\". Journal of Molecular Biology. 1 (2): 87–105. doi:10.1016/S0022-2836(59)80038-3.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":49,"type":"book","title":"Steenhuysen, Julie (2017-12-19). \"U.S. Lifts Funding Ban on Studies That Enhance Dangerous Germs\". U.S. News & World Report. Retrieved 2018-01-15.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":50,"type":"book","title":"Goh AM, Coffill CR, Lane DP (January 2011). \"The role of mutant p53 in human cancer\". The Journal of Pathology. 223 (2): 116–26. doi:10.1002/path.2784. PMID 21125670.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":51,"type":"book","title":"Chenevix-Trench G, Spurdle AB, Gatei M, Kelly H, Marsh A, Chen X, Donn K, Cummings M, Nyholt D, Jenkins MA, Scott C, Pupo GM, Dörk T, Bendix R, Kirk J, Tucker K, McCredie MR, Hopper JL, Sambrook J, Mann GJ, Khanna KK (February 2002). \"Dominant negative ATM mutations in breast cancer families\". Journal of the National Cancer Institute. 94 (3): 205–15. CiteSeerX 10.1.1.557.6394. doi:10.1093/jnci/94.3.205. PMID 11830610.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":52,"type":"book","title":"Paz-Priel I, Friedman A (2011). \"C/EBPα dysregulation in AML and ALL\". Critical Reviews in Oncogenesis. 16 (1–2): 93–102. doi:10.1615/critrevoncog.v16.i1-2.90. PMC 3243939. PMID 22150310.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":53,"type":"book","title":"Capaccio D, Ciccodicola A, Sabatino L, Casamassimi A, Pancione M, Fucci A, Febbraro A, Merlino A, Graziano G, Colantuoni V (June 2010). \"A novel germline mutation in peroxisome proliferator-activated receptor gamma gene associated with large intestine polyp formation and dyslipidemia\". Biochimica et Biophysica Acta. 1802 (6): 572–81. doi:10.1016/j.bbadis.2010.01.012. PMID 20123124.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":54,"type":"book","title":"McKusick VA (July 1991). \"The defect in Marfan syndrome\". Nature. 352 (6333): 279–81. Bibcode:1991Natur.352..279M. doi:10.1038/352279a0. PMID 1852198.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":55,"type":"book","title":"Judge DP, Biery NJ, Keene DR, Geubtner J, Myers L, Huso DL, Sakai LY, Dietz HC (July 2004). \"Evidence for a critical contribution of haploinsufficiency in the complex pathogenesis of Marfan syndrome\". The Journal of Clinical Investigation. 114 (2): 172–81. doi:10.1172/JCI20641. PMC 449744. PMID 15254584.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":56,"type":"book","title":"Judge DP, Dietz HC (December 2005). \"Marfan's syndrome\". Lancet. 366 (9501): 1965–76. doi:10.1016/S0140-6736(05)67789-6. PMC 1513064. PMID 16325700.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":57,"type":"book","title":"Ellis NA, Ciocci S, German J (February 2001). \"Back mutation can produce phenotype reversion in Bloom syndrome somatic cells\". Human Genetics. 108 (2): 167–73. doi:10.1007/s004390000447. PMID 11281456.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":58,"type":"book","title":"Charlesworth D, Charlesworth B, Morgan MT (December 1995). \"The pattern of neutral molecular variation under the background selection model\". Genetics. 141 (4): 1619–32. PMC 1206892. PMID 8601499.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":59,"type":"book","title":"Loewe L (April 2006). \"Quantifying the genomic decay paradox due to Muller's ratchet in human mitochondrial DNA\". Genetical Research. 87 (2): 133–59. doi:10.1017/S0016672306008123. PMID 16709275.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":60,"type":"book","title":"Bernstein H, Hopf FA, Michod RE (1987). The molecular basis of the evolution of sex. Advances in Genetics. 24. pp. 323–70. doi:10.1016/s0065-2660(08)60012-7. ISBN 9780120176243. PMID 3324702.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":61,"type":"book","title":"Peck JR, Barreau G, Heath SC (April 1997). \"Imperfect genes, Fisherian mutation and the evolution of sex\". Genetics. 145 (4): 1171–99. PMC 1207886. PMID 9093868.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":62,"type":"book","title":"Keightley PD, Lynch M (March 2003). \"Toward a realistic model of mutations affecting fitness\". Evolution; International Journal of Organic Evolution. 57 (3): 683–5, discussion 686–9. doi:10.1554/0014-3820(2003)057[0683:tarmom]2.0.co;2. JSTOR 3094781. PMID 12703958.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":63,"type":"book","title":"Barton NH, Keightley PD (January 2002). \"Understanding quantitative genetic variation\". Nature Reviews Genetics. 3 (1): 11–21. doi:10.1038/nrg700. PMID 11823787.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":64,"type":"book","title":"Sanjuán R, Moya A, Elena SF (June 2004). \"The distribution of fitness effects caused by single-nucleotide substitutions in an RNA virus\". Proceedings of the National Academy of Sciences of the United States of America. 101 (22): 8396–401. Bibcode:2004PNAS..101.8396S. doi:10.1073/pnas.0400146101. PMC 420405. PMID 15159545.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":65,"type":"book","title":"Carrasco P, de la Iglesia F, Elena SF (December 2007). \"Distribution of fitness and virulence effects caused by single-nucleotide substitutions in Tobacco Etch virus\". Journal of Virology. 81 (23): 12979–84. doi:10.1128/JVI.00524-07. PMC 2169111. PMID 17898073.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":66,"type":"book","title":"Sanjuán R (June 2010). \"Mutational fitness effects in RNA and single-stranded DNA viruses: common patterns revealed by site-directed mutagenesis studies\". Philosophical Transactions of the Royal Society of London. Series B, Biological Sciences. 365 (1548): 1975–82. doi:10.1098/rstb.2010.0063. PMC 2880115. PMID 20478892.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":67,"type":"book","title":"Peris JB, Davis P, Cuevas JM, Nebot MR, Sanjuán R (June 2010). \"Distribution of fitness effects caused by single-nucleotide substitutions in bacteriophage f1\". Genetics. 185 (2): 603–9. doi:10.1534/genetics.110.115162. PMC 2881140. PMID 20382832.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":68,"type":"book","title":"Elena SF, Ekunwe L, Hajela N, Oden SA, Lenski RE (March 1998). \"Distribution of fitness effects caused by random insertion mutations in Escherichia coli\". Genetica. 102–103 (1–6): 349–58. doi:10.1023/A:1017031008316. PMID 9720287.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":69,"type":"book","title":"Hietpas RT, Jensen JD, Bolon DN (May 2011). \"Experimental illumination of a fitness landscape\". Proceedings of the National Academy of Sciences of the United States of America. 108 (19): 7896–901. Bibcode:2011PNAS..108.7896H. doi:10.1073/pnas.1016024108. PMC 3093508. PMID 21464309.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":70,"type":"book","title":"Davies EK, Peters AD, Keightley PD (September 1999). \"High frequency of cryptic deleterious mutations in Caenorhabditis elegans\". Science. 285 (5434): 1748–51. doi:10.1126/science.285.5434.1748. PMID 10481013.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":71,"type":"book","title":"Loewe L, Charlesworth B (September 2006). \"Inferring the distribution of mutational effects on fitness in Drosophila\". Biology Letters. 2 (3): 426–30. doi:10.1098/rsbl.2006.0481. PMC 1686194. PMID 17148422.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":72,"type":"book","title":"Eyre-Walker A, Woolfit M, Phelps T (June 2006). \"The distribution of fitness effects of new deleterious amino acid mutations in humans\". Genetics. 173 (2): 891–900. doi:10.1534/genetics.106.057570. PMC 1526495. PMID 16547091.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":73,"type":"book","title":"Sawyer SA, Kulathinal RJ, Bustamante CD, Hartl DL (August 2003). \"Bayesian analysis suggests that most amino acid replacements in Drosophila are driven by positive selection\". Journal of Molecular Evolution. 57 Suppl 1 (1): S154–64. Bibcode:2003JMolE..57S.154S. CiteSeerX 10.1.1.78.65. doi:10.1007/s00239-003-0022-3. PMID 15008412.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":74,"type":"book","title":"Piganeau G, Eyre-Walker A (September 2003). \"Estimating the distribution of fitness effects from DNA sequence data: implications for the molecular clock\". Proceedings of the National Academy of Sciences of the United States of America. 100 (18): 10335–40. Bibcode:2003PNAS..10010335P. doi:10.1073/pnas.1833064100. PMC 193562. PMID 12925735.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":75,"type":"book","title":"Eyre-Walker A, Keightley PD (August 2007). \"The distribution of fitness effects of new mutations\" (PDF). Nature Reviews Genetics. 8 (8): 610–8. doi:10.1038/nrg2146. PMID 17637733. Archived (PDF) from the original on 4 March 2016.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":76,"type":"book","title":"Kimura M (1983). The Neutral Theory of Molecular Evolution. Cambridge, UK; New York: Cambridge University Press. ISBN 978-0-521-23109-1. LCCN 82022225. OCLC 9081989.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":77,"type":"book","title":"Eyre-Walker A (October 2006). \"The genomic rate of adaptive evolution\". Trends in Ecology & Evolution. 21 (10): 569–75. doi:10.1016/j.tree.2006.06.015. PMID 16820244.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":78,"type":"book","title":"Orr HA (April 2003). \"The distribution of fitness effects among beneficial mutations\". Genetics. 163 (4): 1519–26. PMC 1462510. PMID 12702694.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":79,"type":"book","title":"Gillespie JH (September 1984). \"Molecular Evolution Over the Mutational Landscape\". Evolution. 38 (5): 1116–1129. doi:10.2307/2408444. JSTOR 2408444. PMID 28555784.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":80,"type":"book","title":"Kassen R, Bataillon T (April 2006). \"Distribution of fitness effects among beneficial mutations before selection in experimental populations of bacteria\". Nature Genetics. 38 (4): 484–8. doi:10.1038/ng1751. PMID 16550173.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":81,"type":"book","title":"Rokyta DR, Joyce P, Caudle SB, Wichman HA (April 2005). \"An empirical test of the mutational landscape model of adaptation using a single-stranded DNA virus\". Nature Genetics. 37 (4): 441–4. doi:10.1038/ng1535. PMID 15778707.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":82,"type":"book","title":"Imhof M, Schlotterer C (January 2001). \"Fitness effects of advantageous mutations in evolving Escherichia coli populations\". Proceedings of the National Academy of Sciences of the United States of America. 98 (3): 1113–7. Bibcode:2001PNAS...98.1113I. doi:10.1073/pnas.98.3.1113. PMC 14717. PMID 11158603.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":83,"type":"book","title":"Hogan, C. Michael (12 October 2010). \"Mutation\". In Monosson, Emily. Encyclopedia of Earth. Washington, D.C.: Environmental Information Coalition, National Council for Science and the Environment. OCLC 72808636. Archived from the original on 14 November 2015. Retrieved 8 October 2015.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":84,"type":"book","title":"Boillée S, Vande Velde C, Cleveland DW (October 2006). \"ALS: a disease of motor neurons and their nonneuronal neighbors\". Neuron. 52 (1): 39–59. CiteSeerX 10.1.1.325.7514. doi:10.1016/j.neuron.2006.09.018. PMID 17015226.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":85,"type":"book","title":"\"RB1 Genetics\". Daisy's Eye Cancer Fund. Oxford, UK. Archived from the original on 2011-11-26. Retrieved 2015-10-09.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":86,"type":"book","title":"\"Compound heterozygote\". MedTerms. New York: WebMD. 14 June 2012. Archived from the original on 4 March 2016. Retrieved 9 October 2015.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":87,"type":"book","title":"Alberts (2014). Molecular Biology of the Cell (6 ed.). Garland Science. p. 487. ISBN 9780815344322.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":88,"type":"book","title":"Chadov BF, Fedorova NB, Chadova EV (2015-07-01). \"Conditional mutations in Drosophila melanogaster: On the occasion of the 150th anniversary of G. Mendel's report in Brünn\". Mutation Research/Reviews in Mutation Research. 765: 40–55. doi:10.1016/j.mrrev.2015.06.001. PMID 26281767.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":89,"type":"book","title":"Landis G, Bhole D, Lu L, Tower J (July 2001). \"High-frequency generation of conditional mutations affecting Drosophila melanogaster development and life span\". Genetics. 158 (3): 1167–76. PMC 1461716. PMID 11454765. Archived from the original on 22 March 2017.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":90,"type":"book","title":"Gierut JJ, Jacks TE, Haigis KM (April 2014). \"Strategies to achieve conditional gene mutation in mice\". Cold Spring Harbor Protocols. 2014 (4): 339–49. doi:10.1101/pdb.top069807. PMC 4142476. PMID 24692485.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":91,"type":"book","title":"Spencer DM (May 1996). \"Creating conditional mutations in mammals\". Trends in Genetics. 12 (5): 181–7. doi:10.1016/0168-9525(96)10013-5. PMID 8984733.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":92,"type":"book","title":"Tan G, Chen M, Foote C, Tan C (September 2009). \"Temperature-sensitive mutations made easy: generating conditional mutations by using temperature-sensitive inteins that function within different temperature ranges\". Genetics. 183 (1): 13–22. doi:10.1534/genetics.109.104794. PMC 2746138. PMID 19596904.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":93,"type":"book","title":"den Dunnen JT, Antonarakis SE (January 2000). \"Mutation nomenclature extensions and suggestions to describe complex mutations: a discussion\". Human Mutation. 15 (1): 7–12. doi:10.1002/(SICI)1098-1004(200001)15:1<7::AID-HUMU4>3.0.CO;2-N. PMID 10612815.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":94,"type":"book","title":"Sawyer SA, Parsch J, Zhang Z, Hartl DL (April 2007). \"Prevalence of positive selection among nearly neutral amino acid replacements in Drosophila\". Proceedings of the National Academy of Sciences of the United States of America. 104 (16): 6504–10. Bibcode:2007PNAS..104.6504S. doi:10.1073/pnas.0701572104. PMC 1871816. PMID 17409186.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":95,"type":"book","title":"Ionov Y, Peinado MA, Malkhosyan S, Shibata D, Perucho M (June 1993). \"Ubiquitous somatic mutations in simple repeated sequences reveal a new mechanism for colonic carcinogenesis\". Nature. 363 (6429): 558–61. Bibcode:1993Natur.363..558I. doi:10.1038/363558a0. PMID 8505985.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":96,"type":"book","title":"Sullivan AD, Wigginton J, Kirschner D (August 2001). \"The coreceptor mutation CCR5Delta32 influences the dynamics of HIV epidemics and is selected for by HIV\". Proceedings of the National Academy of Sciences of the United States of America. 98 (18): 10214–9. Bibcode:2001PNAS...9810214S. doi:10.1073/pnas.181325198. PMC 56941. PMID 11517319.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":97,"type":"book","title":"\"Mystery of the Black Death\". Secrets of the Dead. Season 3. Episode 2. 30 October 2002. PBS. Archived from the original on 12 October 2015. Retrieved 10 October 2015. Episode background.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":98,"type":"book","title":"Galvani AP, Slatkin M (December 2003). \"Evaluating plague and smallpox as historical selective pressures for the CCR5-Delta 32 HIV-resistance allele\". Proceedings of the National Academy of Sciences of the United States of America. 100 (25): 15276–9. Bibcode:2003PNAS..10015276G. doi:10.1073/pnas.2435085100. PMC 299980. PMID 14645720.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":99,"type":"book","title":"Konotey-Ahulu, Felix. \"Frequently Asked Questions [FAQ's]\". sicklecell.md. Archived from the original on 30 April 2011.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":100,"type":"book","title":"Hughes, Diarmaid; Andersson, Dan I. (09 08, 2017). \"Evolutionary Trajectories to Antibiotic Resistance\". Annual Review of Microbiology. 71: 579–596. doi:10.1146/annurev-micro-090816-093813. ISSN 1545-3251. PMID 28697667. Check date values in: |date= (help)","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":101,"type":"book","title":"Ségurel, Laure; Bon, Céline (08 31, 2017). \"On the Evolution of Lactase Persistence in Humans\". Annual Review of Genomics and Human Genetics. 18: 297–319. doi:10.1146/annurev-genom-091416-035340. ISSN 1545-293X. PMID 28426286. Check date values in: |date= (help)","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":102,"type":"book","title":"\"'Lifeless' prion proteins are 'capable of evolution'\". Health. BBC News Online. London. 1 January 2010. Archived from the original on 25 September 2015. Retrieved 10 October 2015.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":103,"type":"book","title":"\"Somatic cell genetic mutation\". Genome Dictionary. Athens, Greece: Information Technology Associates. 30 June 2007. Archived from the original on 24 February 2010. Retrieved 6 June 2010.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":104,"type":"book","title":"Hartl, Jones, Daniel L., Elizabeth W. (1998). Genetics Principles and Analysis. Sudbury, Massachusetts: Jones and Bartlett Publishers. p. 556. ISBN 978-0-7637-0489-6.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":105,"type":"book","title":"Milholland B, Dong X, Zhang L, Hao X, Suh Y, Vijg J (2017). \"Differences between germline and somatic mutation rates in humans and mice\". Nat Commun. 8: 15183. Bibcode:2017NatCo...815183M. doi:10.1038/ncomms15183. PMC 5436103. PMID 28485371.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":106,"type":"book","title":"Araten DJ, Golde DW, Zhang RH, Thaler HT, Gargiulo L, Notaro R, Luzzatto L (September 2005). \"A quantitative measurement of the human somatic mutation rate\". Cancer Research. 65 (18): 8111–7. doi:10.1158/0008-5472.CAN-04-1198. PMID 16166284.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false}],"recommendReferences":null,"auditState":2,"lemmaLevel":1,"origin":0,"originEnTitle":null,"originZhTitle":null,"pv":28058,"auditType":2,"synonyms":["Mutation","失去功能的突变"],"showEditTime":"2020.09.10 16:00","auditors":[{"uid":5567290,"name":"人间小太阳","pic":"https://web.archive.org/web/20221025113755/https://cache.soso.com/qlogo/g?b=oidb&k=4xjGfuUZbbjZEI7OAVIVEg&s=100&t=1609215301","introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":"","jobBrief":"","role":0,"roleName":null,"title":"","professionalTitle":null,"phoneNo":null,"editable":true,"partnerId":250,"partnerIdCreateTime":1598322359,"partnerIdPoped":true}],"hasZhishiNav":false,"auditInfos":{"0":[{"versionId":"0","auditTime":1599729851,"auditorId":5567290,"auditUser":{"uid":5567290,"name":"人间小太阳","pic":"https://web.archive.org/web/20221025113755/https://cache.soso.com/qlogo/g?b=oidb&k=4xjGfuUZbbjZEI7OAVIVEg&s=100&t=1609215301","introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":"","jobBrief":"","role":0,"roleName":null,"title":"","professionalTitle":null,"phoneNo":null,"editable":true,"partnerId":250,"partnerIdCreateTime":1598322359,"partnerIdPoped":true},"auditReason":"","auditSuggest":"简单修改","auditResult":2,"auditAnnotations":null}]},"isHistory":false};</script><script crossorigin="anonymous" src="./402.基因突变 - 搜狗科学百科_files/aegis.min.js.download"></script><script crossorigin="anonymous" src="./402.基因突变 - 搜狗科学百科_files/main_2020092401.js.download"></script><script crossorigin="anonymous" src="./402.基因突变 - 搜狗科学百科_files/react.production.min.js.download"></script><script crossorigin="anonymous" src="./402.基因突变 - 搜狗科学百科_files/react-dom.production.min.js.download"></script><script crossorigin="anonymous" src="./402.基因突变 - 搜狗科学百科_files/jquery-1.11.1.min.js.download"></script><script crossorigin="anonymous" src="./402.基因突变 - 搜狗科学百科_files/main_2022062701.js.download"></script><script crossorigin="anonymous" src="./402.基因突变 - 搜狗科学百科_files/main_66bbe21.js.download"></script><script crossorigin="anonymous" src="./402.基因突变 - 搜狗科学百科_files/react.production.min.js.download"></script><script crossorigin="anonymous" src="./402.基因突变 - 搜狗科学百科_files/react-dom.production.min.js.download"></script><script crossorigin="anonymous" src="./402.基因突变 - 搜狗科学百科_files/main_edf0f08.js.download"></script><div id="popControl"><div class="pop-control-mask" style="display: none;"><div class="pop-container"><i class="close-icon"></i><div class="stop-title">搜狗科学百科停止服务运营公告</div><div class="stop-user">亲爱的用户：</div><div class="stop-content">因业务方向调整，搜狗科学百科将于2022年11月11日正式停止服务与运营。届时产品内所有数据将依据相关法律进行删除并关闭服务器，您将无法登录及使用搜狗科学百科，相关内容在平台下线后将无法找回。</div><div class="stop-content">搜狗科学百科是由搜狗推出的科学领域专业百科产品，与搜狗百科为两个独立产品，本次停止服务与运营仅对搜狗科学百科，并不影响搜狗百科产品正常使用，敬请知悉。</div><div class="stop-content">由此给您带来的不便我们深表歉意，敬请谅解。再次感谢您一直以来的陪伴与支持！</div><div class="stop-team">搜狗科学百科产品&amp;运营团队</div><div class="stop-time">2022年10月11日</div></div></div></div>
</body></html>