<!DOCTYPE html>
<!-- saved from url=(0083)https://web.archive.org/web/20221025084210/https://baike.sogou.com/kexue/d10532.htm -->
<html class="" data-reactroot=""><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><script src="./532.基因敲除 - 搜狗科学百科_files/analytics.js.download" type="text/javascript"></script>
<script type="text/javascript">window.addEventListener('DOMContentLoaded',function(){var v=archive_analytics.values;v.service='wb';v.server_name='wwwb-app210.us.archive.org';v.server_ms=305;archive_analytics.send_pageview({});});</script>
<script type="text/javascript" src="./532.基因敲除 - 搜狗科学百科_files/bundle-playback.js.download" charset="utf-8"></script>
<script type="text/javascript" src="./532.基因敲除 - 搜狗科学百科_files/wombat.js.download" charset="utf-8"></script>
<script type="text/javascript">
  __wm.init("https://web.archive.org/web");
  __wm.wombat("https://baike.sogou.com/kexue/d10532.htm","20221025084210","https://web.archive.org/","web","/_static/",
	      "1666687330");
</script>
<link rel="stylesheet" type="text/css" href="./532.基因敲除 - 搜狗科学百科_files/banner-styles.css">
<link rel="stylesheet" type="text/css" href="./532.基因敲除 - 搜狗科学百科_files/iconochive.css">
<!-- End Wayback Rewrite JS Include -->
<meta name="save" content="history"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="baidu-site-verification" content="VWGb6TyYx8"><meta content="基因敲除 - 搜狗科学百科" name="keywords"><meta content="搜狗科学百科是一部有着平等、协作、分享、自由理念的网络科学全书，为每一个互联网用户创造一个涵盖所有领域知识、服务的中文知识性平台。" name="description"><meta http-equiv="x-dns-prefetch-control" content="on"><meta name="server" baike="235" ip="210" env="online"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025084210/https://cache.soso.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025084210/https://hhy.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025084210/https://pic.baike.soso.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025084210/https://ugc.qpic.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025084210/https://xui.ptlogin2.qq.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025084210/https://q1.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025084210/https://q2.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025084210/https://q3.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025084210/https://q4.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025084210/https://q.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025084210/https://img01.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025084210/https://img02.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025084210/https://img03.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025084210/https://img04.sogoucdn.com/"><link rel="Shortcut Icon" href="https://web.archive.org/web/20221025084210im_/https://www.sogou.com/images/logo/new/favicon.ico?v=4"><link rel="Bookmark" href="https://www.sogou.com/images/logo/new/favicon.ico?v=4"><link href="./532.基因敲除 - 搜狗科学百科_files/base_b849887.css" rel="stylesheet"><link href="./532.基因敲除 - 搜狗科学百科_files/detail_378aed5.css" rel="stylesheet"><link href="./532.基因敲除 - 搜狗科学百科_files/inviteAudit_7894507.css" rel="stylesheet"><link rel="stylesheet" href="./532.基因敲除 - 搜狗科学百科_files/highlight.min.css"><title>基因敲除 - 搜狗科学百科</title><style>.onekey-close {
	position: absolute;
	top: 16px;
	right: 16px;
	width: 24px;
	height: 24px;
	background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	text-indent: -999em;
	background-size: 84px;
	background-position: -63px 0;
}

.onekey-login {
	position: absolute;
	top: 16.4%;
	left: 0;
	right: 0;
	width: 100%;
}

/* .onekey-login-img {
    width: 75px;
    height: 75px;
    background: url("https://web.archive.org/web/20221025084210/https://baike.sogou.com/kexue/images/sprite_wap_baike.png") no-repeat;
    background-size: 100px 91px;
    background-position: 0 0;
    background-repeat: no-repeat;
    margin: 0 auto;
} */

.onekey-login-title {
	text-align: center;
	padding-bottom: 3px;
	font-size: 21px;
	font-weight: bold;
	line-height: 30px;
	color: #000;
}

.onekey-login-txt {
	text-align: center;
	font-family: PingFangSC;
	font-size: 14px;
	line-height: 20px;
	color: #8f8f8f;
}

.onekey-login-qq,
.onekey-login-wx,
.onekey-login-phone {
	display: block;
	width: 245px;
	height: 54px;
	border-radius: 45px;
	text-align: center;

	margin: 0 auto;
	font-size: 17px;
	line-height: 24px;
	color: #000;
	/* padding: 16px 77px; */
	border-radius: 12px;
	border: solid 1px #e0e0e0;
}
.onekey-qq-content,
.onekey-vx-content,
.onekey-phone-content {
	display: inline-block;
	margin-top: 16px;
}
.onekey-qq-content {
	padding: 0 5px;
}

.onekey-login-qq {
	margin-top: 48px;
	margin-bottom: 24px;
}

.onekey-login-qq:before {
	display: inline-block;
	content: "";
	width: 20px;
	height: 20px;
	background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	background-size: 80px;
	background-position: -20px 0;
	vertical-align: top;
	margin: 17px 8px 0 0;
}

.onekey-login-wx {
	margin-bottom: 24px;
}

.onekey-login-wx:before {
	display: inline-block;
	content: "";
	width: 21px;
	height: 21px;
	background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	background-size: 84px;
	background-position: 0 0;
	vertical-align: top;
	margin: 17px 10px 0 0;
}

.onekey-login-phone {
}

.onekey-login-phone:before {
	display: inline-block;
	content: "";
	width: 21px;
	height: 21px;
	background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	background-size: 84px;
	background-position: -42px 0;
	vertical-align: top;
	margin: 17px 10px 0 0;
}

.onekey-fixed {
	z-index: 100;
	position: fixed;
	top: 0;
	bottom: 0;
	left: 0;
	right: 0;
	background: #fff;
	width: 100%;
	height: 100%;
}

.onekey-fixed.forbid {
	z-index: 100;
	position: fixed;
	top: auto;
	bottom: 68px;
	left: 9%;
	right: 0;
	background: rgba(0, 0, 0, 0.7);
	width: 82%;
	height: 43px;
	border-radius: 25px;
	color: #ffffff;
}
.onekey-login-title.forbid {
	text-align: center;
	padding-bottom: 3px;
	font-size: 14px;
	font-weight: normal;
	line-height: 30px;
	color: white;
}
</style><style>#login_mask {
  background: #000;
  opacity: 0.5;
  filter: alpha(opacity=50);
  position: fixed;
  /*fixed好像在哪个IE上有BUG，先用用*/
  left: 0;
  top: 0;
  z-index: 999;
  height: 100%;
}

#login_iframe_container {
  position: fixed;
  width: 550px;
  height: 360px;
  z-index: 1020;
  background-color: #ffffff;
}

@media screen and (max-width: 828px) {
  #login_iframe_container {
    top: 50% !important;
    left: 50% !important;
    transform: translate(-50%, -50%);
  }
}

#login_iframe_container.new-login {
  width: 550px;
  height: 360px;
  background-image: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/background_2a4a8a6.png);
}

#login_iframe_container.new-login.no-bg {
  background: #fff;
}

#login_iframe_container.new-login .login-title {
  width: 100%;
  height: 42px;
  line-height: 42px;
  text-align: center;
  font-size: 30px;
  letter-spacing: 0.19px;
  color: #ffffff;
  margin-top: 62px;
}
#login_iframe_container.new-login .forbid-title {
  width: 100%;
  height: 42px;
  line-height: 42px;
  text-align: center;
  font-size: 24px;
  letter-spacing: 0.19px;
  color: #333333;
  margin-top: 150px;
}

#login_iframe_container.new-login.no-bg .login-title {
  color: #333333;
}

#login_iframe_container.new-login .login-subtitle {
  width: 100%;
  height: 18px;
  line-height: 18px;
  font-size: 13px;
  letter-spacing: 0.08px;
  color: #ffffff;
  text-align: center;
  margin-top: 9px;
  margin-bottom: 43px;
}

#login_iframe_container.new-login.no-bg .login-subtitle {
  color: #999999;
}

#login_iframe_container.new-login .login-subtitle::before {
  content: '';
  display: inline-block;
  width: 10px;
  height: 1px;
  background-color: #ffffff;
  position: relative;
  top: -4px;
  left: -5px;
}

#login_iframe_container.new-login .login-subtitle::after {
  content: '';
  display: inline-block;
  width: 10px;
  height: 1px;
  background-color: #ffffff;
  position: relative;
  top: -4px;
  left: 5px;
}

#login_iframe_container.new-login.no-bg .login-subtitle::before {
  background-color: #999999;
}

#login_iframe_container.new-login.no-bg .login-subtitle::after {
  background-color: #999999;
}

#login_iframe_container.new-login .close-btn {
  position: absolute;
  top: 20px;
  right: 20px;
  width: 12px;
  height: 12px;
  background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/login-sprites_e3853e5.png) -59px -10px;
  background-size: 81px 91px;
  cursor: pointer;
}

#login_iframe_container.new-login .login-btn {
  width: 220px;
  height: 47px;
  border-radius: 24px;
  border: solid 1px #dddddd;
  background-color: #ffffff;
  margin: 0 auto;
  margin-top: 28px;
  position: relative;
  display: block;
}

#login_iframe_container.new-login .login-btn .login-icon {
  position: absolute;
}

#login_iframe_container.new-login .login-btn .login-text {
  width: 61px;
  height: 47px;
  line-height: 47px;
  vertical-align: middle;
  font-size: 15px;
  letter-spacing: 0.1px;
  color: #666666;
  position: absolute;
  right: 62px;
}

#login_iframe_container.new-login .login-btn.qq-btn .login-icon {
  width: 22px;
  height: 27px;
  top: 10px;
  left: 67px;
  background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/login-sprites_e3853e5.png) -10px -54px;
  background-size: 81px 91px;
}

#login_iframe_container.new-login .login-btn.qq-btn .login-text {
  right: 59px;
}

#login_iframe_container.new-login .login-btn.wechat-btn .login-icon {
  width: 29px;
  height: 24px;
  top: 12px;
  left: 62px;
  background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/login-sprites_e3853e5.png) -10px -10px;
  background-size: 81px 91px;
}</style><style>/* -- container -- */
.rodal,
.rodal-mask {
    top: 0;
    left: 0;
    width: 100%;
    height: 100%;
    z-index: 100;
}

.rodal {
    position: fixed;
}

/* -- mask -- */
.rodal-mask {
    position: fixed;
    background: rgba(0, 0, 0, .5);
}

/* -- dialog -- */
.rodal-dialog {
    position: absolute;
    z-index: 101;
    background: #fff;
    border-radius: 3px;
    box-shadow: 0 1px 3px rgba(0, 0, 0, .2);
}

.rodal-center {
    top: 50%;
    transform: translateY(-50%);
    left: 0;
    right: 0;
    margin: 0 auto;
}

.rodal-bottom {
    left: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

.rodal-top {
    left: 0;
    right: 0;
    top: 0;
    margin: auto;
}

.rodal-left {
    top: 0;
    left: 0;
    bottom: 0;
    margin: auto;
}

.rodal-right {
    top: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

/* -- close button -- */
.rodal-close {
    position: absolute;
    cursor: pointer;
    top: 16px;
    right: 16px;
    width: 16px;
    height: 16px;
}

.rodal-close:before,
.rodal-close:after {
    position: absolute;
    content: '';
    height: 2px;
    width: 100%;
    top: 50%;
    left: 0;
    margin-top: -1px;
    background: #999;
    border-radius: 100%;
    -webkit-transition: background .2s;
    transition: background .2s;
}

.rodal-close:before {
    -webkit-transform: rotate(45deg);
    transform: rotate(45deg);
}

.rodal-close:after {
    -webkit-transform: rotate(-45deg);
    transform: rotate(-45deg);
}

.rodal-close:hover:before,
.rodal-close:hover:after {
    background: #333;
}

/* -- fade -- */
/* @-webkit-keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

@keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

.rodal-fade-enter {
    -webkit-animation: rodal-fade-enter both ease-in;
    animation: rodal-fade-enter both ease-in;
} */

@-webkit-keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

@keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

.rodal-fade-leave {
    -webkit-animation: rodal-fade-leave both ease-out;
    animation: rodal-fade-leave both ease-out;
}

/* -- zoom -- */
@-webkit-keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-enter {
    -webkit-animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-leave {
    -webkit-animation: rodal-zoom-leave both;
    animation: rodal-zoom-leave both;
}

/* -- slideDown -- */
@-webkit-keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-enter {
    -webkit-animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-leave {
    -webkit-animation: rodal-slideDown-leave both;
    animation: rodal-slideDown-leave both;
}

/* -- slideLeft -- */
@-webkit-keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-enter {
    -webkit-animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-leave {
    -webkit-animation: rodal-slideLeft-leave both;
    animation: rodal-slideLeft-leave both;
}

/* -- slideRight -- */
@-webkit-keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-enter {
    -webkit-animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-leave {
    -webkit-animation: rodal-slideRight-leave both;
    animation: rodal-slideRight-leave both;
}

/* -- slideUp -- */
@-webkit-keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-enter {
    -webkit-animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
    animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
}

@-webkit-keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-leave {
    -webkit-animation: rodal-slideUp-leave both;
    animation: rodal-slideUp-leave both;
}

/* -- flip -- */
@-webkit-keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

@keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

.rodal-flip-enter {
    -webkit-animation: rodal-flip-enter both ease-in;
    animation: rodal-flip-enter both ease-in;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

@-webkit-keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

@keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

.rodal-flip-leave {
    -webkit-animation: rodal-flip-leave both;
    animation: rodal-flip-leave both;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

/* -- rotate -- */
@-webkit-keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-enter {
    -webkit-animation: rodal-rotate-enter both;
    animation: rodal-rotate-enter both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

@-webkit-keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-leave {
    -webkit-animation: rodal-rotate-leave both;
    animation: rodal-rotate-leave both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

/* -- door -- */
@-webkit-keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

@keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

.rodal-door-enter {
    -webkit-animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

@keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

.rodal-door-leave {
    -webkit-animation: rodal-door-leave both;
    animation: rodal-door-leave both;
}</style><style>/* -- container -- */
.rodal,
.rodal-mask {
    top: 0;
    left: 0;
    width: 100%;
    height: 100%;
    z-index: 100;
}

.rodal {
    position: fixed;
}

/* -- mask -- */
.rodal-mask {
    position: fixed;
    background: rgba(0, 0, 0, .5);
}

/* -- dialog -- */
.rodal-dialog {
    position: absolute;
    z-index: 101;
    background: #fff;
    border-radius: 3px;
    box-shadow: 0 1px 3px rgba(0, 0, 0, .2);
}

.rodal-center {
    top: 50%;
    transform: translateY(-50%);
    left: 0;
    right: 0;
    margin: 0 auto;
}

.rodal-bottom {
    left: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

.rodal-top {
    left: 0;
    right: 0;
    top: 0;
    margin: auto;
}

.rodal-left {
    top: 0;
    left: 0;
    bottom: 0;
    margin: auto;
}

.rodal-right {
    top: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

/* -- close button -- */
.rodal-close {
    position: absolute;
    cursor: pointer;
    top: 16px;
    right: 16px;
    width: 16px;
    height: 16px;
}

.rodal-close:before,
.rodal-close:after {
    position: absolute;
    content: '';
    height: 2px;
    width: 100%;
    top: 50%;
    left: 0;
    margin-top: -1px;
    background: #999;
    border-radius: 100%;
    -webkit-transition: background .2s;
    transition: background .2s;
}

.rodal-close:before {
    -webkit-transform: rotate(45deg);
    transform: rotate(45deg);
}

.rodal-close:after {
    -webkit-transform: rotate(-45deg);
    transform: rotate(-45deg);
}

.rodal-close:hover:before,
.rodal-close:hover:after {
    background: #333;
}

/* -- fade -- */
/* @-webkit-keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

@keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

.rodal-fade-enter {
    -webkit-animation: rodal-fade-enter both ease-in;
    animation: rodal-fade-enter both ease-in;
} */

@-webkit-keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

@keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

.rodal-fade-leave {
    -webkit-animation: rodal-fade-leave both ease-out;
    animation: rodal-fade-leave both ease-out;
}

/* -- zoom -- */
@-webkit-keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-enter {
    -webkit-animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-leave {
    -webkit-animation: rodal-zoom-leave both;
    animation: rodal-zoom-leave both;
}

/* -- slideDown -- */
@-webkit-keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-enter {
    -webkit-animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-leave {
    -webkit-animation: rodal-slideDown-leave both;
    animation: rodal-slideDown-leave both;
}

/* -- slideLeft -- */
@-webkit-keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-enter {
    -webkit-animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-leave {
    -webkit-animation: rodal-slideLeft-leave both;
    animation: rodal-slideLeft-leave both;
}

/* -- slideRight -- */
@-webkit-keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-enter {
    -webkit-animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-leave {
    -webkit-animation: rodal-slideRight-leave both;
    animation: rodal-slideRight-leave both;
}

/* -- slideUp -- */
@-webkit-keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-enter {
    -webkit-animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
    animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
}

@-webkit-keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-leave {
    -webkit-animation: rodal-slideUp-leave both;
    animation: rodal-slideUp-leave both;
}

/* -- flip -- */
@-webkit-keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

@keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

.rodal-flip-enter {
    -webkit-animation: rodal-flip-enter both ease-in;
    animation: rodal-flip-enter both ease-in;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

@-webkit-keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

@keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

.rodal-flip-leave {
    -webkit-animation: rodal-flip-leave both;
    animation: rodal-flip-leave both;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

/* -- rotate -- */
@-webkit-keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-enter {
    -webkit-animation: rodal-rotate-enter both;
    animation: rodal-rotate-enter both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

@-webkit-keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-leave {
    -webkit-animation: rodal-rotate-leave both;
    animation: rodal-rotate-leave both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

/* -- door -- */
@-webkit-keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

@keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

.rodal-door-enter {
    -webkit-animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

@keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

.rodal-door-leave {
    -webkit-animation: rodal-door-leave both;
    animation: rodal-door-leave both;
}</style></head><body class=""><!-- BEGIN WAYBACK TOOLBAR INSERT -->
<style type="text/css">
body {
  margin-top:0 !important;
  padding-top:0 !important;
  /*min-width:800px !important;*/
}
</style>
<script>__wm.rw(0);</script>
<div id="wm-ipp-base" lang="en" style="display: block; direction: ltr;">
</div><div id="wm-ipp-print">The Wayback Machine - https://web.archive.org/web/20221025084210/https://baike.sogou.com/kexue/d10532.htm</div>
<script type="text/javascript">//<![CDATA[
__wm.bt(675,27,25,2,"web","https://baike.sogou.com/kexue/d10532.htm","20221025084210",1996,"/_static/",["/_static/css/banner-styles.css?v=S1zqJCYt","/_static/css/iconochive.css?v=qtvMKcIJ"], false);
  __wm.rw(1);
//]]></script>
<!-- END WAYBACK TOOLBAR INSERT --><script>window._gtag=window._gtag||{};window._gtag.shouldGrayed = false;if ('36785d8e03464d94bf3f1cd37964525a') window._gtag.traceId = '36785d8e03464d94bf3f1cd37964525a';if ({"illegality":true}) window.userInfo = {"illegality":true};</script><div class="topnavbox"><ul class="topnav"><li><a href="https://web.archive.org/web/20221025084210/https://www.sogou.com/web?query=">网页</a></li><li><a href="https://web.archive.org/web/20221025084210/https://weixin.sogou.com/weixin?p=75351201">微信</a></li><li><a href="https://web.archive.org/web/20221025084210/https://zhihu.sogou.com/zhihu?p=75351218">知乎</a></li><li><a href="https://web.archive.org/web/20221025084210/https://pic.sogou.com/pics?query=">图片</a></li><li><a href="https://web.archive.org/web/20221025084210/https://v.sogou.com/v?query=">视频</a></li><li><a href="https://web.archive.org/web/20221025084210/https://mingyi.sogou.com/">医疗</a></li><li class="cur"><strong>科学</strong></li><li><a href="https://web.archive.org/web/20221025084210/https://hanyu.sogou.com/">汉语</a></li><li><a href="https://web.archive.org/web/20221025084210/https://wenwen.sogou.com/">问问</a></li><li><a href="https://web.archive.org/web/20221025084210/https://www.sogou.com/docs/more.htm">更多<span class="topraquo">»</span></a></li></ul></div><div id="header"><div class="header-wrap"><a class="header-logo" href="https://web.archive.org/web/20221025084210/https://baike.sogou.com/kexue"></a><div class="header-search"><div class="querybox" id="suggBox"><form><input id="searchInput" class="query" type="text" placeholder="搜科学领域专业百科词条" name="query" autocomplete="off" value=""><a href="javascript:;" class="query-search"></a></form></div></div><div class="header-rgt"><span class="btn-header-rgt btn-edit" id="editLemma">创建</span><div class="header-user no-login"></div></div></div></div><div class="fixed-placeholder" style="visibility:none"></div><div id="container" class=""><div class="content lemma-level1"><div class="detail-title" id="abstract-title"><h1>基因敲除</h1><a href="https://web.archive.org/web/20221025084210/https://baike.sogou.com/kexue/d10532.htm#!" class="detail-edit">编辑</a></div><div class="section_content" data-id="54571105930764801"><div><p>基因敲除(Knockout, KO) 是一种使生物体的某个基因失去功能的基因技术（从生物体中“敲除”）。然而，KO也可以指被敲除的基因或敲除了某基因的生物体。被敲除基因的生物体或基因敲除通常通过研究某基因缺失造成的影响推得该基因的功能。研究人员从基因敲除个体和正常个体之间的差异中得出推论。</p>
<p>KO技术本质上与基因敲入相反。在生物体中同时敲除两个基因被称为双基因敲除（Double Knockout, DKO）。类似地，术语三敲除(Triple Knockout, TKO)和四敲除(quadruple knockouts, QKO)分别用来描述敲除三个和四个基因。同时，我们需要单等位基因敲除（heterozygous KO，单敲）和双等位基因敲除（homozygous KO，双敲）。前者的两个等位基因中只有一个被敲除，后者的两个等位基因都被敲除。</p></div></div><div id="catalog"><h2 class="title2">目录<a href="javascript:" class="detail-edit">编辑</a></h2><div class="catalog_wrap" style=""><ul class="catalog_list col2"><li><span class="order">1</span><a href="javascript:" data-level="1" data-id="14995393284669954">方法</a></li><li class="secondary_catalog"><span>1.1 </span><a href="javascript:" data-id="14995393284669954">同源重组</a></li><li class="secondary_catalog"><span>1.2 </span><a href="javascript:" data-id="14995393284669954">位点特异性核酸酶</a></li><li class="secondary_catalog"><span>1.3 </span><a href="javascript:" data-id="14995393284669954">基因敲入</a></li><li><span class="order">2</span><a href="javascript:" data-level="1" data-id="14995393284669955">类型</a></li></ul><ul class="catalog_list col2"><li class="secondary_catalog"><span>2.1 </span><a href="javascript:" data-id="14995393284669955">条件性敲除</a></li><li><span class="order">3</span><a href="javascript:" data-level="1" data-id="14995393284669956">应用</a></li><li><span class="order">4</span><a href="javascript:" data-level="1" data-id="references">参考文献</a></li></ul></div></div><div id="paragraphs"><div><div id="par_14995393284669954"><h2 class="title">1 方法<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>基因敲除可通过多种技术完成。最初，自然发生的突变被鉴定后，必须通过基因测序或其他方法确定基因的丢失或失活。<sup><a href="https://web.archive.org/web/20221025084210/https://baike.sogou.com/kexue/d10532.htm#quote_1" class="kx_ref">[1]</a></sup></p><p></p><div class="text_img ed_imgfloat_embedmiddle">
            <a class="ed_image_link" data-src="https://img02.sogoucdn.com/app/a/200698/sogou_science_4420" data-bigsrc="" title="点击查看大图" href="javascript:">
                <img alt="" class="ed_imgfloat_embedmiddle" img_height="195" img_width="300" titlename="左边是一只敲除了影响毛发生长的基因的小鼠，右边是一只正常的小鼠。" data-align="embedmiddle" width="300" data-src="https://img02.sogoucdn.com/app/a/200698/sogou_science_4420" src="./532.基因敲除 - 搜狗科学百科_files/sogou_science_4420" data-observer="true">
            </a>
            <div class="text_img_title" style="text-align: center;">左边是一只敲除了影响毛发生长的基因的小鼠，右边是一只正常的小鼠。</div>
        </div> <p></p><p></p><h3>1.1 <span>同源重组</span></h3><p></p><p>传统上，同源重组是敲除基因的主要方法。这种方法需要构建一段含有目标突变的DNA，用抗药标记基因（drug resistance marker）来代替所要敲除的基因，并包含一段与靶序列至少有2kb同源的序列。<sup><a href="https://web.archive.org/web/20221025084210/https://baike.sogou.com/kexue/d10532.htm#quote_2" class="kx_ref">[2]</a></sup> 该DNA构建片段可通过显微注射或电穿孔转入干细胞，然后依靠细胞自身的修复机制将其重组到现有的DNA中。<sup><a href="https://web.archive.org/web/20221025084210/https://baike.sogou.com/kexue/d10532.htm#quote_2" class="kx_ref">[2]</a></sup> 这导致基因序列的改变，并且在大多数情况下，会被完整翻译成一种无功能蛋白。<sup><a href="https://web.archive.org/web/20221025084210/https://baike.sogou.com/kexue/d10532.htm#quote_2" class="kx_ref">[2]</a></sup> 这然而，这是一个低效过程，因为同源重组只占DNA整合的10<sup>−2</sup>至10<sup>-3</sup>。<sup><a href="https://web.archive.org/web/20221025084210/https://baike.sogou.com/kexue/d10532.htm#quote_2" class="kx_ref">[2]</a></sup><sup><a href="https://web.archive.org/web/20221025084210/https://baike.sogou.com/kexue/d10532.htm#quote_3" class="kx_ref">[3]</a></sup> 构建体上的抗性基因用于选择成功重组的细胞。</p><p></p><div class="text_img ed_imgfloat_embedmiddle">
            <a class="ed_image_link" data-src="https://img01.sogoucdn.com/app/a/200698/sogou_science_4421" data-bigsrc="" title="点击查看大图" href="javascript:">
                <img alt="" class="ed_imgfloat_embedmiddle" img_height="148" img_width="300" titlename="野生型的和基因敲除的小立碗藓：在基因突变文库转化子中诱导的表型变异。小立碗藓的野生型和转化株在最低浓度的Knop培养基上生长，以诱导配子体的分化和发育。每株植物的概观(第一行；比例尺对应于1毫米)和特写镜头(底部一行；比例尺等于0.5毫米)见图。图A：单倍体野生型苔藓植物完全被叶状配子托覆盖以及野生型叶片的特写。图B-D：不同的突变体。[4]" data-align="embedmiddle" width="300" data-src="https://img01.sogoucdn.com/app/a/200698/sogou_science_4421" src="./532.基因敲除 - 搜狗科学百科_files/sogou_science_4421" data-observer="true">
            </a>
            <div class="text_img_title">野生型的和基因敲除的小立碗藓：在基因突变文库转化子中诱导的表型变异。小立碗藓的野生型和转化株在最低浓度的Knop培养基上生长，以诱导配子体的分化和发育。每株植物的概观(第一行；比例尺对应于1毫米)和特写镜头(底部一行；比例尺等于0.5毫米)见图。图A：单倍体野生型苔藓植物完全被叶状配子托覆盖以及野生型叶片的特写。图B-D：不同的突变体。[4]</div>
        </div> <p></p><p>这些敲除了基因的干细胞可以用于体内实验，例如植入小鼠的早期胚胎。<sup><a href="https://web.archive.org/web/20221025084210/https://baike.sogou.com/kexue/d10532.htm#quote_2" class="kx_ref">[2]</a></sup> 如果遗传变化产生在嵌合小鼠的生殖细胞系中，那么这种变化就可以遗传给后代。<sup><a href="https://web.archive.org/web/20221025084210/https://baike.sogou.com/kexue/d10532.htm#quote_2" class="kx_ref">[2]</a></sup> </p><p>在二倍体生物中，大多数基因含有两个等位基因，也可能含有几个相关的基因，它们协同工作，在每个目标基因被敲除之前，还要进行更多轮的转化和选择。产生双等位基因敲除的动物可能需要选择性繁殖。</p><p></p><h3>1.2 <span>位点特异性核酸酶</span></h3><p></p><p>图:单碱基对缺失导致的移码突变引起氨基酸序列改变和提前到来的终止密码子。</p><p>目前在使用的三种方法都需要精确靶向一个DNA序列来引入双链断裂。一旦发生这种情况，细胞的修复机制将试图修复该断裂，通常是通过非同源末端连接(non-homologous end joining, NHEJ)，包括直接将两个切割端连接在一起。<sup><a href="https://web.archive.org/web/20221025084210/https://baike.sogou.com/kexue/d10532.htm#quote_3" class="kx_ref">[3]</a></sup> 这可能是不完美的，因此有时会导致碱基对的插入或缺失，从而导致移码突变。这些突变会使基因失去功能，从而导致该基因被敲除。这个过程比同源重组更有效，因此更常用于产生双等位基因敲除。<sup><a href="https://web.archive.org/web/20221025084210/https://baike.sogou.com/kexue/d10532.htm#quote_3" class="kx_ref">[3]</a></sup> </p><p><strong>锌指</strong></p><p>锌指核酸酶由能够精确定位DNA序列的DNA结合域组成。<sup><a href="https://web.archive.org/web/20221025084210/https://baike.sogou.com/kexue/d10532.htm#quote_3" class="kx_ref">[3]</a></sup> 每个锌指可以识别所需DNA序列的密码子，因此可以模块化组装结合特定的序列。<sup><a href="https://web.archive.org/web/20221025084210/https://baike.sogou.com/kexue/d10532.htm#quote_4" class="kx_ref">[4]</a></sup> 这些结合域与限制性内切酶偶联，可导致双链断裂(double stranded break, DSB)。<sup><a href="https://web.archive.org/web/20221025084210/https://baike.sogou.com/kexue/d10532.htm#quote_3" class="kx_ref">[3]</a></sup> 修复过程可能会引入破坏基因功能的突变。</p><p><strong>TALENS</strong></p><p>转录激活因子样效应物核酸酶(Transcription activator-like effector nucleases, TALENs)也包含一个DNA结合域和一个能切割DNA的核酸酶。<sup><a href="https://web.archive.org/web/20221025084210/https://baike.sogou.com/kexue/d10532.htm#quote_5" class="kx_ref">[5]</a></sup> DNA结合区由氨基酸重复单元组成，每个氨基酸重复单元识别目标DNA的单个碱基对。<sup><a href="https://web.archive.org/web/20221025084210/https://baike.sogou.com/kexue/d10532.htm#quote_4" class="kx_ref">[4]</a></sup> 如果这种切割以基因编码区为目标，并且NHEJ介导的修复引入了插入和缺失，移码突变就会发生，从而基因功能就会被破坏。<sup><a href="https://web.archive.org/web/20221025084210/https://baike.sogou.com/kexue/d10532.htm#quote_5" class="kx_ref">[5]</a></sup> </p><p><strong>CRISPR/Cas9</strong></p><p>规律成簇间隔短回文重复(Clustered regularly interspaced short palindromic repeats, CRISPR)/Cas9是一种基因组编辑方法，包含与Cas9蛋白复合的导向RNA。<sup><a href="https://web.archive.org/web/20221025084210/https://baike.sogou.com/kexue/d10532.htm#quote_4" class="kx_ref">[4]</a></sup> 导向RNA可以通过简单的碱基互补配对被设计成与所需的DNA序列相匹配，而不是锌指或TALENs所需要的耗时的构建体组装。<sup><a href="https://web.archive.org/web/20221025084210/https://baike.sogou.com/kexue/d10532.htm#quote_6" class="kx_ref">[6]</a></sup> 偶联的Cas9将导致DNA双链断裂。<sup><a href="https://web.archive.org/web/20221025084210/https://baike.sogou.com/kexue/d10532.htm#quote_4" class="kx_ref">[4]</a></sup> 遵循与锌指和TALENs相同的原理，修复这些双链断裂的机制经常导致移码突变，从而导致基因失去功能。<sup><a href="https://web.archive.org/web/20221025084210/https://baike.sogou.com/kexue/d10532.htm#quote_4" class="kx_ref">[4]</a></sup> </p><p></p><h3>1.3 <span>基因敲入</span></h3><p></p><p>基因敲入与基因敲除相似，但它是用一个基因代替另一个基因，而不是删除它。</p></div></div><div id="par_14995393284669955"><h2 class="title">2 类型<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p></p><h3>2.1 <span>条件性敲除</span></h3><p></p><p>条件性基因敲除允许组织中基因以特定时间的方式缺失。当无突变会导致胚胎死亡时，这种方法会取代基因敲除。<sup><a href="https://web.archive.org/web/20221025084210/https://baike.sogou.com/kexue/d10532.htm#quote_7" class="kx_ref">[7]</a></sup> 它通过在基因周围引入称为loxP位点的短序列来完成引入方法与敲除相同。然后可以将该种系与另一个含有Cre-重组酶的种系杂交，Cre-重组酶是一种病毒酶，能够识别这些短序列，重组它们并删除这些位点侧翼的基因。</p></div></div><div id="par_14995393284669956"><h2 class="title">3 应用<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p></p><div class="text_img ed_imgfloat_right">
            <a class="ed_image_link lazyLoad" data-src="https://img01.sogoucdn.com/app/a/200698/sogou_science_4422" data-bigsrc="" title="点击查看大图" href="javascript:" data-observer="true"></a>
            <div class="text_img_title">与正常小鼠相比，左边为通过基因敲除获得的肥胖症模型小鼠。</div>   
        </div> <p></p><p>基因敲除主要用于通过将被敲除某特定基因的生物与具有相似遗传背景的野生型进行比较来理解该特定基因或DNA区域的作用。</p><p>基因敲除生物还被用作药物开发中的筛选工具，通过特定敲除来靶向特定的生理过程或缺陷，或者通过使用横跨整个基因组的敲除文库来理解某种药物的作用机制，如用于酿酒酵母中。<sup><a href="https://web.archive.org/web/20221025084210/https://baike.sogou.com/kexue/d10532.htm#quote_8" class="kx_ref">[8]</a></sup> </p></div></div></div></div><div id="references"><h2 class="title" id="par_references">参考文献</h2><ul class="references"><li id="quote_1"><span class="references-num">[1]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Griffiths AJ, Miller JH, Suzuki DT, Lewontin WC, Gelbart WM (2000). An Introduction to Genetic Analysis (7th ed.). New York: W. H. Freeman. ISBN 978-0-7167-3771-1..</span></p></li><li id="quote_2"><span class="references-num">[2]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Hall, Bradford; Limaye, Advait; Kulkarni, Ashok B. (2009-09-01). Current Protocols in Cell Biology. Current Protocols in Cell Biology. 44. Wiley-Blackwell. pp. Unit 19.12 19.12.1–17. doi:10.1002/0471143030.cb1912s44. ISBN 978-0471143031. PMC 2782548. PMID 19731224..</span></p></li><li id="quote_3"><span class="references-num">[3]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Santiago, Yolanda; Chan, Edmond; Liu, Pei-Qi; Orlando, Salvatore; Zhang, Lin; Urnov, Fyodor D.; Holmes, Michael C.; Guschin, Dmitry; Waite, Adam (2008-04-15). "Targeted gene knockout in mammalian cells by using engineered zinc-finger nucleases". Proceedings of the National Academy of Sciences. 105 (15): 5809–5814. doi:10.1073/pnas.0800940105. ISSN 0027-8424. PMC 2299223. PMID 18359850..</span></p></li><li id="quote_4"><span class="references-num">[4]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Gaj, Thomas; Gersbach, Charles A.; Barbas, Carlos F. (2013). "ZFN, TALEN, and CRISPR/Cas-based methods for genome engineering". Trends in Biotechnology. 31 (7): 397–405. doi:10.1016/j.tibtech.2013.04.004. PMC 3694601. PMID 23664777..</span></p></li><li id="quote_5"><span class="references-num">[5]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Joung, J. Keith; Sander, Jeffry D. (January 2013). "TALENs: a widely applicable technology for targeted genome editing". Nature Reviews Molecular Cell Biology. 14 (1): 49–55. doi:10.1038/nrm3486. ISSN 1471-0080. PMC 3547402. PMID 23169466..</span></p></li><li id="quote_6"><span class="references-num">[6]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Ni, Wei; Qiao, Jun; Hu, Shengwei; Zhao, Xinxia; Regouski, Misha; Yang, Min; Polejaeva, Irina A.; Chen, Chuangfu (2014-09-04). "Efficient Gene Knockout in Goats Using CRISPR/Cas9 System". PLOS ONE. 9 (9): e106718. doi:10.1371/journal.pone.0106718. ISSN 1932-6203. PMC 4154755. PMID 25188313..</span></p></li><li id="quote_7"><span class="references-num">[7]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Le, Yunzheng; Sauer, Brian (2001-03-01). "Conditional gene knockout using cre recombinase". Molecular Biotechnology. 17 (3): 269–275. doi:10.1385/MB:17:3:269. ISSN 1073-6085. PMID 11434315..</span></p></li><li id="quote_8"><span class="references-num">[8]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"YeastDeletionWebPages". Retrieved 21 February 2017..</span></p></li></ul></div><div class="read-num">阅读 <!-- -->5038</div></div><div class="right-side" id="rightSide"><div class="side" id="lemma-side"><div class="side-title">版本记录</div><ul class="side-lst"><li><p class="side-lst-txt">暂无</p></li></ul><div class="user-card userCard"></div></div><div class="side"><div class="side-event"></div></div></div></div><div class="footer-box"><div id="footer"><div class="footer-logo-wrap"><div class="footer-logo"></div><div class="footer-logo-text">知识·传播·科普</div></div><div class="footer-info">本网站内容采用<a target="_blank" href="https://web.archive.org/web/20221025084210/https://creativecommons.org/licenses/by-sa/3.0/deed.zh?tdsourcetag=s_pctim_aiomsg">CC-BY-SA 3.0</a>授权</div><div class="footer-btn-wrap"><a target="_blank" href="https://web.archive.org/web/20221025084210/https://baike.sogou.com/help/#user_protocol">用户协议</a><a target="_blank" href="https://web.archive.org/web/20221025084210/http://www.sogou.com/docs/terms.htm?v=1">免责声明</a><a target="_blank" href="https://web.archive.org/web/20221025084210/http://corp.sogou.com/private.html">隐私政策</a><a target="_blank" href="https://web.archive.org/web/20221025084210/https://baike.sogou.com/kexue/intro.htm">关于我们</a></div></div></div><script>window.lemmaInfo ={"lemmaId":"10532","versionId":"54571105913987593","title":"基因敲除","subtitle":"","abstracts":{"paragraphId":"54571105930764801","title":"摘要","versionId":"54571105913987594","lemmaId":10532,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82815118,"name":"iscience","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599822913,"comment":null,"dependVersionId":0,"contentType":2,"content":"<p>基因敲除(Knockout, KO) 是一种使生物体的某个基因失去功能的基因技术（从生物体中“敲除”）。然而，KO也可以指被敲除的基因或敲除了某基因的生物体。被敲除基因的生物体或基因敲除通常通过研究某基因缺失造成的影响推得该基因的功能。研究人员从基因敲除个体和正常个体之间的差异中得出推论。</p>\n<p>KO技术本质上与基因敲入相反。在生物体中同时敲除两个基因被称为双基因敲除（Double Knockout, DKO）。类似地，术语三敲除(Triple Knockout, TKO)和四敲除(quadruple knockouts, QKO)分别用来描述敲除三个和四个基因。同时，我们需要单等位基因敲除（heterozygous KO，单敲）和双等位基因敲除（homozygous KO，双敲）。前者的两个等位基因中只有一个被敲除，后者的两个等位基因都被敲除。</p>","pics":null,"card":null,"references":[],"versionCount":0},"card":{"paragraphId":"0","title":"","versionId":"0","lemmaId":0,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":0,"comment":null,"dependVersionId":0,"contentType":0,"content":"","pics":null,"card":null,"references":null,"versionCount":0},"categories":[{"id":4,"name":"生物学","parents":[]}],"creator":{"uid":76034032,"name":"Jenn","pic":"https://web.archive.org/web/20221025084210/https://cache.soso.com/qlogo/g?b=oidb&k=ib4Xfkrpxibt8Fd4eh5kIuvg&s=100&t=1555396811","introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":"","jobBrief":"","role":0,"roleName":null,"title":"","professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":1568622671,"editor":{"uid":76034032,"name":"Jenn","pic":"https://web.archive.org/web/20221025084210/https://cache.soso.com/qlogo/g?b=oidb&k=ib4Xfkrpxibt8Fd4eh5kIuvg&s=100&t=1555396811","introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":"","jobBrief":"","role":0,"roleName":null,"title":"","professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233950,"state":1,"versionCount":1,"upNum":4,"downNum":0,"pics":[{"originalUrl":"https://web.archive.org/web/20221025084210/https://img02.sogoucdn.com/app/a/200698/sogou_science_4420?w=300&h=195&titlename=%E5%B7%A6%E8%BE%B9%E6%98%AF%E4%B8%80%E5%8F%AA%E6%95%B2%E9%99%A4%E4%BA%86%E5%BD%B1%E5%93%8D%E6%AF%9B%E5%8F%91%E7%94%9F%E9%95%BF%E7%9A%84%E5%9F%BA%E5%9B%A0%E7%9A%84%E5%B0%8F%E9%BC%A0%EF%BC%8C%E5%8F%B3%E8%BE%B9%E6%98%AF%E4%B8%80%E5%8F%AA%E6%AD%A3%E5%B8%B8%E7%9A%84%E5%B0%8F%E9%BC%A0%E3%80%82","url":"https://web.archive.org/web/20221025084210/https://img02.sogoucdn.com/app/a/200698/sogou_science_4420","rw":300,"rh":195,"title":"左边是一只敲除了影响毛发生长的基因的小鼠，右边是一只正常的小鼠。","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221025084210/https://img01.sogoucdn.com/app/a/200698/sogou_science_4421?w=300&h=148&titlename=%E9%87%8E%E7%94%9F%E5%9E%8B%E7%9A%84%E5%92%8C%E5%9F%BA%E5%9B%A0%E6%95%B2%E9%99%A4%E7%9A%84%E5%B0%8F%E7%AB%8B%E7%A2%97%E8%97%93%EF%BC%9A%E5%9C%A8%E5%9F%BA%E5%9B%A0%E7%AA%81%E5%8F%98%E6%96%87%E5%BA%93%E8%BD%AC%E5%8C%96%E5%AD%90%E4%B8%AD%E8%AF%B1%E5%AF%BC%E7%9A%84%E8%A1%A8%E5%9E%8B%E5%8F%98%E5%BC%82%E3%80%82%E5%B0%8F%E7%AB%8B%E7%A2%97%E8%97%93%E7%9A%84%E9%87%8E%E7%94%9F%E5%9E%8B%E5%92%8C%E8%BD%AC%E5%8C%96%E6%A0%AA%E5%9C%A8%E6%9C%80%E4%BD%8E%E6%B5%93%E5%BA%A6%E7%9A%84Knop%E5%9F%B9%E5%85%BB%E5%9F%BA%E4%B8%8A%E7%94%9F%E9%95%BF%EF%BC%8C%E4%BB%A5%E8%AF%B1%E5%AF%BC%E9%85%8D%E5%AD%90%E4%BD%93%E7%9A%84%E5%88%86%E5%8C%96%E5%92%8C%E5%8F%91%E8%82%B2%E3%80%82%E6%AF%8F%E6%A0%AA%E6%A4%8D%E7%89%A9%E7%9A%84%E6%A6%82%E8%A7%82(%E7%AC%AC%E4%B8%80%E8%A1%8C%EF%BC%9B%E6%AF%94%E4%BE%8B%E5%B0%BA%E5%AF%B9%E5%BA%94%E4%BA%8E1%E6%AF%AB%E7%B1%B3)%E5%92%8C%E7%89%B9%E5%86%99%E9%95%9C%E5%A4%B4(%E5%BA%95%E9%83%A8%E4%B8%80%E8%A1%8C%EF%BC%9B%E6%AF%94%E4%BE%8B%E5%B0%BA%E7%AD%89%E4%BA%8E0.5%E6%AF%AB%E7%B1%B3)%E8%A7%81%E5%9B%BE%E3%80%82%E5%9B%BEA%EF%BC%9A%E5%8D%95%E5%80%8D%E4%BD%93%E9%87%8E%E7%94%9F%E5%9E%8B%E8%8B%94%E8%97%93%E6%A4%8D%E7%89%A9%E5%AE%8C%E5%85%A8%E8%A2%AB%E5%8F%B6%E7%8A%B6%E9%85%8D%E5%AD%90%E6%89%98%E8%A6%86%E7%9B%96%E4%BB%A5%E5%8F%8A%E9%87%8E%E7%94%9F%E5%9E%8B%E5%8F%B6%E7%89%87%E7%9A%84%E7%89%B9%E5%86%99%E3%80%82%E5%9B%BEB-D%EF%BC%9A%E4%B8%8D%E5%90%8C%E7%9A%84%E7%AA%81%E5%8F%98%E4%BD%93%E3%80%82%5B4%5D","url":"https://web.archive.org/web/20221025084210/https://img01.sogoucdn.com/app/a/200698/sogou_science_4421","rw":300,"rh":148,"title":"野生型的和基因敲除的小立碗藓：在基因突变文库转化子中诱导的表型变异。小立碗藓的野生型和转化株在最低浓度的Knop培养基上生长，以诱导配子体的分化和发育。每株植物的概观(第一行；比例尺对应于1毫米)和特写镜头(底部一行；比例尺等于0.5毫米)见图。图A：单倍体野生型苔藓植物完全被叶状配子托覆盖以及野生型叶片的特写。图B-D：不同的突变体。[4]","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221025084210/https://img01.sogoucdn.com/app/a/200698/sogou_science_4422?w=300&h=156&titlename=%E4%B8%8E%E6%AD%A3%E5%B8%B8%E5%B0%8F%E9%BC%A0%E7%9B%B8%E6%AF%94%EF%BC%8C%E5%B7%A6%E8%BE%B9%E4%B8%BA%E9%80%9A%E8%BF%87%E5%9F%BA%E5%9B%A0%E6%95%B2%E9%99%A4%E8%8E%B7%E5%BE%97%E7%9A%84%E8%82%A5%E8%83%96%E7%97%87%E6%A8%A1%E5%9E%8B%E5%B0%8F%E9%BC%A0%E3%80%82","url":"https://web.archive.org/web/20221025084210/https://img01.sogoucdn.com/app/a/200698/sogou_science_4422","rw":300,"rh":156,"title":"与正常小鼠相比，左边为通过基因敲除获得的肥胖症模型小鼠。","alt":null,"width":0,"height":0}],"catalogs":[{"level":1,"title":"方法","paragraphId":"14995393284669954","subCatalogs":[{"level":2,"title":"同源重组","paragraphId":"14995393284669954","subCatalogs":null},{"level":2,"title":"位点特异性核酸酶","paragraphId":"14995393284669954","subCatalogs":null},{"level":2,"title":"基因敲入","paragraphId":"14995393284669954","subCatalogs":null}]},{"level":1,"title":"类型","paragraphId":"14995393284669955","subCatalogs":[{"level":2,"title":"条件性敲除","paragraphId":"14995393284669955","subCatalogs":null}]},{"level":1,"title":"应用","paragraphId":"14995393284669956","subCatalogs":null},{"level":1,"title":"参考文献","paragraphId":"-1","subCatalogs":null}],"paragraphs":[{"paragraphId":"14995393284669954","title":"方法","versionId":"54571105913987595","lemmaId":10532,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82815118,"name":"iscience","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599822913,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>基因敲除可通过多种技术完成。最初，自然发生的突变被鉴定后，必须通过基因测序或其他方法确定基因的丢失或失活。<sup><a href=\"#quote_1\" class=\"kx_ref\">[1]</a></sup></p><p><img alt=\"\" class=\"ed_imgfloat_embedmiddle\" img_height=\"195\" img_width=\"300\" titlename=\"左边是一只敲除了影响毛发生长的基因的小鼠，右边是一只正常的小鼠。\" data-align=\"embedmiddle\" width=\"300\" data-src=\"https://img02.sogoucdn.com/app/a/200698/sogou_science_4420\"> </p><p></p><h3>同源重组</h3><p></p><p>传统上，同源重组是敲除基因的主要方法。这种方法需要构建一段含有目标突变的DNA，用抗药标记基因（drug resistance marker）来代替所要敲除的基因，并包含一段与靶序列至少有2kb同源的序列。<sup><a href=\"#quote_2\" class=\"kx_ref\">[2]</a></sup> 该DNA构建片段可通过显微注射或电穿孔转入干细胞，然后依靠细胞自身的修复机制将其重组到现有的DNA中。<sup><a href=\"#quote_2\" class=\"kx_ref\">[2]</a></sup> 这导致基因序列的改变，并且在大多数情况下，会被完整翻译成一种无功能蛋白。<sup><a href=\"#quote_2\" class=\"kx_ref\">[2]</a></sup> 这然而，这是一个低效过程，因为同源重组只占DNA整合的10<sup>−2</sup>至10<sup>-3</sup>。<sup><a href=\"#quote_2\" class=\"kx_ref\">[2]</a></sup><sup><a href=\"#quote_3\" class=\"kx_ref\">[3]</a></sup> 构建体上的抗性基因用于选择成功重组的细胞。</p><p><img alt=\"\" class=\"ed_imgfloat_embedmiddle\" img_height=\"148\" img_width=\"300\" titlename=\"野生型的和基因敲除的小立碗藓：在基因突变文库转化子中诱导的表型变异。小立碗藓的野生型和转化株在最低浓度的Knop培养基上生长，以诱导配子体的分化和发育。每株植物的概观(第一行；比例尺对应于1毫米)和特写镜头(底部一行；比例尺等于0.5毫米)见图。图A：单倍体野生型苔藓植物完全被叶状配子托覆盖以及野生型叶片的特写。图B-D：不同的突变体。[4]\" data-align=\"embedmiddle\" width=\"300\" data-src=\"https://img01.sogoucdn.com/app/a/200698/sogou_science_4421\"> </p><p>这些敲除了基因的干细胞可以用于体内实验，例如植入小鼠的早期胚胎。<sup><a href=\"#quote_2\" class=\"kx_ref\">[2]</a></sup> 如果遗传变化产生在嵌合小鼠的生殖细胞系中，那么这种变化就可以遗传给后代。<sup><a href=\"#quote_2\" class=\"kx_ref\">[2]</a></sup> </p><p>在二倍体生物中，大多数基因含有两个等位基因，也可能含有几个相关的基因，它们协同工作，在每个目标基因被敲除之前，还要进行更多轮的转化和选择。产生双等位基因敲除的动物可能需要选择性繁殖。</p><p></p><h3>位点特异性核酸酶</h3><p></p><p>图:单碱基对缺失导致的移码突变引起氨基酸序列改变和提前到来的终止密码子。</p><p>目前在使用的三种方法都需要精确靶向一个DNA序列来引入双链断裂。一旦发生这种情况，细胞的修复机制将试图修复该断裂，通常是通过非同源末端连接(non-homologous end joining, NHEJ)，包括直接将两个切割端连接在一起。<sup><a href=\"#quote_3\" class=\"kx_ref\">[3]</a></sup> 这可能是不完美的，因此有时会导致碱基对的插入或缺失，从而导致移码突变。这些突变会使基因失去功能，从而导致该基因被敲除。这个过程比同源重组更有效，因此更常用于产生双等位基因敲除。<sup><a href=\"#quote_3\" class=\"kx_ref\">[3]</a></sup> </p><p><strong>锌指</strong></p><p>锌指核酸酶由能够精确定位DNA序列的DNA结合域组成。<sup><a href=\"#quote_3\" class=\"kx_ref\">[3]</a></sup> 每个锌指可以识别所需DNA序列的密码子，因此可以模块化组装结合特定的序列。<sup><a href=\"#quote_4\" class=\"kx_ref\">[4]</a></sup> 这些结合域与限制性内切酶偶联，可导致双链断裂(double stranded break, DSB)。<sup><a href=\"#quote_3\" class=\"kx_ref\">[3]</a></sup> 修复过程可能会引入破坏基因功能的突变。</p><p><strong>TALENS</strong></p><p>转录激活因子样效应物核酸酶(Transcription activator-like effector nucleases, TALENs)也包含一个DNA结合域和一个能切割DNA的核酸酶。<sup><a href=\"#quote_5\" class=\"kx_ref\">[5]</a></sup> DNA结合区由氨基酸重复单元组成，每个氨基酸重复单元识别目标DNA的单个碱基对。<sup><a href=\"#quote_4\" class=\"kx_ref\">[4]</a></sup> 如果这种切割以基因编码区为目标，并且NHEJ介导的修复引入了插入和缺失，移码突变就会发生，从而基因功能就会被破坏。<sup><a href=\"#quote_5\" class=\"kx_ref\">[5]</a></sup> </p><p><strong>CRISPR/Cas9</strong></p><p>规律成簇间隔短回文重复(Clustered regularly interspaced short palindromic repeats, CRISPR)/Cas9是一种基因组编辑方法，包含与Cas9蛋白复合的导向RNA。<sup><a href=\"#quote_4\" class=\"kx_ref\">[4]</a></sup> 导向RNA可以通过简单的碱基互补配对被设计成与所需的DNA序列相匹配，而不是锌指或TALENs所需要的耗时的构建体组装。<sup><a href=\"#quote_6\" class=\"kx_ref\">[6]</a></sup> 偶联的Cas9将导致DNA双链断裂。<sup><a href=\"#quote_4\" class=\"kx_ref\">[4]</a></sup> 遵循与锌指和TALENs相同的原理，修复这些双链断裂的机制经常导致移码突变，从而导致基因失去功能。<sup><a href=\"#quote_4\" class=\"kx_ref\">[4]</a></sup> </p><p></p><h3>基因敲入</h3><p></p><p>基因敲入与基因敲除相似，但它是用一个基因代替另一个基因，而不是删除它。</p>","pics":[{"originalUrl":"https://web.archive.org/web/20221025084210/https://img02.sogoucdn.com/app/a/200698/sogou_science_4420?w=300&h=195&titlename=%E5%B7%A6%E8%BE%B9%E6%98%AF%E4%B8%80%E5%8F%AA%E6%95%B2%E9%99%A4%E4%BA%86%E5%BD%B1%E5%93%8D%E6%AF%9B%E5%8F%91%E7%94%9F%E9%95%BF%E7%9A%84%E5%9F%BA%E5%9B%A0%E7%9A%84%E5%B0%8F%E9%BC%A0%EF%BC%8C%E5%8F%B3%E8%BE%B9%E6%98%AF%E4%B8%80%E5%8F%AA%E6%AD%A3%E5%B8%B8%E7%9A%84%E5%B0%8F%E9%BC%A0%E3%80%82","url":"https://web.archive.org/web/20221025084210/https://img02.sogoucdn.com/app/a/200698/sogou_science_4420","rw":300,"rh":195,"title":"左边是一只敲除了影响毛发生长的基因的小鼠，右边是一只正常的小鼠。","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221025084210/https://img01.sogoucdn.com/app/a/200698/sogou_science_4421?w=300&h=148&titlename=%E9%87%8E%E7%94%9F%E5%9E%8B%E7%9A%84%E5%92%8C%E5%9F%BA%E5%9B%A0%E6%95%B2%E9%99%A4%E7%9A%84%E5%B0%8F%E7%AB%8B%E7%A2%97%E8%97%93%EF%BC%9A%E5%9C%A8%E5%9F%BA%E5%9B%A0%E7%AA%81%E5%8F%98%E6%96%87%E5%BA%93%E8%BD%AC%E5%8C%96%E5%AD%90%E4%B8%AD%E8%AF%B1%E5%AF%BC%E7%9A%84%E8%A1%A8%E5%9E%8B%E5%8F%98%E5%BC%82%E3%80%82%E5%B0%8F%E7%AB%8B%E7%A2%97%E8%97%93%E7%9A%84%E9%87%8E%E7%94%9F%E5%9E%8B%E5%92%8C%E8%BD%AC%E5%8C%96%E6%A0%AA%E5%9C%A8%E6%9C%80%E4%BD%8E%E6%B5%93%E5%BA%A6%E7%9A%84Knop%E5%9F%B9%E5%85%BB%E5%9F%BA%E4%B8%8A%E7%94%9F%E9%95%BF%EF%BC%8C%E4%BB%A5%E8%AF%B1%E5%AF%BC%E9%85%8D%E5%AD%90%E4%BD%93%E7%9A%84%E5%88%86%E5%8C%96%E5%92%8C%E5%8F%91%E8%82%B2%E3%80%82%E6%AF%8F%E6%A0%AA%E6%A4%8D%E7%89%A9%E7%9A%84%E6%A6%82%E8%A7%82(%E7%AC%AC%E4%B8%80%E8%A1%8C%EF%BC%9B%E6%AF%94%E4%BE%8B%E5%B0%BA%E5%AF%B9%E5%BA%94%E4%BA%8E1%E6%AF%AB%E7%B1%B3)%E5%92%8C%E7%89%B9%E5%86%99%E9%95%9C%E5%A4%B4(%E5%BA%95%E9%83%A8%E4%B8%80%E8%A1%8C%EF%BC%9B%E6%AF%94%E4%BE%8B%E5%B0%BA%E7%AD%89%E4%BA%8E0.5%E6%AF%AB%E7%B1%B3)%E8%A7%81%E5%9B%BE%E3%80%82%E5%9B%BEA%EF%BC%9A%E5%8D%95%E5%80%8D%E4%BD%93%E9%87%8E%E7%94%9F%E5%9E%8B%E8%8B%94%E8%97%93%E6%A4%8D%E7%89%A9%E5%AE%8C%E5%85%A8%E8%A2%AB%E5%8F%B6%E7%8A%B6%E9%85%8D%E5%AD%90%E6%89%98%E8%A6%86%E7%9B%96%E4%BB%A5%E5%8F%8A%E9%87%8E%E7%94%9F%E5%9E%8B%E5%8F%B6%E7%89%87%E7%9A%84%E7%89%B9%E5%86%99%E3%80%82%E5%9B%BEB-D%EF%BC%9A%E4%B8%8D%E5%90%8C%E7%9A%84%E7%AA%81%E5%8F%98%E4%BD%93%E3%80%82%5B4%5D","url":"https://web.archive.org/web/20221025084210/https://img01.sogoucdn.com/app/a/200698/sogou_science_4421","rw":300,"rh":148,"title":"野生型的和基因敲除的小立碗藓：在基因突变文库转化子中诱导的表型变异。小立碗藓的野生型和转化株在最低浓度的Knop培养基上生长，以诱导配子体的分化和发育。每株植物的概观(第一行；比例尺对应于1毫米)和特写镜头(底部一行；比例尺等于0.5毫米)见图。图A：单倍体野生型苔藓植物完全被叶状配子托覆盖以及野生型叶片的特写。图B-D：不同的突变体。[4]","alt":null,"width":0,"height":0}],"card":null,"references":[],"versionCount":0},{"paragraphId":"14995393284669955","title":"类型","versionId":"54571105913987596","lemmaId":10532,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82815118,"name":"iscience","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599822913,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p><h3>条件性敲除</h3></p><p>条件性基因敲除允许组织中基因以特定时间的方式缺失。当无突变会导致胚胎死亡时，这种方法会取代基因敲除。<sup><a href=\"#quote_7\" class=\"kx_ref\">[7]</a></sup> 它通过在基因周围引入称为loxP位点的短序列来完成引入方法与敲除相同。然后可以将该种系与另一个含有Cre-重组酶的种系杂交，Cre-重组酶是一种病毒酶，能够识别这些短序列，重组它们并删除这些位点侧翼的基因。</p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14995393284669956","title":"应用","versionId":"54571105913987597","lemmaId":10532,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82815118,"name":"iscience","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599822913,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p><img alt=\"\" class=\"fileimage kx_img ed_imgfloat_right currentPic\" img_height=\"156\" img_width=\"300\" titlename=\"与正常小鼠相比，左边为通过基因敲除获得的肥胖症模型小鼠。\" data-src=\"https://img01.sogoucdn.com/app/a/200698/sogou_science_4422\"> </p><p>基因敲除主要用于通过将被敲除某特定基因的生物与具有相似遗传背景的野生型进行比较来理解该特定基因或DNA区域的作用。</p><p>基因敲除生物还被用作药物开发中的筛选工具，通过特定敲除来靶向特定的生理过程或缺陷，或者通过使用横跨整个基因组的敲除文库来理解某种药物的作用机制，如用于酿酒酵母中。<sup><a href=\"#quote_8\" class=\"kx_ref\">[8]</a></sup> </p>","pics":[{"originalUrl":"https://web.archive.org/web/20221025084210/https://img01.sogoucdn.com/app/a/200698/sogou_science_4422?w=300&h=156&titlename=%E4%B8%8E%E6%AD%A3%E5%B8%B8%E5%B0%8F%E9%BC%A0%E7%9B%B8%E6%AF%94%EF%BC%8C%E5%B7%A6%E8%BE%B9%E4%B8%BA%E9%80%9A%E8%BF%87%E5%9F%BA%E5%9B%A0%E6%95%B2%E9%99%A4%E8%8E%B7%E5%BE%97%E7%9A%84%E8%82%A5%E8%83%96%E7%97%87%E6%A8%A1%E5%9E%8B%E5%B0%8F%E9%BC%A0%E3%80%82","url":"https://web.archive.org/web/20221025084210/https://img01.sogoucdn.com/app/a/200698/sogou_science_4422","rw":300,"rh":156,"title":"与正常小鼠相比，左边为通过基因敲除获得的肥胖症模型小鼠。","alt":null,"width":0,"height":0}],"card":null,"references":[],"versionCount":0}],"references":[{"id":1,"type":"book","title":"Griffiths AJ, Miller JH, Suzuki DT, Lewontin WC, Gelbart WM (2000). An Introduction to Genetic Analysis (7th ed.). New York: W. H. Freeman. ISBN 978-0-7167-3771-1.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":2,"type":"book","title":"Hall, Bradford; Limaye, Advait; Kulkarni, Ashok B. (2009-09-01). Current Protocols in Cell Biology. Current Protocols in Cell Biology. 44. Wiley-Blackwell. pp. Unit 19.12 19.12.1–17. doi:10.1002/0471143030.cb1912s44. ISBN 978-0471143031. PMC 2782548. PMID 19731224.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":3,"type":"book","title":"Santiago, Yolanda; Chan, Edmond; Liu, Pei-Qi; Orlando, Salvatore; Zhang, Lin; Urnov, Fyodor D.; Holmes, Michael C.; Guschin, Dmitry; Waite, Adam (2008-04-15). \"Targeted gene knockout in mammalian cells by using engineered zinc-finger nucleases\". Proceedings of the National Academy of Sciences. 105 (15): 5809–5814. doi:10.1073/pnas.0800940105. ISSN 0027-8424. PMC 2299223. PMID 18359850.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":4,"type":"book","title":"Gaj, Thomas; Gersbach, Charles A.; Barbas, Carlos F. (2013). \"ZFN, TALEN, and CRISPR/Cas-based methods for genome engineering\". Trends in Biotechnology. 31 (7): 397–405. doi:10.1016/j.tibtech.2013.04.004. PMC 3694601. PMID 23664777.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":5,"type":"book","title":"Joung, J. Keith; Sander, Jeffry D. (January 2013). \"TALENs: a widely applicable technology for targeted genome editing\". Nature Reviews Molecular Cell Biology. 14 (1): 49–55. doi:10.1038/nrm3486. ISSN 1471-0080. PMC 3547402. PMID 23169466.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":6,"type":"book","title":"Ni, Wei; Qiao, Jun; Hu, Shengwei; Zhao, Xinxia; Regouski, Misha; Yang, Min; Polejaeva, Irina A.; Chen, Chuangfu (2014-09-04). \"Efficient Gene Knockout in Goats Using CRISPR/Cas9 System\". PLOS ONE. 9 (9): e106718. doi:10.1371/journal.pone.0106718. ISSN 1932-6203. PMC 4154755. PMID 25188313.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":7,"type":"book","title":"Le, Yunzheng; Sauer, Brian (2001-03-01). \"Conditional gene knockout using cre recombinase\". Molecular Biotechnology. 17 (3): 269–275. doi:10.1385/MB:17:3:269. ISSN 1073-6085. PMID 11434315.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":8,"type":"book","title":"\"YeastDeletionWebPages\". Retrieved 21 February 2017.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false}],"recommendReferences":null,"auditState":2,"lemmaLevel":1,"origin":0,"originEnTitle":null,"originZhTitle":null,"pv":5038,"auditType":0,"synonyms":null,"showEditTime":"2019.12.13 18:45","auditors":[{"uid":0,"name":"Ki.κe","pic":"https://web.archive.org/web/20221025084210/https://wx.qlogo.cn/mmopen/vi_32/y67kfr32Doib4wg71Jiau7jVWvharic3nRKgdRRQSl6koeQJCo0GQs2Krw0vwdFRsOWnHIQOwAZsSg5lIkIrFCOcQ/132","introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":"","jobBrief":"","role":0,"roleName":null,"title":"","professionalTitle":null,"phoneNo":null,"editable":true,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false}],"hasZhishiNav":false,"auditInfos":{},"isHistory":false};</script><script crossorigin="anonymous" src="./532.基因敲除 - 搜狗科学百科_files/aegis.min.js.download"></script><script crossorigin="anonymous" src="./532.基因敲除 - 搜狗科学百科_files/main_2020092401.js.download"></script><script crossorigin="anonymous" src="./532.基因敲除 - 搜狗科学百科_files/react.production.min.js.download"></script><script crossorigin="anonymous" src="./532.基因敲除 - 搜狗科学百科_files/react-dom.production.min.js.download"></script><script crossorigin="anonymous" src="./532.基因敲除 - 搜狗科学百科_files/jquery-1.11.1.min.js.download"></script><script crossorigin="anonymous" src="./532.基因敲除 - 搜狗科学百科_files/main_2022062701.js.download"></script><script crossorigin="anonymous" src="./532.基因敲除 - 搜狗科学百科_files/main_66bbe21.js.download"></script><script crossorigin="anonymous" src="./532.基因敲除 - 搜狗科学百科_files/react.production.min.js.download"></script><script crossorigin="anonymous" src="./532.基因敲除 - 搜狗科学百科_files/react-dom.production.min.js.download"></script><script crossorigin="anonymous" src="./532.基因敲除 - 搜狗科学百科_files/main_edf0f08.js.download"></script>
</body></html>