<!DOCTYPE html>
<!-- saved from url=(0083)https://web.archive.org/web/20221025115444/https://baike.sogou.com/kexue/d10555.htm -->
<html class="" data-reactroot=""><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><script src="./555.基因本体论 - 搜狗科学百科_files/analytics.js.download" type="text/javascript"></script>
<script type="text/javascript">window.addEventListener('DOMContentLoaded',function(){var v=archive_analytics.values;v.service='wb';v.server_name='wwwb-app224.us.archive.org';v.server_ms=235;archive_analytics.send_pageview({});});</script>
<script type="text/javascript" src="./555.基因本体论 - 搜狗科学百科_files/bundle-playback.js.download" charset="utf-8"></script>
<script type="text/javascript" src="./555.基因本体论 - 搜狗科学百科_files/wombat.js.download" charset="utf-8"></script>
<script type="text/javascript">
  __wm.init("https://web.archive.org/web");
  __wm.wombat("https://baike.sogou.com/kexue/d10555.htm","20221025115444","https://web.archive.org/","web","/_static/",
	      "1666698884");
</script>
<link rel="stylesheet" type="text/css" href="./555.基因本体论 - 搜狗科学百科_files/banner-styles.css">
<link rel="stylesheet" type="text/css" href="./555.基因本体论 - 搜狗科学百科_files/iconochive.css">
<!-- End Wayback Rewrite JS Include -->
<meta name="save" content="history"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="baidu-site-verification" content="VWGb6TyYx8"><meta content="基因本体论 - 搜狗科学百科" name="keywords"><meta content="搜狗科学百科是一部有着平等、协作、分享、自由理念的网络科学全书，为每一个互联网用户创造一个涵盖所有领域知识、服务的中文知识性平台。" name="description"><meta http-equiv="x-dns-prefetch-control" content="on"><meta name="server" baike="235" ip="210" env="online"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025115444/https://cache.soso.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025115444/https://hhy.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025115444/https://pic.baike.soso.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025115444/https://ugc.qpic.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025115444/https://xui.ptlogin2.qq.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025115444/https://q1.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025115444/https://q2.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025115444/https://q3.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025115444/https://q4.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025115444/https://q.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025115444/https://img01.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025115444/https://img02.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025115444/https://img03.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025115444/https://img04.sogoucdn.com/"><link rel="Shortcut Icon" href="https://web.archive.org/web/20221025115444im_/https://www.sogou.com/images/logo/new/favicon.ico?v=4"><link rel="Bookmark" href="https://www.sogou.com/images/logo/new/favicon.ico?v=4"><link href="./555.基因本体论 - 搜狗科学百科_files/base_b849887.css" rel="stylesheet"><link href="./555.基因本体论 - 搜狗科学百科_files/detail_378aed5.css" rel="stylesheet"><link href="./555.基因本体论 - 搜狗科学百科_files/inviteAudit_7894507.css" rel="stylesheet"><link rel="stylesheet" href="./555.基因本体论 - 搜狗科学百科_files/highlight.min.css"><title>基因本体论 - 搜狗科学百科</title><style>.onekey-close {
	position: absolute;
	top: 16px;
	right: 16px;
	width: 24px;
	height: 24px;
	background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	text-indent: -999em;
	background-size: 84px;
	background-position: -63px 0;
}

.onekey-login {
	position: absolute;
	top: 16.4%;
	left: 0;
	right: 0;
	width: 100%;
}

/* .onekey-login-img {
    width: 75px;
    height: 75px;
    background: url("https://web.archive.org/web/20221025115444/https://baike.sogou.com/kexue/images/sprite_wap_baike.png") no-repeat;
    background-size: 100px 91px;
    background-position: 0 0;
    background-repeat: no-repeat;
    margin: 0 auto;
} */

.onekey-login-title {
	text-align: center;
	padding-bottom: 3px;
	font-size: 21px;
	font-weight: bold;
	line-height: 30px;
	color: #000;
}

.onekey-login-txt {
	text-align: center;
	font-family: PingFangSC;
	font-size: 14px;
	line-height: 20px;
	color: #8f8f8f;
}

.onekey-login-qq,
.onekey-login-wx,
.onekey-login-phone {
	display: block;
	width: 245px;
	height: 54px;
	border-radius: 45px;
	text-align: center;

	margin: 0 auto;
	font-size: 17px;
	line-height: 24px;
	color: #000;
	/* padding: 16px 77px; */
	border-radius: 12px;
	border: solid 1px #e0e0e0;
}
.onekey-qq-content,
.onekey-vx-content,
.onekey-phone-content {
	display: inline-block;
	margin-top: 16px;
}
.onekey-qq-content {
	padding: 0 5px;
}

.onekey-login-qq {
	margin-top: 48px;
	margin-bottom: 24px;
}

.onekey-login-qq:before {
	display: inline-block;
	content: "";
	width: 20px;
	height: 20px;
	background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	background-size: 80px;
	background-position: -20px 0;
	vertical-align: top;
	margin: 17px 8px 0 0;
}

.onekey-login-wx {
	margin-bottom: 24px;
}

.onekey-login-wx:before {
	display: inline-block;
	content: "";
	width: 21px;
	height: 21px;
	background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	background-size: 84px;
	background-position: 0 0;
	vertical-align: top;
	margin: 17px 10px 0 0;
}

.onekey-login-phone {
}

.onekey-login-phone:before {
	display: inline-block;
	content: "";
	width: 21px;
	height: 21px;
	background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	background-size: 84px;
	background-position: -42px 0;
	vertical-align: top;
	margin: 17px 10px 0 0;
}

.onekey-fixed {
	z-index: 100;
	position: fixed;
	top: 0;
	bottom: 0;
	left: 0;
	right: 0;
	background: #fff;
	width: 100%;
	height: 100%;
}

.onekey-fixed.forbid {
	z-index: 100;
	position: fixed;
	top: auto;
	bottom: 68px;
	left: 9%;
	right: 0;
	background: rgba(0, 0, 0, 0.7);
	width: 82%;
	height: 43px;
	border-radius: 25px;
	color: #ffffff;
}
.onekey-login-title.forbid {
	text-align: center;
	padding-bottom: 3px;
	font-size: 14px;
	font-weight: normal;
	line-height: 30px;
	color: white;
}
</style><style>#login_mask {
  background: #000;
  opacity: 0.5;
  filter: alpha(opacity=50);
  position: fixed;
  /*fixed好像在哪个IE上有BUG，先用用*/
  left: 0;
  top: 0;
  z-index: 999;
  height: 100%;
}

#login_iframe_container {
  position: fixed;
  width: 550px;
  height: 360px;
  z-index: 1020;
  background-color: #ffffff;
}

@media screen and (max-width: 828px) {
  #login_iframe_container {
    top: 50% !important;
    left: 50% !important;
    transform: translate(-50%, -50%);
  }
}

#login_iframe_container.new-login {
  width: 550px;
  height: 360px;
  background-image: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/background_2a4a8a6.png);
}

#login_iframe_container.new-login.no-bg {
  background: #fff;
}

#login_iframe_container.new-login .login-title {
  width: 100%;
  height: 42px;
  line-height: 42px;
  text-align: center;
  font-size: 30px;
  letter-spacing: 0.19px;
  color: #ffffff;
  margin-top: 62px;
}
#login_iframe_container.new-login .forbid-title {
  width: 100%;
  height: 42px;
  line-height: 42px;
  text-align: center;
  font-size: 24px;
  letter-spacing: 0.19px;
  color: #333333;
  margin-top: 150px;
}

#login_iframe_container.new-login.no-bg .login-title {
  color: #333333;
}

#login_iframe_container.new-login .login-subtitle {
  width: 100%;
  height: 18px;
  line-height: 18px;
  font-size: 13px;
  letter-spacing: 0.08px;
  color: #ffffff;
  text-align: center;
  margin-top: 9px;
  margin-bottom: 43px;
}

#login_iframe_container.new-login.no-bg .login-subtitle {
  color: #999999;
}

#login_iframe_container.new-login .login-subtitle::before {
  content: '';
  display: inline-block;
  width: 10px;
  height: 1px;
  background-color: #ffffff;
  position: relative;
  top: -4px;
  left: -5px;
}

#login_iframe_container.new-login .login-subtitle::after {
  content: '';
  display: inline-block;
  width: 10px;
  height: 1px;
  background-color: #ffffff;
  position: relative;
  top: -4px;
  left: 5px;
}

#login_iframe_container.new-login.no-bg .login-subtitle::before {
  background-color: #999999;
}

#login_iframe_container.new-login.no-bg .login-subtitle::after {
  background-color: #999999;
}

#login_iframe_container.new-login .close-btn {
  position: absolute;
  top: 20px;
  right: 20px;
  width: 12px;
  height: 12px;
  background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/login-sprites_e3853e5.png) -59px -10px;
  background-size: 81px 91px;
  cursor: pointer;
}

#login_iframe_container.new-login .login-btn {
  width: 220px;
  height: 47px;
  border-radius: 24px;
  border: solid 1px #dddddd;
  background-color: #ffffff;
  margin: 0 auto;
  margin-top: 28px;
  position: relative;
  display: block;
}

#login_iframe_container.new-login .login-btn .login-icon {
  position: absolute;
}

#login_iframe_container.new-login .login-btn .login-text {
  width: 61px;
  height: 47px;
  line-height: 47px;
  vertical-align: middle;
  font-size: 15px;
  letter-spacing: 0.1px;
  color: #666666;
  position: absolute;
  right: 62px;
}

#login_iframe_container.new-login .login-btn.qq-btn .login-icon {
  width: 22px;
  height: 27px;
  top: 10px;
  left: 67px;
  background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/login-sprites_e3853e5.png) -10px -54px;
  background-size: 81px 91px;
}

#login_iframe_container.new-login .login-btn.qq-btn .login-text {
  right: 59px;
}

#login_iframe_container.new-login .login-btn.wechat-btn .login-icon {
  width: 29px;
  height: 24px;
  top: 12px;
  left: 62px;
  background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/login-sprites_e3853e5.png) -10px -10px;
  background-size: 81px 91px;
}</style><style>/* -- container -- */
.rodal,
.rodal-mask {
    top: 0;
    left: 0;
    width: 100%;
    height: 100%;
    z-index: 100;
}

.rodal {
    position: fixed;
}

/* -- mask -- */
.rodal-mask {
    position: fixed;
    background: rgba(0, 0, 0, .5);
}

/* -- dialog -- */
.rodal-dialog {
    position: absolute;
    z-index: 101;
    background: #fff;
    border-radius: 3px;
    box-shadow: 0 1px 3px rgba(0, 0, 0, .2);
}

.rodal-center {
    top: 50%;
    transform: translateY(-50%);
    left: 0;
    right: 0;
    margin: 0 auto;
}

.rodal-bottom {
    left: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

.rodal-top {
    left: 0;
    right: 0;
    top: 0;
    margin: auto;
}

.rodal-left {
    top: 0;
    left: 0;
    bottom: 0;
    margin: auto;
}

.rodal-right {
    top: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

/* -- close button -- */
.rodal-close {
    position: absolute;
    cursor: pointer;
    top: 16px;
    right: 16px;
    width: 16px;
    height: 16px;
}

.rodal-close:before,
.rodal-close:after {
    position: absolute;
    content: '';
    height: 2px;
    width: 100%;
    top: 50%;
    left: 0;
    margin-top: -1px;
    background: #999;
    border-radius: 100%;
    -webkit-transition: background .2s;
    transition: background .2s;
}

.rodal-close:before {
    -webkit-transform: rotate(45deg);
    transform: rotate(45deg);
}

.rodal-close:after {
    -webkit-transform: rotate(-45deg);
    transform: rotate(-45deg);
}

.rodal-close:hover:before,
.rodal-close:hover:after {
    background: #333;
}

/* -- fade -- */
/* @-webkit-keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

@keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

.rodal-fade-enter {
    -webkit-animation: rodal-fade-enter both ease-in;
    animation: rodal-fade-enter both ease-in;
} */

@-webkit-keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

@keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

.rodal-fade-leave {
    -webkit-animation: rodal-fade-leave both ease-out;
    animation: rodal-fade-leave both ease-out;
}

/* -- zoom -- */
@-webkit-keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-enter {
    -webkit-animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-leave {
    -webkit-animation: rodal-zoom-leave both;
    animation: rodal-zoom-leave both;
}

/* -- slideDown -- */
@-webkit-keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-enter {
    -webkit-animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-leave {
    -webkit-animation: rodal-slideDown-leave both;
    animation: rodal-slideDown-leave both;
}

/* -- slideLeft -- */
@-webkit-keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-enter {
    -webkit-animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-leave {
    -webkit-animation: rodal-slideLeft-leave both;
    animation: rodal-slideLeft-leave both;
}

/* -- slideRight -- */
@-webkit-keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-enter {
    -webkit-animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-leave {
    -webkit-animation: rodal-slideRight-leave both;
    animation: rodal-slideRight-leave both;
}

/* -- slideUp -- */
@-webkit-keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-enter {
    -webkit-animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
    animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
}

@-webkit-keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-leave {
    -webkit-animation: rodal-slideUp-leave both;
    animation: rodal-slideUp-leave both;
}

/* -- flip -- */
@-webkit-keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

@keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

.rodal-flip-enter {
    -webkit-animation: rodal-flip-enter both ease-in;
    animation: rodal-flip-enter both ease-in;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

@-webkit-keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

@keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

.rodal-flip-leave {
    -webkit-animation: rodal-flip-leave both;
    animation: rodal-flip-leave both;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

/* -- rotate -- */
@-webkit-keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-enter {
    -webkit-animation: rodal-rotate-enter both;
    animation: rodal-rotate-enter both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

@-webkit-keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-leave {
    -webkit-animation: rodal-rotate-leave both;
    animation: rodal-rotate-leave both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

/* -- door -- */
@-webkit-keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

@keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

.rodal-door-enter {
    -webkit-animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

@keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

.rodal-door-leave {
    -webkit-animation: rodal-door-leave both;
    animation: rodal-door-leave both;
}</style><style>/* -- container -- */
.rodal,
.rodal-mask {
    top: 0;
    left: 0;
    width: 100%;
    height: 100%;
    z-index: 100;
}

.rodal {
    position: fixed;
}

/* -- mask -- */
.rodal-mask {
    position: fixed;
    background: rgba(0, 0, 0, .5);
}

/* -- dialog -- */
.rodal-dialog {
    position: absolute;
    z-index: 101;
    background: #fff;
    border-radius: 3px;
    box-shadow: 0 1px 3px rgba(0, 0, 0, .2);
}

.rodal-center {
    top: 50%;
    transform: translateY(-50%);
    left: 0;
    right: 0;
    margin: 0 auto;
}

.rodal-bottom {
    left: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

.rodal-top {
    left: 0;
    right: 0;
    top: 0;
    margin: auto;
}

.rodal-left {
    top: 0;
    left: 0;
    bottom: 0;
    margin: auto;
}

.rodal-right {
    top: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

/* -- close button -- */
.rodal-close {
    position: absolute;
    cursor: pointer;
    top: 16px;
    right: 16px;
    width: 16px;
    height: 16px;
}

.rodal-close:before,
.rodal-close:after {
    position: absolute;
    content: '';
    height: 2px;
    width: 100%;
    top: 50%;
    left: 0;
    margin-top: -1px;
    background: #999;
    border-radius: 100%;
    -webkit-transition: background .2s;
    transition: background .2s;
}

.rodal-close:before {
    -webkit-transform: rotate(45deg);
    transform: rotate(45deg);
}

.rodal-close:after {
    -webkit-transform: rotate(-45deg);
    transform: rotate(-45deg);
}

.rodal-close:hover:before,
.rodal-close:hover:after {
    background: #333;
}

/* -- fade -- */
/* @-webkit-keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

@keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

.rodal-fade-enter {
    -webkit-animation: rodal-fade-enter both ease-in;
    animation: rodal-fade-enter both ease-in;
} */

@-webkit-keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

@keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

.rodal-fade-leave {
    -webkit-animation: rodal-fade-leave both ease-out;
    animation: rodal-fade-leave both ease-out;
}

/* -- zoom -- */
@-webkit-keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-enter {
    -webkit-animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-leave {
    -webkit-animation: rodal-zoom-leave both;
    animation: rodal-zoom-leave both;
}

/* -- slideDown -- */
@-webkit-keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-enter {
    -webkit-animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-leave {
    -webkit-animation: rodal-slideDown-leave both;
    animation: rodal-slideDown-leave both;
}

/* -- slideLeft -- */
@-webkit-keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-enter {
    -webkit-animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-leave {
    -webkit-animation: rodal-slideLeft-leave both;
    animation: rodal-slideLeft-leave both;
}

/* -- slideRight -- */
@-webkit-keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-enter {
    -webkit-animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-leave {
    -webkit-animation: rodal-slideRight-leave both;
    animation: rodal-slideRight-leave both;
}

/* -- slideUp -- */
@-webkit-keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-enter {
    -webkit-animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
    animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
}

@-webkit-keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-leave {
    -webkit-animation: rodal-slideUp-leave both;
    animation: rodal-slideUp-leave both;
}

/* -- flip -- */
@-webkit-keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

@keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

.rodal-flip-enter {
    -webkit-animation: rodal-flip-enter both ease-in;
    animation: rodal-flip-enter both ease-in;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

@-webkit-keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

@keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

.rodal-flip-leave {
    -webkit-animation: rodal-flip-leave both;
    animation: rodal-flip-leave both;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

/* -- rotate -- */
@-webkit-keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-enter {
    -webkit-animation: rodal-rotate-enter both;
    animation: rodal-rotate-enter both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

@-webkit-keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-leave {
    -webkit-animation: rodal-rotate-leave both;
    animation: rodal-rotate-leave both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

/* -- door -- */
@-webkit-keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

@keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

.rodal-door-enter {
    -webkit-animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

@keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

.rodal-door-leave {
    -webkit-animation: rodal-door-leave both;
    animation: rodal-door-leave both;
}</style></head><body class=""><!-- BEGIN WAYBACK TOOLBAR INSERT -->
<style type="text/css">
body {
  margin-top:0 !important;
  padding-top:0 !important;
  /*min-width:800px !important;*/
}
</style>
<script>__wm.rw(0);</script>
<div id="wm-ipp-base" lang="en" style="display: block; direction: ltr;">
</div><div id="wm-ipp-print">The Wayback Machine - https://web.archive.org/web/20221025115444/https://baike.sogou.com/kexue/d10555.htm</div>
<script type="text/javascript">//<![CDATA[
__wm.bt(675,27,25,2,"web","https://baike.sogou.com/kexue/d10555.htm","20221025115444",1996,"/_static/",["/_static/css/banner-styles.css?v=S1zqJCYt","/_static/css/iconochive.css?v=qtvMKcIJ"], false);
  __wm.rw(1);
//]]></script>
<!-- END WAYBACK TOOLBAR INSERT --><script>window._gtag=window._gtag||{};window._gtag.shouldGrayed = false;if ('c45c273ead8441a183767db04c9971a8') window._gtag.traceId = 'c45c273ead8441a183767db04c9971a8';if ({"illegality":true}) window.userInfo = {"illegality":true};</script><div class="topnavbox"><ul class="topnav"><li><a href="https://web.archive.org/web/20221025115444/https://www.sogou.com/web?query=">网页</a></li><li><a href="https://web.archive.org/web/20221025115444/https://weixin.sogou.com/weixin?p=75351201">微信</a></li><li><a href="https://web.archive.org/web/20221025115444/https://zhihu.sogou.com/zhihu?p=75351218">知乎</a></li><li><a href="https://web.archive.org/web/20221025115444/https://pic.sogou.com/pics?query=">图片</a></li><li><a href="https://web.archive.org/web/20221025115444/https://v.sogou.com/v?query=">视频</a></li><li><a href="https://web.archive.org/web/20221025115444/https://mingyi.sogou.com/">医疗</a></li><li class="cur"><strong>科学</strong></li><li><a href="https://web.archive.org/web/20221025115444/https://hanyu.sogou.com/">汉语</a></li><li><a href="https://web.archive.org/web/20221025115444/https://wenwen.sogou.com/">问问</a></li><li><a href="https://web.archive.org/web/20221025115444/https://www.sogou.com/docs/more.htm">更多<span class="topraquo">»</span></a></li></ul></div><div id="header"><div class="header-wrap"><a class="header-logo" href="https://web.archive.org/web/20221025115444/https://baike.sogou.com/kexue"></a><div class="header-search"><div class="querybox" id="suggBox"><form><input id="searchInput" class="query" type="text" placeholder="搜科学领域专业百科词条" name="query" autocomplete="off" value=""><a href="javascript:;" class="query-search"></a></form></div></div><div class="header-rgt"><span class="btn-header-rgt btn-edit" id="editLemma">创建</span><div class="header-user no-login"></div></div></div></div><div class="fixed-placeholder" style="visibility:none"></div><div id="container" class=""><div class="content lemma-level1"><div class="detail-title" id="abstract-title"><h1>基因本体论</h1><a href="https://web.archive.org/web/20221025115444/https://baike.sogou.com/kexue/d10555.htm#!" class="detail-edit">编辑</a></div><div class="section_content" data-id="53509853687107845"><div class="text_img ed_imgfloat_right"><a class="ed_image_link" data-src="https://img03.sogoucdn.com/app/a/200698/650_390_20200909153537-1183588068.jpg" data-bigsrc="https://img03.sogoucdn.com/app/a/200698/650_390_20200909153537-1183588068.jpg?width=227&amp;height=174&amp;titlename=&amp;w=650&amp;h=390" title="点击查看大图" data-w="650" data-h="390" style="background-image:url(https://web.archive.org/web/20221025115444im_/https://img03.sogoucdn.com/app/a/200698/650_390_20200909153537-1183588068.jpg)" href="https://web.archive.org/web/20221025115444/https://baike.sogou.com/kexue/d10555.htm#!"></a></div><div><p><b>基因本体</b>（GO）是一项重要的生物信息学倡议，旨在统一所有物种的基因和基因产物特性的描述。<sup><a href="https://web.archive.org/web/20221025115444/https://baike.sogou.com/kexue/d10555.htm#quote_1" class="kx_ref">[1]</a></sup> 更具体地说，该项目旨在：</p>
<p>1、保持和发展其基因和基因产物特性的控制字集；</p>
<p>2、注释基因和基因产物，吸收和传播注释数据；</p>
<p>3、提供工具，便于访问项目提供的数据的所有方面，并能够使用GO（例如通过富集分析）对实验数据进行功能性解释。<sup><a href="https://web.archive.org/web/20221025115444/https://baike.sogou.com/kexue/d10555.htm#quote_2" class="kx_ref">[2]</a></sup><sup><a href="https://web.archive.org/web/20221025115444/https://baike.sogou.com/kexue/d10555.htm#quote_3" class="kx_ref">[3]</a></sup>&nbsp;</p>
<p>GO是一项更大的分类结果的一部分，即开放式生物医学本体论（OBO）的一部分。<sup><a href="https://web.archive.org/web/20221025115444/https://baike.sogou.com/kexue/d10555.htm#quote_4" class="kx_ref">[4]</a></sup> </p>
<p>尽管基因命名法本身旨在保持和发展基因和基因产物的控制字集，但基因本体通过使用标记语言使数据（基因、基因产物及其所有特性的数据）机器可读，并以一种在所有物种中统一的方式来实现（而基因命名规则因生物分类而异）。</p></div></div><div id="catalog"><h2 class="title2">目录<a href="javascript:" class="detail-edit">编辑</a></h2><div class="catalog_wrap" style=""><ul class="catalog_list col2"><li><span class="order">1</span><a href="javascript:" data-level="1" data-id="14995403367776522">语义和本体论</a></li><li class="secondary_catalog"><span>1.1 </span><a href="javascript:" data-id="14995403367776522">示例<b>语义</b></a></li><li><span class="order">2</span><a href="javascript:" data-level="1" data-id="14995403384553742">注释</a></li><li class="secondary_catalog"><span>2.1 </span><a href="javascript:" data-id="14995403384553742"><b>示例注释</b></a></li><li><span class="order">3</span><a href="javascript:" data-level="1" data-id="14995403384553743">工具</a></li></ul><ul class="catalog_list col2"><li><span class="order">4</span><a href="javascript:" data-level="1" data-id="14995403384553744">联合会</a></li><li><span class="order">5</span><a href="javascript:" data-level="1" data-id="14995403384553745">历史</a></li><li><span class="order">6</span><a href="javascript:" data-level="1" data-id="references">参考文献</a></li></ul></div></div><div id="paragraphs"><div><div id="par_14995403367776522"><h2 class="title">1 语义和本体论<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>从实际的角度来看，本体是对我们所知道的事物的一种描述。“本体”由可检测或可直接观察的事物的描述以及这些事物之间的关系组成。生物学和相关领域没有通用的标准术语，术语的用法可能特定于某个物种、研究领域甚至特定的研究群体，这使得数据的交流和共享更加困难。基因本体项目提供了描述基因产物特性的已定义语义的本体，本体涵盖三个领域：</p><p>
 </p><ul>
  <li><b>细胞组分</b>，细胞的每个部分或其细胞外环境；</li>
  <li><b>分子功能</b>，基因产物在分子水平上的基本活性，如结合或催化；</li>
  <li><b>生物过程</b>，与细胞、组织、器官和有机体等完整生命单元的功能相关的、具有明确起点和终点的操作或一组分子事件。</li>
 </ul><p></p><p>本体中的每个GO术语都有一个语义名称，可以是一个单词或一串单词、一个独特的字母数字标识符、有引述来源的定义，和指示其所属域的命名空间。语义也可能有同义词，语义的同义词被归类为与语义名称完全等同、更宽、更窄或相关；引用其他数据库中的等效概念；以及对语义含义或用法的评论。GO本体被构造成一个有向无环图，每个语义都定义了与同一个领域中的一个或多个其他语义的关系，有时还定义了与其他领域语义的关系。GO词汇设计为物种中性，包括适用于原核生物和真核生物、单细胞和多细胞生物的语义。</p><p>GO系统不是静态的，研究和注释团体的成员以及直接参与了GO项目的人对GO系统的增加、更正和修改提出了建议并征求了意见。<sup><a href="https://web.archive.org/web/20221025115444/https://baike.sogou.com/kexue/d10555.htm#quote_5" class="kx_ref">[5]</a></sup> 例如，注释者可以请求一个特定语义来表示代谢途径，或者可以在团体专家的帮助下修改本体的一部分<sup><a href="https://web.archive.org/web/20221025115444/https://baike.sogou.com/kexue/d10555.htm#quote_6" class="kx_ref">[6]</a></sup>。被建议的编辑文本由本体修改器审查，合适后就能成功被录用。</p><p>GO本体文件可以从GO网站<sup><a href="https://web.archive.org/web/20221025115444/https://baike.sogou.com/kexue/d10555.htm#quote_7" class="kx_ref">[7]</a></sup> 以多种格式免费获得，或者可以使用GO浏览器——AmiGO在线访问。基因本体项目还提供了其语义到其他分类系统的可下载映射。</p><p></p><h3>1.1 <span>示例语义</span></h3><p></p><p>
 </p><dl>
  <dd>
   Id：GO:0000016
  </dd>
  <dd>
   名称：乳糖酶活性
  </dd>
  <dd>
   命名空间：分子功能
  </dd>
  <dd>
   默认定义：“催化反应：C <sub>12</sub>H <sub>22</sub>O <sub>11</sub>（乳糖）+H <sub>2</sub>O =C <sub>6</sub>H <sub>12</sub>O <sub>6</sub>（D-葡萄糖）+C <sub>6</sub>H <sub>12</sub>O <sub>6</sub>（D-半乳糖）。”
  </dd>
  <dd>
   同义词：“乳糖酶-根皮苷水解酶活性”
  </dd>
  <dd>
   同义词：“乳糖半乳糖水解酶活性”
  </dd>
  <dd>
   相互参照：EC:3.2.1.108
  </dd>
  <dd>
   相互参照：MetaCyc:LACTASE-RXN
  </dd>
  <dd>
   相互参照：Reactome:20536
  </dd>
  <dd>
   <b>is_a：</b>GO:0004553！水解酶活性，水解O-糖基化合物
  </dd>
 </dl><p></p><p>数据来源:<sup><a href="https://web.archive.org/web/20221025115444/https://baike.sogou.com/kexue/d10555.htm#quote_8" class="kx_ref">[8]</a></sup> </p></div></div><div id="par_14995403384553742"><h2 class="title">2 注释<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>基因组注释是捕获基因产物数据的实践，GO注释使用GO本体中的语义来实现。GO联合会的成员在GO网站上提交他们的注释以便整合和传播，在GO网站上他们可以直接下载或使用AmiGO在线查看。除了基因产物标识符和相关的GO语义之外，GO注释还有以下数据：用于进行注释的引用（例如期刊文章）；表示注释所基于的证据类型的证据代码；注释的日期和创建者。</p><p>证据代码来自包含手动和自动注释方法的一组代码控制字集。例如，可追踪作者声明（TAS）意味着管理者已经阅读了一篇已发表的科学论文，并且该注释的元数据带有对该论文的引用；从序列相似性推断（ISS）意味着管理者已经审查了序列相似性搜索的输出，并验证了其生物学意义。来自自动化过程的注释（例如，使用另一个注释字集创建的重新映射注释）被赋予从电子注释推断（IEA）的代码。截至2010年4月1日，98%以上的GO注释是通过计算推断的，而不是人为推断的。<sup><a href="https://web.archive.org/web/20221025115444/https://baike.sogou.com/kexue/d10555.htm#quote_9" class="kx_ref">[9]</a></sup>由于这些注释没有经过人员检查，GO联合会认为它们不太可靠，其在AmiGO的在线数据中只包括一个子集。完整的注释数据集可以从GO网站下载。为了支持注释的开发，GO联合会为新的开发者群体提供学习营地和导师。</p><p>最近，许多机器学习算法已经被设计并实现基因本体注释的预测。<sup><a href="https://web.archive.org/web/20221025115444/https://baike.sogou.com/kexue/d10555.htm#quote_10" class="kx_ref">[10]</a></sup><sup><a href="https://web.archive.org/web/20221025115444/https://baike.sogou.com/kexue/d10555.htm#quote_11" class="kx_ref">[11]</a></sup> </p><p></p><h3>2.1 <span>示例注释</span></h3><p></p><p>
 </p><dl>
  <dd>
   基因产物：肌动蛋白，α心肌1， UniProtKB:P68032
  </dd>
  <dd>
   GO语义：心脏收缩 ; GO:0060047 (生物过程)
  </dd>
  <dd>
   证据代码：从突变表型推断（IMP）
  </dd>
  <dd>
   参考: PMID 17611253 PubMed
  </dd>
  <dd>
   分配: UniProtKB，2008年6月6日
  </dd>
 </dl><p></p><p>数据来源:<sup><a href="https://web.archive.org/web/20221025115444/https://baike.sogou.com/kexue/d10555.htm#quote_12" class="kx_ref">[12]</a></sup> </p></div></div><div id="par_14995403384553743"><h2 class="title">3 工具<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>有大量可用的在线工具和下载工具<sup><a href="https://web.archive.org/web/20221025115444/https://baike.sogou.com/kexue/d10555.htm#quote_13" class="kx_ref">[13]</a></sup> 可以使用GO项目提供的数据。这些软件的绝大多数都来自第三方；GO联合会开发和支持两种工具，<b>AmiGO</b>和<b>OBO-Edit</b>。</p><p>AmiGO<sup><a href="https://web.archive.org/web/20221025115444/https://baike.sogou.com/kexue/d10555.htm#quote_14" class="kx_ref">[14]</a></sup><sup><a href="https://web.archive.org/web/20221025115444/https://baike.sogou.com/kexue/d10555.htm#quote_15" class="kx_ref">[15]</a></sup> 是一个基于网络的应用程序，允许用户查询、浏览和可视化本体和基因产物的注释数据。它还有一个BLAST工具<sup><a href="https://web.archive.org/web/20221025115444/https://baike.sogou.com/kexue/d10555.htm#quote_15" class="kx_ref">[15]</a></sup> （此工具允许分析更大的数据集）<sup><a href="https://web.archive.org/web/20221025115444/https://baike.sogou.com/kexue/d10555.htm#quote_16" class="kx_ref">[16]</a></sup><sup><a href="https://web.archive.org/web/20221025115444/https://baike.sogou.com/kexue/d10555.htm#quote_17" class="kx_ref">[17]</a></sup> ，以及一个直接查询GO数据库的界面。<sup><a href="https://web.archive.org/web/20221025115444/https://baike.sogou.com/kexue/d10555.htm#quote_18" class="kx_ref">[18]</a></sup> </p><p>AmiGO可以在GO网站上在线使用，以访问GO联合会提供的数据，也可以在任何使用GO数据库架构的数据库中下载并安装以供本地使用<sup><a href="https://web.archive.org/web/20221025115444/https://baike.sogou.com/kexue/d10555.htm#quote_19" class="kx_ref">[19]</a></sup>。它是免费的开源软件，可以作为go-dev软件发行版的一部分。<sup><a href="https://web.archive.org/web/20221025115444/https://baike.sogou.com/kexue/d10555.htm#quote_20" class="kx_ref">[20]</a></sup> </p><p>OBO-Edit<sup><a href="https://web.archive.org/web/20221025115444/https://baike.sogou.com/kexue/d10555.htm#quote_21" class="kx_ref">[21]</a></sup> 是一个开源、独立于平台的本体修改器，由基因本体联合会开发和维护。它在Java中实现，并使用面向图形的方法来显示和编辑本体。OBO-Edit包括一个全面的搜索和过滤界面，可以选择呈现术语子集，使它们在视觉上不同；用户界面也可以根据用户偏好定制。OBO-Edit还有一个推理器，可以根据现有的关系和它们的属性推断出没有明确说明的联系。虽然它是为生物医学本体开发的，但是OBO-Edit可以用来查看、搜索和编辑任何本体。它可以免费下载。<sup><a href="https://web.archive.org/web/20221025115444/https://baike.sogou.com/kexue/d10555.htm#quote_20" class="kx_ref">[20]</a></sup> </p></div></div><div id="par_14995403384553744"><h2 class="title">4 联合会<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>基因本体联合会（Gene Ontology Consortium）是积极参与基因本体项目的一组生物数据库和研究团体。<sup><a href="https://web.archive.org/web/20221025115444/https://baike.sogou.com/kexue/d10555.htm#quote_22" class="kx_ref">[22]</a></sup> 这包括许多模式生物数据库和多物种蛋白质数据库、软件开发小组和专门的编辑部。</p></div></div><div id="par_14995403384553745"><h2 class="title">5 历史<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>基因本体最初是在1998年由一群研究三种模式生物基因组的研究人员构建的：黑腹果蝇（果蝇）、小家鼠（小鼠）和酿酒酵母（啤酒酵母或面包酵母）。<sup><a href="https://web.archive.org/web/20221025115444/https://baike.sogou.com/kexue/d10555.htm#quote_23" class="kx_ref">[23]</a></sup> 许多其他模式生物数据库加入了基因本体联合会，不仅贡献了注释数据，还有助于开发用于查看和应用数据的本体和工具。迄今为止，植物、动物和微生物领域的大部分主要数据库都为这个项目做出了贡献。截止2008年1月，GO包含超过24500个广泛适用于多种生物有机体的语义。关于GO的开发和使用有着大量的文献，GO已经成为生物信息学库中的一个标准工具。它们的目标有三个方面：建立基因本体，为基因/基因产物分配本体，为前两个对象开发软件和数据库。</p><p>利用类的形式、域无关属性（元属性）对基因本体的一些分析也开始出现。例如，可以看到对生物本体论的本体论分析。<sup><a href="https://web.archive.org/web/20221025115444/https://baike.sogou.com/kexue/d10555.htm#quote_24" class="kx_ref">[24]</a></sup> </p></div></div></div></div><div id="references"><h2 class="title" id="par_references">参考文献</h2><ul class="references"><li id="quote_1"><span class="references-num">[1]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">The Gene Ontology Consortium (January 2008). "The Gene Ontology project in 2008". Nucleic Acids Research. 36 (Database issue): D440–4. doi:10.1093/nar/gkm883. PMC 2238979. PMID 17984083..</span></p></li><li id="quote_2"><span class="references-num">[2]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Dessimoz, Christophe; Škunca, Nives, eds. (2017). The Gene Ontology Handbook. 1446. doi:10.1007/978-1-4939-3743-1. ISBN 9781493937431. ISSN 1064-3745..</span></p></li><li id="quote_3"><span class="references-num">[3]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Gaudet, Pascale; Škunca, Nives; Hu, James C.; Dessimoz, Christophe (2017). "Primer on the Gene Ontology". 1446: 25–37. doi:10.1007/978-1-4939-3743-1_3. ISSN 1064-3745..</span></p></li><li id="quote_4"><span class="references-num">[4]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Smith B, Ashburner M, Rosse C, Bard J, Bug W, Ceusters W, Goldberg LJ, Eilbeck K, Ireland A, Mungall CJ, Leontis N, Rocca-Serra P, Ruttenberg A, Sansone SA, Scheuermann RH, Shah N, Whetzel PL, Lewis S (November 2007). "The OBO Foundry: coordinated evolution of ontologies to support biomedical data integration". Nature Biotechnology. 25 (11): 1251–5. doi:10.1038/nbt1346. PMC 2814061. PMID 17989687..</span></p></li><li id="quote_5"><span class="references-num">[5]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Lovering, Ruth C. (2017). "How Does the Scientific Community Contribute to Gene Ontology?". In Dessimoz, C; Skunca, N. The Gene Ontology Handbook. Methods in Molecular Biology (in English). 1446. Springer (New York). pp. 85–93. doi:10.1007/978-1-4939-3743-1_7. ISSN 1064-3745.CS1 maint: Unrecognized language (link).</span></p></li><li id="quote_6"><span class="references-num">[6]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Diehl AD, Lee JA, Scheuermann RH, Blake JA (April 2007). "Ontology development for biological systems: immunology". Bioinformatics. 23 (7): 913–5. doi:10.1093/bioinformatics/btm029. PMID 17267433..</span></p></li><li id="quote_7"><span class="references-num">[7]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Gene Ontology Database". Gene Ontology Consortium..</span></p></li><li id="quote_8"><span class="references-num">[8]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">The GO Consortium (2009-03-16). "gene_ontology.1_2.obo" (OBO 1.2 flat file)..</span></p></li><li id="quote_9"><span class="references-num">[9]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">du Plessis L, Skunca N, Dessimoz C (November 2011). "The what, where, how and why of gene ontology--a primer for bioinformaticians". Briefings in Bioinformatics. 12 (6): 723–35. doi:10.1093/bib/bbr002. PMC 3220872. PMID 21330331..</span></p></li><li id="quote_10"><span class="references-num">[10]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Pinoli P, Chicco D, Masseroli M (June 2013). "Computational algorithms to predict Gene Ontology annotation". BMC Bioinformatics. 16 (6): 1–15. doi:10.1186/1471-2105-16-S6-S4. PMC 4416163. PMID 25916950..</span></p></li><li id="quote_11"><span class="references-num">[11]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Cozzetto, Domenico; Jones, David T. (2017). "Computational Methods for Annotation Transfers from Sequence". In Dessimoz, C; Skunca, N. The Gene Ontology Handbook. Methods in Molecular Biology (in English). 1446. Springer (New York). pp. 55–67. doi:10.1007/978-1-4939-3743-1_5. ISSN 1064-3745.CS1 maint: Unrecognized language (link).</span></p></li><li id="quote_12"><span class="references-num">[12]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">The GO Consortium (2009-03-16). "AmiGO: P68032 Associations"..</span></p></li><li id="quote_13"><span class="references-num">[13]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Mosquera JL, Sánchez-Pla A (July 2008). "SerbGO: searching for the best GO tool". Nucleic Acids Research. 36 (Web Server issue): W368–71. doi:10.1093/nar/gkn256. PMC 2447766. PMID 18480123..</span></p></li><li id="quote_14"><span class="references-num">[14]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Carbon S, Ireland A, Mungall CJ, Shu S, Marshall B, Lewis S (January 2009). AmiGO Hub; Web Presence Working Group. "AmiGO: online access to ontology and annotation data". Bioinformatics. 25 (2): 288–9. doi:10.1093/bioinformatics/btn615. PMC 2639003. PMID 19033274..</span></p></li><li id="quote_15"><span class="references-num">[15]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">AmiGO--the current official web-based set of tools for searching and browsing the Gene Ontology database.</span></p></li><li id="quote_16"><span class="references-num">[16]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">AmiGO Term Enrichment tool; finds significant shared GO terms in an annotation set.</span></p></li><li id="quote_17"><span class="references-num">[17]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">AmiGO Slimmer; maps granular annotations up to high-level terms.</span></p></li><li id="quote_18"><span class="references-num">[18]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">GOOSE, GO Online SQL Environment; allows direct SQL querying of the GO database.</span></p></li><li id="quote_19"><span class="references-num">[19]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">The Plant Ontology Consortium (2009-03-16). "Plant Ontology Consortium". Retrieved 2009-03-16..</span></p></li><li id="quote_20"><span class="references-num">[20]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Gene Ontology downloads at SourceForge". Retrieved 2009-03-16..</span></p></li><li id="quote_21"><span class="references-num">[21]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Day-Richter J, Harris MA, Haendel M, Lewis S (August 2007). "OBO-Edit--an ontology editor for biologists". Bioinformatics. 23 (16): 2198–200. doi:10.1093/bioinformatics/btm112. PMID 17545183..</span></p></li><li id="quote_22"><span class="references-num">[22]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"The GO Consortium". Retrieved 2009-03-16..</span></p></li><li id="quote_23"><span class="references-num">[23]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Ashburner M, Ball CA, Blake JA, Botstein D, Butler H, Cherry JM, Davis AP, Dolinski K, Dwight SS, Eppig JT, Harris MA, Hill DP, Issel-Tarver L, Kasarskis A, Lewis S, Matese JC, Richardson JE, Ringwald M, Rubin GM, Sherlock G (May 2000). "Gene ontology: tool for the unification of biology. The Gene Ontology Consortium". Nature Genetics. 25 (1): 25–9. doi:10.1038/75556. PMC 3037419. PMID 10802651..</span></p></li><li id="quote_24"><span class="references-num">[24]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Deb, B. (2012). "An ontological analysis of some biological ontologies". Frontiers in Genetics. 3. doi:10.3389/fgene.2012.00269. PMC 3509948..</span></p></li></ul></div><div class="read-num">阅读 <!-- -->1025</div></div><div class="right-side" id="rightSide"><div class="side" id="lemma-side"><div class="side-title">版本记录</div><ul class="side-lst"><li><p class="side-lst-txt">暂无</p></li></ul><div class="user-card userCard"></div></div><div class="side"><div class="side-event"></div></div></div></div><div class="footer-box"><div id="footer"><div class="footer-logo-wrap"><div class="footer-logo"></div><div class="footer-logo-text">知识·传播·科普</div></div><div class="footer-info">本网站内容采用<a target="_blank" href="https://web.archive.org/web/20221025115444/https://creativecommons.org/licenses/by-sa/3.0/deed.zh?tdsourcetag=s_pctim_aiomsg">CC-BY-SA 3.0</a>授权</div><div class="footer-btn-wrap"><a target="_blank" href="https://web.archive.org/web/20221025115444/https://baike.sogou.com/help/#user_protocol">用户协议</a><a target="_blank" href="https://web.archive.org/web/20221025115444/http://www.sogou.com/docs/terms.htm?v=1">免责声明</a><a target="_blank" href="https://web.archive.org/web/20221025115444/http://corp.sogou.com/private.html">隐私政策</a><a target="_blank" href="https://web.archive.org/web/20221025115444/https://baike.sogou.com/kexue/intro.htm">关于我们</a></div></div></div><script>window.lemmaInfo ={"lemmaId":"10555","versionId":"54259122140406028","title":"基因本体论","subtitle":"","abstracts":{"paragraphId":"53509853687107845","title":"摘要","versionId":"54259122140406029","lemmaId":10555,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82812762,"name":"雷克斯班纳","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599636956,"comment":null,"dependVersionId":0,"contentType":2,"content":"<p><b>基因本体</b>（GO）是一项重要的生物信息学倡议，旨在统一所有物种的基因和基因产物特性的描述。<sup><a href=\"#quote_1\" class=\"kx_ref\">[1]</a></sup> 更具体地说，该项目旨在：</p>\n<p>1、保持和发展其基因和基因产物特性的控制字集；</p>\n<p>2、注释基因和基因产物，吸收和传播注释数据；</p>\n<p>3、提供工具，便于访问项目提供的数据的所有方面，并能够使用GO（例如通过富集分析）对实验数据进行功能性解释。<sup><a href=\"#quote_2\" class=\"kx_ref\">[2]</a></sup><sup><a href=\"#quote_3\" class=\"kx_ref\">[3]</a></sup>&nbsp;</p>\n<p>GO是一项更大的分类结果的一部分，即开放式生物医学本体论（OBO）的一部分。<sup><a href=\"#quote_4\" class=\"kx_ref\">[4]</a></sup> </p>\n<p>尽管基因命名法本身旨在保持和发展基因和基因产物的控制字集，但基因本体通过使用标记语言使数据（基因、基因产物及其所有特性的数据）机器可读，并以一种在所有物种中统一的方式来实现（而基因命名规则因生物分类而异）。</p>","pics":[{"originalUrl":"https://web.archive.org/web/20221025115444/https://img03.sogoucdn.com/app/a/200698/650_390_20200909153537-1183588068.jpg?width=227&height=174&titlename=&w=650&h=390","url":"https://web.archive.org/web/20221025115444/https://img03.sogoucdn.com/app/a/200698/650_390_20200909153537-1183588068.jpg","rw":650,"rh":390,"title":"","alt":null,"width":227,"height":174}],"card":null,"references":[],"versionCount":0},"card":{"paragraphId":"14995403367776521","title":"基本信息","versionId":"14995403367776514","lemmaId":0,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":76032351,"name":"李娇","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233956,"comment":null,"dependVersionId":0,"contentType":3,"content":"","pics":null,"card":{"cardItems":[{"key":"网站","value":"geneontology.org"}]},"references":[],"versionCount":0},"categories":[{"id":4,"name":"生物学","parents":[]}],"creator":{"uid":76032351,"name":"李娇","pic":"https://web.archive.org/web/20221025115444/https://cache.soso.com/qlogo/g?b=oidb&k=1gNdX2634ojFfltPoicTyCQ&s=100&t=1583237397","introduction":null,"educations":[{"schoolName":"中国科学技术大学","major":"","degree":"硕士","universityId":66,"universityLogo":"https://web.archive.org/web/20221025115444/https://img01.sogoucdn.com/app/a/200943/d76be9ee-6011-11e9-bbbc-fc4dd4f70029","majorLevel1":"理学","majorLevel2":"生物学","majorLevel3":"生物化学与分子生物学","majorLevel1Id":1,"majorLevel2Id":189,"majorLevel3Id":218,"state":null,"lab":null,"researchField":null}],"jobs":null,"works":null,"educationBrief":"中国科学技术大学","jobBrief":"","role":0,"roleName":null,"title":"中国科学技术大学 · 生物学硕士","professionalTitle":null,"phoneNo":null,"editable":true,"partnerId":58,"partnerIdCreateTime":1594286506,"partnerIdPoped":true},"createTime":1568626339,"editor":{"uid":76032351,"name":"李娇","pic":"https://web.archive.org/web/20221025115444/https://cache.soso.com/qlogo/g?b=oidb&k=1gNdX2634ojFfltPoicTyCQ&s=100&t=1583237397","introduction":null,"educations":[{"schoolName":"中国科学技术大学","major":"","degree":"硕士","universityId":66,"universityLogo":"https://web.archive.org/web/20221025115444/https://img01.sogoucdn.com/app/a/200943/d76be9ee-6011-11e9-bbbc-fc4dd4f70029","majorLevel1":"理学","majorLevel2":"生物学","majorLevel3":"生物化学与分子生物学","majorLevel1Id":1,"majorLevel2Id":189,"majorLevel3Id":218,"state":null,"lab":null,"researchField":null}],"jobs":null,"works":null,"educationBrief":"中国科学技术大学","jobBrief":"","role":0,"roleName":null,"title":"中国科学技术大学 · 生物学硕士","professionalTitle":null,"phoneNo":null,"editable":true,"partnerId":58,"partnerIdCreateTime":1594286506,"partnerIdPoped":true},"editTime":1576233956,"state":1,"versionCount":1,"upNum":0,"downNum":0,"pics":[{"originalUrl":"https://web.archive.org/web/20221025115444/https://img03.sogoucdn.com/app/a/200698/650_390_20200909153537-1183588068.jpg?width=227&height=174&titlename=&w=650&h=390","url":"https://web.archive.org/web/20221025115444/https://img03.sogoucdn.com/app/a/200698/650_390_20200909153537-1183588068.jpg","rw":650,"rh":390,"title":"","alt":null,"width":227,"height":174}],"catalogs":[{"level":1,"title":"语义和本体论","paragraphId":"14995403367776522","subCatalogs":[{"level":2,"title":"示例<b>语义</b>","paragraphId":"14995403367776522","subCatalogs":null}]},{"level":1,"title":"注释","paragraphId":"14995403384553742","subCatalogs":[{"level":2,"title":"<b>示例注释</b>","paragraphId":"14995403384553742","subCatalogs":null}]},{"level":1,"title":"工具","paragraphId":"14995403384553743","subCatalogs":null},{"level":1,"title":"联合会","paragraphId":"14995403384553744","subCatalogs":null},{"level":1,"title":"历史","paragraphId":"14995403384553745","subCatalogs":null},{"level":1,"title":"参考文献","paragraphId":"-1","subCatalogs":null}],"paragraphs":[{"paragraphId":"14995403367776522","title":"语义和本体论","versionId":"54259122140406030","lemmaId":10555,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82812762,"name":"雷克斯班纳","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599636957,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>从实际的角度来看，本体是对我们所知道的事物的一种描述。“本体”由可检测或可直接观察的事物的描述以及这些事物之间的关系组成。生物学和相关领域没有通用的标准术语，术语的用法可能特定于某个物种、研究领域甚至特定的研究群体，这使得数据的交流和共享更加困难。基因本体项目提供了描述基因产物特性的已定义语义的本体，本体涵盖三个领域：</p><p>\n <ul>\n  <li><b>细胞组分</b>，细胞的每个部分或其细胞外环境；</li>\n  <li><b>分子功能</b>，基因产物在分子水平上的基本活性，如结合或催化；</li>\n  <li><b>生物过程</b>，与细胞、组织、器官和有机体等完整生命单元的功能相关的、具有明确起点和终点的操作或一组分子事件。</li>\n </ul></p><p>本体中的每个GO术语都有一个语义名称，可以是一个单词或一串单词、一个独特的字母数字标识符、有引述来源的定义，和指示其所属域的命名空间。语义也可能有同义词，语义的同义词被归类为与语义名称完全等同、更宽、更窄或相关；引用其他数据库中的等效概念；以及对语义含义或用法的评论。GO本体被构造成一个有向无环图，每个语义都定义了与同一个领域中的一个或多个其他语义的关系，有时还定义了与其他领域语义的关系。GO词汇设计为物种中性，包括适用于原核生物和真核生物、单细胞和多细胞生物的语义。</p><p>GO系统不是静态的，研究和注释团体的成员以及直接参与了GO项目的人对GO系统的增加、更正和修改提出了建议并征求了意见。<sup><a href=\"#quote_5\" class=\"kx_ref\">[5]</a></sup> 例如，注释者可以请求一个特定语义来表示代谢途径，或者可以在团体专家的帮助下修改本体的一部分<sup><a href=\"#quote_6\" class=\"kx_ref\">[6]</a></sup>。被建议的编辑文本由本体修改器审查，合适后就能成功被录用。</p><p>GO本体文件可以从GO网站<sup><a href=\"#quote_7\" class=\"kx_ref\">[7]</a></sup> 以多种格式免费获得，或者可以使用GO浏览器——AmiGO在线访问。基因本体项目还提供了其语义到其他分类系统的可下载映射。</p><p><h3>示例<b>语义</b></h3></p><p>\n <dl>\n  <dd>\n   Id：GO:0000016\n  </dd>\n  <dd>\n   名称：乳糖酶活性\n  </dd>\n  <dd>\n   命名空间：分子功能\n  </dd>\n  <dd>\n   默认定义：“催化反应：C <sub>12</sub>H <sub>22</sub>O <sub>11</sub>（乳糖）+H <sub>2</sub>O =C <sub>6</sub>H <sub>12</sub>O <sub>6</sub>（D-葡萄糖）+C <sub>6</sub>H <sub>12</sub>O <sub>6</sub>（D-半乳糖）。”\n  </dd>\n  <dd>\n   同义词：“乳糖酶-根皮苷水解酶活性”\n  </dd>\n  <dd>\n   同义词：“乳糖半乳糖水解酶活性”\n  </dd>\n  <dd>\n   相互参照：EC:3.2.1.108\n  </dd>\n  <dd>\n   相互参照：MetaCyc:LACTASE-RXN\n  </dd>\n  <dd>\n   相互参照：Reactome:20536\n  </dd>\n  <dd>\n   <b>is_a：</b>GO:0004553！水解酶活性，水解O-糖基化合物\n  </dd>\n </dl></p><p>数据来源:<sup><a href=\"#quote_8\" class=\"kx_ref\">[8]</a></sup> </p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14995403384553742","title":"注释","versionId":"54259122140406031","lemmaId":10555,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82812762,"name":"雷克斯班纳","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599636957,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>基因组注释是捕获基因产物数据的实践，GO注释使用GO本体中的语义来实现。GO联合会的成员在GO网站上提交他们的注释以便整合和传播，在GO网站上他们可以直接下载或使用AmiGO在线查看。除了基因产物标识符和相关的GO语义之外，GO注释还有以下数据：用于进行注释的引用（例如期刊文章）；表示注释所基于的证据类型的证据代码；注释的日期和创建者。</p><p>证据代码来自包含手动和自动注释方法的一组代码控制字集。例如，可追踪作者声明（TAS）意味着管理者已经阅读了一篇已发表的科学论文，并且该注释的元数据带有对该论文的引用；从序列相似性推断（ISS）意味着管理者已经审查了序列相似性搜索的输出，并验证了其生物学意义。来自自动化过程的注释（例如，使用另一个注释字集创建的重新映射注释）被赋予从电子注释推断（IEA）的代码。截至2010年4月1日，98%以上的GO注释是通过计算推断的，而不是人为推断的。<sup><a href=\"#quote_9\" class=\"kx_ref\">[9]</a></sup>由于这些注释没有经过人员检查，GO联合会认为它们不太可靠，其在AmiGO的在线数据中只包括一个子集。完整的注释数据集可以从GO网站下载。为了支持注释的开发，GO联合会为新的开发者群体提供学习营地和导师。</p><p>最近，许多机器学习算法已经被设计并实现基因本体注释的预测。<sup><a href=\"#quote_10\" class=\"kx_ref\">[10]</a></sup><sup><a href=\"#quote_11\" class=\"kx_ref\">[11]</a></sup> </p><p><h3><b>示例注释</b></h3></p><p>\n <dl>\n  <dd>\n   基因产物：肌动蛋白，α心肌1， UniProtKB:P68032\n  </dd>\n  <dd>\n   GO语义：心脏收缩 ; GO:0060047 (生物过程)\n  </dd>\n  <dd>\n   证据代码：从突变表型推断（IMP）\n  </dd>\n  <dd>\n   参考: PMID 17611253 PubMed\n  </dd>\n  <dd>\n   分配: UniProtKB，2008年6月6日\n  </dd>\n </dl></p><p>数据来源:<sup><a href=\"#quote_12\" class=\"kx_ref\">[12]</a></sup> </p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14995403384553743","title":"工具","versionId":"54259122140406032","lemmaId":10555,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82812762,"name":"雷克斯班纳","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599636957,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>有大量可用的在线工具和下载工具<sup><a href=\"#quote_13\" class=\"kx_ref\">[13]</a></sup> 可以使用GO项目提供的数据。这些软件的绝大多数都来自第三方；GO联合会开发和支持两种工具，<b>AmiGO</b>和<b>OBO-Edit</b>。</p><p>AmiGO<sup><a href=\"#quote_14\" class=\"kx_ref\">[14]</a></sup><sup><a href=\"#quote_15\" class=\"kx_ref\">[15]</a></sup> 是一个基于网络的应用程序，允许用户查询、浏览和可视化本体和基因产物的注释数据。它还有一个BLAST工具<sup><a href=\"#quote_15\" class=\"kx_ref\">[15]</a></sup> （此工具允许分析更大的数据集）<sup><a href=\"#quote_16\" class=\"kx_ref\">[16]</a></sup><sup><a href=\"#quote_17\" class=\"kx_ref\">[17]</a></sup> ，以及一个直接查询GO数据库的界面。<sup><a href=\"#quote_18\" class=\"kx_ref\">[18]</a></sup> </p><p>AmiGO可以在GO网站上在线使用，以访问GO联合会提供的数据，也可以在任何使用GO数据库架构的数据库中下载并安装以供本地使用<sup><a href=\"#quote_19\" class=\"kx_ref\">[19]</a></sup>。它是免费的开源软件，可以作为go-dev软件发行版的一部分。<sup><a href=\"#quote_20\" class=\"kx_ref\">[20]</a></sup> </p><p>OBO-Edit<sup><a href=\"#quote_21\" class=\"kx_ref\">[21]</a></sup> 是一个开源、独立于平台的本体修改器，由基因本体联合会开发和维护。它在Java中实现，并使用面向图形的方法来显示和编辑本体。OBO-Edit包括一个全面的搜索和过滤界面，可以选择呈现术语子集，使它们在视觉上不同；用户界面也可以根据用户偏好定制。OBO-Edit还有一个推理器，可以根据现有的关系和它们的属性推断出没有明确说明的联系。虽然它是为生物医学本体开发的，但是OBO-Edit可以用来查看、搜索和编辑任何本体。它可以免费下载。<sup><a href=\"#quote_20\" class=\"kx_ref\">[20]</a></sup> </p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14995403384553744","title":"联合会","versionId":"54259122140406033","lemmaId":10555,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82812762,"name":"雷克斯班纳","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599636957,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>基因本体联合会（Gene Ontology Consortium）是积极参与基因本体项目的一组生物数据库和研究团体。<sup><a href=\"#quote_22\" class=\"kx_ref\">[22]</a></sup> 这包括许多模式生物数据库和多物种蛋白质数据库、软件开发小组和专门的编辑部。</p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14995403384553745","title":"历史","versionId":"54259122140406034","lemmaId":10555,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82812762,"name":"雷克斯班纳","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599636957,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>基因本体最初是在1998年由一群研究三种模式生物基因组的研究人员构建的：黑腹果蝇（果蝇）、小家鼠（小鼠）和酿酒酵母（啤酒酵母或面包酵母）。<sup><a href=\"#quote_23\" class=\"kx_ref\">[23]</a></sup> 许多其他模式生物数据库加入了基因本体联合会，不仅贡献了注释数据，还有助于开发用于查看和应用数据的本体和工具。迄今为止，植物、动物和微生物领域的大部分主要数据库都为这个项目做出了贡献。截止2008年1月，GO包含超过24500个广泛适用于多种生物有机体的语义。关于GO的开发和使用有着大量的文献，GO已经成为生物信息学库中的一个标准工具。它们的目标有三个方面：建立基因本体，为基因/基因产物分配本体，为前两个对象开发软件和数据库。</p><p>利用类的形式、域无关属性（元属性）对基因本体的一些分析也开始出现。例如，可以看到对生物本体论的本体论分析。<sup><a href=\"#quote_24\" class=\"kx_ref\">[24]</a></sup> </p>","pics":null,"card":null,"references":[],"versionCount":0}],"references":[{"id":1,"type":"book","title":"The Gene Ontology Consortium (January 2008). \"The Gene Ontology project in 2008\". Nucleic Acids Research. 36 (Database issue): D440–4. doi:10.1093/nar/gkm883. PMC 2238979. PMID 17984083.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":2,"type":"book","title":"Dessimoz, Christophe; Škunca, Nives, eds. (2017). The Gene Ontology Handbook. 1446. doi:10.1007/978-1-4939-3743-1. ISBN 9781493937431. ISSN 1064-3745.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":3,"type":"book","title":"Gaudet, Pascale; Škunca, Nives; Hu, James C.; Dessimoz, Christophe (2017). \"Primer on the Gene Ontology\". 1446: 25–37. doi:10.1007/978-1-4939-3743-1_3. ISSN 1064-3745.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":4,"type":"book","title":"Smith B, Ashburner M, Rosse C, Bard J, Bug W, Ceusters W, Goldberg LJ, Eilbeck K, Ireland A, Mungall CJ, Leontis N, Rocca-Serra P, Ruttenberg A, Sansone SA, Scheuermann RH, Shah N, Whetzel PL, Lewis S (November 2007). \"The OBO Foundry: coordinated evolution of ontologies to support biomedical data integration\". Nature Biotechnology. 25 (11): 1251–5. doi:10.1038/nbt1346. PMC 2814061. PMID 17989687.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":5,"type":"book","title":"Lovering, Ruth C. (2017). \"How Does the Scientific Community Contribute to Gene Ontology?\". In Dessimoz, C; Skunca, N. The Gene Ontology Handbook. Methods in Molecular Biology (in English). 1446. Springer (New York). pp. 85–93. doi:10.1007/978-1-4939-3743-1_7. ISSN 1064-3745.CS1 maint: Unrecognized language (link)","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":6,"type":"book","title":"Diehl AD, Lee JA, Scheuermann RH, Blake JA (April 2007). \"Ontology development for biological systems: immunology\". Bioinformatics. 23 (7): 913–5. doi:10.1093/bioinformatics/btm029. PMID 17267433.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":7,"type":"book","title":"\"Gene Ontology Database\". Gene Ontology Consortium.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":8,"type":"book","title":"The GO Consortium (2009-03-16). \"gene_ontology.1_2.obo\" (OBO 1.2 flat file).","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":9,"type":"book","title":"du Plessis L, Skunca N, Dessimoz C (November 2011). \"The what, where, how and why of gene ontology--a primer for bioinformaticians\". Briefings in Bioinformatics. 12 (6): 723–35. doi:10.1093/bib/bbr002. PMC 3220872. PMID 21330331.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":10,"type":"book","title":"Pinoli P, Chicco D, Masseroli M (June 2013). \"Computational algorithms to predict Gene Ontology annotation\". BMC Bioinformatics. 16 (6): 1–15. doi:10.1186/1471-2105-16-S6-S4. PMC 4416163. PMID 25916950.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":11,"type":"book","title":"Cozzetto, Domenico; Jones, David T. (2017). \"Computational Methods for Annotation Transfers from Sequence\". In Dessimoz, C; Skunca, N. The Gene Ontology Handbook. Methods in Molecular Biology (in English). 1446. Springer (New York). pp. 55–67. doi:10.1007/978-1-4939-3743-1_5. ISSN 1064-3745.CS1 maint: Unrecognized language (link)","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":12,"type":"book","title":"The GO Consortium (2009-03-16). \"AmiGO: P68032 Associations\".","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":13,"type":"book","title":"Mosquera JL, Sánchez-Pla A (July 2008). \"SerbGO: searching for the best GO tool\". Nucleic Acids Research. 36 (Web Server issue): W368–71. doi:10.1093/nar/gkn256. PMC 2447766. PMID 18480123.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":14,"type":"book","title":"Carbon S, Ireland A, Mungall CJ, Shu S, Marshall B, Lewis S (January 2009). AmiGO Hub; Web Presence Working Group. \"AmiGO: online access to ontology and annotation data\". Bioinformatics. 25 (2): 288–9. doi:10.1093/bioinformatics/btn615. PMC 2639003. PMID 19033274.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":15,"type":"book","title":"AmiGO--the current official web-based set of tools for searching and browsing the Gene Ontology database","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":16,"type":"book","title":"AmiGO Term Enrichment tool; finds significant shared GO terms in an annotation set","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":17,"type":"book","title":"AmiGO Slimmer; maps granular annotations up to high-level terms","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":18,"type":"book","title":"GOOSE, GO Online SQL Environment; allows direct SQL querying of the GO database","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":19,"type":"book","title":"The Plant Ontology Consortium (2009-03-16). \"Plant Ontology Consortium\". Retrieved 2009-03-16.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":20,"type":"book","title":"\"Gene Ontology downloads at SourceForge\". Retrieved 2009-03-16.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":21,"type":"book","title":"Day-Richter J, Harris MA, Haendel M, Lewis S (August 2007). \"OBO-Edit--an ontology editor for biologists\". Bioinformatics. 23 (16): 2198–200. doi:10.1093/bioinformatics/btm112. PMID 17545183.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":22,"type":"book","title":"\"The GO Consortium\". Retrieved 2009-03-16.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":23,"type":"book","title":"Ashburner M, Ball CA, Blake JA, Botstein D, Butler H, Cherry JM, Davis AP, Dolinski K, Dwight SS, Eppig JT, Harris MA, Hill DP, Issel-Tarver L, Kasarskis A, Lewis S, Matese JC, Richardson JE, Ringwald M, Rubin GM, Sherlock G (May 2000). \"Gene ontology: tool for the unification of biology. The Gene Ontology Consortium\". Nature Genetics. 25 (1): 25–9. doi:10.1038/75556. PMC 3037419. PMID 10802651.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":24,"type":"book","title":"Deb, B. (2012). \"An ontological analysis of some biological ontologies\". Frontiers in Genetics. 3. doi:10.3389/fgene.2012.00269. PMC 3509948.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false}],"recommendReferences":null,"auditState":2,"lemmaLevel":1,"origin":0,"originEnTitle":null,"originZhTitle":null,"pv":1025,"auditType":0,"synonyms":["Gene Ontology"],"showEditTime":"2019.12.13 18:45","auditors":[{"uid":0,"name":"Ki.κe","pic":"https://web.archive.org/web/20221025115444/https://wx.qlogo.cn/mmopen/vi_32/y67kfr32Doib4wg71Jiau7jVWvharic3nRKgdRRQSl6koeQJCo0GQs2Krw0vwdFRsOWnHIQOwAZsSg5lIkIrFCOcQ/132","introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":"","jobBrief":"","role":0,"roleName":null,"title":"","professionalTitle":null,"phoneNo":null,"editable":true,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false}],"hasZhishiNav":false,"auditInfos":{},"isHistory":false};</script><script crossorigin="anonymous" src="./555.基因本体论 - 搜狗科学百科_files/aegis.min.js.download"></script><script crossorigin="anonymous" src="./555.基因本体论 - 搜狗科学百科_files/main_2020092401.js.download"></script><script crossorigin="anonymous" src="./555.基因本体论 - 搜狗科学百科_files/react.production.min.js.download"></script><script crossorigin="anonymous" src="./555.基因本体论 - 搜狗科学百科_files/react-dom.production.min.js.download"></script><script crossorigin="anonymous" src="./555.基因本体论 - 搜狗科学百科_files/jquery-1.11.1.min.js.download"></script><script crossorigin="anonymous" src="./555.基因本体论 - 搜狗科学百科_files/main_2022062701.js.download"></script><script crossorigin="anonymous" src="./555.基因本体论 - 搜狗科学百科_files/main_66bbe21.js.download"></script><script crossorigin="anonymous" src="./555.基因本体论 - 搜狗科学百科_files/react.production.min.js.download"></script><script crossorigin="anonymous" src="./555.基因本体论 - 搜狗科学百科_files/react-dom.production.min.js.download"></script><script crossorigin="anonymous" src="./555.基因本体论 - 搜狗科学百科_files/main_edf0f08.js.download"></script><div id="popControl"><div class="pop-control-mask" style="display: none;"><div class="pop-container"><i class="close-icon"></i><div class="stop-title">搜狗科学百科停止服务运营公告</div><div class="stop-user">亲爱的用户：</div><div class="stop-content">因业务方向调整，搜狗科学百科将于2022年11月11日正式停止服务与运营。届时产品内所有数据将依据相关法律进行删除并关闭服务器，您将无法登录及使用搜狗科学百科，相关内容在平台下线后将无法找回。</div><div class="stop-content">搜狗科学百科是由搜狗推出的科学领域专业百科产品，与搜狗百科为两个独立产品，本次停止服务与运营仅对搜狗科学百科，并不影响搜狗百科产品正常使用，敬请知悉。</div><div class="stop-content">由此给您带来的不便我们深表歉意，敬请谅解。再次感谢您一直以来的陪伴与支持！</div><div class="stop-team">搜狗科学百科产品&amp;运营团队</div><div class="stop-time">2022年10月11日</div></div></div></div>
</body></html>