<!DOCTYPE html>
<!-- saved from url=(0083)https://web.archive.org/web/20221025115646/https://baike.sogou.com/kexue/d11029.htm -->
<html class="" data-reactroot=""><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><script src="./1029.染料测序 - 搜狗科学百科_files/analytics.js.download" type="text/javascript"></script>
<script type="text/javascript">window.addEventListener('DOMContentLoaded',function(){var v=archive_analytics.values;v.service='wb';v.server_name='wwwb-app228.us.archive.org';v.server_ms=211;archive_analytics.send_pageview({});});</script>
<script type="text/javascript" src="./1029.染料测序 - 搜狗科学百科_files/bundle-playback.js.download" charset="utf-8"></script>
<script type="text/javascript" src="./1029.染料测序 - 搜狗科学百科_files/wombat.js.download" charset="utf-8"></script>
<script type="text/javascript">
  __wm.init("https://web.archive.org/web");
  __wm.wombat("https://baike.sogou.com/kexue/d11029.htm","20221025115646","https://web.archive.org/","web","/_static/",
	      "1666699006");
</script>
<link rel="stylesheet" type="text/css" href="./1029.染料测序 - 搜狗科学百科_files/banner-styles.css">
<link rel="stylesheet" type="text/css" href="./1029.染料测序 - 搜狗科学百科_files/iconochive.css">
<!-- End Wayback Rewrite JS Include -->
<meta name="save" content="history"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="baidu-site-verification" content="VWGb6TyYx8"><meta content="染料测序 - 搜狗科学百科" name="keywords"><meta content="搜狗科学百科是一部有着平等、协作、分享、自由理念的网络科学全书，为每一个互联网用户创造一个涵盖所有领域知识、服务的中文知识性平台。" name="description"><meta http-equiv="x-dns-prefetch-control" content="on"><meta name="server" baike="235" ip="210" env="online"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025115646/https://cache.soso.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025115646/https://hhy.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025115646/https://pic.baike.soso.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025115646/https://ugc.qpic.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025115646/https://xui.ptlogin2.qq.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025115646/https://q1.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025115646/https://q2.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025115646/https://q3.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025115646/https://q4.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025115646/https://q.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025115646/https://img01.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025115646/https://img02.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025115646/https://img03.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025115646/https://img04.sogoucdn.com/"><link rel="Shortcut Icon" href="https://web.archive.org/web/20221025115646im_/https://www.sogou.com/images/logo/new/favicon.ico?v=4"><link rel="Bookmark" href="https://www.sogou.com/images/logo/new/favicon.ico?v=4"><link href="./1029.染料测序 - 搜狗科学百科_files/base_b849887.css" rel="stylesheet"><link href="./1029.染料测序 - 搜狗科学百科_files/detail_378aed5.css" rel="stylesheet"><link href="./1029.染料测序 - 搜狗科学百科_files/inviteAudit_7894507.css" rel="stylesheet"><link rel="stylesheet" href="./1029.染料测序 - 搜狗科学百科_files/highlight.min.css"><title>染料测序 - 搜狗科学百科</title><style>.onekey-close {
	position: absolute;
	top: 16px;
	right: 16px;
	width: 24px;
	height: 24px;
	background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	text-indent: -999em;
	background-size: 84px;
	background-position: -63px 0;
}

.onekey-login {
	position: absolute;
	top: 16.4%;
	left: 0;
	right: 0;
	width: 100%;
}

/* .onekey-login-img {
    width: 75px;
    height: 75px;
    background: url("https://web.archive.org/web/20221025115646/https://baike.sogou.com/kexue/images/sprite_wap_baike.png") no-repeat;
    background-size: 100px 91px;
    background-position: 0 0;
    background-repeat: no-repeat;
    margin: 0 auto;
} */

.onekey-login-title {
	text-align: center;
	padding-bottom: 3px;
	font-size: 21px;
	font-weight: bold;
	line-height: 30px;
	color: #000;
}

.onekey-login-txt {
	text-align: center;
	font-family: PingFangSC;
	font-size: 14px;
	line-height: 20px;
	color: #8f8f8f;
}

.onekey-login-qq,
.onekey-login-wx,
.onekey-login-phone {
	display: block;
	width: 245px;
	height: 54px;
	border-radius: 45px;
	text-align: center;

	margin: 0 auto;
	font-size: 17px;
	line-height: 24px;
	color: #000;
	/* padding: 16px 77px; */
	border-radius: 12px;
	border: solid 1px #e0e0e0;
}
.onekey-qq-content,
.onekey-vx-content,
.onekey-phone-content {
	display: inline-block;
	margin-top: 16px;
}
.onekey-qq-content {
	padding: 0 5px;
}

.onekey-login-qq {
	margin-top: 48px;
	margin-bottom: 24px;
}

.onekey-login-qq:before {
	display: inline-block;
	content: "";
	width: 20px;
	height: 20px;
	background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	background-size: 80px;
	background-position: -20px 0;
	vertical-align: top;
	margin: 17px 8px 0 0;
}

.onekey-login-wx {
	margin-bottom: 24px;
}

.onekey-login-wx:before {
	display: inline-block;
	content: "";
	width: 21px;
	height: 21px;
	background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	background-size: 84px;
	background-position: 0 0;
	vertical-align: top;
	margin: 17px 10px 0 0;
}

.onekey-login-phone {
}

.onekey-login-phone:before {
	display: inline-block;
	content: "";
	width: 21px;
	height: 21px;
	background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	background-size: 84px;
	background-position: -42px 0;
	vertical-align: top;
	margin: 17px 10px 0 0;
}

.onekey-fixed {
	z-index: 100;
	position: fixed;
	top: 0;
	bottom: 0;
	left: 0;
	right: 0;
	background: #fff;
	width: 100%;
	height: 100%;
}

.onekey-fixed.forbid {
	z-index: 100;
	position: fixed;
	top: auto;
	bottom: 68px;
	left: 9%;
	right: 0;
	background: rgba(0, 0, 0, 0.7);
	width: 82%;
	height: 43px;
	border-radius: 25px;
	color: #ffffff;
}
.onekey-login-title.forbid {
	text-align: center;
	padding-bottom: 3px;
	font-size: 14px;
	font-weight: normal;
	line-height: 30px;
	color: white;
}
</style><style>#login_mask {
  background: #000;
  opacity: 0.5;
  filter: alpha(opacity=50);
  position: fixed;
  /*fixed好像在哪个IE上有BUG，先用用*/
  left: 0;
  top: 0;
  z-index: 999;
  height: 100%;
}

#login_iframe_container {
  position: fixed;
  width: 550px;
  height: 360px;
  z-index: 1020;
  background-color: #ffffff;
}

@media screen and (max-width: 828px) {
  #login_iframe_container {
    top: 50% !important;
    left: 50% !important;
    transform: translate(-50%, -50%);
  }
}

#login_iframe_container.new-login {
  width: 550px;
  height: 360px;
  background-image: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/background_2a4a8a6.png);
}

#login_iframe_container.new-login.no-bg {
  background: #fff;
}

#login_iframe_container.new-login .login-title {
  width: 100%;
  height: 42px;
  line-height: 42px;
  text-align: center;
  font-size: 30px;
  letter-spacing: 0.19px;
  color: #ffffff;
  margin-top: 62px;
}
#login_iframe_container.new-login .forbid-title {
  width: 100%;
  height: 42px;
  line-height: 42px;
  text-align: center;
  font-size: 24px;
  letter-spacing: 0.19px;
  color: #333333;
  margin-top: 150px;
}

#login_iframe_container.new-login.no-bg .login-title {
  color: #333333;
}

#login_iframe_container.new-login .login-subtitle {
  width: 100%;
  height: 18px;
  line-height: 18px;
  font-size: 13px;
  letter-spacing: 0.08px;
  color: #ffffff;
  text-align: center;
  margin-top: 9px;
  margin-bottom: 43px;
}

#login_iframe_container.new-login.no-bg .login-subtitle {
  color: #999999;
}

#login_iframe_container.new-login .login-subtitle::before {
  content: '';
  display: inline-block;
  width: 10px;
  height: 1px;
  background-color: #ffffff;
  position: relative;
  top: -4px;
  left: -5px;
}

#login_iframe_container.new-login .login-subtitle::after {
  content: '';
  display: inline-block;
  width: 10px;
  height: 1px;
  background-color: #ffffff;
  position: relative;
  top: -4px;
  left: 5px;
}

#login_iframe_container.new-login.no-bg .login-subtitle::before {
  background-color: #999999;
}

#login_iframe_container.new-login.no-bg .login-subtitle::after {
  background-color: #999999;
}

#login_iframe_container.new-login .close-btn {
  position: absolute;
  top: 20px;
  right: 20px;
  width: 12px;
  height: 12px;
  background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/login-sprites_e3853e5.png) -59px -10px;
  background-size: 81px 91px;
  cursor: pointer;
}

#login_iframe_container.new-login .login-btn {
  width: 220px;
  height: 47px;
  border-radius: 24px;
  border: solid 1px #dddddd;
  background-color: #ffffff;
  margin: 0 auto;
  margin-top: 28px;
  position: relative;
  display: block;
}

#login_iframe_container.new-login .login-btn .login-icon {
  position: absolute;
}

#login_iframe_container.new-login .login-btn .login-text {
  width: 61px;
  height: 47px;
  line-height: 47px;
  vertical-align: middle;
  font-size: 15px;
  letter-spacing: 0.1px;
  color: #666666;
  position: absolute;
  right: 62px;
}

#login_iframe_container.new-login .login-btn.qq-btn .login-icon {
  width: 22px;
  height: 27px;
  top: 10px;
  left: 67px;
  background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/login-sprites_e3853e5.png) -10px -54px;
  background-size: 81px 91px;
}

#login_iframe_container.new-login .login-btn.qq-btn .login-text {
  right: 59px;
}

#login_iframe_container.new-login .login-btn.wechat-btn .login-icon {
  width: 29px;
  height: 24px;
  top: 12px;
  left: 62px;
  background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/login-sprites_e3853e5.png) -10px -10px;
  background-size: 81px 91px;
}</style><style>/* -- container -- */
.rodal,
.rodal-mask {
    top: 0;
    left: 0;
    width: 100%;
    height: 100%;
    z-index: 100;
}

.rodal {
    position: fixed;
}

/* -- mask -- */
.rodal-mask {
    position: fixed;
    background: rgba(0, 0, 0, .5);
}

/* -- dialog -- */
.rodal-dialog {
    position: absolute;
    z-index: 101;
    background: #fff;
    border-radius: 3px;
    box-shadow: 0 1px 3px rgba(0, 0, 0, .2);
}

.rodal-center {
    top: 50%;
    transform: translateY(-50%);
    left: 0;
    right: 0;
    margin: 0 auto;
}

.rodal-bottom {
    left: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

.rodal-top {
    left: 0;
    right: 0;
    top: 0;
    margin: auto;
}

.rodal-left {
    top: 0;
    left: 0;
    bottom: 0;
    margin: auto;
}

.rodal-right {
    top: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

/* -- close button -- */
.rodal-close {
    position: absolute;
    cursor: pointer;
    top: 16px;
    right: 16px;
    width: 16px;
    height: 16px;
}

.rodal-close:before,
.rodal-close:after {
    position: absolute;
    content: '';
    height: 2px;
    width: 100%;
    top: 50%;
    left: 0;
    margin-top: -1px;
    background: #999;
    border-radius: 100%;
    -webkit-transition: background .2s;
    transition: background .2s;
}

.rodal-close:before {
    -webkit-transform: rotate(45deg);
    transform: rotate(45deg);
}

.rodal-close:after {
    -webkit-transform: rotate(-45deg);
    transform: rotate(-45deg);
}

.rodal-close:hover:before,
.rodal-close:hover:after {
    background: #333;
}

/* -- fade -- */
/* @-webkit-keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

@keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

.rodal-fade-enter {
    -webkit-animation: rodal-fade-enter both ease-in;
    animation: rodal-fade-enter both ease-in;
} */

@-webkit-keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

@keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

.rodal-fade-leave {
    -webkit-animation: rodal-fade-leave both ease-out;
    animation: rodal-fade-leave both ease-out;
}

/* -- zoom -- */
@-webkit-keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-enter {
    -webkit-animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-leave {
    -webkit-animation: rodal-zoom-leave both;
    animation: rodal-zoom-leave both;
}

/* -- slideDown -- */
@-webkit-keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-enter {
    -webkit-animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-leave {
    -webkit-animation: rodal-slideDown-leave both;
    animation: rodal-slideDown-leave both;
}

/* -- slideLeft -- */
@-webkit-keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-enter {
    -webkit-animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-leave {
    -webkit-animation: rodal-slideLeft-leave both;
    animation: rodal-slideLeft-leave both;
}

/* -- slideRight -- */
@-webkit-keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-enter {
    -webkit-animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-leave {
    -webkit-animation: rodal-slideRight-leave both;
    animation: rodal-slideRight-leave both;
}

/* -- slideUp -- */
@-webkit-keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-enter {
    -webkit-animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
    animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
}

@-webkit-keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-leave {
    -webkit-animation: rodal-slideUp-leave both;
    animation: rodal-slideUp-leave both;
}

/* -- flip -- */
@-webkit-keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

@keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

.rodal-flip-enter {
    -webkit-animation: rodal-flip-enter both ease-in;
    animation: rodal-flip-enter both ease-in;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

@-webkit-keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

@keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

.rodal-flip-leave {
    -webkit-animation: rodal-flip-leave both;
    animation: rodal-flip-leave both;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

/* -- rotate -- */
@-webkit-keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-enter {
    -webkit-animation: rodal-rotate-enter both;
    animation: rodal-rotate-enter both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

@-webkit-keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-leave {
    -webkit-animation: rodal-rotate-leave both;
    animation: rodal-rotate-leave both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

/* -- door -- */
@-webkit-keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

@keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

.rodal-door-enter {
    -webkit-animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

@keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

.rodal-door-leave {
    -webkit-animation: rodal-door-leave both;
    animation: rodal-door-leave both;
}</style><style>/* -- container -- */
.rodal,
.rodal-mask {
    top: 0;
    left: 0;
    width: 100%;
    height: 100%;
    z-index: 100;
}

.rodal {
    position: fixed;
}

/* -- mask -- */
.rodal-mask {
    position: fixed;
    background: rgba(0, 0, 0, .5);
}

/* -- dialog -- */
.rodal-dialog {
    position: absolute;
    z-index: 101;
    background: #fff;
    border-radius: 3px;
    box-shadow: 0 1px 3px rgba(0, 0, 0, .2);
}

.rodal-center {
    top: 50%;
    transform: translateY(-50%);
    left: 0;
    right: 0;
    margin: 0 auto;
}

.rodal-bottom {
    left: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

.rodal-top {
    left: 0;
    right: 0;
    top: 0;
    margin: auto;
}

.rodal-left {
    top: 0;
    left: 0;
    bottom: 0;
    margin: auto;
}

.rodal-right {
    top: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

/* -- close button -- */
.rodal-close {
    position: absolute;
    cursor: pointer;
    top: 16px;
    right: 16px;
    width: 16px;
    height: 16px;
}

.rodal-close:before,
.rodal-close:after {
    position: absolute;
    content: '';
    height: 2px;
    width: 100%;
    top: 50%;
    left: 0;
    margin-top: -1px;
    background: #999;
    border-radius: 100%;
    -webkit-transition: background .2s;
    transition: background .2s;
}

.rodal-close:before {
    -webkit-transform: rotate(45deg);
    transform: rotate(45deg);
}

.rodal-close:after {
    -webkit-transform: rotate(-45deg);
    transform: rotate(-45deg);
}

.rodal-close:hover:before,
.rodal-close:hover:after {
    background: #333;
}

/* -- fade -- */
/* @-webkit-keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

@keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

.rodal-fade-enter {
    -webkit-animation: rodal-fade-enter both ease-in;
    animation: rodal-fade-enter both ease-in;
} */

@-webkit-keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

@keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

.rodal-fade-leave {
    -webkit-animation: rodal-fade-leave both ease-out;
    animation: rodal-fade-leave both ease-out;
}

/* -- zoom -- */
@-webkit-keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-enter {
    -webkit-animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-leave {
    -webkit-animation: rodal-zoom-leave both;
    animation: rodal-zoom-leave both;
}

/* -- slideDown -- */
@-webkit-keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-enter {
    -webkit-animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-leave {
    -webkit-animation: rodal-slideDown-leave both;
    animation: rodal-slideDown-leave both;
}

/* -- slideLeft -- */
@-webkit-keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-enter {
    -webkit-animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-leave {
    -webkit-animation: rodal-slideLeft-leave both;
    animation: rodal-slideLeft-leave both;
}

/* -- slideRight -- */
@-webkit-keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-enter {
    -webkit-animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-leave {
    -webkit-animation: rodal-slideRight-leave both;
    animation: rodal-slideRight-leave both;
}

/* -- slideUp -- */
@-webkit-keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-enter {
    -webkit-animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
    animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
}

@-webkit-keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-leave {
    -webkit-animation: rodal-slideUp-leave both;
    animation: rodal-slideUp-leave both;
}

/* -- flip -- */
@-webkit-keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

@keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

.rodal-flip-enter {
    -webkit-animation: rodal-flip-enter both ease-in;
    animation: rodal-flip-enter both ease-in;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

@-webkit-keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

@keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

.rodal-flip-leave {
    -webkit-animation: rodal-flip-leave both;
    animation: rodal-flip-leave both;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

/* -- rotate -- */
@-webkit-keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-enter {
    -webkit-animation: rodal-rotate-enter both;
    animation: rodal-rotate-enter both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

@-webkit-keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-leave {
    -webkit-animation: rodal-rotate-leave both;
    animation: rodal-rotate-leave both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

/* -- door -- */
@-webkit-keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

@keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

.rodal-door-enter {
    -webkit-animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

@keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

.rodal-door-leave {
    -webkit-animation: rodal-door-leave both;
    animation: rodal-door-leave both;
}</style></head><body class=""><!-- BEGIN WAYBACK TOOLBAR INSERT -->
<style type="text/css">
body {
  margin-top:0 !important;
  padding-top:0 !important;
  /*min-width:800px !important;*/
}
</style>
<script>__wm.rw(0);</script>
<div id="wm-ipp-base" lang="en" style="display: block; direction: ltr;">
</div><div id="wm-ipp-print">The Wayback Machine - https://web.archive.org/web/20221025115646/https://baike.sogou.com/kexue/d11029.htm</div>
<script type="text/javascript">//<![CDATA[
__wm.bt(675,27,25,2,"web","https://baike.sogou.com/kexue/d11029.htm","20221025115646",1996,"/_static/",["/_static/css/banner-styles.css?v=S1zqJCYt","/_static/css/iconochive.css?v=qtvMKcIJ"], false);
  __wm.rw(1);
//]]></script>
<!-- END WAYBACK TOOLBAR INSERT --><script>window._gtag=window._gtag||{};window._gtag.shouldGrayed = false;if ('0e6ca22c46bf43c4927f653640230e1b') window._gtag.traceId = '0e6ca22c46bf43c4927f653640230e1b';if ({"illegality":true}) window.userInfo = {"illegality":true};</script><div class="topnavbox"><ul class="topnav"><li><a href="https://web.archive.org/web/20221025115646/https://www.sogou.com/web?query=">网页</a></li><li><a href="https://web.archive.org/web/20221025115646/https://weixin.sogou.com/weixin?p=75351201">微信</a></li><li><a href="https://web.archive.org/web/20221025115646/https://zhihu.sogou.com/zhihu?p=75351218">知乎</a></li><li><a href="https://web.archive.org/web/20221025115646/https://pic.sogou.com/pics?query=">图片</a></li><li><a href="https://web.archive.org/web/20221025115646/https://v.sogou.com/v?query=">视频</a></li><li><a href="https://web.archive.org/web/20221025115646/https://mingyi.sogou.com/">医疗</a></li><li class="cur"><strong>科学</strong></li><li><a href="https://web.archive.org/web/20221025115646/https://hanyu.sogou.com/">汉语</a></li><li><a href="https://web.archive.org/web/20221025115646/https://wenwen.sogou.com/">问问</a></li><li><a href="https://web.archive.org/web/20221025115646/https://www.sogou.com/docs/more.htm">更多<span class="topraquo">»</span></a></li></ul></div><div id="header"><div class="header-wrap"><a class="header-logo" href="https://web.archive.org/web/20221025115646/https://baike.sogou.com/kexue"></a><div class="header-search"><div class="querybox" id="suggBox"><form><input id="searchInput" class="query" type="text" placeholder="搜科学领域专业百科词条" name="query" autocomplete="off" value=""><a href="javascript:;" class="query-search"></a></form></div></div><div class="header-rgt"><span class="btn-header-rgt btn-edit" id="editLemma">创建</span><div class="header-user no-login"></div></div></div></div><div class="fixed-placeholder" style="visibility:none"></div><div id="container" class=""><div class="content lemma-level1"><div class="detail-title" id="abstract-title"><h1>染料测序</h1><a href="https://web.archive.org/web/20221025115646/https://baike.sogou.com/kexue/d11029.htm#!" class="detail-edit">编辑</a></div><div class="section_content" data-id="54562946398871826"><div class="text_img ed_imgfloat_right"><a class="ed_image_link" data-src="https://img02.sogoucdn.com/app/a/200698/sogou_science_16222" data-bigsrc="https://img02.sogoucdn.com/app/a/200698/sogou_science_16222?width=231&amp;height=174&amp;titlename=%E8%84%B1%E6%B0%A7%E6%A0%B8%E7%B3%96%E6%A0%B8%E9%85%B8%E9%80%9A%E8%BF%87%E4%BA%92%E8%A1%A5%E5%BA%8F%E5%88%97%E9%99%84%E7%9D%80%E5%9C%A8%E6%B5%81%E5%8A%A8%E6%B1%A0%E4%B8%8A%E3%80%82%E8%AF%A5%E9%93%BE%E5%BC%AF%E6%9B%B2%E5%B9%B6%E8%BF%9E%E6%8E%A5%E5%88%B0%E7%AC%AC%E4%BA%8C%E5%AF%A1%E8%81%9A%E6%A0%B8%E8%8B%B7%E9%85%B8%E4%B8%8A%EF%BC%8C%E5%BD%A2%E6%88%90%E6%A1%A5%E3%80%82%E8%81%9A%E5%90%88%E9%85%B6%E5%90%88%E6%88%90%E5%8F%8D%E5%90%91%E9%93%BE%E3%80%82%E4%B8%A4%E6%9D%A1%E9%93%BE%E8%A7%A3%E9%93%BE%E5%B9%B6%E4%BC%B8%E7%9B%B4%E3%80%82%E6%AF%8F%E4%B8%80%E6%9D%A1%E9%93%BE%E9%83%BD%E5%BD%A2%E6%88%90%E4%B8%80%E4%B8%AA%E6%96%B0%E6%A1%A5(%E6%A1%A5%E5%BC%8F%E6%89%A9%E5%A2%9E)%E3%80%82%E5%85%B6%E7%BB%93%E6%9E%9C%E6%98%AF%E4%BA%A7%E7%94%9F%E4%B8%80%E7%B0%87DNA%E7%9A%84%E6%AD%A3%E9%93%BE%E5%92%8C%E5%8F%8D%E5%90%91%E4%BA%92%E8%A1%A5%E9%93%BE%E7%9A%84%E5%A4%8D%E5%88%B6%E3%80%82&amp;w=600&amp;h=480" title="点击查看大图" data-w="600" data-h="480" style="background-image:url(https://web.archive.org/web/20221025115646im_/https://img02.sogoucdn.com/app/a/200698/sogou_science_16222)" href="https://web.archive.org/web/20221025115646/https://baike.sogou.com/kexue/d11029.htm#!"></a><div class="text_img_title">脱氧核糖核酸通过互补序列附着在流动池上。该链弯曲并连接到第二寡聚核苷酸上，形成桥。聚合酶合成反向链。两条链解链并伸直。每一条链都形成一个新桥(桥式扩增)。其结果是产生一簇DNA的正链和反向互补链的复制。</div></div><div><p>Illumina染料测序是一种用于测定脱氧核糖核酸中一连串碱基对的技术，也称为脱氧核糖核酸（DNA）测序。可逆终止化学概念由巴黎巴斯德研究所的布鲁诺·卡纳德（Bruno Canard）和西蒙·萨法蒂（Simon Sarfati）提出。<sup><a href="https://web.archive.org/web/20221025115646/https://baike.sogou.com/kexue/d11029.htm#quote_1" class="kx_ref">[1]</a></sup><sup><a href="https://web.archive.org/web/20221025115646/https://baike.sogou.com/kexue/d11029.htm#quote_2" class="kx_ref">[2]</a></sup>后来剑桥大学的尚卡尔·巴拉苏布拉马尼安（Shankar Balasubramanian）和大卫·克莱恩曼（David Klenerman）开发了这一技术，<sup><a href="https://web.archive.org/web/20221025115646/https://baike.sogou.com/kexue/d11029.htm#quote_3" class="kx_ref">[3]</a></sup>他们随后创建了Solexa公司，后来被Illumina收购。这种测序方法基于可逆的染料--终止技术，当它们被引入到脱氧核糖核酸链中时，能够识别单个碱基。它也可用于全基因组和局部测序、转录组分析、宏基因组学、小核糖核酸发现、甲基化谱和全基因组蛋白质-核酸互作分析。<sup><a href="https://web.archive.org/web/20221025115646/https://baike.sogou.com/kexue/d11029.htm#quote_4" class="kx_ref">[4]</a></sup> </p></div></div><div id="catalog"><h2 class="title2">目录<a href="javascript:" class="detail-edit">编辑</a></h2><div class="catalog_wrap" style=""><ul class="catalog_list col2"><li><span class="order">1</span><a href="javascript:" data-level="1" data-id="14996256338215440">概述</a></li><li><span class="order">2</span><a href="javascript:" data-level="1" data-id="14996256354992649">程序</a></li><li class="secondary_catalog"><span>2.1 </span><a href="javascript:" data-id="14996256354992649">标签化</a></li><li class="secondary_catalog"><span>2.2 </span><a href="javascript:" data-id="14996256354992649">低循环扩增</a></li><li class="secondary_catalog"><span>2.3 </span><a href="javascript:" data-id="14996256354992649">桥式扩增</a></li><li class="secondary_catalog"><span>2.4 </span><a href="javascript:" data-id="14996256354992649">克隆扩增</a></li></ul><ul class="catalog_list col2"><li class="secondary_catalog"><span>2.5 </span><a href="javascript:" data-id="14996256354992649">边合成边测序</a></li><li class="secondary_catalog"><span>2.6 </span><a href="javascript:" data-id="14996256354992649">数据分析</a></li><li><span class="order">3</span><a href="javascript:" data-level="1" data-id="14996256354992650">与其他测序方法的比较</a></li><li><span class="order">4</span><a href="javascript:" data-level="1" data-id="14996256354992651">使用示例</a></li><li><span class="order">5</span><a href="javascript:" data-level="1" data-id="references">参考文献</a></li></ul></div></div><div id="paragraphs"><div><div id="par_14996256338215440"><h2 class="title">1 概述<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>Illumina测序技术分为三个基本步骤：扩增、测序和分析。这个过程从纯化的脱氧核糖核酸开始。脱氧核糖核酸被破碎成较小的片段，并加上接头、索引和其他种类的分子修饰——在扩增、测序和分析过程中作为参考点。经过修饰的脱氧核糖核酸被加入到一个专门的芯片上，进行扩增和测序。芯片底部有成千上万的寡核苷酸(人工合成的短脱氧核糖核酸片段)。它们被锚定在芯片上，能够抓取具有互补序列的脱氧核糖核酸片段。一旦片段被连接到芯片上，簇生成阶段就开始了。这一步产生大约一千份脱氧核糖核酸片段的拷贝。接下来，引物和修饰的核苷酸进入芯片。这些核苷酸具有可逆的3’阻断基团，迫使聚合酶一次只能添加一个核苷酸，同时这个核苷酸还有荧光标记。在每一轮合成之后，照相机会给芯片拍一张照片。计算机根据荧光标签的波长确定添加了什么样的碱基，并记录下来芯片上的每个点的位置。每轮反应后，未结合的分子被洗去。用化学剂去封闭步骤，去除（单个碱基）3’末端的封闭基团和染料。这个过程一直持续到完整的脱氧核糖核酸分子被测序。<sup><a href="https://web.archive.org/web/20221025115646/https://baike.sogou.com/kexue/d11029.htm#quote_5" class="kx_ref">[5]</a></sup>利用这项技术，通过大规模并行测序，基因组中的成千上万个位置同时被测序。</p></div></div><div id="par_14996256354992649"><h2 class="title">2 程序<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p></p><h3>2.1 <span>标签化</span></h3><p></p><p>脱氧核糖核酸纯化后的第一步是加上标签。转座酶随机将脱氧核糖核酸切割成小段(“标签”)。接头被添加在切割点的两侧(连接)。未连接接头的DNA链被洗掉。<sup><a href="https://web.archive.org/web/20221025115646/https://baike.sogou.com/kexue/d11029.htm#quote_5" class="kx_ref">[5]</a></sup> </p><p></p><div class="text_img ed_imgfloat_embedmiddle">
            <a class="ed_image_link" data-src="https://img03.sogoucdn.com/app/a/200698/sogou_science_16223" data-bigsrc="" title="点击查看大图" href="javascript:">
                <img alt="" class="ed_imgfloat_embedmiddle" img_height="338" img_width="450" titlename="双链脱氧核糖核酸被转座酶切割。切割的末端被修复，接头、索引、引物结合位点和末端位点被添加到每一条脱氧核糖核酸链上。图示来自illumina测序视频的一部分。[1]" data-align="embedmiddle" width="450" data-src="https://img03.sogoucdn.com/app/a/200698/sogou_science_16223" src="./1029.染料测序 - 搜狗科学百科_files/sogou_science_16223" data-observer="true">
            </a>
            <div class="text_img_title">双链脱氧核糖核酸被转座酶切割。切割的末端被修复，接头、索引、引物结合位点和末端位点被添加到每一条脱氧核糖核酸链上。图示来自illumina测序视频的一部分。[1]</div>
        </div> <p></p><p></p><h3>2.2 <span>低循环扩增</span></h3><p></p><p>下一步被称为低循环扩增。此步骤 (在DNA 片段上)添加引物结合序列、索引和末端序列。索引序列的长度通常是六个碱基对，在脱氧核糖核酸序列分析中用来鉴定样品。索引允许多达96个不同的样本一起进行扩增。在分析过程中，计算机会将所有具有相同索引的读取结果组合在一起。<sup><a href="https://web.archive.org/web/20221025115646/https://baike.sogou.com/kexue/d11029.htm#quote_6" class="kx_ref">[6]</a></sup><sup><a href="https://web.archive.org/web/20221025115646/https://baike.sogou.com/kexue/d11029.htm#quote_7" class="kx_ref">[7]</a></sup>末端序列用于将脱氧核糖核酸链连接到流动池上。Illumina采用“边合成边测序”方法。<sup><a href="https://web.archive.org/web/20221025115646/https://baike.sogou.com/kexue/d11029.htm#quote_7" class="kx_ref">[7]</a></sup>这一过程发生在丙烯酰胺涂层的玻璃流动池内。<sup><a href="https://web.archive.org/web/20221025115646/https://baike.sogou.com/kexue/d11029.htm#quote_8" class="kx_ref">[8]</a></sup>在流动池的底部覆盖有寡核苷酸 (短核苷酸序列)，它们在测序过程中用于将脱氧核糖核酸链固定到适当位置。寡核苷酸与低循环扩增过程中添加到脱氧核糖核酸中的两种末端序列相匹配。当脱氧核糖核酸进入流动池内，其中一个接头连接到与其互补的寡核苷酸上。</p><p></p><div class="text_img ed_imgfloat_embedmiddle">
            <a class="ed_image_link" data-src="https://img01.sogoucdn.com/app/a/200698/sogou_science_16224" data-bigsrc="" title="点击查看大图" href="javascript:">
                <img alt="" class="ed_imgfloat_embedmiddle" img_height="248" img_width="330" titlename="数百万寡核苷酸序列排列在每个流动池的通道的底部" data-align="embedmiddle" width="330" data-src="https://img01.sogoucdn.com/app/a/200698/sogou_science_16224" src="./1029.染料测序 - 搜狗科学百科_files/sogou_science_16224" data-observer="true">
            </a>
            <div class="text_img_title" style="text-align: center;">数百万寡核苷酸序列排列在每个流动池的通道的底部</div>
        </div> <p></p><p></p><h3>2.3 <span>桥式扩增</span></h3><p></p><p>一旦与流动池上寡核苷酸连接，簇生成就可以开始了。目的是创造许许多多相同的脱氧核糖核酸链。有些是正向链；其余的，是反向链。簇是通过桥式扩增产生的。聚合酶沿着一条脱氧核糖核酸链移动，形成互补链。原来的模板链被洗去，只留下反向链。在反向链的顶部有接头序列。脱氧核糖核酸链弯曲并与顶部接头序列互补的寡核苷酸杂交结合。聚合酶附着在反向链上，形成它的互补链(即与模板链相同)。此时的双链脱氧核糖核酸变性解链，每条链单独锚定在流动池上的寡核苷酸序列上。一个是反向链；另一个是先前的链。这一过程被称为桥式扩增，它同时发生在整个流动池的数千个簇中。</p><p></p><h3>2.4 <span>克隆扩增</span></h3><p></p><p>脱氧核糖核酸链一次又一次进行弯曲与寡核苷酸互补结合。聚合酶将合成一条新的链来产生双链片段，然后该片段变性，使得一个区域中的所有脱氧核糖核酸链来自同一个DNA分子(克隆扩增)。克隆扩增对于质量控制非常重要。如果发现一条链有一个奇数序列，那么科学家可以检查反向链，以确保它有相同奇数的互补序列。正向和反向链起到检查防止假象的作用。因为Illumina测序使用的聚合酶，已发现过有碱基错误替换（的现象），<sup><a href="https://web.archive.org/web/20221025115646/https://baike.sogou.com/kexue/d11029.htm#quote_9" class="kx_ref">[9]</a></sup>特别是在3’末端。<sup><a href="https://web.archive.org/web/20221025115646/https://baike.sogou.com/kexue/d11029.htm#quote_10" class="kx_ref">[10]</a></sup>配对的末端读取结果结合簇生成可以确认错误发生（情况）。反向和正向链应该彼此互补，所有反向的读取结果应该相互匹配，所有正向读取结果也应该相互匹配。如果一条reads与其对应副本(通过它进行克隆)不够相似，则可能发生了错误。在一些实验室的分析中使用97%相似度作为最小阈值。<sup><a href="https://web.archive.org/web/20221025115646/https://baike.sogou.com/kexue/d11029.htm#quote_10" class="kx_ref">[10]</a></sup> </p><p></p><h3>2.5 <span>边合成边测序</span></h3><p></p><p>克隆扩增结束时，所有反向链都从流动池中洗掉，只留下正向链。引物结合到正向链上，聚合酶将荧光标记的核苷酸添加到脱氧核糖核酸链上。每轮只增加一个碱基。每一个核苷酸上都有一个可逆的终止剂，以防止一轮中多次添加核苷酸。使用四色化学发光，四个碱基每一个都有独特的荧光信号释放，每一轮之后，机器记录下添加了哪一种碱基。Illumina公司从推出NextSeq和后来的MiniSeq，又推出了一种新的双色测序化学反应方法。与之前的区别在于核苷酸荧光修饰上，新的方法使用两种中一种颜色(红色或绿色)、无颜色(“黑色”)或同时结合两种颜色(红色和绿色的混合后呈现橙色) 来判定碱基种类。</p><p></p><div class="text_img ed_imgfloat_right">
            <a class="ed_image_link lazyLoad" data-src="https://img01.sogoucdn.com/app/a/200698/sogou_science_16225" data-bigsrc="" title="点击查看大图" href="javascript:" data-observer="true"></a>
            <div class="text_img_title">标记的核苷酸被添加到脱氧核糖核酸链中。四种核苷酸中每一种都有一个可以被激发以特征波长发射的识别标记。电脑会记录所有的释放的荧光信号，并根据这些数据转化成碱基。</div>   
        </div> <p></p><p>一旦脱氧核糖核酸链被读取完成，刚刚合成的链就会被洗掉。然后，索引1引物进行结合，合成并测序索引1序列，然后被洗掉。该链再次形成一个桥，并且其3’端连接到流动池上的寡核苷酸上。索引2引物结合，合成并测得索引2的序列，然后被洗掉。</p><p>聚合酶在拱形链上合成互补链。它们由于变性而分开，每条链的3’端被封闭。正向链被洗掉，反向链重复边合成边测序的过程。</p><p></p><h3>2.6 <span>数据分析</span></h3><p></p><p>测序同时发生在数百万个簇上，每个簇有大约1000个相同的脱氧核糖核酸插入片段的拷贝。<sup><a href="https://web.archive.org/web/20221025115646/https://baike.sogou.com/kexue/d11029.htm#quote_9" class="kx_ref">[9]</a></sup>序列数据的分析是通过寻找具有重叠区域(称为重叠群)的片段，并将它们排列起来。如果参考序列是已知的，将重叠群与参考序列进行比较，以识别突变位点。</p><p>尽管从未得到过完整无断的序列，但这种获取零碎短片段的方法让科学家能够得到完整的序列；然而，由于Illumina读取长度不是很长<sup><a href="https://web.archive.org/web/20221025115646/https://baike.sogou.com/kexue/d11029.htm#quote_10" class="kx_ref">[10]</a></sup>（ HiSeq测序可以产生大约90 bp长的读取结果），<sup><a href="https://web.archive.org/web/20221025115646/https://baike.sogou.com/kexue/d11029.htm#quote_6" class="kx_ref">[6]</a></sup>解决短串联重复序列可能是一个困难。<sup><a href="https://web.archive.org/web/20221025115646/https://baike.sogou.com/kexue/d11029.htm#quote_6" class="kx_ref">[6]</a></sup><sup><a href="https://web.archive.org/web/20221025115646/https://baike.sogou.com/kexue/d11029.htm#quote_9" class="kx_ref">[9]</a></sup>此外，如果序列是从头测序，参考基因组不存在，重复序列在序列组装中会造成很多困难。<sup><a href="https://web.archive.org/web/20221025115646/https://baike.sogou.com/kexue/d11029.htm#quote_9" class="kx_ref">[9]</a></sup>还有些其他困难包括不精确的聚合酶反应、嵌合序列和PCR偏好性引起的碱基替换(特别是在读取结果的3’末端)，<sup><a href="https://web.archive.org/web/20221025115646/https://baike.sogou.com/kexue/d11029.htm#quote_10" class="kx_ref">[10]</a></sup>所有这些都可能导致产生不正确的序列。<sup><a href="https://web.archive.org/web/20221025115646/https://baike.sogou.com/kexue/d11029.htm#quote_10" class="kx_ref">[10]</a></sup> </p></div></div><div id="par_14996256354992650"><h2 class="title">3 与其他测序方法的比较<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>这种技术与桑格测序等传统测序方法相比有很多优势。由于Illumina染料测序的自动化性质，使同时测序多条链并快速获得实际测序数据成为可能。此外，这种方法仅使用脱氧核糖核酸聚合酶，而不是像其他测序技术(如焦磷酸测序)需要多种昂贵的酶类。<sup><a href="https://web.archive.org/web/20221025115646/https://baike.sogou.com/kexue/d11029.htm#quote_11" class="kx_ref">[11]</a></sup> </p></div></div><div id="par_14996256354992651"><h2 class="title">4 使用示例<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>Illumina测序已被应用在甘薯和裸子植物红豆杉属的转录组研究中。<sup><a href="https://web.archive.org/web/20221025115646/https://baike.sogou.com/kexue/d11029.htm#quote_12" class="kx_ref">[12]</a></sup><sup><a href="https://web.archive.org/web/20221025115646/https://baike.sogou.com/kexue/d11029.htm#quote_13" class="kx_ref">[13]</a></sup> </p></div></div></div></div><div id="references"><h2 class="title" id="par_references">参考文献</h2><ul class="references"><li id="quote_1"><span class="references-num">[1]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Canard, Bruno; Sarfati, Simon (13 Oct 1994), Novel derivatives usable for the sequencing of nucleic acids, retrieved 2016-03-09.</span></p></li><li id="quote_2"><span class="references-num">[2]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Canard, Bruno; Sarfati, Robert S. (1994-10-11). "DNA polymerase fluorescent substrates with reversible 3′-tags". Gene. 148 (1): 1–6. doi:10.1016/0378-1119(94)90226-7. PMID 7523248..</span></p></li><li id="quote_3"><span class="references-num">[3]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"History of Illumina Sequencing". Archived from the original on 12 October 2014..</span></p></li><li id="quote_4"><span class="references-num">[4]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Illumina - Sequencing and array-based solutions for genetic research". www.illumina.com..</span></p></li><li id="quote_5"><span class="references-num">[5]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Meyer, M.; Kircher, M. (2010). "Illumina Sequencing Library Preparation for Highly Multiplexed Target Capture and Sequencing". Cold Spring Harbor Protocols. 2010 (6): pdb.prot5448. doi:10.1101/pdb.prot5448. PMID 20516186..</span></p></li><li id="quote_6"><span class="references-num">[6]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Feng, Y.-J., Q.-F., Chen, M.-Y., Liang, D. and Zhang, P. (2015). "Parallel tagged amplicon sequencing of relatively long PCR products using the Illumina HiSeq platform and transcriptome assembly". Molecular Ecology Resources. 16 (1): 91–102. doi:10.1111/1755-0998.12429. PMID 25959587..</span></p></li><li id="quote_7"><span class="references-num">[7]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Illumina, Inc. "Multiplexed Sequencing with the Illumina Genome Analyzer System" (PDF). Retrieved 25 September 2015..</span></p></li><li id="quote_8"><span class="references-num">[8]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Quail, Michael A (2012). "A Tale of Three next Generation Sequencing Platforms: Comparison of Ion Torrent, Pacific Biosciences and Illumina MiSeq Sequencers". BMC Genomics. 13: 341. doi:10.1186/1471-2164-13-341. PMC 3431227. PMID 22827831..</span></p></li><li id="quote_9"><span class="references-num">[9]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Morozova, Marra O (Nov 2008). "Applications of next-generation sequencing in functional genomics". Genomics. 92 (5): 255–64. doi:10.1016/j.ygeno.2008.07.001. PMID 18703132..</span></p></li><li id="quote_10"><span class="references-num">[10]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Jeon, YS, Park SC, Lim J, Chun J, Kim BS (January 4, 2015). "Improved pipeline for reducing erroneous identification by 16S rRNA sequences using the Illumina MiSeq platform". Journal of Microbiology. 53 (1): 60–9. doi:10.1007/s12275-015-4601-y. PMID 25557481..</span></p></li><li id="quote_11"><span class="references-num">[11]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Pettersson E .，Lundeberge J .，Ahmadian A. (2008年)。测序技术的产生。基因组学”，第105-111页。.</span></p></li><li id="quote_12"><span class="references-num">[12]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Wang, Z; Fang, B; Chen, J; Zhang, X; Luo, Z; Huang, L; Chen, X; Li, Y (Dec 24, 2010). "De novo assembly and characterization of root transcriptome using Illumina paired-end sequencing and development of cSSR markers in sweet potato (Ipomoea batatas)". BMC Genomics. 11: 726. doi:10.1186/1471-2164-11-726. PMC 3016421. PMID 21182800..</span></p></li><li id="quote_13"><span class="references-num">[13]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Hao, Da Cheng; Ge, GuangBo; Xiao, PeiGen; Zhang, YanYan; Yang, Ling; Ellegren, Hans (22 June 2011). "The First Insight into the Tissue Specific Taxus Transcriptome via Illumina Second Generation Sequencing". PLoS ONE. 6 (6): e21220. doi:10.1371/journal.pone.0021220. PMC 3120849. PMID 21731678..</span></p></li></ul></div><div class="read-num">阅读 <!-- -->2332</div></div><div class="right-side" id="rightSide"><div class="side" id="lemma-side"><div class="side-title">版本记录</div><ul class="side-lst"><li><p class="side-lst-txt">暂无</p></li></ul><div class="user-card userCard"></div></div><div class="side"><div class="side-event"></div></div></div></div><div class="footer-box"><div id="footer"><div class="footer-logo-wrap"><div class="footer-logo"></div><div class="footer-logo-text">知识·传播·科普</div></div><div class="footer-info">本网站内容采用<a target="_blank" href="https://web.archive.org/web/20221025115646/https://creativecommons.org/licenses/by-sa/3.0/deed.zh?tdsourcetag=s_pctim_aiomsg">CC-BY-SA 3.0</a>授权</div><div class="footer-btn-wrap"><a target="_blank" href="https://web.archive.org/web/20221025115646/https://baike.sogou.com/help/#user_protocol">用户协议</a><a target="_blank" href="https://web.archive.org/web/20221025115646/http://www.sogou.com/docs/terms.htm?v=1">免责声明</a><a target="_blank" href="https://web.archive.org/web/20221025115646/http://corp.sogou.com/private.html">隐私政策</a><a target="_blank" href="https://web.archive.org/web/20221025115646/https://baike.sogou.com/kexue/intro.htm">关于我们</a></div></div></div><script>window.lemmaInfo ={"lemmaId":"11029","versionId":"54562946398871820","title":"染料测序","subtitle":"","abstracts":{"paragraphId":"54562946398871826","title":"摘要","versionId":"54562946398871821","lemmaId":11029,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82815118,"name":"iscience","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599818050,"comment":null,"dependVersionId":0,"contentType":2,"content":"<p>Illumina染料测序是一种用于测定脱氧核糖核酸中一连串碱基对的技术，也称为脱氧核糖核酸（DNA）测序。可逆终止化学概念由巴黎巴斯德研究所的布鲁诺·卡纳德（Bruno Canard）和西蒙·萨法蒂（Simon Sarfati）提出。<sup><a href=\"#quote_1\" class=\"kx_ref\">[1]</a></sup><sup><a href=\"#quote_2\" class=\"kx_ref\">[2]</a></sup>后来剑桥大学的尚卡尔·巴拉苏布拉马尼安（Shankar Balasubramanian）和大卫·克莱恩曼（David Klenerman）开发了这一技术，<sup><a href=\"#quote_3\" class=\"kx_ref\">[3]</a></sup>他们随后创建了Solexa公司，后来被Illumina收购。这种测序方法基于可逆的染料--终止技术，当它们被引入到脱氧核糖核酸链中时，能够识别单个碱基。它也可用于全基因组和局部测序、转录组分析、宏基因组学、小核糖核酸发现、甲基化谱和全基因组蛋白质-核酸互作分析。<sup><a href=\"#quote_4\" class=\"kx_ref\">[4]</a></sup> </p>","pics":[{"originalUrl":"https://web.archive.org/web/20221025115646/https://img02.sogoucdn.com/app/a/200698/sogou_science_16222?width=231&height=174&titlename=%E8%84%B1%E6%B0%A7%E6%A0%B8%E7%B3%96%E6%A0%B8%E9%85%B8%E9%80%9A%E8%BF%87%E4%BA%92%E8%A1%A5%E5%BA%8F%E5%88%97%E9%99%84%E7%9D%80%E5%9C%A8%E6%B5%81%E5%8A%A8%E6%B1%A0%E4%B8%8A%E3%80%82%E8%AF%A5%E9%93%BE%E5%BC%AF%E6%9B%B2%E5%B9%B6%E8%BF%9E%E6%8E%A5%E5%88%B0%E7%AC%AC%E4%BA%8C%E5%AF%A1%E8%81%9A%E6%A0%B8%E8%8B%B7%E9%85%B8%E4%B8%8A%EF%BC%8C%E5%BD%A2%E6%88%90%E6%A1%A5%E3%80%82%E8%81%9A%E5%90%88%E9%85%B6%E5%90%88%E6%88%90%E5%8F%8D%E5%90%91%E9%93%BE%E3%80%82%E4%B8%A4%E6%9D%A1%E9%93%BE%E8%A7%A3%E9%93%BE%E5%B9%B6%E4%BC%B8%E7%9B%B4%E3%80%82%E6%AF%8F%E4%B8%80%E6%9D%A1%E9%93%BE%E9%83%BD%E5%BD%A2%E6%88%90%E4%B8%80%E4%B8%AA%E6%96%B0%E6%A1%A5(%E6%A1%A5%E5%BC%8F%E6%89%A9%E5%A2%9E)%E3%80%82%E5%85%B6%E7%BB%93%E6%9E%9C%E6%98%AF%E4%BA%A7%E7%94%9F%E4%B8%80%E7%B0%87DNA%E7%9A%84%E6%AD%A3%E9%93%BE%E5%92%8C%E5%8F%8D%E5%90%91%E4%BA%92%E8%A1%A5%E9%93%BE%E7%9A%84%E5%A4%8D%E5%88%B6%E3%80%82&w=600&h=480","url":"https://web.archive.org/web/20221025115646/https://img02.sogoucdn.com/app/a/200698/sogou_science_16222","rw":600,"rh":480,"title":"脱氧核糖核酸通过互补序列附着在流动池上。该链弯曲并连接到第二寡聚核苷酸上，形成桥。聚合酶合成反向链。两条链解链并伸直。每一条链都形成一个新桥(桥式扩增)。其结果是产生一簇DNA的正链和反向互补链的复制。","alt":null,"width":231,"height":174}],"card":null,"references":[],"versionCount":0},"card":{"paragraphId":"0","title":"","versionId":"0","lemmaId":0,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":0,"comment":null,"dependVersionId":0,"contentType":0,"content":"","pics":null,"card":null,"references":null,"versionCount":0},"categories":[{"id":4,"name":"生物学","parents":[]}],"creator":{"uid":76060375,"name":"郭媛媛","pic":"https://web.archive.org/web/20221025115646/https://cache.soso.com/qlogo/g?b=oidb&k=uCDXrYS8yOVG5n9UN9Ujibw&s=100&t=1556648699","introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":"","jobBrief":"","role":0,"roleName":null,"title":"","professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":1571038441,"editor":{"uid":76060375,"name":"郭媛媛","pic":"https://web.archive.org/web/20221025115646/https://cache.soso.com/qlogo/g?b=oidb&k=uCDXrYS8yOVG5n9UN9Ujibw&s=100&t=1556648699","introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":"","jobBrief":"","role":0,"roleName":null,"title":"","professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576234464,"state":1,"versionCount":1,"upNum":2,"downNum":0,"pics":[{"originalUrl":"https://web.archive.org/web/20221025115646/https://img02.sogoucdn.com/app/a/200698/sogou_science_16222?width=231&height=174&titlename=%E8%84%B1%E6%B0%A7%E6%A0%B8%E7%B3%96%E6%A0%B8%E9%85%B8%E9%80%9A%E8%BF%87%E4%BA%92%E8%A1%A5%E5%BA%8F%E5%88%97%E9%99%84%E7%9D%80%E5%9C%A8%E6%B5%81%E5%8A%A8%E6%B1%A0%E4%B8%8A%E3%80%82%E8%AF%A5%E9%93%BE%E5%BC%AF%E6%9B%B2%E5%B9%B6%E8%BF%9E%E6%8E%A5%E5%88%B0%E7%AC%AC%E4%BA%8C%E5%AF%A1%E8%81%9A%E6%A0%B8%E8%8B%B7%E9%85%B8%E4%B8%8A%EF%BC%8C%E5%BD%A2%E6%88%90%E6%A1%A5%E3%80%82%E8%81%9A%E5%90%88%E9%85%B6%E5%90%88%E6%88%90%E5%8F%8D%E5%90%91%E9%93%BE%E3%80%82%E4%B8%A4%E6%9D%A1%E9%93%BE%E8%A7%A3%E9%93%BE%E5%B9%B6%E4%BC%B8%E7%9B%B4%E3%80%82%E6%AF%8F%E4%B8%80%E6%9D%A1%E9%93%BE%E9%83%BD%E5%BD%A2%E6%88%90%E4%B8%80%E4%B8%AA%E6%96%B0%E6%A1%A5(%E6%A1%A5%E5%BC%8F%E6%89%A9%E5%A2%9E)%E3%80%82%E5%85%B6%E7%BB%93%E6%9E%9C%E6%98%AF%E4%BA%A7%E7%94%9F%E4%B8%80%E7%B0%87DNA%E7%9A%84%E6%AD%A3%E9%93%BE%E5%92%8C%E5%8F%8D%E5%90%91%E4%BA%92%E8%A1%A5%E9%93%BE%E7%9A%84%E5%A4%8D%E5%88%B6%E3%80%82&w=600&h=480","url":"https://web.archive.org/web/20221025115646/https://img02.sogoucdn.com/app/a/200698/sogou_science_16222","rw":600,"rh":480,"title":"脱氧核糖核酸通过互补序列附着在流动池上。该链弯曲并连接到第二寡聚核苷酸上，形成桥。聚合酶合成反向链。两条链解链并伸直。每一条链都形成一个新桥(桥式扩增)。其结果是产生一簇DNA的正链和反向互补链的复制。","alt":null,"width":231,"height":174},{"originalUrl":"https://web.archive.org/web/20221025115646/https://img03.sogoucdn.com/app/a/200698/sogou_science_16223?w=450&h=338&titlename=%E5%8F%8C%E9%93%BE%E8%84%B1%E6%B0%A7%E6%A0%B8%E7%B3%96%E6%A0%B8%E9%85%B8%E8%A2%AB%E8%BD%AC%E5%BA%A7%E9%85%B6%E5%88%87%E5%89%B2%E3%80%82%E5%88%87%E5%89%B2%E7%9A%84%E6%9C%AB%E7%AB%AF%E8%A2%AB%E4%BF%AE%E5%A4%8D%EF%BC%8C%E6%8E%A5%E5%A4%B4%E3%80%81%E7%B4%A2%E5%BC%95%E3%80%81%E5%BC%95%E7%89%A9%E7%BB%93%E5%90%88%E4%BD%8D%E7%82%B9%E5%92%8C%E6%9C%AB%E7%AB%AF%E4%BD%8D%E7%82%B9%E8%A2%AB%E6%B7%BB%E5%8A%A0%E5%88%B0%E6%AF%8F%E4%B8%80%E6%9D%A1%E8%84%B1%E6%B0%A7%E6%A0%B8%E7%B3%96%E6%A0%B8%E9%85%B8%E9%93%BE%E4%B8%8A%E3%80%82%E5%9B%BE%E7%A4%BA%E6%9D%A5%E8%87%AAillumina%E6%B5%8B%E5%BA%8F%E8%A7%86%E9%A2%91%E7%9A%84%E4%B8%80%E9%83%A8%E5%88%86%E3%80%82%5B1%5D","url":"https://web.archive.org/web/20221025115646/https://img03.sogoucdn.com/app/a/200698/sogou_science_16223","rw":450,"rh":338,"title":"双链脱氧核糖核酸被转座酶切割。切割的末端被修复，接头、索引、引物结合位点和末端位点被添加到每一条脱氧核糖核酸链上。图示来自illumina测序视频的一部分。[1]","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221025115646/https://img01.sogoucdn.com/app/a/200698/sogou_science_16224?h=248&titlename=%E6%95%B0%E7%99%BE%E4%B8%87%E5%AF%A1%E6%A0%B8%E8%8B%B7%E9%85%B8%E5%BA%8F%E5%88%97%E6%8E%92%E5%88%97%E5%9C%A8%E6%AF%8F%E4%B8%AA%E6%B5%81%E5%8A%A8%E6%B1%A0%E7%9A%84%E9%80%9A%E9%81%93%E7%9A%84%E5%BA%95%E9%83%A8&w=330","url":"https://web.archive.org/web/20221025115646/https://img01.sogoucdn.com/app/a/200698/sogou_science_16224","rw":330,"rh":248,"title":"数百万寡核苷酸序列排列在每个流动池的通道的底部","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221025115646/https://img01.sogoucdn.com/app/a/200698/sogou_science_16225?h=300&titlename=%E6%A0%87%E8%AE%B0%E7%9A%84%E6%A0%B8%E8%8B%B7%E9%85%B8%E8%A2%AB%E6%B7%BB%E5%8A%A0%E5%88%B0%E8%84%B1%E6%B0%A7%E6%A0%B8%E7%B3%96%E6%A0%B8%E9%85%B8%E9%93%BE%E4%B8%AD%E3%80%82%E5%9B%9B%E7%A7%8D%E6%A0%B8%E8%8B%B7%E9%85%B8%E4%B8%AD%E6%AF%8F%E4%B8%80%E7%A7%8D%E9%83%BD%E6%9C%89%E4%B8%80%E4%B8%AA%E5%8F%AF%E4%BB%A5%E8%A2%AB%E6%BF%80%E5%8F%91%E4%BB%A5%E7%89%B9%E5%BE%81%E6%B3%A2%E9%95%BF%E5%8F%91%E5%B0%84%E7%9A%84%E8%AF%86%E5%88%AB%E6%A0%87%E8%AE%B0%E3%80%82%E7%94%B5%E8%84%91%E4%BC%9A%E8%AE%B0%E5%BD%95%E6%89%80%E6%9C%89%E7%9A%84%E9%87%8A%E6%94%BE%E7%9A%84%E8%8D%A7%E5%85%89%E4%BF%A1%E5%8F%B7%EF%BC%8C%E5%B9%B6%E6%A0%B9%E6%8D%AE%E8%BF%99%E4%BA%9B%E6%95%B0%E6%8D%AE%E8%BD%AC%E5%8C%96%E6%88%90%E7%A2%B1%E5%9F%BA%E3%80%82&w=300","url":"https://web.archive.org/web/20221025115646/https://img01.sogoucdn.com/app/a/200698/sogou_science_16225","rw":300,"rh":300,"title":"标记的核苷酸被添加到脱氧核糖核酸链中。四种核苷酸中每一种都有一个可以被激发以特征波长发射的识别标记。电脑会记录所有的释放的荧光信号，并根据这些数据转化成碱基。","alt":null,"width":0,"height":0}],"catalogs":[{"level":1,"title":"概述","paragraphId":"14996256338215440","subCatalogs":null},{"level":1,"title":"程序","paragraphId":"14996256354992649","subCatalogs":[{"level":2,"title":"标签化","paragraphId":"14996256354992649","subCatalogs":null},{"level":2,"title":"低循环扩增","paragraphId":"14996256354992649","subCatalogs":null},{"level":2,"title":"桥式扩增","paragraphId":"14996256354992649","subCatalogs":null},{"level":2,"title":"克隆扩增","paragraphId":"14996256354992649","subCatalogs":null},{"level":2,"title":"边合成边测序","paragraphId":"14996256354992649","subCatalogs":null},{"level":2,"title":"数据分析","paragraphId":"14996256354992649","subCatalogs":null}]},{"level":1,"title":"与其他测序方法的比较","paragraphId":"14996256354992650","subCatalogs":null},{"level":1,"title":"使用示例","paragraphId":"14996256354992651","subCatalogs":null},{"level":1,"title":"参考文献","paragraphId":"-1","subCatalogs":null}],"paragraphs":[{"paragraphId":"14996256338215440","title":"概述","versionId":"54562946398871822","lemmaId":11029,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82815118,"name":"iscience","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599818050,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>Illumina测序技术分为三个基本步骤：扩增、测序和分析。这个过程从纯化的脱氧核糖核酸开始。脱氧核糖核酸被破碎成较小的片段，并加上接头、索引和其他种类的分子修饰——在扩增、测序和分析过程中作为参考点。经过修饰的脱氧核糖核酸被加入到一个专门的芯片上，进行扩增和测序。芯片底部有成千上万的寡核苷酸(人工合成的短脱氧核糖核酸片段)。它们被锚定在芯片上，能够抓取具有互补序列的脱氧核糖核酸片段。一旦片段被连接到芯片上，簇生成阶段就开始了。这一步产生大约一千份脱氧核糖核酸片段的拷贝。接下来，引物和修饰的核苷酸进入芯片。这些核苷酸具有可逆的3’阻断基团，迫使聚合酶一次只能添加一个核苷酸，同时这个核苷酸还有荧光标记。在每一轮合成之后，照相机会给芯片拍一张照片。计算机根据荧光标签的波长确定添加了什么样的碱基，并记录下来芯片上的每个点的位置。每轮反应后，未结合的分子被洗去。用化学剂去封闭步骤，去除（单个碱基）3’末端的封闭基团和染料。这个过程一直持续到完整的脱氧核糖核酸分子被测序。<sup><a href=\"#quote_5\" class=\"kx_ref\">[5]</a></sup>利用这项技术，通过大规模并行测序，基因组中的成千上万个位置同时被测序。</p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14996256354992649","title":"程序","versionId":"54562946398871823","lemmaId":11029,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82815118,"name":"iscience","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599818050,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p></p><h3>标签化</h3><p></p><p>脱氧核糖核酸纯化后的第一步是加上标签。转座酶随机将脱氧核糖核酸切割成小段(“标签”)。接头被添加在切割点的两侧(连接)。未连接接头的DNA链被洗掉。<sup><a href=\"#quote_5\" class=\"kx_ref\">[5]</a></sup> </p><p><img alt=\"\" class=\"ed_imgfloat_embedmiddle\" img_height=\"338\" img_width=\"450\" titlename=\"双链脱氧核糖核酸被转座酶切割。切割的末端被修复，接头、索引、引物结合位点和末端位点被添加到每一条脱氧核糖核酸链上。图示来自illumina测序视频的一部分。[1]\" data-align=\"embedmiddle\" width=\"450\" data-src=\"https://img03.sogoucdn.com/app/a/200698/sogou_science_16223\"> </p><p></p><h3>低循环扩增</h3><p></p><p>下一步被称为低循环扩增。此步骤 (在DNA 片段上)添加引物结合序列、索引和末端序列。索引序列的长度通常是六个碱基对，在脱氧核糖核酸序列分析中用来鉴定样品。索引允许多达96个不同的样本一起进行扩增。在分析过程中，计算机会将所有具有相同索引的读取结果组合在一起。<sup><a href=\"#quote_6\" class=\"kx_ref\">[6]</a></sup><sup><a href=\"#quote_7\" class=\"kx_ref\">[7]</a></sup>末端序列用于将脱氧核糖核酸链连接到流动池上。Illumina采用“边合成边测序”方法。<sup><a href=\"#quote_7\" class=\"kx_ref\">[7]</a></sup>这一过程发生在丙烯酰胺涂层的玻璃流动池内。<sup><a href=\"#quote_8\" class=\"kx_ref\">[8]</a></sup>在流动池的底部覆盖有寡核苷酸 (短核苷酸序列)，它们在测序过程中用于将脱氧核糖核酸链固定到适当位置。寡核苷酸与低循环扩增过程中添加到脱氧核糖核酸中的两种末端序列相匹配。当脱氧核糖核酸进入流动池内，其中一个接头连接到与其互补的寡核苷酸上。</p><p><img alt=\"\" class=\"ed_imgfloat_embedmiddle\" img_height=\"248\" img_width=\"330\" titlename=\"数百万寡核苷酸序列排列在每个流动池的通道的底部\" data-align=\"embedmiddle\" width=\"330\" data-src=\"https://img01.sogoucdn.com/app/a/200698/sogou_science_16224\"> </p><p></p><h3>桥式扩增</h3><p></p><p>一旦与流动池上寡核苷酸连接，簇生成就可以开始了。目的是创造许许多多相同的脱氧核糖核酸链。有些是正向链；其余的，是反向链。簇是通过桥式扩增产生的。聚合酶沿着一条脱氧核糖核酸链移动，形成互补链。原来的模板链被洗去，只留下反向链。在反向链的顶部有接头序列。脱氧核糖核酸链弯曲并与顶部接头序列互补的寡核苷酸杂交结合。聚合酶附着在反向链上，形成它的互补链(即与模板链相同)。此时的双链脱氧核糖核酸变性解链，每条链单独锚定在流动池上的寡核苷酸序列上。一个是反向链；另一个是先前的链。这一过程被称为桥式扩增，它同时发生在整个流动池的数千个簇中。</p><p></p><h3>克隆扩增</h3><p></p><p>脱氧核糖核酸链一次又一次进行弯曲与寡核苷酸互补结合。聚合酶将合成一条新的链来产生双链片段，然后该片段变性，使得一个区域中的所有脱氧核糖核酸链来自同一个DNA分子(克隆扩增)。克隆扩增对于质量控制非常重要。如果发现一条链有一个奇数序列，那么科学家可以检查反向链，以确保它有相同奇数的互补序列。正向和反向链起到检查防止假象的作用。因为Illumina测序使用的聚合酶，已发现过有碱基错误替换（的现象），<sup><a href=\"#quote_9\" class=\"kx_ref\">[9]</a></sup>特别是在3’末端。<sup><a href=\"#quote_10\" class=\"kx_ref\">[10]</a></sup>配对的末端读取结果结合簇生成可以确认错误发生（情况）。反向和正向链应该彼此互补，所有反向的读取结果应该相互匹配，所有正向读取结果也应该相互匹配。如果一条reads与其对应副本(通过它进行克隆)不够相似，则可能发生了错误。在一些实验室的分析中使用97%相似度作为最小阈值。<sup><a href=\"#quote_10\" class=\"kx_ref\">[10]</a></sup> </p><p></p><h3>边合成边测序</h3><p></p><p>克隆扩增结束时，所有反向链都从流动池中洗掉，只留下正向链。引物结合到正向链上，聚合酶将荧光标记的核苷酸添加到脱氧核糖核酸链上。每轮只增加一个碱基。每一个核苷酸上都有一个可逆的终止剂，以防止一轮中多次添加核苷酸。使用四色化学发光，四个碱基每一个都有独特的荧光信号释放，每一轮之后，机器记录下添加了哪一种碱基。Illumina公司从推出NextSeq和后来的MiniSeq，又推出了一种新的双色测序化学反应方法。与之前的区别在于核苷酸荧光修饰上，新的方法使用两种中一种颜色(红色或绿色)、无颜色(“黑色”)或同时结合两种颜色(红色和绿色的混合后呈现橙色) 来判定碱基种类。</p><p><img alt=\"\" class=\"fileimage kx_img ed_imgfloat_right\" img_height=\"300\" img_width=\"300\" titlename=\"标记的核苷酸被添加到脱氧核糖核酸链中。四种核苷酸中每一种都有一个可以被激发以特征波长发射的识别标记。电脑会记录所有的释放的荧光信号，并根据这些数据转化成碱基。\" data-src=\"https://img01.sogoucdn.com/app/a/200698/sogou_science_16225\"> </p><p>一旦脱氧核糖核酸链被读取完成，刚刚合成的链就会被洗掉。然后，索引1引物进行结合，合成并测序索引1序列，然后被洗掉。该链再次形成一个桥，并且其3’端连接到流动池上的寡核苷酸上。索引2引物结合，合成并测得索引2的序列，然后被洗掉。</p><p>聚合酶在拱形链上合成互补链。它们由于变性而分开，每条链的3’端被封闭。正向链被洗掉，反向链重复边合成边测序的过程。</p><p></p><h3>数据分析</h3><p></p><p>测序同时发生在数百万个簇上，每个簇有大约1000个相同的脱氧核糖核酸插入片段的拷贝。<sup><a href=\"#quote_9\" class=\"kx_ref\">[9]</a></sup>序列数据的分析是通过寻找具有重叠区域(称为重叠群)的片段，并将它们排列起来。如果参考序列是已知的，将重叠群与参考序列进行比较，以识别突变位点。</p><p>尽管从未得到过完整无断的序列，但这种获取零碎短片段的方法让科学家能够得到完整的序列；然而，由于Illumina读取长度不是很长<sup><a href=\"#quote_10\" class=\"kx_ref\">[10]</a></sup>（ HiSeq测序可以产生大约90 bp长的读取结果），<sup><a href=\"#quote_6\" class=\"kx_ref\">[6]</a></sup>解决短串联重复序列可能是一个困难。<sup><a href=\"#quote_6\" class=\"kx_ref\">[6]</a></sup><sup><a href=\"#quote_9\" class=\"kx_ref\">[9]</a></sup>此外，如果序列是从头测序，参考基因组不存在，重复序列在序列组装中会造成很多困难。<sup><a href=\"#quote_9\" class=\"kx_ref\">[9]</a></sup>还有些其他困难包括不精确的聚合酶反应、嵌合序列和PCR偏好性引起的碱基替换(特别是在读取结果的3’末端)，<sup><a href=\"#quote_10\" class=\"kx_ref\">[10]</a></sup>所有这些都可能导致产生不正确的序列。<sup><a href=\"#quote_10\" class=\"kx_ref\">[10]</a></sup> </p>","pics":[{"originalUrl":"https://web.archive.org/web/20221025115646/https://img03.sogoucdn.com/app/a/200698/sogou_science_16223?w=450&h=338&titlename=%E5%8F%8C%E9%93%BE%E8%84%B1%E6%B0%A7%E6%A0%B8%E7%B3%96%E6%A0%B8%E9%85%B8%E8%A2%AB%E8%BD%AC%E5%BA%A7%E9%85%B6%E5%88%87%E5%89%B2%E3%80%82%E5%88%87%E5%89%B2%E7%9A%84%E6%9C%AB%E7%AB%AF%E8%A2%AB%E4%BF%AE%E5%A4%8D%EF%BC%8C%E6%8E%A5%E5%A4%B4%E3%80%81%E7%B4%A2%E5%BC%95%E3%80%81%E5%BC%95%E7%89%A9%E7%BB%93%E5%90%88%E4%BD%8D%E7%82%B9%E5%92%8C%E6%9C%AB%E7%AB%AF%E4%BD%8D%E7%82%B9%E8%A2%AB%E6%B7%BB%E5%8A%A0%E5%88%B0%E6%AF%8F%E4%B8%80%E6%9D%A1%E8%84%B1%E6%B0%A7%E6%A0%B8%E7%B3%96%E6%A0%B8%E9%85%B8%E9%93%BE%E4%B8%8A%E3%80%82%E5%9B%BE%E7%A4%BA%E6%9D%A5%E8%87%AAillumina%E6%B5%8B%E5%BA%8F%E8%A7%86%E9%A2%91%E7%9A%84%E4%B8%80%E9%83%A8%E5%88%86%E3%80%82%5B1%5D","url":"https://web.archive.org/web/20221025115646/https://img03.sogoucdn.com/app/a/200698/sogou_science_16223","rw":450,"rh":338,"title":"双链脱氧核糖核酸被转座酶切割。切割的末端被修复，接头、索引、引物结合位点和末端位点被添加到每一条脱氧核糖核酸链上。图示来自illumina测序视频的一部分。[1]","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221025115646/https://img01.sogoucdn.com/app/a/200698/sogou_science_16224?h=248&titlename=%E6%95%B0%E7%99%BE%E4%B8%87%E5%AF%A1%E6%A0%B8%E8%8B%B7%E9%85%B8%E5%BA%8F%E5%88%97%E6%8E%92%E5%88%97%E5%9C%A8%E6%AF%8F%E4%B8%AA%E6%B5%81%E5%8A%A8%E6%B1%A0%E7%9A%84%E9%80%9A%E9%81%93%E7%9A%84%E5%BA%95%E9%83%A8&w=330","url":"https://web.archive.org/web/20221025115646/https://img01.sogoucdn.com/app/a/200698/sogou_science_16224","rw":330,"rh":248,"title":"数百万寡核苷酸序列排列在每个流动池的通道的底部","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221025115646/https://img01.sogoucdn.com/app/a/200698/sogou_science_16225?h=300&titlename=%E6%A0%87%E8%AE%B0%E7%9A%84%E6%A0%B8%E8%8B%B7%E9%85%B8%E8%A2%AB%E6%B7%BB%E5%8A%A0%E5%88%B0%E8%84%B1%E6%B0%A7%E6%A0%B8%E7%B3%96%E6%A0%B8%E9%85%B8%E9%93%BE%E4%B8%AD%E3%80%82%E5%9B%9B%E7%A7%8D%E6%A0%B8%E8%8B%B7%E9%85%B8%E4%B8%AD%E6%AF%8F%E4%B8%80%E7%A7%8D%E9%83%BD%E6%9C%89%E4%B8%80%E4%B8%AA%E5%8F%AF%E4%BB%A5%E8%A2%AB%E6%BF%80%E5%8F%91%E4%BB%A5%E7%89%B9%E5%BE%81%E6%B3%A2%E9%95%BF%E5%8F%91%E5%B0%84%E7%9A%84%E8%AF%86%E5%88%AB%E6%A0%87%E8%AE%B0%E3%80%82%E7%94%B5%E8%84%91%E4%BC%9A%E8%AE%B0%E5%BD%95%E6%89%80%E6%9C%89%E7%9A%84%E9%87%8A%E6%94%BE%E7%9A%84%E8%8D%A7%E5%85%89%E4%BF%A1%E5%8F%B7%EF%BC%8C%E5%B9%B6%E6%A0%B9%E6%8D%AE%E8%BF%99%E4%BA%9B%E6%95%B0%E6%8D%AE%E8%BD%AC%E5%8C%96%E6%88%90%E7%A2%B1%E5%9F%BA%E3%80%82&w=300","url":"https://web.archive.org/web/20221025115646/https://img01.sogoucdn.com/app/a/200698/sogou_science_16225","rw":300,"rh":300,"title":"标记的核苷酸被添加到脱氧核糖核酸链中。四种核苷酸中每一种都有一个可以被激发以特征波长发射的识别标记。电脑会记录所有的释放的荧光信号，并根据这些数据转化成碱基。","alt":null,"width":0,"height":0}],"card":null,"references":[],"versionCount":0},{"paragraphId":"14996256354992650","title":"与其他测序方法的比较","versionId":"54562946398871824","lemmaId":11029,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82815118,"name":"iscience","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599818050,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>这种技术与桑格测序等传统测序方法相比有很多优势。由于Illumina染料测序的自动化性质，使同时测序多条链并快速获得实际测序数据成为可能。此外，这种方法仅使用脱氧核糖核酸聚合酶，而不是像其他测序技术(如焦磷酸测序)需要多种昂贵的酶类。<sup><a href=\"#quote_11\" class=\"kx_ref\">[11]</a></sup> </p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14996256354992651","title":"使用示例","versionId":"54562946398871825","lemmaId":11029,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82815118,"name":"iscience","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599818050,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>Illumina测序已被应用在甘薯和裸子植物红豆杉属的转录组研究中。<sup><a href=\"#quote_12\" class=\"kx_ref\">[12]</a></sup><sup><a href=\"#quote_13\" class=\"kx_ref\">[13]</a></sup> </p>","pics":null,"card":null,"references":[],"versionCount":0}],"references":[{"id":1,"type":"book","title":"Canard, Bruno; Sarfati, Simon (13 Oct 1994), Novel derivatives usable for the sequencing of nucleic acids, retrieved 2016-03-09","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":2,"type":"book","title":"Canard, Bruno; Sarfati, Robert S. (1994-10-11). \"DNA polymerase fluorescent substrates with reversible 3′-tags\". Gene. 148 (1): 1–6. doi:10.1016/0378-1119(94)90226-7. PMID 7523248.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":3,"type":"book","title":"\"History of Illumina Sequencing\". Archived from the original on 12 October 2014.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":4,"type":"book","title":"\"Illumina - Sequencing and array-based solutions for genetic research\". www.illumina.com.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":5,"type":"book","title":"Meyer, M.; Kircher, M. (2010). \"Illumina Sequencing Library Preparation for Highly Multiplexed Target Capture and Sequencing\". Cold Spring Harbor Protocols. 2010 (6): pdb.prot5448. doi:10.1101/pdb.prot5448. PMID 20516186.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":6,"type":"book","title":"Feng, Y.-J., Q.-F., Chen, M.-Y., Liang, D. and Zhang, P. (2015). \"Parallel tagged amplicon sequencing of relatively long PCR products using the Illumina HiSeq platform and transcriptome assembly\". Molecular Ecology Resources. 16 (1): 91–102. doi:10.1111/1755-0998.12429. PMID 25959587.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":7,"type":"book","title":"Illumina, Inc. \"Multiplexed Sequencing with the Illumina Genome Analyzer System\" (PDF). Retrieved 25 September 2015.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":8,"type":"book","title":"Quail, Michael A (2012). \"A Tale of Three next Generation Sequencing Platforms: Comparison of Ion Torrent, Pacific Biosciences and Illumina MiSeq Sequencers\". BMC Genomics. 13: 341. doi:10.1186/1471-2164-13-341. PMC 3431227. PMID 22827831.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":9,"type":"book","title":"Morozova, Marra O (Nov 2008). \"Applications of next-generation sequencing in functional genomics\". Genomics. 92 (5): 255–64. doi:10.1016/j.ygeno.2008.07.001. PMID 18703132.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":10,"type":"book","title":"Jeon, YS, Park SC, Lim J, Chun J, Kim BS (January 4, 2015). \"Improved pipeline for reducing erroneous identification by 16S rRNA sequences using the Illumina MiSeq platform\". Journal of Microbiology. 53 (1): 60–9. doi:10.1007/s12275-015-4601-y. PMID 25557481.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":11,"type":"book","title":"Pettersson E .，Lundeberge J .，Ahmadian A. (2008年)。测序技术的产生。基因组学”，第105-111页。","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":12,"type":"book","title":"Wang, Z; Fang, B; Chen, J; Zhang, X; Luo, Z; Huang, L; Chen, X; Li, Y (Dec 24, 2010). \"De novo assembly and characterization of root transcriptome using Illumina paired-end sequencing and development of cSSR markers in sweet potato (Ipomoea batatas)\". BMC Genomics. 11: 726. doi:10.1186/1471-2164-11-726. PMC 3016421. PMID 21182800.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":13,"type":"book","title":"Hao, Da Cheng; Ge, GuangBo; Xiao, PeiGen; Zhang, YanYan; Yang, Ling; Ellegren, Hans (22 June 2011). \"The First Insight into the Tissue Specific Taxus Transcriptome via Illumina Second Generation Sequencing\". PLoS ONE. 6 (6): e21220. doi:10.1371/journal.pone.0021220. PMC 3120849. PMID 21731678.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false}],"recommendReferences":null,"auditState":2,"lemmaLevel":1,"origin":0,"originEnTitle":null,"originZhTitle":null,"pv":2332,"auditType":0,"synonyms":null,"showEditTime":"2019.12.13 18:54","auditors":[{"uid":0,"name":"Ki.κe","pic":"https://web.archive.org/web/20221025115646/https://wx.qlogo.cn/mmopen/vi_32/y67kfr32Doib4wg71Jiau7jVWvharic3nRKgdRRQSl6koeQJCo0GQs2Krw0vwdFRsOWnHIQOwAZsSg5lIkIrFCOcQ/132","introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":"","jobBrief":"","role":0,"roleName":null,"title":"","professionalTitle":null,"phoneNo":null,"editable":true,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false}],"hasZhishiNav":false,"auditInfos":{},"isHistory":false};</script><script crossorigin="anonymous" src="./1029.染料测序 - 搜狗科学百科_files/aegis.min.js.download"></script><script crossorigin="anonymous" src="./1029.染料测序 - 搜狗科学百科_files/main_2020092401.js.download"></script><script crossorigin="anonymous" src="./1029.染料测序 - 搜狗科学百科_files/react.production.min.js.download"></script><script crossorigin="anonymous" src="./1029.染料测序 - 搜狗科学百科_files/react-dom.production.min.js.download"></script><script crossorigin="anonymous" src="./1029.染料测序 - 搜狗科学百科_files/jquery-1.11.1.min.js.download"></script><script crossorigin="anonymous" src="./1029.染料测序 - 搜狗科学百科_files/main_2022062701.js.download"></script><script crossorigin="anonymous" src="./1029.染料测序 - 搜狗科学百科_files/main_66bbe21.js.download"></script><script crossorigin="anonymous" src="./1029.染料测序 - 搜狗科学百科_files/react.production.min.js.download"></script><script crossorigin="anonymous" src="./1029.染料测序 - 搜狗科学百科_files/react-dom.production.min.js.download"></script><script crossorigin="anonymous" src="./1029.染料测序 - 搜狗科学百科_files/main_edf0f08.js.download"></script>
</body></html>