<!DOCTYPE html>
<!-- saved from url=(0083)https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm -->
<html class="" data-reactroot=""><script async="false" type="text/javascript" src="chrome-extension://fnjhmkhhmkbjkkabndcnnogagogbneec/in-page.js"></script><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><script src="./1182.聚合酶链反应 - 搜狗科学百科_files/analytics.js.download" type="text/javascript"></script>
<script type="text/javascript">window.addEventListener('DOMContentLoaded',function(){var v=archive_analytics.values;v.service='wb';v.server_name='wwwb-app219.us.archive.org';v.server_ms=11484;archive_analytics.send_pageview({});});</script>
<script type="text/javascript" src="./1182.聚合酶链反应 - 搜狗科学百科_files/bundle-playback.js.download" charset="utf-8"></script>
<script type="text/javascript" src="./1182.聚合酶链反应 - 搜狗科学百科_files/wombat.js.download" charset="utf-8"></script>
<script type="text/javascript">
  __wm.init("https://web.archive.org/web");
  __wm.wombat("https://baike.sogou.com/kexue/d11182.htm","20221028220934","https://web.archive.org/","web","/_static/",
	      "1666994974");
</script>
<link rel="stylesheet" type="text/css" href="./1182.聚合酶链反应 - 搜狗科学百科_files/banner-styles.css">
<link rel="stylesheet" type="text/css" href="./1182.聚合酶链反应 - 搜狗科学百科_files/iconochive.css">
<!-- End Wayback Rewrite JS Include -->
<meta name="save" content="history"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="baidu-site-verification" content="VWGb6TyYx8"><meta content="聚合酶链反应 - 搜狗科学百科" name="keywords"><meta content="搜狗科学百科是一部有着平等、协作、分享、自由理念的网络科学全书，为每一个互联网用户创造一个涵盖所有领域知识、服务的中文知识性平台。" name="description"><meta http-equiv="x-dns-prefetch-control" content="on"><meta name="server" baike="235" ip="210" env="online"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028220934/https://cache.soso.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028220934/https://hhy.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028220934/https://pic.baike.soso.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028220934/https://ugc.qpic.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028220934/https://xui.ptlogin2.qq.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028220934/https://q1.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028220934/https://q2.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028220934/https://q3.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028220934/https://q4.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028220934/https://q.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028220934/https://img01.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028220934/https://img02.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028220934/https://img03.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028220934/https://img04.sogoucdn.com/"><link rel="Shortcut Icon" href="https://web.archive.org/web/20221028220934im_/https://www.sogou.com/images/logo/new/favicon.ico?v=4"><link rel="Bookmark" href="https://www.sogou.com/images/logo/new/favicon.ico?v=4"><link href="./1182.聚合酶链反应 - 搜狗科学百科_files/base_b849887.css" rel="stylesheet"><link href="./1182.聚合酶链反应 - 搜狗科学百科_files/detail_378aed5.css" rel="stylesheet"><link href="./1182.聚合酶链反应 - 搜狗科学百科_files/inviteAudit_7894507.css" rel="stylesheet"><link rel="stylesheet" href="./1182.聚合酶链反应 - 搜狗科学百科_files/highlight.min.css"><title>聚合酶链反应 - 搜狗科学百科</title><style>.onekey-close {
	position: absolute;
	top: 16px;
	right: 16px;
	width: 24px;
	height: 24px;
	background: url(//web.archive.org/web/20221028205211/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	text-indent: -999em;
	background-size: 84px;
	background-position: -63px 0;
}

.onekey-login {
	position: absolute;
	top: 16.4%;
	left: 0;
	right: 0;
	width: 100%;
}

/* .onekey-login-img {
    width: 75px;
    height: 75px;
    background: url("https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/images/sprite_wap_baike.png") no-repeat;
    background-size: 100px 91px;
    background-position: 0 0;
    background-repeat: no-repeat;
    margin: 0 auto;
} */

.onekey-login-title {
	text-align: center;
	padding-bottom: 3px;
	font-size: 21px;
	font-weight: bold;
	line-height: 30px;
	color: #000;
}

.onekey-login-txt {
	text-align: center;
	font-family: PingFangSC;
	font-size: 14px;
	line-height: 20px;
	color: #8f8f8f;
}

.onekey-login-qq,
.onekey-login-wx,
.onekey-login-phone {
	display: block;
	width: 245px;
	height: 54px;
	border-radius: 45px;
	text-align: center;

	margin: 0 auto;
	font-size: 17px;
	line-height: 24px;
	color: #000;
	/* padding: 16px 77px; */
	border-radius: 12px;
	border: solid 1px #e0e0e0;
}
.onekey-qq-content,
.onekey-vx-content,
.onekey-phone-content {
	display: inline-block;
	margin-top: 16px;
}
.onekey-qq-content {
	padding: 0 5px;
}

.onekey-login-qq {
	margin-top: 48px;
	margin-bottom: 24px;
}

.onekey-login-qq:before {
	display: inline-block;
	content: "";
	width: 20px;
	height: 20px;
	background: url(//web.archive.org/web/20221028205211/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	background-size: 80px;
	background-position: -20px 0;
	vertical-align: top;
	margin: 17px 8px 0 0;
}

.onekey-login-wx {
	margin-bottom: 24px;
}

.onekey-login-wx:before {
	display: inline-block;
	content: "";
	width: 21px;
	height: 21px;
	background: url(//web.archive.org/web/20221028205211/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	background-size: 84px;
	background-position: 0 0;
	vertical-align: top;
	margin: 17px 10px 0 0;
}

.onekey-login-phone {
}

.onekey-login-phone:before {
	display: inline-block;
	content: "";
	width: 21px;
	height: 21px;
	background: url(//web.archive.org/web/20221028205211/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	background-size: 84px;
	background-position: -42px 0;
	vertical-align: top;
	margin: 17px 10px 0 0;
}

.onekey-fixed {
	z-index: 100;
	position: fixed;
	top: 0;
	bottom: 0;
	left: 0;
	right: 0;
	background: #fff;
	width: 100%;
	height: 100%;
}

.onekey-fixed.forbid {
	z-index: 100;
	position: fixed;
	top: auto;
	bottom: 68px;
	left: 9%;
	right: 0;
	background: rgba(0, 0, 0, 0.7);
	width: 82%;
	height: 43px;
	border-radius: 25px;
	color: #ffffff;
}
.onekey-login-title.forbid {
	text-align: center;
	padding-bottom: 3px;
	font-size: 14px;
	font-weight: normal;
	line-height: 30px;
	color: white;
}
</style><style>#login_mask {
  background: #000;
  opacity: 0.5;
  filter: alpha(opacity=50);
  position: fixed;
  /*fixed好像在哪个IE上有BUG，先用用*/
  left: 0;
  top: 0;
  z-index: 999;
  height: 100%;
}

#login_iframe_container {
  position: fixed;
  width: 550px;
  height: 360px;
  z-index: 1020;
  background-color: #ffffff;
}

@media screen and (max-width: 828px) {
  #login_iframe_container {
    top: 50% !important;
    left: 50% !important;
    transform: translate(-50%, -50%);
  }
}

#login_iframe_container.new-login {
  width: 550px;
  height: 360px;
  background-image: url(//web.archive.org/web/20221028205211/https://hhy.sogoucdn.com/js/common/hhy/background_2a4a8a6.png);
}

#login_iframe_container.new-login.no-bg {
  background: #fff;
}

#login_iframe_container.new-login .login-title {
  width: 100%;
  height: 42px;
  line-height: 42px;
  text-align: center;
  font-size: 30px;
  letter-spacing: 0.19px;
  color: #ffffff;
  margin-top: 62px;
}
#login_iframe_container.new-login .forbid-title {
  width: 100%;
  height: 42px;
  line-height: 42px;
  text-align: center;
  font-size: 24px;
  letter-spacing: 0.19px;
  color: #333333;
  margin-top: 150px;
}

#login_iframe_container.new-login.no-bg .login-title {
  color: #333333;
}

#login_iframe_container.new-login .login-subtitle {
  width: 100%;
  height: 18px;
  line-height: 18px;
  font-size: 13px;
  letter-spacing: 0.08px;
  color: #ffffff;
  text-align: center;
  margin-top: 9px;
  margin-bottom: 43px;
}

#login_iframe_container.new-login.no-bg .login-subtitle {
  color: #999999;
}

#login_iframe_container.new-login .login-subtitle::before {
  content: '';
  display: inline-block;
  width: 10px;
  height: 1px;
  background-color: #ffffff;
  position: relative;
  top: -4px;
  left: -5px;
}

#login_iframe_container.new-login .login-subtitle::after {
  content: '';
  display: inline-block;
  width: 10px;
  height: 1px;
  background-color: #ffffff;
  position: relative;
  top: -4px;
  left: 5px;
}

#login_iframe_container.new-login.no-bg .login-subtitle::before {
  background-color: #999999;
}

#login_iframe_container.new-login.no-bg .login-subtitle::after {
  background-color: #999999;
}

#login_iframe_container.new-login .close-btn {
  position: absolute;
  top: 20px;
  right: 20px;
  width: 12px;
  height: 12px;
  background: url(//web.archive.org/web/20221028205211/https://hhy.sogoucdn.com/js/common/hhy/login-sprites_e3853e5.png) -59px -10px;
  background-size: 81px 91px;
  cursor: pointer;
}

#login_iframe_container.new-login .login-btn {
  width: 220px;
  height: 47px;
  border-radius: 24px;
  border: solid 1px #dddddd;
  background-color: #ffffff;
  margin: 0 auto;
  margin-top: 28px;
  position: relative;
  display: block;
}

#login_iframe_container.new-login .login-btn .login-icon {
  position: absolute;
}

#login_iframe_container.new-login .login-btn .login-text {
  width: 61px;
  height: 47px;
  line-height: 47px;
  vertical-align: middle;
  font-size: 15px;
  letter-spacing: 0.1px;
  color: #666666;
  position: absolute;
  right: 62px;
}

#login_iframe_container.new-login .login-btn.qq-btn .login-icon {
  width: 22px;
  height: 27px;
  top: 10px;
  left: 67px;
  background: url(//web.archive.org/web/20221028205211/https://hhy.sogoucdn.com/js/common/hhy/login-sprites_e3853e5.png) -10px -54px;
  background-size: 81px 91px;
}

#login_iframe_container.new-login .login-btn.qq-btn .login-text {
  right: 59px;
}

#login_iframe_container.new-login .login-btn.wechat-btn .login-icon {
  width: 29px;
  height: 24px;
  top: 12px;
  left: 62px;
  background: url(//web.archive.org/web/20221028205211/https://hhy.sogoucdn.com/js/common/hhy/login-sprites_e3853e5.png) -10px -10px;
  background-size: 81px 91px;
}</style><style>/* -- container -- */
.rodal,
.rodal-mask {
    top: 0;
    left: 0;
    width: 100%;
    height: 100%;
    z-index: 100;
}

.rodal {
    position: fixed;
}

/* -- mask -- */
.rodal-mask {
    position: fixed;
    background: rgba(0, 0, 0, .5);
}

/* -- dialog -- */
.rodal-dialog {
    position: absolute;
    z-index: 101;
    background: #fff;
    border-radius: 3px;
    box-shadow: 0 1px 3px rgba(0, 0, 0, .2);
}

.rodal-center {
    top: 50%;
    transform: translateY(-50%);
    left: 0;
    right: 0;
    margin: 0 auto;
}

.rodal-bottom {
    left: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

.rodal-top {
    left: 0;
    right: 0;
    top: 0;
    margin: auto;
}

.rodal-left {
    top: 0;
    left: 0;
    bottom: 0;
    margin: auto;
}

.rodal-right {
    top: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

/* -- close button -- */
.rodal-close {
    position: absolute;
    cursor: pointer;
    top: 16px;
    right: 16px;
    width: 16px;
    height: 16px;
}

.rodal-close:before,
.rodal-close:after {
    position: absolute;
    content: '';
    height: 2px;
    width: 100%;
    top: 50%;
    left: 0;
    margin-top: -1px;
    background: #999;
    border-radius: 100%;
    -webkit-transition: background .2s;
    transition: background .2s;
}

.rodal-close:before {
    -webkit-transform: rotate(45deg);
    transform: rotate(45deg);
}

.rodal-close:after {
    -webkit-transform: rotate(-45deg);
    transform: rotate(-45deg);
}

.rodal-close:hover:before,
.rodal-close:hover:after {
    background: #333;
}

/* -- fade -- */
/* @-webkit-keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

@keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

.rodal-fade-enter {
    -webkit-animation: rodal-fade-enter both ease-in;
    animation: rodal-fade-enter both ease-in;
} */

@-webkit-keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

@keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

.rodal-fade-leave {
    -webkit-animation: rodal-fade-leave both ease-out;
    animation: rodal-fade-leave both ease-out;
}

/* -- zoom -- */
@-webkit-keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-enter {
    -webkit-animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-leave {
    -webkit-animation: rodal-zoom-leave both;
    animation: rodal-zoom-leave both;
}

/* -- slideDown -- */
@-webkit-keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-enter {
    -webkit-animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-leave {
    -webkit-animation: rodal-slideDown-leave both;
    animation: rodal-slideDown-leave both;
}

/* -- slideLeft -- */
@-webkit-keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-enter {
    -webkit-animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-leave {
    -webkit-animation: rodal-slideLeft-leave both;
    animation: rodal-slideLeft-leave both;
}

/* -- slideRight -- */
@-webkit-keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-enter {
    -webkit-animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-leave {
    -webkit-animation: rodal-slideRight-leave both;
    animation: rodal-slideRight-leave both;
}

/* -- slideUp -- */
@-webkit-keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-enter {
    -webkit-animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
    animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
}

@-webkit-keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-leave {
    -webkit-animation: rodal-slideUp-leave both;
    animation: rodal-slideUp-leave both;
}

/* -- flip -- */
@-webkit-keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

@keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

.rodal-flip-enter {
    -webkit-animation: rodal-flip-enter both ease-in;
    animation: rodal-flip-enter both ease-in;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

@-webkit-keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

@keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

.rodal-flip-leave {
    -webkit-animation: rodal-flip-leave both;
    animation: rodal-flip-leave both;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

/* -- rotate -- */
@-webkit-keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-enter {
    -webkit-animation: rodal-rotate-enter both;
    animation: rodal-rotate-enter both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

@-webkit-keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-leave {
    -webkit-animation: rodal-rotate-leave both;
    animation: rodal-rotate-leave both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

/* -- door -- */
@-webkit-keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

@keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

.rodal-door-enter {
    -webkit-animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

@keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

.rodal-door-leave {
    -webkit-animation: rodal-door-leave both;
    animation: rodal-door-leave both;
}</style><style>/* -- container -- */
.rodal,
.rodal-mask {
    top: 0;
    left: 0;
    width: 100%;
    height: 100%;
    z-index: 100;
}

.rodal {
    position: fixed;
}

/* -- mask -- */
.rodal-mask {
    position: fixed;
    background: rgba(0, 0, 0, .5);
}

/* -- dialog -- */
.rodal-dialog {
    position: absolute;
    z-index: 101;
    background: #fff;
    border-radius: 3px;
    box-shadow: 0 1px 3px rgba(0, 0, 0, .2);
}

.rodal-center {
    top: 50%;
    transform: translateY(-50%);
    left: 0;
    right: 0;
    margin: 0 auto;
}

.rodal-bottom {
    left: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

.rodal-top {
    left: 0;
    right: 0;
    top: 0;
    margin: auto;
}

.rodal-left {
    top: 0;
    left: 0;
    bottom: 0;
    margin: auto;
}

.rodal-right {
    top: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

/* -- close button -- */
.rodal-close {
    position: absolute;
    cursor: pointer;
    top: 16px;
    right: 16px;
    width: 16px;
    height: 16px;
}

.rodal-close:before,
.rodal-close:after {
    position: absolute;
    content: '';
    height: 2px;
    width: 100%;
    top: 50%;
    left: 0;
    margin-top: -1px;
    background: #999;
    border-radius: 100%;
    -webkit-transition: background .2s;
    transition: background .2s;
}

.rodal-close:before {
    -webkit-transform: rotate(45deg);
    transform: rotate(45deg);
}

.rodal-close:after {
    -webkit-transform: rotate(-45deg);
    transform: rotate(-45deg);
}

.rodal-close:hover:before,
.rodal-close:hover:after {
    background: #333;
}

/* -- fade -- */
/* @-webkit-keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

@keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

.rodal-fade-enter {
    -webkit-animation: rodal-fade-enter both ease-in;
    animation: rodal-fade-enter both ease-in;
} */

@-webkit-keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

@keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

.rodal-fade-leave {
    -webkit-animation: rodal-fade-leave both ease-out;
    animation: rodal-fade-leave both ease-out;
}

/* -- zoom -- */
@-webkit-keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-enter {
    -webkit-animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-leave {
    -webkit-animation: rodal-zoom-leave both;
    animation: rodal-zoom-leave both;
}

/* -- slideDown -- */
@-webkit-keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-enter {
    -webkit-animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-leave {
    -webkit-animation: rodal-slideDown-leave both;
    animation: rodal-slideDown-leave both;
}

/* -- slideLeft -- */
@-webkit-keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-enter {
    -webkit-animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-leave {
    -webkit-animation: rodal-slideLeft-leave both;
    animation: rodal-slideLeft-leave both;
}

/* -- slideRight -- */
@-webkit-keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-enter {
    -webkit-animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-leave {
    -webkit-animation: rodal-slideRight-leave both;
    animation: rodal-slideRight-leave both;
}

/* -- slideUp -- */
@-webkit-keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-enter {
    -webkit-animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
    animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
}

@-webkit-keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-leave {
    -webkit-animation: rodal-slideUp-leave both;
    animation: rodal-slideUp-leave both;
}

/* -- flip -- */
@-webkit-keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

@keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

.rodal-flip-enter {
    -webkit-animation: rodal-flip-enter both ease-in;
    animation: rodal-flip-enter both ease-in;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

@-webkit-keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

@keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

.rodal-flip-leave {
    -webkit-animation: rodal-flip-leave both;
    animation: rodal-flip-leave both;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

/* -- rotate -- */
@-webkit-keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-enter {
    -webkit-animation: rodal-rotate-enter both;
    animation: rodal-rotate-enter both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

@-webkit-keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-leave {
    -webkit-animation: rodal-rotate-leave both;
    animation: rodal-rotate-leave both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

/* -- door -- */
@-webkit-keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

@keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

.rodal-door-enter {
    -webkit-animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

@keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

.rodal-door-leave {
    -webkit-animation: rodal-door-leave both;
    animation: rodal-door-leave both;
}</style></head><body class=""><!-- BEGIN WAYBACK TOOLBAR INSERT -->
<style type="text/css">
body {
  margin-top:0 !important;
  padding-top:0 !important;
  /*min-width:800px !important;*/
}
</style>
<script>__wm.rw(0);</script>
<div id="wm-ipp-base" lang="en" style="display: block; direction: ltr;">
</div><div id="wm-ipp-print">The Wayback Machine - https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm</div>
<script type="text/javascript">//<![CDATA[
__wm.bt(675,27,25,2,"web","https://baike.sogou.com/kexue/d11182.htm","20221028220934",1996,"/_static/",["/_static/css/banner-styles.css?v=S1zqJCYt","/_static/css/iconochive.css?v=qtvMKcIJ"], false);
  __wm.rw(1);
//]]></script>
<!-- END WAYBACK TOOLBAR INSERT --><script>window._gtag=window._gtag||{};window._gtag.shouldGrayed = false;if ('151e3e6b1e5f444b99e39a45f5cb8cd0') window._gtag.traceId = '151e3e6b1e5f444b99e39a45f5cb8cd0';if ({"illegality":true}) window.userInfo = {"illegality":true};</script><div class="topnavbox"><ul class="topnav"><li><a href="https://web.archive.org/web/20221028220934/https://www.sogou.com/web?query=">网页</a></li><li><a href="https://web.archive.org/web/20221028220934/https://weixin.sogou.com/weixin?p=75351201">微信</a></li><li><a href="https://web.archive.org/web/20221028220934/https://zhihu.sogou.com/zhihu?p=75351218">知乎</a></li><li><a href="https://web.archive.org/web/20221028220934/https://pic.sogou.com/pics?query=">图片</a></li><li><a href="https://web.archive.org/web/20221028220934/https://v.sogou.com/v?query=">视频</a></li><li><a href="https://web.archive.org/web/20221028220934/https://mingyi.sogou.com/">医疗</a></li><li class="cur"><strong>科学</strong></li><li><a href="https://web.archive.org/web/20221028220934/https://hanyu.sogou.com/">汉语</a></li><li><a href="https://web.archive.org/web/20221028220934/https://wenwen.sogou.com/">问问</a></li><li><a href="https://web.archive.org/web/20221028220934/https://www.sogou.com/docs/more.htm">更多<span class="topraquo">»</span></a></li></ul></div><div id="header"><div class="header-wrap"><a class="header-logo" href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue"></a><div class="header-search"><div class="querybox" id="suggBox"><form><input id="searchInput" class="query" type="text" placeholder="搜科学领域专业百科词条" name="query" autocomplete="off" value="" control-id="ControlID-1"><a href="javascript:;" class="query-search"></a></form></div></div><div class="header-rgt"><span class="btn-header-rgt btn-edit" id="editLemma">创建</span><div class="header-user no-login"></div></div></div></div><div class="fixed-placeholder" style="visibility:none"></div><div id="container" class=""><div class="content lemma-level1"><div class="detail-title" id="abstract-title"><h1>聚合酶链反应</h1><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#!" class="detail-edit">编辑</a></div><div class="section_content" data-id="54234671596233220"><div class="text_img ed_imgfloat_right"><a class="ed_image_link" data-src="https://img01.sogoucdn.com/app/a/200698/sogou_science_17113" data-bigsrc="https://img01.sogoucdn.com/app/a/200698/sogou_science_17113?width=231&amp;height=174&amp;titlename=%E4%B8%80%E6%9D%A1%E5%85%AB%E4%B8%AA%E8%81%9A%E5%90%88%E9%85%B6%E9%93%BE%E5%8F%8D%E5%BA%94%E8%AF%95%E7%AE%A1%EF%BC%8C%E6%AF%8F%E4%B8%AA%E8%AF%95%E7%AE%A1%E5%8C%85%E5%90%AB%E4%B8%80%E4%B8%AA100%20%CE%BCl%E5%8F%8D%E5%BA%94%E6%B7%B7%E5%90%88%E7%89%A9&amp;w=300&amp;h=246" title="点击查看大图" data-w="300" data-h="246" style="background-image:url(https://web.archive.org/web/20221028220934im_/https://img01.sogoucdn.com/app/a/200698/sogou_science_17113)" href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#!"></a><div class="text_img_title">一条八个聚合酶链反应试管，每个试管包含一个100 μl反应混合物</div></div><div><p>聚合酶链式反应（英文：Polymerase chain reaction，缩写：PCR，又称多聚酶链式反应），是一项利用DNA双链复制的原理，在生物体外复制特定DNA片段的核酸合成技术。透过这项技术，可在短时间内大量扩增目的基因，而不必依赖大肠杆菌或酵母菌等生物体。</p>
<p>聚合酶链反应是由凯利·穆利斯（Kary Mullis）于1983年开发，当时他是Cetus公司的雇员，也是1993年诺贝尔化学奖的获得者，它是一种简单，廉价和可靠的方法复制DNA片段，这个概念适用于现代生物学和相关科学的许多领域。 PCR可能是分子生物学中使用最广泛的技术。这种技术被用于生物医学研究，犯罪取证和分子考古学</p>
<p>绝大多数聚合酶链反应方法依赖于热循环。热循环将反应物暴露于加热和冷却的重复循环中，以允许不同的温度依赖性反应——具体地说，脱氧核糖核酸融化和酶-驱动DNA复制。聚合酶链反应使用两种主要试剂-引物(引物是短的单链DNA片段，称为寡核苷酸，是目标DNA区域的互补序列)和DNA聚合酶。在PCR反应的第一步，DNA双螺旋结构的两条链在高温下物理分离，这个过程称为脱氧核糖核酸变性。第二步，降低温度，引物与互补的脱氧核糖核酸序列结合。这两条DNA链就变成了模板，以酶促的方式从构成DNA的自由核苷酸中组装出一条新的DNA链。随着聚合酶链反应的进行，产生的DNA本身被用作复制的模板，启动了一个连锁反应，原始的DNA模板是以指数形式放大。</p></div></div><div id="catalog"><h2 class="title2">目录<a href="javascript:" class="detail-edit">编辑</a></h2><div class="catalog_wrap" style=""><ul class="catalog_list col3"><li><span class="order">1</span><a href="javascript:" data-level="1" data-id="14996399716303369">原则</a></li><li class="secondary_catalog"><span>1.1 </span><a href="javascript:" data-id="14996399716303369">程序</a></li><li class="secondary_catalog"><span>1.2 </span><a href="javascript:" data-id="14996399716303369">阶段</a></li><li><span class="order">2</span><a href="javascript:" data-level="1" data-id="14996399716303370">优化</a></li><li><span class="order">3</span><a href="javascript:" data-level="1" data-id="14996399716303371">应用程序</a></li><li class="secondary_catalog"><span>3.1 </span><a href="javascript:" data-id="14996399716303371">选择性脱氧核糖核酸分离</a></li></ul><ul class="catalog_list col3"><li class="secondary_catalog"><span>3.2 </span><a href="javascript:" data-id="14996399716303371">DNA的扩增和定量</a></li><li class="secondary_catalog"><span>3.3 </span><a href="javascript:" data-id="14996399716303371">医疗和诊断应用</a></li><li class="secondary_catalog"><span>3.4 </span><a href="javascript:" data-id="14996399716303371">传染病应用</a></li><li class="secondary_catalog"><span>3.5 </span><a href="javascript:" data-id="14996399716303371">法医应用</a></li><li class="secondary_catalog"><span>3.6 </span><a href="javascript:" data-id="14996399716303371">研究应用</a></li><li><span class="order">4</span><a href="javascript:" data-level="1" data-id="14996399716303372">优势</a></li></ul><ul class="catalog_list col3"><li><span class="order">5</span><a href="javascript:" data-level="1" data-id="14996399733080591">限制</a></li><li><span class="order">6</span><a href="javascript:" data-level="1" data-id="14996399733080592">变化</a></li><li><span class="order">7</span><a href="javascript:" data-level="1" data-id="14996399733080593">历史</a></li><li class="secondary_catalog"><span>7.1 </span><a href="javascript:" data-id="14996399733080593">专利纠纷</a></li><li><span class="order">8</span><a href="javascript:" data-level="1" data-id="references">参考文献</a></li></ul></div></div><div id="paragraphs"><div><div id="par_14996399716303369"><h2 class="title">1 原则<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p></p><div class="text_img ed_imgfloat_right">
            <a class="ed_image_link lazyLoad" data-src="https://img02.sogoucdn.com/app/a/200698/sogou_science_17115" data-bigsrc="" title="点击查看大图" href="javascript:" data-observer="true"></a>
            <div class="text_img_title" style="text-align: center;">聚合酶链反应的热循环仪</div>   
        </div> <p></p><p></p><div class="text_img ed_imgfloat_right">
            <a class="ed_image_link lazyLoad" data-src="https://img03.sogoucdn.com/app/a/200698/sogou_science_17116" data-bigsrc="" title="点击查看大图" href="javascript:" data-observer="true"></a>
            <div class="text_img_title">一种用于聚合酶链反应的老式三温热循环仪</div>   
        </div> <p></p><p>PCR扩增DNA链的特定区域(DNA靶)。大多数聚合酶链反应方法扩增长度在0.1至10千碱基对（kbp）之间的DNA片段，尽管一些技术允许扩增高达40 kbp的片段。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_1" class="kx_ref">[1]</a></sup>扩增产物的量由反应中可用的底物决定，随着反应的进行，底物的数量变得有限。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_2" class="kx_ref">[2]</a></sup> </p><p>基本的聚合酶链反应设置需要几种组分和试剂，包括包含要扩增的DNA目标区域的<i>DNA模板</i>；<i> DNA聚合酶</i>；一种聚合新DNA链的酶；耐热的 Taq聚合酶特别常见，<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_3" class="kx_ref">[3]</a></sup>因为它更有可能在高温DNA变性过程中保持完整；两个DNA<i>引物分别</i>与DNA靶的每条正义链和反义链的 3’末端互补(DNA聚合酶只能与DNA双链区域结合并从双链区域延伸；没有引物，就没有聚合酶可以结合的双链起始位点)；<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_4" class="kx_ref">[4]</a></sup>预先选择与DNA靶区互补的特异性引物，通常在实验室定制或从商业生化供应商处购买；<i>脱氧核苷三磷酸</i>，或dNTPs(有时称为“脱氧核苷酸三磷酸”；核苷酸含有三磷酸基团)，DNA聚合酶从中合成新的DNA链的构建模块；a<i>缓冲溶液</i>为DNA聚合酶的最佳活性和稳定性提供合适的化学环境；<i>二价 阳离子</i>通常为镁或锰离子；其中镁离子是最常见的，但是Mn<sup>2+</sup>可作为高锰酸盐用于PCR介导的DNA诱变， 锰离子浓度会增加DNA合成过程中的错误率；<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_5" class="kx_ref">[5]</a></sup>其中<i>单价阳离子</i>通常为钾离子</p><p>PCR反应通常在10-200&nbsp;μL体积内进行，在小反应管中(0.2-0.5&nbsp;μL体积)放入热循环仪中。热循环仪加热和冷却反应管，以达到反应中每个步骤所需的温度(见下文)。许多现代热循环仪利用珀耳帖效应，该效应允许通过简单地逆转电流来加热和冷却PCR管。薄壁反应管允许良好的导热性，以实现快速热平衡。大多数热循环仪都有加热的盖子，以防止反应管顶部冷凝。没有加热盖的老式热循环仪需要在反应混合物顶部涂一层油，或者在管内涂一团蜡。</p><p></p><h3>1.1 <span>程序</span></h3><p></p><p>通常，PCR由一系列20-40次重复的温度变化组成，称为热循环，每个循环通常由两个或三个离散的温度步骤组成(见下图)。循环通常在非常高的温度下进行单个温度步骤(&gt;90 degrees Celsius (194 degrees Fahrenheit))，然后在最后保持一次，以便最终产品扩展或短暂存储。在每个循环中使用的温度和时间取决于各种参数，包括用于DNA合成的酶、反应中二价离子和dNTPs的浓度以及引物的熔解温度(<i>Tm</i>)。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_6" class="kx_ref">[6]</a></sup>大多数聚合酶链反应方法共有的步骤如下:</p><p>
 </p><ul>
  <li><i>初始化</i>:此步骤仅适用于需要热激活的DNA聚合酶热启动聚合酶链反应。它包括将反应室加热至94–96 °C (201–205 °F)，或 98 °C (208 °F)，如果使用极端耐热的聚合酶，然后保持1-10分钟。</li>
  <li><i>变性</i>:该步骤是第一次常规循环活动，包括将反应室加热至94-98°C(201-208°F) 持续20-30秒。通过破坏互补碱基之间的氢键，导致脱氧核糖核酸熔解或变性，产生两个单链DNA分子。</li>
  <li><i>退火</i>:在下一步中，反应温度降低到50-65℃(122-149°F)持续20-40秒，使每个单链DNA模板上的引物退火。反应混合物中通常包含两种不同的引物:包含靶区的两种单链互补物各一种。引物本身是单链序列，但比靶区的长度短得多，只能补充每个链3'端非常短的序列。</li>
 </ul><p></p><p>
 </p><dl>
  <dd>
   确定合适的退火温度至关重要，因为退火温度对影响效率和特异性有很大影响。该温度必须足够低以允许引物与链的杂交，但是足够高以使杂交具有特异性，即引物应该 <i>仅仅与链中</i>完全互补的部分结合，而不是其他地方。如果温度过低，引物可能结合不完全。如果太高，引物可能根本不会结合。典型的退火温度大约在引物 <i>Tm</i>以下3–5 °C。只有当引物序列与模板序列非常接近时，互补碱基之间才会形成稳定的氢键。在这一步骤中，聚合酶结合引物-模板杂交体，并开始DNA形成。
  </dd>
 </dl><p></p><p>
 </p><ul>
  <li><i>延伸/伸长</i>:这一步的温度取决于所用的DNA聚合酶； Taq(水生栖热菌)聚合酶耐热DNA聚合酶的最佳活性温度约为75-80°C(167-176°F),尽管该酶通常使用72°C(162°F)的温度。在该步骤中，DNA聚合酶通过从反应混合物中添加在5’-至-3’方向上与模板互补的游离DNA链来合成与DNA模板链互补的新DNA链，在新生(伸长)DNA链的末端，将dNTPs的5'-磷酸基与3'-羟基缩合。延伸所需的精确时间取决于所使用的DNA聚合酶和要扩增的DNA目标区域的长度。根据经验，在最佳温度下，大多数DNA聚合酶每分钟聚合一千个碱基。在最佳条件下(即，如果没有限制底物或试剂的限制)，在每个延伸/延伸步骤中，DNA靶序列的数量加倍。随着每一个连续的循环，原始模板链加上所有新产生的链成为下一轮延伸的模板链，导致特定DNA目标区域的指数(几何)扩增。</li>
 </ul><p></p><p>
 </p><dl>
  <dd>
   变性、退火和伸长的过程构成一个单一的循环。将DNA目标扩增到数百万个拷贝需要多个周期。用于计算给定循环次数后形成的DNA拷贝数的公式是2 <sup>n</sup>，其中 <i>n</i>是周期数。因此，30个循环的反应集产生2个 <sup>30</sup>，或1073741824原始双链DNA目标区域的拷贝。
  </dd>
 </dl><p></p><p>
 </p><ul>
  <li><i>最终伸长率</i>:此单一步骤是可选的，但在70 - 74°C的温度(158 - 165°F)下执行(PCR中大多数聚合酶最佳活性所需的温度范围)在最后一个PCR循环后5-15分钟，以确保任何剩余的单链DNA被完全延长。</li>
  <li><i>最终保持</i>:最后一步将反应室冷却至4–15 °C (39–59 °F)，并且可以用于PCR产物的短期储存。</li>
 </ul><p></p><p></p><div class="text_img ed_imgfloat_embedmiddle">
            <a class="ed_image_link" data-src="https://img04.sogoucdn.com/app/a/200698/sogou_science_17117" data-bigsrc="" title="点击查看大图" href="javascript:">
                <img alt="完整聚合酶链反应循环示意图" class="ed_imgfloat_embedmiddle" img_height="435" img_width="835" titlename="" data-align="embedmiddle" width="480" data-src="https://img04.sogoucdn.com/app/a/200698/sogou_science_17117" src="./1182.聚合酶链反应 - 搜狗科学百科_files/sogou_science_17117" data-observer="true">
            </a>
            <div class="text_img_title" style="text-align: center;"></div>
        </div> <p></p><p></p><div class="text_img ed_imgfloat_embedmiddle">
            <a class="ed_image_link" data-src="https://img03.sogoucdn.com/app/a/200698/sogou_science_17118" data-bigsrc="" title="点击查看大图" href="javascript:">
                <img alt="" class="ed_imgfloat_embedmiddle" img_height="283" img_width="300" titlename="溴化乙锭-染色后的聚合酶链反应产物凝胶电泳。两组引物用于扩增来自三种不同组织样品的靶序列。样品#1中不存在扩增；样品#2和#3中的DNA条带表明目标序列成功扩增。凝胶还显示出阳性对照，以及包含定义长度的DNA片段的DNA梯子，用于确定实验性PCRs中条带的大小。" data-align="embedmiddle" width="300" data-src="https://img03.sogoucdn.com/app/a/200698/sogou_science_17118" src="./1182.聚合酶链反应 - 搜狗科学百科_files/sogou_science_17118" data-observer="true">
            </a>
            <div class="text_img_title">溴化乙锭-染色后的聚合酶链反应产物凝胶电泳。两组引物用于扩增来自三种不同组织样品的靶序列。样品#1中不存在扩增；样品#2和#3中的DNA条带表明目标序列成功扩增。凝胶还显示出阳性对照，以及包含定义长度的DNA片段的DNA梯子，用于确定实验性PCRs中条带的大小。</div>
        </div> <p></p><p>为了检查聚合酶链反应是否成功产生预期的目的DNA(有时也称为扩增子或扩增子)，可以使用琼脂糖凝胶电泳进行聚合酶链反应产物的大小分离。聚合酶链反应产物的大小是通过与 DNA梯比较来确定的，DNA梯是一种分子量标记，包含与聚合酶链反应产物一起在凝胶上运行的已知大小的DNA片段。</p><p></p><div class="text_img ed_imgfloat_embedmiddle">
            <a class="ed_image_link" data-src="https://img03.sogoucdn.com/app/a/200698/sogou_science_17119" data-bigsrc="" title="点击查看大图" href="javascript:">
                <img alt="塔克聚合酶链反应" class="ed_imgfloat_embedmiddle" img_height="707" img_width="773" titlename="" data-align="embedmiddle" width="480" data-src="https://img03.sogoucdn.com/app/a/200698/sogou_science_17119" src="./1182.聚合酶链反应 - 搜狗科学百科_files/sogou_science_17119" data-observer="true">
            </a>
            <div class="text_img_title" style="text-align: center;"></div>
        </div> <p></p><p></p><h3>1.2 <span>阶段</span></h3><p></p><p>与其他化学反应一样，聚合酶链反应的反应速率和效率受到限制因素的影响。因此，整个聚合酶链反应过程可以根据反应进程进一步分为三个阶段:</p><p>
 </p><ul>
  <li><i>指数放大</i>:在每个循环中，产物的量加倍(假设100%反应效率)。经过30个周期后，DNA的单个拷贝可以增加到10亿个拷贝。从某种意义上说，一条离散DNA链的复制是在试管中于受控条件下进行的。该反应非常敏感:只需要少量的DNA。</li>
  <li>稳定阶段当DNA聚合酶失去活性时，以及DNA聚合酶和引物等试剂的消耗会导致他们变得更加有限，反应会变慢。</li>
  <li><i>平台期</i>:由于试剂和酶耗尽，不再有产品积累。</li>
 </ul><p></p></div></div><div id="par_14996399716303370"><h2 class="title">2 优化<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>实际上，PCR可能由于各种原因失败，部分原因是它对污染的敏感性导致虚假DNA产物的扩增。因此，已经开发了许多技术和程序来优化聚合酶链反应条件。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_7" class="kx_ref">[7]</a></sup><sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_8" class="kx_ref">[8]</a></sup>外来DNA的污染通过实验室协议和程序来解决，这些协议和程序将PCR前混合物与潜在的DNA污染物分开。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_9" class="kx_ref">[9]</a></sup>这通常包括将聚合酶链式反应设置区域与聚合酶链式反应产品的分析或纯化区域空间分离，使用一次性塑料制品，以及彻底清洁反应设置之间的工作表面。引物设计技术对于提高PCR产物产量和避免假产物的形成非常重要，使用替代缓冲组分或聚合酶可以帮助扩增DNA的长区域或其他有问题的区域。在缓冲体系中加入试剂，如甲酰胺，可以提高聚合酶链反应的特异性和产率。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_9" class="kx_ref">[9]</a></sup>可以对理论聚合酶链反应结果进行计算机模拟(电子聚合酶链反应)，以帮助引物设计。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_10" class="kx_ref">[10]</a></sup> </p></div></div><div id="par_14996399716303371"><h2 class="title">3 应用程序<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p></p><h3>3.1 <span>选择性脱氧核糖核酸分离</span></h3><p></p><p>PCR允许从基因组DNA中通过选择性扩增DNA的特定区域分离DNA片段。聚合酶链反应的使用增加了许多方法，例如用于Southern或 Northern杂交的杂交探针和 DNA克隆，这需要更大量的DNA，代表特定的DNA区域。PCR为这些技术提供了大量的纯DNA，使得即使从非常少量的原始材料中也能够分析DNA样品。</p><p>聚合酶链反应的其他应用包括脱氧核糖核酸测序以确定未知的聚合酶链反应扩增序列，其中一个扩增引物可用于Sanger测序、分离脱氧核糖核酸序列以加速重组脱氧核糖核酸技术，该技术涉及将脱氧核糖核酸序列插入到质粒、噬菌体或粘粒(取决于大小)或另一生物体的遗传物质中。细菌菌落<i>(如大肠杆菌)</i>可以通过聚合酶链反应快速筛选正确的DNA 载体构建体。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_11" class="kx_ref">[11]</a></sup>聚合酶链反应也可用于遗传指纹；一种法医技术，用于通过不同的PCR方法比较实验性DNA来鉴定人或其他生物。</p><p>一些聚合酶链反应的“指纹”方法具有很高的辨别能力，可以用于识别个体之间的遗传关系，例如亲子关系或兄弟姐妹之间的遗传关系，并用于亲子鉴定(图4)。当使用特定的分子钟(即微生物的16S rRNA 和recA基因)时，该技术也可以用于确定生物体之间的进化关系。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_12" class="kx_ref">[12]</a></sup> </p><p></p><div class="text_img ed_imgfloat_embedmiddle">
            <a class="ed_image_link" data-src="https://img02.sogoucdn.com/app/a/200698/sogou_science_17120" data-bigsrc="" title="点击查看大图" href="javascript:">
                <img alt="" class="ed_imgfloat_embedmiddle" img_height="301" img_width="200" titlename="聚合酶链反应扩增的DNA片段的电泳。(1)父亲。(2)儿童。(3)母亲。这个孩子继承了父母的一些指纹，但不是全部，给了它一个新的独特的指纹。" data-align="embedmiddle" width="200" data-src="https://img02.sogoucdn.com/app/a/200698/sogou_science_17120" src="./1182.聚合酶链反应 - 搜狗科学百科_files/sogou_science_17120" data-observer="true">
            </a>
            <div class="text_img_title">聚合酶链反应扩增的DNA片段的电泳。(1)父亲。(2)儿童。(3)母亲。这个孩子继承了父母的一些指纹，但不是全部，给了它一个新的独特的指纹。</div>
        </div> <p></p><p></p><h3>3.2 <span>DNA的扩增和定量</span></h3><p></p><p>因为PCR扩增了目的DNA区域，所以PCR可以用于分析极少量的样品。这对于法医检定法，当时只有微量的DNA可作为证据。聚合酶链反应也可用于分析古代DNA那已经有数万年的历史了。这些基于聚合酶链反应的技术已经成功地应用于动物身上，例如四万年前猛犸，以及人类DNA，应用范围从分析埃及木乃伊识别一个俄罗斯沙皇和英国国王理查三世遗体的鉴定。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_13" class="kx_ref">[13]</a></sup> </p><p>定量聚合酶链反应或者实时聚合酶链反应<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_14" class="kx_ref">[14]</a></sup>不要与逆转录-聚合酶链反应方法混淆)允许估计样品中存在的给定序列的量——这种技术通常用于定量确定基因表达的水平。定量PCR是一种已建立的DNA定量工具，用于测量每轮PCR扩增后DNA产物的积累。</p><p>定量PCR允许实时定量和检测特定的DNA序列，因为它在合成过程中测量浓度。有两种方法可以同时检测和定量。第一种方法是使用在DNA双链之间非特异性保留的荧光染料。第二种方法涉及编码特定序列并被荧光标记的探针。只有在探针与其互补DNA杂交后，才能使用这些方法检测DNA。一种有趣的技术组合是实时PCR和逆转录。这种复杂的技术被称为RT-qPCR，允许定量少量RNA。通过这种组合技术，mRNA被转化为cDNA，并用qPCR进一步定量。这种技术降低了聚合酶链反应终点出错的可能性，<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_15" class="kx_ref">[15]</a></sup>增加了检测与遗传疾病如癌症相关的基因的机会。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_16" class="kx_ref">[16]</a></sup>实验室使用RT-qPCR来灵敏地测量基因调控。</p><p></p><h3>3.3 <span>医疗和诊断应用</span></h3><p></p><p>未来的父母可以被测试是否是遗传携带者，或者他们的孩子可以被测试是否真的受到了疾病的影响。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_16" class="kx_ref">[16]</a></sup>用于产前检测的DNA样品可以通过羊膜穿刺术、绒毛取样或甚至通过分析在母亲血流中循环的稀有胎儿细胞获得。聚合酶链反应分析对于植入前遗传诊断也是至关重要的，在植入前遗传诊断中，对发育中胚胎的单个细胞进行突变测试。</p><p>
 </p><ul>
  <li>聚合酶链反应也可以用作以下敏感试验的一部分<i>组织分型</i>，对器官移植至关重要。截至2008年，甚至有人提议用聚合酶链反应检测取代传统的血型抗体检测。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_16" class="kx_ref">[16]</a></sup></li>
  <li>许多形式的癌症涉及致癌基因的改变。通过使用基于聚合酶链反应的测试来研究这些突变，治疗方案有时可以根据患者的不同而不同。聚合酶链反应可以早期诊断恶性疾病，如白血病和淋巴瘤，这是目前癌症研究中发展最快的，并且已经被常规使用。PCR检测可以直接在基因组DNA样品上进行，以检测易位特异性恶性细胞，其灵敏度至少是其他方法的10，000倍。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_17" class="kx_ref">[17]</a></sup>聚合酶链反应在医学领域非常有用，因为它允许分离和扩增肿瘤抑制剂。例如，定量PCR可以用于量化和分析单细胞，以及识别DNA、mRNA和蛋白质的确认和组合。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_15" class="kx_ref">[15]</a></sup></li>
 </ul><p></p><p></p><h3>3.4 <span>传染病应用</span></h3><p></p><p>聚合酶链反应可以快速和高度特异性地诊断传染疾病病，包括由细菌或病毒引起的疾病。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_18" class="kx_ref">[18]</a></sup>聚合酶链反应还允许从组织培养试验和动物模型中鉴定不可培养或生长缓慢的微生物，例如分枝杆菌、厌氧菌或病毒。聚合酶链反应在微生物学中诊断应用的基础是检测感染因子和借助特定基因从病原菌株中区分非病原性。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_18" class="kx_ref">[18]</a></sup><sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_19" class="kx_ref">[19]</a></sup> </p><p>通过PCR，传染病生物体的表征和检测已经在以下方面发生了革命性的变化:</p><p>
 </p><ul>
  <li>这<i>人类免疫缺陷病毒</i>(或<i>艾滋病毒</i>)，是一个很难找到和根除的目标。最早的感染测试依赖于血液中循环的病毒抗体的存在。然而，抗体直到感染后许多周才会出现，母体抗体掩盖了新生儿的感染，而对抗感染的治疗药物不影响抗体。PCR 测试已经被开发出来，它可以检测超过50，000个宿主细胞的DNA中的一个病毒基因组。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_20" class="kx_ref">[20]</a></sup>可以更早地检测感染，可以直接筛查捐赠的血液中的病毒，可以立即检测新生儿的感染，并且可以对抗病毒治疗的效果进行量化。</li>
  <li>一些疾病生物，如<i>肺结核</i>，很难从病人身上取样，而且在实验室里生长缓慢。基于聚合酶链反应的检测可以方便地检测少量疾病生物(活的或死的)样品。详细的遗传分析也可以用来检测抗生素耐药性，从而允许立即有效的治疗。治疗效果也可以立即评估。</li>
  <li>可通过PCR检测监测疾病病菌通过家畜或野生动物种群的传播。在许多情况下，可以检测和监测新毒性亚型的出现。也可以通过聚合酶链反应分析来确定导致早期流行的生物体的亚型。</li>
  <li>病毒DNA可以通过PCR检测。所使用的引物必须对病毒DNA中的目标序列具有特异性，并且PCR可以用于病毒基因组的诊断分析或DNA测序。聚合酶链反应的高灵敏度允许在感染后不久甚至在疾病开始之前检测病毒。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_18" class="kx_ref">[18]</a></sup>这种早期发现可能会给医生治疗带来很长的准备时间。患者体内的病毒量(“病毒载量”)也可以通过基于聚合酶链反应的DNA定量技术进行定量(见下文)。</li>
  <li>百日咳等疾病是由细菌百日咳杆菌引起的。这种细菌的特点是严重的急性呼吸道感染，影响到各种动物和人类，并导致许多幼儿死亡。百日咳毒素是一种蛋白质外毒素，通过两个二聚体与细胞受体结合，并与在细胞免疫中发挥作用的不同细胞类型如T淋巴细胞反应。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_21" class="kx_ref">[21]</a></sup>聚合酶链反应是一种重要的检测工具，可以检测百日咳毒素基因中的序列。这是因为聚合酶链反应对毒素有很高的灵敏度，并显示出快速的周转时间。与常规培养相比，聚合酶链反应对百日咳的诊断非常有效。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_22" class="kx_ref">[22]</a></sup></li>
 </ul><p></p><p></p><h3>3.5 <span>法医应用</span></h3><p></p><p>基于聚合酶链反应的遗传(或脱氧核糖核酸)指纹图谱协议的发展已在法医学中得到广泛应用:</p><p>
 </p><ul>
  <li><i>遗传指纹</i>以其最具辨别力的形式，可以从世界上所有人群中独一无二地区分任何一个人。微小的DNA样本可以从犯罪现场，和比较的从嫌疑人那里，或者从DNA资料库早期的证据或罪犯。这些测试的简单版本通常用于在刑事调查中快速排除嫌疑人。几十年前的犯罪证据可以被检验，确认或免责最初被定罪的人。</li>
  <li>通过分析犯罪现场发现的证据，法医DNA分型已经成为鉴定犯罪嫌疑人或为其开脱的有效方法。人类基因组有许多重复区域，可以在基因序列中或基因组的非编码区域中发现。具体来说，高达40%的人类DNA是重复的。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_16" class="kx_ref">[16]</a></sup>基因组中这些重复的非编码区域有两种不同的类别。第一类称为可变数目串联重复序列(VNTR)，长度为10-100个碱基对，第二类称为短串联重复序列(STR)，由重复的2-10个碱基对部分组成。PCR用于使用位于每个重复区域侧翼的引物扩增几个众所周知的VNTRs和str。从任何个体获得的每个str片段的大小将表明存在哪些等位基因。通过分析一个人的几个str，会发现每个人的一组等位基因在统计上可能是唯一的。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_16" class="kx_ref">[16]</a></sup>研究人员已经确定了人类基因组的完整序列。该序列可以通过NCBI网站轻松访问，并用于许多实际应用中。例如，联邦调查局已经汇编了一组用于识别的DNA标记位点，这些位点被称为组合DNA索引系统(CODIS) DNA数据库。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_16" class="kx_ref">[16]</a></sup>使用该数据库可以使用统计分析来确定DNA样品匹配的概率。PCR是一种非常强大和重要的分析工具，用于法医DNA分型，因为研究人员只需要非常少量的目标DNA用于分析。例如，一根附着毛囊的人的头发有足够的DNA进行分析。类似地，一些精子、指甲下的皮肤样本或少量血液可以为结论性分析提供足够的DNA。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_16" class="kx_ref">[16]</a></sup></li>
  <li>脱氧核糖核酸指纹中较小的区别有助于<i>亲子鉴定，在亲子鉴定中，</i>一个人和他的近亲相匹配。可以测试未知人类遗骸的DNA，并与可能的父母、兄弟姐妹或儿童进行比较。类似的测试可以用来确认被收养(或绑架)孩子的亲生父母。新生儿的实际生父也可以被确认(或排除)。</li>
  <li>PCR AMGX/AMGY设计已被证明不仅有助于从非常少量的基因组中扩增DNA序列。然而，它也可以用于法医骨骼样本的实时性别确定。这为我们提供了一种强大而有效的方法，不仅可以确定古代标本的性别，还可以确定当前犯罪嫌疑人的性别。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_23" class="kx_ref">[23]</a></sup></li>
 </ul><p></p><p></p><h3>3.6 <span>研究应用</span></h3><p></p><p>PCR已经应用于分子遗传学的许多研究领域:</p><p>
 </p><ul>
  <li>聚合酶链反应允许快速生产短DNA片段，即使已知的引物序列不超过两个。聚合酶链反应的这种能力增强了许多方法，例如生成<i>杂交 探针</i>用于 Southern 或northern blot 杂交。聚合酶链反应为这些技术提供了大量的纯DNA，有时是单链的，甚至可以从非常少量的起始材料中进行分析。</li>
  <li><i>脱氧核糖核酸测序</i>的任务也可以通过聚合酶链反应来辅助完成。已知的DNA片段可以很容易地从患有遗传疾病突变的患者体内产生。对扩增技术的修饰可以从完全未知的基因组中提取片段，或者可以仅产生感兴趣区域的单链。</li>
  <li>聚合酶链反应有许多应用于传统的<i> DNA克隆</i>。它可以从更大的基因组中提取片段以插入载体，这可能只有少量可用。使用一组“载体引物”，它还可以分析或提取已经插入载体的片段。聚合酶链反应方案的一些改变可以<i>产生突变</i>(通用的或定点的)插入片段。</li>
  <li><i>序列标签站点</i>是一个过程，其中PCR被用作基因组的特定片段存在于特定克隆中的指示物。人类基因组计划发现这一应用对于绘制他们测序的粘粒克隆以及协调不同实验室的结果至关重要。</li>
  <li>聚合酶链反应的一个令人兴奋的应用是对来自远古来源的DNA进行系统进化分析，例如在尼安德特人的复原骨骼中、从猛犸的冷冻组织中或从埃及木乃伊的大脑中发现的DNA已经被放大和测序。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_24" class="kx_ref">[24]</a></sup>在某些情况下，这些来源的高度降解的DNA可能在扩增的早期阶段重新组装。</li>
  <li>聚合酶链反应的一个常见应用是对以下<i>基因表达模式的研究</i>。组织(甚至单个细胞)可以在不同阶段进行分析，以了解哪些基因变得活跃，哪些基因被关闭。该应用还可以使用定量聚合酶链反应来定量表达的实际水平</li>
  <li>聚合酶链反应同时扩增单个精子中几个基因座的能力<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_24" class="kx_ref">[24]</a></sup>大大增强了极大地增强了通过研究减数分裂后染色体交叉来进行基因定位的传统任务。通过分析数千个单个精子，已经直接观察到非常紧密基因座之间罕见的交叉事件。类似地，可以分析异常的缺失、插入、易位或倒位，所有这些都无需等待(或支付)漫长而艰苦的受精、胚胎发生等过程。</li>
  <li>定点突变:聚合酶链反应可用于产生突变基因，突变由科学家随意选择。可以选择这些突变来理解蛋白质是如何完成其功能的，并改变或改善蛋白质功能。</li>
 </ul><p></p></div></div><div id="par_14996399716303372"><h2 class="title">4 优势<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>聚合酶链反应有许多优点。它很容易理解和使用，并迅速产生结果。这项技术非常敏感，有可能产生数百万到数十亿份特定产品的拷贝，用于测序、克隆和分析。qRT-PCR具有与PCR相同的优点，同时还具有定量合成产物的优点。因此，它可以用于分析肿瘤、微生物或其他疾病状态中基因表达水平的变化。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_25" class="kx_ref">[25]</a></sup> </p><p>聚合酶链反应是一种非常强大和实用的研究工具。许多疾病的未知病因的测序正在通过聚合酶链反应得到解决。这项技术可以帮助我们识别与已知病毒相关的未知病毒序列，从而让我们更好地了解疾病本身。如果该过程可以进一步简化，并且可以开发灵敏的非辐射检测系统，PCR将在未来几年在临床实验室中占据显著地位。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_26" class="kx_ref">[26]</a></sup> </p></div></div><div id="par_14996399733080591"><h2 class="title">5 限制<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>聚合酶链反应的一个主要限制是，为了产生允许其选择性扩增的引物，需要关于目标序列的先前信息。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_26" class="kx_ref">[26]</a></sup>这意味着，通常情况下，PCR用户必须知道两个单链模板中每个模板上目标区域上游的精确序列，以确保DNA聚合酶正确结合引物-模板杂交体，并随后在DNA合成过程中产生整个目标区域。</p><p>像所有酶一样，DNA聚合酶也容易出错，这反过来会导致产生的PCR片段发生突变。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_27" class="kx_ref">[27]</a></sup> </p><p>PCR的另一个限制是，即使是最少量的污染DNA也可以被扩增，导致误导或模糊的结果。为了最大限度地减少污染的可能性，调查人员应该为试剂制备、聚合酶链反应和产品分析预留单独的房间。试剂应分配到一次性的等分试样中。应经常使用带有一次性柱塞和超长移液器吸头的移液器。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_28" class="kx_ref">[28]</a></sup> </p></div></div><div id="par_14996399733080592"><h2 class="title">6 变化<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>
 </p><ul>
  <li><i>等位基因特异性聚合酶链反应</i>:基于单核苷酸变异的诊断或克隆技术(snv不要与 SNPs 混淆)(患者的单碱基差异)。它需要事先知道DNA序列，包括等位基因之间的差异，并使用3’端包含SNV的引物(通常包含SNV周围的碱基对缓冲液)。在模板和引物不匹配的情况下，严格条件下的PCR扩增效率要低得多，因此用单核苷酸多态性特异性引物成功扩增表明序列中存在特异性单核苷酸多态性。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_28" class="kx_ref">[28]</a></sup>有关更多信息，请参见单核苷酸多态性基因分型。</li>
  <li><i>装配聚合酶链反应</i>或者<i>聚合酶循环组件</i>:通过对具有短重叠片段的长寡核苷酸池进行PCR来人工合成长DNA序列。寡核苷酸在有义和反义方向之间交替，重叠片段决定聚合酶链反应片段的顺序，从而选择性地产生最终的长DNA产物。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_29" class="kx_ref">[29]</a></sup></li>
  <li><i>不对称聚合酶链反应</i>:优先扩增双链DNA模板中的一条DNA链。它用于测序和杂交探测，其中只需要扩增两条互补链中的一条。聚合酶链反应照常进行，但扩增目标链的引物过量。由于限制性引物用完后，反应后期的扩增速度较慢(算术)，因此需要额外的PCR循环。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_30" class="kx_ref">[30]</a></sup>这个过程最近的一个修改，称为<i>L²</i>线性-<i>A</i>之后-<i>T</i>他-<i>E</i>xponential-PCR (LATE-PCR)使用比过量引物熔点更高的限制性引物( Tm )，以保持反应效率，因为限制性引物浓度在反应中期降低。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_31" class="kx_ref">[31]</a></sup></li>
  <li><i>对流聚合酶链反应</i>:一种伪等温PCR方法。将溶液置于热梯度下，而不是反复加热和冷却聚合酶链反应混合物。由此产生的热不稳定性驱动对流流动自动将聚合酶链反应试剂从热区域和冷区域打乱，从而反复启动聚合酶链反应。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_32" class="kx_ref">[32]</a></sup>通过利用混沌流场的出现，可以优化热边界条件和PCR外壳的几何形状等参数，以产生特异和快速的PCR。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_33" class="kx_ref">[33]</a></sup>这种对流聚合酶链反应设置显著降低了设备功率需求和操作时间。</li>
  <li><i>拨号聚合酶链反应</i>:一种高度并行的方法，用于为基因合成检索精确的DNA分子。在大规模平行测序之前，用独特的侧翼标签修饰复杂的DNA分子库。然后，标签导向引物能够通过聚合酶链反应检索具有所需序列的分子。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_34" class="kx_ref">[34]</a></sup></li>
  <li><i>数字聚合酶链反应 (dPCR)</i>:用于测量DNA样品中目标DNA序列的数量。DNA样品被高度稀释，使得在并行运行许多PCR后，其中一些PCR不会接收到目标DNA的单个分子。目标DNA浓度是使用负结果的比例计算的。因此得名“数字聚合酶链反应”。</li>
  <li><i>Helicase-dependent amplification</i>:与传统PCR相似，但使用恒温而不是通过变性和退火/延伸循环。 DNA解旋酶，一种解开DNA的酶，被用来代替热变性。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_35" class="kx_ref">[35]</a></sup></li>
  <li><i>热启动聚合酶链式反应</i>:一种在PCR的初始建立阶段减少非特异性扩增的技术。它可以通过将反应组分加热到变性温度(例如95&nbsp;c)在加入聚合酶之前。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_36" class="kx_ref">[36]</a></sup>已经开发了特殊的酶系统，通过结合抗体来抑制聚合酶在环境温度下的活性<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_37" class="kx_ref">[37]</a></sup><sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_37" class="kx_ref">[37]</a></sup>或者通过共价键结合的抑制剂的存在，该抑制剂仅在高温活化步骤后解离。热启动/冷完成聚合酶链反应是通过新的杂合聚合酶实现的，这些酶在环境温度下不活跃，在伸长温度下立即激活。</li>
  <li><i>电子聚合酶链反应</i>(数字聚合酶链反应、虚拟聚合酶链反应、电子聚合酶链反应、电子聚合酶链反应)是指计算工具使用给定的一组引物 ( 探针)来扩增来自测序的基因组或转录组的DNA 序列，用于计算理论聚合酶链反应结果。电子PCR被提出作为分子生物学的教育工具。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_38" class="kx_ref">[38]</a></sup></li>
  <li><i>序列间特异性聚合酶链反应</i>(ISSR):一种用于DNA指纹识别的聚合酶链反应方法，它放大简单重复序列之间的区域，以产生扩增片段长度的独特指纹。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_39" class="kx_ref">[39]</a></sup></li>
  <li><i>反向聚合酶链反应</i>:通常用于识别基因组插入片段周围的侧翼序列。它涉及一系列的脱氧核糖核酸消化和自连接，在未知序列的两端产生已知序列。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_40" class="kx_ref">[40]</a></sup></li>
  <li><i>连接介导的聚合酶链反应</i>:使用连接到感兴趣的DNA的小DNA接头和退火到DNA接头的多个引物；它已被用于 DNA测序、基因组行走和 DNA足迹法。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_41" class="kx_ref">[41]</a></sup></li>
  <li><i>甲基化特异性聚合酶链反应</i>(MSP):由史蒂芬·贝林和詹姆斯·赫尔曼在约翰·霍普金斯大学医学院开发，<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_42" class="kx_ref">[42]</a></sup>用于检测基因组DNA中CpG岛的甲基化。DNA首先用亚硫酸氢钠处理，亚硫酸氢钠将未甲基化的胞嘧啶碱基转化为尿嘧啶，尿嘧啶被PCR引物识别为胸腺嘧啶。然后在修饰的DNA上进行两次PCR，使用除引物序列中任何CpG岛以外相同的引物组。在这些点上，一组引物用胞嘧啶识别DNA以扩增甲基化的DNA，一组引物用尿嘧啶或胸腺嘧啶识别DNA以扩增未甲基化的DNA。使用qPCR进行MSP也可以获得甲基化的定量而不是定性信息。</li>
  <li><i>微型引物聚合酶链反应</i>:使用耐热聚合酶(S-Tbr)，可以从短引物延伸到9或10个核苷酸。这种方法允许PCR靶向较小的引物结合区域，并用于扩增保守的DNA序列，如16S(或真核18S) rRNA基因。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_43" class="kx_ref">[43]</a></sup></li>
  <li><i>多重连接依赖探针扩增</i>（<i>MLPA</i>):允许用单个引物对扩增多个靶标，从而避免多重PCR的分辨率限制(见下文)。</li>
  <li><i>多重聚合酶链反应</i>:由单个PCR混合物中的多个引物组组成，以产生对不同DNA序列特异的不同大小的扩增子。通过同时靶向多个基因，可以从单次测试中获得额外的信息，否则将需要几次试剂和更多时间来执行。每个引物组的退火温度必须优化，以在单个反应中正确工作，并符合扩增子尺寸。也就是说，当通过凝胶电泳可视化时，它们的碱基对长度应该足够不同以形成不同的条带。</li>
  <li><i>纳米粒子辅助聚合酶链反应</i>:一些纳米粒子(NPs)可以提高PCR的效率(因此被称为纳米PCR)，一些甚至可以优于原始的PCR增强剂。据报道，量子点可以提高聚合酶链反应的特异性和效率。单壁碳纳米管和多壁碳纳米管能有效增强长聚合酶链反应的扩增。纳米碳粉(成交堆积)可以提高重复聚合酶链反应和长聚合酶链反应的效率氧化锌,二氧化钛发现银纳米粒子能提高聚合酶链反应的产率。以前的数据表明非金属NP保留了可接受的放大保真度。鉴于许多纳米粒子能够提高聚合酶链反应效率，很明显纳米粒子技术的改进和产品开发有很大的潜力。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_44" class="kx_ref">[44]</a></sup><sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_45" class="kx_ref">[45]</a></sup></li>
  <li><i>嵌套聚合酶链反应</i>:通过减少DNA非特异性扩增的背景，提高DNA扩增的特异性。两组引物用于两个连续的PCR。在第一个反应中，一对引物用于产生DNA产物，除了预期的靶之外，该产物可能仍然由非特异性扩增的DNA片段组成。然后用一组引物将产物用于第二次聚合酶链反应，所述引物的结合位点完全或部分不同于第一次反应中使用的每个引物，并且位于其中的3’。嵌套式PCR在特异性扩增长DNA片段方面通常比传统PCR更成功，但它需要更详细的目标序列知识。</li>
  <li><i>重叠延伸聚合酶链反应</i>或者<i>通过重叠延伸拼接(SOEing)</i>:一种用于将两个或多个含有互补序列的DNA片段拼接在一起的基因工程技术。它用于连接含有基因、调节序列或突变的DNA片段；这项技术可以创造特定的长DNA构建体。它还可以将缺失、插入或点突变引入DNA序列。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_46" class="kx_ref">[46]</a></sup><sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_47" class="kx_ref">[47]</a></sup></li>
  <li><i>PAN-AC</i>:使用等温条件进行扩增，并可用于活细胞。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_48" class="kx_ref">[48]</a></sup><sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_49" class="kx_ref">[49]</a></sup></li>
  <li><i>定量聚合酶链反应</i>(qPCR):用于测量目标序列的数量(通常是实时的)。它定量测量DNA、cDNA或RNA的起始量。定量PCR 通常用于确定样品中是否存在DNA序列及其在样品中的拷贝数。<i>定量聚合酶链反应</i>具有非常高的精确度。定量PCR方法使用荧光染料，如Sybr Green、EvaGreen或含荧光团的DNA探针，如TaqMan ，实时测量扩增产物的量。它有时也被缩写为逆转录-聚合酶链反应(<i>实时的</i>聚合酶链反应)，但这个缩写只能用于 逆转录PCR 。定量PCR是定量PCR 的合适缩写。</li>
  <li><i>逆转录聚合酶链反应(逆转录-聚合酶链反应)</i>:用于从RNA中扩增DNA。逆转录酶将核糖核酸逆转录成cDNA ，然后通过聚合酶链反应进行扩增。逆转录-聚合酶链反应广泛用于表达谱，以确定基因的表达或鉴定RNA转录物的序列，包括转录起始和终止位点。如果基因的基因组DNA序列是已知的，逆转录-聚合酶链反应可以用来绘制基因中外显子和内含子的位置。基因的5’端(对应于转录起始位点)通常通过 RACE-PCR (<i>快速扩增cDNA末端</i>)中。</li>
  <li><i>核糖核酸酶H依赖性聚合酶链反应</i>(rhPCR):PCR的一种改进，利用带有3’延伸区的引物，该延伸区可以被耐热核糖核酸酶HII酶去除。该系统减少了引物二聚体，并允许用更高数量的引物进行多重反应。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_50" class="kx_ref">[50]</a></sup></li>
  <li><i>单一特异性引物聚合酶链反应</i>(SSP-PCR):即使序列信息仅在一端可用，也允许双链DNA的扩增。这种方法允许扩增只有部分序列信息可用的基因，并允许单向基因组从染色体的已知区域步行到未知区域。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_51" class="kx_ref">[51]</a></sup></li>
  <li><i>固相聚合酶链反应</i>:包含多种含义，包括 Polony Amplification (例如，聚合酶链反应菌落来源于凝胶基质)，桥聚合酶链反应<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_52" class="kx_ref">[52]</a></sup>(引物共价连接到固相支持物表面)、常规固相聚合酶链反应(其中不对称聚合酶链反应在带有固相支持物的引物存在下应用，所述引物的序列与其中一个水性引物匹配)和增强型固相聚合酶链反应<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_53" class="kx_ref">[53]</a></sup>(其中传统的固相聚合酶链反应可以通过使用高Tm和嵌套的固相支持物引物并任选应用热“步骤”来促进固相支持物引发来改进)。</li>
  <li><i>自杀聚合酶链反应</i>:通常用于古遗传学或其他研究中，其中避免假阳性并确保扩增片段的特异性是最高优先级。最初是在一项研究中描述的，该研究旨在验证从14世纪的坟墓中获得的牙科样本中是否存在微生物鼠疫耶尔森菌，据说这些坟墓是在中世纪的黑死病流行期间被鼠疫杀死的。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_54" class="kx_ref">[54]</a></sup>该方法规定在聚合酶链反应中只使用一次任何引物组合(因此称为“自杀”)，这种组合不应用于任何阳性对照聚合酶链反应，引物应始终针对以前在实验室中从未使用该引物或任何其他引物组扩增的基因组区域。这确保了实验室中不存在来自先前PCR反应的污染DNA，否则这些DNA可能会产生假阳性。</li>
 </ul><p></p><p>
 </p><ul>
  <li><i>热不对称交错聚合酶链反应</i>:用于分离已知序列侧翼的未知序列。在已知序列中，TAIL-PCR使用一对具有不同退火温度的嵌套引物；简并引物用于从未知序列向另一个方向扩增。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_55" class="kx_ref">[55]</a></sup></li>
  <li><i>触地聚合酶链反应</i>（<i>逐步下降聚合酶链反应</i>):PCR的一种变体，旨在通过随着PCR循环的进行逐渐降低退火温度来减少非特异性背景。初始循环的退火温度通常为几度(3-5&nbsp;c)高于T<sub>m</sub>在使用的引物中，在后期的循环中，它是几度(3-5&nbsp;c)低于底漆T<sub>m</sub>。较高的温度赋予引物结合更大的特异性，较低的温度允许从初始循环中形成的特定产物中进行更有效的扩增。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_56" class="kx_ref">[56]</a></sup></li>
  <li><i>万能快走</i>:对于基因组行走和遗传指纹识别，使用比传统的“单侧”方法更特异性的“双侧”PCR(仅使用一个基因特异性引物和一个通用引物——这可能导致动脉事实上的“噪音”)<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_57" class="kx_ref">[57]</a></sup>这是由于套索结构形成的机制。UFW的流线型衍生物是LaNe RAGE(用于快速扩增基因组DNA末端的套索依赖性巢式PCR)，<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_58" class="kx_ref">[58]</a></sup>' 5'RACE LaNe<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_59" class="kx_ref">[59]</a></sup>和3'RACE LaNe。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_60" class="kx_ref">[60]</a></sup></li>
 </ul><p></p></div></div><div id="par_14996399733080593"><h2 class="title">7 历史<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p></p><div class="text_img ed_imgfloat_right">
            <a class="ed_image_link lazyLoad" data-src="https://img04.sogoucdn.com/app/a/200698/sogou_science_17121" data-bigsrc="" title="点击查看大图" href="javascript:" data-observer="true"></a>
            <div class="text_img_title">示例引物对的图解表示。在体外分析中使用引物以允许DNA合成是允许PCR发展的一项重大创新。</div>   
        </div> <p></p><p>1971年的一篇论文<i>《分子生物学杂志》</i>作者:Kjell Kleppe&nbsp;(挪威语)高宾德·科兰纳实验室的同事首先描述了一种<i>在试管内</i>使用酶分析法用引物复制短DNA模板的方法。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_61" class="kx_ref">[61]</a></sup>然而，这种基本聚合酶链反应原理的早期表现在当时并没有受到太多关注，聚合酶链式反应在1983年的发明通常归功于凯利·穆利斯。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_62" class="kx_ref">[62]</a></sup> </p><p></p><div class="text_img ed_imgfloat_right">
            <a class="ed_image_link lazyLoad" data-src="https://img01.sogoucdn.com/app/a/200698/sogou_science_17123" data-bigsrc="" title="点击查看大图" href="javascript:" data-observer="true"></a>
            <div class="text_img_title">“浅蓝色”，1986年用于聚合酶链反应的样机</div>   
        </div> <p></p><p>当穆利斯在1983年开发聚合酶链反应时，他在加利福尼亚州的 Emeryville 为塞特斯公司工作，该公司是首批生物技术公司之一，他负责合成DNA短链。穆利斯写道，有一天晚上，他开车沿着太平洋海岸高速公路巡航时，第一次想到了聚合酶链反应的想法。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_63" class="kx_ref">[63]</a></sup>当他意识到他已经发明了一种通过由DNA聚合酶驱动的重复复制周期来扩增任何DNA区域的方法时，他正在用一种新的分析DNA变化(突变)方法。在<i>科学美国人</i>穆利斯总结道:“从遗传物质DNA的一个分子开始，聚合酶链反应可以在一个下午产生1000亿个类似的分子。这个反应很容易执行。它只需要一个试管、一些简单的试剂和一个热源。”<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_64" class="kx_ref">[64]</a></sup>1988年，DNA指纹首次用于亲子鉴定。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_65" class="kx_ref">[65]</a></sup> </p><p>穆利斯因其发明于1993年被授予诺贝尔化学奖，七年前他和他在塞特斯的同事首次将他的建议付诸实践。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_66" class="kx_ref">[66]</a></sup>mullis 1985年与R. K .斋祀和H. A. Erlich合著的论文《β-珠蛋白基因组序列的酶促扩增和用于镰刀形红细胞贫血症诊断的限制性位点分析》——聚合酶链式反应发明(PCR)——荣获2017年美国化学学会历史分期化学奖化学突破奖。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_66" class="kx_ref">[66]</a></sup><sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_67" class="kx_ref">[67]</a></sup> </p><p>关于其他科学家对穆利斯工作的智力和实践贡献，以及他是否是PCR原理的唯一发明者，仍然存在一些争议(见下文)。</p><p>聚合酶链反应方法的核心是使用合适的能够耐受高温的脱氧核糖核酸聚合酶90&nbsp;°C (194&nbsp;°F)在每一个复制循环后，在 DNA双螺旋中分离两条DNA链所必需的。最初用于体外实验的DNA聚合酶预示着PCR不能耐受这些高温。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_68" class="kx_ref">[68]</a></sup>因此，早期的DNA复制过程非常低效和耗时，并且需要大量的DNA聚合酶和在整个过程中的连续处理。</p><p>1976年发现了 Taq聚合酶——一种从嗜热细菌中纯化的DNA聚合酶，<i>水生栖热菌</i>，它自然地生活在诸如<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_68" class="kx_ref">[68]</a></sup>温泉之类的高温(50至80°C(122至176°F))环境中，为PCR方法的显著改进铺平了道路。从<i>T.aquaticus</i>分离出的DNA聚合酶在高温下稳定，即使在DNA变性之后仍保持活性，<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_68" class="kx_ref">[68]</a></sup>因此不需要在每个循环后添加新的DNA聚合酶。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_68" class="kx_ref">[68]</a></sup>这允许基于自动热循环仪的DNA扩增过程。</p><p></p><h3>7.1 <span>专利纠纷</span></h3><p></p><p>聚合酶链反应技术由凯利·穆利斯获得专利，并转让给塞特斯公司，穆利斯1983年发明这项技术时就在该公司工作。<i>Taq</i>聚合酶也被专利所保护。已经有几起与该技术相关的高调诉讼，包括杜邦提起的一起不成功的诉讼。制药公司罗氏在1992年购买了这些专利的权利，目前拥有那些仍然受到保护的权利。</p><p>罗氏和Promega 之间围绕Taq聚合酶酶的相关专利战仍在世界各地的几个管辖区进行。法律争论已经超出了2005年3月28日到期的原始PCR和Taq聚合酶专利的寿命。<sup><a href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/d11182.htm#quote_68" class="kx_ref">[68]</a></sup> </p></div></div></div></div><div id="references"><h2 class="title" id="par_references">参考文献</h2><ul class="references"><li id="quote_1"><span class="references-num">[1]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Cheng, S.; Fockler, C.; Barnes, W. M.; Higuchi, R. (1994). "Effective Amplification of Long Targets from Cloned Inserts and Human Genomic DNA". Proceedings of the National Academy of Sciences. 91 (12): 5695–5699. Bibcode:1994PNAS...91.5695C. doi:10.1073/pnas.91.12.5695. PMC 44063. PMID 8202550..</span></p></li><li id="quote_2"><span class="references-num">[2]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Carr AC, Moore SD (2012). Lucia A, ed. "Robust quantification of polymerase chain reactions using global fitting". PLoS ONE. 7 (5): e37640. Bibcode:2012PLoSO...737640C. doi:10.1371/journal.pone.0037640. PMC 3365123. PMID 22701526..</span></p></li><li id="quote_3"><span class="references-num">[3]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Polymerase Chain Reaction (PCR)". National Center for Biotechnology Information, U.S. National Library of Medicine..</span></p></li><li id="quote_4"><span class="references-num">[4]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"PCR". Genetic Science Learning Center, University of Utah..</span></p></li><li id="quote_5"><span class="references-num">[5]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Pavlov, A. R.; Pavlova, N. V.; Kozyavkin, S. A.; Slesarev, A. I. (2004). "Recent developments in the optimization of thermostable DNA polymerases for efficient applications☆". Trends in Biotechnology. 22 (5): 253–260. doi:10.1016/j.tibtech.2004.02.011. PMID 15109812..</span></p></li><li id="quote_6"><span class="references-num">[6]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Rychlik W, Spencer WJ, Rhoads RE (1990). "Optimization of the annealing temperature for DNA amplification in vitro". Nucleic Acids Res. 18 (21): 6409–6412. doi:10.1093/nar/18.21.6409. PMC 332522. PMID 2243783..</span></p></li><li id="quote_7"><span class="references-num">[7]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Borman, Jon; Schuster, David; Li, Wu-bo; Jessee, Joel; Rashtchian, Ayoub (2000). "PCR from problematic templates" (PDF). Focus. 22 (1): 10..</span></p></li><li id="quote_8"><span class="references-num">[8]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Bogetto, Prachi and Waidne, Lisa (2000). "Helpful tips for PCR" (PDF). Focus. 22 (1): 12.CS1 maint: Multiple names: authors list (link).</span></p></li><li id="quote_9"><span class="references-num">[9]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Joseph Sambrook &amp; David W. Russel (2001). Molecular Cloning: A Laboratory Manual (3rd ed.). Cold Spring Harbor, N.Y.: Cold Spring Harbor Laboratory Press. ISBN 978-0-879-69576-7.第八章:聚合酶链反应体外扩增脱氧核糖核酸.</span></p></li><li id="quote_10"><span class="references-num">[10]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Electronic PCR". NCBI – National Center for Biotechnology Information. Retrieved 13 March 2012..</span></p></li><li id="quote_11"><span class="references-num">[11]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Pavlov AR, Pavlova NV, Kozyavkin SA, Slesarev AI (2006). "Thermostable DNA Polymerases for a Wide Spectrum of Applications: Comparison of a Robust Hybrid TopoTaq to other enzymes". In Kieleczawa J. DNA Sequencing II: Optimizing Preparation and Cleanup. Jones and Bartlett. pp. 241–257. ISBN 978-0-7637-3383-4..</span></p></li><li id="quote_12"><span class="references-num">[12]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Pombert JF, Sistek V, Boissinot M, Frenette M (2009). "Evolutionary relationships among salivarius streptococci as inferred from multilocus phylogenies based on 16S rRNA-encoding, recA, secA, and secY gene sequences". BMC Microbiol. 9: 232. doi:10.1186/1471-2180-9-232. PMC 2777182. PMID 19878555..</span></p></li><li id="quote_13"><span class="references-num">[13]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Chemical Synthesis, Sequencing, and Amplification of DNA (class notes on MBB/BIO 343)". Arizona State University. Archived from the original on 9 October 1997. Retrieved 29 October 2007..</span></p></li><li id="quote_14"><span class="references-num">[14]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Bustin, S. A.; Benes, V.; Garson, J. A.; Hellemans, J.; Huggett, J.; Kubista, M.; Mueller, R.; Nolan, T.; Pfaffl, M. W.; Shipley, G. L.; Vandesompele, J.; Wittwer, C. T. (2009). "The MIQE Guidelines: Minimum Information for Publication of Quantitative Real-Time PCR Experiments". Clinical Chemistry. 55 (4): 611–622. doi:10.1373/clinchem.2008.112797. PMID 19246619..</span></p></li><li id="quote_15"><span class="references-num">[15]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Garibyan, Avashia (March 2013). "Polymerase Chain Reaction". Journal of Investigative Dermatology. 133 (3): 1–4. doi:10.1038/jid.2013.1. PMC 4102308. PMID 23399825..</span></p></li><li id="quote_16"><span class="references-num">[16]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Ninfa, Alexander; Ballou, David; Benore, Marilee (2009). Fundamental Laboratory Approaches for Biochemistry and Biotechnology. United States: Wiley. pp. 408–410. ISBN 978-0470087664..</span></p></li><li id="quote_17"><span class="references-num">[17]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Tomar, Rukam (2010). Molecular Markers and Plant Biotechnology. Pitman Pura, New Delhi: New India Publishing Agency. p. 188. ISBN 978-93-80235-25-7..</span></p></li><li id="quote_18"><span class="references-num">[18]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Cai, H; Caswell JL; Prescott JF (March 2014). "Nonculture Molecular Techniques for Diagnosis of Bacterial Disease in Animals: A Diagnostic Laboratory Perspective". Veterinary Pathology. 51 (2): 341–350. doi:10.1177/0300985813511132. PMID 24569613..</span></p></li><li id="quote_19"><span class="references-num">[19]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Salis AD (2009). "Applications in Clinical Microbiology". Real-Time PCR: Current Technology and Applications. Caister Academic Press. ISBN 978-1-904455-39-4..</span></p></li><li id="quote_20"><span class="references-num">[20]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Kwok, S.; Mack, D. H.; Mullis, K. B.; Poiesz, B.; Ehrlich, G.; Blair, D.; Friedman-Kien, A.; Sninsky, J. J. (1987). "Identification of human immunodeficiency virus sequences by using in vitro enzymatic amplification and oligomer cleavage detection". Journal of Virology. 61 (5): 1690–4. PMC 254157. PMID 2437321..</span></p></li><li id="quote_21"><span class="references-num">[21]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Finger, Horst; von Koenig, Carl Heinz Wirsing (1996). Baron, Samuel, ed. Medical Microbiology (4th ed.). Galveston (TX): University of Texas Medical Branch at Galveston. ISBN 978-0963117212. PMID 21413270..</span></p></li><li id="quote_22"><span class="references-num">[22]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Yeh, Sylvia H.; Mink, ChrisAnna M. (2012). "Netter's Infectious Diseases". Netter's Infectious Diseases: 11–14. doi:10.1016/B978-1-4377-0126-5.00003-3..</span></p></li><li id="quote_23"><span class="references-num">[23]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Alonso, A (2004-01-28). "Real-time PCR designs to estimate nuclear and mitochondrial DNA copy number in forensic and ancient DNA studies". Forensic Science International. 139 (2–3): 141–149. doi:10.1016/j.forsciint.2003.10.008. PMID 15040907..</span></p></li><li id="quote_24"><span class="references-num">[24]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Boehnke, M.; Arnheim, N.; Li, H.; Collins, F. S. (1989). "Fine-structure genetic mapping of human chromosomes using the polymerase chain reaction on single sperm: Experimental design considerations". American Journal of Human Genetics. 45 (1): 21–32. PMC 1683385. PMID 2568090..</span></p></li><li id="quote_25"><span class="references-num">[25]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Garibyan, Lilit; Avashia, Nidhi (2013-03-01). "Polymerase Chain Reaction". Journal of Investigative Dermatology. 133 (3): 1–4. doi:10.1038/jid.2013.1. PMC 4102308. PMID 23399825..</span></p></li><li id="quote_26"><span class="references-num">[26]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Garibyan L, Avashia N (2013). "Polymerase Chain Reaction". Journal of Investigative Dermatology. 133 (3): 1–4. doi:10.1038/jid.2013.1. PMC 4102308. PMID 23399825..</span></p></li><li id="quote_27"><span class="references-num">[27]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Zhou, Y H; Zhang, X P; Ebright, R H (1991-11-11). "Random mutagenesis of gene-sized DNA molecules by use of PCR with Taq DNA polymerase". Nucleic Acids Research. 19 (21): 6052. doi:10.1093/nar/19.21.6052. PMC 329070. PMID 1658751..</span></p></li><li id="quote_28"><span class="references-num">[28]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Schochetman, Gerald; Ou, Chin-Yih; Jones, Wanda K. (1988). "Polymerase Chain Reaction". The Journal of Infectious Diseases. 158 (6): 1154–1157. doi:10.1093/infdis/158.6.1154. JSTOR 30137034..</span></p></li><li id="quote_29"><span class="references-num">[29]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Stemmer WP, Crameri A, Ha KD, Brennan TM, Heyneker HL (1995). "Single-step assembly of a gene and entire plasmid from large numbers of oligodeoxyribonucleotides". Gene. 164 (1): 49–53. doi:10.1016/0378-1119(95)00511-4. PMID 7590320..</span></p></li><li id="quote_30"><span class="references-num">[30]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Innis MA, Myambo KB, Gelfand DH, Brow MA (1988). "DNA sequencing with Thermus aquaticus DNA polymerase and direct sequencing of polymerase chain reaction-amplified DNA". Proc Natl Acad Sci USA. 85 (24): 9436–9440. Bibcode:1988PNAS...85.9436I. doi:10.1073/pnas.85.24.9436. PMC 282767. PMID 3200828..</span></p></li><li id="quote_31"><span class="references-num">[31]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Pierce KE &amp; Wangh LJ (2007). Linear-after-the-exponential polymerase chain reaction and allied technologies Real-time detection strategies for rapid, reliable diagnosis from single cells. Methods in Molecular Medicine. 132. pp. 65–85. doi:10.1007/978-1-59745-298-4_7. ISBN 978-1-58829-578-1. PMID 17876077..</span></p></li><li id="quote_32"><span class="references-num">[32]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Krishnan, Madhavi; Ugaz, Victor; Burns, Mark (2002). "PCR in a Rayleigh-Benard convection cell". Science. 298 (5594): 793. doi:10.1126/science.298.5594.793. PMID 12399582..</span></p></li><li id="quote_33"><span class="references-num">[33]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Priye, Aashish; Hassan, Yassin; Ugaz, Victor (2013). "Microscale chaotic advection enables robust convective DNA replication". Analytical Chemistry. 85 (21): 10536–10541. doi:10.1021/ac402611s. PMID 24083802..</span></p></li><li id="quote_34"><span class="references-num">[34]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Schwartz JJ, Lee C, Shendure J (2012). "Accurate gene synthesis with tag-directed retrieval of sequence-verified DNA molecules". Nature Methods. 9 (9): 913–915. doi:10.1038/nmeth.2137. PMC 3433648. PMID 22886093..</span></p></li><li id="quote_35"><span class="references-num">[35]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Vincent M, Xu Y, Kong H (2004). "Helicase-dependent isothermal DNA amplification". EMBO Reports. 5 (8): 795–800. doi:10.1038/sj.embor.7400200. PMC 1249482. PMID 15247927..</span></p></li><li id="quote_36"><span class="references-num">[36]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Chou Q, Russell M, Birch DE, Raymond J, Bloch W (1992). "Prevention of pre-PCR mis-priming and primer dimerization improves low-copy-number amplifications". Nucleic Acids Research. 20 (7): 1717–1723. doi:10.1093/nar/20.7.1717. PMC 312262. PMID 1579465..</span></p></li><li id="quote_37"><span class="references-num">[37]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Sharkey, D. J.; Scalice, E. R.; Christy, K. G.; Atwood, S. M.; Daiss, J. L. (1994). "Antibodies as Thermolabile Switches: High Temperature Triggering for the Polymerase Chain Reaction". Bio/Technology. 12 (5): 506–509. doi:10.1038/nbt0594-506..</span></p></li><li id="quote_38"><span class="references-num">[38]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">San Millan RM, Martinez-Ballesteros I, Rementeria A, Garaizar J, Bikandi J (2013). "Online exercise for the design and simulation of PCR and PCR-RFLP experiments". BMC Research Notes. 6: 513. doi:10.1186/1756-0500-6-513. PMC 4029544. PMID 24314313..</span></p></li><li id="quote_39"><span class="references-num">[39]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Zietkiewicz, E.; Rafalski, A.; Labuda, D. (1994). "Genome fingerprinting by simple sequence repeat (SSR)-anchored polymerase chain reaction amplification". Genomics. 20 (2): 176–83. doi:10.1006/geno.1994.1151. PMID 8020964..</span></p></li><li id="quote_40"><span class="references-num">[40]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Ochman H, Gerber AS, Hartl DL (1988). "Genetic Applications of an Inverse Polymerase Chain Reaction". Genetics. 120 (3): 621–623. PMC 1203539. PMID 2852134..</span></p></li><li id="quote_41"><span class="references-num">[41]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Mueller PR, Wold B (1988). "In vivo footprinting of a muscle specific enhancer by ligation mediated PCR". Science. 246 (4931): 780–786. doi:10.1126/science.2814500. PMID 2814500..</span></p></li><li id="quote_42"><span class="references-num">[42]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Herman JG, Graff JR, Myöhänen S, Nelkin BD, Baylin SB (1996). "Methylation-specific PCR: a novel PCR assay for methylation status of CpG islands". Proc Natl Acad Sci USA. 93 (13): 9821–9826. Bibcode:1996PNAS...93.9821H. doi:10.1073/pnas.93.18.9821. PMC 38513. PMID 8790415..</span></p></li><li id="quote_43"><span class="references-num">[43]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Isenbarger TA, Finney M, Ríos-Velázquez C, Handelsman J, Ruvkun G (2008). "Miniprimer PCR, a New Lens for Viewing the Microbial World". Applied and Environmental Microbiology. 74 (3): 840–9. doi:10.1128/AEM.01933-07. PMC 2227730. PMID 18083877..</span></p></li><li id="quote_44"><span class="references-num">[44]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Cenchao Shen; Wenjuan Yang; Qiaoli Ji; Hisaji Maki; Anjie Dong; Zhizhou Zhang (2009). "NanoPCR observation: different levels of DNA replication fidelity in nanoparticle-enhanced polymerase chain reactions". Nanotechnology. 20 (45): 455103. Bibcode:2009Nanot..20S5103S. doi:10.1088/0957-4484/20/45/455103. PMID 19822925..</span></p></li><li id="quote_45"><span class="references-num">[45]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Shen, Cenchao (2013). "An Overview of Nanoparticle-Assisted Polymerase Chain Reaction Technology". An Overview of Nanoparticle‐Assisted Polymerase Chain Reaction Technology. US: Wiley-Blackwell Publishing Ltd. pp. 97–106. doi:10.1002/9781118451915.ch5. ISBN 9781118451915..</span></p></li><li id="quote_46"><span class="references-num">[46]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Horton RM, Hunt HD, Ho SN, Pullen JK, Pease LR (1989). "Engineering hybrid genes without the use of restriction enzymes: gene splicing by overlap exten-sion". Gene. 77 (1): 61–68. doi:10.1016/0378-1119(89)90359-4. PMID 2744488..</span></p></li><li id="quote_47"><span class="references-num">[47]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Moller, Simon (2006). PCR: The Basics. US: Taylor &amp; Francis Group. p. 144. ISBN 9780415355476..</span></p></li><li id="quote_48"><span class="references-num">[48]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">David F, Turlotte E (1998). "Une méthode d'amplification génique isotherme" [An Isothermal Amplification Method]. Comptes Rendus de l'Académie des Sciences, Série III. 321 (11): 909–914. Bibcode:1998CRASG.321..909D. doi:10.1016/S0764-4469(99)80005-5..</span></p></li><li id="quote_49"><span class="references-num">[49]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Fabrice David (September–October 2002). "Utiliser les propriétés topologiques de l'ADN: une nouvelle arme contre les agents pathogènes" (PDF). Fusion. Archived from the original (PDF) on 2007-11-28.(法语).</span></p></li><li id="quote_50"><span class="references-num">[50]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Dobosy JR, Rose SD, Beltz KR, Rupp SM, Powers KM, Behlke MA, Walder JA (August 2011). "RNase H-dependent PCR (rhPCR): improved specificity and single nucleotide polymorphism detection using blocked cleavable primers". BMC Biotechnology. 11: 80. doi:10.1186/1472-6750-11-80. PMC 3224242. PMID 21831278..</span></p></li><li id="quote_51"><span class="references-num">[51]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Shyamala, V.; Ferro-Luzzi, Ames G. (1993). Single Specific Primer-Polymerase Chain Reaction (SSP-PCR) and Genome Walking. Methods in Molecular Biology. 15. pp. 339–48. doi:10.1385/0-89603-244-2:339. ISBN 978-0-89603-244-6. PMID 21400290..</span></p></li><li id="quote_52"><span class="references-num">[52]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Bing DH, Boles C, Rehman FN, Audeh M, Belmarsh M, Kelley B, Adams CP (1996). "Bridge amplification: a solid phase PCR system for the amplification and detection of allelic differences in single copy genes". Genetic Identity Conference Proceedings, Seventh International Symposium on Human Identification. Archived from the original on 7 May 2001..</span></p></li><li id="quote_53"><span class="references-num">[53]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Khan Z, Poetter K, Park DJ (2008). "Enhanced solid phase PCR: mechanisms to increase priming by solid support primers". Analytical Biochemistry. 375 (2): 391–393. doi:10.1016/j.ab.2008.01.021. PMID 18267099..</span></p></li><li id="quote_54"><span class="references-num">[54]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Raoult, D; G Aboudharam; E Crubezy; G Larrouy; B Ludes; M Drancourt (2000-11-07). "Molecular identification by "suicide PCR" of Yersinia pestis as the agent of medieval black death". Proc. Natl. Acad. Sci. U.S.A. 97 (23): 12800–12803. Bibcode:2000PNAS...9712800R. doi:10.1073/pnas.220225197. PMC 18844. PMID 11058154..</span></p></li><li id="quote_55"><span class="references-num">[55]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Y.G. Liu &amp; R. F. Whittier (1995). "Thermal asymmetric interlaced PCR: automatable amplification and sequencing of insert end fragments from P1 and YAC clones for chromosome walking". Genomics. 25 (3): 674–81. doi:10.1016/0888-7543(95)80010-J. PMID 7759102..</span></p></li><li id="quote_56"><span class="references-num">[56]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Don RH, Cox PT, Wainwright BJ, Baker K, Mattick JS (1991). "'Touchdown' PCR to circumvent spurious priming during gene amplification". Nucleic Acids Res. 19 (14): 4008. doi:10.1093/nar/19.14.4008. PMC 328507. PMID 1861999..</span></p></li><li id="quote_57"><span class="references-num">[57]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Myrick KV, Gelbart WM (2002). "Universal Fast Walking for direct and versatile determination of flanking sequence". Gene. 284 (1–2): 125–131. doi:10.1016/S0378-1119(02)00384-0. PMID 11891053..</span></p></li><li id="quote_58"><span class="references-num">[58]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Full Text – LaNe RAGE: a new tool for genomic DNA flanking sequence determination"..</span></p></li><li id="quote_59"><span class="references-num">[59]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Park DJ (2005). "A new 5' terminal murine GAPDH exon identified using 5'RACE LaNe". Molecular Biotechnology. 29 (1): 39–46. doi:10.1385/MB:29:1:39. PMID 15668518..</span></p></li><li id="quote_60"><span class="references-num">[60]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Park DJ (2004). "3'RACE LaNe: a simple and rapid fully nested PCR method to determine 3'-terminal cDNA sequence". BioTechniques. 36 (4): 586–588, 590. doi:10.2144/04364BM04. PMID 15088375..</span></p></li><li id="quote_61"><span class="references-num">[61]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Kleppe K, Ohtsuka E, Kleppe R, Molineux I, Khorana HG (1971). "Studies on polynucleotides. XCVI. Repair replications of short synthetic DNA's as catalyzed by DNA polymerases". J. Mol. Biol. 56 (2): 341–361. doi:10.1016/0022-2836(71)90469-4. PMID 4927950..</span></p></li><li id="quote_62"><span class="references-num">[62]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Rabinow, Paul (1996). Making PCR: A Story of Biotechnology. Chicago: University of Chicago Press. ISBN 978-0-226-70146-2..</span></p></li><li id="quote_63"><span class="references-num">[63]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Mullis, Kary (1998). Dancing Naked in the Mind Field. New York: Pantheon Books. ISBN 978-0-679-44255-4..</span></p></li><li id="quote_64"><span class="references-num">[64]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Mullis, Kary (1990). "The unusual origin of the polymerase chain reaction". Scientific American. 262 (4): 56–61, 64–5. Bibcode:1990SciAm.262d..56M. doi:10.1038/scientificamerican0490-56. PMID 2315679..</span></p></li><li id="quote_65"><span class="references-num">[65]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Patidar, Madhvika; Agrawal, Suraksha; Parveen, Farah; Khare, Parul (2015). "Molecular insights of saliva in solving paternity dispute". Journal of Forensic Dental Sciences. 7 (1): 76–79. doi:10.4103/0975-1475.150325. PMC 4330625. PMID 25709326..</span></p></li><li id="quote_66"><span class="references-num">[66]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Kary B. Mullis – Nobel Lecture: The Polymerase Chain Reaction"..</span></p></li><li id="quote_67"><span class="references-num">[67]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Saiki, R.; Scharf, S; Faloona, F; Mullis, K.; Horn, G.; Erlich, H.; Arnheim, N (1985). "Enzymatic amplification of beta-globin genomic sequences and restriction site analysis for diagnosis of sickle cell anemia". Science. 230 (4732): 1350–1354. doi:10.1126/science.2999980..</span></p></li><li id="quote_68"><span class="references-num">[68]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Saiki, R.; Scharf, S.; Faloona, F.; Mullis, K.; Horn, G.; Erlich, H.; Arnheim, N. (1985). "Enzymatic amplification of beta-globin genomic sequences and restriction site analysis for diagnosis of sickle cell anemia". Science. 230 (4732): 1350–1354. doi:10.1126/science.2999980. PMID 2999980..</span></p></li></ul></div><div class="read-num">阅读 <!-- -->1065</div></div><div class="right-side" id="rightSide"><div class="side" id="lemma-side"><div class="side-title">版本记录</div><ul class="side-lst"><li><p class="side-lst-txt">暂无</p></li></ul><div class="user-card userCard"></div></div><div class="side"><div class="side-event"></div></div></div></div><div class="footer-box"><div id="footer"><div class="footer-logo-wrap"><div class="footer-logo"></div><div class="footer-logo-text">知识·传播·科普</div></div><div class="footer-info">本网站内容采用<a target="_blank" href="https://web.archive.org/web/20221028220934/https://creativecommons.org/licenses/by-sa/3.0/deed.zh?tdsourcetag=s_pctim_aiomsg">CC-BY-SA 3.0</a>授权</div><div class="footer-btn-wrap"><a target="_blank" href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/help/#user_protocol">用户协议</a><a target="_blank" href="https://web.archive.org/web/20221028220934/http://www.sogou.com/docs/terms.htm?v=1">免责声明</a><a target="_blank" href="https://web.archive.org/web/20221028220934/http://corp.sogou.com/private.html">隐私政策</a><a target="_blank" href="https://web.archive.org/web/20221028220934/https://baike.sogou.com/kexue/intro.htm">关于我们</a></div></div></div><script>window.lemmaInfo ={"lemmaId":"11182","versionId":"54568173541843210","title":"聚合酶链反应","subtitle":"","abstracts":{"paragraphId":"54234671596233220","title":"摘要","versionId":"54568173541843211","lemmaId":11182,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82815118,"name":"iscience","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599821165,"comment":null,"dependVersionId":0,"contentType":2,"content":"<p>聚合酶链式反应（英文：Polymerase chain reaction，缩写：PCR，又称多聚酶链式反应），是一项利用DNA双链复制的原理，在生物体外复制特定DNA片段的核酸合成技术。透过这项技术，可在短时间内大量扩增目的基因，而不必依赖大肠杆菌或酵母菌等生物体。</p>\n<p>聚合酶链反应是由凯利·穆利斯（Kary Mullis）于1983年开发，当时他是Cetus公司的雇员，也是1993年诺贝尔化学奖的获得者，它是一种简单，廉价和可靠的方法复制DNA片段，这个概念适用于现代生物学和相关科学的许多领域。 PCR可能是分子生物学中使用最广泛的技术。这种技术被用于生物医学研究，犯罪取证和分子考古学</p>\n<p>绝大多数聚合酶链反应方法依赖于热循环。热循环将反应物暴露于加热和冷却的重复循环中，以允许不同的温度依赖性反应——具体地说，脱氧核糖核酸融化和酶-驱动DNA复制。聚合酶链反应使用两种主要试剂-引物(引物是短的单链DNA片段，称为寡核苷酸，是目标DNA区域的互补序列)和DNA聚合酶。在PCR反应的第一步，DNA双螺旋结构的两条链在高温下物理分离，这个过程称为脱氧核糖核酸变性。第二步，降低温度，引物与互补的脱氧核糖核酸序列结合。这两条DNA链就变成了模板，以酶促的方式从构成DNA的自由核苷酸中组装出一条新的DNA链。随着聚合酶链反应的进行，产生的DNA本身被用作复制的模板，启动了一个连锁反应，原始的DNA模板是以指数形式放大。</p>","pics":[{"originalUrl":"https://web.archive.org/web/20221028220934/https://img01.sogoucdn.com/app/a/200698/sogou_science_17113?width=231&height=174&titlename=%E4%B8%80%E6%9D%A1%E5%85%AB%E4%B8%AA%E8%81%9A%E5%90%88%E9%85%B6%E9%93%BE%E5%8F%8D%E5%BA%94%E8%AF%95%E7%AE%A1%EF%BC%8C%E6%AF%8F%E4%B8%AA%E8%AF%95%E7%AE%A1%E5%8C%85%E5%90%AB%E4%B8%80%E4%B8%AA100%20%CE%BCl%E5%8F%8D%E5%BA%94%E6%B7%B7%E5%90%88%E7%89%A9&w=300&h=246","url":"https://web.archive.org/web/20221028220934/https://img01.sogoucdn.com/app/a/200698/sogou_science_17113","rw":300,"rh":246,"title":"一条八个聚合酶链反应试管，每个试管包含一个100 μl反应混合物","alt":null,"width":231,"height":174}],"card":null,"references":[],"versionCount":0},"card":{"paragraphId":"54234671545901578","title":"基本信息","versionId":"54234671545901570","lemmaId":11182,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":78021691,"name":"没时间到白头","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599622383,"comment":null,"dependVersionId":0,"contentType":3,"content":"","pics":null,"card":{"cardItems":[{"key":"属性","value":"链式反应"},{"key":"目前状态","value":"第三代技术"}]},"references":[],"versionCount":0},"categories":[{"id":4,"name":"生物学","parents":[]}],"creator":{"uid":76032140,"name":"Ivan","pic":"https://web.archive.org/web/20221028220934/https://cache.soso.com/qlogo/g?b=oidb&k=R4Tp0e1bbEohYdh6HMicSVw&s=100&t=1557019253","introduction":null,"educations":[{"schoolName":"山东大学","major":"","degree":"硕士","universityId":70,"universityLogo":"https://web.archive.org/web/20221028220934/https://img01.sogoucdn.com/app/a/200943/7cf4032e-6012-11e9-aa89-fc4dd4f70029","majorLevel1":"理学","majorLevel2":"生物学","majorLevel3":"生物化学与分子生物学","majorLevel1Id":1,"majorLevel2Id":189,"majorLevel3Id":218,"state":null,"lab":null,"researchField":null}],"jobs":null,"works":null,"educationBrief":"山东大学","jobBrief":"","role":0,"roleName":null,"title":"山东大学 · 生物学硕士","professionalTitle":null,"phoneNo":null,"editable":true,"partnerId":37,"partnerIdCreateTime":1594286501,"partnerIdPoped":false},"createTime":1572578566,"editor":{"uid":76032140,"name":"Ivan","pic":"https://web.archive.org/web/20221028220934/https://cache.soso.com/qlogo/g?b=oidb&k=R4Tp0e1bbEohYdh6HMicSVw&s=100&t=1557019253","introduction":null,"educations":[{"schoolName":"山东大学","major":"","degree":"硕士","universityId":70,"universityLogo":"https://web.archive.org/web/20221028220934/https://img01.sogoucdn.com/app/a/200943/7cf4032e-6012-11e9-aa89-fc4dd4f70029","majorLevel1":"理学","majorLevel2":"生物学","majorLevel3":"生物化学与分子生物学","majorLevel1Id":1,"majorLevel2Id":189,"majorLevel3Id":218,"state":null,"lab":null,"researchField":null}],"jobs":null,"works":null,"educationBrief":"山东大学","jobBrief":"","role":0,"roleName":null,"title":"山东大学 · 生物学硕士","professionalTitle":null,"phoneNo":null,"editable":true,"partnerId":37,"partnerIdCreateTime":1594286501,"partnerIdPoped":false},"editTime":1576234550,"state":1,"versionCount":1,"upNum":3,"downNum":0,"pics":[{"originalUrl":"https://web.archive.org/web/20221028220934/https://img01.sogoucdn.com/app/a/200698/sogou_science_17113?width=231&height=174&titlename=%E4%B8%80%E6%9D%A1%E5%85%AB%E4%B8%AA%E8%81%9A%E5%90%88%E9%85%B6%E9%93%BE%E5%8F%8D%E5%BA%94%E8%AF%95%E7%AE%A1%EF%BC%8C%E6%AF%8F%E4%B8%AA%E8%AF%95%E7%AE%A1%E5%8C%85%E5%90%AB%E4%B8%80%E4%B8%AA100%20%CE%BCl%E5%8F%8D%E5%BA%94%E6%B7%B7%E5%90%88%E7%89%A9&w=300&h=246","url":"https://web.archive.org/web/20221028220934/https://img01.sogoucdn.com/app/a/200698/sogou_science_17113","rw":300,"rh":246,"title":"一条八个聚合酶链反应试管，每个试管包含一个100 μl反应混合物","alt":null,"width":231,"height":174},{"originalUrl":"https://web.archive.org/web/20221028220934/https://img02.sogoucdn.com/app/a/200698/sogou_science_17115?w=300&h=225&titlename=%E8%81%9A%E5%90%88%E9%85%B6%E9%93%BE%E5%8F%8D%E5%BA%94%E7%9A%84%E7%83%AD%E5%BE%AA%E7%8E%AF%E4%BB%AA","url":"https://web.archive.org/web/20221028220934/https://img02.sogoucdn.com/app/a/200698/sogou_science_17115","rw":300,"rh":225,"title":"聚合酶链反应的热循环仪","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221028220934/https://img03.sogoucdn.com/app/a/200698/sogou_science_17116?w=300&h=338&titlename=%E4%B8%80%E7%A7%8D%E7%94%A8%E4%BA%8E%E8%81%9A%E5%90%88%E9%85%B6%E9%93%BE%E5%8F%8D%E5%BA%94%E7%9A%84%E8%80%81%E5%BC%8F%E4%B8%89%E6%B8%A9%E7%83%AD%E5%BE%AA%E7%8E%AF%E4%BB%AA","url":"https://web.archive.org/web/20221028220934/https://img03.sogoucdn.com/app/a/200698/sogou_science_17116","rw":300,"rh":338,"title":"一种用于聚合酶链反应的老式三温热循环仪","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221028220934/https://img04.sogoucdn.com/app/a/200698/sogou_science_17117?titlename=&w=835&h=435","url":"https://web.archive.org/web/20221028220934/https://img04.sogoucdn.com/app/a/200698/sogou_science_17117","rw":835,"rh":435,"title":"","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221028220934/https://img03.sogoucdn.com/app/a/200698/sogou_science_17118?w=300&h=283&titlename=%E6%BA%B4%E5%8C%96%E4%B9%99%E9%94%AD-%E6%9F%93%E8%89%B2%E5%90%8E%E7%9A%84%E8%81%9A%E5%90%88%E9%85%B6%E9%93%BE%E5%8F%8D%E5%BA%94%E4%BA%A7%E7%89%A9%E5%87%9D%E8%83%B6%E7%94%B5%E6%B3%B3%E3%80%82%E4%B8%A4%E7%BB%84%E5%BC%95%E7%89%A9%E7%94%A8%E4%BA%8E%E6%89%A9%E5%A2%9E%E6%9D%A5%E8%87%AA%E4%B8%89%E7%A7%8D%E4%B8%8D%E5%90%8C%E7%BB%84%E7%BB%87%E6%A0%B7%E5%93%81%E7%9A%84%E9%9D%B6%E5%BA%8F%E5%88%97%E3%80%82%E6%A0%B7%E5%93%81%231%E4%B8%AD%E4%B8%8D%E5%AD%98%E5%9C%A8%E6%89%A9%E5%A2%9E%EF%BC%9B%E6%A0%B7%E5%93%81%232%E5%92%8C%233%E4%B8%AD%E7%9A%84DNA%E6%9D%A1%E5%B8%A6%E8%A1%A8%E6%98%8E%E7%9B%AE%E6%A0%87%E5%BA%8F%E5%88%97%E6%88%90%E5%8A%9F%E6%89%A9%E5%A2%9E%E3%80%82%E5%87%9D%E8%83%B6%E8%BF%98%E6%98%BE%E7%A4%BA%E5%87%BA%E9%98%B3%E6%80%A7%E5%AF%B9%E7%85%A7%EF%BC%8C%E4%BB%A5%E5%8F%8A%E5%8C%85%E5%90%AB%E5%AE%9A%E4%B9%89%E9%95%BF%E5%BA%A6%E7%9A%84DNA%E7%89%87%E6%AE%B5%E7%9A%84DNA%E6%A2%AF%E5%AD%90%EF%BC%8C%E7%94%A8%E4%BA%8E%E7%A1%AE%E5%AE%9A%E5%AE%9E%E9%AA%8C%E6%80%A7PCRs%E4%B8%AD%E6%9D%A1%E5%B8%A6%E7%9A%84%E5%A4%A7%E5%B0%8F%E3%80%82","url":"https://web.archive.org/web/20221028220934/https://img03.sogoucdn.com/app/a/200698/sogou_science_17118","rw":300,"rh":283,"title":"溴化乙锭-染色后的聚合酶链反应产物凝胶电泳。两组引物用于扩增来自三种不同组织样品的靶序列。样品#1中不存在扩增；样品#2和#3中的DNA条带表明目标序列成功扩增。凝胶还显示出阳性对照，以及包含定义长度的DNA片段的DNA梯子，用于确定实验性PCRs中条带的大小。","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221028220934/https://img03.sogoucdn.com/app/a/200698/sogou_science_17119?w=773&h=707&titlename=","url":"https://web.archive.org/web/20221028220934/https://img03.sogoucdn.com/app/a/200698/sogou_science_17119","rw":773,"rh":707,"title":"","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221028220934/https://img02.sogoucdn.com/app/a/200698/sogou_science_17120?h=301&titlename=%E8%81%9A%E5%90%88%E9%85%B6%E9%93%BE%E5%8F%8D%E5%BA%94%E6%89%A9%E5%A2%9E%E7%9A%84DNA%E7%89%87%E6%AE%B5%E7%9A%84%E7%94%B5%E6%B3%B3%E3%80%82(1)%E7%88%B6%E4%BA%B2%E3%80%82(2)%E5%84%BF%E7%AB%A5%E3%80%82(3)%E6%AF%8D%E4%BA%B2%E3%80%82%E8%BF%99%E4%B8%AA%E5%AD%A9%E5%AD%90%E7%BB%A7%E6%89%BF%E4%BA%86%E7%88%B6%E6%AF%8D%E7%9A%84%E4%B8%80%E4%BA%9B%E6%8C%87%E7%BA%B9%EF%BC%8C%E4%BD%86%E4%B8%8D%E6%98%AF%E5%85%A8%E9%83%A8%EF%BC%8C%E7%BB%99%E4%BA%86%E5%AE%83%E4%B8%80%E4%B8%AA%E6%96%B0%E7%9A%84%E7%8B%AC%E7%89%B9%E7%9A%84%E6%8C%87%E7%BA%B9%E3%80%82&w=200","url":"https://web.archive.org/web/20221028220934/https://img02.sogoucdn.com/app/a/200698/sogou_science_17120","rw":200,"rh":301,"title":"聚合酶链反应扩增的DNA片段的电泳。(1)父亲。(2)儿童。(3)母亲。这个孩子继承了父母的一些指纹，但不是全部，给了它一个新的独特的指纹。","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221028220934/https://img04.sogoucdn.com/app/a/200698/sogou_science_17121?w=300&h=156&titlename=%E7%A4%BA%E4%BE%8B%E5%BC%95%E7%89%A9%E5%AF%B9%E7%9A%84%E5%9B%BE%E8%A7%A3%E8%A1%A8%E7%A4%BA%E3%80%82%E5%9C%A8%E4%BD%93%E5%A4%96%E5%88%86%E6%9E%90%E4%B8%AD%E4%BD%BF%E7%94%A8%E5%BC%95%E7%89%A9%E4%BB%A5%E5%85%81%E8%AE%B8DNA%E5%90%88%E6%88%90%E6%98%AF%E5%85%81%E8%AE%B8PCR%E5%8F%91%E5%B1%95%E7%9A%84%E4%B8%80%E9%A1%B9%E9%87%8D%E5%A4%A7%E5%88%9B%E6%96%B0%E3%80%82","url":"https://web.archive.org/web/20221028220934/https://img04.sogoucdn.com/app/a/200698/sogou_science_17121","rw":300,"rh":156,"title":"示例引物对的图解表示。在体外分析中使用引物以允许DNA合成是允许PCR发展的一项重大创新。","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221028220934/https://img01.sogoucdn.com/app/a/200698/sogou_science_17123?w=300&h=193&titlename=%E2%80%9C%E6%B5%85%E8%93%9D%E8%89%B2%E2%80%9D%EF%BC%8C1986%E5%B9%B4%E7%94%A8%E4%BA%8E%E8%81%9A%E5%90%88%E9%85%B6%E9%93%BE%E5%8F%8D%E5%BA%94%E7%9A%84%E6%A0%B7%E6%9C%BA","url":"https://web.archive.org/web/20221028220934/https://img01.sogoucdn.com/app/a/200698/sogou_science_17123","rw":300,"rh":193,"title":"“浅蓝色”，1986年用于聚合酶链反应的样机","alt":null,"width":0,"height":0}],"catalogs":[{"level":1,"title":"原则","paragraphId":"14996399716303369","subCatalogs":[{"level":2,"title":"程序","paragraphId":"14996399716303369","subCatalogs":null},{"level":2,"title":"阶段","paragraphId":"14996399716303369","subCatalogs":null}]},{"level":1,"title":"优化","paragraphId":"14996399716303370","subCatalogs":null},{"level":1,"title":"应用程序","paragraphId":"14996399716303371","subCatalogs":[{"level":2,"title":"选择性脱氧核糖核酸分离","paragraphId":"14996399716303371","subCatalogs":null},{"level":2,"title":"DNA的扩增和定量","paragraphId":"14996399716303371","subCatalogs":null},{"level":2,"title":"医疗和诊断应用","paragraphId":"14996399716303371","subCatalogs":null},{"level":2,"title":"传染病应用","paragraphId":"14996399716303371","subCatalogs":null},{"level":2,"title":"法医应用","paragraphId":"14996399716303371","subCatalogs":null},{"level":2,"title":"研究应用","paragraphId":"14996399716303371","subCatalogs":null}]},{"level":1,"title":"优势","paragraphId":"14996399716303372","subCatalogs":null},{"level":1,"title":"限制","paragraphId":"14996399733080591","subCatalogs":null},{"level":1,"title":"变化","paragraphId":"14996399733080592","subCatalogs":null},{"level":1,"title":"历史","paragraphId":"14996399733080593","subCatalogs":[{"level":2,"title":"专利纠纷","paragraphId":"14996399733080593","subCatalogs":null}]},{"level":1,"title":"参考文献","paragraphId":"-1","subCatalogs":null}],"paragraphs":[{"paragraphId":"14996399716303369","title":"原则","versionId":"54568173541843212","lemmaId":11182,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82815118,"name":"iscience","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599821165,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p><img alt=\"\" class=\"fileimage kx_img ed_imgfloat_right\" img_height=\"225\" img_width=\"300\" titlename=\"聚合酶链反应的热循环仪\" data-src=\"https://img02.sogoucdn.com/app/a/200698/sogou_science_17115\"> </p><p><img alt=\"\" class=\"fileimage kx_img ed_imgfloat_right\" img_height=\"338\" img_width=\"300\" titlename=\"一种用于聚合酶链反应的老式三温热循环仪\" data-src=\"https://img03.sogoucdn.com/app/a/200698/sogou_science_17116\"> </p><p>PCR扩增DNA链的特定区域(DNA靶)。大多数聚合酶链反应方法扩增长度在0.1至10千碱基对（kbp）之间的DNA片段，尽管一些技术允许扩增高达40 kbp的片段。<sup><a href=\"#quote_1\" class=\"kx_ref\">[1]</a></sup>扩增产物的量由反应中可用的底物决定，随着反应的进行，底物的数量变得有限。<sup><a href=\"#quote_2\" class=\"kx_ref\">[2]</a></sup> </p><p>基本的聚合酶链反应设置需要几种组分和试剂，包括包含要扩增的DNA目标区域的<i>DNA模板</i>；<i> DNA聚合酶</i>；一种聚合新DNA链的酶；耐热的 Taq聚合酶特别常见，<sup><a href=\"#quote_3\" class=\"kx_ref\">[3]</a></sup>因为它更有可能在高温DNA变性过程中保持完整；两个DNA<i>引物分别</i>与DNA靶的每条正义链和反义链的 3’末端互补(DNA聚合酶只能与DNA双链区域结合并从双链区域延伸；没有引物，就没有聚合酶可以结合的双链起始位点)；<sup><a href=\"#quote_4\" class=\"kx_ref\">[4]</a></sup>预先选择与DNA靶区互补的特异性引物，通常在实验室定制或从商业生化供应商处购买；<i>脱氧核苷三磷酸</i>，或dNTPs(有时称为“脱氧核苷酸三磷酸”；核苷酸含有三磷酸基团)，DNA聚合酶从中合成新的DNA链的构建模块；a<i>缓冲溶液</i>为DNA聚合酶的最佳活性和稳定性提供合适的化学环境；<i>二价 阳离子</i>通常为镁或锰离子；其中镁离子是最常见的，但是Mn<sup>2+</sup>可作为高锰酸盐用于PCR介导的DNA诱变， 锰离子浓度会增加DNA合成过程中的错误率；<sup><a href=\"#quote_5\" class=\"kx_ref\">[5]</a></sup>其中<i>单价阳离子</i>通常为钾离子</p><p>PCR反应通常在10-200&nbsp;μL体积内进行，在小反应管中(0.2-0.5&nbsp;μL体积)放入热循环仪中。热循环仪加热和冷却反应管，以达到反应中每个步骤所需的温度(见下文)。许多现代热循环仪利用珀耳帖效应，该效应允许通过简单地逆转电流来加热和冷却PCR管。薄壁反应管允许良好的导热性，以实现快速热平衡。大多数热循环仪都有加热的盖子，以防止反应管顶部冷凝。没有加热盖的老式热循环仪需要在反应混合物顶部涂一层油，或者在管内涂一团蜡。</p><p></p><h3>程序</h3><p></p><p>通常，PCR由一系列20-40次重复的温度变化组成，称为热循环，每个循环通常由两个或三个离散的温度步骤组成(见下图)。循环通常在非常高的温度下进行单个温度步骤(&gt;90 degrees Celsius (194 degrees Fahrenheit))，然后在最后保持一次，以便最终产品扩展或短暂存储。在每个循环中使用的温度和时间取决于各种参数，包括用于DNA合成的酶、反应中二价离子和dNTPs的浓度以及引物的熔解温度(<i>Tm</i>)。<sup><a href=\"#quote_6\" class=\"kx_ref\">[6]</a></sup>大多数聚合酶链反应方法共有的步骤如下:</p><p>\n </p><ul>\n  <li><i>初始化</i>:此步骤仅适用于需要热激活的DNA聚合酶热启动聚合酶链反应。它包括将反应室加热至94–96 °C (201–205 °F)，或 98 °C (208 °F)，如果使用极端耐热的聚合酶，然后保持1-10分钟。</li>\n  <li><i>变性</i>:该步骤是第一次常规循环活动，包括将反应室加热至94-98°C(201-208°F) 持续20-30秒。通过破坏互补碱基之间的氢键，导致脱氧核糖核酸熔解或变性，产生两个单链DNA分子。</li>\n  <li><i>退火</i>:在下一步中，反应温度降低到50-65℃(122-149°F)持续20-40秒，使每个单链DNA模板上的引物退火。反应混合物中通常包含两种不同的引物:包含靶区的两种单链互补物各一种。引物本身是单链序列，但比靶区的长度短得多，只能补充每个链3'端非常短的序列。</li>\n </ul><p></p><p>\n </p><dl>\n  <dd>\n   确定合适的退火温度至关重要，因为退火温度对影响效率和特异性有很大影响。该温度必须足够低以允许引物与链的杂交，但是足够高以使杂交具有特异性，即引物应该 <i>仅仅与链中</i>完全互补的部分结合，而不是其他地方。如果温度过低，引物可能结合不完全。如果太高，引物可能根本不会结合。典型的退火温度大约在引物 <i>Tm</i>以下3–5 °C。只有当引物序列与模板序列非常接近时，互补碱基之间才会形成稳定的氢键。在这一步骤中，聚合酶结合引物-模板杂交体，并开始DNA形成。\n  </dd>\n </dl><p></p><p>\n </p><ul>\n  <li><i>延伸/伸长</i>:这一步的温度取决于所用的DNA聚合酶； Taq(水生栖热菌)聚合酶耐热DNA聚合酶的最佳活性温度约为75-80°C(167-176°F),尽管该酶通常使用72°C(162°F)的温度。在该步骤中，DNA聚合酶通过从反应混合物中添加在5’-至-3’方向上与模板互补的游离DNA链来合成与DNA模板链互补的新DNA链，在新生(伸长)DNA链的末端，将dNTPs的5'-磷酸基与3'-羟基缩合。延伸所需的精确时间取决于所使用的DNA聚合酶和要扩增的DNA目标区域的长度。根据经验，在最佳温度下，大多数DNA聚合酶每分钟聚合一千个碱基。在最佳条件下(即，如果没有限制底物或试剂的限制)，在每个延伸/延伸步骤中，DNA靶序列的数量加倍。随着每一个连续的循环，原始模板链加上所有新产生的链成为下一轮延伸的模板链，导致特定DNA目标区域的指数(几何)扩增。</li>\n </ul><p></p><p>\n </p><dl>\n  <dd>\n   变性、退火和伸长的过程构成一个单一的循环。将DNA目标扩增到数百万个拷贝需要多个周期。用于计算给定循环次数后形成的DNA拷贝数的公式是2 <sup>n</sup>，其中 <i>n</i>是周期数。因此，30个循环的反应集产生2个 <sup>30</sup>，或1073741824原始双链DNA目标区域的拷贝。\n  </dd>\n </dl><p></p><p>\n </p><ul>\n  <li><i>最终伸长率</i>:此单一步骤是可选的，但在70 - 74°C的温度(158 - 165°F)下执行(PCR中大多数聚合酶最佳活性所需的温度范围)在最后一个PCR循环后5-15分钟，以确保任何剩余的单链DNA被完全延长。</li>\n  <li><i>最终保持</i>:最后一步将反应室冷却至4–15 °C (39–59 °F)，并且可以用于PCR产物的短期储存。</li>\n </ul><p></p><p><img alt=\"完整聚合酶链反应循环示意图\" class=\"ed_imgfloat_embedmiddle\" img_height=\"435\" img_width=\"835\" titlename=\"\" data-align=\"embedmiddle\" width=\"480\" data-src=\"https://img04.sogoucdn.com/app/a/200698/sogou_science_17117\"> </p><p><img alt=\"\" class=\"ed_imgfloat_embedmiddle\" img_height=\"283\" img_width=\"300\" titlename=\"溴化乙锭-染色后的聚合酶链反应产物凝胶电泳。两组引物用于扩增来自三种不同组织样品的靶序列。样品#1中不存在扩增；样品#2和#3中的DNA条带表明目标序列成功扩增。凝胶还显示出阳性对照，以及包含定义长度的DNA片段的DNA梯子，用于确定实验性PCRs中条带的大小。\" data-align=\"embedmiddle\" width=\"300\" data-src=\"https://img03.sogoucdn.com/app/a/200698/sogou_science_17118\"> </p><p>为了检查聚合酶链反应是否成功产生预期的目的DNA(有时也称为扩增子或扩增子)，可以使用琼脂糖凝胶电泳进行聚合酶链反应产物的大小分离。聚合酶链反应产物的大小是通过与 DNA梯比较来确定的，DNA梯是一种分子量标记，包含与聚合酶链反应产物一起在凝胶上运行的已知大小的DNA片段。</p><p><img alt=\"塔克聚合酶链反应\" class=\"ed_imgfloat_embedmiddle\" img_height=\"707\" img_width=\"773\" titlename=\"\" data-align=\"embedmiddle\" width=\"480\" data-src=\"https://img03.sogoucdn.com/app/a/200698/sogou_science_17119\"> </p><p></p><h3>阶段</h3><p></p><p>与其他化学反应一样，聚合酶链反应的反应速率和效率受到限制因素的影响。因此，整个聚合酶链反应过程可以根据反应进程进一步分为三个阶段:</p><p>\n </p><ul>\n  <li><i>指数放大</i>:在每个循环中，产物的量加倍(假设100%反应效率)。经过30个周期后，DNA的单个拷贝可以增加到10亿个拷贝。从某种意义上说，一条离散DNA链的复制是在试管中于受控条件下进行的。该反应非常敏感:只需要少量的DNA。</li>\n  <li>稳定阶段当DNA聚合酶失去活性时，以及DNA聚合酶和引物等试剂的消耗会导致他们变得更加有限，反应会变慢。</li>\n  <li><i>平台期</i>:由于试剂和酶耗尽，不再有产品积累。</li>\n </ul><p></p>","pics":[{"originalUrl":"https://web.archive.org/web/20221028220934/https://img02.sogoucdn.com/app/a/200698/sogou_science_17115?w=300&h=225&titlename=%E8%81%9A%E5%90%88%E9%85%B6%E9%93%BE%E5%8F%8D%E5%BA%94%E7%9A%84%E7%83%AD%E5%BE%AA%E7%8E%AF%E4%BB%AA","url":"https://web.archive.org/web/20221028220934/https://img02.sogoucdn.com/app/a/200698/sogou_science_17115","rw":300,"rh":225,"title":"聚合酶链反应的热循环仪","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221028220934/https://img03.sogoucdn.com/app/a/200698/sogou_science_17116?w=300&h=338&titlename=%E4%B8%80%E7%A7%8D%E7%94%A8%E4%BA%8E%E8%81%9A%E5%90%88%E9%85%B6%E9%93%BE%E5%8F%8D%E5%BA%94%E7%9A%84%E8%80%81%E5%BC%8F%E4%B8%89%E6%B8%A9%E7%83%AD%E5%BE%AA%E7%8E%AF%E4%BB%AA","url":"https://web.archive.org/web/20221028220934/https://img03.sogoucdn.com/app/a/200698/sogou_science_17116","rw":300,"rh":338,"title":"一种用于聚合酶链反应的老式三温热循环仪","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221028220934/https://img04.sogoucdn.com/app/a/200698/sogou_science_17117?titlename=&w=835&h=435","url":"https://web.archive.org/web/20221028220934/https://img04.sogoucdn.com/app/a/200698/sogou_science_17117","rw":835,"rh":435,"title":"","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221028220934/https://img03.sogoucdn.com/app/a/200698/sogou_science_17118?w=300&h=283&titlename=%E6%BA%B4%E5%8C%96%E4%B9%99%E9%94%AD-%E6%9F%93%E8%89%B2%E5%90%8E%E7%9A%84%E8%81%9A%E5%90%88%E9%85%B6%E9%93%BE%E5%8F%8D%E5%BA%94%E4%BA%A7%E7%89%A9%E5%87%9D%E8%83%B6%E7%94%B5%E6%B3%B3%E3%80%82%E4%B8%A4%E7%BB%84%E5%BC%95%E7%89%A9%E7%94%A8%E4%BA%8E%E6%89%A9%E5%A2%9E%E6%9D%A5%E8%87%AA%E4%B8%89%E7%A7%8D%E4%B8%8D%E5%90%8C%E7%BB%84%E7%BB%87%E6%A0%B7%E5%93%81%E7%9A%84%E9%9D%B6%E5%BA%8F%E5%88%97%E3%80%82%E6%A0%B7%E5%93%81%231%E4%B8%AD%E4%B8%8D%E5%AD%98%E5%9C%A8%E6%89%A9%E5%A2%9E%EF%BC%9B%E6%A0%B7%E5%93%81%232%E5%92%8C%233%E4%B8%AD%E7%9A%84DNA%E6%9D%A1%E5%B8%A6%E8%A1%A8%E6%98%8E%E7%9B%AE%E6%A0%87%E5%BA%8F%E5%88%97%E6%88%90%E5%8A%9F%E6%89%A9%E5%A2%9E%E3%80%82%E5%87%9D%E8%83%B6%E8%BF%98%E6%98%BE%E7%A4%BA%E5%87%BA%E9%98%B3%E6%80%A7%E5%AF%B9%E7%85%A7%EF%BC%8C%E4%BB%A5%E5%8F%8A%E5%8C%85%E5%90%AB%E5%AE%9A%E4%B9%89%E9%95%BF%E5%BA%A6%E7%9A%84DNA%E7%89%87%E6%AE%B5%E7%9A%84DNA%E6%A2%AF%E5%AD%90%EF%BC%8C%E7%94%A8%E4%BA%8E%E7%A1%AE%E5%AE%9A%E5%AE%9E%E9%AA%8C%E6%80%A7PCRs%E4%B8%AD%E6%9D%A1%E5%B8%A6%E7%9A%84%E5%A4%A7%E5%B0%8F%E3%80%82","url":"https://web.archive.org/web/20221028220934/https://img03.sogoucdn.com/app/a/200698/sogou_science_17118","rw":300,"rh":283,"title":"溴化乙锭-染色后的聚合酶链反应产物凝胶电泳。两组引物用于扩增来自三种不同组织样品的靶序列。样品#1中不存在扩增；样品#2和#3中的DNA条带表明目标序列成功扩增。凝胶还显示出阳性对照，以及包含定义长度的DNA片段的DNA梯子，用于确定实验性PCRs中条带的大小。","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221028220934/https://img03.sogoucdn.com/app/a/200698/sogou_science_17119?w=773&h=707&titlename=","url":"https://web.archive.org/web/20221028220934/https://img03.sogoucdn.com/app/a/200698/sogou_science_17119","rw":773,"rh":707,"title":"","alt":null,"width":0,"height":0}],"card":null,"references":[],"versionCount":0},{"paragraphId":"14996399716303370","title":"优化","versionId":"54568173541843213","lemmaId":11182,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82815118,"name":"iscience","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599821165,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>实际上，PCR可能由于各种原因失败，部分原因是它对污染的敏感性导致虚假DNA产物的扩增。因此，已经开发了许多技术和程序来优化聚合酶链反应条件。<sup><a href=\"#quote_7\" class=\"kx_ref\">[7]</a></sup><sup><a href=\"#quote_8\" class=\"kx_ref\">[8]</a></sup>外来DNA的污染通过实验室协议和程序来解决，这些协议和程序将PCR前混合物与潜在的DNA污染物分开。<sup><a href=\"#quote_9\" class=\"kx_ref\">[9]</a></sup>这通常包括将聚合酶链式反应设置区域与聚合酶链式反应产品的分析或纯化区域空间分离，使用一次性塑料制品，以及彻底清洁反应设置之间的工作表面。引物设计技术对于提高PCR产物产量和避免假产物的形成非常重要，使用替代缓冲组分或聚合酶可以帮助扩增DNA的长区域或其他有问题的区域。在缓冲体系中加入试剂，如甲酰胺，可以提高聚合酶链反应的特异性和产率。<sup><a href=\"#quote_9\" class=\"kx_ref\">[9]</a></sup>可以对理论聚合酶链反应结果进行计算机模拟(电子聚合酶链反应)，以帮助引物设计。<sup><a href=\"#quote_10\" class=\"kx_ref\">[10]</a></sup> </p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14996399716303371","title":"应用程序","versionId":"54568173541843214","lemmaId":11182,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82815118,"name":"iscience","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599821165,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p></p><h3>选择性脱氧核糖核酸分离</h3><p></p><p>PCR允许从基因组DNA中通过选择性扩增DNA的特定区域分离DNA片段。聚合酶链反应的使用增加了许多方法，例如用于Southern或 Northern杂交的杂交探针和 DNA克隆，这需要更大量的DNA，代表特定的DNA区域。PCR为这些技术提供了大量的纯DNA，使得即使从非常少量的原始材料中也能够分析DNA样品。</p><p>聚合酶链反应的其他应用包括脱氧核糖核酸测序以确定未知的聚合酶链反应扩增序列，其中一个扩增引物可用于Sanger测序、分离脱氧核糖核酸序列以加速重组脱氧核糖核酸技术，该技术涉及将脱氧核糖核酸序列插入到质粒、噬菌体或粘粒(取决于大小)或另一生物体的遗传物质中。细菌菌落<i>(如大肠杆菌)</i>可以通过聚合酶链反应快速筛选正确的DNA 载体构建体。<sup><a href=\"#quote_11\" class=\"kx_ref\">[11]</a></sup>聚合酶链反应也可用于遗传指纹；一种法医技术，用于通过不同的PCR方法比较实验性DNA来鉴定人或其他生物。</p><p>一些聚合酶链反应的“指纹”方法具有很高的辨别能力，可以用于识别个体之间的遗传关系，例如亲子关系或兄弟姐妹之间的遗传关系，并用于亲子鉴定(图4)。当使用特定的分子钟(即微生物的16S rRNA 和recA基因)时，该技术也可以用于确定生物体之间的进化关系。<sup><a href=\"#quote_12\" class=\"kx_ref\">[12]</a></sup> </p><p><img alt=\"\" class=\"ed_imgfloat_embedmiddle\" img_height=\"301\" img_width=\"200\" titlename=\"聚合酶链反应扩增的DNA片段的电泳。(1)父亲。(2)儿童。(3)母亲。这个孩子继承了父母的一些指纹，但不是全部，给了它一个新的独特的指纹。\" data-align=\"embedmiddle\" width=\"200\" data-src=\"https://img02.sogoucdn.com/app/a/200698/sogou_science_17120\"> </p><p></p><h3>DNA的扩增和定量</h3><p></p><p>因为PCR扩增了目的DNA区域，所以PCR可以用于分析极少量的样品。这对于法医检定法，当时只有微量的DNA可作为证据。聚合酶链反应也可用于分析古代DNA那已经有数万年的历史了。这些基于聚合酶链反应的技术已经成功地应用于动物身上，例如四万年前猛犸，以及人类DNA，应用范围从分析埃及木乃伊识别一个俄罗斯沙皇和英国国王理查三世遗体的鉴定。<sup><a href=\"#quote_13\" class=\"kx_ref\">[13]</a></sup> </p><p>定量聚合酶链反应或者实时聚合酶链反应<sup><a href=\"#quote_14\" class=\"kx_ref\">[14]</a></sup>不要与逆转录-聚合酶链反应方法混淆)允许估计样品中存在的给定序列的量——这种技术通常用于定量确定基因表达的水平。定量PCR是一种已建立的DNA定量工具，用于测量每轮PCR扩增后DNA产物的积累。</p><p>定量PCR允许实时定量和检测特定的DNA序列，因为它在合成过程中测量浓度。有两种方法可以同时检测和定量。第一种方法是使用在DNA双链之间非特异性保留的荧光染料。第二种方法涉及编码特定序列并被荧光标记的探针。只有在探针与其互补DNA杂交后，才能使用这些方法检测DNA。一种有趣的技术组合是实时PCR和逆转录。这种复杂的技术被称为RT-qPCR，允许定量少量RNA。通过这种组合技术，mRNA被转化为cDNA，并用qPCR进一步定量。这种技术降低了聚合酶链反应终点出错的可能性，<sup><a href=\"#quote_15\" class=\"kx_ref\">[15]</a></sup>增加了检测与遗传疾病如癌症相关的基因的机会。<sup><a href=\"#quote_16\" class=\"kx_ref\">[16]</a></sup>实验室使用RT-qPCR来灵敏地测量基因调控。</p><p></p><h3>医疗和诊断应用</h3><p></p><p>未来的父母可以被测试是否是遗传携带者，或者他们的孩子可以被测试是否真的受到了疾病的影响。<sup><a href=\"#quote_16\" class=\"kx_ref\">[16]</a></sup>用于产前检测的DNA样品可以通过羊膜穿刺术、绒毛取样或甚至通过分析在母亲血流中循环的稀有胎儿细胞获得。聚合酶链反应分析对于植入前遗传诊断也是至关重要的，在植入前遗传诊断中，对发育中胚胎的单个细胞进行突变测试。</p><p>\n </p><ul>\n  <li>聚合酶链反应也可以用作以下敏感试验的一部分<i>组织分型</i>，对器官移植至关重要。截至2008年，甚至有人提议用聚合酶链反应检测取代传统的血型抗体检测。<sup><a href=\"#quote_16\" class=\"kx_ref\">[16]</a></sup></li>\n  <li>许多形式的癌症涉及致癌基因的改变。通过使用基于聚合酶链反应的测试来研究这些突变，治疗方案有时可以根据患者的不同而不同。聚合酶链反应可以早期诊断恶性疾病，如白血病和淋巴瘤，这是目前癌症研究中发展最快的，并且已经被常规使用。PCR检测可以直接在基因组DNA样品上进行，以检测易位特异性恶性细胞，其灵敏度至少是其他方法的10，000倍。<sup><a href=\"#quote_17\" class=\"kx_ref\">[17]</a></sup>聚合酶链反应在医学领域非常有用，因为它允许分离和扩增肿瘤抑制剂。例如，定量PCR可以用于量化和分析单细胞，以及识别DNA、mRNA和蛋白质的确认和组合。<sup><a href=\"#quote_15\" class=\"kx_ref\">[15]</a></sup></li>\n </ul><p></p><p></p><h3>传染病应用</h3><p></p><p>聚合酶链反应可以快速和高度特异性地诊断传染疾病病，包括由细菌或病毒引起的疾病。<sup><a href=\"#quote_18\" class=\"kx_ref\">[18]</a></sup>聚合酶链反应还允许从组织培养试验和动物模型中鉴定不可培养或生长缓慢的微生物，例如分枝杆菌、厌氧菌或病毒。聚合酶链反应在微生物学中诊断应用的基础是检测感染因子和借助特定基因从病原菌株中区分非病原性。<sup><a href=\"#quote_18\" class=\"kx_ref\">[18]</a></sup><sup><a href=\"#quote_19\" class=\"kx_ref\">[19]</a></sup> </p><p>通过PCR，传染病生物体的表征和检测已经在以下方面发生了革命性的变化:</p><p>\n </p><ul>\n  <li>这<i>人类免疫缺陷病毒</i>(或<i>艾滋病毒</i>)，是一个很难找到和根除的目标。最早的感染测试依赖于血液中循环的病毒抗体的存在。然而，抗体直到感染后许多周才会出现，母体抗体掩盖了新生儿的感染，而对抗感染的治疗药物不影响抗体。PCR 测试已经被开发出来，它可以检测超过50，000个宿主细胞的DNA中的一个病毒基因组。<sup><a href=\"#quote_20\" class=\"kx_ref\">[20]</a></sup>可以更早地检测感染，可以直接筛查捐赠的血液中的病毒，可以立即检测新生儿的感染，并且可以对抗病毒治疗的效果进行量化。</li>\n  <li>一些疾病生物，如<i>肺结核</i>，很难从病人身上取样，而且在实验室里生长缓慢。基于聚合酶链反应的检测可以方便地检测少量疾病生物(活的或死的)样品。详细的遗传分析也可以用来检测抗生素耐药性，从而允许立即有效的治疗。治疗效果也可以立即评估。</li>\n  <li>可通过PCR检测监测疾病病菌通过家畜或野生动物种群的传播。在许多情况下，可以检测和监测新毒性亚型的出现。也可以通过聚合酶链反应分析来确定导致早期流行的生物体的亚型。</li>\n  <li>病毒DNA可以通过PCR检测。所使用的引物必须对病毒DNA中的目标序列具有特异性，并且PCR可以用于病毒基因组的诊断分析或DNA测序。聚合酶链反应的高灵敏度允许在感染后不久甚至在疾病开始之前检测病毒。<sup><a href=\"#quote_18\" class=\"kx_ref\">[18]</a></sup>这种早期发现可能会给医生治疗带来很长的准备时间。患者体内的病毒量(“病毒载量”)也可以通过基于聚合酶链反应的DNA定量技术进行定量(见下文)。</li>\n  <li>百日咳等疾病是由细菌百日咳杆菌引起的。这种细菌的特点是严重的急性呼吸道感染，影响到各种动物和人类，并导致许多幼儿死亡。百日咳毒素是一种蛋白质外毒素，通过两个二聚体与细胞受体结合，并与在细胞免疫中发挥作用的不同细胞类型如T淋巴细胞反应。<sup><a href=\"#quote_21\" class=\"kx_ref\">[21]</a></sup>聚合酶链反应是一种重要的检测工具，可以检测百日咳毒素基因中的序列。这是因为聚合酶链反应对毒素有很高的灵敏度，并显示出快速的周转时间。与常规培养相比，聚合酶链反应对百日咳的诊断非常有效。<sup><a href=\"#quote_22\" class=\"kx_ref\">[22]</a></sup></li>\n </ul><p></p><p></p><h3>法医应用</h3><p></p><p>基于聚合酶链反应的遗传(或脱氧核糖核酸)指纹图谱协议的发展已在法医学中得到广泛应用:</p><p>\n </p><ul>\n  <li><i>遗传指纹</i>以其最具辨别力的形式，可以从世界上所有人群中独一无二地区分任何一个人。微小的DNA样本可以从犯罪现场，和比较的从嫌疑人那里，或者从DNA资料库早期的证据或罪犯。这些测试的简单版本通常用于在刑事调查中快速排除嫌疑人。几十年前的犯罪证据可以被检验，确认或免责最初被定罪的人。</li>\n  <li>通过分析犯罪现场发现的证据，法医DNA分型已经成为鉴定犯罪嫌疑人或为其开脱的有效方法。人类基因组有许多重复区域，可以在基因序列中或基因组的非编码区域中发现。具体来说，高达40%的人类DNA是重复的。<sup><a href=\"#quote_16\" class=\"kx_ref\">[16]</a></sup>基因组中这些重复的非编码区域有两种不同的类别。第一类称为可变数目串联重复序列(VNTR)，长度为10-100个碱基对，第二类称为短串联重复序列(STR)，由重复的2-10个碱基对部分组成。PCR用于使用位于每个重复区域侧翼的引物扩增几个众所周知的VNTRs和str。从任何个体获得的每个str片段的大小将表明存在哪些等位基因。通过分析一个人的几个str，会发现每个人的一组等位基因在统计上可能是唯一的。<sup><a href=\"#quote_16\" class=\"kx_ref\">[16]</a></sup>研究人员已经确定了人类基因组的完整序列。该序列可以通过NCBI网站轻松访问，并用于许多实际应用中。例如，联邦调查局已经汇编了一组用于识别的DNA标记位点，这些位点被称为组合DNA索引系统(CODIS) DNA数据库。<sup><a href=\"#quote_16\" class=\"kx_ref\">[16]</a></sup>使用该数据库可以使用统计分析来确定DNA样品匹配的概率。PCR是一种非常强大和重要的分析工具，用于法医DNA分型，因为研究人员只需要非常少量的目标DNA用于分析。例如，一根附着毛囊的人的头发有足够的DNA进行分析。类似地，一些精子、指甲下的皮肤样本或少量血液可以为结论性分析提供足够的DNA。<sup><a href=\"#quote_16\" class=\"kx_ref\">[16]</a></sup></li>\n  <li>脱氧核糖核酸指纹中较小的区别有助于<i>亲子鉴定，在亲子鉴定中，</i>一个人和他的近亲相匹配。可以测试未知人类遗骸的DNA，并与可能的父母、兄弟姐妹或儿童进行比较。类似的测试可以用来确认被收养(或绑架)孩子的亲生父母。新生儿的实际生父也可以被确认(或排除)。</li>\n  <li>PCR AMGX/AMGY设计已被证明不仅有助于从非常少量的基因组中扩增DNA序列。然而，它也可以用于法医骨骼样本的实时性别确定。这为我们提供了一种强大而有效的方法，不仅可以确定古代标本的性别，还可以确定当前犯罪嫌疑人的性别。<sup><a href=\"#quote_23\" class=\"kx_ref\">[23]</a></sup></li>\n </ul><p></p><p></p><h3>研究应用</h3><p></p><p>PCR已经应用于分子遗传学的许多研究领域:</p><p>\n </p><ul>\n  <li>聚合酶链反应允许快速生产短DNA片段，即使已知的引物序列不超过两个。聚合酶链反应的这种能力增强了许多方法，例如生成<i>杂交 探针</i>用于 Southern 或northern blot 杂交。聚合酶链反应为这些技术提供了大量的纯DNA，有时是单链的，甚至可以从非常少量的起始材料中进行分析。</li>\n  <li><i>脱氧核糖核酸测序</i>的任务也可以通过聚合酶链反应来辅助完成。已知的DNA片段可以很容易地从患有遗传疾病突变的患者体内产生。对扩增技术的修饰可以从完全未知的基因组中提取片段，或者可以仅产生感兴趣区域的单链。</li>\n  <li>聚合酶链反应有许多应用于传统的<i> DNA克隆</i>。它可以从更大的基因组中提取片段以插入载体，这可能只有少量可用。使用一组“载体引物”，它还可以分析或提取已经插入载体的片段。聚合酶链反应方案的一些改变可以<i>产生突变</i>(通用的或定点的)插入片段。</li>\n  <li><i>序列标签站点</i>是一个过程，其中PCR被用作基因组的特定片段存在于特定克隆中的指示物。人类基因组计划发现这一应用对于绘制他们测序的粘粒克隆以及协调不同实验室的结果至关重要。</li>\n  <li>聚合酶链反应的一个令人兴奋的应用是对来自远古来源的DNA进行系统进化分析，例如在尼安德特人的复原骨骼中、从猛犸的冷冻组织中或从埃及木乃伊的大脑中发现的DNA已经被放大和测序。<sup><a href=\"#quote_24\" class=\"kx_ref\">[24]</a></sup>在某些情况下，这些来源的高度降解的DNA可能在扩增的早期阶段重新组装。</li>\n  <li>聚合酶链反应的一个常见应用是对以下<i>基因表达模式的研究</i>。组织(甚至单个细胞)可以在不同阶段进行分析，以了解哪些基因变得活跃，哪些基因被关闭。该应用还可以使用定量聚合酶链反应来定量表达的实际水平</li>\n  <li>聚合酶链反应同时扩增单个精子中几个基因座的能力<sup><a href=\"#quote_24\" class=\"kx_ref\">[24]</a></sup>大大增强了极大地增强了通过研究减数分裂后染色体交叉来进行基因定位的传统任务。通过分析数千个单个精子，已经直接观察到非常紧密基因座之间罕见的交叉事件。类似地，可以分析异常的缺失、插入、易位或倒位，所有这些都无需等待(或支付)漫长而艰苦的受精、胚胎发生等过程。</li>\n  <li>定点突变:聚合酶链反应可用于产生突变基因，突变由科学家随意选择。可以选择这些突变来理解蛋白质是如何完成其功能的，并改变或改善蛋白质功能。</li>\n </ul><p></p>","pics":[{"originalUrl":"https://web.archive.org/web/20221028220934/https://img02.sogoucdn.com/app/a/200698/sogou_science_17120?h=301&titlename=%E8%81%9A%E5%90%88%E9%85%B6%E9%93%BE%E5%8F%8D%E5%BA%94%E6%89%A9%E5%A2%9E%E7%9A%84DNA%E7%89%87%E6%AE%B5%E7%9A%84%E7%94%B5%E6%B3%B3%E3%80%82(1)%E7%88%B6%E4%BA%B2%E3%80%82(2)%E5%84%BF%E7%AB%A5%E3%80%82(3)%E6%AF%8D%E4%BA%B2%E3%80%82%E8%BF%99%E4%B8%AA%E5%AD%A9%E5%AD%90%E7%BB%A7%E6%89%BF%E4%BA%86%E7%88%B6%E6%AF%8D%E7%9A%84%E4%B8%80%E4%BA%9B%E6%8C%87%E7%BA%B9%EF%BC%8C%E4%BD%86%E4%B8%8D%E6%98%AF%E5%85%A8%E9%83%A8%EF%BC%8C%E7%BB%99%E4%BA%86%E5%AE%83%E4%B8%80%E4%B8%AA%E6%96%B0%E7%9A%84%E7%8B%AC%E7%89%B9%E7%9A%84%E6%8C%87%E7%BA%B9%E3%80%82&w=200","url":"https://web.archive.org/web/20221028220934/https://img02.sogoucdn.com/app/a/200698/sogou_science_17120","rw":200,"rh":301,"title":"聚合酶链反应扩增的DNA片段的电泳。(1)父亲。(2)儿童。(3)母亲。这个孩子继承了父母的一些指纹，但不是全部，给了它一个新的独特的指纹。","alt":null,"width":0,"height":0}],"card":null,"references":[],"versionCount":0},{"paragraphId":"14996399716303372","title":"优势","versionId":"54568173541843215","lemmaId":11182,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82815118,"name":"iscience","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599821165,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>聚合酶链反应有许多优点。它很容易理解和使用，并迅速产生结果。这项技术非常敏感，有可能产生数百万到数十亿份特定产品的拷贝，用于测序、克隆和分析。qRT-PCR具有与PCR相同的优点，同时还具有定量合成产物的优点。因此，它可以用于分析肿瘤、微生物或其他疾病状态中基因表达水平的变化。<sup><a href=\"#quote_25\" class=\"kx_ref\">[25]</a></sup> </p><p>聚合酶链反应是一种非常强大和实用的研究工具。许多疾病的未知病因的测序正在通过聚合酶链反应得到解决。这项技术可以帮助我们识别与已知病毒相关的未知病毒序列，从而让我们更好地了解疾病本身。如果该过程可以进一步简化，并且可以开发灵敏的非辐射检测系统，PCR将在未来几年在临床实验室中占据显著地位。<sup><a href=\"#quote_26\" class=\"kx_ref\">[26]</a></sup> </p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14996399733080591","title":"限制","versionId":"54568173541843216","lemmaId":11182,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82815118,"name":"iscience","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599821165,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>聚合酶链反应的一个主要限制是，为了产生允许其选择性扩增的引物，需要关于目标序列的先前信息。<sup><a href=\"#quote_26\" class=\"kx_ref\">[26]</a></sup>这意味着，通常情况下，PCR用户必须知道两个单链模板中每个模板上目标区域上游的精确序列，以确保DNA聚合酶正确结合引物-模板杂交体，并随后在DNA合成过程中产生整个目标区域。</p><p>像所有酶一样，DNA聚合酶也容易出错，这反过来会导致产生的PCR片段发生突变。<sup><a href=\"#quote_27\" class=\"kx_ref\">[27]</a></sup> </p><p>PCR的另一个限制是，即使是最少量的污染DNA也可以被扩增，导致误导或模糊的结果。为了最大限度地减少污染的可能性，调查人员应该为试剂制备、聚合酶链反应和产品分析预留单独的房间。试剂应分配到一次性的等分试样中。应经常使用带有一次性柱塞和超长移液器吸头的移液器。<sup><a href=\"#quote_28\" class=\"kx_ref\">[28]</a></sup> </p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14996399733080592","title":"变化","versionId":"54568173541843217","lemmaId":11182,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82815118,"name":"iscience","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599821165,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>\n <ul>\n  <li><i>等位基因特异性聚合酶链反应</i>:基于单核苷酸变异的诊断或克隆技术(snv不要与 SNPs 混淆)(患者的单碱基差异)。它需要事先知道DNA序列，包括等位基因之间的差异，并使用3’端包含SNV的引物(通常包含SNV周围的碱基对缓冲液)。在模板和引物不匹配的情况下，严格条件下的PCR扩增效率要低得多，因此用单核苷酸多态性特异性引物成功扩增表明序列中存在特异性单核苷酸多态性。<sup><a href=\"#quote_28\" class=\"kx_ref\">[28]</a></sup>有关更多信息，请参见单核苷酸多态性基因分型。</li>\n  <li><i>装配聚合酶链反应</i>或者<i>聚合酶循环组件</i>:通过对具有短重叠片段的长寡核苷酸池进行PCR来人工合成长DNA序列。寡核苷酸在有义和反义方向之间交替，重叠片段决定聚合酶链反应片段的顺序，从而选择性地产生最终的长DNA产物。<sup><a href=\"#quote_29\" class=\"kx_ref\">[29]</a></sup></li>\n  <li><i>不对称聚合酶链反应</i>:优先扩增双链DNA模板中的一条DNA链。它用于测序和杂交探测，其中只需要扩增两条互补链中的一条。聚合酶链反应照常进行，但扩增目标链的引物过量。由于限制性引物用完后，反应后期的扩增速度较慢(算术)，因此需要额外的PCR循环。<sup><a href=\"#quote_30\" class=\"kx_ref\">[30]</a></sup>这个过程最近的一个修改，称为<i>L²</i>线性-<i>A</i>之后-<i>T</i>他-<i>E</i>xponential-PCR (LATE-PCR)使用比过量引物熔点更高的限制性引物( Tm )，以保持反应效率，因为限制性引物浓度在反应中期降低。<sup><a href=\"#quote_31\" class=\"kx_ref\">[31]</a></sup></li>\n  <li><i>对流聚合酶链反应</i>:一种伪等温PCR方法。将溶液置于热梯度下，而不是反复加热和冷却聚合酶链反应混合物。由此产生的热不稳定性驱动对流流动自动将聚合酶链反应试剂从热区域和冷区域打乱，从而反复启动聚合酶链反应。<sup><a href=\"#quote_32\" class=\"kx_ref\">[32]</a></sup>通过利用混沌流场的出现，可以优化热边界条件和PCR外壳的几何形状等参数，以产生特异和快速的PCR。<sup><a href=\"#quote_33\" class=\"kx_ref\">[33]</a></sup>这种对流聚合酶链反应设置显著降低了设备功率需求和操作时间。</li>\n  <li><i>拨号聚合酶链反应</i>:一种高度并行的方法，用于为基因合成检索精确的DNA分子。在大规模平行测序之前，用独特的侧翼标签修饰复杂的DNA分子库。然后，标签导向引物能够通过聚合酶链反应检索具有所需序列的分子。<sup><a href=\"#quote_34\" class=\"kx_ref\">[34]</a></sup></li>\n  <li><i>数字聚合酶链反应 (dPCR)</i>:用于测量DNA样品中目标DNA序列的数量。DNA样品被高度稀释，使得在并行运行许多PCR后，其中一些PCR不会接收到目标DNA的单个分子。目标DNA浓度是使用负结果的比例计算的。因此得名“数字聚合酶链反应”。</li>\n  <li><i>Helicase-dependent amplification</i>:与传统PCR相似，但使用恒温而不是通过变性和退火/延伸循环。 DNA解旋酶，一种解开DNA的酶，被用来代替热变性。<sup><a href=\"#quote_35\" class=\"kx_ref\">[35]</a></sup></li>\n  <li><i>热启动聚合酶链式反应</i>:一种在PCR的初始建立阶段减少非特异性扩增的技术。它可以通过将反应组分加热到变性温度(例如95&nbsp;c)在加入聚合酶之前。<sup><a href=\"#quote_36\" class=\"kx_ref\">[36]</a></sup>已经开发了特殊的酶系统，通过结合抗体来抑制聚合酶在环境温度下的活性<sup><a href=\"#quote_37\" class=\"kx_ref\">[37]</a></sup><sup><a href=\"#quote_37\" class=\"kx_ref\">[37]</a></sup>或者通过共价键结合的抑制剂的存在，该抑制剂仅在高温活化步骤后解离。热启动/冷完成聚合酶链反应是通过新的杂合聚合酶实现的，这些酶在环境温度下不活跃，在伸长温度下立即激活。</li>\n  <li><i>电子聚合酶链反应</i>(数字聚合酶链反应、虚拟聚合酶链反应、电子聚合酶链反应、电子聚合酶链反应)是指计算工具使用给定的一组引物 ( 探针)来扩增来自测序的基因组或转录组的DNA 序列，用于计算理论聚合酶链反应结果。电子PCR被提出作为分子生物学的教育工具。<sup><a href=\"#quote_38\" class=\"kx_ref\">[38]</a></sup></li>\n  <li><i>序列间特异性聚合酶链反应</i>(ISSR):一种用于DNA指纹识别的聚合酶链反应方法，它放大简单重复序列之间的区域，以产生扩增片段长度的独特指纹。<sup><a href=\"#quote_39\" class=\"kx_ref\">[39]</a></sup></li>\n  <li><i>反向聚合酶链反应</i>:通常用于识别基因组插入片段周围的侧翼序列。它涉及一系列的脱氧核糖核酸消化和自连接，在未知序列的两端产生已知序列。<sup><a href=\"#quote_40\" class=\"kx_ref\">[40]</a></sup></li>\n  <li><i>连接介导的聚合酶链反应</i>:使用连接到感兴趣的DNA的小DNA接头和退火到DNA接头的多个引物；它已被用于 DNA测序、基因组行走和 DNA足迹法。<sup><a href=\"#quote_41\" class=\"kx_ref\">[41]</a></sup></li>\n  <li><i>甲基化特异性聚合酶链反应</i>(MSP):由史蒂芬·贝林和詹姆斯·赫尔曼在约翰·霍普金斯大学医学院开发，<sup><a href=\"#quote_42\" class=\"kx_ref\">[42]</a></sup>用于检测基因组DNA中CpG岛的甲基化。DNA首先用亚硫酸氢钠处理，亚硫酸氢钠将未甲基化的胞嘧啶碱基转化为尿嘧啶，尿嘧啶被PCR引物识别为胸腺嘧啶。然后在修饰的DNA上进行两次PCR，使用除引物序列中任何CpG岛以外相同的引物组。在这些点上，一组引物用胞嘧啶识别DNA以扩增甲基化的DNA，一组引物用尿嘧啶或胸腺嘧啶识别DNA以扩增未甲基化的DNA。使用qPCR进行MSP也可以获得甲基化的定量而不是定性信息。</li>\n  <li><i>微型引物聚合酶链反应</i>:使用耐热聚合酶(S-Tbr)，可以从短引物延伸到9或10个核苷酸。这种方法允许PCR靶向较小的引物结合区域，并用于扩增保守的DNA序列，如16S(或真核18S) rRNA基因。<sup><a href=\"#quote_43\" class=\"kx_ref\">[43]</a></sup></li>\n  <li><i>多重连接依赖探针扩增</i>（<i>MLPA</i>):允许用单个引物对扩增多个靶标，从而避免多重PCR的分辨率限制(见下文)。</li>\n  <li><i>多重聚合酶链反应</i>:由单个PCR混合物中的多个引物组组成，以产生对不同DNA序列特异的不同大小的扩增子。通过同时靶向多个基因，可以从单次测试中获得额外的信息，否则将需要几次试剂和更多时间来执行。每个引物组的退火温度必须优化，以在单个反应中正确工作，并符合扩增子尺寸。也就是说，当通过凝胶电泳可视化时，它们的碱基对长度应该足够不同以形成不同的条带。</li>\n  <li><i>纳米粒子辅助聚合酶链反应</i>:一些纳米粒子(NPs)可以提高PCR的效率(因此被称为纳米PCR)，一些甚至可以优于原始的PCR增强剂。据报道，量子点可以提高聚合酶链反应的特异性和效率。单壁碳纳米管和多壁碳纳米管能有效增强长聚合酶链反应的扩增。纳米碳粉(成交堆积)可以提高重复聚合酶链反应和长聚合酶链反应的效率氧化锌,二氧化钛发现银纳米粒子能提高聚合酶链反应的产率。以前的数据表明非金属NP保留了可接受的放大保真度。鉴于许多纳米粒子能够提高聚合酶链反应效率，很明显纳米粒子技术的改进和产品开发有很大的潜力。<sup><a href=\"#quote_44\" class=\"kx_ref\">[44]</a></sup><sup><a href=\"#quote_45\" class=\"kx_ref\">[45]</a></sup></li>\n  <li><i>嵌套聚合酶链反应</i>:通过减少DNA非特异性扩增的背景，提高DNA扩增的特异性。两组引物用于两个连续的PCR。在第一个反应中，一对引物用于产生DNA产物，除了预期的靶之外，该产物可能仍然由非特异性扩增的DNA片段组成。然后用一组引物将产物用于第二次聚合酶链反应，所述引物的结合位点完全或部分不同于第一次反应中使用的每个引物，并且位于其中的3’。嵌套式PCR在特异性扩增长DNA片段方面通常比传统PCR更成功，但它需要更详细的目标序列知识。</li>\n  <li><i>重叠延伸聚合酶链反应</i>或者<i>通过重叠延伸拼接(SOEing)</i>:一种用于将两个或多个含有互补序列的DNA片段拼接在一起的基因工程技术。它用于连接含有基因、调节序列或突变的DNA片段；这项技术可以创造特定的长DNA构建体。它还可以将缺失、插入或点突变引入DNA序列。<sup><a href=\"#quote_46\" class=\"kx_ref\">[46]</a></sup><sup><a href=\"#quote_47\" class=\"kx_ref\">[47]</a></sup></li>\n  <li><i>PAN-AC</i>:使用等温条件进行扩增，并可用于活细胞。<sup><a href=\"#quote_48\" class=\"kx_ref\">[48]</a></sup><sup><a href=\"#quote_49\" class=\"kx_ref\">[49]</a></sup></li>\n  <li><i>定量聚合酶链反应</i>(qPCR):用于测量目标序列的数量(通常是实时的)。它定量测量DNA、cDNA或RNA的起始量。定量PCR 通常用于确定样品中是否存在DNA序列及其在样品中的拷贝数。<i>定量聚合酶链反应</i>具有非常高的精确度。定量PCR方法使用荧光染料，如Sybr Green、EvaGreen或含荧光团的DNA探针，如TaqMan ，实时测量扩增产物的量。它有时也被缩写为逆转录-聚合酶链反应(<i>实时的</i>聚合酶链反应)，但这个缩写只能用于 逆转录PCR 。定量PCR是定量PCR 的合适缩写。</li>\n  <li><i>逆转录聚合酶链反应(逆转录-聚合酶链反应)</i>:用于从RNA中扩增DNA。逆转录酶将核糖核酸逆转录成cDNA ，然后通过聚合酶链反应进行扩增。逆转录-聚合酶链反应广泛用于表达谱，以确定基因的表达或鉴定RNA转录物的序列，包括转录起始和终止位点。如果基因的基因组DNA序列是已知的，逆转录-聚合酶链反应可以用来绘制基因中外显子和内含子的位置。基因的5’端(对应于转录起始位点)通常通过 RACE-PCR (<i>快速扩增cDNA末端</i>)中。</li>\n  <li><i>核糖核酸酶H依赖性聚合酶链反应</i>(rhPCR):PCR的一种改进，利用带有3’延伸区的引物，该延伸区可以被耐热核糖核酸酶HII酶去除。该系统减少了引物二聚体，并允许用更高数量的引物进行多重反应。<sup><a href=\"#quote_50\" class=\"kx_ref\">[50]</a></sup></li>\n  <li><i>单一特异性引物聚合酶链反应</i>(SSP-PCR):即使序列信息仅在一端可用，也允许双链DNA的扩增。这种方法允许扩增只有部分序列信息可用的基因，并允许单向基因组从染色体的已知区域步行到未知区域。<sup><a href=\"#quote_51\" class=\"kx_ref\">[51]</a></sup></li>\n  <li><i>固相聚合酶链反应</i>:包含多种含义，包括 Polony Amplification (例如，聚合酶链反应菌落来源于凝胶基质)，桥聚合酶链反应<sup><a href=\"#quote_52\" class=\"kx_ref\">[52]</a></sup>(引物共价连接到固相支持物表面)、常规固相聚合酶链反应(其中不对称聚合酶链反应在带有固相支持物的引物存在下应用，所述引物的序列与其中一个水性引物匹配)和增强型固相聚合酶链反应<sup><a href=\"#quote_53\" class=\"kx_ref\">[53]</a></sup>(其中传统的固相聚合酶链反应可以通过使用高Tm和嵌套的固相支持物引物并任选应用热“步骤”来促进固相支持物引发来改进)。</li>\n  <li><i>自杀聚合酶链反应</i>:通常用于古遗传学或其他研究中，其中避免假阳性并确保扩增片段的特异性是最高优先级。最初是在一项研究中描述的，该研究旨在验证从14世纪的坟墓中获得的牙科样本中是否存在微生物鼠疫耶尔森菌，据说这些坟墓是在中世纪的黑死病流行期间被鼠疫杀死的。<sup><a href=\"#quote_54\" class=\"kx_ref\">[54]</a></sup>该方法规定在聚合酶链反应中只使用一次任何引物组合(因此称为“自杀”)，这种组合不应用于任何阳性对照聚合酶链反应，引物应始终针对以前在实验室中从未使用该引物或任何其他引物组扩增的基因组区域。这确保了实验室中不存在来自先前PCR反应的污染DNA，否则这些DNA可能会产生假阳性。</li>\n </ul></p><p>\n <ul>\n  <li><i>热不对称交错聚合酶链反应</i>:用于分离已知序列侧翼的未知序列。在已知序列中，TAIL-PCR使用一对具有不同退火温度的嵌套引物；简并引物用于从未知序列向另一个方向扩增。<sup><a href=\"#quote_55\" class=\"kx_ref\">[55]</a></sup></li>\n  <li><i>触地聚合酶链反应</i>（<i>逐步下降聚合酶链反应</i>):PCR的一种变体，旨在通过随着PCR循环的进行逐渐降低退火温度来减少非特异性背景。初始循环的退火温度通常为几度(3-5&nbsp;c)高于T<sub>m</sub>在使用的引物中，在后期的循环中，它是几度(3-5&nbsp;c)低于底漆T<sub>m</sub>。较高的温度赋予引物结合更大的特异性，较低的温度允许从初始循环中形成的特定产物中进行更有效的扩增。<sup><a href=\"#quote_56\" class=\"kx_ref\">[56]</a></sup></li>\n  <li><i>万能快走</i>:对于基因组行走和遗传指纹识别，使用比传统的“单侧”方法更特异性的“双侧”PCR(仅使用一个基因特异性引物和一个通用引物——这可能导致动脉事实上的“噪音”)<sup><a href=\"#quote_57\" class=\"kx_ref\">[57]</a></sup>这是由于套索结构形成的机制。UFW的流线型衍生物是LaNe RAGE(用于快速扩增基因组DNA末端的套索依赖性巢式PCR)，<sup><a href=\"#quote_58\" class=\"kx_ref\">[58]</a></sup>' 5'RACE LaNe<sup><a href=\"#quote_59\" class=\"kx_ref\">[59]</a></sup>和3'RACE LaNe。<sup><a href=\"#quote_60\" class=\"kx_ref\">[60]</a></sup></li>\n </ul></p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14996399733080593","title":"历史","versionId":"54568173541843218","lemmaId":11182,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82815118,"name":"iscience","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599821165,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p><img alt=\"\" class=\"fileimage kx_img ed_imgfloat_right currentPic\" img_height=\"156\" img_width=\"300\" titlename=\"示例引物对的图解表示。在体外分析中使用引物以允许DNA合成是允许PCR发展的一项重大创新。\" data-src=\"https://img04.sogoucdn.com/app/a/200698/sogou_science_17121\"> </p><p>1971年的一篇论文<i>《分子生物学杂志》</i>作者:Kjell Kleppe&nbsp;(挪威语)高宾德·科兰纳实验室的同事首先描述了一种<i>在试管内</i>使用酶分析法用引物复制短DNA模板的方法。<sup><a href=\"#quote_61\" class=\"kx_ref\">[61]</a></sup>然而，这种基本聚合酶链反应原理的早期表现在当时并没有受到太多关注，聚合酶链式反应在1983年的发明通常归功于凯利·穆利斯。<sup><a href=\"#quote_62\" class=\"kx_ref\">[62]</a></sup> </p><p><img alt=\"\" class=\"fileimage kx_img ed_imgfloat_right\" img_height=\"193\" img_width=\"300\" titlename=\"“浅蓝色”，1986年用于聚合酶链反应的样机\" data-src=\"https://img01.sogoucdn.com/app/a/200698/sogou_science_17123\"> </p><p>当穆利斯在1983年开发聚合酶链反应时，他在加利福尼亚州的 Emeryville 为塞特斯公司工作，该公司是首批生物技术公司之一，他负责合成DNA短链。穆利斯写道，有一天晚上，他开车沿着太平洋海岸高速公路巡航时，第一次想到了聚合酶链反应的想法。<sup><a href=\"#quote_63\" class=\"kx_ref\">[63]</a></sup>当他意识到他已经发明了一种通过由DNA聚合酶驱动的重复复制周期来扩增任何DNA区域的方法时，他正在用一种新的分析DNA变化(突变)方法。在<i>科学美国人</i>穆利斯总结道:“从遗传物质DNA的一个分子开始，聚合酶链反应可以在一个下午产生1000亿个类似的分子。这个反应很容易执行。它只需要一个试管、一些简单的试剂和一个热源。”<sup><a href=\"#quote_64\" class=\"kx_ref\">[64]</a></sup>1988年，DNA指纹首次用于亲子鉴定。<sup><a href=\"#quote_65\" class=\"kx_ref\">[65]</a></sup> </p><p>穆利斯因其发明于1993年被授予诺贝尔化学奖，七年前他和他在塞特斯的同事首次将他的建议付诸实践。<sup><a href=\"#quote_66\" class=\"kx_ref\">[66]</a></sup>mullis 1985年与R. K .斋祀和H. A. Erlich合著的论文《β-珠蛋白基因组序列的酶促扩增和用于镰刀形红细胞贫血症诊断的限制性位点分析》——聚合酶链式反应发明(PCR)——荣获2017年美国化学学会历史分期化学奖化学突破奖。<sup><a href=\"#quote_66\" class=\"kx_ref\">[66]</a></sup><sup><a href=\"#quote_67\" class=\"kx_ref\">[67]</a></sup> </p><p>关于其他科学家对穆利斯工作的智力和实践贡献，以及他是否是PCR原理的唯一发明者，仍然存在一些争议(见下文)。</p><p>聚合酶链反应方法的核心是使用合适的能够耐受高温的脱氧核糖核酸聚合酶90&nbsp;°C (194&nbsp;°F)在每一个复制循环后，在 DNA双螺旋中分离两条DNA链所必需的。最初用于体外实验的DNA聚合酶预示着PCR不能耐受这些高温。<sup><a href=\"#quote_68\" class=\"kx_ref\">[68]</a></sup>因此，早期的DNA复制过程非常低效和耗时，并且需要大量的DNA聚合酶和在整个过程中的连续处理。</p><p>1976年发现了 Taq聚合酶——一种从嗜热细菌中纯化的DNA聚合酶，<i>水生栖热菌</i>，它自然地生活在诸如<sup><a href=\"#quote_68\" class=\"kx_ref\">[68]</a></sup>温泉之类的高温(50至80°C(122至176°F))环境中，为PCR方法的显著改进铺平了道路。从<i>T.aquaticus</i>分离出的DNA聚合酶在高温下稳定，即使在DNA变性之后仍保持活性，<sup><a href=\"#quote_68\" class=\"kx_ref\">[68]</a></sup>因此不需要在每个循环后添加新的DNA聚合酶。<sup><a href=\"#quote_68\" class=\"kx_ref\">[68]</a></sup>这允许基于自动热循环仪的DNA扩增过程。</p><p></p><h3>专利纠纷</h3><p></p><p>聚合酶链反应技术由凯利·穆利斯获得专利，并转让给塞特斯公司，穆利斯1983年发明这项技术时就在该公司工作。<i>Taq</i>聚合酶也被专利所保护。已经有几起与该技术相关的高调诉讼，包括杜邦提起的一起不成功的诉讼。制药公司罗氏在1992年购买了这些专利的权利，目前拥有那些仍然受到保护的权利。</p><p>罗氏和Promega 之间围绕Taq聚合酶酶的相关专利战仍在世界各地的几个管辖区进行。法律争论已经超出了2005年3月28日到期的原始PCR和Taq聚合酶专利的寿命。<sup><a href=\"#quote_68\" class=\"kx_ref\">[68]</a></sup> </p>","pics":[{"originalUrl":"https://web.archive.org/web/20221028220934/https://img04.sogoucdn.com/app/a/200698/sogou_science_17121?w=300&h=156&titlename=%E7%A4%BA%E4%BE%8B%E5%BC%95%E7%89%A9%E5%AF%B9%E7%9A%84%E5%9B%BE%E8%A7%A3%E8%A1%A8%E7%A4%BA%E3%80%82%E5%9C%A8%E4%BD%93%E5%A4%96%E5%88%86%E6%9E%90%E4%B8%AD%E4%BD%BF%E7%94%A8%E5%BC%95%E7%89%A9%E4%BB%A5%E5%85%81%E8%AE%B8DNA%E5%90%88%E6%88%90%E6%98%AF%E5%85%81%E8%AE%B8PCR%E5%8F%91%E5%B1%95%E7%9A%84%E4%B8%80%E9%A1%B9%E9%87%8D%E5%A4%A7%E5%88%9B%E6%96%B0%E3%80%82","url":"https://web.archive.org/web/20221028220934/https://img04.sogoucdn.com/app/a/200698/sogou_science_17121","rw":300,"rh":156,"title":"示例引物对的图解表示。在体外分析中使用引物以允许DNA合成是允许PCR发展的一项重大创新。","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221028220934/https://img01.sogoucdn.com/app/a/200698/sogou_science_17123?w=300&h=193&titlename=%E2%80%9C%E6%B5%85%E8%93%9D%E8%89%B2%E2%80%9D%EF%BC%8C1986%E5%B9%B4%E7%94%A8%E4%BA%8E%E8%81%9A%E5%90%88%E9%85%B6%E9%93%BE%E5%8F%8D%E5%BA%94%E7%9A%84%E6%A0%B7%E6%9C%BA","url":"https://web.archive.org/web/20221028220934/https://img01.sogoucdn.com/app/a/200698/sogou_science_17123","rw":300,"rh":193,"title":"“浅蓝色”，1986年用于聚合酶链反应的样机","alt":null,"width":0,"height":0}],"card":null,"references":[],"versionCount":0}],"references":[{"id":1,"type":"book","title":"Cheng, S.; Fockler, C.; Barnes, W. M.; Higuchi, R. (1994). \"Effective Amplification of Long Targets from Cloned Inserts and Human Genomic DNA\". Proceedings of the National Academy of Sciences. 91 (12): 5695–5699. Bibcode:1994PNAS...91.5695C. doi:10.1073/pnas.91.12.5695. PMC 44063. PMID 8202550.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":2,"type":"book","title":"Carr AC, Moore SD (2012). Lucia A, ed. \"Robust quantification of polymerase chain reactions using global fitting\". PLoS ONE. 7 (5): e37640. Bibcode:2012PLoSO...737640C. doi:10.1371/journal.pone.0037640. PMC 3365123. PMID 22701526.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":3,"type":"book","title":"\"Polymerase Chain Reaction (PCR)\". National Center for Biotechnology Information, U.S. National Library of Medicine.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":4,"type":"book","title":"\"PCR\". Genetic Science Learning Center, University of Utah.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":5,"type":"book","title":"Pavlov, A. R.; Pavlova, N. V.; Kozyavkin, S. A.; Slesarev, A. I. (2004). \"Recent developments in the optimization of thermostable DNA polymerases for efficient applications☆\". Trends in Biotechnology. 22 (5): 253–260. doi:10.1016/j.tibtech.2004.02.011. PMID 15109812.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":6,"type":"book","title":"Rychlik W, Spencer WJ, Rhoads RE (1990). \"Optimization of the annealing temperature for DNA amplification in vitro\". Nucleic Acids Res. 18 (21): 6409–6412. doi:10.1093/nar/18.21.6409. PMC 332522. PMID 2243783.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":7,"type":"book","title":"Borman, Jon; Schuster, David; Li, Wu-bo; Jessee, Joel; Rashtchian, Ayoub (2000). \"PCR from problematic templates\" (PDF). Focus. 22 (1): 10.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":8,"type":"book","title":"Bogetto, Prachi and Waidne, Lisa (2000). \"Helpful tips for PCR\" (PDF). Focus. 22 (1): 12.CS1 maint: Multiple names: authors list (link)","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":9,"type":"book","title":"Joseph Sambrook & David W. Russel (2001). Molecular Cloning: A Laboratory Manual (3rd ed.). Cold Spring Harbor, N.Y.: Cold Spring Harbor Laboratory Press. ISBN 978-0-879-69576-7.第八章:聚合酶链反应体外扩增脱氧核糖核酸","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":10,"type":"book","title":"\"Electronic PCR\". NCBI – National Center for Biotechnology Information. Retrieved 13 March 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":11,"type":"book","title":"Pavlov AR, Pavlova NV, Kozyavkin SA, Slesarev AI (2006). \"Thermostable DNA Polymerases for a Wide Spectrum of Applications: Comparison of a Robust Hybrid TopoTaq to other enzymes\". In Kieleczawa J. DNA Sequencing II: Optimizing Preparation and Cleanup. Jones and Bartlett. pp. 241–257. ISBN 978-0-7637-3383-4.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":12,"type":"book","title":"Pombert JF, Sistek V, Boissinot M, Frenette M (2009). \"Evolutionary relationships among salivarius streptococci as inferred from multilocus phylogenies based on 16S rRNA-encoding, recA, secA, and secY gene sequences\". BMC Microbiol. 9: 232. doi:10.1186/1471-2180-9-232. PMC 2777182. PMID 19878555.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":13,"type":"book","title":"\"Chemical Synthesis, Sequencing, and Amplification of DNA (class notes on MBB/BIO 343)\". Arizona State University. Archived from the original on 9 October 1997. Retrieved 29 October 2007.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":14,"type":"book","title":"Bustin, S. A.; Benes, V.; Garson, J. A.; Hellemans, J.; Huggett, J.; Kubista, M.; Mueller, R.; Nolan, T.; Pfaffl, M. W.; Shipley, G. L.; Vandesompele, J.; Wittwer, C. T. (2009). \"The MIQE Guidelines: Minimum Information for Publication of Quantitative Real-Time PCR Experiments\". Clinical Chemistry. 55 (4): 611–622. doi:10.1373/clinchem.2008.112797. PMID 19246619.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":15,"type":"book","title":"Garibyan, Avashia (March 2013). \"Polymerase Chain Reaction\". Journal of Investigative Dermatology. 133 (3): 1–4. doi:10.1038/jid.2013.1. PMC 4102308. PMID 23399825.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":16,"type":"book","title":"Ninfa, Alexander; Ballou, David; Benore, Marilee (2009). Fundamental Laboratory Approaches for Biochemistry and Biotechnology. United States: Wiley. pp. 408–410. ISBN 978-0470087664.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":17,"type":"book","title":"Tomar, Rukam (2010). Molecular Markers and Plant Biotechnology. Pitman Pura, New Delhi: New India Publishing Agency. p. 188. ISBN 978-93-80235-25-7.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":18,"type":"book","title":"Cai, H; Caswell JL; Prescott JF (March 2014). \"Nonculture Molecular Techniques for Diagnosis of Bacterial Disease in Animals: A Diagnostic Laboratory Perspective\". Veterinary Pathology. 51 (2): 341–350. doi:10.1177/0300985813511132. PMID 24569613.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":19,"type":"book","title":"Salis AD (2009). \"Applications in Clinical Microbiology\". Real-Time PCR: Current Technology and Applications. Caister Academic Press. ISBN 978-1-904455-39-4.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":20,"type":"book","title":"Kwok, S.; Mack, D. H.; Mullis, K. B.; Poiesz, B.; Ehrlich, G.; Blair, D.; Friedman-Kien, A.; Sninsky, J. J. (1987). \"Identification of human immunodeficiency virus sequences by using in vitro enzymatic amplification and oligomer cleavage detection\". Journal of Virology. 61 (5): 1690–4. PMC 254157. PMID 2437321.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":21,"type":"book","title":"Finger, Horst; von Koenig, Carl Heinz Wirsing (1996). Baron, Samuel, ed. Medical Microbiology (4th ed.). Galveston (TX): University of Texas Medical Branch at Galveston. ISBN 978-0963117212. PMID 21413270.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":22,"type":"book","title":"Yeh, Sylvia H.; Mink, ChrisAnna M. (2012). \"Netter's Infectious Diseases\". Netter's Infectious Diseases: 11–14. doi:10.1016/B978-1-4377-0126-5.00003-3.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":23,"type":"book","title":"Alonso, A (2004-01-28). \"Real-time PCR designs to estimate nuclear and mitochondrial DNA copy number in forensic and ancient DNA studies\". Forensic Science International. 139 (2–3): 141–149. doi:10.1016/j.forsciint.2003.10.008. PMID 15040907.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":24,"type":"book","title":"Boehnke, M.; Arnheim, N.; Li, H.; Collins, F. S. (1989). \"Fine-structure genetic mapping of human chromosomes using the polymerase chain reaction on single sperm: Experimental design considerations\". American Journal of Human Genetics. 45 (1): 21–32. PMC 1683385. PMID 2568090.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":25,"type":"book","title":"Garibyan, Lilit; Avashia, Nidhi (2013-03-01). \"Polymerase Chain Reaction\". Journal of Investigative Dermatology. 133 (3): 1–4. doi:10.1038/jid.2013.1. PMC 4102308. PMID 23399825.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":26,"type":"book","title":"Garibyan L, Avashia N (2013). \"Polymerase Chain Reaction\". Journal of Investigative Dermatology. 133 (3): 1–4. doi:10.1038/jid.2013.1. PMC 4102308. PMID 23399825.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":27,"type":"book","title":"Zhou, Y H; Zhang, X P; Ebright, R H (1991-11-11). \"Random mutagenesis of gene-sized DNA molecules by use of PCR with Taq DNA polymerase\". Nucleic Acids Research. 19 (21): 6052. doi:10.1093/nar/19.21.6052. PMC 329070. PMID 1658751.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":28,"type":"book","title":"Schochetman, Gerald; Ou, Chin-Yih; Jones, Wanda K. (1988). \"Polymerase Chain Reaction\". The Journal of Infectious Diseases. 158 (6): 1154–1157. doi:10.1093/infdis/158.6.1154. JSTOR 30137034.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":29,"type":"book","title":"Stemmer WP, Crameri A, Ha KD, Brennan TM, Heyneker HL (1995). \"Single-step assembly of a gene and entire plasmid from large numbers of oligodeoxyribonucleotides\". Gene. 164 (1): 49–53. doi:10.1016/0378-1119(95)00511-4. PMID 7590320.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":30,"type":"book","title":"Innis MA, Myambo KB, Gelfand DH, Brow MA (1988). \"DNA sequencing with Thermus aquaticus DNA polymerase and direct sequencing of polymerase chain reaction-amplified DNA\". Proc Natl Acad Sci USA. 85 (24): 9436–9440. Bibcode:1988PNAS...85.9436I. doi:10.1073/pnas.85.24.9436. PMC 282767. PMID 3200828.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":31,"type":"book","title":"Pierce KE & Wangh LJ (2007). Linear-after-the-exponential polymerase chain reaction and allied technologies Real-time detection strategies for rapid, reliable diagnosis from single cells. Methods in Molecular Medicine. 132. pp. 65–85. doi:10.1007/978-1-59745-298-4_7. ISBN 978-1-58829-578-1. PMID 17876077.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":32,"type":"book","title":"Krishnan, Madhavi; Ugaz, Victor; Burns, Mark (2002). \"PCR in a Rayleigh-Benard convection cell\". Science. 298 (5594): 793. doi:10.1126/science.298.5594.793. PMID 12399582.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":33,"type":"book","title":"Priye, Aashish; Hassan, Yassin; Ugaz, Victor (2013). \"Microscale chaotic advection enables robust convective DNA replication\". Analytical Chemistry. 85 (21): 10536–10541. doi:10.1021/ac402611s. PMID 24083802.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":34,"type":"book","title":"Schwartz JJ, Lee C, Shendure J (2012). \"Accurate gene synthesis with tag-directed retrieval of sequence-verified DNA molecules\". Nature Methods. 9 (9): 913–915. doi:10.1038/nmeth.2137. PMC 3433648. PMID 22886093.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":35,"type":"book","title":"Vincent M, Xu Y, Kong H (2004). \"Helicase-dependent isothermal DNA amplification\". EMBO Reports. 5 (8): 795–800. doi:10.1038/sj.embor.7400200. PMC 1249482. PMID 15247927.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":36,"type":"book","title":"Chou Q, Russell M, Birch DE, Raymond J, Bloch W (1992). \"Prevention of pre-PCR mis-priming and primer dimerization improves low-copy-number amplifications\". Nucleic Acids Research. 20 (7): 1717–1723. doi:10.1093/nar/20.7.1717. PMC 312262. PMID 1579465.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":37,"type":"book","title":"Sharkey, D. J.; Scalice, E. R.; Christy, K. G.; Atwood, S. M.; Daiss, J. L. (1994). \"Antibodies as Thermolabile Switches: High Temperature Triggering for the Polymerase Chain Reaction\". Bio/Technology. 12 (5): 506–509. doi:10.1038/nbt0594-506.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":38,"type":"book","title":"San Millan RM, Martinez-Ballesteros I, Rementeria A, Garaizar J, Bikandi J (2013). \"Online exercise for the design and simulation of PCR and PCR-RFLP experiments\". BMC Research Notes. 6: 513. doi:10.1186/1756-0500-6-513. PMC 4029544. PMID 24314313.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":39,"type":"book","title":"Zietkiewicz, E.; Rafalski, A.; Labuda, D. (1994). \"Genome fingerprinting by simple sequence repeat (SSR)-anchored polymerase chain reaction amplification\". Genomics. 20 (2): 176–83. doi:10.1006/geno.1994.1151. PMID 8020964.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":40,"type":"book","title":"Ochman H, Gerber AS, Hartl DL (1988). \"Genetic Applications of an Inverse Polymerase Chain Reaction\". Genetics. 120 (3): 621–623. PMC 1203539. PMID 2852134.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":41,"type":"book","title":"Mueller PR, Wold B (1988). \"In vivo footprinting of a muscle specific enhancer by ligation mediated PCR\". Science. 246 (4931): 780–786. doi:10.1126/science.2814500. PMID 2814500.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":42,"type":"book","title":"Herman JG, Graff JR, Myöhänen S, Nelkin BD, Baylin SB (1996). \"Methylation-specific PCR: a novel PCR assay for methylation status of CpG islands\". Proc Natl Acad Sci USA. 93 (13): 9821–9826. Bibcode:1996PNAS...93.9821H. doi:10.1073/pnas.93.18.9821. PMC 38513. PMID 8790415.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":43,"type":"book","title":"Isenbarger TA, Finney M, Ríos-Velázquez C, Handelsman J, Ruvkun G (2008). \"Miniprimer PCR, a New Lens for Viewing the Microbial World\". Applied and Environmental Microbiology. 74 (3): 840–9. doi:10.1128/AEM.01933-07. PMC 2227730. PMID 18083877.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":44,"type":"book","title":"Cenchao Shen; Wenjuan Yang; Qiaoli Ji; Hisaji Maki; Anjie Dong; Zhizhou Zhang (2009). \"NanoPCR observation: different levels of DNA replication fidelity in nanoparticle-enhanced polymerase chain reactions\". Nanotechnology. 20 (45): 455103. Bibcode:2009Nanot..20S5103S. doi:10.1088/0957-4484/20/45/455103. PMID 19822925.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":45,"type":"book","title":"Shen, Cenchao (2013). \"An Overview of Nanoparticle-Assisted Polymerase Chain Reaction Technology\". An Overview of Nanoparticle‐Assisted Polymerase Chain Reaction Technology. US: Wiley-Blackwell Publishing Ltd. pp. 97–106. doi:10.1002/9781118451915.ch5. ISBN 9781118451915.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":46,"type":"book","title":"Horton RM, Hunt HD, Ho SN, Pullen JK, Pease LR (1989). \"Engineering hybrid genes without the use of restriction enzymes: gene splicing by overlap exten-sion\". Gene. 77 (1): 61–68. doi:10.1016/0378-1119(89)90359-4. PMID 2744488.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":47,"type":"book","title":"Moller, Simon (2006). PCR: The Basics. US: Taylor & Francis Group. p. 144. ISBN 9780415355476.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":48,"type":"book","title":"David F, Turlotte E (1998). \"Une méthode d'amplification génique isotherme\" [An Isothermal Amplification Method]. Comptes Rendus de l'Académie des Sciences, Série III. 321 (11): 909–914. Bibcode:1998CRASG.321..909D. doi:10.1016/S0764-4469(99)80005-5.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":49,"type":"book","title":"Fabrice David (September–October 2002). \"Utiliser les propriétés topologiques de l'ADN: une nouvelle arme contre les agents pathogènes\" (PDF). Fusion. Archived from the original (PDF) on 2007-11-28.(法语)","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":50,"type":"book","title":"Dobosy JR, Rose SD, Beltz KR, Rupp SM, Powers KM, Behlke MA, Walder JA (August 2011). \"RNase H-dependent PCR (rhPCR): improved specificity and single nucleotide polymorphism detection using blocked cleavable primers\". BMC Biotechnology. 11: 80. doi:10.1186/1472-6750-11-80. PMC 3224242. PMID 21831278.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":51,"type":"book","title":"Shyamala, V.; Ferro-Luzzi, Ames G. (1993). Single Specific Primer-Polymerase Chain Reaction (SSP-PCR) and Genome Walking. Methods in Molecular Biology. 15. pp. 339–48. doi:10.1385/0-89603-244-2:339. ISBN 978-0-89603-244-6. PMID 21400290.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":52,"type":"book","title":"Bing DH, Boles C, Rehman FN, Audeh M, Belmarsh M, Kelley B, Adams CP (1996). \"Bridge amplification: a solid phase PCR system for the amplification and detection of allelic differences in single copy genes\". Genetic Identity Conference Proceedings, Seventh International Symposium on Human Identification. Archived from the original on 7 May 2001.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":53,"type":"book","title":"Khan Z, Poetter K, Park DJ (2008). \"Enhanced solid phase PCR: mechanisms to increase priming by solid support primers\". Analytical Biochemistry. 375 (2): 391–393. doi:10.1016/j.ab.2008.01.021. PMID 18267099.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":54,"type":"book","title":"Raoult, D; G Aboudharam; E Crubezy; G Larrouy; B Ludes; M Drancourt (2000-11-07). \"Molecular identification by \"suicide PCR\" of Yersinia pestis as the agent of medieval black death\". Proc. Natl. Acad. Sci. U.S.A. 97 (23): 12800–12803. Bibcode:2000PNAS...9712800R. doi:10.1073/pnas.220225197. PMC 18844. PMID 11058154.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":55,"type":"book","title":"Y.G. Liu & R. F. Whittier (1995). \"Thermal asymmetric interlaced PCR: automatable amplification and sequencing of insert end fragments from P1 and YAC clones for chromosome walking\". Genomics. 25 (3): 674–81. doi:10.1016/0888-7543(95)80010-J. PMID 7759102.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":56,"type":"book","title":"Don RH, Cox PT, Wainwright BJ, Baker K, Mattick JS (1991). \"'Touchdown' PCR to circumvent spurious priming during gene amplification\". Nucleic Acids Res. 19 (14): 4008. doi:10.1093/nar/19.14.4008. PMC 328507. PMID 1861999.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":57,"type":"book","title":"Myrick KV, Gelbart WM (2002). \"Universal Fast Walking for direct and versatile determination of flanking sequence\". Gene. 284 (1–2): 125–131. doi:10.1016/S0378-1119(02)00384-0. PMID 11891053.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":58,"type":"book","title":"\"Full Text – LaNe RAGE: a new tool for genomic DNA flanking sequence determination\".","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":59,"type":"book","title":"Park DJ (2005). \"A new 5' terminal murine GAPDH exon identified using 5'RACE LaNe\". Molecular Biotechnology. 29 (1): 39–46. doi:10.1385/MB:29:1:39. PMID 15668518.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":60,"type":"book","title":"Park DJ (2004). \"3'RACE LaNe: a simple and rapid fully nested PCR method to determine 3'-terminal cDNA sequence\". BioTechniques. 36 (4): 586–588, 590. doi:10.2144/04364BM04. PMID 15088375.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":61,"type":"book","title":"Kleppe K, Ohtsuka E, Kleppe R, Molineux I, Khorana HG (1971). \"Studies on polynucleotides. XCVI. Repair replications of short synthetic DNA's as catalyzed by DNA polymerases\". J. Mol. Biol. 56 (2): 341–361. doi:10.1016/0022-2836(71)90469-4. PMID 4927950.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":62,"type":"book","title":"Rabinow, Paul (1996). Making PCR: A Story of Biotechnology. Chicago: University of Chicago Press. ISBN 978-0-226-70146-2.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":63,"type":"book","title":"Mullis, Kary (1998). Dancing Naked in the Mind Field. New York: Pantheon Books. ISBN 978-0-679-44255-4.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":64,"type":"book","title":"Mullis, Kary (1990). \"The unusual origin of the polymerase chain reaction\". Scientific American. 262 (4): 56–61, 64–5. Bibcode:1990SciAm.262d..56M. doi:10.1038/scientificamerican0490-56. PMID 2315679.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":65,"type":"book","title":"Patidar, Madhvika; Agrawal, Suraksha; Parveen, Farah; Khare, Parul (2015). \"Molecular insights of saliva in solving paternity dispute\". Journal of Forensic Dental Sciences. 7 (1): 76–79. doi:10.4103/0975-1475.150325. PMC 4330625. PMID 25709326.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":66,"type":"book","title":"\"Kary B. Mullis – Nobel Lecture: The Polymerase Chain Reaction\".","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":67,"type":"book","title":"Saiki, R.; Scharf, S; Faloona, F; Mullis, K.; Horn, G.; Erlich, H.; Arnheim, N (1985). \"Enzymatic amplification of beta-globin genomic sequences and restriction site analysis for diagnosis of sickle cell anemia\". Science. 230 (4732): 1350–1354. doi:10.1126/science.2999980.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":68,"type":"book","title":"Saiki, R.; Scharf, S.; Faloona, F.; Mullis, K.; Horn, G.; Erlich, H.; Arnheim, N. (1985). \"Enzymatic amplification of beta-globin genomic sequences and restriction site analysis for diagnosis of sickle cell anemia\". Science. 230 (4732): 1350–1354. doi:10.1126/science.2999980. PMID 2999980.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false}],"recommendReferences":null,"auditState":2,"lemmaLevel":1,"origin":0,"originEnTitle":null,"originZhTitle":null,"pv":1065,"auditType":0,"synonyms":null,"showEditTime":"2019.12.13 18:55","auditors":[{"uid":0,"name":"Ki.κe","pic":"https://web.archive.org/web/20221028220934/https://wx.qlogo.cn/mmopen/vi_32/y67kfr32Doib4wg71Jiau7jVWvharic3nRKgdRRQSl6koeQJCo0GQs2Krw0vwdFRsOWnHIQOwAZsSg5lIkIrFCOcQ/132","introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":"","jobBrief":"","role":0,"roleName":null,"title":"","professionalTitle":null,"phoneNo":null,"editable":true,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false}],"hasZhishiNav":false,"auditInfos":{},"isHistory":false};</script><script crossorigin="anonymous" src="./1182.聚合酶链反应 - 搜狗科学百科_files/aegis.min.js.download"></script><script crossorigin="anonymous" src="./1182.聚合酶链反应 - 搜狗科学百科_files/main_2020092401.js.download"></script><script crossorigin="anonymous" src="./1182.聚合酶链反应 - 搜狗科学百科_files/react.production.min.js.download"></script><script crossorigin="anonymous" src="./1182.聚合酶链反应 - 搜狗科学百科_files/react-dom.production.min.js.download"></script><script crossorigin="anonymous" src="./1182.聚合酶链反应 - 搜狗科学百科_files/jquery-1.11.1.min.js.download"></script><script crossorigin="anonymous" src="./1182.聚合酶链反应 - 搜狗科学百科_files/main_2022062701.js.download"></script><script crossorigin="anonymous" src="./1182.聚合酶链反应 - 搜狗科学百科_files/main_66bbe21.js.download"></script><script crossorigin="anonymous" src="./1182.聚合酶链反应 - 搜狗科学百科_files/react.production.min.js.download"></script><script crossorigin="anonymous" src="./1182.聚合酶链反应 - 搜狗科学百科_files/react-dom.production.min.js.download"></script><script crossorigin="anonymous" src="./1182.聚合酶链反应 - 搜狗科学百科_files/main_edf0f08.js.download"></script>
</body></html>