<!DOCTYPE html>
<!-- saved from url=(0083)https://web.archive.org/web/20221028222850/https://baike.sogou.com/kexue/d10191.htm -->
<html class="" data-reactroot=""><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><script src="./191.肽核酸 - 搜狗科学百科_files/analytics.js.download" type="text/javascript"></script>
<script type="text/javascript">window.addEventListener('DOMContentLoaded',function(){var v=archive_analytics.values;v.service='wb';v.server_name='wwwb-app213.us.archive.org';v.server_ms=3781;archive_analytics.send_pageview({});});</script>
<script type="text/javascript" src="./191.肽核酸 - 搜狗科学百科_files/bundle-playback.js.download" charset="utf-8"></script>
<script type="text/javascript" src="./191.肽核酸 - 搜狗科学百科_files/wombat.js.download" charset="utf-8"></script>
<script type="text/javascript">
  __wm.init("https://web.archive.org/web");
  __wm.wombat("https://baike.sogou.com/kexue/d10191.htm","20221028222850","https://web.archive.org/","web","/_static/",
	      "1666996130");
</script>
<link rel="stylesheet" type="text/css" href="./191.肽核酸 - 搜狗科学百科_files/banner-styles.css">
<link rel="stylesheet" type="text/css" href="./191.肽核酸 - 搜狗科学百科_files/iconochive.css">
<!-- End Wayback Rewrite JS Include -->
<meta name="save" content="history"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="baidu-site-verification" content="VWGb6TyYx8"><meta content="肽核酸 - 搜狗科学百科" name="keywords"><meta content="搜狗科学百科是一部有着平等、协作、分享、自由理念的网络科学全书，为每一个互联网用户创造一个涵盖所有领域知识、服务的中文知识性平台。" name="description"><meta http-equiv="x-dns-prefetch-control" content="on"><meta name="server" baike="235" ip="210" env="online"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028222850/https://cache.soso.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028222850/https://hhy.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028222850/https://pic.baike.soso.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028222850/https://ugc.qpic.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028222850/https://xui.ptlogin2.qq.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028222850/https://q1.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028222850/https://q2.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028222850/https://q3.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028222850/https://q4.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028222850/https://q.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028222850/https://img01.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028222850/https://img02.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028222850/https://img03.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028222850/https://img04.sogoucdn.com/"><link rel="Shortcut Icon" href="https://web.archive.org/web/20221028222850im_/https://www.sogou.com/images/logo/new/favicon.ico?v=4"><link rel="Bookmark" href="https://www.sogou.com/images/logo/new/favicon.ico?v=4"><link href="./191.肽核酸 - 搜狗科学百科_files/base_b849887.css" rel="stylesheet"><link href="./191.肽核酸 - 搜狗科学百科_files/detail_378aed5.css" rel="stylesheet"><link href="./191.肽核酸 - 搜狗科学百科_files/inviteAudit_7894507.css" rel="stylesheet"><link rel="stylesheet" href="./191.肽核酸 - 搜狗科学百科_files/highlight.min.css"><title>肽核酸 - 搜狗科学百科</title><style>.onekey-close {
	position: absolute;
	top: 16px;
	right: 16px;
	width: 24px;
	height: 24px;
	background: url(//web.archive.org/web/20221028205211/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	text-indent: -999em;
	background-size: 84px;
	background-position: -63px 0;
}

.onekey-login {
	position: absolute;
	top: 16.4%;
	left: 0;
	right: 0;
	width: 100%;
}

/* .onekey-login-img {
    width: 75px;
    height: 75px;
    background: url("https://web.archive.org/web/20221028222850/https://baike.sogou.com/kexue/images/sprite_wap_baike.png") no-repeat;
    background-size: 100px 91px;
    background-position: 0 0;
    background-repeat: no-repeat;
    margin: 0 auto;
} */

.onekey-login-title {
	text-align: center;
	padding-bottom: 3px;
	font-size: 21px;
	font-weight: bold;
	line-height: 30px;
	color: #000;
}

.onekey-login-txt {
	text-align: center;
	font-family: PingFangSC;
	font-size: 14px;
	line-height: 20px;
	color: #8f8f8f;
}

.onekey-login-qq,
.onekey-login-wx,
.onekey-login-phone {
	display: block;
	width: 245px;
	height: 54px;
	border-radius: 45px;
	text-align: center;

	margin: 0 auto;
	font-size: 17px;
	line-height: 24px;
	color: #000;
	/* padding: 16px 77px; */
	border-radius: 12px;
	border: solid 1px #e0e0e0;
}
.onekey-qq-content,
.onekey-vx-content,
.onekey-phone-content {
	display: inline-block;
	margin-top: 16px;
}
.onekey-qq-content {
	padding: 0 5px;
}

.onekey-login-qq {
	margin-top: 48px;
	margin-bottom: 24px;
}

.onekey-login-qq:before {
	display: inline-block;
	content: "";
	width: 20px;
	height: 20px;
	background: url(//web.archive.org/web/20221028205211/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	background-size: 80px;
	background-position: -20px 0;
	vertical-align: top;
	margin: 17px 8px 0 0;
}

.onekey-login-wx {
	margin-bottom: 24px;
}

.onekey-login-wx:before {
	display: inline-block;
	content: "";
	width: 21px;
	height: 21px;
	background: url(//web.archive.org/web/20221028205211/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	background-size: 84px;
	background-position: 0 0;
	vertical-align: top;
	margin: 17px 10px 0 0;
}

.onekey-login-phone {
}

.onekey-login-phone:before {
	display: inline-block;
	content: "";
	width: 21px;
	height: 21px;
	background: url(//web.archive.org/web/20221028205211/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	background-size: 84px;
	background-position: -42px 0;
	vertical-align: top;
	margin: 17px 10px 0 0;
}

.onekey-fixed {
	z-index: 100;
	position: fixed;
	top: 0;
	bottom: 0;
	left: 0;
	right: 0;
	background: #fff;
	width: 100%;
	height: 100%;
}

.onekey-fixed.forbid {
	z-index: 100;
	position: fixed;
	top: auto;
	bottom: 68px;
	left: 9%;
	right: 0;
	background: rgba(0, 0, 0, 0.7);
	width: 82%;
	height: 43px;
	border-radius: 25px;
	color: #ffffff;
}
.onekey-login-title.forbid {
	text-align: center;
	padding-bottom: 3px;
	font-size: 14px;
	font-weight: normal;
	line-height: 30px;
	color: white;
}
</style><style>#login_mask {
  background: #000;
  opacity: 0.5;
  filter: alpha(opacity=50);
  position: fixed;
  /*fixed好像在哪个IE上有BUG，先用用*/
  left: 0;
  top: 0;
  z-index: 999;
  height: 100%;
}

#login_iframe_container {
  position: fixed;
  width: 550px;
  height: 360px;
  z-index: 1020;
  background-color: #ffffff;
}

@media screen and (max-width: 828px) {
  #login_iframe_container {
    top: 50% !important;
    left: 50% !important;
    transform: translate(-50%, -50%);
  }
}

#login_iframe_container.new-login {
  width: 550px;
  height: 360px;
  background-image: url(//web.archive.org/web/20221028205211/https://hhy.sogoucdn.com/js/common/hhy/background_2a4a8a6.png);
}

#login_iframe_container.new-login.no-bg {
  background: #fff;
}

#login_iframe_container.new-login .login-title {
  width: 100%;
  height: 42px;
  line-height: 42px;
  text-align: center;
  font-size: 30px;
  letter-spacing: 0.19px;
  color: #ffffff;
  margin-top: 62px;
}
#login_iframe_container.new-login .forbid-title {
  width: 100%;
  height: 42px;
  line-height: 42px;
  text-align: center;
  font-size: 24px;
  letter-spacing: 0.19px;
  color: #333333;
  margin-top: 150px;
}

#login_iframe_container.new-login.no-bg .login-title {
  color: #333333;
}

#login_iframe_container.new-login .login-subtitle {
  width: 100%;
  height: 18px;
  line-height: 18px;
  font-size: 13px;
  letter-spacing: 0.08px;
  color: #ffffff;
  text-align: center;
  margin-top: 9px;
  margin-bottom: 43px;
}

#login_iframe_container.new-login.no-bg .login-subtitle {
  color: #999999;
}

#login_iframe_container.new-login .login-subtitle::before {
  content: '';
  display: inline-block;
  width: 10px;
  height: 1px;
  background-color: #ffffff;
  position: relative;
  top: -4px;
  left: -5px;
}

#login_iframe_container.new-login .login-subtitle::after {
  content: '';
  display: inline-block;
  width: 10px;
  height: 1px;
  background-color: #ffffff;
  position: relative;
  top: -4px;
  left: 5px;
}

#login_iframe_container.new-login.no-bg .login-subtitle::before {
  background-color: #999999;
}

#login_iframe_container.new-login.no-bg .login-subtitle::after {
  background-color: #999999;
}

#login_iframe_container.new-login .close-btn {
  position: absolute;
  top: 20px;
  right: 20px;
  width: 12px;
  height: 12px;
  background: url(//web.archive.org/web/20221028205211/https://hhy.sogoucdn.com/js/common/hhy/login-sprites_e3853e5.png) -59px -10px;
  background-size: 81px 91px;
  cursor: pointer;
}

#login_iframe_container.new-login .login-btn {
  width: 220px;
  height: 47px;
  border-radius: 24px;
  border: solid 1px #dddddd;
  background-color: #ffffff;
  margin: 0 auto;
  margin-top: 28px;
  position: relative;
  display: block;
}

#login_iframe_container.new-login .login-btn .login-icon {
  position: absolute;
}

#login_iframe_container.new-login .login-btn .login-text {
  width: 61px;
  height: 47px;
  line-height: 47px;
  vertical-align: middle;
  font-size: 15px;
  letter-spacing: 0.1px;
  color: #666666;
  position: absolute;
  right: 62px;
}

#login_iframe_container.new-login .login-btn.qq-btn .login-icon {
  width: 22px;
  height: 27px;
  top: 10px;
  left: 67px;
  background: url(//web.archive.org/web/20221028205211/https://hhy.sogoucdn.com/js/common/hhy/login-sprites_e3853e5.png) -10px -54px;
  background-size: 81px 91px;
}

#login_iframe_container.new-login .login-btn.qq-btn .login-text {
  right: 59px;
}

#login_iframe_container.new-login .login-btn.wechat-btn .login-icon {
  width: 29px;
  height: 24px;
  top: 12px;
  left: 62px;
  background: url(//web.archive.org/web/20221028205211/https://hhy.sogoucdn.com/js/common/hhy/login-sprites_e3853e5.png) -10px -10px;
  background-size: 81px 91px;
}</style><style>/* -- container -- */
.rodal,
.rodal-mask {
    top: 0;
    left: 0;
    width: 100%;
    height: 100%;
    z-index: 100;
}

.rodal {
    position: fixed;
}

/* -- mask -- */
.rodal-mask {
    position: fixed;
    background: rgba(0, 0, 0, .5);
}

/* -- dialog -- */
.rodal-dialog {
    position: absolute;
    z-index: 101;
    background: #fff;
    border-radius: 3px;
    box-shadow: 0 1px 3px rgba(0, 0, 0, .2);
}

.rodal-center {
    top: 50%;
    transform: translateY(-50%);
    left: 0;
    right: 0;
    margin: 0 auto;
}

.rodal-bottom {
    left: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

.rodal-top {
    left: 0;
    right: 0;
    top: 0;
    margin: auto;
}

.rodal-left {
    top: 0;
    left: 0;
    bottom: 0;
    margin: auto;
}

.rodal-right {
    top: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

/* -- close button -- */
.rodal-close {
    position: absolute;
    cursor: pointer;
    top: 16px;
    right: 16px;
    width: 16px;
    height: 16px;
}

.rodal-close:before,
.rodal-close:after {
    position: absolute;
    content: '';
    height: 2px;
    width: 100%;
    top: 50%;
    left: 0;
    margin-top: -1px;
    background: #999;
    border-radius: 100%;
    -webkit-transition: background .2s;
    transition: background .2s;
}

.rodal-close:before {
    -webkit-transform: rotate(45deg);
    transform: rotate(45deg);
}

.rodal-close:after {
    -webkit-transform: rotate(-45deg);
    transform: rotate(-45deg);
}

.rodal-close:hover:before,
.rodal-close:hover:after {
    background: #333;
}

/* -- fade -- */
/* @-webkit-keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

@keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

.rodal-fade-enter {
    -webkit-animation: rodal-fade-enter both ease-in;
    animation: rodal-fade-enter both ease-in;
} */

@-webkit-keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

@keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

.rodal-fade-leave {
    -webkit-animation: rodal-fade-leave both ease-out;
    animation: rodal-fade-leave both ease-out;
}

/* -- zoom -- */
@-webkit-keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-enter {
    -webkit-animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-leave {
    -webkit-animation: rodal-zoom-leave both;
    animation: rodal-zoom-leave both;
}

/* -- slideDown -- */
@-webkit-keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-enter {
    -webkit-animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-leave {
    -webkit-animation: rodal-slideDown-leave both;
    animation: rodal-slideDown-leave both;
}

/* -- slideLeft -- */
@-webkit-keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-enter {
    -webkit-animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-leave {
    -webkit-animation: rodal-slideLeft-leave both;
    animation: rodal-slideLeft-leave both;
}

/* -- slideRight -- */
@-webkit-keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-enter {
    -webkit-animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-leave {
    -webkit-animation: rodal-slideRight-leave both;
    animation: rodal-slideRight-leave both;
}

/* -- slideUp -- */
@-webkit-keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-enter {
    -webkit-animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
    animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
}

@-webkit-keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-leave {
    -webkit-animation: rodal-slideUp-leave both;
    animation: rodal-slideUp-leave both;
}

/* -- flip -- */
@-webkit-keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

@keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

.rodal-flip-enter {
    -webkit-animation: rodal-flip-enter both ease-in;
    animation: rodal-flip-enter both ease-in;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

@-webkit-keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

@keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

.rodal-flip-leave {
    -webkit-animation: rodal-flip-leave both;
    animation: rodal-flip-leave both;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

/* -- rotate -- */
@-webkit-keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-enter {
    -webkit-animation: rodal-rotate-enter both;
    animation: rodal-rotate-enter both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

@-webkit-keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-leave {
    -webkit-animation: rodal-rotate-leave both;
    animation: rodal-rotate-leave both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

/* -- door -- */
@-webkit-keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

@keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

.rodal-door-enter {
    -webkit-animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

@keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

.rodal-door-leave {
    -webkit-animation: rodal-door-leave both;
    animation: rodal-door-leave both;
}</style><style>/* -- container -- */
.rodal,
.rodal-mask {
    top: 0;
    left: 0;
    width: 100%;
    height: 100%;
    z-index: 100;
}

.rodal {
    position: fixed;
}

/* -- mask -- */
.rodal-mask {
    position: fixed;
    background: rgba(0, 0, 0, .5);
}

/* -- dialog -- */
.rodal-dialog {
    position: absolute;
    z-index: 101;
    background: #fff;
    border-radius: 3px;
    box-shadow: 0 1px 3px rgba(0, 0, 0, .2);
}

.rodal-center {
    top: 50%;
    transform: translateY(-50%);
    left: 0;
    right: 0;
    margin: 0 auto;
}

.rodal-bottom {
    left: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

.rodal-top {
    left: 0;
    right: 0;
    top: 0;
    margin: auto;
}

.rodal-left {
    top: 0;
    left: 0;
    bottom: 0;
    margin: auto;
}

.rodal-right {
    top: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

/* -- close button -- */
.rodal-close {
    position: absolute;
    cursor: pointer;
    top: 16px;
    right: 16px;
    width: 16px;
    height: 16px;
}

.rodal-close:before,
.rodal-close:after {
    position: absolute;
    content: '';
    height: 2px;
    width: 100%;
    top: 50%;
    left: 0;
    margin-top: -1px;
    background: #999;
    border-radius: 100%;
    -webkit-transition: background .2s;
    transition: background .2s;
}

.rodal-close:before {
    -webkit-transform: rotate(45deg);
    transform: rotate(45deg);
}

.rodal-close:after {
    -webkit-transform: rotate(-45deg);
    transform: rotate(-45deg);
}

.rodal-close:hover:before,
.rodal-close:hover:after {
    background: #333;
}

/* -- fade -- */
/* @-webkit-keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

@keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

.rodal-fade-enter {
    -webkit-animation: rodal-fade-enter both ease-in;
    animation: rodal-fade-enter both ease-in;
} */

@-webkit-keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

@keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

.rodal-fade-leave {
    -webkit-animation: rodal-fade-leave both ease-out;
    animation: rodal-fade-leave both ease-out;
}

/* -- zoom -- */
@-webkit-keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-enter {
    -webkit-animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-leave {
    -webkit-animation: rodal-zoom-leave both;
    animation: rodal-zoom-leave both;
}

/* -- slideDown -- */
@-webkit-keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-enter {
    -webkit-animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-leave {
    -webkit-animation: rodal-slideDown-leave both;
    animation: rodal-slideDown-leave both;
}

/* -- slideLeft -- */
@-webkit-keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-enter {
    -webkit-animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-leave {
    -webkit-animation: rodal-slideLeft-leave both;
    animation: rodal-slideLeft-leave both;
}

/* -- slideRight -- */
@-webkit-keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-enter {
    -webkit-animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-leave {
    -webkit-animation: rodal-slideRight-leave both;
    animation: rodal-slideRight-leave both;
}

/* -- slideUp -- */
@-webkit-keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-enter {
    -webkit-animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
    animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
}

@-webkit-keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-leave {
    -webkit-animation: rodal-slideUp-leave both;
    animation: rodal-slideUp-leave both;
}

/* -- flip -- */
@-webkit-keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

@keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

.rodal-flip-enter {
    -webkit-animation: rodal-flip-enter both ease-in;
    animation: rodal-flip-enter both ease-in;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

@-webkit-keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

@keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

.rodal-flip-leave {
    -webkit-animation: rodal-flip-leave both;
    animation: rodal-flip-leave both;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

/* -- rotate -- */
@-webkit-keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-enter {
    -webkit-animation: rodal-rotate-enter both;
    animation: rodal-rotate-enter both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

@-webkit-keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-leave {
    -webkit-animation: rodal-rotate-leave both;
    animation: rodal-rotate-leave both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

/* -- door -- */
@-webkit-keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

@keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

.rodal-door-enter {
    -webkit-animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

@keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

.rodal-door-leave {
    -webkit-animation: rodal-door-leave both;
    animation: rodal-door-leave both;
}</style></head><body class=""><!-- BEGIN WAYBACK TOOLBAR INSERT -->
<style type="text/css">
body {
  margin-top:0 !important;
  padding-top:0 !important;
  /*min-width:800px !important;*/
}
</style>
<script>__wm.rw(0);</script>
<div id="wm-ipp-base" lang="en" style="display: block; direction: ltr;">
</div><div id="wm-ipp-print">The Wayback Machine - https://web.archive.org/web/20221028222850/https://baike.sogou.com/kexue/d10191.htm</div>
<script type="text/javascript">//<![CDATA[
__wm.bt(675,27,25,2,"web","https://baike.sogou.com/kexue/d10191.htm","20221028222850",1996,"/_static/",["/_static/css/banner-styles.css?v=S1zqJCYt","/_static/css/iconochive.css?v=qtvMKcIJ"], false);
  __wm.rw(1);
//]]></script>
<!-- END WAYBACK TOOLBAR INSERT --><script>window._gtag=window._gtag||{};window._gtag.shouldGrayed = false;if ('fe2c6d5249724b68a07f6c1274627842') window._gtag.traceId = 'fe2c6d5249724b68a07f6c1274627842';if ({"illegality":true}) window.userInfo = {"illegality":true};</script><div class="topnavbox"><ul class="topnav"><li><a href="https://web.archive.org/web/20221028222850/https://www.sogou.com/web?query=">网页</a></li><li><a href="https://web.archive.org/web/20221028222850/https://weixin.sogou.com/weixin?p=75351201">微信</a></li><li><a href="https://web.archive.org/web/20221028222850/https://zhihu.sogou.com/zhihu?p=75351218">知乎</a></li><li><a href="https://web.archive.org/web/20221028222850/https://pic.sogou.com/pics?query=">图片</a></li><li><a href="https://web.archive.org/web/20221028222850/https://v.sogou.com/v?query=">视频</a></li><li><a href="https://web.archive.org/web/20221028222850/https://mingyi.sogou.com/">医疗</a></li><li class="cur"><strong>科学</strong></li><li><a href="https://web.archive.org/web/20221028222850/https://hanyu.sogou.com/">汉语</a></li><li><a href="https://web.archive.org/web/20221028222850/https://wenwen.sogou.com/">问问</a></li><li><a href="https://web.archive.org/web/20221028222850/https://www.sogou.com/docs/more.htm">更多<span class="topraquo">»</span></a></li></ul></div><div id="header"><div class="header-wrap"><a class="header-logo" href="https://web.archive.org/web/20221028222850/https://baike.sogou.com/kexue"></a><div class="header-search"><div class="querybox" id="suggBox"><form><input id="searchInput" class="query" type="text" placeholder="搜科学领域专业百科词条" name="query" autocomplete="off" value=""><a href="javascript:;" class="query-search"></a></form></div></div><div class="header-rgt"><span class="btn-header-rgt btn-edit" id="editLemma">创建</span><div class="header-user no-login"></div></div></div></div><div class="fixed-placeholder" style="visibility:none"></div><div id="container" class=""><div class="content lemma-level1"><div class="detail-title" id="abstract-title"><h1>肽核酸</h1><a href="https://web.archive.org/web/20221028222850/https://baike.sogou.com/kexue/d10191.htm#!" class="detail-edit">编辑</a></div><div class="section_content" data-id="14994954862461197"><div class="text_img ed_imgfloat_right"><a class="ed_image_link" data-src="https://img03.sogoucdn.com/app/a/200698/sogou_science_2668" data-bigsrc="https://img03.sogoucdn.com/app/a/200698/sogou_science_2668?w=203&amp;h=238&amp;titlename=" title="点击查看大图" data-w="203" data-h="238" style="background-image:url(https://web.archive.org/web/20221028222850im_/https://img03.sogoucdn.com/app/a/200698/sogou_science_2668)" href="https://web.archive.org/web/20221028222850/https://baike.sogou.com/kexue/d10191.htm#!"></a></div><div><p>肽核酸（PNA）是一种人工合成的聚合物，类似于脱氧核糖核酸或核糖核酸。<sup><a href="https://web.archive.org/web/20221028222850/https://baike.sogou.com/kexue/d10191.htm#quote_1" class="kx_ref">[1]</a></sup> 它是由彼得·尼尔森Peter E. Nielsen(哥本哈根大学)、迈克尔·埃霍姆Michael Egholm(哥本哈根大学)、罗尔夫·伯格Rolf H. Berg(里瑟国家实验室)和奥莱·布加勒斯特Ole Buchardt(哥本哈根大学)在1991年发明的。 </p>
<p>近年来，合成肽核酸寡聚体已用于分子生物学流程、诊断分析和反义疗法。<sup><a href="https://web.archive.org/web/20221028222850/https://baike.sogou.com/kexue/d10191.htm#quote_2" class="kx_ref">[2]</a></sup> 由于其较高的结合强度，没有必要设计用于这些应用的长肽核酸寡聚物，这些应用通常需要20-25个碱基的寡核苷酸探针。肽核酸寡聚物长度的主要考虑是保证特异性。肽核酸寡聚物在与互补脱氧核糖核酸结合时也表现出更强的特异性，肽核酸/脱氧核糖核酸（PNA/DNA)碱基错配比脱氧核糖核酸/脱氧核糖核酸双链体中类似的错配更不稳定。这种结合强度和特异性也适用于肽核酸/核糖核酸双链体。肽核酸不容易被核酸酶或蛋白酶识别，因此它们能抵抗酶的降解。肽核酸在很宽的酸碱度范围内也是稳定的。尽管未修饰的肽核酸不能稳定地穿过细胞膜进入胞质溶胶，但是细胞穿透肽与肽核酸的共价偶联可以提高胞质溶胶的递送效率。<sup><a href="https://web.archive.org/web/20221028222850/https://baike.sogou.com/kexue/d10191.htm#quote_3" class="kx_ref">[3]</a></sup> </p>
<p>目前还不知道肽核酸是否天然存在，但是肽核酸的骨架——N-(2-氨基乙基)-甘氨酸(AEG)被认为是地球上生命遗传分子的早期形式，由蓝细菌产生。<sup><a href="https://web.archive.org/web/20221028222850/https://baike.sogou.com/kexue/d10191.htm#quote_4" class="kx_ref">[4]</a></sup> </p></div></div><div id="catalog"><h2 class="title2">目录<a href="javascript:" class="detail-edit">编辑</a></h2><div class="catalog_wrap" style=""><ul class="catalog_list col2"><li><span class="order">1</span><a href="javascript:" data-level="1" data-id="14994954862461198">结构</a></li><li><span class="order">2</span><a href="javascript:" data-level="1" data-id="14994954862461199">综合能力</a></li><li><span class="order">3</span><a href="javascript:" data-level="1" data-id="14994954879238404">从其他核酸翻译成肽核酸</a></li><li><span class="order">4</span><a href="javascript:" data-level="1" data-id="14994954879238405">传递</a></li></ul><ul class="catalog_list col2"><li><span class="order">5</span><a href="javascript:" data-level="1" data-id="14994954879238406">肽核酸世界假说</a></li><li><span class="order">6</span><a href="javascript:" data-level="1" data-id="14994954879238407">应用</a></li><li><span class="order">7</span><a href="javascript:" data-level="1" data-id="references">参考文献</a></li></ul></div></div><div id="paragraphs"><div><div id="par_14994954862461198"><h2 class="title">1 结构<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>脱氧核糖和核糖核酸分别有脱氧核糖和核糖骨架，而肽核酸的骨架由肽键连接的重复的氮-(2-氨基乙基)-甘氨酸单元组成。各种嘌呤和嘧啶碱基通过亚甲基桥(-CH2-)和羰基(-(碳=氧)-)连接到骨架上。肽核酸被描绘成类肽，其氮末端在第一(左)位置，碳末端在最后(右)位置。<sup><a href="https://web.archive.org/web/20221028222850/https://baike.sogou.com/kexue/d10191.htm#quote_5" class="kx_ref">[5]</a></sup> </p></div></div><div id="par_14994954862461199"><h2 class="title">2 综合能力<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>由于肽核酸的骨架不含带电荷的磷酸基团，由于缺乏静电排斥，肽核酸/脱氧核糖核酸链之间的结合比脱氧核糖核酸/脱氧核糖核酸链之间的结合更强。不幸的是，这也导致它相当疏水，这使得它不被冲出体外的话就很难转运到溶液中的体细胞。同嘧啶链(仅由一个重复嘧啶碱基组成的链)的早期实验表明，6-碱基胸腺嘧啶脱氧核糖核酸/腺嘌呤脱氧核糖核酸双螺旋的Tm(“熔化”温度)为31℃，与之相比，6-碱基脱氧核糖核酸/脱氧核糖核酸双螺旋在低于10℃的温度下变性。在碱基对识别方面，混合碱基PNA分子更加类似于DNA分子。肽核酸/肽核酸结合比肽核酸/脱氧核糖核酸结合更强。 </p></div></div><div id="par_14994954879238404"><h2 class="title">3 从其他核酸翻译成肽核酸<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>几个实验室已经报道了来自脱氧核糖核酸或核糖核酸模板的肽核酸的序列特异性聚合。<sup><a href="https://web.archive.org/web/20221028222850/https://baike.sogou.com/kexue/d10191.htm#quote_6" class="kx_ref">[6]</a></sup><sup><a href="https://web.archive.org/web/20221028222850/https://baike.sogou.com/kexue/d10191.htm#quote_7" class="kx_ref">[7]</a></sup><sup><a href="https://web.archive.org/web/20221028222850/https://baike.sogou.com/kexue/d10191.htm#quote_8" class="kx_ref">[8]</a></sup> Liu和同事使用这些聚合方法来进化功能性肽核酸，使其能够折叠成三维结构，类似于蛋白质、适体和核酶。<sup><a href="https://web.archive.org/web/20221028222850/https://baike.sogou.com/kexue/d10191.htm#quote_6" class="kx_ref">[6]</a></sup> </p></div></div><div id="par_14994954879238405"><h2 class="title">4 传递<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>2015年，Jain等人描述了一种基于反式作用的脱氧核糖核酸的两亲性传递系统，用于方便地传递聚腺苷酸尾不带电荷的核酸，如脱氧核糖核酸和吗啉类，从而可以在体外容易地筛选几种不带电荷的核酸。<sup><a href="https://web.archive.org/web/20221028222850/https://baike.sogou.com/kexue/d10191.htm#quote_9" class="kx_ref">[9]</a></sup> </p></div></div><div id="par_14994954879238406"><h2 class="title">5 肽核酸世界假说<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>据推测，地球上最早的生命可能已经肽核酸用作遗传物质，因为肽核酸具有极强的稳定性，结构简单，在100℃<sup><a href="https://web.archive.org/web/20221028222850/https://baike.sogou.com/kexue/d10191.htm#quote_10" class="kx_ref">[10]</a></sup> 时可能会发生自发聚合(标准压力下的水在这个温度沸腾，而高压下的水——如深海中的水——在更高的温度沸腾)。如果是这样的话，生命只是在后期才进化成基于脱氧核糖核酸/核糖核酸的系统。<sup><a href="https://web.archive.org/web/20221028222850/https://baike.sogou.com/kexue/d10191.htm#quote_11" class="kx_ref">[11]</a></sup><sup><a href="https://web.archive.org/web/20221028222850/https://baike.sogou.com/kexue/d10191.htm#quote_12" class="kx_ref">[12]</a></sup> 然而，肽核酸世界假说没有证据。<sup><a href="https://web.archive.org/web/20221028222850/https://baike.sogou.com/kexue/d10191.htm#quote_13" class="kx_ref">[13]</a></sup> </p></div></div><div id="par_14994954879238407"><h2 class="title">6 应用<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>其应用包括基因表达的改变——在不同情况下作为抑制剂和启动子，抗原和反义治疗剂，抗癌剂，抗病毒剂，抗细菌剂和抗寄生虫剂，生物传感器的分子工具和探针，检测脱氧核糖核酸序列，以及纳米技术。<sup><a href="https://web.archive.org/web/20221028222850/https://baike.sogou.com/kexue/d10191.htm#quote_14" class="kx_ref">[14]</a></sup><sup><a href="https://web.archive.org/web/20221028222850/https://baike.sogou.com/kexue/d10191.htm#quote_15" class="kx_ref">[15]</a></sup> </p>
<p>肽核酸可用于通过阻断污染物质体和线粒体序列的扩增来提高植物和土壤样品的高通量16S核糖体核糖核酸基因测序。<sup><a href="https://web.archive.org/web/20221028222850/https://baike.sogou.com/kexue/d10191.htm#quote_16" class="kx_ref">[16]</a></sup> </p></div></div></div></div><div id="references"><h2 class="title" id="par_references">参考文献</h2><ul class="references"><li id="quote_1"><span class="references-num">[1]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Nielsen PE, Egholm M, Berg RH, Buchardt O (December 1991). "Sequence-selective recognition of DNA by strand displacement with a thymine-substituted polyamide". Science. 254 (5037): 1497–500. Bibcode:1991Sci...254.1497N. doi:10.1126/science.1962210. PMID 1962210..</span></p></li><li id="quote_2"><span class="references-num">[2]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Gupta A, Mishra A, Puri N (October 2017). "Peptide nucleic acids: Advanced tools for biomedical applications". Journal of Biotechnology. 259: 148–159. doi:10.1016/j.jbiotec.2017.07.026. PMID 28764969..</span></p></li><li id="quote_3"><span class="references-num">[3]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Zhao XL, Chen BC, Han JC, Wei L, Pan XB (November 2015). "Delivery of cell-penetrating peptide-peptide nucleic acid conjugates by assembly on an oligonucleotide scaffold". Scientific Reports. 5: 17640. doi:10.1038/srep17640. PMC 4661726. PMID 26612536..</span></p></li><li id="quote_4"><span class="references-num">[4]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Cyanobacteria Produce N-(2-Aminoethyl)Glycine, a Backbone for Peptide Nucleic Acids Which May Have Been the First Genetic Molecules for Life on Earth.</span></p></li><li id="quote_5"><span class="references-num">[5]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Egholm M, Buchardt O, Christensen L, Behrens C, Freier SM, Driver DA, Berg RH, Kim SK, Norden B, Nielsen PE (October 1993). "PNA hybridizes to complementary oligonucleotides obeying the Watson-Crick hydrogen-bonding rules". Nature. 365 (6446): 566–8. Bibcode:1993Natur.365..566E. doi:10.1038/365566a0. PMID 7692304..</span></p></li><li id="quote_6"><span class="references-num">[6]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Brudno Y, Birnbaum ME, Kleiner RE, Liu DR (February 2010). "An in vitro translation, selection and amplification system for peptide nucleic acids". Nature Chemical Biology. 6 (2): 148–55. doi:10.1038/nchembio.280. PMC 2808706. PMID 20081830..</span></p></li><li id="quote_7"><span class="references-num">[7]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Kleiner RE, Brudno Y, Birnbaum ME, Liu DR (April 2008). "DNA-templated polymerization of side-chain-functionalized peptide nucleic acid aldehydes". Journal of the American Chemical Society. 130 (14): 4646–59. doi:10.1021/ja0753997. PMC 2748799. PMID 18341334..</span></p></li><li id="quote_8"><span class="references-num">[8]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Ura Y, Beierle JM, Leman LJ, Orgel LE, Ghadiri MR (July 2009). "Self-assembling sequence-adaptive peptide nucleic acids". Science. 325 (5936): 73–7. Bibcode:2009Sci...325...73U. doi:10.1126/science.1174577. PMID 19520909..</span></p></li><li id="quote_9"><span class="references-num">[9]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Jain HV, Verthelyi D, Beaucage SL (2015). "Amphipathic trans-acting phosphorothioate DNA elements mediate the delivery of uncharged nucleic acid sequences in mammalian cells". RSC Advances. 5 (80): 65245–65254. doi:10.1039/C5RA12038A..</span></p></li><li id="quote_10"><span class="references-num">[10]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Wittung P, Nielsen PE, Buchardt O, Egholm M &amp; Nordén B (April 1994). "DNA-like double helix formed by peptide nucleic acid". Nature. 368 (6471): 561–3. Bibcode:1994Natur.368..561W. doi:10.1038/368561a0. PMID 8139692..</span></p></li><li id="quote_11"><span class="references-num">[11]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Nelson KE, Levy M, Miller SL (April 2000). "Peptide nucleic acids rather than RNA may have been the first genetic molecule". Proceedings of the National Academy of Sciences of the United States of America. 97 (8): 3868–71. Bibcode:2000PNAS...97.3868N. doi:10.1073/pnas.97.8.3868. PMC 18108. PMID 10760258..</span></p></li><li id="quote_12"><span class="references-num">[12]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Alberts B, Johnson A, Lewis J (March 2002). Molecular Biology of the Cell (4th ed.). Routledge. ISBN 978-0-8153-3218-3..</span></p></li><li id="quote_13"><span class="references-num">[13]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Zimmer C (January 2009). "Evolutionary roots. On the origin of life on Earth". Science. 323 (5911): 198–9. doi:10.1126/science.323.5911.198. PMID 19131603..</span></p></li><li id="quote_14"><span class="references-num">[14]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Anstaett P, Gasser G (2014). "Peptide nucleic acid - an opportunity for bio-nanotechnology". Chimia. 68 (4): 264–8. doi:10.2533/chimia.2014.264. PMID 24983612..</span></p></li><li id="quote_15"><span class="references-num">[15]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">D'Souza AD, Belotserkovskii BP, Hanawalt PC (February 2018). "A novel mode for transcription inhibition mediated by PNA-induced R-loops with a model in vitro system". Biochimica et Biophysica Acta. Gene Regulatory Mechanisms. 1861 (2): 158–166. doi:10.1016/j.bbagrm.2017.12.008. PMC 5820110. PMID 29357316..</span></p></li><li id="quote_16"><span class="references-num">[16]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Lundberg DS, Yourstone S, Mieczkowski P, Jones CD, Dangl JL (October 2013). "Practical innovations for high-throughput amplicon sequencing". Nature Methods. 10 (10): 999–1002. doi:10.1038/nmeth.2634. PMID 23995388..</span></p></li></ul></div><div class="read-num">阅读 <!-- -->372</div></div><div class="right-side" id="rightSide"><div class="side" id="lemma-side"><div class="side-title">版本记录</div><ul class="side-lst"><li><p class="side-lst-txt">暂无</p></li></ul><div class="user-card userCard"></div></div><div class="side"><div class="side-event"></div></div></div></div><div class="footer-box"><div id="footer"><div class="footer-logo-wrap"><div class="footer-logo"></div><div class="footer-logo-text">知识·传播·科普</div></div><div class="footer-info">本网站内容采用<a target="_blank" href="https://web.archive.org/web/20221028222850/https://creativecommons.org/licenses/by-sa/3.0/deed.zh?tdsourcetag=s_pctim_aiomsg">CC-BY-SA 3.0</a>授权</div><div class="footer-btn-wrap"><a target="_blank" href="https://web.archive.org/web/20221028222850/https://baike.sogou.com/help/#user_protocol">用户协议</a><a target="_blank" href="https://web.archive.org/web/20221028222850/http://www.sogou.com/docs/terms.htm?v=1">免责声明</a><a target="_blank" href="https://web.archive.org/web/20221028222850/http://corp.sogou.com/private.html">隐私政策</a><a target="_blank" href="https://web.archive.org/web/20221028222850/https://baike.sogou.com/kexue/intro.htm">关于我们</a></div></div></div><script>window.lemmaInfo ={"lemmaId":"10191","versionId":"14994954862461189","title":"肽核酸","subtitle":"","abstracts":{"paragraphId":"14994954862461197","title":"简介","versionId":"14994954862461190","lemmaId":10191,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":53099137,"name":"孟凡康","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233689,"comment":null,"dependVersionId":0,"contentType":2,"content":"<p>肽核酸（PNA）是一种人工合成的聚合物，类似于脱氧核糖核酸或核糖核酸。<sup><a href=\"#quote_1\" class=\"kx_ref\">[1]</a></sup> 它是由彼得·尼尔森Peter E. Nielsen(哥本哈根大学)、迈克尔·埃霍姆Michael Egholm(哥本哈根大学)、罗尔夫·伯格Rolf H. Berg(里瑟国家实验室)和奥莱·布加勒斯特Ole Buchardt(哥本哈根大学)在1991年发明的。 </p>\n<p>近年来，合成肽核酸寡聚体已用于分子生物学流程、诊断分析和反义疗法。<sup><a href=\"#quote_2\" class=\"kx_ref\">[2]</a></sup> 由于其较高的结合强度，没有必要设计用于这些应用的长肽核酸寡聚物，这些应用通常需要20-25个碱基的寡核苷酸探针。肽核酸寡聚物长度的主要考虑是保证特异性。肽核酸寡聚物在与互补脱氧核糖核酸结合时也表现出更强的特异性，肽核酸/脱氧核糖核酸（PNA/DNA)碱基错配比脱氧核糖核酸/脱氧核糖核酸双链体中类似的错配更不稳定。这种结合强度和特异性也适用于肽核酸/核糖核酸双链体。肽核酸不容易被核酸酶或蛋白酶识别，因此它们能抵抗酶的降解。肽核酸在很宽的酸碱度范围内也是稳定的。尽管未修饰的肽核酸不能稳定地穿过细胞膜进入胞质溶胶，但是细胞穿透肽与肽核酸的共价偶联可以提高胞质溶胶的递送效率。<sup><a href=\"#quote_3\" class=\"kx_ref\">[3]</a></sup> </p>\n<p>目前还不知道肽核酸是否天然存在，但是肽核酸的骨架——N-(2-氨基乙基)-甘氨酸(AEG)被认为是地球上生命遗传分子的早期形式，由蓝细菌产生。<sup><a href=\"#quote_4\" class=\"kx_ref\">[4]</a></sup> </p>","pics":[{"originalUrl":"https://web.archive.org/web/20221028222850/https://img03.sogoucdn.com/app/a/200698/sogou_science_2668?w=203&h=238&titlename=","url":"https://web.archive.org/web/20221028222850/https://img03.sogoucdn.com/app/a/200698/sogou_science_2668","rw":203,"rh":238,"title":"","alt":null,"width":0,"height":0}],"card":null,"references":[],"versionCount":0},"card":{"paragraphId":"0","title":null,"versionId":"0","lemmaId":0,"createType":0,"creator":null,"createTime":0,"versionEditor":null,"editTime":0,"comment":null,"dependVersionId":0,"contentType":0,"content":null,"pics":null,"card":null,"references":null,"versionCount":0},"categories":[{"id":4,"name":"生物学","parents":[]}],"creator":{"uid":53099137,"name":"孟凡康","pic":"https://web.archive.org/web/20221028222850/https://cache.soso.com/qlogo/g?b=oidb&k=gfMphd6l1FFGwj0oC386Fg&s=100&t=1561211753","introduction":null,"educations":[{"schoolName":"中国科学院大学","major":"","degree":"博士","universityId":111,"universityLogo":"https://web.archive.org/web/20221028222850/https://img01.sogoucdn.com/app/a/200943/d7a2d86e-6011-11e9-a2ce-fc4dd4f70029","majorLevel1":"理学","majorLevel2":"生物学","majorLevel3":"合成生物学","majorLevel1Id":1,"majorLevel2Id":189,"majorLevel3Id":204,"state":null,"lab":null,"researchField":null}],"jobs":null,"works":null,"educationBrief":"中国科学院大学","jobBrief":"","role":0,"roleName":null,"title":"中国科学院大学 · 生物学博士","professionalTitle":null,"phoneNo":null,"editable":true,"partnerId":99,"partnerIdCreateTime":1594286514,"partnerIdPoped":false},"createTime":1571038083,"editor":{"uid":53099137,"name":"孟凡康","pic":"https://web.archive.org/web/20221028222850/https://cache.soso.com/qlogo/g?b=oidb&k=gfMphd6l1FFGwj0oC386Fg&s=100&t=1561211753","introduction":null,"educations":[{"schoolName":"中国科学院大学","major":"","degree":"博士","universityId":111,"universityLogo":"https://web.archive.org/web/20221028222850/https://img01.sogoucdn.com/app/a/200943/d7a2d86e-6011-11e9-a2ce-fc4dd4f70029","majorLevel1":"理学","majorLevel2":"生物学","majorLevel3":"合成生物学","majorLevel1Id":1,"majorLevel2Id":189,"majorLevel3Id":204,"state":null,"lab":null,"researchField":null}],"jobs":null,"works":null,"educationBrief":"中国科学院大学","jobBrief":"","role":0,"roleName":null,"title":"中国科学院大学 · 生物学博士","professionalTitle":null,"phoneNo":null,"editable":true,"partnerId":99,"partnerIdCreateTime":1594286514,"partnerIdPoped":false},"editTime":1576233689,"state":1,"versionCount":1,"upNum":1,"downNum":0,"pics":[{"originalUrl":"https://web.archive.org/web/20221028222850/https://img03.sogoucdn.com/app/a/200698/sogou_science_2668?w=203&h=238&titlename=","url":"https://web.archive.org/web/20221028222850/https://img03.sogoucdn.com/app/a/200698/sogou_science_2668","rw":203,"rh":238,"title":"","alt":null,"width":0,"height":0}],"catalogs":[{"level":1,"title":"结构","paragraphId":"14994954862461198","subCatalogs":null},{"level":1,"title":"综合能力","paragraphId":"14994954862461199","subCatalogs":null},{"level":1,"title":"从其他核酸翻译成肽核酸","paragraphId":"14994954879238404","subCatalogs":null},{"level":1,"title":"传递","paragraphId":"14994954879238405","subCatalogs":null},{"level":1,"title":"肽核酸世界假说","paragraphId":"14994954879238406","subCatalogs":null},{"level":1,"title":"应用","paragraphId":"14994954879238407","subCatalogs":null},{"level":1,"title":"参考文献","paragraphId":"-1","subCatalogs":null}],"paragraphs":[{"paragraphId":"14994954862461198","title":"结构","versionId":"14994954862461191","lemmaId":10191,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":53099137,"name":"孟凡康","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233689,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>脱氧核糖和核糖核酸分别有脱氧核糖和核糖骨架，而肽核酸的骨架由肽键连接的重复的氮-(2-氨基乙基)-甘氨酸单元组成。各种嘌呤和嘧啶碱基通过亚甲基桥(-CH2-)和羰基(-(碳=氧)-)连接到骨架上。肽核酸被描绘成类肽，其氮末端在第一(左)位置，碳末端在最后(右)位置。<sup><a href=\"#quote_5\" class=\"kx_ref\">[5]</a></sup> </p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14994954862461199","title":"综合能力","versionId":"14994954862461192","lemmaId":10191,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":53099137,"name":"孟凡康","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233689,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>由于肽核酸的骨架不含带电荷的磷酸基团，由于缺乏静电排斥，肽核酸/脱氧核糖核酸链之间的结合比脱氧核糖核酸/脱氧核糖核酸链之间的结合更强。不幸的是，这也导致它相当疏水，这使得它不被冲出体外的话就很难转运到溶液中的体细胞。同嘧啶链(仅由一个重复嘧啶碱基组成的链)的早期实验表明，6-碱基胸腺嘧啶脱氧核糖核酸/腺嘌呤脱氧核糖核酸双螺旋的Tm(“熔化”温度)为31℃，与之相比，6-碱基脱氧核糖核酸/脱氧核糖核酸双螺旋在低于10℃的温度下变性。在碱基对识别方面，混合碱基PNA分子更加类似于DNA分子。肽核酸/肽核酸结合比肽核酸/脱氧核糖核酸结合更强。 </p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14994954879238404","title":"从其他核酸翻译成肽核酸","versionId":"14994954862461193","lemmaId":10191,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":53099137,"name":"孟凡康","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233689,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>几个实验室已经报道了来自脱氧核糖核酸或核糖核酸模板的肽核酸的序列特异性聚合。<sup><a href=\"#quote_6\" class=\"kx_ref\">[6]</a></sup><sup><a href=\"#quote_7\" class=\"kx_ref\">[7]</a></sup><sup><a href=\"#quote_8\" class=\"kx_ref\">[8]</a></sup> Liu和同事使用这些聚合方法来进化功能性肽核酸，使其能够折叠成三维结构，类似于蛋白质、适体和核酶。<sup><a href=\"#quote_6\" class=\"kx_ref\">[6]</a></sup> </p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14994954879238405","title":"传递","versionId":"14994954862461194","lemmaId":10191,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":53099137,"name":"孟凡康","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233689,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>2015年，Jain等人描述了一种基于反式作用的脱氧核糖核酸的两亲性传递系统，用于方便地传递聚腺苷酸尾不带电荷的核酸，如脱氧核糖核酸和吗啉类，从而可以在体外容易地筛选几种不带电荷的核酸。<sup><a href=\"#quote_9\" class=\"kx_ref\">[9]</a></sup> </p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14994954879238406","title":"肽核酸世界假说","versionId":"14994954862461195","lemmaId":10191,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":53099137,"name":"孟凡康","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233689,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>据推测，地球上最早的生命可能已经肽核酸用作遗传物质，因为肽核酸具有极强的稳定性，结构简单，在100℃<sup><a href=\"#quote_10\" class=\"kx_ref\">[10]</a></sup> 时可能会发生自发聚合(标准压力下的水在这个温度沸腾，而高压下的水——如深海中的水——在更高的温度沸腾)。如果是这样的话，生命只是在后期才进化成基于脱氧核糖核酸/核糖核酸的系统。<sup><a href=\"#quote_11\" class=\"kx_ref\">[11]</a></sup><sup><a href=\"#quote_12\" class=\"kx_ref\">[12]</a></sup> 然而，肽核酸世界假说没有证据。<sup><a href=\"#quote_13\" class=\"kx_ref\">[13]</a></sup> </p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14994954879238407","title":"应用","versionId":"14994954862461196","lemmaId":10191,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":53099137,"name":"孟凡康","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233689,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>其应用包括基因表达的改变——在不同情况下作为抑制剂和启动子，抗原和反义治疗剂，抗癌剂，抗病毒剂，抗细菌剂和抗寄生虫剂，生物传感器的分子工具和探针，检测脱氧核糖核酸序列，以及纳米技术。<sup><a href=\"#quote_14\" class=\"kx_ref\">[14]</a></sup><sup><a href=\"#quote_15\" class=\"kx_ref\">[15]</a></sup> </p>\n<p>肽核酸可用于通过阻断污染物质体和线粒体序列的扩增来提高植物和土壤样品的高通量16S核糖体核糖核酸基因测序。<sup><a href=\"#quote_16\" class=\"kx_ref\">[16]</a></sup> </p>","pics":null,"card":null,"references":[],"versionCount":0}],"references":[{"id":1,"type":"book","title":"Nielsen PE, Egholm M, Berg RH, Buchardt O (December 1991). \"Sequence-selective recognition of DNA by strand displacement with a thymine-substituted polyamide\". Science. 254 (5037): 1497–500. Bibcode:1991Sci...254.1497N. doi:10.1126/science.1962210. PMID 1962210.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":2,"type":"book","title":"Gupta A, Mishra A, Puri N (October 2017). \"Peptide nucleic acids: Advanced tools for biomedical applications\". Journal of Biotechnology. 259: 148–159. doi:10.1016/j.jbiotec.2017.07.026. PMID 28764969.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":3,"type":"book","title":"Zhao XL, Chen BC, Han JC, Wei L, Pan XB (November 2015). \"Delivery of cell-penetrating peptide-peptide nucleic acid conjugates by assembly on an oligonucleotide scaffold\". Scientific Reports. 5: 17640. doi:10.1038/srep17640. PMC 4661726. PMID 26612536.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":4,"type":"book","title":"Cyanobacteria Produce N-(2-Aminoethyl)Glycine, a Backbone for Peptide Nucleic Acids Which May Have Been the First Genetic Molecules for Life on Earth","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":5,"type":"book","title":"Egholm M, Buchardt O, Christensen L, Behrens C, Freier SM, Driver DA, Berg RH, Kim SK, Norden B, Nielsen PE (October 1993). \"PNA hybridizes to complementary oligonucleotides obeying the Watson-Crick hydrogen-bonding rules\". Nature. 365 (6446): 566–8. Bibcode:1993Natur.365..566E. doi:10.1038/365566a0. PMID 7692304.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":6,"type":"book","title":"Brudno Y, Birnbaum ME, Kleiner RE, Liu DR (February 2010). \"An in vitro translation, selection and amplification system for peptide nucleic acids\". Nature Chemical Biology. 6 (2): 148–55. doi:10.1038/nchembio.280. PMC 2808706. PMID 20081830.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":7,"type":"book","title":"Kleiner RE, Brudno Y, Birnbaum ME, Liu DR (April 2008). \"DNA-templated polymerization of side-chain-functionalized peptide nucleic acid aldehydes\". Journal of the American Chemical Society. 130 (14): 4646–59. doi:10.1021/ja0753997. PMC 2748799. PMID 18341334.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":8,"type":"book","title":"Ura Y, Beierle JM, Leman LJ, Orgel LE, Ghadiri MR (July 2009). \"Self-assembling sequence-adaptive peptide nucleic acids\". Science. 325 (5936): 73–7. Bibcode:2009Sci...325...73U. doi:10.1126/science.1174577. PMID 19520909.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":9,"type":"book","title":"Jain HV, Verthelyi D, Beaucage SL (2015). \"Amphipathic trans-acting phosphorothioate DNA elements mediate the delivery of uncharged nucleic acid sequences in mammalian cells\". RSC Advances. 5 (80): 65245–65254. doi:10.1039/C5RA12038A.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":10,"type":"book","title":"Wittung P, Nielsen PE, Buchardt O, Egholm M & Nordén B (April 1994). \"DNA-like double helix formed by peptide nucleic acid\". Nature. 368 (6471): 561–3. Bibcode:1994Natur.368..561W. doi:10.1038/368561a0. PMID 8139692.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":11,"type":"book","title":"Nelson KE, Levy M, Miller SL (April 2000). \"Peptide nucleic acids rather than RNA may have been the first genetic molecule\". Proceedings of the National Academy of Sciences of the United States of America. 97 (8): 3868–71. Bibcode:2000PNAS...97.3868N. doi:10.1073/pnas.97.8.3868. PMC 18108. PMID 10760258.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":12,"type":"book","title":"Alberts B, Johnson A, Lewis J (March 2002). Molecular Biology of the Cell (4th ed.). Routledge. ISBN 978-0-8153-3218-3.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":13,"type":"book","title":"Zimmer C (January 2009). \"Evolutionary roots. On the origin of life on Earth\". Science. 323 (5911): 198–9. doi:10.1126/science.323.5911.198. PMID 19131603.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":14,"type":"book","title":"Anstaett P, Gasser G (2014). \"Peptide nucleic acid - an opportunity for bio-nanotechnology\". Chimia. 68 (4): 264–8. doi:10.2533/chimia.2014.264. PMID 24983612.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":15,"type":"book","title":"D'Souza AD, Belotserkovskii BP, Hanawalt PC (February 2018). \"A novel mode for transcription inhibition mediated by PNA-induced R-loops with a model in vitro system\". Biochimica et Biophysica Acta. Gene Regulatory Mechanisms. 1861 (2): 158–166. doi:10.1016/j.bbagrm.2017.12.008. PMC 5820110. PMID 29357316.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":16,"type":"book","title":"Lundberg DS, Yourstone S, Mieczkowski P, Jones CD, Dangl JL (October 2013). \"Practical innovations for high-throughput amplicon sequencing\". Nature Methods. 10 (10): 999–1002. doi:10.1038/nmeth.2634. PMID 23995388.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false}],"recommendReferences":null,"auditState":2,"lemmaLevel":1,"origin":0,"originEnTitle":null,"originZhTitle":null,"pv":372,"auditType":0,"synonyms":null,"showEditTime":"2019.12.13 18:41","auditors":[{"uid":0,"name":"Ki.κe","pic":"https://web.archive.org/web/20221028222850/https://wx.qlogo.cn/mmopen/vi_32/y67kfr32Doib4wg71Jiau7jVWvharic3nRKgdRRQSl6koeQJCo0GQs2Krw0vwdFRsOWnHIQOwAZsSg5lIkIrFCOcQ/132","introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":"","jobBrief":"","role":0,"roleName":null,"title":"","professionalTitle":null,"phoneNo":null,"editable":true,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false}],"hasZhishiNav":false,"auditInfos":{},"isHistory":false};</script><script crossorigin="anonymous" src="./191.肽核酸 - 搜狗科学百科_files/aegis.min.js.download"></script><script crossorigin="anonymous" src="./191.肽核酸 - 搜狗科学百科_files/main_2020092401.js.download"></script><script crossorigin="anonymous" src="./191.肽核酸 - 搜狗科学百科_files/react.production.min.js.download"></script><script crossorigin="anonymous" src="./191.肽核酸 - 搜狗科学百科_files/react-dom.production.min.js.download"></script><script crossorigin="anonymous" src="./191.肽核酸 - 搜狗科学百科_files/jquery-1.11.1.min.js.download"></script><script crossorigin="anonymous" src="./191.肽核酸 - 搜狗科学百科_files/main_2022062701.js.download"></script><script crossorigin="anonymous" src="./191.肽核酸 - 搜狗科学百科_files/main_66bbe21.js.download"></script><script crossorigin="anonymous" src="./191.肽核酸 - 搜狗科学百科_files/react.production.min.js.download"></script><script crossorigin="anonymous" src="./191.肽核酸 - 搜狗科学百科_files/react-dom.production.min.js.download"></script><script crossorigin="anonymous" src="./191.肽核酸 - 搜狗科学百科_files/main_edf0f08.js.download"></script>
</body></html>