<!DOCTYPE html>
<!-- saved from url=(0083)https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm -->
<html class="" data-reactroot=""><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><script src="./248.Folding@home - 搜狗科学百科_files/analytics.js.download" type="text/javascript"></script>
<script type="text/javascript">window.addEventListener('DOMContentLoaded',function(){var v=archive_analytics.values;v.service='wb';v.server_name='wwwb-app210.us.archive.org';v.server_ms=212;archive_analytics.send_pageview({});});</script>
<script type="text/javascript" src="./248.Folding@home - 搜狗科学百科_files/bundle-playback.js.download" charset="utf-8"></script>
<script type="text/javascript" src="./248.Folding@home - 搜狗科学百科_files/wombat.js.download" charset="utf-8"></script>
<script type="text/javascript">
  __wm.init("https://web.archive.org/web");
  __wm.wombat("https://baike.sogou.com/kexue/d10248.htm","20221029033459","https://web.archive.org/","web","/_static/",
	      "1667014499");
</script>
<link rel="stylesheet" type="text/css" href="./248.Folding@home - 搜狗科学百科_files/banner-styles.css">
<link rel="stylesheet" type="text/css" href="./248.Folding@home - 搜狗科学百科_files/iconochive.css">
<!-- End Wayback Rewrite JS Include -->
<meta name="save" content="history"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="baidu-site-verification" content="VWGb6TyYx8"><meta content="Folding@home - 搜狗科学百科" name="keywords"><meta content="搜狗科学百科是一部有着平等、协作、分享、自由理念的网络科学全书，为每一个互联网用户创造一个涵盖所有领域知识、服务的中文知识性平台。" name="description"><meta http-equiv="x-dns-prefetch-control" content="on"><meta name="server" baike="235" ip="210" env="online"><link rel="dns-prefetch" href="https://web.archive.org/web/20221029033459/https://cache.soso.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221029033459/https://hhy.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221029033459/https://pic.baike.soso.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221029033459/https://ugc.qpic.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221029033459/https://xui.ptlogin2.qq.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221029033459/https://q1.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221029033459/https://q2.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221029033459/https://q3.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221029033459/https://q4.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221029033459/https://q.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221029033459/https://img01.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221029033459/https://img02.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221029033459/https://img03.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221029033459/https://img04.sogoucdn.com/"><link rel="Shortcut Icon" href="https://web.archive.org/web/20221029033459im_/https://www.sogou.com/images/logo/new/favicon.ico?v=4"><link rel="Bookmark" href="https://www.sogou.com/images/logo/new/favicon.ico?v=4"><link href="./248.Folding@home - 搜狗科学百科_files/base_b849887.css" rel="stylesheet"><link href="./248.Folding@home - 搜狗科学百科_files/detail_378aed5.css" rel="stylesheet"><link href="./248.Folding@home - 搜狗科学百科_files/inviteAudit_7894507.css" rel="stylesheet"><link rel="stylesheet" href="./248.Folding@home - 搜狗科学百科_files/highlight.min.css"><title>Folding@home - 搜狗科学百科</title><style>.onekey-close {
	position: absolute;
	top: 16px;
	right: 16px;
	width: 24px;
	height: 24px;
	background: url(//web.archive.org/web/20221029045211/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	text-indent: -999em;
	background-size: 84px;
	background-position: -63px 0;
}

.onekey-login {
	position: absolute;
	top: 16.4%;
	left: 0;
	right: 0;
	width: 100%;
}

/* .onekey-login-img {
    width: 75px;
    height: 75px;
    background: url("https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/images/sprite_wap_baike.png") no-repeat;
    background-size: 100px 91px;
    background-position: 0 0;
    background-repeat: no-repeat;
    margin: 0 auto;
} */

.onekey-login-title {
	text-align: center;
	padding-bottom: 3px;
	font-size: 21px;
	font-weight: bold;
	line-height: 30px;
	color: #000;
}

.onekey-login-txt {
	text-align: center;
	font-family: PingFangSC;
	font-size: 14px;
	line-height: 20px;
	color: #8f8f8f;
}

.onekey-login-qq,
.onekey-login-wx,
.onekey-login-phone {
	display: block;
	width: 245px;
	height: 54px;
	border-radius: 45px;
	text-align: center;

	margin: 0 auto;
	font-size: 17px;
	line-height: 24px;
	color: #000;
	/* padding: 16px 77px; */
	border-radius: 12px;
	border: solid 1px #e0e0e0;
}
.onekey-qq-content,
.onekey-vx-content,
.onekey-phone-content {
	display: inline-block;
	margin-top: 16px;
}
.onekey-qq-content {
	padding: 0 5px;
}

.onekey-login-qq {
	margin-top: 48px;
	margin-bottom: 24px;
}

.onekey-login-qq:before {
	display: inline-block;
	content: "";
	width: 20px;
	height: 20px;
	background: url(//web.archive.org/web/20221029045211/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	background-size: 80px;
	background-position: -20px 0;
	vertical-align: top;
	margin: 17px 8px 0 0;
}

.onekey-login-wx {
	margin-bottom: 24px;
}

.onekey-login-wx:before {
	display: inline-block;
	content: "";
	width: 21px;
	height: 21px;
	background: url(//web.archive.org/web/20221029045211/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	background-size: 84px;
	background-position: 0 0;
	vertical-align: top;
	margin: 17px 10px 0 0;
}

.onekey-login-phone {
}

.onekey-login-phone:before {
	display: inline-block;
	content: "";
	width: 21px;
	height: 21px;
	background: url(//web.archive.org/web/20221029045211/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	background-size: 84px;
	background-position: -42px 0;
	vertical-align: top;
	margin: 17px 10px 0 0;
}

.onekey-fixed {
	z-index: 100;
	position: fixed;
	top: 0;
	bottom: 0;
	left: 0;
	right: 0;
	background: #fff;
	width: 100%;
	height: 100%;
}

.onekey-fixed.forbid {
	z-index: 100;
	position: fixed;
	top: auto;
	bottom: 68px;
	left: 9%;
	right: 0;
	background: rgba(0, 0, 0, 0.7);
	width: 82%;
	height: 43px;
	border-radius: 25px;
	color: #ffffff;
}
.onekey-login-title.forbid {
	text-align: center;
	padding-bottom: 3px;
	font-size: 14px;
	font-weight: normal;
	line-height: 30px;
	color: white;
}
</style><style>#login_mask {
  background: #000;
  opacity: 0.5;
  filter: alpha(opacity=50);
  position: fixed;
  /*fixed好像在哪个IE上有BUG，先用用*/
  left: 0;
  top: 0;
  z-index: 999;
  height: 100%;
}

#login_iframe_container {
  position: fixed;
  width: 550px;
  height: 360px;
  z-index: 1020;
  background-color: #ffffff;
}

@media screen and (max-width: 828px) {
  #login_iframe_container {
    top: 50% !important;
    left: 50% !important;
    transform: translate(-50%, -50%);
  }
}

#login_iframe_container.new-login {
  width: 550px;
  height: 360px;
  background-image: url(//web.archive.org/web/20221029045211/https://hhy.sogoucdn.com/js/common/hhy/background_2a4a8a6.png);
}

#login_iframe_container.new-login.no-bg {
  background: #fff;
}

#login_iframe_container.new-login .login-title {
  width: 100%;
  height: 42px;
  line-height: 42px;
  text-align: center;
  font-size: 30px;
  letter-spacing: 0.19px;
  color: #ffffff;
  margin-top: 62px;
}
#login_iframe_container.new-login .forbid-title {
  width: 100%;
  height: 42px;
  line-height: 42px;
  text-align: center;
  font-size: 24px;
  letter-spacing: 0.19px;
  color: #333333;
  margin-top: 150px;
}

#login_iframe_container.new-login.no-bg .login-title {
  color: #333333;
}

#login_iframe_container.new-login .login-subtitle {
  width: 100%;
  height: 18px;
  line-height: 18px;
  font-size: 13px;
  letter-spacing: 0.08px;
  color: #ffffff;
  text-align: center;
  margin-top: 9px;
  margin-bottom: 43px;
}

#login_iframe_container.new-login.no-bg .login-subtitle {
  color: #999999;
}

#login_iframe_container.new-login .login-subtitle::before {
  content: '';
  display: inline-block;
  width: 10px;
  height: 1px;
  background-color: #ffffff;
  position: relative;
  top: -4px;
  left: -5px;
}

#login_iframe_container.new-login .login-subtitle::after {
  content: '';
  display: inline-block;
  width: 10px;
  height: 1px;
  background-color: #ffffff;
  position: relative;
  top: -4px;
  left: 5px;
}

#login_iframe_container.new-login.no-bg .login-subtitle::before {
  background-color: #999999;
}

#login_iframe_container.new-login.no-bg .login-subtitle::after {
  background-color: #999999;
}

#login_iframe_container.new-login .close-btn {
  position: absolute;
  top: 20px;
  right: 20px;
  width: 12px;
  height: 12px;
  background: url(//web.archive.org/web/20221029045211/https://hhy.sogoucdn.com/js/common/hhy/login-sprites_e3853e5.png) -59px -10px;
  background-size: 81px 91px;
  cursor: pointer;
}

#login_iframe_container.new-login .login-btn {
  width: 220px;
  height: 47px;
  border-radius: 24px;
  border: solid 1px #dddddd;
  background-color: #ffffff;
  margin: 0 auto;
  margin-top: 28px;
  position: relative;
  display: block;
}

#login_iframe_container.new-login .login-btn .login-icon {
  position: absolute;
}

#login_iframe_container.new-login .login-btn .login-text {
  width: 61px;
  height: 47px;
  line-height: 47px;
  vertical-align: middle;
  font-size: 15px;
  letter-spacing: 0.1px;
  color: #666666;
  position: absolute;
  right: 62px;
}

#login_iframe_container.new-login .login-btn.qq-btn .login-icon {
  width: 22px;
  height: 27px;
  top: 10px;
  left: 67px;
  background: url(//web.archive.org/web/20221029045211/https://hhy.sogoucdn.com/js/common/hhy/login-sprites_e3853e5.png) -10px -54px;
  background-size: 81px 91px;
}

#login_iframe_container.new-login .login-btn.qq-btn .login-text {
  right: 59px;
}

#login_iframe_container.new-login .login-btn.wechat-btn .login-icon {
  width: 29px;
  height: 24px;
  top: 12px;
  left: 62px;
  background: url(//web.archive.org/web/20221029045211/https://hhy.sogoucdn.com/js/common/hhy/login-sprites_e3853e5.png) -10px -10px;
  background-size: 81px 91px;
}</style><style>/* -- container -- */
.rodal,
.rodal-mask {
    top: 0;
    left: 0;
    width: 100%;
    height: 100%;
    z-index: 100;
}

.rodal {
    position: fixed;
}

/* -- mask -- */
.rodal-mask {
    position: fixed;
    background: rgba(0, 0, 0, .5);
}

/* -- dialog -- */
.rodal-dialog {
    position: absolute;
    z-index: 101;
    background: #fff;
    border-radius: 3px;
    box-shadow: 0 1px 3px rgba(0, 0, 0, .2);
}

.rodal-center {
    top: 50%;
    transform: translateY(-50%);
    left: 0;
    right: 0;
    margin: 0 auto;
}

.rodal-bottom {
    left: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

.rodal-top {
    left: 0;
    right: 0;
    top: 0;
    margin: auto;
}

.rodal-left {
    top: 0;
    left: 0;
    bottom: 0;
    margin: auto;
}

.rodal-right {
    top: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

/* -- close button -- */
.rodal-close {
    position: absolute;
    cursor: pointer;
    top: 16px;
    right: 16px;
    width: 16px;
    height: 16px;
}

.rodal-close:before,
.rodal-close:after {
    position: absolute;
    content: '';
    height: 2px;
    width: 100%;
    top: 50%;
    left: 0;
    margin-top: -1px;
    background: #999;
    border-radius: 100%;
    -webkit-transition: background .2s;
    transition: background .2s;
}

.rodal-close:before {
    -webkit-transform: rotate(45deg);
    transform: rotate(45deg);
}

.rodal-close:after {
    -webkit-transform: rotate(-45deg);
    transform: rotate(-45deg);
}

.rodal-close:hover:before,
.rodal-close:hover:after {
    background: #333;
}

/* -- fade -- */
/* @-webkit-keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

@keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

.rodal-fade-enter {
    -webkit-animation: rodal-fade-enter both ease-in;
    animation: rodal-fade-enter both ease-in;
} */

@-webkit-keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

@keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

.rodal-fade-leave {
    -webkit-animation: rodal-fade-leave both ease-out;
    animation: rodal-fade-leave both ease-out;
}

/* -- zoom -- */
@-webkit-keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-enter {
    -webkit-animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-leave {
    -webkit-animation: rodal-zoom-leave both;
    animation: rodal-zoom-leave both;
}

/* -- slideDown -- */
@-webkit-keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-enter {
    -webkit-animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-leave {
    -webkit-animation: rodal-slideDown-leave both;
    animation: rodal-slideDown-leave both;
}

/* -- slideLeft -- */
@-webkit-keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-enter {
    -webkit-animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-leave {
    -webkit-animation: rodal-slideLeft-leave both;
    animation: rodal-slideLeft-leave both;
}

/* -- slideRight -- */
@-webkit-keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-enter {
    -webkit-animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-leave {
    -webkit-animation: rodal-slideRight-leave both;
    animation: rodal-slideRight-leave both;
}

/* -- slideUp -- */
@-webkit-keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-enter {
    -webkit-animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
    animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
}

@-webkit-keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-leave {
    -webkit-animation: rodal-slideUp-leave both;
    animation: rodal-slideUp-leave both;
}

/* -- flip -- */
@-webkit-keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

@keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

.rodal-flip-enter {
    -webkit-animation: rodal-flip-enter both ease-in;
    animation: rodal-flip-enter both ease-in;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

@-webkit-keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

@keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

.rodal-flip-leave {
    -webkit-animation: rodal-flip-leave both;
    animation: rodal-flip-leave both;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

/* -- rotate -- */
@-webkit-keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-enter {
    -webkit-animation: rodal-rotate-enter both;
    animation: rodal-rotate-enter both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

@-webkit-keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-leave {
    -webkit-animation: rodal-rotate-leave both;
    animation: rodal-rotate-leave both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

/* -- door -- */
@-webkit-keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

@keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

.rodal-door-enter {
    -webkit-animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

@keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

.rodal-door-leave {
    -webkit-animation: rodal-door-leave both;
    animation: rodal-door-leave both;
}</style><style>/* -- container -- */
.rodal,
.rodal-mask {
    top: 0;
    left: 0;
    width: 100%;
    height: 100%;
    z-index: 100;
}

.rodal {
    position: fixed;
}

/* -- mask -- */
.rodal-mask {
    position: fixed;
    background: rgba(0, 0, 0, .5);
}

/* -- dialog -- */
.rodal-dialog {
    position: absolute;
    z-index: 101;
    background: #fff;
    border-radius: 3px;
    box-shadow: 0 1px 3px rgba(0, 0, 0, .2);
}

.rodal-center {
    top: 50%;
    transform: translateY(-50%);
    left: 0;
    right: 0;
    margin: 0 auto;
}

.rodal-bottom {
    left: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

.rodal-top {
    left: 0;
    right: 0;
    top: 0;
    margin: auto;
}

.rodal-left {
    top: 0;
    left: 0;
    bottom: 0;
    margin: auto;
}

.rodal-right {
    top: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

/* -- close button -- */
.rodal-close {
    position: absolute;
    cursor: pointer;
    top: 16px;
    right: 16px;
    width: 16px;
    height: 16px;
}

.rodal-close:before,
.rodal-close:after {
    position: absolute;
    content: '';
    height: 2px;
    width: 100%;
    top: 50%;
    left: 0;
    margin-top: -1px;
    background: #999;
    border-radius: 100%;
    -webkit-transition: background .2s;
    transition: background .2s;
}

.rodal-close:before {
    -webkit-transform: rotate(45deg);
    transform: rotate(45deg);
}

.rodal-close:after {
    -webkit-transform: rotate(-45deg);
    transform: rotate(-45deg);
}

.rodal-close:hover:before,
.rodal-close:hover:after {
    background: #333;
}

/* -- fade -- */
/* @-webkit-keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

@keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

.rodal-fade-enter {
    -webkit-animation: rodal-fade-enter both ease-in;
    animation: rodal-fade-enter both ease-in;
} */

@-webkit-keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

@keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

.rodal-fade-leave {
    -webkit-animation: rodal-fade-leave both ease-out;
    animation: rodal-fade-leave both ease-out;
}

/* -- zoom -- */
@-webkit-keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-enter {
    -webkit-animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-leave {
    -webkit-animation: rodal-zoom-leave both;
    animation: rodal-zoom-leave both;
}

/* -- slideDown -- */
@-webkit-keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-enter {
    -webkit-animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-leave {
    -webkit-animation: rodal-slideDown-leave both;
    animation: rodal-slideDown-leave both;
}

/* -- slideLeft -- */
@-webkit-keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-enter {
    -webkit-animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-leave {
    -webkit-animation: rodal-slideLeft-leave both;
    animation: rodal-slideLeft-leave both;
}

/* -- slideRight -- */
@-webkit-keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-enter {
    -webkit-animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-leave {
    -webkit-animation: rodal-slideRight-leave both;
    animation: rodal-slideRight-leave both;
}

/* -- slideUp -- */
@-webkit-keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-enter {
    -webkit-animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
    animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
}

@-webkit-keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-leave {
    -webkit-animation: rodal-slideUp-leave both;
    animation: rodal-slideUp-leave both;
}

/* -- flip -- */
@-webkit-keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

@keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

.rodal-flip-enter {
    -webkit-animation: rodal-flip-enter both ease-in;
    animation: rodal-flip-enter both ease-in;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

@-webkit-keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

@keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

.rodal-flip-leave {
    -webkit-animation: rodal-flip-leave both;
    animation: rodal-flip-leave both;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

/* -- rotate -- */
@-webkit-keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-enter {
    -webkit-animation: rodal-rotate-enter both;
    animation: rodal-rotate-enter both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

@-webkit-keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-leave {
    -webkit-animation: rodal-rotate-leave both;
    animation: rodal-rotate-leave both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

/* -- door -- */
@-webkit-keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

@keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

.rodal-door-enter {
    -webkit-animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

@keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

.rodal-door-leave {
    -webkit-animation: rodal-door-leave both;
    animation: rodal-door-leave both;
}</style>
<!--百度统计-->
<script>
   var _hmt = _hmt || [];
   (function() {
      var hm = document.createElement("script");
      hm.src = "https://hm.baidu.com/hm.js?3c7614be3026469d5a60f41ab30b5082";
      var s = document.getElementsByTagName("script")[0]; 
      s.parentNode.insertBefore(hm, s);
      })();
</script>
</head>
<body class=""><!-- BEGIN WAYBACK TOOLBAR INSERT -->
<style type="text/css">
body {
  margin-top:0 !important;
  padding-top:0 !important;
  /*min-width:800px !important;*/
}
</style>
<script>__wm.rw(0);</script>
<div id="wm-ipp-base" lang="en" style="display: block; direction: ltr;">
</div><div id="wm-ipp-print">The Wayback Machine - https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm</div>
<script type="text/javascript">//<![CDATA[
__wm.bt(675,27,25,2,"web","https://baike.sogou.com/kexue/d10248.htm","20221029033459",1996,"/_static/",["/_static/css/banner-styles.css?v=S1zqJCYt","/_static/css/iconochive.css?v=qtvMKcIJ"], false);
  __wm.rw(1);
//]]></script>
<!-- END WAYBACK TOOLBAR INSERT --><script>window._gtag=window._gtag||{};window._gtag.shouldGrayed = false;if ('a5e79d1be0b141499f2bc1706a8fdd8c') window._gtag.traceId = 'a5e79d1be0b141499f2bc1706a8fdd8c';if ({"illegality":true}) window.userInfo = {"illegality":true};</script><div class="topnavbox"><ul class="topnav"><li><a href="https://web.archive.org/web/20221029033459/https://www.sogou.com/web?query=">网页</a></li><li><a href="https://web.archive.org/web/20221029033459/https://weixin.sogou.com/weixin?p=75351201">微信</a></li><li><a href="https://web.archive.org/web/20221029033459/https://zhihu.sogou.com/zhihu?p=75351218">知乎</a></li><li><a href="https://web.archive.org/web/20221029033459/https://pic.sogou.com/pics?query=">图片</a></li><li><a href="https://web.archive.org/web/20221029033459/https://v.sogou.com/v?query=">视频</a></li><li><a href="https://web.archive.org/web/20221029033459/https://mingyi.sogou.com/">医疗</a></li><li class="cur"><strong>科学</strong></li><li><a href="https://web.archive.org/web/20221029033459/https://hanyu.sogou.com/">汉语</a></li><li><a href="https://web.archive.org/web/20221029033459/https://wenwen.sogou.com/">问问</a></li><li><a href="https://web.archive.org/web/20221029033459/https://www.sogou.com/docs/more.htm">更多<span class="topraquo">»</span></a></li></ul></div><div id="header"><div class="header-wrap"><a class="header-logo" href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue"></a><div class="header-search"><div class="querybox" id="suggBox"><form><input id="searchInput" class="query" type="text" placeholder="搜科学领域专业百科词条" name="query" autocomplete="off" value=""><a href="javascript:;" class="query-search"></a></form></div></div><div class="header-rgt"><span class="btn-header-rgt btn-edit" id="editLemma">创建</span><div class="header-user no-login"></div></div></div></div><div class="fixed-placeholder" style="visibility:none"></div><div id="container" class=""><div class="content lemma-level1"><div class="detail-title" id="abstract-title"><h1>Folding@home</h1><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#!" class="detail-edit">编辑</a></div><div class="section_content" data-id="14994961137139988"><div><p>Folding@home (FAH或F@h)是一个用于疾病研究的分布式计算项目，它模拟蛋白质折叠、计算药物设计和其他类型的分子动力学。该项目利用了志愿者拥有的数千台个人电脑的闲置处理资源，并且已经在他们的系统上安装了软件。其主要目的是确定蛋白质的折叠机制，即蛋白质到达其最终三维结构的过程，并检查蛋白质错误折叠的原因。这具有重要的学术意义，对阿尔茨海默氏病、亨廷顿氏病、多种癌症以及其他疾病的医学研究具有重要意义。在较小的程度上，Folding@home还试图预测蛋白质的最终结构，并确定其他分子如何与之相互作用，这一研究在药物设计中有一定的应用。Folding @home由斯坦福大学的潘德（Pande）实验室在维贾伊·潘德（Prof. Vijay Pande）教授的指导下开发和运营，并由世界各地的各种科学机构和研究实验室共享。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_1" class="kx_ref">[1]</a></sup> </p>
<p>该项目率先将图形处理单元(GPU)、PlayStation 3s、消息传递接口(用于多核处理器上的计算)和一些SNOY-Xperia智能手机用于分布式计算和科学研究。该项目使用统计模拟方法，这是传统计算方法的一种范式转变。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_2" class="kx_ref">[2]</a></sup>作为客户机-服务器模型网络体系结构的一部分，志愿者机器各自接收模拟(工作单元)的片段并完成它们，随后将其返回到项目的数据库服务器，在那里单元被编译成一个整体模拟。志愿者可以在Folding @home网站上实时查询他们的贡献，这使得志愿者之间产生竞争力，从而鼓励他们长期参与。 </p>
<p>Folding @home是世界上最快的计算系统之一，截至2019年6月，其运算速度约为98.7千兆位。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_3" class="kx_ref">[3]</a></sup>其大规模的高性能计算网络使得研究人员能够运行计算高成本的原子级蛋白质折叠模拟，比以前长几千倍。自2000年10月1日成立以来，潘德实验室已经基于Folding @home发表了206篇科学研究论文。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_4" class="kx_ref">[4]</a></sup> 该项目的仿真模拟结果与实验结果非常吻合。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_5" class="kx_ref">[5]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_3" class="kx_ref">[3]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_6" class="kx_ref">[6]</a></sup> </p></div></div><div id="catalog"><h2 class="title2">目录<a href="javascript:" class="detail-edit">编辑</a></h2><div class="catalog_wrap" style=""><ul class="catalog_list col3"><li><span class="order">1</span><a href="javascript:" data-level="1" data-id="14994961137139990">项目意义</a></li><li><span class="order">2</span><a href="javascript:" data-level="1" data-id="14994961153917196">生物医学研究中的应用实例</a></li><li class="secondary_catalog"><span>2.1 </span><a href="javascript:" data-id="14994961153917196">阿尔茨海默病</a></li><li class="secondary_catalog"><span>2.2 </span><a href="javascript:" data-id="14994961153917196">亨廷顿氏病</a></li><li class="secondary_catalog"><span>2.3 </span><a href="javascript:" data-id="14994961153917196">癌症</a></li><li class="secondary_catalog"><span>2.4 </span><a href="javascript:" data-id="14994961153917196">成骨不全</a></li><li class="secondary_catalog"><span>2.5 </span><a href="javascript:" data-id="14994961153917196">病毒</a></li></ul><ul class="catalog_list col3"><li class="secondary_catalog"><span>2.6 </span><a href="javascript:" data-id="14994961153917196">药物设计</a></li><li><span class="order">3</span><a href="javascript:" data-level="1" data-id="14994961153917197">生物医学研究的潜在应用</a></li><li class="secondary_catalog"><span>3.1 </span><a href="javascript:" data-id="14994961153917197">朊病毒疾病</a></li><li><span class="order">4</span><a href="javascript:" data-level="1" data-id="14994961153917198">参与模式</a></li><li class="secondary_catalog"><span>4.1 </span><a href="javascript:" data-id="14994961153917198">性能</a></li><li class="secondary_catalog"><span>4.2 </span><a href="javascript:" data-id="14994961153917198">积分</a></li><li><span class="order">5</span><a href="javascript:" data-level="1" data-id="14994961153917199">软件</a></li></ul><ul class="catalog_list col3"><li class="secondary_catalog"><span>5.1 </span><a href="javascript:" data-id="14994961153917199">工作单位</a></li><li class="secondary_catalog"><span>5.2 </span><a href="javascript:" data-id="14994961153917199">核心</a></li><li class="secondary_catalog"><span>5.3 </span><a href="javascript:" data-id="14994961153917199">客户</a></li><li><span class="order">6</span><a href="javascript:" data-level="1" data-id="14994961170694412">与其他分子模拟器的比较</a></li><li><span class="order">7</span><a href="javascript:" data-level="1" data-id="14994961170694413">笔记</a></li><li><span class="order">8</span><a href="javascript:" data-level="1" data-id="references">参考文献</a></li></ul></div></div><div id="paragraphs"><div><div id="par_14994961137139990"><h2 class="title">1 项目意义<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p></p><p></p><div class="text_img ed_imgfloat_right">
            <a class="ed_image_link lazyLoad" data-src="https://img04.sogoucdn.com/app/a/200698/sogou_science_11046" data-bigsrc="" title="点击查看大图" href="javascript:" data-observer="true"></a>
            <div class="text_img_title">折叠前后的一条蛋白质。它一开始是不稳定的随机蜷曲形态，折叠之后变为它的天然态。</div>   
        </div> <p></p><p></p> 
<p>蛋白质是许多生物细胞功能的重要组成部分，几乎参与生物细胞内的所有过程。它们通常扮演催化酶的角色进行生化反应，包括细胞信号、分子运输和细胞调节。作为结构元素，一些蛋白质充当细胞的骨架和抗体，而其他蛋白质参与免疫系统。在蛋白质发挥这些作用之前，它必须折叠成一个功能性的三维结构，这一过程经常是自发的，并依赖于其氨基酸序列内的相互作用以及氨基酸与其周围环境的相互作用。蛋白质折叠是通过寻找蛋白质最有利的构象（即其天然状态）来驱动进行的。因此，理解蛋白质折叠对于理解蛋白质的功能和工作方式至关重要，被认为是计算生物学的圣杯。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_7" class="kx_ref">[7]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_8" class="kx_ref">[8]</a></sup>尽管折叠发生在拥挤的细胞环境中，但它通常进展顺利。然而，由于蛋白质的化学性质或其他因素，蛋白质可能会错误折叠，也就是说，沿着错误的路径折叠并最终畸形。除非细胞机制能够破坏或重新折叠错误折叠的蛋白质，否则它们会随后聚集并导致各种衰弱性疾病。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_9" class="kx_ref">[9]</a></sup>研究这些过程的实验可能在范围和原子细节上受到限制，这就导致科学家使用基于物理的计算模型，这些模型在补充实验时，会试图提供蛋白质折叠、错误折叠和聚集的更完整的图像。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_10" class="kx_ref">[10]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_11" class="kx_ref">[11]</a></sup> </p>
<p>由于蛋白质构象或构型空间的复杂性(蛋白质可以采取的一组可能形状)，以及计算能力的限制，全原子分子动力学模拟在它们可以研究的时间尺度上受到严重限制。虽然大多数蛋白质通常折叠在毫秒量级，<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_10" class="kx_ref">[10]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_12" class="kx_ref">[12]</a></sup>但在2010年之前，模拟只能达到纳秒到微秒的时间尺度。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_5" class="kx_ref">[5]</a></sup>通用超级计算机已经被用来模拟蛋白质折叠，但是这种系统是昂贵的，并且通常由许多研究小组共享。此外，因为动力学模型中的计算是连续发生的，所以传统分子模拟对这些结构的强缩放是异常困难的。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_13" class="kx_ref">[13]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_14" class="kx_ref">[14]</a></sup>此外，由于蛋白质折叠是一个随机过程，可以随时间统计变化，因此这是一个在计算上具有挑战性的综合视图折叠过程。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_15" class="kx_ref">[15]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_16" class="kx_ref">[16]</a></sup> </p> 
<p></p><p></p><div class="text_img ed_imgfloat_right">
            <a class="ed_image_link lazyLoad" data-src="https://img04.sogoucdn.com/app/a/200698/sogou_science_11047" data-bigsrc="" title="点击查看大图" href="javascript:" data-observer="true"></a>
            <div class="text_img_title">如图所示，Folding@home使用马尔科夫模型来模拟蛋白质从其初始的随机蜷曲状态（左）凝结成其三维天然态（右）过程中可能的形状与折叠方式。</div>   
        </div> <p></p><p></p> 
<p>蛋白质折叠不是一步完成的。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_9" class="kx_ref">[9]</a></sup>相反，蛋白质花费大部分折叠时间，在某些情况下接近96%，<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_17" class="kx_ref">[17]</a></sup>在各种中间构象状态下等待，每种状态都是蛋白质能量景观中的局部热力学自由能的最小值。通过一个被称为自适应采样的过程，这些构象被Folding@home用作一组模拟轨迹的起点。随着模拟发现更多的构象，轨迹从这些构象重新开始，并从这个循环过程中逐渐创建马尔可夫状态模型。MSM是离散时间主方程模型，它将生物分子的构象和能量景观描述为一组不同的结构以及它们之间的短暂跃迁。自适应采样马尔可夫状态模型方法显著提高了仿真的效率，因为它避免了局部能量最小值本身内部的计算，并且适合分布式计算(包括在GPUGRID上)，因为它允许短的、独立的仿真轨迹的统计聚集。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_18" class="kx_ref">[18]</a></sup>构建马尔可夫状态模型所需的时间与并行模拟运行的数量成反比，即可用处理器的数量。换句话说，它实现了线性并行化，在整个串行计算时间中导致大约四个数量级的放大。一个完整的质谱可能包含成千上万个来自蛋白质相空间的样品状态(蛋白质可以采取的所有构象)以及它们之间的转换。该模型说明了折叠事件和路径(即路线)，研究人员随后可以使用动力学聚类来查看本应高度详细的模型的粗粒度表示。他们可以使用这些质谱来揭示蛋白质是如何错误折叠的，并定量比较模拟和实验。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_2" class="kx_ref">[2]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_15" class="kx_ref">[15]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_19" class="kx_ref">[19]</a></sup> </p>
<p>2000年至2010年间，蛋白质Folding@home研究的长度增加了四倍，而蛋白质折叠模拟的时间尺度增加了六个数量级。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_20" class="kx_ref">[20]</a></sup> 2002年，Folding@home使用马尔可夫状态模型在几个月的时间里完成了大约一百万个CPU日的模拟，<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_6" class="kx_ref">[6]</a></sup>2011年，MSMs并行化了另一个需要总计1000万个CPU小时计算的模拟。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_21" class="kx_ref">[21]</a></sup>2010年1月，Folding@home使用MSMs模拟慢折叠的32残基NTL9蛋白的动力学过程，时间跨度为1.52毫秒，这与实验性折叠速率预测一致，但比以前的预测长一千倍。该模型由许多单独的轨迹组成，每条轨迹都短了两个数量级，为蛋白质的能量景观提供了前所未有的细节。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_2" class="kx_ref">[2]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_5" class="kx_ref">[5]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_22" class="kx_ref">[22]</a></sup> 2010年，Folding@home研究员格雷戈里·鲍曼(Gregory Bowman)因开发开源MSMBuilder软件和实现理论与实验之间的定量一致而获得美国化学学会颁发的托马斯·库恩范式转换奖（Thomas Kuhn Paradigm Shift Award）。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_23" class="kx_ref">[23]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_24" class="kx_ref">[24]</a></sup> 由于他的工作，潘德获得了2012年迈克尔和凯特·巴恩青年研究员奖（Michael and Kate Bárány Award），因为他“开发了定义场和改变场的计算方法来产生蛋白质和核糖核酸折叠的领先理论模型”，<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_25" class="kx_ref">[25]</a></sup> 和2006年欧文·西格勒青年研究员奖(Irving Sigal Young Investigator Award)，这两项奖“激发了对整体和单分子测量意义的重新审视，使潘德博士在模拟方法学方面做出了开创性贡献。”<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_26" class="kx_ref">[26]</a></sup> </p></div></div><div id="par_14994961153917196"><h2 class="title">2 生物医学研究中的应用实例<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>蛋白质错误折叠会导致多种疾病，包括阿尔茨海默病、癌症、克罗伊茨费尔特-雅各布病、囊性纤维化、亨廷顿氏病、镰状细胞贫血和二型糖尿病。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_9" class="kx_ref">[9]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_27" class="kx_ref">[27]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_28" class="kx_ref">[28]</a></sup>艾滋病毒和流感等病毒引起的细胞感染也涉及细胞膜折叠事件。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_29" class="kx_ref">[29]</a></sup>一旦蛋白质错误折叠得到更好的理解，就可以开发出增强细胞调节蛋白质折叠的天然能力的疗法。这种疗法包括使用工程分子来改变给定蛋白质的产量，帮助破坏错误折叠的蛋白质，或帮助折叠过程。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_30" class="kx_ref">[30]</a></sup>计算分子建模和实验分析的结合有可能从根本上塑造分子医学的未来和治疗学的合理设计，<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_11" class="kx_ref">[11]</a></sup>例如加快和降低药物发现的成本。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_31" class="kx_ref">[31]</a></sup> Folding@home的前五年的目标是在理解折叠方面取得进展，而当前的目标是理解错误折叠和相关疾病，尤其是阿尔茨海默氏症。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_32" class="kx_ref">[32]</a></sup> </p>
<p>在Folding@home上运行的模拟与实验室实验结合使用，<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_15" class="kx_ref">[15]</a></sup>但是研究人员可以用它们来研究体外折叠与天然细胞环境中折叠有何不同。这有利于研究折叠、错误折叠及其与难以通过实验观察到的疾病的关系。例如，2011年，Folding@home模拟核糖体出口通道内的蛋白质折叠，帮助科学家更好地理解自然限制和拥挤可能如何影响折叠过程。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_33" class="kx_ref">[33]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_34" class="kx_ref">[34]</a></sup>此外，科学家通常使用化学变性剂将蛋白质从稳定的天然状态中展开。但是，通常不知道变性剂如何影响蛋白质的重折叠，并且很难通过实验确定这些变性状态是否包含可能影响折叠行为的残余结构。2010年，Folding@home利用图形处理器模拟蛋白质L的展开状态，并预测其折叠速率，与实验结果非常吻合。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_35" class="kx_ref">[35]</a></sup> </p>
<p>潘德实验室是斯坦福大学的一部分，斯坦福大学是一个非营利实体，不出售Folding@home产生的结果。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_36" class="kx_ref">[36]</a></sup>该项目的大型数据集可免费供其他研究人员索取使用，有些数据集可从Folding@home网站上访问。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_37" class="kx_ref">[37]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_38" class="kx_ref">[38]</a></sup>潘德实验室与其他分子动力学系统合作，如的蓝色基因超级计算机，<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_39" class="kx_ref">[39]</a></sup> 他们与其他研究人员共享Folding@home的关键软件，因此使Folding@home受益的算法可能有助于其他科学领域。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_37" class="kx_ref">[37]</a></sup>2011年，他们发布了开源哥白尼软件（open-source Copernicus software），该软件基于Folding@home的MSM和其他并行化方法，旨在提高大型计算机集群或超级计算机上分子模拟的效率和规模。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_40" class="kx_ref">[40]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_41" class="kx_ref">[41]</a></sup> Folding@home的所有科学发现的摘要在出版后都张贴在Folding@home的网站上。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_4" class="kx_ref">[4]</a></sup> </p> 
<h3>2.1 <span>阿尔茨海默病</span></h3> 
<p></p><p></p><div class="text_img ed_imgfloat_right">
            <a class="ed_image_link lazyLoad" data-src="https://img03.sogoucdn.com/app/a/200698/sogou_science_11048" data-bigsrc="" title="点击查看大图" href="javascript:" data-observer="true"></a>
            <div class="text_img_title" style="text-align: center;"></div>   
        </div><div class="text_img ed_imgfloat_right">
            <a class="ed_image_link lazyLoad" data-src="https://img02.sogoucdn.com/app/a/200698/sogou_science_11049" data-bigsrc="" title="点击查看大图" href="javascript:" data-observer="true"></a>
            <div class="text_img_title" style="text-align: center;"></div>   
        </div><div class="text_img ed_imgfloat_right">
            <a class="ed_image_link lazyLoad" data-src="https://img04.sogoucdn.com/app/a/200698/sogou_science_11050" data-bigsrc="" title="点击查看大图" href="javascript:" data-observer="true"></a>
            <div class="text_img_title" style="text-align: center;"></div>   
        </div><p></p><p>阿尔茨海默症与大脑中β-淀粉样蛋白片段的聚集有关（右图）。研究人员使用Folding@home模拟这一聚集过程，以更好的了解这种疾病的起因。</p><p></p><p></p> 
<p>阿尔茨海默病是一种不可治愈的神经退行性疾病，最常影响老年人，占所有痴呆症病例的一半以上。其确切原因尚不清楚，但这种疾病被确定为蛋白质错误折叠疾病。阿尔茨海默病与淀粉样β肽的毒性聚集有关，该聚集是由淀粉样β错误折叠和与其他淀粉样β肽聚集在一起引起的。然后，这些β-淀粉样蛋白聚集成明显更大的老年斑，这是阿尔茨海默病的病理标志。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_42" class="kx_ref">[42]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_43" class="kx_ref">[43]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_44" class="kx_ref">[44]</a></sup> 由于这些聚集体的异质性，诸如x光结晶学和核磁共振等实验方法很难表征它们的结构。此外，由于β聚集的大小和复杂性，原子模拟在计算上要求很高。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_45" class="kx_ref">[45]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_46" class="kx_ref">[46]</a></sup> </p>
<p>根据Naeem博士和法兹利（Fazili）博士在一篇文献综述文章中的观点，预防α-β聚集是开发阿尔茨海默病治疗药物的一种有前途的方法。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_47" class="kx_ref">[47]</a></sup>2008年，Folding@home在几十秒的时间尺度内以原子细节模拟了β聚集的动力学。先前的研究只能模拟大约10微秒。Folding@home能够模拟比以前长六个数量级的β折叠。研究人员利用这项研究的结果来鉴定β发夹，它是结构内分子相互作用的主要来源。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_48" class="kx_ref">[48]</a></sup>这项研究有助于潘德实验室为未来的聚集研究做准备，并为进一步研究寻找一种可以稳定聚集过程的小肽做准备。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_45" class="kx_ref">[45]</a></sup> </p>
<p>2008年12月，Folding@home发现了几种小的候选药物，它们似乎抑制了β-淀粉样聚集体的毒性。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_49" class="kx_ref">[49]</a></sup>2010年，与蛋白质折叠机械中心密切合作，这些药物引线开始在生物组织上测试。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_28" class="kx_ref">[28]</a></sup>2011年，Folding@home完成了几个α-β突变的模拟，这些突变似乎稳定了聚集物的形成，这可能有助于该疾病治疗药物的开发，并极大地有助于β-低聚物的实验核磁共振波谱研究。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_46" class="kx_ref">[46]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_50" class="kx_ref">[50]</a></sup> 同年，“Folding@home”开始模拟各种β淀粉样蛋白片段，以确定各种天然酶如何影响β淀粉样蛋白的结构和折叠。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_51" class="kx_ref">[51]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_52" class="kx_ref">[52]</a></sup> </p> 
<h3>2.2 <span>亨廷顿氏病</span></h3> 
<p>亨廷顿氏病是一种神经退行性遗传疾病，与蛋白质错误折叠和聚集有关。亨廷顿蛋白氮末端谷氨酰胺氨基酸的过量重复导致聚集，尽管重复的行为还没有完全被理解，但它确实导致了与疾病相关的认知下降。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_53" class="kx_ref">[53]</a></sup>同其他集料一样，很难通过实验确定其结构。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_54" class="kx_ref">[54]</a></sup> 科学家正在使用Folding@home来研究亨廷顿蛋白聚集体的结构，并预测其形成方式，协助合理的药物设计方法来阻止该聚集体的形成。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_28" class="kx_ref">[28]</a></sup>亨廷顿蛋白的N17片段加速了这种聚集，虽然已经提出了几种机制，但其在这一过程中的实质性作用仍大部分未知。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_55" class="kx_ref">[55]</a></sup> Folding@home模拟了这一片段和其他片段，以阐明它们在疾病中的作用。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_56" class="kx_ref">[56]</a></sup>自2008年以来，其针对阿尔茨海默病的药物设计方法已经应用于亨廷顿氏症。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_28" class="kx_ref">[28]</a></sup> </p> 
<h3>2.3 <span>癌症</span></h3> 
<p>一半以上的已知癌症涉及p53突变，p53是一种存在于每个细胞中的肿瘤抑制蛋白，它调节细胞周期，并在脱氧核糖核酸受损时发出细胞死亡信号。p53的特定突变可以破坏这些功能，允许异常细胞不受抑制地继续生长，导致肿瘤的发展。对这些突变的分析有助于解释p53相关癌症的根本原因。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_57" class="kx_ref">[57]</a></sup> 2004年，Folding@home被用于在全原子模拟水中进行p53蛋白二聚体重折叠的第一次分子动力学研究。模拟的结果与实验观察一致，并对以前无法获得的二聚体的再折叠提供了参考。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_58" class="kx_ref">[58]</a></sup>这是第一份来自分布式计算项目的关于癌症的同行评议出版物。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_59" class="kx_ref">[59]</a></sup> 第二年，“Folding@home”提供了一种新的方法来鉴定对特定蛋白质的稳定性有着至关重要的氨基酸，然后用于研究p53的突变。该方法在识别致癌突变方面相当成功，并通过实验测量确定了特定突变的影响。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_60" class="kx_ref">[60]</a></sup> </p>
<p>Folding@home也用于研究蛋白质伴侣，<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_28" class="kx_ref">[28]</a></sup>热休克蛋白，在细胞内的极端环境中环境中，通过协助折叠其他蛋白质对细胞的存活发挥重要作用。快速生长的癌细胞依赖于特定的伴侣，一些伴侣在化疗耐药性中起着关键作用。对这些特定伴侣的抑制被视为有效化疗药物或减少癌症扩散的潜在作用模式。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_61" class="kx_ref">[61]</a></sup>潘德实验室希望通过“Folding@home”并与蛋白质折叠机械中心密切合作，找到一种抑制参与癌细胞的伴侣的药物。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_62" class="kx_ref">[62]</a></sup>研究人员也在使用Folding@home来研究其他与癌症相关的分子，如Src激酶和某些形式的植入同源域:一种可能与包括癌症在内的许多疾病有关的大蛋白质。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_63" class="kx_ref">[63]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_64" class="kx_ref">[64]</a></sup> 2011年，Folding@home开始模拟小打结蛋白EETI的动力学，这种蛋白可以通过结合癌细胞的表面受体在成像扫描中识别癌症。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_65" class="kx_ref">[65]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_66" class="kx_ref">[66]</a></sup> </p>
<p>白细胞介素2(il2)是一种帮助免疫系统的T细胞攻击病原体和肿瘤的蛋白质。然而，由于严重的副作用，如肺水肿，其作为癌症治疗的用途受到限制。白细胞介素-2与这些肺细胞的结合不同于与T细胞的结合，因此白细胞介素-2的研究包括了解这些结合机制之间的差异。2012年，Folding@home协助发现了一种白细胞介素-2的突变形式，这种突变形式的白细胞介素-2在免疫系统中的作用是原来的三百倍，但副作用更少。在实验中，这种改变的形式在阻止肿瘤生长方面明显优于天然白细胞介素-2。制药公司已经表达了对这种突变分子的兴趣，国家卫生研究院正在用多种肿瘤模型对其进行测试，试图加速其作为治疗药物的发展。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_67" class="kx_ref">[67]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_68" class="kx_ref">[68]</a></sup> </p> 
<h3>2.4 <span>成骨不全</span></h3> 
<p>成骨不全，被称为脆性骨病，是一种不可治愈的遗传性骨病，可以致命。患有这种疾病的人不能制造功能性结缔组织。这最常见的原因是一型胶原蛋白的突变，<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_69" class="kx_ref">[69]</a></sup> 该蛋白具有多种结构功能，是哺乳动物中最丰富的蛋白质。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_70" class="kx_ref">[70]</a></sup>这种突变导致胶原蛋白的三螺旋结构变形，如果不自然破坏，就会导致骨组织异常和弱化。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_71" class="kx_ref">[71]</a></sup>2005年，Folding@home测试了一种新的量子力学方法，这种方法对以前的模拟方法进行了改进，可能对未来胶原蛋白的计算研究有用。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_72" class="kx_ref">[72]</a></sup>尽管研究人员已经使用Folding@home来研究胶原折叠和错误折叠，但与阿尔茨海默氏症和亨廷顿氏症的研究相比，这种兴趣只是一个试点项目。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_28" class="kx_ref">[28]</a></sup> </p> 
<h3>2.5 <span>病毒</span></h3> 
<p>Folding@home正在协助防止流感和艾滋病毒等病毒识别和进入生物细胞的研究。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_28" class="kx_ref">[28]</a></sup>2011年，“Folding@home”开始模拟核糖核酸酶H的动态，核糖核酸酶H是艾滋病毒的一个关键成分，试图设计药物使其失活。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_73" class="kx_ref">[73]</a></sup> Folding@home也被用于研究膜融合，膜融合是病毒感染的重要事件，具有广泛的生物学功能。这种融合涉及病毒融合蛋白的构象变化和蛋白对接，<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_29" class="kx_ref">[29]</a></sup>但融合背后的确切分子机制仍大部分未知。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_74" class="kx_ref">[74]</a></sup>聚变事件可能由50多万个原子相互作用数百微秒组成。这种复杂性将典型的计算机模拟限制在几十纳秒内大约一万个原子:相差几个数量级。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_48" class="kx_ref">[48]</a></sup>预测膜融合机制的模型的开发将有助于科学理解如何用抗病毒药物靶向该过程。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_75" class="kx_ref">[75]</a></sup>2006年，科学家应用马尔可夫状态模型和Folding@home网络发现了两条融合途径，并获得了其他的机制上的见解。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_48" class="kx_ref">[48]</a></sup> </p>
<p>2007年，在折叠小细胞之家囊泡的详细模拟之后，潘德实验室引入了一种新的计算方法来测量融合过程中其结构变化的拓扑结构。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_76" class="kx_ref">[76]</a></sup>2009年，研究人员使用折叠@home来研究流感血凝素的突变，流感血凝素是一种将病毒附着于宿主细胞并帮助病毒进入的蛋白质。血凝素的突变影响蛋白质与宿主细胞表面受体分子的结合程度，这决定了病毒株对宿主有机体的感染性。血凝素突变作用的知识有助于抗病毒药物的开发。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_77" class="kx_ref">[77]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_78" class="kx_ref">[78]</a></sup>截至2012年，Folding@home继续模拟血凝素的折叠和相互作用，补充了弗吉尼亚大学的实验研究。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_28" class="kx_ref">[28]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_79" class="kx_ref">[79]</a></sup> </p> 
<h3>2.6 <span>药物设计</span></h3> 
<p>药物通过结合目标分子上的特定位置而发挥作用，并引起一些期望的变化，例如使目标失效或引起构象变化。理想情况下，一种药物应该非常特异地发挥作用，并且只与其靶标结合，而不干扰其他生物功能。然而，很难精确确定两个分子结合的位置和紧密程度。由于计算能力的限制，目前的电子方法通常必须以速度换取准确性；例如使用快速蛋白质对接方法，而不是计算成本高的自由能计算。Folding@home的计算性能允许研究人员使用这两种方法，并评估它们的效率和可靠性。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_32" class="kx_ref">[32]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_80" class="kx_ref">[80]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_81" class="kx_ref">[81]</a></sup>计算机辅助药物设计有可能加快和降低药物发现的成本。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_31" class="kx_ref">[31]</a></sup>2010年，Folding@home使用质谱和自由能计算来预测绒毛蛋白的天然状态，其与通过x光晶体学实验确定的晶体结构的均方根偏差(RMSD)在1.8埃以内。这种准确性对未来的蛋白质结构包括内在非结构蛋白质的预测方法有影响。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_48" class="kx_ref">[48]</a></sup>科学家利用折叠技术研究耐药性，研究万可霉素和β-内酰胺酶。万可霉素是最后一种抗生素，β-内酰胺酶是一种能分解青霉素等抗生素的蛋白质。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_82" class="kx_ref">[82]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_83" class="kx_ref">[83]</a></sup> </p>
<p>化学活性发生在蛋白质的活性部位。传统的药物设计方法包括紧密结合该位点并阻断其活性，假设靶蛋白存在于一个刚性结构中。然而，这种方法只适用于大约15%的蛋白质。蛋白质含有变构位点，当被小分子结合时，可以改变蛋白质的构象，最终影响蛋白质的活性。这些位点是有吸引力的药物靶点，但是定位它们的计算成本非常高。2012年，Folding@home和MSMs被用于鉴定三种医学相关蛋白质的变构位点:β-内酰胺酶、白细胞介素-2和核糖核酸酶h。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_83" class="kx_ref">[83]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_84" class="kx_ref">[84]</a></sup> </p>
<p>大约一半的已知抗生素会干扰细菌核糖体的工作，核糖体是一种大型复杂的生化机器，通过将信使核糖核酸转化为蛋白质来进行蛋白质生物合成。大环内酯类抗生素堵塞核糖体的出口通道，阻止必需细菌蛋白质的合成。2007年，潘德实验室获得了研究和设计新抗生素的资助。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_28" class="kx_ref">[28]</a></sup>2008年，他们使用Folding@home来研究这个隧道的内部以及特定分子可能如何影响它。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_85" class="kx_ref">[85]</a></sup>核糖体的完整结构直到2011年才被确定，Folding@home也模拟了核糖体蛋白，因为它们的许多功能仍大部分未知。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_86" class="kx_ref">[86]</a></sup> </p></div></div><div id="par_14994961153917197"><h2 class="title">3 生物医学研究的潜在应用<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>有更多蛋白质错误折叠促进的疾病可以从Folding@home中受益，从而识别错误折叠的蛋白质结构或错误折叠动力学，并有助于未来的药物设计。通常致命的朊病毒疾病是最重要的疾病之一。 </p> 
<h3>3.1 <span>朊病毒疾病</span></h3> 
<p>朊病毒(PrP)是一种广泛存在于真核细胞中的跨膜蛋白。在哺乳动物中，它在中枢神经系统中更丰富。尽管它的功能未知，但它在物种间的高度保守性表明它在细胞功能中起着重要作用。从正常朊病毒蛋白(PrPc，代表细胞)到引起同种型PrPSc(代表原型朊病毒疾病——羊瘙痒病)的疾病的构象变化，导致了一系列被统称为传染性海绵状脑病的疾病，包括牛海绵状脑病、克罗伊茨费尔特-雅各布病(CJD)和人类致命失眠、鹿家族慢性消耗性疾病(CWD)。构象变化被广泛认为是蛋白质错误折叠的结果。TSEs与其他蛋白质错误折叠疾病的区别在于其可传播性。传染性PrPSc的“播种”，无论是自发产生的、遗传的还是通过暴露于受污染的组织而获得的，<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_87" class="kx_ref">[87]</a></sup>都可以引起将正常PrPSc转化为由PrPsC组成的纤维聚集体或淀粉样斑块的连锁反应。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_88" class="kx_ref">[88]</a></sup> </p>
<p>PrPSc的分子结构由于其聚集性质尚未完全表征。对蛋白质错误折叠的机制和动力学都知之甚少。利用PrPc的已知结构和下文所述的体外和体内研究结果，Folding@home可能有助于阐明PrPSc是如何形成的，以及感染性蛋白质是如何排列自身形成原纤维和淀粉样斑块的，绕过了纯化PrPSc或溶解聚集体的要求。 </p>
<p>PrPc已从膜上酶促分离并纯化，其结构用结构表征技术如核磁共振波谱和x光结晶学研究。翻译后PrPc在小鼠体内有231个氨基酸(aa)。该分子由一个长达aa残基121的长而非结构化的氨基末端区域和一个结构化的羧基末端结构域组成。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_88" class="kx_ref">[88]</a></sup>这个球状结构域包含两条短片状反平行β链(鼠PrPc中aa 128-130和aa 160-162)和三条α螺旋(螺旋ⅰ:aa 143-153；螺旋二:aa 171至192；螺旋ⅲ:鼠PrPc中的aa 199至226)，<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_89" class="kx_ref">[89]</a></sup>螺旋ⅱ和ⅲ是反平行取向的，并通过短环连接。它们的结构稳定性由平行于两个成片β链的二硫键支撑。这些α螺旋和β片形成PrPc球状畴的刚性核。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_90" class="kx_ref">[90]</a></sup> </p>
<p>引起PrPSc的疾病是蛋白酶K抗性的和不溶性的。试图从受感染动物的大脑中纯化它总是产生不均匀的混合物和聚集状态，这些不适合核磁共振波谱或x光晶体学的表征。然而，人们普遍认为PrPSc比正常PrPsC含有更高百分比的紧密堆积的β-片，这使得蛋白不溶并对蛋白酶有抗性。使用冷冻电子显微镜技术和基于相似普通蛋白质结构的结构建模技术，已经发现PrPSc在aa 81-95至aa 171区域包含β-片，而羧基末端结构被假定保留，在正常PrPc中保留二硫键连接的α-螺旋构象。这些β片形成平行的左旋β螺旋。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_88" class="kx_ref">[88]</a></sup>三种PrPSc分子被认为形成了一个初级单位，因此为所谓的羊瘙痒病相关原纤维奠定了基础。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_91" class="kx_ref">[91]</a></sup>催化活性取决于颗粒的大小。仅由14-28个PrPc分子组成的PrPSc颗粒显示出最高的传染性和转化率。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_92" class="kx_ref">[92]</a></sup> </p>
<p>尽管纯化和鉴定PrPSc有困难，但是从PrPSc的已知分子结构和使用转基因小鼠和N-末端缺失，<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_93" class="kx_ref">[93]</a></sup> 可以推断导致致病PrPSc的蛋白质错误折叠的潜在原因， Folding@home在证实这些方面可能有很大价值。研究发现朊病毒蛋白的一级和二级结构对转化都有重要意义。 </p>
<p>朊病毒蛋白基因(PRNP)有20多个突变，已知与人类TSEs的遗传形式相关或直接相关，表明PrPc的某些位置(可能在羧基结构域内）<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_89" class="kx_ref">[89]</a></sup> 的单个氨基酸可以影响TSEs的易感性。 </p>
<p>PrPc翻译后氨基末端区域由残基23-120组成，这些残基构成全长成熟PrPc氨基序列的近一半。氨基末端区域有两个部分可能影响转化。首先，残基52-90包含八肽重复(5次)区域，其可能影响初始结合(通过八肽重复)以及通过aa 108-124的第二部分的实际转化。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_94" class="kx_ref">[94]</a></sup>高度疏水的AGAAAAGA位于aa残基113和120之间，被描述为推定的聚集位点，<sup>[98]</sup>尽管该序列需要其侧翼部分形成原纤维聚集体。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_95" class="kx_ref">[95]</a></sup> </p>
<p>在羧基球状结构域中，<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_90" class="kx_ref">[90]</a></sup>在三个螺旋中，研究表明螺旋ⅱ具有显著更高的β链构象倾向。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_96" class="kx_ref">[96]</a></sup>由于残基114-125(非结构化氮末端链的一部分)之间的高构象柔性和螺旋ⅱ的高β链倾向，只有环境条件或相互作用的适度变化才足以诱导PrPc的错误折叠和随后的原纤维形成。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_88" class="kx_ref">[88]</a></sup> </p>
<p>PrPc核磁共振结构的其他研究表明，这些残基(~ 108–189)包含大部分折叠结构域，包括β链、前两个α螺旋和连接它们的环/转区，但不包括螺旋ⅲ。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_93" class="kx_ref">[93]</a></sup>PrPC本身的环路/转弯结构中的微小变化在转换中也很重要。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_97" class="kx_ref">[97]</a></sup>在另一项研究中，里克等人（Riek et al）表明，环区上游的两个小β链区充当了PrPc中的环/匝和α螺旋结构向β片构象转化的成核位点。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_89" class="kx_ref">[89]</a></sup> </p>
<p>转换的能量阈值不一定很高。折叠稳定性，即球状蛋白质在其环境中的自由能在一个或两个氢键的范围内，因此允许向异构体的转变而不需要高的转变能。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_88" class="kx_ref">[88]</a></sup> </p>
<p>从PrPc分子之间的相互作用来看，疏水相互作用在β-片的形成中起着至关重要的作用，β-片是PrPc的标志，因为β-片使多肽链片段非常接近。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_98" class="kx_ref">[98]</a></sup>事实上，库兹涅佐夫和拉科夫斯基（Kutznetsov and Rackovsky） <sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_99" class="kx_ref">[99]</a></sup>表明人类PrPc中促进疾病的突变具有显著增加局部疏水性的统计学趋势。 </p>
<p>体外实验表明，错误折叠的动力学有一个初始滞后期，随后是纤维形成的快速生长期。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_100" class="kx_ref">[100]</a></sup> PrPC在最终成为淀粉样原纤维的一部分之前，可能会经历一些部分展开或降解的中间状态。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_88" class="kx_ref">[88]</a></sup> </p></div></div><div id="par_14994961153917198"><h2 class="title">4 参与模式<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>像其他分布式计算项目一样，Folding@home是一个在线公民科学项目。在这些项目中，非专家贡献计算机处理能力或帮助分析专业科学家产生的数据。这些项目的参与者在促进研究方面发挥着宝贵的作用，只获得很少或不是很明显的回报。 </p>
<p>对公民科学家的动机进行了研究，这些研究中的大多数发现，参与者之所以有动机参与是因为利他主义的原因，也就是说，他们希望帮助科学家并为他们的研究进步做出贡献。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_101" class="kx_ref">[101]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_102" class="kx_ref">[102]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_103" class="kx_ref">[103]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_104" class="kx_ref">[104]</a></sup>公民科学的许多参与者对研究主题有着潜在的兴趣，并被他们感兴趣的学科项目所吸引。Folding@home在这方面没有什么不同。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_105" class="kx_ref">[105]</a></sup>最近对400多名积极参与者进行的研究显示，他们希望为研究做出贡献，许多人的朋友或亲戚都受到了Folding@home科学家关于疾病影响的调查。 </p>
<p>Folding@home吸引了计算机硬件爱好者(有时被称为“超频者”)。这些小组给项目带来了相当多的专业知识，并且能够制造具有先进处理能力的计算机。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_106" class="kx_ref">[106]</a></sup>其他分布式计算项目吸引了这些类型的参与者，项目通常被用来衡量改进后的计算机的性能，这方面的爱好通过项目的竞争性得到满足。个人和团队可以竞争谁能处理最多的计算机处理单元。 </p>
<p>这项关于Folding@home的最新研究涉及在线小组的采访和人种观察，研究表明硬件爱好者团队有时可以一起工作，分享最大化处理输出的最佳实践。这样的团队可以成为实践社区，共享语言和网络文化。这种参与模式已经有应用于其他分布式计算项目中。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_107" class="kx_ref">[107]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_108" class="kx_ref">[108]</a></sup> </p>
<p>Folding@home参与者的另一个重要特征是大部分参与者为男性。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_105" class="kx_ref">[105]</a></sup> 同样在其他分布式项目中也如此。此外，许多参与者是从事基于计算机和技术的工作和职业。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_105" class="kx_ref">[105]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_109" class="kx_ref">[109]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_110" class="kx_ref">[110]</a></sup> </p>
<p>并非所有的Folding@home参与者都是硬件爱好者。许多参与者在未经修改的机器上运行项目软件，并且参与竞争。超过100，000名参与者参与了Folding@home。然而，很难确定有多少参与者是硬件爱好者。尽管根据项目经理的说法，就处理能力而言，发烧友社区的贡献要大得多。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_111" class="kx_ref">[111]</a></sup> </p> 
<h3>4.1 <span>性能</span></h3> 
<p></p><p></p><div class="text_img ed_imgfloat_right">
            <a class="ed_image_link lazyLoad" data-src="https://img02.sogoucdn.com/app/a/200698/sogou_science_11051" data-bigsrc="" title="点击查看大图" href="javascript:" data-observer="true"></a>
            <div class="text_img_title">2004年四月到2012年十月期间，Folding@home计划与当时最快的超级计算机的计算能力对比图。2007年6至2011年6月期间，Folding@home的计算能力（红）超过了世界前500超级计算机的算力（黑）。但是与2011年十一月的日本“京”超级计算机和2012年6月的IBM蓝色基因超级计算机相比，该项目的运算能力就变得黯然失色了[a]</div>   
        </div> <p></p><p></p> 
<p>2007年9月16日，在很大程度上由于PlayStation 3控制台的参与，Folding@home项目正式达到了比一个本机petaFLOPS（千兆位触发器）更高的持续性能水平，成为第一个这样做的计算系统。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_112" class="kx_ref">[112]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_113" class="kx_ref">[113]</a></sup>当时500强最快的超级计算机是BlueGene/L，0.280千兆次。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_114" class="kx_ref">[114]</a></sup>第二年，2008年5月7日，该项目达到了持续的性能水平，高于两个petaFLOPS，<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_115" class="kx_ref">[115]</a></sup>随后分别在2008年8月<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_116" class="kx_ref">[116]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_117" class="kx_ref">[117]</a></sup>和2008年9月28日达到了三个和四个本地petaFLOPS。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_118" class="kx_ref">[118]</a></sup>2009年2月18日，Folding@home实现了五个本地petaFLOPS，<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_119" class="kx_ref">[119]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_120" class="kx_ref">[120]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_121" class="kx_ref">[121]</a></sup>是第一个满足这五个级别的计算项目。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_122" class="kx_ref">[122]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_123" class="kx_ref">[123]</a></sup>相比之下，2008年11月最快的超级计算机是IBM的Roadrunner，速度为1.105千兆次。2011年11月10日，Folding@home的性能超过了6个本地petaFLOPS，相当于近8个x86petaFLOPS。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_113" class="kx_ref">[113]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_124" class="kx_ref">[124]</a></sup>2013年5月中旬，Folding@home获得了超过7个本机千兆位触发器，相当于14.87个x86千兆位触发器。随后在6月21日，它达到了8个本机千兆位触发器，随后在同年9月9日达到了9个，拥有17.9个x86千兆位触发器。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_125" class="kx_ref">[125]</a></sup>2016年5月11日，Folding@home宣布，它正朝着100 x86千兆位的目标迈进。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_126" class="kx_ref">[126]</a></sup> </p> 
<h3>4.2 <span>积分</span></h3> 
<p>与其他分布式计算项目类似，Folding@home通过信用系统定量评估用户计算对项目的贡献。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_127" class="kx_ref">[127]</a></sup>给定蛋白质项目的所有单位都有统一的基本信用，这是通过在项目发布前在官方参考机器上对该项目的一个或多个工作单位进行基准测试来确定的。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_127" class="kx_ref">[127]</a></sup>每个用户都可以获得完成每个工作单元的基本分数，尽管通过使用密钥，他们可以获得可靠快速完成计算要求更高或科学优先级更高的单元的额外加分。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_128" class="kx_ref">[128]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_129" class="kx_ref">[129]</a></sup>用户也可以在多台机器上获得客户对其工作的认可。<sup>[39]</sup>这一积分系统试图将授予的学分与科学成果的价值联系起来。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_127" class="kx_ref">[127]</a></sup> </p>
<p>用户可以在一个团队中注册他们的贡献，这个团队将他们所有成员的观点结合在一起。同时，用户可以创建自己的团队，也可以加入现有的团队。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_130" class="kx_ref">[130]</a></sup>在某些情况下，一个团队可能有自己的社区驱动的帮助来源或招聘来源，如互联网论坛。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_131" class="kx_ref">[131]</a></sup>这些积分可以促进个人和团队之间的友好竞争，从而为项目计算出最大的价值，这将有利于Folding@home并加速科学研究。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_127" class="kx_ref">[127]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_132" class="kx_ref">[132]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_133" class="kx_ref">[133]</a></sup>个人和团队统计数据发布在Folding@home网站上。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_127" class="kx_ref">[127]</a></sup> </p>
<p>如果用户没有组成新团队，或者没有加入现有团队，该用户将自动成为“默认”团队的一部分。该“默认”团队的团队编号为“0”。为这个“默认”团队以及特别命名的团队积累统计数据。 </p></div></div><div id="par_14994961153917199"><h2 class="title">5 软件<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>在用户端Folding@home软件包括三个主要组件:工作单元、核心和客户端。 </p> 
<h3>5.1 <span>工作单位</span></h3> 
<p>工作单元是要求客户处理的蛋白质数据。工作单元是马尔可夫状态模型中状态之间模拟的一部分。志愿者的电脑下载并完全处理完工作单元后，它会返回到Folding@home服务器，然后由该服务器向志愿者发放积分。这个循环会自动重复。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_132" class="kx_ref">[132]</a></sup>所有工作单位都有相关的截止日期，如果超过该截止日期，用户可能无法获得积分，该单位将自动重新颁发给另一个参与者。由于蛋白质折叠是连续发生的，并且许多工作单元是从它们的前身生成的，如果一个工作单元在一段合理的时间后没有返回，这就允许整个模拟过程正常进行。由于这些截止日期，Folding@home的最低系统要求是一个奔腾3 450兆赫的中央处理器与流式SIMD扩展(SSE)。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_36" class="kx_ref">[36]</a></sup> 然而，高性能客户端的工作单元的截止时间比单处理器客户端短得多，因为科学利益的一个主要部分取决于快速完成模拟。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_134" class="kx_ref">[134]</a></sup> </p>
<p>在公开发布之前，工作单元要经历几个质量保证步骤，以防止有问题的步骤完全可用。这些测试阶段包括内部测试、测试和高级测试，直到最终完全发布。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_135" class="kx_ref">[135]</a></sup> Folding@home的工作单元通常只处理一次，除非在处理过程中出现罕见的错误。如果这种情况发生在三个不同的用户身上，设备将自动退出分发。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_136" class="kx_ref">[136]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_137" class="kx_ref">[137]</a></sup> Folding@home支持论坛可用于区分问题硬件和坏工作单元引起的问题。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_138" class="kx_ref">[138]</a></sup> </p> 
<h3>5.2 <span>核心</span></h3> 
<p>专门的分子动力学程序，称为“FahCores”，通常缩写为“Cores”，在工作单元上执行计算，作为后台进程。Folding@home的大部分内核都是基于GROOMACS，<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_132" class="kx_ref">[132]</a></sup> 这是最快、最流行的分子动力学软件包之一，它主要由手动优化的汇编语言代码和硬件优化组成。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_139" class="kx_ref">[139]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_140" class="kx_ref">[140]</a></sup>虽然GROOMACS是开源软件，潘德实验室和GROOMACS开发者之间也有合作，但是Folding@home使用了一个开源许可证来帮助确保数据的有效性。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_141" class="kx_ref">[141]</a></sup>活性较低的内核包括ProtoMol和GREEP。Folding@home使用了AMBER、CPMD、德斯蒙德和TINKER，但这些已经退役，不再服役。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_142" class="kx_ref">[142]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_143" class="kx_ref">[143]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_144" class="kx_ref">[144]</a></sup>这些核中的一些执行显式溶剂化计算，其中周围的溶剂(通常是水)被逐个原子地模拟；而另一些实施隐式溶剂化方法，其中溶剂被视为数学连续体。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_145" class="kx_ref">[145]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_146" class="kx_ref">[146]</a></sup>核心与客户分离，使科学方法能够自动更新，而无需客户更新。内核定期创建计算检查点，这样，如果它们被中断，就可以在启动时从该点恢复工作。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_132" class="kx_ref">[132]</a></sup> </p> 
<h3>5.3 <span>客户</span></h3> 
<p>Folding@home参与者在他们的个人电脑上安装一个客户端程序。用户与客户端交互，客户端在后台管理其他软件组件。通过客户端，用户可以暂停折叠过程、打开事件日志、检查工作进度或查看个人统计数据。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_147" class="kx_ref">[147]</a></sup>计算机客户端以非常低的优先级在后台连续运行，使用空闲的处理能力，因此正常的计算机使用不受影响。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_130" class="kx_ref">[130]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_36" class="kx_ref">[36]</a></sup>最大CPU使用量可以通过客户端设置进行调整。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_147" class="kx_ref">[147]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_148" class="kx_ref">[148]</a></sup> 客户端连接到Folding@home服务器，检索工作单元，还可以下载适合客户端设置、操作系统和底层硬件架构的内核。处理后，工作单元返回到Folding@home服务器。计算机客户端是为单处理器和多核处理器系统以及图形处理单元量身定制的。每种硬件架构的多样性和强大功能使Folding@home能够及时(在几周或几个月而不是几年内)高效完成多种类型的模拟，这具有重要的科学价值。这些客户一起就允许研究人员研究以前被认为难以采用计算来解决的生物医学问题。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_32" class="kx_ref">[32]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_132" class="kx_ref">[132]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_134" class="kx_ref">[134]</a></sup> </p>
<p>专业软件开发人员负责Folding@home的大部分代码，包括客户端和服务器端。开发团队包括英伟达、ATI、索尼和大锅开发公司（Cauldron Development）的程序员。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_149" class="kx_ref">[149]</a></sup>客户端只能从官方的Folding@home网站或其商业伙伴处下载，并且只能与Folding@home电脑文件交互。他们将通过斯坦福的Folding@home数据服务器上传和下载数据(通过端口8080，以80作为替代)，并使用2048位数字签名验证通信。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_36" class="kx_ref">[36]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_150" class="kx_ref">[150]</a></sup> 虽然客户端的图形用户界面是开源的，<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_151" class="kx_ref">[151]</a></sup> 但客户端是以安全性和科学完整性为理由的专有软件。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_152" class="kx_ref">[152]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_153" class="kx_ref">[153]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_154" class="kx_ref">[154]</a></sup> </p>
<p>然而，这种使用专有软件的基本原理是有争议的，因为尽管许可证可以在法律领域追溯执行，但它实际上并不阻止对可执行二进制文件的修改(也称为修补)。同样，纯二进制分发并不能防止恶意修改可执行二进制代码，无论是通过中间人攻击，同时通过互联网下载，<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_155" class="kx_ref">[155]</a></sup>还是通过由第三方重新分发二进制代码，这些二进制代码先前已被修改为二进制状态(即打补丁)，<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_156" class="kx_ref">[156]</a></sup> 或通过反编译译<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_157" class="kx_ref">[157]</a></sup> 并在修改后重新编译它们。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_158" class="kx_ref">[158]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_159" class="kx_ref">[159]</a></sup>除非二进制文件(和传输通道)已签名，并且接收人/系统能够验证数字签名，在这种情况下，不必要的修改应该是可检测的，但并不总是这样。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_160" class="kx_ref">[160]</a></sup>无论哪种方式，由于在Folding@home的情况下，客户端软件处理的输入数据和输出结果都是数字签名的，<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_36" class="kx_ref">[36]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_150" class="kx_ref">[150]</a></sup> 因此可以独立于客户端软件本身的完整性来验证工作的完整性。 </p>
<p>Folding@home使用Coms软件库进行联网。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_132" class="kx_ref">[132]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_149" class="kx_ref">[149]</a></sup> Folding@home于2000年10月1日启动，是首个针对生物分子系统的分布式计算项目。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_161" class="kx_ref">[161]</a></sup>它的第一个客户端是一个屏幕保护程序，它可以在电脑不用时运行。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_162" class="kx_ref">[162]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_163" class="kx_ref">[163]</a></sup>2004年，潘德实验室与大卫·安德森（ David P. Anderson）合作，在开源BOINC框架上测试一个补充客户端。该客户于2005年4月被发布到封闭测试版；<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_164" class="kx_ref">[164]</a></sup>然而，这种方法变得行不通，并于2006年6月被搁置。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_165" class="kx_ref">[165]</a></sup> </p> 
<p><strong>图形处理单元</strong></p> 
<p>图形处理单元(GPU)的专用硬件旨在加速视频游戏等三维图形应用的渲染，并且在某些类型的计算中可以显著优于图形处理单元。图形处理器是最强大、发展最快的计算平台之一，许多科学家和研究人员正在图形处理单元上进行通用计算(GPGPU)。然而，图形处理器硬件很难用于非图形任务，通常需要大量的算法重组和对底层架构的高级理解。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_166" class="kx_ref">[166]</a></sup>这种定制具有挑战性，对于软件开发资源有限的研究人员来说更是如此。Folding@home使用开源OpenMM库，该库使用具有两个应用编程接口(API)级别的桥接设计模式，将分子模拟软件与底层硬件架构接口。随着硬件优化的增加，基于OpenMM的图形处理器仿真不需要进行重大修改，但实现的性能几乎等同于手动调整的图形处理器代码，并且大大优于CPU实现。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_145" class="kx_ref">[145]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_167" class="kx_ref">[167]</a></sup> </p>
<p>在2010年之前，GPGPU消费级硬件的计算可靠性在很大程度上是未知的，与GPU内存中缺乏内置错误检测和纠正相关的间接证据引发了可靠性问题。在第一次大规模的GPU科学准确性测试中，2010年对Folding@home网络上20，000多台主机的研究发现，三分之二被测试的GPU内存子系统存在软件错误。尽管研究得出结论，只要不是硬件问题，可靠的GPU计算对于诸如软件端的错误检测都是可以处理的，但这些错误与板架构实质上是密切相关。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_168" class="kx_ref">[168]</a></sup> </p>
<p>Folding@home的第一代图形处理器客户端(GPU1)于2006年10月2日在向公众发布，<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_165" class="kx_ref">[165]</a></sup> 其计算速度是基于中央处理器的GROMACS的同类产品的20-30倍。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_169" class="kx_ref">[169]</a></sup>这是通用处理器首次用于分布式计算或主要分子动力学计算。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_170" class="kx_ref">[170]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_171" class="kx_ref">[171]</a></sup> GPU1为研究人员提供了开发GPU软件的丰富知识和经验，但由于DirectX编程接口的科学误差，2008年4月10日，它被第二代客户端GPU2所取代。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_169" class="kx_ref">[169]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_172" class="kx_ref">[172]</a></sup>继GPU2推出后，GPU1于6月6日正式退役。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_169" class="kx_ref">[169]</a></sup>与GPU1相比，GPU2在科学上更加可靠和高效，运行在ATI和CUDA支持的Nvidia通用处理器上，支持更高级的算法、更大的蛋白质和蛋白质模拟的实时可视化。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_173" class="kx_ref">[173]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_174" class="kx_ref">[174]</a></sup>随后，第三代Folding@home的GPU客户端(GPU3)于2010年5月25日发布。虽然与GPU2向后兼容，但GPU3在科学能力上更稳定、高效和灵活，<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_175" class="kx_ref">[175]</a></sup>并在OpenCL框架上使用OpenMM。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_175" class="kx_ref">[175]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_176" class="kx_ref">[176]</a></sup> 虽然这些GPU3客户端本身不支持操作系统Linux和macOS，但拥有英伟达显卡的Linux用户能够通过Wine软件应用程序运行它们。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_177" class="kx_ref">[177]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_178" class="kx_ref">[178]</a></sup> GPU仍然是Folding@home中FLOPS浮点运算的最强大平台。截至2012年11月，GPU客户端占整个项目x86 FLOPS吞吐量的87%。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_179" class="kx_ref">[179]</a></sup> </p>
<p>在Linux下对Nvidia和AMD显卡的本机支持是用FahCore 17引入的，它使用的是OpenCL而不是CUDA。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_180" class="kx_ref">[180]</a></sup> </p> 
<p><strong>PlayStation 3</strong></p> 
<p></p><p></p><div class="text_img ed_imgfloat_right">
            <a class="ed_image_link lazyLoad" data-src="https://img03.sogoucdn.com/app/a/200698/sogou_science_11052" data-bigsrc="" title="点击查看大图" href="javascript:" data-observer="true"></a>
            <div class="text_img_title">PlayStation 3平台的Life With PlayStation客户端展示了蛋白质被折叠的三维动画。</div>   
        </div> <p></p><p></p> 
<p>PlayStation 3s的强大功能。在其成立之初，其主要流式细胞处理器在某些计算方面的速度是个人电脑的20倍，处理能力在Xbox 360等其他系统上是找不到的。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_32" class="kx_ref">[32]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_181" class="kx_ref">[181]</a></sup> PS3的高速和高效率为根据阿姆达尔定律（ Amdahl's law）进行有价值的优化带来了其他机会，并极大地改变了计算效率和整体精度之间的权衡，允许以很少的额外计算成本使用更复杂的分子模型。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_182" class="kx_ref">[182]</a></sup> 这使得Folding@home能够运行生物医学计算，否则这些计算是不可行的。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_183" class="kx_ref">[183]</a></sup> </p>
<p>PS3客户端是索尼和潘德实验室合作开发的，于2007年3月23日首次作为独立客户端发布。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_32" class="kx_ref">[32]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_184" class="kx_ref">[184]</a></sup>它的发布使Folding@home成为第一个使用PS3的分布式计算项目。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_185" class="kx_ref">[185]</a></sup>第二年9月18日，PS3客户端成为了PlayStation推出后的生活频道。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_186" class="kx_ref">[186]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_187" class="kx_ref">[187]</a></sup>在推出时，客户端可以执行的计算类型介于中央处理器的灵活性和图形处理器的速度之间。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_132" class="kx_ref">[132]</a></sup> 然而，与运行在个人电脑上的客户端不同，用户在运行Folding@home时无法在PS3上执行其他活动。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_183" class="kx_ref">[183]</a></sup>PS3的统一控制台环境使技术支持变得更加容易，并使Folding@home更加用户友好。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_32" class="kx_ref">[32]</a></sup> PS3还能够将数据快速传输到图形处理器，图形处理器用于当前蛋白质动态的实时原子级可视化。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_182" class="kx_ref">[182]</a></sup> </p>
<p>2012年11月6日，索尼终止了对Folding@home PS3客户端的支持，以及PlayStation Life提供的其他服务。在其五年零七个月的生命周期中，超过1500万用户为Folding@home贡献了超过1亿小时的计算时间，极大地帮助了该项目的疾病研究。在与潘德实验室讨论后，索尼决定终止该应用。潘德认为PlayStation 3客户端是该项目的“游戏改变者”。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_188" class="kx_ref">[188]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_189" class="kx_ref">[189]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_190" class="kx_ref">[190]</a></sup> </p> 
<p><strong>多核处理客户端</strong></p> 
<p>Folding@home可以利用现代多核处理器的并行计算能力。同时使用几个中央处理器内核的能力允许更快地完成完整的模拟。这些中央处理器内核协同工作，完成单个工作单元的速度要比标准的单处理器客户端快得多。这种方法在科学上很有价值，因为它能够在相同的时间内执行更长的仿真轨迹，并且降低了将大型仿真扩展到许多独立处理器的传统性困难。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_191" class="kx_ref">[191]</a></sup> 2007年发表在《分子生物学杂志》上的一篇论文依赖多核处理来模拟绒毛蛋白部分的折叠，其长度大约是单处理器客户端的10倍，与实验折叠速率一致。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_192" class="kx_ref">[192]</a></sup> </p>
<p>2006年11月，第一代对称多处理(SMP)客户端被公开发布用于开放测试，称为SMP1。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_165" class="kx_ref">[165]</a></sup>这些客户端使用消息传递接口(MPI)通信协议进行并行处理，因为当时GROMACS内核不是设计用于多线程的。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_134" class="kx_ref">[134]</a></sup> 这是分布式计算项目第一次使用MPI。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_193" class="kx_ref">[193]</a></sup>虽然客户端在基于Unix的操作系统(如Linux和macOS)中表现良好，但在windows系统下却很麻烦。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_191" class="kx_ref">[191]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_193" class="kx_ref">[193]</a></sup>2010年1月24日，第二代SMP客户端和SMP1的后继者SMP2作为开放测试版发布，并以更可靠的基于线程的实现取代复杂的MPI。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_129" class="kx_ref">[129]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_149" class="kx_ref">[149]</a></sup> </p>
<p>SMP2支持一种特殊类别的bigadv工作单元的试验，该工作单元被设计用来模拟异常大且计算密集并且具有高度科学优先权的蛋白质。这些单元最初至少需要八个中央处理器核心，<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_194" class="kx_ref">[194]</a></sup>后来在2012年2月7日提高到十六个。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_195" class="kx_ref">[195]</a></sup>除了标准SMP2工作单元的这些额外硬件要求外，它们还需要更多的系统资源，如随机存取存储器和互联网带宽。作为回报，运行这些程序的用户将获得比SMP2系统奖励积分增加20%的奖励。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_196" class="kx_ref">[196]</a></sup>bigav类别允许Folding@home长时间运行特别苛刻的模拟，这种模拟以前需要使用超级计算集群，在Folding@home上的任何其他地方都无法执行。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_194" class="kx_ref">[194]</a></sup>许多拥有能够运行大adv单元的硬件的用户后来发现，当CPU核心最小值增加时，他们的硬件设置被认为不符合大adv工作单元的要求，这使得他们只能运行正常的SMP工作单元。这让许多用户感到沮丧，他们在程序中投入了大量资金，却在不久后就因为bigadv的目的而使他们的硬件过时了。因此，潘德在2014年1月宣布bigadv计划将于2015年1月31日结束。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_197" class="kx_ref">[197]</a></sup> </p> 
<p><strong>V7</strong></p> 
<p></p><p></p><div class="text_img ed_imgfloat_right">
            <a class="ed_image_link lazyLoad" data-src="https://img04.sogoucdn.com/app/a/200698/sogou_science_11053" data-bigsrc="" title="点击查看大图" href="javascript:" data-observer="true"></a>
            <div class="text_img_title">在Windows 7下运行的V7客户端初始模式的示例图。除了各种控件和用户详细信息外，V7客户端还呈现了工作单元信息，例如它的状态，计算进度，ETA，信用点，ID号和描述。</div>   
        </div> <p></p><p></p> 
<p>V7客户端是第七代也是最新一代的Folding@home客户端软件，是对以前的Windows、macOS和Linux操作系统客户端的完全重写和统一。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_198" class="kx_ref">[198]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_199" class="kx_ref">[199]</a></sup> 它于2012年3月22日发布。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_200" class="kx_ref">[200]</a></sup>像它的前辈一样，V7可以在后台以非常低的优先级运行Folding@home，允许其他应用程序根据需要使用中央处理器资源。它的设计是为了让新手更容易安装、启动和操作，并为研究人员提供比以前客户更大的科学灵活性。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_201" class="kx_ref">[201]</a></sup> V7使用Trac来管理它的bug票证，这样用户就可以看到它的开发过程并提供反馈。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_199" class="kx_ref">[199]</a></sup> </p>
<p>V7由四个集成元件组成。用户通常与V7的开源图形用户界面交互，名为FAHControl。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_151" class="kx_ref">[151]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_202" class="kx_ref">[202]</a></sup>它具有新手、高级和专家用户界面模式，并且能够从一台计算机监控、配置和控制许多远程折叠客户端。FAHControl指导FAHClient，这是一个后端应用程序，它依次管理每个FahClient。每个插槽都可以替代以前不同的Folding@home v6单处理器、SMP或GPU计算机客户端，因为它可以独立下载、处理和上传工作单元。FAHViewer函数以PS3的浏览器为模型，显示当前正在加工的蛋白质的实时三维渲染模型 (如果有的话)。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_198" class="kx_ref">[198]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_199" class="kx_ref">[199]</a></sup> </p> 
<p><strong>谷歌浏览器（<b>Google Chrome</b>）</strong></p> 
<p>2014年，谷歌浏览器和谷歌网络浏览器的客户端发布，允许用户在他们的网络浏览器中运行Folding@home。客户端在基于<b>Google Chrome</b>上使用谷歌的本地客户端(NaCl)功能，在用户机器上的沙盒中以接近本地的速度运行Folding@home代码。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_203" class="kx_ref">[203]</a></sup> </p> 
<p><strong>机器人</strong></p> 
<p>2015年7月，一款安卓手机客户端在谷歌播放器上发布，适用于运行安卓4.4 KitKat或更高版本的设备。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_204" class="kx_ref">[204]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_205" class="kx_ref">[205]</a></sup> </p></div></div><div id="par_14994961170694412"><h2 class="title">6 与其他分子模拟器的比较<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>Rosetta@home是一个面向蛋白质结构预测的分布式计算项目，是最精确的三级结构预测器之一。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_206" class="kx_ref">[206]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_207" class="kx_ref">[207]</a></sup> Rosetta软件中的构象状态可用于初始化马尔可夫状态模型，作为Folding@home模拟的起点。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_18" class="kx_ref">[18]</a></sup> 相反，结构预测算法可以从热力学和动力学模型以及蛋白质折叠模拟的取样方面得到改进。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_208" class="kx_ref">[208]</a></sup> 由于Rosetta只试图预测最终折叠状态，而不是折叠如何进行，Rosetta@home和Folding@home是互补的，并解决相当不同的分子问题。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_18" class="kx_ref">[18]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_209" class="kx_ref">[209]</a></sup> </p>
<p>安东（Anton ）是为分子动力学模拟而建造的专用超级计算机。2011年10月，安东和Folding@home是两个最强大的分子动力学系统。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_210" class="kx_ref">[210]</a></sup> Anton 的独特之处在于它能够产生计算成本高昂的单个超长分子轨道，<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_211" class="kx_ref">[211]</a></sup> 例如2010年达到毫秒级的轨道。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_212" class="kx_ref">[212]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_213" class="kx_ref">[213]</a></sup> 这些长轨迹可能对某些类型的生化问题特别有帮助。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_214" class="kx_ref">[214]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_215" class="kx_ref">[215]</a></sup> 但是，Anton 不使用马尔可夫状态模型进行分析。2011年，潘德实验室从两个100微秒Anton 模拟中构建了一个质谱，并发现了Anton 传统分析中不可见的替代折叠路径。他们得出的结论是，由有限数量的长轨道构建的多管轨迹和由许多短轨道组装的多管轨迹之间没有什么区别。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_211" class="kx_ref">[211]</a></sup>2011年6月，Folding@home开始增加Anton 模拟的采样，以更好地确定其方法与Anton 的方法相比如何。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_216" class="kx_ref">[216]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_217" class="kx_ref">[217]</a></sup> 然而，不像Folding@home的较短轨迹更适合分布式计算和其他并行化方法，较长轨迹不需要自适应采样来充分采样蛋白质的相空间。因此，Anton 和Folding@home的模拟方法的结合有可能提供对该空间更彻底的采样。<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_211" class="kx_ref">[211]</a></sup> </p></div></div><div id="par_14994961170694413"><h2 class="title">7 笔记<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><ol> 
 <li id="cite-ref-2"> Supercomputer FLOPS performance is assessed by running the legacy LINPACK benchmark. This short-term testing has difficulty in accurately reflecting sustained performance on real-world tasks because LINPACK more efficiently maps to supercomputer hardware. Computing systems vary in architecture and design, so direct comparison is difficult. Despite this, FLOPS remain the primary speed metric used in supercomputing.<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_218" class="kx_ref">[218]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_219" class="kx_ref">[219]</a></sup> In contrast, Folding@home determines its FLOPS using wall-clock time by measuring how much time its work units take to complete.<sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_220" class="kx_ref">[220]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_221" class="kx_ref">[221]</a></sup><sup><a href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/d10248.htm#quote_222" class="kx_ref">[222]</a></sup> </li> 
</ol></div></div></div></div><div id="references"><h2 class="title" id="par_references">参考文献</h2><ul class="references"><li id="quote_1"><span class="references-num">[1]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Pande lab (2010-07-26). "About Folding@home". Folding@home. Stanford University. Retrieved 2017-06-30..</span></p></li><li id="quote_2"><span class="references-num">[2]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">V. S. Pande; K. Beauchamp; G. R. Bowman (2010). "Everything you wanted to know about Markov State Models but were afraid to ask". Methods. 52 (1): 99–105. doi:10.1016/j.ymeth.2010.06.002. PMC 2933958. PMID 20570730..</span></p></li><li id="quote_3"><span class="references-num">[3]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Gregory R. Bowman; Vijay S. Pande (2010). "Protein folded states are kinetic hubs". Proceedings of the National Academy of Sciences. 107 (24): 10890–5. Bibcode:2010PNAS..10710890B. doi:10.1073/pnas.1003962107. PMC 2890711. PMID 20534497..</span></p></li><li id="quote_4"><span class="references-num">[4]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Pande lab (July 27, 2012). "Papers &amp; Results from Folding@home". Folding@home. Stanford University. Archived from the original on September 21, 2012. Retrieved 1 February 2019..</span></p></li><li id="quote_5"><span class="references-num">[5]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Vincent A. Voelz; Gregory R. Bowman; Kyle Beauchamp; Vijay S. Pande (2010). "Molecular simulation of ab initio protein folding for a millisecond folder NTL9(1–39)". Journal of the American Chemical Society. 132 (5): 1526–1528. doi:10.1021/ja9090353. PMC 2835335. PMID 20070076..</span></p></li><li id="quote_6"><span class="references-num">[6]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Christopher D. Snow; Houbi Ngyen; Vijay S. Pande; Martin Gruebele (2002). "Absolute comparison of simulated and experimental protein-folding dynamics" (PDF). Nature. 420 (6911): 102–106. Bibcode:2002Natur.420..102S. doi:10.1038/nature01160. PMID 12422224. Archived from the original (PDF) on 2012-03-24..</span></p></li><li id="quote_7"><span class="references-num">[7]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Fabrizio Marinelli, Fabio Pietrucci, Alessandro Laio, Stefano Piana (2009). Pande, Vijay S., ed. "A Kinetic Model of Trp-Cage Folding from Multiple Biased Molecular Dynamics Simulations". PLoS Computational Biology. 5 (8): e1000452. Bibcode:2009PLSCB...5E0452M. doi:10.1371/journal.pcbi.1000452. PMC 2711228. PMID 19662155.CS1 maint: Multiple names: authors list (link).</span></p></li><li id="quote_8"><span class="references-num">[8]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"So Much More to Know". Science. 309 (5731): 78–102. 2005. doi:10.1126/science.309.5731.78b. PMID 15994524..</span></p></li><li id="quote_9"><span class="references-num">[9]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Heath Ecroyd; John A. Carver (2008). "Unraveling the mysteries of protein folding and misfolding". IUBMB Life (review). 60 (12): 769–774. doi:10.1002/iub.117. PMID 18767168..</span></p></li><li id="quote_10"><span class="references-num">[10]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Yiwen Chen; Feng Ding; Huifen Nie; Adrian W. Serohijos; Shantanu Sharma; Kyle C. Wilcox; Shuangye Yin; Nikolay V. Dokholyan (2008). "Protein folding: Then and now". Archives of Biochemistry and Biophysics. 469 (1): 4–19. doi:10.1016/j.abb.2007.05.014. PMC 2173875. PMID 17585870..</span></p></li><li id="quote_11"><span class="references-num">[11]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Leila M Luheshi; Damian Crowther; Christopher Dobson (2008). "Protein misfolding and disease: from the test tube to the organism". Current Opinion in Chemical Biology. 12 (1): 25–31. doi:10.1016/j.cbpa.2008.02.011. PMID 18295611..</span></p></li><li id="quote_12"><span class="references-num">[12]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">C. D. Snow; E. J. Sorin; Y. M. Rhee; V. S. Pande. (2005). "How well can simulation predict protein folding kinetics and thermodynamics?". Annual Review of Biophysics (review). 34: 43–69. doi:10.1146/annurev.biophys.34.040204.144447. PMID 15869383..</span></p></li><li id="quote_13"><span class="references-num">[13]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">A. Verma; S.M. Gopal; A. Schug; J.S. Oh; K.V. Klenin; K.H. Lee; W. Wenzel (2008). Massively Parallel All Atom Protein Folding in a Single Day. Advances in Parallel Computing. 15. pp. 527–534. ISBN 978-1-58603-796-3. ISSN 0927-5452..</span></p></li><li id="quote_14"><span class="references-num">[14]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Vijay S. Pande; Ian Baker; Jarrod Chapman; Sidney P. Elmer; Siraj Khaliq; Stefan M. Larson; Young Min Rhee; Michael R. Shirts; Christopher D. Snow; Eric J. Sorin; Bojan Zagrovic (2002). "Atomistic protein folding simulations on the submillisecond timescale using worldwide distributed computing". Biopolymers. 68 (1): 91–109. doi:10.1002/bip.10219. PMID 12579582..</span></p></li><li id="quote_15"><span class="references-num">[15]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">G. Bowman; V. Volez; V. S. Pande (2011). "Taming the complexity of protein folding". Current Opinion in Structural Biology. 21 (1): 4–11. doi:10.1016/j.sbi.2010.10.006. PMC 3042729. PMID 21081274..</span></p></li><li id="quote_16"><span class="references-num">[16]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Chodera, John D.; Swope, William C.; Pitera, Jed W.; Dill, Ken A. (1 January 2006). "Long‐Time Protein Folding Dynamics from Short‐Time Molecular Dynamics Simulations". Multiscale Modeling &amp; Simulation. 5 (4): 1214–1226. doi:10.1137/06065146X..</span></p></li><li id="quote_17"><span class="references-num">[17]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Robert B Best (2012). "Atomistic molecular simulations of protein folding". Current Opinion in Structural Biology (review). 22 (1): 52–61. doi:10.1016/j.sbi.2011.12.001. PMID 22257762..</span></p></li><li id="quote_18"><span class="references-num">[18]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">TJ Lane; Gregory Bowman; Robert McGibbon; Christian Schwantes; Vijay Pande; Bruce Borden (September 10, 2012). "Folding@home Simulation FAQ". Folding@home. Stanford University. Archived from the original on September 21, 2012. Retrieved July 8, 2013..</span></p></li><li id="quote_19"><span class="references-num">[19]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Gregory R. Bowman; Daniel L. Ensign; Vijay S. Pande (2010). "Enhanced Modeling via Network Theory: Adaptive Sampling of Markov State Models". Journal of Chemical Theory and Computation. 6 (3): 787–794. doi:10.1021/ct900620b. PMC 3637129. PMID 23626502..</span></p></li><li id="quote_20"><span class="references-num">[20]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Vijay Pande (June 8, 2012). "FAHcon 2012: Thinking about how far FAH has come". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved June 12, 2012..</span></p></li><li id="quote_21"><span class="references-num">[21]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Kyle A. Beauchamp; Daniel L. Ensign; Rhiju Das; Vijay S. Pande (2011). "Quantitative comparison of villin headpiece subdomain simulations and triplet–triplet energy transfer experiments". Proceedings of the National Academy of Sciences. 108 (31): 12734–9. Bibcode:2011PNAS..10812734B. doi:10.1073/pnas.1010880108. PMC 3150881. PMID 21768345..</span></p></li><li id="quote_22"><span class="references-num">[22]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Timothy H. Click; Debabani Ganguly; Jianhan Chen (2010). "Intrinsically Disordered Proteins in a Physics-Based World". International Journal of Molecular Sciences. 11 (12): 919–27. doi:10.3390/ijms11125292. PMC 3100817. PMID 21614208..</span></p></li><li id="quote_23"><span class="references-num">[23]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Greg Bowman awarded the 2010 Kuhn Paradigm Shift Award". simtk.org. SimTK: MSMBuilder. March 29, 2010. Archived from the original on September 21, 2012. Retrieved September 20, 2012..</span></p></li><li id="quote_24"><span class="references-num">[24]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"MSMBuilder Source Code Repository". MSMBuilder. simtk.org. 2012. Archived from the original on October 12, 2012. Retrieved October 12, 2012..</span></p></li><li id="quote_25"><span class="references-num">[25]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Biophysical Society Names Five 2012 Award Recipients". Biophysics.org. Biophysical Society. August 17, 2011. Archived from the original on September 21, 2012. Retrieved September 20, 2012..</span></p></li><li id="quote_26"><span class="references-num">[26]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Folding@home – Awards". Folding@home. Stanford University. August 2011. Archived from the original (FAQ) on September 21, 2012. Retrieved July 8, 2013..</span></p></li><li id="quote_27"><span class="references-num">[27]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Vittorio Bellotti; Monica Stoppini (2009). "Protein Misfolding Diseases" (PDF). The Open Biology Journal. 2 (2): 228–234. doi:10.2174/1874196700902020228. Archived from the original on February 22, 2014.CS1 maint: BOT: original-url status unknown (link).</span></p></li><li id="quote_28"><span class="references-num">[28]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Pande lab (May 30, 2012). "Folding@home Diseases Studied FAQ". Folding@home. Stanford University. Archived from the original (FAQ) on September 21, 2012. Retrieved July 8, 2013..</span></p></li><li id="quote_29"><span class="references-num">[29]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Collier, Leslie; Balows, Albert; Sussman, Max (1998). Mahy, Brian; Collier, Leslie, eds. Topley and Wilson's Microbiology and Microbial Infections. 1, Virology (ninth ed.). London: Arnold. pp. 75–91. ISBN 978-0-340-66316-5..</span></p></li><li id="quote_30"><span class="references-num">[30]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Fred E. Cohen; Jeffery W. Kelly (2003). "Therapeutic approaches to protein misfolding diseases". Nature (review). 426 (6968): 905–9. Bibcode:2003Natur.426..905C. doi:10.1038/nature02265. PMID 14685252..</span></p></li><li id="quote_31"><span class="references-num">[31]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Chun Song; Shen Lim; Joo Tong (2009). "Recent advances in computer-aided drug design". Briefings in Bioinformatics (review). 10 (5): 579–91. doi:10.1093/bib/bbp023. PMID 19433475..</span></p></li><li id="quote_32"><span class="references-num">[32]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Pande lab (2012). "Folding@Home Press FAQ". Folding@home. Stanford University. Archived from the original (FAQ) on September 21, 2012. Retrieved July 8, 2013..</span></p></li><li id="quote_33"><span class="references-num">[33]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Christian "schwancr" Schwantes (Pande lab member) (August 15, 2011). "Projects 7808 and 7809 to full fah". Folding@home. phpBB Group. Archived from the original on September 21, 2012. Retrieved October 16, 2011..</span></p></li><li id="quote_34"><span class="references-num">[34]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Del Lucent; V. Vishal; Vijay S. Pande (2007). "Protein folding under confinement: A role for solvent". Proceedings of the National Academy of Sciences of the United States of America. 104 (25): 10430–10434. Bibcode:2007PNAS..10410430L. doi:10.1073/pnas.0608256104. PMC 1965530. PMID 17563390..</span></p></li><li id="quote_35"><span class="references-num">[35]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Vincent A. Voelz; Vijay R. Singh; William J. Wedemeyer; Lisa J. Lapidus; Vijay S. Pande (2010). "Unfolded-State Dynamics and Structure of Protein L Characterized by Simulation and Experiment". Journal of the American Chemical Society. 132 (13): 4702–4709. doi:10.1021/ja908369h. PMC 2853762. PMID 20218718..</span></p></li><li id="quote_36"><span class="references-num">[36]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Pande lab (August 18, 2011). "Folding@home Main FAQ" (FAQ). Folding@home. Stanford University. Archived from the original on September 21, 2012. Retrieved July 8, 2013..</span></p></li><li id="quote_37"><span class="references-num">[37]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Vijay Pande (April 23, 2008). "Folding@home and Simbios". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved November 9, 2011..</span></p></li><li id="quote_38"><span class="references-num">[38]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Vijay Pande (October 25, 2011). "Re: Suggested Changes to F@h Website". Folding@home. phpBB Group. Archived from the original on September 21, 2012. Retrieved October 25, 2011..</span></p></li><li id="quote_39"><span class="references-num">[39]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Caroline Hadley (2004). "Biologists think bigger". EMBO Reports. 5 (3): 236–238. doi:10.1038/sj.embor.7400108. PMC 1299019. PMID 14993921..</span></p></li><li id="quote_40"><span class="references-num">[40]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">S. Pronk; P. Larsson; I. Pouya; G.R. Bowman; I.S. Haque; K. Beauchamp; B. Hess; V.S. Pande; P.M. Kasson; E. Lindahl (2011). "Copernicus: A new paradigm for parallel adaptive molecular dynamics". 2011 International Conference for High Performance Computing, Networking, Storage and Analysis: 1–10, 12–18..</span></p></li><li id="quote_41"><span class="references-num">[41]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Sander Pronk; Iman Pouya; Per Larsson; Peter Kasson; Erik Lindahl (November 17, 2011). "Copernicus Download". copernicus-computing.org. Copernicus. Archived from the original on October 12, 2012. Retrieved October 2, 2012..</span></p></li><li id="quote_42"><span class="references-num">[42]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">G Brent Irvine; Omar M El-Agnaf; Ganesh M Shankar; Dominic M Walsh (2008). "Protein Aggregation in the Brain: The Molecular Basis for Alzheimer's and Parkinson's Diseases". Molecular Medicine (review). 14 (7–8): 451–464. doi:10.2119/2007-00100.Irvine. PMC 2274891. PMID 18368143..</span></p></li><li id="quote_43"><span class="references-num">[43]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Claudio Soto; Lisbell D. Estrada (2008). "Protein Misfolding and Neurodegeneration". Archives of Neurology (review). 65 (2): 184–189. doi:10.1001/archneurol.2007.56. PMID 18268186..</span></p></li><li id="quote_44"><span class="references-num">[44]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Robin Roychaudhuri; Mingfeng Yang; Minako M. Hoshi; David B. Teplow (2008). "Amyloid β-Protein Assembly and Alzheimer Disease". Journal of Biological Chemistry. 284 (8): 4749–53. doi:10.1074/jbc.R800036200. PMC 3837440. PMID 18845536..</span></p></li><li id="quote_45"><span class="references-num">[45]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Nicholas W. Kelley; V. Vishal; Grant A. Krafft; Vijay S. Pande. (2008). "Simulating oligomerization at experimental concentrations and long timescales: A Markov state model approach". Journal of Chemical Physics. 129 (21): 214707. Bibcode:2008JChPh.129u4707K. doi:10.1063/1.3010881. PMC 2674793. PMID 19063575..</span></p></li><li id="quote_46"><span class="references-num">[46]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">P. Novick, J. Rajadas, C.W. Liu, N. W. Kelley, M. Inayathullah, and V. S. Pande (2011). Buehler, Markus J., ed. "Rationally Designed Turn Promoting Mutation in the Amyloid-β Peptide Sequence Stabilizes Oligomers in Solution". PLoS ONE. 6 (7): e21776. Bibcode:2011PLoSO...621776R. doi:10.1371/journal.pone.0021776. PMC 3142112. PMID 21799748.CS1 maint: Multiple names: authors list (link).</span></p></li><li id="quote_47"><span class="references-num">[47]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Aabgeena Naeem; Naveed Ahmad Fazili (2011). "Defective Protein Folding and Aggregation as the Basis of Neurodegenerative Diseases: The Darker Aspect of Proteins". Cell Biochemistry and Biophysics (review). 61 (2): 237–50. doi:10.1007/s12013-011-9200-x. PMID 21573992..</span></p></li><li id="quote_48"><span class="references-num">[48]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Gregory R Bowman; Xuhui Huang; Vijay S Pande (2010). "Network models for molecular kinetics and their initial applications to human health". Cell Research (review). 20 (6): 622–630. doi:10.1038/cr.2010.57. PMC 4441225. PMID 20421891..</span></p></li><li id="quote_49"><span class="references-num">[49]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Vijay Pande (December 18, 2008). "New FAH results on possible new Alzheimer's drug presented". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved September 23, 2011..</span></p></li><li id="quote_50"><span class="references-num">[50]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Paul A. Novick; Dahabada H. Lopes; Kim M. Branson; Alexandra Esteras-Chopo; Isabella A. Graef; Gal Bitan; Vijay S. Pande (2012). "Design of β-Amyloid Aggregation Inhibitors from a Predicted Structural Motif". Journal of Medicinal Chemistry. 55 (7): 3002–10. doi:10.1021/jm201332p. PMC 3766731. PMID 22420626..</span></p></li><li id="quote_51"><span class="references-num">[51]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">yslin (Pande lab member) (July 22, 2011). "New project p6871 [Classic]". Folding@home. phpBB Group. Archived from the original on September 21, 2012. Retrieved March 17, 2012..</span></p></li><li id="quote_52"><span class="references-num">[52]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Pande lab. "Project 6871 Description". Folding@home. Stanford University. Archived from the original on September 21, 2012. Retrieved September 27, 2011..</span></p></li><li id="quote_53"><span class="references-num">[53]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Walker FO (2007). "Huntington's disease". Lancet. 369 (9557): 218–28 [220]. doi:10.1016/S0140-6736(07)60111-1. PMID 17240289..</span></p></li><li id="quote_54"><span class="references-num">[54]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Nicholas W. Kelley; Xuhui Huang; Stephen Tam; Christoph Spiess; Judith Frydman; Vijay S. Pande (2009). "The predicted structure of the headpiece of the Huntingtin protein and its implications on Huntingtin aggregation". Journal of Molecular Biology. 388 (5): 919–27. doi:10.1016/j.jmb.2009.01.032. PMC 2677131. PMID 19361448..</span></p></li><li id="quote_55"><span class="references-num">[55]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Susan W Liebman; Stephen C Meredith (2010). "Protein folding: Sticky N17 speeds huntingtin pile-up". Nature Chemical Biology. 6 (1): 7–8. doi:10.1038/nchembio.279. PMID 20016493..</span></p></li><li id="quote_56"><span class="references-num">[56]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Diwakar Shukla (Pande lab member) (February 10, 2012). "Project 8021 released to beta". Folding@home. phpBB Group. Archived from the original on September 21, 2012. Retrieved March 17, 2012..</span></p></li><li id="quote_57"><span class="references-num">[57]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">M Hollstein; D Sidransky; B Vogelstein; CC Harris (1991). "p53 mutations in human cancers". Science. 253 (5015): 49–53. Bibcode:1991Sci...253...49H. doi:10.1126/science.1905840. PMID 1905840..</span></p></li><li id="quote_58"><span class="references-num">[58]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">L. T. Chong; C. D. Snow; Y. M. Rhee; V. S. Pande. (2004). "Dimerization of the p53 Oligomerization Domain: Identification of a Folding Nucleus by Molecular Dynamics Simulations". Journal of Molecular Biology. 345 (4): 869–878. CiteSeerX 10.1.1.132.1174. doi:10.1016/j.jmb.2004.10.083. PMID 15588832..</span></p></li><li id="quote_59"><span class="references-num">[59]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">mah3, Vijay Pande (September 24, 2004). "F@H project publishes results of cancer related research". MaximumPC.com. Future US, Inc. Archived from the original on September 21, 2012. Retrieved September 20, 2012. To our knowledge, this is the first peer-reviewed results from a distributed computing project related to cancer..</span></p></li><li id="quote_60"><span class="references-num">[60]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Lillian T. Chong; William C. Swope; Jed W. Pitera; Vijay S. Pande (2005). "Kinetic Computational Alanine Scanning: Application to p53 Oligomerization". Journal of Molecular Biology. 357 (3): 1039–1049. doi:10.1016/j.jmb.2005.12.083. PMID 16457841..</span></p></li><li id="quote_61"><span class="references-num">[61]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Almeida MB, do Nascimento JL, Herculano AM, Crespo-López ME (2011). "Molecular chaperones: toward new therapeutic tools". Journal of Molecular Biology (review). 65 (4): 239–43. doi:10.1016/j.biopha.2011.04.025. PMID 21737228..</span></p></li><li id="quote_62"><span class="references-num">[62]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Vijay Pande (September 28, 2007). "Nanomedicine center". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved September 23, 2011..</span></p></li><li id="quote_63"><span class="references-num">[63]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Vijay Pande (December 22, 2009). "Release of new Protomol (Core B4) WUs". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved September 23, 2011..</span></p></li><li id="quote_64"><span class="references-num">[64]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Pande lab. "Project 180 Description". Folding@home. Stanford University. Archived from the original on September 21, 2012. Retrieved September 27, 2011..</span></p></li><li id="quote_65"><span class="references-num">[65]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">TJ Lane (Pande lab member) (June 8, 2011). "Project 7600 in Beta". Folding@home. phpBB Group. Archived from the original on September 21, 2012. Retrieved September 27, 2011..</span></p></li><li id="quote_66"><span class="references-num">[66]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">TJ Lane (Pande lab member) (June 8, 2011). "Project 7600 Description". Folding@home. Stanford University. Archived from the original on September 21, 2012. Retrieved March 31, 2012..</span></p></li><li id="quote_67"><span class="references-num">[67]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Scientists boost potency, reduce side effects of IL-2 protein used to treat cancer". MedicalXpress.com. Medical Xpress. March 18, 2012. Archived from the original on September 21, 2012. Retrieved September 20, 2012..</span></p></li><li id="quote_68"><span class="references-num">[68]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Aron M. Levin; Darren L. Bates; Aaron M. Ring; Carsten Krieg; Jack T. Lin; Leon Su; Ignacio Moraga; Miro E. Raeber; Gregory R. Bowman; Paul Novick; Vijay S. Pande; C. Garrison Fathman; Onur Boyman; K. Christopher Garcia (2012). "Exploiting a natural conformational switch to engineer an interleukin-2 'superkine'". Nature. 484 (7395): 529–33. Bibcode:2012Natur.484..529L. doi:10.1038/nature10975. PMC 3338870. PMID 22446627..</span></p></li><li id="quote_69"><span class="references-num">[69]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Rauch F, Glorieux FH (2004). "Osteogenesis imperfecta". Lancet. 363 (9418): 1377–85. doi:10.1016/S0140-6736(04)16051-0. PMID 15110498..</span></p></li><li id="quote_70"><span class="references-num">[70]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Fratzl, Peter (2008). Collagen: structure and mechanics. ISBN 978-0-387-73905-2. Retrieved March 17, 2012..</span></p></li><li id="quote_71"><span class="references-num">[71]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Gautieri A, Uzel S, Vesentini S, Redaelli A, Buehler MJ (2009). "Molecular and mesoscale disease mechanisms of Osteogenesis Imperfecta". Biophysical Journal. 97 (3): 857–865. Bibcode:2009BpJ....97..857G. doi:10.1016/j.bpj.2009.04.059. PMC 2718154. PMID 19651044..</span></p></li><li id="quote_72"><span class="references-num">[72]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Sanghyun Park; Randall J. Radmer; Teri E. Klein; Vijay S. Pande (2005). "A New Set of Molecular Mechanics Parameters for Hydroxyproline and Its Use in Molecular Dynamics Simulations of Collagen-Like Peptides". Journal of Computational Chemistry. 26 (15): 1612–1616. CiteSeerX 10.1.1.142.6781. doi:10.1002/jcc.20301. PMID 16170799..</span></p></li><li id="quote_73"><span class="references-num">[73]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Gregory Bowman (Pande lab Member). "Project 10125". Folding@home. phpBB Group. Retrieved December 2, 2011..</span></p></li><li id="quote_74"><span class="references-num">[74]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Hana Robson Marsden; Itsuro Tomatsu; Alexander Kros (2011). "Model systems for membrane fusion". Chemical Society Reviews (review). 40 (3): 1572–1585. doi:10.1039/c0cs00115e. PMID 21152599..</span></p></li><li id="quote_75"><span class="references-num">[75]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Peter Kasson (2012). "Peter M. Kasson". Kasson lab. University of Virginia. Archived from the original on September 21, 2012. Retrieved September 20, 2012..</span></p></li><li id="quote_76"><span class="references-num">[76]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Peter M. Kasson; Afra Zomorodian; Sanghyun Park; Nina Singhal; Leonidas J. Guibas; Vijay S. Pande (2007). "Persistent voids: a new structural metric for membrane fusion". Bioinformatics. 23 (14): 1753–1759. doi:10.1093/bioinformatics/btm250. PMID 17488753..</span></p></li><li id="quote_77"><span class="references-num">[77]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Peter M. Kasson; Daniel L. Ensign; Vijay S. Pande (2009). "Combining Molecular Dynamics with Bayesian Analysis To Predict and Evaluate Ligand-Binding Mutations in Influenza Hemagglutinin". Journal of the American Chemical Society. 131 (32): 11338–11340. doi:10.1021/ja904557w. PMC 2737089. PMID 19637916..</span></p></li><li id="quote_78"><span class="references-num">[78]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Peter M. Kasson; Vijay S. Pande (2009). "Combining mutual information with structural analysis to screen for functionally important residues in influenza hemagglutinin". Pacific Symposium on Biocomputing: 492–503. PMC 2811693. PMID 19209725..</span></p></li><li id="quote_79"><span class="references-num">[79]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Vijay Pande (February 24, 2012). "Protein folding and viral infection". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved March 4, 2012..</span></p></li><li id="quote_80"><span class="references-num">[80]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Vijay Pande (February 27, 2012). "New methods for computational drug design". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved April 1, 2012..</span></p></li><li id="quote_81"><span class="references-num">[81]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Guha Jayachandran; M. R. Shirts; S. Park; V. S. Pande (2006). "Parallelized-Over-Parts Computation of Absolute Binding Free Energy with Docking and Molecular Dynamics". Journal of Chemical Physics. 125 (8): 084901. Bibcode:2006JChPh.125h4901J. doi:10.1063/1.2221680. PMID 16965051..</span></p></li><li id="quote_82"><span class="references-num">[82]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Pande lab. "Project 10721 Description". Folding@home. Stanford University. Archived from the original on September 21, 2012. Retrieved September 27, 2011..</span></p></li><li id="quote_83"><span class="references-num">[83]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Gregory Bowman (July 23, 2012). "Searching for new drug targets". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved September 27, 2011..</span></p></li><li id="quote_84"><span class="references-num">[84]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Gregory R. Bowman; Phillip L. Geissler (July 2012). "Equilibrium fluctuations of a single folded protein reveal a multitude of potential cryptic allosteric sites". PNAS. 109 (29): 11681–6. Bibcode:2012PNAS..10911681B. doi:10.1073/pnas.1209309109. PMC 3406870. PMID 22753506..</span></p></li><li id="quote_85"><span class="references-num">[85]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Paula M. Petrone; Christopher D. Snow; Del Lucent; Vijay S. Pande (2008). "Side-chain recognition and gating in the ribosome exit tunnel". Proceedings of the National Academy of Sciences. 105 (43): 16549–54. Bibcode:2008PNAS..10516549P. doi:10.1073/pnas.0801795105. PMC 2575457. PMID 18946046..</span></p></li><li id="quote_86"><span class="references-num">[86]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Pande lab. "Project 5765 Description". Folding@home. Stanford University. Archived from the original on September 21, 2012. Retrieved December 2, 2011..</span></p></li><li id="quote_87"><span class="references-num">[87]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">National Institute of Neurological Disorders and Stroke (21 August 2018). "Creutzfeldt-Jakob Disease Fact Sheet". NIH. Retrieved 2 March 2019..</span></p></li><li id="quote_88"><span class="references-num">[88]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Kupfer, L; Hinrichs, W; Groschup, M.H. (2009). "Prion Protein Misfolding". Current Molecular Medicine. Bentham Science Publishers. 9 (7): 826–835. doi:10.2174/156652409789105543. PMC 3330701. PMID 19860662..</span></p></li><li id="quote_89"><span class="references-num">[89]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Riek, Poland; Hornemann, Simone; Wider, Gerhard; Billeter, Martin; Glockshuber, Rudi; Wüthrich, Kurt (1996). "NMR structure of the mouse prion protein domain in PrP(121-231)". Nature. Nature Research. 382 (6587): 180–182. doi:10.1038/382180a0. PMID 8700211..</span></p></li><li id="quote_90"><span class="references-num">[90]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Ziegler, J; Sticht, H; Marx, UC; Müller, W; Rösch, P; Schwarzinger, S (2003). "CD and NMR studies of prion protein (PrP) helix1. Novel implications for its role in the PrPC--&gt;PrPSc conversion process". J Biol Chem. American Society for Biochemistry and Molecular Biology. 278 (50): 50175–81. doi:10.1074/jbc.M305234200. PMID 12952977..</span></p></li><li id="quote_91"><span class="references-num">[91]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Govaerts, Cedric; Wile, Holger; Brusiner, Stanley B.; Cohen, Fred (2004). "Evidence for assembly of prions with left-handed β-helices into trimers". Proc Natl Acad Sci USA. National Academy of Sciences. 101 (22): 8342–47. doi:10.1073/pnas.0402254101. PMC 420396. PMID 15155909..</span></p></li><li id="quote_92"><span class="references-num">[92]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Silveira, Jay; Raymond, Gregory; Hughson, Andrew; Race, Richard; Sim, Valerie; Caughey, Byron; Hayes, Stanley (2005). "The most infectious prion protein particles". Nature. Nature Research. 437 (7056): 257–261. doi:10.1038/nature03989. PMC 1513539. PMID 16148934..</span></p></li><li id="quote_93"><span class="references-num">[93]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Moore, Roger A.; Taubner, Lara M.; Priola, Suzette (2009). "Prion Protein Misfolding and Disease". Curr Opin Struct Biol. Elsevier. 19 (1): 14–22. doi:10.1016/j.sbi.2008.12.007. PMC 2674794. PMID 19157856..</span></p></li><li id="quote_94"><span class="references-num">[94]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Moore, Roger A.; Herzog, Christian; Errett, John; Kocisko, David A.; Arnold, Kevin M.; Hayes, Stanley F.; Priola, Suzette A. (2006). "Octapeptide repeat insertions increase the rate of protease-resistant prion protein formation". Protein Science. Wiley-Blackwell. 15 (3): 609–619. doi:10.1110/ps.051822606. PMC 2249780. PMID 16452616..</span></p></li><li id="quote_95"><span class="references-num">[95]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Ziegler, Jan; Viehrig, Christine; Geimer, Stefan; Rosch, Paul; Schwarzinger, Stephan (2006). "Putative aggregation initiation sites in prion protein". FEBS Letters. FEBS Press. 580 (8): 2033–40. doi:10.1016/j.febslet.2006.03.002. PMID 16545382..</span></p></li><li id="quote_96"><span class="references-num">[96]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Brusiner, Stanley (1998). "Prions". Proc Natl Acad Sci USA. National Academy of Sciences. 95 (23): 13363–83. doi:10.1073/pnas.95.23.13363. PMC 33918. PMID 9811807..</span></p></li><li id="quote_97"><span class="references-num">[97]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Vorberg, I; Groschup, MH; Pfaff, E; Priola, SA (2003). "Multiple amino acid residues within the rabbit prion protein inhibit formation of its abnormal isoform". J Virol. American Society for Microbiology. 77 (3): 2003–9. doi:10.1128/JVI.77.3.2003-2009.2003. PMC 140934. PMID 12525634..</span></p></li><li id="quote_98"><span class="references-num">[98]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Barrow, CJ; Yasuda, A; Kenny, PT; Zagorski, MG (1992). "Solution conformations and aggregational properties of synthetic amyloid beta-peptides of Alzheimer's disease. Analysis of circular dichroism spectra". J Biol Chem. American Society for Biochemistry and Molecular Biology. 225 (4): 1075–93. PMID 1613791..</span></p></li><li id="quote_99"><span class="references-num">[99]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Kuznetsov, Igor; Rackovsky, Shalom (2004). "Comparative computational analysis of prion proteins reveals two fragments with unusual structural properties and a pattern of increase in hydrophobicity associated with disease-promoting mutations". Protein Science. Wiley-Blackwell. 13 (12): 3230–44. doi:10.1110/ps.04833404. PMC 2287303. PMID 15557265..</span></p></li><li id="quote_100"><span class="references-num">[100]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Baskakov, IV; Legname, G; Baldwin, MA; Prusiner, SB; Cohen, FE (2002). "Pathway complexity of prion protein assembly into amyloid". J Biol Chem. American Society for Biochemistry and Molecular Biology. 277 (24): 21140–8. doi:10.1074/jbc.M111402200. PMID 11912192..</span></p></li><li id="quote_101"><span class="references-num">[101]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Raddick, M. Jordan; Bracey, Georgia; Gay, Pamela L.; Lintott, Chris J.; Murray, Phil; Schawinski, Kevin; Szalay, Alexander S.; Vandenberg, Jan (December 2010). "Galaxy Zoo: Exploring the Motivations of Citizen Science Volunteers". Astronomy Education Review. 9 (1). arXiv:0909.2925. doi:10.3847/AER2009036..</span></p></li><li id="quote_102"><span class="references-num">[102]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Vickie, Curtis (2018-04-20). Online citizen science and the widening of academia : distributed engagement with research and knowledge production. Cham, Switzerland. ISBN 9783319776644. OCLC 1034547418..</span></p></li><li id="quote_103"><span class="references-num">[103]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Nov, Oded; Arazy, Ofer; Anderson, David (2011). "Dusting for science: motivation and participation of digital citizen science volunteers". Proceedings of the 2011 IConference on - IConference '11. IConference '11. Seattle, Washington: ACM Press: 68–74. doi:10.1145/1940761.1940771. ISBN 9781450301213..</span></p></li><li id="quote_104"><span class="references-num">[104]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Curtis, Vickie (December 2015). "Motivation to Participate in an Online Citizen Science Game: A Study of Foldit" (PDF). Science Communication. 37 (6): 723–746. doi:10.1177/1075547015609322. ISSN 1075-5470..</span></p></li><li id="quote_105"><span class="references-num">[105]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Curtis, Vickie (2018-04-27). "Patterns of Participation and Motivation in Folding@home: The Contribution of Hardware Enthusiasts and Overclockers". Citizen Science: Theory and Practice. 3 (1): 5. doi:10.5334/cstp.109. ISSN 2057-4991..</span></p></li><li id="quote_106"><span class="references-num">[106]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Colwell, B. (March 2004). "The Zen of overclocking". Computer. 37 (3): 9–12. doi:10.1109/MC.2004.1273994. ISSN 0018-9162..</span></p></li><li id="quote_107"><span class="references-num">[107]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Kloetzer, Laure; Da Costa, Julien; Schneider, Daniel K. (2016-12-31). "Not so passive: engagement and learning in Volunteer Computing projects". Human Computation. 3 (1). doi:10.15346/hc.v3i1.4. ISSN 2330-8001..</span></p></li><li id="quote_108"><span class="references-num">[108]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Darch Peter; Carusi Annamaria (2010-09-13). "Retaining volunteers in volunteer computing projects". Philosophical Transactions of the Royal Society A: Mathematical, Physical and Engineering Sciences. 368 (1926): 4177–4192. doi:10.1098/rsta.2010.0163. PMID 20679130..</span></p></li><li id="quote_109"><span class="references-num">[109]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"2013 Member Study: Findings and Next Steps". World Community Grid..</span></p></li><li id="quote_110"><span class="references-num">[110]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Krebs, Viola (2010-01-31). "Motivations of cybervolunteers in an applied distributed computing environment: MalariaControl.net as an example". First Monday. 15 (2). doi:10.5210/fm.v15i2.2783..</span></p></li><li id="quote_111"><span class="references-num">[111]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Curtis, Vickie (2015). Online citizen science projects: an exploration of motivation, contribution and participation, PhD Thesis (PDF). United Kingdom: The Open University..</span></p></li><li id="quote_112"><span class="references-num">[112]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Vijay Pande (September 16, 2007). "Crossing the petaFLOPS barrier". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved August 28, 2011..</span></p></li><li id="quote_113"><span class="references-num">[113]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Michael Gross (2012). "Folding research recruits unconventional help". Current Biology. 22 (2): R35–R38. doi:10.1016/j.cub.2012.01.008. PMID 22389910..</span></p></li><li id="quote_114"><span class="references-num">[114]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"TOP500 List — June 2007". top500.org. Top500. June 2007. Retrieved September 20, 2012..</span></p></li><li id="quote_115"><span class="references-num">[115]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Folding@Home reach 2 Petaflops". n4g.com. HAVAmedia. May 8, 2008. Archived from the original on September 21, 2012. Retrieved September 20, 2012..</span></p></li><li id="quote_116"><span class="references-num">[116]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"3 PetaFLOP barrier". longecity.org. Longecity. August 19, 2008. Archived from the original on September 21, 2012. Retrieved September 20, 2012..</span></p></li><li id="quote_117"><span class="references-num">[117]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"NVIDIA Achieves Monumental Folding@Home Milestone With Cuda". nvidia.com. NVIDIA Corporation. August 26, 2008. Retrieved September 20, 2012..</span></p></li><li id="quote_118"><span class="references-num">[118]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Increase in 'active' PS3 folders pushes Folding@home past 4 Petaflops!". team52735.blogspot.com. Blogspot. September 29, 2008. Archived from the original on September 21, 2012. Retrieved September 20, 2012..</span></p></li><li id="quote_119"><span class="references-num">[119]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"TOP500 List — November 2008". top500.org. Top500. November 2008. Retrieved September 20, 2012..</span></p></li><li id="quote_120"><span class="references-num">[120]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Vijay Pande (February 18, 2009). "Folding@home Passes the 5 petaFLOP Mark". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved August 31, 2011..</span></p></li><li id="quote_121"><span class="references-num">[121]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Crossing the 5 petaFLOPS barrier". longecity.org. Longecity. February 18, 2009. Archived from the original on September 21, 2012. Retrieved September 20, 2012..</span></p></li><li id="quote_122"><span class="references-num">[122]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Dragan Zakic (May 2009). "Community Grid Computing — Studies in Parallel and Distributed Systems" (PDF). Massey University College of Sciences. Massey University. Archived from the original (PDF) on September 21, 2012. Retrieved September 20, 2012..</span></p></li><li id="quote_123"><span class="references-num">[123]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">William Ito. "A review of recent advances in ab initio protein folding by the Folding@home project" (PDF). Stanford University. Archived (PDF) from the original on September 22, 2012. Retrieved September 22, 2012..</span></p></li><li id="quote_124"><span class="references-num">[124]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Jesse Victors (November 10, 2011). "Six Native PetaFLOPS". Folding@home. phpBB Group. Archived from the original on September 21, 2012. Retrieved November 11, 2011..</span></p></li><li id="quote_125"><span class="references-num">[125]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Risto Kantonen (September 23, 2013). "Folding@home Stats - Google Docs". Folding@home. Retrieved September 23, 2013..</span></p></li><li id="quote_126"><span class="references-num">[126]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"100 Petaflops nearly reached". Stanford University. 2016-05-11. Retrieved 9 August 2016..</span></p></li><li id="quote_127"><span class="references-num">[127]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Pande lab (August 20, 2012). "Folding@home Points FAQ". Folding@home. Stanford University. Archived from the original (FAQ) on September 21, 2012. Retrieved July 8, 2013..</span></p></li><li id="quote_128"><span class="references-num">[128]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Pande lab (July 23, 2012). "Folding@home Passkey FAQ". Folding@home. Stanford University. Archived from the original (FAQ) on September 21, 2012. Retrieved July 8, 2013..</span></p></li><li id="quote_129"><span class="references-num">[129]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Peter Kasson (Pande lab member) (January 24, 2010). "upcoming release of SMP2 cores". Folding@home. phpBB Group. Archived from the original on September 21, 2012. Retrieved September 30, 2011..</span></p></li><li id="quote_130"><span class="references-num">[130]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Pande lab (2012). "Folding@home homepage". Folding@home. Stanford University. Archived from the original on September 21, 2012. Retrieved July 8, 2013..</span></p></li><li id="quote_131"><span class="references-num">[131]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Official Extreme Overclocking Folding@home Team Forum". forums.extremeoverclocking.com. Extreme Overclocking. Archived from the original on September 21, 2012. Retrieved September 20, 2012..</span></p></li><li id="quote_132"><span class="references-num">[132]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Adam Beberg; Daniel Ensign; Guha Jayachandran; Siraj Khaliq; Vijay Pande (2009). Folding@home: Lessons From Eight Years of Volunteer Distributed Computing (PDF). Parallel &amp; Distributed Processing, IEEE International Symposium. pp. 1–8. doi:10.1109/IPDPS.2009.5160922. ISBN 978-1-4244-3751-1. ISSN 1530-2075..</span></p></li><li id="quote_133"><span class="references-num">[133]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Norman Chan (April 6, 2009). "Help Maximum PC's Folding Team Win the Next Chimp Challenge!". Maximumpc.com. Future US, Inc. Archived from the original on September 21, 2012. Retrieved September 20, 2012..</span></p></li><li id="quote_134"><span class="references-num">[134]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Pande lab (June 11, 2012). "Folding@home SMP FAQ". Folding@home. Stanford University. Archived from the original (FAQ) on September 21, 2012. Retrieved July 8, 2013..</span></p></li><li id="quote_135"><span class="references-num">[135]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Vijay Pande (April 5, 2011). "More transparency in testing". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved October 14, 2011..</span></p></li><li id="quote_136"><span class="references-num">[136]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Bruce Borden (August 7, 2011). "Re: Gromacs Cannot Continue Further". Folding@home. phpBB Group. Archived from the original on September 21, 2012. Retrieved August 7, 2011..</span></p></li><li id="quote_137"><span class="references-num">[137]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">PantherX (October 1, 2011). "Re: Project 6803: (Run 4, Clone 66, Gen 255)". Folding@home. phpBB Group. Archived from the original on September 21, 2012. Retrieved October 9, 2011..</span></p></li><li id="quote_138"><span class="references-num">[138]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">PantherX (October 31, 2010). "Troubleshooting Bad WUs". Folding@home. phpBB Group. Archived from the original on September 21, 2012. Retrieved August 7, 2011..</span></p></li><li id="quote_139"><span class="references-num">[139]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Carsten Kutzner; David Van Der Spoel; Martin Fechner; Erik Lindahl; Udo W. Schmitt; Bert L. De Groot; Helmut Grubmüller (2007). "Speeding up parallel GROMACS on high-latency networks". Journal of Computational Chemistry. 28 (12): 2075–2084. doi:10.1002/jcc.20703. PMID 17405124..</span></p></li><li id="quote_140"><span class="references-num">[140]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Berk Hess; Carsten Kutzner; David van der Spoel; Erik Lindahl (2008). "GROMACS 4: Algorithms for Highly Efficient, Load-Balanced, and Scalable Molecular Simulation". Journal of Chemical Theory and Computation. 4 (3): 435–447. doi:10.1021/ct700301q. PMID 26620784..</span></p></li><li id="quote_141"><span class="references-num">[141]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Pande lab (August 19, 2012). "Folding@home Gromacs FAQ". Folding@home. Stanford University. Archived from the original (FAQ) on September 21, 2012. Retrieved July 8, 2013..</span></p></li><li id="quote_142"><span class="references-num">[142]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Pande lab (August 2, 2012). "Folding@home Open Source FAQ". Folding@home. Stanford University. Archived from the original (FAQ) on September 21, 2012. Retrieved July 8, 2013..</span></p></li><li id="quote_143"><span class="references-num">[143]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Pande lab (August 7, 2012). "Folding@home Frequently Asked Questions (FAQ) Index". Folding@home. Stanford University. Archived from the original on September 21, 2012. Retrieved July 8, 2013..</span></p></li><li id="quote_144"><span class="references-num">[144]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Vijay Pande (September 25, 2009). "Update on new FAH cores and clients". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved February 24, 2012..</span></p></li><li id="quote_145"><span class="references-num">[145]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">M. S. Friedrichs; P. Eastman; V. Vaidyanathan; M. Houston; S. LeGrand; A. L. Beberg; D. L. Ensign; C. M. Bruns; V. S. Pande (2009). "Accelerating Molecular Dynamic Simulation on Graphics Processing Units". Journal of Computational Chemistry. 30 (6): 864–72. doi:10.1002/jcc.21209. PMC 2724265. PMID 19191337..</span></p></li><li id="quote_146"><span class="references-num">[146]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Pande lab (August 19, 2012). "Folding@home Petaflop Initiative". Folding@home. Stanford University. Archived from the original (FAQ) on September 21, 2012. Retrieved July 8, 2013..</span></p></li><li id="quote_147"><span class="references-num">[147]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Pande lab (February 10, 2011). "Windows Uniprocessor Client Installation Guide". Folding@home. Stanford University. Archived from the original (Guide) on September 21, 2012. Retrieved July 8, 2013..</span></p></li><li id="quote_148"><span class="references-num">[148]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">PantherX (September 2, 2010). "Re: Can Folding@home damage any part of my PC?". Folding@home. phpBB Group. Archived from the original on September 21, 2012. Retrieved February 25, 2012..</span></p></li><li id="quote_149"><span class="references-num">[149]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Vijay Pande (June 17, 2009). "How does FAH code development and sysadmin get done?". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved October 14, 2011..</span></p></li><li id="quote_150"><span class="references-num">[150]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Pande lab (May 30, 2012). "Uninstalling Folding@home Guide". Folding@home. Stanford University. Archived from the original (Guide) on September 21, 2012. Retrieved July 8, 2013..</span></p></li><li id="quote_151"><span class="references-num">[151]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Folding@home developers. "FAHControl source code repository". Stanford University. Archived from the original on December 12, 2012. Retrieved October 15, 2012..</span></p></li><li id="quote_152"><span class="references-num">[152]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Pande lab. "Folding@home Distributed Computing Client". Folding@home. Stanford University. Archived from the original on September 21, 2012. Retrieved July 8, 2013..</span></p></li><li id="quote_153"><span class="references-num">[153]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Vijay Pande (June 28, 2008). "Folding@home's End User License Agreement (EULA)". Folding@home. Archived from the original on September 21, 2012. Retrieved May 15, 2012..</span></p></li><li id="quote_154"><span class="references-num">[154]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">unikuser (August 7, 2011). "FoldingAtHome". Ubuntu Documentation. help.ubuntu.com. Archived from the original on September 22, 2012. Retrieved September 22, 2012..</span></p></li><li id="quote_155"><span class="references-num">[155]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">The Case of the Modified Binaries, Josh Pitts, October 23, 2014; retrieved on 2016-11-23.</span></p></li><li id="quote_156"><span class="references-num">[156]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Fixing/Making Holes in ELF Binaries/Programs - Black Hat.</span></p></li><li id="quote_157"><span class="references-num">[157]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">probably using tools such as ERESI.</span></p></li><li id="quote_158"><span class="references-num">[158]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">x86 - Disassembling, modifying and then reassembling a Linux executable - Stack Overflow.</span></p></li><li id="quote_159"><span class="references-num">[159]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">linux - How do I add functionality to an existing binary executable? - Reverse Engineering Stack Exchange.</span></p></li><li id="quote_160"><span class="references-num">[160]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Certificate Bypass: Hiding and Executing Malware from a Digitally Signed Executable" (PDF). BlackHat.com. Deep Instinct. August 2016..</span></p></li><li id="quote_161"><span class="references-num">[161]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Phineus R. L. Markwick; J. Andrew McCammon (2011). "Studying functional dynamics in bio-molecules using accelerated molecular dynamics". Physical Chemistry Chemical Physics. 13 (45): 20053–65. Bibcode:2011PCCP...1320053M. doi:10.1039/C1CP22100K. PMID 22015376..</span></p></li><li id="quote_162"><span class="references-num">[162]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">M. R. Shirts; V. S. Pande. (2000). "Screen Savers of the World, Unite!". Science. 290 (5498): 1903–1904. doi:10.1126/science.290.5498.1903. PMID 17742054..</span></p></li><li id="quote_163"><span class="references-num">[163]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Pande lab. "Folding@Home Executive summary" (PDF). Folding@home. Stanford University. Archived (PDF) from the original on September 21, 2012. Retrieved October 4, 2011..</span></p></li><li id="quote_164"><span class="references-num">[164]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Rattledagger, Vijay Pande (April 1, 2005). "Folding@home client for BOINC in beta "soon"". Boarddigger.com. Anandtech.com. Archived from the original on September 21, 2012. Retrieved September 20, 2012..</span></p></li><li id="quote_165"><span class="references-num">[165]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Pande lab (May 30, 2012). "High Performance FAQ". Folding@home. Stanford University. Archived from the original (FAQ) on September 21, 2012. Retrieved July 8, 2013..</span></p></li><li id="quote_166"><span class="references-num">[166]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">John D. Owens; David Luebke; Naga Govindaraju; Mark Harris; Jens Krüger; Aaron Lefohn; Timothy J. Purcell (2007). "A Survey of General-Purpose Computation on Graphics Hardware". Computer Graphics Forum. 26 (1): 80–113. CiteSeerX 10.1.1.215.426. doi:10.1111/j.1467-8659.2007.01012.x..</span></p></li><li id="quote_167"><span class="references-num">[167]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">P. Eastman; V. S. Pande (2010). "OpenMM: A Hardware-Independent Framework for Molecular Simulations". Computing in Science and Engineering. 12 (4): 34–39. doi:10.1109/MCSE.2010.27. ISSN 1521-9615. PMC 4486654. PMID 26146490..</span></p></li><li id="quote_168"><span class="references-num">[168]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">I. Haque; V. S. Pande (2010). Hard Data on Soft Errors: A Large-Scale Assessment of Real-World Error Rates in GPGPU. 2010 10th IEEE/ACM International Conference on Cluster, Butt and Grid Computing (CCGrid). pp. 691–696. arXiv:0910.0505. doi:10.1109/CCGRID.2010.84. ISBN 978-1-4244-6987-1..</span></p></li><li id="quote_169"><span class="references-num">[169]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Pande lab (March 18, 2011). "ATI FAQ". Folding@home. Stanford University. Archived from the original (FAQ) on September 21, 2012. Retrieved July 8, 2013..</span></p></li><li id="quote_170"><span class="references-num">[170]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Vijay Pande (May 23, 2008). "GPU news (about GPU1, GPU2, &amp; NVIDIA support)". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved September 8, 2011..</span></p></li><li id="quote_171"><span class="references-num">[171]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Travis Desell; Anthony Waters; Malik Magdon-Ismail; Boleslaw K. Szymanski; Carlos A. Varela; Matthew Newby; Heidi Newberg; Andreas Przystawik; David Anderson (2009). "Accelerating the MilkyWay@Home volunteer computing project with GPUs". 8th International Conference on Parallel Processing and Applied Mathematics (PPAM 2009) Part I. pp. 276–288. CiteSeerX 10.1.1.158.7614. ISBN 978-3-642-14389-2..</span></p></li><li id="quote_172"><span class="references-num">[172]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Vijay Pande (April 10, 2008). "GPU2 open beta". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved September 7, 2011..</span></p></li><li id="quote_173"><span class="references-num">[173]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Vijay Pande (April 15, 2008). "Updates to the Download page/GPU2 goes live". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved September 7, 2011..</span></p></li><li id="quote_174"><span class="references-num">[174]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Vijay Pande (April 11, 2008). "GPU2 open beta going well". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved September 7, 2011..</span></p></li><li id="quote_175"><span class="references-num">[175]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Vijay Pande (April 24, 2010). "Prepping for the GPU3 rolling: new client and NVIDIA FAH GPU clients will (in the future) need CUDA 2.2 or later". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved September 8, 2011..</span></p></li><li id="quote_176"><span class="references-num">[176]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Vijay Pande (May 25, 2010). "Folding@home: Open beta release of the GPU3 client/core". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved September 7, 2011..</span></p></li><li id="quote_177"><span class="references-num">[177]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Joseph Coffland (CEO of Cauldron Development LLC &amp; lead developer at Folding@home) (October 13, 2011). "Re: FAHClient V7.1.38 released (4th Open-Beta)". Folding@home. phpBB Group. Archived from the original on September 21, 2012. Retrieved October 15, 2011..</span></p></li><li id="quote_178"><span class="references-num">[178]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"NVIDIA GPU3 Linux/Wine Headless Install Guide". Folding@home. phpBB Group. November 8, 2008. Archived from the original on September 21, 2012. Retrieved September 5, 2011..</span></p></li><li id="quote_179"><span class="references-num">[179]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Pande lab. "Client Statistics by OS". Folding@home. Stanford University. Archived from the original on November 28, 2012. Retrieved July 8, 2013..</span></p></li><li id="quote_180"><span class="references-num">[180]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Bruce Borden (June 25, 2013). "GPU FahCore_17 is now available on Windows &amp; native Linux". Folding@home. phpBB Group. Retrieved September 30, 2014..</span></p></li><li id="quote_181"><span class="references-num">[181]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Futures in Biotech 27: Folding@home at 1.3 Petaflops". Castroller.com. CastRoller. December 28, 2007. Archived from the original (Interview, webcast) on September 21, 2012. Retrieved September 20, 2012..</span></p></li><li id="quote_182"><span class="references-num">[182]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Edgar Luttmann; Daniel L. Ensign; Vishal Vaidyanathan; Mike Houston; Noam Rimon; Jeppe Øland; Guha Jayachandran; Mark Friedrichs; Vijay S. Pande (2008). "Accelerating Molecular Dynamic Simulation on the Cell processor and PlayStation 3". Journal of Computational Chemistry. 30 (2): 268–274. doi:10.1002/jcc.21054. PMID 18615421..</span></p></li><li id="quote_183"><span class="references-num">[183]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">David E. Williams (October 20, 2006). "PlayStation's serious side: Fighting disease". CNN. Archived from the original on September 21, 2012. Retrieved September 20, 2012..</span></p></li><li id="quote_184"><span class="references-num">[184]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Jerry Liao (March 23, 2007). "The Home Cure: PlayStation 3 to Help Study Causes of Cancer". mb.com. Manila Bulletin Publishing Corporation. Archived from the original on September 21, 2012. Retrieved September 20, 2012..</span></p></li><li id="quote_185"><span class="references-num">[185]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Lou Kesten, Associated Press (March 26, 2007). "Week in video-game news: 'God of War II' storms the PS2; a PS3 research project". Post-Gazette.com. Pittsburgh Post-Gazette. Archived from the original on September 21, 2012. Retrieved September 20, 2012..</span></p></li><li id="quote_186"><span class="references-num">[186]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Elaine Chow (September 18, 2008). "PS3 News Service, Life With Playstation, Now Up For Download". Gizmodo.com. Gizmodo. Archived from the original on September 21, 2012. Retrieved September 20, 2012..</span></p></li><li id="quote_187"><span class="references-num">[187]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Vijay Pande (September 18, 2008). "Life with Playstation – a new update to the FAH/PS3 client". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved February 24, 2012..</span></p></li><li id="quote_188"><span class="references-num">[188]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Pande lab (May 30, 2012). "PS3 FAQ". Folding@home. Stanford University. Archived from the original (FAQ) on September 21, 2012. Retrieved July 8, 2013..</span></p></li><li id="quote_189"><span class="references-num">[189]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Eric Lempel (October 21, 2012). "PS3 System Software Update (v4.30)". PlayStation blog. Sony. Archived from the original on October 22, 2012. Retrieved October 21, 2012..</span></p></li><li id="quote_190"><span class="references-num">[190]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Termination of Life with PlayStation". Life with PlayStation. Sony. November 6, 2012. Archived from the original on November 9, 2012. Retrieved November 8, 2012..</span></p></li><li id="quote_191"><span class="references-num">[191]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Vijay Pande (June 15, 2008). "What does the SMP core do?". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved September 7, 2011..</span></p></li><li id="quote_192"><span class="references-num">[192]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Daniel L. Ensign; Peter M. Kasson; Vijay S. Pande (2007). "Heterogeneity Even at the Speed Limit of Folding: Large-scale Molecular Dynamics Study of a Fast-folding Variant of the Villin Headpiece". Journal of Molecular Biology. 374 (3): 806–816. doi:10.1016/j.jmb.2007.09.069. PMC 3689540. PMID 17950314..</span></p></li><li id="quote_193"><span class="references-num">[193]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Vijay Pande (March 8, 2008). "New Windows client/core development (SMP and classic clients)". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved September 30, 2011..</span></p></li><li id="quote_194"><span class="references-num">[194]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Peter Kasson (Pande lab member) (July 15, 2009). "new release: extra-large work units". Folding@home. phpBB Group. Archived from the original on September 21, 2012. Retrieved October 9, 2011..</span></p></li><li id="quote_195"><span class="references-num">[195]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Vijay Pande (February 7, 2012). "Update on "bigadv-16", the new bigadv rollout". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved February 9, 2012..</span></p></li><li id="quote_196"><span class="references-num">[196]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Vijay Pande (July 2, 2011). "Change in the points system for bigadv work units". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved February 24, 2012..</span></p></li><li id="quote_197"><span class="references-num">[197]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Vijay Pande (January 15, 2014). "Revised plans for BigAdv (BA) experiment". Retrieved October 6, 2014..</span></p></li><li id="quote_198"><span class="references-num">[198]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Pande lab (March 23, 2012). "Windows (FAH V7) Installation Guide". Folding@home. Stanford University. Archived from the original (Guide) on September 21, 2012. Retrieved July 8, 2013..</span></p></li><li id="quote_199"><span class="references-num">[199]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Vijay Pande (March 29, 2011). "Client version 7 now in open beta". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved August 14, 2011..</span></p></li><li id="quote_200"><span class="references-num">[200]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Vijay Pande (March 22, 2012). "Web page revamp and v7 rollout". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved March 22, 2012..</span></p></li><li id="quote_201"><span class="references-num">[201]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Vijay Pande (March 31, 2011). "Core 16 for ATI released; also note on NVIDIA GPU support for older boards". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved September 7, 2011..</span></p></li><li id="quote_202"><span class="references-num">[202]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">aschofield and jcoffland (October 3, 2011). "Ticket #736 (Link to GPL in FAHControl)". Folding@home. Trac. Archived from the original on October 12, 2012. Retrieved October 12, 2012..</span></p></li><li id="quote_203"><span class="references-num">[203]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Pande, Vijay (24 February 2014). "Adding a completely new way to fold, directly in the browser". folding.stanford.edu. Pande Lab, Stanford University. Retrieved 13 February 2015..</span></p></li><li id="quote_204"><span class="references-num">[204]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Pande, Vijay (July 7, 2015). "First full version of our Folding@Home client for Android Mobile phones". Folding@Home. Stanford University. Retrieved May 31, 2016..</span></p></li><li id="quote_205"><span class="references-num">[205]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Folding@Home". Google Play. 2016. Retrieved May 31, 2016..</span></p></li><li id="quote_206"><span class="references-num">[206]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Lensink MF, Méndez R, Wodak SJ (December 2007). "Docking and scoring protein complexes: CAPRI 3rd Edition". Proteins. 69 (4): 704–18. doi:10.1002/prot.21804. PMID 17918726..</span></p></li><li id="quote_207"><span class="references-num">[207]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Gregory R. Bowman; Vijay S. Pande (2009). "Simulated tempering yields insight into the low-resolution Rosetta scoring function". Proteins: Structure, Function, and Bioinformatics. 74 (3): 777–88. doi:10.1002/prot.22210. PMID 18767152..</span></p></li><li id="quote_208"><span class="references-num">[208]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">G. R. Bowman and V. S. Pande (2009). Hofmann, Andreas, ed. "The Roles of Entropy and Kinetics in Structure Prediction". PLoS ONE. 4 (6): e5840. Bibcode:2009PLoSO...4.5840B. doi:10.1371/journal.pone.0005840. PMC 2688754. PMID 19513117..</span></p></li><li id="quote_209"><span class="references-num">[209]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Gen_X_Accord, Vijay Pande (June 11, 2006). "Folding@home vs. Rosetta@home". Rosetta@home forums. University of Washington. Archived from the original on September 21, 2012. Retrieved September 20, 2012..</span></p></li><li id="quote_210"><span class="references-num">[210]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Vijay Pande (October 13, 2011). "Comparison between FAH and Anton's approaches". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved February 25, 2012..</span></p></li><li id="quote_211"><span class="references-num">[211]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Thomas J. Lane; Gregory R. Bowman; Kyle A Beauchamp; Vincent Alvin Voelz; Vijay S. Pande (2011). "Markov State Model Reveals Folding and Functional Dynamics in Ultra-Long MD Trajectories". Journal of the American Chemical Society. 133 (45): 18413–9. doi:10.1021/ja207470h. PMC 3227799. PMID 21988563..</span></p></li><li id="quote_212"><span class="references-num">[212]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">David E. Shaw; et al. (2009). Millisecond-scale molecular dynamics simulations on Anton. Proceedings of the Conference on High Performance Computing Networking, Storage and Analysis. pp. 1–11. doi:10.1145/1654059.1654099. ISBN 978-1-60558-744-8..</span></p></li><li id="quote_213"><span class="references-num">[213]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">David E. Shaw; et al. (2010). "Atomic-Level Characterization of the Structural Dynamics of Proteins". Science. 330 (6002): 341–346. Bibcode:2010Sci...330..341S. doi:10.1126/science.1187409. PMID 20947758..</span></p></li><li id="quote_214"><span class="references-num">[214]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">David E. Shaw; Martin M. Deneroff; Ron O. Dror; Jeffrey S. Kuskin; Richard H. Larson; John K. Salmon; Cliff Young; Brannon Batson; Kevin J. Bowers; Jack C. Chao; Michael P. Eastwood; Joseph Gagliardo; J. P. Grossman; C. Richard Ho; Douglas J. Ierardi; et al. (2008). "Anton, A Special-Purpose Machine for Molecular Dynamics Simulation". Communications of the ACM. 51 (7): 91–97. doi:10.1145/1364782.1364802..</span></p></li><li id="quote_215"><span class="references-num">[215]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Ron O. Dror; Robert M. Dirks; J.P. Grossman; Huafeng Xu; David E. Shaw (2012). "Biomolecular Simulation: A Computational Microscope for Molecular Biology". Annual Review of Biophysics. 41: 429–52. doi:10.1146/annurev-biophys-042910-155245. PMID 22577825..</span></p></li><li id="quote_216"><span class="references-num">[216]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">TJ Lane (Pande lab member) (June 6, 2011). "Project 7610 &amp; 7611 in Beta". Folding@home. phpBB Group. Archived from the original on September 21, 2012. Retrieved February 25, 2012..</span></p></li><li id="quote_217"><span class="references-num">[217]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Pande lab. "Project 7610 Description". Folding@home. Archived from the original on September 21, 2012. Retrieved February 26, 2012..</span></p></li><li id="quote_218"><span class="references-num">[218]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Imran Haque (Pande lab member) (July 13, 2011). "Re: Are my conversion for GPU flops relativly[sic] correct?". Folding@home. phpBB Group. Archived from the original on September 21, 2012. Retrieved September 12, 2012..</span></p></li><li id="quote_219"><span class="references-num">[219]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Christopher Mims (November 8, 2010). "Why China's New Supercomputer Is Only Technically the World's Fastest". Technology Review. MIT. Archived from the original on September 21, 2012. Retrieved September 20, 2012..</span></p></li><li id="quote_220"><span class="references-num">[220]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Vijay Pande (November 9, 2008). "Re: ATI and NVIDIA stats vs. PPD numbers". Folding@home. phpBB Group. Archived from the original on September 21, 2012. Retrieved September 12, 2012..</span></p></li><li id="quote_221"><span class="references-num">[221]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Vijay Pande (November 9, 2008). "Re: ATI and NVIDIA stats vs. PPD numbers". Folding@home. phpBB Group. Archived from the original on September 21, 2012. Retrieved February 25, 2012..</span></p></li><li id="quote_222"><span class="references-num">[222]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Bruce Borden (July 12, 2011). "Re: Are my conversion for GPU flops relativly[sic] correct?". Folding@home. phpBB Group. Archived from the original on September 21, 2012. Retrieved September 12, 2012..</span></p></li></ul></div><div class="read-num">阅读 <!-- -->263</div></div><div class="right-side" id="rightSide"><div class="side" id="lemma-side"><div class="side-title">版本记录</div><ul class="side-lst"><li><p class="side-lst-txt">暂无</p></li></ul><div class="user-card userCard"></div></div><div class="side"><div class="side-event"></div></div></div></div><div class="footer-box"><div id="footer"><div class="footer-logo-wrap"><div class="footer-logo"></div><div class="footer-logo-text">知识·传播·科普</div></div><div class="footer-info">本网站内容采用<a target="_blank" href="https://web.archive.org/web/20221029033459/https://creativecommons.org/licenses/by-sa/3.0/deed.zh?tdsourcetag=s_pctim_aiomsg">CC-BY-SA 3.0</a>授权</div><div class="footer-btn-wrap"><a target="_blank" href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/help/#user_protocol">用户协议</a><a target="_blank" href="https://web.archive.org/web/20221029033459/http://www.sogou.com/docs/terms.htm?v=1">免责声明</a><a target="_blank" href="https://web.archive.org/web/20221029033459/http://corp.sogou.com/private.html">隐私政策</a><a target="_blank" href="https://web.archive.org/web/20221029033459/https://baike.sogou.com/kexue/intro.htm">关于我们</a></div></div></div><script>window.lemmaInfo ={"lemmaId":"10248","versionId":"14994961137139978","title":"Folding@home","subtitle":"","abstracts":{"paragraphId":"14994961137139988","title":"简介","versionId":"14994961137139979","lemmaId":10248,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":10145103,"name":"科学百科编辑组","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233692,"comment":null,"dependVersionId":0,"contentType":2,"content":"<p>Folding@home (FAH或F@h)是一个用于疾病研究的分布式计算项目，它模拟蛋白质折叠、计算药物设计和其他类型的分子动力学。该项目利用了志愿者拥有的数千台个人电脑的闲置处理资源，并且已经在他们的系统上安装了软件。其主要目的是确定蛋白质的折叠机制，即蛋白质到达其最终三维结构的过程，并检查蛋白质错误折叠的原因。这具有重要的学术意义，对阿尔茨海默氏病、亨廷顿氏病、多种癌症以及其他疾病的医学研究具有重要意义。在较小的程度上，Folding@home还试图预测蛋白质的最终结构，并确定其他分子如何与之相互作用，这一研究在药物设计中有一定的应用。Folding @home由斯坦福大学的潘德（Pande）实验室在维贾伊·潘德（Prof. Vijay Pande）教授的指导下开发和运营，并由世界各地的各种科学机构和研究实验室共享。<sup><a href=\"#quote_1\" class=\"kx_ref\">[1]</a></sup> </p>\n<p>该项目率先将图形处理单元(GPU)、PlayStation 3s、消息传递接口(用于多核处理器上的计算)和一些SNOY-Xperia智能手机用于分布式计算和科学研究。该项目使用统计模拟方法，这是传统计算方法的一种范式转变。<sup><a href=\"#quote_2\" class=\"kx_ref\">[2]</a></sup>作为客户机-服务器模型网络体系结构的一部分，志愿者机器各自接收模拟(工作单元)的片段并完成它们，随后将其返回到项目的数据库服务器，在那里单元被编译成一个整体模拟。志愿者可以在Folding @home网站上实时查询他们的贡献，这使得志愿者之间产生竞争力，从而鼓励他们长期参与。 </p>\n<p>Folding @home是世界上最快的计算系统之一，截至2019年6月，其运算速度约为98.7千兆位。<sup><a href=\"#quote_3\" class=\"kx_ref\">[3]</a></sup>其大规模的高性能计算网络使得研究人员能够运行计算高成本的原子级蛋白质折叠模拟，比以前长几千倍。自2000年10月1日成立以来，潘德实验室已经基于Folding @home发表了206篇科学研究论文。<sup><a href=\"#quote_4\" class=\"kx_ref\">[4]</a></sup> 该项目的仿真模拟结果与实验结果非常吻合。<sup><a href=\"#quote_5\" class=\"kx_ref\">[5]</a></sup><sup><a href=\"#quote_3\" class=\"kx_ref\">[3]</a></sup><sup><a href=\"#quote_6\" class=\"kx_ref\">[6]</a></sup> </p>","pics":null,"card":null,"references":[],"versionCount":0},"card":{"paragraphId":"14994961137139989","title":"基本信息","versionId":"14994961137139980","lemmaId":0,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":10145103,"name":"科学百科编辑组","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233692,"comment":null,"dependVersionId":0,"contentType":3,"content":"","pics":null,"card":{"cardItems":[{"key":"原作者","value":"Vijay Pande"},{"key":"开发者","value":"Pande Laboratory, Sony, Nvidia, ATI, Cauldron Development<sup class=\"reference\"></sup>"},{"key":"初始版本","value":"2000年10月1日，​19 years ago"},{"key":"稳定版本","value":"7.5.1 （2018年5月31日，​18 months ago<sup class=\"reference\"></sup> ）"},{"key":"操作系统","value":"Windows, macOS, Linux, Android (4.4 onward)"},{"key":"系统平台","value":"Cross-platform: IA-32, x86-64; ARM"},{"key":"语言","value":"English"},{"key":"类型","value":"Distributed computing"},{"key":"许可协议","value":"Mix: GPL, proprietary<sup class=\"reference\"></sup>"},{"key":"网站","value":"foldingathome.org"}]},"references":[],"versionCount":0},"categories":[{"id":1,"name":"计算机","parents":[]}],"creator":{"uid":10145103,"name":"柚子otto","pic":"https://web.archive.org/web/20221029033459/https://img02.sogoucdn.com/app/a/200698/1152_1152_1864088_20200427232849-1518843971.png","introduction":"","educations":[{"schoolName":"中国地质大学（北京）","major":"","degree":"本科","universityId":22,"universityLogo":"https://web.archive.org/web/20221029033459/https://img01.sogoucdn.com/app/a/200943/d3465c1c-6011-11e9-b353-fc4dd4f70029","majorLevel1":"理学","majorLevel2":"地理学","majorLevel3":"地图学与地理信息系统","majorLevel1Id":1,"majorLevel2Id":103,"majorLevel3Id":109,"state":"毕业","lab":"","researchField":""}],"jobs":[{"company":"搜狗","title":"产品经理"}],"works":null,"educationBrief":"中国地质大学（北京）","jobBrief":"产品经理","role":0,"roleName":null,"title":"中国地质大学（北京） · 地理学本科","professionalTitle":null,"phoneNo":null,"editable":true,"partnerId":139,"partnerIdCreateTime":1595844881,"partnerIdPoped":true},"createTime":1568622505,"editor":{"uid":10145103,"name":"柚子otto","pic":"https://web.archive.org/web/20221029033459/https://img02.sogoucdn.com/app/a/200698/1152_1152_1864088_20200427232849-1518843971.png","introduction":"","educations":[{"schoolName":"中国地质大学（北京）","major":"","degree":"本科","universityId":22,"universityLogo":"https://web.archive.org/web/20221029033459/https://img01.sogoucdn.com/app/a/200943/d3465c1c-6011-11e9-b353-fc4dd4f70029","majorLevel1":"理学","majorLevel2":"地理学","majorLevel3":"地图学与地理信息系统","majorLevel1Id":1,"majorLevel2Id":103,"majorLevel3Id":109,"state":"毕业","lab":"","researchField":""}],"jobs":[{"company":"搜狗","title":"产品经理"}],"works":null,"educationBrief":"中国地质大学（北京）","jobBrief":"产品经理","role":0,"roleName":null,"title":"中国地质大学（北京） · 地理学本科","professionalTitle":null,"phoneNo":null,"editable":true,"partnerId":139,"partnerIdCreateTime":1595844881,"partnerIdPoped":true},"editTime":1576233692,"state":1,"versionCount":1,"upNum":1,"downNum":0,"pics":[{"originalUrl":"https://web.archive.org/web/20221029033459/https://img04.sogoucdn.com/app/a/200698/sogou_science_11046?w=300&h=133&titlename=%E6%8A%98%E5%8F%A0%E5%89%8D%E5%90%8E%E7%9A%84%E4%B8%80%E6%9D%A1%E8%9B%8B%E7%99%BD%E8%B4%A8%E3%80%82%E5%AE%83%E4%B8%80%E5%BC%80%E5%A7%8B%E6%98%AF%E4%B8%8D%E7%A8%B3%E5%AE%9A%E7%9A%84%E9%9A%8F%E6%9C%BA%E8%9C%B7%E6%9B%B2%E5%BD%A2%E6%80%81%EF%BC%8C%E6%8A%98%E5%8F%A0%E4%B9%8B%E5%90%8E%E5%8F%98%E4%B8%BA%E5%AE%83%E7%9A%84%E5%A4%A9%E7%84%B6%E6%80%81%E3%80%82","url":"https://web.archive.org/web/20221029033459/https://img04.sogoucdn.com/app/a/200698/sogou_science_11046","rw":300,"rh":133,"title":"折叠前后的一条蛋白质。它一开始是不稳定的随机蜷曲形态，折叠之后变为它的天然态。","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221029033459/https://img04.sogoucdn.com/app/a/200698/sogou_science_11047?w=400&h=239&titlename=%E5%A6%82%E5%9B%BE%E6%89%80%E7%A4%BA%EF%BC%8CFolding@home%E4%BD%BF%E7%94%A8%E9%A9%AC%E5%B0%94%E7%A7%91%E5%A4%AB%E6%A8%A1%E5%9E%8B%E6%9D%A5%E6%A8%A1%E6%8B%9F%E8%9B%8B%E7%99%BD%E8%B4%A8%E4%BB%8E%E5%85%B6%E5%88%9D%E5%A7%8B%E7%9A%84%E9%9A%8F%E6%9C%BA%E8%9C%B7%E6%9B%B2%E7%8A%B6%E6%80%81%EF%BC%88%E5%B7%A6%EF%BC%89%E5%87%9D%E7%BB%93%E6%88%90%E5%85%B6%E4%B8%89%E7%BB%B4%E5%A4%A9%E7%84%B6%E6%80%81%EF%BC%88%E5%8F%B3%EF%BC%89%E8%BF%87%E7%A8%8B%E4%B8%AD%E5%8F%AF%E8%83%BD%E7%9A%84%E5%BD%A2%E7%8A%B6%E4%B8%8E%E6%8A%98%E5%8F%A0%E6%96%B9%E5%BC%8F%E3%80%82","url":"https://web.archive.org/web/20221029033459/https://img04.sogoucdn.com/app/a/200698/sogou_science_11047","rw":400,"rh":239,"title":"如图所示，Folding@home使用马尔科夫模型来模拟蛋白质从其初始的随机蜷曲状态（左）凝结成其三维天然态（右）过程中可能的形状与折叠方式。","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221029033459/https://img03.sogoucdn.com/app/a/200698/sogou_science_11048?w=200&h=134&titlename=","url":"https://web.archive.org/web/20221029033459/https://img03.sogoucdn.com/app/a/200698/sogou_science_11048","rw":200,"rh":134,"title":"","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221029033459/https://img02.sogoucdn.com/app/a/200698/sogou_science_11049?w=200&h=133&titlename=","url":"https://web.archive.org/web/20221029033459/https://img02.sogoucdn.com/app/a/200698/sogou_science_11049","rw":200,"rh":133,"title":"","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221029033459/https://img04.sogoucdn.com/app/a/200698/sogou_science_11050?w=200&h=133&titlename=","url":"https://web.archive.org/web/20221029033459/https://img04.sogoucdn.com/app/a/200698/sogou_science_11050","rw":200,"rh":133,"title":"","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221029033459/https://img02.sogoucdn.com/app/a/200698/sogou_science_11051?w=400&h=181&titlename=2004%E5%B9%B4%E5%9B%9B%E6%9C%88%E5%88%B02012%E5%B9%B4%E5%8D%81%E6%9C%88%E6%9C%9F%E9%97%B4%EF%BC%8CFolding@home%E8%AE%A1%E5%88%92%E4%B8%8E%E5%BD%93%E6%97%B6%E6%9C%80%E5%BF%AB%E7%9A%84%E8%B6%85%E7%BA%A7%E8%AE%A1%E7%AE%97%E6%9C%BA%E7%9A%84%E8%AE%A1%E7%AE%97%E8%83%BD%E5%8A%9B%E5%AF%B9%E6%AF%94%E5%9B%BE%E3%80%822007%E5%B9%B46%E8%87%B32011%E5%B9%B46%E6%9C%88%E6%9C%9F%E9%97%B4%EF%BC%8CFolding@home%E7%9A%84%E8%AE%A1%E7%AE%97%E8%83%BD%E5%8A%9B%EF%BC%88%E7%BA%A2%EF%BC%89%E8%B6%85%E8%BF%87%E4%BA%86%E4%B8%96%E7%95%8C%E5%89%8D500%E8%B6%85%E7%BA%A7%E8%AE%A1%E7%AE%97%E6%9C%BA%E7%9A%84%E7%AE%97%E5%8A%9B%EF%BC%88%E9%BB%91%EF%BC%89%E3%80%82%E4%BD%86%E6%98%AF%E4%B8%8E2011%E5%B9%B4%E5%8D%81%E4%B8%80%E6%9C%88%E7%9A%84%E6%97%A5%E6%9C%AC%E2%80%9C%E4%BA%AC%E2%80%9D%E8%B6%85%E7%BA%A7%E8%AE%A1%E7%AE%97%E6%9C%BA%E5%92%8C2012%E5%B9%B46%E6%9C%88%E7%9A%84IBM%E8%93%9D%E8%89%B2%E5%9F%BA%E5%9B%A0%E8%B6%85%E7%BA%A7%E8%AE%A1%E7%AE%97%E6%9C%BA%E7%9B%B8%E6%AF%94%EF%BC%8C%E8%AF%A5%E9%A1%B9%E7%9B%AE%E7%9A%84%E8%BF%90%E7%AE%97%E8%83%BD%E5%8A%9B%E5%B0%B1%E5%8F%98%E5%BE%97%E9%BB%AF%E7%84%B6%E5%A4%B1%E8%89%B2%E4%BA%86%5Ba%5D","url":"https://web.archive.org/web/20221029033459/https://img02.sogoucdn.com/app/a/200698/sogou_science_11051","rw":400,"rh":181,"title":"2004年四月到2012年十月期间，Folding@home计划与当时最快的超级计算机的计算能力对比图。2007年6至2011年6月期间，Folding@home的计算能力（红）超过了世界前500超级计算机的算力（黑）。但是与2011年十一月的日本“京”超级计算机和2012年6月的IBM蓝色基因超级计算机相比，该项目的运算能力就变得黯然失色了[a]","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221029033459/https://img03.sogoucdn.com/app/a/200698/sogou_science_11052?w=280&h=157&titlename=PlayStation%203%E5%B9%B3%E5%8F%B0%E7%9A%84Life%20With%20PlayStation%E5%AE%A2%E6%88%B7%E7%AB%AF%E5%B1%95%E7%A4%BA%E4%BA%86%E8%9B%8B%E7%99%BD%E8%B4%A8%E8%A2%AB%E6%8A%98%E5%8F%A0%E7%9A%84%E4%B8%89%E7%BB%B4%E5%8A%A8%E7%94%BB%E3%80%82","url":"https://web.archive.org/web/20221029033459/https://img03.sogoucdn.com/app/a/200698/sogou_science_11052","rw":280,"rh":157,"title":"PlayStation 3平台的Life With PlayStation客户端展示了蛋白质被折叠的三维动画。","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221029033459/https://img04.sogoucdn.com/app/a/200698/sogou_science_11053?w=400&h=280&titlename=%E5%9C%A8Windows%207%E4%B8%8B%E8%BF%90%E8%A1%8C%E7%9A%84V7%E5%AE%A2%E6%88%B7%E7%AB%AF%E5%88%9D%E5%A7%8B%E6%A8%A1%E5%BC%8F%E7%9A%84%E7%A4%BA%E4%BE%8B%E5%9B%BE%E3%80%82%E9%99%A4%E4%BA%86%E5%90%84%E7%A7%8D%E6%8E%A7%E4%BB%B6%E5%92%8C%E7%94%A8%E6%88%B7%E8%AF%A6%E7%BB%86%E4%BF%A1%E6%81%AF%E5%A4%96%EF%BC%8CV7%E5%AE%A2%E6%88%B7%E7%AB%AF%E8%BF%98%E5%91%88%E7%8E%B0%E4%BA%86%E5%B7%A5%E4%BD%9C%E5%8D%95%E5%85%83%E4%BF%A1%E6%81%AF%EF%BC%8C%E4%BE%8B%E5%A6%82%E5%AE%83%E7%9A%84%E7%8A%B6%E6%80%81%EF%BC%8C%E8%AE%A1%E7%AE%97%E8%BF%9B%E5%BA%A6%EF%BC%8CETA%EF%BC%8C%E4%BF%A1%E7%94%A8%E7%82%B9%EF%BC%8CID%E5%8F%B7%E5%92%8C%E6%8F%8F%E8%BF%B0%E3%80%82","url":"https://web.archive.org/web/20221029033459/https://img04.sogoucdn.com/app/a/200698/sogou_science_11053","rw":400,"rh":280,"title":"在Windows 7下运行的V7客户端初始模式的示例图。除了各种控件和用户详细信息外，V7客户端还呈现了工作单元信息，例如它的状态，计算进度，ETA，信用点，ID号和描述。","alt":null,"width":0,"height":0}],"catalogs":[{"level":1,"title":"项目意义","paragraphId":"14994961137139990","subCatalogs":null},{"level":1,"title":"生物医学研究中的应用实例","paragraphId":"14994961153917196","subCatalogs":[{"level":2,"title":"阿尔茨海默病","paragraphId":"14994961153917196","subCatalogs":null},{"level":2,"title":"亨廷顿氏病","paragraphId":"14994961153917196","subCatalogs":null},{"level":2,"title":"癌症","paragraphId":"14994961153917196","subCatalogs":null},{"level":2,"title":"成骨不全","paragraphId":"14994961153917196","subCatalogs":null},{"level":2,"title":"病毒","paragraphId":"14994961153917196","subCatalogs":null},{"level":2,"title":"药物设计","paragraphId":"14994961153917196","subCatalogs":null}]},{"level":1,"title":"生物医学研究的潜在应用","paragraphId":"14994961153917197","subCatalogs":[{"level":2,"title":"朊病毒疾病","paragraphId":"14994961153917197","subCatalogs":null}]},{"level":1,"title":"参与模式","paragraphId":"14994961153917198","subCatalogs":[{"level":2,"title":"性能","paragraphId":"14994961153917198","subCatalogs":null},{"level":2,"title":"积分","paragraphId":"14994961153917198","subCatalogs":null}]},{"level":1,"title":"软件","paragraphId":"14994961153917199","subCatalogs":[{"level":2,"title":"工作单位","paragraphId":"14994961153917199","subCatalogs":null},{"level":2,"title":"核心","paragraphId":"14994961153917199","subCatalogs":null},{"level":2,"title":"客户","paragraphId":"14994961153917199","subCatalogs":null}]},{"level":1,"title":"与其他分子模拟器的比较","paragraphId":"14994961170694412","subCatalogs":null},{"level":1,"title":"笔记","paragraphId":"14994961170694413","subCatalogs":null},{"level":1,"title":"参考文献","paragraphId":"-1","subCatalogs":null}],"paragraphs":[{"paragraphId":"14994961137139990","title":"项目意义","versionId":"14994961137139981","lemmaId":10248,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":10145103,"name":"科学百科编辑组","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233692,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p></p><p><img alt=\"\" class=\"fileimage kx_img ed_imgfloat_right\" img_height=\"133\" img_width=\"300\" titlename=\"折叠前后的一条蛋白质。它一开始是不稳定的随机蜷曲形态，折叠之后变为它的天然态。\" data-src=\"https://img04.sogoucdn.com/app/a/200698/sogou_science_11046\"> </p><p></p> \n<p>蛋白质是许多生物细胞功能的重要组成部分，几乎参与生物细胞内的所有过程。它们通常扮演催化酶的角色进行生化反应，包括细胞信号、分子运输和细胞调节。作为结构元素，一些蛋白质充当细胞的骨架和抗体，而其他蛋白质参与免疫系统。在蛋白质发挥这些作用之前，它必须折叠成一个功能性的三维结构，这一过程经常是自发的，并依赖于其氨基酸序列内的相互作用以及氨基酸与其周围环境的相互作用。蛋白质折叠是通过寻找蛋白质最有利的构象（即其天然状态）来驱动进行的。因此，理解蛋白质折叠对于理解蛋白质的功能和工作方式至关重要，被认为是计算生物学的圣杯。<sup><a href=\"#quote_7\" class=\"kx_ref\">[7]</a></sup><sup><a href=\"#quote_8\" class=\"kx_ref\">[8]</a></sup>尽管折叠发生在拥挤的细胞环境中，但它通常进展顺利。然而，由于蛋白质的化学性质或其他因素，蛋白质可能会错误折叠，也就是说，沿着错误的路径折叠并最终畸形。除非细胞机制能够破坏或重新折叠错误折叠的蛋白质，否则它们会随后聚集并导致各种衰弱性疾病。<sup><a href=\"#quote_9\" class=\"kx_ref\">[9]</a></sup>研究这些过程的实验可能在范围和原子细节上受到限制，这就导致科学家使用基于物理的计算模型，这些模型在补充实验时，会试图提供蛋白质折叠、错误折叠和聚集的更完整的图像。<sup><a href=\"#quote_10\" class=\"kx_ref\">[10]</a></sup><sup><a href=\"#quote_11\" class=\"kx_ref\">[11]</a></sup> </p>\n<p>由于蛋白质构象或构型空间的复杂性(蛋白质可以采取的一组可能形状)，以及计算能力的限制，全原子分子动力学模拟在它们可以研究的时间尺度上受到严重限制。虽然大多数蛋白质通常折叠在毫秒量级，<sup><a href=\"#quote_10\" class=\"kx_ref\">[10]</a></sup><sup><a href=\"#quote_12\" class=\"kx_ref\">[12]</a></sup>但在2010年之前，模拟只能达到纳秒到微秒的时间尺度。<sup><a href=\"#quote_5\" class=\"kx_ref\">[5]</a></sup>通用超级计算机已经被用来模拟蛋白质折叠，但是这种系统是昂贵的，并且通常由许多研究小组共享。此外，因为动力学模型中的计算是连续发生的，所以传统分子模拟对这些结构的强缩放是异常困难的。<sup><a href=\"#quote_13\" class=\"kx_ref\">[13]</a></sup><sup><a href=\"#quote_14\" class=\"kx_ref\">[14]</a></sup>此外，由于蛋白质折叠是一个随机过程，可以随时间统计变化，因此这是一个在计算上具有挑战性的综合视图折叠过程。<sup><a href=\"#quote_15\" class=\"kx_ref\">[15]</a></sup><sup><a href=\"#quote_16\" class=\"kx_ref\">[16]</a></sup> </p> \n<p></p><p><img alt=\"\" class=\"fileimage kx_img ed_imgfloat_right\" img_height=\"239\" img_width=\"400\" titlename=\"如图所示，Folding@home使用马尔科夫模型来模拟蛋白质从其初始的随机蜷曲状态（左）凝结成其三维天然态（右）过程中可能的形状与折叠方式。\" data-src=\"https://img04.sogoucdn.com/app/a/200698/sogou_science_11047\"> </p><p></p> \n<p>蛋白质折叠不是一步完成的。<sup><a href=\"#quote_9\" class=\"kx_ref\">[9]</a></sup>相反，蛋白质花费大部分折叠时间，在某些情况下接近96%，<sup><a href=\"#quote_17\" class=\"kx_ref\">[17]</a></sup>在各种中间构象状态下等待，每种状态都是蛋白质能量景观中的局部热力学自由能的最小值。通过一个被称为自适应采样的过程，这些构象被Folding@home用作一组模拟轨迹的起点。随着模拟发现更多的构象，轨迹从这些构象重新开始，并从这个循环过程中逐渐创建马尔可夫状态模型。MSM是离散时间主方程模型，它将生物分子的构象和能量景观描述为一组不同的结构以及它们之间的短暂跃迁。自适应采样马尔可夫状态模型方法显著提高了仿真的效率，因为它避免了局部能量最小值本身内部的计算，并且适合分布式计算(包括在GPUGRID上)，因为它允许短的、独立的仿真轨迹的统计聚集。<sup><a href=\"#quote_18\" class=\"kx_ref\">[18]</a></sup>构建马尔可夫状态模型所需的时间与并行模拟运行的数量成反比，即可用处理器的数量。换句话说，它实现了线性并行化，在整个串行计算时间中导致大约四个数量级的放大。一个完整的质谱可能包含成千上万个来自蛋白质相空间的样品状态(蛋白质可以采取的所有构象)以及它们之间的转换。该模型说明了折叠事件和路径(即路线)，研究人员随后可以使用动力学聚类来查看本应高度详细的模型的粗粒度表示。他们可以使用这些质谱来揭示蛋白质是如何错误折叠的，并定量比较模拟和实验。<sup><a href=\"#quote_2\" class=\"kx_ref\">[2]</a></sup><sup><a href=\"#quote_15\" class=\"kx_ref\">[15]</a></sup><sup><a href=\"#quote_19\" class=\"kx_ref\">[19]</a></sup> </p>\n<p>2000年至2010年间，蛋白质Folding@home研究的长度增加了四倍，而蛋白质折叠模拟的时间尺度增加了六个数量级。<sup><a href=\"#quote_20\" class=\"kx_ref\">[20]</a></sup> 2002年，Folding@home使用马尔可夫状态模型在几个月的时间里完成了大约一百万个CPU日的模拟，<sup><a href=\"#quote_6\" class=\"kx_ref\">[6]</a></sup>2011年，MSMs并行化了另一个需要总计1000万个CPU小时计算的模拟。<sup><a href=\"#quote_21\" class=\"kx_ref\">[21]</a></sup>2010年1月，Folding@home使用MSMs模拟慢折叠的32残基NTL9蛋白的动力学过程，时间跨度为1.52毫秒，这与实验性折叠速率预测一致，但比以前的预测长一千倍。该模型由许多单独的轨迹组成，每条轨迹都短了两个数量级，为蛋白质的能量景观提供了前所未有的细节。<sup><a href=\"#quote_2\" class=\"kx_ref\">[2]</a></sup><sup><a href=\"#quote_5\" class=\"kx_ref\">[5]</a></sup><sup><a href=\"#quote_22\" class=\"kx_ref\">[22]</a></sup> 2010年，Folding@home研究员格雷戈里·鲍曼(Gregory Bowman)因开发开源MSMBuilder软件和实现理论与实验之间的定量一致而获得美国化学学会颁发的托马斯·库恩范式转换奖（Thomas Kuhn Paradigm Shift Award）。<sup><a href=\"#quote_23\" class=\"kx_ref\">[23]</a></sup><sup><a href=\"#quote_24\" class=\"kx_ref\">[24]</a></sup> 由于他的工作，潘德获得了2012年迈克尔和凯特·巴恩青年研究员奖（Michael and Kate Bárány Award），因为他“开发了定义场和改变场的计算方法来产生蛋白质和核糖核酸折叠的领先理论模型”，<sup><a href=\"#quote_25\" class=\"kx_ref\">[25]</a></sup> 和2006年欧文·西格勒青年研究员奖(Irving Sigal Young Investigator Award)，这两项奖“激发了对整体和单分子测量意义的重新审视，使潘德博士在模拟方法学方面做出了开创性贡献。”<sup><a href=\"#quote_26\" class=\"kx_ref\">[26]</a></sup> </p>","pics":[{"originalUrl":"https://web.archive.org/web/20221029033459/https://img04.sogoucdn.com/app/a/200698/sogou_science_11046?w=300&h=133&titlename=%E6%8A%98%E5%8F%A0%E5%89%8D%E5%90%8E%E7%9A%84%E4%B8%80%E6%9D%A1%E8%9B%8B%E7%99%BD%E8%B4%A8%E3%80%82%E5%AE%83%E4%B8%80%E5%BC%80%E5%A7%8B%E6%98%AF%E4%B8%8D%E7%A8%B3%E5%AE%9A%E7%9A%84%E9%9A%8F%E6%9C%BA%E8%9C%B7%E6%9B%B2%E5%BD%A2%E6%80%81%EF%BC%8C%E6%8A%98%E5%8F%A0%E4%B9%8B%E5%90%8E%E5%8F%98%E4%B8%BA%E5%AE%83%E7%9A%84%E5%A4%A9%E7%84%B6%E6%80%81%E3%80%82","url":"https://web.archive.org/web/20221029033459/https://img04.sogoucdn.com/app/a/200698/sogou_science_11046","rw":300,"rh":133,"title":"折叠前后的一条蛋白质。它一开始是不稳定的随机蜷曲形态，折叠之后变为它的天然态。","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221029033459/https://img04.sogoucdn.com/app/a/200698/sogou_science_11047?w=400&h=239&titlename=%E5%A6%82%E5%9B%BE%E6%89%80%E7%A4%BA%EF%BC%8CFolding@home%E4%BD%BF%E7%94%A8%E9%A9%AC%E5%B0%94%E7%A7%91%E5%A4%AB%E6%A8%A1%E5%9E%8B%E6%9D%A5%E6%A8%A1%E6%8B%9F%E8%9B%8B%E7%99%BD%E8%B4%A8%E4%BB%8E%E5%85%B6%E5%88%9D%E5%A7%8B%E7%9A%84%E9%9A%8F%E6%9C%BA%E8%9C%B7%E6%9B%B2%E7%8A%B6%E6%80%81%EF%BC%88%E5%B7%A6%EF%BC%89%E5%87%9D%E7%BB%93%E6%88%90%E5%85%B6%E4%B8%89%E7%BB%B4%E5%A4%A9%E7%84%B6%E6%80%81%EF%BC%88%E5%8F%B3%EF%BC%89%E8%BF%87%E7%A8%8B%E4%B8%AD%E5%8F%AF%E8%83%BD%E7%9A%84%E5%BD%A2%E7%8A%B6%E4%B8%8E%E6%8A%98%E5%8F%A0%E6%96%B9%E5%BC%8F%E3%80%82","url":"https://web.archive.org/web/20221029033459/https://img04.sogoucdn.com/app/a/200698/sogou_science_11047","rw":400,"rh":239,"title":"如图所示，Folding@home使用马尔科夫模型来模拟蛋白质从其初始的随机蜷曲状态（左）凝结成其三维天然态（右）过程中可能的形状与折叠方式。","alt":null,"width":0,"height":0}],"card":null,"references":[],"versionCount":0},{"paragraphId":"14994961153917196","title":"生物医学研究中的应用实例","versionId":"14994961137139982","lemmaId":10248,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":10145103,"name":"科学百科编辑组","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233692,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>蛋白质错误折叠会导致多种疾病，包括阿尔茨海默病、癌症、克罗伊茨费尔特-雅各布病、囊性纤维化、亨廷顿氏病、镰状细胞贫血和二型糖尿病。<sup><a href=\"#quote_9\" class=\"kx_ref\">[9]</a></sup><sup><a href=\"#quote_27\" class=\"kx_ref\">[27]</a></sup><sup><a href=\"#quote_28\" class=\"kx_ref\">[28]</a></sup>艾滋病毒和流感等病毒引起的细胞感染也涉及细胞膜折叠事件。<sup><a href=\"#quote_29\" class=\"kx_ref\">[29]</a></sup>一旦蛋白质错误折叠得到更好的理解，就可以开发出增强细胞调节蛋白质折叠的天然能力的疗法。这种疗法包括使用工程分子来改变给定蛋白质的产量，帮助破坏错误折叠的蛋白质，或帮助折叠过程。<sup><a href=\"#quote_30\" class=\"kx_ref\">[30]</a></sup>计算分子建模和实验分析的结合有可能从根本上塑造分子医学的未来和治疗学的合理设计，<sup><a href=\"#quote_11\" class=\"kx_ref\">[11]</a></sup>例如加快和降低药物发现的成本。<sup><a href=\"#quote_31\" class=\"kx_ref\">[31]</a></sup> Folding@home的前五年的目标是在理解折叠方面取得进展，而当前的目标是理解错误折叠和相关疾病，尤其是阿尔茨海默氏症。<sup><a href=\"#quote_32\" class=\"kx_ref\">[32]</a></sup> </p>\n<p>在Folding@home上运行的模拟与实验室实验结合使用，<sup><a href=\"#quote_15\" class=\"kx_ref\">[15]</a></sup>但是研究人员可以用它们来研究体外折叠与天然细胞环境中折叠有何不同。这有利于研究折叠、错误折叠及其与难以通过实验观察到的疾病的关系。例如，2011年，Folding@home模拟核糖体出口通道内的蛋白质折叠，帮助科学家更好地理解自然限制和拥挤可能如何影响折叠过程。<sup><a href=\"#quote_33\" class=\"kx_ref\">[33]</a></sup><sup><a href=\"#quote_34\" class=\"kx_ref\">[34]</a></sup>此外，科学家通常使用化学变性剂将蛋白质从稳定的天然状态中展开。但是，通常不知道变性剂如何影响蛋白质的重折叠，并且很难通过实验确定这些变性状态是否包含可能影响折叠行为的残余结构。2010年，Folding@home利用图形处理器模拟蛋白质L的展开状态，并预测其折叠速率，与实验结果非常吻合。<sup><a href=\"#quote_35\" class=\"kx_ref\">[35]</a></sup> </p>\n<p>潘德实验室是斯坦福大学的一部分，斯坦福大学是一个非营利实体，不出售Folding@home产生的结果。<sup><a href=\"#quote_36\" class=\"kx_ref\">[36]</a></sup>该项目的大型数据集可免费供其他研究人员索取使用，有些数据集可从Folding@home网站上访问。<sup><a href=\"#quote_37\" class=\"kx_ref\">[37]</a></sup><sup><a href=\"#quote_38\" class=\"kx_ref\">[38]</a></sup>潘德实验室与其他分子动力学系统合作，如的蓝色基因超级计算机，<sup><a href=\"#quote_39\" class=\"kx_ref\">[39]</a></sup> 他们与其他研究人员共享Folding@home的关键软件，因此使Folding@home受益的算法可能有助于其他科学领域。<sup><a href=\"#quote_37\" class=\"kx_ref\">[37]</a></sup>2011年，他们发布了开源哥白尼软件（open-source Copernicus software），该软件基于Folding@home的MSM和其他并行化方法，旨在提高大型计算机集群或超级计算机上分子模拟的效率和规模。<sup><a href=\"#quote_40\" class=\"kx_ref\">[40]</a></sup><sup><a href=\"#quote_41\" class=\"kx_ref\">[41]</a></sup> Folding@home的所有科学发现的摘要在出版后都张贴在Folding@home的网站上。<sup><a href=\"#quote_4\" class=\"kx_ref\">[4]</a></sup> </p> \n<h3>阿尔茨海默病</h3> \n<p></p><p><img alt=\"\" class=\"fileimage kx_img ed_imgfloat_right\" img_height=\"134\" img_width=\"200\" titlename=\"\" data-src=\"https://img03.sogoucdn.com/app/a/200698/sogou_science_11048\"><img alt=\"\" class=\"fileimage kx_img ed_imgfloat_right\" img_height=\"133\" img_width=\"200\" titlename=\"\" data-src=\"https://img02.sogoucdn.com/app/a/200698/sogou_science_11049\"><img alt=\"\" class=\"fileimage kx_img ed_imgfloat_right\" img_height=\"133\" img_width=\"200\" titlename=\"\" data-src=\"https://img04.sogoucdn.com/app/a/200698/sogou_science_11050\"></p><p>阿尔茨海默症与大脑中β-淀粉样蛋白片段的聚集有关（右图）。研究人员使用Folding@home模拟这一聚集过程，以更好的了解这种疾病的起因。</p><p></p><p></p> \n<p>阿尔茨海默病是一种不可治愈的神经退行性疾病，最常影响老年人，占所有痴呆症病例的一半以上。其确切原因尚不清楚，但这种疾病被确定为蛋白质错误折叠疾病。阿尔茨海默病与淀粉样β肽的毒性聚集有关，该聚集是由淀粉样β错误折叠和与其他淀粉样β肽聚集在一起引起的。然后，这些β-淀粉样蛋白聚集成明显更大的老年斑，这是阿尔茨海默病的病理标志。<sup><a href=\"#quote_42\" class=\"kx_ref\">[42]</a></sup><sup><a href=\"#quote_43\" class=\"kx_ref\">[43]</a></sup><sup><a href=\"#quote_44\" class=\"kx_ref\">[44]</a></sup> 由于这些聚集体的异质性，诸如x光结晶学和核磁共振等实验方法很难表征它们的结构。此外，由于β聚集的大小和复杂性，原子模拟在计算上要求很高。<sup><a href=\"#quote_45\" class=\"kx_ref\">[45]</a></sup><sup><a href=\"#quote_46\" class=\"kx_ref\">[46]</a></sup> </p>\n<p>根据Naeem博士和法兹利（Fazili）博士在一篇文献综述文章中的观点，预防α-β聚集是开发阿尔茨海默病治疗药物的一种有前途的方法。<sup><a href=\"#quote_47\" class=\"kx_ref\">[47]</a></sup>2008年，Folding@home在几十秒的时间尺度内以原子细节模拟了β聚集的动力学。先前的研究只能模拟大约10微秒。Folding@home能够模拟比以前长六个数量级的β折叠。研究人员利用这项研究的结果来鉴定β发夹，它是结构内分子相互作用的主要来源。<sup><a href=\"#quote_48\" class=\"kx_ref\">[48]</a></sup>这项研究有助于潘德实验室为未来的聚集研究做准备，并为进一步研究寻找一种可以稳定聚集过程的小肽做准备。<sup><a href=\"#quote_45\" class=\"kx_ref\">[45]</a></sup> </p>\n<p>2008年12月，Folding@home发现了几种小的候选药物，它们似乎抑制了β-淀粉样聚集体的毒性。<sup><a href=\"#quote_49\" class=\"kx_ref\">[49]</a></sup>2010年，与蛋白质折叠机械中心密切合作，这些药物引线开始在生物组织上测试。<sup><a href=\"#quote_28\" class=\"kx_ref\">[28]</a></sup>2011年，Folding@home完成了几个α-β突变的模拟，这些突变似乎稳定了聚集物的形成，这可能有助于该疾病治疗药物的开发，并极大地有助于β-低聚物的实验核磁共振波谱研究。<sup><a href=\"#quote_46\" class=\"kx_ref\">[46]</a></sup><sup><a href=\"#quote_50\" class=\"kx_ref\">[50]</a></sup> 同年，“Folding@home”开始模拟各种β淀粉样蛋白片段，以确定各种天然酶如何影响β淀粉样蛋白的结构和折叠。<sup><a href=\"#quote_51\" class=\"kx_ref\">[51]</a></sup><sup><a href=\"#quote_52\" class=\"kx_ref\">[52]</a></sup> </p> \n<h3>亨廷顿氏病</h3> \n<p>亨廷顿氏病是一种神经退行性遗传疾病，与蛋白质错误折叠和聚集有关。亨廷顿蛋白氮末端谷氨酰胺氨基酸的过量重复导致聚集，尽管重复的行为还没有完全被理解，但它确实导致了与疾病相关的认知下降。<sup><a href=\"#quote_53\" class=\"kx_ref\">[53]</a></sup>同其他集料一样，很难通过实验确定其结构。<sup><a href=\"#quote_54\" class=\"kx_ref\">[54]</a></sup> 科学家正在使用Folding@home来研究亨廷顿蛋白聚集体的结构，并预测其形成方式，协助合理的药物设计方法来阻止该聚集体的形成。<sup><a href=\"#quote_28\" class=\"kx_ref\">[28]</a></sup>亨廷顿蛋白的N17片段加速了这种聚集，虽然已经提出了几种机制，但其在这一过程中的实质性作用仍大部分未知。<sup><a href=\"#quote_55\" class=\"kx_ref\">[55]</a></sup> Folding@home模拟了这一片段和其他片段，以阐明它们在疾病中的作用。<sup><a href=\"#quote_56\" class=\"kx_ref\">[56]</a></sup>自2008年以来，其针对阿尔茨海默病的药物设计方法已经应用于亨廷顿氏症。<sup><a href=\"#quote_28\" class=\"kx_ref\">[28]</a></sup> </p> \n<h3>癌症</h3> \n<p>一半以上的已知癌症涉及p53突变，p53是一种存在于每个细胞中的肿瘤抑制蛋白，它调节细胞周期，并在脱氧核糖核酸受损时发出细胞死亡信号。p53的特定突变可以破坏这些功能，允许异常细胞不受抑制地继续生长，导致肿瘤的发展。对这些突变的分析有助于解释p53相关癌症的根本原因。<sup><a href=\"#quote_57\" class=\"kx_ref\">[57]</a></sup> 2004年，Folding@home被用于在全原子模拟水中进行p53蛋白二聚体重折叠的第一次分子动力学研究。模拟的结果与实验观察一致，并对以前无法获得的二聚体的再折叠提供了参考。<sup><a href=\"#quote_58\" class=\"kx_ref\">[58]</a></sup>这是第一份来自分布式计算项目的关于癌症的同行评议出版物。<sup><a href=\"#quote_59\" class=\"kx_ref\">[59]</a></sup> 第二年，“Folding@home”提供了一种新的方法来鉴定对特定蛋白质的稳定性有着至关重要的氨基酸，然后用于研究p53的突变。该方法在识别致癌突变方面相当成功，并通过实验测量确定了特定突变的影响。<sup><a href=\"#quote_60\" class=\"kx_ref\">[60]</a></sup> </p>\n<p>Folding@home也用于研究蛋白质伴侣，<sup><a href=\"#quote_28\" class=\"kx_ref\">[28]</a></sup>热休克蛋白，在细胞内的极端环境中环境中，通过协助折叠其他蛋白质对细胞的存活发挥重要作用。快速生长的癌细胞依赖于特定的伴侣，一些伴侣在化疗耐药性中起着关键作用。对这些特定伴侣的抑制被视为有效化疗药物或减少癌症扩散的潜在作用模式。<sup><a href=\"#quote_61\" class=\"kx_ref\">[61]</a></sup>潘德实验室希望通过“Folding@home”并与蛋白质折叠机械中心密切合作，找到一种抑制参与癌细胞的伴侣的药物。<sup><a href=\"#quote_62\" class=\"kx_ref\">[62]</a></sup>研究人员也在使用Folding@home来研究其他与癌症相关的分子，如Src激酶和某些形式的植入同源域:一种可能与包括癌症在内的许多疾病有关的大蛋白质。<sup><a href=\"#quote_63\" class=\"kx_ref\">[63]</a></sup><sup><a href=\"#quote_64\" class=\"kx_ref\">[64]</a></sup> 2011年，Folding@home开始模拟小打结蛋白EETI的动力学，这种蛋白可以通过结合癌细胞的表面受体在成像扫描中识别癌症。<sup><a href=\"#quote_65\" class=\"kx_ref\">[65]</a></sup><sup><a href=\"#quote_66\" class=\"kx_ref\">[66]</a></sup> </p>\n<p>白细胞介素2(il2)是一种帮助免疫系统的T细胞攻击病原体和肿瘤的蛋白质。然而，由于严重的副作用，如肺水肿，其作为癌症治疗的用途受到限制。白细胞介素-2与这些肺细胞的结合不同于与T细胞的结合，因此白细胞介素-2的研究包括了解这些结合机制之间的差异。2012年，Folding@home协助发现了一种白细胞介素-2的突变形式，这种突变形式的白细胞介素-2在免疫系统中的作用是原来的三百倍，但副作用更少。在实验中，这种改变的形式在阻止肿瘤生长方面明显优于天然白细胞介素-2。制药公司已经表达了对这种突变分子的兴趣，国家卫生研究院正在用多种肿瘤模型对其进行测试，试图加速其作为治疗药物的发展。<sup><a href=\"#quote_67\" class=\"kx_ref\">[67]</a></sup><sup><a href=\"#quote_68\" class=\"kx_ref\">[68]</a></sup> </p> \n<h3>成骨不全</h3> \n<p>成骨不全，被称为脆性骨病，是一种不可治愈的遗传性骨病，可以致命。患有这种疾病的人不能制造功能性结缔组织。这最常见的原因是一型胶原蛋白的突变，<sup><a href=\"#quote_69\" class=\"kx_ref\">[69]</a></sup> 该蛋白具有多种结构功能，是哺乳动物中最丰富的蛋白质。<sup><a href=\"#quote_70\" class=\"kx_ref\">[70]</a></sup>这种突变导致胶原蛋白的三螺旋结构变形，如果不自然破坏，就会导致骨组织异常和弱化。<sup><a href=\"#quote_71\" class=\"kx_ref\">[71]</a></sup>2005年，Folding@home测试了一种新的量子力学方法，这种方法对以前的模拟方法进行了改进，可能对未来胶原蛋白的计算研究有用。<sup><a href=\"#quote_72\" class=\"kx_ref\">[72]</a></sup>尽管研究人员已经使用Folding@home来研究胶原折叠和错误折叠，但与阿尔茨海默氏症和亨廷顿氏症的研究相比，这种兴趣只是一个试点项目。<sup><a href=\"#quote_28\" class=\"kx_ref\">[28]</a></sup> </p> \n<h3>病毒</h3> \n<p>Folding@home正在协助防止流感和艾滋病毒等病毒识别和进入生物细胞的研究。<sup><a href=\"#quote_28\" class=\"kx_ref\">[28]</a></sup>2011年，“Folding@home”开始模拟核糖核酸酶H的动态，核糖核酸酶H是艾滋病毒的一个关键成分，试图设计药物使其失活。<sup><a href=\"#quote_73\" class=\"kx_ref\">[73]</a></sup> Folding@home也被用于研究膜融合，膜融合是病毒感染的重要事件，具有广泛的生物学功能。这种融合涉及病毒融合蛋白的构象变化和蛋白对接，<sup><a href=\"#quote_29\" class=\"kx_ref\">[29]</a></sup>但融合背后的确切分子机制仍大部分未知。<sup><a href=\"#quote_74\" class=\"kx_ref\">[74]</a></sup>聚变事件可能由50多万个原子相互作用数百微秒组成。这种复杂性将典型的计算机模拟限制在几十纳秒内大约一万个原子:相差几个数量级。<sup><a href=\"#quote_48\" class=\"kx_ref\">[48]</a></sup>预测膜融合机制的模型的开发将有助于科学理解如何用抗病毒药物靶向该过程。<sup><a href=\"#quote_75\" class=\"kx_ref\">[75]</a></sup>2006年，科学家应用马尔可夫状态模型和Folding@home网络发现了两条融合途径，并获得了其他的机制上的见解。<sup><a href=\"#quote_48\" class=\"kx_ref\">[48]</a></sup> </p>\n<p>2007年，在折叠小细胞之家囊泡的详细模拟之后，潘德实验室引入了一种新的计算方法来测量融合过程中其结构变化的拓扑结构。<sup><a href=\"#quote_76\" class=\"kx_ref\">[76]</a></sup>2009年，研究人员使用折叠@home来研究流感血凝素的突变，流感血凝素是一种将病毒附着于宿主细胞并帮助病毒进入的蛋白质。血凝素的突变影响蛋白质与宿主细胞表面受体分子的结合程度，这决定了病毒株对宿主有机体的感染性。血凝素突变作用的知识有助于抗病毒药物的开发。<sup><a href=\"#quote_77\" class=\"kx_ref\">[77]</a></sup><sup><a href=\"#quote_78\" class=\"kx_ref\">[78]</a></sup>截至2012年，Folding@home继续模拟血凝素的折叠和相互作用，补充了弗吉尼亚大学的实验研究。<sup><a href=\"#quote_28\" class=\"kx_ref\">[28]</a></sup><sup><a href=\"#quote_79\" class=\"kx_ref\">[79]</a></sup> </p> \n<h3>药物设计</h3> \n<p>药物通过结合目标分子上的特定位置而发挥作用，并引起一些期望的变化，例如使目标失效或引起构象变化。理想情况下，一种药物应该非常特异地发挥作用，并且只与其靶标结合，而不干扰其他生物功能。然而，很难精确确定两个分子结合的位置和紧密程度。由于计算能力的限制，目前的电子方法通常必须以速度换取准确性；例如使用快速蛋白质对接方法，而不是计算成本高的自由能计算。Folding@home的计算性能允许研究人员使用这两种方法，并评估它们的效率和可靠性。<sup><a href=\"#quote_32\" class=\"kx_ref\">[32]</a></sup><sup><a href=\"#quote_80\" class=\"kx_ref\">[80]</a></sup><sup><a href=\"#quote_81\" class=\"kx_ref\">[81]</a></sup>计算机辅助药物设计有可能加快和降低药物发现的成本。<sup><a href=\"#quote_31\" class=\"kx_ref\">[31]</a></sup>2010年，Folding@home使用质谱和自由能计算来预测绒毛蛋白的天然状态，其与通过x光晶体学实验确定的晶体结构的均方根偏差(RMSD)在1.8埃以内。这种准确性对未来的蛋白质结构包括内在非结构蛋白质的预测方法有影响。<sup><a href=\"#quote_48\" class=\"kx_ref\">[48]</a></sup>科学家利用折叠技术研究耐药性，研究万可霉素和β-内酰胺酶。万可霉素是最后一种抗生素，β-内酰胺酶是一种能分解青霉素等抗生素的蛋白质。<sup><a href=\"#quote_82\" class=\"kx_ref\">[82]</a></sup><sup><a href=\"#quote_83\" class=\"kx_ref\">[83]</a></sup> </p>\n<p>化学活性发生在蛋白质的活性部位。传统的药物设计方法包括紧密结合该位点并阻断其活性，假设靶蛋白存在于一个刚性结构中。然而，这种方法只适用于大约15%的蛋白质。蛋白质含有变构位点，当被小分子结合时，可以改变蛋白质的构象，最终影响蛋白质的活性。这些位点是有吸引力的药物靶点，但是定位它们的计算成本非常高。2012年，Folding@home和MSMs被用于鉴定三种医学相关蛋白质的变构位点:β-内酰胺酶、白细胞介素-2和核糖核酸酶h。<sup><a href=\"#quote_83\" class=\"kx_ref\">[83]</a></sup><sup><a href=\"#quote_84\" class=\"kx_ref\">[84]</a></sup> </p>\n<p>大约一半的已知抗生素会干扰细菌核糖体的工作，核糖体是一种大型复杂的生化机器，通过将信使核糖核酸转化为蛋白质来进行蛋白质生物合成。大环内酯类抗生素堵塞核糖体的出口通道，阻止必需细菌蛋白质的合成。2007年，潘德实验室获得了研究和设计新抗生素的资助。<sup><a href=\"#quote_28\" class=\"kx_ref\">[28]</a></sup>2008年，他们使用Folding@home来研究这个隧道的内部以及特定分子可能如何影响它。<sup><a href=\"#quote_85\" class=\"kx_ref\">[85]</a></sup>核糖体的完整结构直到2011年才被确定，Folding@home也模拟了核糖体蛋白，因为它们的许多功能仍大部分未知。<sup><a href=\"#quote_86\" class=\"kx_ref\">[86]</a></sup> </p>","pics":[{"originalUrl":"https://web.archive.org/web/20221029033459/https://img03.sogoucdn.com/app/a/200698/sogou_science_11048?w=200&h=134&titlename=","url":"https://web.archive.org/web/20221029033459/https://img03.sogoucdn.com/app/a/200698/sogou_science_11048","rw":200,"rh":134,"title":"","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221029033459/https://img02.sogoucdn.com/app/a/200698/sogou_science_11049?w=200&h=133&titlename=","url":"https://web.archive.org/web/20221029033459/https://img02.sogoucdn.com/app/a/200698/sogou_science_11049","rw":200,"rh":133,"title":"","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221029033459/https://img04.sogoucdn.com/app/a/200698/sogou_science_11050?w=200&h=133&titlename=","url":"https://web.archive.org/web/20221029033459/https://img04.sogoucdn.com/app/a/200698/sogou_science_11050","rw":200,"rh":133,"title":"","alt":null,"width":0,"height":0}],"card":null,"references":[],"versionCount":0},{"paragraphId":"14994961153917197","title":"生物医学研究的潜在应用","versionId":"14994961137139983","lemmaId":10248,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":10145103,"name":"科学百科编辑组","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233692,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>有更多蛋白质错误折叠促进的疾病可以从Folding@home中受益，从而识别错误折叠的蛋白质结构或错误折叠动力学，并有助于未来的药物设计。通常致命的朊病毒疾病是最重要的疾病之一。 </p> \n<h3>朊病毒疾病</h3> \n<p>朊病毒(PrP)是一种广泛存在于真核细胞中的跨膜蛋白。在哺乳动物中，它在中枢神经系统中更丰富。尽管它的功能未知，但它在物种间的高度保守性表明它在细胞功能中起着重要作用。从正常朊病毒蛋白(PrPc，代表细胞)到引起同种型PrPSc(代表原型朊病毒疾病——羊瘙痒病)的疾病的构象变化，导致了一系列被统称为传染性海绵状脑病的疾病，包括牛海绵状脑病、克罗伊茨费尔特-雅各布病(CJD)和人类致命失眠、鹿家族慢性消耗性疾病(CWD)。构象变化被广泛认为是蛋白质错误折叠的结果。TSEs与其他蛋白质错误折叠疾病的区别在于其可传播性。传染性PrPSc的“播种”，无论是自发产生的、遗传的还是通过暴露于受污染的组织而获得的，<sup><a href=\"#quote_87\" class=\"kx_ref\">[87]</a></sup>都可以引起将正常PrPSc转化为由PrPsC组成的纤维聚集体或淀粉样斑块的连锁反应。<sup><a href=\"#quote_88\" class=\"kx_ref\">[88]</a></sup> </p>\n<p>PrPSc的分子结构由于其聚集性质尚未完全表征。对蛋白质错误折叠的机制和动力学都知之甚少。利用PrPc的已知结构和下文所述的体外和体内研究结果，Folding@home可能有助于阐明PrPSc是如何形成的，以及感染性蛋白质是如何排列自身形成原纤维和淀粉样斑块的，绕过了纯化PrPSc或溶解聚集体的要求。 </p>\n<p>PrPc已从膜上酶促分离并纯化，其结构用结构表征技术如核磁共振波谱和x光结晶学研究。翻译后PrPc在小鼠体内有231个氨基酸(aa)。该分子由一个长达aa残基121的长而非结构化的氨基末端区域和一个结构化的羧基末端结构域组成。<sup><a href=\"#quote_88\" class=\"kx_ref\">[88]</a></sup>这个球状结构域包含两条短片状反平行β链(鼠PrPc中aa 128-130和aa 160-162)和三条α螺旋(螺旋ⅰ:aa 143-153；螺旋二:aa 171至192；螺旋ⅲ:鼠PrPc中的aa 199至226)，<sup><a href=\"#quote_89\" class=\"kx_ref\">[89]</a></sup>螺旋ⅱ和ⅲ是反平行取向的，并通过短环连接。它们的结构稳定性由平行于两个成片β链的二硫键支撑。这些α螺旋和β片形成PrPc球状畴的刚性核。<sup><a href=\"#quote_90\" class=\"kx_ref\">[90]</a></sup> </p>\n<p>引起PrPSc的疾病是蛋白酶K抗性的和不溶性的。试图从受感染动物的大脑中纯化它总是产生不均匀的混合物和聚集状态，这些不适合核磁共振波谱或x光晶体学的表征。然而，人们普遍认为PrPSc比正常PrPsC含有更高百分比的紧密堆积的β-片，这使得蛋白不溶并对蛋白酶有抗性。使用冷冻电子显微镜技术和基于相似普通蛋白质结构的结构建模技术，已经发现PrPSc在aa 81-95至aa 171区域包含β-片，而羧基末端结构被假定保留，在正常PrPc中保留二硫键连接的α-螺旋构象。这些β片形成平行的左旋β螺旋。<sup><a href=\"#quote_88\" class=\"kx_ref\">[88]</a></sup>三种PrPSc分子被认为形成了一个初级单位，因此为所谓的羊瘙痒病相关原纤维奠定了基础。<sup><a href=\"#quote_91\" class=\"kx_ref\">[91]</a></sup>催化活性取决于颗粒的大小。仅由14-28个PrPc分子组成的PrPSc颗粒显示出最高的传染性和转化率。<sup><a href=\"#quote_92\" class=\"kx_ref\">[92]</a></sup> </p>\n<p>尽管纯化和鉴定PrPSc有困难，但是从PrPSc的已知分子结构和使用转基因小鼠和N-末端缺失，<sup><a href=\"#quote_93\" class=\"kx_ref\">[93]</a></sup> 可以推断导致致病PrPSc的蛋白质错误折叠的潜在原因， Folding@home在证实这些方面可能有很大价值。研究发现朊病毒蛋白的一级和二级结构对转化都有重要意义。 </p>\n<p>朊病毒蛋白基因(PRNP)有20多个突变，已知与人类TSEs的遗传形式相关或直接相关，表明PrPc的某些位置(可能在羧基结构域内）<sup><a href=\"#quote_89\" class=\"kx_ref\">[89]</a></sup> 的单个氨基酸可以影响TSEs的易感性。 </p>\n<p>PrPc翻译后氨基末端区域由残基23-120组成，这些残基构成全长成熟PrPc氨基序列的近一半。氨基末端区域有两个部分可能影响转化。首先，残基52-90包含八肽重复(5次)区域，其可能影响初始结合(通过八肽重复)以及通过aa 108-124的第二部分的实际转化。<sup><a href=\"#quote_94\" class=\"kx_ref\">[94]</a></sup>高度疏水的AGAAAAGA位于aa残基113和120之间，被描述为推定的聚集位点，<sup>[98]</sup>尽管该序列需要其侧翼部分形成原纤维聚集体。<sup><a href=\"#quote_95\" class=\"kx_ref\">[95]</a></sup> </p>\n<p>在羧基球状结构域中，<sup><a href=\"#quote_90\" class=\"kx_ref\">[90]</a></sup>在三个螺旋中，研究表明螺旋ⅱ具有显著更高的β链构象倾向。<sup><a href=\"#quote_96\" class=\"kx_ref\">[96]</a></sup>由于残基114-125(非结构化氮末端链的一部分)之间的高构象柔性和螺旋ⅱ的高β链倾向，只有环境条件或相互作用的适度变化才足以诱导PrPc的错误折叠和随后的原纤维形成。<sup><a href=\"#quote_88\" class=\"kx_ref\">[88]</a></sup> </p>\n<p>PrPc核磁共振结构的其他研究表明，这些残基(~ 108–189)包含大部分折叠结构域，包括β链、前两个α螺旋和连接它们的环/转区，但不包括螺旋ⅲ。<sup><a href=\"#quote_93\" class=\"kx_ref\">[93]</a></sup>PrPC本身的环路/转弯结构中的微小变化在转换中也很重要。<sup><a href=\"#quote_97\" class=\"kx_ref\">[97]</a></sup>在另一项研究中，里克等人（Riek et al）表明，环区上游的两个小β链区充当了PrPc中的环/匝和α螺旋结构向β片构象转化的成核位点。<sup><a href=\"#quote_89\" class=\"kx_ref\">[89]</a></sup> </p>\n<p>转换的能量阈值不一定很高。折叠稳定性，即球状蛋白质在其环境中的自由能在一个或两个氢键的范围内，因此允许向异构体的转变而不需要高的转变能。<sup><a href=\"#quote_88\" class=\"kx_ref\">[88]</a></sup> </p>\n<p>从PrPc分子之间的相互作用来看，疏水相互作用在β-片的形成中起着至关重要的作用，β-片是PrPc的标志，因为β-片使多肽链片段非常接近。<sup><a href=\"#quote_98\" class=\"kx_ref\">[98]</a></sup>事实上，库兹涅佐夫和拉科夫斯基（Kutznetsov and Rackovsky） <sup><a href=\"#quote_99\" class=\"kx_ref\">[99]</a></sup>表明人类PrPc中促进疾病的突变具有显著增加局部疏水性的统计学趋势。 </p>\n<p>体外实验表明，错误折叠的动力学有一个初始滞后期，随后是纤维形成的快速生长期。<sup><a href=\"#quote_100\" class=\"kx_ref\">[100]</a></sup> PrPC在最终成为淀粉样原纤维的一部分之前，可能会经历一些部分展开或降解的中间状态。<sup><a href=\"#quote_88\" class=\"kx_ref\">[88]</a></sup> </p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14994961153917198","title":"参与模式","versionId":"14994961137139984","lemmaId":10248,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":10145103,"name":"科学百科编辑组","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233692,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>像其他分布式计算项目一样，Folding@home是一个在线公民科学项目。在这些项目中，非专家贡献计算机处理能力或帮助分析专业科学家产生的数据。这些项目的参与者在促进研究方面发挥着宝贵的作用，只获得很少或不是很明显的回报。 </p>\n<p>对公民科学家的动机进行了研究，这些研究中的大多数发现，参与者之所以有动机参与是因为利他主义的原因，也就是说，他们希望帮助科学家并为他们的研究进步做出贡献。<sup><a href=\"#quote_101\" class=\"kx_ref\">[101]</a></sup><sup><a href=\"#quote_102\" class=\"kx_ref\">[102]</a></sup><sup><a href=\"#quote_103\" class=\"kx_ref\">[103]</a></sup><sup><a href=\"#quote_104\" class=\"kx_ref\">[104]</a></sup>公民科学的许多参与者对研究主题有着潜在的兴趣，并被他们感兴趣的学科项目所吸引。Folding@home在这方面没有什么不同。<sup><a href=\"#quote_105\" class=\"kx_ref\">[105]</a></sup>最近对400多名积极参与者进行的研究显示，他们希望为研究做出贡献，许多人的朋友或亲戚都受到了Folding@home科学家关于疾病影响的调查。 </p>\n<p>Folding@home吸引了计算机硬件爱好者(有时被称为“超频者”)。这些小组给项目带来了相当多的专业知识，并且能够制造具有先进处理能力的计算机。<sup><a href=\"#quote_106\" class=\"kx_ref\">[106]</a></sup>其他分布式计算项目吸引了这些类型的参与者，项目通常被用来衡量改进后的计算机的性能，这方面的爱好通过项目的竞争性得到满足。个人和团队可以竞争谁能处理最多的计算机处理单元。 </p>\n<p>这项关于Folding@home的最新研究涉及在线小组的采访和人种观察，研究表明硬件爱好者团队有时可以一起工作，分享最大化处理输出的最佳实践。这样的团队可以成为实践社区，共享语言和网络文化。这种参与模式已经有应用于其他分布式计算项目中。<sup><a href=\"#quote_107\" class=\"kx_ref\">[107]</a></sup><sup><a href=\"#quote_108\" class=\"kx_ref\">[108]</a></sup> </p>\n<p>Folding@home参与者的另一个重要特征是大部分参与者为男性。<sup><a href=\"#quote_105\" class=\"kx_ref\">[105]</a></sup> 同样在其他分布式项目中也如此。此外，许多参与者是从事基于计算机和技术的工作和职业。<sup><a href=\"#quote_105\" class=\"kx_ref\">[105]</a></sup><sup><a href=\"#quote_109\" class=\"kx_ref\">[109]</a></sup><sup><a href=\"#quote_110\" class=\"kx_ref\">[110]</a></sup> </p>\n<p>并非所有的Folding@home参与者都是硬件爱好者。许多参与者在未经修改的机器上运行项目软件，并且参与竞争。超过100，000名参与者参与了Folding@home。然而，很难确定有多少参与者是硬件爱好者。尽管根据项目经理的说法，就处理能力而言，发烧友社区的贡献要大得多。<sup><a href=\"#quote_111\" class=\"kx_ref\">[111]</a></sup> </p> \n<h3>性能</h3> \n<p></p><p><img alt=\"\" class=\"fileimage kx_img ed_imgfloat_right\" img_height=\"181\" img_width=\"400\" titlename=\"2004年四月到2012年十月期间，Folding@home计划与当时最快的超级计算机的计算能力对比图。2007年6至2011年6月期间，Folding@home的计算能力（红）超过了世界前500超级计算机的算力（黑）。但是与2011年十一月的日本“京”超级计算机和2012年6月的IBM蓝色基因超级计算机相比，该项目的运算能力就变得黯然失色了[a]\" data-src=\"https://img02.sogoucdn.com/app/a/200698/sogou_science_11051\"> </p><p></p> \n<p>2007年9月16日，在很大程度上由于PlayStation 3控制台的参与，Folding@home项目正式达到了比一个本机petaFLOPS（千兆位触发器）更高的持续性能水平，成为第一个这样做的计算系统。<sup><a href=\"#quote_112\" class=\"kx_ref\">[112]</a></sup><sup><a href=\"#quote_113\" class=\"kx_ref\">[113]</a></sup>当时500强最快的超级计算机是BlueGene/L，0.280千兆次。<sup><a href=\"#quote_114\" class=\"kx_ref\">[114]</a></sup>第二年，2008年5月7日，该项目达到了持续的性能水平，高于两个petaFLOPS，<sup><a href=\"#quote_115\" class=\"kx_ref\">[115]</a></sup>随后分别在2008年8月<sup><a href=\"#quote_116\" class=\"kx_ref\">[116]</a></sup><sup><a href=\"#quote_117\" class=\"kx_ref\">[117]</a></sup>和2008年9月28日达到了三个和四个本地petaFLOPS。<sup><a href=\"#quote_118\" class=\"kx_ref\">[118]</a></sup>2009年2月18日，Folding@home实现了五个本地petaFLOPS，<sup><a href=\"#quote_119\" class=\"kx_ref\">[119]</a></sup><sup><a href=\"#quote_120\" class=\"kx_ref\">[120]</a></sup><sup><a href=\"#quote_121\" class=\"kx_ref\">[121]</a></sup>是第一个满足这五个级别的计算项目。<sup><a href=\"#quote_122\" class=\"kx_ref\">[122]</a></sup><sup><a href=\"#quote_123\" class=\"kx_ref\">[123]</a></sup>相比之下，2008年11月最快的超级计算机是IBM的Roadrunner，速度为1.105千兆次。2011年11月10日，Folding@home的性能超过了6个本地petaFLOPS，相当于近8个x86petaFLOPS。<sup><a href=\"#quote_113\" class=\"kx_ref\">[113]</a></sup><sup><a href=\"#quote_124\" class=\"kx_ref\">[124]</a></sup>2013年5月中旬，Folding@home获得了超过7个本机千兆位触发器，相当于14.87个x86千兆位触发器。随后在6月21日，它达到了8个本机千兆位触发器，随后在同年9月9日达到了9个，拥有17.9个x86千兆位触发器。<sup><a href=\"#quote_125\" class=\"kx_ref\">[125]</a></sup>2016年5月11日，Folding@home宣布，它正朝着100 x86千兆位的目标迈进。<sup><a href=\"#quote_126\" class=\"kx_ref\">[126]</a></sup> </p> \n<h3>积分</h3> \n<p>与其他分布式计算项目类似，Folding@home通过信用系统定量评估用户计算对项目的贡献。<sup><a href=\"#quote_127\" class=\"kx_ref\">[127]</a></sup>给定蛋白质项目的所有单位都有统一的基本信用，这是通过在项目发布前在官方参考机器上对该项目的一个或多个工作单位进行基准测试来确定的。<sup><a href=\"#quote_127\" class=\"kx_ref\">[127]</a></sup>每个用户都可以获得完成每个工作单元的基本分数，尽管通过使用密钥，他们可以获得可靠快速完成计算要求更高或科学优先级更高的单元的额外加分。<sup><a href=\"#quote_128\" class=\"kx_ref\">[128]</a></sup><sup><a href=\"#quote_129\" class=\"kx_ref\">[129]</a></sup>用户也可以在多台机器上获得客户对其工作的认可。<sup>[39]</sup>这一积分系统试图将授予的学分与科学成果的价值联系起来。<sup><a href=\"#quote_127\" class=\"kx_ref\">[127]</a></sup> </p>\n<p>用户可以在一个团队中注册他们的贡献，这个团队将他们所有成员的观点结合在一起。同时，用户可以创建自己的团队，也可以加入现有的团队。<sup><a href=\"#quote_130\" class=\"kx_ref\">[130]</a></sup>在某些情况下，一个团队可能有自己的社区驱动的帮助来源或招聘来源，如互联网论坛。<sup><a href=\"#quote_131\" class=\"kx_ref\">[131]</a></sup>这些积分可以促进个人和团队之间的友好竞争，从而为项目计算出最大的价值，这将有利于Folding@home并加速科学研究。<sup><a href=\"#quote_127\" class=\"kx_ref\">[127]</a></sup><sup><a href=\"#quote_132\" class=\"kx_ref\">[132]</a></sup><sup><a href=\"#quote_133\" class=\"kx_ref\">[133]</a></sup>个人和团队统计数据发布在Folding@home网站上。<sup><a href=\"#quote_127\" class=\"kx_ref\">[127]</a></sup> </p>\n<p>如果用户没有组成新团队，或者没有加入现有团队，该用户将自动成为“默认”团队的一部分。该“默认”团队的团队编号为“0”。为这个“默认”团队以及特别命名的团队积累统计数据。 </p>","pics":[{"originalUrl":"https://web.archive.org/web/20221029033459/https://img02.sogoucdn.com/app/a/200698/sogou_science_11051?w=400&h=181&titlename=2004%E5%B9%B4%E5%9B%9B%E6%9C%88%E5%88%B02012%E5%B9%B4%E5%8D%81%E6%9C%88%E6%9C%9F%E9%97%B4%EF%BC%8CFolding@home%E8%AE%A1%E5%88%92%E4%B8%8E%E5%BD%93%E6%97%B6%E6%9C%80%E5%BF%AB%E7%9A%84%E8%B6%85%E7%BA%A7%E8%AE%A1%E7%AE%97%E6%9C%BA%E7%9A%84%E8%AE%A1%E7%AE%97%E8%83%BD%E5%8A%9B%E5%AF%B9%E6%AF%94%E5%9B%BE%E3%80%822007%E5%B9%B46%E8%87%B32011%E5%B9%B46%E6%9C%88%E6%9C%9F%E9%97%B4%EF%BC%8CFolding@home%E7%9A%84%E8%AE%A1%E7%AE%97%E8%83%BD%E5%8A%9B%EF%BC%88%E7%BA%A2%EF%BC%89%E8%B6%85%E8%BF%87%E4%BA%86%E4%B8%96%E7%95%8C%E5%89%8D500%E8%B6%85%E7%BA%A7%E8%AE%A1%E7%AE%97%E6%9C%BA%E7%9A%84%E7%AE%97%E5%8A%9B%EF%BC%88%E9%BB%91%EF%BC%89%E3%80%82%E4%BD%86%E6%98%AF%E4%B8%8E2011%E5%B9%B4%E5%8D%81%E4%B8%80%E6%9C%88%E7%9A%84%E6%97%A5%E6%9C%AC%E2%80%9C%E4%BA%AC%E2%80%9D%E8%B6%85%E7%BA%A7%E8%AE%A1%E7%AE%97%E6%9C%BA%E5%92%8C2012%E5%B9%B46%E6%9C%88%E7%9A%84IBM%E8%93%9D%E8%89%B2%E5%9F%BA%E5%9B%A0%E8%B6%85%E7%BA%A7%E8%AE%A1%E7%AE%97%E6%9C%BA%E7%9B%B8%E6%AF%94%EF%BC%8C%E8%AF%A5%E9%A1%B9%E7%9B%AE%E7%9A%84%E8%BF%90%E7%AE%97%E8%83%BD%E5%8A%9B%E5%B0%B1%E5%8F%98%E5%BE%97%E9%BB%AF%E7%84%B6%E5%A4%B1%E8%89%B2%E4%BA%86%5Ba%5D","url":"https://web.archive.org/web/20221029033459/https://img02.sogoucdn.com/app/a/200698/sogou_science_11051","rw":400,"rh":181,"title":"2004年四月到2012年十月期间，Folding@home计划与当时最快的超级计算机的计算能力对比图。2007年6至2011年6月期间，Folding@home的计算能力（红）超过了世界前500超级计算机的算力（黑）。但是与2011年十一月的日本“京”超级计算机和2012年6月的IBM蓝色基因超级计算机相比，该项目的运算能力就变得黯然失色了[a]","alt":null,"width":0,"height":0}],"card":null,"references":[],"versionCount":0},{"paragraphId":"14994961153917199","title":"软件","versionId":"14994961137139985","lemmaId":10248,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":10145103,"name":"科学百科编辑组","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233692,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>在用户端Folding@home软件包括三个主要组件:工作单元、核心和客户端。 </p> \n<h3>工作单位</h3> \n<p>工作单元是要求客户处理的蛋白质数据。工作单元是马尔可夫状态模型中状态之间模拟的一部分。志愿者的电脑下载并完全处理完工作单元后，它会返回到Folding@home服务器，然后由该服务器向志愿者发放积分。这个循环会自动重复。<sup><a href=\"#quote_132\" class=\"kx_ref\">[132]</a></sup>所有工作单位都有相关的截止日期，如果超过该截止日期，用户可能无法获得积分，该单位将自动重新颁发给另一个参与者。由于蛋白质折叠是连续发生的，并且许多工作单元是从它们的前身生成的，如果一个工作单元在一段合理的时间后没有返回，这就允许整个模拟过程正常进行。由于这些截止日期，Folding@home的最低系统要求是一个奔腾3 450兆赫的中央处理器与流式SIMD扩展(SSE)。<sup><a href=\"#quote_36\" class=\"kx_ref\">[36]</a></sup> 然而，高性能客户端的工作单元的截止时间比单处理器客户端短得多，因为科学利益的一个主要部分取决于快速完成模拟。<sup><a href=\"#quote_134\" class=\"kx_ref\">[134]</a></sup> </p>\n<p>在公开发布之前，工作单元要经历几个质量保证步骤，以防止有问题的步骤完全可用。这些测试阶段包括内部测试、测试和高级测试，直到最终完全发布。<sup><a href=\"#quote_135\" class=\"kx_ref\">[135]</a></sup> Folding@home的工作单元通常只处理一次，除非在处理过程中出现罕见的错误。如果这种情况发生在三个不同的用户身上，设备将自动退出分发。<sup><a href=\"#quote_136\" class=\"kx_ref\">[136]</a></sup><sup><a href=\"#quote_137\" class=\"kx_ref\">[137]</a></sup> Folding@home支持论坛可用于区分问题硬件和坏工作单元引起的问题。<sup><a href=\"#quote_138\" class=\"kx_ref\">[138]</a></sup> </p> \n<h3>核心</h3> \n<p>专门的分子动力学程序，称为“FahCores”，通常缩写为“Cores”，在工作单元上执行计算，作为后台进程。Folding@home的大部分内核都是基于GROOMACS，<sup><a href=\"#quote_132\" class=\"kx_ref\">[132]</a></sup> 这是最快、最流行的分子动力学软件包之一，它主要由手动优化的汇编语言代码和硬件优化组成。<sup><a href=\"#quote_139\" class=\"kx_ref\">[139]</a></sup><sup><a href=\"#quote_140\" class=\"kx_ref\">[140]</a></sup>虽然GROOMACS是开源软件，潘德实验室和GROOMACS开发者之间也有合作，但是Folding@home使用了一个开源许可证来帮助确保数据的有效性。<sup><a href=\"#quote_141\" class=\"kx_ref\">[141]</a></sup>活性较低的内核包括ProtoMol和GREEP。Folding@home使用了AMBER、CPMD、德斯蒙德和TINKER，但这些已经退役，不再服役。<sup><a href=\"#quote_142\" class=\"kx_ref\">[142]</a></sup><sup><a href=\"#quote_143\" class=\"kx_ref\">[143]</a></sup><sup><a href=\"#quote_144\" class=\"kx_ref\">[144]</a></sup>这些核中的一些执行显式溶剂化计算，其中周围的溶剂(通常是水)被逐个原子地模拟；而另一些实施隐式溶剂化方法，其中溶剂被视为数学连续体。<sup><a href=\"#quote_145\" class=\"kx_ref\">[145]</a></sup><sup><a href=\"#quote_146\" class=\"kx_ref\">[146]</a></sup>核心与客户分离，使科学方法能够自动更新，而无需客户更新。内核定期创建计算检查点，这样，如果它们被中断，就可以在启动时从该点恢复工作。<sup><a href=\"#quote_132\" class=\"kx_ref\">[132]</a></sup> </p> \n<h3>客户</h3> \n<p>Folding@home参与者在他们的个人电脑上安装一个客户端程序。用户与客户端交互，客户端在后台管理其他软件组件。通过客户端，用户可以暂停折叠过程、打开事件日志、检查工作进度或查看个人统计数据。<sup><a href=\"#quote_147\" class=\"kx_ref\">[147]</a></sup>计算机客户端以非常低的优先级在后台连续运行，使用空闲的处理能力，因此正常的计算机使用不受影响。<sup><a href=\"#quote_130\" class=\"kx_ref\">[130]</a></sup><sup><a href=\"#quote_36\" class=\"kx_ref\">[36]</a></sup>最大CPU使用量可以通过客户端设置进行调整。<sup><a href=\"#quote_147\" class=\"kx_ref\">[147]</a></sup><sup><a href=\"#quote_148\" class=\"kx_ref\">[148]</a></sup> 客户端连接到Folding@home服务器，检索工作单元，还可以下载适合客户端设置、操作系统和底层硬件架构的内核。处理后，工作单元返回到Folding@home服务器。计算机客户端是为单处理器和多核处理器系统以及图形处理单元量身定制的。每种硬件架构的多样性和强大功能使Folding@home能够及时(在几周或几个月而不是几年内)高效完成多种类型的模拟，这具有重要的科学价值。这些客户一起就允许研究人员研究以前被认为难以采用计算来解决的生物医学问题。<sup><a href=\"#quote_32\" class=\"kx_ref\">[32]</a></sup><sup><a href=\"#quote_132\" class=\"kx_ref\">[132]</a></sup><sup><a href=\"#quote_134\" class=\"kx_ref\">[134]</a></sup> </p>\n<p>专业软件开发人员负责Folding@home的大部分代码，包括客户端和服务器端。开发团队包括英伟达、ATI、索尼和大锅开发公司（Cauldron Development）的程序员。<sup><a href=\"#quote_149\" class=\"kx_ref\">[149]</a></sup>客户端只能从官方的Folding@home网站或其商业伙伴处下载，并且只能与Folding@home电脑文件交互。他们将通过斯坦福的Folding@home数据服务器上传和下载数据(通过端口8080，以80作为替代)，并使用2048位数字签名验证通信。<sup><a href=\"#quote_36\" class=\"kx_ref\">[36]</a></sup><sup><a href=\"#quote_150\" class=\"kx_ref\">[150]</a></sup> 虽然客户端的图形用户界面是开源的，<sup><a href=\"#quote_151\" class=\"kx_ref\">[151]</a></sup> 但客户端是以安全性和科学完整性为理由的专有软件。<sup><a href=\"#quote_152\" class=\"kx_ref\">[152]</a></sup><sup><a href=\"#quote_153\" class=\"kx_ref\">[153]</a></sup><sup><a href=\"#quote_154\" class=\"kx_ref\">[154]</a></sup> </p>\n<p>然而，这种使用专有软件的基本原理是有争议的，因为尽管许可证可以在法律领域追溯执行，但它实际上并不阻止对可执行二进制文件的修改(也称为修补)。同样，纯二进制分发并不能防止恶意修改可执行二进制代码，无论是通过中间人攻击，同时通过互联网下载，<sup><a href=\"#quote_155\" class=\"kx_ref\">[155]</a></sup>还是通过由第三方重新分发二进制代码，这些二进制代码先前已被修改为二进制状态(即打补丁)，<sup><a href=\"#quote_156\" class=\"kx_ref\">[156]</a></sup> 或通过反编译译<sup><a href=\"#quote_157\" class=\"kx_ref\">[157]</a></sup> 并在修改后重新编译它们。<sup><a href=\"#quote_158\" class=\"kx_ref\">[158]</a></sup><sup><a href=\"#quote_159\" class=\"kx_ref\">[159]</a></sup>除非二进制文件(和传输通道)已签名，并且接收人/系统能够验证数字签名，在这种情况下，不必要的修改应该是可检测的，但并不总是这样。<sup><a href=\"#quote_160\" class=\"kx_ref\">[160]</a></sup>无论哪种方式，由于在Folding@home的情况下，客户端软件处理的输入数据和输出结果都是数字签名的，<sup><a href=\"#quote_36\" class=\"kx_ref\">[36]</a></sup><sup><a href=\"#quote_150\" class=\"kx_ref\">[150]</a></sup> 因此可以独立于客户端软件本身的完整性来验证工作的完整性。 </p>\n<p>Folding@home使用Coms软件库进行联网。<sup><a href=\"#quote_132\" class=\"kx_ref\">[132]</a></sup><sup><a href=\"#quote_149\" class=\"kx_ref\">[149]</a></sup> Folding@home于2000年10月1日启动，是首个针对生物分子系统的分布式计算项目。<sup><a href=\"#quote_161\" class=\"kx_ref\">[161]</a></sup>它的第一个客户端是一个屏幕保护程序，它可以在电脑不用时运行。<sup><a href=\"#quote_162\" class=\"kx_ref\">[162]</a></sup><sup><a href=\"#quote_163\" class=\"kx_ref\">[163]</a></sup>2004年，潘德实验室与大卫·安德森（ David P. Anderson）合作，在开源BOINC框架上测试一个补充客户端。该客户于2005年4月被发布到封闭测试版；<sup><a href=\"#quote_164\" class=\"kx_ref\">[164]</a></sup>然而，这种方法变得行不通，并于2006年6月被搁置。<sup><a href=\"#quote_165\" class=\"kx_ref\">[165]</a></sup> </p> \n<p><strong>图形处理单元</strong></p> \n<p>图形处理单元(GPU)的专用硬件旨在加速视频游戏等三维图形应用的渲染，并且在某些类型的计算中可以显著优于图形处理单元。图形处理器是最强大、发展最快的计算平台之一，许多科学家和研究人员正在图形处理单元上进行通用计算(GPGPU)。然而，图形处理器硬件很难用于非图形任务，通常需要大量的算法重组和对底层架构的高级理解。<sup><a href=\"#quote_166\" class=\"kx_ref\">[166]</a></sup>这种定制具有挑战性，对于软件开发资源有限的研究人员来说更是如此。Folding@home使用开源OpenMM库，该库使用具有两个应用编程接口(API)级别的桥接设计模式，将分子模拟软件与底层硬件架构接口。随着硬件优化的增加，基于OpenMM的图形处理器仿真不需要进行重大修改，但实现的性能几乎等同于手动调整的图形处理器代码，并且大大优于CPU实现。<sup><a href=\"#quote_145\" class=\"kx_ref\">[145]</a></sup><sup><a href=\"#quote_167\" class=\"kx_ref\">[167]</a></sup> </p>\n<p>在2010年之前，GPGPU消费级硬件的计算可靠性在很大程度上是未知的，与GPU内存中缺乏内置错误检测和纠正相关的间接证据引发了可靠性问题。在第一次大规模的GPU科学准确性测试中，2010年对Folding@home网络上20，000多台主机的研究发现，三分之二被测试的GPU内存子系统存在软件错误。尽管研究得出结论，只要不是硬件问题，可靠的GPU计算对于诸如软件端的错误检测都是可以处理的，但这些错误与板架构实质上是密切相关。<sup><a href=\"#quote_168\" class=\"kx_ref\">[168]</a></sup> </p>\n<p>Folding@home的第一代图形处理器客户端(GPU1)于2006年10月2日在向公众发布，<sup><a href=\"#quote_165\" class=\"kx_ref\">[165]</a></sup> 其计算速度是基于中央处理器的GROMACS的同类产品的20-30倍。<sup><a href=\"#quote_169\" class=\"kx_ref\">[169]</a></sup>这是通用处理器首次用于分布式计算或主要分子动力学计算。<sup><a href=\"#quote_170\" class=\"kx_ref\">[170]</a></sup><sup><a href=\"#quote_171\" class=\"kx_ref\">[171]</a></sup> GPU1为研究人员提供了开发GPU软件的丰富知识和经验，但由于DirectX编程接口的科学误差，2008年4月10日，它被第二代客户端GPU2所取代。<sup><a href=\"#quote_169\" class=\"kx_ref\">[169]</a></sup><sup><a href=\"#quote_172\" class=\"kx_ref\">[172]</a></sup>继GPU2推出后，GPU1于6月6日正式退役。<sup><a href=\"#quote_169\" class=\"kx_ref\">[169]</a></sup>与GPU1相比，GPU2在科学上更加可靠和高效，运行在ATI和CUDA支持的Nvidia通用处理器上，支持更高级的算法、更大的蛋白质和蛋白质模拟的实时可视化。<sup><a href=\"#quote_173\" class=\"kx_ref\">[173]</a></sup><sup><a href=\"#quote_174\" class=\"kx_ref\">[174]</a></sup>随后，第三代Folding@home的GPU客户端(GPU3)于2010年5月25日发布。虽然与GPU2向后兼容，但GPU3在科学能力上更稳定、高效和灵活，<sup><a href=\"#quote_175\" class=\"kx_ref\">[175]</a></sup>并在OpenCL框架上使用OpenMM。<sup><a href=\"#quote_175\" class=\"kx_ref\">[175]</a></sup><sup><a href=\"#quote_176\" class=\"kx_ref\">[176]</a></sup> 虽然这些GPU3客户端本身不支持操作系统Linux和macOS，但拥有英伟达显卡的Linux用户能够通过Wine软件应用程序运行它们。<sup><a href=\"#quote_177\" class=\"kx_ref\">[177]</a></sup><sup><a href=\"#quote_178\" class=\"kx_ref\">[178]</a></sup> GPU仍然是Folding@home中FLOPS浮点运算的最强大平台。截至2012年11月，GPU客户端占整个项目x86 FLOPS吞吐量的87%。<sup><a href=\"#quote_179\" class=\"kx_ref\">[179]</a></sup> </p>\n<p>在Linux下对Nvidia和AMD显卡的本机支持是用FahCore 17引入的，它使用的是OpenCL而不是CUDA。<sup><a href=\"#quote_180\" class=\"kx_ref\">[180]</a></sup> </p> \n<p><strong>PlayStation 3</strong></p> \n<p></p><p><img alt=\"\" class=\"fileimage kx_img ed_imgfloat_right\" img_height=\"157\" img_width=\"280\" titlename=\"PlayStation 3平台的Life With PlayStation客户端展示了蛋白质被折叠的三维动画。\" data-src=\"https://img03.sogoucdn.com/app/a/200698/sogou_science_11052\"> </p><p></p> \n<p>PlayStation 3s的强大功能。在其成立之初，其主要流式细胞处理器在某些计算方面的速度是个人电脑的20倍，处理能力在Xbox 360等其他系统上是找不到的。<sup><a href=\"#quote_32\" class=\"kx_ref\">[32]</a></sup><sup><a href=\"#quote_181\" class=\"kx_ref\">[181]</a></sup> PS3的高速和高效率为根据阿姆达尔定律（ Amdahl's law）进行有价值的优化带来了其他机会，并极大地改变了计算效率和整体精度之间的权衡，允许以很少的额外计算成本使用更复杂的分子模型。<sup><a href=\"#quote_182\" class=\"kx_ref\">[182]</a></sup> 这使得Folding@home能够运行生物医学计算，否则这些计算是不可行的。<sup><a href=\"#quote_183\" class=\"kx_ref\">[183]</a></sup> </p>\n<p>PS3客户端是索尼和潘德实验室合作开发的，于2007年3月23日首次作为独立客户端发布。<sup><a href=\"#quote_32\" class=\"kx_ref\">[32]</a></sup><sup><a href=\"#quote_184\" class=\"kx_ref\">[184]</a></sup>它的发布使Folding@home成为第一个使用PS3的分布式计算项目。<sup><a href=\"#quote_185\" class=\"kx_ref\">[185]</a></sup>第二年9月18日，PS3客户端成为了PlayStation推出后的生活频道。<sup><a href=\"#quote_186\" class=\"kx_ref\">[186]</a></sup><sup><a href=\"#quote_187\" class=\"kx_ref\">[187]</a></sup>在推出时，客户端可以执行的计算类型介于中央处理器的灵活性和图形处理器的速度之间。<sup><a href=\"#quote_132\" class=\"kx_ref\">[132]</a></sup> 然而，与运行在个人电脑上的客户端不同，用户在运行Folding@home时无法在PS3上执行其他活动。<sup><a href=\"#quote_183\" class=\"kx_ref\">[183]</a></sup>PS3的统一控制台环境使技术支持变得更加容易，并使Folding@home更加用户友好。<sup><a href=\"#quote_32\" class=\"kx_ref\">[32]</a></sup> PS3还能够将数据快速传输到图形处理器，图形处理器用于当前蛋白质动态的实时原子级可视化。<sup><a href=\"#quote_182\" class=\"kx_ref\">[182]</a></sup> </p>\n<p>2012年11月6日，索尼终止了对Folding@home PS3客户端的支持，以及PlayStation Life提供的其他服务。在其五年零七个月的生命周期中，超过1500万用户为Folding@home贡献了超过1亿小时的计算时间，极大地帮助了该项目的疾病研究。在与潘德实验室讨论后，索尼决定终止该应用。潘德认为PlayStation 3客户端是该项目的“游戏改变者”。<sup><a href=\"#quote_188\" class=\"kx_ref\">[188]</a></sup><sup><a href=\"#quote_189\" class=\"kx_ref\">[189]</a></sup><sup><a href=\"#quote_190\" class=\"kx_ref\">[190]</a></sup> </p> \n<p><strong>多核处理客户端</strong></p> \n<p>Folding@home可以利用现代多核处理器的并行计算能力。同时使用几个中央处理器内核的能力允许更快地完成完整的模拟。这些中央处理器内核协同工作，完成单个工作单元的速度要比标准的单处理器客户端快得多。这种方法在科学上很有价值，因为它能够在相同的时间内执行更长的仿真轨迹，并且降低了将大型仿真扩展到许多独立处理器的传统性困难。<sup><a href=\"#quote_191\" class=\"kx_ref\">[191]</a></sup> 2007年发表在《分子生物学杂志》上的一篇论文依赖多核处理来模拟绒毛蛋白部分的折叠，其长度大约是单处理器客户端的10倍，与实验折叠速率一致。<sup><a href=\"#quote_192\" class=\"kx_ref\">[192]</a></sup> </p>\n<p>2006年11月，第一代对称多处理(SMP)客户端被公开发布用于开放测试，称为SMP1。<sup><a href=\"#quote_165\" class=\"kx_ref\">[165]</a></sup>这些客户端使用消息传递接口(MPI)通信协议进行并行处理，因为当时GROMACS内核不是设计用于多线程的。<sup><a href=\"#quote_134\" class=\"kx_ref\">[134]</a></sup> 这是分布式计算项目第一次使用MPI。<sup><a href=\"#quote_193\" class=\"kx_ref\">[193]</a></sup>虽然客户端在基于Unix的操作系统(如Linux和macOS)中表现良好，但在windows系统下却很麻烦。<sup><a href=\"#quote_191\" class=\"kx_ref\">[191]</a></sup><sup><a href=\"#quote_193\" class=\"kx_ref\">[193]</a></sup>2010年1月24日，第二代SMP客户端和SMP1的后继者SMP2作为开放测试版发布，并以更可靠的基于线程的实现取代复杂的MPI。<sup><a href=\"#quote_129\" class=\"kx_ref\">[129]</a></sup><sup><a href=\"#quote_149\" class=\"kx_ref\">[149]</a></sup> </p>\n<p>SMP2支持一种特殊类别的bigadv工作单元的试验，该工作单元被设计用来模拟异常大且计算密集并且具有高度科学优先权的蛋白质。这些单元最初至少需要八个中央处理器核心，<sup><a href=\"#quote_194\" class=\"kx_ref\">[194]</a></sup>后来在2012年2月7日提高到十六个。<sup><a href=\"#quote_195\" class=\"kx_ref\">[195]</a></sup>除了标准SMP2工作单元的这些额外硬件要求外，它们还需要更多的系统资源，如随机存取存储器和互联网带宽。作为回报，运行这些程序的用户将获得比SMP2系统奖励积分增加20%的奖励。<sup><a href=\"#quote_196\" class=\"kx_ref\">[196]</a></sup>bigav类别允许Folding@home长时间运行特别苛刻的模拟，这种模拟以前需要使用超级计算集群，在Folding@home上的任何其他地方都无法执行。<sup><a href=\"#quote_194\" class=\"kx_ref\">[194]</a></sup>许多拥有能够运行大adv单元的硬件的用户后来发现，当CPU核心最小值增加时，他们的硬件设置被认为不符合大adv工作单元的要求，这使得他们只能运行正常的SMP工作单元。这让许多用户感到沮丧，他们在程序中投入了大量资金，却在不久后就因为bigadv的目的而使他们的硬件过时了。因此，潘德在2014年1月宣布bigadv计划将于2015年1月31日结束。<sup><a href=\"#quote_197\" class=\"kx_ref\">[197]</a></sup> </p> \n<p><strong>V7</strong></p> \n<p></p><p><img alt=\"\" class=\"fileimage kx_img ed_imgfloat_right\" img_height=\"280\" img_width=\"400\" titlename=\"在Windows 7下运行的V7客户端初始模式的示例图。除了各种控件和用户详细信息外，V7客户端还呈现了工作单元信息，例如它的状态，计算进度，ETA，信用点，ID号和描述。\" data-src=\"https://img04.sogoucdn.com/app/a/200698/sogou_science_11053\"> </p><p></p> \n<p>V7客户端是第七代也是最新一代的Folding@home客户端软件，是对以前的Windows、macOS和Linux操作系统客户端的完全重写和统一。<sup><a href=\"#quote_198\" class=\"kx_ref\">[198]</a></sup><sup><a href=\"#quote_199\" class=\"kx_ref\">[199]</a></sup> 它于2012年3月22日发布。<sup><a href=\"#quote_200\" class=\"kx_ref\">[200]</a></sup>像它的前辈一样，V7可以在后台以非常低的优先级运行Folding@home，允许其他应用程序根据需要使用中央处理器资源。它的设计是为了让新手更容易安装、启动和操作，并为研究人员提供比以前客户更大的科学灵活性。<sup><a href=\"#quote_201\" class=\"kx_ref\">[201]</a></sup> V7使用Trac来管理它的bug票证，这样用户就可以看到它的开发过程并提供反馈。<sup><a href=\"#quote_199\" class=\"kx_ref\">[199]</a></sup> </p>\n<p>V7由四个集成元件组成。用户通常与V7的开源图形用户界面交互，名为FAHControl。<sup><a href=\"#quote_151\" class=\"kx_ref\">[151]</a></sup><sup><a href=\"#quote_202\" class=\"kx_ref\">[202]</a></sup>它具有新手、高级和专家用户界面模式，并且能够从一台计算机监控、配置和控制许多远程折叠客户端。FAHControl指导FAHClient，这是一个后端应用程序，它依次管理每个FahClient。每个插槽都可以替代以前不同的Folding@home v6单处理器、SMP或GPU计算机客户端，因为它可以独立下载、处理和上传工作单元。FAHViewer函数以PS3的浏览器为模型，显示当前正在加工的蛋白质的实时三维渲染模型 (如果有的话)。<sup><a href=\"#quote_198\" class=\"kx_ref\">[198]</a></sup><sup><a href=\"#quote_199\" class=\"kx_ref\">[199]</a></sup> </p> \n<p><strong>谷歌浏览器（<b>Google Chrome</b>）</strong></p> \n<p>2014年，谷歌浏览器和谷歌网络浏览器的客户端发布，允许用户在他们的网络浏览器中运行Folding@home。客户端在基于<b>Google Chrome</b>上使用谷歌的本地客户端(NaCl)功能，在用户机器上的沙盒中以接近本地的速度运行Folding@home代码。<sup><a href=\"#quote_203\" class=\"kx_ref\">[203]</a></sup> </p> \n<p><strong>机器人</strong></p> \n<p>2015年7月，一款安卓手机客户端在谷歌播放器上发布，适用于运行安卓4.4 KitKat或更高版本的设备。<sup><a href=\"#quote_204\" class=\"kx_ref\">[204]</a></sup><sup><a href=\"#quote_205\" class=\"kx_ref\">[205]</a></sup> </p>","pics":[{"originalUrl":"https://web.archive.org/web/20221029033459/https://img03.sogoucdn.com/app/a/200698/sogou_science_11052?w=280&h=157&titlename=PlayStation%203%E5%B9%B3%E5%8F%B0%E7%9A%84Life%20With%20PlayStation%E5%AE%A2%E6%88%B7%E7%AB%AF%E5%B1%95%E7%A4%BA%E4%BA%86%E8%9B%8B%E7%99%BD%E8%B4%A8%E8%A2%AB%E6%8A%98%E5%8F%A0%E7%9A%84%E4%B8%89%E7%BB%B4%E5%8A%A8%E7%94%BB%E3%80%82","url":"https://web.archive.org/web/20221029033459/https://img03.sogoucdn.com/app/a/200698/sogou_science_11052","rw":280,"rh":157,"title":"PlayStation 3平台的Life With PlayStation客户端展示了蛋白质被折叠的三维动画。","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221029033459/https://img04.sogoucdn.com/app/a/200698/sogou_science_11053?w=400&h=280&titlename=%E5%9C%A8Windows%207%E4%B8%8B%E8%BF%90%E8%A1%8C%E7%9A%84V7%E5%AE%A2%E6%88%B7%E7%AB%AF%E5%88%9D%E5%A7%8B%E6%A8%A1%E5%BC%8F%E7%9A%84%E7%A4%BA%E4%BE%8B%E5%9B%BE%E3%80%82%E9%99%A4%E4%BA%86%E5%90%84%E7%A7%8D%E6%8E%A7%E4%BB%B6%E5%92%8C%E7%94%A8%E6%88%B7%E8%AF%A6%E7%BB%86%E4%BF%A1%E6%81%AF%E5%A4%96%EF%BC%8CV7%E5%AE%A2%E6%88%B7%E7%AB%AF%E8%BF%98%E5%91%88%E7%8E%B0%E4%BA%86%E5%B7%A5%E4%BD%9C%E5%8D%95%E5%85%83%E4%BF%A1%E6%81%AF%EF%BC%8C%E4%BE%8B%E5%A6%82%E5%AE%83%E7%9A%84%E7%8A%B6%E6%80%81%EF%BC%8C%E8%AE%A1%E7%AE%97%E8%BF%9B%E5%BA%A6%EF%BC%8CETA%EF%BC%8C%E4%BF%A1%E7%94%A8%E7%82%B9%EF%BC%8CID%E5%8F%B7%E5%92%8C%E6%8F%8F%E8%BF%B0%E3%80%82","url":"https://web.archive.org/web/20221029033459/https://img04.sogoucdn.com/app/a/200698/sogou_science_11053","rw":400,"rh":280,"title":"在Windows 7下运行的V7客户端初始模式的示例图。除了各种控件和用户详细信息外，V7客户端还呈现了工作单元信息，例如它的状态，计算进度，ETA，信用点，ID号和描述。","alt":null,"width":0,"height":0}],"card":null,"references":[],"versionCount":0},{"paragraphId":"14994961170694412","title":"与其他分子模拟器的比较","versionId":"14994961137139986","lemmaId":10248,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":10145103,"name":"科学百科编辑组","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233692,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>Rosetta@home是一个面向蛋白质结构预测的分布式计算项目，是最精确的三级结构预测器之一。<sup><a href=\"#quote_206\" class=\"kx_ref\">[206]</a></sup><sup><a href=\"#quote_207\" class=\"kx_ref\">[207]</a></sup> Rosetta软件中的构象状态可用于初始化马尔可夫状态模型，作为Folding@home模拟的起点。<sup><a href=\"#quote_18\" class=\"kx_ref\">[18]</a></sup> 相反，结构预测算法可以从热力学和动力学模型以及蛋白质折叠模拟的取样方面得到改进。<sup><a href=\"#quote_208\" class=\"kx_ref\">[208]</a></sup> 由于Rosetta只试图预测最终折叠状态，而不是折叠如何进行，Rosetta@home和Folding@home是互补的，并解决相当不同的分子问题。<sup><a href=\"#quote_18\" class=\"kx_ref\">[18]</a></sup><sup><a href=\"#quote_209\" class=\"kx_ref\">[209]</a></sup> </p>\n<p>安东（Anton ）是为分子动力学模拟而建造的专用超级计算机。2011年10月，安东和Folding@home是两个最强大的分子动力学系统。<sup><a href=\"#quote_210\" class=\"kx_ref\">[210]</a></sup> Anton 的独特之处在于它能够产生计算成本高昂的单个超长分子轨道，<sup><a href=\"#quote_211\" class=\"kx_ref\">[211]</a></sup> 例如2010年达到毫秒级的轨道。<sup><a href=\"#quote_212\" class=\"kx_ref\">[212]</a></sup><sup><a href=\"#quote_213\" class=\"kx_ref\">[213]</a></sup> 这些长轨迹可能对某些类型的生化问题特别有帮助。<sup><a href=\"#quote_214\" class=\"kx_ref\">[214]</a></sup><sup><a href=\"#quote_215\" class=\"kx_ref\">[215]</a></sup> 但是，Anton 不使用马尔可夫状态模型进行分析。2011年，潘德实验室从两个100微秒Anton 模拟中构建了一个质谱，并发现了Anton 传统分析中不可见的替代折叠路径。他们得出的结论是，由有限数量的长轨道构建的多管轨迹和由许多短轨道组装的多管轨迹之间没有什么区别。<sup><a href=\"#quote_211\" class=\"kx_ref\">[211]</a></sup>2011年6月，Folding@home开始增加Anton 模拟的采样，以更好地确定其方法与Anton 的方法相比如何。<sup><a href=\"#quote_216\" class=\"kx_ref\">[216]</a></sup><sup><a href=\"#quote_217\" class=\"kx_ref\">[217]</a></sup> 然而，不像Folding@home的较短轨迹更适合分布式计算和其他并行化方法，较长轨迹不需要自适应采样来充分采样蛋白质的相空间。因此，Anton 和Folding@home的模拟方法的结合有可能提供对该空间更彻底的采样。<sup><a href=\"#quote_211\" class=\"kx_ref\">[211]</a></sup> </p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14994961170694413","title":"笔记","versionId":"14994961137139987","lemmaId":10248,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":10145103,"name":"科学百科编辑组","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233692,"comment":null,"dependVersionId":0,"contentType":1,"content":"<ol> \n <li id=\"cite-ref-2\"> Supercomputer FLOPS performance is assessed by running the legacy LINPACK benchmark. This short-term testing has difficulty in accurately reflecting sustained performance on real-world tasks because LINPACK more efficiently maps to supercomputer hardware. Computing systems vary in architecture and design, so direct comparison is difficult. Despite this, FLOPS remain the primary speed metric used in supercomputing.<sup><a href=\"#quote_218\" class=\"kx_ref\">[218]</a></sup><sup><a href=\"#quote_219\" class=\"kx_ref\">[219]</a></sup> In contrast, Folding@home determines its FLOPS using wall-clock time by measuring how much time its work units take to complete.<sup><a href=\"#quote_220\" class=\"kx_ref\">[220]</a></sup><sup><a href=\"#quote_221\" class=\"kx_ref\">[221]</a></sup><sup><a href=\"#quote_222\" class=\"kx_ref\">[222]</a></sup> </li> \n</ol>","pics":null,"card":null,"references":[],"versionCount":0}],"references":[{"id":1,"type":"book","title":"Pande lab (2010-07-26). \"About Folding@home\". Folding@home. Stanford University. Retrieved 2017-06-30.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":2,"type":"book","title":"V. S. Pande; K. Beauchamp; G. R. Bowman (2010). \"Everything you wanted to know about Markov State Models but were afraid to ask\". Methods. 52 (1): 99–105. doi:10.1016/j.ymeth.2010.06.002. PMC 2933958. PMID 20570730.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":3,"type":"book","title":"Gregory R. Bowman; Vijay S. Pande (2010). \"Protein folded states are kinetic hubs\". Proceedings of the National Academy of Sciences. 107 (24): 10890–5. Bibcode:2010PNAS..10710890B. doi:10.1073/pnas.1003962107. PMC 2890711. PMID 20534497.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":4,"type":"book","title":"Pande lab (July 27, 2012). \"Papers & Results from Folding@home\". Folding@home. Stanford University. Archived from the original on September 21, 2012. Retrieved 1 February 2019.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":5,"type":"book","title":"Vincent A. Voelz; Gregory R. Bowman; Kyle Beauchamp; Vijay S. Pande (2010). \"Molecular simulation of ab initio protein folding for a millisecond folder NTL9(1–39)\". Journal of the American Chemical Society. 132 (5): 1526–1528. doi:10.1021/ja9090353. PMC 2835335. PMID 20070076.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":6,"type":"book","title":"Christopher D. Snow; Houbi Ngyen; Vijay S. Pande; Martin Gruebele (2002). \"Absolute comparison of simulated and experimental protein-folding dynamics\" (PDF). Nature. 420 (6911): 102–106. Bibcode:2002Natur.420..102S. doi:10.1038/nature01160. PMID 12422224. Archived from the original (PDF) on 2012-03-24.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":7,"type":"book","title":"Fabrizio Marinelli, Fabio Pietrucci, Alessandro Laio, Stefano Piana (2009). Pande, Vijay S., ed. \"A Kinetic Model of Trp-Cage Folding from Multiple Biased Molecular Dynamics Simulations\". PLoS Computational Biology. 5 (8): e1000452. Bibcode:2009PLSCB...5E0452M. doi:10.1371/journal.pcbi.1000452. PMC 2711228. PMID 19662155.CS1 maint: Multiple names: authors list (link)","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":8,"type":"book","title":"\"So Much More to Know\". Science. 309 (5731): 78–102. 2005. doi:10.1126/science.309.5731.78b. PMID 15994524.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":9,"type":"book","title":"Heath Ecroyd; John A. Carver (2008). \"Unraveling the mysteries of protein folding and misfolding\". IUBMB Life (review). 60 (12): 769–774. doi:10.1002/iub.117. PMID 18767168.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":10,"type":"book","title":"Yiwen Chen; Feng Ding; Huifen Nie; Adrian W. Serohijos; Shantanu Sharma; Kyle C. Wilcox; Shuangye Yin; Nikolay V. Dokholyan (2008). \"Protein folding: Then and now\". Archives of Biochemistry and Biophysics. 469 (1): 4–19. doi:10.1016/j.abb.2007.05.014. PMC 2173875. PMID 17585870.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":11,"type":"book","title":"Leila M Luheshi; Damian Crowther; Christopher Dobson (2008). \"Protein misfolding and disease: from the test tube to the organism\". Current Opinion in Chemical Biology. 12 (1): 25–31. doi:10.1016/j.cbpa.2008.02.011. PMID 18295611.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":12,"type":"book","title":"C. D. Snow; E. J. Sorin; Y. M. Rhee; V. S. Pande. (2005). \"How well can simulation predict protein folding kinetics and thermodynamics?\". Annual Review of Biophysics (review). 34: 43–69. doi:10.1146/annurev.biophys.34.040204.144447. PMID 15869383.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":13,"type":"book","title":"A. Verma; S.M. Gopal; A. Schug; J.S. Oh; K.V. Klenin; K.H. Lee; W. Wenzel (2008). Massively Parallel All Atom Protein Folding in a Single Day. Advances in Parallel Computing. 15. pp. 527–534. ISBN 978-1-58603-796-3. ISSN 0927-5452.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":14,"type":"book","title":"Vijay S. Pande; Ian Baker; Jarrod Chapman; Sidney P. Elmer; Siraj Khaliq; Stefan M. Larson; Young Min Rhee; Michael R. Shirts; Christopher D. Snow; Eric J. Sorin; Bojan Zagrovic (2002). \"Atomistic protein folding simulations on the submillisecond timescale using worldwide distributed computing\". Biopolymers. 68 (1): 91–109. doi:10.1002/bip.10219. PMID 12579582.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":15,"type":"book","title":"G. Bowman; V. Volez; V. S. Pande (2011). \"Taming the complexity of protein folding\". Current Opinion in Structural Biology. 21 (1): 4–11. doi:10.1016/j.sbi.2010.10.006. PMC 3042729. PMID 21081274.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":16,"type":"book","title":"Chodera, John D.; Swope, William C.; Pitera, Jed W.; Dill, Ken A. (1 January 2006). \"Long‐Time Protein Folding Dynamics from Short‐Time Molecular Dynamics Simulations\". Multiscale Modeling & Simulation. 5 (4): 1214–1226. doi:10.1137/06065146X.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":17,"type":"book","title":"Robert B Best (2012). \"Atomistic molecular simulations of protein folding\". Current Opinion in Structural Biology (review). 22 (1): 52–61. doi:10.1016/j.sbi.2011.12.001. PMID 22257762.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":18,"type":"book","title":"TJ Lane; Gregory Bowman; Robert McGibbon; Christian Schwantes; Vijay Pande; Bruce Borden (September 10, 2012). \"Folding@home Simulation FAQ\". Folding@home. Stanford University. Archived from the original on September 21, 2012. Retrieved July 8, 2013.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":19,"type":"book","title":"Gregory R. Bowman; Daniel L. Ensign; Vijay S. Pande (2010). \"Enhanced Modeling via Network Theory: Adaptive Sampling of Markov State Models\". Journal of Chemical Theory and Computation. 6 (3): 787–794. doi:10.1021/ct900620b. PMC 3637129. PMID 23626502.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":20,"type":"book","title":"Vijay Pande (June 8, 2012). \"FAHcon 2012: Thinking about how far FAH has come\". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved June 12, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":21,"type":"book","title":"Kyle A. Beauchamp; Daniel L. Ensign; Rhiju Das; Vijay S. Pande (2011). \"Quantitative comparison of villin headpiece subdomain simulations and triplet–triplet energy transfer experiments\". Proceedings of the National Academy of Sciences. 108 (31): 12734–9. Bibcode:2011PNAS..10812734B. doi:10.1073/pnas.1010880108. PMC 3150881. PMID 21768345.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":22,"type":"book","title":"Timothy H. Click; Debabani Ganguly; Jianhan Chen (2010). \"Intrinsically Disordered Proteins in a Physics-Based World\". International Journal of Molecular Sciences. 11 (12): 919–27. doi:10.3390/ijms11125292. PMC 3100817. PMID 21614208.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":23,"type":"book","title":"\"Greg Bowman awarded the 2010 Kuhn Paradigm Shift Award\". simtk.org. SimTK: MSMBuilder. March 29, 2010. Archived from the original on September 21, 2012. Retrieved September 20, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":24,"type":"book","title":"\"MSMBuilder Source Code Repository\". MSMBuilder. simtk.org. 2012. Archived from the original on October 12, 2012. Retrieved October 12, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":25,"type":"book","title":"\"Biophysical Society Names Five 2012 Award Recipients\". Biophysics.org. Biophysical Society. August 17, 2011. Archived from the original on September 21, 2012. Retrieved September 20, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":26,"type":"book","title":"\"Folding@home – Awards\". Folding@home. Stanford University. August 2011. Archived from the original (FAQ) on September 21, 2012. Retrieved July 8, 2013.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":27,"type":"book","title":"Vittorio Bellotti; Monica Stoppini (2009). \"Protein Misfolding Diseases\" (PDF). The Open Biology Journal. 2 (2): 228–234. doi:10.2174/1874196700902020228. Archived from the original on February 22, 2014.CS1 maint: BOT: original-url status unknown (link)","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":28,"type":"book","title":"Pande lab (May 30, 2012). \"Folding@home Diseases Studied FAQ\". Folding@home. Stanford University. Archived from the original (FAQ) on September 21, 2012. Retrieved July 8, 2013.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":29,"type":"book","title":"Collier, Leslie; Balows, Albert; Sussman, Max (1998). Mahy, Brian; Collier, Leslie, eds. Topley and Wilson's Microbiology and Microbial Infections. 1, Virology (ninth ed.). London: Arnold. pp. 75–91. ISBN 978-0-340-66316-5.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":30,"type":"book","title":"Fred E. Cohen; Jeffery W. Kelly (2003). \"Therapeutic approaches to protein misfolding diseases\". Nature (review). 426 (6968): 905–9. Bibcode:2003Natur.426..905C. doi:10.1038/nature02265. PMID 14685252.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":31,"type":"book","title":"Chun Song; Shen Lim; Joo Tong (2009). \"Recent advances in computer-aided drug design\". Briefings in Bioinformatics (review). 10 (5): 579–91. doi:10.1093/bib/bbp023. PMID 19433475.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":32,"type":"book","title":"Pande lab (2012). \"Folding@Home Press FAQ\". Folding@home. Stanford University. Archived from the original (FAQ) on September 21, 2012. Retrieved July 8, 2013.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":33,"type":"book","title":"Christian \"schwancr\" Schwantes (Pande lab member) (August 15, 2011). \"Projects 7808 and 7809 to full fah\". Folding@home. phpBB Group. Archived from the original on September 21, 2012. Retrieved October 16, 2011.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":34,"type":"book","title":"Del Lucent; V. Vishal; Vijay S. Pande (2007). \"Protein folding under confinement: A role for solvent\". Proceedings of the National Academy of Sciences of the United States of America. 104 (25): 10430–10434. Bibcode:2007PNAS..10410430L. doi:10.1073/pnas.0608256104. PMC 1965530. PMID 17563390.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":35,"type":"book","title":"Vincent A. Voelz; Vijay R. Singh; William J. Wedemeyer; Lisa J. Lapidus; Vijay S. Pande (2010). \"Unfolded-State Dynamics and Structure of Protein L Characterized by Simulation and Experiment\". Journal of the American Chemical Society. 132 (13): 4702–4709. doi:10.1021/ja908369h. PMC 2853762. PMID 20218718.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":36,"type":"book","title":"Pande lab (August 18, 2011). \"Folding@home Main FAQ\" (FAQ). Folding@home. Stanford University. Archived from the original on September 21, 2012. Retrieved July 8, 2013.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":37,"type":"book","title":"Vijay Pande (April 23, 2008). \"Folding@home and Simbios\". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved November 9, 2011.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":38,"type":"book","title":"Vijay Pande (October 25, 2011). \"Re: Suggested Changes to F@h Website\". Folding@home. phpBB Group. Archived from the original on September 21, 2012. Retrieved October 25, 2011.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":39,"type":"book","title":"Caroline Hadley (2004). \"Biologists think bigger\". EMBO Reports. 5 (3): 236–238. doi:10.1038/sj.embor.7400108. PMC 1299019. PMID 14993921.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":40,"type":"book","title":"S. Pronk; P. Larsson; I. Pouya; G.R. Bowman; I.S. Haque; K. Beauchamp; B. Hess; V.S. Pande; P.M. Kasson; E. Lindahl (2011). \"Copernicus: A new paradigm for parallel adaptive molecular dynamics\". 2011 International Conference for High Performance Computing, Networking, Storage and Analysis: 1–10, 12–18.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":41,"type":"book","title":"Sander Pronk; Iman Pouya; Per Larsson; Peter Kasson; Erik Lindahl (November 17, 2011). \"Copernicus Download\". copernicus-computing.org. Copernicus. Archived from the original on October 12, 2012. Retrieved October 2, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":42,"type":"book","title":"G Brent Irvine; Omar M El-Agnaf; Ganesh M Shankar; Dominic M Walsh (2008). \"Protein Aggregation in the Brain: The Molecular Basis for Alzheimer's and Parkinson's Diseases\". Molecular Medicine (review). 14 (7–8): 451–464. doi:10.2119/2007-00100.Irvine. PMC 2274891. PMID 18368143.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":43,"type":"book","title":"Claudio Soto; Lisbell D. Estrada (2008). \"Protein Misfolding and Neurodegeneration\". Archives of Neurology (review). 65 (2): 184–189. doi:10.1001/archneurol.2007.56. PMID 18268186.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":44,"type":"book","title":"Robin Roychaudhuri; Mingfeng Yang; Minako M. Hoshi; David B. Teplow (2008). \"Amyloid β-Protein Assembly and Alzheimer Disease\". Journal of Biological Chemistry. 284 (8): 4749–53. doi:10.1074/jbc.R800036200. PMC 3837440. PMID 18845536.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":45,"type":"book","title":"Nicholas W. Kelley; V. Vishal; Grant A. Krafft; Vijay S. Pande. (2008). \"Simulating oligomerization at experimental concentrations and long timescales: A Markov state model approach\". Journal of Chemical Physics. 129 (21): 214707. Bibcode:2008JChPh.129u4707K. doi:10.1063/1.3010881. PMC 2674793. PMID 19063575.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":46,"type":"book","title":"P. Novick, J. Rajadas, C.W. Liu, N. W. Kelley, M. Inayathullah, and V. S. Pande (2011). Buehler, Markus J., ed. \"Rationally Designed Turn Promoting Mutation in the Amyloid-β Peptide Sequence Stabilizes Oligomers in Solution\". PLoS ONE. 6 (7): e21776. Bibcode:2011PLoSO...621776R. doi:10.1371/journal.pone.0021776. PMC 3142112. PMID 21799748.CS1 maint: Multiple names: authors list (link)","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":47,"type":"book","title":"Aabgeena Naeem; Naveed Ahmad Fazili (2011). \"Defective Protein Folding and Aggregation as the Basis of Neurodegenerative Diseases: The Darker Aspect of Proteins\". Cell Biochemistry and Biophysics (review). 61 (2): 237–50. doi:10.1007/s12013-011-9200-x. PMID 21573992.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":48,"type":"book","title":"Gregory R Bowman; Xuhui Huang; Vijay S Pande (2010). \"Network models for molecular kinetics and their initial applications to human health\". Cell Research (review). 20 (6): 622–630. doi:10.1038/cr.2010.57. PMC 4441225. PMID 20421891.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":49,"type":"book","title":"Vijay Pande (December 18, 2008). \"New FAH results on possible new Alzheimer's drug presented\". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved September 23, 2011.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":50,"type":"book","title":"Paul A. Novick; Dahabada H. Lopes; Kim M. Branson; Alexandra Esteras-Chopo; Isabella A. Graef; Gal Bitan; Vijay S. Pande (2012). \"Design of β-Amyloid Aggregation Inhibitors from a Predicted Structural Motif\". Journal of Medicinal Chemistry. 55 (7): 3002–10. doi:10.1021/jm201332p. PMC 3766731. PMID 22420626.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":51,"type":"book","title":"yslin (Pande lab member) (July 22, 2011). \"New project p6871 [Classic]\". Folding@home. phpBB Group. Archived from the original on September 21, 2012. Retrieved March 17, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":52,"type":"book","title":"Pande lab. \"Project 6871 Description\". Folding@home. Stanford University. Archived from the original on September 21, 2012. Retrieved September 27, 2011.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":53,"type":"book","title":"Walker FO (2007). \"Huntington's disease\". Lancet. 369 (9557): 218–28 [220]. doi:10.1016/S0140-6736(07)60111-1. PMID 17240289.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":54,"type":"book","title":"Nicholas W. Kelley; Xuhui Huang; Stephen Tam; Christoph Spiess; Judith Frydman; Vijay S. Pande (2009). \"The predicted structure of the headpiece of the Huntingtin protein and its implications on Huntingtin aggregation\". Journal of Molecular Biology. 388 (5): 919–27. doi:10.1016/j.jmb.2009.01.032. PMC 2677131. PMID 19361448.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":55,"type":"book","title":"Susan W Liebman; Stephen C Meredith (2010). \"Protein folding: Sticky N17 speeds huntingtin pile-up\". Nature Chemical Biology. 6 (1): 7–8. doi:10.1038/nchembio.279. PMID 20016493.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":56,"type":"book","title":"Diwakar Shukla (Pande lab member) (February 10, 2012). \"Project 8021 released to beta\". Folding@home. phpBB Group. Archived from the original on September 21, 2012. Retrieved March 17, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":57,"type":"book","title":"M Hollstein; D Sidransky; B Vogelstein; CC Harris (1991). \"p53 mutations in human cancers\". Science. 253 (5015): 49–53. Bibcode:1991Sci...253...49H. doi:10.1126/science.1905840. PMID 1905840.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":58,"type":"book","title":"L. T. Chong; C. D. Snow; Y. M. Rhee; V. S. Pande. (2004). \"Dimerization of the p53 Oligomerization Domain: Identification of a Folding Nucleus by Molecular Dynamics Simulations\". Journal of Molecular Biology. 345 (4): 869–878. CiteSeerX 10.1.1.132.1174. doi:10.1016/j.jmb.2004.10.083. PMID 15588832.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":59,"type":"book","title":"mah3, Vijay Pande (September 24, 2004). \"F@H project publishes results of cancer related research\". MaximumPC.com. Future US, Inc. Archived from the original on September 21, 2012. Retrieved September 20, 2012. To our knowledge, this is the first peer-reviewed results from a distributed computing project related to cancer.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":60,"type":"book","title":"Lillian T. Chong; William C. Swope; Jed W. Pitera; Vijay S. Pande (2005). \"Kinetic Computational Alanine Scanning: Application to p53 Oligomerization\". Journal of Molecular Biology. 357 (3): 1039–1049. doi:10.1016/j.jmb.2005.12.083. PMID 16457841.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":61,"type":"book","title":"Almeida MB, do Nascimento JL, Herculano AM, Crespo-López ME (2011). \"Molecular chaperones: toward new therapeutic tools\". Journal of Molecular Biology (review). 65 (4): 239–43. doi:10.1016/j.biopha.2011.04.025. PMID 21737228.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":62,"type":"book","title":"Vijay Pande (September 28, 2007). \"Nanomedicine center\". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved September 23, 2011.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":63,"type":"book","title":"Vijay Pande (December 22, 2009). \"Release of new Protomol (Core B4) WUs\". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved September 23, 2011.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":64,"type":"book","title":"Pande lab. \"Project 180 Description\". Folding@home. Stanford University. Archived from the original on September 21, 2012. Retrieved September 27, 2011.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":65,"type":"book","title":"TJ Lane (Pande lab member) (June 8, 2011). \"Project 7600 in Beta\". Folding@home. phpBB Group. Archived from the original on September 21, 2012. Retrieved September 27, 2011.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":66,"type":"book","title":"TJ Lane (Pande lab member) (June 8, 2011). \"Project 7600 Description\". Folding@home. Stanford University. Archived from the original on September 21, 2012. Retrieved March 31, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":67,"type":"book","title":"\"Scientists boost potency, reduce side effects of IL-2 protein used to treat cancer\". MedicalXpress.com. Medical Xpress. March 18, 2012. Archived from the original on September 21, 2012. Retrieved September 20, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":68,"type":"book","title":"Aron M. Levin; Darren L. Bates; Aaron M. Ring; Carsten Krieg; Jack T. Lin; Leon Su; Ignacio Moraga; Miro E. Raeber; Gregory R. Bowman; Paul Novick; Vijay S. Pande; C. Garrison Fathman; Onur Boyman; K. Christopher Garcia (2012). \"Exploiting a natural conformational switch to engineer an interleukin-2 'superkine'\". Nature. 484 (7395): 529–33. Bibcode:2012Natur.484..529L. doi:10.1038/nature10975. PMC 3338870. PMID 22446627.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":69,"type":"book","title":"Rauch F, Glorieux FH (2004). \"Osteogenesis imperfecta\". Lancet. 363 (9418): 1377–85. doi:10.1016/S0140-6736(04)16051-0. PMID 15110498.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":70,"type":"book","title":"Fratzl, Peter (2008). Collagen: structure and mechanics. ISBN 978-0-387-73905-2. Retrieved March 17, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":71,"type":"book","title":"Gautieri A, Uzel S, Vesentini S, Redaelli A, Buehler MJ (2009). \"Molecular and mesoscale disease mechanisms of Osteogenesis Imperfecta\". Biophysical Journal. 97 (3): 857–865. Bibcode:2009BpJ....97..857G. doi:10.1016/j.bpj.2009.04.059. PMC 2718154. PMID 19651044.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":72,"type":"book","title":"Sanghyun Park; Randall J. Radmer; Teri E. Klein; Vijay S. Pande (2005). \"A New Set of Molecular Mechanics Parameters for Hydroxyproline and Its Use in Molecular Dynamics Simulations of Collagen-Like Peptides\". Journal of Computational Chemistry. 26 (15): 1612–1616. CiteSeerX 10.1.1.142.6781. doi:10.1002/jcc.20301. PMID 16170799.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":73,"type":"book","title":"Gregory Bowman (Pande lab Member). \"Project 10125\". Folding@home. phpBB Group. Retrieved December 2, 2011.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":74,"type":"book","title":"Hana Robson Marsden; Itsuro Tomatsu; Alexander Kros (2011). \"Model systems for membrane fusion\". Chemical Society Reviews (review). 40 (3): 1572–1585. doi:10.1039/c0cs00115e. PMID 21152599.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":75,"type":"book","title":"Peter Kasson (2012). \"Peter M. Kasson\". Kasson lab. University of Virginia. Archived from the original on September 21, 2012. Retrieved September 20, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":76,"type":"book","title":"Peter M. Kasson; Afra Zomorodian; Sanghyun Park; Nina Singhal; Leonidas J. Guibas; Vijay S. Pande (2007). \"Persistent voids: a new structural metric for membrane fusion\". Bioinformatics. 23 (14): 1753–1759. doi:10.1093/bioinformatics/btm250. PMID 17488753.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":77,"type":"book","title":"Peter M. Kasson; Daniel L. Ensign; Vijay S. Pande (2009). \"Combining Molecular Dynamics with Bayesian Analysis To Predict and Evaluate Ligand-Binding Mutations in Influenza Hemagglutinin\". Journal of the American Chemical Society. 131 (32): 11338–11340. doi:10.1021/ja904557w. PMC 2737089. PMID 19637916.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":78,"type":"book","title":"Peter M. Kasson; Vijay S. Pande (2009). \"Combining mutual information with structural analysis to screen for functionally important residues in influenza hemagglutinin\". Pacific Symposium on Biocomputing: 492–503. PMC 2811693. PMID 19209725.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":79,"type":"book","title":"Vijay Pande (February 24, 2012). \"Protein folding and viral infection\". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved March 4, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":80,"type":"book","title":"Vijay Pande (February 27, 2012). \"New methods for computational drug design\". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved April 1, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":81,"type":"book","title":"Guha Jayachandran; M. R. Shirts; S. Park; V. S. Pande (2006). \"Parallelized-Over-Parts Computation of Absolute Binding Free Energy with Docking and Molecular Dynamics\". Journal of Chemical Physics. 125 (8): 084901. Bibcode:2006JChPh.125h4901J. doi:10.1063/1.2221680. PMID 16965051.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":82,"type":"book","title":"Pande lab. \"Project 10721 Description\". Folding@home. Stanford University. Archived from the original on September 21, 2012. Retrieved September 27, 2011.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":83,"type":"book","title":"Gregory Bowman (July 23, 2012). \"Searching for new drug targets\". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved September 27, 2011.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":84,"type":"book","title":"Gregory R. Bowman; Phillip L. Geissler (July 2012). \"Equilibrium fluctuations of a single folded protein reveal a multitude of potential cryptic allosteric sites\". PNAS. 109 (29): 11681–6. Bibcode:2012PNAS..10911681B. doi:10.1073/pnas.1209309109. PMC 3406870. PMID 22753506.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":85,"type":"book","title":"Paula M. Petrone; Christopher D. Snow; Del Lucent; Vijay S. Pande (2008). \"Side-chain recognition and gating in the ribosome exit tunnel\". Proceedings of the National Academy of Sciences. 105 (43): 16549–54. Bibcode:2008PNAS..10516549P. doi:10.1073/pnas.0801795105. PMC 2575457. PMID 18946046.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":86,"type":"book","title":"Pande lab. \"Project 5765 Description\". Folding@home. Stanford University. Archived from the original on September 21, 2012. Retrieved December 2, 2011.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":87,"type":"book","title":"National Institute of Neurological Disorders and Stroke (21 August 2018). \"Creutzfeldt-Jakob Disease Fact Sheet\". NIH. Retrieved 2 March 2019.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":88,"type":"book","title":"Kupfer, L; Hinrichs, W; Groschup, M.H. (2009). \"Prion Protein Misfolding\". Current Molecular Medicine. Bentham Science Publishers. 9 (7): 826–835. doi:10.2174/156652409789105543. PMC 3330701. PMID 19860662.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":89,"type":"book","title":"Riek, Poland; Hornemann, Simone; Wider, Gerhard; Billeter, Martin; Glockshuber, Rudi; Wüthrich, Kurt (1996). \"NMR structure of the mouse prion protein domain in PrP(121-231)\". Nature. Nature Research. 382 (6587): 180–182. doi:10.1038/382180a0. PMID 8700211.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":90,"type":"book","title":"Ziegler, J; Sticht, H; Marx, UC; Müller, W; Rösch, P; Schwarzinger, S (2003). \"CD and NMR studies of prion protein (PrP) helix1. Novel implications for its role in the PrPC-->PrPSc conversion process\". J Biol Chem. American Society for Biochemistry and Molecular Biology. 278 (50): 50175–81. doi:10.1074/jbc.M305234200. PMID 12952977.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":91,"type":"book","title":"Govaerts, Cedric; Wile, Holger; Brusiner, Stanley B.; Cohen, Fred (2004). \"Evidence for assembly of prions with left-handed β-helices into trimers\". Proc Natl Acad Sci USA. National Academy of Sciences. 101 (22): 8342–47. doi:10.1073/pnas.0402254101. PMC 420396. PMID 15155909.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":92,"type":"book","title":"Silveira, Jay; Raymond, Gregory; Hughson, Andrew; Race, Richard; Sim, Valerie; Caughey, Byron; Hayes, Stanley (2005). \"The most infectious prion protein particles\". Nature. Nature Research. 437 (7056): 257–261. doi:10.1038/nature03989. PMC 1513539. PMID 16148934.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":93,"type":"book","title":"Moore, Roger A.; Taubner, Lara M.; Priola, Suzette (2009). \"Prion Protein Misfolding and Disease\". Curr Opin Struct Biol. Elsevier. 19 (1): 14–22. doi:10.1016/j.sbi.2008.12.007. PMC 2674794. PMID 19157856.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":94,"type":"book","title":"Moore, Roger A.; Herzog, Christian; Errett, John; Kocisko, David A.; Arnold, Kevin M.; Hayes, Stanley F.; Priola, Suzette A. (2006). \"Octapeptide repeat insertions increase the rate of protease-resistant prion protein formation\". Protein Science. Wiley-Blackwell. 15 (3): 609–619. doi:10.1110/ps.051822606. PMC 2249780. PMID 16452616.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":95,"type":"book","title":"Ziegler, Jan; Viehrig, Christine; Geimer, Stefan; Rosch, Paul; Schwarzinger, Stephan (2006). \"Putative aggregation initiation sites in prion protein\". FEBS Letters. FEBS Press. 580 (8): 2033–40. doi:10.1016/j.febslet.2006.03.002. PMID 16545382.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":96,"type":"book","title":"Brusiner, Stanley (1998). \"Prions\". Proc Natl Acad Sci USA. National Academy of Sciences. 95 (23): 13363–83. doi:10.1073/pnas.95.23.13363. PMC 33918. PMID 9811807.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":97,"type":"book","title":"Vorberg, I; Groschup, MH; Pfaff, E; Priola, SA (2003). \"Multiple amino acid residues within the rabbit prion protein inhibit formation of its abnormal isoform\". J Virol. American Society for Microbiology. 77 (3): 2003–9. doi:10.1128/JVI.77.3.2003-2009.2003. PMC 140934. PMID 12525634.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":98,"type":"book","title":"Barrow, CJ; Yasuda, A; Kenny, PT; Zagorski, MG (1992). \"Solution conformations and aggregational properties of synthetic amyloid beta-peptides of Alzheimer's disease. Analysis of circular dichroism spectra\". J Biol Chem. American Society for Biochemistry and Molecular Biology. 225 (4): 1075–93. PMID 1613791.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":99,"type":"book","title":"Kuznetsov, Igor; Rackovsky, Shalom (2004). \"Comparative computational analysis of prion proteins reveals two fragments with unusual structural properties and a pattern of increase in hydrophobicity associated with disease-promoting mutations\". Protein Science. Wiley-Blackwell. 13 (12): 3230–44. doi:10.1110/ps.04833404. PMC 2287303. PMID 15557265.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":100,"type":"book","title":"Baskakov, IV; Legname, G; Baldwin, MA; Prusiner, SB; Cohen, FE (2002). \"Pathway complexity of prion protein assembly into amyloid\". J Biol Chem. American Society for Biochemistry and Molecular Biology. 277 (24): 21140–8. doi:10.1074/jbc.M111402200. PMID 11912192.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":101,"type":"book","title":"Raddick, M. Jordan; Bracey, Georgia; Gay, Pamela L.; Lintott, Chris J.; Murray, Phil; Schawinski, Kevin; Szalay, Alexander S.; Vandenberg, Jan (December 2010). \"Galaxy Zoo: Exploring the Motivations of Citizen Science Volunteers\". Astronomy Education Review. 9 (1). arXiv:0909.2925. doi:10.3847/AER2009036.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":102,"type":"book","title":"Vickie, Curtis (2018-04-20). Online citizen science and the widening of academia : distributed engagement with research and knowledge production. Cham, Switzerland. ISBN 9783319776644. OCLC 1034547418.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":103,"type":"book","title":"Nov, Oded; Arazy, Ofer; Anderson, David (2011). \"Dusting for science: motivation and participation of digital citizen science volunteers\". Proceedings of the 2011 IConference on - IConference '11. IConference '11. Seattle, Washington: ACM Press: 68–74. doi:10.1145/1940761.1940771. ISBN 9781450301213.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":104,"type":"book","title":"Curtis, Vickie (December 2015). \"Motivation to Participate in an Online Citizen Science Game: A Study of Foldit\" (PDF). Science Communication. 37 (6): 723–746. doi:10.1177/1075547015609322. ISSN 1075-5470.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":105,"type":"book","title":"Curtis, Vickie (2018-04-27). \"Patterns of Participation and Motivation in Folding@home: The Contribution of Hardware Enthusiasts and Overclockers\". Citizen Science: Theory and Practice. 3 (1): 5. doi:10.5334/cstp.109. ISSN 2057-4991.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":106,"type":"book","title":"Colwell, B. (March 2004). \"The Zen of overclocking\". Computer. 37 (3): 9–12. doi:10.1109/MC.2004.1273994. ISSN 0018-9162.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":107,"type":"book","title":"Kloetzer, Laure; Da Costa, Julien; Schneider, Daniel K. (2016-12-31). \"Not so passive: engagement and learning in Volunteer Computing projects\". Human Computation. 3 (1). doi:10.15346/hc.v3i1.4. ISSN 2330-8001.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":108,"type":"book","title":"Darch Peter; Carusi Annamaria (2010-09-13). \"Retaining volunteers in volunteer computing projects\". Philosophical Transactions of the Royal Society A: Mathematical, Physical and Engineering Sciences. 368 (1926): 4177–4192. doi:10.1098/rsta.2010.0163. PMID 20679130.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":109,"type":"book","title":"\"2013 Member Study: Findings and Next Steps\". World Community Grid.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":110,"type":"book","title":"Krebs, Viola (2010-01-31). \"Motivations of cybervolunteers in an applied distributed computing environment: MalariaControl.net as an example\". First Monday. 15 (2). doi:10.5210/fm.v15i2.2783.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":111,"type":"book","title":"Curtis, Vickie (2015). Online citizen science projects: an exploration of motivation, contribution and participation, PhD Thesis (PDF). United Kingdom: The Open University.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":112,"type":"book","title":"Vijay Pande (September 16, 2007). \"Crossing the petaFLOPS barrier\". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved August 28, 2011.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":113,"type":"book","title":"Michael Gross (2012). \"Folding research recruits unconventional help\". Current Biology. 22 (2): R35–R38. doi:10.1016/j.cub.2012.01.008. PMID 22389910.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":114,"type":"book","title":"\"TOP500 List — June 2007\". top500.org. Top500. June 2007. Retrieved September 20, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":115,"type":"book","title":"\"Folding@Home reach 2 Petaflops\". n4g.com. HAVAmedia. May 8, 2008. Archived from the original on September 21, 2012. Retrieved September 20, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":116,"type":"book","title":"\"3 PetaFLOP barrier\". longecity.org. Longecity. August 19, 2008. Archived from the original on September 21, 2012. Retrieved September 20, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":117,"type":"book","title":"\"NVIDIA Achieves Monumental Folding@Home Milestone With Cuda\". nvidia.com. NVIDIA Corporation. August 26, 2008. Retrieved September 20, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":118,"type":"book","title":"\"Increase in 'active' PS3 folders pushes Folding@home past 4 Petaflops!\". team52735.blogspot.com. Blogspot. September 29, 2008. Archived from the original on September 21, 2012. Retrieved September 20, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":119,"type":"book","title":"\"TOP500 List — November 2008\". top500.org. Top500. November 2008. Retrieved September 20, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":120,"type":"book","title":"Vijay Pande (February 18, 2009). \"Folding@home Passes the 5 petaFLOP Mark\". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved August 31, 2011.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":121,"type":"book","title":"\"Crossing the 5 petaFLOPS barrier\". longecity.org. Longecity. February 18, 2009. Archived from the original on September 21, 2012. Retrieved September 20, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":122,"type":"book","title":"Dragan Zakic (May 2009). \"Community Grid Computing — Studies in Parallel and Distributed Systems\" (PDF). Massey University College of Sciences. Massey University. Archived from the original (PDF) on September 21, 2012. Retrieved September 20, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":123,"type":"book","title":"William Ito. \"A review of recent advances in ab initio protein folding by the Folding@home project\" (PDF). Stanford University. Archived (PDF) from the original on September 22, 2012. Retrieved September 22, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":124,"type":"book","title":"Jesse Victors (November 10, 2011). \"Six Native PetaFLOPS\". Folding@home. phpBB Group. Archived from the original on September 21, 2012. Retrieved November 11, 2011.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":125,"type":"book","title":"Risto Kantonen (September 23, 2013). \"Folding@home Stats - Google Docs\". Folding@home. Retrieved September 23, 2013.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":126,"type":"book","title":"\"100 Petaflops nearly reached\". Stanford University. 2016-05-11. Retrieved 9 August 2016.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":127,"type":"book","title":"Pande lab (August 20, 2012). \"Folding@home Points FAQ\". Folding@home. Stanford University. Archived from the original (FAQ) on September 21, 2012. Retrieved July 8, 2013.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":128,"type":"book","title":"Pande lab (July 23, 2012). \"Folding@home Passkey FAQ\". Folding@home. Stanford University. Archived from the original (FAQ) on September 21, 2012. Retrieved July 8, 2013.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":129,"type":"book","title":"Peter Kasson (Pande lab member) (January 24, 2010). \"upcoming release of SMP2 cores\". Folding@home. phpBB Group. Archived from the original on September 21, 2012. Retrieved September 30, 2011.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":130,"type":"book","title":"Pande lab (2012). \"Folding@home homepage\". Folding@home. Stanford University. Archived from the original on September 21, 2012. Retrieved July 8, 2013.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":131,"type":"book","title":"\"Official Extreme Overclocking Folding@home Team Forum\". forums.extremeoverclocking.com. Extreme Overclocking. Archived from the original on September 21, 2012. Retrieved September 20, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":132,"type":"book","title":"Adam Beberg; Daniel Ensign; Guha Jayachandran; Siraj Khaliq; Vijay Pande (2009). Folding@home: Lessons From Eight Years of Volunteer Distributed Computing (PDF). Parallel & Distributed Processing, IEEE International Symposium. pp. 1–8. doi:10.1109/IPDPS.2009.5160922. ISBN 978-1-4244-3751-1. ISSN 1530-2075.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":133,"type":"book","title":"Norman Chan (April 6, 2009). \"Help Maximum PC's Folding Team Win the Next Chimp Challenge!\". Maximumpc.com. Future US, Inc. Archived from the original on September 21, 2012. Retrieved September 20, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":134,"type":"book","title":"Pande lab (June 11, 2012). \"Folding@home SMP FAQ\". Folding@home. Stanford University. Archived from the original (FAQ) on September 21, 2012. Retrieved July 8, 2013.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":135,"type":"book","title":"Vijay Pande (April 5, 2011). \"More transparency in testing\". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved October 14, 2011.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":136,"type":"book","title":"Bruce Borden (August 7, 2011). \"Re: Gromacs Cannot Continue Further\". Folding@home. phpBB Group. Archived from the original on September 21, 2012. Retrieved August 7, 2011.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":137,"type":"book","title":"PantherX (October 1, 2011). \"Re: Project 6803: (Run 4, Clone 66, Gen 255)\". Folding@home. phpBB Group. Archived from the original on September 21, 2012. Retrieved October 9, 2011.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":138,"type":"book","title":"PantherX (October 31, 2010). \"Troubleshooting Bad WUs\". Folding@home. phpBB Group. Archived from the original on September 21, 2012. Retrieved August 7, 2011.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":139,"type":"book","title":"Carsten Kutzner; David Van Der Spoel; Martin Fechner; Erik Lindahl; Udo W. Schmitt; Bert L. De Groot; Helmut Grubmüller (2007). \"Speeding up parallel GROMACS on high-latency networks\". Journal of Computational Chemistry. 28 (12): 2075–2084. doi:10.1002/jcc.20703. PMID 17405124.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":140,"type":"book","title":"Berk Hess; Carsten Kutzner; David van der Spoel; Erik Lindahl (2008). \"GROMACS 4: Algorithms for Highly Efficient, Load-Balanced, and Scalable Molecular Simulation\". Journal of Chemical Theory and Computation. 4 (3): 435–447. doi:10.1021/ct700301q. PMID 26620784.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":141,"type":"book","title":"Pande lab (August 19, 2012). \"Folding@home Gromacs FAQ\". Folding@home. Stanford University. Archived from the original (FAQ) on September 21, 2012. Retrieved July 8, 2013.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":142,"type":"book","title":"Pande lab (August 2, 2012). \"Folding@home Open Source FAQ\". Folding@home. Stanford University. Archived from the original (FAQ) on September 21, 2012. Retrieved July 8, 2013.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":143,"type":"book","title":"Pande lab (August 7, 2012). \"Folding@home Frequently Asked Questions (FAQ) Index\". Folding@home. Stanford University. Archived from the original on September 21, 2012. Retrieved July 8, 2013.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":144,"type":"book","title":"Vijay Pande (September 25, 2009). \"Update on new FAH cores and clients\". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved February 24, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":145,"type":"book","title":"M. S. Friedrichs; P. Eastman; V. Vaidyanathan; M. Houston; S. LeGrand; A. L. Beberg; D. L. Ensign; C. M. Bruns; V. S. Pande (2009). \"Accelerating Molecular Dynamic Simulation on Graphics Processing Units\". Journal of Computational Chemistry. 30 (6): 864–72. doi:10.1002/jcc.21209. PMC 2724265. PMID 19191337.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":146,"type":"book","title":"Pande lab (August 19, 2012). \"Folding@home Petaflop Initiative\". Folding@home. Stanford University. Archived from the original (FAQ) on September 21, 2012. Retrieved July 8, 2013.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":147,"type":"book","title":"Pande lab (February 10, 2011). \"Windows Uniprocessor Client Installation Guide\". Folding@home. Stanford University. Archived from the original (Guide) on September 21, 2012. Retrieved July 8, 2013.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":148,"type":"book","title":"PantherX (September 2, 2010). \"Re: Can Folding@home damage any part of my PC?\". Folding@home. phpBB Group. Archived from the original on September 21, 2012. Retrieved February 25, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":149,"type":"book","title":"Vijay Pande (June 17, 2009). \"How does FAH code development and sysadmin get done?\". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved October 14, 2011.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":150,"type":"book","title":"Pande lab (May 30, 2012). \"Uninstalling Folding@home Guide\". Folding@home. Stanford University. Archived from the original (Guide) on September 21, 2012. Retrieved July 8, 2013.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":151,"type":"book","title":"Folding@home developers. \"FAHControl source code repository\". Stanford University. Archived from the original on December 12, 2012. Retrieved October 15, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":152,"type":"book","title":"Pande lab. \"Folding@home Distributed Computing Client\". Folding@home. Stanford University. Archived from the original on September 21, 2012. Retrieved July 8, 2013.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":153,"type":"book","title":"Vijay Pande (June 28, 2008). \"Folding@home's End User License Agreement (EULA)\". Folding@home. Archived from the original on September 21, 2012. Retrieved May 15, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":154,"type":"book","title":"unikuser (August 7, 2011). \"FoldingAtHome\". Ubuntu Documentation. help.ubuntu.com. Archived from the original on September 22, 2012. Retrieved September 22, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":155,"type":"book","title":"The Case of the Modified Binaries, Josh Pitts, October 23, 2014; retrieved on 2016-11-23","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":156,"type":"book","title":"Fixing/Making Holes in ELF Binaries/Programs - Black Hat","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":157,"type":"book","title":"probably using tools such as ERESI","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":158,"type":"book","title":"x86 - Disassembling, modifying and then reassembling a Linux executable - Stack Overflow","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":159,"type":"book","title":"linux - How do I add functionality to an existing binary executable? - Reverse Engineering Stack Exchange","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":160,"type":"book","title":"\"Certificate Bypass: Hiding and Executing Malware from a Digitally Signed Executable\" (PDF). BlackHat.com. Deep Instinct. August 2016.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":161,"type":"book","title":"Phineus R. L. Markwick; J. Andrew McCammon (2011). \"Studying functional dynamics in bio-molecules using accelerated molecular dynamics\". Physical Chemistry Chemical Physics. 13 (45): 20053–65. Bibcode:2011PCCP...1320053M. doi:10.1039/C1CP22100K. PMID 22015376.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":162,"type":"book","title":"M. R. Shirts; V. S. Pande. (2000). \"Screen Savers of the World, Unite!\". Science. 290 (5498): 1903–1904. doi:10.1126/science.290.5498.1903. PMID 17742054.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":163,"type":"book","title":"Pande lab. \"Folding@Home Executive summary\" (PDF). Folding@home. Stanford University. Archived (PDF) from the original on September 21, 2012. Retrieved October 4, 2011.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":164,"type":"book","title":"Rattledagger, Vijay Pande (April 1, 2005). \"Folding@home client for BOINC in beta \"soon\"\". Boarddigger.com. Anandtech.com. Archived from the original on September 21, 2012. Retrieved September 20, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":165,"type":"book","title":"Pande lab (May 30, 2012). \"High Performance FAQ\". Folding@home. Stanford University. Archived from the original (FAQ) on September 21, 2012. Retrieved July 8, 2013.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":166,"type":"book","title":"John D. Owens; David Luebke; Naga Govindaraju; Mark Harris; Jens Krüger; Aaron Lefohn; Timothy J. Purcell (2007). \"A Survey of General-Purpose Computation on Graphics Hardware\". Computer Graphics Forum. 26 (1): 80–113. CiteSeerX 10.1.1.215.426. doi:10.1111/j.1467-8659.2007.01012.x.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":167,"type":"book","title":"P. Eastman; V. S. Pande (2010). \"OpenMM: A Hardware-Independent Framework for Molecular Simulations\". Computing in Science and Engineering. 12 (4): 34–39. doi:10.1109/MCSE.2010.27. ISSN 1521-9615. PMC 4486654. PMID 26146490.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":168,"type":"book","title":"I. Haque; V. S. Pande (2010). Hard Data on Soft Errors: A Large-Scale Assessment of Real-World Error Rates in GPGPU. 2010 10th IEEE/ACM International Conference on Cluster, Butt and Grid Computing (CCGrid). pp. 691–696. arXiv:0910.0505. doi:10.1109/CCGRID.2010.84. ISBN 978-1-4244-6987-1.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":169,"type":"book","title":"Pande lab (March 18, 2011). \"ATI FAQ\". Folding@home. Stanford University. Archived from the original (FAQ) on September 21, 2012. Retrieved July 8, 2013.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":170,"type":"book","title":"Vijay Pande (May 23, 2008). \"GPU news (about GPU1, GPU2, & NVIDIA support)\". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved September 8, 2011.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":171,"type":"book","title":"Travis Desell; Anthony Waters; Malik Magdon-Ismail; Boleslaw K. Szymanski; Carlos A. Varela; Matthew Newby; Heidi Newberg; Andreas Przystawik; David Anderson (2009). \"Accelerating the MilkyWay@Home volunteer computing project with GPUs\". 8th International Conference on Parallel Processing and Applied Mathematics (PPAM 2009) Part I. pp. 276–288. CiteSeerX 10.1.1.158.7614. ISBN 978-3-642-14389-2.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":172,"type":"book","title":"Vijay Pande (April 10, 2008). \"GPU2 open beta\". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved September 7, 2011.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":173,"type":"book","title":"Vijay Pande (April 15, 2008). \"Updates to the Download page/GPU2 goes live\". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved September 7, 2011.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":174,"type":"book","title":"Vijay Pande (April 11, 2008). \"GPU2 open beta going well\". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved September 7, 2011.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":175,"type":"book","title":"Vijay Pande (April 24, 2010). \"Prepping for the GPU3 rolling: new client and NVIDIA FAH GPU clients will (in the future) need CUDA 2.2 or later\". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved September 8, 2011.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":176,"type":"book","title":"Vijay Pande (May 25, 2010). \"Folding@home: Open beta release of the GPU3 client/core\". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved September 7, 2011.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":177,"type":"book","title":"Joseph Coffland (CEO of Cauldron Development LLC & lead developer at Folding@home) (October 13, 2011). \"Re: FAHClient V7.1.38 released (4th Open-Beta)\". Folding@home. phpBB Group. Archived from the original on September 21, 2012. Retrieved October 15, 2011.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":178,"type":"book","title":"\"NVIDIA GPU3 Linux/Wine Headless Install Guide\". Folding@home. phpBB Group. November 8, 2008. Archived from the original on September 21, 2012. Retrieved September 5, 2011.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":179,"type":"book","title":"Pande lab. \"Client Statistics by OS\". Folding@home. Stanford University. Archived from the original on November 28, 2012. Retrieved July 8, 2013.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":180,"type":"book","title":"Bruce Borden (June 25, 2013). \"GPU FahCore_17 is now available on Windows & native Linux\". Folding@home. phpBB Group. Retrieved September 30, 2014.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":181,"type":"book","title":"\"Futures in Biotech 27: Folding@home at 1.3 Petaflops\". Castroller.com. CastRoller. December 28, 2007. Archived from the original (Interview, webcast) on September 21, 2012. Retrieved September 20, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":182,"type":"book","title":"Edgar Luttmann; Daniel L. Ensign; Vishal Vaidyanathan; Mike Houston; Noam Rimon; Jeppe Øland; Guha Jayachandran; Mark Friedrichs; Vijay S. Pande (2008). \"Accelerating Molecular Dynamic Simulation on the Cell processor and PlayStation 3\". Journal of Computational Chemistry. 30 (2): 268–274. doi:10.1002/jcc.21054. PMID 18615421.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":183,"type":"book","title":"David E. Williams (October 20, 2006). \"PlayStation's serious side: Fighting disease\". CNN. Archived from the original on September 21, 2012. Retrieved September 20, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":184,"type":"book","title":"Jerry Liao (March 23, 2007). \"The Home Cure: PlayStation 3 to Help Study Causes of Cancer\". mb.com. Manila Bulletin Publishing Corporation. Archived from the original on September 21, 2012. Retrieved September 20, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":185,"type":"book","title":"Lou Kesten, Associated Press (March 26, 2007). \"Week in video-game news: 'God of War II' storms the PS2; a PS3 research project\". Post-Gazette.com. Pittsburgh Post-Gazette. Archived from the original on September 21, 2012. Retrieved September 20, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":186,"type":"book","title":"Elaine Chow (September 18, 2008). \"PS3 News Service, Life With Playstation, Now Up For Download\". Gizmodo.com. Gizmodo. Archived from the original on September 21, 2012. Retrieved September 20, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":187,"type":"book","title":"Vijay Pande (September 18, 2008). \"Life with Playstation – a new update to the FAH/PS3 client\". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved February 24, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":188,"type":"book","title":"Pande lab (May 30, 2012). \"PS3 FAQ\". Folding@home. Stanford University. Archived from the original (FAQ) on September 21, 2012. Retrieved July 8, 2013.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":189,"type":"book","title":"Eric Lempel (October 21, 2012). \"PS3 System Software Update (v4.30)\". PlayStation blog. Sony. Archived from the original on October 22, 2012. Retrieved October 21, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":190,"type":"book","title":"\"Termination of Life with PlayStation\". Life with PlayStation. Sony. November 6, 2012. Archived from the original on November 9, 2012. Retrieved November 8, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":191,"type":"book","title":"Vijay Pande (June 15, 2008). \"What does the SMP core do?\". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved September 7, 2011.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":192,"type":"book","title":"Daniel L. Ensign; Peter M. Kasson; Vijay S. Pande (2007). \"Heterogeneity Even at the Speed Limit of Folding: Large-scale Molecular Dynamics Study of a Fast-folding Variant of the Villin Headpiece\". Journal of Molecular Biology. 374 (3): 806–816. doi:10.1016/j.jmb.2007.09.069. PMC 3689540. PMID 17950314.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":193,"type":"book","title":"Vijay Pande (March 8, 2008). \"New Windows client/core development (SMP and classic clients)\". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved September 30, 2011.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":194,"type":"book","title":"Peter Kasson (Pande lab member) (July 15, 2009). \"new release: extra-large work units\". Folding@home. phpBB Group. Archived from the original on September 21, 2012. Retrieved October 9, 2011.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":195,"type":"book","title":"Vijay Pande (February 7, 2012). \"Update on \"bigadv-16\", the new bigadv rollout\". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved February 9, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":196,"type":"book","title":"Vijay Pande (July 2, 2011). \"Change in the points system for bigadv work units\". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved February 24, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":197,"type":"book","title":"Vijay Pande (January 15, 2014). \"Revised plans for BigAdv (BA) experiment\". Retrieved October 6, 2014.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":198,"type":"book","title":"Pande lab (March 23, 2012). \"Windows (FAH V7) Installation Guide\". Folding@home. Stanford University. Archived from the original (Guide) on September 21, 2012. Retrieved July 8, 2013.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":199,"type":"book","title":"Vijay Pande (March 29, 2011). \"Client version 7 now in open beta\". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved August 14, 2011.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":200,"type":"book","title":"Vijay Pande (March 22, 2012). \"Web page revamp and v7 rollout\". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved March 22, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":201,"type":"book","title":"Vijay Pande (March 31, 2011). \"Core 16 for ATI released; also note on NVIDIA GPU support for older boards\". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved September 7, 2011.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":202,"type":"book","title":"aschofield and jcoffland (October 3, 2011). \"Ticket #736 (Link to GPL in FAHControl)\". Folding@home. Trac. Archived from the original on October 12, 2012. Retrieved October 12, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":203,"type":"book","title":"Pande, Vijay (24 February 2014). \"Adding a completely new way to fold, directly in the browser\". folding.stanford.edu. Pande Lab, Stanford University. Retrieved 13 February 2015.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":204,"type":"book","title":"Pande, Vijay (July 7, 2015). \"First full version of our Folding@Home client for Android Mobile phones\". Folding@Home. Stanford University. Retrieved May 31, 2016.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":205,"type":"book","title":"\"Folding@Home\". Google Play. 2016. Retrieved May 31, 2016.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":206,"type":"book","title":"Lensink MF, Méndez R, Wodak SJ (December 2007). \"Docking and scoring protein complexes: CAPRI 3rd Edition\". Proteins. 69 (4): 704–18. doi:10.1002/prot.21804. PMID 17918726.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":207,"type":"book","title":"Gregory R. Bowman; Vijay S. Pande (2009). \"Simulated tempering yields insight into the low-resolution Rosetta scoring function\". Proteins: Structure, Function, and Bioinformatics. 74 (3): 777–88. doi:10.1002/prot.22210. PMID 18767152.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":208,"type":"book","title":"G. R. Bowman and V. S. Pande (2009). Hofmann, Andreas, ed. \"The Roles of Entropy and Kinetics in Structure Prediction\". PLoS ONE. 4 (6): e5840. Bibcode:2009PLoSO...4.5840B. doi:10.1371/journal.pone.0005840. PMC 2688754. PMID 19513117.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":209,"type":"book","title":"Gen_X_Accord, Vijay Pande (June 11, 2006). \"Folding@home vs. Rosetta@home\". Rosetta@home forums. University of Washington. Archived from the original on September 21, 2012. Retrieved September 20, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":210,"type":"book","title":"Vijay Pande (October 13, 2011). \"Comparison between FAH and Anton's approaches\". Folding@home. typepad.com. Archived from the original on September 21, 2012. Retrieved February 25, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":211,"type":"book","title":"Thomas J. Lane; Gregory R. Bowman; Kyle A Beauchamp; Vincent Alvin Voelz; Vijay S. Pande (2011). \"Markov State Model Reveals Folding and Functional Dynamics in Ultra-Long MD Trajectories\". Journal of the American Chemical Society. 133 (45): 18413–9. doi:10.1021/ja207470h. PMC 3227799. PMID 21988563.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":212,"type":"book","title":"David E. Shaw; et al. (2009). Millisecond-scale molecular dynamics simulations on Anton. Proceedings of the Conference on High Performance Computing Networking, Storage and Analysis. pp. 1–11. doi:10.1145/1654059.1654099. ISBN 978-1-60558-744-8.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":213,"type":"book","title":"David E. Shaw; et al. (2010). \"Atomic-Level Characterization of the Structural Dynamics of Proteins\". Science. 330 (6002): 341–346. Bibcode:2010Sci...330..341S. doi:10.1126/science.1187409. PMID 20947758.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":214,"type":"book","title":"David E. Shaw; Martin M. Deneroff; Ron O. Dror; Jeffrey S. Kuskin; Richard H. Larson; John K. Salmon; Cliff Young; Brannon Batson; Kevin J. Bowers; Jack C. Chao; Michael P. Eastwood; Joseph Gagliardo; J. P. Grossman; C. Richard Ho; Douglas J. Ierardi; et al. (2008). \"Anton, A Special-Purpose Machine for Molecular Dynamics Simulation\". Communications of the ACM. 51 (7): 91–97. doi:10.1145/1364782.1364802.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":215,"type":"book","title":"Ron O. Dror; Robert M. Dirks; J.P. Grossman; Huafeng Xu; David E. Shaw (2012). \"Biomolecular Simulation: A Computational Microscope for Molecular Biology\". Annual Review of Biophysics. 41: 429–52. doi:10.1146/annurev-biophys-042910-155245. PMID 22577825.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":216,"type":"book","title":"TJ Lane (Pande lab member) (June 6, 2011). \"Project 7610 & 7611 in Beta\". Folding@home. phpBB Group. Archived from the original on September 21, 2012. Retrieved February 25, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":217,"type":"book","title":"Pande lab. \"Project 7610 Description\". Folding@home. Archived from the original on September 21, 2012. Retrieved February 26, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":218,"type":"book","title":"Imran Haque (Pande lab member) (July 13, 2011). \"Re: Are my conversion for GPU flops relativly[sic] correct?\". Folding@home. phpBB Group. Archived from the original on September 21, 2012. Retrieved September 12, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":219,"type":"book","title":"Christopher Mims (November 8, 2010). \"Why China's New Supercomputer Is Only Technically the World's Fastest\". Technology Review. MIT. Archived from the original on September 21, 2012. Retrieved September 20, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":220,"type":"book","title":"Vijay Pande (November 9, 2008). \"Re: ATI and NVIDIA stats vs. PPD numbers\". Folding@home. phpBB Group. Archived from the original on September 21, 2012. Retrieved September 12, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":221,"type":"book","title":"Vijay Pande (November 9, 2008). \"Re: ATI and NVIDIA stats vs. PPD numbers\". Folding@home. phpBB Group. Archived from the original on September 21, 2012. Retrieved February 25, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":222,"type":"book","title":"Bruce Borden (July 12, 2011). \"Re: Are my conversion for GPU flops relativly[sic] correct?\". Folding@home. phpBB Group. Archived from the original on September 21, 2012. Retrieved September 12, 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false}],"recommendReferences":null,"auditState":2,"lemmaLevel":1,"origin":0,"originEnTitle":null,"originZhTitle":null,"pv":263,"auditType":0,"synonyms":null,"showEditTime":"2019.12.13 18:41","auditors":[{"uid":0,"name":"Ki.κe","pic":"https://web.archive.org/web/20221029033459/https://wx.qlogo.cn/mmopen/vi_32/y67kfr32Doib4wg71Jiau7jVWvharic3nRKgdRRQSl6koeQJCo0GQs2Krw0vwdFRsOWnHIQOwAZsSg5lIkIrFCOcQ/132","introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":"","jobBrief":"","role":0,"roleName":null,"title":"","professionalTitle":null,"phoneNo":null,"editable":true,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false}],"hasZhishiNav":false,"auditInfos":{},"isHistory":false};</script><script crossorigin="anonymous" src="./248.Folding@home - 搜狗科学百科_files/aegis.min.js.download"></script><script crossorigin="anonymous" src="./248.Folding@home - 搜狗科学百科_files/main_2020092401.js.download"></script><script crossorigin="anonymous" src="./248.Folding@home - 搜狗科学百科_files/react.production.min.js.download"></script><script crossorigin="anonymous" src="./248.Folding@home - 搜狗科学百科_files/react-dom.production.min.js.download"></script><script crossorigin="anonymous" src="./248.Folding@home - 搜狗科学百科_files/jquery-1.11.1.min.js.download"></script><script crossorigin="anonymous" src="./248.Folding@home - 搜狗科学百科_files/main_2022062701.js.download"></script><script crossorigin="anonymous" src="./248.Folding@home - 搜狗科学百科_files/main_66bbe21.js.download"></script><script crossorigin="anonymous" src="./248.Folding@home - 搜狗科学百科_files/react.production.min.js.download"></script><script crossorigin="anonymous" src="./248.Folding@home - 搜狗科学百科_files/react-dom.production.min.js.download"></script><script crossorigin="anonymous" src="./248.Folding@home - 搜狗科学百科_files/main_edf0f08.js.download"></script>
</body></html>