<!DOCTYPE html>
<!-- saved from url=(0083)https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm -->
<html class="" data-reactroot=""><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><script src="./519.生物信息学 - 搜狗科学百科_files/analytics.js.download" type="text/javascript"></script>
<script type="text/javascript">window.addEventListener('DOMContentLoaded',function(){var v=archive_analytics.values;v.service='wb';v.server_name='wwwb-app226.us.archive.org';v.server_ms=212;archive_analytics.send_pageview({});});</script>
<script type="text/javascript" src="./519.生物信息学 - 搜狗科学百科_files/bundle-playback.js.download" charset="utf-8"></script>
<script type="text/javascript" src="./519.生物信息学 - 搜狗科学百科_files/wombat.js.download" charset="utf-8"></script>
<script type="text/javascript">
  __wm.init("https://web.archive.org/web");
  __wm.wombat("https://baike.sogou.com/kexue/d10519.htm","20221025085755","https://web.archive.org/","web","/_static/",
	      "1666688275");
</script>
<link rel="stylesheet" type="text/css" href="./519.生物信息学 - 搜狗科学百科_files/banner-styles.css">
<link rel="stylesheet" type="text/css" href="./519.生物信息学 - 搜狗科学百科_files/iconochive.css">
<!-- End Wayback Rewrite JS Include -->
<meta name="save" content="history"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="baidu-site-verification" content="VWGb6TyYx8"><meta content="生物信息学 - 搜狗科学百科" name="keywords"><meta content="搜狗科学百科是一部有着平等、协作、分享、自由理念的网络科学全书，为每一个互联网用户创造一个涵盖所有领域知识、服务的中文知识性平台。" name="description"><meta http-equiv="x-dns-prefetch-control" content="on"><meta name="server" baike="235" ip="210" env="online"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025085755/https://cache.soso.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025085755/https://hhy.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025085755/https://pic.baike.soso.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025085755/https://ugc.qpic.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025085755/https://xui.ptlogin2.qq.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025085755/https://q1.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025085755/https://q2.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025085755/https://q3.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025085755/https://q4.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025085755/https://q.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025085755/https://img01.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025085755/https://img02.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025085755/https://img03.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025085755/https://img04.sogoucdn.com/"><link rel="Shortcut Icon" href="https://web.archive.org/web/20221025085755im_/https://www.sogou.com/images/logo/new/favicon.ico?v=4"><link rel="Bookmark" href="https://www.sogou.com/images/logo/new/favicon.ico?v=4"><link href="./519.生物信息学 - 搜狗科学百科_files/base_b849887.css" rel="stylesheet"><link href="./519.生物信息学 - 搜狗科学百科_files/detail_378aed5.css" rel="stylesheet"><link href="./519.生物信息学 - 搜狗科学百科_files/inviteAudit_7894507.css" rel="stylesheet"><link rel="stylesheet" href="./519.生物信息学 - 搜狗科学百科_files/highlight.min.css"><title>生物信息学 - 搜狗科学百科</title><style>.onekey-close {
	position: absolute;
	top: 16px;
	right: 16px;
	width: 24px;
	height: 24px;
	background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	text-indent: -999em;
	background-size: 84px;
	background-position: -63px 0;
}

.onekey-login {
	position: absolute;
	top: 16.4%;
	left: 0;
	right: 0;
	width: 100%;
}

/* .onekey-login-img {
    width: 75px;
    height: 75px;
    background: url("https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/images/sprite_wap_baike.png") no-repeat;
    background-size: 100px 91px;
    background-position: 0 0;
    background-repeat: no-repeat;
    margin: 0 auto;
} */

.onekey-login-title {
	text-align: center;
	padding-bottom: 3px;
	font-size: 21px;
	font-weight: bold;
	line-height: 30px;
	color: #000;
}

.onekey-login-txt {
	text-align: center;
	font-family: PingFangSC;
	font-size: 14px;
	line-height: 20px;
	color: #8f8f8f;
}

.onekey-login-qq,
.onekey-login-wx,
.onekey-login-phone {
	display: block;
	width: 245px;
	height: 54px;
	border-radius: 45px;
	text-align: center;

	margin: 0 auto;
	font-size: 17px;
	line-height: 24px;
	color: #000;
	/* padding: 16px 77px; */
	border-radius: 12px;
	border: solid 1px #e0e0e0;
}
.onekey-qq-content,
.onekey-vx-content,
.onekey-phone-content {
	display: inline-block;
	margin-top: 16px;
}
.onekey-qq-content {
	padding: 0 5px;
}

.onekey-login-qq {
	margin-top: 48px;
	margin-bottom: 24px;
}

.onekey-login-qq:before {
	display: inline-block;
	content: "";
	width: 20px;
	height: 20px;
	background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	background-size: 80px;
	background-position: -20px 0;
	vertical-align: top;
	margin: 17px 8px 0 0;
}

.onekey-login-wx {
	margin-bottom: 24px;
}

.onekey-login-wx:before {
	display: inline-block;
	content: "";
	width: 21px;
	height: 21px;
	background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	background-size: 84px;
	background-position: 0 0;
	vertical-align: top;
	margin: 17px 10px 0 0;
}

.onekey-login-phone {
}

.onekey-login-phone:before {
	display: inline-block;
	content: "";
	width: 21px;
	height: 21px;
	background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	background-size: 84px;
	background-position: -42px 0;
	vertical-align: top;
	margin: 17px 10px 0 0;
}

.onekey-fixed {
	z-index: 100;
	position: fixed;
	top: 0;
	bottom: 0;
	left: 0;
	right: 0;
	background: #fff;
	width: 100%;
	height: 100%;
}

.onekey-fixed.forbid {
	z-index: 100;
	position: fixed;
	top: auto;
	bottom: 68px;
	left: 9%;
	right: 0;
	background: rgba(0, 0, 0, 0.7);
	width: 82%;
	height: 43px;
	border-radius: 25px;
	color: #ffffff;
}
.onekey-login-title.forbid {
	text-align: center;
	padding-bottom: 3px;
	font-size: 14px;
	font-weight: normal;
	line-height: 30px;
	color: white;
}
</style><style>#login_mask {
  background: #000;
  opacity: 0.5;
  filter: alpha(opacity=50);
  position: fixed;
  /*fixed好像在哪个IE上有BUG，先用用*/
  left: 0;
  top: 0;
  z-index: 999;
  height: 100%;
}

#login_iframe_container {
  position: fixed;
  width: 550px;
  height: 360px;
  z-index: 1020;
  background-color: #ffffff;
}

@media screen and (max-width: 828px) {
  #login_iframe_container {
    top: 50% !important;
    left: 50% !important;
    transform: translate(-50%, -50%);
  }
}

#login_iframe_container.new-login {
  width: 550px;
  height: 360px;
  background-image: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/background_2a4a8a6.png);
}

#login_iframe_container.new-login.no-bg {
  background: #fff;
}

#login_iframe_container.new-login .login-title {
  width: 100%;
  height: 42px;
  line-height: 42px;
  text-align: center;
  font-size: 30px;
  letter-spacing: 0.19px;
  color: #ffffff;
  margin-top: 62px;
}
#login_iframe_container.new-login .forbid-title {
  width: 100%;
  height: 42px;
  line-height: 42px;
  text-align: center;
  font-size: 24px;
  letter-spacing: 0.19px;
  color: #333333;
  margin-top: 150px;
}

#login_iframe_container.new-login.no-bg .login-title {
  color: #333333;
}

#login_iframe_container.new-login .login-subtitle {
  width: 100%;
  height: 18px;
  line-height: 18px;
  font-size: 13px;
  letter-spacing: 0.08px;
  color: #ffffff;
  text-align: center;
  margin-top: 9px;
  margin-bottom: 43px;
}

#login_iframe_container.new-login.no-bg .login-subtitle {
  color: #999999;
}

#login_iframe_container.new-login .login-subtitle::before {
  content: '';
  display: inline-block;
  width: 10px;
  height: 1px;
  background-color: #ffffff;
  position: relative;
  top: -4px;
  left: -5px;
}

#login_iframe_container.new-login .login-subtitle::after {
  content: '';
  display: inline-block;
  width: 10px;
  height: 1px;
  background-color: #ffffff;
  position: relative;
  top: -4px;
  left: 5px;
}

#login_iframe_container.new-login.no-bg .login-subtitle::before {
  background-color: #999999;
}

#login_iframe_container.new-login.no-bg .login-subtitle::after {
  background-color: #999999;
}

#login_iframe_container.new-login .close-btn {
  position: absolute;
  top: 20px;
  right: 20px;
  width: 12px;
  height: 12px;
  background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/login-sprites_e3853e5.png) -59px -10px;
  background-size: 81px 91px;
  cursor: pointer;
}

#login_iframe_container.new-login .login-btn {
  width: 220px;
  height: 47px;
  border-radius: 24px;
  border: solid 1px #dddddd;
  background-color: #ffffff;
  margin: 0 auto;
  margin-top: 28px;
  position: relative;
  display: block;
}

#login_iframe_container.new-login .login-btn .login-icon {
  position: absolute;
}

#login_iframe_container.new-login .login-btn .login-text {
  width: 61px;
  height: 47px;
  line-height: 47px;
  vertical-align: middle;
  font-size: 15px;
  letter-spacing: 0.1px;
  color: #666666;
  position: absolute;
  right: 62px;
}

#login_iframe_container.new-login .login-btn.qq-btn .login-icon {
  width: 22px;
  height: 27px;
  top: 10px;
  left: 67px;
  background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/login-sprites_e3853e5.png) -10px -54px;
  background-size: 81px 91px;
}

#login_iframe_container.new-login .login-btn.qq-btn .login-text {
  right: 59px;
}

#login_iframe_container.new-login .login-btn.wechat-btn .login-icon {
  width: 29px;
  height: 24px;
  top: 12px;
  left: 62px;
  background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/login-sprites_e3853e5.png) -10px -10px;
  background-size: 81px 91px;
}</style><style>/* -- container -- */
.rodal,
.rodal-mask {
    top: 0;
    left: 0;
    width: 100%;
    height: 100%;
    z-index: 100;
}

.rodal {
    position: fixed;
}

/* -- mask -- */
.rodal-mask {
    position: fixed;
    background: rgba(0, 0, 0, .5);
}

/* -- dialog -- */
.rodal-dialog {
    position: absolute;
    z-index: 101;
    background: #fff;
    border-radius: 3px;
    box-shadow: 0 1px 3px rgba(0, 0, 0, .2);
}

.rodal-center {
    top: 50%;
    transform: translateY(-50%);
    left: 0;
    right: 0;
    margin: 0 auto;
}

.rodal-bottom {
    left: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

.rodal-top {
    left: 0;
    right: 0;
    top: 0;
    margin: auto;
}

.rodal-left {
    top: 0;
    left: 0;
    bottom: 0;
    margin: auto;
}

.rodal-right {
    top: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

/* -- close button -- */
.rodal-close {
    position: absolute;
    cursor: pointer;
    top: 16px;
    right: 16px;
    width: 16px;
    height: 16px;
}

.rodal-close:before,
.rodal-close:after {
    position: absolute;
    content: '';
    height: 2px;
    width: 100%;
    top: 50%;
    left: 0;
    margin-top: -1px;
    background: #999;
    border-radius: 100%;
    -webkit-transition: background .2s;
    transition: background .2s;
}

.rodal-close:before {
    -webkit-transform: rotate(45deg);
    transform: rotate(45deg);
}

.rodal-close:after {
    -webkit-transform: rotate(-45deg);
    transform: rotate(-45deg);
}

.rodal-close:hover:before,
.rodal-close:hover:after {
    background: #333;
}

/* -- fade -- */
/* @-webkit-keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

@keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

.rodal-fade-enter {
    -webkit-animation: rodal-fade-enter both ease-in;
    animation: rodal-fade-enter both ease-in;
} */

@-webkit-keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

@keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

.rodal-fade-leave {
    -webkit-animation: rodal-fade-leave both ease-out;
    animation: rodal-fade-leave both ease-out;
}

/* -- zoom -- */
@-webkit-keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-enter {
    -webkit-animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-leave {
    -webkit-animation: rodal-zoom-leave both;
    animation: rodal-zoom-leave both;
}

/* -- slideDown -- */
@-webkit-keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-enter {
    -webkit-animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-leave {
    -webkit-animation: rodal-slideDown-leave both;
    animation: rodal-slideDown-leave both;
}

/* -- slideLeft -- */
@-webkit-keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-enter {
    -webkit-animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-leave {
    -webkit-animation: rodal-slideLeft-leave both;
    animation: rodal-slideLeft-leave both;
}

/* -- slideRight -- */
@-webkit-keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-enter {
    -webkit-animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-leave {
    -webkit-animation: rodal-slideRight-leave both;
    animation: rodal-slideRight-leave both;
}

/* -- slideUp -- */
@-webkit-keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-enter {
    -webkit-animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
    animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
}

@-webkit-keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-leave {
    -webkit-animation: rodal-slideUp-leave both;
    animation: rodal-slideUp-leave both;
}

/* -- flip -- */
@-webkit-keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

@keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

.rodal-flip-enter {
    -webkit-animation: rodal-flip-enter both ease-in;
    animation: rodal-flip-enter both ease-in;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

@-webkit-keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

@keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

.rodal-flip-leave {
    -webkit-animation: rodal-flip-leave both;
    animation: rodal-flip-leave both;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

/* -- rotate -- */
@-webkit-keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-enter {
    -webkit-animation: rodal-rotate-enter both;
    animation: rodal-rotate-enter both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

@-webkit-keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-leave {
    -webkit-animation: rodal-rotate-leave both;
    animation: rodal-rotate-leave both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

/* -- door -- */
@-webkit-keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

@keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

.rodal-door-enter {
    -webkit-animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

@keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

.rodal-door-leave {
    -webkit-animation: rodal-door-leave both;
    animation: rodal-door-leave both;
}</style><style>/* -- container -- */
.rodal,
.rodal-mask {
    top: 0;
    left: 0;
    width: 100%;
    height: 100%;
    z-index: 100;
}

.rodal {
    position: fixed;
}

/* -- mask -- */
.rodal-mask {
    position: fixed;
    background: rgba(0, 0, 0, .5);
}

/* -- dialog -- */
.rodal-dialog {
    position: absolute;
    z-index: 101;
    background: #fff;
    border-radius: 3px;
    box-shadow: 0 1px 3px rgba(0, 0, 0, .2);
}

.rodal-center {
    top: 50%;
    transform: translateY(-50%);
    left: 0;
    right: 0;
    margin: 0 auto;
}

.rodal-bottom {
    left: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

.rodal-top {
    left: 0;
    right: 0;
    top: 0;
    margin: auto;
}

.rodal-left {
    top: 0;
    left: 0;
    bottom: 0;
    margin: auto;
}

.rodal-right {
    top: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

/* -- close button -- */
.rodal-close {
    position: absolute;
    cursor: pointer;
    top: 16px;
    right: 16px;
    width: 16px;
    height: 16px;
}

.rodal-close:before,
.rodal-close:after {
    position: absolute;
    content: '';
    height: 2px;
    width: 100%;
    top: 50%;
    left: 0;
    margin-top: -1px;
    background: #999;
    border-radius: 100%;
    -webkit-transition: background .2s;
    transition: background .2s;
}

.rodal-close:before {
    -webkit-transform: rotate(45deg);
    transform: rotate(45deg);
}

.rodal-close:after {
    -webkit-transform: rotate(-45deg);
    transform: rotate(-45deg);
}

.rodal-close:hover:before,
.rodal-close:hover:after {
    background: #333;
}

/* -- fade -- */
/* @-webkit-keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

@keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

.rodal-fade-enter {
    -webkit-animation: rodal-fade-enter both ease-in;
    animation: rodal-fade-enter both ease-in;
} */

@-webkit-keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

@keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

.rodal-fade-leave {
    -webkit-animation: rodal-fade-leave both ease-out;
    animation: rodal-fade-leave both ease-out;
}

/* -- zoom -- */
@-webkit-keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-enter {
    -webkit-animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-leave {
    -webkit-animation: rodal-zoom-leave both;
    animation: rodal-zoom-leave both;
}

/* -- slideDown -- */
@-webkit-keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-enter {
    -webkit-animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-leave {
    -webkit-animation: rodal-slideDown-leave both;
    animation: rodal-slideDown-leave both;
}

/* -- slideLeft -- */
@-webkit-keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-enter {
    -webkit-animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-leave {
    -webkit-animation: rodal-slideLeft-leave both;
    animation: rodal-slideLeft-leave both;
}

/* -- slideRight -- */
@-webkit-keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-enter {
    -webkit-animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-leave {
    -webkit-animation: rodal-slideRight-leave both;
    animation: rodal-slideRight-leave both;
}

/* -- slideUp -- */
@-webkit-keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-enter {
    -webkit-animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
    animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
}

@-webkit-keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-leave {
    -webkit-animation: rodal-slideUp-leave both;
    animation: rodal-slideUp-leave both;
}

/* -- flip -- */
@-webkit-keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

@keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

.rodal-flip-enter {
    -webkit-animation: rodal-flip-enter both ease-in;
    animation: rodal-flip-enter both ease-in;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

@-webkit-keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

@keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

.rodal-flip-leave {
    -webkit-animation: rodal-flip-leave both;
    animation: rodal-flip-leave both;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

/* -- rotate -- */
@-webkit-keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-enter {
    -webkit-animation: rodal-rotate-enter both;
    animation: rodal-rotate-enter both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

@-webkit-keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-leave {
    -webkit-animation: rodal-rotate-leave both;
    animation: rodal-rotate-leave both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

/* -- door -- */
@-webkit-keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

@keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

.rodal-door-enter {
    -webkit-animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

@keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

.rodal-door-leave {
    -webkit-animation: rodal-door-leave both;
    animation: rodal-door-leave both;
}</style>
<!--百度统计-->
<script>
   var _hmt = _hmt || [];
   (function() {
      var hm = document.createElement("script");
      hm.src = "https://hm.baidu.com/hm.js?3c7614be3026469d5a60f41ab30b5082";
      var s = document.getElementsByTagName("script")[0]; 
      s.parentNode.insertBefore(hm, s);
      })();
</script>
</head>
<body class=""><!-- BEGIN WAYBACK TOOLBAR INSERT -->
<style type="text/css">
body {
  margin-top:0 !important;
  padding-top:0 !important;
  /*min-width:800px !important;*/
}
</style>
<script>__wm.rw(0);</script>
<div id="wm-ipp-base" lang="en" style="display: block; direction: ltr;">
</div><div id="wm-ipp-print">The Wayback Machine - https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm</div>
<script type="text/javascript">//<![CDATA[
__wm.bt(675,27,25,2,"web","https://baike.sogou.com/kexue/d10519.htm","20221025085755",1996,"/_static/",["/_static/css/banner-styles.css?v=S1zqJCYt","/_static/css/iconochive.css?v=qtvMKcIJ"], false);
  __wm.rw(1);
//]]></script>
<!-- END WAYBACK TOOLBAR INSERT --><script>window._gtag=window._gtag||{};window._gtag.shouldGrayed = false;if ('759b4d3db7d3488b862f5116071259b9') window._gtag.traceId = '759b4d3db7d3488b862f5116071259b9';if ({"illegality":true}) window.userInfo = {"illegality":true};</script><div class="topnavbox"><ul class="topnav"><li><a href="https://web.archive.org/web/20221025085755/https://www.sogou.com/web?query=">网页</a></li><li><a href="https://web.archive.org/web/20221025085755/https://weixin.sogou.com/weixin?p=75351201">微信</a></li><li><a href="https://web.archive.org/web/20221025085755/https://zhihu.sogou.com/zhihu?p=75351218">知乎</a></li><li><a href="https://web.archive.org/web/20221025085755/https://pic.sogou.com/pics?query=">图片</a></li><li><a href="https://web.archive.org/web/20221025085755/https://v.sogou.com/v?query=">视频</a></li><li><a href="https://web.archive.org/web/20221025085755/https://mingyi.sogou.com/">医疗</a></li><li class="cur"><strong>科学</strong></li><li><a href="https://web.archive.org/web/20221025085755/https://hanyu.sogou.com/">汉语</a></li><li><a href="https://web.archive.org/web/20221025085755/https://wenwen.sogou.com/">问问</a></li><li><a href="https://web.archive.org/web/20221025085755/https://www.sogou.com/docs/more.htm">更多<span class="topraquo">»</span></a></li></ul></div><div id="header"><div class="header-wrap"><a class="header-logo" href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue"></a><div class="header-search"><div class="querybox" id="suggBox"><form><input id="searchInput" class="query" type="text" placeholder="搜科学领域专业百科词条" name="query" autocomplete="off" value=""><a href="javascript:;" class="query-search"></a></form></div></div><div class="header-rgt"><span class="btn-header-rgt btn-edit" id="editLemma">创建</span><div class="header-user no-login"></div></div></div></div><div class="fixed-placeholder" style="visibility:none"></div><div id="container" class=""><div class="content lemma-level1"><div class="detail-title" id="abstract-title"><h1>生物信息学</h1><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#!" class="detail-edit">编辑</a></div><div class="section_content" data-id="14995387513307666"><div class="text_img ed_imgfloat_right"><a class="ed_image_link" data-src="https://img01.sogoucdn.com/app/a/200698/sogou_science_489" data-bigsrc="https://img01.sogoucdn.com/app/a/200698/sogou_science_489?w=220&amp;h=242&amp;titlename=%E4%BA%BA%E7%B1%BBX%E6%9F%93%E8%89%B2%E4%BD%93%E5%9B%BE%E8%B0%B1(%E6%9D%A5%E8%87%AA%E5%9B%BD%E5%AE%B6%E7%94%9F%E7%89%A9%E6%8A%80%E6%9C%AF%E4%BF%A1%E6%81%AF%E4%B8%AD%E5%BF%83%E7%BD%91%E7%AB%99)%E3%80%82" title="点击查看大图" data-w="220" data-h="242" style="background-image:url(https://web.archive.org/web/20221025085755im_/https://img01.sogoucdn.com/app/a/200698/sogou_science_489)" href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#!"></a><div class="text_img_title">人类X染色体图谱(来自国家生物技术信息中心网站)。</div></div><div><p></p> 
<p><b>生物信息学</b>是一个跨学科的领域，目的是开发理解生物数据的方法和软件工具。生物信息学作为一个跨学科的科学领域，结合了生物学、计算机科学、信息工程、数学和统计学的相关知识用于分析和解释生物数据。通过数学和统计技术，生物信息学已经被用于对生物数据库进行计算机分析。 </p>
<p>生物信息学既是生物研究主体的总称，该研究主体使用计算机编程作为其方法论的一部分；也是对重复使用的特定分析“管道”的引用，特别是在基因组学领域。生物信息学的常见用途包括候选基因的鉴定和单核苷酸多态性(SNPs)。通常，这种鉴定的目的是为了更好地理解疾病的遗传基础、独特的适应性、理想的特性(特别是农业物种)或种群间的差异。以一种不太正式的方式，生物信息学也试图理解核酸和蛋白质序列中的组织原则，称为蛋白质组学。<sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_1" class="kx_ref">[1]</a></sup> </p></div></div><div id="catalog"><h2 class="title2">目录<a href="javascript:" class="detail-edit">编辑</a></h2><div class="catalog_wrap" style=""><ul class="catalog_list col4"><li><span class="order">1</span><a href="javascript:" data-level="1" data-id="14995387513307667">介绍</a></li><li class="secondary_catalog"><span>1.1 </span><a href="javascript:" data-id="14995387513307667">历史</a></li><li class="secondary_catalog"><span>1.2 </span><a href="javascript:" data-id="14995387513307667">目标</a></li><li class="secondary_catalog"><span>1.3 </span><a href="javascript:" data-id="14995387513307667">与其他领域的关系</a></li><li><span class="order">2</span><a href="javascript:" data-level="1" data-id="14995387530084878">序列分析</a></li><li class="secondary_catalog"><span>2.1 </span><a href="javascript:" data-id="14995387530084878">DNA测序</a></li><li class="secondary_catalog"><span>2.2 </span><a href="javascript:" data-id="14995387530084878">序列组装</a></li><li class="secondary_catalog"><span>2.3 </span><a href="javascript:" data-id="14995387530084878">基因组注释</a></li><li class="secondary_catalog"><span>2.4 </span><a href="javascript:" data-id="14995387530084878">计算进化生物学</a></li><li class="secondary_catalog"><span>2.5 </span><a href="javascript:" data-id="14995387530084878">比较基因组学</a></li><li class="secondary_catalog"><span>2.6 </span><a href="javascript:" data-id="14995387530084878">泛基因组学</a></li></ul><ul class="catalog_list col4"><li class="secondary_catalog"><span>2.7 </span><a href="javascript:" data-id="14995387530084878">疾病遗传学</a></li><li class="secondary_catalog"><span>2.8 </span><a href="javascript:" data-id="14995387530084878">癌症突变分析</a></li><li><span class="order">3</span><a href="javascript:" data-level="1" data-id="14995387530084879">基因和蛋白质表达</a></li><li class="secondary_catalog"><span>3.1 </span><a href="javascript:" data-id="14995387530084879">基因表达分析</a></li><li class="secondary_catalog"><span>3.2 </span><a href="javascript:" data-id="14995387530084879">蛋白质表达分析</a></li><li class="secondary_catalog"><span>3.3 </span><a href="javascript:" data-id="14995387530084879">调控分析</a></li><li><span class="order">4</span><a href="javascript:" data-level="1" data-id="14995387530084880">细胞组织分析</a></li><li class="secondary_catalog"><span>4.1 </span><a href="javascript:" data-id="14995387530084880">显微镜和图像分析</a></li><li class="secondary_catalog"><span>4.2 </span><a href="javascript:" data-id="14995387530084880">蛋白质定位</a></li><li class="secondary_catalog"><span>4.3 </span><a href="javascript:" data-id="14995387530084880">染色质的核组织</a></li><li><span class="order">5</span><a href="javascript:" data-level="1" data-id="14995387530084881">结构生物信息学</a></li></ul><ul class="catalog_list col4"><li><span class="order">6</span><a href="javascript:" data-level="1" data-id="14995387530084882">网络和系统生物学</a></li><li class="secondary_catalog"><span>6.1 </span><a href="javascript:" data-id="14995387530084882">分子相互作用网络</a></li><li><span class="order">7</span><a href="javascript:" data-level="1" data-id="14995387530084883">其他</a></li><li class="secondary_catalog"><span>7.1 </span><a href="javascript:" data-id="14995387530084883">文献分析</a></li><li class="secondary_catalog"><span>7.2 </span><a href="javascript:" data-id="14995387530084883">高通量图像分析</a></li><li class="secondary_catalog"><span>7.3 </span><a href="javascript:" data-id="14995387530084883">高通量单细胞数据分析</a></li><li class="secondary_catalog"><span>7.4 </span><a href="javascript:" data-id="14995387530084883">生物多样性信息学</a></li><li class="secondary_catalog"><span>7.5 </span><a href="javascript:" data-id="14995387530084883">本体论和数据集成</a></li><li><span class="order">8</span><a href="javascript:" data-level="1" data-id="14995387546862090">数据库</a></li><li><span class="order">9</span><a href="javascript:" data-level="1" data-id="14995387546862091">软件和工具</a></li><li class="secondary_catalog"><span>9.1 </span><a href="javascript:" data-id="14995387546862091">开源生物信息学软件</a></li></ul><ul class="catalog_list col4"><li class="secondary_catalog"><span>9.2 </span><a href="javascript:" data-id="14995387546862091">生物信息学中的Web服务</a></li><li class="secondary_catalog"><span>9.3 </span><a href="javascript:" data-id="14995387546862091">生物信息学工作流管理系统</a></li><li class="secondary_catalog"><span>9.4 </span><a href="javascript:" data-id="14995387546862091">生物计算机和生物计算机对象</a></li><li><span class="order">10</span><a href="javascript:" data-level="1" data-id="14995387546862092">教育平台</a></li><li><span class="order">11</span><a href="javascript:" data-level="1" data-id="14995387546862093">会议</a></li><li><span class="order">12</span><a href="javascript:" data-level="1" data-id="references">参考文献</a></li></ul></div></div><div id="paragraphs"><div><div id="par_14995387513307667"><h2 class="title">1 介绍<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>生物信息学已经成为生物学许多领域的重要组成部分。在实验分子生物学中，图像和信号处理等生物信息学技术允许从大量原始数据中提取有用的结果。在遗传学领域，它有助于对基因组及其观察到的突变进行测序和注释。它在生物文献的文本挖掘以及生物和基因本体的发展中起着组织和查询生物数据的作用。它还在基因和蛋白质表达和调节的分析中发挥作用。生物信息学工具有助于比较遗传和基因组数据，更概括的说，有助于理解分子生物学的进化方面。在更综合的层面上，它有助于分析和编目作为系统生物学重要组成部分的生物路径和网络。在结构生物学中，它有助于对DNA、RNA、<sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_2" class="kx_ref">[2]</a></sup><sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_3" class="kx_ref">[3]</a></sup> 蛋白质<sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_4" class="kx_ref">[4]</a></sup> 以及生物分子间的相互作用进行模拟和建模。<sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_5" class="kx_ref">[5]</a></sup><sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_6" class="kx_ref">[6]</a></sup><sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_7" class="kx_ref">[7]</a></sup><sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_8" class="kx_ref">[8]</a></sup> </p> 
<h3>1.1 <span>历史</span></h3> 
<p>历史上，生物信息学这个术语和它今天的意义并不一样。波利恩·霍格威和本·海茨帕在1970年创造了这个词，用来指对生物系统中信息过程的研究。<sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_9" class="kx_ref">[9]</a></sup><sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_10" class="kx_ref">[10]</a></sup><sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_11" class="kx_ref">[11]</a></sup> 这一定义将生物信息学定位为一个平行于生物化学(研究生物系统中的化学过程)的领域。<sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_9" class="kx_ref">[9]</a></sup> </p> 
<p><strong>序列</strong></p> 
<p></p><p></p><div class="text_img ed_imgfloat_right">
            <a class="ed_image_link lazyLoad" data-src="https://img04.sogoucdn.com/app/a/200698/sogou_science_4425" data-bigsrc="" title="点击查看大图" href="javascript:" data-observer="true"></a>
            <div class="text_img_title">遗传物质序列在生物信息学中经常使用，使用计算机比手工更容易管理。</div>   
        </div> <p></p><p></p> 
<p>20世纪50年代初，弗雷德里克·桑格确定胰岛素序列后，蛋白质序列的获取成为可能，计算机成为分子生物学中的关键。手动比较多个序列被证明是不切实际的。这一领域的先驱是玛格丽特·奥克利·戴霍夫。<sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_12" class="kx_ref">[12]</a></sup> 她编译了第一批蛋白质序列数据库，最初作为书籍出版，<sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_13" class="kx_ref">[13]</a></sup> 并开创了序列比对和分子进化的方法。<sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_14" class="kx_ref">[14]</a></sup> 生物信息学的另一个早期贡献者是艾文·卡巴特，他在1970年开创了生物序列分析方法，他在1980年至1991年间与泰特·吴一起发布了大量的抗体序列。<sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_15" class="kx_ref">[15]</a></sup> </p> 
<h3>1.2 <span>目标</span></h3> 
<p>为了研究正常细胞活动在不同疾病状态下是如何改变的，生物数据必须结合起来，形成这些活动的综合图像。因此，生物信息学领域已经发展到现在最紧迫的任务是分析和解释各种类型的数据。这包括核苷酸和氨基酸序列、蛋白质结构域和蛋白质结构。<sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_16" class="kx_ref">[16]</a></sup> 分析和解释数据的实际过程被称为计算生物学。生物信息学和计算生物学中的重要子学科包括: </p> 
<ul>
 <li>开发和执行计算机程序，使各种类型的信息能够得到有效的访问、使用和管理</li> 
 <li>开发评估大数据集之间关系的新算法(数学公式)和统计方法。例如，定位序列中的基因，预测蛋白质结构和/或功能，并将蛋白质序列聚类到相关序列的家族中等的各种方法。</li>
</ul> 
<p>生物信息学的主要目标是增加对生物过程的理解。然而，它与其他方法的不同之处在于它专注于开发和应用计算密集型技术来实现这一目标。例子包括:模式识别、数据挖掘、机器学习算法和可视化。该领域的主要研究工作包括序列比对、基因挖掘、基因组组装、药物设计、药物发现、蛋白质结构比对、蛋白质结构预测、基因表达预测和蛋白质-蛋白质相互作用、全基因组关联研究、进化和细胞分裂/有丝分裂建模。 </p>
<p>生物信息学现在需要创建和发展数据库、算法、计算和统计技术以及理论，用于解决生物数据管理和分析中出现的形式和实际问题。 </p>
<p>在过去的几十年里，基因组和其他分子研究技术的快速发展以及信息技术的发展结合起来，产生了大量与分子生物学相关的信息。生物信息学是这些数学和计算方法的名称，用来收集对生物过程的理解。 </p>
<p>生物信息学中常见的形式包括绘制和分析DNA和蛋白质序列，对DNA和蛋白质序列进行比较，以及创建和查看蛋白质结构的三维模型。 </p> 
<h3>1.3 <span>与其他领域的关系</span></h3> 
<p>生物信息学是一个类似于但不同于生物计算的科学领域，虽然它通常被认为是计算生物学的同义词。生物计算使用生物工程和生物学来构建生物计算机，而生物信息学使用计算来更好地理解生物学。生物信息学和计算生物学涉及生物数据的分析，特别是DNA、RNA和蛋白质序列。生物信息学领域从20世纪90年代中期开始经历了爆炸性的增长，这主要是由人类基因组计划和DNA测序技术的快速进步推动的。 </p>
<p>分析生物数据产生有意义的信息包括编写和运行软件程序，这些程序使用来自图论、人工智能、软计算、数据挖掘、图像处理和计算机模拟的算法。算法又依赖于理论基础，如离散数学、控制理论、系统理论、信息论和统计学。 </p></div></div><div id="par_14995387530084878"><h2 class="title">2 序列分析<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p></p><p></p><div class="text_img ed_imgfloat_right">
            <a class="ed_image_link lazyLoad" data-src="https://img01.sogoucdn.com/app/a/200698/sogou_science_498" data-bigsrc="" title="点击查看大图" href="javascript:" data-observer="true"></a>
            <div class="text_img_title">不同基因或蛋白质的序列可以并排排列以测量它们的相似性。这种比对比较了蛋白质序列和含有WPP结构域的基因组序列。</div>   
        </div> <p></p><p></p> 
<p>自从噬菌体φ-X174在1977年被测序以来，<sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_17" class="kx_ref">[17]</a></sup> 分析序列信息可以确定编码蛋白质、RNA基因、调节序列、结构基序和重复序列的基因。一个物种内或不同物种之间的基因比较可以显示蛋白质功能之间的相似性，或物种之间的关系(使用分子系统学构建系统进化树)。随着数据量的增长，很久以前手工分析DNA序列变得不切实际。今天，计算机程序如BLAST每天都被用来从超过260,000个生物体中搜索序列，<sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_18" class="kx_ref">[18]</a></sup>包含超过1900亿个核苷酸。 这些程序可以补偿基因序列中的突变(交换、删除或插入碱基)，以识别相关但不相同的序列。这种序列比对的变体用于测序过程本身。对于序列片段分类的特殊任务，现代基于k-mer的软件如Kraken实现了比对方法无法达到的通量。 </p> 
<h3>2.1 <span>DNA测序</span></h3> 
<p>在分析序列之前，必须先获得它们。DNA测序仍然是一个不小的问题，因为原始数据可能有噪声或受到微弱信号的影响。多种算法已经被开发出来用于分析多种测序方法的结果。 </p> 
<h3>2.2 <span>序列组装</span></h3> 
<p>大多数DNA测序技术产生的短序列片段需要组装才能获得完整的基因或基因组序列。所谓的鸟枪测序技术(例如，由基因组研究所(TIGR)用来对第一个细菌基因组流感嗜血杆菌进行测序)<sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_19" class="kx_ref">[19]</a></sup>产生了成千上万个小的DNA片段的序列(根据测序技术，长度从35到900个核苷酸不等)。这些片段的末端重叠，当通过基因组组装程序正确排列时，可以用来重建完整的基因组。鸟枪测序能快速产生序列数据，但是对于更大的基因组来说，组装片段的任务可能相当复杂。对于像人类基因组一样大的基因组来说，在大内存、多处理器计算机上组装片段可能需要很多天的CPU时间，而最终的组装通常包含许多必须在以后填补的空白。鸟枪法测序是当今几乎所有测序基因组的首选方法。基因组组装算法是生物信息学研究的一个关键领域。 </p> 
<h3>2.3 <span>基因组注释</span></h3> 
<p>在基因组学的背景下，注释是在基因序列中标记基因和其他生物特征的过程。这个过程需要自动化，因为大多数基因组太大，不能用手工注释，更不用说注释尽可能多的基因组了，因为测序的速度已经不再是瓶颈。注释之所以成为可能，是因为基因有可识别的起始和终止区域，尽管在这些区域中发现的确切序列可能因基因而异。 </p>
<p>基因组研究所的团队于1995年 <sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_19" class="kx_ref">[19]</a></sup> 首次描述了第一个综合基因组注释系统。该团队对一种自然生物——流感嗜血杆菌<sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_19" class="kx_ref">[19]</a></sup>——的基因组进行了第一次完整的测序和分析。欧文·怀特(Owen White)设计并构建了一个软件系统用于识别编码所有蛋白质的基因，转运RNA、核糖体RNA(以及其他位点)，并进行初始功能分配。目前大多数基因组注释系统的工作原理相似，但可用于基因组DNA分析的程序，如经过训练并用于在流感嗜血杆菌中发现蛋白质编码基因的GeneMark程序，正在不断变化和改进。 </p>
<p>继人类基因组计划在2003年结束后留下来需要实现的目标之后，美国国家人类基因组研究所开发的一个新项目出现了。所谓的编码项目(ENCODE project)是人类基因组功能元素的协作数据收集，它使用下一代DNA测序技术和基因组平铺阵列，这些技术能够以大幅降低的单位成本自动生成大量数据，同时保证相同水平的精确度(基本调用误差)和保真度(装配误差)。 </p> 
<h3>2.4 <span>计算进化生物学</span></h3> 
<p>进化生物学研究物种的起源和世代，以及它们随时间的变化。生物信息学辅助进化生物学家通过使研究人员能够: </p> 
<ul>
 <li>通过测量大量生物的DNA的变化来追踪它们的进化，而不是仅仅通过物理分类或生理观察，</li> 
 <li>比较整个基因组，这允许研究更复杂的进化事件，例如基因复制、水平基因转移和预测细菌物种形成中的重要因素，</li> 
 <li>建立复杂的计算群体遗传学模型来预测系统随时间变化的结果<sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_20" class="kx_ref">[20]</a></sup></li> 
 <li>跟踪和共享越来越多的物种和有机体的信息</li>
</ul> 
<p>进一步的工作正在努力重建比现在更加复杂的生命树。 </p>
<p>计算机科学中使用遗传算法的研究领域有时与计算进化生物学混淆，但这两个领域不一定相关。 </p> 
<h3>2.5 <span>比较基因组学</span></h3> 
<p>比较基因组分析的核心是建立不同生物体中基因(正交性分析)或其他基因组特征之间的对应关系。正是这些属间图使得追踪导致两个基因组差异的进化过程成为可能。在不同的组织层次上发生的大量进化事件塑造了基因组进化。在最低水平，点突变影响单个核苷酸。在更高水平上，大染色体片段经历复制、侧向转移、倒位、转位、缺失和插入。<sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_21" class="kx_ref">[21]</a></sup> 最终，整个基因组都参与杂交、多倍体化和内共生的过程，往往导致快速物种形成。基因组进化的复杂性给数学模型和算法的开发者带来了许多令人兴奋的挑战，他们求助于从精确的、启发式的、基于简约模型的问题的固定参数和逼近算法到基于概率模型的问题贝叶斯分析的马尔可夫链蒙特卡罗算法等一系列算法、统计和数学技术。 </p>
<p>这些中许多的研究是基于序列同源性的检测，以将序列分配给不同的蛋白质家族。<sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_22" class="kx_ref">[22]</a></sup> </p> 
<h3>2.6 <span>泛基因组学</span></h3> 
<p>泛基因组学是泰特林和麦地那在2005年引入的一个概念，最终扎根于生物信息学。泛基因组是特定分类群的完整基因库:虽然最初应用于一个物种的密切相关的菌株，但它也可以应用于更大的环境，如属、门等。它分为两个部分——核心基因组:所有被研究基因组共有的一组基因(这些通常是对生存至关重要的管家基因)和可有可无/灵活基因组:除了一个或一些被研究基因组之外，不存在于所有基因组中的一组基因。BPGA的生物信息学工具可以用来表征细菌物种的泛基因组。<sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_23" class="kx_ref">[23]</a></sup> </p> 
<h3>2.7 <span>疾病遗传学</span></h3> 
<p>随着下一代测序技术的出现，我们获得了足够的序列数据来绘制复杂疾病如不育、乳腺癌<sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_24" class="kx_ref">[24]</a></sup>或阿尔茨海默病<sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_25" class="kx_ref">[25]</a></sup>的基因图。<sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_26" class="kx_ref">[26]</a></sup>全基因组关联研究是查明导致这种复杂疾病的突变的有用方法。<sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_27" class="kx_ref">[27]</a></sup> 通过这些研究，数以千计的与类似疾病和特征相关的基因变异被识别出来。<sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_28" class="kx_ref">[28]</a></sup>此外，基因用于预后、诊断或治疗的可能性是最重要的应用之一。许多研究正在讨论选择要使用的基因的有前景的方法，以及使用基因预测疾病存在或预后的问题和缺陷。<sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_29" class="kx_ref">[29]</a></sup> </p> 
<h3>2.8 <span>癌症突变分析</span></h3> 
<p>在癌症中，受影响细胞的基因组以复杂甚至不可预测的方式重新排列。大规模测序工作被用来鉴定癌症中各种基因先前未知的点突变。生物信息学家继续设计专门的自动化系统来管理产生的大量序列数据，他们创造了新的算法和软件来将测序结果与不断增长的人类基因组序列和种系多态性集合进行比较。采用新的物理检测技术，例如识别染色体增加或者缺失的寡核苷酸微阵列(称为比较基因组杂交)，以及检测已知点突变的单核苷酸多态性阵列。这些检测方法同时测量整个基因组中的几十万个位点，当用于高通量测量数千个样本时，每个实验产生万亿字节的数据。大量新类型的数据再次为生物信息学家创造了新的机会。数据通常、包含相当大的可变性或噪声，因此隐马尔可夫模型和变化点分析方法被开发来推断真实拷贝数的变化。 </p>
<p>两个重要的原则可用于癌症基因组的生物信息学分析，涉及外显子组突变的识别。首先，癌症是一种体细胞基因突变累积的疾病。第二种癌症包含司机突变，需要与乘客突变区分开来。<sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_30" class="kx_ref">[30]</a></sup> </p>
<p>随着这种下一代测序技术在生物信息学领域的突破，癌症基因组学可能会发生巨大变化。这些新方法和软件允许生物信息学家快速且经济地对许多癌症基因组进行测序。这可以通过分析基因组中癌症驱动的突变，创建一个更灵活的癌症分类过程。此外，利用癌症样本的序列在疾病发展的同时跟踪患者可能在将来成为可能。<sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_31" class="kx_ref">[31]</a></sup> </p>
<p>另一种需要新生物信息学发展的数据类型是对许多肿瘤中复发的病变的分析。 </p></div></div><div id="par_14995387530084879"><h2 class="title">3 基因和蛋白质表达<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><h3>3.1 <span>基因表达分析</span></h3> 
<p>许多基因的表达可以通过用多种技术测量基因水平来确定，包括微阵列、表达的基因序列标签(EST)测序、基因表达的序列分析(SAGE)标签测序、大规模平行标记测序(MPSS)、RNA测序(RNA-Seq)，也称为“全转录组鸟枪法测序”(WTSS)，或多重原位杂交的各种应用。所有这些技术都极易产生噪声和/或在生物测量中受到偏差的影响，计算生物学的一个主要研究领域是开发统计工具，在高通量基因表达研究中从噪声中分离信号。<sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_32" class="kx_ref">[32]</a></sup> 这种研究经常被用来确定与疾病有关的基因:人们可以将来自癌性上皮细胞的微阵列数据与来自非癌性细胞的数据进行比较，以确定特定癌细胞群体中上调和下调的转录产物。 </p> 
<h3>3.2 <span>蛋白质表达分析</span></h3> 
<p>蛋白质微阵列和高通量质谱可以提供生物样品中蛋白质的信息。生物信息学在理解蛋白质微阵列和高温质谱数据方面非常重要；前一种方法面临着与靶向基因的微阵列相似的问题，后一种方法涉及将大量质谱数据与来自蛋白质序列数据库的预测质谱相匹配的问题，以及对样品进行复杂的统计分析，其中检测的蛋白质是数量多但不完整的肽。组织环境中的细胞蛋白质定位可以通过亲和蛋白质组学实现，亲和蛋白质组学显示为基于免疫组织化学和组织微阵列的空间数据。<sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_33" class="kx_ref">[33]</a></sup> </p> 
<h3>3.3 <span>调控分析</span></h3> 
<p>基因调控是一种复杂的事件协调过程，通过这种过程，一种信号，潜在的细胞外信号，如激素，最终导致一种或多种蛋白质活性的增加或减少。生物信息学技术已经被用于探索这个过程中的各个步骤。 </p>
<p>例如，基因表达可以由基因组中附近的元素调节。启动子分析包括识别和研究基因编码区周围的序列模体。这些模体影响该区域转录成基因的程度。远离启动子的增强子元件也可以通过三维环状相互作用来调节基因表达。这些相互作用可以通过染色体构象捕获实验的生物信息学分析来确定。 </p>
<p>表达数据可用于推断基因调控:人们可以比较来自生物体各种状态的微阵列数据，以形成关于每种状态所涉及基因的假设。在单细胞生物中，人们可能会比较细胞周期的各个阶段，以及各种应激条件(热休克、饥饿等)。然后可以对表达数据应用聚类算法来确定哪些基因被共同表达。例如，可以在共表达基因的上游区域(启动子)搜索过度表达的调控元件。基因聚类中应用的聚类算法的例子有k均值聚类、自组织映射(SOMs)、层次聚类和一致性聚类方法。 </p></div></div><div id="par_14995387530084880"><h2 class="title">4 细胞组织分析<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>研究人员已经开发了几种方法用于分析细胞器、基因、蛋白质和细胞内其他成分的位置。这是相关的，因为这些成分的位置影响细胞内的事件，从而帮助我们预测生物系统的行为。研究人员在许多生物数据库中捕捉亚细胞定位（基因本体论范畴的细胞隔间）。 </p> 
<h3>4.1 <span>显微镜和图像分析</span></h3> 
<p>显微照片使我们能够定位细胞器和分子。它还可以帮助我们区分正常细胞和异常细胞，例如癌症细胞。 </p> 
<h3>4.2 <span>蛋白质定位</span></h3> 
<p>蛋白质的定位有助于我们评估蛋白质的作用。例如，如果在细胞核中发现一种蛋白质，它可能参与基因调控或分裂。相反，如果在线粒体中发现蛋白质，它可能参与呼吸或其他代谢过程。因此，蛋白质定位是蛋白质功能预测的重要组成部分。现在已经有成熟的蛋白质亚细胞定位预测资源，包括蛋白质亚细胞定位数据库和预测工具。<sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_34" class="kx_ref">[34]</a></sup><sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_35" class="kx_ref">[35]</a></sup> </p> 
<h3>4.3 <span>染色质的核组织</span></h3> 
<p>来自高通量染色体构象捕获实验的数据，例如Hi-C和ChIA-PET，可以提供关于基因座空间邻近性的信息。分析这些实验可以确定染色质的三维结构和核组织。这一领域的生物信息学挑战包括将基因组分成在三维空间中组织在一起的域，如拓扑关联域(TADs)。<sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_36" class="kx_ref">[36]</a></sup> </p></div></div><div id="par_14995387530084881"><h2 class="title">5 结构生物信息学<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p></p><p></p><div class="text_img ed_imgfloat_right">
            <a class="ed_image_link lazyLoad" data-src="https://img03.sogoucdn.com/app/a/200698/sogou_science_4426" data-bigsrc="" title="点击查看大图" href="javascript:" data-observer="true"></a>
            <div class="text_img_title">像这样的三维蛋白质结构是生物信息学分析中常见的课题。</div>   
        </div> <p></p><p></p> 
<p>蛋白质结构预测是生物信息学的另一个重要应用。蛋白质的氨基酸序列，即所谓的初级结构，可以很容易地从编码它的基因序列中确定出来。在绝大多数情况下，这种主要结构独特地决定了其自然环境中的结构。(当然，也有例外，比如牛海绵状脑病——又称疯牛病——朊病毒。)了解这种结构对于理解蛋白质的功能至关重要。结构信息通常分为二级、三级和四级结构。这种预测的可行的总体解决方案仍然是一个悬而未决的问题。迄今为止，大多数努力都是针对大多数情况下有效的启发式方法。 </p>
<p>生物信息学的关键思想之一是同源性的概念。在生物信息学的基因组分支中，同源性被用来预测基因的功能:如果功能已知的基因A序列与功能未知的基因B序列同源，就可以推断出基因B可能共享基因A的功能。在生物信息学的结构分支中，同源性被用来确定蛋白质的哪些部分在结构形成和与其他蛋白质的相互作用中是重要的。在一种称为同源建模的技术中，一旦同源蛋白质的结构已知，这些信息就被用来预测蛋白质的结构。这是目前可靠预测蛋白质结构的唯一方法。 </p>
<p>这方面的一个例子是人类的血红蛋白和豆科植物的血红蛋白(豆血红蛋白)，它们是同一蛋白质超家族的远亲。两者在生物体中输送氧气的目的相同。尽管这两种蛋白质具有完全不同的氨基酸序列，但它们的蛋白质结构实际上是相同的，这反映了它们近乎相同的目的和共同的祖先。<sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_37" class="kx_ref">[37]</a></sup> </p>
<p>预测蛋白质结构的其他技术包括蛋白质穿引和从头开始的基于物理的建模。 </p>
<p>结构生物信息学的另一个方面包括将蛋白质结构用于虚拟筛选模型，如定量结构-活性关系模型和蛋白质化学计量模型。此外，蛋白质的晶体结构可用于模拟例如配体结合研究和计算机诱变研究。 </p></div></div><div id="par_14995387530084882"><h2 class="title">6 网络和系统生物学<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>网络分析寻求理解生物网络内的关系，如代谢或蛋白质-蛋白质相互作用网络。虽然生物网络可以由单一类型的分子或实体(如基因)构建，但网络生物学经常试图整合许多不同的数据类型，如蛋白质、小分子、基因表达数据等，它们都在物理上、功能上或两者都有关联。 </p>
<p>系统生物学包括使用细胞子系统(如代谢物和酶网络，包括代谢、信号转导途径和基因调控网络)的计算机模拟来分析和可视化这些细胞过程的复杂联系。人工生命或虚拟进化试图通过简单(人工)生命形式的计算机模拟来理解进化过程。 </p> 
<h3>6.1 <span>分子相互作用网络</span></h3> 
<p></p><p></p><div class="text_img ed_imgfloat_right">
            <a class="ed_image_link lazyLoad" data-src="https://img03.sogoucdn.com/app/a/200698/sogou_science_493" data-bigsrc="" title="点击查看大图" href="javascript:" data-observer="true"></a>
            <div class="text_img_title">蛋白质之间的相互作用经常用网络可视化和分析。这个网络由梅毒和其他疾病的病原体梅毒螺旋体的蛋白质相互作用组成。</div>   
        </div> <p></p><p></p>
<p>成千上万的三维蛋白质结构已经由x光晶体学和蛋白质核磁共振谱(蛋白质核磁共振)确定，结构生物信息学的一个中心问题是，仅仅基于这些三维形状而不进行蛋白质-蛋白质相互作用实验预测可能的蛋白质-蛋白质相互作用是否可行。。研究人员已经开发了多种方法来解决蛋白质-蛋白质对接问题，尽管在这一领域似乎还有许多工作要做。 </p>
<p>该领域遇到的其他相互作用包括蛋白质-配体(包括药物)和蛋白质-肽。原子围绕旋转键运动的分子动力学模拟是研究分子相互作用的计算算法(称为对接算法)背后的基本原理。 </p></div></div><div id="par_14995387530084883"><h2 class="title">7 其他<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><h3>7.1 <span>文献分析</span></h3> 
<p>出版文献数量的增长使得研究人员几乎不可能阅读每一篇论文，导致研究的子领域脱节。文献分析旨在利用计算和统计语言学来挖掘这个不断增长的文本资源库。例如: </p> 
<ul>
 <li>缩写识别–识别生物学术语的长形式和缩写</li> 
 <li>命名实体识别——识别生物术语，如基因名称</li> 
 <li>蛋白质-蛋白质相互作用-识别文本中哪些蛋白质与哪些蛋白质相互作用</li>
</ul> 
<p>研究领域来自统计学和计算语言学。 </p> 
<h3>7.2 <span>高通量图像分析</span></h3> 
<p>计算技术用于加速或完全自动化大量高信息含量生物医学图像的处理、量化和分析。现代图像分析系统通过提高精确度、客观性或速度，增强了观察者从大量或复杂图像中进行测量的能力。一个开发完全的分析系统可以完全取代观察者。尽管这些系统并非生物医学成像所独有，但生物医学成像对诊断和研究都变得越来越重要。一些例子是: </p> 
<ul>
 <li>高通量和高保真定量和亚细胞定位(高含量筛选、细胞组织病理学、生物图像信息学)</li> 
 <li>形态特征</li> 
 <li>临床图像分析和可视化</li> 
 <li>确定活体动物呼吸肺中的实时气流模式</li> 
 <li>从动脉损伤的发展和恢复中量化实时图像中的闭塞大小</li> 
 <li>从实验动物的扩展视频记录中进行行为观察</li> 
 <li>用于代谢活动测定的红外测量</li> 
 <li>推断克隆在DNA图谱中的重叠，例如苏尔斯顿分数</li>
</ul> 
<h3>7.3 <span>高通量单细胞数据分析</span></h3> 
<p>计算技术用于分析高通量、低测量的单细胞数据，例如从流式细胞仪获得的数据。这些方法通常包括寻找与特定疾病状态或实验条件相关的细胞群。 </p> 
<h3>7.4 <span>生物多样性信息学</span></h3> 
<p>生物多样性信息学处理生物多样性数据的收集和分析，如分类数据库或微生物群数据。这种分析的例子包括遗传基因、生态位建模、物种丰富度绘图、DNA条形码或物种识别工具。 </p> 
<h3>7.5 <span>本体论和数据集成</span></h3> 
<p>生物本体是受控词汇的有向无环图。它们旨在捕捉生物学概念和描述，便于用计算机进行分类和分析。当以这种方式分类时，从整体和综合分析中获得附加值是可能的。 </p>
<p>OBOFoundry致力于标准化某些本体。其中最广泛的是描述基因功能的基因本体论。也有描述表型的本体论。 </p></div></div><div id="par_14995387546862090"><h2 class="title">8 数据库<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>数据库对于生物信息学研究和应用至关重要。存在许多数据库，涵盖各种信息类型:例如，DNA和蛋白质序列、分子结构、表型和生物多样性。数据库可能包含经验数据(直接从实验中获得)、预测数据(从分析中获得)，或最常见的两者。它们可能对特定的生物体、途径或感兴趣的分子具有特异性。或者，它们可以合并从多个其他数据库编译的数据。这些数据库在格式、访问机制以及它们是否公开方面各不相同。 </p>
<p>下面列出了一些最常用的数据库。 </p> 
<ul>
 <li>用于生物序列分析:Genbank，UniProt</li> 
 <li>用于结构分析:蛋白质数据库(PDB)</li> 
 <li>用于寻找蛋白质家族和基序:InterPro，Pfam</li> 
 <li>用于下一代测序:序列读取存档</li> 
 <li>用于网络分析:代谢途径数据库(KEGG，BioCyc)，相互作用分析数据库，功能网络</li> 
 <li>用于合成遗传电路的设计:GenoCAD</li>
</ul></div></div><div id="par_14995387546862091"><h2 class="title">9 软件和工具<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>生物信息学软件工具的范围从简单的命令行工具到更复杂的图形程序和各种生物信息学公司或公共机构提供的独立网络服务。 </p> 
<h3>9.1 <span>开源生物信息学软件</span></h3> 
<p>自20世纪80年代以来，许多免费和开源软件工具已经存在并继续增长。<sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_38" class="kx_ref">[38]</a></sup> 对用于分析新兴类型生物读数的新算法的持续需求、创新电子实验的潜力和免费开放代码库的结合，为了所有研究团体创造新的机会，为生物信息学和一系列可用的开放源代码软件做出贡献，无论它们的资助是如何获得的。开源工具通常是想法的孵化器，或者商业应用中社区支持的插件。它们还可以提供业界标准和共享的对象模型来帮助研究人员应对生物信息集成的挑战。 </p>
<p>开源软件包的范围包括诸如Bioconductor、BioPerl、 Biopython、 BioJava、BioJS、BioRuby、Bioclipse、EMBOSS、.NET Bio、Orange及其生物信息学插件，Apache Taverna，UGENE和GenoCAD。<sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_38" class="kx_ref">[38]</a></sup> 为了保持这一传统并创造更多的机会，非营利的开放生物信息学基金会自2000年以来一直支持一年一度的生物信息学开放源码会议(BOSC)。<sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_39" class="kx_ref">[39]</a></sup> </p>
<p>构建公共生物信息学数据库的另一种方法是使用带有维基百科扩展的媒体维基引擎。该系统允许该领域的所有专家访问和更新数据库。<sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_40" class="kx_ref">[40]</a></sup> </p> 
<h3>9.2 <span>生物信息学中的Web服务</span></h3> 
<p>基于SOAP和REST的接口已经为各种各样的生物信息学应用程序开发出来，允许运行在世界一个地方的一台计算机上的应用程序在世界其他地方的服务器上使用算法、数据和计算资源。期主要优势是终端用户不必进行软件和数据库维护开销。 </p>
<p>EBI将基本生物信息学服务分为三类:序列搜索服务、多序列比对服务和生物序列分析服务。<sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_41" class="kx_ref">[41]</a></sup> 这些面向服务的生物信息学资源的可用性证明了基于网络的生物信息学解决方案的适用性，范围从在单一、独立或基于网络的界面下具有通用数据格式的独立工具的集合，到集成、分布式和可扩展的生物信息学工作流管理系统。 </p> 
<h3>9.3 <span>生物信息学工作流管理系统</span></h3> 
<p>生物信息学工作流管理系统是工作流管理系统的一种特殊形式，专门设计用于在生物信息学应用程序中组成和执行一系列计算或数据操作步骤或工作流。这种系统旨在 </p> 
<ul>
 <li>为单个应用科学家自己创建自己的工作流提供一个易于使用的环境，</li> 
 <li>为科学家提供交互式工具，使他们能够执行工作流并实时查看结果，</li> 
 <li>简化科学家之间共享和重用工作流的过程，</li> 
 <li>使科学家能够跟踪工作流执行结果和工作流创建步骤的来源。</li>
</ul> 
<p>提供这项服务的一些平台有: Galaxy、Kepler、Taverna、UGENE、Anduril、HIVE。 </p> 
<h3>9.4 <span>生物计算机和生物计算机对象</span></h3> 
<p>2014年，美国食品和药物管理局在国家卫生研究院贝塞斯达校区主办了一次会议，讨论生物信息学中的再现性。<sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_42" class="kx_ref">[42]</a></sup> 在接下来的三年里，一群利益相关者定期会面，讨论什么将成为生物计算机范例。<sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_43" class="kx_ref">[43]</a></sup> 这些利益攸关方包括政府、行业和学术实体的代表。会议领导人代表了美国食品和药物管理局和美国国立卫生研究院的许多分支机构和中心，包括人类变异组项目和欧洲医学信息学联合会在内的非营利实体，以及包括斯坦福大学、纽约基因组中心和乔治华盛顿大学在内的研究机构。 </p>
<p>会议决定已经决定生物计算机范例将采用数字“实验室笔记本”的形式，允许生物信息学协议的再现性、复制性、审查性和重用性。这是为了使研究小组在正常人员流动过程中保持更大的连续性，同时促进小组之间的思想交流。美国食品和药物管理局资助了这项工作，以使管道信息更加透明，便于监管人员获取。<sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_44" class="kx_ref">[44]</a></sup> </p>
<p>2016年，该小组在贝塞斯达的国立卫生研究院再次召开会议，讨论了生物计算机对象的潜力，这是生物计算机范例的一个实例。这项工作被复制为“标准试用”文件和上传到bioRxiv的预印本文件。生物计算机对象允许员工、合作者和监管者共享JSON化的记录。<sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_45" class="kx_ref">[45]</a></sup><sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_46" class="kx_ref">[46]</a></sup> </p></div></div><div id="par_14995387546862092"><h2 class="title">10 教育平台<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>旨在教授生物信息学概念和方法的软件平台包括Rosalind和通过瑞士生物信息学研究所培训门户网站提供的在线课程。加拿大生物信息学研讨会根据知识共享许可在其网站上提供培训研讨会的视频和幻灯片。4273π项目或4273pi项目<sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_47" class="kx_ref">[47]</a></sup> 也免费提供开源教育材料。该课程在低成本的 Raspberry Pi电脑上运行，并已被用于教育成人和小学生。<sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_48" class="kx_ref">[48]</a></sup><sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_49" class="kx_ref">[49]</a></sup>4273π由一个学术团体和研究人员积极开发，他们使用Raspberry Pi计算机和4273π操作系统运行研究级生物信息学。<sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_50" class="kx_ref">[50]</a></sup><sup><a href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/d10519.htm#quote_51" class="kx_ref">[51]</a></sup> </p>
<p>MOOC平台还提供生物信息学和相关学科的在线认证，包括Coursera的生物信息学专业(加州大学圣地亚哥分校)和基因组数据科学专业(约翰·霍普金斯大学)以及EdX的生命科学系列数据分析(哈佛大学)。南加州大学提供侧重于生物医学应用的翻译生物信息学硕士学位。 </p></div></div><div id="par_14995387546862093"><h2 class="title">11 会议<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>有几个关于生物信息学的大型会议。一些最著名的例子是分子生物学智能系统(ISMB)、欧洲计算生物学会议(ECCB)和计算分子生物学研究会议（RECOMB）。 </p></div></div></div></div><div id="references"><h2 class="title" id="par_references">参考文献</h2><ul class="references"><li id="quote_1"><span class="references-num">[1]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Lesk, A. M. (26 July 2013). "Bioinformatics". Encyclopaedia Britannica. Retrieved 17 April 2017..</span></p></li><li id="quote_2"><span class="references-num">[2]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Sim, A. Y. L.; Minary, P.; Levitt, M. (2012). "Modeling nucleic acids". Current Opinion in Structural Biology. 22 (3): 273–278. doi:10.1016/j.sbi.2012.03.012. PMC 4028509. PMID 22538125..</span></p></li><li id="quote_3"><span class="references-num">[3]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Dawson, W. K.; Maciejczyk, M.; Jankowska, E. J.; Bujnicki, J. M. (2016). "Coarse-grained modeling of RNA 3D structure" (PDF). Methods. 103: 138–156. doi:10.1016/j.ymeth.2016.04.026. PMID 27125734..</span></p></li><li id="quote_4"><span class="references-num">[4]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Kmiecik, S.; Gront, D.; Kolinski, M.; Wieteska, L.; Dawid, A. E.; Kolinski, A. (2016). "Coarse-Grained Protein Models and Their Applications". Chemical Reviews. 116 (14): 7898–936. doi:10.1021/acs.chemrev.6b00163. PMID 27333362..</span></p></li><li id="quote_5"><span class="references-num">[5]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Wong, K. C. (2016). Computational Biology and Bioinformatics: Gene Regulation. CRC Press/Taylor &amp; Francis Group. ISBN 9781498724975..</span></p></li><li id="quote_6"><span class="references-num">[6]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Joyce, A. P.; Zhang, C.; Bradley, P.; Havranek, J. J. (2015). "Structure-based modeling of protein: DNA specificity". Briefings in Functional Genomics. 14 (1): 39–49. doi:10.1093/bfgp/elu044. PMC 4366589. PMID 25414269..</span></p></li><li id="quote_7"><span class="references-num">[7]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Spiga, E.; Degiacomi, M. T.; Dal Peraro, M. (2014). "New Strategies for Integrative Dynamic Modeling of Macromolecular Assembly". In Karabencheva-Christova, T. Biomolecular Modelling and Simulations. Advances in Protein Chemistry and Structural Biology. 96. Academic Press. pp. 77–111. doi:10.1016/bs.apcsb.2014.06.008. ISBN 9780128000137. PMID 25443955..</span></p></li><li id="quote_8"><span class="references-num">[8]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Ciemny, Maciej; Kurcinski, Mateusz; Kamel, Karol; Kolinski, Andrzej; Alam, Nawsad; Schueler-Furman, Ora; Kmiecik, Sebastian (2018-05-04). "Protein–peptide docking: opportunities and challenges". Drug Discovery Today (in 英语). 23 (8): 1530–1537. doi:10.1016/j.drudis.2018.05.006. ISSN 1359-6446. PMID 29733895..</span></p></li><li id="quote_9"><span class="references-num">[9]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Hogeweg P (2011). Searls, David B., ed. "The Roots of Bioinformatics in Theoretical Biology". PLoS Computational Biology. 7 (3): e1002021. Bibcode:2011PLSCB...7E2021H. doi:10.1371/journal.pcbi.1002021. PMC 3068925. PMID 21483479..</span></p></li><li id="quote_10"><span class="references-num">[10]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Hesper B, Hogeweg P (1970). "Bioinformatica: een werkconcept". 1 (6). Kameleon: 28–29..</span></p></li><li id="quote_11"><span class="references-num">[11]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Hogeweg P (1978). "Simulating the growth of cellular forms". Simulation. 31 (3): 90–96. doi:10.1177/003754977803100305..</span></p></li><li id="quote_12"><span class="references-num">[12]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Moody, Glyn (2004). Digital Code of Life: How Bioinformatics is Revolutionizing Science, Medicine, and Business. ISBN 978-0-471-32788-2..</span></p></li><li id="quote_13"><span class="references-num">[13]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Dayhoff, M.O. (1966) Atlas of protein sequence and structure. National Biomedical Research Foundation, 215 pp..</span></p></li><li id="quote_14"><span class="references-num">[14]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Eck RV, Dayhoff MO (1966). "Evolution of the structure of ferredoxin based on living relics of primitive amino Acid sequences". Science. 152 (3720): 363–6. Bibcode:1966Sci...152..363E. doi:10.1126/science.152.3720.363. PMID 17775169..</span></p></li><li id="quote_15"><span class="references-num">[15]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Johnson G, Wu TT (January 2000). "Kabat Database and its applications: 30 years after the first variability plot". Nucleic Acids Res. 28 (1): 214–218. doi:10.1093/nar/28.1.214. PMC 102431. PMID 10592229..</span></p></li><li id="quote_16"><span class="references-num">[16]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Attwood TK, Gisel A, Eriksson NE, Bongcam-Rudloff E (2011). "Concepts, Historical Milestones and the Central Place of Bioinformatics in Modern Biology: A European Perspective". Bioinformatics - Trends and Methodologies. Bioinformatics – Trends and Methodologies. InTech. doi:10.5772/23535. ISBN 978-953-307-282-1. Retrieved 8 Jan 2012..</span></p></li><li id="quote_17"><span class="references-num">[17]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Sanger F, Air GM, Barrell BG, Brown NL, Coulson AR, Fiddes CA, Hutchison CA, Slocombe PM, Smith M (February 1977). "Nucleotide sequence of bacteriophage phi X174 DNA". Nature. 265 (5596): 687–95. Bibcode:1977Natur.265..687S. doi:10.1038/265687a0. PMID 870828..</span></p></li><li id="quote_18"><span class="references-num">[18]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Benson DA, Karsch-Mizrachi I, Lipman DJ, Ostell J, Wheeler DL (January 2008). "GenBank". Nucleic Acids Res. 36 (Database issue): D25–30. doi:10.1093/nar/gkm929. PMC 2238942. PMID 18073190..</span></p></li><li id="quote_19"><span class="references-num">[19]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Fleischmann RD, Adams MD, White O, Clayton RA, Kirkness EF, Kerlavage AR, Bult CJ, Tomb JF, Dougherty BA, Merrick JM (July 1995). "Whole-genome random sequencing and assembly of Haemophilus influenzae Rd". Science. 269 (5223): 496–512. Bibcode:1995Sci...269..496F. doi:10.1126/science.7542800. PMID 7542800..</span></p></li><li id="quote_20"><span class="references-num">[20]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Carvajal-Rodríguez A (2012). "Simulation of Genes and Genomes Forward in Time". Current Genomics. 11 (1): 58–61. doi:10.2174/138920210790218007. PMC 2851118. PMID 20808525..</span></p></li><li id="quote_21"><span class="references-num">[21]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Brown, TA (2002). "Mutation, Repair and Recombination". Genomes (2nd ed.). Manchester (UK): Oxford..</span></p></li><li id="quote_22"><span class="references-num">[22]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Carter, N. P.; Fiegler, H.; Piper, J. (2002). "Comparative analysis of comparative genomic hybridization microarray technologies: Report of a workshop sponsored by the Wellcome trust". Cytometry Part A. 49 (2): 43–8. doi:10.1002/cyto.10153. PMID 12357458..</span></p></li><li id="quote_23"><span class="references-num">[23]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Chaudhari Narendrakumar M., Kumar Gupta Vinod, Dutta Chitra (2016). "BPGA-an ultra-fast pan-genome analysis pipeline". Scientific Reports. 6: 24373. Bibcode:2016NatSR...624373C. doi:10.1038/srep24373. PMC 4829868. PMID 27071527.CS1 maint: Multiple names: authors list (link).</span></p></li><li id="quote_24"><span class="references-num">[24]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Véron A, Blein S, Cox DG (2014). "Genome-wide association studies and the clinic: A focus on breast cancer". Biomarkers in Medicine. 8 (2): 287–96. doi:10.2217/bmm.13.121. PMID 24521025..</span></p></li><li id="quote_25"><span class="references-num">[25]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Tosto G, Reitz C (2013). "Genome-wide association studies in Alzheimer's disease: A review". Current Neurology and Neuroscience Reports. 13 (10): 381. doi:10.1007/s11910-013-0381-0. PMC 3809844. PMID 23954969..</span></p></li><li id="quote_26"><span class="references-num">[26]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Aston KI (2014). "Genetic susceptibility to male infertility: News from genome-wide association studies". Andrology. 2 (3): 315–21. doi:10.1111/j.2047-2927.2014.00188.x. PMID 24574159..</span></p></li><li id="quote_27"><span class="references-num">[27]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Londin E, Yadav P, Surrey S, Kricka LJ, Fortina P (2013). Use of Linkage Analysis, Genome-Wide Association Studies, and Next-Generation Sequencing in the Identification of Disease-Causing Mutations. Pharmacogenomics. Methods in Molecular Biology. 1015. pp. 127–46. doi:10.1007/978-1-62703-435-7_8. ISBN 978-1-62703-434-0. PMID 23824853..</span></p></li><li id="quote_28"><span class="references-num">[28]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Hindorff, L.A.; et al. (2009). "Potential etiologic and functional implications of genome-wide association loci for human diseases and traits". Proc. Natl. Acad. Sci. USA. 106 (23): 9362–9367. Bibcode:2009PNAS..106.9362H. doi:10.1073/pnas.0903103106. PMC 2687147. PMID 19474294..</span></p></li><li id="quote_29"><span class="references-num">[29]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Hall, L.O. (2010). Finding the right genes for disease and prognosis prediction. System Science and Engineering (ICSSE),2010 International Conference. pp. 1–2. doi:10.1109/ICSSE.2010.5551766. ISBN 978-1-4244-6472-2..</span></p></li><li id="quote_30"><span class="references-num">[30]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Vazquez, Miguel; Torre, Victor de la; Valencia, Alfonso (2012-12-27). "Chapter 14: Cancer Genome Analysis". PLOS Computational Biology (in 英语). 8 (12): e1002824. Bibcode:2012PLSCB...8E2824V. doi:10.1371/journal.pcbi.1002824. ISSN 1553-7358. PMC 3531315. PMID 23300415..</span></p></li><li id="quote_31"><span class="references-num">[31]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Hye-Jung, E.C.; Jaswinder, K.; Martin, K.; Samuel, A.A; Marco, A.M (2014). "Second-Generation Sequencing for Cancer Genome Analysis". In Dellaire, Graham; Berman, Jason N.; Arceci, Robert J. Cancer Genomics. Boston (US): Academic Press. pp. 13–30. doi:10.1016/B978-0-12-396967-5.00002-5. ISBN 9780123969675..</span></p></li><li id="quote_32"><span class="references-num">[32]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Grau, J.; Ben-Gal, I.; Posch, S.; Grosse, I. (1 July 2006). "VOMBAT: prediction of transcription factor binding sites using variable order Bayesian trees" (PDF). Nucleic Acids Research. 34 (Web Server): W529–W533. doi:10.1093/nar/gkl212. PMC 1538886. PMID 16845064..</span></p></li><li id="quote_33"><span class="references-num">[33]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"The Human Protein Atlas". www.proteinatlas.org. Retrieved 2017-10-02..</span></p></li><li id="quote_34"><span class="references-num">[34]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"The human cell". www.proteinatlas.org. Retrieved 2017-10-02..</span></p></li><li id="quote_35"><span class="references-num">[35]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Thul, Peter J.; Åkesson, Lovisa; Wiking, Mikaela; Mahdessian, Diana; Geladaki, Aikaterini; Blal, Hammou Ait; Alm, Tove; Asplund, Anna; Björk, Lars (2017-05-26). "A subcellular map of the human proteome". Science. 356 (6340): eaal3321. doi:10.1126/science.aal3321. PMID 28495876..</span></p></li><li id="quote_36"><span class="references-num">[36]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Ay, Ferhat; Noble, William S. (2 September 2015). "Analysis methods for studying the 3D architecture of the genome". Genome Biology. 16 (1): 183. doi:10.1186/s13059-015-0745-7. PMC 4556012. PMID 26328929..</span></p></li><li id="quote_37"><span class="references-num">[37]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Hoy, JA; Robinson, H; Trent JT, 3rd; Kakar, S; Smagghe, BJ; Hargrove, MS (3 August 2007). "Plant hemoglobins: a molecular fossil record for the evolution of oxygen transport". Journal of Molecular Biology. 371 (1): 168–79. doi:10.1016/j.jmb.2007.05.029. PMID 17560601..</span></p></li><li id="quote_38"><span class="references-num">[38]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Open Bioinformatics Foundation: About us". Official website. Open Bioinformatics Foundation. Retrieved 10 May 2011..</span></p></li><li id="quote_39"><span class="references-num">[39]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Open Bioinformatics Foundation: BOSC". Official website. Open Bioinformatics Foundation. Retrieved 10 May 2011..</span></p></li><li id="quote_40"><span class="references-num">[40]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Brohée, Sylvain; Barriot, Roland; Moreau, Yves (2010). "Biological knowledge bases using Wikis: combining the flexibility of Wikis with the structure of databases". Bioinformatics. 26 (17): 2210–2211. doi:10.1093/bioinformatics/btq348. PMID 20591906. Retrieved 5 May 2015..</span></p></li><li id="quote_41"><span class="references-num">[41]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Nisbet, Robert (14 May 2009). "BIOINFORMATICS". Handbook of Statistical Analysis and Data Mining Applications. John Elder IV, Gary Miner. Academic Press. p. 328. ISBN 9780080912035. Retrieved 9 May 2014..</span></p></li><li id="quote_42"><span class="references-num">[42]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Commissioner, Office of the. "Advancing Regulatory Science - Sept. 24-25, 2014 Public Workshop: Next Generation Sequencing Standards". www.fda.gov (in 英语). Retrieved 2017-11-30..</span></p></li><li id="quote_43"><span class="references-num">[43]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Simonyan, Vahan; Goecks, Jeremy; Mazumder, Raja (2017). "Biocompute Objects—A Step towards Evaluation and Validation of Biomedical Scientific Computations". PDA Journal of Pharmaceutical Science and Technology. 71 (2): 136–146. doi:10.5731/pdajpst.2016.006734. ISSN 1079-7440. PMC 5510742. PMID 27974626..</span></p></li><li id="quote_44"><span class="references-num">[44]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Commissioner, Office of the. "Advancing Regulatory Science - Community-based development of HTS standards for validating data and computation and encouraging interoperability". www.fda.gov (in 英语). Retrieved 2017-11-30..</span></p></li><li id="quote_45"><span class="references-num">[45]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Alterovitz, Gil; Dean, Dennis A.; Goble, Carole; Crusoe, Michael R.; Soiland-Reyes, Stian; Bell, Amanda; Hayes, Anais; King, Charles Hadley S.; Johanson, Elaine (2017-10-04). "Enabling Precision Medicine via standard communication of NGS provenance, analysis, and results". bioRxiv 191783..</span></p></li><li id="quote_46"><span class="references-num">[46]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">BioCompute Object (BCO) project is a collaborative and community-driven framework to standardize HTS computational data. 1. BCO Specification Document: user manual for understanding and creating B., biocompute-objects, 2017-09-03, retrieved 2017-11-30.</span></p></li><li id="quote_47"><span class="references-num">[47]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Barker, D; Ferrier, D.E.K.; Holland, P.W; Mitchell, J.B.O; Plaisier, H; Ritchie, M.G; Smart, S.D. (2013). "4273π : bioinformatics education on low cost ARM hardware". BMC Bioinformatics. 14: 243. doi:10.1186/1471-2105-14-243. PMC 3751261. PMID 23937194..</span></p></li><li id="quote_48"><span class="references-num">[48]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Barker, D; Alderson, R.G; McDonagh, J.L; Plaisier, H; Comrie, M.M; Duncan, L; Muirhead, G.T.P; Sweeny, S.D. (2015). "University-level practical activities in bioinformatics benefit voluntary groups of pupils in the last 2 years of school". International Journal of STEM Education. 2 (17). doi:10.1186/s40594-015-0030-z..</span></p></li><li id="quote_49"><span class="references-num">[49]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">McDonagh, J.L; Barker, D; Alderson, R.G. (2016). "Bringing computational science to the public". SpringerPlus. 5 (259): 259. doi:10.1186/s40064-016-1856-7. PMC 4775721. PMID 27006868..</span></p></li><li id="quote_50"><span class="references-num">[50]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Robson, J.F.; Barker, D (2015). "Comparison of the protein-coding gene content of Chlamydia trachomatis and Protochlamydia amoebophila using a Raspberry Pi computer". BMC Research Notes. 8 (561): 561. doi:10.1186/s13104-015-1476-2. PMC 4604092. PMID 26462790..</span></p></li><li id="quote_51"><span class="references-num">[51]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Wregglesworth, K.M; Barker, D (2015). "A comparison of the protein-coding genomes of two green sulphur bacteria, Chlorobium tepidum TLS and Pelodictyon phaeoclathratiforme BU-1". BMC Research Notes. 8 (565): 565. doi:10.1186/s13104-015-1535-8. PMC 4606965. PMID 26467441..</span></p></li></ul></div><div class="read-num">阅读 <!-- -->5105</div></div><div class="right-side" id="rightSide"><div class="side" id="lemma-side"><div class="side-title">版本记录</div><ul class="side-lst"><li><p class="side-lst-txt">暂无</p></li></ul><div class="user-card userCard"></div></div><div class="side"><div class="side-event"></div></div></div></div><div class="footer-box"><div id="footer"><div class="footer-logo-wrap"><div class="footer-logo"></div><div class="footer-logo-text">知识·传播·科普</div></div><div class="footer-info">本网站内容采用<a target="_blank" href="https://web.archive.org/web/20221025085755/https://creativecommons.org/licenses/by-sa/3.0/deed.zh?tdsourcetag=s_pctim_aiomsg">CC-BY-SA 3.0</a>授权</div><div class="footer-btn-wrap"><a target="_blank" href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/help/#user_protocol">用户协议</a><a target="_blank" href="https://web.archive.org/web/20221025085755/http://www.sogou.com/docs/terms.htm?v=1">免责声明</a><a target="_blank" href="https://web.archive.org/web/20221025085755/http://corp.sogou.com/private.html">隐私政策</a><a target="_blank" href="https://web.archive.org/web/20221025085755/https://baike.sogou.com/kexue/intro.htm">关于我们</a></div></div></div><script>window.lemmaInfo ={"lemmaId":"10519","versionId":"14995387513307653","title":"生物信息学","subtitle":"","abstracts":{"paragraphId":"14995387513307666","title":"简介","versionId":"14995387513307654","lemmaId":10519,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":53099137,"name":"孟凡康","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233946,"comment":null,"dependVersionId":0,"contentType":2,"content":"<p></p> \n<p><b>生物信息学</b>是一个跨学科的领域，目的是开发理解生物数据的方法和软件工具。生物信息学作为一个跨学科的科学领域，结合了生物学、计算机科学、信息工程、数学和统计学的相关知识用于分析和解释生物数据。通过数学和统计技术，生物信息学已经被用于对生物数据库进行计算机分析。 </p>\n<p>生物信息学既是生物研究主体的总称，该研究主体使用计算机编程作为其方法论的一部分；也是对重复使用的特定分析“管道”的引用，特别是在基因组学领域。生物信息学的常见用途包括候选基因的鉴定和单核苷酸多态性(SNPs)。通常，这种鉴定的目的是为了更好地理解疾病的遗传基础、独特的适应性、理想的特性(特别是农业物种)或种群间的差异。以一种不太正式的方式，生物信息学也试图理解核酸和蛋白质序列中的组织原则，称为蛋白质组学。<sup><a href=\"#quote_1\" class=\"kx_ref\">[1]</a></sup> </p>","pics":[{"originalUrl":"https://web.archive.org/web/20221025085755/https://img01.sogoucdn.com/app/a/200698/sogou_science_489?w=220&h=242&titlename=%E4%BA%BA%E7%B1%BBX%E6%9F%93%E8%89%B2%E4%BD%93%E5%9B%BE%E8%B0%B1(%E6%9D%A5%E8%87%AA%E5%9B%BD%E5%AE%B6%E7%94%9F%E7%89%A9%E6%8A%80%E6%9C%AF%E4%BF%A1%E6%81%AF%E4%B8%AD%E5%BF%83%E7%BD%91%E7%AB%99)%E3%80%82","url":"https://web.archive.org/web/20221025085755/https://img01.sogoucdn.com/app/a/200698/sogou_science_489","rw":220,"rh":242,"title":"人类X染色体图谱(来自国家生物技术信息中心网站)。","alt":null,"width":0,"height":0}],"card":null,"references":[],"versionCount":0},"card":{"paragraphId":"0","title":null,"versionId":"0","lemmaId":0,"createType":0,"creator":null,"createTime":0,"versionEditor":null,"editTime":0,"comment":null,"dependVersionId":0,"contentType":0,"content":null,"pics":null,"card":null,"references":null,"versionCount":0},"categories":[{"id":4,"name":"生物学","parents":[]}],"creator":{"uid":53099137,"name":"孟凡康","pic":"https://web.archive.org/web/20221025085755/https://cache.soso.com/qlogo/g?b=oidb&k=gfMphd6l1FFGwj0oC386Fg&s=100&t=1561211753","introduction":null,"educations":[{"schoolName":"中国科学院大学","major":"","degree":"博士","universityId":111,"universityLogo":"https://web.archive.org/web/20221025085755/https://img01.sogoucdn.com/app/a/200943/d7a2d86e-6011-11e9-a2ce-fc4dd4f70029","majorLevel1":"理学","majorLevel2":"生物学","majorLevel3":"合成生物学","majorLevel1Id":1,"majorLevel2Id":189,"majorLevel3Id":204,"state":null,"lab":null,"researchField":null}],"jobs":null,"works":null,"educationBrief":"中国科学院大学","jobBrief":"","role":0,"roleName":null,"title":"中国科学院大学 · 生物学博士","professionalTitle":null,"phoneNo":null,"editable":true,"partnerId":99,"partnerIdCreateTime":1594286514,"partnerIdPoped":false},"createTime":1568622667,"editor":{"uid":53099137,"name":"孟凡康","pic":"https://web.archive.org/web/20221025085755/https://cache.soso.com/qlogo/g?b=oidb&k=gfMphd6l1FFGwj0oC386Fg&s=100&t=1561211753","introduction":null,"educations":[{"schoolName":"中国科学院大学","major":"","degree":"博士","universityId":111,"universityLogo":"https://web.archive.org/web/20221025085755/https://img01.sogoucdn.com/app/a/200943/d7a2d86e-6011-11e9-a2ce-fc4dd4f70029","majorLevel1":"理学","majorLevel2":"生物学","majorLevel3":"合成生物学","majorLevel1Id":1,"majorLevel2Id":189,"majorLevel3Id":204,"state":null,"lab":null,"researchField":null}],"jobs":null,"works":null,"educationBrief":"中国科学院大学","jobBrief":"","role":0,"roleName":null,"title":"中国科学院大学 · 生物学博士","professionalTitle":null,"phoneNo":null,"editable":true,"partnerId":99,"partnerIdCreateTime":1594286514,"partnerIdPoped":false},"editTime":1576233946,"state":1,"versionCount":1,"upNum":4,"downNum":0,"pics":[{"originalUrl":"https://web.archive.org/web/20221025085755/https://img01.sogoucdn.com/app/a/200698/sogou_science_489?w=220&h=242&titlename=%E4%BA%BA%E7%B1%BBX%E6%9F%93%E8%89%B2%E4%BD%93%E5%9B%BE%E8%B0%B1(%E6%9D%A5%E8%87%AA%E5%9B%BD%E5%AE%B6%E7%94%9F%E7%89%A9%E6%8A%80%E6%9C%AF%E4%BF%A1%E6%81%AF%E4%B8%AD%E5%BF%83%E7%BD%91%E7%AB%99)%E3%80%82","url":"https://web.archive.org/web/20221025085755/https://img01.sogoucdn.com/app/a/200698/sogou_science_489","rw":220,"rh":242,"title":"人类X染色体图谱(来自国家生物技术信息中心网站)。","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221025085755/https://img04.sogoucdn.com/app/a/200698/sogou_science_4425?w=300&h=86&titlename=%E9%81%97%E4%BC%A0%E7%89%A9%E8%B4%A8%E5%BA%8F%E5%88%97%E5%9C%A8%E7%94%9F%E7%89%A9%E4%BF%A1%E6%81%AF%E5%AD%A6%E4%B8%AD%E7%BB%8F%E5%B8%B8%E4%BD%BF%E7%94%A8%EF%BC%8C%E4%BD%BF%E7%94%A8%E8%AE%A1%E7%AE%97%E6%9C%BA%E6%AF%94%E6%89%8B%E5%B7%A5%E6%9B%B4%E5%AE%B9%E6%98%93%E7%AE%A1%E7%90%86%E3%80%82","url":"https://web.archive.org/web/20221025085755/https://img04.sogoucdn.com/app/a/200698/sogou_science_4425","rw":300,"rh":86,"title":"遗传物质序列在生物信息学中经常使用，使用计算机比手工更容易管理。","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221025085755/https://img01.sogoucdn.com/app/a/200698/sogou_science_498?w=500&h=116&titlename=%E4%B8%8D%E5%90%8C%E5%9F%BA%E5%9B%A0%E6%88%96%E8%9B%8B%E7%99%BD%E8%B4%A8%E7%9A%84%E5%BA%8F%E5%88%97%E5%8F%AF%E4%BB%A5%E5%B9%B6%E6%8E%92%E6%8E%92%E5%88%97%E4%BB%A5%E6%B5%8B%E9%87%8F%E5%AE%83%E4%BB%AC%E7%9A%84%E7%9B%B8%E4%BC%BC%E6%80%A7%E3%80%82%E8%BF%99%E7%A7%8D%E6%AF%94%E5%AF%B9%E6%AF%94%E8%BE%83%E4%BA%86%E8%9B%8B%E7%99%BD%E8%B4%A8%E5%BA%8F%E5%88%97%E5%92%8C%E5%90%AB%E6%9C%89WPP%E7%BB%93%E6%9E%84%E5%9F%9F%E7%9A%84%E5%9F%BA%E5%9B%A0%E7%BB%84%E5%BA%8F%E5%88%97%E3%80%82","url":"https://web.archive.org/web/20221025085755/https://img01.sogoucdn.com/app/a/200698/sogou_science_498","rw":500,"rh":116,"title":"不同基因或蛋白质的序列可以并排排列以测量它们的相似性。这种比对比较了蛋白质序列和含有WPP结构域的基因组序列。","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221025085755/https://img03.sogoucdn.com/app/a/200698/sogou_science_4426?w=300&h=357&titlename=%E5%83%8F%E8%BF%99%E6%A0%B7%E7%9A%84%E4%B8%89%E7%BB%B4%E8%9B%8B%E7%99%BD%E8%B4%A8%E7%BB%93%E6%9E%84%E6%98%AF%E7%94%9F%E7%89%A9%E4%BF%A1%E6%81%AF%E5%AD%A6%E5%88%86%E6%9E%90%E4%B8%AD%E5%B8%B8%E8%A7%81%E7%9A%84%E8%AF%BE%E9%A2%98%E3%80%82","url":"https://web.archive.org/web/20221025085755/https://img03.sogoucdn.com/app/a/200698/sogou_science_4426","rw":300,"rh":357,"title":"像这样的三维蛋白质结构是生物信息学分析中常见的课题。","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221025085755/https://img03.sogoucdn.com/app/a/200698/sogou_science_493?w=200&h=188&titlename=%E8%9B%8B%E7%99%BD%E8%B4%A8%E4%B9%8B%E9%97%B4%E7%9A%84%E7%9B%B8%E4%BA%92%E4%BD%9C%E7%94%A8%E7%BB%8F%E5%B8%B8%E7%94%A8%E7%BD%91%E7%BB%9C%E5%8F%AF%E8%A7%86%E5%8C%96%E5%92%8C%E5%88%86%E6%9E%90%E3%80%82%E8%BF%99%E4%B8%AA%E7%BD%91%E7%BB%9C%E7%94%B1%E6%A2%85%E6%AF%92%E5%92%8C%E5%85%B6%E4%BB%96%E7%96%BE%E7%97%85%E7%9A%84%E7%97%85%E5%8E%9F%E4%BD%93%E6%A2%85%E6%AF%92%E8%9E%BA%E6%97%8B%E4%BD%93%E7%9A%84%E8%9B%8B%E7%99%BD%E8%B4%A8%E7%9B%B8%E4%BA%92%E4%BD%9C%E7%94%A8%E7%BB%84%E6%88%90%E3%80%82","url":"https://web.archive.org/web/20221025085755/https://img03.sogoucdn.com/app/a/200698/sogou_science_493","rw":200,"rh":188,"title":"蛋白质之间的相互作用经常用网络可视化和分析。这个网络由梅毒和其他疾病的病原体梅毒螺旋体的蛋白质相互作用组成。","alt":null,"width":0,"height":0}],"catalogs":[{"level":1,"title":"介绍","paragraphId":"14995387513307667","subCatalogs":[{"level":2,"title":"历史","paragraphId":"14995387513307667","subCatalogs":null},{"level":2,"title":"目标","paragraphId":"14995387513307667","subCatalogs":null},{"level":2,"title":"与其他领域的关系","paragraphId":"14995387513307667","subCatalogs":null}]},{"level":1,"title":"序列分析","paragraphId":"14995387530084878","subCatalogs":[{"level":2,"title":"DNA测序","paragraphId":"14995387530084878","subCatalogs":null},{"level":2,"title":"序列组装","paragraphId":"14995387530084878","subCatalogs":null},{"level":2,"title":"基因组注释","paragraphId":"14995387530084878","subCatalogs":null},{"level":2,"title":"计算进化生物学","paragraphId":"14995387530084878","subCatalogs":null},{"level":2,"title":"比较基因组学","paragraphId":"14995387530084878","subCatalogs":null},{"level":2,"title":"泛基因组学","paragraphId":"14995387530084878","subCatalogs":null},{"level":2,"title":"疾病遗传学","paragraphId":"14995387530084878","subCatalogs":null},{"level":2,"title":"癌症突变分析","paragraphId":"14995387530084878","subCatalogs":null}]},{"level":1,"title":"基因和蛋白质表达","paragraphId":"14995387530084879","subCatalogs":[{"level":2,"title":"基因表达分析","paragraphId":"14995387530084879","subCatalogs":null},{"level":2,"title":"蛋白质表达分析","paragraphId":"14995387530084879","subCatalogs":null},{"level":2,"title":"调控分析","paragraphId":"14995387530084879","subCatalogs":null}]},{"level":1,"title":"细胞组织分析","paragraphId":"14995387530084880","subCatalogs":[{"level":2,"title":"显微镜和图像分析","paragraphId":"14995387530084880","subCatalogs":null},{"level":2,"title":"蛋白质定位","paragraphId":"14995387530084880","subCatalogs":null},{"level":2,"title":"染色质的核组织","paragraphId":"14995387530084880","subCatalogs":null}]},{"level":1,"title":"结构生物信息学","paragraphId":"14995387530084881","subCatalogs":null},{"level":1,"title":"网络和系统生物学","paragraphId":"14995387530084882","subCatalogs":[{"level":2,"title":"分子相互作用网络","paragraphId":"14995387530084882","subCatalogs":null}]},{"level":1,"title":"其他","paragraphId":"14995387530084883","subCatalogs":[{"level":2,"title":"文献分析","paragraphId":"14995387530084883","subCatalogs":null},{"level":2,"title":"高通量图像分析","paragraphId":"14995387530084883","subCatalogs":null},{"level":2,"title":"高通量单细胞数据分析","paragraphId":"14995387530084883","subCatalogs":null},{"level":2,"title":"生物多样性信息学","paragraphId":"14995387530084883","subCatalogs":null},{"level":2,"title":"本体论和数据集成","paragraphId":"14995387530084883","subCatalogs":null}]},{"level":1,"title":"数据库","paragraphId":"14995387546862090","subCatalogs":null},{"level":1,"title":"软件和工具","paragraphId":"14995387546862091","subCatalogs":[{"level":2,"title":"开源生物信息学软件","paragraphId":"14995387546862091","subCatalogs":null},{"level":2,"title":"生物信息学中的Web服务","paragraphId":"14995387546862091","subCatalogs":null},{"level":2,"title":"生物信息学工作流管理系统","paragraphId":"14995387546862091","subCatalogs":null},{"level":2,"title":"生物计算机和生物计算机对象","paragraphId":"14995387546862091","subCatalogs":null}]},{"level":1,"title":"教育平台","paragraphId":"14995387546862092","subCatalogs":null},{"level":1,"title":"会议","paragraphId":"14995387546862093","subCatalogs":null},{"level":1,"title":"参考文献","paragraphId":"-1","subCatalogs":null}],"paragraphs":[{"paragraphId":"14995387513307667","title":"介绍","versionId":"14995387513307655","lemmaId":10519,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":53099137,"name":"孟凡康","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233946,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>生物信息学已经成为生物学许多领域的重要组成部分。在实验分子生物学中，图像和信号处理等生物信息学技术允许从大量原始数据中提取有用的结果。在遗传学领域，它有助于对基因组及其观察到的突变进行测序和注释。它在生物文献的文本挖掘以及生物和基因本体的发展中起着组织和查询生物数据的作用。它还在基因和蛋白质表达和调节的分析中发挥作用。生物信息学工具有助于比较遗传和基因组数据，更概括的说，有助于理解分子生物学的进化方面。在更综合的层面上，它有助于分析和编目作为系统生物学重要组成部分的生物路径和网络。在结构生物学中，它有助于对DNA、RNA、<sup><a href=\"#quote_2\" class=\"kx_ref\">[2]</a></sup><sup><a href=\"#quote_3\" class=\"kx_ref\">[3]</a></sup> 蛋白质<sup><a href=\"#quote_4\" class=\"kx_ref\">[4]</a></sup> 以及生物分子间的相互作用进行模拟和建模。<sup><a href=\"#quote_5\" class=\"kx_ref\">[5]</a></sup><sup><a href=\"#quote_6\" class=\"kx_ref\">[6]</a></sup><sup><a href=\"#quote_7\" class=\"kx_ref\">[7]</a></sup><sup><a href=\"#quote_8\" class=\"kx_ref\">[8]</a></sup> </p> \n<h3>历史</h3> \n<p>历史上，生物信息学这个术语和它今天的意义并不一样。波利恩·霍格威和本·海茨帕在1970年创造了这个词，用来指对生物系统中信息过程的研究。<sup><a href=\"#quote_9\" class=\"kx_ref\">[9]</a></sup><sup><a href=\"#quote_10\" class=\"kx_ref\">[10]</a></sup><sup><a href=\"#quote_11\" class=\"kx_ref\">[11]</a></sup> 这一定义将生物信息学定位为一个平行于生物化学(研究生物系统中的化学过程)的领域。<sup><a href=\"#quote_9\" class=\"kx_ref\">[9]</a></sup> </p> \n<p><strong>序列</strong></p> \n<p></p><p><img alt=\"\" class=\"fileimage kx_img ed_imgfloat_right\" img_height=\"86\" img_width=\"300\" titlename=\"遗传物质序列在生物信息学中经常使用，使用计算机比手工更容易管理。\" data-src=\"https://img04.sogoucdn.com/app/a/200698/sogou_science_4425\"> </p><p></p> \n<p>20世纪50年代初，弗雷德里克·桑格确定胰岛素序列后，蛋白质序列的获取成为可能，计算机成为分子生物学中的关键。手动比较多个序列被证明是不切实际的。这一领域的先驱是玛格丽特·奥克利·戴霍夫。<sup><a href=\"#quote_12\" class=\"kx_ref\">[12]</a></sup> 她编译了第一批蛋白质序列数据库，最初作为书籍出版，<sup><a href=\"#quote_13\" class=\"kx_ref\">[13]</a></sup> 并开创了序列比对和分子进化的方法。<sup><a href=\"#quote_14\" class=\"kx_ref\">[14]</a></sup> 生物信息学的另一个早期贡献者是艾文·卡巴特，他在1970年开创了生物序列分析方法，他在1980年至1991年间与泰特·吴一起发布了大量的抗体序列。<sup><a href=\"#quote_15\" class=\"kx_ref\">[15]</a></sup> </p> \n<h3>目标</h3> \n<p>为了研究正常细胞活动在不同疾病状态下是如何改变的，生物数据必须结合起来，形成这些活动的综合图像。因此，生物信息学领域已经发展到现在最紧迫的任务是分析和解释各种类型的数据。这包括核苷酸和氨基酸序列、蛋白质结构域和蛋白质结构。<sup><a href=\"#quote_16\" class=\"kx_ref\">[16]</a></sup> 分析和解释数据的实际过程被称为计算生物学。生物信息学和计算生物学中的重要子学科包括: </p> \n<ul>\n <li>开发和执行计算机程序，使各种类型的信息能够得到有效的访问、使用和管理</li> \n <li>开发评估大数据集之间关系的新算法(数学公式)和统计方法。例如，定位序列中的基因，预测蛋白质结构和/或功能，并将蛋白质序列聚类到相关序列的家族中等的各种方法。</li>\n</ul> \n<p>生物信息学的主要目标是增加对生物过程的理解。然而，它与其他方法的不同之处在于它专注于开发和应用计算密集型技术来实现这一目标。例子包括:模式识别、数据挖掘、机器学习算法和可视化。该领域的主要研究工作包括序列比对、基因挖掘、基因组组装、药物设计、药物发现、蛋白质结构比对、蛋白质结构预测、基因表达预测和蛋白质-蛋白质相互作用、全基因组关联研究、进化和细胞分裂/有丝分裂建模。 </p>\n<p>生物信息学现在需要创建和发展数据库、算法、计算和统计技术以及理论，用于解决生物数据管理和分析中出现的形式和实际问题。 </p>\n<p>在过去的几十年里，基因组和其他分子研究技术的快速发展以及信息技术的发展结合起来，产生了大量与分子生物学相关的信息。生物信息学是这些数学和计算方法的名称，用来收集对生物过程的理解。 </p>\n<p>生物信息学中常见的形式包括绘制和分析DNA和蛋白质序列，对DNA和蛋白质序列进行比较，以及创建和查看蛋白质结构的三维模型。 </p> \n<h3>与其他领域的关系</h3> \n<p>生物信息学是一个类似于但不同于生物计算的科学领域，虽然它通常被认为是计算生物学的同义词。生物计算使用生物工程和生物学来构建生物计算机，而生物信息学使用计算来更好地理解生物学。生物信息学和计算生物学涉及生物数据的分析，特别是DNA、RNA和蛋白质序列。生物信息学领域从20世纪90年代中期开始经历了爆炸性的增长，这主要是由人类基因组计划和DNA测序技术的快速进步推动的。 </p>\n<p>分析生物数据产生有意义的信息包括编写和运行软件程序，这些程序使用来自图论、人工智能、软计算、数据挖掘、图像处理和计算机模拟的算法。算法又依赖于理论基础，如离散数学、控制理论、系统理论、信息论和统计学。 </p>","pics":[{"originalUrl":"https://web.archive.org/web/20221025085755/https://img04.sogoucdn.com/app/a/200698/sogou_science_4425?w=300&h=86&titlename=%E9%81%97%E4%BC%A0%E7%89%A9%E8%B4%A8%E5%BA%8F%E5%88%97%E5%9C%A8%E7%94%9F%E7%89%A9%E4%BF%A1%E6%81%AF%E5%AD%A6%E4%B8%AD%E7%BB%8F%E5%B8%B8%E4%BD%BF%E7%94%A8%EF%BC%8C%E4%BD%BF%E7%94%A8%E8%AE%A1%E7%AE%97%E6%9C%BA%E6%AF%94%E6%89%8B%E5%B7%A5%E6%9B%B4%E5%AE%B9%E6%98%93%E7%AE%A1%E7%90%86%E3%80%82","url":"https://web.archive.org/web/20221025085755/https://img04.sogoucdn.com/app/a/200698/sogou_science_4425","rw":300,"rh":86,"title":"遗传物质序列在生物信息学中经常使用，使用计算机比手工更容易管理。","alt":null,"width":0,"height":0}],"card":null,"references":[],"versionCount":0},{"paragraphId":"14995387530084878","title":"序列分析","versionId":"14995387513307656","lemmaId":10519,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":53099137,"name":"孟凡康","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233946,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p></p><p><img alt=\"\" class=\"fileimage kx_img ed_imgfloat_right\" img_height=\"116\" img_width=\"500\" titlename=\"不同基因或蛋白质的序列可以并排排列以测量它们的相似性。这种比对比较了蛋白质序列和含有WPP结构域的基因组序列。\" data-src=\"https://img01.sogoucdn.com/app/a/200698/sogou_science_498\"> </p><p></p> \n<p>自从噬菌体φ-X174在1977年被测序以来，<sup><a href=\"#quote_17\" class=\"kx_ref\">[17]</a></sup> 分析序列信息可以确定编码蛋白质、RNA基因、调节序列、结构基序和重复序列的基因。一个物种内或不同物种之间的基因比较可以显示蛋白质功能之间的相似性，或物种之间的关系(使用分子系统学构建系统进化树)。随着数据量的增长，很久以前手工分析DNA序列变得不切实际。今天，计算机程序如BLAST每天都被用来从超过260,000个生物体中搜索序列，<sup><a href=\"#quote_18\" class=\"kx_ref\">[18]</a></sup>包含超过1900亿个核苷酸。 这些程序可以补偿基因序列中的突变(交换、删除或插入碱基)，以识别相关但不相同的序列。这种序列比对的变体用于测序过程本身。对于序列片段分类的特殊任务，现代基于k-mer的软件如Kraken实现了比对方法无法达到的通量。 </p> \n<h3>DNA测序</h3> \n<p>在分析序列之前，必须先获得它们。DNA测序仍然是一个不小的问题，因为原始数据可能有噪声或受到微弱信号的影响。多种算法已经被开发出来用于分析多种测序方法的结果。 </p> \n<h3>序列组装</h3> \n<p>大多数DNA测序技术产生的短序列片段需要组装才能获得完整的基因或基因组序列。所谓的鸟枪测序技术(例如，由基因组研究所(TIGR)用来对第一个细菌基因组流感嗜血杆菌进行测序)<sup><a href=\"#quote_19\" class=\"kx_ref\">[19]</a></sup>产生了成千上万个小的DNA片段的序列(根据测序技术，长度从35到900个核苷酸不等)。这些片段的末端重叠，当通过基因组组装程序正确排列时，可以用来重建完整的基因组。鸟枪测序能快速产生序列数据，但是对于更大的基因组来说，组装片段的任务可能相当复杂。对于像人类基因组一样大的基因组来说，在大内存、多处理器计算机上组装片段可能需要很多天的CPU时间，而最终的组装通常包含许多必须在以后填补的空白。鸟枪法测序是当今几乎所有测序基因组的首选方法。基因组组装算法是生物信息学研究的一个关键领域。 </p> \n<h3>基因组注释</h3> \n<p>在基因组学的背景下，注释是在基因序列中标记基因和其他生物特征的过程。这个过程需要自动化，因为大多数基因组太大，不能用手工注释，更不用说注释尽可能多的基因组了，因为测序的速度已经不再是瓶颈。注释之所以成为可能，是因为基因有可识别的起始和终止区域，尽管在这些区域中发现的确切序列可能因基因而异。 </p>\n<p>基因组研究所的团队于1995年 <sup><a href=\"#quote_19\" class=\"kx_ref\">[19]</a></sup> 首次描述了第一个综合基因组注释系统。该团队对一种自然生物——流感嗜血杆菌<sup><a href=\"#quote_19\" class=\"kx_ref\">[19]</a></sup>——的基因组进行了第一次完整的测序和分析。欧文·怀特(Owen White)设计并构建了一个软件系统用于识别编码所有蛋白质的基因，转运RNA、核糖体RNA(以及其他位点)，并进行初始功能分配。目前大多数基因组注释系统的工作原理相似，但可用于基因组DNA分析的程序，如经过训练并用于在流感嗜血杆菌中发现蛋白质编码基因的GeneMark程序，正在不断变化和改进。 </p>\n<p>继人类基因组计划在2003年结束后留下来需要实现的目标之后，美国国家人类基因组研究所开发的一个新项目出现了。所谓的编码项目(ENCODE project)是人类基因组功能元素的协作数据收集，它使用下一代DNA测序技术和基因组平铺阵列，这些技术能够以大幅降低的单位成本自动生成大量数据，同时保证相同水平的精确度(基本调用误差)和保真度(装配误差)。 </p> \n<h3>计算进化生物学</h3> \n<p>进化生物学研究物种的起源和世代，以及它们随时间的变化。生物信息学辅助进化生物学家通过使研究人员能够: </p> \n<ul>\n <li>通过测量大量生物的DNA的变化来追踪它们的进化，而不是仅仅通过物理分类或生理观察，</li> \n <li>比较整个基因组，这允许研究更复杂的进化事件，例如基因复制、水平基因转移和预测细菌物种形成中的重要因素，</li> \n <li>建立复杂的计算群体遗传学模型来预测系统随时间变化的结果<sup><a href=\"#quote_20\" class=\"kx_ref\">[20]</a></sup></li> \n <li>跟踪和共享越来越多的物种和有机体的信息</li>\n</ul> \n<p>进一步的工作正在努力重建比现在更加复杂的生命树。 </p>\n<p>计算机科学中使用遗传算法的研究领域有时与计算进化生物学混淆，但这两个领域不一定相关。 </p> \n<h3>比较基因组学</h3> \n<p>比较基因组分析的核心是建立不同生物体中基因(正交性分析)或其他基因组特征之间的对应关系。正是这些属间图使得追踪导致两个基因组差异的进化过程成为可能。在不同的组织层次上发生的大量进化事件塑造了基因组进化。在最低水平，点突变影响单个核苷酸。在更高水平上，大染色体片段经历复制、侧向转移、倒位、转位、缺失和插入。<sup><a href=\"#quote_21\" class=\"kx_ref\">[21]</a></sup> 最终，整个基因组都参与杂交、多倍体化和内共生的过程，往往导致快速物种形成。基因组进化的复杂性给数学模型和算法的开发者带来了许多令人兴奋的挑战，他们求助于从精确的、启发式的、基于简约模型的问题的固定参数和逼近算法到基于概率模型的问题贝叶斯分析的马尔可夫链蒙特卡罗算法等一系列算法、统计和数学技术。 </p>\n<p>这些中许多的研究是基于序列同源性的检测，以将序列分配给不同的蛋白质家族。<sup><a href=\"#quote_22\" class=\"kx_ref\">[22]</a></sup> </p> \n<h3>泛基因组学</h3> \n<p>泛基因组学是泰特林和麦地那在2005年引入的一个概念，最终扎根于生物信息学。泛基因组是特定分类群的完整基因库:虽然最初应用于一个物种的密切相关的菌株，但它也可以应用于更大的环境，如属、门等。它分为两个部分——核心基因组:所有被研究基因组共有的一组基因(这些通常是对生存至关重要的管家基因)和可有可无/灵活基因组:除了一个或一些被研究基因组之外，不存在于所有基因组中的一组基因。BPGA的生物信息学工具可以用来表征细菌物种的泛基因组。<sup><a href=\"#quote_23\" class=\"kx_ref\">[23]</a></sup> </p> \n<h3>疾病遗传学</h3> \n<p>随着下一代测序技术的出现，我们获得了足够的序列数据来绘制复杂疾病如不育、乳腺癌<sup><a href=\"#quote_24\" class=\"kx_ref\">[24]</a></sup>或阿尔茨海默病<sup><a href=\"#quote_25\" class=\"kx_ref\">[25]</a></sup>的基因图。<sup><a href=\"#quote_26\" class=\"kx_ref\">[26]</a></sup>全基因组关联研究是查明导致这种复杂疾病的突变的有用方法。<sup><a href=\"#quote_27\" class=\"kx_ref\">[27]</a></sup> 通过这些研究，数以千计的与类似疾病和特征相关的基因变异被识别出来。<sup><a href=\"#quote_28\" class=\"kx_ref\">[28]</a></sup>此外，基因用于预后、诊断或治疗的可能性是最重要的应用之一。许多研究正在讨论选择要使用的基因的有前景的方法，以及使用基因预测疾病存在或预后的问题和缺陷。<sup><a href=\"#quote_29\" class=\"kx_ref\">[29]</a></sup> </p> \n<h3>癌症突变分析</h3> \n<p>在癌症中，受影响细胞的基因组以复杂甚至不可预测的方式重新排列。大规模测序工作被用来鉴定癌症中各种基因先前未知的点突变。生物信息学家继续设计专门的自动化系统来管理产生的大量序列数据，他们创造了新的算法和软件来将测序结果与不断增长的人类基因组序列和种系多态性集合进行比较。采用新的物理检测技术，例如识别染色体增加或者缺失的寡核苷酸微阵列(称为比较基因组杂交)，以及检测已知点突变的单核苷酸多态性阵列。这些检测方法同时测量整个基因组中的几十万个位点，当用于高通量测量数千个样本时，每个实验产生万亿字节的数据。大量新类型的数据再次为生物信息学家创造了新的机会。数据通常、包含相当大的可变性或噪声，因此隐马尔可夫模型和变化点分析方法被开发来推断真实拷贝数的变化。 </p>\n<p>两个重要的原则可用于癌症基因组的生物信息学分析，涉及外显子组突变的识别。首先，癌症是一种体细胞基因突变累积的疾病。第二种癌症包含司机突变，需要与乘客突变区分开来。<sup><a href=\"#quote_30\" class=\"kx_ref\">[30]</a></sup> </p>\n<p>随着这种下一代测序技术在生物信息学领域的突破，癌症基因组学可能会发生巨大变化。这些新方法和软件允许生物信息学家快速且经济地对许多癌症基因组进行测序。这可以通过分析基因组中癌症驱动的突变，创建一个更灵活的癌症分类过程。此外，利用癌症样本的序列在疾病发展的同时跟踪患者可能在将来成为可能。<sup><a href=\"#quote_31\" class=\"kx_ref\">[31]</a></sup> </p>\n<p>另一种需要新生物信息学发展的数据类型是对许多肿瘤中复发的病变的分析。 </p>","pics":[{"originalUrl":"https://web.archive.org/web/20221025085755/https://img01.sogoucdn.com/app/a/200698/sogou_science_498?w=500&h=116&titlename=%E4%B8%8D%E5%90%8C%E5%9F%BA%E5%9B%A0%E6%88%96%E8%9B%8B%E7%99%BD%E8%B4%A8%E7%9A%84%E5%BA%8F%E5%88%97%E5%8F%AF%E4%BB%A5%E5%B9%B6%E6%8E%92%E6%8E%92%E5%88%97%E4%BB%A5%E6%B5%8B%E9%87%8F%E5%AE%83%E4%BB%AC%E7%9A%84%E7%9B%B8%E4%BC%BC%E6%80%A7%E3%80%82%E8%BF%99%E7%A7%8D%E6%AF%94%E5%AF%B9%E6%AF%94%E8%BE%83%E4%BA%86%E8%9B%8B%E7%99%BD%E8%B4%A8%E5%BA%8F%E5%88%97%E5%92%8C%E5%90%AB%E6%9C%89WPP%E7%BB%93%E6%9E%84%E5%9F%9F%E7%9A%84%E5%9F%BA%E5%9B%A0%E7%BB%84%E5%BA%8F%E5%88%97%E3%80%82","url":"https://web.archive.org/web/20221025085755/https://img01.sogoucdn.com/app/a/200698/sogou_science_498","rw":500,"rh":116,"title":"不同基因或蛋白质的序列可以并排排列以测量它们的相似性。这种比对比较了蛋白质序列和含有WPP结构域的基因组序列。","alt":null,"width":0,"height":0}],"card":null,"references":[],"versionCount":0},{"paragraphId":"14995387530084879","title":"基因和蛋白质表达","versionId":"14995387513307657","lemmaId":10519,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":53099137,"name":"孟凡康","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233946,"comment":null,"dependVersionId":0,"contentType":1,"content":"<h3>基因表达分析</h3> \n<p>许多基因的表达可以通过用多种技术测量基因水平来确定，包括微阵列、表达的基因序列标签(EST)测序、基因表达的序列分析(SAGE)标签测序、大规模平行标记测序(MPSS)、RNA测序(RNA-Seq)，也称为“全转录组鸟枪法测序”(WTSS)，或多重原位杂交的各种应用。所有这些技术都极易产生噪声和/或在生物测量中受到偏差的影响，计算生物学的一个主要研究领域是开发统计工具，在高通量基因表达研究中从噪声中分离信号。<sup><a href=\"#quote_32\" class=\"kx_ref\">[32]</a></sup> 这种研究经常被用来确定与疾病有关的基因:人们可以将来自癌性上皮细胞的微阵列数据与来自非癌性细胞的数据进行比较，以确定特定癌细胞群体中上调和下调的转录产物。 </p> \n<h3>蛋白质表达分析</h3> \n<p>蛋白质微阵列和高通量质谱可以提供生物样品中蛋白质的信息。生物信息学在理解蛋白质微阵列和高温质谱数据方面非常重要；前一种方法面临着与靶向基因的微阵列相似的问题，后一种方法涉及将大量质谱数据与来自蛋白质序列数据库的预测质谱相匹配的问题，以及对样品进行复杂的统计分析，其中检测的蛋白质是数量多但不完整的肽。组织环境中的细胞蛋白质定位可以通过亲和蛋白质组学实现，亲和蛋白质组学显示为基于免疫组织化学和组织微阵列的空间数据。<sup><a href=\"#quote_33\" class=\"kx_ref\">[33]</a></sup> </p> \n<h3>调控分析</h3> \n<p>基因调控是一种复杂的事件协调过程，通过这种过程，一种信号，潜在的细胞外信号，如激素，最终导致一种或多种蛋白质活性的增加或减少。生物信息学技术已经被用于探索这个过程中的各个步骤。 </p>\n<p>例如，基因表达可以由基因组中附近的元素调节。启动子分析包括识别和研究基因编码区周围的序列模体。这些模体影响该区域转录成基因的程度。远离启动子的增强子元件也可以通过三维环状相互作用来调节基因表达。这些相互作用可以通过染色体构象捕获实验的生物信息学分析来确定。 </p>\n<p>表达数据可用于推断基因调控:人们可以比较来自生物体各种状态的微阵列数据，以形成关于每种状态所涉及基因的假设。在单细胞生物中，人们可能会比较细胞周期的各个阶段，以及各种应激条件(热休克、饥饿等)。然后可以对表达数据应用聚类算法来确定哪些基因被共同表达。例如，可以在共表达基因的上游区域(启动子)搜索过度表达的调控元件。基因聚类中应用的聚类算法的例子有k均值聚类、自组织映射(SOMs)、层次聚类和一致性聚类方法。 </p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14995387530084880","title":"细胞组织分析","versionId":"14995387513307658","lemmaId":10519,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":53099137,"name":"孟凡康","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233946,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>研究人员已经开发了几种方法用于分析细胞器、基因、蛋白质和细胞内其他成分的位置。这是相关的，因为这些成分的位置影响细胞内的事件，从而帮助我们预测生物系统的行为。研究人员在许多生物数据库中捕捉亚细胞定位（基因本体论范畴的细胞隔间）。 </p> \n<h3>显微镜和图像分析</h3> \n<p>显微照片使我们能够定位细胞器和分子。它还可以帮助我们区分正常细胞和异常细胞，例如癌症细胞。 </p> \n<h3>蛋白质定位</h3> \n<p>蛋白质的定位有助于我们评估蛋白质的作用。例如，如果在细胞核中发现一种蛋白质，它可能参与基因调控或分裂。相反，如果在线粒体中发现蛋白质，它可能参与呼吸或其他代谢过程。因此，蛋白质定位是蛋白质功能预测的重要组成部分。现在已经有成熟的蛋白质亚细胞定位预测资源，包括蛋白质亚细胞定位数据库和预测工具。<sup><a href=\"#quote_34\" class=\"kx_ref\">[34]</a></sup><sup><a href=\"#quote_35\" class=\"kx_ref\">[35]</a></sup> </p> \n<h3>染色质的核组织</h3> \n<p>来自高通量染色体构象捕获实验的数据，例如Hi-C和ChIA-PET，可以提供关于基因座空间邻近性的信息。分析这些实验可以确定染色质的三维结构和核组织。这一领域的生物信息学挑战包括将基因组分成在三维空间中组织在一起的域，如拓扑关联域(TADs)。<sup><a href=\"#quote_36\" class=\"kx_ref\">[36]</a></sup> </p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14995387530084881","title":"结构生物信息学","versionId":"14995387513307659","lemmaId":10519,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":53099137,"name":"孟凡康","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233946,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p></p><p><img alt=\"\" class=\"fileimage kx_img ed_imgfloat_right\" img_height=\"357\" img_width=\"300\" titlename=\"像这样的三维蛋白质结构是生物信息学分析中常见的课题。\" data-src=\"https://img03.sogoucdn.com/app/a/200698/sogou_science_4426\"> </p><p></p> \n<p>蛋白质结构预测是生物信息学的另一个重要应用。蛋白质的氨基酸序列，即所谓的初级结构，可以很容易地从编码它的基因序列中确定出来。在绝大多数情况下，这种主要结构独特地决定了其自然环境中的结构。(当然，也有例外，比如牛海绵状脑病——又称疯牛病——朊病毒。)了解这种结构对于理解蛋白质的功能至关重要。结构信息通常分为二级、三级和四级结构。这种预测的可行的总体解决方案仍然是一个悬而未决的问题。迄今为止，大多数努力都是针对大多数情况下有效的启发式方法。 </p>\n<p>生物信息学的关键思想之一是同源性的概念。在生物信息学的基因组分支中，同源性被用来预测基因的功能:如果功能已知的基因A序列与功能未知的基因B序列同源，就可以推断出基因B可能共享基因A的功能。在生物信息学的结构分支中，同源性被用来确定蛋白质的哪些部分在结构形成和与其他蛋白质的相互作用中是重要的。在一种称为同源建模的技术中，一旦同源蛋白质的结构已知，这些信息就被用来预测蛋白质的结构。这是目前可靠预测蛋白质结构的唯一方法。 </p>\n<p>这方面的一个例子是人类的血红蛋白和豆科植物的血红蛋白(豆血红蛋白)，它们是同一蛋白质超家族的远亲。两者在生物体中输送氧气的目的相同。尽管这两种蛋白质具有完全不同的氨基酸序列，但它们的蛋白质结构实际上是相同的，这反映了它们近乎相同的目的和共同的祖先。<sup><a href=\"#quote_37\" class=\"kx_ref\">[37]</a></sup> </p>\n<p>预测蛋白质结构的其他技术包括蛋白质穿引和从头开始的基于物理的建模。 </p>\n<p>结构生物信息学的另一个方面包括将蛋白质结构用于虚拟筛选模型，如定量结构-活性关系模型和蛋白质化学计量模型。此外，蛋白质的晶体结构可用于模拟例如配体结合研究和计算机诱变研究。 </p>","pics":[{"originalUrl":"https://web.archive.org/web/20221025085755/https://img03.sogoucdn.com/app/a/200698/sogou_science_4426?w=300&h=357&titlename=%E5%83%8F%E8%BF%99%E6%A0%B7%E7%9A%84%E4%B8%89%E7%BB%B4%E8%9B%8B%E7%99%BD%E8%B4%A8%E7%BB%93%E6%9E%84%E6%98%AF%E7%94%9F%E7%89%A9%E4%BF%A1%E6%81%AF%E5%AD%A6%E5%88%86%E6%9E%90%E4%B8%AD%E5%B8%B8%E8%A7%81%E7%9A%84%E8%AF%BE%E9%A2%98%E3%80%82","url":"https://web.archive.org/web/20221025085755/https://img03.sogoucdn.com/app/a/200698/sogou_science_4426","rw":300,"rh":357,"title":"像这样的三维蛋白质结构是生物信息学分析中常见的课题。","alt":null,"width":0,"height":0}],"card":null,"references":[],"versionCount":0},{"paragraphId":"14995387530084882","title":"网络和系统生物学","versionId":"14995387513307660","lemmaId":10519,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":53099137,"name":"孟凡康","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233946,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>网络分析寻求理解生物网络内的关系，如代谢或蛋白质-蛋白质相互作用网络。虽然生物网络可以由单一类型的分子或实体(如基因)构建，但网络生物学经常试图整合许多不同的数据类型，如蛋白质、小分子、基因表达数据等，它们都在物理上、功能上或两者都有关联。 </p>\n<p>系统生物学包括使用细胞子系统(如代谢物和酶网络，包括代谢、信号转导途径和基因调控网络)的计算机模拟来分析和可视化这些细胞过程的复杂联系。人工生命或虚拟进化试图通过简单(人工)生命形式的计算机模拟来理解进化过程。 </p> \n<h3>分子相互作用网络</h3> \n<p></p><p><img alt=\"\" class=\"fileimage kx_img ed_imgfloat_right\" img_height=\"188\" img_width=\"200\" titlename=\"蛋白质之间的相互作用经常用网络可视化和分析。这个网络由梅毒和其他疾病的病原体梅毒螺旋体的蛋白质相互作用组成。\" data-src=\"https://img03.sogoucdn.com/app/a/200698/sogou_science_493\"> </p><p></p>\n<p>成千上万的三维蛋白质结构已经由x光晶体学和蛋白质核磁共振谱(蛋白质核磁共振)确定，结构生物信息学的一个中心问题是，仅仅基于这些三维形状而不进行蛋白质-蛋白质相互作用实验预测可能的蛋白质-蛋白质相互作用是否可行。。研究人员已经开发了多种方法来解决蛋白质-蛋白质对接问题，尽管在这一领域似乎还有许多工作要做。 </p>\n<p>该领域遇到的其他相互作用包括蛋白质-配体(包括药物)和蛋白质-肽。原子围绕旋转键运动的分子动力学模拟是研究分子相互作用的计算算法(称为对接算法)背后的基本原理。 </p>","pics":[{"originalUrl":"https://web.archive.org/web/20221025085755/https://img03.sogoucdn.com/app/a/200698/sogou_science_493?w=200&h=188&titlename=%E8%9B%8B%E7%99%BD%E8%B4%A8%E4%B9%8B%E9%97%B4%E7%9A%84%E7%9B%B8%E4%BA%92%E4%BD%9C%E7%94%A8%E7%BB%8F%E5%B8%B8%E7%94%A8%E7%BD%91%E7%BB%9C%E5%8F%AF%E8%A7%86%E5%8C%96%E5%92%8C%E5%88%86%E6%9E%90%E3%80%82%E8%BF%99%E4%B8%AA%E7%BD%91%E7%BB%9C%E7%94%B1%E6%A2%85%E6%AF%92%E5%92%8C%E5%85%B6%E4%BB%96%E7%96%BE%E7%97%85%E7%9A%84%E7%97%85%E5%8E%9F%E4%BD%93%E6%A2%85%E6%AF%92%E8%9E%BA%E6%97%8B%E4%BD%93%E7%9A%84%E8%9B%8B%E7%99%BD%E8%B4%A8%E7%9B%B8%E4%BA%92%E4%BD%9C%E7%94%A8%E7%BB%84%E6%88%90%E3%80%82","url":"https://web.archive.org/web/20221025085755/https://img03.sogoucdn.com/app/a/200698/sogou_science_493","rw":200,"rh":188,"title":"蛋白质之间的相互作用经常用网络可视化和分析。这个网络由梅毒和其他疾病的病原体梅毒螺旋体的蛋白质相互作用组成。","alt":null,"width":0,"height":0}],"card":null,"references":[],"versionCount":0},{"paragraphId":"14995387530084883","title":"其他","versionId":"14995387513307661","lemmaId":10519,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":53099137,"name":"孟凡康","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233946,"comment":null,"dependVersionId":0,"contentType":1,"content":"<h3>文献分析</h3> \n<p>出版文献数量的增长使得研究人员几乎不可能阅读每一篇论文，导致研究的子领域脱节。文献分析旨在利用计算和统计语言学来挖掘这个不断增长的文本资源库。例如: </p> \n<ul>\n <li>缩写识别–识别生物学术语的长形式和缩写</li> \n <li>命名实体识别——识别生物术语，如基因名称</li> \n <li>蛋白质-蛋白质相互作用-识别文本中哪些蛋白质与哪些蛋白质相互作用</li>\n</ul> \n<p>研究领域来自统计学和计算语言学。 </p> \n<h3>高通量图像分析</h3> \n<p>计算技术用于加速或完全自动化大量高信息含量生物医学图像的处理、量化和分析。现代图像分析系统通过提高精确度、客观性或速度，增强了观察者从大量或复杂图像中进行测量的能力。一个开发完全的分析系统可以完全取代观察者。尽管这些系统并非生物医学成像所独有，但生物医学成像对诊断和研究都变得越来越重要。一些例子是: </p> \n<ul>\n <li>高通量和高保真定量和亚细胞定位(高含量筛选、细胞组织病理学、生物图像信息学)</li> \n <li>形态特征</li> \n <li>临床图像分析和可视化</li> \n <li>确定活体动物呼吸肺中的实时气流模式</li> \n <li>从动脉损伤的发展和恢复中量化实时图像中的闭塞大小</li> \n <li>从实验动物的扩展视频记录中进行行为观察</li> \n <li>用于代谢活动测定的红外测量</li> \n <li>推断克隆在DNA图谱中的重叠，例如苏尔斯顿分数</li>\n</ul> \n<h3>高通量单细胞数据分析</h3> \n<p>计算技术用于分析高通量、低测量的单细胞数据，例如从流式细胞仪获得的数据。这些方法通常包括寻找与特定疾病状态或实验条件相关的细胞群。 </p> \n<h3>生物多样性信息学</h3> \n<p>生物多样性信息学处理生物多样性数据的收集和分析，如分类数据库或微生物群数据。这种分析的例子包括遗传基因、生态位建模、物种丰富度绘图、DNA条形码或物种识别工具。 </p> \n<h3>本体论和数据集成</h3> \n<p>生物本体是受控词汇的有向无环图。它们旨在捕捉生物学概念和描述，便于用计算机进行分类和分析。当以这种方式分类时，从整体和综合分析中获得附加值是可能的。 </p>\n<p>OBOFoundry致力于标准化某些本体。其中最广泛的是描述基因功能的基因本体论。也有描述表型的本体论。 </p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14995387546862090","title":"数据库","versionId":"14995387513307662","lemmaId":10519,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":53099137,"name":"孟凡康","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233946,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>数据库对于生物信息学研究和应用至关重要。存在许多数据库，涵盖各种信息类型:例如，DNA和蛋白质序列、分子结构、表型和生物多样性。数据库可能包含经验数据(直接从实验中获得)、预测数据(从分析中获得)，或最常见的两者。它们可能对特定的生物体、途径或感兴趣的分子具有特异性。或者，它们可以合并从多个其他数据库编译的数据。这些数据库在格式、访问机制以及它们是否公开方面各不相同。 </p>\n<p>下面列出了一些最常用的数据库。 </p> \n<ul>\n <li>用于生物序列分析:Genbank，UniProt</li> \n <li>用于结构分析:蛋白质数据库(PDB)</li> \n <li>用于寻找蛋白质家族和基序:InterPro，Pfam</li> \n <li>用于下一代测序:序列读取存档</li> \n <li>用于网络分析:代谢途径数据库(KEGG，BioCyc)，相互作用分析数据库，功能网络</li> \n <li>用于合成遗传电路的设计:GenoCAD</li>\n</ul>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14995387546862091","title":"软件和工具","versionId":"14995387513307663","lemmaId":10519,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":53099137,"name":"孟凡康","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233946,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>生物信息学软件工具的范围从简单的命令行工具到更复杂的图形程序和各种生物信息学公司或公共机构提供的独立网络服务。 </p> \n<h3>开源生物信息学软件</h3> \n<p>自20世纪80年代以来，许多免费和开源软件工具已经存在并继续增长。<sup><a href=\"#quote_38\" class=\"kx_ref\">[38]</a></sup> 对用于分析新兴类型生物读数的新算法的持续需求、创新电子实验的潜力和免费开放代码库的结合，为了所有研究团体创造新的机会，为生物信息学和一系列可用的开放源代码软件做出贡献，无论它们的资助是如何获得的。开源工具通常是想法的孵化器，或者商业应用中社区支持的插件。它们还可以提供业界标准和共享的对象模型来帮助研究人员应对生物信息集成的挑战。 </p>\n<p>开源软件包的范围包括诸如Bioconductor、BioPerl、 Biopython、 BioJava、BioJS、BioRuby、Bioclipse、EMBOSS、.NET Bio、Orange及其生物信息学插件，Apache Taverna，UGENE和GenoCAD。<sup><a href=\"#quote_38\" class=\"kx_ref\">[38]</a></sup> 为了保持这一传统并创造更多的机会，非营利的开放生物信息学基金会自2000年以来一直支持一年一度的生物信息学开放源码会议(BOSC)。<sup><a href=\"#quote_39\" class=\"kx_ref\">[39]</a></sup> </p>\n<p>构建公共生物信息学数据库的另一种方法是使用带有维基百科扩展的媒体维基引擎。该系统允许该领域的所有专家访问和更新数据库。<sup><a href=\"#quote_40\" class=\"kx_ref\">[40]</a></sup> </p> \n<h3>生物信息学中的Web服务</h3> \n<p>基于SOAP和REST的接口已经为各种各样的生物信息学应用程序开发出来，允许运行在世界一个地方的一台计算机上的应用程序在世界其他地方的服务器上使用算法、数据和计算资源。期主要优势是终端用户不必进行软件和数据库维护开销。 </p>\n<p>EBI将基本生物信息学服务分为三类:序列搜索服务、多序列比对服务和生物序列分析服务。<sup><a href=\"#quote_41\" class=\"kx_ref\">[41]</a></sup> 这些面向服务的生物信息学资源的可用性证明了基于网络的生物信息学解决方案的适用性，范围从在单一、独立或基于网络的界面下具有通用数据格式的独立工具的集合，到集成、分布式和可扩展的生物信息学工作流管理系统。 </p> \n<h3>生物信息学工作流管理系统</h3> \n<p>生物信息学工作流管理系统是工作流管理系统的一种特殊形式，专门设计用于在生物信息学应用程序中组成和执行一系列计算或数据操作步骤或工作流。这种系统旨在 </p> \n<ul>\n <li>为单个应用科学家自己创建自己的工作流提供一个易于使用的环境，</li> \n <li>为科学家提供交互式工具，使他们能够执行工作流并实时查看结果，</li> \n <li>简化科学家之间共享和重用工作流的过程，</li> \n <li>使科学家能够跟踪工作流执行结果和工作流创建步骤的来源。</li>\n</ul> \n<p>提供这项服务的一些平台有: Galaxy、Kepler、Taverna、UGENE、Anduril、HIVE。 </p> \n<h3>生物计算机和生物计算机对象</h3> \n<p>2014年，美国食品和药物管理局在国家卫生研究院贝塞斯达校区主办了一次会议，讨论生物信息学中的再现性。<sup><a href=\"#quote_42\" class=\"kx_ref\">[42]</a></sup> 在接下来的三年里，一群利益相关者定期会面，讨论什么将成为生物计算机范例。<sup><a href=\"#quote_43\" class=\"kx_ref\">[43]</a></sup> 这些利益攸关方包括政府、行业和学术实体的代表。会议领导人代表了美国食品和药物管理局和美国国立卫生研究院的许多分支机构和中心，包括人类变异组项目和欧洲医学信息学联合会在内的非营利实体，以及包括斯坦福大学、纽约基因组中心和乔治华盛顿大学在内的研究机构。 </p>\n<p>会议决定已经决定生物计算机范例将采用数字“实验室笔记本”的形式，允许生物信息学协议的再现性、复制性、审查性和重用性。这是为了使研究小组在正常人员流动过程中保持更大的连续性，同时促进小组之间的思想交流。美国食品和药物管理局资助了这项工作，以使管道信息更加透明，便于监管人员获取。<sup><a href=\"#quote_44\" class=\"kx_ref\">[44]</a></sup> </p>\n<p>2016年，该小组在贝塞斯达的国立卫生研究院再次召开会议，讨论了生物计算机对象的潜力，这是生物计算机范例的一个实例。这项工作被复制为“标准试用”文件和上传到bioRxiv的预印本文件。生物计算机对象允许员工、合作者和监管者共享JSON化的记录。<sup><a href=\"#quote_45\" class=\"kx_ref\">[45]</a></sup><sup><a href=\"#quote_46\" class=\"kx_ref\">[46]</a></sup> </p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14995387546862092","title":"教育平台","versionId":"14995387513307664","lemmaId":10519,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":53099137,"name":"孟凡康","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233946,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>旨在教授生物信息学概念和方法的软件平台包括Rosalind和通过瑞士生物信息学研究所培训门户网站提供的在线课程。加拿大生物信息学研讨会根据知识共享许可在其网站上提供培训研讨会的视频和幻灯片。4273π项目或4273pi项目<sup><a href=\"#quote_47\" class=\"kx_ref\">[47]</a></sup> 也免费提供开源教育材料。该课程在低成本的 Raspberry Pi电脑上运行，并已被用于教育成人和小学生。<sup><a href=\"#quote_48\" class=\"kx_ref\">[48]</a></sup><sup><a href=\"#quote_49\" class=\"kx_ref\">[49]</a></sup>4273π由一个学术团体和研究人员积极开发，他们使用Raspberry Pi计算机和4273π操作系统运行研究级生物信息学。<sup><a href=\"#quote_50\" class=\"kx_ref\">[50]</a></sup><sup><a href=\"#quote_51\" class=\"kx_ref\">[51]</a></sup> </p>\n<p>MOOC平台还提供生物信息学和相关学科的在线认证，包括Coursera的生物信息学专业(加州大学圣地亚哥分校)和基因组数据科学专业(约翰·霍普金斯大学)以及EdX的生命科学系列数据分析(哈佛大学)。南加州大学提供侧重于生物医学应用的翻译生物信息学硕士学位。 </p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14995387546862093","title":"会议","versionId":"14995387513307665","lemmaId":10519,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":53099137,"name":"孟凡康","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233946,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>有几个关于生物信息学的大型会议。一些最著名的例子是分子生物学智能系统(ISMB)、欧洲计算生物学会议(ECCB)和计算分子生物学研究会议（RECOMB）。 </p>","pics":null,"card":null,"references":[],"versionCount":0}],"references":[{"id":1,"type":"book","title":"Lesk, A. M. (26 July 2013). \"Bioinformatics\". Encyclopaedia Britannica. Retrieved 17 April 2017.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":2,"type":"book","title":"Sim, A. Y. L.; Minary, P.; Levitt, M. (2012). \"Modeling nucleic acids\". Current Opinion in Structural Biology. 22 (3): 273–278. doi:10.1016/j.sbi.2012.03.012. PMC 4028509. PMID 22538125.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":3,"type":"book","title":"Dawson, W. K.; Maciejczyk, M.; Jankowska, E. J.; Bujnicki, J. M. (2016). \"Coarse-grained modeling of RNA 3D structure\" (PDF). Methods. 103: 138–156. doi:10.1016/j.ymeth.2016.04.026. PMID 27125734.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":4,"type":"book","title":"Kmiecik, S.; Gront, D.; Kolinski, M.; Wieteska, L.; Dawid, A. E.; Kolinski, A. (2016). \"Coarse-Grained Protein Models and Their Applications\". Chemical Reviews. 116 (14): 7898–936. doi:10.1021/acs.chemrev.6b00163. PMID 27333362.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":5,"type":"book","title":"Wong, K. C. (2016). Computational Biology and Bioinformatics: Gene Regulation. CRC Press/Taylor & Francis Group. ISBN 9781498724975.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":6,"type":"book","title":"Joyce, A. P.; Zhang, C.; Bradley, P.; Havranek, J. J. (2015). \"Structure-based modeling of protein: DNA specificity\". Briefings in Functional Genomics. 14 (1): 39–49. doi:10.1093/bfgp/elu044. PMC 4366589. PMID 25414269.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":7,"type":"book","title":"Spiga, E.; Degiacomi, M. T.; Dal Peraro, M. (2014). \"New Strategies for Integrative Dynamic Modeling of Macromolecular Assembly\". In Karabencheva-Christova, T. Biomolecular Modelling and Simulations. Advances in Protein Chemistry and Structural Biology. 96. Academic Press. pp. 77–111. doi:10.1016/bs.apcsb.2014.06.008. ISBN 9780128000137. PMID 25443955.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":8,"type":"book","title":"Ciemny, Maciej; Kurcinski, Mateusz; Kamel, Karol; Kolinski, Andrzej; Alam, Nawsad; Schueler-Furman, Ora; Kmiecik, Sebastian (2018-05-04). \"Protein–peptide docking: opportunities and challenges\". Drug Discovery Today (in 英语). 23 (8): 1530–1537. doi:10.1016/j.drudis.2018.05.006. ISSN 1359-6446. PMID 29733895.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":9,"type":"book","title":"Hogeweg P (2011). Searls, David B., ed. \"The Roots of Bioinformatics in Theoretical Biology\". PLoS Computational Biology. 7 (3): e1002021. Bibcode:2011PLSCB...7E2021H. doi:10.1371/journal.pcbi.1002021. PMC 3068925. PMID 21483479.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":10,"type":"book","title":"Hesper B, Hogeweg P (1970). \"Bioinformatica: een werkconcept\". 1 (6). Kameleon: 28–29.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":11,"type":"book","title":"Hogeweg P (1978). \"Simulating the growth of cellular forms\". Simulation. 31 (3): 90–96. doi:10.1177/003754977803100305.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":12,"type":"book","title":"Moody, Glyn (2004). Digital Code of Life: How Bioinformatics is Revolutionizing Science, Medicine, and Business. ISBN 978-0-471-32788-2.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":13,"type":"book","title":"Dayhoff, M.O. (1966) Atlas of protein sequence and structure. National Biomedical Research Foundation, 215 pp.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":14,"type":"book","title":"Eck RV, Dayhoff MO (1966). \"Evolution of the structure of ferredoxin based on living relics of primitive amino Acid sequences\". Science. 152 (3720): 363–6. Bibcode:1966Sci...152..363E. doi:10.1126/science.152.3720.363. PMID 17775169.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":15,"type":"book","title":"Johnson G, Wu TT (January 2000). \"Kabat Database and its applications: 30 years after the first variability plot\". Nucleic Acids Res. 28 (1): 214–218. doi:10.1093/nar/28.1.214. PMC 102431. PMID 10592229.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":16,"type":"book","title":"Attwood TK, Gisel A, Eriksson NE, Bongcam-Rudloff E (2011). \"Concepts, Historical Milestones and the Central Place of Bioinformatics in Modern Biology: A European Perspective\". Bioinformatics - Trends and Methodologies. Bioinformatics – Trends and Methodologies. InTech. doi:10.5772/23535. ISBN 978-953-307-282-1. Retrieved 8 Jan 2012.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":17,"type":"book","title":"Sanger F, Air GM, Barrell BG, Brown NL, Coulson AR, Fiddes CA, Hutchison CA, Slocombe PM, Smith M (February 1977). \"Nucleotide sequence of bacteriophage phi X174 DNA\". Nature. 265 (5596): 687–95. Bibcode:1977Natur.265..687S. doi:10.1038/265687a0. PMID 870828.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":18,"type":"book","title":"Benson DA, Karsch-Mizrachi I, Lipman DJ, Ostell J, Wheeler DL (January 2008). \"GenBank\". Nucleic Acids Res. 36 (Database issue): D25–30. doi:10.1093/nar/gkm929. PMC 2238942. PMID 18073190.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":19,"type":"book","title":"Fleischmann RD, Adams MD, White O, Clayton RA, Kirkness EF, Kerlavage AR, Bult CJ, Tomb JF, Dougherty BA, Merrick JM (July 1995). \"Whole-genome random sequencing and assembly of Haemophilus influenzae Rd\". Science. 269 (5223): 496–512. Bibcode:1995Sci...269..496F. doi:10.1126/science.7542800. PMID 7542800.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":20,"type":"book","title":"Carvajal-Rodríguez A (2012). \"Simulation of Genes and Genomes Forward in Time\". Current Genomics. 11 (1): 58–61. doi:10.2174/138920210790218007. PMC 2851118. PMID 20808525.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":21,"type":"book","title":"Brown, TA (2002). \"Mutation, Repair and Recombination\". Genomes (2nd ed.). Manchester (UK): Oxford.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":22,"type":"book","title":"Carter, N. P.; Fiegler, H.; Piper, J. (2002). \"Comparative analysis of comparative genomic hybridization microarray technologies: Report of a workshop sponsored by the Wellcome trust\". Cytometry Part A. 49 (2): 43–8. doi:10.1002/cyto.10153. PMID 12357458.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":23,"type":"book","title":"Chaudhari Narendrakumar M., Kumar Gupta Vinod, Dutta Chitra (2016). \"BPGA-an ultra-fast pan-genome analysis pipeline\". Scientific Reports. 6: 24373. Bibcode:2016NatSR...624373C. doi:10.1038/srep24373. PMC 4829868. PMID 27071527.CS1 maint: Multiple names: authors list (link)","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":24,"type":"book","title":"Véron A, Blein S, Cox DG (2014). \"Genome-wide association studies and the clinic: A focus on breast cancer\". Biomarkers in Medicine. 8 (2): 287–96. doi:10.2217/bmm.13.121. PMID 24521025.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":25,"type":"book","title":"Tosto G, Reitz C (2013). \"Genome-wide association studies in Alzheimer's disease: A review\". Current Neurology and Neuroscience Reports. 13 (10): 381. doi:10.1007/s11910-013-0381-0. PMC 3809844. PMID 23954969.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":26,"type":"book","title":"Aston KI (2014). \"Genetic susceptibility to male infertility: News from genome-wide association studies\". Andrology. 2 (3): 315–21. doi:10.1111/j.2047-2927.2014.00188.x. PMID 24574159.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":27,"type":"book","title":"Londin E, Yadav P, Surrey S, Kricka LJ, Fortina P (2013). Use of Linkage Analysis, Genome-Wide Association Studies, and Next-Generation Sequencing in the Identification of Disease-Causing Mutations. Pharmacogenomics. Methods in Molecular Biology. 1015. pp. 127–46. doi:10.1007/978-1-62703-435-7_8. ISBN 978-1-62703-434-0. PMID 23824853.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":28,"type":"book","title":"Hindorff, L.A.; et al. (2009). \"Potential etiologic and functional implications of genome-wide association loci for human diseases and traits\". Proc. Natl. Acad. Sci. USA. 106 (23): 9362–9367. Bibcode:2009PNAS..106.9362H. doi:10.1073/pnas.0903103106. PMC 2687147. PMID 19474294.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":29,"type":"book","title":"Hall, L.O. (2010). Finding the right genes for disease and prognosis prediction. System Science and Engineering (ICSSE),2010 International Conference. pp. 1–2. doi:10.1109/ICSSE.2010.5551766. ISBN 978-1-4244-6472-2.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":30,"type":"book","title":"Vazquez, Miguel; Torre, Victor de la; Valencia, Alfonso (2012-12-27). \"Chapter 14: Cancer Genome Analysis\". PLOS Computational Biology (in 英语). 8 (12): e1002824. Bibcode:2012PLSCB...8E2824V. doi:10.1371/journal.pcbi.1002824. ISSN 1553-7358. PMC 3531315. PMID 23300415.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":31,"type":"book","title":"Hye-Jung, E.C.; Jaswinder, K.; Martin, K.; Samuel, A.A; Marco, A.M (2014). \"Second-Generation Sequencing for Cancer Genome Analysis\". In Dellaire, Graham; Berman, Jason N.; Arceci, Robert J. Cancer Genomics. Boston (US): Academic Press. pp. 13–30. doi:10.1016/B978-0-12-396967-5.00002-5. ISBN 9780123969675.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":32,"type":"book","title":"Grau, J.; Ben-Gal, I.; Posch, S.; Grosse, I. (1 July 2006). \"VOMBAT: prediction of transcription factor binding sites using variable order Bayesian trees\" (PDF). Nucleic Acids Research. 34 (Web Server): W529–W533. doi:10.1093/nar/gkl212. PMC 1538886. PMID 16845064.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":33,"type":"book","title":"\"The Human Protein Atlas\". www.proteinatlas.org. Retrieved 2017-10-02.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":34,"type":"book","title":"\"The human cell\". www.proteinatlas.org. Retrieved 2017-10-02.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":35,"type":"book","title":"Thul, Peter J.; Åkesson, Lovisa; Wiking, Mikaela; Mahdessian, Diana; Geladaki, Aikaterini; Blal, Hammou Ait; Alm, Tove; Asplund, Anna; Björk, Lars (2017-05-26). \"A subcellular map of the human proteome\". Science. 356 (6340): eaal3321. doi:10.1126/science.aal3321. PMID 28495876.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":36,"type":"book","title":"Ay, Ferhat; Noble, William S. (2 September 2015). \"Analysis methods for studying the 3D architecture of the genome\". Genome Biology. 16 (1): 183. doi:10.1186/s13059-015-0745-7. PMC 4556012. PMID 26328929.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":37,"type":"book","title":"Hoy, JA; Robinson, H; Trent JT, 3rd; Kakar, S; Smagghe, BJ; Hargrove, MS (3 August 2007). \"Plant hemoglobins: a molecular fossil record for the evolution of oxygen transport\". Journal of Molecular Biology. 371 (1): 168–79. doi:10.1016/j.jmb.2007.05.029. PMID 17560601.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":38,"type":"book","title":"\"Open Bioinformatics Foundation: About us\". Official website. Open Bioinformatics Foundation. Retrieved 10 May 2011.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":39,"type":"book","title":"\"Open Bioinformatics Foundation: BOSC\". Official website. Open Bioinformatics Foundation. Retrieved 10 May 2011.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":40,"type":"book","title":"Brohée, Sylvain; Barriot, Roland; Moreau, Yves (2010). \"Biological knowledge bases using Wikis: combining the flexibility of Wikis with the structure of databases\". Bioinformatics. 26 (17): 2210–2211. doi:10.1093/bioinformatics/btq348. PMID 20591906. Retrieved 5 May 2015.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":41,"type":"book","title":"Nisbet, Robert (14 May 2009). \"BIOINFORMATICS\". Handbook of Statistical Analysis and Data Mining Applications. John Elder IV, Gary Miner. Academic Press. p. 328. ISBN 9780080912035. Retrieved 9 May 2014.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":42,"type":"book","title":"Commissioner, Office of the. \"Advancing Regulatory Science - Sept. 24-25, 2014 Public Workshop: Next Generation Sequencing Standards\". www.fda.gov (in 英语). Retrieved 2017-11-30.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":43,"type":"book","title":"Simonyan, Vahan; Goecks, Jeremy; Mazumder, Raja (2017). \"Biocompute Objects—A Step towards Evaluation and Validation of Biomedical Scientific Computations\". PDA Journal of Pharmaceutical Science and Technology. 71 (2): 136–146. doi:10.5731/pdajpst.2016.006734. ISSN 1079-7440. PMC 5510742. PMID 27974626.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":44,"type":"book","title":"Commissioner, Office of the. \"Advancing Regulatory Science - Community-based development of HTS standards for validating data and computation and encouraging interoperability\". www.fda.gov (in 英语). Retrieved 2017-11-30.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":45,"type":"book","title":"Alterovitz, Gil; Dean, Dennis A.; Goble, Carole; Crusoe, Michael R.; Soiland-Reyes, Stian; Bell, Amanda; Hayes, Anais; King, Charles Hadley S.; Johanson, Elaine (2017-10-04). \"Enabling Precision Medicine via standard communication of NGS provenance, analysis, and results\". bioRxiv 191783.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":46,"type":"book","title":"BioCompute Object (BCO) project is a collaborative and community-driven framework to standardize HTS computational data. 1. BCO Specification Document: user manual for understanding and creating B., biocompute-objects, 2017-09-03, retrieved 2017-11-30","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":47,"type":"book","title":"Barker, D; Ferrier, D.E.K.; Holland, P.W; Mitchell, J.B.O; Plaisier, H; Ritchie, M.G; Smart, S.D. (2013). \"4273π : bioinformatics education on low cost ARM hardware\". BMC Bioinformatics. 14: 243. doi:10.1186/1471-2105-14-243. PMC 3751261. PMID 23937194.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":48,"type":"book","title":"Barker, D; Alderson, R.G; McDonagh, J.L; Plaisier, H; Comrie, M.M; Duncan, L; Muirhead, G.T.P; Sweeny, S.D. (2015). \"University-level practical activities in bioinformatics benefit voluntary groups of pupils in the last 2 years of school\". International Journal of STEM Education. 2 (17). doi:10.1186/s40594-015-0030-z.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":49,"type":"book","title":"McDonagh, J.L; Barker, D; Alderson, R.G. (2016). \"Bringing computational science to the public\". SpringerPlus. 5 (259): 259. doi:10.1186/s40064-016-1856-7. PMC 4775721. PMID 27006868.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":50,"type":"book","title":"Robson, J.F.; Barker, D (2015). \"Comparison of the protein-coding gene content of Chlamydia trachomatis and Protochlamydia amoebophila using a Raspberry Pi computer\". BMC Research Notes. 8 (561): 561. doi:10.1186/s13104-015-1476-2. PMC 4604092. PMID 26462790.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":51,"type":"book","title":"Wregglesworth, K.M; Barker, D (2015). \"A comparison of the protein-coding genomes of two green sulphur bacteria, Chlorobium tepidum TLS and Pelodictyon phaeoclathratiforme BU-1\". BMC Research Notes. 8 (565): 565. doi:10.1186/s13104-015-1535-8. PMC 4606965. PMID 26467441.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false}],"recommendReferences":null,"auditState":2,"lemmaLevel":1,"origin":0,"originEnTitle":null,"originZhTitle":null,"pv":5105,"auditType":0,"synonyms":["生物信息","BI","bioinformatics","生物学信息"],"showEditTime":"2019.12.13 18:45","auditors":[{"uid":0,"name":"Ki.κe","pic":"https://web.archive.org/web/20221025085755/https://wx.qlogo.cn/mmopen/vi_32/y67kfr32Doib4wg71Jiau7jVWvharic3nRKgdRRQSl6koeQJCo0GQs2Krw0vwdFRsOWnHIQOwAZsSg5lIkIrFCOcQ/132","introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":"","jobBrief":"","role":0,"roleName":null,"title":"","professionalTitle":null,"phoneNo":null,"editable":true,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false}],"hasZhishiNav":true,"auditInfos":{},"isHistory":false};</script><script crossorigin="anonymous" src="./519.生物信息学 - 搜狗科学百科_files/aegis.min.js.download"></script><script crossorigin="anonymous" src="./519.生物信息学 - 搜狗科学百科_files/main_2020092401.js.download"></script><script crossorigin="anonymous" src="./519.生物信息学 - 搜狗科学百科_files/react.production.min.js.download"></script><script crossorigin="anonymous" src="./519.生物信息学 - 搜狗科学百科_files/react-dom.production.min.js.download"></script><script crossorigin="anonymous" src="./519.生物信息学 - 搜狗科学百科_files/jquery-1.11.1.min.js.download"></script><script crossorigin="anonymous" src="./519.生物信息学 - 搜狗科学百科_files/main_2022062701.js.download"></script><script crossorigin="anonymous" src="./519.生物信息学 - 搜狗科学百科_files/main_66bbe21.js.download"></script><script crossorigin="anonymous" src="./519.生物信息学 - 搜狗科学百科_files/react.production.min.js.download"></script><script crossorigin="anonymous" src="./519.生物信息学 - 搜狗科学百科_files/react-dom.production.min.js.download"></script><script crossorigin="anonymous" src="./519.生物信息学 - 搜狗科学百科_files/main_edf0f08.js.download"></script>
</body></html>