<!DOCTYPE html>
<!-- saved from url=(0083)https://web.archive.org/web/20221028212443/https://baike.sogou.com/kexue/d10063.htm -->
<html class="" data-reactroot=""><script async="false" type="text/javascript" src="chrome-extension://fnjhmkhhmkbjkkabndcnnogagogbneec/in-page.js"></script><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><script src="./63.纺锤体 - 搜狗科学百科_files/analytics.js.download" type="text/javascript"></script>
<script type="text/javascript">window.addEventListener('DOMContentLoaded',function(){var v=archive_analytics.values;v.service='wb';v.server_name='wwwb-app220.us.archive.org';v.server_ms=199;archive_analytics.send_pageview({});});</script>
<script type="text/javascript" src="./63.纺锤体 - 搜狗科学百科_files/bundle-playback.js.download" charset="utf-8"></script>
<script type="text/javascript" src="./63.纺锤体 - 搜狗科学百科_files/wombat.js.download" charset="utf-8"></script>
<script type="text/javascript">
  __wm.init("https://web.archive.org/web");
  __wm.wombat("https://baike.sogou.com/kexue/d10063.htm","20221028212443","https://web.archive.org/","web","/_static/",
	      "1666992283");
</script>
<link rel="stylesheet" type="text/css" href="./63.纺锤体 - 搜狗科学百科_files/banner-styles.css">
<link rel="stylesheet" type="text/css" href="./63.纺锤体 - 搜狗科学百科_files/iconochive.css">
<!-- End Wayback Rewrite JS Include -->
<meta name="save" content="history"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="baidu-site-verification" content="VWGb6TyYx8"><meta content="纺锤体 - 搜狗科学百科" name="keywords"><meta content="搜狗科学百科是一部有着平等、协作、分享、自由理念的网络科学全书，为每一个互联网用户创造一个涵盖所有领域知识、服务的中文知识性平台。" name="description"><meta http-equiv="x-dns-prefetch-control" content="on"><meta name="server" baike="235" ip="210" env="online"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028212443/https://cache.soso.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028212443/https://hhy.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028212443/https://pic.baike.soso.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028212443/https://ugc.qpic.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028212443/https://xui.ptlogin2.qq.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028212443/https://q1.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028212443/https://q2.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028212443/https://q3.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028212443/https://q4.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028212443/https://q.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028212443/https://img01.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028212443/https://img02.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028212443/https://img03.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221028212443/https://img04.sogoucdn.com/"><link rel="Shortcut Icon" href="https://web.archive.org/web/20221028212443im_/https://www.sogou.com/images/logo/new/favicon.ico?v=4"><link rel="Bookmark" href="https://www.sogou.com/images/logo/new/favicon.ico?v=4"><link href="./63.纺锤体 - 搜狗科学百科_files/base_b849887.css" rel="stylesheet"><link href="./63.纺锤体 - 搜狗科学百科_files/detail_378aed5.css" rel="stylesheet"><link href="./63.纺锤体 - 搜狗科学百科_files/inviteAudit_7894507.css" rel="stylesheet"><link rel="stylesheet" href="./63.纺锤体 - 搜狗科学百科_files/highlight.min.css"><title>纺锤体 - 搜狗科学百科</title><style>.onekey-close {
	position: absolute;
	top: 16px;
	right: 16px;
	width: 24px;
	height: 24px;
	background: url(//web.archive.org/web/20221028205211/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	text-indent: -999em;
	background-size: 84px;
	background-position: -63px 0;
}

.onekey-login {
	position: absolute;
	top: 16.4%;
	left: 0;
	right: 0;
	width: 100%;
}

/* .onekey-login-img {
    width: 75px;
    height: 75px;
    background: url("https://web.archive.org/web/20221028212443/https://baike.sogou.com/kexue/images/sprite_wap_baike.png") no-repeat;
    background-size: 100px 91px;
    background-position: 0 0;
    background-repeat: no-repeat;
    margin: 0 auto;
} */

.onekey-login-title {
	text-align: center;
	padding-bottom: 3px;
	font-size: 21px;
	font-weight: bold;
	line-height: 30px;
	color: #000;
}

.onekey-login-txt {
	text-align: center;
	font-family: PingFangSC;
	font-size: 14px;
	line-height: 20px;
	color: #8f8f8f;
}

.onekey-login-qq,
.onekey-login-wx,
.onekey-login-phone {
	display: block;
	width: 245px;
	height: 54px;
	border-radius: 45px;
	text-align: center;

	margin: 0 auto;
	font-size: 17px;
	line-height: 24px;
	color: #000;
	/* padding: 16px 77px; */
	border-radius: 12px;
	border: solid 1px #e0e0e0;
}
.onekey-qq-content,
.onekey-vx-content,
.onekey-phone-content {
	display: inline-block;
	margin-top: 16px;
}
.onekey-qq-content {
	padding: 0 5px;
}

.onekey-login-qq {
	margin-top: 48px;
	margin-bottom: 24px;
}

.onekey-login-qq:before {
	display: inline-block;
	content: "";
	width: 20px;
	height: 20px;
	background: url(//web.archive.org/web/20221028205211/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	background-size: 80px;
	background-position: -20px 0;
	vertical-align: top;
	margin: 17px 8px 0 0;
}

.onekey-login-wx {
	margin-bottom: 24px;
}

.onekey-login-wx:before {
	display: inline-block;
	content: "";
	width: 21px;
	height: 21px;
	background: url(//web.archive.org/web/20221028205211/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	background-size: 84px;
	background-position: 0 0;
	vertical-align: top;
	margin: 17px 10px 0 0;
}

.onekey-login-phone {
}

.onekey-login-phone:before {
	display: inline-block;
	content: "";
	width: 21px;
	height: 21px;
	background: url(//web.archive.org/web/20221028205211/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	background-size: 84px;
	background-position: -42px 0;
	vertical-align: top;
	margin: 17px 10px 0 0;
}

.onekey-fixed {
	z-index: 100;
	position: fixed;
	top: 0;
	bottom: 0;
	left: 0;
	right: 0;
	background: #fff;
	width: 100%;
	height: 100%;
}

.onekey-fixed.forbid {
	z-index: 100;
	position: fixed;
	top: auto;
	bottom: 68px;
	left: 9%;
	right: 0;
	background: rgba(0, 0, 0, 0.7);
	width: 82%;
	height: 43px;
	border-radius: 25px;
	color: #ffffff;
}
.onekey-login-title.forbid {
	text-align: center;
	padding-bottom: 3px;
	font-size: 14px;
	font-weight: normal;
	line-height: 30px;
	color: white;
}
</style><style>#login_mask {
  background: #000;
  opacity: 0.5;
  filter: alpha(opacity=50);
  position: fixed;
  /*fixed好像在哪个IE上有BUG，先用用*/
  left: 0;
  top: 0;
  z-index: 999;
  height: 100%;
}

#login_iframe_container {
  position: fixed;
  width: 550px;
  height: 360px;
  z-index: 1020;
  background-color: #ffffff;
}

@media screen and (max-width: 828px) {
  #login_iframe_container {
    top: 50% !important;
    left: 50% !important;
    transform: translate(-50%, -50%);
  }
}

#login_iframe_container.new-login {
  width: 550px;
  height: 360px;
  background-image: url(//web.archive.org/web/20221028205211/https://hhy.sogoucdn.com/js/common/hhy/background_2a4a8a6.png);
}

#login_iframe_container.new-login.no-bg {
  background: #fff;
}

#login_iframe_container.new-login .login-title {
  width: 100%;
  height: 42px;
  line-height: 42px;
  text-align: center;
  font-size: 30px;
  letter-spacing: 0.19px;
  color: #ffffff;
  margin-top: 62px;
}
#login_iframe_container.new-login .forbid-title {
  width: 100%;
  height: 42px;
  line-height: 42px;
  text-align: center;
  font-size: 24px;
  letter-spacing: 0.19px;
  color: #333333;
  margin-top: 150px;
}

#login_iframe_container.new-login.no-bg .login-title {
  color: #333333;
}

#login_iframe_container.new-login .login-subtitle {
  width: 100%;
  height: 18px;
  line-height: 18px;
  font-size: 13px;
  letter-spacing: 0.08px;
  color: #ffffff;
  text-align: center;
  margin-top: 9px;
  margin-bottom: 43px;
}

#login_iframe_container.new-login.no-bg .login-subtitle {
  color: #999999;
}

#login_iframe_container.new-login .login-subtitle::before {
  content: '';
  display: inline-block;
  width: 10px;
  height: 1px;
  background-color: #ffffff;
  position: relative;
  top: -4px;
  left: -5px;
}

#login_iframe_container.new-login .login-subtitle::after {
  content: '';
  display: inline-block;
  width: 10px;
  height: 1px;
  background-color: #ffffff;
  position: relative;
  top: -4px;
  left: 5px;
}

#login_iframe_container.new-login.no-bg .login-subtitle::before {
  background-color: #999999;
}

#login_iframe_container.new-login.no-bg .login-subtitle::after {
  background-color: #999999;
}

#login_iframe_container.new-login .close-btn {
  position: absolute;
  top: 20px;
  right: 20px;
  width: 12px;
  height: 12px;
  background: url(//web.archive.org/web/20221028205211/https://hhy.sogoucdn.com/js/common/hhy/login-sprites_e3853e5.png) -59px -10px;
  background-size: 81px 91px;
  cursor: pointer;
}

#login_iframe_container.new-login .login-btn {
  width: 220px;
  height: 47px;
  border-radius: 24px;
  border: solid 1px #dddddd;
  background-color: #ffffff;
  margin: 0 auto;
  margin-top: 28px;
  position: relative;
  display: block;
}

#login_iframe_container.new-login .login-btn .login-icon {
  position: absolute;
}

#login_iframe_container.new-login .login-btn .login-text {
  width: 61px;
  height: 47px;
  line-height: 47px;
  vertical-align: middle;
  font-size: 15px;
  letter-spacing: 0.1px;
  color: #666666;
  position: absolute;
  right: 62px;
}

#login_iframe_container.new-login .login-btn.qq-btn .login-icon {
  width: 22px;
  height: 27px;
  top: 10px;
  left: 67px;
  background: url(//web.archive.org/web/20221028205211/https://hhy.sogoucdn.com/js/common/hhy/login-sprites_e3853e5.png) -10px -54px;
  background-size: 81px 91px;
}

#login_iframe_container.new-login .login-btn.qq-btn .login-text {
  right: 59px;
}

#login_iframe_container.new-login .login-btn.wechat-btn .login-icon {
  width: 29px;
  height: 24px;
  top: 12px;
  left: 62px;
  background: url(//web.archive.org/web/20221028205211/https://hhy.sogoucdn.com/js/common/hhy/login-sprites_e3853e5.png) -10px -10px;
  background-size: 81px 91px;
}</style><style>/* -- container -- */
.rodal,
.rodal-mask {
    top: 0;
    left: 0;
    width: 100%;
    height: 100%;
    z-index: 100;
}

.rodal {
    position: fixed;
}

/* -- mask -- */
.rodal-mask {
    position: fixed;
    background: rgba(0, 0, 0, .5);
}

/* -- dialog -- */
.rodal-dialog {
    position: absolute;
    z-index: 101;
    background: #fff;
    border-radius: 3px;
    box-shadow: 0 1px 3px rgba(0, 0, 0, .2);
}

.rodal-center {
    top: 50%;
    transform: translateY(-50%);
    left: 0;
    right: 0;
    margin: 0 auto;
}

.rodal-bottom {
    left: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

.rodal-top {
    left: 0;
    right: 0;
    top: 0;
    margin: auto;
}

.rodal-left {
    top: 0;
    left: 0;
    bottom: 0;
    margin: auto;
}

.rodal-right {
    top: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

/* -- close button -- */
.rodal-close {
    position: absolute;
    cursor: pointer;
    top: 16px;
    right: 16px;
    width: 16px;
    height: 16px;
}

.rodal-close:before,
.rodal-close:after {
    position: absolute;
    content: '';
    height: 2px;
    width: 100%;
    top: 50%;
    left: 0;
    margin-top: -1px;
    background: #999;
    border-radius: 100%;
    -webkit-transition: background .2s;
    transition: background .2s;
}

.rodal-close:before {
    -webkit-transform: rotate(45deg);
    transform: rotate(45deg);
}

.rodal-close:after {
    -webkit-transform: rotate(-45deg);
    transform: rotate(-45deg);
}

.rodal-close:hover:before,
.rodal-close:hover:after {
    background: #333;
}

/* -- fade -- */
/* @-webkit-keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

@keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

.rodal-fade-enter {
    -webkit-animation: rodal-fade-enter both ease-in;
    animation: rodal-fade-enter both ease-in;
} */

@-webkit-keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

@keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

.rodal-fade-leave {
    -webkit-animation: rodal-fade-leave both ease-out;
    animation: rodal-fade-leave both ease-out;
}

/* -- zoom -- */
@-webkit-keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-enter {
    -webkit-animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-leave {
    -webkit-animation: rodal-zoom-leave both;
    animation: rodal-zoom-leave both;
}

/* -- slideDown -- */
@-webkit-keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-enter {
    -webkit-animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-leave {
    -webkit-animation: rodal-slideDown-leave both;
    animation: rodal-slideDown-leave both;
}

/* -- slideLeft -- */
@-webkit-keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-enter {
    -webkit-animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-leave {
    -webkit-animation: rodal-slideLeft-leave both;
    animation: rodal-slideLeft-leave both;
}

/* -- slideRight -- */
@-webkit-keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-enter {
    -webkit-animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-leave {
    -webkit-animation: rodal-slideRight-leave both;
    animation: rodal-slideRight-leave both;
}

/* -- slideUp -- */
@-webkit-keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-enter {
    -webkit-animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
    animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
}

@-webkit-keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-leave {
    -webkit-animation: rodal-slideUp-leave both;
    animation: rodal-slideUp-leave both;
}

/* -- flip -- */
@-webkit-keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

@keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

.rodal-flip-enter {
    -webkit-animation: rodal-flip-enter both ease-in;
    animation: rodal-flip-enter both ease-in;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

@-webkit-keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

@keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

.rodal-flip-leave {
    -webkit-animation: rodal-flip-leave both;
    animation: rodal-flip-leave both;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

/* -- rotate -- */
@-webkit-keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-enter {
    -webkit-animation: rodal-rotate-enter both;
    animation: rodal-rotate-enter both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

@-webkit-keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-leave {
    -webkit-animation: rodal-rotate-leave both;
    animation: rodal-rotate-leave both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

/* -- door -- */
@-webkit-keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

@keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

.rodal-door-enter {
    -webkit-animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

@keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

.rodal-door-leave {
    -webkit-animation: rodal-door-leave both;
    animation: rodal-door-leave both;
}</style><style>/* -- container -- */
.rodal,
.rodal-mask {
    top: 0;
    left: 0;
    width: 100%;
    height: 100%;
    z-index: 100;
}

.rodal {
    position: fixed;
}

/* -- mask -- */
.rodal-mask {
    position: fixed;
    background: rgba(0, 0, 0, .5);
}

/* -- dialog -- */
.rodal-dialog {
    position: absolute;
    z-index: 101;
    background: #fff;
    border-radius: 3px;
    box-shadow: 0 1px 3px rgba(0, 0, 0, .2);
}

.rodal-center {
    top: 50%;
    transform: translateY(-50%);
    left: 0;
    right: 0;
    margin: 0 auto;
}

.rodal-bottom {
    left: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

.rodal-top {
    left: 0;
    right: 0;
    top: 0;
    margin: auto;
}

.rodal-left {
    top: 0;
    left: 0;
    bottom: 0;
    margin: auto;
}

.rodal-right {
    top: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

/* -- close button -- */
.rodal-close {
    position: absolute;
    cursor: pointer;
    top: 16px;
    right: 16px;
    width: 16px;
    height: 16px;
}

.rodal-close:before,
.rodal-close:after {
    position: absolute;
    content: '';
    height: 2px;
    width: 100%;
    top: 50%;
    left: 0;
    margin-top: -1px;
    background: #999;
    border-radius: 100%;
    -webkit-transition: background .2s;
    transition: background .2s;
}

.rodal-close:before {
    -webkit-transform: rotate(45deg);
    transform: rotate(45deg);
}

.rodal-close:after {
    -webkit-transform: rotate(-45deg);
    transform: rotate(-45deg);
}

.rodal-close:hover:before,
.rodal-close:hover:after {
    background: #333;
}

/* -- fade -- */
/* @-webkit-keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

@keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

.rodal-fade-enter {
    -webkit-animation: rodal-fade-enter both ease-in;
    animation: rodal-fade-enter both ease-in;
} */

@-webkit-keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

@keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

.rodal-fade-leave {
    -webkit-animation: rodal-fade-leave both ease-out;
    animation: rodal-fade-leave both ease-out;
}

/* -- zoom -- */
@-webkit-keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-enter {
    -webkit-animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-leave {
    -webkit-animation: rodal-zoom-leave both;
    animation: rodal-zoom-leave both;
}

/* -- slideDown -- */
@-webkit-keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-enter {
    -webkit-animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-leave {
    -webkit-animation: rodal-slideDown-leave both;
    animation: rodal-slideDown-leave both;
}

/* -- slideLeft -- */
@-webkit-keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-enter {
    -webkit-animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-leave {
    -webkit-animation: rodal-slideLeft-leave both;
    animation: rodal-slideLeft-leave both;
}

/* -- slideRight -- */
@-webkit-keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-enter {
    -webkit-animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-leave {
    -webkit-animation: rodal-slideRight-leave both;
    animation: rodal-slideRight-leave both;
}

/* -- slideUp -- */
@-webkit-keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-enter {
    -webkit-animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
    animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
}

@-webkit-keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-leave {
    -webkit-animation: rodal-slideUp-leave both;
    animation: rodal-slideUp-leave both;
}

/* -- flip -- */
@-webkit-keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

@keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

.rodal-flip-enter {
    -webkit-animation: rodal-flip-enter both ease-in;
    animation: rodal-flip-enter both ease-in;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

@-webkit-keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

@keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

.rodal-flip-leave {
    -webkit-animation: rodal-flip-leave both;
    animation: rodal-flip-leave both;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

/* -- rotate -- */
@-webkit-keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-enter {
    -webkit-animation: rodal-rotate-enter both;
    animation: rodal-rotate-enter both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

@-webkit-keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-leave {
    -webkit-animation: rodal-rotate-leave both;
    animation: rodal-rotate-leave both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

/* -- door -- */
@-webkit-keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

@keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

.rodal-door-enter {
    -webkit-animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

@keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

.rodal-door-leave {
    -webkit-animation: rodal-door-leave both;
    animation: rodal-door-leave both;
}</style>
<!--百度统计-->
<script>
   var _hmt = _hmt || [];
   (function() {
      var hm = document.createElement("script");
      hm.src = "https://hm.baidu.com/hm.js?3c7614be3026469d5a60f41ab30b5082";
      var s = document.getElementsByTagName("script")[0]; 
      s.parentNode.insertBefore(hm, s);
      })();
</script>

	<!--百度统计-->
	<script>
		var _hmt = _hmt || [];
		(function() {
			var hm = document.createElement("script");
			hm.src = "https://hm.baidu.com/hm.js?3c7614be3026469d5a60f41ab30b5082";
			var s = document.getElementsByTagName("script")[0]; 
			s.parentNode.insertBefore(hm, s);
			})();
	</script>
</head>
<body class=""><!-- BEGIN WAYBACK TOOLBAR INSERT -->
<style type="text/css">
body {
  margin-top:0 !important;
  padding-top:0 !important;
  /*min-width:800px !important;*/
}
</style>
<script>__wm.rw(0);</script>
<div id="wm-ipp-base" lang="en" style="display: block; direction: ltr;">
</div><div id="wm-ipp-print">The Wayback Machine - https://web.archive.org/web/20221028212443/https://baike.sogou.com/kexue/d10063.htm</div>
<script type="text/javascript">//<![CDATA[
__wm.bt(675,27,25,2,"web","https://baike.sogou.com/kexue/d10063.htm","20221028212443",1996,"/_static/",["/_static/css/banner-styles.css?v=S1zqJCYt","/_static/css/iconochive.css?v=qtvMKcIJ"], false);
  __wm.rw(1);
//]]></script>
<!-- END WAYBACK TOOLBAR INSERT --><script>window._gtag=window._gtag||{};window._gtag.shouldGrayed = false;if ('a2c434e17ccf4160b0f43e77816985f1') window._gtag.traceId = 'a2c434e17ccf4160b0f43e77816985f1';if ({"illegality":true}) window.userInfo = {"illegality":true};</script><div class="topnavbox"><ul class="topnav"><li><a href="https://web.archive.org/web/20221028212443/https://www.sogou.com/web?query=">网页</a></li><li><a href="https://web.archive.org/web/20221028212443/https://weixin.sogou.com/weixin?p=75351201">微信</a></li><li><a href="https://web.archive.org/web/20221028212443/https://zhihu.sogou.com/zhihu?p=75351218">知乎</a></li><li><a href="https://web.archive.org/web/20221028212443/https://pic.sogou.com/pics?query=">图片</a></li><li><a href="https://web.archive.org/web/20221028212443/https://v.sogou.com/v?query=">视频</a></li><li><a href="https://web.archive.org/web/20221028212443/https://mingyi.sogou.com/">医疗</a></li><li class="cur"><strong>科学</strong></li><li><a href="https://web.archive.org/web/20221028212443/https://hanyu.sogou.com/">汉语</a></li><li><a href="https://web.archive.org/web/20221028212443/https://wenwen.sogou.com/">问问</a></li><li><a href="https://web.archive.org/web/20221028212443/https://www.sogou.com/docs/more.htm">更多<span class="topraquo">»</span></a></li></ul></div><div id="header"><div class="header-wrap"><a class="header-logo" href="https://web.archive.org/web/20221028212443/https://baike.sogou.com/kexue"></a><div class="header-search"><div class="querybox" id="suggBox"><form><input id="searchInput" class="query" type="text" placeholder="搜科学领域专业百科词条" name="query" autocomplete="off" value="" control-id="ControlID-1"><a href="javascript:;" class="query-search"></a></form></div></div><div class="header-rgt"><span class="btn-header-rgt btn-edit" id="editLemma">创建</span><div class="header-user no-login"></div></div></div></div><div class="fixed-placeholder" style="visibility:none"></div><div id="container" class=""><div class="content lemma-level1"><div class="detail-title" id="abstract-title"><h1>纺锤体</h1><a href="https://web.archive.org/web/20221028212443/https://baike.sogou.com/kexue/d10063.htm#!" class="detail-edit">编辑</a></div><div class="section_content" data-id="54080905291030538"><div class="text_img ed_imgfloat_right"><a class="ed_image_link" data-src="https://img04.sogoucdn.com/app/a/200698/sogou_science_3342" data-bigsrc="https://img04.sogoucdn.com/app/a/200698/sogou_science_3342?width=231&amp;height=174&amp;titlename=%E6%98%BE%E5%BE%AE%E7%85%A7%E7%89%87%E6%98%BE%E7%A4%BA%E7%9A%84%E6%98%AF%E6%9C%89%E4%B8%9D%E5%88%86%E8%A3%82%E4%B8%AD%E6%9C%9F%E7%9A%84%E7%BB%93%E6%9E%84%EF%BC%8C%E8%93%9D%E8%89%B2%E7%9A%84%E6%98%AF%E6%B5%93%E7%BC%A9%E7%9A%84%E6%9F%93%E8%89%B2%E8%B4%A8%EF%BC%8C%E7%B2%89%E7%BA%A2%E8%89%B2%E7%9A%84%E6%98%AF%E7%9D%80%E4%B8%9D%E7%B2%92%EF%BC%8C%E7%BB%BF%E8%89%B2%E7%9A%84%E6%98%AF%E5%BE%AE%E7%AE%A1%E3%80%82&amp;w=250&amp;h=186" title="点击查看大图" data-w="250" data-h="186" style="background-image:url(https://web.archive.org/web/20221028212443im_/https://img04.sogoucdn.com/app/a/200698/sogou_science_3342)" href="https://web.archive.org/web/20221028212443/https://baike.sogou.com/kexue/d10063.htm#!"></a><div class="text_img_title">显微照片显示的是有丝分裂中期的结构，蓝色的是浓缩的染色质，粉红色的是着丝粒，绿色的是微管。</div></div><div><p>纺锤体是真核细胞有丝分裂或减数分裂过程中形成的中间宽两端窄的纺锤状细胞结构，主要由大量纵向排列的微管构成。纺锤体一般产生于早前期，并在分裂末期消失。纺锤体主要元件包括极间丝、着丝点丝、星体丝及区间丝四种微管和附着在微管上的动力分子分子马达以及一系列复杂的超分子结构组成。<sup><a href="https://web.archive.org/web/20221028212443/https://baike.sogou.com/kexue/d10063.htm#quote_1" class="kx_ref">[1]</a></sup><sup><a href="https://web.archive.org/web/20221028212443/https://baike.sogou.com/kexue/d10063.htm#quote_2" class="kx_ref">[2]</a></sup></p></div></div><div id="catalog"><h2 class="title2">目录<a href="javascript:" class="detail-edit">编辑</a></h2><div class="catalog_wrap" style=""><ul class="catalog_list col2"><li><span class="order">1</span><a href="javascript:" data-level="1" data-id="14994014365286675">纺锤体结构</a></li><li class="secondary_catalog"><span>1.1 </span><a href="javascript:" data-id="14994014365286675">微管相关蛋白与纺锤体动力学</a></li><li><span class="order">2</span><a href="javascript:" data-level="1" data-id="14994014365286676">组织纺锤体</a></li><li class="secondary_catalog"><span>2.1 </span><a href="javascript:" data-id="14994014365286676"><b>中心体介导的“搜索和捕获”模型</b></a></li><li class="secondary_catalog"><span>2.2 </span><a href="javascript:" data-id="14994014365286676"><b>染色质介导的有丝分裂纺锤体的自组织</b></a></li><li><span class="order">3</span><a href="javascript:" data-level="1" data-id="14994014382063887">Ran GTP梯度法研究染色质介导的微管成核</a></li></ul><ul class="catalog_list col2"><li><span class="order">4</span><a href="javascript:" data-level="1" data-id="14994014382063888">纺锤体组装的调节</a></li><li><span class="order">5</span><a href="javascript:" data-level="1" data-id="14994014382063889">有丝分裂染色体结构</a></li><li><span class="order">6</span><a href="javascript:" data-level="1" data-id="14994014398841092">有丝分裂纺锤体检查点</a></li><li><span class="order">7</span><a href="javascript:" data-level="1" data-id="14994014398841093">染色体定向</a></li><li><span class="order">8</span><a href="javascript:" data-level="1" data-id="references">参考文献</a></li></ul></div></div><div id="paragraphs"><div><div id="par_14994014365286675"><h2 class="title">1 纺锤体结构<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p></p><div class="text_img ed_imgfloat_right">
            <a class="ed_image_link lazyLoad" data-src="https://img01.sogoucdn.com/app/a/200698/sogou_science_3343" data-bigsrc="" title="点击查看大图" href="javascript:" data-observer="true"></a>
            <div class="text_img_title">这幅图描绘的是动物细胞中典型的有丝分裂纺锤体的结构。染色体通过被称为着丝粒的一个多蛋白复合体附着在着丝粒微管上。极性微管在纺锤体的中间区域交叉，通过马达蛋白将纺锤体的两极分开。星体微管将纺锤体的两极锚定在细胞膜上。接着，微管聚合在微管组织中心成核。</div>   
        </div> <p></p><p>微管与染色体的连接是由动粒介导的。动粒能主动监测纺锤体的形成并防止过早的进入细胞分裂后期。微管聚合和解聚的动态变化驱动染色体聚集。微管解聚时会在动粒上产生张力；<sup><a href="https://web.archive.org/web/20221028212443/https://baike.sogou.com/kexue/d10063.htm#quote_3" class="kx_ref">[3]</a></sup> 姐妹动粒与相反细胞两极所产生微管的双极连接伴生了相反的张力，使染色体在细胞赤道对齐，并使它们与子细胞分离。一旦每条染色体都变成双向的，细胞分裂进入后期，连接姐妹染色单体的黏连蛋白（cohesin）被切断，允许姐妹染色单体移动到相反的两极。</p><p>细胞纺锤体包括纺锤体微管、相关蛋白(包括驱动蛋白和动力蛋白分子马达)、凝聚的染色体和根据细胞类型可能存在于纺锤极的中心体或星状体。<sup><a href="https://web.archive.org/web/20221028212443/https://baike.sogou.com/kexue/d10063.htm#quote_4" class="kx_ref">[4]</a></sup> 纺锤体的横截面是模糊的椭球体，端部呈锥形。在被称为纺锤体中间带的中间较宽部分，反向平行的微管被驱动蛋白捆绑在一起。在称为纺锤极的尖端，微管在大多数动物细胞中被中心体凝聚成核。无中心体或无星状体纺锤体顾名思义，是指分别在纺锤极体缺少中心体或星状体，它们会出现在例如大多数雌性动物的减数分裂期间。<sup><a href="https://web.archive.org/web/20221028212443/https://baike.sogou.com/kexue/d10063.htm#quote_5" class="kx_ref">[5]</a></sup> 在这种情况下，Ran GTP浓度梯度是纺锤体微管组织和组装的主要调节因素。在真菌中，纺锤体形成于嵌入核膜的纺锤极体之间，在有丝分裂期间不会崩解。</p><p></p><h3>1.1 <span>微管相关蛋白与纺锤体动力学</span></h3><p></p><p>纺锤体微管的动态延长和缩短，通过动态不稳定性的过程，在很大程度上决定有丝分裂纺锤体的形状，并促进纺锤体中段染色体的正确排列。微管结合蛋白(MAPs)在中间区和纺锤体极与微管结合，以调节它们的动态变化。γ-微管蛋白是一种特殊的微管蛋白变体，它组装成γ-TuRC的环状复合物，该复合物可以将α/β微管蛋白异二聚体聚合成微管。γ-TuRC被募集到中心体周围区域来稳定微管负端，并将它们锚定在微管组织中心附近。微管相关蛋白Augmin与γ-TURC共同作用，使新微管从现有微管分离而成核。<sup><a href="https://web.archive.org/web/20221028212443/https://baike.sogou.com/kexue/d10063.htm#quote_6" class="kx_ref">[6]</a></sup> </p><p>微管正端跟踪蛋白(+TIPs)可以保护微管的生长末端免受崩解，以促进微管与中间区的动粒结合。在海拉(HeLa) 细胞中CLIP170被认为定位于微管正端附近， <sup><a href="https://web.archive.org/web/20221028212443/https://baike.sogou.com/kexue/d10063.htm#quote_7" class="kx_ref">[7]</a></sup> 并于细胞分裂前期在动粒中积累。<sup><a href="https://web.archive.org/web/20221028212443/https://baike.sogou.com/kexue/d10063.htm#quote_8" class="kx_ref">[8]</a></sup> 尽管还不清楚CLIP170是如何识别正端，但已经表明它的同源物可以防止微管崩解并促进其生长，<sup><a href="https://web.archive.org/web/20221028212443/https://baike.sogou.com/kexue/d10063.htm#quote_9" class="kx_ref">[9]</a></sup><sup><a href="https://web.archive.org/web/20221028212443/https://baike.sogou.com/kexue/d10063.htm#quote_10" class="kx_ref">[10]</a></sup> 这表明CLIP170在稳定正端和可能介导它们与动粒的直接连接方面发挥了作用。<sup><a href="https://web.archive.org/web/20221028212443/https://baike.sogou.com/kexue/d10063.htm#quote_11" class="kx_ref">[11]</a></sup>CLIP相关蛋白，如人类的CLASP1，也被证明定位于正端和外动粒，并调节动粒微管的动态变化。果蝇、爪蟾和酵母中的CLASP同源物是纺锤体正确组装所必需的；在哺乳动物中，CLASP1和CLASP2都有助于后期纺锤体的正确组装和微管动态变化。<sup><a href="https://web.archive.org/web/20221028212443/https://baike.sogou.com/kexue/d10063.htm#quote_12" class="kx_ref">[12]</a></sup> EB1蛋白可以进一步调节正端微管聚合，EB1蛋白直接结合微管的生长端，并协调其他+TIPs的结合。<sup><a href="https://web.archive.org/web/20221028212443/https://baike.sogou.com/kexue/d10063.htm#quote_13" class="kx_ref">[13]</a></sup><sup><a href="https://web.archive.org/web/20221028212443/https://baike.sogou.com/kexue/d10063.htm#quote_14" class="kx_ref">[14]</a></sup> </p><p>与这些微管稳定蛋白的作用相反的是数量众多的微管解聚因子，微管解聚因子允许有丝分裂纺锤体的动态重塑，以促进染色体聚集和实现双极化。MAPs的驱动蛋白-13超家族包含一类具有相关微管解聚活性的正端导向马达蛋白，包括被深入研究的哺乳动物MCAK和爪蟾XKCM1。MCAK定位于动粒微管生长的尖端，在那里它可以通过破坏+TIP活性而引发微管崩解。<sup><a href="https://web.archive.org/web/20221028212443/https://baike.sogou.com/kexue/d10063.htm#quote_15" class="kx_ref">[15]</a></sup> 这些蛋白质利用ATP水解的能量诱导原丝结构的不稳定构象变化，从而导致驱动蛋白释放和微管解聚。<sup><a href="https://web.archive.org/web/20221028212443/https://baike.sogou.com/kexue/d10063.htm#quote_16" class="kx_ref">[16]</a></sup> 它们活性的丧失导致许多有丝分裂缺陷。其他微管不稳定蛋白包括Op18/stathmin和katanin，它们在有丝分裂纺锤体重塑和细胞分裂后期染色体分离中起作用。<sup><a href="https://web.archive.org/web/20221028212443/https://baike.sogou.com/kexue/d10063.htm#quote_17" class="kx_ref">[17]</a></sup> </p><p>这些MAPs的活性受到细致的调节，以在纺锤体组装过程中维持适当的微管动态，其中许多蛋白质是Aurora和Polo-like激酶底物。<sup><a href="https://web.archive.org/web/20221028212443/https://baike.sogou.com/kexue/d10063.htm#quote_17" class="kx_ref">[17]</a></sup><sup><a href="https://web.archive.org/web/20221028212443/https://baike.sogou.com/kexue/d10063.htm#quote_18" class="kx_ref">[18]</a></sup> </p></div></div><div id="par_14994014365286676"><h2 class="title">2 组织纺锤体<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p></p><div class="text_img ed_imgfloat_right">
            <a class="ed_image_link lazyLoad" data-src="https://img03.sogoucdn.com/app/a/200698/sogou_science_4522" data-bigsrc="" title="点击查看大图" href="javascript:" data-observer="true"></a>
            <div class="text_img_title">在中心体介导的“搜索和捕获”模型（左）中，中心体和染色体偶然的接触使微管成核，并且微管稳定在着丝粒上形成纺锤体。在染色质介导的“自组织”模型（右）中，微管在有丝分裂的染色质附近成核，并由马达蛋白将它们组织成一个具有两极的阵列。</div>   
        </div> <p></p><p>在一个正确形成的有丝分裂纺锤体中，双向染色体沿着细胞赤道面上排列，纺锤体微管大致垂直于染色体，它们的正端嵌入在微粒中，而负端锚定在细胞两极。为了确保准确的染色体分离和确定细胞分裂平面，这种复合体的精确定位是必需的。然而，尚不清楚纺锤体是如何组织的。两种可能的模型在领域内处于主流地位，不过它们是相互补充的，而不是相互排斥的。在搜索-捕捉模型中，纺锤体主要由中心体微管组织中心的极向分离来组织。纺锤体微管从中心体发出，并“寻找”动粒；当它们与动粒结合时，它们变得稳定并对染色体施加张力。在另一种自组装模型中，微管在压缩的染色体中进行无中心体成核。受细胞尺寸、运动蛋白与反平行微管的横向连接以及与动粒末端连接的限制，微管自主采用纺锤体状结构，染色体沿细胞赤道排列。</p><p></p><h3>2.1 <span>中心体介导的“搜索和捕获”模型</span></h3><p></p><p>在这个模型中，微管在微管组织中心成核，并经历快速生长和崩解，在细胞质中“寻找”动粒。一旦与动粒结合，它们就稳定下来，其动态变化程度也会减弱。新的单极染色体在它所附着的极体附近的空间振荡，直到来自相反极体的微管结合这个姐妹动粒。这第二次结合进一步使动粒在有丝分裂纺锤体上稳定附着。逐渐地，双向染色体被拉向细胞中心，直到微管张力在着丝粒两侧平衡；集合的染色体然后在中期板振荡，直到细胞分裂后期启动释放了姐妹染色单体的凝聚体。</p><p>在该模型中，微管组织中心定位于细胞极，它们的分离由微管聚合驱动，而纺锤体中间区反向平行纺锤体微管相对于彼此的“滑动”是由双极和正端定向驱动蛋白所介导的。<sup><a href="https://web.archive.org/web/20221028212443/https://baike.sogou.com/kexue/d10063.htm#quote_19" class="kx_ref">[19]</a></sup><sup><a href="https://web.archive.org/web/20221028212443/https://baike.sogou.com/kexue/d10063.htm#quote_20" class="kx_ref">[20]</a></sup> 这种滑动力不仅可以解释有丝分裂早期的纺锤体极分离，还可以解释有丝分裂后期的纺锤体伸长。</p><p></p><h3>2.2 <span>染色质介导的有丝分裂纺锤体的自组织</span></h3><p></p><p>与中心体在很大程度上决定有丝分裂纺锤体组织的搜索和捕获机制相反，该模型提出微管在染色体附近可以无中心粒成核并自发组装成反向平行束并采用纺锤体样结构。<sup><a href="https://web.archive.org/web/20221028212443/https://baike.sogou.com/kexue/d10063.htm#quote_21" class="kx_ref">[21]</a></sup> 希尔德（Heald）和卡森蒂（karsenti）的经典实验表明，在非洲爪蟾卵提取物中孵育的带有DNA的小珠周围可以形成有功能的有丝分裂纺锤体和细胞核，微管的双极排列在没有中心体和动粒的情况下也可以形成。<sup><a href="https://web.archive.org/web/20221028212443/https://baike.sogou.com/kexue/d10063.htm#quote_22" class="kx_ref">[22]</a></sup> 事实上，业已证明在脊椎动物细胞中，利用激光消融中心体，既不抑制纺锤体组装，也不抑制染色体分离。<sup><a href="https://web.archive.org/web/20221028212443/https://baike.sogou.com/kexue/d10063.htm#quote_23" class="kx_ref">[23]</a></sup> 在这个框架下，有丝分裂纺锤体的形状和大小是由交联马达蛋白的生物物理特性所决定的。<sup><a href="https://web.archive.org/web/20221028212443/https://baike.sogou.com/kexue/d10063.htm#quote_24" class="kx_ref">[24]</a></sup> </p></div></div><div id="par_14994014382063887"><h2 class="title">3 Ran GTP梯度法研究染色质介导的微管成核<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>小GTPase Ran(染色体凝聚调节因子1或RCC1)的鸟嘌呤核苷酸交换因子通过核心组蛋白H2A和H2B与核小体相连。<sup><a href="https://web.archive.org/web/20221028212443/https://baike.sogou.com/kexue/d10063.htm#quote_25" class="kx_ref">[25]</a></sup> 因此，在有丝分裂染色质附近产生了GTP结合的Ran的浓度梯度。在爪蟾卵提取物中，包裹着RCC1的玻璃珠会诱导微管成核和双极纺锤体形成，这表明单独Ran GTP梯度就足以进行纺锤体组装。<sup><a href="https://web.archive.org/web/20221028212443/https://baike.sogou.com/kexue/d10063.htm#quote_26" class="kx_ref">[26]</a></sup> Ran GTP梯度通过转运蛋白importin β/α触发纺锤体组装因子(SAFs)从抑制性相互作用中释放。然后，未结合的SAFs促进有丝分裂染色质周围微管成核和稳定，纺锤体双极化由微管马达蛋白组织。<sup><a href="https://web.archive.org/web/20221028212443/https://baike.sogou.com/kexue/d10063.htm#quote_27" class="kx_ref">[27]</a></sup> </p></div></div><div id="par_14994014382063888"><h2 class="title">4 纺锤体组装的调节<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>纺锤体组装主要受有丝分裂激酶催化的磷酸化事件调节。细胞周期蛋白依赖激酶复合物(CDKs)由有丝分裂细胞周期蛋白激活，其翻译在有丝分裂期间增加。CDK1(也称为CDC2)被认为是哺乳动物细胞中主要的有丝分裂激酶，由细胞周期蛋白B1激活。Aurora激酶是纺锤体正确组装和分离所必需的。<sup><a href="https://web.archive.org/web/20221028212443/https://baike.sogou.com/kexue/d10063.htm#quote_28" class="kx_ref">[28]</a></sup> Aurora A与中心体相关联，被认为调节有丝分裂的起始。Aurora B是染色体乘客复合体的一员，介导染色体-微管连接和姐妹染色单体凝聚。Polo-like激酶，又称PLK，尤其是PLK1，通过调节微管动态在纺锤体维持中具有重要作用。<sup><a href="https://web.archive.org/web/20221028212443/https://baike.sogou.com/kexue/d10063.htm#quote_29" class="kx_ref">[29]</a></sup> </p></div></div><div id="par_14994014382063889"><h2 class="title">5 有丝分裂染色体结构<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>在DNA复制结束时，姐妹染色单体结合在一起，形成一团杂乱的DNA和蛋白质混合物，几乎不可能平均分割进每个子细胞。为了避免这个问题，有丝分裂起始触发了已复制基因组的大规模重组。姐妹染色单体被解开并彼此分离。染色体长度也缩短了，在动物细胞中缩短了10,000倍，<sup><a href="https://web.archive.org/web/20221028212443/https://baike.sogou.com/kexue/d10063.htm#quote_30" class="kx_ref">[30]</a></sup> 这一过程称为染色体凝聚。染色体凝聚开始于有丝分裂前期，并在有丝分裂中期排列在纺锤体中间时被最大程度地压缩成棒状结构。这使有丝分裂染色体具有核型中典型“X”形，每个浓缩的姐妹染色单体通过凝聚蛋白沿其长度相连，并通常在着丝粒中心附近的连接。<sup><a href="https://web.archive.org/web/20221028212443/https://baike.sogou.com/kexue/d10063.htm#quote_30" class="kx_ref">[30]</a></sup><sup><a href="https://web.archive.org/web/20221028212443/https://baike.sogou.com/kexue/d10063.htm#quote_31" class="kx_ref">[31]</a></sup><sup><a href="https://web.archive.org/web/20221028212443/https://baike.sogou.com/kexue/d10063.htm#quote_32" class="kx_ref">[32]</a></sup> </p><p>虽然这些动态重排对于确保基因组的精确和高保真分离至关重要，但我们对有丝分裂染色体结构的理解仍然很不完整。不过，已经可以确定一小部分特定分子的功能: 拓扑异构酶II利用ATP水解来催化DNA纠缠的解除，促进姐妹染色单体的分解。<sup><a href="https://web.archive.org/web/20221028212443/https://baike.sogou.com/kexue/d10063.htm#quote_33" class="kx_ref">[33]</a></sup> 凝聚素是5-亚单位复合物，也利用三磷酸腺苷水解促进染色体凝聚。爪蟾卵提取物的实验也暗示了组蛋白H1作为连接者，是有丝分裂染色体紧密度的重要调节因子。<sup><a href="https://web.archive.org/web/20221028212443/https://baike.sogou.com/kexue/d10063.htm#quote_34" class="kx_ref">[34]</a></sup> </p></div></div><div id="par_14994014398841092"><h2 class="title">6 有丝分裂纺锤体检查点<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>纺锤体形成的完成是细胞周期中一个关键的转变点，称为纺锤体组装检查点。如果在这个检查点之前染色体没有正确地附着在有丝分裂纺锤体上，后期的开始将会延迟。<sup><a href="https://web.archive.org/web/20221028212443/https://baike.sogou.com/kexue/d10063.htm#quote_35" class="kx_ref">[35]</a></sup> 纺锤体组装检查点的故障会导致非整倍体，这可能与衰老和癌症的形成有关。<sup><a href="https://web.archive.org/web/20221028212443/https://baike.sogou.com/kexue/d10063.htm#quote_36" class="kx_ref">[36]</a></sup> </p></div></div><div id="par_14994014398841093"><h2 class="title">7 染色体定向<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p></p><div class="text_img ed_imgfloat_right">
            <a class="ed_image_link lazyLoad" data-src="https://img02.sogoucdn.com/app/a/200698/sogou_science_4523" data-bigsrc="" title="点击查看大图" href="javascript:" data-observer="true"></a>
            <div class="text_img_title">这个图片显示的是被上皮组织围绕的一个上皮细胞正在分裂的卡通图画。纺锤体结构在细胞内旋转。旋转式星体微管像三细胞结点（TCJ）方向拉动的结果，信号中心就位于三个细胞接触的区域。</div>   
        </div> <p></p><p>细胞分裂定向对组织结构、细胞命运和形态发生具有重要意义。细胞倾向于根据所谓的赫蒂威格（Hertwig）法则沿其长轴分裂。细胞分裂的轴由纺锤体的方向决定。细胞沿着连接纺锤体的两个中心体的线分裂。纺锤体形成后，它会在细胞内部经历旋转过程。起源于中心体的星形微管到达细胞膜，在那里它们被拉向特定的皮质线。<sup><a href="https://web.archive.org/web/20221028212443/https://baike.sogou.com/kexue/d10063.htm#quote_37" class="kx_ref">[37]</a></sup> 在体外，皮质线的分布是由粘附模式建立的。皮质线由位于细胞顶点的三细胞结点的定位决定。<sup><a href="https://web.archive.org/web/20221028212443/https://baike.sogou.com/kexue/d10063.htm#quote_38" class="kx_ref">[38]</a></sup> 皮质线的空间分布决定最终纺锤体方向和随后细胞分裂的方向。</p></div></div></div></div><div id="references"><h2 class="title" id="par_references">参考文献</h2><ul class="references"><li id="quote_1"><span class="references-num">[1]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">C. E. Walczak; R. Heald (2008). "Mechanisms of Mitotic Spindle Assembly and Function". International Review of Cytology. 265: 111–158. doi:10.1016/s0074-7696(07)65003-7..</span></p></li><li id="quote_2"><span class="references-num">[2]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Helmke KJ, Heald R, Wilbur JD (2013). "Interplay between spindle architecture and function". Int. Rev. Cell Mol. Biol. 306: 83–125. doi:10.1016/B978-0-12-407694-5.00003-1. PMID 24016524..</span></p></li><li id="quote_3"><span class="references-num">[3]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">E. Nogales; V. H. Ramey (1 November 2009). "Structure-function insights into the yeast Dam1 kinetochore complex". J Cell Sci. 122: 3831–3836. doi:10.1242/jcs.004689. PMC 2773187..</span></p></li><li id="quote_4"><span class="references-num">[4]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Campbell, Neil A.; Jane B. Reece (2005). Biology, 7th Edition. San Francisco: Benjamin Cummings. pp. 221–224. ISBN 0-8053-7171-0..</span></p></li><li id="quote_5"><span class="references-num">[5]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Manandhar Gf; Schatten H; Sutovsky P (2005). "Centrosome reduction during gametogenesis and its significance". Biol. Reprod. 72 (1): 2–13. doi:10.1095/biolreprod.104.031245. PMID 15385423..</span></p></li><li id="quote_6"><span class="references-num">[6]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Petry S, et al. (2013). "Branching microtubule nucleation in Xenopus egg extracts mediated by augmin and TPX2". Cell. 152: 768–777. doi:10.1016/j.cell.2012.12.044. PMC 3680348. PMID 23415226..</span></p></li><li id="quote_7"><span class="references-num">[7]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">J.E. Rickard; T.E. Kreis (1990). "Identification of a novel nucleotide-sensitive microtubule-binding protein in HeLa cells". J Cell Biol. 110 (5): 1623–1633. doi:10.1083/jcb.110.5.1623. PMC 2200191. PMID 1970824..</span></p></li><li id="quote_8"><span class="references-num">[8]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">D. Dujardin; U.I. Wacker; A. Moreau; T.A. Schroer; J.E. Rickard; J.R. DeMey (1998). "Evidence for a role of CLIP-170 in the establishment of metaphase chromosome alignment". J Cell Biol. 141 (4): 849–862. doi:10.1083/jcb.141.4.849. PMC 2132766. PMID 9585405..</span></p></li><li id="quote_9"><span class="references-num">[9]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">D. Brunner; P. Nurse (2000). "CLIP-170-like tip1p spatially organizes microtubular dynamics in fission yeast". Cell. 102 (5): 695–704. doi:10.1016/S0092-8674(00)00091-X. PMID 11007487..</span></p></li><li id="quote_10"><span class="references-num">[10]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Y.A. Komarova; A.S. Kojima; et al. (2002). "Cytoplasmic linker proteins promote microtubule rescue in vivo". J Cell Biol. 159: 589–599. doi:10.1083/jcb.200208058. PMC 2173097. PMID 12446741..</span></p></li><li id="quote_11"><span class="references-num">[11]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">S. Goldstone; C. Reyes; G. Gay; T. Courthéoux; M. Dubarry; et al. (2010). "Tip1/CLIP-170 Protein Is Required for Correct Chromosome Poleward Movement in Fission Yeast". PLoS ONE. 5: e10634. doi:10.1371/journal.pone.0010634. PMC 2869355. PMID 20498706..</span></p></li><li id="quote_12"><span class="references-num">[12]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">A.L. Pereira; A.J. Pereira; A.R.R. Maia; et al. (1 October 2006). "Mammalian CLASP1 and CLASP2 Cooperate to Ensure Mitotic Fidelity by Regulating Spindle and Kinetochore Function". Mol Biol Cell. 17: 4526–4542. doi:10.1091/mbc.E06-07-0579. PMC 1635371. PMID 16914514..</span></p></li><li id="quote_13"><span class="references-num">[13]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">A. Akhmanova; M.O. Steinmetz (April 2008). "Tracking the ends: a dynamic protein network controls the fate of microtubule tips". Nat Rev Mol Cell Biol. 9 (4): 309–322. doi:10.1038/nrm2369. PMID 18322465..</span></p></li><li id="quote_14"><span class="references-num">[14]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">J.S. Tirnauer; S. Grego; E.D. Salmon; T.J. Mitchison (1 October 2002). "EB1-microtubule interactions in Xenopus egg extracts: Role of EB1 in microtubule stabilization and mechanisms of targeting to microtubules". Mol Biol Cell. 13 (10): 3614–3626. doi:10.1091/mbc.02-04-0210. PMC 129970. PMID 12388761..</span></p></li><li id="quote_15"><span class="references-num">[15]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">M.E. Tanenbaum; R.H. Medema; A. Akhmanova (2011). "Regulation of localization and activity of the microtubule depolymerase MCAK". Bioarchitecture. 1 (2): 80–87. doi:10.4161/bioa.1.2.15807. PMC 3158623. PMID 21866268..</span></p></li><li id="quote_16"><span class="references-num">[16]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">H. Niederstrasser; H. Salehi-Had; E.C. Gan; C. Walczak; E. Nogales (2002). "XKCM1 acts on a single protofilament and requires the C terminus of tubulin". J Mol Biol. 316 (3): 817–828. doi:10.1006/jmbi.2001.5360. PMID 11866534..</span></p></li><li id="quote_17"><span class="references-num">[17]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">H. Maiato; P Sampaio; C.E. Sunkel (2004). "Microtubule-associated proteins and their essential roles during mitosis". Int Rev Cytol. 241: 53–153. doi:10.1016/S0074-7696(04)41002-X. PMID 15548419..</span></p></li><li id="quote_18"><span class="references-num">[18]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">R. Tournebize; A. Popov; K. Kinoshita; A.J. Ashford; et al. (2000). "Control of microtubule dynamics by the antagonistic activities of XMAP215 and XKCM1 in Xenopus egg extracts". Nat Cell Biol. 2: 13–19. doi:10.1038/71330. PMID 10620801..</span></p></li><li id="quote_19"><span class="references-num">[19]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">J. McIntosh; S.C. Landis (1971). "The distribution of spindle microtubules during mitosis in cultured human cells". J Cell Biol. 49 (2): 468–497. doi:10.1083/jcb.49.2.468. PMC 2108320. PMID 19866774..</span></p></li><li id="quote_20"><span class="references-num">[20]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">D.J. Sharp; K.L. McDonald; H.M. Brown; et al. (1999). "The bipolar kinesin, CLP61F, cross-links microtubules within interpolar microtubule bundles of Drosophila embryonic mitotic spindles". J Cell Biol. 144 (1): 125–138. doi:10.1083/jcb.144.1.125. PMC 2148119. PMID 9885249..</span></p></li><li id="quote_21"><span class="references-num">[21]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">M.A. Hallen; S.A. Endow (2009). "Anastral spindle assembly: a mathematical model". Biophys J. 97 (8): 2191–2201. doi:10.1016/j.bpj.2009.08.008. PMC 2764103. PMID 19843451..</span></p></li><li id="quote_22"><span class="references-num">[22]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">R. Heald; R. Tournebize; et al. (1996). "Self-organization of microtubules into bipolar spindles around artificial chromosomes in Xenopus egg extracts". Nature. 382 (6590): 420–425. doi:10.1038/382420a0. PMID 8684481..</span></p></li><li id="quote_23"><span class="references-num">[23]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">A. Khodjakov; R.W. Cole; B.R. Oakley; C.L. Rieder (2000). "Centrosome-independent mitotic spindle formation in vertebrates". Curr Biol. 10 (2): 59–67. doi:10.1016/S0960-9822(99)00276-6. PMID 10662665..</span></p></li><li id="quote_24"><span class="references-num">[24]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">K.S. Burbank; T.J. Mitchison; D.S. Fisher (2007). "Slide-and-cluster models for spindle assembly". Curr Biol. 17 (16): 1373–1383. doi:10.1016/j.cub.2007.07.058. PMID 17702580..</span></p></li><li id="quote_25"><span class="references-num">[25]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Makde R, England J, Yennawar H, Tan S (2010). "Structure of the RCC1 chromatin factor bound to the nucleosome core particle". Nature. 467 (7315): 562–566. doi:10.1038/nature09321. PMC 3168546. PMID 20739938..</span></p></li><li id="quote_26"><span class="references-num">[26]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Halpin D, Kalab P, Wang J, Weis K, Heald R (2011). "Mitotic spindle assembly around RCC1-coated beads in Xenopus egg extracts". PLoS Biol. 9 (12): e1001225. doi:10.1371/journal.pbio.1001225. PMC 3246454. PMID 22215983..</span></p></li><li id="quote_27"><span class="references-num">[27]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Fu J, Jiang Q, Zhang C (2010). "Coordination of Cell Cycle Events by Ran GTPase". Nature Education. 3 (9): 32..</span></p></li><li id="quote_28"><span class="references-num">[28]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">A.R. Barr; F. Gergely (2007). "Aurora A: The maker and breaker of spindle poles". J Cell Sci. 120: 2987–2996. doi:10.1242/jcs.013136..</span></p></li><li id="quote_29"><span class="references-num">[29]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Peters, U., J. Cherian; et al. (2006). "Probing cell-division phenotype space and Polo-like kinase function using small molecules". Nat Chem Biol. 2 (11): 618–26. doi:10.1038/nchembio826. PMID 17028580.CS1 maint: Multiple names: authors list (link).</span></p></li><li id="quote_30"><span class="references-num">[30]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Morgan DO: The Cell Cycle: Principles of Control (Primers inBiology) London: New Science Press Ltd; 2007:297. ISBN 978-0-9539181-2-6.</span></p></li><li id="quote_31"><span class="references-num">[31]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Belmont AS (2010). "Large-scale chromatin organization: The good, the surprising, and the still perplexing". Curr Opin Cell Biol. 26: 69–78. doi:10.1016/j.ceb.2013.10.002. PMC 3927141. PMID 24529248..</span></p></li><li id="quote_32"><span class="references-num">[32]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Marko, JF. The mitotic chromosome: structure and Mechanics. 2012. Genome Organization and Function in the Cell Nucleus. Wiley-VCH, Ch. 18, 449-485. doi:10.1002/9783527639991.ch18.</span></p></li><li id="quote_33"><span class="references-num">[33]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Champoux JJ (2001). "DNA TOPOISOMERASES: Structure, Function, and Mechanism". Annu Rev Biochem. 70 (1): 369–413. doi:10.1146/annurev.biochem.70.1.369. PMID 11395412..</span></p></li><li id="quote_34"><span class="references-num">[34]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Maresca TJ, Freedman BS, Heald R (2005). "Histone H1 is essential for mitotic chromosome architecture and segregation in Xenopus laevis egg extracts". J. Cell Biol. 169 (6): 859–69. doi:10.1083/jcb.200503031. PMC 2171634. PMID 15967810..</span></p></li><li id="quote_35"><span class="references-num">[35]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Raven, Peter H.; Ray F. Evert; Susan E. Eichhorn (2005). Biology of Plants, 7th Edition. New York: W.H. Freeman and Company Publishers. p. 59. ISBN 0-7167-1007-2..</span></p></li><li id="quote_36"><span class="references-num">[36]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Baker DJ, Chen J, van Deursen JM (2005). "The mitotic checkpoint in cancer and aging: what have mice taught us?". Curr. Opin. Cell Biol. 17 (6): 583–9. doi:10.1016/j.ceb.2005.09.011. PMID 16226453..</span></p></li><li id="quote_37"><span class="references-num">[37]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Thery M, Jimenez-Dalmaroni A, Racine V, Bornens M, Julicher F (2007). "Experimental and theoretical study of mitotic spindle orientation". Nature. 447: 493–6. doi:10.1038/nature05786. PMID 17495931..</span></p></li><li id="quote_38"><span class="references-num">[38]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Bosveld F, Markova O, Guirao B, Martin C, Wang Z, Pierre A, Balakireva M, Gaugue I, Ainslie A, Christophorou N, Lubensky DK, Minc N, Bellaïche Y (2016). "Epithelial tricellular junctions act as interphase cell shape sensors to orient mitosis". Nature. 530: 496–8. doi:10.1038/nature16970. PMC 5450930. PMID 26886796..</span></p></li></ul></div><div class="read-num">阅读 <!-- -->5603</div></div><div class="right-side" id="rightSide"><div class="side" id="lemma-side"><div class="side-title">版本记录</div><ul class="side-lst"><li><p class="side-lst-txt">暂无</p></li></ul><div class="user-card userCard"></div></div><div class="side"><div class="side-event"></div></div></div></div><div class="footer-box"><div id="footer"><div class="footer-logo-wrap"><div class="footer-logo"></div><div class="footer-logo-text">知识·传播·科普</div></div><div class="footer-info">本网站内容采用<a target="_blank" href="https://web.archive.org/web/20221028212443/https://creativecommons.org/licenses/by-sa/3.0/deed.zh?tdsourcetag=s_pctim_aiomsg">CC-BY-SA 3.0</a>授权</div><div class="footer-btn-wrap"><a target="_blank" href="https://web.archive.org/web/20221028212443/https://baike.sogou.com/help/#user_protocol">用户协议</a><a target="_blank" href="https://web.archive.org/web/20221028212443/http://www.sogou.com/docs/terms.htm?v=1">免责声明</a><a target="_blank" href="https://web.archive.org/web/20221028212443/http://corp.sogou.com/private.html">隐私政策</a><a target="_blank" href="https://web.archive.org/web/20221028212443/https://baike.sogou.com/kexue/intro.htm">关于我们</a></div></div></div><script>window.lemmaInfo ={"lemmaId":"10063","versionId":"54413046671532298","title":"纺锤体","subtitle":"","abstracts":{"paragraphId":"54080905291030538","title":"摘要","versionId":"54413046671532299","lemmaId":10063,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82812762,"name":"雷克斯班纳","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599728703,"comment":null,"dependVersionId":0,"contentType":2,"content":"<p>纺锤体是真核细胞有丝分裂或减数分裂过程中形成的中间宽两端窄的纺锤状细胞结构，主要由大量纵向排列的微管构成。纺锤体一般产生于早前期，并在分裂末期消失。纺锤体主要元件包括极间丝、着丝点丝、星体丝及区间丝四种微管和附着在微管上的动力分子分子马达以及一系列复杂的超分子结构组成。<sup><a href=\"#quote_1\" class=\"kx_ref\">[1]</a></sup><sup><a href=\"#quote_2\" class=\"kx_ref\">[2]</a></sup></p>","pics":[{"originalUrl":"https://web.archive.org/web/20221028212443/https://img04.sogoucdn.com/app/a/200698/sogou_science_3342?width=231&height=174&titlename=%E6%98%BE%E5%BE%AE%E7%85%A7%E7%89%87%E6%98%BE%E7%A4%BA%E7%9A%84%E6%98%AF%E6%9C%89%E4%B8%9D%E5%88%86%E8%A3%82%E4%B8%AD%E6%9C%9F%E7%9A%84%E7%BB%93%E6%9E%84%EF%BC%8C%E8%93%9D%E8%89%B2%E7%9A%84%E6%98%AF%E6%B5%93%E7%BC%A9%E7%9A%84%E6%9F%93%E8%89%B2%E8%B4%A8%EF%BC%8C%E7%B2%89%E7%BA%A2%E8%89%B2%E7%9A%84%E6%98%AF%E7%9D%80%E4%B8%9D%E7%B2%92%EF%BC%8C%E7%BB%BF%E8%89%B2%E7%9A%84%E6%98%AF%E5%BE%AE%E7%AE%A1%E3%80%82&w=250&h=186","url":"https://web.archive.org/web/20221028212443/https://img04.sogoucdn.com/app/a/200698/sogou_science_3342","rw":250,"rh":186,"title":"显微照片显示的是有丝分裂中期的结构，蓝色的是浓缩的染色质，粉红色的是着丝粒，绿色的是微管。","alt":null,"width":231,"height":174}],"card":null,"references":[],"versionCount":0},"card":{"paragraphId":"54080905291030539","title":"基本信息","versionId":"54080905274253320","lemmaId":10063,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82812762,"name":"雷克斯班纳","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599530731,"comment":null,"dependVersionId":0,"contentType":3,"content":"","pics":null,"card":{"cardItems":[{"key":"外文名","value":"Spindle Apparatus"},{"key":"所属领域","value":"生物学"},{"key":"时期","value":"有丝分裂初期到末期"},{"key":"结构","value":"形似纺锤结构"}]},"references":[],"versionCount":0},"categories":[{"id":4,"name":"生物学","parents":[]}],"creator":{"uid":76032831,"name":"凌俊杰","pic":"https://web.archive.org/web/20221028212443/https://cache.soso.com/qlogo/g?b=oidb&k=8jKiclbZtgfRn3gOYcdrayg&s=100&t=1557234458","introduction":"科研与科普，与你共赏。","educations":[{"schoolName":"北京大学","major":"","degree":"博士","universityId":2,"universityLogo":"https://web.archive.org/web/20221028212443/https://img01.sogoucdn.com/app/a/200943/ff1b854c-6011-11e9-aa79-fc4dd4f70029","majorLevel1":"理学","majorLevel2":"生物学","majorLevel3":"植物学","majorLevel1Id":1,"majorLevel2Id":189,"majorLevel3Id":211,"state":"毕业","lab":"邓兴旺实验室","researchField":"植物光信号转导"},{"schoolName":"中国农业大学","major":"","degree":"本科","universityId":18,"universityLogo":"https://web.archive.org/web/20221028212443/https://img01.sogoucdn.com/app/a/200943/d2593cfe-6011-11e9-87fe-fc4dd4f70029","majorLevel1":"农学","majorLevel2":"植物保护","majorLevel3":"植物病理学","majorLevel1Id":1358,"majorLevel2Id":1425,"majorLevel3Id":1439,"state":"毕业","lab":"孙文献","researchField":"细菌病害的分子机理"}],"jobs":[{"company":"清华大学","title":"博士后"}],"works":[{"title":"&#26680;&#31958;&#20307;","url":"https://web.archive.org/web/20221028212443/https://baike.sogou.com/kexue/d15518088841658890.htm?vid=15565917480026629"},{"title":"&#30701;&#26438;&#33740;&#32957;","url":"https://web.archive.org/web/20221028212443/https://baike.sogou.com/kexue/d38599767999144456.htm"}],"educationBrief":"北京大学","jobBrief":"博士后","role":0,"roleName":null,"title":"北京大学 · 生物学博士","professionalTitle":null,"phoneNo":null,"editable":true,"partnerId":8,"partnerIdCreateTime":1594286496,"partnerIdPoped":true},"createTime":1571038112,"editor":{"uid":76061299,"name":"刘佳俊","pic":"https://web.archive.org/web/20221028212443/https://cache.soso.com/qlogo/g?b=oidb&k=dI476bNpMicnzEicQynicV5yg&s=100&t=1556346144","introduction":null,"educations":[{"schoolName":"西安交通大学","major":"","degree":"博士","universityId":98,"universityLogo":"https://web.archive.org/web/20221028212443/https://img01.sogoucdn.com/app/a/200943/7e3f044c-6013-11e9-8a86-fc4dd4f70029","majorLevel1":null,"majorLevel2":null,"majorLevel3":null,"majorLevel1Id":0,"majorLevel2Id":0,"majorLevel3Id":0,"state":null,"lab":null,"researchField":null}],"jobs":null,"works":null,"educationBrief":"西安交通大学","jobBrief":"","role":0,"roleName":null,"title":"西安交通大学 · 博士","professionalTitle":null,"phoneNo":null,"editable":true,"partnerId":32,"partnerIdCreateTime":1594286500,"partnerIdPoped":true},"editTime":1576233128,"state":1,"versionCount":2,"upNum":3,"downNum":0,"pics":[{"originalUrl":"https://web.archive.org/web/20221028212443/https://img04.sogoucdn.com/app/a/200698/sogou_science_3342?width=231&height=174&titlename=%E6%98%BE%E5%BE%AE%E7%85%A7%E7%89%87%E6%98%BE%E7%A4%BA%E7%9A%84%E6%98%AF%E6%9C%89%E4%B8%9D%E5%88%86%E8%A3%82%E4%B8%AD%E6%9C%9F%E7%9A%84%E7%BB%93%E6%9E%84%EF%BC%8C%E8%93%9D%E8%89%B2%E7%9A%84%E6%98%AF%E6%B5%93%E7%BC%A9%E7%9A%84%E6%9F%93%E8%89%B2%E8%B4%A8%EF%BC%8C%E7%B2%89%E7%BA%A2%E8%89%B2%E7%9A%84%E6%98%AF%E7%9D%80%E4%B8%9D%E7%B2%92%EF%BC%8C%E7%BB%BF%E8%89%B2%E7%9A%84%E6%98%AF%E5%BE%AE%E7%AE%A1%E3%80%82&w=250&h=186","url":"https://web.archive.org/web/20221028212443/https://img04.sogoucdn.com/app/a/200698/sogou_science_3342","rw":250,"rh":186,"title":"显微照片显示的是有丝分裂中期的结构，蓝色的是浓缩的染色质，粉红色的是着丝粒，绿色的是微管。","alt":null,"width":231,"height":174},{"originalUrl":"https://web.archive.org/web/20221028212443/https://img01.sogoucdn.com/app/a/200698/sogou_science_3343?titlename=%E8%BF%99%E5%B9%85%E5%9B%BE%E6%8F%8F%E7%BB%98%E7%9A%84%E6%98%AF%E5%8A%A8%E7%89%A9%E7%BB%86%E8%83%9E%E4%B8%AD%E5%85%B8%E5%9E%8B%E7%9A%84%E6%9C%89%E4%B8%9D%E5%88%86%E8%A3%82%E7%BA%BA%E9%94%A4%E4%BD%93%E7%9A%84%E7%BB%93%E6%9E%84%E3%80%82%E6%9F%93%E8%89%B2%E4%BD%93%E9%80%9A%E8%BF%87%E8%A2%AB%E7%A7%B0%E4%B8%BA%E7%9D%80%E4%B8%9D%E7%B2%92%E7%9A%84%E4%B8%80%E4%B8%AA%E5%A4%9A%E8%9B%8B%E7%99%BD%E5%A4%8D%E5%90%88%E4%BD%93%E9%99%84%E7%9D%80%E5%9C%A8%E7%9D%80%E4%B8%9D%E7%B2%92%E5%BE%AE%E7%AE%A1%E4%B8%8A%E3%80%82%E6%9E%81%E6%80%A7%E5%BE%AE%E7%AE%A1%E5%9C%A8%E7%BA%BA%E9%94%A4%E4%BD%93%E7%9A%84%E4%B8%AD%E9%97%B4%E5%8C%BA%E5%9F%9F%E4%BA%A4%E5%8F%89%EF%BC%8C%E9%80%9A%E8%BF%87%E9%A9%AC%E8%BE%BE%E8%9B%8B%E7%99%BD%E5%B0%86%E7%BA%BA%E9%94%A4%E4%BD%93%E7%9A%84%E4%B8%A4%E6%9E%81%E5%88%86%E5%BC%80%E3%80%82%E6%98%9F%E4%BD%93%E5%BE%AE%E7%AE%A1%E5%B0%86%E7%BA%BA%E9%94%A4%E4%BD%93%E7%9A%84%E4%B8%A4%E6%9E%81%E9%94%9A%E5%AE%9A%E5%9C%A8%E7%BB%86%E8%83%9E%E8%86%9C%E4%B8%8A%E3%80%82%E6%8E%A5%E7%9D%80%EF%BC%8C%E5%BE%AE%E7%AE%A1%E8%81%9A%E5%90%88%E5%9C%A8%E5%BE%AE%E7%AE%A1%E7%BB%84%E7%BB%87%E4%B8%AD%E5%BF%83%E6%88%90%E6%A0%B8%E3%80%82&w=400&h=279","url":"https://web.archive.org/web/20221028212443/https://img01.sogoucdn.com/app/a/200698/sogou_science_3343","rw":400,"rh":279,"title":"这幅图描绘的是动物细胞中典型的有丝分裂纺锤体的结构。染色体通过被称为着丝粒的一个多蛋白复合体附着在着丝粒微管上。极性微管在纺锤体的中间区域交叉，通过马达蛋白将纺锤体的两极分开。星体微管将纺锤体的两极锚定在细胞膜上。接着，微管聚合在微管组织中心成核。","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221028212443/https://img03.sogoucdn.com/app/a/200698/sogou_science_4522?w=300&h=630&titlename=%E5%9C%A8%E4%B8%AD%E5%BF%83%E4%BD%93%E4%BB%8B%E5%AF%BC%E7%9A%84%E2%80%9C%E6%90%9C%E7%B4%A2%E5%92%8C%E6%8D%95%E8%8E%B7%E2%80%9D%E6%A8%A1%E5%9E%8B%EF%BC%88%E5%B7%A6%EF%BC%89%E4%B8%AD%EF%BC%8C%E4%B8%AD%E5%BF%83%E4%BD%93%E5%92%8C%E6%9F%93%E8%89%B2%E4%BD%93%E5%81%B6%E7%84%B6%E7%9A%84%E6%8E%A5%E8%A7%A6%E4%BD%BF%E5%BE%AE%E7%AE%A1%E6%88%90%E6%A0%B8%EF%BC%8C%E5%B9%B6%E4%B8%94%E5%BE%AE%E7%AE%A1%E7%A8%B3%E5%AE%9A%E5%9C%A8%E7%9D%80%E4%B8%9D%E7%B2%92%E4%B8%8A%E5%BD%A2%E6%88%90%E7%BA%BA%E9%94%A4%E4%BD%93%E3%80%82%E5%9C%A8%E6%9F%93%E8%89%B2%E8%B4%A8%E4%BB%8B%E5%AF%BC%E7%9A%84%E2%80%9C%E8%87%AA%E7%BB%84%E7%BB%87%E2%80%9D%E6%A8%A1%E5%9E%8B%EF%BC%88%E5%8F%B3%EF%BC%89%E4%B8%AD%EF%BC%8C%E5%BE%AE%E7%AE%A1%E5%9C%A8%E6%9C%89%E4%B8%9D%E5%88%86%E8%A3%82%E7%9A%84%E6%9F%93%E8%89%B2%E8%B4%A8%E9%99%84%E8%BF%91%E6%88%90%E6%A0%B8%EF%BC%8C%E5%B9%B6%E7%94%B1%E9%A9%AC%E8%BE%BE%E8%9B%8B%E7%99%BD%E5%B0%86%E5%AE%83%E4%BB%AC%E7%BB%84%E7%BB%87%E6%88%90%E4%B8%80%E4%B8%AA%E5%85%B7%E6%9C%89%E4%B8%A4%E6%9E%81%E7%9A%84%E9%98%B5%E5%88%97%E3%80%82","url":"https://web.archive.org/web/20221028212443/https://img03.sogoucdn.com/app/a/200698/sogou_science_4522","rw":300,"rh":630,"title":"在中心体介导的“搜索和捕获”模型（左）中，中心体和染色体偶然的接触使微管成核，并且微管稳定在着丝粒上形成纺锤体。在染色质介导的“自组织”模型（右）中，微管在有丝分裂的染色质附近成核，并由马达蛋白将它们组织成一个具有两极的阵列。","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221028212443/https://img02.sogoucdn.com/app/a/200698/sogou_science_4523?titlename=%E8%BF%99%E4%B8%AA%E5%9B%BE%E7%89%87%E6%98%BE%E7%A4%BA%E7%9A%84%E6%98%AF%E8%A2%AB%E4%B8%8A%E7%9A%AE%E7%BB%84%E7%BB%87%E5%9B%B4%E7%BB%95%E7%9A%84%E4%B8%80%E4%B8%AA%E4%B8%8A%E7%9A%AE%E7%BB%86%E8%83%9E%E6%AD%A3%E5%9C%A8%E5%88%86%E8%A3%82%E7%9A%84%E5%8D%A1%E9%80%9A%E5%9B%BE%E7%94%BB%E3%80%82%E7%BA%BA%E9%94%A4%E4%BD%93%E7%BB%93%E6%9E%84%E5%9C%A8%E7%BB%86%E8%83%9E%E5%86%85%E6%97%8B%E8%BD%AC%E3%80%82%E6%97%8B%E8%BD%AC%E5%BC%8F%E6%98%9F%E4%BD%93%E5%BE%AE%E7%AE%A1%E5%83%8F%E4%B8%89%E7%BB%86%E8%83%9E%E7%BB%93%E7%82%B9%EF%BC%88TCJ%EF%BC%89%E6%96%B9%E5%90%91%E6%8B%89%E5%8A%A8%E7%9A%84%E7%BB%93%E6%9E%9C%EF%BC%8C%E4%BF%A1%E5%8F%B7%E4%B8%AD%E5%BF%83%E5%B0%B1%E4%BD%8D%E4%BA%8E%E4%B8%89%E4%B8%AA%E7%BB%86%E8%83%9E%E6%8E%A5%E8%A7%A6%E7%9A%84%E5%8C%BA%E5%9F%9F%E3%80%82&w=300&h=212","url":"https://web.archive.org/web/20221028212443/https://img02.sogoucdn.com/app/a/200698/sogou_science_4523","rw":300,"rh":212,"title":"这个图片显示的是被上皮组织围绕的一个上皮细胞正在分裂的卡通图画。纺锤体结构在细胞内旋转。旋转式星体微管像三细胞结点（TCJ）方向拉动的结果，信号中心就位于三个细胞接触的区域。","alt":null,"width":0,"height":0}],"catalogs":[{"level":1,"title":"纺锤体结构","paragraphId":"14994014365286675","subCatalogs":[{"level":2,"title":"微管相关蛋白与纺锤体动力学","paragraphId":"14994014365286675","subCatalogs":null}]},{"level":1,"title":"组织纺锤体","paragraphId":"14994014365286676","subCatalogs":[{"level":2,"title":"<b>中心体介导的“搜索和捕获”模型</b>","paragraphId":"14994014365286676","subCatalogs":null},{"level":2,"title":"<b>染色质介导的有丝分裂纺锤体的自组织</b>","paragraphId":"14994014365286676","subCatalogs":null}]},{"level":1,"title":"Ran GTP梯度法研究染色质介导的微管成核","paragraphId":"14994014382063887","subCatalogs":null},{"level":1,"title":"纺锤体组装的调节","paragraphId":"14994014382063888","subCatalogs":null},{"level":1,"title":"有丝分裂染色体结构","paragraphId":"14994014382063889","subCatalogs":null},{"level":1,"title":"有丝分裂纺锤体检查点","paragraphId":"14994014398841092","subCatalogs":null},{"level":1,"title":"染色体定向","paragraphId":"14994014398841093","subCatalogs":null},{"level":1,"title":"参考文献","paragraphId":"-1","subCatalogs":null}],"paragraphs":[{"paragraphId":"14994014365286675","title":"纺锤体结构","versionId":"54080905274253321","lemmaId":10063,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82812762,"name":"雷克斯班纳","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599530731,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p><img alt=\"\" class=\"fileimage kx_img ed_imgfloat_right currentPic\" img_height=\"279\" img_width=\"400\" titlename=\"这幅图描绘的是动物细胞中典型的有丝分裂纺锤体的结构。染色体通过被称为着丝粒的一个多蛋白复合体附着在着丝粒微管上。极性微管在纺锤体的中间区域交叉，通过马达蛋白将纺锤体的两极分开。星体微管将纺锤体的两极锚定在细胞膜上。接着，微管聚合在微管组织中心成核。\" data-src=\"https://img01.sogoucdn.com/app/a/200698/sogou_science_3343\"> </p><p>微管与染色体的连接是由动粒介导的。动粒能主动监测纺锤体的形成并防止过早的进入细胞分裂后期。微管聚合和解聚的动态变化驱动染色体聚集。微管解聚时会在动粒上产生张力；<sup><a href=\"#quote_3\" class=\"kx_ref\">[3]</a></sup> 姐妹动粒与相反细胞两极所产生微管的双极连接伴生了相反的张力，使染色体在细胞赤道对齐，并使它们与子细胞分离。一旦每条染色体都变成双向的，细胞分裂进入后期，连接姐妹染色单体的黏连蛋白（cohesin）被切断，允许姐妹染色单体移动到相反的两极。</p><p>细胞纺锤体包括纺锤体微管、相关蛋白(包括驱动蛋白和动力蛋白分子马达)、凝聚的染色体和根据细胞类型可能存在于纺锤极的中心体或星状体。<sup><a href=\"#quote_4\" class=\"kx_ref\">[4]</a></sup> 纺锤体的横截面是模糊的椭球体，端部呈锥形。在被称为纺锤体中间带的中间较宽部分，反向平行的微管被驱动蛋白捆绑在一起。在称为纺锤极的尖端，微管在大多数动物细胞中被中心体凝聚成核。无中心体或无星状体纺锤体顾名思义，是指分别在纺锤极体缺少中心体或星状体，它们会出现在例如大多数雌性动物的减数分裂期间。<sup><a href=\"#quote_5\" class=\"kx_ref\">[5]</a></sup> 在这种情况下，Ran GTP浓度梯度是纺锤体微管组织和组装的主要调节因素。在真菌中，纺锤体形成于嵌入核膜的纺锤极体之间，在有丝分裂期间不会崩解。</p><p></p><h3>微管相关蛋白与纺锤体动力学</h3><p></p><p>纺锤体微管的动态延长和缩短，通过动态不稳定性的过程，在很大程度上决定有丝分裂纺锤体的形状，并促进纺锤体中段染色体的正确排列。微管结合蛋白(MAPs)在中间区和纺锤体极与微管结合，以调节它们的动态变化。γ-微管蛋白是一种特殊的微管蛋白变体，它组装成γ-TuRC的环状复合物，该复合物可以将α/β微管蛋白异二聚体聚合成微管。γ-TuRC被募集到中心体周围区域来稳定微管负端，并将它们锚定在微管组织中心附近。微管相关蛋白Augmin与γ-TURC共同作用，使新微管从现有微管分离而成核。<sup><a href=\"#quote_6\" class=\"kx_ref\">[6]</a></sup> </p><p>微管正端跟踪蛋白(+TIPs)可以保护微管的生长末端免受崩解，以促进微管与中间区的动粒结合。在海拉(HeLa) 细胞中CLIP170被认为定位于微管正端附近， <sup><a href=\"#quote_7\" class=\"kx_ref\">[7]</a></sup> 并于细胞分裂前期在动粒中积累。<sup><a href=\"#quote_8\" class=\"kx_ref\">[8]</a></sup> 尽管还不清楚CLIP170是如何识别正端，但已经表明它的同源物可以防止微管崩解并促进其生长，<sup><a href=\"#quote_9\" class=\"kx_ref\">[9]</a></sup><sup><a href=\"#quote_10\" class=\"kx_ref\">[10]</a></sup> 这表明CLIP170在稳定正端和可能介导它们与动粒的直接连接方面发挥了作用。<sup><a href=\"#quote_11\" class=\"kx_ref\">[11]</a></sup>CLIP相关蛋白，如人类的CLASP1，也被证明定位于正端和外动粒，并调节动粒微管的动态变化。果蝇、爪蟾和酵母中的CLASP同源物是纺锤体正确组装所必需的；在哺乳动物中，CLASP1和CLASP2都有助于后期纺锤体的正确组装和微管动态变化。<sup><a href=\"#quote_12\" class=\"kx_ref\">[12]</a></sup> EB1蛋白可以进一步调节正端微管聚合，EB1蛋白直接结合微管的生长端，并协调其他+TIPs的结合。<sup><a href=\"#quote_13\" class=\"kx_ref\">[13]</a></sup><sup><a href=\"#quote_14\" class=\"kx_ref\">[14]</a></sup> </p><p>与这些微管稳定蛋白的作用相反的是数量众多的微管解聚因子，微管解聚因子允许有丝分裂纺锤体的动态重塑，以促进染色体聚集和实现双极化。MAPs的驱动蛋白-13超家族包含一类具有相关微管解聚活性的正端导向马达蛋白，包括被深入研究的哺乳动物MCAK和爪蟾XKCM1。MCAK定位于动粒微管生长的尖端，在那里它可以通过破坏+TIP活性而引发微管崩解。<sup><a href=\"#quote_15\" class=\"kx_ref\">[15]</a></sup> 这些蛋白质利用ATP水解的能量诱导原丝结构的不稳定构象变化，从而导致驱动蛋白释放和微管解聚。<sup><a href=\"#quote_16\" class=\"kx_ref\">[16]</a></sup> 它们活性的丧失导致许多有丝分裂缺陷。其他微管不稳定蛋白包括Op18/stathmin和katanin，它们在有丝分裂纺锤体重塑和细胞分裂后期染色体分离中起作用。<sup><a href=\"#quote_17\" class=\"kx_ref\">[17]</a></sup> </p><p>这些MAPs的活性受到细致的调节，以在纺锤体组装过程中维持适当的微管动态，其中许多蛋白质是Aurora和Polo-like激酶底物。<sup><a href=\"#quote_17\" class=\"kx_ref\">[17]</a></sup><sup><a href=\"#quote_18\" class=\"kx_ref\">[18]</a></sup> </p>","pics":[{"originalUrl":"https://web.archive.org/web/20221028212443/https://img01.sogoucdn.com/app/a/200698/sogou_science_3343?titlename=%E8%BF%99%E5%B9%85%E5%9B%BE%E6%8F%8F%E7%BB%98%E7%9A%84%E6%98%AF%E5%8A%A8%E7%89%A9%E7%BB%86%E8%83%9E%E4%B8%AD%E5%85%B8%E5%9E%8B%E7%9A%84%E6%9C%89%E4%B8%9D%E5%88%86%E8%A3%82%E7%BA%BA%E9%94%A4%E4%BD%93%E7%9A%84%E7%BB%93%E6%9E%84%E3%80%82%E6%9F%93%E8%89%B2%E4%BD%93%E9%80%9A%E8%BF%87%E8%A2%AB%E7%A7%B0%E4%B8%BA%E7%9D%80%E4%B8%9D%E7%B2%92%E7%9A%84%E4%B8%80%E4%B8%AA%E5%A4%9A%E8%9B%8B%E7%99%BD%E5%A4%8D%E5%90%88%E4%BD%93%E9%99%84%E7%9D%80%E5%9C%A8%E7%9D%80%E4%B8%9D%E7%B2%92%E5%BE%AE%E7%AE%A1%E4%B8%8A%E3%80%82%E6%9E%81%E6%80%A7%E5%BE%AE%E7%AE%A1%E5%9C%A8%E7%BA%BA%E9%94%A4%E4%BD%93%E7%9A%84%E4%B8%AD%E9%97%B4%E5%8C%BA%E5%9F%9F%E4%BA%A4%E5%8F%89%EF%BC%8C%E9%80%9A%E8%BF%87%E9%A9%AC%E8%BE%BE%E8%9B%8B%E7%99%BD%E5%B0%86%E7%BA%BA%E9%94%A4%E4%BD%93%E7%9A%84%E4%B8%A4%E6%9E%81%E5%88%86%E5%BC%80%E3%80%82%E6%98%9F%E4%BD%93%E5%BE%AE%E7%AE%A1%E5%B0%86%E7%BA%BA%E9%94%A4%E4%BD%93%E7%9A%84%E4%B8%A4%E6%9E%81%E9%94%9A%E5%AE%9A%E5%9C%A8%E7%BB%86%E8%83%9E%E8%86%9C%E4%B8%8A%E3%80%82%E6%8E%A5%E7%9D%80%EF%BC%8C%E5%BE%AE%E7%AE%A1%E8%81%9A%E5%90%88%E5%9C%A8%E5%BE%AE%E7%AE%A1%E7%BB%84%E7%BB%87%E4%B8%AD%E5%BF%83%E6%88%90%E6%A0%B8%E3%80%82&w=400&h=279","url":"https://web.archive.org/web/20221028212443/https://img01.sogoucdn.com/app/a/200698/sogou_science_3343","rw":400,"rh":279,"title":"这幅图描绘的是动物细胞中典型的有丝分裂纺锤体的结构。染色体通过被称为着丝粒的一个多蛋白复合体附着在着丝粒微管上。极性微管在纺锤体的中间区域交叉，通过马达蛋白将纺锤体的两极分开。星体微管将纺锤体的两极锚定在细胞膜上。接着，微管聚合在微管组织中心成核。","alt":null,"width":0,"height":0}],"card":null,"references":[],"versionCount":0},{"paragraphId":"14994014365286676","title":"组织纺锤体","versionId":"54080905274253322","lemmaId":10063,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82812762,"name":"雷克斯班纳","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599530731,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p><img alt=\"\" class=\"fileimage kx_img ed_imgfloat_right\" img_height=\"630\" img_width=\"300\" titlename=\"在中心体介导的“搜索和捕获”模型（左）中，中心体和染色体偶然的接触使微管成核，并且微管稳定在着丝粒上形成纺锤体。在染色质介导的“自组织”模型（右）中，微管在有丝分裂的染色质附近成核，并由马达蛋白将它们组织成一个具有两极的阵列。\" data-src=\"https://img03.sogoucdn.com/app/a/200698/sogou_science_4522\"> </p><p>在一个正确形成的有丝分裂纺锤体中，双向染色体沿着细胞赤道面上排列，纺锤体微管大致垂直于染色体，它们的正端嵌入在微粒中，而负端锚定在细胞两极。为了确保准确的染色体分离和确定细胞分裂平面，这种复合体的精确定位是必需的。然而，尚不清楚纺锤体是如何组织的。两种可能的模型在领域内处于主流地位，不过它们是相互补充的，而不是相互排斥的。在搜索-捕捉模型中，纺锤体主要由中心体微管组织中心的极向分离来组织。纺锤体微管从中心体发出，并“寻找”动粒；当它们与动粒结合时，它们变得稳定并对染色体施加张力。在另一种自组装模型中，微管在压缩的染色体中进行无中心体成核。受细胞尺寸、运动蛋白与反平行微管的横向连接以及与动粒末端连接的限制，微管自主采用纺锤体状结构，染色体沿细胞赤道排列。</p><p></p><h3><b>中心体介导的“搜索和捕获”模型</b></h3><p></p><p>在这个模型中，微管在微管组织中心成核，并经历快速生长和崩解，在细胞质中“寻找”动粒。一旦与动粒结合，它们就稳定下来，其动态变化程度也会减弱。新的单极染色体在它所附着的极体附近的空间振荡，直到来自相反极体的微管结合这个姐妹动粒。这第二次结合进一步使动粒在有丝分裂纺锤体上稳定附着。逐渐地，双向染色体被拉向细胞中心，直到微管张力在着丝粒两侧平衡；集合的染色体然后在中期板振荡，直到细胞分裂后期启动释放了姐妹染色单体的凝聚体。</p><p>在该模型中，微管组织中心定位于细胞极，它们的分离由微管聚合驱动，而纺锤体中间区反向平行纺锤体微管相对于彼此的“滑动”是由双极和正端定向驱动蛋白所介导的。<sup><a href=\"#quote_19\" class=\"kx_ref\">[19]</a></sup><sup><a href=\"#quote_20\" class=\"kx_ref\">[20]</a></sup> 这种滑动力不仅可以解释有丝分裂早期的纺锤体极分离，还可以解释有丝分裂后期的纺锤体伸长。</p><p></p><h3><b>染色质介导的有丝分裂纺锤体的自组织</b></h3><p></p><p>与中心体在很大程度上决定有丝分裂纺锤体组织的搜索和捕获机制相反，该模型提出微管在染色体附近可以无中心粒成核并自发组装成反向平行束并采用纺锤体样结构。<sup><a href=\"#quote_21\" class=\"kx_ref\">[21]</a></sup> 希尔德（Heald）和卡森蒂（karsenti）的经典实验表明，在非洲爪蟾卵提取物中孵育的带有DNA的小珠周围可以形成有功能的有丝分裂纺锤体和细胞核，微管的双极排列在没有中心体和动粒的情况下也可以形成。<sup><a href=\"#quote_22\" class=\"kx_ref\">[22]</a></sup> 事实上，业已证明在脊椎动物细胞中，利用激光消融中心体，既不抑制纺锤体组装，也不抑制染色体分离。<sup><a href=\"#quote_23\" class=\"kx_ref\">[23]</a></sup> 在这个框架下，有丝分裂纺锤体的形状和大小是由交联马达蛋白的生物物理特性所决定的。<sup><a href=\"#quote_24\" class=\"kx_ref\">[24]</a></sup> </p>","pics":[{"originalUrl":"https://web.archive.org/web/20221028212443/https://img03.sogoucdn.com/app/a/200698/sogou_science_4522?w=300&h=630&titlename=%E5%9C%A8%E4%B8%AD%E5%BF%83%E4%BD%93%E4%BB%8B%E5%AF%BC%E7%9A%84%E2%80%9C%E6%90%9C%E7%B4%A2%E5%92%8C%E6%8D%95%E8%8E%B7%E2%80%9D%E6%A8%A1%E5%9E%8B%EF%BC%88%E5%B7%A6%EF%BC%89%E4%B8%AD%EF%BC%8C%E4%B8%AD%E5%BF%83%E4%BD%93%E5%92%8C%E6%9F%93%E8%89%B2%E4%BD%93%E5%81%B6%E7%84%B6%E7%9A%84%E6%8E%A5%E8%A7%A6%E4%BD%BF%E5%BE%AE%E7%AE%A1%E6%88%90%E6%A0%B8%EF%BC%8C%E5%B9%B6%E4%B8%94%E5%BE%AE%E7%AE%A1%E7%A8%B3%E5%AE%9A%E5%9C%A8%E7%9D%80%E4%B8%9D%E7%B2%92%E4%B8%8A%E5%BD%A2%E6%88%90%E7%BA%BA%E9%94%A4%E4%BD%93%E3%80%82%E5%9C%A8%E6%9F%93%E8%89%B2%E8%B4%A8%E4%BB%8B%E5%AF%BC%E7%9A%84%E2%80%9C%E8%87%AA%E7%BB%84%E7%BB%87%E2%80%9D%E6%A8%A1%E5%9E%8B%EF%BC%88%E5%8F%B3%EF%BC%89%E4%B8%AD%EF%BC%8C%E5%BE%AE%E7%AE%A1%E5%9C%A8%E6%9C%89%E4%B8%9D%E5%88%86%E8%A3%82%E7%9A%84%E6%9F%93%E8%89%B2%E8%B4%A8%E9%99%84%E8%BF%91%E6%88%90%E6%A0%B8%EF%BC%8C%E5%B9%B6%E7%94%B1%E9%A9%AC%E8%BE%BE%E8%9B%8B%E7%99%BD%E5%B0%86%E5%AE%83%E4%BB%AC%E7%BB%84%E7%BB%87%E6%88%90%E4%B8%80%E4%B8%AA%E5%85%B7%E6%9C%89%E4%B8%A4%E6%9E%81%E7%9A%84%E9%98%B5%E5%88%97%E3%80%82","url":"https://web.archive.org/web/20221028212443/https://img03.sogoucdn.com/app/a/200698/sogou_science_4522","rw":300,"rh":630,"title":"在中心体介导的“搜索和捕获”模型（左）中，中心体和染色体偶然的接触使微管成核，并且微管稳定在着丝粒上形成纺锤体。在染色质介导的“自组织”模型（右）中，微管在有丝分裂的染色质附近成核，并由马达蛋白将它们组织成一个具有两极的阵列。","alt":null,"width":0,"height":0}],"card":null,"references":[],"versionCount":0},{"paragraphId":"14994014382063887","title":"Ran GTP梯度法研究染色质介导的微管成核","versionId":"54413046671532300","lemmaId":10063,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82812762,"name":"雷克斯班纳","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599728703,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>小GTPase Ran(染色体凝聚调节因子1或RCC1)的鸟嘌呤核苷酸交换因子通过核心组蛋白H2A和H2B与核小体相连。<sup><a href=\"#quote_25\" class=\"kx_ref\">[25]</a></sup> 因此，在有丝分裂染色质附近产生了GTP结合的Ran的浓度梯度。在爪蟾卵提取物中，包裹着RCC1的玻璃珠会诱导微管成核和双极纺锤体形成，这表明单独Ran GTP梯度就足以进行纺锤体组装。<sup><a href=\"#quote_26\" class=\"kx_ref\">[26]</a></sup> Ran GTP梯度通过转运蛋白importin β/α触发纺锤体组装因子(SAFs)从抑制性相互作用中释放。然后，未结合的SAFs促进有丝分裂染色质周围微管成核和稳定，纺锤体双极化由微管马达蛋白组织。<sup><a href=\"#quote_27\" class=\"kx_ref\">[27]</a></sup> </p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14994014382063888","title":"纺锤体组装的调节","versionId":"54413046671532301","lemmaId":10063,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82812762,"name":"雷克斯班纳","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599728703,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>纺锤体组装主要受有丝分裂激酶催化的磷酸化事件调节。细胞周期蛋白依赖激酶复合物(CDKs)由有丝分裂细胞周期蛋白激活，其翻译在有丝分裂期间增加。CDK1(也称为CDC2)被认为是哺乳动物细胞中主要的有丝分裂激酶，由细胞周期蛋白B1激活。Aurora激酶是纺锤体正确组装和分离所必需的。<sup><a href=\"#quote_28\" class=\"kx_ref\">[28]</a></sup> Aurora A与中心体相关联，被认为调节有丝分裂的起始。Aurora B是染色体乘客复合体的一员，介导染色体-微管连接和姐妹染色单体凝聚。Polo-like激酶，又称PLK，尤其是PLK1，通过调节微管动态在纺锤体维持中具有重要作用。<sup><a href=\"#quote_29\" class=\"kx_ref\">[29]</a></sup> </p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14994014382063889","title":"有丝分裂染色体结构","versionId":"54413046671532302","lemmaId":10063,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82812762,"name":"雷克斯班纳","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599728703,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>在DNA复制结束时，姐妹染色单体结合在一起，形成一团杂乱的DNA和蛋白质混合物，几乎不可能平均分割进每个子细胞。为了避免这个问题，有丝分裂起始触发了已复制基因组的大规模重组。姐妹染色单体被解开并彼此分离。染色体长度也缩短了，在动物细胞中缩短了10,000倍，<sup><a href=\"#quote_30\" class=\"kx_ref\">[30]</a></sup> 这一过程称为染色体凝聚。染色体凝聚开始于有丝分裂前期，并在有丝分裂中期排列在纺锤体中间时被最大程度地压缩成棒状结构。这使有丝分裂染色体具有核型中典型“X”形，每个浓缩的姐妹染色单体通过凝聚蛋白沿其长度相连，并通常在着丝粒中心附近的连接。<sup><a href=\"#quote_30\" class=\"kx_ref\">[30]</a></sup><sup><a href=\"#quote_31\" class=\"kx_ref\">[31]</a></sup><sup><a href=\"#quote_32\" class=\"kx_ref\">[32]</a></sup> </p><p>虽然这些动态重排对于确保基因组的精确和高保真分离至关重要，但我们对有丝分裂染色体结构的理解仍然很不完整。不过，已经可以确定一小部分特定分子的功能: 拓扑异构酶II利用ATP水解来催化DNA纠缠的解除，促进姐妹染色单体的分解。<sup><a href=\"#quote_33\" class=\"kx_ref\">[33]</a></sup> 凝聚素是5-亚单位复合物，也利用三磷酸腺苷水解促进染色体凝聚。爪蟾卵提取物的实验也暗示了组蛋白H1作为连接者，是有丝分裂染色体紧密度的重要调节因子。<sup><a href=\"#quote_34\" class=\"kx_ref\">[34]</a></sup> </p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14994014398841092","title":"有丝分裂纺锤体检查点","versionId":"54413046671532303","lemmaId":10063,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82812762,"name":"雷克斯班纳","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599728703,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>纺锤体形成的完成是细胞周期中一个关键的转变点，称为纺锤体组装检查点。如果在这个检查点之前染色体没有正确地附着在有丝分裂纺锤体上，后期的开始将会延迟。<sup><a href=\"#quote_35\" class=\"kx_ref\">[35]</a></sup> 纺锤体组装检查点的故障会导致非整倍体，这可能与衰老和癌症的形成有关。<sup><a href=\"#quote_36\" class=\"kx_ref\">[36]</a></sup> </p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14994014398841093","title":"染色体定向","versionId":"54080905274253327","lemmaId":10063,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82812762,"name":"雷克斯班纳","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599530731,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p><img alt=\"\" class=\"fileimage kx_img ed_imgfloat_right\" img_height=\"212\" img_width=\"300\" titlename=\"这个图片显示的是被上皮组织围绕的一个上皮细胞正在分裂的卡通图画。纺锤体结构在细胞内旋转。旋转式星体微管像三细胞结点（TCJ）方向拉动的结果，信号中心就位于三个细胞接触的区域。\" data-src=\"https://img02.sogoucdn.com/app/a/200698/sogou_science_4523\"> </p><p>细胞分裂定向对组织结构、细胞命运和形态发生具有重要意义。细胞倾向于根据所谓的赫蒂威格（Hertwig）法则沿其长轴分裂。细胞分裂的轴由纺锤体的方向决定。细胞沿着连接纺锤体的两个中心体的线分裂。纺锤体形成后，它会在细胞内部经历旋转过程。起源于中心体的星形微管到达细胞膜，在那里它们被拉向特定的皮质线。<sup><a href=\"#quote_37\" class=\"kx_ref\">[37]</a></sup> 在体外，皮质线的分布是由粘附模式建立的。皮质线由位于细胞顶点的三细胞结点的定位决定。<sup><a href=\"#quote_38\" class=\"kx_ref\">[38]</a></sup> 皮质线的空间分布决定最终纺锤体方向和随后细胞分裂的方向。</p>","pics":[{"originalUrl":"https://web.archive.org/web/20221028212443/https://img02.sogoucdn.com/app/a/200698/sogou_science_4523?titlename=%E8%BF%99%E4%B8%AA%E5%9B%BE%E7%89%87%E6%98%BE%E7%A4%BA%E7%9A%84%E6%98%AF%E8%A2%AB%E4%B8%8A%E7%9A%AE%E7%BB%84%E7%BB%87%E5%9B%B4%E7%BB%95%E7%9A%84%E4%B8%80%E4%B8%AA%E4%B8%8A%E7%9A%AE%E7%BB%86%E8%83%9E%E6%AD%A3%E5%9C%A8%E5%88%86%E8%A3%82%E7%9A%84%E5%8D%A1%E9%80%9A%E5%9B%BE%E7%94%BB%E3%80%82%E7%BA%BA%E9%94%A4%E4%BD%93%E7%BB%93%E6%9E%84%E5%9C%A8%E7%BB%86%E8%83%9E%E5%86%85%E6%97%8B%E8%BD%AC%E3%80%82%E6%97%8B%E8%BD%AC%E5%BC%8F%E6%98%9F%E4%BD%93%E5%BE%AE%E7%AE%A1%E5%83%8F%E4%B8%89%E7%BB%86%E8%83%9E%E7%BB%93%E7%82%B9%EF%BC%88TCJ%EF%BC%89%E6%96%B9%E5%90%91%E6%8B%89%E5%8A%A8%E7%9A%84%E7%BB%93%E6%9E%9C%EF%BC%8C%E4%BF%A1%E5%8F%B7%E4%B8%AD%E5%BF%83%E5%B0%B1%E4%BD%8D%E4%BA%8E%E4%B8%89%E4%B8%AA%E7%BB%86%E8%83%9E%E6%8E%A5%E8%A7%A6%E7%9A%84%E5%8C%BA%E5%9F%9F%E3%80%82&w=300&h=212","url":"https://web.archive.org/web/20221028212443/https://img02.sogoucdn.com/app/a/200698/sogou_science_4523","rw":300,"rh":212,"title":"这个图片显示的是被上皮组织围绕的一个上皮细胞正在分裂的卡通图画。纺锤体结构在细胞内旋转。旋转式星体微管像三细胞结点（TCJ）方向拉动的结果，信号中心就位于三个细胞接触的区域。","alt":null,"width":0,"height":0}],"card":null,"references":[],"versionCount":0}],"references":[{"id":1,"type":"book","title":"C. E. Walczak; R. Heald (2008). \"Mechanisms of Mitotic Spindle Assembly and Function\". International Review of Cytology. 265: 111–158. doi:10.1016/s0074-7696(07)65003-7.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":2,"type":"book","title":"Helmke KJ, Heald R, Wilbur JD (2013). \"Interplay between spindle architecture and function\". Int. Rev. Cell Mol. Biol. 306: 83–125. doi:10.1016/B978-0-12-407694-5.00003-1. PMID 24016524.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":3,"type":"book","title":"E. Nogales; V. H. Ramey (1 November 2009). \"Structure-function insights into the yeast Dam1 kinetochore complex\". J Cell Sci. 122: 3831–3836. doi:10.1242/jcs.004689. PMC 2773187.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":4,"type":"book","title":"Campbell, Neil A.; Jane B. Reece (2005). Biology, 7th Edition. San Francisco: Benjamin Cummings. pp. 221–224. ISBN 0-8053-7171-0.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":5,"type":"book","title":"Manandhar Gf; Schatten H; Sutovsky P (2005). \"Centrosome reduction during gametogenesis and its significance\". Biol. Reprod. 72 (1): 2–13. doi:10.1095/biolreprod.104.031245. PMID 15385423.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":6,"type":"book","title":"Petry S, et al. (2013). \"Branching microtubule nucleation in Xenopus egg extracts mediated by augmin and TPX2\". Cell. 152: 768–777. doi:10.1016/j.cell.2012.12.044. PMC 3680348. PMID 23415226.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":7,"type":"book","title":"J.E. Rickard; T.E. Kreis (1990). \"Identification of a novel nucleotide-sensitive microtubule-binding protein in HeLa cells\". J Cell Biol. 110 (5): 1623–1633. doi:10.1083/jcb.110.5.1623. PMC 2200191. PMID 1970824.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":8,"type":"book","title":"D. Dujardin; U.I. Wacker; A. Moreau; T.A. Schroer; J.E. Rickard; J.R. DeMey (1998). \"Evidence for a role of CLIP-170 in the establishment of metaphase chromosome alignment\". J Cell Biol. 141 (4): 849–862. doi:10.1083/jcb.141.4.849. PMC 2132766. PMID 9585405.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":9,"type":"book","title":"D. Brunner; P. Nurse (2000). \"CLIP-170-like tip1p spatially organizes microtubular dynamics in fission yeast\". Cell. 102 (5): 695–704. doi:10.1016/S0092-8674(00)00091-X. PMID 11007487.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":10,"type":"book","title":"Y.A. Komarova; A.S. Kojima; et al. (2002). \"Cytoplasmic linker proteins promote microtubule rescue in vivo\". J Cell Biol. 159: 589–599. doi:10.1083/jcb.200208058. PMC 2173097. PMID 12446741.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":11,"type":"book","title":"S. Goldstone; C. Reyes; G. Gay; T. Courthéoux; M. Dubarry; et al. (2010). \"Tip1/CLIP-170 Protein Is Required for Correct Chromosome Poleward Movement in Fission Yeast\". PLoS ONE. 5: e10634. doi:10.1371/journal.pone.0010634. PMC 2869355. PMID 20498706.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":12,"type":"book","title":"A.L. Pereira; A.J. Pereira; A.R.R. Maia; et al. (1 October 2006). \"Mammalian CLASP1 and CLASP2 Cooperate to Ensure Mitotic Fidelity by Regulating Spindle and Kinetochore Function\". Mol Biol Cell. 17: 4526–4542. doi:10.1091/mbc.E06-07-0579. PMC 1635371. PMID 16914514.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":13,"type":"book","title":"A. Akhmanova; M.O. Steinmetz (April 2008). \"Tracking the ends: a dynamic protein network controls the fate of microtubule tips\". Nat Rev Mol Cell Biol. 9 (4): 309–322. doi:10.1038/nrm2369. PMID 18322465.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":14,"type":"book","title":"J.S. Tirnauer; S. Grego; E.D. Salmon; T.J. Mitchison (1 October 2002). \"EB1-microtubule interactions in Xenopus egg extracts: Role of EB1 in microtubule stabilization and mechanisms of targeting to microtubules\". Mol Biol Cell. 13 (10): 3614–3626. doi:10.1091/mbc.02-04-0210. PMC 129970. PMID 12388761.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":15,"type":"book","title":"M.E. Tanenbaum; R.H. Medema; A. Akhmanova (2011). \"Regulation of localization and activity of the microtubule depolymerase MCAK\". Bioarchitecture. 1 (2): 80–87. doi:10.4161/bioa.1.2.15807. PMC 3158623. PMID 21866268.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":16,"type":"book","title":"H. Niederstrasser; H. Salehi-Had; E.C. Gan; C. Walczak; E. Nogales (2002). \"XKCM1 acts on a single protofilament and requires the C terminus of tubulin\". J Mol Biol. 316 (3): 817–828. doi:10.1006/jmbi.2001.5360. PMID 11866534.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":17,"type":"book","title":"H. Maiato; P Sampaio; C.E. Sunkel (2004). \"Microtubule-associated proteins and their essential roles during mitosis\". Int Rev Cytol. 241: 53–153. doi:10.1016/S0074-7696(04)41002-X. PMID 15548419.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":18,"type":"book","title":"R. Tournebize; A. Popov; K. Kinoshita; A.J. Ashford; et al. (2000). \"Control of microtubule dynamics by the antagonistic activities of XMAP215 and XKCM1 in Xenopus egg extracts\". Nat Cell Biol. 2: 13–19. doi:10.1038/71330. PMID 10620801.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":19,"type":"book","title":"J. McIntosh; S.C. Landis (1971). \"The distribution of spindle microtubules during mitosis in cultured human cells\". J Cell Biol. 49 (2): 468–497. doi:10.1083/jcb.49.2.468. PMC 2108320. PMID 19866774.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":20,"type":"book","title":"D.J. Sharp; K.L. McDonald; H.M. Brown; et al. (1999). \"The bipolar kinesin, CLP61F, cross-links microtubules within interpolar microtubule bundles of Drosophila embryonic mitotic spindles\". J Cell Biol. 144 (1): 125–138. doi:10.1083/jcb.144.1.125. PMC 2148119. PMID 9885249.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":21,"type":"book","title":"M.A. Hallen; S.A. Endow (2009). \"Anastral spindle assembly: a mathematical model\". Biophys J. 97 (8): 2191–2201. doi:10.1016/j.bpj.2009.08.008. PMC 2764103. PMID 19843451.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":22,"type":"book","title":"R. Heald; R. Tournebize; et al. (1996). \"Self-organization of microtubules into bipolar spindles around artificial chromosomes in Xenopus egg extracts\". Nature. 382 (6590): 420–425. doi:10.1038/382420a0. PMID 8684481.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":23,"type":"book","title":"A. Khodjakov; R.W. Cole; B.R. Oakley; C.L. Rieder (2000). \"Centrosome-independent mitotic spindle formation in vertebrates\". Curr Biol. 10 (2): 59–67. doi:10.1016/S0960-9822(99)00276-6. PMID 10662665.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":24,"type":"book","title":"K.S. Burbank; T.J. Mitchison; D.S. Fisher (2007). \"Slide-and-cluster models for spindle assembly\". Curr Biol. 17 (16): 1373–1383. doi:10.1016/j.cub.2007.07.058. PMID 17702580.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":25,"type":"book","title":"Makde R, England J, Yennawar H, Tan S (2010). \"Structure of the RCC1 chromatin factor bound to the nucleosome core particle\". Nature. 467 (7315): 562–566. doi:10.1038/nature09321. PMC 3168546. PMID 20739938.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":26,"type":"book","title":"Halpin D, Kalab P, Wang J, Weis K, Heald R (2011). \"Mitotic spindle assembly around RCC1-coated beads in Xenopus egg extracts\". PLoS Biol. 9 (12): e1001225. doi:10.1371/journal.pbio.1001225. PMC 3246454. PMID 22215983.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":27,"type":"book","title":"Fu J, Jiang Q, Zhang C (2010). \"Coordination of Cell Cycle Events by Ran GTPase\". Nature Education. 3 (9): 32.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":28,"type":"book","title":"A.R. Barr; F. Gergely (2007). \"Aurora A: The maker and breaker of spindle poles\". J Cell Sci. 120: 2987–2996. doi:10.1242/jcs.013136.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":29,"type":"book","title":"Peters, U., J. Cherian; et al. (2006). \"Probing cell-division phenotype space and Polo-like kinase function using small molecules\". Nat Chem Biol. 2 (11): 618–26. doi:10.1038/nchembio826. PMID 17028580.CS1 maint: Multiple names: authors list (link)","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":30,"type":"book","title":"Morgan DO: The Cell Cycle: Principles of Control (Primers inBiology) London: New Science Press Ltd; 2007:297. ISBN 978-0-9539181-2-6","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":31,"type":"book","title":"Belmont AS (2010). \"Large-scale chromatin organization: The good, the surprising, and the still perplexing\". Curr Opin Cell Biol. 26: 69–78. doi:10.1016/j.ceb.2013.10.002. PMC 3927141. PMID 24529248.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":32,"type":"book","title":"Marko, JF. The mitotic chromosome: structure and Mechanics. 2012. Genome Organization and Function in the Cell Nucleus. Wiley-VCH, Ch. 18, 449-485. doi:10.1002/9783527639991.ch18","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":33,"type":"book","title":"Champoux JJ (2001). \"DNA TOPOISOMERASES: Structure, Function, and Mechanism\". Annu Rev Biochem. 70 (1): 369–413. doi:10.1146/annurev.biochem.70.1.369. PMID 11395412.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":34,"type":"book","title":"Maresca TJ, Freedman BS, Heald R (2005). \"Histone H1 is essential for mitotic chromosome architecture and segregation in Xenopus laevis egg extracts\". J. Cell Biol. 169 (6): 859–69. doi:10.1083/jcb.200503031. PMC 2171634. PMID 15967810.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":35,"type":"book","title":"Raven, Peter H.; Ray F. Evert; Susan E. Eichhorn (2005). Biology of Plants, 7th Edition. New York: W.H. Freeman and Company Publishers. p. 59. ISBN 0-7167-1007-2.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":36,"type":"book","title":"Baker DJ, Chen J, van Deursen JM (2005). \"The mitotic checkpoint in cancer and aging: what have mice taught us?\". Curr. Opin. Cell Biol. 17 (6): 583–9. doi:10.1016/j.ceb.2005.09.011. PMID 16226453.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":37,"type":"book","title":"Thery M, Jimenez-Dalmaroni A, Racine V, Bornens M, Julicher F (2007). \"Experimental and theoretical study of mitotic spindle orientation\". Nature. 447: 493–6. doi:10.1038/nature05786. PMID 17495931.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":38,"type":"book","title":"Bosveld F, Markova O, Guirao B, Martin C, Wang Z, Pierre A, Balakireva M, Gaugue I, Ainslie A, Christophorou N, Lubensky DK, Minc N, Bellaïche Y (2016). \"Epithelial tricellular junctions act as interphase cell shape sensors to orient mitosis\". Nature. 530: 496–8. doi:10.1038/nature16970. PMC 5450930. PMID 26886796.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false}],"recommendReferences":null,"auditState":2,"lemmaLevel":1,"origin":0,"originEnTitle":null,"originZhTitle":null,"pv":5603,"auditType":0,"synonyms":["Spindle Apparatus"],"showEditTime":"2019.12.13 18:32","auditors":[{"uid":0,"name":"Ki.κe","pic":"https://web.archive.org/web/20221028212443/https://wx.qlogo.cn/mmopen/vi_32/y67kfr32Doib4wg71Jiau7jVWvharic3nRKgdRRQSl6koeQJCo0GQs2Krw0vwdFRsOWnHIQOwAZsSg5lIkIrFCOcQ/132","introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":"","jobBrief":"","role":0,"roleName":null,"title":"","professionalTitle":null,"phoneNo":null,"editable":true,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false}],"hasZhishiNav":false,"auditInfos":{},"isHistory":false};</script><script crossorigin="anonymous" src="./63.纺锤体 - 搜狗科学百科_files/aegis.min.js.download"></script><script crossorigin="anonymous" src="./63.纺锤体 - 搜狗科学百科_files/main_2020092401.js.download"></script><script crossorigin="anonymous" src="./63.纺锤体 - 搜狗科学百科_files/react.production.min.js.download"></script><script crossorigin="anonymous" src="./63.纺锤体 - 搜狗科学百科_files/react-dom.production.min.js.download"></script><script crossorigin="anonymous" src="./63.纺锤体 - 搜狗科学百科_files/jquery-1.11.1.min.js.download"></script><script crossorigin="anonymous" src="./63.纺锤体 - 搜狗科学百科_files/main_2022062701.js.download"></script><script crossorigin="anonymous" src="./63.纺锤体 - 搜狗科学百科_files/main_66bbe21.js.download"></script><script crossorigin="anonymous" src="./63.纺锤体 - 搜狗科学百科_files/react.production.min.js.download"></script><script crossorigin="anonymous" src="./63.纺锤体 - 搜狗科学百科_files/react-dom.production.min.js.download"></script><script crossorigin="anonymous" src="./63.纺锤体 - 搜狗科学百科_files/main_edf0f08.js.download"></script>
</body></html>