<!DOCTYPE html>
<!-- saved from url=(0083)https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm -->
<html class="" data-reactroot=""><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><script src="./636.肿瘤基因组学 - 搜狗科学百科_files/analytics.js.download" type="text/javascript"></script>
<script type="text/javascript">window.addEventListener('DOMContentLoaded',function(){var v=archive_analytics.values;v.service='wb';v.server_name='wwwb-app210.us.archive.org';v.server_ms=234;archive_analytics.send_pageview({});});</script>
<script type="text/javascript" src="./636.肿瘤基因组学 - 搜狗科学百科_files/bundle-playback.js.download" charset="utf-8"></script>
<script type="text/javascript" src="./636.肿瘤基因组学 - 搜狗科学百科_files/wombat.js.download" charset="utf-8"></script>
<script type="text/javascript">
  __wm.init("https://web.archive.org/web");
  __wm.wombat("https://baike.sogou.com/kexue/d10636.htm","20221025184839","https://web.archive.org/","web","/_static/",
	      "1666723719");
</script>
<link rel="stylesheet" type="text/css" href="./636.肿瘤基因组学 - 搜狗科学百科_files/banner-styles.css">
<link rel="stylesheet" type="text/css" href="./636.肿瘤基因组学 - 搜狗科学百科_files/iconochive.css">
<!-- End Wayback Rewrite JS Include -->
<meta name="save" content="history"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="baidu-site-verification" content="VWGb6TyYx8"><meta content="肿瘤基因组学 - 搜狗科学百科" name="keywords"><meta content="搜狗科学百科是一部有着平等、协作、分享、自由理念的网络科学全书，为每一个互联网用户创造一个涵盖所有领域知识、服务的中文知识性平台。" name="description"><meta http-equiv="x-dns-prefetch-control" content="on"><meta name="server" baike="235" ip="210" env="online"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025184839/https://cache.soso.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025184839/https://hhy.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025184839/https://pic.baike.soso.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025184839/https://ugc.qpic.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025184839/https://xui.ptlogin2.qq.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025184839/https://q1.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025184839/https://q2.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025184839/https://q3.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025184839/https://q4.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025184839/https://q.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025184839/https://img01.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025184839/https://img02.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025184839/https://img03.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025184839/https://img04.sogoucdn.com/"><link rel="Shortcut Icon" href="https://web.archive.org/web/20221025184839im_/https://www.sogou.com/images/logo/new/favicon.ico?v=4"><link rel="Bookmark" href="https://www.sogou.com/images/logo/new/favicon.ico?v=4"><link href="./636.肿瘤基因组学 - 搜狗科学百科_files/base_b849887.css" rel="stylesheet"><link href="./636.肿瘤基因组学 - 搜狗科学百科_files/detail_378aed5.css" rel="stylesheet"><link href="./636.肿瘤基因组学 - 搜狗科学百科_files/inviteAudit_7894507.css" rel="stylesheet"><link rel="stylesheet" href="./636.肿瘤基因组学 - 搜狗科学百科_files/highlight.min.css"><title>肿瘤基因组学 - 搜狗科学百科</title><style>.onekey-close {
	position: absolute;
	top: 16px;
	right: 16px;
	width: 24px;
	height: 24px;
	background: url(//web.archive.org/web/20221025193758/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	text-indent: -999em;
	background-size: 84px;
	background-position: -63px 0;
}

.onekey-login {
	position: absolute;
	top: 16.4%;
	left: 0;
	right: 0;
	width: 100%;
}

/* .onekey-login-img {
    width: 75px;
    height: 75px;
    background: url("https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/images/sprite_wap_baike.png") no-repeat;
    background-size: 100px 91px;
    background-position: 0 0;
    background-repeat: no-repeat;
    margin: 0 auto;
} */

.onekey-login-title {
	text-align: center;
	padding-bottom: 3px;
	font-size: 21px;
	font-weight: bold;
	line-height: 30px;
	color: #000;
}

.onekey-login-txt {
	text-align: center;
	font-family: PingFangSC;
	font-size: 14px;
	line-height: 20px;
	color: #8f8f8f;
}

.onekey-login-qq,
.onekey-login-wx,
.onekey-login-phone {
	display: block;
	width: 245px;
	height: 54px;
	border-radius: 45px;
	text-align: center;

	margin: 0 auto;
	font-size: 17px;
	line-height: 24px;
	color: #000;
	/* padding: 16px 77px; */
	border-radius: 12px;
	border: solid 1px #e0e0e0;
}
.onekey-qq-content,
.onekey-vx-content,
.onekey-phone-content {
	display: inline-block;
	margin-top: 16px;
}
.onekey-qq-content {
	padding: 0 5px;
}

.onekey-login-qq {
	margin-top: 48px;
	margin-bottom: 24px;
}

.onekey-login-qq:before {
	display: inline-block;
	content: "";
	width: 20px;
	height: 20px;
	background: url(//web.archive.org/web/20221025193758/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	background-size: 80px;
	background-position: -20px 0;
	vertical-align: top;
	margin: 17px 8px 0 0;
}

.onekey-login-wx {
	margin-bottom: 24px;
}

.onekey-login-wx:before {
	display: inline-block;
	content: "";
	width: 21px;
	height: 21px;
	background: url(//web.archive.org/web/20221025193758/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	background-size: 84px;
	background-position: 0 0;
	vertical-align: top;
	margin: 17px 10px 0 0;
}

.onekey-login-phone {
}

.onekey-login-phone:before {
	display: inline-block;
	content: "";
	width: 21px;
	height: 21px;
	background: url(//web.archive.org/web/20221025193758/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	background-size: 84px;
	background-position: -42px 0;
	vertical-align: top;
	margin: 17px 10px 0 0;
}

.onekey-fixed {
	z-index: 100;
	position: fixed;
	top: 0;
	bottom: 0;
	left: 0;
	right: 0;
	background: #fff;
	width: 100%;
	height: 100%;
}

.onekey-fixed.forbid {
	z-index: 100;
	position: fixed;
	top: auto;
	bottom: 68px;
	left: 9%;
	right: 0;
	background: rgba(0, 0, 0, 0.7);
	width: 82%;
	height: 43px;
	border-radius: 25px;
	color: #ffffff;
}
.onekey-login-title.forbid {
	text-align: center;
	padding-bottom: 3px;
	font-size: 14px;
	font-weight: normal;
	line-height: 30px;
	color: white;
}
</style><style>#login_mask {
  background: #000;
  opacity: 0.5;
  filter: alpha(opacity=50);
  position: fixed;
  /*fixed好像在哪个IE上有BUG，先用用*/
  left: 0;
  top: 0;
  z-index: 999;
  height: 100%;
}

#login_iframe_container {
  position: fixed;
  width: 550px;
  height: 360px;
  z-index: 1020;
  background-color: #ffffff;
}

@media screen and (max-width: 828px) {
  #login_iframe_container {
    top: 50% !important;
    left: 50% !important;
    transform: translate(-50%, -50%);
  }
}

#login_iframe_container.new-login {
  width: 550px;
  height: 360px;
  background-image: url(//web.archive.org/web/20221025193758/https://hhy.sogoucdn.com/js/common/hhy/background_2a4a8a6.png);
}

#login_iframe_container.new-login.no-bg {
  background: #fff;
}

#login_iframe_container.new-login .login-title {
  width: 100%;
  height: 42px;
  line-height: 42px;
  text-align: center;
  font-size: 30px;
  letter-spacing: 0.19px;
  color: #ffffff;
  margin-top: 62px;
}
#login_iframe_container.new-login .forbid-title {
  width: 100%;
  height: 42px;
  line-height: 42px;
  text-align: center;
  font-size: 24px;
  letter-spacing: 0.19px;
  color: #333333;
  margin-top: 150px;
}

#login_iframe_container.new-login.no-bg .login-title {
  color: #333333;
}

#login_iframe_container.new-login .login-subtitle {
  width: 100%;
  height: 18px;
  line-height: 18px;
  font-size: 13px;
  letter-spacing: 0.08px;
  color: #ffffff;
  text-align: center;
  margin-top: 9px;
  margin-bottom: 43px;
}

#login_iframe_container.new-login.no-bg .login-subtitle {
  color: #999999;
}

#login_iframe_container.new-login .login-subtitle::before {
  content: '';
  display: inline-block;
  width: 10px;
  height: 1px;
  background-color: #ffffff;
  position: relative;
  top: -4px;
  left: -5px;
}

#login_iframe_container.new-login .login-subtitle::after {
  content: '';
  display: inline-block;
  width: 10px;
  height: 1px;
  background-color: #ffffff;
  position: relative;
  top: -4px;
  left: 5px;
}

#login_iframe_container.new-login.no-bg .login-subtitle::before {
  background-color: #999999;
}

#login_iframe_container.new-login.no-bg .login-subtitle::after {
  background-color: #999999;
}

#login_iframe_container.new-login .close-btn {
  position: absolute;
  top: 20px;
  right: 20px;
  width: 12px;
  height: 12px;
  background: url(//web.archive.org/web/20221025193758/https://hhy.sogoucdn.com/js/common/hhy/login-sprites_e3853e5.png) -59px -10px;
  background-size: 81px 91px;
  cursor: pointer;
}

#login_iframe_container.new-login .login-btn {
  width: 220px;
  height: 47px;
  border-radius: 24px;
  border: solid 1px #dddddd;
  background-color: #ffffff;
  margin: 0 auto;
  margin-top: 28px;
  position: relative;
  display: block;
}

#login_iframe_container.new-login .login-btn .login-icon {
  position: absolute;
}

#login_iframe_container.new-login .login-btn .login-text {
  width: 61px;
  height: 47px;
  line-height: 47px;
  vertical-align: middle;
  font-size: 15px;
  letter-spacing: 0.1px;
  color: #666666;
  position: absolute;
  right: 62px;
}

#login_iframe_container.new-login .login-btn.qq-btn .login-icon {
  width: 22px;
  height: 27px;
  top: 10px;
  left: 67px;
  background: url(//web.archive.org/web/20221025193758/https://hhy.sogoucdn.com/js/common/hhy/login-sprites_e3853e5.png) -10px -54px;
  background-size: 81px 91px;
}

#login_iframe_container.new-login .login-btn.qq-btn .login-text {
  right: 59px;
}

#login_iframe_container.new-login .login-btn.wechat-btn .login-icon {
  width: 29px;
  height: 24px;
  top: 12px;
  left: 62px;
  background: url(//web.archive.org/web/20221025193758/https://hhy.sogoucdn.com/js/common/hhy/login-sprites_e3853e5.png) -10px -10px;
  background-size: 81px 91px;
}</style><style>/* -- container -- */
.rodal,
.rodal-mask {
    top: 0;
    left: 0;
    width: 100%;
    height: 100%;
    z-index: 100;
}

.rodal {
    position: fixed;
}

/* -- mask -- */
.rodal-mask {
    position: fixed;
    background: rgba(0, 0, 0, .5);
}

/* -- dialog -- */
.rodal-dialog {
    position: absolute;
    z-index: 101;
    background: #fff;
    border-radius: 3px;
    box-shadow: 0 1px 3px rgba(0, 0, 0, .2);
}

.rodal-center {
    top: 50%;
    transform: translateY(-50%);
    left: 0;
    right: 0;
    margin: 0 auto;
}

.rodal-bottom {
    left: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

.rodal-top {
    left: 0;
    right: 0;
    top: 0;
    margin: auto;
}

.rodal-left {
    top: 0;
    left: 0;
    bottom: 0;
    margin: auto;
}

.rodal-right {
    top: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

/* -- close button -- */
.rodal-close {
    position: absolute;
    cursor: pointer;
    top: 16px;
    right: 16px;
    width: 16px;
    height: 16px;
}

.rodal-close:before,
.rodal-close:after {
    position: absolute;
    content: '';
    height: 2px;
    width: 100%;
    top: 50%;
    left: 0;
    margin-top: -1px;
    background: #999;
    border-radius: 100%;
    -webkit-transition: background .2s;
    transition: background .2s;
}

.rodal-close:before {
    -webkit-transform: rotate(45deg);
    transform: rotate(45deg);
}

.rodal-close:after {
    -webkit-transform: rotate(-45deg);
    transform: rotate(-45deg);
}

.rodal-close:hover:before,
.rodal-close:hover:after {
    background: #333;
}

/* -- fade -- */
/* @-webkit-keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

@keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

.rodal-fade-enter {
    -webkit-animation: rodal-fade-enter both ease-in;
    animation: rodal-fade-enter both ease-in;
} */

@-webkit-keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

@keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

.rodal-fade-leave {
    -webkit-animation: rodal-fade-leave both ease-out;
    animation: rodal-fade-leave both ease-out;
}

/* -- zoom -- */
@-webkit-keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-enter {
    -webkit-animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-leave {
    -webkit-animation: rodal-zoom-leave both;
    animation: rodal-zoom-leave both;
}

/* -- slideDown -- */
@-webkit-keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-enter {
    -webkit-animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-leave {
    -webkit-animation: rodal-slideDown-leave both;
    animation: rodal-slideDown-leave both;
}

/* -- slideLeft -- */
@-webkit-keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-enter {
    -webkit-animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-leave {
    -webkit-animation: rodal-slideLeft-leave both;
    animation: rodal-slideLeft-leave both;
}

/* -- slideRight -- */
@-webkit-keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-enter {
    -webkit-animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-leave {
    -webkit-animation: rodal-slideRight-leave both;
    animation: rodal-slideRight-leave both;
}

/* -- slideUp -- */
@-webkit-keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-enter {
    -webkit-animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
    animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
}

@-webkit-keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-leave {
    -webkit-animation: rodal-slideUp-leave both;
    animation: rodal-slideUp-leave both;
}

/* -- flip -- */
@-webkit-keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

@keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

.rodal-flip-enter {
    -webkit-animation: rodal-flip-enter both ease-in;
    animation: rodal-flip-enter both ease-in;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

@-webkit-keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

@keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

.rodal-flip-leave {
    -webkit-animation: rodal-flip-leave both;
    animation: rodal-flip-leave both;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

/* -- rotate -- */
@-webkit-keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-enter {
    -webkit-animation: rodal-rotate-enter both;
    animation: rodal-rotate-enter both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

@-webkit-keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-leave {
    -webkit-animation: rodal-rotate-leave both;
    animation: rodal-rotate-leave both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

/* -- door -- */
@-webkit-keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

@keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

.rodal-door-enter {
    -webkit-animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

@keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

.rodal-door-leave {
    -webkit-animation: rodal-door-leave both;
    animation: rodal-door-leave both;
}</style><style>/* -- container -- */
.rodal,
.rodal-mask {
    top: 0;
    left: 0;
    width: 100%;
    height: 100%;
    z-index: 100;
}

.rodal {
    position: fixed;
}

/* -- mask -- */
.rodal-mask {
    position: fixed;
    background: rgba(0, 0, 0, .5);
}

/* -- dialog -- */
.rodal-dialog {
    position: absolute;
    z-index: 101;
    background: #fff;
    border-radius: 3px;
    box-shadow: 0 1px 3px rgba(0, 0, 0, .2);
}

.rodal-center {
    top: 50%;
    transform: translateY(-50%);
    left: 0;
    right: 0;
    margin: 0 auto;
}

.rodal-bottom {
    left: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

.rodal-top {
    left: 0;
    right: 0;
    top: 0;
    margin: auto;
}

.rodal-left {
    top: 0;
    left: 0;
    bottom: 0;
    margin: auto;
}

.rodal-right {
    top: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

/* -- close button -- */
.rodal-close {
    position: absolute;
    cursor: pointer;
    top: 16px;
    right: 16px;
    width: 16px;
    height: 16px;
}

.rodal-close:before,
.rodal-close:after {
    position: absolute;
    content: '';
    height: 2px;
    width: 100%;
    top: 50%;
    left: 0;
    margin-top: -1px;
    background: #999;
    border-radius: 100%;
    -webkit-transition: background .2s;
    transition: background .2s;
}

.rodal-close:before {
    -webkit-transform: rotate(45deg);
    transform: rotate(45deg);
}

.rodal-close:after {
    -webkit-transform: rotate(-45deg);
    transform: rotate(-45deg);
}

.rodal-close:hover:before,
.rodal-close:hover:after {
    background: #333;
}

/* -- fade -- */
/* @-webkit-keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

@keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

.rodal-fade-enter {
    -webkit-animation: rodal-fade-enter both ease-in;
    animation: rodal-fade-enter both ease-in;
} */

@-webkit-keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

@keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

.rodal-fade-leave {
    -webkit-animation: rodal-fade-leave both ease-out;
    animation: rodal-fade-leave both ease-out;
}

/* -- zoom -- */
@-webkit-keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-enter {
    -webkit-animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-leave {
    -webkit-animation: rodal-zoom-leave both;
    animation: rodal-zoom-leave both;
}

/* -- slideDown -- */
@-webkit-keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-enter {
    -webkit-animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-leave {
    -webkit-animation: rodal-slideDown-leave both;
    animation: rodal-slideDown-leave both;
}

/* -- slideLeft -- */
@-webkit-keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-enter {
    -webkit-animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-leave {
    -webkit-animation: rodal-slideLeft-leave both;
    animation: rodal-slideLeft-leave both;
}

/* -- slideRight -- */
@-webkit-keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-enter {
    -webkit-animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-leave {
    -webkit-animation: rodal-slideRight-leave both;
    animation: rodal-slideRight-leave both;
}

/* -- slideUp -- */
@-webkit-keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-enter {
    -webkit-animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
    animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
}

@-webkit-keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-leave {
    -webkit-animation: rodal-slideUp-leave both;
    animation: rodal-slideUp-leave both;
}

/* -- flip -- */
@-webkit-keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

@keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

.rodal-flip-enter {
    -webkit-animation: rodal-flip-enter both ease-in;
    animation: rodal-flip-enter both ease-in;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

@-webkit-keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

@keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

.rodal-flip-leave {
    -webkit-animation: rodal-flip-leave both;
    animation: rodal-flip-leave both;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

/* -- rotate -- */
@-webkit-keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-enter {
    -webkit-animation: rodal-rotate-enter both;
    animation: rodal-rotate-enter both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

@-webkit-keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-leave {
    -webkit-animation: rodal-rotate-leave both;
    animation: rodal-rotate-leave both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

/* -- door -- */
@-webkit-keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

@keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

.rodal-door-enter {
    -webkit-animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

@keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

.rodal-door-leave {
    -webkit-animation: rodal-door-leave both;
    animation: rodal-door-leave both;
}</style>
<!--百度统计-->
<script>
   var _hmt = _hmt || [];
   (function() {
      var hm = document.createElement("script");
      hm.src = "https://hm.baidu.com/hm.js?3c7614be3026469d5a60f41ab30b5082";
      var s = document.getElementsByTagName("script")[0]; 
      s.parentNode.insertBefore(hm, s);
      })();
</script>
</head>
<body class=""><!-- BEGIN WAYBACK TOOLBAR INSERT -->
<style type="text/css">
body {
  margin-top:0 !important;
  padding-top:0 !important;
  /*min-width:800px !important;*/
}
</style>
<script>__wm.rw(0);</script>
<div id="wm-ipp-base" lang="en" style="display: block; direction: ltr;">
</div><div id="wm-ipp-print">The Wayback Machine - https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm</div>
<script type="text/javascript">//<![CDATA[
__wm.bt(675,27,25,2,"web","https://baike.sogou.com/kexue/d10636.htm","20221025184839",1996,"/_static/",["/_static/css/banner-styles.css?v=S1zqJCYt","/_static/css/iconochive.css?v=qtvMKcIJ"], false);
  __wm.rw(1);
//]]></script>
<!-- END WAYBACK TOOLBAR INSERT --><script>window._gtag=window._gtag||{};window._gtag.shouldGrayed = false;if ('63dcb93d274148138c97c0eada3491ba') window._gtag.traceId = '63dcb93d274148138c97c0eada3491ba';if ({"illegality":true}) window.userInfo = {"illegality":true};</script><div class="topnavbox"><ul class="topnav"><li><a href="https://web.archive.org/web/20221025184839/https://www.sogou.com/web?query=">网页</a></li><li><a href="https://web.archive.org/web/20221025184839/https://weixin.sogou.com/weixin?p=75351201">微信</a></li><li><a href="https://web.archive.org/web/20221025184839/https://zhihu.sogou.com/zhihu?p=75351218">知乎</a></li><li><a href="https://web.archive.org/web/20221025184839/https://pic.sogou.com/pics?query=">图片</a></li><li><a href="https://web.archive.org/web/20221025184839/https://v.sogou.com/v?query=">视频</a></li><li><a href="https://web.archive.org/web/20221025184839/https://mingyi.sogou.com/">医疗</a></li><li class="cur"><strong>科学</strong></li><li><a href="https://web.archive.org/web/20221025184839/https://hanyu.sogou.com/">汉语</a></li><li><a href="https://web.archive.org/web/20221025184839/https://wenwen.sogou.com/">问问</a></li><li><a href="https://web.archive.org/web/20221025184839/https://www.sogou.com/docs/more.htm">更多<span class="topraquo">»</span></a></li></ul></div><div id="header"><div class="header-wrap"><a class="header-logo" href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue"></a><div class="header-search"><div class="querybox" id="suggBox"><form><input id="searchInput" class="query" type="text" placeholder="搜科学领域专业百科词条" name="query" autocomplete="off" value=""><a href="javascript:;" class="query-search"></a></form></div></div><div class="header-rgt"><span class="btn-header-rgt btn-edit" id="editLemma">创建</span><div class="header-user no-login"></div></div></div></div><div class="fixed-placeholder" style="visibility:none"></div><div id="container" class=""><div class="content lemma-level1"><div class="detail-title" id="abstract-title"><h1>肿瘤基因组学</h1><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#!" class="detail-edit">编辑</a></div><div class="section_content" data-id="14995439136801295"><div class="text_img ed_imgfloat_right"><a class="ed_image_link" data-src="https://img02.sogoucdn.com/app/a/200698/sogou_science_13465" data-bigsrc="https://img02.sogoucdn.com/app/a/200698/sogou_science_13465?w=500&amp;h=277&amp;titlename=%E8%82%BF%E7%98%A4%E5%9F%BA%E5%9B%A0%E7%BB%84%E5%AD%A6%E7%9A%84%E6%95%B4%E4%BD%93%E7%9B%AE%E6%A0%87" title="点击查看大图" data-w="500" data-h="277" style="background-image:url(https://web.archive.org/web/20221025184839im_/https://img02.sogoucdn.com/app/a/200698/sogou_science_13465)" href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#!"></a><div class="text_img_title" style="text-align: center;">肿瘤基因组学的整体目标</div></div><div><p>肿瘤基因组学是基因组学描述癌症相关基因的一个子领域。它关注癌症的基因组、表观基因组和转录组的水平变化。 </p>
<p> 癌症是一种基因病，由脱氧核糖核酸突变的积累和表观遗传水平的改变导致细胞无限制增殖形成肿瘤。肿瘤基因组学的目标是鉴定新的致癌基因或肿瘤抑制基因，这些基因可能为癌症诊断、癌症的临床结果预测和癌症靶向治疗提供新的方法。格列卫、赫赛汀和阿瓦斯丁等靶向癌症治疗的成功为肿瘤基因组学阐明癌症治疗的新靶点带来了希望。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_1" class="kx_ref">[1]</a></sup></p>
<p></p> 
<p>除了了解引发或驱动癌症进展的潜在遗传机制，肿瘤基因组学还以个性化癌症治疗为目标。癌症是由于随机积累的基因突变和表观遗传改变而发展的。识别和定位个体患者的突变可能会提高治疗效果。 </p>
<p>人类基因组计划的完成促进了肿瘤组基因组学领域的发展，并提高了研究人员发现肿瘤基因的能力。测序技术和全基因组甲基化图谱技术已经应用于肿瘤基因组学的研究。 </p></div></div><div id="catalog"><h2 class="title2">目录<a href="javascript:" class="detail-edit">编辑</a></h2><div class="catalog_wrap" style=""><ul class="catalog_list col3"><li><span class="order">1</span><a href="javascript:" data-level="1" data-id="14995439136801296">历史</a></li><li class="secondary_catalog"><span>1.1 </span><a href="javascript:" data-id="14995439136801296">全基因组测序</a></li><li><span class="order">2</span><a href="javascript:" data-level="1" data-id="14995439153578500">技术</a></li><li class="secondary_catalog"><span>2.1 </span><a href="javascript:" data-id="14995439153578500">基因组测序</a></li><li class="secondary_catalog"><span>2.2 </span><a href="javascript:" data-id="14995439153578500">转录组</a></li><li class="secondary_catalog"><span>2.3 </span><a href="javascript:" data-id="14995439153578500">生物信息学与肿瘤基因的功能分析</a></li><li class="secondary_catalog"><span>2.4 </span><a href="javascript:" data-id="14995439153578500">操纵子组</a></li></ul><ul class="catalog_list col3"><li><span class="order">3</span><a href="javascript:" data-level="1" data-id="14995439153578501">比较肿瘤基因组学</a></li><li><span class="order">4</span><a href="javascript:" data-level="1" data-id="14995439153578502">癌症驱动突变的来源，癌症突变形成</a></li><li><span class="order">5</span><a href="javascript:" data-level="1" data-id="14995439170355718">合成致死</a></li><li><span class="order">6</span><a href="javascript:" data-level="1" data-id="14995439170355719">癌症研究数据库</a></li><li><span class="order">7</span><a href="javascript:" data-level="1" data-id="14995439187132936">基因家族</a></li><li><span class="order">8</span><a href="javascript:" data-level="1" data-id="14995439187132937">线粒体dna</a></li><li class="secondary_catalog"><span>8.1 </span><a href="javascript:" data-id="14995439187132937">点突变</a></li></ul><ul class="catalog_list col3"><li class="secondary_catalog"><span>8.2 </span><a href="javascript:" data-id="14995439187132937">删除</a></li><li class="secondary_catalog"><span>8.3 </span><a href="javascript:" data-id="14995439187132937">插入</a></li><li class="secondary_catalog"><span>8.4 </span><a href="javascript:" data-id="14995439187132937">拷贝数突变</a></li><li class="secondary_catalog"><span>8.5 </span><a href="javascript:" data-id="14995439187132937">诊断应用</a></li><li><span class="order">9</span><a href="javascript:" data-level="1" data-id="14995439187132938">肿瘤生物标志物</a></li><li><span class="order">10</span><a href="javascript:" data-level="1" data-id="references">参考文献</a></li></ul></div></div><div id="paragraphs"><div><div id="par_14995439136801296"><h2 class="title">1 历史<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>基因组学始于20世纪90年代，伴随着许多生物基因序列的产生。在21世纪，人类基因组计划的完成使得功能基因组学的研究和肿瘤基因组检测成为可能。癌症是一个主要焦点。 </p>
<p>表观基因组学时代最近才开始，约在2000年。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_2" class="kx_ref">[2]</a></sup><sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_3" class="kx_ref">[3]</a></sup>表观遗传变化的一个主要来源是基因启动子区CpG岛甲基化的改变。许多最近设计的方法可以评估癌症和正常组织中的脱氧核糖核酸甲基化状态。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_4" class="kx_ref">[4]</a></sup>一些方法评估位于不同基因座类别的CpG甲基化，包括CpG岛、岛岸和岛架以及启动子、基因体和基因间区域。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_5" class="kx_ref">[5]</a></sup>癌症也是表观遗传学研究的一个主要焦点。 </p>
<p>获得癌症全基因组测序对癌症(或癌症基因组)研究很重要，因为: </p> 
<ul>
 <li>突变是引发癌症的直接原因，并决定了肿瘤表型。</li> 
 <li>从同一名患者获取癌症和正常组织样本，根据大多数突变代表体细胞变异的事实，来识别癌症特异性突变。</li> 
 <li>癌症突变是累积的，有时与疾病分期有关。转移和耐药性是可以区分的。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_6" class="kx_ref">[6]</a></sup> 获得甲基化谱对癌症研究很重要，因为:</li> 
 <li>表观驱动和突变驱动可以作为癌症的直接原因。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_7" class="kx_ref">[7]</a></sup></li> 
 <li>癌症表突变是累积的，有时与疾病阶段相关。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_8" class="kx_ref">[8]</a></sup></li>
</ul> 
<h3>1.1 <span>全基因组测序</span></h3> 
<p>第一个癌症基因组于2008年被测序。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_6" class="kx_ref">[6]</a></sup>这项研究测序了一个典型的急性髓细胞白血病(AML)基因组及其从同一患者身上获得的正常对应基因组。通过比较揭示了十个突变基因。其中两个已经被认为是肿瘤进展的原因：FLT3受体酪氨酸激酶基因的内部串联重复（激活激酶信号通路并与不良预后相关）以及NPM1基因第12外显子的四个碱基插入。这些突变在25-30%的急性髓细胞白血病肿瘤中都有发现，被认为促进了疾病进展，而不是直接导致疾病发生的原因。 </p>
<p>其余8个是新突变，都是单碱基变化：4个是与癌症发病机制密切相关的家族(PTPRT、CDH24、PCLKC和SLC15A1)。另外四个与癌症发病机制没有关联。它们在代谢通路中确实有潜在的作用，提示了它们可以通过某种机制促进癌症 (KNDC1，GPR124，EB12，GRINC1B) </p>
<p>这些基因参与了已知的导致癌症发病的途径，但是在这项研究之前，大多数基因都不是靶向基因治疗的候选基因。该分析验证了癌症全基因组测序在鉴定体细胞突变中的方法以及正常和肿瘤细胞基因组平行测序的重要性。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_9" class="kx_ref">[9]</a></sup> </p>
<p>2011年，对一名已被依维莫司药物清除的特殊膀胱癌患者的基因组进行了测序，揭示了两个基因的突变，即TSC1和NF2。这些突变使mTOR失调（mTOR是依维莫司抑制的蛋白质）造成无限制地繁殖。因此，2015年，美国国家癌症研究所创建了“特殊反应者倡议”。该倡议允许这类特殊患者(对通常无效的癌症药物有至少六个月明确反应)进行基因组测序，以识别相关突变。一旦鉴定出来，其他患者就可以进行突变筛查，然后服用药物。为此，2015年开始了一项全国性的癌症药物试验，涉及多达2400个中心。具有适当突变的患者匹配到了四十多种药物中合适的一种药物。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_10" class="kx_ref">[10]</a></sup> </p>
<p>2014年，分子肿瘤学中心推出了MSK-IMPACT试验，这是一种寻找341个癌症相关基因突变的筛查工具。到2015年，超过5000名患者接受了筛查。具有适当突变的患者有资格参加提供靶向治疗的临床试验。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_10" class="kx_ref">[10]</a></sup> </p></div></div><div id="par_14995439153578500"><h2 class="title">2 技术<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p></p><p></p><div class="text_img ed_imgfloat_right">
            <a class="ed_image_link lazyLoad" data-src="https://img04.sogoucdn.com/app/a/200698/sogou_science_13466" data-bigsrc="" title="点击查看大图" href="javascript:" data-observer="true"></a>
            <div class="text_img_title" style="text-align: center;">肿瘤基因组学中使用的技术</div>   
        </div> <p></p><p></p>
<p>基因组技术包括: </p>
<h3>2.1 <span>基因组测序</span></h3> 
<ul>
 <li>脱氧核糖核酸测序：基于焦磷酸测序的测序仪提供了一种相对低成本的方法来生成测序数据。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_1" class="kx_ref">[1]</a></sup><sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_11" class="kx_ref">[11]</a></sup><sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_12" class="kx_ref">[12]</a></sup></li> 
 <li>微阵列比较基因组杂交：这项技术检测正常基因组和癌症基因组之间的基因拷贝数差异。它使用荧光标记样本的荧光强度值，这些样本可与微阵列上的已知探针杂交。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_13" class="kx_ref">[13]</a></sup><sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_14" class="kx_ref">[14]</a></sup></li> 
 <li>代表性寡核苷酸微阵列分析：使用扩增后的限制性酶消化基因组片段，与阵列上人类寡核苷酸杂交来检测拷贝数变化，达到30-35 kbit/秒的分辨率。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_15" class="kx_ref">[15]</a></sup></li> 
 <li>数字核型分析：通过限制性酶消化获得的基因组标签，然后将这些标签连接到双标签中，连接、克隆、测序并回贴到参考基因组，以评估标签密度，以此来检测拷贝数变化。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_16" class="kx_ref">[16]</a></sup><sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_17" class="kx_ref">[17]</a></sup></li> 
 <li>细菌人工染色体(BAC)-末端测序(末端序列分析)：通过从癌症基因组中生成BAC文库并对其末端测序来识别染色体断点。含有染色体异常的BAC克隆的末端序列不与参考基因组的相似区域相匹配，从而识别染色体断点。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_18" class="kx_ref">[18]</a></sup></li>
</ul> 
<h3>2.2 <span>转录组</span></h3> 
<ul>
 <li>微阵列：评估转录本丰度。在癌症分类、预后、提高不同治疗方法的可能性以及帮助识别蛋白质编码区的突变等方面十分有用。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_19" class="kx_ref">[19]</a></sup><sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_20" class="kx_ref">[20]</a></sup> 可变剪切转录本的相对丰富已经成为癌症研究的一个重要特征。特定的可变剪切转录本形式与特定的癌症类型相关。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_21" class="kx_ref">[21]</a></sup></li> 
 <li><i>RNA-Seq</i></li>
</ul> 
<h3>2.3 <span>生物信息学与肿瘤基因的功能分析</span></h3> 
<p>生物信息学技术主要用来对基因组数据进行统计分析。到目前为止，癌基因的功能特征尚未确定。例如，它们与肿瘤形成相关的转化能力以及在癌症发展的每个阶段的特定作用相关。 </p>
<p>在对一组癌症样本进行体细胞癌突变检测后，可以进行生物信息学计算分析，以确定可能的功能突变和可能的驱动突变。通常鉴定的方法主要有以下三种：绘制突变图，评估蛋白质或调节元件功能突变的影响，以及在肿瘤群中寻找阳性标记。这些方法不一定是顺序的，但是不同方法的元素之间有重要的优先级关系。每个步骤使用不同的工具。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_22" class="kx_ref">[22]</a></sup> </p> 
<h3>2.4 <span>操纵子组</span></h3> 
<p>操纵子组旨在整合基因组学、转录组学和蛋白质组学，以了解癌症发展的分子机制。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_23" class="kx_ref">[23]</a></sup> </p></div></div><div id="par_14995439153578501"><h2 class="title">3 比较肿瘤基因组学<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>比较肿瘤基因组学通过使用跨物种的基因组比较来识别癌基因。这项研究包括研究模型生物(如小鼠)的癌症基因组、转录组和蛋白质组，从而识别潜在的癌基因，然后参考人类癌症样本来鉴别这些癌基因的同源基因是否对人类癌症的发生扮演重要的角色。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_24" class="kx_ref">[24]</a></sup> 研究发现老鼠模型中的基因改变与人类癌症相似。这些模型是通过逆转录病毒介导插入导致变异或癌细胞转染等方法来产生。 </p></div></div><div id="par_14995439153578502"><h2 class="title">4 癌症驱动突变的来源，癌症突变形成<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>突变为进化中的自然选择提供了原材料，可能是由基因复制错误、外源诱变剂的作用或内源基因损伤引起。复制和基因组维持的机制可能因突变而受损，或因癌症的生理条件和表达水平的差异而改变。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_25" class="kx_ref">[25]</a></sup> </p>
<p>正如Gao等人所指出的那样，<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_26" class="kx_ref">[26]</a></sup>人类基因组的稳定性和完整性由脱氧核糖核酸损伤应答调控系统来维持。未修复的脱氧核糖核酸损伤是导致致癌的突变的主要原因<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_27" class="kx_ref">[27]</a></sup><sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_28" class="kx_ref">[28]</a></sup>。如果脱氧核糖核酸修复有缺陷，脱氧核糖核酸损伤就会逐渐累积。由于跨损伤修复错误的合成反应，这种过度的脱氧核糖核酸损伤会增加脱氧核糖核酸复制过程中的突变错误。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_29" class="kx_ref">[29]</a></sup><sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_30" class="kx_ref">[30]</a></sup>由于DNA修复过程中的错误，过度脱氧核糖核酸损伤也会增加表观遗传的改变。这样的突变和表观遗传改变会导致癌症的发生。DDR基因通常在人类癌症中被表观遗传机制抑制。这种抑制可能涉及启动子区域的脱氧核糖核酸甲基化或小分子核糖核酸对DDR基因的抑制。在许多类型的癌症中，DDR基因的表观遗传学抑制比基因突变更频繁发生。因此，表观遗传学抑制在减少DDR基因表达方面通常比突变发挥更重要的作用。DDR基因表达的减少可能是癌症发生的一个重要驱动因素。 </p>
<p>核苷酸序列周围环境影响突变概率<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_31" class="kx_ref">[31]</a></sup><sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_32" class="kx_ref">[32]</a></sup><sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_33" class="kx_ref">[33]</a></sup>，因此对突变(可变)的脱氧核糖核酸基序的分析对于理解癌症中突变的机制是必不可少的。这些基序代表了脱氧核糖核酸和诱变剂之间、脱氧核糖核酸和修复/复制/修饰酶之间相互作用的指纹图谱。关于基序的例子，如AID作用于WRCY/RGYW基序，将C通过各种作用引入T/G/A的突变（W = A或T，R = 嘌呤 ，Y = 嘧啶）<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_33" class="kx_ref">[33]</a></sup>，WA/TW基序中易错的DNA聚合酶修复机制通过转换或颠换引发AID相关的突变（A转成G/C/G）。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_34" class="kx_ref">[34]</a></sup> </p>
<p>另一种(不可知的)分析肿瘤突变观察到的突变谱和脱氧核糖核酸序列背景的方法是将来自癌症样本中不同突变类型和背景的所有突变汇集成离散分布。如果有多个癌症样本，它们的背景相关突变可以用非负矩阵的形式表示。这个矩阵可以进一步分解成理想状态下描述个体诱变因素的子矩阵(突变信号)。已经提出了几种计算方法来解决这个分解问题。非负矩阵分解(NMF)方法的第一个实现是Mutational Signature Framework，由桑格研究所开发的一个MATLAB包。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_35" class="kx_ref">[35]</a></sup>另一方面，如果仅有来自单个肿瘤样本的突变，DeconstructSigs 包（一个R包）<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_36" class="kx_ref">[36]</a></sup>和MutaGene 服务器<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_37" class="kx_ref">[37]</a></sup>可以提供对单个肿瘤样本的不同突变信号贡献的识别能力。此外，在线MutaGene服务器提供诱变剂或癌症特异性突变背景模型和信号，可用于计算目的脱氧核糖核酸和蛋白质位点突变型，以消除突变和选择在癌症发生中的相对作用。 </p></div></div><div id="par_14995439170355718"><h2 class="title">5 合成致死<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>当两种或多种基因表达缺陷的组合导致细胞死亡，而这些基因中只有一种缺陷不会导致细胞死亡时，合成致死性就会出现。这种缺陷可能是由其中一个基因的突变、表观遗传改变或抑制剂引起的。 </p>
<p>合成致死作为潜在的一种有效抗癌策略，治疗潜力正在不断提高。最近，由于罗纳德·德平霍(Ronald A. DePinho)及其同事在内的科学家在所谓的“附带性致死”方面的工作，合成致死性在靶向癌症治疗中的适用性有所提高。穆勒（Muller）等人发现，染色体上接近肿瘤抑制基因的乘客基因在某些癌症中被间接删除。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_38" class="kx_ref">[38]</a></sup>因此，对执行基本细胞功能的间接删除的冗余基因的鉴定可能是随后寻求合成致死方法的尚未开发的储存库。因此，间接致死在肿瘤学中识别新的选择性治疗靶点方面具有巨大潜力。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_39" class="kx_ref">[39]</a></sup>2012年，穆勒（Muller）等人发现人类胶质母细胞瘤(GBM)中冗余必需糖酵解ENO1基因纯合缺失是接近1p36肿瘤抑制基因位点缺失的结果，并可能为合成GBM抑制提供一种合成致死方法。 <sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_38" class="kx_ref">[38]</a></sup>ENO1是编码哺乳动物α-烯醇酶的三个同源基因(ENO2，ENO3)之一。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_40" class="kx_ref">[40]</a></sup>编码烯醇化酶2的ENO2主要在神经组织中表达，因此，推测在ENO1缺失的GBM中，ENO2可能是作为ENO1冗余同源基因的理想靶位。 穆勒（Muller）发现，在具有纯合EN1缺失的GBM细胞中，基因和药理学上的EN2抑制通过选择性杀死GBM细胞而引发合成致死。2016年，穆勒（Muller）及其同事发现抗生素SF2312是一种高效的纳米级烯醇化酶抑制剂，优先抑制神经胶质瘤细胞增殖和ENO1缺失细胞的糖酵解速率。SF2312被证明比泛烯醇化酶抑制剂PhAH更有效，并且对ENO1抑制比ENO1具有更高的特异性。同一小组随后的工作表明，同样的方法可以应用于胰腺癌，纯合缺失的SMAD4导致线粒体苹果酸酶2 (ME2)的侧支缺失，线粒体苹果酸酶2是氧化还原体内平衡所必需的氧化脱羧酶。 Dey等人表明，胰腺导管腺癌细胞中的ME2基因组缺失导致高内源性活性氧，与KRAS驱动的胰腺癌相一致，并且基本上通过耗尽多余的NAD(P)+-依赖亚型ME3来激发ME2-null细胞合成致死性。发现ME3缺失的影响是通过抑制由AMPK活化和线粒体活性氧介导的凋亡导致的从头核苷酸合成来介导的。与此同时，奥科（Oike）等人通过在代谢以外的过程中定位多余的必须基因，即染色质重塑SWI/SNF复合体中的SMARCA4和SMARCA2亚单位，证明了这一概念的普遍性。 </p>
<p>合成致命性作为有效抗癌策略的治疗潜力正在不断提高。最近，由于包括罗纳德·德平霍和他的同事在内的科学家最近在所谓的“间接致死”方面的工作，合成致死在靶向癌症治疗中的适用性得到了提高。Muller等人发现，染色体接近肿瘤抑制基因的乘客基因在一些癌症中被间接删除。 因此，识别执行基本细胞功能的并列缺失的冗余基因可能是随后寻求合成致死方法的未开发的储库。因此，间接致死在肿瘤学中识别新的选择性治疗靶点方面具有巨大潜力。 2012年，Muller等人发现人类胶质母细胞瘤(GBM)中冗余必需糖酵解ENO1基因纯合缺失是接近1p36肿瘤抑制基因位点缺失的结果，并可能对GBM抑制具有合成致死性途径的潜力。 ENO1是编码哺乳动物α-烯醇酶的三个同源基因(ENO2，ENO3)之一。 编码烯醇酶2的ENO2主要在神经组织中表达，导致推测在ENO1缺失的GBM中，ENO2可能是ENO1的冗余同源物的理想靶标。 穆勒发现，基因和药理学上抑制具有纯合ENO1缺失的GBM细胞的ENO2，通过选择性杀死GBM细胞，引发合成致死结果。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_38" class="kx_ref">[38]</a></sup> 2016年，穆勒和他的同事发现抗生素SF2312是一种高效的纳米级烯醇化酶抑制剂，优先抑制神经胶质瘤细胞增殖和ENO1缺失细胞的糖酵解通量。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_41" class="kx_ref">[41]</a></sup> SF2312被证明比泛烯醇化酶抑制剂PhAH更有效，并且对ENO2的抑制比ENO1更特异。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_41" class="kx_ref">[41]</a></sup> 同一个小组随后的工作表明，同样的方法可以应用于胰腺癌，由此纯合缺失的SMAD4导致线粒体苹果酸酶2 (ME2)的侧枝缺失，线粒体2是氧化还原体内平衡所必需的氧化脱羧酶。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_42" class="kx_ref">[42]</a></sup> Dey等人表明，胰腺导管腺癌细胞中的ME2基因组缺失导致高内源性活性氧物种，这与KRAS驱动的胰腺癌相一致，并且基本上通过耗尽多余的NAD(P)+-依赖性亚型ME3来激发ME2-空细胞的合成致死性。发现ME3耗竭的影响是通过抑制由AMPK激活和线粒体活性氧介导的凋亡引起的从头核苷酸合成来介导的。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_43" class="kx_ref">[43]</a></sup><sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_42" class="kx_ref">[42]</a></sup> 同时，Oike等人通过靶向代谢以外过程中的冗余必需基因，即染色质重塑SWI/SNF复合物中的SMARCA4和SMARCA2亚单位，证明了该概念的普遍性。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_44" class="kx_ref">[44]</a></sup> </p>
<p>一些癌基因对所有细胞(不仅仅是癌细胞)的生存至关重要。因此，敲除这些致癌基因(从而杀死癌细胞)的药物也可能损害正常细胞，导致重大疾病。然而，其他基因可能对癌细胞至关重要，但对健康细胞却不是。 </p>
<p>基于合成致死原理的治疗延长了癌症患者的生存期，并显示出逆转癌症发生的未来进展的前景。 一种主要的合成致命性作用于通常引发癌症的DNA修复缺陷，并且仍然存在于肿瘤细胞中。 这里给出了一些例子。 </p>
<p>BRCA1或BRCA2在大多数高级乳腺癌和卵巢癌中表达不足，通常是由于其启动子的表观遗传甲基化或过度表达的microRNA的表观遗传抑制(见BRCA1和BRCA2文章)。 BRCA1和BRCA2是双链断裂同源重组修复主要途径的重要组成部分。 如果其中一个有缺陷，它会增加患癌症的风险，尤其是乳腺癌或卵巢癌。 一条备用的脱氧核糖核酸修复途径，对于一些通常由BRCA1和BRCA2修复的损伤，依赖于PARP1。 因此，许多卵巢癌对美国食品和药物管理局批准的PARP抑制剂治疗有反应，导致BRCA1或BRCA2缺陷的癌细胞合成致死。 在2016年的三期临床试验中，乳腺癌和许多其他癌症也在评估这种治疗方法。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_45" class="kx_ref">[45]</a></sup> </p>
<p>双链断裂的同源重组修复有两种途径。 主要途径依赖于BRCA1、PALB2和BRCA2，而另一种途径依赖于RAD52。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_46" class="kx_ref">[46]</a></sup> 临床前研究，包括表观遗传学上减少或突变的BRCA缺陷细胞(在培养中或注射到小鼠体内)，表明RAD52的抑制对BRCA缺陷综合致死。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_47" class="kx_ref">[47]</a></sup> </p>
<p>用于DNA错配修复的基因突变导致高突变率。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_48" class="kx_ref">[48]</a></sup> 在肿瘤中，这种频繁的后续突变通常会产生“非自身”免疫原性抗原。 一项有41名患者参与的人类二期临床试验评估了一种针对有或无基质金属蛋白酶缺陷肿瘤的合成致死方法。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_49" class="kx_ref">[49]</a></sup> 基因的产物 <i>PD-1</i> 通常抑制细胞毒性免疫反应。 抑制这种基因可以产生更强的免疫反应。 当肿瘤中有基质金属蛋白酶缺陷的癌症患者暴露于PD-1抑制剂时，67% - 78%的患者经历了免疫相关的无恶化生存期。 相比之下，对于MMR无缺陷的患者，PD-1抑制剂的添加仅产生11%的免疫相关无恶化生存期病患者。 因此，PD-1的抑制主要是由于基质金属蛋白酶缺陷而综合致死。 </p>
<p>ARID1A，一种染色质修饰物，是非同源末端连接所必需的，非同源末端连接是修复DNA双链断裂的主要途径，<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_50" class="kx_ref">[50]</a></sup> 并且还具有转录调节作用。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_51" class="kx_ref">[51]</a></sup> ARID1A突变是12种最常见的致癌突变之一。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_52" class="kx_ref">[52]</a></sup> 突变或表观遗传学降低的表达<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_53" class="kx_ref">[53]</a></sup> 在17种癌症中发现了ARID1A。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_54" class="kx_ref">[54]</a></sup> 在细胞和小鼠中进行的临床前研究表明，ARID1A缺乏症的合成致死率是通过抑制EZ2的甲基转移酶活性，<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_55" class="kx_ref">[55]</a></sup><sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_56" class="kx_ref">[56]</a></sup> 或者加入激酶抑制剂达沙替尼。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_57" class="kx_ref">[57]</a></sup> </p>
<p>另一种方法是单独敲除基因组中的每个基因，观察对正常细胞和癌细胞的影响。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_58" class="kx_ref">[58]</a></sup><sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_59" class="kx_ref">[59]</a></sup> 如果一个非必需基因的敲除对健康细胞影响很小或者没有影响，但是对含有突变癌基因的癌细胞是致命的，那么系统范围内对被抑制基因的抑制可以破坏癌细胞，同时使健康细胞相对完好无损。该技术被用于鉴定PARP-1抑制剂以治疗BRCA1/BRCA2相关癌症。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_60" class="kx_ref">[60]</a></sup><sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_61" class="kx_ref">[61]</a></sup> 在这种情况下，PARP-1抑制和癌症相关突变在BRCA基因中的联合存在仅对癌细胞是致命的。 </p></div></div><div id="par_14995439170355719"><h2 class="title">6 癌症研究数据库<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>“Cancer Genome Project”是一项绘制癌症所有体细胞突变的倡议。该项目系统地对原发性肿瘤和癌细胞系基因组的外显子和侧翼剪接点进行测序。 宇宙软件显示这些实验产生的数据。截至2008年2月，CGP已经在1，848个肿瘤中鉴定出4，746个基因和2，985个突变。 </p>
<p>癌症基因组解剖学项目包括癌症基因组、转录组和蛋白质组的研究信息。 </p>
<p>Progenetix是一个致癌基因参考数据库，提供细胞遗传学和分子细胞遗传学肿瘤数据。 </p>
<p>安康尼从癌症转录组概况中收集了数据。 </p>
<p>IntOGen和Gitools数据集的整合肿瘤组学数据库整合了按肿瘤类型分类的多维人类肿瘤组学数据。IntOGen的第一个版本关注的是基因表达失控和CNV在癌症中的作用。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_62" class="kx_ref">[62]</a></sup> 后来的版本强调了28种肿瘤类型的突变癌症驱动基因。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_63" class="kx_ref">[63]</a></sup><sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_64" class="kx_ref">[64]</a></sup> IntOGen数据的所有版本都可以在IntOGen数据库中获得。 </p>
<p>国际癌症基因组联盟是收集人类癌症基因组数据的最大项目。数据可通过ICGC网站获取。生物表达肿瘤学套件包含来自原发性、转移性和良性肿瘤样品以及正常样品(包括匹配的相邻对照)的基因表达数据。该套件包括许多著名癌症的血液恶性肿瘤样本。 </p>
<p>模型动物的特定数据库包括逆转录病毒标记癌症基因数据库(RTCGD)，该数据库汇集了对小鼠肿瘤中逆转录病毒和转座子插入突变的研究。 </p></div></div><div id="par_14995439187132936"><h2 class="title">7 基因家族<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>对整个基因家族的突变分析表明，同一个家族的基因具有相似的功能，正如相似的编码序列和蛋白质结构域所预测的那样。激酶家族和磷酸酶家族是两个这样的类别，激酶家族参与向蛋白质中添加磷酸基团，磷酸酶家族参与从蛋白质中去除磷酸基团。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_65" class="kx_ref">[65]</a></sup> 因为这些家族在传导细胞生长或死亡的细胞信号中的明显作用，所以首先对其进行了检查。特别是，超过50%的结直肠癌携带激酶或磷酸酶基因突变。磷脂酰肌醇d 3激酶(磷脂酰肌醇-3-激酶)基因编码脂质激酶，其通常包含结肠直肠癌、乳腺癌、胃癌、肺癌和各种其他癌症的突变。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_66" class="kx_ref">[66]</a></sup><sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_67" class="kx_ref">[67]</a></sup> 药物疗法可以抑制磷脂酰肌醇-3-激酶。另一个例子是BRAF基因，它是第一个涉及黑色素瘤的基因之一。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_68" class="kx_ref">[68]</a></sup> BRAF编码参与RAS-RAF-MAPK生长信号通路的丝氨酸/苏氨酸激酶。在59%的黑素瘤中，BRAF突变导致组成性磷酸化和活性。在BRAF之前，黑色素瘤发展的遗传机制尚不清楚，因此患者的预后很差。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_69" class="kx_ref">[69]</a></sup> </p></div></div><div id="par_14995439187132937"><h2 class="title">8 线粒体dna<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>线粒体DNA (mtDNA)突变与肿瘤的形成有关。已经确定了四种类型的mtDNA突变:<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_70" class="kx_ref">[70]</a></sup> </p> 
<h3>8.1 <span>点突变</span></h3> 
<p>在癌细胞包含的mtDNA编码区和非编码区观察到了点突变。在患有膀胱癌、头颈癌和肺癌的个体中，编码区内的点突变显示出彼此相似的迹象。这表明，当健康细胞转化为肿瘤细胞(肿瘤转化)时，线粒体似乎变得同质。位于癌线粒体非编码区D环的大量点突变表明，该区的突变可能是某些癌症的重要特征。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_70" class="kx_ref">[70]</a></sup> </p> 
<h3>8.2 <span>删除</span></h3> 
<p>这种类型的突变由于其小尺寸(&lt; 1kb)而偶尔被检测到。在多种类型的癌症中出现特定的mtDNA突变(复合物1亚基基因ND1中的264-bp缺失和66-bp缺失)提供了一些证据，表明小mtDNA缺失可能出现在肿瘤发生的开始。这也表明，随着肿瘤的进展，含有这些缺失的线粒体数量增加。一个例外是出现在许多癌症中的相对较大的缺失(称为“常见缺失”)，但是与肿瘤细胞相比，在正常细胞中发现了更多的mtDNA大规模缺失。这可能是由于肿瘤细胞表面上的适应过程消除了任何含有这些大规模缺失的线粒体(“常见缺失”大于4kb)。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_70" class="kx_ref">[70]</a></sup> </p> 
<h3>8.3 <span>插入</span></h3> 
<p>乳腺癌、胃癌、肝细胞癌(HCC)和结肠癌以及正常细胞中可能存在约260和约520 bp的两个小mtDNA插入。这些插入物和癌症之间没有相关性。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_71" class="kx_ref">[71]</a></sup> </p> 
<h3>8.4 <span>拷贝数突变</span></h3> 
<p>通过实时聚合酶链反应试验对mtDNA进行的表征表明，在许多癌症中存在mtDNA拷贝数的定量变化。由于氧化应激，拷贝数预计会增加。另一方面，减少被认为是由在复制起点的H链和/或在D环区域的D310均聚c-伸展的体细胞点突变，p53(肿瘤抑制基因)介导的途径的突变和/或由于POLG突变的低效酶活性引起的。拷贝数的任何增加/减少在肿瘤细胞内保持不变。肿瘤细胞中的mtDNA量是恒定的这一事实表明，肿瘤细胞中的mtDNA量是由一个复杂得多的系统控制的，而不是简单地由于异常细胞增殖而改变。对于特定的肿瘤类型或部位，mtDNA含量在人类癌症中的作用明显不同。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_70" class="kx_ref">[70]</a></sup> </p> 
<table class="wikitable"> 
 <caption>
  不同癌症中线粒体基因的突变 
 </caption> 
 <tbody>
  <tr> 
   <th rowspan="2">癌症类型 </th> 
   <th colspan="4">点突变的位置 </th> 
   <th rowspan="2">核苷酸缺失位置 </th> 
   <th rowspan="2">mtDNA拷贝数增加 </th> 
   <th rowspan="2">mtDNA拷贝数减少 </th>
  </tr> 
  <tr> 
   <th>mtDNA控制区 </th> 
   <th>mRNAs </th> 
   <th>tRNAs </th> 
   <th>rRNAs </th>
  </tr> 
  <tr> 
   <td>膀胱癌<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_72" class="kx_ref">[72]</a></sup></td> 
   <td>X</td> 
   <td>X</td> 
   <td></td> 
   <td>X</td> 
   <td>15,642-15,662</td> 
   <td></td> 
   <td> </td>
  </tr> 
  <tr> 
   <td>乳腺癌<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_73" class="kx_ref">[73]</a></sup><sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_74" class="kx_ref">[74]</a></sup><sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_75" class="kx_ref">[75]</a></sup><sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_76" class="kx_ref">[76]</a></sup></td> 
   <td>X</td> 
   <td>X</td> 
   <td>X</td> 
   <td>X</td> 
   <td>8470-13,447和8482-13459</td> 
   <td></td> 
   <td>X </td>
  </tr> 
  <tr> 
   <td>头颈癌<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_73" class="kx_ref">[73]</a></sup><sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_77" class="kx_ref">[77]</a></sup><sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_78" class="kx_ref">[78]</a></sup></td> 
   <td>X</td> 
   <td>X</td> 
   <td>X</td> 
   <td>X</td> 
   <td>8470-13,447和 8482-13459</td> 
   <td>X</td> 
   <td> </td>
  </tr> 
  <tr> 
   <td>口腔癌<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_79" class="kx_ref">[79]</a></sup></td> 
   <td>X</td> 
   <td>X</td> 
   <td></td> 
   <td></td> 
   <td>8470-13,447和8482-13459</td> 
   <td></td> 
   <td> </td>
  </tr> 
  <tr> 
   <td>肝细胞癌(HCC)<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_80" class="kx_ref">[80]</a></sup><sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_81" class="kx_ref">[81]</a></sup></td> 
   <td>X</td> 
   <td>X</td> 
   <td>X</td> 
   <td>X</td> 
   <td>306-556和3894-3960</td> 
   <td></td> 
   <td>X </td>
  </tr> 
  <tr> 
   <td>食管癌<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_82" class="kx_ref">[82]</a></sup></td> 
   <td>X</td> 
   <td>X</td> 
   <td></td> 
   <td>X</td> 
   <td>8470-13,447和8482-13459</td> 
   <td>X</td> 
   <td> </td>
  </tr> 
  <tr> 
   <td>胃癌 <sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_83" class="kx_ref">[83]</a></sup><sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_84" class="kx_ref">[84]</a></sup><sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_85" class="kx_ref">[85]</a></sup></td> 
   <td>X</td> 
   <td>X</td> 
   <td>X</td> 
   <td></td> 
   <td>298-348</td> 
   <td></td> 
   <td>X </td>
  </tr> 
  <tr> 
   <td>前列腺癌<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_86" class="kx_ref">[86]</a></sup><sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_87" class="kx_ref">[87]</a></sup></td> 
   <td>X</td> 
   <td></td> 
   <td></td> 
   <td>X</td> 
   <td>8470-13,447和 8482-13459</td> 
   <td>X</td> 
   <td> </td>
  </tr>
 </tbody>
</table> 
<p>57.7% (500/867)的癌症含体细胞点突变，在调查的1172个突变中，37.8% (443/1127)位于mtDNA控制区，13.1% (154/1172)位于tRNA或rRNA基因，49.1% (575/1127)位于负责产生线粒体呼吸所需复合物所需的mRNA基因上。 </p> 
<h3>8.5 <span>诊断应用</span></h3> 
<p>一些抗癌药物以mtDNA为目标，并显示出杀死肿瘤细胞的积极效果。因此，一些研究使用线粒体突变作为癌细胞治疗的生物标记。线粒体基因组比细胞核基因组更小，更容易筛选特定的突变，因此更容易定位线粒体基因组中的突变。血液样本中发现的mtDNA含量变化可以作为预测未来癌症易感性以及跟踪恶性肿瘤进展的筛选标记。除了mtDNA这些潜在的有益特性外，mtDNA不受细胞周期的控制，对维持三磷酸腺苷的生成和线粒体内环境稳定很重要。这些特点使得靶向mtDNA成为一种可行的治疗策略。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_70" class="kx_ref">[70]</a></sup> </p></div></div><div id="par_14995439187132938"><h2 class="title">9 肿瘤生物标志物<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>目前，多种生物标志物可用于癌症分期、预后和治疗。它们的范围分布广泛，例如：单核苷酸多态性、染色体畸变、DNA拷贝数的变化、微卫星不稳定性、启动子区甲基化，甚至是高/低蛋白质表达水平。<sup><a href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/d10636.htm#quote_88" class="kx_ref">[88]</a></sup> </p></div></div></div></div><div id="references"><h2 class="title" id="par_references">参考文献</h2><ul class="references"><li id="quote_1"><span class="references-num">[1]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Strausberg RL, Simpson AJ, Old LJ, Riggins GJ (May 2004). "Oncogenomics and the development of new cancer therapies". Nature. 429 (6990): 469–74. Bibcode:2004Natur.429..469S. doi:10.1038/nature02627. PMID 15164073..</span></p></li><li id="quote_2"><span class="references-num">[2]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Ting AH, McGarvey KM, Baylin SB (2006). "The cancer epigenome--components and functional correlates". Genes Dev. 20 (23): 3215–31. doi:10.1101/gad.1464906. PMID 17158741..</span></p></li><li id="quote_3"><span class="references-num">[3]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Jones PA, Baylin SB (2007). "The epigenomics of cancer". Cell. 128 (4): 683–92. doi:10.1016/j.cell.2007.01.029. PMC 3894624. PMID 17320506..</span></p></li><li id="quote_4"><span class="references-num">[4]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Li D, Zhang B, Xing X, Wang T (2015). "Combining MeDIP-seq and MRE-seq to investigate genome-wide CpG methylation". Methods. 72: 29–40. doi:10.1016/j.ymeth.2014.10.032. PMC 4300244. PMID 25448294..</span></p></li><li id="quote_5"><span class="references-num">[5]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Wei J, Li G, Dang S, Zhou Y, Zeng K, Liu M (2016). "Discovery and Validation of Hypermethylated Markers for Colorectal Cancer". Dis. Markers. 2016: 2192853. doi:10.1155/2016/2192853. PMC 4963574. PMID 27493446..</span></p></li><li id="quote_6"><span class="references-num">[6]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Strausberg RL, Simpson AJ (January 2010). "Whole-genome cancer analysis as an approach to deeper understanding of tumour biology". Br. J. Cancer. 102 (2): 243–8. doi:10.1038/sj.bjc.6605497. PMC 2816661. PMID 20029419..</span></p></li><li id="quote_7"><span class="references-num">[7]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Vogelstein B, Papadopoulos N, Velculescu VE, Zhou S, Diaz LA, Kinzler KW (2013). "Cancer genome landscapes". Science. 339 (6127): 1546–58. Bibcode:2013Sci...339.1546V. doi:10.1126/science.1235122. PMC 3749880. PMID 23539594..</span></p></li><li id="quote_8"><span class="references-num">[8]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Luo Y, Wong CJ, Kaz AM, Dzieciatkowski S, Carter KT, Morris SM, Wang J, Willis JE, Makar KW, Ulrich CM, Lutterbaugh JD, Shrubsole MJ, Zheng W, Markowitz SD, Grady WM (2014). "Differences in DNA methylation signatures reveal multiple pathways of progression from adenoma to colorectal cancer". Gastroenterology. 147 (2): 418–29.e8. doi:10.1053/j.gastro.2014.04.039. PMC 4107146. PMID 24793120..</span></p></li><li id="quote_9"><span class="references-num">[9]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Ley TJ, Mardis ER, Ding L, Fulton B, McLellan MD, Chen K, et al. (November 2008). "DNA sequencing of a cytogenetically normal acute myeloid leukaemia genome". Nature. 456 (7218): 66–72. Bibcode:2008Natur.456...66L. doi:10.1038/nature07485. PMC 2603574. PMID 18987736..</span></p></li><li id="quote_10"><span class="references-num">[10]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Peikoff, Kira (2015-10-16). "What Miraculous Recoveries Tell Us About Beating Cancer". Popular Mechanics. Retrieved 2016-04-28..</span></p></li><li id="quote_11"><span class="references-num">[11]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Bardelli A.; Velculescu V.E. (2005). "Mutational analysis of gene families in human cancer". Current Opinion in Genetics &amp; Development. 15 (1): 5–12. doi:10.1016/j.gde.2004.12.009..</span></p></li><li id="quote_12"><span class="references-num">[12]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Benvenuti S.; Arena S.; Bardelli A. (2005). "Identification of cancer genes by mutational profiling of tumor genomes". FEBS Letters. 579 (8): 1884–1890. doi:10.1016/j.febslet.2005.02.015. PMID 15763568..</span></p></li><li id="quote_13"><span class="references-num">[13]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Shih I.M.; Wang T.L. (2005). "Apply innovative technologies to explore cancer genome". Current Opinion in Oncology. 17 (1): 33–38. doi:10.1097/01.cco.0000147382.97085.e4. PMID 15608510..</span></p></li><li id="quote_14"><span class="references-num">[14]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Greshock J, Naylor TL, Margolin A, Diskin S, Cleaver SH, Futreal PA, et al. (January 2004). "1-Mb resolution array-based comparative genomic hybridization using a BAC clone set optimized for cancer gene analysis". Genome Res. 14 (1): 179–87. doi:10.1101/gr.1847304. PMC 314295. PMID 14672980..</span></p></li><li id="quote_15"><span class="references-num">[15]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Lucito R, Healy J, Alexander J, Reiner A, Esposito D, Chi M, et al. (October 2003). "Representational oligonucleotide microarray analysis: a high-resolution method to detect genome copy number variation". Genome Res. 13 (10): 2291–305. doi:10.1101/gr.1349003. PMC 403708. PMID 12975311..</span></p></li><li id="quote_16"><span class="references-num">[16]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Hu M, Yao J, Polyak K (2006). "Methylation-specific digital karyotyping". Nat Protoc. 1 (3): 1621–36. doi:10.1038/nprot.2006.278. PMID 17406428..</span></p></li><li id="quote_17"><span class="references-num">[17]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Körner H, Epanchintsev A, Berking C, Schuler-Thurner B, Speicher MR, Menssen A, Hermeking H (January 2007). "Digital karyotyping reveals frequent inactivation of the dystrophin/DMD gene in malignant melanoma". Cell Cycle. 6 (2): 189–98. doi:10.4161/cc.6.2.3733. PMID 17314512..</span></p></li><li id="quote_18"><span class="references-num">[18]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Volik S, Zhao S, Chin K, Brebner JH, Herndon DR, Tao Q, et al. (June 2003). "End-sequence profiling: sequence-based analysis of aberrant genomes". Proc. Natl. Acad. Sci. U.S.A. 100 (13): 7696–701. doi:10.1073/pnas.1232418100. PMC 164650. PMID 12788976..</span></p></li><li id="quote_19"><span class="references-num">[19]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Van, de Vijver M.J.; et al. (2002). "A gene-expression signature as a predictor of survival in breast cancer". New England Journal of Medicine. 347 (25): 1999–2009. doi:10.1056/NEJMoa021967. PMID 12490681..</span></p></li><li id="quote_20"><span class="references-num">[20]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">van 't Veer LJ, Dai H, van de Vijver MJ, He YD, Hart AA, Bernards R, Friend SH (2003). "Expression profiling predicts outcome in breast cancer". Breast Cancer Res. 5 (1): 57–8. doi:10.1186/bcr562. PMC 154139. PMID 12559048..</span></p></li><li id="quote_21"><span class="references-num">[21]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Xu Q.; Lee C. (2003). "Discovery of novel splice forms and functional analysis of cancer-specific alternative splicing in human expressed sequences". Nucleic Acids Research. 31 (19): 5635–5643. doi:10.1093/nar/gkg786. PMC 206480. PMID 14500827..</span></p></li><li id="quote_22"><span class="references-num">[22]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Gonzalez-Perez A, Mustonen V, Reva B, Ritchie GR, Creixell P, Karchin R, et al. (August 2013). "Computational approaches to identify functional genetic variants in cancer genomes". Nat. Methods. 10 (8): 723–9. doi:10.1038/nmeth.2562. PMC 3919555. PMID 23900255..</span></p></li><li id="quote_23"><span class="references-num">[23]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Hanash SM (September 2000). "Operomics: molecular analysis of tissues from DNA to RNA to protein". Clin. Chem. Lab. Med. 38 (9): 805–13. doi:10.1515/CCLM.2000.116. PMID 11097332..</span></p></li><li id="quote_24"><span class="references-num">[24]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Peeper D, Berns A (June 2006). "Cross-species oncogenomics in cancer gene identification". Cell. 125 (7): 1230–3. doi:10.1016/j.cell.2006.06.018. PMID 16814709..</span></p></li><li id="quote_25"><span class="references-num">[25]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Rogozin IB, Pavlov YI, Goncearenco A, De S, Lada AG, Poliakov E, Panchenko AR, Cooper DN (November 2018). "Mutational signatures and mutable motifs in cancer genomes". Brief. Bioinformatics. 19 (6): 1085–1101. doi:10.1093/bib/bbx049. PMID 28498882..</span></p></li><li id="quote_26"><span class="references-num">[26]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Gao D, Herman JG, Guo M (2016). "The clinical value of aberrant epigenetic changes of DNA damage repair genes in human cancer". Oncotarget. 7: 37331–37346. doi:10.18632/oncotarget.7949. PMC 5095080. PMID 26967246..</span></p></li><li id="quote_27"><span class="references-num">[27]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Kastan MB (2008). "DNA damage responses: mechanisms and roles in human disease: 2007 G.H.A. Clowes Memorial Award Lecture". Mol. Cancer Res. 6 (4): 517–24. doi:10.1158/1541-7786.MCR-08-0020. PMID 18403632..</span></p></li><li id="quote_28"><span class="references-num">[28]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Bernstein, C; Prasad, AR; Nfonsam, V; Bernstein, H. (2013). "Chapter 16: DNA Damage, DNA Repair and Cancer". In Chen, Clark. New Research Directions in DNA Repair,. p. 413. ISBN 978-953-51-1114-6..</span></p></li><li id="quote_29"><span class="references-num">[29]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">O'Hagan HM, Mohammad HP, Baylin SB (2008). "Double strand breaks can initiate gene silencing and SIRT1-dependent onset of DNA methylation in an exogenous promoter CpG island". PLoS Genetics. 4 (8): e1000155. doi:10.1371/journal.pgen.1000155. PMC 2491723. PMID 18704159..</span></p></li><li id="quote_30"><span class="references-num">[30]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Cuozzo C, Porcellini A, Angrisano T, et al. (July 2007). "DNA damage, homology-directed repair, and DNA methylation". PLoS Genetics. 3 (7): e110. doi:10.1371/journal.pgen.0030110. PMC 1913100. PMID 17616978..</span></p></li><li id="quote_31"><span class="references-num">[31]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Coulondre C, Miller JH, Farabaugh PJ, Gilbert W (August 1978). "Molecular basis of base substitution hotspots in Escherichia coli". Nature. 274 (5673): 775–80. Bibcode:1978Natur.274..775C. doi:10.1038/274775a0. PMID 355893..</span></p></li><li id="quote_32"><span class="references-num">[32]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Cooper DN, Youssoufian H (February 1988). "The CpG dinucleotide and human genetic disease". Hum. Genet. 78 (2): 151–5. doi:10.1007/bf00278187. PMID 3338800..</span></p></li><li id="quote_33"><span class="references-num">[33]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Rogozin IB, Kolchanov NA (November 1992). "Somatic hypermutagenesis in immunoglobulin genes. II. Influence of neighbouring base sequences on mutagenesis". Biochim. Biophys. Acta. 1171 (1): 11–8. doi:10.1016/0167-4781(92)90134-l. ISSN 0006-3002. PMID 1420357..</span></p></li><li id="quote_34"><span class="references-num">[34]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Rogozin IB, Pavlov YI, Bebenek K, Matsuda T, Kunkel TA (June 2001). "Somatic mutation hotspots correlate with DNA polymerase eta error spectrum". Nat. Immunol. 2 (6): 530–6. doi:10.1038/88732. ISSN 1529-2908. PMID 11376340..</span></p></li><li id="quote_35"><span class="references-num">[35]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Alexandrov LB, Nik-Zainal S, Wedge DC, Campbell PJ, Stratton MR (January 2013). "Deciphering signatures of mutational processes operative in human cancer". Cell Rep. 3 (1): 246–59. doi:10.1016/j.celrep.2012.12.008. PMC 3588146. PMID 23318258..</span></p></li><li id="quote_36"><span class="references-num">[36]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Rosenthal R, McGranahan N, Herrero J, Taylor BS, Swanton C (February 2016). "DeconstructSigs: delineating mutational processes in single tumors distinguishes DNA repair deficiencies and patterns of carcinoma evolution". Genome Biol. 17: 31. doi:10.1186/s13059-016-0893-4. PMC 4762164. PMID 26899170..</span></p></li><li id="quote_37"><span class="references-num">[37]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Goncearenco A, Rager SL, Li M, Sang QX, Rogozin IB, Panchenko AR (July 2017). "Exploring background mutational processes to decipher cancer genetic heterogeneity". Nucleic Acids Res. 45 (W1): W514–W522. doi:10.1093/nar/gkx367. PMC 5793731. PMID 28472504..</span></p></li><li id="quote_38"><span class="references-num">[38]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Muller FL, Colla S, Aquilanti E, Manzo VE, Genovese G, Lee J, et al. (August 2012). "Passenger deletions generate therapeutic vulnerabilities in cancer". Nature. 488 (7411): 337–42. Bibcode:2012Natur.488..337M. doi:10.1038/nature11331. PMC 3712624. PMID 22895339..</span></p></li><li id="quote_39"><span class="references-num">[39]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Muller FL, Aquilanti EA, DePinho RA (November 2015). "Collateral Lethality: A new therapeutic strategy in oncology". Trends Cancer. 1 (3): 161–173. doi:10.1016/j.trecan.2015.10.002. PMC 4746004. PMID 26870836..</span></p></li><li id="quote_40"><span class="references-num">[40]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Poyner RR, Reed GH (August 1992). "Structure of the bis divalent cation complex with phosphonoacetohydroxamate at the active site of enolase". Biochemistry. 31 (31): 7166–73. doi:10.1021/bi00146a020. PMID 1322695..</span></p></li><li id="quote_41"><span class="references-num">[41]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Leonard PG, Satani N, Maxwell D, Lin YH, Hammoudi N, Peng Z, et al. (December 2016). "SF2312 is a natural phosphonate inhibitor of enolase". Nat. Chem. Biol. 12 (12): 1053–1058. doi:10.1038/nchembio.2195. PMC 5110371. PMID 27723749..</span></p></li><li id="quote_42"><span class="references-num">[42]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Dey P, Baddour J, Muller F, Wu CC, Wang H, Liao WT, et al. (February 2017). "Genomic deletion of malic enzyme 2 confers collateral lethality in pancreatic cancer". Nature. 542 (7639): 119–123. Bibcode:2017Natur.542..119D. doi:10.1038/nature21052. PMC 5398413. PMID 28099419..</span></p></li><li id="quote_43"><span class="references-num">[43]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Liou GY, Döppler H, DelGiorno KE, Zhang L, Leitges M, Crawford HC, Murphy MP, Storz P (March 2016). "Mutant KRas-Induced Mitochondrial Oxidative Stress in Acinar Cells Upregulates EGFR Signaling to Drive Formation of Pancreatic Precancerous Lesions". Cell Rep. 14 (10): 2325–36. doi:10.1016/j.celrep.2016.02.029. PMC 4794374. PMID 26947075..</span></p></li><li id="quote_44"><span class="references-num">[44]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Oike T, Ogiwara H, Tominaga Y, Ito K, Ando O, Tsuta K, et al. (September 2013). "A synthetic lethality-based strategy to treat cancers harboring a genetic deficiency in the chromatin remodeling factor BRG1". Cancer Res. 73 (17): 5508–18. doi:10.1158/0008-5472.CAN-12-4593. PMID 23872584..</span></p></li><li id="quote_45"><span class="references-num">[45]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Murata S, Zhang C, Finch N, Zhang K, Campo L, Breuer EK (2016). "Predictors and Modulators of Synthetic Lethality: An Update on PARP Inhibitors and Personalized Medicine". Biomed Res Int. 2016: 2346585. doi:10.1155/2016/2346585. PMC 5013223. PMID 27642590..</span></p></li><li id="quote_46"><span class="references-num">[46]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Lok BH, Carley AC, Tchang B, Powell SN (2013). "RAD52 inactivation is synthetically lethal with deficiencies in BRCA1 and PALB2 in addition to BRCA2 through RAD51-mediated homologous recombination". Oncogene. 32 (30): 3552–8. doi:10.1038/onc.2012.391. PMC 5730454. PMID 22964643..</span></p></li><li id="quote_47"><span class="references-num">[47]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Cramer-Morales K, Nieborowska-Skorska M, Scheibner K, Padget M, Irvine DA, Sliwinski T, Haas K, Lee J, Geng H, Roy D, Slupianek A, Rassool FV, Wasik MA, Childers W, Copland M, Müschen M, Civin CI, Skorski T (2013). "Personalized synthetic lethality induced by targeting RAD52 in leukemias identified by gene mutation and expression profile". Blood. 122 (7): 1293–304. doi:10.1182/blood-2013-05-501072. PMC 3744994. PMID 23836560..</span></p></li><li id="quote_48"><span class="references-num">[48]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Eshleman JR, Lang EZ, Bowerfind GK, Parsons R, Vogelstein B, Willson JK, Veigl ML, Sedwick WD, Markowitz SD (1995). "Increased mutation rate at the hprt locus accompanies microsatellite instability in colon cancer". Oncogene. 10 (1): 33–7. PMID 7824277..</span></p></li><li id="quote_49"><span class="references-num">[49]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Le DT, Uram JN, Wang H, Bartlett BR, Kemberling H, Eyring AD, et al. (2015). "PD-1 Blockade in Tumors with Mismatch-Repair Deficiency". N. Engl. J. Med. 372 (26): 2509–20. doi:10.1056/NEJMoa1500596. PMC 4481136. PMID 26028255..</span></p></li><li id="quote_50"><span class="references-num">[50]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Watanabe R, Ui A, Kanno S, Ogiwara H, Nagase T, Kohno T, Yasui A (2014). "SWI/SNF factors required for cellular resistance to DNA damage include ARID1A and ARID1B and show interdependent protein stability". Cancer Res. 74 (9): 2465–75. doi:10.1158/0008-5472.CAN-13-3608. PMID 24788099..</span></p></li><li id="quote_51"><span class="references-num">[51]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Raab JR, Resnick S, Magnuson T (2015). "Genome-Wide Transcriptional Regulation Mediated by Biochemically Distinct SWI/SNF Complexes". PLoS Genet. 11 (12): e1005748. doi:10.1371/journal.pgen.1005748. PMC 4699898. PMID 26716708..</span></p></li><li id="quote_52"><span class="references-num">[52]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Lawrence MS, Stojanov P, Mermel CH, Robinson JT, Garraway LA, Golub TR, Meyerson M, Gabriel SB, Lander ES, Getz G (2014). "Discovery and saturation analysis of cancer genes across 21 tumour types". Nature. 505 (7484): 495–501. Bibcode:2014Natur.505..495L. doi:10.1038/nature12912. PMC 4048962. PMID 24390350..</span></p></li><li id="quote_53"><span class="references-num">[53]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Zhang X, Sun Q, Shan M, Niu M, Liu T, Xia B, Liang X, Wei W, Sun S, Zhang Y, Liu XS, Song Q, Yang Y, Ma Y, Liu Y, Yang L, Ren Y, Zhang G, Pang D (2013). "Promoter hypermethylation of ARID1A gene is responsible for its low mRNA expression in many invasive breast cancers". PLoS ONE. 8 (1): e53931. Bibcode:2013PLoSO...853931Z. doi:10.1371/journal.pone.0053931. PMC 3549982. PMID 23349767..</span></p></li><li id="quote_54"><span class="references-num">[54]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Wu JN, Roberts CW (2013). "ARID1A mutations in cancer: another epigenetic tumor suppressor?". Cancer Discov. 3 (1): 35–43. doi:10.1158/2159-8290.CD-12-0361. PMC 3546152. PMID 23208470..</span></p></li><li id="quote_55"><span class="references-num">[55]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Bitler BG, Aird KM, Garipov A, Li H, Amatangelo M, Kossenkov AV, Schultz DC, Liu Q, Shih IeM, Conejo-Garcia JR, Speicher DW, Zhang R (2015). "Synthetic lethality by targeting EZH2 methyltransferase activity in ARID1A-mutated cancers". Nat. Med. 21 (3): 231–8. doi:10.1038/nm.3799. PMC 4352133. PMID 25686104..</span></p></li><li id="quote_56"><span class="references-num">[56]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Kim KH, Kim W, Howard TP, Vazquez F, Tsherniak A, Wu JN, Wang W, Haswell JR, Walensky LD, Hahn WC, Orkin SH, Roberts CW (2015). "SWI/SNF-mutant cancers depend on catalytic and non-catalytic activity of EZH2". Nat. Med. 21 (12): 1491–6. doi:10.1038/nm.3968. PMC 4886303. PMID 26552009..</span></p></li><li id="quote_57"><span class="references-num">[57]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Miller RE, Brough R, Bajrami I, Williamson CT, McDade S, Campbell J, et al. (2016). "Synthetic Lethal Targeting of ARID1A-Mutant Ovarian Clear Cell Tumors with Dasatinib". Mol. Cancer Ther. 15 (7): 1472–84. doi:10.1158/1535-7163.MCT-15-0554. PMID 27364904..</span></p></li><li id="quote_58"><span class="references-num">[58]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Kaelin W.G. (2005). "The concept of synthetic lethality in the context of anticancer therapy". Nature Reviews Cancer. 5 (9): 689–698. doi:10.1038/nrc1691. PMID 16110319..</span></p></li><li id="quote_59"><span class="references-num">[59]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">O'Connor M.J.; Martin N.M.B.; Smith G.C.M. (2007). "Targeted cancer therapies based on the inhibition of DNA strand break repair". Oncogene. 26 (56): 7816–7824. doi:10.1038/sj.onc.1210879. PMID 18066095..</span></p></li><li id="quote_60"><span class="references-num">[60]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Farmer H, McCabe N, Lord CJ, Tutt AN, Johnson DA, Richardson TB, Santarosa M, Dillon KJ, Hickson I, Knights C, Martin NM, Jackson SP, Smith GC, Ashworth A (April 2005). "Targeting the DNA repair defect in BRCA mutant cells as a therapeutic strategy". Nature. 434 (7035): 917–21. Bibcode:2005Natur.434..917F. doi:10.1038/nature03445. PMID 15829967..</span></p></li><li id="quote_61"><span class="references-num">[61]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Bryant H.E.; Schultz, Niklas; Thomas, Huw D.; Parker, Kayan M.; Flower, Dan; Lopez, Elena; Kyle, Suzanne; Meuth, Mark; Curtin, Nicola J.; Helleday, Thomas; et al. (2005). "Specific killing of BRCA2-deficient tumours with inhibitors of poly(ADP-ribose) polymerase". Nature. 434 (7035): 913–917. Bibcode:2005Natur.434..913B. doi:10.1038/nature03443. PMID 15829966..</span></p></li><li id="quote_62"><span class="references-num">[62]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Gundem G, Perez-Llamas C, Jene-Sanz A, Kedzierska A, Islam A, Deu-Pons J, Furney SJ, Lopez-Bigas N (February 2010). "IntOGen: integration and data mining of multidimensional oncogenomic data". Nat. Methods. 7 (2): 92–3. doi:10.1038/nmeth0210-92. PMID 20111033..</span></p></li><li id="quote_63"><span class="references-num">[63]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Gonzalez-Perez A, Perez-Llamas C, Deu-Pons J, Tamborero D, Schroeder MP, Jene-Sanz A, Santos A, Lopez-Bigas N (November 2013). "IntOGen-mutations identifies cancer drivers across tumor types". Nat. Methods. 10 (11): 1081–2. doi:10.1038/nmeth.2642. PMC 5758042. PMID 24037244..</span></p></li><li id="quote_64"><span class="references-num">[64]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Rubio-Perez C, Tamborero D, Schroeder MP, Antolín AA, Deu-Pons J, Perez-Llamas C, Mestres J, Gonzalez-Perez A, Lopez-Bigas N (March 2015). "In silico prescription of anticancer drugs to cohorts of 28 tumor types reveals targeting opportunities". Cancer Cell. 27 (3): 382–96. doi:10.1016/j.ccell.2015.02.007. PMID 25759023..</span></p></li><li id="quote_65"><span class="references-num">[65]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Blume-Jensen P, Hunter T (May 2001). "Oncogenic kinase signalling". Nature. 411 (6835): 355–65. doi:10.1038/35077225. PMID 11357143..</span></p></li><li id="quote_66"><span class="references-num">[66]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Bardelli A, et al. (2003). "Mutational analysis of the tyrosine kinome in colorectal cancers". Science. 300 (5621): 949–949. doi:10.1126/science.1082596. PMID 12738854..</span></p></li><li id="quote_67"><span class="references-num">[67]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Samuels Y, Wang Z, Bardelli A, Silliman N, Ptak J, Szabo S, Yan H, Gazdar A, Powell SM, Riggins GJ, Willson JK, Markowitz S, Kinzler KW, Vogelstein B, Velculescu VE (April 2004). "High frequency of mutations of the PIK3CA gene in human cancers". Science. 304 (5670): 554. doi:10.1126/science.1096502. PMID 15016963..</span></p></li><li id="quote_68"><span class="references-num">[68]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Davies H, Bignell GR, Cox C, Stephens P, Edkins S, Clegg S, et al. (June 2002). "Mutations of the BRAF gene in human cancer". Nature. 417 (6892): 949–54. doi:10.1038/nature00766. PMID 12068308..</span></p></li><li id="quote_69"><span class="references-num">[69]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Danson S.; Lorigan P. (2005). "Improving outcomes in advanced malignant melanoma - Update on systemic therapy". Drugs. 65 (6): 733–743. doi:10.2165/00003495-200565060-00002. PMID 15819587..</span></p></li><li id="quote_70"><span class="references-num">[70]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Yu, Man (2012). "Somatic Mitochondrial DNA Mutations in Human Cancers". Advances in Clinical Chemistry. Advances in Clinical Chemistry. 57: 99–138. doi:10.1016/B978-0-12-394384-2.00004-8. ISBN 9780123943842. PMID 22870588..</span></p></li><li id="quote_71"><span class="references-num">[71]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Hung, W.Y.; J.C. Lin; L.M. Lee; et al. (2008). "Tandem duplication/triplication correlated with poly-cytosine stretch variation in human mitochondrial DNA D-loop region". Mutagenesis. 23 (2): 137–142. doi:10.1093/mutage/gen002. PMID 18252697..</span></p></li><li id="quote_72"><span class="references-num">[72]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Fliss, M. S.; Usadel, H.; Caballero, O. L.; et al. (2000). "Facile detection of mitochondrial DNA mutations in tumors and bodily fluids". Science. 287 (5460): 2017–2019. Bibcode:2000Sci...287.2017F. doi:10.1126/science.287.5460.2017. PMID 10720328..</span></p></li><li id="quote_73"><span class="references-num">[73]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Dani, M.A.; S.U. Dani; S.P. Lima; et al. (2004). "Less ΔmtDNA4977 than normal in various types of tumors suggests that cancer cells are essentially free of this mutation". Genet. Mol. Res. 3 (3): 395–409. PMID 15614730..</span></p></li><li id="quote_74"><span class="references-num">[74]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Ye, C.; X.O. Shu; W. Wen; et al. (2008). "Quantitative analysis of mitochondrial DNA 4977-bp deletion in sporadic breast cancer and benign breast diseases". Breast Cancer Res. Treat. 108 (3): 427–434. doi:10.1007/s10549-007-9613-9. PMC 3836503. PMID 17541740..</span></p></li><li id="quote_75"><span class="references-num">[75]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Tseng, L.M.; P.H. Yin; C.W. Chi; et al. (2006). "Mitochondrial DNA mutations and mitochondrial DNA depletion in breast cancer". Genes Chromosomes Cancer. 45 (7): 629–638. doi:10.1002/gcc.20326. PMID 16568452..</span></p></li><li id="quote_76"><span class="references-num">[76]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Zhu, W.; W. Qin; P. Bradley; A. Wessel; C.L. Puckett; E.R. Sauter (2005). "Mitochondrial DNA mutations in breast cancer tissue and in matched nipple aspirate fluid". Carcinogenesis. 26 (1): 145–152. doi:10.1093/carcin/bgh282. PMID 15375011..</span></p></li><li id="quote_77"><span class="references-num">[77]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Zhou S, Kachhap S, Sun W, Wu G, Chuang A, Poeta L, Grumbine L, Mithani SK, Chatterjee A, Koch W, Westra WH, Maitra A, Glazer C, Carducci M, Sidransky D, McFate T, Verma A, Califano JA (May 2007). "Frequency and phenotypic implications of mitochondrial DNA mutations in human squamous cell cancers of the head and neck". Proc. Natl. Acad. Sci. U.S.A. 104 (18): 7540–5. doi:10.1073/pnas.0610818104. PMC 1863503. PMID 17456604..</span></p></li><li id="quote_78"><span class="references-num">[78]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Poetsch M, Petersmann A, Lignitz E, Kleist B (March 2004). "Relationship between mitochondrial DNA instability, mitochondrial DNA large deletions, and nuclear microsatellite instability in head and neck squamous cell carcinomas". Diagn. Mol. Pathol. 13 (1): 26–32. doi:10.1097/00019606-200403000-00005. PMID 15163006..</span></p></li><li id="quote_79"><span class="references-num">[79]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Tan, D.J.; J. Chang; W.L. Chen; et al. (2004). "Somatic mitochondrial DNA mutations in oral cancer of betel quid chewers". Ann. N. Y. Acad. Sci. 1011: 310–316. Bibcode:2004NYASA1011..310T. doi:10.1196/annals.1293.030. PMID 15126307..</span></p></li><li id="quote_80"><span class="references-num">[80]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Lee, H.C.; S.H. Li; J.C. Lin; C.C. Wu; D.C. Yeh; Y.H. Wei (2004). "Somatic mutations in the D-loop and decrease in the copy number of mitochondrial DNA in human hepatocellular carcinoma". Mutation Research. 547 (1–2): 71–78. doi:10.1016/j.mrfmmm.2003.12.011. PMID 15013701..</span></p></li><li id="quote_81"><span class="references-num">[81]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Yin, P.H.; C.C. Wu; J.C. Lin; C.W. Chi; Y.H. Wei; H.C. Lee (2010). "Somatic mutations of mitochondrial genome in hepatocellular carcinoma". Mitochondrion. 10 (2): 174–182. doi:10.1016/j.mito.2009.12.147. PMID 20006738..</span></p></li><li id="quote_82"><span class="references-num">[82]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Tan, D.J.; J. Chang; L.L. Liu; et al. (2006). "Significance of somatic mutations and content alteration of mitochondrial DNA in esophageal cancer". BMC Cancer. 6: 93. doi:10.1186/1471-2407-6-93. PMC 1459869. PMID 16620376..</span></p></li><li id="quote_83"><span class="references-num">[83]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Kassauei K, Habbe N, Mullendore ME, Karikari CA, Maitra A, Feldmann G (2006). "Mitochondrial DNA mutations in pancreatic cancer". Int J Gastrointest Cancer. 37 (2–3): 57–64. doi:10.1007/s12029-007-0008-2. PMID 17827523..</span></p></li><li id="quote_84"><span class="references-num">[84]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Hung WY, Wu CW, Yin PH, Chang CJ, Li AF, Chi CW, Wei YH, Lee HC (March 2010). "Somatic mutations in mitochondrial genome and their potential roles in the progression of human gastric cancer". Biochim. Biophys. Acta. 1800 (3): 264–70. doi:10.1016/j.bbagen.2009.06.006. PMID 19527772..</span></p></li><li id="quote_85"><span class="references-num">[85]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Wu CW, Yin PH, Hung WY, Li AF, Li SH, Chi CW, Wei YH, Lee HC (September 2005). "Mitochondrial DNA mutations and mitochondrial DNA depletion in gastric cancer". Genes Chromosomes Cancer. 44 (1): 19–28. doi:10.1002/gcc.20213. PMID 15892105..</span></p></li><li id="quote_86"><span class="references-num">[86]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Yu, J.J.; T. Yan (2010). "Effect of mtDNA mutation on tumor malignant degree in patients with prostate cancer". Aging Male. 13 (3): 159–165. doi:10.3109/13685530903536668. PMID 20136572..</span></p></li><li id="quote_87"><span class="references-num">[87]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Gomez-Zaera, M.; J. Abril; L. Gonzalez; et al. (2006). "Identification of somatic and germline mitochondrial DNA sequence variants in prostate cancer patients". Mutation Research. 595 (1–2): 42–51. doi:10.1016/j.mrfmmm.2005.10.012. PMID 16472830..</span></p></li><li id="quote_88"><span class="references-num">[88]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Ludwig JA, Weinstein JN (November 2005). "Biomarkers in cancer staging, prognosis and treatment selection". Nat. Rev. Cancer. 5 (11): 845–56. doi:10.1038/nrc1739. PMID 16239904..</span></p></li></ul></div><div class="read-num">阅读 <!-- -->39</div></div><div class="right-side" id="rightSide"><div class="side" id="lemma-side"><div class="side-title">版本记录</div><ul class="side-lst"><li><p class="side-lst-txt">暂无</p></li></ul><div class="user-card userCard"></div></div><div class="side"><div class="side-event"></div></div></div></div><div class="footer-box"><div id="footer"><div class="footer-logo-wrap"><div class="footer-logo"></div><div class="footer-logo-text">知识·传播·科普</div></div><div class="footer-info">本网站内容采用<a target="_blank" href="https://web.archive.org/web/20221025184839/https://creativecommons.org/licenses/by-sa/3.0/deed.zh?tdsourcetag=s_pctim_aiomsg">CC-BY-SA 3.0</a>授权</div><div class="footer-btn-wrap"><a target="_blank" href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/help/#user_protocol">用户协议</a><a target="_blank" href="https://web.archive.org/web/20221025184839/http://www.sogou.com/docs/terms.htm?v=1">免责声明</a><a target="_blank" href="https://web.archive.org/web/20221025184839/http://corp.sogou.com/private.html">隐私政策</a><a target="_blank" href="https://web.archive.org/web/20221025184839/https://baike.sogou.com/kexue/intro.htm">关于我们</a></div></div></div><script>window.lemmaInfo ={"lemmaId":"10636","versionId":"14995439136801284","title":"肿瘤基因组学","subtitle":"","abstracts":{"paragraphId":"14995439136801295","title":"简介","versionId":"14995439136801285","lemmaId":10636,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":10145103,"name":"科学百科编辑组","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233977,"comment":null,"dependVersionId":0,"contentType":2,"content":"<p>肿瘤基因组学是基因组学描述癌症相关基因的一个子领域。它关注癌症的基因组、表观基因组和转录组的水平变化。 </p>\n<p> 癌症是一种基因病，由脱氧核糖核酸突变的积累和表观遗传水平的改变导致细胞无限制增殖形成肿瘤。肿瘤基因组学的目标是鉴定新的致癌基因或肿瘤抑制基因，这些基因可能为癌症诊断、癌症的临床结果预测和癌症靶向治疗提供新的方法。格列卫、赫赛汀和阿瓦斯丁等靶向癌症治疗的成功为肿瘤基因组学阐明癌症治疗的新靶点带来了希望。<sup><a href=\"#quote_1\" class=\"kx_ref\">[1]</a></sup></p>\n<p></p> \n<p>除了了解引发或驱动癌症进展的潜在遗传机制，肿瘤基因组学还以个性化癌症治疗为目标。癌症是由于随机积累的基因突变和表观遗传改变而发展的。识别和定位个体患者的突变可能会提高治疗效果。 </p>\n<p>人类基因组计划的完成促进了肿瘤组基因组学领域的发展，并提高了研究人员发现肿瘤基因的能力。测序技术和全基因组甲基化图谱技术已经应用于肿瘤基因组学的研究。 </p>","pics":[{"originalUrl":"https://web.archive.org/web/20221025184839/https://img02.sogoucdn.com/app/a/200698/sogou_science_13465?w=500&h=277&titlename=%E8%82%BF%E7%98%A4%E5%9F%BA%E5%9B%A0%E7%BB%84%E5%AD%A6%E7%9A%84%E6%95%B4%E4%BD%93%E7%9B%AE%E6%A0%87","url":"https://web.archive.org/web/20221025184839/https://img02.sogoucdn.com/app/a/200698/sogou_science_13465","rw":500,"rh":277,"title":"肿瘤基因组学的整体目标","alt":null,"width":0,"height":0}],"card":null,"references":[],"versionCount":0},"card":{"paragraphId":"0","title":null,"versionId":"0","lemmaId":0,"createType":0,"creator":null,"createTime":0,"versionEditor":null,"editTime":0,"comment":null,"dependVersionId":0,"contentType":0,"content":null,"pics":null,"card":null,"references":null,"versionCount":0},"categories":[{"id":13,"name":"药学","parents":[]}],"creator":{"uid":10145103,"name":"柚子otto","pic":"https://web.archive.org/web/20221025184839/https://img02.sogoucdn.com/app/a/200698/1152_1152_1864088_20200427232849-1518843971.png","introduction":"","educations":[{"schoolName":"中国地质大学（北京）","major":"","degree":"本科","universityId":22,"universityLogo":"https://web.archive.org/web/20221025184839/https://img01.sogoucdn.com/app/a/200943/d3465c1c-6011-11e9-b353-fc4dd4f70029","majorLevel1":"理学","majorLevel2":"地理学","majorLevel3":"地图学与地理信息系统","majorLevel1Id":1,"majorLevel2Id":103,"majorLevel3Id":109,"state":"毕业","lab":"","researchField":""}],"jobs":[{"company":"搜狗","title":"产品经理"}],"works":null,"educationBrief":"中国地质大学（北京）","jobBrief":"产品经理","role":0,"roleName":null,"title":"中国地质大学（北京） · 地理学本科","professionalTitle":null,"phoneNo":null,"editable":true,"partnerId":139,"partnerIdCreateTime":1595844881,"partnerIdPoped":true},"createTime":1568626657,"editor":{"uid":10145103,"name":"柚子otto","pic":"https://web.archive.org/web/20221025184839/https://img02.sogoucdn.com/app/a/200698/1152_1152_1864088_20200427232849-1518843971.png","introduction":"","educations":[{"schoolName":"中国地质大学（北京）","major":"","degree":"本科","universityId":22,"universityLogo":"https://web.archive.org/web/20221025184839/https://img01.sogoucdn.com/app/a/200943/d3465c1c-6011-11e9-b353-fc4dd4f70029","majorLevel1":"理学","majorLevel2":"地理学","majorLevel3":"地图学与地理信息系统","majorLevel1Id":1,"majorLevel2Id":103,"majorLevel3Id":109,"state":"毕业","lab":"","researchField":""}],"jobs":[{"company":"搜狗","title":"产品经理"}],"works":null,"educationBrief":"中国地质大学（北京）","jobBrief":"产品经理","role":0,"roleName":null,"title":"中国地质大学（北京） · 地理学本科","professionalTitle":null,"phoneNo":null,"editable":true,"partnerId":139,"partnerIdCreateTime":1595844881,"partnerIdPoped":true},"editTime":1576233977,"state":1,"versionCount":1,"upNum":0,"downNum":0,"pics":[{"originalUrl":"https://web.archive.org/web/20221025184839/https://img02.sogoucdn.com/app/a/200698/sogou_science_13465?w=500&h=277&titlename=%E8%82%BF%E7%98%A4%E5%9F%BA%E5%9B%A0%E7%BB%84%E5%AD%A6%E7%9A%84%E6%95%B4%E4%BD%93%E7%9B%AE%E6%A0%87","url":"https://web.archive.org/web/20221025184839/https://img02.sogoucdn.com/app/a/200698/sogou_science_13465","rw":500,"rh":277,"title":"肿瘤基因组学的整体目标","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221025184839/https://img04.sogoucdn.com/app/a/200698/sogou_science_13466?w=600&h=456&titlename=%E8%82%BF%E7%98%A4%E5%9F%BA%E5%9B%A0%E7%BB%84%E5%AD%A6%E4%B8%AD%E4%BD%BF%E7%94%A8%E7%9A%84%E6%8A%80%E6%9C%AF","url":"https://web.archive.org/web/20221025184839/https://img04.sogoucdn.com/app/a/200698/sogou_science_13466","rw":600,"rh":456,"title":"肿瘤基因组学中使用的技术","alt":null,"width":0,"height":0}],"catalogs":[{"level":1,"title":"历史","paragraphId":"14995439136801296","subCatalogs":[{"level":2,"title":"全基因组测序","paragraphId":"14995439136801296","subCatalogs":null}]},{"level":1,"title":"技术","paragraphId":"14995439153578500","subCatalogs":[{"level":2,"title":"基因组测序","paragraphId":"14995439153578500","subCatalogs":null},{"level":2,"title":"转录组","paragraphId":"14995439153578500","subCatalogs":null},{"level":2,"title":"生物信息学与肿瘤基因的功能分析","paragraphId":"14995439153578500","subCatalogs":null},{"level":2,"title":"操纵子组","paragraphId":"14995439153578500","subCatalogs":null}]},{"level":1,"title":"比较肿瘤基因组学","paragraphId":"14995439153578501","subCatalogs":null},{"level":1,"title":"癌症驱动突变的来源，癌症突变形成","paragraphId":"14995439153578502","subCatalogs":null},{"level":1,"title":"合成致死","paragraphId":"14995439170355718","subCatalogs":null},{"level":1,"title":"癌症研究数据库","paragraphId":"14995439170355719","subCatalogs":null},{"level":1,"title":"基因家族","paragraphId":"14995439187132936","subCatalogs":null},{"level":1,"title":"线粒体dna","paragraphId":"14995439187132937","subCatalogs":[{"level":2,"title":"点突变","paragraphId":"14995439187132937","subCatalogs":null},{"level":2,"title":"删除","paragraphId":"14995439187132937","subCatalogs":null},{"level":2,"title":"插入","paragraphId":"14995439187132937","subCatalogs":null},{"level":2,"title":"拷贝数突变","paragraphId":"14995439187132937","subCatalogs":null},{"level":2,"title":"诊断应用","paragraphId":"14995439187132937","subCatalogs":null}]},{"level":1,"title":"肿瘤生物标志物","paragraphId":"14995439187132938","subCatalogs":null},{"level":1,"title":"参考文献","paragraphId":"-1","subCatalogs":null}],"paragraphs":[{"paragraphId":"14995439136801296","title":"历史","versionId":"14995439136801286","lemmaId":10636,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":10145103,"name":"科学百科编辑组","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233977,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>基因组学始于20世纪90年代，伴随着许多生物基因序列的产生。在21世纪，人类基因组计划的完成使得功能基因组学的研究和肿瘤基因组检测成为可能。癌症是一个主要焦点。 </p>\n<p>表观基因组学时代最近才开始，约在2000年。<sup><a href=\"#quote_2\" class=\"kx_ref\">[2]</a></sup><sup><a href=\"#quote_3\" class=\"kx_ref\">[3]</a></sup>表观遗传变化的一个主要来源是基因启动子区CpG岛甲基化的改变。许多最近设计的方法可以评估癌症和正常组织中的脱氧核糖核酸甲基化状态。<sup><a href=\"#quote_4\" class=\"kx_ref\">[4]</a></sup>一些方法评估位于不同基因座类别的CpG甲基化，包括CpG岛、岛岸和岛架以及启动子、基因体和基因间区域。<sup><a href=\"#quote_5\" class=\"kx_ref\">[5]</a></sup>癌症也是表观遗传学研究的一个主要焦点。 </p>\n<p>获得癌症全基因组测序对癌症(或癌症基因组)研究很重要，因为: </p> \n<ul>\n <li>突变是引发癌症的直接原因，并决定了肿瘤表型。</li> \n <li>从同一名患者获取癌症和正常组织样本，根据大多数突变代表体细胞变异的事实，来识别癌症特异性突变。</li> \n <li>癌症突变是累积的，有时与疾病分期有关。转移和耐药性是可以区分的。<sup><a href=\"#quote_6\" class=\"kx_ref\">[6]</a></sup> 获得甲基化谱对癌症研究很重要，因为:</li> \n <li>表观驱动和突变驱动可以作为癌症的直接原因。<sup><a href=\"#quote_7\" class=\"kx_ref\">[7]</a></sup></li> \n <li>癌症表突变是累积的，有时与疾病阶段相关。<sup><a href=\"#quote_8\" class=\"kx_ref\">[8]</a></sup></li>\n</ul> \n<h3>全基因组测序</h3> \n<p>第一个癌症基因组于2008年被测序。<sup><a href=\"#quote_6\" class=\"kx_ref\">[6]</a></sup>这项研究测序了一个典型的急性髓细胞白血病(AML)基因组及其从同一患者身上获得的正常对应基因组。通过比较揭示了十个突变基因。其中两个已经被认为是肿瘤进展的原因：FLT3受体酪氨酸激酶基因的内部串联重复（激活激酶信号通路并与不良预后相关）以及NPM1基因第12外显子的四个碱基插入。这些突变在25-30%的急性髓细胞白血病肿瘤中都有发现，被认为促进了疾病进展，而不是直接导致疾病发生的原因。 </p>\n<p>其余8个是新突变，都是单碱基变化：4个是与癌症发病机制密切相关的家族(PTPRT、CDH24、PCLKC和SLC15A1)。另外四个与癌症发病机制没有关联。它们在代谢通路中确实有潜在的作用，提示了它们可以通过某种机制促进癌症 (KNDC1，GPR124，EB12，GRINC1B) </p>\n<p>这些基因参与了已知的导致癌症发病的途径，但是在这项研究之前，大多数基因都不是靶向基因治疗的候选基因。该分析验证了癌症全基因组测序在鉴定体细胞突变中的方法以及正常和肿瘤细胞基因组平行测序的重要性。<sup><a href=\"#quote_9\" class=\"kx_ref\">[9]</a></sup> </p>\n<p>2011年，对一名已被依维莫司药物清除的特殊膀胱癌患者的基因组进行了测序，揭示了两个基因的突变，即TSC1和NF2。这些突变使mTOR失调（mTOR是依维莫司抑制的蛋白质）造成无限制地繁殖。因此，2015年，美国国家癌症研究所创建了“特殊反应者倡议”。该倡议允许这类特殊患者(对通常无效的癌症药物有至少六个月明确反应)进行基因组测序，以识别相关突变。一旦鉴定出来，其他患者就可以进行突变筛查，然后服用药物。为此，2015年开始了一项全国性的癌症药物试验，涉及多达2400个中心。具有适当突变的患者匹配到了四十多种药物中合适的一种药物。<sup><a href=\"#quote_10\" class=\"kx_ref\">[10]</a></sup> </p>\n<p>2014年，分子肿瘤学中心推出了MSK-IMPACT试验，这是一种寻找341个癌症相关基因突变的筛查工具。到2015年，超过5000名患者接受了筛查。具有适当突变的患者有资格参加提供靶向治疗的临床试验。<sup><a href=\"#quote_10\" class=\"kx_ref\">[10]</a></sup> </p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14995439153578500","title":"技术","versionId":"14995439136801287","lemmaId":10636,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":10145103,"name":"科学百科编辑组","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233977,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p></p><p><img alt=\"\" class=\"fileimage kx_img ed_imgfloat_right\" img_height=\"456\" img_width=\"600\" titlename=\"肿瘤基因组学中使用的技术\" data-src=\"https://img04.sogoucdn.com/app/a/200698/sogou_science_13466\"> </p><p></p>\n<p>基因组技术包括: </p>\n<h3>基因组测序</h3> \n<ul>\n <li>脱氧核糖核酸测序：基于焦磷酸测序的测序仪提供了一种相对低成本的方法来生成测序数据。<sup><a href=\"#quote_1\" class=\"kx_ref\">[1]</a></sup><sup><a href=\"#quote_11\" class=\"kx_ref\">[11]</a></sup><sup><a href=\"#quote_12\" class=\"kx_ref\">[12]</a></sup></li> \n <li>微阵列比较基因组杂交：这项技术检测正常基因组和癌症基因组之间的基因拷贝数差异。它使用荧光标记样本的荧光强度值，这些样本可与微阵列上的已知探针杂交。<sup><a href=\"#quote_13\" class=\"kx_ref\">[13]</a></sup><sup><a href=\"#quote_14\" class=\"kx_ref\">[14]</a></sup></li> \n <li>代表性寡核苷酸微阵列分析：使用扩增后的限制性酶消化基因组片段，与阵列上人类寡核苷酸杂交来检测拷贝数变化，达到30-35 kbit/秒的分辨率。<sup><a href=\"#quote_15\" class=\"kx_ref\">[15]</a></sup></li> \n <li>数字核型分析：通过限制性酶消化获得的基因组标签，然后将这些标签连接到双标签中，连接、克隆、测序并回贴到参考基因组，以评估标签密度，以此来检测拷贝数变化。<sup><a href=\"#quote_16\" class=\"kx_ref\">[16]</a></sup><sup><a href=\"#quote_17\" class=\"kx_ref\">[17]</a></sup></li> \n <li>细菌人工染色体(BAC)-末端测序(末端序列分析)：通过从癌症基因组中生成BAC文库并对其末端测序来识别染色体断点。含有染色体异常的BAC克隆的末端序列不与参考基因组的相似区域相匹配，从而识别染色体断点。<sup><a href=\"#quote_18\" class=\"kx_ref\">[18]</a></sup></li>\n</ul> \n<h3>转录组</h3> \n<ul>\n <li>微阵列：评估转录本丰度。在癌症分类、预后、提高不同治疗方法的可能性以及帮助识别蛋白质编码区的突变等方面十分有用。<sup><a href=\"#quote_19\" class=\"kx_ref\">[19]</a></sup><sup><a href=\"#quote_20\" class=\"kx_ref\">[20]</a></sup> 可变剪切转录本的相对丰富已经成为癌症研究的一个重要特征。特定的可变剪切转录本形式与特定的癌症类型相关。<sup><a href=\"#quote_21\" class=\"kx_ref\">[21]</a></sup></li> \n <li><i>RNA-Seq</i></li>\n</ul> \n<h3>生物信息学与肿瘤基因的功能分析</h3> \n<p>生物信息学技术主要用来对基因组数据进行统计分析。到目前为止，癌基因的功能特征尚未确定。例如，它们与肿瘤形成相关的转化能力以及在癌症发展的每个阶段的特定作用相关。 </p>\n<p>在对一组癌症样本进行体细胞癌突变检测后，可以进行生物信息学计算分析，以确定可能的功能突变和可能的驱动突变。通常鉴定的方法主要有以下三种：绘制突变图，评估蛋白质或调节元件功能突变的影响，以及在肿瘤群中寻找阳性标记。这些方法不一定是顺序的，但是不同方法的元素之间有重要的优先级关系。每个步骤使用不同的工具。<sup><a href=\"#quote_22\" class=\"kx_ref\">[22]</a></sup> </p> \n<h3>操纵子组</h3> \n<p>操纵子组旨在整合基因组学、转录组学和蛋白质组学，以了解癌症发展的分子机制。<sup><a href=\"#quote_23\" class=\"kx_ref\">[23]</a></sup> </p>","pics":[{"originalUrl":"https://web.archive.org/web/20221025184839/https://img04.sogoucdn.com/app/a/200698/sogou_science_13466?w=600&h=456&titlename=%E8%82%BF%E7%98%A4%E5%9F%BA%E5%9B%A0%E7%BB%84%E5%AD%A6%E4%B8%AD%E4%BD%BF%E7%94%A8%E7%9A%84%E6%8A%80%E6%9C%AF","url":"https://web.archive.org/web/20221025184839/https://img04.sogoucdn.com/app/a/200698/sogou_science_13466","rw":600,"rh":456,"title":"肿瘤基因组学中使用的技术","alt":null,"width":0,"height":0}],"card":null,"references":[],"versionCount":0},{"paragraphId":"14995439153578501","title":"比较肿瘤基因组学","versionId":"14995439136801288","lemmaId":10636,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":10145103,"name":"科学百科编辑组","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233977,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>比较肿瘤基因组学通过使用跨物种的基因组比较来识别癌基因。这项研究包括研究模型生物(如小鼠)的癌症基因组、转录组和蛋白质组，从而识别潜在的癌基因，然后参考人类癌症样本来鉴别这些癌基因的同源基因是否对人类癌症的发生扮演重要的角色。<sup><a href=\"#quote_24\" class=\"kx_ref\">[24]</a></sup> 研究发现老鼠模型中的基因改变与人类癌症相似。这些模型是通过逆转录病毒介导插入导致变异或癌细胞转染等方法来产生。 </p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14995439153578502","title":"癌症驱动突变的来源，癌症突变形成","versionId":"14995439136801289","lemmaId":10636,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":10145103,"name":"科学百科编辑组","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233977,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>突变为进化中的自然选择提供了原材料，可能是由基因复制错误、外源诱变剂的作用或内源基因损伤引起。复制和基因组维持的机制可能因突变而受损，或因癌症的生理条件和表达水平的差异而改变。<sup><a href=\"#quote_25\" class=\"kx_ref\">[25]</a></sup> </p>\n<p>正如Gao等人所指出的那样，<sup><a href=\"#quote_26\" class=\"kx_ref\">[26]</a></sup>人类基因组的稳定性和完整性由脱氧核糖核酸损伤应答调控系统来维持。未修复的脱氧核糖核酸损伤是导致致癌的突变的主要原因<sup><a href=\"#quote_27\" class=\"kx_ref\">[27]</a></sup><sup><a href=\"#quote_28\" class=\"kx_ref\">[28]</a></sup>。如果脱氧核糖核酸修复有缺陷，脱氧核糖核酸损伤就会逐渐累积。由于跨损伤修复错误的合成反应，这种过度的脱氧核糖核酸损伤会增加脱氧核糖核酸复制过程中的突变错误。<sup><a href=\"#quote_29\" class=\"kx_ref\">[29]</a></sup><sup><a href=\"#quote_30\" class=\"kx_ref\">[30]</a></sup>由于DNA修复过程中的错误，过度脱氧核糖核酸损伤也会增加表观遗传的改变。这样的突变和表观遗传改变会导致癌症的发生。DDR基因通常在人类癌症中被表观遗传机制抑制。这种抑制可能涉及启动子区域的脱氧核糖核酸甲基化或小分子核糖核酸对DDR基因的抑制。在许多类型的癌症中，DDR基因的表观遗传学抑制比基因突变更频繁发生。因此，表观遗传学抑制在减少DDR基因表达方面通常比突变发挥更重要的作用。DDR基因表达的减少可能是癌症发生的一个重要驱动因素。 </p>\n<p>核苷酸序列周围环境影响突变概率<sup><a href=\"#quote_31\" class=\"kx_ref\">[31]</a></sup><sup><a href=\"#quote_32\" class=\"kx_ref\">[32]</a></sup><sup><a href=\"#quote_33\" class=\"kx_ref\">[33]</a></sup>，因此对突变(可变)的脱氧核糖核酸基序的分析对于理解癌症中突变的机制是必不可少的。这些基序代表了脱氧核糖核酸和诱变剂之间、脱氧核糖核酸和修复/复制/修饰酶之间相互作用的指纹图谱。关于基序的例子，如AID作用于WRCY/RGYW基序，将C通过各种作用引入T/G/A的突变（W = A或T，R = 嘌呤 ，Y = 嘧啶）<sup><a href=\"#quote_33\" class=\"kx_ref\">[33]</a></sup>，WA/TW基序中易错的DNA聚合酶修复机制通过转换或颠换引发AID相关的突变（A转成G/C/G）。<sup><a href=\"#quote_34\" class=\"kx_ref\">[34]</a></sup> </p>\n<p>另一种(不可知的)分析肿瘤突变观察到的突变谱和脱氧核糖核酸序列背景的方法是将来自癌症样本中不同突变类型和背景的所有突变汇集成离散分布。如果有多个癌症样本，它们的背景相关突变可以用非负矩阵的形式表示。这个矩阵可以进一步分解成理想状态下描述个体诱变因素的子矩阵(突变信号)。已经提出了几种计算方法来解决这个分解问题。非负矩阵分解(NMF)方法的第一个实现是Mutational Signature Framework，由桑格研究所开发的一个MATLAB包。<sup><a href=\"#quote_35\" class=\"kx_ref\">[35]</a></sup>另一方面，如果仅有来自单个肿瘤样本的突变，DeconstructSigs 包（一个R包）<sup><a href=\"#quote_36\" class=\"kx_ref\">[36]</a></sup>和MutaGene 服务器<sup><a href=\"#quote_37\" class=\"kx_ref\">[37]</a></sup>可以提供对单个肿瘤样本的不同突变信号贡献的识别能力。此外，在线MutaGene服务器提供诱变剂或癌症特异性突变背景模型和信号，可用于计算目的脱氧核糖核酸和蛋白质位点突变型，以消除突变和选择在癌症发生中的相对作用。 </p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14995439170355718","title":"合成致死","versionId":"14995439136801290","lemmaId":10636,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":10145103,"name":"科学百科编辑组","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233977,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>当两种或多种基因表达缺陷的组合导致细胞死亡，而这些基因中只有一种缺陷不会导致细胞死亡时，合成致死性就会出现。这种缺陷可能是由其中一个基因的突变、表观遗传改变或抑制剂引起的。 </p>\n<p>合成致死作为潜在的一种有效抗癌策略，治疗潜力正在不断提高。最近，由于罗纳德·德平霍(Ronald A. DePinho)及其同事在内的科学家在所谓的“附带性致死”方面的工作，合成致死性在靶向癌症治疗中的适用性有所提高。穆勒（Muller）等人发现，染色体上接近肿瘤抑制基因的乘客基因在某些癌症中被间接删除。<sup><a href=\"#quote_38\" class=\"kx_ref\">[38]</a></sup>因此，对执行基本细胞功能的间接删除的冗余基因的鉴定可能是随后寻求合成致死方法的尚未开发的储存库。因此，间接致死在肿瘤学中识别新的选择性治疗靶点方面具有巨大潜力。<sup><a href=\"#quote_39\" class=\"kx_ref\">[39]</a></sup>2012年，穆勒（Muller）等人发现人类胶质母细胞瘤(GBM)中冗余必需糖酵解ENO1基因纯合缺失是接近1p36肿瘤抑制基因位点缺失的结果，并可能为合成GBM抑制提供一种合成致死方法。 <sup><a href=\"#quote_38\" class=\"kx_ref\">[38]</a></sup>ENO1是编码哺乳动物α-烯醇酶的三个同源基因(ENO2，ENO3)之一。<sup><a href=\"#quote_40\" class=\"kx_ref\">[40]</a></sup>编码烯醇化酶2的ENO2主要在神经组织中表达，因此，推测在ENO1缺失的GBM中，ENO2可能是作为ENO1冗余同源基因的理想靶位。 穆勒（Muller）发现，在具有纯合EN1缺失的GBM细胞中，基因和药理学上的EN2抑制通过选择性杀死GBM细胞而引发合成致死。2016年，穆勒（Muller）及其同事发现抗生素SF2312是一种高效的纳米级烯醇化酶抑制剂，优先抑制神经胶质瘤细胞增殖和ENO1缺失细胞的糖酵解速率。SF2312被证明比泛烯醇化酶抑制剂PhAH更有效，并且对ENO1抑制比ENO1具有更高的特异性。同一小组随后的工作表明，同样的方法可以应用于胰腺癌，纯合缺失的SMAD4导致线粒体苹果酸酶2 (ME2)的侧支缺失，线粒体苹果酸酶2是氧化还原体内平衡所必需的氧化脱羧酶。 Dey等人表明，胰腺导管腺癌细胞中的ME2基因组缺失导致高内源性活性氧，与KRAS驱动的胰腺癌相一致，并且基本上通过耗尽多余的NAD(P)+-依赖亚型ME3来激发ME2-null细胞合成致死性。发现ME3缺失的影响是通过抑制由AMPK活化和线粒体活性氧介导的凋亡导致的从头核苷酸合成来介导的。与此同时，奥科（Oike）等人通过在代谢以外的过程中定位多余的必须基因，即染色质重塑SWI/SNF复合体中的SMARCA4和SMARCA2亚单位，证明了这一概念的普遍性。 </p>\n<p>合成致命性作为有效抗癌策略的治疗潜力正在不断提高。最近，由于包括罗纳德·德平霍和他的同事在内的科学家最近在所谓的“间接致死”方面的工作，合成致死在靶向癌症治疗中的适用性得到了提高。Muller等人发现，染色体接近肿瘤抑制基因的乘客基因在一些癌症中被间接删除。 因此，识别执行基本细胞功能的并列缺失的冗余基因可能是随后寻求合成致死方法的未开发的储库。因此，间接致死在肿瘤学中识别新的选择性治疗靶点方面具有巨大潜力。 2012年，Muller等人发现人类胶质母细胞瘤(GBM)中冗余必需糖酵解ENO1基因纯合缺失是接近1p36肿瘤抑制基因位点缺失的结果，并可能对GBM抑制具有合成致死性途径的潜力。 ENO1是编码哺乳动物α-烯醇酶的三个同源基因(ENO2，ENO3)之一。 编码烯醇酶2的ENO2主要在神经组织中表达，导致推测在ENO1缺失的GBM中，ENO2可能是ENO1的冗余同源物的理想靶标。 穆勒发现，基因和药理学上抑制具有纯合ENO1缺失的GBM细胞的ENO2，通过选择性杀死GBM细胞，引发合成致死结果。<sup><a href=\"#quote_38\" class=\"kx_ref\">[38]</a></sup> 2016年，穆勒和他的同事发现抗生素SF2312是一种高效的纳米级烯醇化酶抑制剂，优先抑制神经胶质瘤细胞增殖和ENO1缺失细胞的糖酵解通量。<sup><a href=\"#quote_41\" class=\"kx_ref\">[41]</a></sup> SF2312被证明比泛烯醇化酶抑制剂PhAH更有效，并且对ENO2的抑制比ENO1更特异。<sup><a href=\"#quote_41\" class=\"kx_ref\">[41]</a></sup> 同一个小组随后的工作表明，同样的方法可以应用于胰腺癌，由此纯合缺失的SMAD4导致线粒体苹果酸酶2 (ME2)的侧枝缺失，线粒体2是氧化还原体内平衡所必需的氧化脱羧酶。<sup><a href=\"#quote_42\" class=\"kx_ref\">[42]</a></sup> Dey等人表明，胰腺导管腺癌细胞中的ME2基因组缺失导致高内源性活性氧物种，这与KRAS驱动的胰腺癌相一致，并且基本上通过耗尽多余的NAD(P)+-依赖性亚型ME3来激发ME2-空细胞的合成致死性。发现ME3耗竭的影响是通过抑制由AMPK激活和线粒体活性氧介导的凋亡引起的从头核苷酸合成来介导的。<sup><a href=\"#quote_43\" class=\"kx_ref\">[43]</a></sup><sup><a href=\"#quote_42\" class=\"kx_ref\">[42]</a></sup> 同时，Oike等人通过靶向代谢以外过程中的冗余必需基因，即染色质重塑SWI/SNF复合物中的SMARCA4和SMARCA2亚单位，证明了该概念的普遍性。<sup><a href=\"#quote_44\" class=\"kx_ref\">[44]</a></sup> </p>\n<p>一些癌基因对所有细胞(不仅仅是癌细胞)的生存至关重要。因此，敲除这些致癌基因(从而杀死癌细胞)的药物也可能损害正常细胞，导致重大疾病。然而，其他基因可能对癌细胞至关重要，但对健康细胞却不是。 </p>\n<p>基于合成致死原理的治疗延长了癌症患者的生存期，并显示出逆转癌症发生的未来进展的前景。 一种主要的合成致命性作用于通常引发癌症的DNA修复缺陷，并且仍然存在于肿瘤细胞中。 这里给出了一些例子。 </p>\n<p>BRCA1或BRCA2在大多数高级乳腺癌和卵巢癌中表达不足，通常是由于其启动子的表观遗传甲基化或过度表达的microRNA的表观遗传抑制(见BRCA1和BRCA2文章)。 BRCA1和BRCA2是双链断裂同源重组修复主要途径的重要组成部分。 如果其中一个有缺陷，它会增加患癌症的风险，尤其是乳腺癌或卵巢癌。 一条备用的脱氧核糖核酸修复途径，对于一些通常由BRCA1和BRCA2修复的损伤，依赖于PARP1。 因此，许多卵巢癌对美国食品和药物管理局批准的PARP抑制剂治疗有反应，导致BRCA1或BRCA2缺陷的癌细胞合成致死。 在2016年的三期临床试验中，乳腺癌和许多其他癌症也在评估这种治疗方法。<sup><a href=\"#quote_45\" class=\"kx_ref\">[45]</a></sup> </p>\n<p>双链断裂的同源重组修复有两种途径。 主要途径依赖于BRCA1、PALB2和BRCA2，而另一种途径依赖于RAD52。<sup><a href=\"#quote_46\" class=\"kx_ref\">[46]</a></sup> 临床前研究，包括表观遗传学上减少或突变的BRCA缺陷细胞(在培养中或注射到小鼠体内)，表明RAD52的抑制对BRCA缺陷综合致死。<sup><a href=\"#quote_47\" class=\"kx_ref\">[47]</a></sup> </p>\n<p>用于DNA错配修复的基因突变导致高突变率。<sup><a href=\"#quote_48\" class=\"kx_ref\">[48]</a></sup> 在肿瘤中，这种频繁的后续突变通常会产生“非自身”免疫原性抗原。 一项有41名患者参与的人类二期临床试验评估了一种针对有或无基质金属蛋白酶缺陷肿瘤的合成致死方法。<sup><a href=\"#quote_49\" class=\"kx_ref\">[49]</a></sup> 基因的产物 <i>PD-1</i> 通常抑制细胞毒性免疫反应。 抑制这种基因可以产生更强的免疫反应。 当肿瘤中有基质金属蛋白酶缺陷的癌症患者暴露于PD-1抑制剂时，67% - 78%的患者经历了免疫相关的无恶化生存期。 相比之下，对于MMR无缺陷的患者，PD-1抑制剂的添加仅产生11%的免疫相关无恶化生存期病患者。 因此，PD-1的抑制主要是由于基质金属蛋白酶缺陷而综合致死。 </p>\n<p>ARID1A，一种染色质修饰物，是非同源末端连接所必需的，非同源末端连接是修复DNA双链断裂的主要途径，<sup><a href=\"#quote_50\" class=\"kx_ref\">[50]</a></sup> 并且还具有转录调节作用。<sup><a href=\"#quote_51\" class=\"kx_ref\">[51]</a></sup> ARID1A突变是12种最常见的致癌突变之一。<sup><a href=\"#quote_52\" class=\"kx_ref\">[52]</a></sup> 突变或表观遗传学降低的表达<sup><a href=\"#quote_53\" class=\"kx_ref\">[53]</a></sup> 在17种癌症中发现了ARID1A。<sup><a href=\"#quote_54\" class=\"kx_ref\">[54]</a></sup> 在细胞和小鼠中进行的临床前研究表明，ARID1A缺乏症的合成致死率是通过抑制EZ2的甲基转移酶活性，<sup><a href=\"#quote_55\" class=\"kx_ref\">[55]</a></sup><sup><a href=\"#quote_56\" class=\"kx_ref\">[56]</a></sup> 或者加入激酶抑制剂达沙替尼。<sup><a href=\"#quote_57\" class=\"kx_ref\">[57]</a></sup> </p>\n<p>另一种方法是单独敲除基因组中的每个基因，观察对正常细胞和癌细胞的影响。<sup><a href=\"#quote_58\" class=\"kx_ref\">[58]</a></sup><sup><a href=\"#quote_59\" class=\"kx_ref\">[59]</a></sup> 如果一个非必需基因的敲除对健康细胞影响很小或者没有影响，但是对含有突变癌基因的癌细胞是致命的，那么系统范围内对被抑制基因的抑制可以破坏癌细胞，同时使健康细胞相对完好无损。该技术被用于鉴定PARP-1抑制剂以治疗BRCA1/BRCA2相关癌症。<sup><a href=\"#quote_60\" class=\"kx_ref\">[60]</a></sup><sup><a href=\"#quote_61\" class=\"kx_ref\">[61]</a></sup> 在这种情况下，PARP-1抑制和癌症相关突变在BRCA基因中的联合存在仅对癌细胞是致命的。 </p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14995439170355719","title":"癌症研究数据库","versionId":"14995439136801291","lemmaId":10636,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":10145103,"name":"科学百科编辑组","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233977,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>“Cancer Genome Project”是一项绘制癌症所有体细胞突变的倡议。该项目系统地对原发性肿瘤和癌细胞系基因组的外显子和侧翼剪接点进行测序。 宇宙软件显示这些实验产生的数据。截至2008年2月，CGP已经在1，848个肿瘤中鉴定出4，746个基因和2，985个突变。 </p>\n<p>癌症基因组解剖学项目包括癌症基因组、转录组和蛋白质组的研究信息。 </p>\n<p>Progenetix是一个致癌基因参考数据库，提供细胞遗传学和分子细胞遗传学肿瘤数据。 </p>\n<p>安康尼从癌症转录组概况中收集了数据。 </p>\n<p>IntOGen和Gitools数据集的整合肿瘤组学数据库整合了按肿瘤类型分类的多维人类肿瘤组学数据。IntOGen的第一个版本关注的是基因表达失控和CNV在癌症中的作用。<sup><a href=\"#quote_62\" class=\"kx_ref\">[62]</a></sup> 后来的版本强调了28种肿瘤类型的突变癌症驱动基因。<sup><a href=\"#quote_63\" class=\"kx_ref\">[63]</a></sup><sup><a href=\"#quote_64\" class=\"kx_ref\">[64]</a></sup> IntOGen数据的所有版本都可以在IntOGen数据库中获得。 </p>\n<p>国际癌症基因组联盟是收集人类癌症基因组数据的最大项目。数据可通过ICGC网站获取。生物表达肿瘤学套件包含来自原发性、转移性和良性肿瘤样品以及正常样品(包括匹配的相邻对照)的基因表达数据。该套件包括许多著名癌症的血液恶性肿瘤样本。 </p>\n<p>模型动物的特定数据库包括逆转录病毒标记癌症基因数据库(RTCGD)，该数据库汇集了对小鼠肿瘤中逆转录病毒和转座子插入突变的研究。 </p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14995439187132936","title":"基因家族","versionId":"14995439136801292","lemmaId":10636,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":10145103,"name":"科学百科编辑组","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233977,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>对整个基因家族的突变分析表明，同一个家族的基因具有相似的功能，正如相似的编码序列和蛋白质结构域所预测的那样。激酶家族和磷酸酶家族是两个这样的类别，激酶家族参与向蛋白质中添加磷酸基团，磷酸酶家族参与从蛋白质中去除磷酸基团。<sup><a href=\"#quote_65\" class=\"kx_ref\">[65]</a></sup> 因为这些家族在传导细胞生长或死亡的细胞信号中的明显作用，所以首先对其进行了检查。特别是，超过50%的结直肠癌携带激酶或磷酸酶基因突变。磷脂酰肌醇d 3激酶(磷脂酰肌醇-3-激酶)基因编码脂质激酶，其通常包含结肠直肠癌、乳腺癌、胃癌、肺癌和各种其他癌症的突变。<sup><a href=\"#quote_66\" class=\"kx_ref\">[66]</a></sup><sup><a href=\"#quote_67\" class=\"kx_ref\">[67]</a></sup> 药物疗法可以抑制磷脂酰肌醇-3-激酶。另一个例子是BRAF基因，它是第一个涉及黑色素瘤的基因之一。<sup><a href=\"#quote_68\" class=\"kx_ref\">[68]</a></sup> BRAF编码参与RAS-RAF-MAPK生长信号通路的丝氨酸/苏氨酸激酶。在59%的黑素瘤中，BRAF突变导致组成性磷酸化和活性。在BRAF之前，黑色素瘤发展的遗传机制尚不清楚，因此患者的预后很差。<sup><a href=\"#quote_69\" class=\"kx_ref\">[69]</a></sup> </p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14995439187132937","title":"线粒体dna","versionId":"14995439136801293","lemmaId":10636,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":10145103,"name":"科学百科编辑组","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233977,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>线粒体DNA (mtDNA)突变与肿瘤的形成有关。已经确定了四种类型的mtDNA突变:<sup><a href=\"#quote_70\" class=\"kx_ref\">[70]</a></sup> </p> \n<h3>点突变</h3> \n<p>在癌细胞包含的mtDNA编码区和非编码区观察到了点突变。在患有膀胱癌、头颈癌和肺癌的个体中，编码区内的点突变显示出彼此相似的迹象。这表明，当健康细胞转化为肿瘤细胞(肿瘤转化)时，线粒体似乎变得同质。位于癌线粒体非编码区D环的大量点突变表明，该区的突变可能是某些癌症的重要特征。<sup><a href=\"#quote_70\" class=\"kx_ref\">[70]</a></sup> </p> \n<h3>删除</h3> \n<p>这种类型的突变由于其小尺寸(&lt; 1kb)而偶尔被检测到。在多种类型的癌症中出现特定的mtDNA突变(复合物1亚基基因ND1中的264-bp缺失和66-bp缺失)提供了一些证据，表明小mtDNA缺失可能出现在肿瘤发生的开始。这也表明，随着肿瘤的进展，含有这些缺失的线粒体数量增加。一个例外是出现在许多癌症中的相对较大的缺失(称为“常见缺失”)，但是与肿瘤细胞相比，在正常细胞中发现了更多的mtDNA大规模缺失。这可能是由于肿瘤细胞表面上的适应过程消除了任何含有这些大规模缺失的线粒体(“常见缺失”大于4kb)。<sup><a href=\"#quote_70\" class=\"kx_ref\">[70]</a></sup> </p> \n<h3>插入</h3> \n<p>乳腺癌、胃癌、肝细胞癌(HCC)和结肠癌以及正常细胞中可能存在约260和约520 bp的两个小mtDNA插入。这些插入物和癌症之间没有相关性。<sup><a href=\"#quote_71\" class=\"kx_ref\">[71]</a></sup> </p> \n<h3>拷贝数突变</h3> \n<p>通过实时聚合酶链反应试验对mtDNA进行的表征表明，在许多癌症中存在mtDNA拷贝数的定量变化。由于氧化应激，拷贝数预计会增加。另一方面，减少被认为是由在复制起点的H链和/或在D环区域的D310均聚c-伸展的体细胞点突变，p53(肿瘤抑制基因)介导的途径的突变和/或由于POLG突变的低效酶活性引起的。拷贝数的任何增加/减少在肿瘤细胞内保持不变。肿瘤细胞中的mtDNA量是恒定的这一事实表明，肿瘤细胞中的mtDNA量是由一个复杂得多的系统控制的，而不是简单地由于异常细胞增殖而改变。对于特定的肿瘤类型或部位，mtDNA含量在人类癌症中的作用明显不同。<sup><a href=\"#quote_70\" class=\"kx_ref\">[70]</a></sup> </p> \n<table class=\"wikitable\"> \n <caption>\n  不同癌症中线粒体基因的突变 \n </caption> \n <tbody>\n  <tr> \n   <th rowspan=\"2\">癌症类型 </th> \n   <th colspan=\"4\">点突变的位置 </th> \n   <th rowspan=\"2\">核苷酸缺失位置 </th> \n   <th rowspan=\"2\">mtDNA拷贝数增加 </th> \n   <th rowspan=\"2\">mtDNA拷贝数减少 </th>\n  </tr> \n  <tr> \n   <th>mtDNA控制区 </th> \n   <th>mRNAs </th> \n   <th>tRNAs </th> \n   <th>rRNAs </th>\n  </tr> \n  <tr> \n   <td>膀胱癌<sup><a href=\"#quote_72\" class=\"kx_ref\">[72]</a></sup></td> \n   <td>X</td> \n   <td>X</td> \n   <td></td> \n   <td>X</td> \n   <td>15,642-15,662</td> \n   <td></td> \n   <td> </td>\n  </tr> \n  <tr> \n   <td>乳腺癌<sup><a href=\"#quote_73\" class=\"kx_ref\">[73]</a></sup><sup><a href=\"#quote_74\" class=\"kx_ref\">[74]</a></sup><sup><a href=\"#quote_75\" class=\"kx_ref\">[75]</a></sup><sup><a href=\"#quote_76\" class=\"kx_ref\">[76]</a></sup></td> \n   <td>X</td> \n   <td>X</td> \n   <td>X</td> \n   <td>X</td> \n   <td>8470-13,447和8482-13459</td> \n   <td></td> \n   <td>X </td>\n  </tr> \n  <tr> \n   <td>头颈癌<sup><a href=\"#quote_73\" class=\"kx_ref\">[73]</a></sup><sup><a href=\"#quote_77\" class=\"kx_ref\">[77]</a></sup><sup><a href=\"#quote_78\" class=\"kx_ref\">[78]</a></sup></td> \n   <td>X</td> \n   <td>X</td> \n   <td>X</td> \n   <td>X</td> \n   <td>8470-13,447和 8482-13459</td> \n   <td>X</td> \n   <td> </td>\n  </tr> \n  <tr> \n   <td>口腔癌<sup><a href=\"#quote_79\" class=\"kx_ref\">[79]</a></sup></td> \n   <td>X</td> \n   <td>X</td> \n   <td></td> \n   <td></td> \n   <td>8470-13,447和8482-13459</td> \n   <td></td> \n   <td> </td>\n  </tr> \n  <tr> \n   <td>肝细胞癌(HCC)<sup><a href=\"#quote_80\" class=\"kx_ref\">[80]</a></sup><sup><a href=\"#quote_81\" class=\"kx_ref\">[81]</a></sup></td> \n   <td>X</td> \n   <td>X</td> \n   <td>X</td> \n   <td>X</td> \n   <td>306-556和3894-3960</td> \n   <td></td> \n   <td>X </td>\n  </tr> \n  <tr> \n   <td>食管癌<sup><a href=\"#quote_82\" class=\"kx_ref\">[82]</a></sup></td> \n   <td>X</td> \n   <td>X</td> \n   <td></td> \n   <td>X</td> \n   <td>8470-13,447和8482-13459</td> \n   <td>X</td> \n   <td> </td>\n  </tr> \n  <tr> \n   <td>胃癌 <sup><a href=\"#quote_83\" class=\"kx_ref\">[83]</a></sup><sup><a href=\"#quote_84\" class=\"kx_ref\">[84]</a></sup><sup><a href=\"#quote_85\" class=\"kx_ref\">[85]</a></sup></td> \n   <td>X</td> \n   <td>X</td> \n   <td>X</td> \n   <td></td> \n   <td>298-348</td> \n   <td></td> \n   <td>X </td>\n  </tr> \n  <tr> \n   <td>前列腺癌<sup><a href=\"#quote_86\" class=\"kx_ref\">[86]</a></sup><sup><a href=\"#quote_87\" class=\"kx_ref\">[87]</a></sup></td> \n   <td>X</td> \n   <td></td> \n   <td></td> \n   <td>X</td> \n   <td>8470-13,447和 8482-13459</td> \n   <td>X</td> \n   <td> </td>\n  </tr>\n </tbody>\n</table> \n<p>57.7% (500/867)的癌症含体细胞点突变，在调查的1172个突变中，37.8% (443/1127)位于mtDNA控制区，13.1% (154/1172)位于tRNA或rRNA基因，49.1% (575/1127)位于负责产生线粒体呼吸所需复合物所需的mRNA基因上。 </p> \n<h3>诊断应用</h3> \n<p>一些抗癌药物以mtDNA为目标，并显示出杀死肿瘤细胞的积极效果。因此，一些研究使用线粒体突变作为癌细胞治疗的生物标记。线粒体基因组比细胞核基因组更小，更容易筛选特定的突变，因此更容易定位线粒体基因组中的突变。血液样本中发现的mtDNA含量变化可以作为预测未来癌症易感性以及跟踪恶性肿瘤进展的筛选标记。除了mtDNA这些潜在的有益特性外，mtDNA不受细胞周期的控制，对维持三磷酸腺苷的生成和线粒体内环境稳定很重要。这些特点使得靶向mtDNA成为一种可行的治疗策略。<sup><a href=\"#quote_70\" class=\"kx_ref\">[70]</a></sup> </p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14995439187132938","title":"肿瘤生物标志物","versionId":"14995439136801294","lemmaId":10636,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":10145103,"name":"科学百科编辑组","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233977,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>目前，多种生物标志物可用于癌症分期、预后和治疗。它们的范围分布广泛，例如：单核苷酸多态性、染色体畸变、DNA拷贝数的变化、微卫星不稳定性、启动子区甲基化，甚至是高/低蛋白质表达水平。<sup><a href=\"#quote_88\" class=\"kx_ref\">[88]</a></sup> </p>","pics":null,"card":null,"references":[],"versionCount":0}],"references":[{"id":1,"type":"book","title":"Strausberg RL, Simpson AJ, Old LJ, Riggins GJ (May 2004). \"Oncogenomics and the development of new cancer therapies\". Nature. 429 (6990): 469–74. Bibcode:2004Natur.429..469S. doi:10.1038/nature02627. PMID 15164073.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":2,"type":"book","title":"Ting AH, McGarvey KM, Baylin SB (2006). \"The cancer epigenome--components and functional correlates\". Genes Dev. 20 (23): 3215–31. doi:10.1101/gad.1464906. PMID 17158741.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":3,"type":"book","title":"Jones PA, Baylin SB (2007). \"The epigenomics of cancer\". Cell. 128 (4): 683–92. doi:10.1016/j.cell.2007.01.029. PMC 3894624. PMID 17320506.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":4,"type":"book","title":"Li D, Zhang B, Xing X, Wang T (2015). \"Combining MeDIP-seq and MRE-seq to investigate genome-wide CpG methylation\". Methods. 72: 29–40. doi:10.1016/j.ymeth.2014.10.032. PMC 4300244. PMID 25448294.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":5,"type":"book","title":"Wei J, Li G, Dang S, Zhou Y, Zeng K, Liu M (2016). \"Discovery and Validation of Hypermethylated Markers for Colorectal Cancer\". Dis. Markers. 2016: 2192853. doi:10.1155/2016/2192853. PMC 4963574. PMID 27493446.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":6,"type":"book","title":"Strausberg RL, Simpson AJ (January 2010). \"Whole-genome cancer analysis as an approach to deeper understanding of tumour biology\". Br. J. Cancer. 102 (2): 243–8. doi:10.1038/sj.bjc.6605497. PMC 2816661. PMID 20029419.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":7,"type":"book","title":"Vogelstein B, Papadopoulos N, Velculescu VE, Zhou S, Diaz LA, Kinzler KW (2013). \"Cancer genome landscapes\". Science. 339 (6127): 1546–58. Bibcode:2013Sci...339.1546V. doi:10.1126/science.1235122. PMC 3749880. PMID 23539594.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":8,"type":"book","title":"Luo Y, Wong CJ, Kaz AM, Dzieciatkowski S, Carter KT, Morris SM, Wang J, Willis JE, Makar KW, Ulrich CM, Lutterbaugh JD, Shrubsole MJ, Zheng W, Markowitz SD, Grady WM (2014). \"Differences in DNA methylation signatures reveal multiple pathways of progression from adenoma to colorectal cancer\". Gastroenterology. 147 (2): 418–29.e8. doi:10.1053/j.gastro.2014.04.039. PMC 4107146. PMID 24793120.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":9,"type":"book","title":"Ley TJ, Mardis ER, Ding L, Fulton B, McLellan MD, Chen K, et al. (November 2008). \"DNA sequencing of a cytogenetically normal acute myeloid leukaemia genome\". Nature. 456 (7218): 66–72. Bibcode:2008Natur.456...66L. doi:10.1038/nature07485. PMC 2603574. PMID 18987736.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":10,"type":"book","title":"Peikoff, Kira (2015-10-16). \"What Miraculous Recoveries Tell Us About Beating Cancer\". Popular Mechanics. Retrieved 2016-04-28.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":11,"type":"book","title":"Bardelli A.; Velculescu V.E. (2005). \"Mutational analysis of gene families in human cancer\". Current Opinion in Genetics & Development. 15 (1): 5–12. doi:10.1016/j.gde.2004.12.009.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":12,"type":"book","title":"Benvenuti S.; Arena S.; Bardelli A. (2005). \"Identification of cancer genes by mutational profiling of tumor genomes\". FEBS Letters. 579 (8): 1884–1890. doi:10.1016/j.febslet.2005.02.015. PMID 15763568.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":13,"type":"book","title":"Shih I.M.; Wang T.L. (2005). \"Apply innovative technologies to explore cancer genome\". Current Opinion in Oncology. 17 (1): 33–38. doi:10.1097/01.cco.0000147382.97085.e4. PMID 15608510.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":14,"type":"book","title":"Greshock J, Naylor TL, Margolin A, Diskin S, Cleaver SH, Futreal PA, et al. (January 2004). \"1-Mb resolution array-based comparative genomic hybridization using a BAC clone set optimized for cancer gene analysis\". Genome Res. 14 (1): 179–87. doi:10.1101/gr.1847304. PMC 314295. PMID 14672980.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":15,"type":"book","title":"Lucito R, Healy J, Alexander J, Reiner A, Esposito D, Chi M, et al. (October 2003). \"Representational oligonucleotide microarray analysis: a high-resolution method to detect genome copy number variation\". Genome Res. 13 (10): 2291–305. doi:10.1101/gr.1349003. PMC 403708. PMID 12975311.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":16,"type":"book","title":"Hu M, Yao J, Polyak K (2006). \"Methylation-specific digital karyotyping\". Nat Protoc. 1 (3): 1621–36. doi:10.1038/nprot.2006.278. PMID 17406428.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":17,"type":"book","title":"Körner H, Epanchintsev A, Berking C, Schuler-Thurner B, Speicher MR, Menssen A, Hermeking H (January 2007). \"Digital karyotyping reveals frequent inactivation of the dystrophin/DMD gene in malignant melanoma\". Cell Cycle. 6 (2): 189–98. doi:10.4161/cc.6.2.3733. PMID 17314512.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":18,"type":"book","title":"Volik S, Zhao S, Chin K, Brebner JH, Herndon DR, Tao Q, et al. (June 2003). \"End-sequence profiling: sequence-based analysis of aberrant genomes\". Proc. Natl. Acad. Sci. U.S.A. 100 (13): 7696–701. doi:10.1073/pnas.1232418100. PMC 164650. PMID 12788976.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":19,"type":"book","title":"Van, de Vijver M.J.; et al. (2002). \"A gene-expression signature as a predictor of survival in breast cancer\". New England Journal of Medicine. 347 (25): 1999–2009. doi:10.1056/NEJMoa021967. PMID 12490681.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":20,"type":"book","title":"van 't Veer LJ, Dai H, van de Vijver MJ, He YD, Hart AA, Bernards R, Friend SH (2003). \"Expression profiling predicts outcome in breast cancer\". Breast Cancer Res. 5 (1): 57–8. doi:10.1186/bcr562. PMC 154139. PMID 12559048.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":21,"type":"book","title":"Xu Q.; Lee C. (2003). \"Discovery of novel splice forms and functional analysis of cancer-specific alternative splicing in human expressed sequences\". Nucleic Acids Research. 31 (19): 5635–5643. doi:10.1093/nar/gkg786. PMC 206480. PMID 14500827.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":22,"type":"book","title":"Gonzalez-Perez A, Mustonen V, Reva B, Ritchie GR, Creixell P, Karchin R, et al. (August 2013). \"Computational approaches to identify functional genetic variants in cancer genomes\". Nat. Methods. 10 (8): 723–9. doi:10.1038/nmeth.2562. PMC 3919555. PMID 23900255.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":23,"type":"book","title":"Hanash SM (September 2000). \"Operomics: molecular analysis of tissues from DNA to RNA to protein\". Clin. Chem. Lab. Med. 38 (9): 805–13. doi:10.1515/CCLM.2000.116. PMID 11097332.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":24,"type":"book","title":"Peeper D, Berns A (June 2006). \"Cross-species oncogenomics in cancer gene identification\". Cell. 125 (7): 1230–3. doi:10.1016/j.cell.2006.06.018. PMID 16814709.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":25,"type":"book","title":"Rogozin IB, Pavlov YI, Goncearenco A, De S, Lada AG, Poliakov E, Panchenko AR, Cooper DN (November 2018). \"Mutational signatures and mutable motifs in cancer genomes\". Brief. Bioinformatics. 19 (6): 1085–1101. doi:10.1093/bib/bbx049. PMID 28498882.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":26,"type":"book","title":"Gao D, Herman JG, Guo M (2016). \"The clinical value of aberrant epigenetic changes of DNA damage repair genes in human cancer\". Oncotarget. 7: 37331–37346. doi:10.18632/oncotarget.7949. PMC 5095080. PMID 26967246.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":27,"type":"book","title":"Kastan MB (2008). \"DNA damage responses: mechanisms and roles in human disease: 2007 G.H.A. Clowes Memorial Award Lecture\". Mol. Cancer Res. 6 (4): 517–24. doi:10.1158/1541-7786.MCR-08-0020. PMID 18403632.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":28,"type":"book","title":"Bernstein, C; Prasad, AR; Nfonsam, V; Bernstein, H. (2013). \"Chapter 16: DNA Damage, DNA Repair and Cancer\". In Chen, Clark. New Research Directions in DNA Repair,. p. 413. ISBN 978-953-51-1114-6.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":29,"type":"book","title":"O'Hagan HM, Mohammad HP, Baylin SB (2008). \"Double strand breaks can initiate gene silencing and SIRT1-dependent onset of DNA methylation in an exogenous promoter CpG island\". PLoS Genetics. 4 (8): e1000155. doi:10.1371/journal.pgen.1000155. PMC 2491723. PMID 18704159.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":30,"type":"book","title":"Cuozzo C, Porcellini A, Angrisano T, et al. (July 2007). \"DNA damage, homology-directed repair, and DNA methylation\". PLoS Genetics. 3 (7): e110. doi:10.1371/journal.pgen.0030110. PMC 1913100. PMID 17616978.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":31,"type":"book","title":"Coulondre C, Miller JH, Farabaugh PJ, Gilbert W (August 1978). \"Molecular basis of base substitution hotspots in Escherichia coli\". Nature. 274 (5673): 775–80. Bibcode:1978Natur.274..775C. doi:10.1038/274775a0. PMID 355893.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":32,"type":"book","title":"Cooper DN, Youssoufian H (February 1988). \"The CpG dinucleotide and human genetic disease\". Hum. Genet. 78 (2): 151–5. doi:10.1007/bf00278187. PMID 3338800.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":33,"type":"book","title":"Rogozin IB, Kolchanov NA (November 1992). \"Somatic hypermutagenesis in immunoglobulin genes. II. Influence of neighbouring base sequences on mutagenesis\". Biochim. Biophys. Acta. 1171 (1): 11–8. doi:10.1016/0167-4781(92)90134-l. ISSN 0006-3002. PMID 1420357.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":34,"type":"book","title":"Rogozin IB, Pavlov YI, Bebenek K, Matsuda T, Kunkel TA (June 2001). \"Somatic mutation hotspots correlate with DNA polymerase eta error spectrum\". Nat. Immunol. 2 (6): 530–6. doi:10.1038/88732. ISSN 1529-2908. PMID 11376340.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":35,"type":"book","title":"Alexandrov LB, Nik-Zainal S, Wedge DC, Campbell PJ, Stratton MR (January 2013). \"Deciphering signatures of mutational processes operative in human cancer\". Cell Rep. 3 (1): 246–59. doi:10.1016/j.celrep.2012.12.008. PMC 3588146. PMID 23318258.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":36,"type":"book","title":"Rosenthal R, McGranahan N, Herrero J, Taylor BS, Swanton C (February 2016). \"DeconstructSigs: delineating mutational processes in single tumors distinguishes DNA repair deficiencies and patterns of carcinoma evolution\". Genome Biol. 17: 31. doi:10.1186/s13059-016-0893-4. PMC 4762164. PMID 26899170.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":37,"type":"book","title":"Goncearenco A, Rager SL, Li M, Sang QX, Rogozin IB, Panchenko AR (July 2017). \"Exploring background mutational processes to decipher cancer genetic heterogeneity\". Nucleic Acids Res. 45 (W1): W514–W522. doi:10.1093/nar/gkx367. PMC 5793731. PMID 28472504.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":38,"type":"book","title":"Muller FL, Colla S, Aquilanti E, Manzo VE, Genovese G, Lee J, et al. (August 2012). \"Passenger deletions generate therapeutic vulnerabilities in cancer\". Nature. 488 (7411): 337–42. Bibcode:2012Natur.488..337M. doi:10.1038/nature11331. PMC 3712624. PMID 22895339.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":39,"type":"book","title":"Muller FL, Aquilanti EA, DePinho RA (November 2015). \"Collateral Lethality: A new therapeutic strategy in oncology\". Trends Cancer. 1 (3): 161–173. doi:10.1016/j.trecan.2015.10.002. PMC 4746004. PMID 26870836.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":40,"type":"book","title":"Poyner RR, Reed GH (August 1992). \"Structure of the bis divalent cation complex with phosphonoacetohydroxamate at the active site of enolase\". Biochemistry. 31 (31): 7166–73. doi:10.1021/bi00146a020. PMID 1322695.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":41,"type":"book","title":"Leonard PG, Satani N, Maxwell D, Lin YH, Hammoudi N, Peng Z, et al. (December 2016). \"SF2312 is a natural phosphonate inhibitor of enolase\". Nat. Chem. Biol. 12 (12): 1053–1058. doi:10.1038/nchembio.2195. PMC 5110371. PMID 27723749.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":42,"type":"book","title":"Dey P, Baddour J, Muller F, Wu CC, Wang H, Liao WT, et al. (February 2017). \"Genomic deletion of malic enzyme 2 confers collateral lethality in pancreatic cancer\". Nature. 542 (7639): 119–123. Bibcode:2017Natur.542..119D. doi:10.1038/nature21052. PMC 5398413. PMID 28099419.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":43,"type":"book","title":"Liou GY, Döppler H, DelGiorno KE, Zhang L, Leitges M, Crawford HC, Murphy MP, Storz P (March 2016). \"Mutant KRas-Induced Mitochondrial Oxidative Stress in Acinar Cells Upregulates EGFR Signaling to Drive Formation of Pancreatic Precancerous Lesions\". Cell Rep. 14 (10): 2325–36. doi:10.1016/j.celrep.2016.02.029. PMC 4794374. PMID 26947075.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":44,"type":"book","title":"Oike T, Ogiwara H, Tominaga Y, Ito K, Ando O, Tsuta K, et al. (September 2013). \"A synthetic lethality-based strategy to treat cancers harboring a genetic deficiency in the chromatin remodeling factor BRG1\". Cancer Res. 73 (17): 5508–18. doi:10.1158/0008-5472.CAN-12-4593. PMID 23872584.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":45,"type":"book","title":"Murata S, Zhang C, Finch N, Zhang K, Campo L, Breuer EK (2016). \"Predictors and Modulators of Synthetic Lethality: An Update on PARP Inhibitors and Personalized Medicine\". Biomed Res Int. 2016: 2346585. doi:10.1155/2016/2346585. PMC 5013223. PMID 27642590.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":46,"type":"book","title":"Lok BH, Carley AC, Tchang B, Powell SN (2013). \"RAD52 inactivation is synthetically lethal with deficiencies in BRCA1 and PALB2 in addition to BRCA2 through RAD51-mediated homologous recombination\". Oncogene. 32 (30): 3552–8. doi:10.1038/onc.2012.391. PMC 5730454. PMID 22964643.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":47,"type":"book","title":"Cramer-Morales K, Nieborowska-Skorska M, Scheibner K, Padget M, Irvine DA, Sliwinski T, Haas K, Lee J, Geng H, Roy D, Slupianek A, Rassool FV, Wasik MA, Childers W, Copland M, Müschen M, Civin CI, Skorski T (2013). \"Personalized synthetic lethality induced by targeting RAD52 in leukemias identified by gene mutation and expression profile\". Blood. 122 (7): 1293–304. doi:10.1182/blood-2013-05-501072. PMC 3744994. PMID 23836560.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":48,"type":"book","title":"Eshleman JR, Lang EZ, Bowerfind GK, Parsons R, Vogelstein B, Willson JK, Veigl ML, Sedwick WD, Markowitz SD (1995). \"Increased mutation rate at the hprt locus accompanies microsatellite instability in colon cancer\". Oncogene. 10 (1): 33–7. PMID 7824277.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":49,"type":"book","title":"Le DT, Uram JN, Wang H, Bartlett BR, Kemberling H, Eyring AD, et al. (2015). \"PD-1 Blockade in Tumors with Mismatch-Repair Deficiency\". N. Engl. J. Med. 372 (26): 2509–20. doi:10.1056/NEJMoa1500596. PMC 4481136. PMID 26028255.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":50,"type":"book","title":"Watanabe R, Ui A, Kanno S, Ogiwara H, Nagase T, Kohno T, Yasui A (2014). \"SWI/SNF factors required for cellular resistance to DNA damage include ARID1A and ARID1B and show interdependent protein stability\". Cancer Res. 74 (9): 2465–75. doi:10.1158/0008-5472.CAN-13-3608. PMID 24788099.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":51,"type":"book","title":"Raab JR, Resnick S, Magnuson T (2015). \"Genome-Wide Transcriptional Regulation Mediated by Biochemically Distinct SWI/SNF Complexes\". PLoS Genet. 11 (12): e1005748. doi:10.1371/journal.pgen.1005748. PMC 4699898. PMID 26716708.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":52,"type":"book","title":"Lawrence MS, Stojanov P, Mermel CH, Robinson JT, Garraway LA, Golub TR, Meyerson M, Gabriel SB, Lander ES, Getz G (2014). \"Discovery and saturation analysis of cancer genes across 21 tumour types\". Nature. 505 (7484): 495–501. Bibcode:2014Natur.505..495L. doi:10.1038/nature12912. PMC 4048962. PMID 24390350.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":53,"type":"book","title":"Zhang X, Sun Q, Shan M, Niu M, Liu T, Xia B, Liang X, Wei W, Sun S, Zhang Y, Liu XS, Song Q, Yang Y, Ma Y, Liu Y, Yang L, Ren Y, Zhang G, Pang D (2013). \"Promoter hypermethylation of ARID1A gene is responsible for its low mRNA expression in many invasive breast cancers\". PLoS ONE. 8 (1): e53931. Bibcode:2013PLoSO...853931Z. doi:10.1371/journal.pone.0053931. PMC 3549982. PMID 23349767.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":54,"type":"book","title":"Wu JN, Roberts CW (2013). \"ARID1A mutations in cancer: another epigenetic tumor suppressor?\". Cancer Discov. 3 (1): 35–43. doi:10.1158/2159-8290.CD-12-0361. PMC 3546152. PMID 23208470.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":55,"type":"book","title":"Bitler BG, Aird KM, Garipov A, Li H, Amatangelo M, Kossenkov AV, Schultz DC, Liu Q, Shih IeM, Conejo-Garcia JR, Speicher DW, Zhang R (2015). \"Synthetic lethality by targeting EZH2 methyltransferase activity in ARID1A-mutated cancers\". Nat. Med. 21 (3): 231–8. doi:10.1038/nm.3799. PMC 4352133. PMID 25686104.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":56,"type":"book","title":"Kim KH, Kim W, Howard TP, Vazquez F, Tsherniak A, Wu JN, Wang W, Haswell JR, Walensky LD, Hahn WC, Orkin SH, Roberts CW (2015). \"SWI/SNF-mutant cancers depend on catalytic and non-catalytic activity of EZH2\". Nat. Med. 21 (12): 1491–6. doi:10.1038/nm.3968. PMC 4886303. PMID 26552009.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":57,"type":"book","title":"Miller RE, Brough R, Bajrami I, Williamson CT, McDade S, Campbell J, et al. (2016). \"Synthetic Lethal Targeting of ARID1A-Mutant Ovarian Clear Cell Tumors with Dasatinib\". Mol. Cancer Ther. 15 (7): 1472–84. doi:10.1158/1535-7163.MCT-15-0554. PMID 27364904.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":58,"type":"book","title":"Kaelin W.G. (2005). \"The concept of synthetic lethality in the context of anticancer therapy\". Nature Reviews Cancer. 5 (9): 689–698. doi:10.1038/nrc1691. PMID 16110319.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":59,"type":"book","title":"O'Connor M.J.; Martin N.M.B.; Smith G.C.M. (2007). \"Targeted cancer therapies based on the inhibition of DNA strand break repair\". Oncogene. 26 (56): 7816–7824. doi:10.1038/sj.onc.1210879. PMID 18066095.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":60,"type":"book","title":"Farmer H, McCabe N, Lord CJ, Tutt AN, Johnson DA, Richardson TB, Santarosa M, Dillon KJ, Hickson I, Knights C, Martin NM, Jackson SP, Smith GC, Ashworth A (April 2005). \"Targeting the DNA repair defect in BRCA mutant cells as a therapeutic strategy\". Nature. 434 (7035): 917–21. Bibcode:2005Natur.434..917F. doi:10.1038/nature03445. PMID 15829967.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":61,"type":"book","title":"Bryant H.E.; Schultz, Niklas; Thomas, Huw D.; Parker, Kayan M.; Flower, Dan; Lopez, Elena; Kyle, Suzanne; Meuth, Mark; Curtin, Nicola J.; Helleday, Thomas; et al. (2005). \"Specific killing of BRCA2-deficient tumours with inhibitors of poly(ADP-ribose) polymerase\". Nature. 434 (7035): 913–917. Bibcode:2005Natur.434..913B. doi:10.1038/nature03443. PMID 15829966.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":62,"type":"book","title":"Gundem G, Perez-Llamas C, Jene-Sanz A, Kedzierska A, Islam A, Deu-Pons J, Furney SJ, Lopez-Bigas N (February 2010). \"IntOGen: integration and data mining of multidimensional oncogenomic data\". Nat. Methods. 7 (2): 92–3. doi:10.1038/nmeth0210-92. PMID 20111033.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":63,"type":"book","title":"Gonzalez-Perez A, Perez-Llamas C, Deu-Pons J, Tamborero D, Schroeder MP, Jene-Sanz A, Santos A, Lopez-Bigas N (November 2013). \"IntOGen-mutations identifies cancer drivers across tumor types\". Nat. Methods. 10 (11): 1081–2. doi:10.1038/nmeth.2642. PMC 5758042. PMID 24037244.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":64,"type":"book","title":"Rubio-Perez C, Tamborero D, Schroeder MP, Antolín AA, Deu-Pons J, Perez-Llamas C, Mestres J, Gonzalez-Perez A, Lopez-Bigas N (March 2015). \"In silico prescription of anticancer drugs to cohorts of 28 tumor types reveals targeting opportunities\". Cancer Cell. 27 (3): 382–96. doi:10.1016/j.ccell.2015.02.007. PMID 25759023.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":65,"type":"book","title":"Blume-Jensen P, Hunter T (May 2001). \"Oncogenic kinase signalling\". Nature. 411 (6835): 355–65. doi:10.1038/35077225. PMID 11357143.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":66,"type":"book","title":"Bardelli A, et al. (2003). \"Mutational analysis of the tyrosine kinome in colorectal cancers\". Science. 300 (5621): 949–949. doi:10.1126/science.1082596. PMID 12738854.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":67,"type":"book","title":"Samuels Y, Wang Z, Bardelli A, Silliman N, Ptak J, Szabo S, Yan H, Gazdar A, Powell SM, Riggins GJ, Willson JK, Markowitz S, Kinzler KW, Vogelstein B, Velculescu VE (April 2004). \"High frequency of mutations of the PIK3CA gene in human cancers\". Science. 304 (5670): 554. doi:10.1126/science.1096502. PMID 15016963.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":68,"type":"book","title":"Davies H, Bignell GR, Cox C, Stephens P, Edkins S, Clegg S, et al. (June 2002). \"Mutations of the BRAF gene in human cancer\". Nature. 417 (6892): 949–54. doi:10.1038/nature00766. PMID 12068308.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":69,"type":"book","title":"Danson S.; Lorigan P. (2005). \"Improving outcomes in advanced malignant melanoma - Update on systemic therapy\". Drugs. 65 (6): 733–743. doi:10.2165/00003495-200565060-00002. PMID 15819587.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":70,"type":"book","title":"Yu, Man (2012). \"Somatic Mitochondrial DNA Mutations in Human Cancers\". Advances in Clinical Chemistry. Advances in Clinical Chemistry. 57: 99–138. doi:10.1016/B978-0-12-394384-2.00004-8. ISBN 9780123943842. PMID 22870588.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":71,"type":"book","title":"Hung, W.Y.; J.C. Lin; L.M. Lee; et al. (2008). \"Tandem duplication/triplication correlated with poly-cytosine stretch variation in human mitochondrial DNA D-loop region\". Mutagenesis. 23 (2): 137–142. doi:10.1093/mutage/gen002. PMID 18252697.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":72,"type":"book","title":"Fliss, M. S.; Usadel, H.; Caballero, O. L.; et al. (2000). \"Facile detection of mitochondrial DNA mutations in tumors and bodily fluids\". Science. 287 (5460): 2017–2019. Bibcode:2000Sci...287.2017F. doi:10.1126/science.287.5460.2017. PMID 10720328.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":73,"type":"book","title":"Dani, M.A.; S.U. Dani; S.P. Lima; et al. (2004). \"Less ΔmtDNA4977 than normal in various types of tumors suggests that cancer cells are essentially free of this mutation\". Genet. Mol. Res. 3 (3): 395–409. PMID 15614730.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":74,"type":"book","title":"Ye, C.; X.O. Shu; W. Wen; et al. (2008). \"Quantitative analysis of mitochondrial DNA 4977-bp deletion in sporadic breast cancer and benign breast diseases\". Breast Cancer Res. Treat. 108 (3): 427–434. doi:10.1007/s10549-007-9613-9. PMC 3836503. PMID 17541740.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":75,"type":"book","title":"Tseng, L.M.; P.H. Yin; C.W. Chi; et al. (2006). \"Mitochondrial DNA mutations and mitochondrial DNA depletion in breast cancer\". Genes Chromosomes Cancer. 45 (7): 629–638. doi:10.1002/gcc.20326. PMID 16568452.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":76,"type":"book","title":"Zhu, W.; W. Qin; P. Bradley; A. Wessel; C.L. Puckett; E.R. Sauter (2005). \"Mitochondrial DNA mutations in breast cancer tissue and in matched nipple aspirate fluid\". Carcinogenesis. 26 (1): 145–152. doi:10.1093/carcin/bgh282. PMID 15375011.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":77,"type":"book","title":"Zhou S, Kachhap S, Sun W, Wu G, Chuang A, Poeta L, Grumbine L, Mithani SK, Chatterjee A, Koch W, Westra WH, Maitra A, Glazer C, Carducci M, Sidransky D, McFate T, Verma A, Califano JA (May 2007). \"Frequency and phenotypic implications of mitochondrial DNA mutations in human squamous cell cancers of the head and neck\". Proc. Natl. Acad. Sci. U.S.A. 104 (18): 7540–5. doi:10.1073/pnas.0610818104. PMC 1863503. PMID 17456604.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":78,"type":"book","title":"Poetsch M, Petersmann A, Lignitz E, Kleist B (March 2004). \"Relationship between mitochondrial DNA instability, mitochondrial DNA large deletions, and nuclear microsatellite instability in head and neck squamous cell carcinomas\". Diagn. Mol. Pathol. 13 (1): 26–32. doi:10.1097/00019606-200403000-00005. PMID 15163006.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":79,"type":"book","title":"Tan, D.J.; J. Chang; W.L. Chen; et al. (2004). \"Somatic mitochondrial DNA mutations in oral cancer of betel quid chewers\". Ann. N. Y. Acad. Sci. 1011: 310–316. Bibcode:2004NYASA1011..310T. doi:10.1196/annals.1293.030. PMID 15126307.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":80,"type":"book","title":"Lee, H.C.; S.H. Li; J.C. Lin; C.C. Wu; D.C. Yeh; Y.H. Wei (2004). \"Somatic mutations in the D-loop and decrease in the copy number of mitochondrial DNA in human hepatocellular carcinoma\". Mutation Research. 547 (1–2): 71–78. doi:10.1016/j.mrfmmm.2003.12.011. PMID 15013701.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":81,"type":"book","title":"Yin, P.H.; C.C. Wu; J.C. Lin; C.W. Chi; Y.H. Wei; H.C. Lee (2010). \"Somatic mutations of mitochondrial genome in hepatocellular carcinoma\". Mitochondrion. 10 (2): 174–182. doi:10.1016/j.mito.2009.12.147. PMID 20006738.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":82,"type":"book","title":"Tan, D.J.; J. Chang; L.L. Liu; et al. (2006). \"Significance of somatic mutations and content alteration of mitochondrial DNA in esophageal cancer\". BMC Cancer. 6: 93. doi:10.1186/1471-2407-6-93. PMC 1459869. PMID 16620376.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":83,"type":"book","title":"Kassauei K, Habbe N, Mullendore ME, Karikari CA, Maitra A, Feldmann G (2006). \"Mitochondrial DNA mutations in pancreatic cancer\". Int J Gastrointest Cancer. 37 (2–3): 57–64. doi:10.1007/s12029-007-0008-2. PMID 17827523.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":84,"type":"book","title":"Hung WY, Wu CW, Yin PH, Chang CJ, Li AF, Chi CW, Wei YH, Lee HC (March 2010). \"Somatic mutations in mitochondrial genome and their potential roles in the progression of human gastric cancer\". Biochim. Biophys. Acta. 1800 (3): 264–70. doi:10.1016/j.bbagen.2009.06.006. PMID 19527772.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":85,"type":"book","title":"Wu CW, Yin PH, Hung WY, Li AF, Li SH, Chi CW, Wei YH, Lee HC (September 2005). \"Mitochondrial DNA mutations and mitochondrial DNA depletion in gastric cancer\". Genes Chromosomes Cancer. 44 (1): 19–28. doi:10.1002/gcc.20213. PMID 15892105.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":86,"type":"book","title":"Yu, J.J.; T. Yan (2010). \"Effect of mtDNA mutation on tumor malignant degree in patients with prostate cancer\". Aging Male. 13 (3): 159–165. doi:10.3109/13685530903536668. PMID 20136572.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":87,"type":"book","title":"Gomez-Zaera, M.; J. Abril; L. Gonzalez; et al. (2006). \"Identification of somatic and germline mitochondrial DNA sequence variants in prostate cancer patients\". Mutation Research. 595 (1–2): 42–51. doi:10.1016/j.mrfmmm.2005.10.012. PMID 16472830.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":88,"type":"book","title":"Ludwig JA, Weinstein JN (November 2005). \"Biomarkers in cancer staging, prognosis and treatment selection\". Nat. Rev. Cancer. 5 (11): 845–56. doi:10.1038/nrc1739. PMID 16239904.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false}],"recommendReferences":null,"auditState":2,"lemmaLevel":1,"origin":0,"originEnTitle":null,"originZhTitle":null,"pv":39,"auditType":0,"synonyms":null,"showEditTime":"2019.12.13 18:46","auditors":[{"uid":0,"name":"Ki.κe","pic":"https://web.archive.org/web/20221025184839/https://wx.qlogo.cn/mmopen/vi_32/y67kfr32Doib4wg71Jiau7jVWvharic3nRKgdRRQSl6koeQJCo0GQs2Krw0vwdFRsOWnHIQOwAZsSg5lIkIrFCOcQ/132","introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":"","jobBrief":"","role":0,"roleName":null,"title":"","professionalTitle":null,"phoneNo":null,"editable":true,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false}],"hasZhishiNav":false,"auditInfos":{},"isHistory":false};</script><script crossorigin="anonymous" src="./636.肿瘤基因组学 - 搜狗科学百科_files/aegis.min.js.download"></script><script crossorigin="anonymous" src="./636.肿瘤基因组学 - 搜狗科学百科_files/main_2020092401.js.download"></script><script crossorigin="anonymous" src="./636.肿瘤基因组学 - 搜狗科学百科_files/react.production.min.js.download"></script><script crossorigin="anonymous" src="./636.肿瘤基因组学 - 搜狗科学百科_files/react-dom.production.min.js.download"></script><script crossorigin="anonymous" src="./636.肿瘤基因组学 - 搜狗科学百科_files/jquery-1.11.1.min.js.download"></script><script crossorigin="anonymous" src="./636.肿瘤基因组学 - 搜狗科学百科_files/main_2022062701.js.download"></script><script crossorigin="anonymous" src="./636.肿瘤基因组学 - 搜狗科学百科_files/main_66bbe21.js.download"></script><script crossorigin="anonymous" src="./636.肿瘤基因组学 - 搜狗科学百科_files/react.production.min.js.download"></script><script crossorigin="anonymous" src="./636.肿瘤基因组学 - 搜狗科学百科_files/react-dom.production.min.js.download"></script><script crossorigin="anonymous" src="./636.肿瘤基因组学 - 搜狗科学百科_files/main_edf0f08.js.download"></script>
</body></html>