<!DOCTYPE html>
<!-- saved from url=(0083)https://web.archive.org/web/20221025183652/https://baike.sogou.com/kexue/d10661.htm -->
<html class="" data-reactroot=""><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><script src="./661.重亚硫酸盐测序 - 搜狗科学百科_files/analytics.js.download" type="text/javascript"></script>
<script type="text/javascript">window.addEventListener('DOMContentLoaded',function(){var v=archive_analytics.values;v.service='wb';v.server_name='wwwb-app224.us.archive.org';v.server_ms=266;archive_analytics.send_pageview({});});</script>
<script type="text/javascript" src="./661.重亚硫酸盐测序 - 搜狗科学百科_files/bundle-playback.js.download" charset="utf-8"></script>
<script type="text/javascript" src="./661.重亚硫酸盐测序 - 搜狗科学百科_files/wombat.js.download" charset="utf-8"></script>
<script type="text/javascript">
  __wm.init("https://web.archive.org/web");
  __wm.wombat("https://baike.sogou.com/kexue/d10661.htm","20221025183652","https://web.archive.org/","web","/_static/",
	      "1666723012");
</script>
<link rel="stylesheet" type="text/css" href="./661.重亚硫酸盐测序 - 搜狗科学百科_files/banner-styles.css">
<link rel="stylesheet" type="text/css" href="./661.重亚硫酸盐测序 - 搜狗科学百科_files/iconochive.css">
<!-- End Wayback Rewrite JS Include -->
<meta name="save" content="history"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="baidu-site-verification" content="VWGb6TyYx8"><meta content="重亚硫酸盐测序 - 搜狗科学百科" name="keywords"><meta content="搜狗科学百科是一部有着平等、协作、分享、自由理念的网络科学全书，为每一个互联网用户创造一个涵盖所有领域知识、服务的中文知识性平台。" name="description"><meta http-equiv="x-dns-prefetch-control" content="on"><meta name="server" baike="235" ip="210" env="online"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025183652/https://cache.soso.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025183652/https://hhy.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025183652/https://pic.baike.soso.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025183652/https://ugc.qpic.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025183652/https://xui.ptlogin2.qq.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025183652/https://q1.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025183652/https://q2.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025183652/https://q3.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025183652/https://q4.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025183652/https://q.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025183652/https://img01.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025183652/https://img02.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025183652/https://img03.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025183652/https://img04.sogoucdn.com/"><link rel="Shortcut Icon" href="https://web.archive.org/web/20221025183652im_/https://www.sogou.com/images/logo/new/favicon.ico?v=4"><link rel="Bookmark" href="https://www.sogou.com/images/logo/new/favicon.ico?v=4"><link href="./661.重亚硫酸盐测序 - 搜狗科学百科_files/base_b849887.css" rel="stylesheet"><link href="./661.重亚硫酸盐测序 - 搜狗科学百科_files/detail_378aed5.css" rel="stylesheet"><link href="./661.重亚硫酸盐测序 - 搜狗科学百科_files/inviteAudit_7894507.css" rel="stylesheet"><link rel="stylesheet" href="./661.重亚硫酸盐测序 - 搜狗科学百科_files/highlight.min.css"><title>重亚硫酸盐测序 - 搜狗科学百科</title><style>.onekey-close {
	position: absolute;
	top: 16px;
	right: 16px;
	width: 24px;
	height: 24px;
	background: url(//web.archive.org/web/20221025193758/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	text-indent: -999em;
	background-size: 84px;
	background-position: -63px 0;
}

.onekey-login {
	position: absolute;
	top: 16.4%;
	left: 0;
	right: 0;
	width: 100%;
}

/* .onekey-login-img {
    width: 75px;
    height: 75px;
    background: url("https://web.archive.org/web/20221025183652/https://baike.sogou.com/kexue/images/sprite_wap_baike.png") no-repeat;
    background-size: 100px 91px;
    background-position: 0 0;
    background-repeat: no-repeat;
    margin: 0 auto;
} */

.onekey-login-title {
	text-align: center;
	padding-bottom: 3px;
	font-size: 21px;
	font-weight: bold;
	line-height: 30px;
	color: #000;
}

.onekey-login-txt {
	text-align: center;
	font-family: PingFangSC;
	font-size: 14px;
	line-height: 20px;
	color: #8f8f8f;
}

.onekey-login-qq,
.onekey-login-wx,
.onekey-login-phone {
	display: block;
	width: 245px;
	height: 54px;
	border-radius: 45px;
	text-align: center;

	margin: 0 auto;
	font-size: 17px;
	line-height: 24px;
	color: #000;
	/* padding: 16px 77px; */
	border-radius: 12px;
	border: solid 1px #e0e0e0;
}
.onekey-qq-content,
.onekey-vx-content,
.onekey-phone-content {
	display: inline-block;
	margin-top: 16px;
}
.onekey-qq-content {
	padding: 0 5px;
}

.onekey-login-qq {
	margin-top: 48px;
	margin-bottom: 24px;
}

.onekey-login-qq:before {
	display: inline-block;
	content: "";
	width: 20px;
	height: 20px;
	background: url(//web.archive.org/web/20221025193758/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	background-size: 80px;
	background-position: -20px 0;
	vertical-align: top;
	margin: 17px 8px 0 0;
}

.onekey-login-wx {
	margin-bottom: 24px;
}

.onekey-login-wx:before {
	display: inline-block;
	content: "";
	width: 21px;
	height: 21px;
	background: url(//web.archive.org/web/20221025193758/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	background-size: 84px;
	background-position: 0 0;
	vertical-align: top;
	margin: 17px 10px 0 0;
}

.onekey-login-phone {
}

.onekey-login-phone:before {
	display: inline-block;
	content: "";
	width: 21px;
	height: 21px;
	background: url(//web.archive.org/web/20221025193758/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	background-size: 84px;
	background-position: -42px 0;
	vertical-align: top;
	margin: 17px 10px 0 0;
}

.onekey-fixed {
	z-index: 100;
	position: fixed;
	top: 0;
	bottom: 0;
	left: 0;
	right: 0;
	background: #fff;
	width: 100%;
	height: 100%;
}

.onekey-fixed.forbid {
	z-index: 100;
	position: fixed;
	top: auto;
	bottom: 68px;
	left: 9%;
	right: 0;
	background: rgba(0, 0, 0, 0.7);
	width: 82%;
	height: 43px;
	border-radius: 25px;
	color: #ffffff;
}
.onekey-login-title.forbid {
	text-align: center;
	padding-bottom: 3px;
	font-size: 14px;
	font-weight: normal;
	line-height: 30px;
	color: white;
}
</style><style>#login_mask {
  background: #000;
  opacity: 0.5;
  filter: alpha(opacity=50);
  position: fixed;
  /*fixed好像在哪个IE上有BUG，先用用*/
  left: 0;
  top: 0;
  z-index: 999;
  height: 100%;
}

#login_iframe_container {
  position: fixed;
  width: 550px;
  height: 360px;
  z-index: 1020;
  background-color: #ffffff;
}

@media screen and (max-width: 828px) {
  #login_iframe_container {
    top: 50% !important;
    left: 50% !important;
    transform: translate(-50%, -50%);
  }
}

#login_iframe_container.new-login {
  width: 550px;
  height: 360px;
  background-image: url(//web.archive.org/web/20221025193758/https://hhy.sogoucdn.com/js/common/hhy/background_2a4a8a6.png);
}

#login_iframe_container.new-login.no-bg {
  background: #fff;
}

#login_iframe_container.new-login .login-title {
  width: 100%;
  height: 42px;
  line-height: 42px;
  text-align: center;
  font-size: 30px;
  letter-spacing: 0.19px;
  color: #ffffff;
  margin-top: 62px;
}
#login_iframe_container.new-login .forbid-title {
  width: 100%;
  height: 42px;
  line-height: 42px;
  text-align: center;
  font-size: 24px;
  letter-spacing: 0.19px;
  color: #333333;
  margin-top: 150px;
}

#login_iframe_container.new-login.no-bg .login-title {
  color: #333333;
}

#login_iframe_container.new-login .login-subtitle {
  width: 100%;
  height: 18px;
  line-height: 18px;
  font-size: 13px;
  letter-spacing: 0.08px;
  color: #ffffff;
  text-align: center;
  margin-top: 9px;
  margin-bottom: 43px;
}

#login_iframe_container.new-login.no-bg .login-subtitle {
  color: #999999;
}

#login_iframe_container.new-login .login-subtitle::before {
  content: '';
  display: inline-block;
  width: 10px;
  height: 1px;
  background-color: #ffffff;
  position: relative;
  top: -4px;
  left: -5px;
}

#login_iframe_container.new-login .login-subtitle::after {
  content: '';
  display: inline-block;
  width: 10px;
  height: 1px;
  background-color: #ffffff;
  position: relative;
  top: -4px;
  left: 5px;
}

#login_iframe_container.new-login.no-bg .login-subtitle::before {
  background-color: #999999;
}

#login_iframe_container.new-login.no-bg .login-subtitle::after {
  background-color: #999999;
}

#login_iframe_container.new-login .close-btn {
  position: absolute;
  top: 20px;
  right: 20px;
  width: 12px;
  height: 12px;
  background: url(//web.archive.org/web/20221025193758/https://hhy.sogoucdn.com/js/common/hhy/login-sprites_e3853e5.png) -59px -10px;
  background-size: 81px 91px;
  cursor: pointer;
}

#login_iframe_container.new-login .login-btn {
  width: 220px;
  height: 47px;
  border-radius: 24px;
  border: solid 1px #dddddd;
  background-color: #ffffff;
  margin: 0 auto;
  margin-top: 28px;
  position: relative;
  display: block;
}

#login_iframe_container.new-login .login-btn .login-icon {
  position: absolute;
}

#login_iframe_container.new-login .login-btn .login-text {
  width: 61px;
  height: 47px;
  line-height: 47px;
  vertical-align: middle;
  font-size: 15px;
  letter-spacing: 0.1px;
  color: #666666;
  position: absolute;
  right: 62px;
}

#login_iframe_container.new-login .login-btn.qq-btn .login-icon {
  width: 22px;
  height: 27px;
  top: 10px;
  left: 67px;
  background: url(//web.archive.org/web/20221025193758/https://hhy.sogoucdn.com/js/common/hhy/login-sprites_e3853e5.png) -10px -54px;
  background-size: 81px 91px;
}

#login_iframe_container.new-login .login-btn.qq-btn .login-text {
  right: 59px;
}

#login_iframe_container.new-login .login-btn.wechat-btn .login-icon {
  width: 29px;
  height: 24px;
  top: 12px;
  left: 62px;
  background: url(//web.archive.org/web/20221025193758/https://hhy.sogoucdn.com/js/common/hhy/login-sprites_e3853e5.png) -10px -10px;
  background-size: 81px 91px;
}</style><style>/* -- container -- */
.rodal,
.rodal-mask {
    top: 0;
    left: 0;
    width: 100%;
    height: 100%;
    z-index: 100;
}

.rodal {
    position: fixed;
}

/* -- mask -- */
.rodal-mask {
    position: fixed;
    background: rgba(0, 0, 0, .5);
}

/* -- dialog -- */
.rodal-dialog {
    position: absolute;
    z-index: 101;
    background: #fff;
    border-radius: 3px;
    box-shadow: 0 1px 3px rgba(0, 0, 0, .2);
}

.rodal-center {
    top: 50%;
    transform: translateY(-50%);
    left: 0;
    right: 0;
    margin: 0 auto;
}

.rodal-bottom {
    left: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

.rodal-top {
    left: 0;
    right: 0;
    top: 0;
    margin: auto;
}

.rodal-left {
    top: 0;
    left: 0;
    bottom: 0;
    margin: auto;
}

.rodal-right {
    top: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

/* -- close button -- */
.rodal-close {
    position: absolute;
    cursor: pointer;
    top: 16px;
    right: 16px;
    width: 16px;
    height: 16px;
}

.rodal-close:before,
.rodal-close:after {
    position: absolute;
    content: '';
    height: 2px;
    width: 100%;
    top: 50%;
    left: 0;
    margin-top: -1px;
    background: #999;
    border-radius: 100%;
    -webkit-transition: background .2s;
    transition: background .2s;
}

.rodal-close:before {
    -webkit-transform: rotate(45deg);
    transform: rotate(45deg);
}

.rodal-close:after {
    -webkit-transform: rotate(-45deg);
    transform: rotate(-45deg);
}

.rodal-close:hover:before,
.rodal-close:hover:after {
    background: #333;
}

/* -- fade -- */
/* @-webkit-keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

@keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

.rodal-fade-enter {
    -webkit-animation: rodal-fade-enter both ease-in;
    animation: rodal-fade-enter both ease-in;
} */

@-webkit-keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

@keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

.rodal-fade-leave {
    -webkit-animation: rodal-fade-leave both ease-out;
    animation: rodal-fade-leave both ease-out;
}

/* -- zoom -- */
@-webkit-keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-enter {
    -webkit-animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-leave {
    -webkit-animation: rodal-zoom-leave both;
    animation: rodal-zoom-leave both;
}

/* -- slideDown -- */
@-webkit-keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-enter {
    -webkit-animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-leave {
    -webkit-animation: rodal-slideDown-leave both;
    animation: rodal-slideDown-leave both;
}

/* -- slideLeft -- */
@-webkit-keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-enter {
    -webkit-animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-leave {
    -webkit-animation: rodal-slideLeft-leave both;
    animation: rodal-slideLeft-leave both;
}

/* -- slideRight -- */
@-webkit-keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-enter {
    -webkit-animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-leave {
    -webkit-animation: rodal-slideRight-leave both;
    animation: rodal-slideRight-leave both;
}

/* -- slideUp -- */
@-webkit-keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-enter {
    -webkit-animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
    animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
}

@-webkit-keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-leave {
    -webkit-animation: rodal-slideUp-leave both;
    animation: rodal-slideUp-leave both;
}

/* -- flip -- */
@-webkit-keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

@keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

.rodal-flip-enter {
    -webkit-animation: rodal-flip-enter both ease-in;
    animation: rodal-flip-enter both ease-in;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

@-webkit-keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

@keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

.rodal-flip-leave {
    -webkit-animation: rodal-flip-leave both;
    animation: rodal-flip-leave both;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

/* -- rotate -- */
@-webkit-keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-enter {
    -webkit-animation: rodal-rotate-enter both;
    animation: rodal-rotate-enter both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

@-webkit-keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-leave {
    -webkit-animation: rodal-rotate-leave both;
    animation: rodal-rotate-leave both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

/* -- door -- */
@-webkit-keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

@keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

.rodal-door-enter {
    -webkit-animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

@keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

.rodal-door-leave {
    -webkit-animation: rodal-door-leave both;
    animation: rodal-door-leave both;
}</style><style>/* -- container -- */
.rodal,
.rodal-mask {
    top: 0;
    left: 0;
    width: 100%;
    height: 100%;
    z-index: 100;
}

.rodal {
    position: fixed;
}

/* -- mask -- */
.rodal-mask {
    position: fixed;
    background: rgba(0, 0, 0, .5);
}

/* -- dialog -- */
.rodal-dialog {
    position: absolute;
    z-index: 101;
    background: #fff;
    border-radius: 3px;
    box-shadow: 0 1px 3px rgba(0, 0, 0, .2);
}

.rodal-center {
    top: 50%;
    transform: translateY(-50%);
    left: 0;
    right: 0;
    margin: 0 auto;
}

.rodal-bottom {
    left: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

.rodal-top {
    left: 0;
    right: 0;
    top: 0;
    margin: auto;
}

.rodal-left {
    top: 0;
    left: 0;
    bottom: 0;
    margin: auto;
}

.rodal-right {
    top: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

/* -- close button -- */
.rodal-close {
    position: absolute;
    cursor: pointer;
    top: 16px;
    right: 16px;
    width: 16px;
    height: 16px;
}

.rodal-close:before,
.rodal-close:after {
    position: absolute;
    content: '';
    height: 2px;
    width: 100%;
    top: 50%;
    left: 0;
    margin-top: -1px;
    background: #999;
    border-radius: 100%;
    -webkit-transition: background .2s;
    transition: background .2s;
}

.rodal-close:before {
    -webkit-transform: rotate(45deg);
    transform: rotate(45deg);
}

.rodal-close:after {
    -webkit-transform: rotate(-45deg);
    transform: rotate(-45deg);
}

.rodal-close:hover:before,
.rodal-close:hover:after {
    background: #333;
}

/* -- fade -- */
/* @-webkit-keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

@keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

.rodal-fade-enter {
    -webkit-animation: rodal-fade-enter both ease-in;
    animation: rodal-fade-enter both ease-in;
} */

@-webkit-keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

@keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

.rodal-fade-leave {
    -webkit-animation: rodal-fade-leave both ease-out;
    animation: rodal-fade-leave both ease-out;
}

/* -- zoom -- */
@-webkit-keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-enter {
    -webkit-animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-leave {
    -webkit-animation: rodal-zoom-leave both;
    animation: rodal-zoom-leave both;
}

/* -- slideDown -- */
@-webkit-keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-enter {
    -webkit-animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-leave {
    -webkit-animation: rodal-slideDown-leave both;
    animation: rodal-slideDown-leave both;
}

/* -- slideLeft -- */
@-webkit-keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-enter {
    -webkit-animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-leave {
    -webkit-animation: rodal-slideLeft-leave both;
    animation: rodal-slideLeft-leave both;
}

/* -- slideRight -- */
@-webkit-keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-enter {
    -webkit-animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-leave {
    -webkit-animation: rodal-slideRight-leave both;
    animation: rodal-slideRight-leave both;
}

/* -- slideUp -- */
@-webkit-keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-enter {
    -webkit-animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
    animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
}

@-webkit-keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-leave {
    -webkit-animation: rodal-slideUp-leave both;
    animation: rodal-slideUp-leave both;
}

/* -- flip -- */
@-webkit-keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

@keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

.rodal-flip-enter {
    -webkit-animation: rodal-flip-enter both ease-in;
    animation: rodal-flip-enter both ease-in;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

@-webkit-keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

@keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

.rodal-flip-leave {
    -webkit-animation: rodal-flip-leave both;
    animation: rodal-flip-leave both;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

/* -- rotate -- */
@-webkit-keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-enter {
    -webkit-animation: rodal-rotate-enter both;
    animation: rodal-rotate-enter both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

@-webkit-keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-leave {
    -webkit-animation: rodal-rotate-leave both;
    animation: rodal-rotate-leave both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

/* -- door -- */
@-webkit-keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

@keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

.rodal-door-enter {
    -webkit-animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

@keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

.rodal-door-leave {
    -webkit-animation: rodal-door-leave both;
    animation: rodal-door-leave both;
}</style></head><body class=""><!-- BEGIN WAYBACK TOOLBAR INSERT -->
<style type="text/css">
body {
  margin-top:0 !important;
  padding-top:0 !important;
  /*min-width:800px !important;*/
}
</style>
<script>__wm.rw(0);</script>
<div id="wm-ipp-base" lang="en" style="display: block; direction: ltr;">
</div><div id="wm-ipp-print">The Wayback Machine - https://web.archive.org/web/20221025183652/https://baike.sogou.com/kexue/d10661.htm</div>
<script type="text/javascript">//<![CDATA[
__wm.bt(675,27,25,2,"web","https://baike.sogou.com/kexue/d10661.htm","20221025183652",1996,"/_static/",["/_static/css/banner-styles.css?v=S1zqJCYt","/_static/css/iconochive.css?v=qtvMKcIJ"], false);
  __wm.rw(1);
//]]></script>
<!-- END WAYBACK TOOLBAR INSERT --><script>window._gtag=window._gtag||{};window._gtag.shouldGrayed = false;if ('5a4fdb639a9f49858b0f524c717427ef') window._gtag.traceId = '5a4fdb639a9f49858b0f524c717427ef';if ({"illegality":true}) window.userInfo = {"illegality":true};</script><div class="topnavbox"><ul class="topnav"><li><a href="https://web.archive.org/web/20221025183652/https://www.sogou.com/web?query=">网页</a></li><li><a href="https://web.archive.org/web/20221025183652/https://weixin.sogou.com/weixin?p=75351201">微信</a></li><li><a href="https://web.archive.org/web/20221025183652/https://zhihu.sogou.com/zhihu?p=75351218">知乎</a></li><li><a href="https://web.archive.org/web/20221025183652/https://pic.sogou.com/pics?query=">图片</a></li><li><a href="https://web.archive.org/web/20221025183652/https://v.sogou.com/v?query=">视频</a></li><li><a href="https://web.archive.org/web/20221025183652/https://mingyi.sogou.com/">医疗</a></li><li class="cur"><strong>科学</strong></li><li><a href="https://web.archive.org/web/20221025183652/https://hanyu.sogou.com/">汉语</a></li><li><a href="https://web.archive.org/web/20221025183652/https://wenwen.sogou.com/">问问</a></li><li><a href="https://web.archive.org/web/20221025183652/https://www.sogou.com/docs/more.htm">更多<span class="topraquo">»</span></a></li></ul></div><div id="header"><div class="header-wrap"><a class="header-logo" href="https://web.archive.org/web/20221025183652/https://baike.sogou.com/kexue"></a><div class="header-search"><div class="querybox" id="suggBox"><form><input id="searchInput" class="query" type="text" placeholder="搜科学领域专业百科词条" name="query" autocomplete="off" value=""><a href="javascript:;" class="query-search"></a></form></div></div><div class="header-rgt"><span class="btn-header-rgt btn-edit" id="editLemma">创建</span><div class="header-user no-login"></div></div></div></div><div class="fixed-placeholder" style="visibility:none"></div><div id="container" class=""><div class="content lemma-level1"><div class="detail-title" id="abstract-title"><h1>重亚硫酸盐测序</h1><a href="https://web.archive.org/web/20221025183652/https://baike.sogou.com/kexue/d10661.htm#!" class="detail-edit">编辑</a></div><div class="section_content" data-id="14995449974882577"><div class="text_img ed_imgfloat_right"><a class="ed_image_link" data-src="https://img03.sogoucdn.com/app/a/200698/sogou_science_13618" data-bigsrc="https://img03.sogoucdn.com/app/a/200698/sogou_science_13618?w=300&amp;h=209&amp;titlename=%E5%9B%BE1:%20%E9%87%8D%E4%BA%9A%E7%A1%AB%E9%85%B8%E7%9B%90%E8%BD%AC%E5%8C%96%E5%9F%BA%E5%9B%A0%E7%BB%84DNA%E5%BA%8F%E5%88%97%E6%A1%88%E4%BE%8B%E6%A6%82%E8%BF%B0%E3%80%82%E8%93%9D%E8%89%B2%E6%A0%B8%E8%8B%B7%E9%85%B8%E6%98%AF%E9%87%8D%E4%BA%9A%E7%A1%AB%E9%85%B8%E7%9B%90%E8%BD%AC%E5%8C%96%E4%B8%BA%E5%B0%BF%E5%98%A7%E5%95%B6%E7%9A%84%E6%9C%AA%E7%94%B2%E5%9F%BA%E5%8C%96%E8%83%9E%E5%98%A7%E5%95%B6%EF%BC%8C%E8%80%8C%E7%BA%A2%E8%89%B2%E6%A0%B8%E8%8B%B7%E9%85%B8%E6%98%AF%E5%AF%B9%E8%BD%AC%E5%8C%96%E6%9C%89%E6%8A%97%E6%80%A7%E7%9A%845-%E7%94%B2%E5%9F%BA%E8%83%9E%E5%98%A7%E5%95%B6%E3%80%82" title="点击查看大图" data-w="300" data-h="209" style="background-image:url(https://web.archive.org/web/20221025183652im_/https://img03.sogoucdn.com/app/a/200698/sogou_science_13618)" href="https://web.archive.org/web/20221025183652/https://baike.sogou.com/kexue/d10661.htm#!"></a><div class="text_img_title">图1: 重亚硫酸盐转化基因组DNA序列案例概述。蓝色核苷酸是重亚硫酸盐转化为尿嘧啶的未甲基化胞嘧啶，而红色核苷酸是对转化有抗性的5-甲基胞嘧啶。</div></div><div><p></p> 
<p></p> 
<p>重亚硫酸盐测序(也称为亚硫酸氢盐测序)<sup><a href="https://web.archive.org/web/20221025183652/https://baike.sogou.com/kexue/d10661.htm#quote_1" class="kx_ref">[1]</a></sup> 是在常规测序前使用重亚硫酸盐处理来测定甲基化情形的方法。脱氧核糖核酸 (DNA)甲基化是第一个被发现的表观遗传标记，并且到现在为止仍然是研究最多的表观遗传改变。在动物中，DNA甲基化主要是指在二核苷酸CpG的胞嘧啶残基的5号碳位添加一个甲基，它可以对基因的转录活性起抑制作用。 </p>
<p>用重亚硫酸盐处理DNA可以将胞嘧啶残基转化为尿嘧啶，但5-甲基胞嘧啶残基不受影响。因此，用重亚硫酸盐处理过的DNA只保留甲基化胞嘧啶。基于此原理，重亚硫酸盐处理引入了依赖于单个胞嘧啶残基甲基化状态的特定的DNA序列变化，产生了关于DNA片段甲基化状态的单核苷酸分辨率信息。可以对改变后的序列进行各种分析来检索这些信息。因此，该分析的目的被简化为区分重亚硫酸盐转化产生的单核苷酸多态性(胞嘧啶和<b>胸腺嘧啶</b>)(图1)。 </p></div></div><div id="catalog"><h2 class="title2">目录<a href="javascript:" class="detail-edit">编辑</a></h2><div class="catalog_wrap" style=""><ul class="catalog_list col2"><li><span class="order">1</span><a href="javascript:" data-level="1" data-id="14995449974882578">方法</a></li><li class="secondary_catalog"><span>1.1 </span><a href="javascript:" data-id="14995449974882578">基于非甲基化特异性PCR的方法</a></li><li class="secondary_catalog"><span>1.2 </span><a href="javascript:" data-id="14995449974882578">甲基化特异性PCR</a></li><li class="secondary_catalog"><span>1.3 </span><a href="javascript:" data-id="14995449974882578">基于微阵列的方法</a></li><li><span class="order">2</span><a href="javascript:" data-level="1" data-id="14995449991659785">局限性</a></li><li class="secondary_catalog"><span>2.1 </span><a href="javascript:" data-id="14995449991659785">5-羟甲基胞嘧啶</a></li><li class="secondary_catalog"><span>2.2 </span><a href="javascript:" data-id="14995449991659785">不完全转化</a></li></ul><ul class="catalog_list col2"><li class="secondary_catalog"><span>2.3 </span><a href="javascript:" data-id="14995449991659785">重亚硫酸盐处理过程中DNA的降解</a></li><li class="secondary_catalog"><span>2.4 </span><a href="javascript:" data-id="14995449991659785">其他关注点</a></li><li><span class="order">3</span><a href="javascript:" data-level="1" data-id="14995449991659786">应用：全基因组甲基化分析</a></li><li><span class="order">4</span><a href="javascript:" data-level="1" data-id="14995449991659787">氧化重亚硫酸盐测序</a></li><li><span class="order">5</span><a href="javascript:" data-level="1" data-id="references">参考文献</a></li></ul></div></div><div id="paragraphs"><div><div id="par_14995449974882578"><h2 class="title">1 方法<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>重亚硫酸盐测序将常规测序方法应用于重亚硫酸盐处理后的基因组DNA，以确定CpG二核苷酸的甲基化状态。其他非测序的检测DNA甲基化的策略也被用于在特定位点或全基因组水平上检测甲基化情况。所有的检测策略都假定重亚硫酸盐诱导的未甲基化胞嘧啶向尿嘧啶的转化已完成，这是所有后续技术的基础。理想情况下，使用的方法将分别确定每个等位基因的甲基化状态。除了重亚硫酸盐测序方法外，还有联合亚硫酸氢钠限制性内切酶分析法和甲基化DNA免疫沉淀等方法。 </p>
<p>分析重亚硫酸盐处理后的DNA甲基化的方法一直在不断发展。为了总结这些快速发展的方法，已经写了许多综述文章。<sup><a href="https://web.archive.org/web/20221025183652/https://baike.sogou.com/kexue/d10661.htm#quote_2" class="kx_ref">[2]</a></sup><sup><a href="https://web.archive.org/web/20221025183652/https://baike.sogou.com/kexue/d10661.htm#quote_3" class="kx_ref">[3]</a></sup><sup><a href="https://web.archive.org/web/20221025183652/https://baike.sogou.com/kexue/d10661.htm#quote_4" class="kx_ref">[4]</a></sup><sup><a href="https://web.archive.org/web/20221025183652/https://baike.sogou.com/kexue/d10661.htm#quote_5" class="kx_ref">[5]</a></sup> </p>
<p>这些方法通常可以分为基于甲基化特异性聚合酶链式反应 (MSP)的策略和基于非甲基化特异性PCR(聚合酶链式反应)的策略。以微阵列为基础的检测方法使用的是基于非甲基化特异性PCR。 </p> 
<h3>1.1 <span>基于非甲基化特异性PCR的方法</span></h3> 
<p></p><p></p><div class="text_img ed_imgfloat_right">
            <a class="ed_image_link lazyLoad" data-src="https://img03.sogoucdn.com/app/a/200698/sogou_science_13620" data-bigsrc="" title="点击查看大图" href="javascript:" data-observer="true"></a>
            <div class="text_img_title">图3:不基于甲基化特异性PCR的DNA甲基化分析方法。重亚硫酸盐转化后，用不区分甲基化和非甲基化序列的PCR扩增基因组DNA。然后利用现有的多种方法根据重亚硫酸盐转化导致的扩增子内的变化进行鉴别。</div>   
        </div> <p></p><p></p> 
<p><strong>直接测序</strong></p> 
<p>第一个报道的使用重亚硫酸盐检测DNA甲基化方法是利用PCR和标准双脱氧核苷酸测序直接测定对重亚硫酸盐转化有抗性的核苷酸。<sup><a href="https://web.archive.org/web/20221025183652/https://baike.sogou.com/kexue/d10661.htm#quote_6" class="kx_ref">[6]</a></sup> 引物被设计成链特异性和重亚硫酸盐特异性(即，引物含有非CpG胞嘧啶，因此它们与非重亚硫酸盐处理的DNA不互补)，位于需要检测的甲基化位点的侧翼(但不交叉)。因此，与甲基化特异性PCR相比，它将扩增甲基化和非甲基化序列。所有未甲基化胞嘧啶的位点在有义链的扩增序列中显示为胸腺嘧啶，在扩增的反义链中显示为腺嘌呤。通过将高通量测序衔接子结合到PCR引物中，PCR产物可以用大规模平行测序进行测序。或者，人工将PCR产物克隆之后进行测序。巢式PCR方法可用于提高扩增产物质量。 </p>
<p>所有后续的使用重亚硫酸盐处理过DNA的DNA甲基化分析技术都是基于Frommer等人的成果(图2)<sup><a href="https://web.archive.org/web/20221025183652/https://baike.sogou.com/kexue/d10661.htm#quote_7" class="kx_ref">[7]</a></sup>。尽管大多数其他模式不是真正的测序技术，但是术语“重亚硫酸盐测序”通常用于描述重亚硫酸盐转化的DNA甲基化分析技术。 </p> 
<p><strong>焦磷酸测序</strong></p> 
<p>焦磷酸测序也是非甲基化特异性PCR测序的一种类型，用于分析重亚硫酸盐处理的DNA。<sup><a href="https://web.archive.org/web/20221025183652/https://baike.sogou.com/kexue/d10661.htm#quote_8" class="kx_ref">[8]</a></sup><sup><a href="https://web.archive.org/web/20221025183652/https://baike.sogou.com/kexue/d10661.htm#quote_9" class="kx_ref">[9]</a></sup> 在PCR扩增目的片段后，焦磷酸测序用于测定该区域特定CpG位点的重亚硫酸盐转化序列。在单个位点的C-T比率可以根据序列延伸过程中C-T结合的量来定量确定。这种方法的主要限制条件是技术成本。然而，焦磷酸测序很好地扩展应用到高通量筛选方法中。 </p>
<p>Wong等人最近描述了对该技术的进一步改进，即使用等位基因特异性引物。该引物将单核苷酸多态性结合到测序引物的序列中，从而允许对母系和父系等位基因进行单独分析。<sup><a href="https://web.archive.org/web/20221025183652/https://baike.sogou.com/kexue/d10661.htm#quote_10" class="kx_ref">[10]</a></sup> 这项技术对基因组印记分析特别有用。 </p> 
<p><strong>甲基化敏感的单链构象分析(MS-SSCA)</strong></p> 
<p>该方法是基于单链构象多态性分析(SSCA)方法为分析单核苷酸多态性而开发的。<sup><a href="https://web.archive.org/web/20221025183652/https://baike.sogou.com/kexue/d10661.htm#quote_11" class="kx_ref">[11]</a></sup> SSCA基于非变性电泳中的差异迁移来区分大小相同但序列不同的单链DNA片段。在MS-SSCA中，这被用来区分重亚硫酸盐处理的、含有目的CpG位点的PCR扩增区域。尽管SSCA在只有单个核苷酸差异时缺乏敏感性，但重亚硫酸盐处理的DNA甲基化检测中，在大多数目的区域中往往有进行大量的C-T转化，因此，产生的敏感性接近100%。MS-SSCA还提供了基于条带强度比率的半定量分析。然而，该方法旨在评估目的区域内的所有CpG位点，而不是单个甲基化位点。 </p> 
<p><strong>高分辨率熔融分析</strong></p> 
<p>另一种区分转化的和未转化的重亚硫酸盐处理的DNA的方法是使用高分辨率熔解分析，这是一种基于定量PCR的技术，最初设计用于区分单核苷酸多态性。<sup><a href="https://web.archive.org/web/20221025183652/https://baike.sogou.com/kexue/d10661.htm#quote_12" class="kx_ref">[12]</a></sup> PCR扩增子直接通过温度升高和熔化过程中嵌入荧光染料的释放进行分析。甲基化程度，如扩增子中的C-T含量所代表的，决定了染料的熔化速度和随后的释放。这种方法允许在单管分析中直接定量，是评估扩增区域整体的甲基化，而不是特定CpG位点的甲基化。 </p> 
<p><strong>甲基化敏感的单核苷酸引物延伸(MS-SnuPE)</strong></p> 
<p>MS-SnuPE采用最初设计用于分析单核苷酸多态性的引物延伸方法。<sup><a href="https://web.archive.org/web/20221025183652/https://baike.sogou.com/kexue/d10661.htm#quote_13" class="kx_ref">[13]</a></sup> DNA被重亚硫酸盐转化，重亚硫酸盐特异性引物被退火到目标CpG之前的碱基对序列上。允许引物使用终止双脱氧核苷酸的DNA聚合酶将一个碱基对延伸到C(或T)中，并定量测定C与T的比率。 </p>
<p>有许多方法可以用来确定这一C-T比率。起初，MS-SnuP依赖放射性ddNTPs作为引物延伸的标记。后来，荧光标记和焦磷酸测序也被使用。<sup><a href="https://web.archive.org/web/20221025183652/https://baike.sogou.com/kexue/d10661.htm#quote_14" class="kx_ref">[14]</a></sup> 此外，基质辅助激光解吸电离/飞行时间(MALDI-TOF)质谱分析可用于区分两种多态性引物延伸产物，本质上是基于为单核苷酸多态性基因分型设计的良好分析。反相离子对高效液相色谱也已经被用于区分引物延伸产物。<sup><a href="https://web.archive.org/web/20221025183652/https://baike.sogou.com/kexue/d10661.htm#quote_15" class="kx_ref">[15]</a></sup> </p> 
<p><strong>碱基特异性裂解/MALDI-TOF</strong></p> 
<p>Ehrich等人最近描述的方法通过添加碱基特异性切割步骤进一步利用重亚硫酸盐转化来增强从核苷酸变化中获得的信息。<sup><a href="https://web.archive.org/web/20221025183652/https://baike.sogou.com/kexue/d10661.htm#quote_16" class="kx_ref">[16]</a></sup> 首先，通过将目的区域体外转录成核糖核酸(通过在初始扩增中向PCR引物添加核糖核酸聚合酶启动子位点)，核糖核酸酶A可用于切割核糖核酸转录酶碱基特异性位点。由于核糖核酸酶A在胞嘧啶和尿嘧啶核糖核苷酸处特异性切割核糖核酸，当需要胞嘧啶特异性(C特异性)切割时，通过加入抗切割的dTTP，当需要尿嘧啶特异性(U特异性)切割时，加入dCTP，从而实现碱基特异性。然后裂解的片段可以通过MALDI-TOF进行分析。重亚硫酸盐处理导致通过C-U转化引入/去除切割位点，或者在扩增的反向链中通过G-A转化转移片段质量。C特异性切割将在所有甲基化CpG位点特异性切割。通过分析所得片段的大小，可以确定区域内CpG位点的特定甲基化模式，而不是确定整个区域的甲基化程度。这种方法证明了高通量筛选的有效性，允许以低成本的方式对多个组织中的多个CpG位点进行检测。 </p> 
<h3>1.2 <span>甲基化特异性PCR</span></h3> 
<p></p><p></p><div class="text_img ed_imgfloat_right">
            <a class="ed_image_link lazyLoad" data-src="https://img04.sogoucdn.com/app/a/200698/sogou_science_13621" data-bigsrc="" title="点击查看大图" href="javascript:" data-observer="true"></a>
            <div class="text_img_title">图4:甲基化特异性PCR是一种灵敏的方法，可在重亚硫酸盐转化的基因组DNA上使用甲基化特异性引物鉴别扩增以及检测目的甲基化区域。这种引物将只与甲基化的序列退火，因此仅包含有对重亚硫酸盐转化有抗性的5-甲基胞嘧啶。在另一种方式中，可以使用非甲基化特异性引物。</div>   
        </div> <p></p><p></p> 
<p>这种甲基化分析的替代方法也使用重亚硫酸盐处理的DNA，但是避免了对目的区域测序的需要。<sup><a href="https://web.archive.org/web/20221025183652/https://baike.sogou.com/kexue/d10661.htm#quote_17" class="kx_ref">[17]</a></sup> 相反，引物对本身被设计为“甲基化特异性的”， 仅包括互补未转化的5-甲基胞嘧啶的序列，或者相反，设计为“非甲基化特异性的”，互补由非甲基化胞嘧啶转化而来的胸腺嘧啶。甲基化是由特定引物实现扩增的能力决定的。由于引物中CpG对数量的增加提高了检测的特异性，因此这种方法对于检测可能具有高甲基化密度的CpG岛特别有用。将CpG对置于引物的3’端能增强灵敏度。MSP的初始报告描述了足够灵敏度可以检测0.1%等位基因甲基化。一般来说，当检测特定位点的甲基化状态时，MSP及其相关方案被认为是最敏感的。 </p>
<p>MethyLight方法基于MSP，但使用定量PCR提供定量分析。<sup><a href="https://web.archive.org/web/20221025183652/https://baike.sogou.com/kexue/d10661.htm#quote_18" class="kx_ref">[18]</a></sup> 使用甲基化特异性引物，并使用甲基化特异性荧光报告探针退火到扩增区域。在另一种方式中，如果需要区分相关序列中的CpG对，引物或探针可以设计成没有甲基化特异性。定量是参照甲基化的参考DNA进行的。为了提高PCR对成功重亚硫酸盐转化的DNA的特异性，对该方案的一项修改是使用重亚硫酸盐未转化的DNA的额外探针来定量这种非特异性扩增。<sup><a href="https://web.archive.org/web/20221025183652/https://baike.sogou.com/kexue/d10661.htm#quote_19" class="kx_ref">[19]</a></sup> </p>
<p>对MSP扩增DNA的进一步方法是使用熔解曲线分析(Mc-MSP)来分析产物。<sup><a href="https://web.archive.org/web/20221025183652/https://baike.sogou.com/kexue/d10661.htm#quote_20" class="kx_ref">[20]</a></sup> 该方法用甲基化特异性引物和非甲基化特异性引物扩增重亚硫酸盐转化的DNA，并通过比较熔解曲线分析中产生的差异峰来确定两种产物的定量比。这是一种同时使用定量PCR和熔解分析的高分辨率熔解分析方法，特别是用于低水平甲基化的灵敏检测。<sup><a href="https://web.archive.org/web/20221025183652/https://baike.sogou.com/kexue/d10661.htm#quote_21" class="kx_ref">[21]</a></sup> </p> 
<h3>1.3 <span>基于微阵列的方法</span></h3> 
<p>基于微阵列的方法是分析重亚硫酸盐处理DNA的技术的逻辑延伸，可用于整个基因组的甲基化水平的检测。<sup><a href="https://web.archive.org/web/20221025183652/https://baike.sogou.com/kexue/d10661.htm#quote_22" class="kx_ref">[22]</a></sup> 寡核苷酸微阵列是利用靶向目标CpG位点的寡核苷酸杂交探针对设计的。一个与未改变的甲基化序列互补，另一个与C到T转化的未甲基化序列互补。探针也是重亚硫酸盐特异性的，以防止与重亚硫酸盐未完全转化的DNA结合。Illumina公司甲基化分析是一种在微阵列水平上应用重亚硫酸盐测序技术来产生全基因组甲基化数据的分析。 </p></div></div><div id="par_14995449991659785"><h2 class="title">2 局限性<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><h3>2.1 <span>5-羟甲基胞嘧啶</span></h3> 
<p>重亚硫酸盐测序在哺乳动物基因组中被广泛使用，但是随着一种新的哺乳动物DNA修饰——5-羟甲基胞嘧啶的发现，情况变得复杂化。<sup><a href="https://web.archive.org/web/20221025183652/https://baike.sogou.com/kexue/d10661.htm#quote_23" class="kx_ref">[23]</a></sup><sup><a href="https://web.archive.org/web/20221025183652/https://baike.sogou.com/kexue/d10661.htm#quote_24" class="kx_ref">[24]</a></sup> 5-羟甲基胞嘧啶在重亚硫酸盐处理后转化为胞嘧啶-5-甲基磺酸盐，测序后被读成胞嘧啶碱基。<sup><a href="https://web.archive.org/web/20221025183652/https://baike.sogou.com/kexue/d10661.htm#quote_25" class="kx_ref">[25]</a></sup> 因此，重亚硫酸盐测序不能区分5-甲基胞嘧啶和5-羟甲基胞嘧啶。这意味着重亚硫酸盐测序的结果不能再定义为单一的DNA甲基化，而是5-甲基胞嘧啶和5-羟甲基胞嘧啶的复合物。芝加哥大学何川教授(Chuan He)开发的Tet辅助重亚硫酸盐测序现在能够在单碱基分辨率下区分这两种修饰。<sup><a href="https://web.archive.org/web/20221025183652/https://baike.sogou.com/kexue/d10661.htm#quote_26" class="kx_ref">[26]</a></sup> </p> 
<h3>2.2 <span>不完全转化</span></h3> 
<p>重亚硫酸盐测序依赖于每个未甲基化胞嘧啶残基转化为尿嘧啶。如果转化不完全，后续分析将错误地将未成功转化的未甲基化胞嘧啶解释为甲基化胞嘧啶，导致甲基化的假阳性结果。只有单链DNA中的胞嘧啶容易受到重亚硫酸盐的攻击，因此DNA的变性是至关重要的。<sup><a href="https://web.archive.org/web/20221025183652/https://baike.sogou.com/kexue/d10661.htm#quote_2" class="kx_ref">[2]</a></sup> 转化中确保适合的反应参数，如温度和盐浓度来维持DNA的单链构象，尽量使非甲基化胞嘧啶完全转化。据报道，将DNA包埋在琼脂糖凝胶中，通过保持DNA链物理分离的状态，可以提高转化率。<sup><a href="https://web.archive.org/web/20221025183652/https://baike.sogou.com/kexue/d10661.htm#quote_27" class="kx_ref">[27]</a></sup> </p> 
<h3>2.3 <span>重亚硫酸盐处理过程中DNA的降解</span></h3> 
<p>重亚硫酸盐测序的一个主要挑战是与转化同时发生的DNA降解。完全转化所需的条件，如长的培养时间、高温和高重亚硫酸盐浓度，可导致约90%的孵育的DNA降解。<sup><a href="https://web.archive.org/web/20221025183652/https://baike.sogou.com/kexue/d10661.htm#quote_28" class="kx_ref">[28]</a></sup> 鉴于DNA的起始量往往有限，这种大量的降解可能是有问题的。降解伴随脱嘌呤作用发生，导致DNA链发生随机断裂。<sup><a href="https://web.archive.org/web/20221025183652/https://baike.sogou.com/kexue/d10661.htm#quote_29" class="kx_ref">[29]</a></sup> 因此，期望得到的PCR扩增子越长，完整模板分子的数量可能越有限。这可能导致PCR扩增失败，或者由于模板分子取样有限而丢失甲基化水平定量的准确信息。因此，评估由所用反应条件导致的DNA降解量，并考虑这将如何影响所需的扩增子是很重要的。也可以从技术层面来减少DNA的降解，例如调整循环中培养温度。<sup><a href="https://web.archive.org/web/20221025183652/https://baike.sogou.com/kexue/d10661.htm#quote_29" class="kx_ref">[29]</a></sup> </p> 
<h3>2.4 <span>其他关注点</span></h3> 
<p>重亚硫酸盐处理后一个潜在的重要问题是由于溶液碱化不充分导致嘧啶残基的不完全脱磺酸基。这可能会抑制某些DNA聚合酶，使后续的PCR很难扩增。然而，这种情况可以通过监测溶液的酸碱度来避免，以确保脱磺酸基作用充分完成。<sup><a href="https://web.archive.org/web/20221025183652/https://baike.sogou.com/kexue/d10661.htm#quote_2" class="kx_ref">[2]</a></sup> </p>
<p>最后一个要关注的方面是重亚硫酸盐处理大大降低了样品的复杂程度，如果要进行多重PCR，这可能是个问题(2006年)。<sup><a href="https://web.archive.org/web/20221025183652/https://baike.sogou.com/kexue/d10661.htm#quote_5" class="kx_ref">[5]</a></sup> 引物设计更加困难，不恰当的杂交发生更加频繁。 </p></div></div><div id="par_14995449991659786"><h2 class="title">3 应用：全基因组甲基化分析<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>重亚硫酸盐测序的发展使全基因组范围内应用这一技术成为可能，然而在此之前，对DNA甲基化进行全局测量只能使用其他技术，例如限制性标记基因组扫描。人类表观基因组的绘制被许多科学家视为人类基因组计划完成的延伸。<sup><a href="https://web.archive.org/web/20221025183652/https://baike.sogou.com/kexue/d10661.htm#quote_30" class="kx_ref">[30]</a></sup><sup><a href="https://web.archive.org/web/20221025183652/https://baike.sogou.com/kexue/d10661.htm#quote_31" class="kx_ref">[31]</a></sup> 这些表观遗传信息对于理解基因序列的功能是如何实现和调控非常重要。由于表观基因组不如基因组稳定，它被认为在基因-环境相互作用中很重要。<sup><a href="https://web.archive.org/web/20221025183652/https://baike.sogou.com/kexue/d10661.htm#quote_32" class="kx_ref">[32]</a></sup> </p>
<p>然而，表观基因组图谱绘制本质上比基因组测序更复杂，因为表观基因组比基因组更多变。一个人的表观基因组随年龄的增长而变化，不同组织之间也会有所不同，还会受环境因素的影响，并在疾病中表现出异常。然而，这种丰富的表观遗传图谱代表了不同的年龄、组织类型和疾病状态，将为表观遗传标记的正常功能以及导致衰老和疾病的机制方面产生有价值的信息。 </p>
<p>表观基因组图谱的直接好处包括对克隆技术可能带来的进步。人们认为，未能生产出具有正常生存能力和寿命的克隆动物是表观遗传标记模式不当造成的。此外，异常甲基化模式在许多癌症中都有明确的特征。整体低甲基化导致基因组稳定性降低，而肿瘤抑制基因启动子的局部高甲基化通常导致其功能丧失。甲基化的特定模式指示特定的癌症类型，具有预后价值，并有助于指导最佳的治疗方案。<sup><a href="https://web.archive.org/web/20221025183652/https://baike.sogou.com/kexue/d10661.htm#quote_31" class="kx_ref">[31]</a></sup> </p>
<p>世界各地正在进行大规模表观基因组绘图工作，并已在人类表观基因组计划下组织起来。<sup><a href="https://web.archive.org/web/20221025183652/https://baike.sogou.com/kexue/d10661.htm#quote_32" class="kx_ref">[32]</a></sup> 基于多层策略，即重亚硫酸盐测序被用来为有限数量的参考表观基因组获得高分辨率甲基化谱，而对更大范围的样品进行的分析则不太彻底。这种方法旨在最大限度从给定资源中获得的表观信息，因为完成全基因组高分辨率绘制仍然是一项昂贵的任务。 </p>
<p>当应用于高通量甲基化数据(例如全基因组重亚硫酸盐测序)时，基因集分析(例如使用像DAVID和GoSeq这样的工具)已经显示出严重的偏差；有人建议，这可以通过使用样品标记排列或应用统计模型来矫正针对每个基因的CpG探针/ CpG位点数量的差异。<sup><a href="https://web.archive.org/web/20221025183652/https://baike.sogou.com/kexue/d10661.htm#quote_33" class="kx_ref">[33]</a></sup> </p></div></div><div id="par_14995449991659787"><h2 class="title">4 氧化重亚硫酸盐测序<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>5-甲基胞嘧啶和5-羟甲基胞嘧啶在重亚硫酸盐测序中都被读作胞嘧啶。<sup><a href="https://web.archive.org/web/20221025183652/https://baike.sogou.com/kexue/d10661.htm#quote_25" class="kx_ref">[25]</a></sup> 氧化重亚硫酸盐测序是一种以单碱基分辨率区分5-甲基胞嘧啶和5-羟甲基胞嘧啶的方法。该方法采用一种特异性化学氧化反应将5-羟甲基胞嘧啶转成5-甲酰胞嘧啶，然后在重亚硫酸盐处理过程中转化成尿嘧啶。<sup><a href="https://web.archive.org/web/20221025183652/https://baike.sogou.com/kexue/d10661.htm#quote_34" class="kx_ref">[34]</a></sup> 这样，5-甲基胞嘧啶和5-羟甲基胞嘧啶中唯一一个读作胞嘧啶的碱基是5-甲基胞嘧啶，这种方法给出了DNA样品中真正甲基化状态的图谱。5-羟甲基胞嘧啶的水平也可以通过测量重亚硫酸盐和氧化重亚硫酸盐测序之间的差异来定量。 </p></div></div></div></div><div id="references"><h2 class="title" id="par_references">参考文献</h2><ul class="references"><li id="quote_1"><span class="references-num">[1]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Chatterjee A, Stockwell PA, Rodger EJ and Morison IM 2012. Comparison of alignment software for genome-wide bisulphite sequence data. Nucleic Acids Research 40(10): e79..</span></p></li><li id="quote_2"><span class="references-num">[2]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Fraga MF, Esteller M (September 2002). "DNA methylation: a profile of methods and applications". BioTechniques. 33 (3): 632, 634, 636–49. doi:10.2144/02333rv01. PMID 12238773..</span></p></li><li id="quote_3"><span class="references-num">[3]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">El-Maarri O (2003). "Methods: DNA methylation". Adv. Exp. Med. Biol. Advances in Experimental Medicine and Biology. 544: 197–204. doi:10.1007/978-1-4419-9072-3_23. ISBN 978-0-306-48174-1. PMID 14713229..</span></p></li><li id="quote_4"><span class="references-num">[4]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Laird PW (April 2003). "The power and the promise of DNA methylation markers". Nat. Rev. Cancer. 3 (4): 253–66. doi:10.1038/nrc1045. PMID 12671664..</span></p></li><li id="quote_5"><span class="references-num">[5]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Callinan PA, Feinberg AP (2006). "The emerging science of epigenomics". Hum Mol Genet. 15 (90001): R95–101. doi:10.1093/hmg/ddl095. PMID 16651376..</span></p></li><li id="quote_6"><span class="references-num">[6]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Frommer M, McDonald LE, Millar DS, et al. (March 1992). "A genomic sequencing protocol that yields a positive display of 5-methylcytosine residues in individual DNA strands". Proc. Natl. Acad. Sci. U.S.A. 89 (5): 1827–31. doi:10.1073/pnas.89.5.1827. PMC 48546. PMID 1542678..</span></p></li><li id="quote_7"><span class="references-num">[7]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Frommer M, McDonald LE, Millar DS, et al. (March 1992). "A genomic sequencing protocol that yields a positive display of 5-methylcytosine residues in individual DNA strands". Proc. Natl. Acad. Sci. U.S.A. 89 (5): 1827–31. doi:10.1073/pnas.89.5.1827. 位于PMC的全文本内容：48546. PubMed..</span></p></li><li id="quote_8"><span class="references-num">[8]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Colella S, Shen L, Baggerly KA, Issa JP, Krahe R (July 2003). "Sensitive and quantitative universal Pyrosequencing methylation analysis of CpG sites". BioTechniques. 35 (1): 146–50. doi:10.2144/03351md01. PMID 12866414..</span></p></li><li id="quote_9"><span class="references-num">[9]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Tost J, Dunker J, Gut IG (July 2003). "Analysis and quantification of multiple methylation variable positions in CpG islands by Pyrosequencing". BioTechniques. 35 (1): 152–6. doi:10.2144/03351md02. PMID 12866415..</span></p></li><li id="quote_10"><span class="references-num">[10]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Wong HL, Byun HM, Kwan JM, et al. (December 2006). "Rapid and quantitative method of allele-specific DNA methylation analysis". BioTechniques. 41 (6): 734–9. doi:10.2144/000112305. PMID 17191619..</span></p></li><li id="quote_11"><span class="references-num">[11]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Bianco T, Hussey D, Dobrovic A (1999). "Methylation-sensitive, single-strand conformation analysis (MS-SSCA): A rapid method to screen for and analyze methylation". Hum. Mutat. 14 (4): 289–93. doi:10.1002/(SICI)1098-1004(199910)14:4&lt;289::AID-HUMU3&gt;3.0.CO;2-A. PMID 10502775..</span></p></li><li id="quote_12"><span class="references-num">[12]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Wojdacz TK, Dobrovic A (2007). "Methylation-sensitive high resolution melting (MS-HRM): a new approach for sensitive and high-throughput assessment of methylation". Nucleic Acids Res. 35 (6): e41. doi:10.1093/nar/gkm013. PMC 1874596. PMID 17289753..</span></p></li><li id="quote_13"><span class="references-num">[13]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Gonzalgo ML, Jones PA (June 1997). "Rapid quantitation of methylation differences at specific sites using methylation-sensitive single nucleotide primer extension (Ms-SNuPE)". Nucleic Acids Res. 25 (12): 2529–31. doi:10.1093/nar/25.12.2529. PMC 146734. PMID 9171109..</span></p></li><li id="quote_14"><span class="references-num">[14]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Uhlmann K, Brinckmann A, Toliat MR, Ritter H, Nürnberg P (December 2002). "Evaluation of a potential epigenetic biomarker by quantitative methyl-single nucleotide polymorphism analysis". Electrophoresis. 23 (24): 4072–9. doi:10.1002/elps.200290023. PMID 12481262..</span></p></li><li id="quote_15"><span class="references-num">[15]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Matin MM, Baumer A, Hornby DP (October 2002). "An analytical method for the detection of methylation differences at specific chromosomal loci using primer extension and ion pair reverse phase HPLC". Hum. Mutat. 20 (4): 305–11. doi:10.1002/humu.10118. PMID 12325026..</span></p></li><li id="quote_16"><span class="references-num">[16]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Ehrich M, Nelson MR, Stanssens P, et al. (November 2005). "Quantitative high-throughput analysis of DNA methylation patterns by base-specific cleavage and mass spectrometry". Proc. Natl. Acad. Sci. U.S.A. 102 (44): 15785–90. doi:10.1073/pnas.0507816102. PMC 1276092. PMID 16243968..</span></p></li><li id="quote_17"><span class="references-num">[17]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Herman JG, Graff JR, Myöhänen S, Nelkin BD, Baylin SB (September 1996). "Methylation-specific PCR: a novel PCR assay for methylation status of CpG islands". Proc. Natl. Acad. Sci. U.S.A. 93 (18): 9821–6. doi:10.1073/pnas.93.18.9821. PMC 38513. PMID 8790415..</span></p></li><li id="quote_18"><span class="references-num">[18]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Eads CA, Danenberg KD, Kawakami K, et al. (April 2000). "MethyLight: a high-throughput assay to measure DNA methylation". Nucleic Acids Res. 28 (8): 32e–0. doi:10.1093/nar/28.8.e32. PMC 102836. PMID 10734209..</span></p></li><li id="quote_19"><span class="references-num">[19]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Rand K, Qu W, Ho T, Clark SJ, Molloy P (June 2002). "Conversion-specific detection of DNA methylation using real-time polymerase chain reaction (ConLight-MSP) to avoid false positives". Methods. 27 (2): 114–20. doi:10.1016/S1046-2023(02)00062-2. PMID 12095268..</span></p></li><li id="quote_20"><span class="references-num">[20]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Akey DT, Akey JM, Zhang K, Jin L (October 2002). "Assaying DNA methylation based on high-throughput melting curve approaches". Genomics. 80 (4): 376–84. doi:10.1006/geno.2002.6851. PMID 12376091..</span></p></li><li id="quote_21"><span class="references-num">[21]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Kristensen LS, Mikeska T, Krypuy M, Dobrovic A (April 2008). "Sensitive Melting Analysis after Real Time- Methylation Specific PCR (SMART-MSP): high-throughput and probe-free quantitative DNA methylation detection". Nucleic Acids Res. 36 (7): e42. doi:10.1093/nar/gkn113. PMC 2367707. PMID 18344521..</span></p></li><li id="quote_22"><span class="references-num">[22]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Adorján P, Distler J, Lipscher E, et al. (March 2002). "Tumour class prediction and discovery by microarray-based DNA methylation analysis". Nucleic Acids Res. 30 (5): 21e–21. doi:10.1093/nar/30.5.e21. PMC 101257. PMID 11861926..</span></p></li><li id="quote_23"><span class="references-num">[23]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Tahiliani M, Koh KP, Shen Y, Pastor WA, Bandukwala H, BrudnoY, et al. Conversion of 5-methylcytosine to 5-hydroxymethylcytosine inmammalian DNA by MLL partner TET1. Science. 2009;324(5929):930-5..</span></p></li><li id="quote_24"><span class="references-num">[24]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Kriaucionis S, Heintz N. The nuclear DNA base 5-hydroxymethylcytosine is present in Purkinje neurons and the brain. Science.2009;324(5929):929-30..</span></p></li><li id="quote_25"><span class="references-num">[25]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Huang Y, Pastor WA, Shen Y, Tahiliani M, Liu DR, Rao A. The Behaviour of 5-Hydroxymethylcytosine in Bisulfite Sequencing. PLOS ONE.2010;5(1):e8888..</span></p></li><li id="quote_26"><span class="references-num">[26]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Yu, M., Hon, G. C., Szulwach, K. E., Song, C., Jin, P., Ren, B., He, C. Tet-assisted bisulfite sequencing of 5-hydroxymethylcytosine. Nat. Protocols 2012, 7, 2159..</span></p></li><li id="quote_27"><span class="references-num">[27]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Olek A, Oswald J, Walter J (December 1996). "A modified and improved method for bisulphite based cytosine methylation analysis". Nucleic Acids Res. 24 (24): 5064–6. doi:10.1093/nar/24.24.5064. PMC 146326. PMID 9016686. Archived from the original on 2012-07-14..</span></p></li><li id="quote_28"><span class="references-num">[28]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Grunau C, Clark SJ, Rosenthal A (July 2001). "Bisulfite genomic sequencing: systematic investigation of critical experimental parameters". Nucleic Acids Res. 29 (13): E65–5. doi:10.1093/nar/29.13.e65. PMC 55789. PMID 11433041..</span></p></li><li id="quote_29"><span class="references-num">[29]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Ehrich M, Zoll S, Sur S, van den Boom D (2007). "A new method for accurate assessment of DNA quality after bisulfite treatment". Nucleic Acids Res. 35 (5): e29. doi:10.1093/nar/gkl1134. PMC 1865059. PMID 17259213..</span></p></li><li id="quote_30"><span class="references-num">[30]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Esteller M (June 2006). "The necessity of a human epigenome project". Carcinogenesis. 27 (6): 1121–5. doi:10.1093/carcin/bgl033. PMID 16699174..</span></p></li><li id="quote_31"><span class="references-num">[31]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Bradbury J (December 2003). "Human Epigenome Project—Up and Running". PLoS Biol. 1 (3): E82. doi:10.1371/journal.pbio.0000082. PMC 300691. PMID 14691553..</span></p></li><li id="quote_32"><span class="references-num">[32]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Jones PA, Martienssen R (December 2005). "A blueprint for a Human Epigenome Project: the AACR Human Epigenome Workshop". Cancer Res. 65 (24): 11241–6. doi:10.1158/0008-5472.CAN-05-3865. PMID 16357125..</span></p></li><li id="quote_33"><span class="references-num">[33]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Geeleher P, Hartnett L, Egan LJ, Golden A, Raja Ali RA, Seoighe C (June 2013). "Gene-Set Analysis is Severely Biased When Applied to Genome-wide Methylation Data". Bioinformatics. 29 (15): 1851–7. doi:10.1093/bioinformatics/btt311. PMID 23732277..</span></p></li><li id="quote_34"><span class="references-num">[34]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Booth MJ, Branco MR, Ficz G, Oxley D, Krueger F, Reik W, et al. quantitative sequencing of 5-methylcytosine and 5-hydroxymethylcytosine at single-base resolution. Science. 2012;336(6083):934-7..</span></p></li></ul></div><div class="read-num">阅读 <!-- -->94</div></div><div class="right-side" id="rightSide"><div class="side" id="lemma-side"><div class="side-title">版本记录</div><ul class="side-lst"><li><p class="side-lst-txt">暂无</p></li></ul><div class="user-card userCard"></div></div><div class="side"><div class="side-event"></div></div></div></div><div class="footer-box"><div id="footer"><div class="footer-logo-wrap"><div class="footer-logo"></div><div class="footer-logo-text">知识·传播·科普</div></div><div class="footer-info">本网站内容采用<a target="_blank" href="https://web.archive.org/web/20221025183652/https://creativecommons.org/licenses/by-sa/3.0/deed.zh?tdsourcetag=s_pctim_aiomsg">CC-BY-SA 3.0</a>授权</div><div class="footer-btn-wrap"><a target="_blank" href="https://web.archive.org/web/20221025183652/https://baike.sogou.com/help/#user_protocol">用户协议</a><a target="_blank" href="https://web.archive.org/web/20221025183652/http://www.sogou.com/docs/terms.htm?v=1">免责声明</a><a target="_blank" href="https://web.archive.org/web/20221025183652/http://corp.sogou.com/private.html">隐私政策</a><a target="_blank" href="https://web.archive.org/web/20221025183652/https://baike.sogou.com/kexue/intro.htm">关于我们</a></div></div></div><script>window.lemmaInfo ={"lemmaId":"10661","versionId":"14995449974882571","title":"重亚硫酸盐测序","subtitle":"","abstracts":{"paragraphId":"14995449974882577","title":"简介","versionId":"14995449974882572","lemmaId":10661,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":10145103,"name":"科学百科编辑组","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233984,"comment":null,"dependVersionId":0,"contentType":2,"content":"<p></p> \n<p></p> \n<p>重亚硫酸盐测序(也称为亚硫酸氢盐测序)<sup><a href=\"#quote_1\" class=\"kx_ref\">[1]</a></sup> 是在常规测序前使用重亚硫酸盐处理来测定甲基化情形的方法。脱氧核糖核酸 (DNA)甲基化是第一个被发现的表观遗传标记，并且到现在为止仍然是研究最多的表观遗传改变。在动物中，DNA甲基化主要是指在二核苷酸CpG的胞嘧啶残基的5号碳位添加一个甲基，它可以对基因的转录活性起抑制作用。 </p>\n<p>用重亚硫酸盐处理DNA可以将胞嘧啶残基转化为尿嘧啶，但5-甲基胞嘧啶残基不受影响。因此，用重亚硫酸盐处理过的DNA只保留甲基化胞嘧啶。基于此原理，重亚硫酸盐处理引入了依赖于单个胞嘧啶残基甲基化状态的特定的DNA序列变化，产生了关于DNA片段甲基化状态的单核苷酸分辨率信息。可以对改变后的序列进行各种分析来检索这些信息。因此，该分析的目的被简化为区分重亚硫酸盐转化产生的单核苷酸多态性(胞嘧啶和<b>胸腺嘧啶</b>)(图1)。 </p>","pics":[{"originalUrl":"https://web.archive.org/web/20221025183652/https://img03.sogoucdn.com/app/a/200698/sogou_science_13618?w=300&h=209&titlename=%E5%9B%BE1:%20%E9%87%8D%E4%BA%9A%E7%A1%AB%E9%85%B8%E7%9B%90%E8%BD%AC%E5%8C%96%E5%9F%BA%E5%9B%A0%E7%BB%84DNA%E5%BA%8F%E5%88%97%E6%A1%88%E4%BE%8B%E6%A6%82%E8%BF%B0%E3%80%82%E8%93%9D%E8%89%B2%E6%A0%B8%E8%8B%B7%E9%85%B8%E6%98%AF%E9%87%8D%E4%BA%9A%E7%A1%AB%E9%85%B8%E7%9B%90%E8%BD%AC%E5%8C%96%E4%B8%BA%E5%B0%BF%E5%98%A7%E5%95%B6%E7%9A%84%E6%9C%AA%E7%94%B2%E5%9F%BA%E5%8C%96%E8%83%9E%E5%98%A7%E5%95%B6%EF%BC%8C%E8%80%8C%E7%BA%A2%E8%89%B2%E6%A0%B8%E8%8B%B7%E9%85%B8%E6%98%AF%E5%AF%B9%E8%BD%AC%E5%8C%96%E6%9C%89%E6%8A%97%E6%80%A7%E7%9A%845-%E7%94%B2%E5%9F%BA%E8%83%9E%E5%98%A7%E5%95%B6%E3%80%82","url":"https://web.archive.org/web/20221025183652/https://img03.sogoucdn.com/app/a/200698/sogou_science_13618","rw":300,"rh":209,"title":"图1: 重亚硫酸盐转化基因组DNA序列案例概述。蓝色核苷酸是重亚硫酸盐转化为尿嘧啶的未甲基化胞嘧啶，而红色核苷酸是对转化有抗性的5-甲基胞嘧啶。","alt":null,"width":0,"height":0}],"card":null,"references":[],"versionCount":0},"card":{"paragraphId":"0","title":null,"versionId":"0","lemmaId":0,"createType":0,"creator":null,"createTime":0,"versionEditor":null,"editTime":0,"comment":null,"dependVersionId":0,"contentType":0,"content":null,"pics":null,"card":null,"references":null,"versionCount":0},"categories":[{"id":4,"name":"生物学","parents":[]}],"creator":{"uid":10145103,"name":"柚子otto","pic":"https://web.archive.org/web/20221025183652/https://img02.sogoucdn.com/app/a/200698/1152_1152_1864088_20200427232849-1518843971.png","introduction":"","educations":[{"schoolName":"中国地质大学（北京）","major":"","degree":"本科","universityId":22,"universityLogo":"https://web.archive.org/web/20221025183652/https://img01.sogoucdn.com/app/a/200943/d3465c1c-6011-11e9-b353-fc4dd4f70029","majorLevel1":"理学","majorLevel2":"地理学","majorLevel3":"地图学与地理信息系统","majorLevel1Id":1,"majorLevel2Id":103,"majorLevel3Id":109,"state":"毕业","lab":"","researchField":""}],"jobs":[{"company":"搜狗","title":"产品经理"}],"works":null,"educationBrief":"中国地质大学（北京）","jobBrief":"产品经理","role":0,"roleName":null,"title":"中国地质大学（北京） · 地理学本科","professionalTitle":null,"phoneNo":null,"editable":true,"partnerId":139,"partnerIdCreateTime":1595844881,"partnerIdPoped":true},"createTime":1568626774,"editor":{"uid":10145103,"name":"柚子otto","pic":"https://web.archive.org/web/20221025183652/https://img02.sogoucdn.com/app/a/200698/1152_1152_1864088_20200427232849-1518843971.png","introduction":"","educations":[{"schoolName":"中国地质大学（北京）","major":"","degree":"本科","universityId":22,"universityLogo":"https://web.archive.org/web/20221025183652/https://img01.sogoucdn.com/app/a/200943/d3465c1c-6011-11e9-b353-fc4dd4f70029","majorLevel1":"理学","majorLevel2":"地理学","majorLevel3":"地图学与地理信息系统","majorLevel1Id":1,"majorLevel2Id":103,"majorLevel3Id":109,"state":"毕业","lab":"","researchField":""}],"jobs":[{"company":"搜狗","title":"产品经理"}],"works":null,"educationBrief":"中国地质大学（北京）","jobBrief":"产品经理","role":0,"roleName":null,"title":"中国地质大学（北京） · 地理学本科","professionalTitle":null,"phoneNo":null,"editable":true,"partnerId":139,"partnerIdCreateTime":1595844881,"partnerIdPoped":true},"editTime":1576233984,"state":1,"versionCount":1,"upNum":0,"downNum":0,"pics":[{"originalUrl":"https://web.archive.org/web/20221025183652/https://img03.sogoucdn.com/app/a/200698/sogou_science_13618?w=300&h=209&titlename=%E5%9B%BE1:%20%E9%87%8D%E4%BA%9A%E7%A1%AB%E9%85%B8%E7%9B%90%E8%BD%AC%E5%8C%96%E5%9F%BA%E5%9B%A0%E7%BB%84DNA%E5%BA%8F%E5%88%97%E6%A1%88%E4%BE%8B%E6%A6%82%E8%BF%B0%E3%80%82%E8%93%9D%E8%89%B2%E6%A0%B8%E8%8B%B7%E9%85%B8%E6%98%AF%E9%87%8D%E4%BA%9A%E7%A1%AB%E9%85%B8%E7%9B%90%E8%BD%AC%E5%8C%96%E4%B8%BA%E5%B0%BF%E5%98%A7%E5%95%B6%E7%9A%84%E6%9C%AA%E7%94%B2%E5%9F%BA%E5%8C%96%E8%83%9E%E5%98%A7%E5%95%B6%EF%BC%8C%E8%80%8C%E7%BA%A2%E8%89%B2%E6%A0%B8%E8%8B%B7%E9%85%B8%E6%98%AF%E5%AF%B9%E8%BD%AC%E5%8C%96%E6%9C%89%E6%8A%97%E6%80%A7%E7%9A%845-%E7%94%B2%E5%9F%BA%E8%83%9E%E5%98%A7%E5%95%B6%E3%80%82","url":"https://web.archive.org/web/20221025183652/https://img03.sogoucdn.com/app/a/200698/sogou_science_13618","rw":300,"rh":209,"title":"图1: 重亚硫酸盐转化基因组DNA序列案例概述。蓝色核苷酸是重亚硫酸盐转化为尿嘧啶的未甲基化胞嘧啶，而红色核苷酸是对转化有抗性的5-甲基胞嘧啶。","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221025183652/https://img03.sogoucdn.com/app/a/200698/sogou_science_13620?w=450&h=213&titlename=%E5%9B%BE3:%E4%B8%8D%E5%9F%BA%E4%BA%8E%E7%94%B2%E5%9F%BA%E5%8C%96%E7%89%B9%E5%BC%82%E6%80%A7PCR%E7%9A%84DNA%E7%94%B2%E5%9F%BA%E5%8C%96%E5%88%86%E6%9E%90%E6%96%B9%E6%B3%95%E3%80%82%E9%87%8D%E4%BA%9A%E7%A1%AB%E9%85%B8%E7%9B%90%E8%BD%AC%E5%8C%96%E5%90%8E%EF%BC%8C%E7%94%A8%E4%B8%8D%E5%8C%BA%E5%88%86%E7%94%B2%E5%9F%BA%E5%8C%96%E5%92%8C%E9%9D%9E%E7%94%B2%E5%9F%BA%E5%8C%96%E5%BA%8F%E5%88%97%E7%9A%84PCR%E6%89%A9%E5%A2%9E%E5%9F%BA%E5%9B%A0%E7%BB%84DNA%E3%80%82%E7%84%B6%E5%90%8E%E5%88%A9%E7%94%A8%E7%8E%B0%E6%9C%89%E7%9A%84%E5%A4%9A%E7%A7%8D%E6%96%B9%E6%B3%95%E6%A0%B9%E6%8D%AE%E9%87%8D%E4%BA%9A%E7%A1%AB%E9%85%B8%E7%9B%90%E8%BD%AC%E5%8C%96%E5%AF%BC%E8%87%B4%E7%9A%84%E6%89%A9%E5%A2%9E%E5%AD%90%E5%86%85%E7%9A%84%E5%8F%98%E5%8C%96%E8%BF%9B%E8%A1%8C%E9%89%B4%E5%88%AB%E3%80%82","url":"https://web.archive.org/web/20221025183652/https://img03.sogoucdn.com/app/a/200698/sogou_science_13620","rw":450,"rh":213,"title":"图3:不基于甲基化特异性PCR的DNA甲基化分析方法。重亚硫酸盐转化后，用不区分甲基化和非甲基化序列的PCR扩增基因组DNA。然后利用现有的多种方法根据重亚硫酸盐转化导致的扩增子内的变化进行鉴别。","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221025183652/https://img04.sogoucdn.com/app/a/200698/sogou_science_13621?w=400&h=238&titlename=%E5%9B%BE4:%E7%94%B2%E5%9F%BA%E5%8C%96%E7%89%B9%E5%BC%82%E6%80%A7PCR%E6%98%AF%E4%B8%80%E7%A7%8D%E7%81%B5%E6%95%8F%E7%9A%84%E6%96%B9%E6%B3%95%EF%BC%8C%E5%8F%AF%E5%9C%A8%E9%87%8D%E4%BA%9A%E7%A1%AB%E9%85%B8%E7%9B%90%E8%BD%AC%E5%8C%96%E7%9A%84%E5%9F%BA%E5%9B%A0%E7%BB%84DNA%E4%B8%8A%E4%BD%BF%E7%94%A8%E7%94%B2%E5%9F%BA%E5%8C%96%E7%89%B9%E5%BC%82%E6%80%A7%E5%BC%95%E7%89%A9%E9%89%B4%E5%88%AB%E6%89%A9%E5%A2%9E%E4%BB%A5%E5%8F%8A%E6%A3%80%E6%B5%8B%E7%9B%AE%E7%9A%84%E7%94%B2%E5%9F%BA%E5%8C%96%E5%8C%BA%E5%9F%9F%E3%80%82%E8%BF%99%E7%A7%8D%E5%BC%95%E7%89%A9%E5%B0%86%E5%8F%AA%E4%B8%8E%E7%94%B2%E5%9F%BA%E5%8C%96%E7%9A%84%E5%BA%8F%E5%88%97%E9%80%80%E7%81%AB%EF%BC%8C%E5%9B%A0%E6%AD%A4%E4%BB%85%E5%8C%85%E5%90%AB%E6%9C%89%E5%AF%B9%E9%87%8D%E4%BA%9A%E7%A1%AB%E9%85%B8%E7%9B%90%E8%BD%AC%E5%8C%96%E6%9C%89%E6%8A%97%E6%80%A7%E7%9A%845-%E7%94%B2%E5%9F%BA%E8%83%9E%E5%98%A7%E5%95%B6%E3%80%82%E5%9C%A8%E5%8F%A6%E4%B8%80%E7%A7%8D%E6%96%B9%E5%BC%8F%E4%B8%AD%EF%BC%8C%E5%8F%AF%E4%BB%A5%E4%BD%BF%E7%94%A8%E9%9D%9E%E7%94%B2%E5%9F%BA%E5%8C%96%E7%89%B9%E5%BC%82%E6%80%A7%E5%BC%95%E7%89%A9%E3%80%82","url":"https://web.archive.org/web/20221025183652/https://img04.sogoucdn.com/app/a/200698/sogou_science_13621","rw":400,"rh":238,"title":"图4:甲基化特异性PCR是一种灵敏的方法，可在重亚硫酸盐转化的基因组DNA上使用甲基化特异性引物鉴别扩增以及检测目的甲基化区域。这种引物将只与甲基化的序列退火，因此仅包含有对重亚硫酸盐转化有抗性的5-甲基胞嘧啶。在另一种方式中，可以使用非甲基化特异性引物。","alt":null,"width":0,"height":0}],"catalogs":[{"level":1,"title":"方法","paragraphId":"14995449974882578","subCatalogs":[{"level":2,"title":"基于非甲基化特异性PCR的方法","paragraphId":"14995449974882578","subCatalogs":null},{"level":2,"title":"甲基化特异性PCR","paragraphId":"14995449974882578","subCatalogs":null},{"level":2,"title":"基于微阵列的方法","paragraphId":"14995449974882578","subCatalogs":null}]},{"level":1,"title":"局限性","paragraphId":"14995449991659785","subCatalogs":[{"level":2,"title":"5-羟甲基胞嘧啶","paragraphId":"14995449991659785","subCatalogs":null},{"level":2,"title":"不完全转化","paragraphId":"14995449991659785","subCatalogs":null},{"level":2,"title":"重亚硫酸盐处理过程中DNA的降解","paragraphId":"14995449991659785","subCatalogs":null},{"level":2,"title":"其他关注点","paragraphId":"14995449991659785","subCatalogs":null}]},{"level":1,"title":"应用：全基因组甲基化分析","paragraphId":"14995449991659786","subCatalogs":null},{"level":1,"title":"氧化重亚硫酸盐测序","paragraphId":"14995449991659787","subCatalogs":null},{"level":1,"title":"参考文献","paragraphId":"-1","subCatalogs":null}],"paragraphs":[{"paragraphId":"14995449974882578","title":"方法","versionId":"14995449974882573","lemmaId":10661,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":10145103,"name":"科学百科编辑组","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233984,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>重亚硫酸盐测序将常规测序方法应用于重亚硫酸盐处理后的基因组DNA，以确定CpG二核苷酸的甲基化状态。其他非测序的检测DNA甲基化的策略也被用于在特定位点或全基因组水平上检测甲基化情况。所有的检测策略都假定重亚硫酸盐诱导的未甲基化胞嘧啶向尿嘧啶的转化已完成，这是所有后续技术的基础。理想情况下，使用的方法将分别确定每个等位基因的甲基化状态。除了重亚硫酸盐测序方法外，还有联合亚硫酸氢钠限制性内切酶分析法和甲基化DNA免疫沉淀等方法。 </p>\n<p>分析重亚硫酸盐处理后的DNA甲基化的方法一直在不断发展。为了总结这些快速发展的方法，已经写了许多综述文章。<sup><a href=\"#quote_2\" class=\"kx_ref\">[2]</a></sup><sup><a href=\"#quote_3\" class=\"kx_ref\">[3]</a></sup><sup><a href=\"#quote_4\" class=\"kx_ref\">[4]</a></sup><sup><a href=\"#quote_5\" class=\"kx_ref\">[5]</a></sup> </p>\n<p>这些方法通常可以分为基于甲基化特异性聚合酶链式反应 (MSP)的策略和基于非甲基化特异性PCR(聚合酶链式反应)的策略。以微阵列为基础的检测方法使用的是基于非甲基化特异性PCR。 </p> \n<h3>基于非甲基化特异性PCR的方法</h3> \n<p></p><p><img alt=\"\" class=\"fileimage kx_img ed_imgfloat_right\" img_height=\"213\" img_width=\"450\" titlename=\"图3:不基于甲基化特异性PCR的DNA甲基化分析方法。重亚硫酸盐转化后，用不区分甲基化和非甲基化序列的PCR扩增基因组DNA。然后利用现有的多种方法根据重亚硫酸盐转化导致的扩增子内的变化进行鉴别。\" data-src=\"https://img03.sogoucdn.com/app/a/200698/sogou_science_13620\"> </p><p></p> \n<p><strong>直接测序</strong></p> \n<p>第一个报道的使用重亚硫酸盐检测DNA甲基化方法是利用PCR和标准双脱氧核苷酸测序直接测定对重亚硫酸盐转化有抗性的核苷酸。<sup><a href=\"#quote_6\" class=\"kx_ref\">[6]</a></sup> 引物被设计成链特异性和重亚硫酸盐特异性(即，引物含有非CpG胞嘧啶，因此它们与非重亚硫酸盐处理的DNA不互补)，位于需要检测的甲基化位点的侧翼(但不交叉)。因此，与甲基化特异性PCR相比，它将扩增甲基化和非甲基化序列。所有未甲基化胞嘧啶的位点在有义链的扩增序列中显示为胸腺嘧啶，在扩增的反义链中显示为腺嘌呤。通过将高通量测序衔接子结合到PCR引物中，PCR产物可以用大规模平行测序进行测序。或者，人工将PCR产物克隆之后进行测序。巢式PCR方法可用于提高扩增产物质量。 </p>\n<p>所有后续的使用重亚硫酸盐处理过DNA的DNA甲基化分析技术都是基于Frommer等人的成果(图2)<sup><a href=\"#quote_7\" class=\"kx_ref\">[7]</a></sup>。尽管大多数其他模式不是真正的测序技术，但是术语“重亚硫酸盐测序”通常用于描述重亚硫酸盐转化的DNA甲基化分析技术。 </p> \n<p><strong>焦磷酸测序</strong></p> \n<p>焦磷酸测序也是非甲基化特异性PCR测序的一种类型，用于分析重亚硫酸盐处理的DNA。<sup><a href=\"#quote_8\" class=\"kx_ref\">[8]</a></sup><sup><a href=\"#quote_9\" class=\"kx_ref\">[9]</a></sup> 在PCR扩增目的片段后，焦磷酸测序用于测定该区域特定CpG位点的重亚硫酸盐转化序列。在单个位点的C-T比率可以根据序列延伸过程中C-T结合的量来定量确定。这种方法的主要限制条件是技术成本。然而，焦磷酸测序很好地扩展应用到高通量筛选方法中。 </p>\n<p>Wong等人最近描述了对该技术的进一步改进，即使用等位基因特异性引物。该引物将单核苷酸多态性结合到测序引物的序列中，从而允许对母系和父系等位基因进行单独分析。<sup><a href=\"#quote_10\" class=\"kx_ref\">[10]</a></sup> 这项技术对基因组印记分析特别有用。 </p> \n<p><strong>甲基化敏感的单链构象分析(MS-SSCA)</strong></p> \n<p>该方法是基于单链构象多态性分析(SSCA)方法为分析单核苷酸多态性而开发的。<sup><a href=\"#quote_11\" class=\"kx_ref\">[11]</a></sup> SSCA基于非变性电泳中的差异迁移来区分大小相同但序列不同的单链DNA片段。在MS-SSCA中，这被用来区分重亚硫酸盐处理的、含有目的CpG位点的PCR扩增区域。尽管SSCA在只有单个核苷酸差异时缺乏敏感性，但重亚硫酸盐处理的DNA甲基化检测中，在大多数目的区域中往往有进行大量的C-T转化，因此，产生的敏感性接近100%。MS-SSCA还提供了基于条带强度比率的半定量分析。然而，该方法旨在评估目的区域内的所有CpG位点，而不是单个甲基化位点。 </p> \n<p><strong>高分辨率熔融分析</strong></p> \n<p>另一种区分转化的和未转化的重亚硫酸盐处理的DNA的方法是使用高分辨率熔解分析，这是一种基于定量PCR的技术，最初设计用于区分单核苷酸多态性。<sup><a href=\"#quote_12\" class=\"kx_ref\">[12]</a></sup> PCR扩增子直接通过温度升高和熔化过程中嵌入荧光染料的释放进行分析。甲基化程度，如扩增子中的C-T含量所代表的，决定了染料的熔化速度和随后的释放。这种方法允许在单管分析中直接定量，是评估扩增区域整体的甲基化，而不是特定CpG位点的甲基化。 </p> \n<p><strong>甲基化敏感的单核苷酸引物延伸(MS-SnuPE)</strong></p> \n<p>MS-SnuPE采用最初设计用于分析单核苷酸多态性的引物延伸方法。<sup><a href=\"#quote_13\" class=\"kx_ref\">[13]</a></sup> DNA被重亚硫酸盐转化，重亚硫酸盐特异性引物被退火到目标CpG之前的碱基对序列上。允许引物使用终止双脱氧核苷酸的DNA聚合酶将一个碱基对延伸到C(或T)中，并定量测定C与T的比率。 </p>\n<p>有许多方法可以用来确定这一C-T比率。起初，MS-SnuP依赖放射性ddNTPs作为引物延伸的标记。后来，荧光标记和焦磷酸测序也被使用。<sup><a href=\"#quote_14\" class=\"kx_ref\">[14]</a></sup> 此外，基质辅助激光解吸电离/飞行时间(MALDI-TOF)质谱分析可用于区分两种多态性引物延伸产物，本质上是基于为单核苷酸多态性基因分型设计的良好分析。反相离子对高效液相色谱也已经被用于区分引物延伸产物。<sup><a href=\"#quote_15\" class=\"kx_ref\">[15]</a></sup> </p> \n<p><strong>碱基特异性裂解/MALDI-TOF</strong></p> \n<p>Ehrich等人最近描述的方法通过添加碱基特异性切割步骤进一步利用重亚硫酸盐转化来增强从核苷酸变化中获得的信息。<sup><a href=\"#quote_16\" class=\"kx_ref\">[16]</a></sup> 首先，通过将目的区域体外转录成核糖核酸(通过在初始扩增中向PCR引物添加核糖核酸聚合酶启动子位点)，核糖核酸酶A可用于切割核糖核酸转录酶碱基特异性位点。由于核糖核酸酶A在胞嘧啶和尿嘧啶核糖核苷酸处特异性切割核糖核酸，当需要胞嘧啶特异性(C特异性)切割时，通过加入抗切割的dTTP，当需要尿嘧啶特异性(U特异性)切割时，加入dCTP，从而实现碱基特异性。然后裂解的片段可以通过MALDI-TOF进行分析。重亚硫酸盐处理导致通过C-U转化引入/去除切割位点，或者在扩增的反向链中通过G-A转化转移片段质量。C特异性切割将在所有甲基化CpG位点特异性切割。通过分析所得片段的大小，可以确定区域内CpG位点的特定甲基化模式，而不是确定整个区域的甲基化程度。这种方法证明了高通量筛选的有效性，允许以低成本的方式对多个组织中的多个CpG位点进行检测。 </p> \n<h3>甲基化特异性PCR</h3> \n<p></p><p><img alt=\"\" class=\"fileimage kx_img ed_imgfloat_right\" img_height=\"238\" img_width=\"400\" titlename=\"图4:甲基化特异性PCR是一种灵敏的方法，可在重亚硫酸盐转化的基因组DNA上使用甲基化特异性引物鉴别扩增以及检测目的甲基化区域。这种引物将只与甲基化的序列退火，因此仅包含有对重亚硫酸盐转化有抗性的5-甲基胞嘧啶。在另一种方式中，可以使用非甲基化特异性引物。\" data-src=\"https://img04.sogoucdn.com/app/a/200698/sogou_science_13621\"> </p><p></p> \n<p>这种甲基化分析的替代方法也使用重亚硫酸盐处理的DNA，但是避免了对目的区域测序的需要。<sup><a href=\"#quote_17\" class=\"kx_ref\">[17]</a></sup> 相反，引物对本身被设计为“甲基化特异性的”， 仅包括互补未转化的5-甲基胞嘧啶的序列，或者相反，设计为“非甲基化特异性的”，互补由非甲基化胞嘧啶转化而来的胸腺嘧啶。甲基化是由特定引物实现扩增的能力决定的。由于引物中CpG对数量的增加提高了检测的特异性，因此这种方法对于检测可能具有高甲基化密度的CpG岛特别有用。将CpG对置于引物的3’端能增强灵敏度。MSP的初始报告描述了足够灵敏度可以检测0.1%等位基因甲基化。一般来说，当检测特定位点的甲基化状态时，MSP及其相关方案被认为是最敏感的。 </p>\n<p>MethyLight方法基于MSP，但使用定量PCR提供定量分析。<sup><a href=\"#quote_18\" class=\"kx_ref\">[18]</a></sup> 使用甲基化特异性引物，并使用甲基化特异性荧光报告探针退火到扩增区域。在另一种方式中，如果需要区分相关序列中的CpG对，引物或探针可以设计成没有甲基化特异性。定量是参照甲基化的参考DNA进行的。为了提高PCR对成功重亚硫酸盐转化的DNA的特异性，对该方案的一项修改是使用重亚硫酸盐未转化的DNA的额外探针来定量这种非特异性扩增。<sup><a href=\"#quote_19\" class=\"kx_ref\">[19]</a></sup> </p>\n<p>对MSP扩增DNA的进一步方法是使用熔解曲线分析(Mc-MSP)来分析产物。<sup><a href=\"#quote_20\" class=\"kx_ref\">[20]</a></sup> 该方法用甲基化特异性引物和非甲基化特异性引物扩增重亚硫酸盐转化的DNA，并通过比较熔解曲线分析中产生的差异峰来确定两种产物的定量比。这是一种同时使用定量PCR和熔解分析的高分辨率熔解分析方法，特别是用于低水平甲基化的灵敏检测。<sup><a href=\"#quote_21\" class=\"kx_ref\">[21]</a></sup> </p> \n<h3>基于微阵列的方法</h3> \n<p>基于微阵列的方法是分析重亚硫酸盐处理DNA的技术的逻辑延伸，可用于整个基因组的甲基化水平的检测。<sup><a href=\"#quote_22\" class=\"kx_ref\">[22]</a></sup> 寡核苷酸微阵列是利用靶向目标CpG位点的寡核苷酸杂交探针对设计的。一个与未改变的甲基化序列互补，另一个与C到T转化的未甲基化序列互补。探针也是重亚硫酸盐特异性的，以防止与重亚硫酸盐未完全转化的DNA结合。Illumina公司甲基化分析是一种在微阵列水平上应用重亚硫酸盐测序技术来产生全基因组甲基化数据的分析。 </p>","pics":[{"originalUrl":"https://web.archive.org/web/20221025183652/https://img03.sogoucdn.com/app/a/200698/sogou_science_13620?w=450&h=213&titlename=%E5%9B%BE3:%E4%B8%8D%E5%9F%BA%E4%BA%8E%E7%94%B2%E5%9F%BA%E5%8C%96%E7%89%B9%E5%BC%82%E6%80%A7PCR%E7%9A%84DNA%E7%94%B2%E5%9F%BA%E5%8C%96%E5%88%86%E6%9E%90%E6%96%B9%E6%B3%95%E3%80%82%E9%87%8D%E4%BA%9A%E7%A1%AB%E9%85%B8%E7%9B%90%E8%BD%AC%E5%8C%96%E5%90%8E%EF%BC%8C%E7%94%A8%E4%B8%8D%E5%8C%BA%E5%88%86%E7%94%B2%E5%9F%BA%E5%8C%96%E5%92%8C%E9%9D%9E%E7%94%B2%E5%9F%BA%E5%8C%96%E5%BA%8F%E5%88%97%E7%9A%84PCR%E6%89%A9%E5%A2%9E%E5%9F%BA%E5%9B%A0%E7%BB%84DNA%E3%80%82%E7%84%B6%E5%90%8E%E5%88%A9%E7%94%A8%E7%8E%B0%E6%9C%89%E7%9A%84%E5%A4%9A%E7%A7%8D%E6%96%B9%E6%B3%95%E6%A0%B9%E6%8D%AE%E9%87%8D%E4%BA%9A%E7%A1%AB%E9%85%B8%E7%9B%90%E8%BD%AC%E5%8C%96%E5%AF%BC%E8%87%B4%E7%9A%84%E6%89%A9%E5%A2%9E%E5%AD%90%E5%86%85%E7%9A%84%E5%8F%98%E5%8C%96%E8%BF%9B%E8%A1%8C%E9%89%B4%E5%88%AB%E3%80%82","url":"https://web.archive.org/web/20221025183652/https://img03.sogoucdn.com/app/a/200698/sogou_science_13620","rw":450,"rh":213,"title":"图3:不基于甲基化特异性PCR的DNA甲基化分析方法。重亚硫酸盐转化后，用不区分甲基化和非甲基化序列的PCR扩增基因组DNA。然后利用现有的多种方法根据重亚硫酸盐转化导致的扩增子内的变化进行鉴别。","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221025183652/https://img04.sogoucdn.com/app/a/200698/sogou_science_13621?w=400&h=238&titlename=%E5%9B%BE4:%E7%94%B2%E5%9F%BA%E5%8C%96%E7%89%B9%E5%BC%82%E6%80%A7PCR%E6%98%AF%E4%B8%80%E7%A7%8D%E7%81%B5%E6%95%8F%E7%9A%84%E6%96%B9%E6%B3%95%EF%BC%8C%E5%8F%AF%E5%9C%A8%E9%87%8D%E4%BA%9A%E7%A1%AB%E9%85%B8%E7%9B%90%E8%BD%AC%E5%8C%96%E7%9A%84%E5%9F%BA%E5%9B%A0%E7%BB%84DNA%E4%B8%8A%E4%BD%BF%E7%94%A8%E7%94%B2%E5%9F%BA%E5%8C%96%E7%89%B9%E5%BC%82%E6%80%A7%E5%BC%95%E7%89%A9%E9%89%B4%E5%88%AB%E6%89%A9%E5%A2%9E%E4%BB%A5%E5%8F%8A%E6%A3%80%E6%B5%8B%E7%9B%AE%E7%9A%84%E7%94%B2%E5%9F%BA%E5%8C%96%E5%8C%BA%E5%9F%9F%E3%80%82%E8%BF%99%E7%A7%8D%E5%BC%95%E7%89%A9%E5%B0%86%E5%8F%AA%E4%B8%8E%E7%94%B2%E5%9F%BA%E5%8C%96%E7%9A%84%E5%BA%8F%E5%88%97%E9%80%80%E7%81%AB%EF%BC%8C%E5%9B%A0%E6%AD%A4%E4%BB%85%E5%8C%85%E5%90%AB%E6%9C%89%E5%AF%B9%E9%87%8D%E4%BA%9A%E7%A1%AB%E9%85%B8%E7%9B%90%E8%BD%AC%E5%8C%96%E6%9C%89%E6%8A%97%E6%80%A7%E7%9A%845-%E7%94%B2%E5%9F%BA%E8%83%9E%E5%98%A7%E5%95%B6%E3%80%82%E5%9C%A8%E5%8F%A6%E4%B8%80%E7%A7%8D%E6%96%B9%E5%BC%8F%E4%B8%AD%EF%BC%8C%E5%8F%AF%E4%BB%A5%E4%BD%BF%E7%94%A8%E9%9D%9E%E7%94%B2%E5%9F%BA%E5%8C%96%E7%89%B9%E5%BC%82%E6%80%A7%E5%BC%95%E7%89%A9%E3%80%82","url":"https://web.archive.org/web/20221025183652/https://img04.sogoucdn.com/app/a/200698/sogou_science_13621","rw":400,"rh":238,"title":"图4:甲基化特异性PCR是一种灵敏的方法，可在重亚硫酸盐转化的基因组DNA上使用甲基化特异性引物鉴别扩增以及检测目的甲基化区域。这种引物将只与甲基化的序列退火，因此仅包含有对重亚硫酸盐转化有抗性的5-甲基胞嘧啶。在另一种方式中，可以使用非甲基化特异性引物。","alt":null,"width":0,"height":0}],"card":null,"references":[],"versionCount":0},{"paragraphId":"14995449991659785","title":"局限性","versionId":"14995449974882574","lemmaId":10661,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":10145103,"name":"科学百科编辑组","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233984,"comment":null,"dependVersionId":0,"contentType":1,"content":"<h3>5-羟甲基胞嘧啶</h3> \n<p>重亚硫酸盐测序在哺乳动物基因组中被广泛使用，但是随着一种新的哺乳动物DNA修饰——5-羟甲基胞嘧啶的发现，情况变得复杂化。<sup><a href=\"#quote_23\" class=\"kx_ref\">[23]</a></sup><sup><a href=\"#quote_24\" class=\"kx_ref\">[24]</a></sup> 5-羟甲基胞嘧啶在重亚硫酸盐处理后转化为胞嘧啶-5-甲基磺酸盐，测序后被读成胞嘧啶碱基。<sup><a href=\"#quote_25\" class=\"kx_ref\">[25]</a></sup> 因此，重亚硫酸盐测序不能区分5-甲基胞嘧啶和5-羟甲基胞嘧啶。这意味着重亚硫酸盐测序的结果不能再定义为单一的DNA甲基化，而是5-甲基胞嘧啶和5-羟甲基胞嘧啶的复合物。芝加哥大学何川教授(Chuan He)开发的Tet辅助重亚硫酸盐测序现在能够在单碱基分辨率下区分这两种修饰。<sup><a href=\"#quote_26\" class=\"kx_ref\">[26]</a></sup> </p> \n<h3>不完全转化</h3> \n<p>重亚硫酸盐测序依赖于每个未甲基化胞嘧啶残基转化为尿嘧啶。如果转化不完全，后续分析将错误地将未成功转化的未甲基化胞嘧啶解释为甲基化胞嘧啶，导致甲基化的假阳性结果。只有单链DNA中的胞嘧啶容易受到重亚硫酸盐的攻击，因此DNA的变性是至关重要的。<sup><a href=\"#quote_2\" class=\"kx_ref\">[2]</a></sup> 转化中确保适合的反应参数，如温度和盐浓度来维持DNA的单链构象，尽量使非甲基化胞嘧啶完全转化。据报道，将DNA包埋在琼脂糖凝胶中，通过保持DNA链物理分离的状态，可以提高转化率。<sup><a href=\"#quote_27\" class=\"kx_ref\">[27]</a></sup> </p> \n<h3>重亚硫酸盐处理过程中DNA的降解</h3> \n<p>重亚硫酸盐测序的一个主要挑战是与转化同时发生的DNA降解。完全转化所需的条件，如长的培养时间、高温和高重亚硫酸盐浓度，可导致约90%的孵育的DNA降解。<sup><a href=\"#quote_28\" class=\"kx_ref\">[28]</a></sup> 鉴于DNA的起始量往往有限，这种大量的降解可能是有问题的。降解伴随脱嘌呤作用发生，导致DNA链发生随机断裂。<sup><a href=\"#quote_29\" class=\"kx_ref\">[29]</a></sup> 因此，期望得到的PCR扩增子越长，完整模板分子的数量可能越有限。这可能导致PCR扩增失败，或者由于模板分子取样有限而丢失甲基化水平定量的准确信息。因此，评估由所用反应条件导致的DNA降解量，并考虑这将如何影响所需的扩增子是很重要的。也可以从技术层面来减少DNA的降解，例如调整循环中培养温度。<sup><a href=\"#quote_29\" class=\"kx_ref\">[29]</a></sup> </p> \n<h3>其他关注点</h3> \n<p>重亚硫酸盐处理后一个潜在的重要问题是由于溶液碱化不充分导致嘧啶残基的不完全脱磺酸基。这可能会抑制某些DNA聚合酶，使后续的PCR很难扩增。然而，这种情况可以通过监测溶液的酸碱度来避免，以确保脱磺酸基作用充分完成。<sup><a href=\"#quote_2\" class=\"kx_ref\">[2]</a></sup> </p>\n<p>最后一个要关注的方面是重亚硫酸盐处理大大降低了样品的复杂程度，如果要进行多重PCR，这可能是个问题(2006年)。<sup><a href=\"#quote_5\" class=\"kx_ref\">[5]</a></sup> 引物设计更加困难，不恰当的杂交发生更加频繁。 </p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14995449991659786","title":"应用：全基因组甲基化分析","versionId":"14995449974882575","lemmaId":10661,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":10145103,"name":"科学百科编辑组","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233984,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>重亚硫酸盐测序的发展使全基因组范围内应用这一技术成为可能，然而在此之前，对DNA甲基化进行全局测量只能使用其他技术，例如限制性标记基因组扫描。人类表观基因组的绘制被许多科学家视为人类基因组计划完成的延伸。<sup><a href=\"#quote_30\" class=\"kx_ref\">[30]</a></sup><sup><a href=\"#quote_31\" class=\"kx_ref\">[31]</a></sup> 这些表观遗传信息对于理解基因序列的功能是如何实现和调控非常重要。由于表观基因组不如基因组稳定，它被认为在基因-环境相互作用中很重要。<sup><a href=\"#quote_32\" class=\"kx_ref\">[32]</a></sup> </p>\n<p>然而，表观基因组图谱绘制本质上比基因组测序更复杂，因为表观基因组比基因组更多变。一个人的表观基因组随年龄的增长而变化，不同组织之间也会有所不同，还会受环境因素的影响，并在疾病中表现出异常。然而，这种丰富的表观遗传图谱代表了不同的年龄、组织类型和疾病状态，将为表观遗传标记的正常功能以及导致衰老和疾病的机制方面产生有价值的信息。 </p>\n<p>表观基因组图谱的直接好处包括对克隆技术可能带来的进步。人们认为，未能生产出具有正常生存能力和寿命的克隆动物是表观遗传标记模式不当造成的。此外，异常甲基化模式在许多癌症中都有明确的特征。整体低甲基化导致基因组稳定性降低，而肿瘤抑制基因启动子的局部高甲基化通常导致其功能丧失。甲基化的特定模式指示特定的癌症类型，具有预后价值，并有助于指导最佳的治疗方案。<sup><a href=\"#quote_31\" class=\"kx_ref\">[31]</a></sup> </p>\n<p>世界各地正在进行大规模表观基因组绘图工作，并已在人类表观基因组计划下组织起来。<sup><a href=\"#quote_32\" class=\"kx_ref\">[32]</a></sup> 基于多层策略，即重亚硫酸盐测序被用来为有限数量的参考表观基因组获得高分辨率甲基化谱，而对更大范围的样品进行的分析则不太彻底。这种方法旨在最大限度从给定资源中获得的表观信息，因为完成全基因组高分辨率绘制仍然是一项昂贵的任务。 </p>\n<p>当应用于高通量甲基化数据(例如全基因组重亚硫酸盐测序)时，基因集分析(例如使用像DAVID和GoSeq这样的工具)已经显示出严重的偏差；有人建议，这可以通过使用样品标记排列或应用统计模型来矫正针对每个基因的CpG探针/ CpG位点数量的差异。<sup><a href=\"#quote_33\" class=\"kx_ref\">[33]</a></sup> </p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14995449991659787","title":"氧化重亚硫酸盐测序","versionId":"14995449974882576","lemmaId":10661,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":10145103,"name":"科学百科编辑组","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233984,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>5-甲基胞嘧啶和5-羟甲基胞嘧啶在重亚硫酸盐测序中都被读作胞嘧啶。<sup><a href=\"#quote_25\" class=\"kx_ref\">[25]</a></sup> 氧化重亚硫酸盐测序是一种以单碱基分辨率区分5-甲基胞嘧啶和5-羟甲基胞嘧啶的方法。该方法采用一种特异性化学氧化反应将5-羟甲基胞嘧啶转成5-甲酰胞嘧啶，然后在重亚硫酸盐处理过程中转化成尿嘧啶。<sup><a href=\"#quote_34\" class=\"kx_ref\">[34]</a></sup> 这样，5-甲基胞嘧啶和5-羟甲基胞嘧啶中唯一一个读作胞嘧啶的碱基是5-甲基胞嘧啶，这种方法给出了DNA样品中真正甲基化状态的图谱。5-羟甲基胞嘧啶的水平也可以通过测量重亚硫酸盐和氧化重亚硫酸盐测序之间的差异来定量。 </p>","pics":null,"card":null,"references":[],"versionCount":0}],"references":[{"id":1,"type":"book","title":"Chatterjee A, Stockwell PA, Rodger EJ and Morison IM 2012. Comparison of alignment software for genome-wide bisulphite sequence data. Nucleic Acids Research 40(10): e79.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":2,"type":"book","title":"Fraga MF, Esteller M (September 2002). \"DNA methylation: a profile of methods and applications\". BioTechniques. 33 (3): 632, 634, 636–49. doi:10.2144/02333rv01. PMID 12238773.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":3,"type":"book","title":"El-Maarri O (2003). \"Methods: DNA methylation\". Adv. Exp. Med. Biol. Advances in Experimental Medicine and Biology. 544: 197–204. doi:10.1007/978-1-4419-9072-3_23. ISBN 978-0-306-48174-1. PMID 14713229.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":4,"type":"book","title":"Laird PW (April 2003). \"The power and the promise of DNA methylation markers\". Nat. Rev. Cancer. 3 (4): 253–66. doi:10.1038/nrc1045. PMID 12671664.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":5,"type":"book","title":"Callinan PA, Feinberg AP (2006). \"The emerging science of epigenomics\". Hum Mol Genet. 15 (90001): R95–101. doi:10.1093/hmg/ddl095. PMID 16651376.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":6,"type":"book","title":"Frommer M, McDonald LE, Millar DS, et al. (March 1992). \"A genomic sequencing protocol that yields a positive display of 5-methylcytosine residues in individual DNA strands\". Proc. Natl. Acad. Sci. U.S.A. 89 (5): 1827–31. doi:10.1073/pnas.89.5.1827. PMC 48546. PMID 1542678.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":7,"type":"book","title":"Frommer M, McDonald LE, Millar DS, et al. (March 1992). \"A genomic sequencing protocol that yields a positive display of 5-methylcytosine residues in individual DNA strands\". Proc. Natl. Acad. Sci. U.S.A. 89 (5): 1827–31. doi:10.1073/pnas.89.5.1827. 位于PMC的全文本内容：48546. PubMed.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":8,"type":"book","title":"Colella S, Shen L, Baggerly KA, Issa JP, Krahe R (July 2003). \"Sensitive and quantitative universal Pyrosequencing methylation analysis of CpG sites\". BioTechniques. 35 (1): 146–50. doi:10.2144/03351md01. PMID 12866414.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":9,"type":"book","title":"Tost J, Dunker J, Gut IG (July 2003). \"Analysis and quantification of multiple methylation variable positions in CpG islands by Pyrosequencing\". BioTechniques. 35 (1): 152–6. doi:10.2144/03351md02. PMID 12866415.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":10,"type":"book","title":"Wong HL, Byun HM, Kwan JM, et al. (December 2006). \"Rapid and quantitative method of allele-specific DNA methylation analysis\". BioTechniques. 41 (6): 734–9. doi:10.2144/000112305. PMID 17191619.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":11,"type":"book","title":"Bianco T, Hussey D, Dobrovic A (1999). \"Methylation-sensitive, single-strand conformation analysis (MS-SSCA): A rapid method to screen for and analyze methylation\". Hum. Mutat. 14 (4): 289–93. doi:10.1002/(SICI)1098-1004(199910)14:4<289::AID-HUMU3>3.0.CO;2-A. PMID 10502775.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":12,"type":"book","title":"Wojdacz TK, Dobrovic A (2007). \"Methylation-sensitive high resolution melting (MS-HRM): a new approach for sensitive and high-throughput assessment of methylation\". Nucleic Acids Res. 35 (6): e41. doi:10.1093/nar/gkm013. PMC 1874596. PMID 17289753.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":13,"type":"book","title":"Gonzalgo ML, Jones PA (June 1997). \"Rapid quantitation of methylation differences at specific sites using methylation-sensitive single nucleotide primer extension (Ms-SNuPE)\". Nucleic Acids Res. 25 (12): 2529–31. doi:10.1093/nar/25.12.2529. PMC 146734. PMID 9171109.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":14,"type":"book","title":"Uhlmann K, Brinckmann A, Toliat MR, Ritter H, Nürnberg P (December 2002). \"Evaluation of a potential epigenetic biomarker by quantitative methyl-single nucleotide polymorphism analysis\". Electrophoresis. 23 (24): 4072–9. doi:10.1002/elps.200290023. PMID 12481262.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":15,"type":"book","title":"Matin MM, Baumer A, Hornby DP (October 2002). \"An analytical method for the detection of methylation differences at specific chromosomal loci using primer extension and ion pair reverse phase HPLC\". Hum. Mutat. 20 (4): 305–11. doi:10.1002/humu.10118. PMID 12325026.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":16,"type":"book","title":"Ehrich M, Nelson MR, Stanssens P, et al. (November 2005). \"Quantitative high-throughput analysis of DNA methylation patterns by base-specific cleavage and mass spectrometry\". Proc. Natl. Acad. Sci. U.S.A. 102 (44): 15785–90. doi:10.1073/pnas.0507816102. PMC 1276092. PMID 16243968.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":17,"type":"book","title":"Herman JG, Graff JR, Myöhänen S, Nelkin BD, Baylin SB (September 1996). \"Methylation-specific PCR: a novel PCR assay for methylation status of CpG islands\". Proc. Natl. Acad. Sci. U.S.A. 93 (18): 9821–6. doi:10.1073/pnas.93.18.9821. PMC 38513. PMID 8790415.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":18,"type":"book","title":"Eads CA, Danenberg KD, Kawakami K, et al. (April 2000). \"MethyLight: a high-throughput assay to measure DNA methylation\". Nucleic Acids Res. 28 (8): 32e–0. doi:10.1093/nar/28.8.e32. PMC 102836. PMID 10734209.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":19,"type":"book","title":"Rand K, Qu W, Ho T, Clark SJ, Molloy P (June 2002). \"Conversion-specific detection of DNA methylation using real-time polymerase chain reaction (ConLight-MSP) to avoid false positives\". Methods. 27 (2): 114–20. doi:10.1016/S1046-2023(02)00062-2. PMID 12095268.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":20,"type":"book","title":"Akey DT, Akey JM, Zhang K, Jin L (October 2002). \"Assaying DNA methylation based on high-throughput melting curve approaches\". Genomics. 80 (4): 376–84. doi:10.1006/geno.2002.6851. PMID 12376091.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":21,"type":"book","title":"Kristensen LS, Mikeska T, Krypuy M, Dobrovic A (April 2008). \"Sensitive Melting Analysis after Real Time- Methylation Specific PCR (SMART-MSP): high-throughput and probe-free quantitative DNA methylation detection\". Nucleic Acids Res. 36 (7): e42. doi:10.1093/nar/gkn113. PMC 2367707. PMID 18344521.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":22,"type":"book","title":"Adorján P, Distler J, Lipscher E, et al. (March 2002). \"Tumour class prediction and discovery by microarray-based DNA methylation analysis\". Nucleic Acids Res. 30 (5): 21e–21. doi:10.1093/nar/30.5.e21. PMC 101257. PMID 11861926.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":23,"type":"book","title":"Tahiliani M, Koh KP, Shen Y, Pastor WA, Bandukwala H, BrudnoY, et al. Conversion of 5-methylcytosine to 5-hydroxymethylcytosine inmammalian DNA by MLL partner TET1. Science. 2009;324(5929):930-5.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":24,"type":"book","title":"Kriaucionis S, Heintz N. The nuclear DNA base 5-hydroxymethylcytosine is present in Purkinje neurons and the brain. Science.2009;324(5929):929-30.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":25,"type":"book","title":"Huang Y, Pastor WA, Shen Y, Tahiliani M, Liu DR, Rao A. The Behaviour of 5-Hydroxymethylcytosine in Bisulfite Sequencing. PLOS ONE.2010;5(1):e8888.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":26,"type":"book","title":"Yu, M., Hon, G. C., Szulwach, K. E., Song, C., Jin, P., Ren, B., He, C. Tet-assisted bisulfite sequencing of 5-hydroxymethylcytosine. Nat. Protocols 2012, 7, 2159.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":27,"type":"book","title":"Olek A, Oswald J, Walter J (December 1996). \"A modified and improved method for bisulphite based cytosine methylation analysis\". Nucleic Acids Res. 24 (24): 5064–6. doi:10.1093/nar/24.24.5064. PMC 146326. PMID 9016686. Archived from the original on 2012-07-14.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":28,"type":"book","title":"Grunau C, Clark SJ, Rosenthal A (July 2001). \"Bisulfite genomic sequencing: systematic investigation of critical experimental parameters\". Nucleic Acids Res. 29 (13): E65–5. doi:10.1093/nar/29.13.e65. PMC 55789. PMID 11433041.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":29,"type":"book","title":"Ehrich M, Zoll S, Sur S, van den Boom D (2007). \"A new method for accurate assessment of DNA quality after bisulfite treatment\". Nucleic Acids Res. 35 (5): e29. doi:10.1093/nar/gkl1134. PMC 1865059. PMID 17259213.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":30,"type":"book","title":"Esteller M (June 2006). \"The necessity of a human epigenome project\". Carcinogenesis. 27 (6): 1121–5. doi:10.1093/carcin/bgl033. PMID 16699174.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":31,"type":"book","title":"Bradbury J (December 2003). \"Human Epigenome Project—Up and Running\". PLoS Biol. 1 (3): E82. doi:10.1371/journal.pbio.0000082. PMC 300691. PMID 14691553.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":32,"type":"book","title":"Jones PA, Martienssen R (December 2005). \"A blueprint for a Human Epigenome Project: the AACR Human Epigenome Workshop\". Cancer Res. 65 (24): 11241–6. doi:10.1158/0008-5472.CAN-05-3865. PMID 16357125.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":33,"type":"book","title":"Geeleher P, Hartnett L, Egan LJ, Golden A, Raja Ali RA, Seoighe C (June 2013). \"Gene-Set Analysis is Severely Biased When Applied to Genome-wide Methylation Data\". Bioinformatics. 29 (15): 1851–7. doi:10.1093/bioinformatics/btt311. PMID 23732277.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":34,"type":"book","title":"Booth MJ, Branco MR, Ficz G, Oxley D, Krueger F, Reik W, et al. quantitative sequencing of 5-methylcytosine and 5-hydroxymethylcytosine at single-base resolution. Science. 2012;336(6083):934-7.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false}],"recommendReferences":null,"auditState":2,"lemmaLevel":1,"origin":0,"originEnTitle":null,"originZhTitle":null,"pv":94,"auditType":0,"synonyms":null,"showEditTime":"2019.12.13 18:46","auditors":[{"uid":0,"name":"Ki.κe","pic":"https://web.archive.org/web/20221025183652/https://wx.qlogo.cn/mmopen/vi_32/y67kfr32Doib4wg71Jiau7jVWvharic3nRKgdRRQSl6koeQJCo0GQs2Krw0vwdFRsOWnHIQOwAZsSg5lIkIrFCOcQ/132","introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":"","jobBrief":"","role":0,"roleName":null,"title":"","professionalTitle":null,"phoneNo":null,"editable":true,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false}],"hasZhishiNav":false,"auditInfos":{},"isHistory":false};</script><script crossorigin="anonymous" src="./661.重亚硫酸盐测序 - 搜狗科学百科_files/aegis.min.js.download"></script><script crossorigin="anonymous" src="./661.重亚硫酸盐测序 - 搜狗科学百科_files/main_2020092401.js.download"></script><script crossorigin="anonymous" src="./661.重亚硫酸盐测序 - 搜狗科学百科_files/react.production.min.js.download"></script><script crossorigin="anonymous" src="./661.重亚硫酸盐测序 - 搜狗科学百科_files/react-dom.production.min.js.download"></script><script crossorigin="anonymous" src="./661.重亚硫酸盐测序 - 搜狗科学百科_files/jquery-1.11.1.min.js.download"></script><script crossorigin="anonymous" src="./661.重亚硫酸盐测序 - 搜狗科学百科_files/main_2022062701.js.download"></script><script crossorigin="anonymous" src="./661.重亚硫酸盐测序 - 搜狗科学百科_files/main_66bbe21.js.download"></script><script crossorigin="anonymous" src="./661.重亚硫酸盐测序 - 搜狗科学百科_files/react.production.min.js.download"></script><script crossorigin="anonymous" src="./661.重亚硫酸盐测序 - 搜狗科学百科_files/react-dom.production.min.js.download"></script><script crossorigin="anonymous" src="./661.重亚硫酸盐测序 - 搜狗科学百科_files/main_edf0f08.js.download"></script>
</body></html>