<!DOCTYPE html>
<!-- saved from url=(0083)https://web.archive.org/web/20221025122658/https://baike.sogou.com/kexue/d10681.htm -->
<html class="" data-reactroot=""><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><script src="./681.基因复制 - 搜狗科学百科_files/analytics.js.download" type="text/javascript"></script>
<script type="text/javascript">window.addEventListener('DOMContentLoaded',function(){var v=archive_analytics.values;v.service='wb';v.server_name='wwwb-app220.us.archive.org';v.server_ms=192;archive_analytics.send_pageview({});});</script>
<script type="text/javascript" src="./681.基因复制 - 搜狗科学百科_files/bundle-playback.js.download" charset="utf-8"></script>
<script type="text/javascript" src="./681.基因复制 - 搜狗科学百科_files/wombat.js.download" charset="utf-8"></script>
<script type="text/javascript">
  __wm.init("https://web.archive.org/web");
  __wm.wombat("https://baike.sogou.com/kexue/d10681.htm","20221025122658","https://web.archive.org/","web","/_static/",
	      "1666700818");
</script>
<link rel="stylesheet" type="text/css" href="./681.基因复制 - 搜狗科学百科_files/banner-styles.css">
<link rel="stylesheet" type="text/css" href="./681.基因复制 - 搜狗科学百科_files/iconochive.css">
<!-- End Wayback Rewrite JS Include -->
<meta name="save" content="history"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="baidu-site-verification" content="VWGb6TyYx8"><meta content="基因复制 - 搜狗科学百科" name="keywords"><meta content="搜狗科学百科是一部有着平等、协作、分享、自由理念的网络科学全书，为每一个互联网用户创造一个涵盖所有领域知识、服务的中文知识性平台。" name="description"><meta http-equiv="x-dns-prefetch-control" content="on"><meta name="server" baike="235" ip="210" env="online"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025122658/https://cache.soso.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025122658/https://hhy.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025122658/https://pic.baike.soso.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025122658/https://ugc.qpic.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025122658/https://xui.ptlogin2.qq.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025122658/https://q1.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025122658/https://q2.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025122658/https://q3.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025122658/https://q4.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025122658/https://q.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025122658/https://img01.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025122658/https://img02.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025122658/https://img03.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025122658/https://img04.sogoucdn.com/"><link rel="Shortcut Icon" href="https://web.archive.org/web/20221025122658im_/https://www.sogou.com/images/logo/new/favicon.ico?v=4"><link rel="Bookmark" href="https://www.sogou.com/images/logo/new/favicon.ico?v=4"><link href="./681.基因复制 - 搜狗科学百科_files/base_b849887.css" rel="stylesheet"><link href="./681.基因复制 - 搜狗科学百科_files/detail_378aed5.css" rel="stylesheet"><link href="./681.基因复制 - 搜狗科学百科_files/inviteAudit_7894507.css" rel="stylesheet"><link rel="stylesheet" href="./681.基因复制 - 搜狗科学百科_files/highlight.min.css"><title>基因复制 - 搜狗科学百科</title><style>.onekey-close {
	position: absolute;
	top: 16px;
	right: 16px;
	width: 24px;
	height: 24px;
	background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	text-indent: -999em;
	background-size: 84px;
	background-position: -63px 0;
}

.onekey-login {
	position: absolute;
	top: 16.4%;
	left: 0;
	right: 0;
	width: 100%;
}

/* .onekey-login-img {
    width: 75px;
    height: 75px;
    background: url("https://web.archive.org/web/20221025122658/https://baike.sogou.com/kexue/images/sprite_wap_baike.png") no-repeat;
    background-size: 100px 91px;
    background-position: 0 0;
    background-repeat: no-repeat;
    margin: 0 auto;
} */

.onekey-login-title {
	text-align: center;
	padding-bottom: 3px;
	font-size: 21px;
	font-weight: bold;
	line-height: 30px;
	color: #000;
}

.onekey-login-txt {
	text-align: center;
	font-family: PingFangSC;
	font-size: 14px;
	line-height: 20px;
	color: #8f8f8f;
}

.onekey-login-qq,
.onekey-login-wx,
.onekey-login-phone {
	display: block;
	width: 245px;
	height: 54px;
	border-radius: 45px;
	text-align: center;

	margin: 0 auto;
	font-size: 17px;
	line-height: 24px;
	color: #000;
	/* padding: 16px 77px; */
	border-radius: 12px;
	border: solid 1px #e0e0e0;
}
.onekey-qq-content,
.onekey-vx-content,
.onekey-phone-content {
	display: inline-block;
	margin-top: 16px;
}
.onekey-qq-content {
	padding: 0 5px;
}

.onekey-login-qq {
	margin-top: 48px;
	margin-bottom: 24px;
}

.onekey-login-qq:before {
	display: inline-block;
	content: "";
	width: 20px;
	height: 20px;
	background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	background-size: 80px;
	background-position: -20px 0;
	vertical-align: top;
	margin: 17px 8px 0 0;
}

.onekey-login-wx {
	margin-bottom: 24px;
}

.onekey-login-wx:before {
	display: inline-block;
	content: "";
	width: 21px;
	height: 21px;
	background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	background-size: 84px;
	background-position: 0 0;
	vertical-align: top;
	margin: 17px 10px 0 0;
}

.onekey-login-phone {
}

.onekey-login-phone:before {
	display: inline-block;
	content: "";
	width: 21px;
	height: 21px;
	background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	background-size: 84px;
	background-position: -42px 0;
	vertical-align: top;
	margin: 17px 10px 0 0;
}

.onekey-fixed {
	z-index: 100;
	position: fixed;
	top: 0;
	bottom: 0;
	left: 0;
	right: 0;
	background: #fff;
	width: 100%;
	height: 100%;
}

.onekey-fixed.forbid {
	z-index: 100;
	position: fixed;
	top: auto;
	bottom: 68px;
	left: 9%;
	right: 0;
	background: rgba(0, 0, 0, 0.7);
	width: 82%;
	height: 43px;
	border-radius: 25px;
	color: #ffffff;
}
.onekey-login-title.forbid {
	text-align: center;
	padding-bottom: 3px;
	font-size: 14px;
	font-weight: normal;
	line-height: 30px;
	color: white;
}
</style><style>#login_mask {
  background: #000;
  opacity: 0.5;
  filter: alpha(opacity=50);
  position: fixed;
  /*fixed好像在哪个IE上有BUG，先用用*/
  left: 0;
  top: 0;
  z-index: 999;
  height: 100%;
}

#login_iframe_container {
  position: fixed;
  width: 550px;
  height: 360px;
  z-index: 1020;
  background-color: #ffffff;
}

@media screen and (max-width: 828px) {
  #login_iframe_container {
    top: 50% !important;
    left: 50% !important;
    transform: translate(-50%, -50%);
  }
}

#login_iframe_container.new-login {
  width: 550px;
  height: 360px;
  background-image: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/background_2a4a8a6.png);
}

#login_iframe_container.new-login.no-bg {
  background: #fff;
}

#login_iframe_container.new-login .login-title {
  width: 100%;
  height: 42px;
  line-height: 42px;
  text-align: center;
  font-size: 30px;
  letter-spacing: 0.19px;
  color: #ffffff;
  margin-top: 62px;
}
#login_iframe_container.new-login .forbid-title {
  width: 100%;
  height: 42px;
  line-height: 42px;
  text-align: center;
  font-size: 24px;
  letter-spacing: 0.19px;
  color: #333333;
  margin-top: 150px;
}

#login_iframe_container.new-login.no-bg .login-title {
  color: #333333;
}

#login_iframe_container.new-login .login-subtitle {
  width: 100%;
  height: 18px;
  line-height: 18px;
  font-size: 13px;
  letter-spacing: 0.08px;
  color: #ffffff;
  text-align: center;
  margin-top: 9px;
  margin-bottom: 43px;
}

#login_iframe_container.new-login.no-bg .login-subtitle {
  color: #999999;
}

#login_iframe_container.new-login .login-subtitle::before {
  content: '';
  display: inline-block;
  width: 10px;
  height: 1px;
  background-color: #ffffff;
  position: relative;
  top: -4px;
  left: -5px;
}

#login_iframe_container.new-login .login-subtitle::after {
  content: '';
  display: inline-block;
  width: 10px;
  height: 1px;
  background-color: #ffffff;
  position: relative;
  top: -4px;
  left: 5px;
}

#login_iframe_container.new-login.no-bg .login-subtitle::before {
  background-color: #999999;
}

#login_iframe_container.new-login.no-bg .login-subtitle::after {
  background-color: #999999;
}

#login_iframe_container.new-login .close-btn {
  position: absolute;
  top: 20px;
  right: 20px;
  width: 12px;
  height: 12px;
  background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/login-sprites_e3853e5.png) -59px -10px;
  background-size: 81px 91px;
  cursor: pointer;
}

#login_iframe_container.new-login .login-btn {
  width: 220px;
  height: 47px;
  border-radius: 24px;
  border: solid 1px #dddddd;
  background-color: #ffffff;
  margin: 0 auto;
  margin-top: 28px;
  position: relative;
  display: block;
}

#login_iframe_container.new-login .login-btn .login-icon {
  position: absolute;
}

#login_iframe_container.new-login .login-btn .login-text {
  width: 61px;
  height: 47px;
  line-height: 47px;
  vertical-align: middle;
  font-size: 15px;
  letter-spacing: 0.1px;
  color: #666666;
  position: absolute;
  right: 62px;
}

#login_iframe_container.new-login .login-btn.qq-btn .login-icon {
  width: 22px;
  height: 27px;
  top: 10px;
  left: 67px;
  background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/login-sprites_e3853e5.png) -10px -54px;
  background-size: 81px 91px;
}

#login_iframe_container.new-login .login-btn.qq-btn .login-text {
  right: 59px;
}

#login_iframe_container.new-login .login-btn.wechat-btn .login-icon {
  width: 29px;
  height: 24px;
  top: 12px;
  left: 62px;
  background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/login-sprites_e3853e5.png) -10px -10px;
  background-size: 81px 91px;
}</style><style>/* -- container -- */
.rodal,
.rodal-mask {
    top: 0;
    left: 0;
    width: 100%;
    height: 100%;
    z-index: 100;
}

.rodal {
    position: fixed;
}

/* -- mask -- */
.rodal-mask {
    position: fixed;
    background: rgba(0, 0, 0, .5);
}

/* -- dialog -- */
.rodal-dialog {
    position: absolute;
    z-index: 101;
    background: #fff;
    border-radius: 3px;
    box-shadow: 0 1px 3px rgba(0, 0, 0, .2);
}

.rodal-center {
    top: 50%;
    transform: translateY(-50%);
    left: 0;
    right: 0;
    margin: 0 auto;
}

.rodal-bottom {
    left: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

.rodal-top {
    left: 0;
    right: 0;
    top: 0;
    margin: auto;
}

.rodal-left {
    top: 0;
    left: 0;
    bottom: 0;
    margin: auto;
}

.rodal-right {
    top: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

/* -- close button -- */
.rodal-close {
    position: absolute;
    cursor: pointer;
    top: 16px;
    right: 16px;
    width: 16px;
    height: 16px;
}

.rodal-close:before,
.rodal-close:after {
    position: absolute;
    content: '';
    height: 2px;
    width: 100%;
    top: 50%;
    left: 0;
    margin-top: -1px;
    background: #999;
    border-radius: 100%;
    -webkit-transition: background .2s;
    transition: background .2s;
}

.rodal-close:before {
    -webkit-transform: rotate(45deg);
    transform: rotate(45deg);
}

.rodal-close:after {
    -webkit-transform: rotate(-45deg);
    transform: rotate(-45deg);
}

.rodal-close:hover:before,
.rodal-close:hover:after {
    background: #333;
}

/* -- fade -- */
/* @-webkit-keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

@keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

.rodal-fade-enter {
    -webkit-animation: rodal-fade-enter both ease-in;
    animation: rodal-fade-enter both ease-in;
} */

@-webkit-keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

@keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

.rodal-fade-leave {
    -webkit-animation: rodal-fade-leave both ease-out;
    animation: rodal-fade-leave both ease-out;
}

/* -- zoom -- */
@-webkit-keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-enter {
    -webkit-animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-leave {
    -webkit-animation: rodal-zoom-leave both;
    animation: rodal-zoom-leave both;
}

/* -- slideDown -- */
@-webkit-keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-enter {
    -webkit-animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-leave {
    -webkit-animation: rodal-slideDown-leave both;
    animation: rodal-slideDown-leave both;
}

/* -- slideLeft -- */
@-webkit-keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-enter {
    -webkit-animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-leave {
    -webkit-animation: rodal-slideLeft-leave both;
    animation: rodal-slideLeft-leave both;
}

/* -- slideRight -- */
@-webkit-keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-enter {
    -webkit-animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-leave {
    -webkit-animation: rodal-slideRight-leave both;
    animation: rodal-slideRight-leave both;
}

/* -- slideUp -- */
@-webkit-keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-enter {
    -webkit-animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
    animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
}

@-webkit-keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-leave {
    -webkit-animation: rodal-slideUp-leave both;
    animation: rodal-slideUp-leave both;
}

/* -- flip -- */
@-webkit-keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

@keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

.rodal-flip-enter {
    -webkit-animation: rodal-flip-enter both ease-in;
    animation: rodal-flip-enter both ease-in;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

@-webkit-keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

@keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

.rodal-flip-leave {
    -webkit-animation: rodal-flip-leave both;
    animation: rodal-flip-leave both;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

/* -- rotate -- */
@-webkit-keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-enter {
    -webkit-animation: rodal-rotate-enter both;
    animation: rodal-rotate-enter both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

@-webkit-keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-leave {
    -webkit-animation: rodal-rotate-leave both;
    animation: rodal-rotate-leave both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

/* -- door -- */
@-webkit-keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

@keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

.rodal-door-enter {
    -webkit-animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

@keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

.rodal-door-leave {
    -webkit-animation: rodal-door-leave both;
    animation: rodal-door-leave both;
}</style><style>/* -- container -- */
.rodal,
.rodal-mask {
    top: 0;
    left: 0;
    width: 100%;
    height: 100%;
    z-index: 100;
}

.rodal {
    position: fixed;
}

/* -- mask -- */
.rodal-mask {
    position: fixed;
    background: rgba(0, 0, 0, .5);
}

/* -- dialog -- */
.rodal-dialog {
    position: absolute;
    z-index: 101;
    background: #fff;
    border-radius: 3px;
    box-shadow: 0 1px 3px rgba(0, 0, 0, .2);
}

.rodal-center {
    top: 50%;
    transform: translateY(-50%);
    left: 0;
    right: 0;
    margin: 0 auto;
}

.rodal-bottom {
    left: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

.rodal-top {
    left: 0;
    right: 0;
    top: 0;
    margin: auto;
}

.rodal-left {
    top: 0;
    left: 0;
    bottom: 0;
    margin: auto;
}

.rodal-right {
    top: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

/* -- close button -- */
.rodal-close {
    position: absolute;
    cursor: pointer;
    top: 16px;
    right: 16px;
    width: 16px;
    height: 16px;
}

.rodal-close:before,
.rodal-close:after {
    position: absolute;
    content: '';
    height: 2px;
    width: 100%;
    top: 50%;
    left: 0;
    margin-top: -1px;
    background: #999;
    border-radius: 100%;
    -webkit-transition: background .2s;
    transition: background .2s;
}

.rodal-close:before {
    -webkit-transform: rotate(45deg);
    transform: rotate(45deg);
}

.rodal-close:after {
    -webkit-transform: rotate(-45deg);
    transform: rotate(-45deg);
}

.rodal-close:hover:before,
.rodal-close:hover:after {
    background: #333;
}

/* -- fade -- */
/* @-webkit-keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

@keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

.rodal-fade-enter {
    -webkit-animation: rodal-fade-enter both ease-in;
    animation: rodal-fade-enter both ease-in;
} */

@-webkit-keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

@keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

.rodal-fade-leave {
    -webkit-animation: rodal-fade-leave both ease-out;
    animation: rodal-fade-leave both ease-out;
}

/* -- zoom -- */
@-webkit-keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-enter {
    -webkit-animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-leave {
    -webkit-animation: rodal-zoom-leave both;
    animation: rodal-zoom-leave both;
}

/* -- slideDown -- */
@-webkit-keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-enter {
    -webkit-animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-leave {
    -webkit-animation: rodal-slideDown-leave both;
    animation: rodal-slideDown-leave both;
}

/* -- slideLeft -- */
@-webkit-keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-enter {
    -webkit-animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-leave {
    -webkit-animation: rodal-slideLeft-leave both;
    animation: rodal-slideLeft-leave both;
}

/* -- slideRight -- */
@-webkit-keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-enter {
    -webkit-animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-leave {
    -webkit-animation: rodal-slideRight-leave both;
    animation: rodal-slideRight-leave both;
}

/* -- slideUp -- */
@-webkit-keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-enter {
    -webkit-animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
    animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
}

@-webkit-keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-leave {
    -webkit-animation: rodal-slideUp-leave both;
    animation: rodal-slideUp-leave both;
}

/* -- flip -- */
@-webkit-keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

@keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

.rodal-flip-enter {
    -webkit-animation: rodal-flip-enter both ease-in;
    animation: rodal-flip-enter both ease-in;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

@-webkit-keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

@keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

.rodal-flip-leave {
    -webkit-animation: rodal-flip-leave both;
    animation: rodal-flip-leave both;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

/* -- rotate -- */
@-webkit-keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-enter {
    -webkit-animation: rodal-rotate-enter both;
    animation: rodal-rotate-enter both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

@-webkit-keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-leave {
    -webkit-animation: rodal-rotate-leave both;
    animation: rodal-rotate-leave both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

/* -- door -- */
@-webkit-keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

@keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

.rodal-door-enter {
    -webkit-animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

@keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

.rodal-door-leave {
    -webkit-animation: rodal-door-leave both;
    animation: rodal-door-leave both;
}</style>
<!--百度统计-->
<script>
   var _hmt = _hmt || [];
   (function() {
      var hm = document.createElement("script");
      hm.src = "https://hm.baidu.com/hm.js?3c7614be3026469d5a60f41ab30b5082";
      var s = document.getElementsByTagName("script")[0]; 
      s.parentNode.insertBefore(hm, s);
      })();
</script>

	<!--百度统计-->
	<script>
		var _hmt = _hmt || [];
		(function() {
			var hm = document.createElement("script");
			hm.src = "https://hm.baidu.com/hm.js?3c7614be3026469d5a60f41ab30b5082";
			var s = document.getElementsByTagName("script")[0]; 
			s.parentNode.insertBefore(hm, s);
			})();
	</script>
</head>
<body class=""><!-- BEGIN WAYBACK TOOLBAR INSERT -->
<style type="text/css">
body {
  margin-top:0 !important;
  padding-top:0 !important;
  /*min-width:800px !important;*/
}
</style>
<script>__wm.rw(0);</script>
<div id="wm-ipp-base" lang="en" style="display: block; direction: ltr;">
</div><div id="wm-ipp-print">The Wayback Machine - https://web.archive.org/web/20221025122658/https://baike.sogou.com/kexue/d10681.htm</div>
<script type="text/javascript">//<![CDATA[
__wm.bt(675,27,25,2,"web","https://baike.sogou.com/kexue/d10681.htm","20221025122658",1996,"/_static/",["/_static/css/banner-styles.css?v=S1zqJCYt","/_static/css/iconochive.css?v=qtvMKcIJ"], false);
  __wm.rw(1);
//]]></script>
<!-- END WAYBACK TOOLBAR INSERT --><script>window._gtag=window._gtag||{};window._gtag.shouldGrayed = false;if ('2c3e8e3eb1934ccf91bb8a7e116098fd') window._gtag.traceId = '2c3e8e3eb1934ccf91bb8a7e116098fd';if ({"illegality":true}) window.userInfo = {"illegality":true};</script><div class="topnavbox"><ul class="topnav"><li><a href="https://web.archive.org/web/20221025122658/https://www.sogou.com/web?query=">网页</a></li><li><a href="https://web.archive.org/web/20221025122658/https://weixin.sogou.com/weixin?p=75351201">微信</a></li><li><a href="https://web.archive.org/web/20221025122658/https://zhihu.sogou.com/zhihu?p=75351218">知乎</a></li><li><a href="https://web.archive.org/web/20221025122658/https://pic.sogou.com/pics?query=">图片</a></li><li><a href="https://web.archive.org/web/20221025122658/https://v.sogou.com/v?query=">视频</a></li><li><a href="https://web.archive.org/web/20221025122658/https://mingyi.sogou.com/">医疗</a></li><li class="cur"><strong>科学</strong></li><li><a href="https://web.archive.org/web/20221025122658/https://hanyu.sogou.com/">汉语</a></li><li><a href="https://web.archive.org/web/20221025122658/https://wenwen.sogou.com/">问问</a></li><li><a href="https://web.archive.org/web/20221025122658/https://www.sogou.com/docs/more.htm">更多<span class="topraquo">»</span></a></li></ul></div><div id="header"><div class="header-wrap"><a class="header-logo" href="https://web.archive.org/web/20221025122658/https://baike.sogou.com/kexue"></a><div class="header-search"><div class="querybox" id="suggBox"><form><input id="searchInput" class="query" type="text" placeholder="搜科学领域专业百科词条" name="query" autocomplete="off" value=""><a href="javascript:;" class="query-search"></a></form></div></div><div class="header-rgt"><span class="btn-header-rgt btn-edit" id="editLemma">创建</span><div class="header-user no-login"></div></div></div></div><div class="fixed-placeholder" style="visibility:none"></div><div id="container" class=""><div class="content lemma-level1"><div class="detail-title" id="abstract-title"><h1>基因复制</h1><a href="https://web.archive.org/web/20221025122658/https://baike.sogou.com/kexue/d10681.htm#!" class="detail-edit">编辑</a></div><div class="section_content" data-id="54554453033935109"><div class="text_img ed_imgfloat_right"><a class="ed_image_link" data-src="https://img02.sogoucdn.com/app/a/200698/sogou_science_13741" data-bigsrc="https://img02.sogoucdn.com/app/a/200698/sogou_science_13741?width=231&amp;height=174&amp;titlename=DNA%E5%A4%8D%E5%88%B6%EF%BC%9A%E5%8F%8C%E8%9E%BA%E6%97%8B%E6%98%AF%E6%9C%AA%E5%8E%8B%E7%BC%A9%E4%B8%94%E6%9C%AA%E5%8D%B7%E7%BB%95%E7%9A%84%EF%BC%8C%E9%9A%8F%E5%90%8E%E6%AF%8F%E6%9D%A1%E5%88%86%E7%A6%BB%E7%9A%84%E9%93%BE%EF%BC%88%E9%9D%92%E7%BB%BF%E8%89%B2%EF%BC%89%E9%83%BD%E4%BD%9C%E4%B8%BA%E6%A8%A1%E6%9D%BF%EF%BC%8C%E5%A4%8D%E5%88%B6%E4%B8%80%E6%9D%A1%E6%96%B0%E7%9A%84%E4%BC%99%E4%BC%B4%E9%93%BE%EF%BC%88%E7%BB%BF%E8%89%B2%EF%BC%89%E3%80%82%E6%A0%B8%E8%8B%B7%E9%85%B8%EF%BC%88%E7%A2%B1%E5%9F%BA%EF%BC%89%E5%8C%B9%E9%85%8D%E4%BB%A5%E5%90%88%E6%88%90%E6%96%B0%E7%9A%84%E4%BC%99%E4%BC%B4%E9%93%BE%EF%BC%8C%E5%BD%A2%E6%88%90%E4%B8%A4%E4%B8%AA%E6%96%B0%E7%9A%84%E5%8F%8C%E8%9E%BA%E6%97%8B%E3%80%82&amp;w=200&amp;h=403" title="点击查看大图" data-w="200" data-h="403" style="background-image:url(https://web.archive.org/web/20221025122658im_/https://img02.sogoucdn.com/app/a/200698/sogou_science_13741)" href="https://web.archive.org/web/20221025122658/https://baike.sogou.com/kexue/d10681.htm#!"></a><div class="text_img_title">DNA复制：双螺旋是未压缩且未卷绕的，随后每条分离的链（青绿色）都作为模板，复制一条新的伙伴链（绿色）。核苷酸（碱基）匹配以合成新的伙伴链，形成两个新的双螺旋。</div></div><div><p>在分子生物学中，DNA复制是从一个原始DNA分子中产生两个完全相同的子代DNA的生物学过程。DNA复制发生在所有生物体内，作为生物遗传的基础。细胞具有独特的分裂特性，这使得DNA复制过程不是必需的。</p>
<p>DNA由两条互补的双螺旋链组成。在复制过程中，这些链被分离。然后，原始DNA分子的每条链都可作为模板，用于产生对应的DNA分子，这一过程被称为半保守复制。作为半保守复制的结果，新螺旋将由原始的DNA链和新合成的链组成。<sup><a href="https://web.archive.org/web/20221025122658/https://baike.sogou.com/kexue/d10681.htm#quote_1" class="kx_ref">[1]</a></sup> 细胞校对和错误检查机制确保了DNA复制高保真性。<sup><a href="https://web.archive.org/web/20221025122658/https://baike.sogou.com/kexue/d10681.htm#quote_2" class="kx_ref">[2]</a></sup> </p>
<p>在细胞中，DNA复制始于基因组中的特定位置或复制起点。 在复制起点解开DNA从而合成子链，由一种被称为解旋酶的酶调节，导致复制叉从起点双向生长。许多蛋白质与复制叉相关联，有助于启动和延伸DNA合成。最显著的是，DNA聚合酶通过添加核苷酸来补充每条(模板)链，从而合成新的链。DNA复制发生在分裂间期的S期。</p>
<p>DNA复制(DNA扩增)也可以在体外进行(人工地，在细胞外)。从细胞中分离的DNA聚合酶和人工DNA引物可用于在模板DNA分子中以已知序列开始DNA合成。例如，聚合酶链式反应、连接酶链式反应(LCR)和转录介导的扩增。</p></div></div><div id="catalog"><h2 class="title2">目录<a href="javascript:" class="detail-edit">编辑</a></h2><div class="catalog_wrap" style=""><ul class="catalog_list col3"><li><span class="order">1</span><a href="javascript:" data-level="1" data-id="14995458648703489">DNA结构</a></li><li><span class="order">2</span><a href="javascript:" data-level="1" data-id="14995458648703490">DNA聚合酶</a></li><li><span class="order">3</span><a href="javascript:" data-level="1" data-id="14995458665480719">复制过程</a></li><li class="secondary_catalog"><span>3.1 </span><a href="javascript:" data-id="14995458665480719">开始</a></li><li class="secondary_catalog"><span>3.2 </span><a href="javascript:" data-id="14995458665480719">延长</a></li><li class="secondary_catalog"><span>3.3 </span><a href="javascript:" data-id="14995458665480719">复制叉</a></li></ul><ul class="catalog_list col3"><li class="secondary_catalog"><span>3.4 </span><a href="javascript:" data-id="14995458665480719">DNA复制蛋白</a></li><li class="secondary_catalog"><span>3.5 </span><a href="javascript:" data-id="14995458665480719">复制机械</a></li><li class="secondary_catalog"><span>3.6 </span><a href="javascript:" data-id="14995458665480719">终止</a></li><li><span class="order">4</span><a href="javascript:" data-level="1" data-id="14995458665480720">管理</a></li><li class="secondary_catalog"><span>4.1 </span><a href="javascript:" data-id="14995458665480720">真核生物</a></li><li class="secondary_catalog"><span>4.2 </span><a href="javascript:" data-id="14995458665480720">细菌</a></li></ul><ul class="catalog_list col3"><li><span class="order">5</span><a href="javascript:" data-level="1" data-id="14995458665480721">聚合酶链反应</a></li><li><span class="order">6</span><a href="javascript:" data-level="1" data-id="14995458665480722">笔记</a></li><li><span class="order">7</span><a href="javascript:" data-level="1" data-id="references">参考文献</a></li></ul></div></div><div id="paragraphs"><div><div id="par_14995458648703489"><h2 class="title">1 DNA结构<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>DNA以双链结构存在，两条链盘绕在一起形成特有的双螺旋结构。每一股DNA都是由四种核苷酸组成的链。DNA中的核苷酸含有脱氧核糖、磷酸和碱基。这四种类型的核苷酸对应于四种核酸碱基：腺嘌呤、胞嘧啶、鸟嘌呤和胸腺嘧啶，通常缩写为A、C、G和T。腺嘌呤和鸟嘌呤是嘌呤碱基，而胞嘧啶和胸腺嘧啶是嘧啶。这些核苷酸形成磷酸二酯键，形成核酸双螺旋的磷酸-脱氧核糖骨架，核酸碱基指向内部(即，朝向相反的链)。碱基通过氢键在链之间互补配对形成碱基对。腺嘌呤与胸腺嘧啶通过两个氢键互补配对，鸟嘌呤与胞嘧啶通过三个氢键互补配对。</p><p>DNA链具有方向性，单链的不同末端被称为“3′末端”和“5′末端”。按照惯例，如果给出单条DNA链的碱基序列，序列的左端是5’端，而右端是3’端。双螺旋的链是反向平行的，一条是5’到3’，另一条是3’到5’。这些术语指脱氧核糖中的碳原子，链中的下一个磷酸连接到该碳原子上。方向性对DNA合成有影响，因为DNA聚合酶只能从DNA链的5’到3’添加核苷酸从而合成DNA。</p><p>DNA中互补碱基的配对(通过氢键)意味着每条链中包含的信息是多余的。磷酸二酯(链内)键比氢(链间)键更强，这允许双链之间彼此分离。因此，单链上的核苷酸可用于在新合成的配对链上重建核苷酸。<sup><a href="https://web.archive.org/web/20221025122658/https://baike.sogou.com/kexue/d10681.htm#quote_3" class="kx_ref">[3]</a></sup> </p></div></div><div id="par_14995458648703490"><h2 class="title">2 DNA聚合酶<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p></p><div class="text_img ed_imgfloat_right">
            <a class="ed_image_link lazyLoad" data-src="https://img02.sogoucdn.com/app/a/200698/sogou_science_13743" data-bigsrc="" title="点击查看大图" href="javascript:" data-observer="true"></a>
            <div class="text_img_title">DNA聚合酶将核苷酸添加到DNA链的3 '端。[1] 如果不小心引入错配，聚合酶就会受到抑制，无法进一步扩展。校对删除不匹配的核苷酸，扩展就会继续进行。</div>   
        </div> <p></p><p>DNA聚合酶是一个进行各种形式的DNA复制的酶家族。<sup><a href="https://web.archive.org/web/20221025122658/https://baike.sogou.com/kexue/d10681.htm#quote_4" class="kx_ref">[4]</a></sup> 一般来说，DNA聚合酶不能引发新链的合成，但只能延伸与模板链配对的现有DNA或核糖核酸链。要开始合成，必须合成一个短的核糖核酸片段，称为引物，并与模板DNA链配对。</p><p>DNA聚合酶通过延伸现有核苷酸链的3’端来添加新的DNA链，通过磷酸二酯键的产生一次添加一个与模板链匹配的新核苷酸。这个DNA聚合过程的能量来自于附着在每个未掺入碱基上的三个磷酸盐之间的高能磷酸盐(磷酸酐)键的水解。带有磷酸基团的游离碱基称为核苷酸；特别是，带有三个磷酸基团的碱基被称为核苷三磷酸。当核苷酸被添加到生长的DNA链中时，核苷酸的近端磷酸酯与生长链之间磷酸二酯键的形成伴随着高能磷酸酯键的水解以及两个远端磷酸基团作为焦磷酸盐的释放。将所得焦磷酸酶解成无机磷酸盐消耗第二个高能磷酸键，并使反应有效地不可逆。</p><p>首先测量活细胞中的基因复制速率，即噬菌体T4DNA在噬菌体感染的大肠杆菌中的延伸速率。<sup><a href="https://web.archive.org/web/20221025122658/https://baike.sogou.com/kexue/d10681.htm#quote_5" class="kx_ref">[5]</a></sup> 此外，一些DNA聚合酶也具有校对能力；他们可以去除生长链末端的核苷酸，以纠正错配的碱基。最后，复制后错配修复机制监测DNA的错误，能够区分新合成的DNA链和模板链序列中的错配。总之，这三个鉴别步骤使得每添加109个核苷酸，复制保真度不到一个错误。<sup><a href="https://web.archive.org/web/20221025122658/https://baike.sogou.com/kexue/d10681.htm#quote_5" class="kx_ref">[5]</a></sup> </p><p>在活细胞中的基因复制速率首先被测量为噬菌体T4基因在噬菌体感染中的延伸速率 <i>E.大肠杆菌</i>。<sup><a href="https://web.archive.org/web/20221025122658/https://baike.sogou.com/kexue/d10681.htm#quote_6" class="kx_ref">[6]</a></sup> 在37℃下，DNA呈指数增长期间，速率为每秒749个核苷酸。噬菌体T4DNA合成过程中每复制一个碱基对的突变率为1.7/108。<sup><a href="https://web.archive.org/web/20221025122658/https://baike.sogou.com/kexue/d10681.htm#quote_7" class="kx_ref">[7]</a></sup> </p></div></div><div id="par_14995458665480719"><h2 class="title">3 复制过程<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>像所有生物聚合过程一样，DNA复制在三个酶催化和协调的步骤中进行:起始、延伸和终止。</p><p></p><h3>3.1 <span>开始</span></h3><p></p><p></p><div class="text_img ed_imgfloat_right">
            <a class="ed_image_link lazyLoad" data-src="https://img03.sogoucdn.com/app/a/200698/sogou_science_13745" data-bigsrc="" title="点击查看大图" href="javascript:" data-observer="true"></a>
            <div class="text_img_title" style="text-align: center;">发起DNA复制的引发剂的作用。</div>   
        </div> <p></p><p></p><div class="text_img ed_imgfloat_right">
            <a class="ed_image_link lazyLoad" data-src="https://img02.sogoucdn.com/app/a/200698/sogou_science_13746" data-bigsrc="" title="点击查看大图" href="javascript:" data-observer="true"></a>
            <div class="text_img_title" style="text-align: center;">复制前复合物的形成。</div>   
        </div> <p></p><p>细胞要分裂，首先必须复制它的DNA。<sup><a href="https://web.archive.org/web/20221025122658/https://baike.sogou.com/kexue/d10681.htm#quote_8" class="kx_ref">[8]</a></sup> 这一过程是在被称为“起源”的DNA中的特定点开始的，这些点与起始蛋白靶向结合。<sup><a href="https://web.archive.org/web/20221025122658/https://baike.sogou.com/kexue/d10681.htm#quote_9" class="kx_ref">[9]</a></sup> 在大肠杆菌中，这种蛋白质是DnaA；在酵母中，这是起源识别复合体。<sup><a href="https://web.archive.org/web/20221025122658/https://baike.sogou.com/kexue/d10681.htm#quote_9" class="kx_ref">[9]</a></sup> 引发蛋白结合的DNA序列往往“富含腺苷酸和胸腺嘧啶碱基”，因为A-T有两个氢键(而不是三个形成的C-G)，因此双链更容易分离。<sup><a href="https://web.archive.org/web/20221025122658/https://baike.sogou.com/kexue/d10681.htm#quote_10" class="kx_ref">[10]</a></sup> 一旦找到了起点，这些起始蛋白就会招募其他蛋白质，形成复制前复合体，解开双链DNA。</p><p></p><h3>3.2 <span>延长</span></h3><p></p><p>DNA聚合酶具有5′–3′活性。在开始合成之前，所有已知的DNA复制系统都需要一个游离的3’羟基(注意:DNA模板是在3’到5’方向上读取的，而新的链是在5’到3’方向上合成的——（这一点经常会被混淆）。存在四种不同的DNA合成机制:</p><p>
 </p><ol>
  <li>所有细胞生命形式和许多DNA病毒、噬菌体和质粒都使用一种引物酶来合成一种带有游离3’羟基的短RNA引物，该引物随后被DNA聚合酶延长。</li>
  <li>反转录因子(包括反转录病毒)采用tRNA，通过提供一种被反转录酶延长的游离3’羟基来启动DNA复制。</li>
  <li>在腺病毒和φ29噬菌体家族中，3’羟基由基因组连接蛋白(末端蛋白)的氨基酸侧链提供，核苷酸通过DNA聚合酶添加到该蛋白上形成新的链。</li>
  <li>在单链DNA病毒——包括圆环病毒、双生病毒、细小病毒和其他病毒——以及许多使用滚环复制(RCR)机制的噬菌体和质粒中，RCR核酸内切酶在基因组链(单链病毒)或其中一条DNA链(质粒)上产生缺口。缺口链的5’端转移到核酸酶上的酪氨酸残基，然后游离的3’羟基被DNA聚合酶用来合成新链。</li>
 </ol><p></p><p>第一种是这些机制中最广为人知的，被细胞生物体所使用。在这种机制中，一旦两条链被分离，引物酶将RNA引物添加到模板链中。前导链接收一个RNA引物，而滞后链接收多个RNA引物。前导链通过高加工性的DNA聚合酶从引物中连续延伸，而滞后链从形成冈崎片段的每个引物中不连续延伸。核糖核酸酶去除引物核糖核酸片段，一个不同于复制聚合酶的低加工性DNA聚合酶进入以填补缺口。当该过程完成后，可以在前导链上找到一个缺口，在滞后链上找到几个缺口。连接酶可以填补这些缺口，从而完成新复制的DNA分子。</p><p>细菌和古细菌/真核生物在这一过程中使用的引物酶有显著差异。细菌使用一种属于DnaG蛋白超家族的引物酶，它含有TOPRIM折叠型催化结构域。<sup><a href="https://web.archive.org/web/20221025122658/https://baike.sogou.com/kexue/d10681.htm#quote_11" class="kx_ref">[11]</a></sup> TOPRIM折叠型催化结构域包含一个α/β核，在Rossmann样拓扑中有四个保守链。这种结构也存在于拓扑异构酶Ia、拓扑异构酶II、OLD家族核酸酶和与RecR蛋白相关的DNA修复蛋白的催化结构域中。</p><p>相比之下，古细菌和真核生物使用的引物酶含有高度衍生的RNA识别基序(RRM)。这种引物酶在结构上类似于许多病毒核糖核酸依赖性核糖核酸聚合酶、逆转录酶、环核苷酸生成环化酶和参与DNA复制和修复的A/B/Y家族DNA聚合酶。在真核复制中，引物酶与Pol α形成复合物。<sup><a href="https://web.archive.org/web/20221025122658/https://baike.sogou.com/kexue/d10681.htm#quote_12" class="kx_ref">[12]</a></sup> </p><p>多种DNA聚合酶在DNA复制过程中扮演不同的角色。在大肠杆菌中，DNAPol III是主要负责DNA复制的聚合酶。它在复制叉处组装成一个复制复合体，表现出极高的可加工性，在整个复制周期中保持不变。相反，DNAPol 1是负责用DNA取代核糖核酸引物的酶。DNAPol除了聚合酶活性外，还具有5’至3’的核酸外切酶活性，并利用其核酸外切酶活性降解其前面的核糖核酸引物，同时延伸其后面的DNA链，这一过程称为缺口翻译。Pol I比Pol III的加工性差得多，因为它在DNA复制中的主要功能是创造许多短的DNA区域，而不是几个很长的区域。</p><p>在真核生物中，低加工性酶Pol α有助于启动复制，因为它与引物酶形成复合物。<sup><a href="https://web.archive.org/web/20221025122658/https://baike.sogou.com/kexue/d10681.htm#quote_13" class="kx_ref">[13]</a></sup>在真核生物中，主导链被认为是由Pol ε指导合成；然而，这一观点最近受到了挑战，表明了Pol δ的作用。<sup><a href="https://web.archive.org/web/20221025122658/https://baike.sogou.com/kexue/d10681.htm#quote_14" class="kx_ref">[14]</a></sup> 引物去除完成Pol δ<sup><a href="https://web.archive.org/web/20221025122658/https://baike.sogou.com/kexue/d10681.htm#quote_15" class="kx_ref">[15]</a></sup>，而复制过程中的DNA修复由Pol ε完成。</p><p>随着DNA合成的继续，原始DNA链继续在气泡的每一侧展开，形成一个有两个叉的复制叉。在圆形染色体上有单一复制起点的细菌中，这一过程产生了“θ结构”(类似希腊字母θ)。相比之下，真核生物具有更长的线性染色体，并在这些染色体中的多个起点开始复制。<sup><a href="https://web.archive.org/web/20221025122658/https://baike.sogou.com/kexue/d10681.htm#quote_16" class="kx_ref">[16]</a></sup> </p><p></p><h3>3.3 <span>复制叉</span></h3><p></p><p></p><div class="text_img ed_imgfloat_right">
            <a class="ed_image_link lazyLoad" data-src="https://img04.sogoucdn.com/app/a/200698/sogou_science_13748" data-bigsrc="" title="点击查看大图" href="javascript:" data-observer="true"></a>
            <div class="text_img_title">复制叉方案。 a： 模板，b： 前导链，c： 后随链，d： 复制叉，e： 引物，f： 冈崎片段</div>   
        </div> <p></p><p>复制叉是在DNA复制过程中在长螺旋DNA内形成的一种结构。它是由解旋酶产生的，解旋酶打破了将两条DNA链连接在一起的氢键。由此产生的结构有两个分支“叉”，每个叉由一股DNA组成。这两条链作为前导链和滞后链的模板，当DNA聚合酶将互补核苷酸与模板匹配时，就会产生前导链和滞后链；模板可以恰当地称为前导链模板和滞后链模板。</p><p>DNA总是在5’到3’方向合成。由于前导链和滞后链模板在复制叉上方向相反，一个主要问题是如何实现新生(新)滞后链DNA的合成，其合成方向与生长的复制叉的方向相反。</p><p><strong>前导链</strong></p><p>主导链是新生的DNA链，其合成方向与复制叉延伸方向相同。这种基因复制是连续的。。</p><p><strong>后随链</strong></p><p>滞后链是新生的DNA链，其合成方向与复制叉的延伸方向相反。由于滞后链的定向，其复制过程比前导链的复制更复杂。因此，这条链上的DNA聚合酶被认为“落后”于另一条链。</p><p>滞后链是在短的、分离的片段中合成的。在滞后链模板上，引物酶“读取”模板的DNA，并开始合成一个短的互补核糖核酸引物。DNA聚合酶延伸引物片段，形成冈崎片段。然后移除核糖核酸引物并用DNA替换，DNA片段通过DNA连接酶连接在一起。</p><p><strong>复制叉处的动态变化</strong></p><p></p><div class="text_img ed_imgfloat_right">
            <a class="ed_image_link lazyLoad" data-src="https://img04.sogoucdn.com/app/a/200698/sogou_science_13750" data-bigsrc="" title="点击查看大图" href="javascript:" data-observer="true"></a>
            <div class="text_img_title">组装的人类DNA钳，它是PCNA蛋白的三聚体。</div>   
        </div> <p></p><p>在所有情况下，解旋酶都是由六种多肽组成的，它们只包绕一条被复制的DNA链。这两种聚合酶结合到解旋酶六聚体上。在真核生物中，螺旋酶缠绕在前导链上，而在原核生物中，解旋酶缠绕在滞后链上。<sup><a href="https://web.archive.org/web/20221025122658/https://baike.sogou.com/kexue/d10681.htm#quote_17" class="kx_ref">[17]</a></sup> </p><p>当解旋酶在复制叉上解开DNA时，前方的DNA被迫旋转。这个过程导致前方的DNA扭曲堆积。<sup><a href="https://web.archive.org/web/20221025122658/https://baike.sogou.com/kexue/d10681.htm#quote_18" class="kx_ref">[18]</a></sup>这种积聚形成了扭转阻力，最终会阻止复制叉的前进。拓扑异构酶是暂时断裂DNA链的酶，缓解了由于解开DNA螺旋的两条链而引起的张力；拓扑异构酶(包括DNA旋转酶)通过在DNA螺旋中添加负的超螺旋来实现这一点。<sup><a href="https://web.archive.org/web/20221025122658/https://baike.sogou.com/kexue/d10681.htm#quote_19" class="kx_ref">[19]</a></sup> </p><p>裸露的单链DNA往往会折回，形成二级结构；这些结构会干扰DNA聚合酶的运动。为了防止这种情况，单链结合蛋白与DNA结合，直到合成第二链，防止二级结构形成。<sup><a href="https://web.archive.org/web/20221025122658/https://baike.sogou.com/kexue/d10681.htm#quote_20" class="kx_ref">[20]</a></sup> </p><p>钳蛋白质在DNA周围形成滑动夹钳，帮助DNA聚合酶与其模板保持接触，从而有助于加工。夹子的内表面使DNA能够穿过它。一旦聚合酶到达模板末端或检测到双链DNA，滑动夹就会发生构象变化，释放DNA聚合酶。夹钳装载蛋白用于最初装载夹钳，识别模板和核糖核酸引物之间的连接。<sup><a href="https://web.archive.org/web/20221025122658/https://baike.sogou.com/kexue/d10681.htm#quote_21" class="kx_ref">[21]</a></sup><sup>：274-5</sup> </p><p></p><h3>3.4 <span>DNA复制蛋白</span></h3><p></p><p>在复制叉上，许多复制酶在DNA上组装成一个复杂的分子机器，称为复制体。以下是参与复制子体的主要DNA复制酶的列表:<sup><a href="https://web.archive.org/web/20221025122658/https://baike.sogou.com/kexue/d10681.htm#quote_21" class="kx_ref">[21]</a></sup> </p><p>
 </p><table class="wikitable">
  <tbody>
   <tr>
    <th>酶</th>
    <th>在DNA复制中的作用</th>
   </tr>
   <tr>
    <td>DNA解旋酶</td>
    <td>也被称为螺旋去稳定酶。螺旋酶在拓扑异构酶后面的复制叉上分离出两条DNA链。</td>
   </tr>
   <tr>
    <td>DNA聚合酶</td>
    <td>在DNA复制过程中，负责在5’到3’方向催化核苷酸底物添加到DNA的酶。还执行校对和纠错。存在许多不同类型的DNA聚合酶，每种聚合酶在不同类型的细胞中执行不同的功能。</td>
   </tr>
   <tr>
    <td>DNA钳</td>
    <td>一种防止延伸的DNA聚合酶从DNA母链上解离的蛋白质。</td>
   </tr>
   <tr>
    <td>单链结合蛋白</td>
    <td>与ssDNA结合，防止DNA双螺旋在DNA解旋后重新退火，从而保持链分离，促进新生链的合成。</td>
   </tr>
   <tr>
    <td>拓扑异构酶</td>
    <td>放松DNA的超螺旋性质。</td>
   </tr>
   <tr>
    <td>DNA旋转酶</td>
    <td>通过解旋酶释放解旋张力；这是一种特殊类型的拓扑异构酶</td>
   </tr>
   <tr>
    <td>DNA连接酶</td>
    <td>重新退火半保守链，并加入滞后链的冈崎片段。</td>
   </tr>
   <tr>
    <td>引物酶</td>
    <td>为DNA聚合酶开始合成新的DNA链提供了核糖核酸(或DNA)的起点。</td>
   </tr>
   <tr>
    <td>端粒酶</td>
    <td>通过在真核生物染色体末端添加重复的核苷酸序列来延长端粒。这允许生殖细胞和干细胞避免Hayflick界限对细胞分裂的影响。<sup><a href="https://web.archive.org/web/20221025122658/https://baike.sogou.com/kexue/d10681.htm#quote_22" class="kx_ref">[22]</a></sup> </td>
   </tr>
  </tbody>
 </table><p></p><p></p><h3>3.5 <span>复制机械</span></h3><p></p><p>复制机制由参与DNA复制并出现在模板ssDNA上的因素组成。复制机制包括原体是复制酶；DNA聚合酶、DNA螺旋酶、DNA钳和DNA拓扑异构酶以及复制蛋白；例如单链DNA结合蛋白。在复制机制中，这些组件相互协调。在大多数细菌中，所有参与复制的因素都位于复制叉上，复合物在复制过程中停留在叉上。这些复制机制被称为复制体或DNA复制系统。这些术语是位于复制叉上的蛋白质的通用术语。在真核细胞和一些细菌细胞中，不形成复制体。</p><p>由于复制机制不会相对于模板DNA（如工厂）移动，因此它们被称为复制工厂。<sup><a href="https://web.archive.org/web/20221025122658/https://baike.sogou.com/kexue/d10681.htm#quote_23" class="kx_ref">[23]</a></sup> 在另一个数字中，DNA工厂类似于放映机，DNA就像电影胶片一样不断进入放映机。在复制工厂模型中，在前导链和滞后链的DNA解旋酶都装载到模板DNA上之后，解旋酶沿着DNA链相互运行。解旋酶与复制过程的剩余部分保持关联。彼得·梅斯特(Peter Meister)等人通过监测绿色荧光蛋白标记的DNA聚合酶α，直接观察出芽酵母中的复制位点。他们检测到与复制起点对称隔开的标记基因座对的DNA复制，并发现成对基因座之间的距离随着时间的推移显著减小。 这一发现表明，基因复制的机制与基因工厂有关。也就是说，复制工厂加载在复制源和彼此关联的工厂上。此外，模板DNA进入工厂，导致模板ssDNA和新生DNA的挤压。梅斯特的发现首次证明了复制工厂模式。随后的研究表明，在许多真核细胞中，DNA解旋酶形成二聚体，细菌复制机制在DNA合成过程中停留在单个核内位置。<sup><a href="https://web.archive.org/web/20221025122658/https://baike.sogou.com/kexue/d10681.htm#quote_23" class="kx_ref">[23]</a></sup> </p><p>复制工厂完成解开姐妹染色单体。姐妹染色单体的解开对于在DNA复制后将染色质分布到子细胞中是必不可少的。因为在DNA复制后姐妹染色单体通过黏连蛋白环相互结合，所以在DNA复制中只有一个解开缠结的机会。将复制机器固定为复制工厂可以提高DNA复制的成功率。如果复制叉在染色体中自由移动，细胞核的连锁会加剧并阻碍有丝分裂分离。 </p><p></p><h3>3.6 <span>终止</span></h3><p></p><p>真核生物在染色体的多个点启动DNA复制，因此复制叉在染色体的许多点相遇并终止。因为真核生物有线性染色体，所以基因复制无法到达染色体的最末端。由于这个问题，在从染色体末端开始的每个复制周期中，DNA都会丢失。端粒是靠近末端的重复DNA区域，有助于防止基因因这种缩短而丢失。端粒缩短是体细胞的正常过程。这缩短了子代DNA染色体的端粒。因此，细胞DNA在失去阻止进一步分裂之前，只能分裂一定次数。（这就是所谓的Hayflick界限。）在将DNA传递给下一代的生殖细胞系中，端粒酶延伸端粒区域的重复序列以防止降解。端粒酶可能在体细胞中异常活跃，有时会导致癌症的形成。端粒酶活性增加是癌症的标志之一。</p><p>终止要求DNA复制叉的进程必须停止或被阻止。当在特定位点发生终止时，涉及两个组分之间的相互作用:(1)在DNA中的终止位点序列，和(2)结合到该序列上以物理停止DNA复制的蛋白质。在各种细菌中，这被称为DNA复制末端位点结合蛋白，或称Ter蛋白。</p><p>因为细菌有环状染色体，当两个复制叉在亲代染色体的另一端相遇时，此时复制终止。大肠杆菌通过使用终止序列来调节这一过程，当终止序列与Tus蛋白结合时，只允许复制叉的一个方向通过。因此，复制叉被限制为总是在染色体的终止区域内相遇。<sup><a href="https://web.archive.org/web/20221025122658/https://baike.sogou.com/kexue/d10681.htm#quote_24" class="kx_ref">[24]</a></sup> </p></div></div><div id="par_14995458665480720"><h2 class="title">4 管理<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p></p><div class="text_img ed_imgfloat_embedmiddle">
            <a class="ed_image_link" data-src="https://img01.sogoucdn.com/app/a/200698/sogou_science_13751" data-bigsrc="" title="点击查看大图" href="javascript:">
                <img alt="" class="ed_imgfloat_embedmiddle" img_height="156" img_width="150" titlename="真核细胞的细胞周期。" data-align="embedmiddle" width="150" data-src="https://img01.sogoucdn.com/app/a/200698/sogou_science_13751" src="./681.基因复制 - 搜狗科学百科_files/sogou_science_13751" data-observer="true">
            </a>
            <div class="text_img_title" style="text-align: center;">真核细胞的细胞周期。</div>
        </div> <p></p><p></p><h3>4.1 <span>真核生物</span></h3><p></p><p>在真核生物中，DNA的复制受细胞周期调控。随着细胞的生长和分裂，它在细胞周期的各个阶段进行；DNA复制发生在S期（合成期）。真核细胞整个周期进程由细胞周期检查点来控制。通过检查点的进展是通过各种蛋白质之间复杂的相互作用来控制的，包括细胞周期蛋白和细胞周期蛋白依赖性激酶。<sup><a href="https://web.archive.org/web/20221025122658/https://baike.sogou.com/kexue/d10681.htm#quote_25" class="kx_ref">[25]</a></sup> 与细菌不同，真核生物的DNA在细胞核内复制。<sup><a href="https://web.archive.org/web/20221025122658/https://baike.sogou.com/kexue/d10681.htm#quote_26" class="kx_ref">[26]</a></sup> </p><p>G1/S期检查点(或限制性检查点)调节真核细胞是否进入DNA复制和随后分裂的过程。没有通过这个检查点的细胞仍然处于G0期，并且不复制它们的DNA。</p><p>叶绿体和线粒体基因组的复制通过D环复制过程独立于细胞周期发生。</p><p><strong>复制焦点</strong></p><p>在脊椎动物细胞中，复制位点集中在被称为复制焦点的位置。<sup><a href="https://web.archive.org/web/20221025122658/https://baike.sogou.com/kexue/d10681.htm#quote_27" class="kx_ref">[27]</a></sup> 复制位点可以通过子链免疫染色和复制酶以及监测绿色荧光蛋白标记的复制因子来检测。通过这些方法发现，不同大小和位置的复制焦点出现在细胞分裂的S期，每个细胞核的复制焦点数目远小于基因组复制叉的数目。</p><p>P.赫恩等人(2001年)跟踪了芽殖酵母细胞中绿色荧光蛋白标记的复制焦点，并发现了复制起点在G1期和S期不断移动，在S期活力显著下降。<sup><a href="https://web.archive.org/web/20221025122658/https://baike.sogou.com/kexue/d10681.htm#quote_27" class="kx_ref">[27]</a></sup> 传统上，复制位点由核基质或核纤层蛋白固定在染色体的空间结构上。赫恩的结果否定了传统概念，出芽酵母不含核纤层蛋白，并支持复制起源自组装并形成复制焦点。</p><p>通过激发复制起点，在空间和时间上进行控制，调控复制焦点的形成。杰克逊等人(1998年)揭示了哺乳动物细胞中相邻的起源同时启动。<sup><a href="https://web.archive.org/web/20221025122658/https://baike.sogou.com/kexue/d10681.htm#quote_27" class="kx_ref">[27]</a></sup> 复制站点的空间并列带来了复制分叉的群集。群集确实挽救了停滞的复制叉，并有利于复制叉的正常进展。复制分叉的进展受到许多因素的抑制；与蛋白质或与强结合于DNA的复合物的碰撞、dNTPs的缺乏、模板DNA上的缺口等。如果复制叉停止，并且停止的叉中的剩余序列没有被复制，子链将获得未复制的位点。母链上的未复制位点将另一条链连接在一起，而不是子链。因此，产生的姐妹染色单体不能彼此分离，也不能分裂成两个子细胞。当相邻原点启动并且一个原点的复制叉停止时，另一个原点的复制叉从停止复制叉的相反方向进入并复制未复制的位点。作为另一种拯救机制，有休眠复制起源的应用，即多余的起点不会在正常的DNA复制中触发。</p><p></p><h3>4.2 <span>细菌</span></h3><p></p><p></p><div class="text_img ed_imgfloat_right">
            <a class="ed_image_link lazyLoad" data-src="https://img02.sogoucdn.com/app/a/200698/sogou_science_13752" data-bigsrc="" title="点击查看大图" href="javascript:" data-observer="true"></a>
            <div class="text_img_title">复制后的GATC位点的Dam甲基化腺嘌呤。</div>   
        </div> <p></p><p>大多数细菌并不经历一个明确的细胞周期，而是不断复制它们的DNA；在快速增长期间，这可能会导致多轮复制同时发生。<sup><a href="https://web.archive.org/web/20221025122658/https://baike.sogou.com/kexue/d10681.htm#quote_27" class="kx_ref">[27]</a></sup> 在大肠杆菌这种最具特征的细菌中，DNA的复制是通过几种机制来调节的，包括:起始序列的半甲基化和隔离，三磷酸腺苷与二磷酸腺苷的比例，以及DnaA蛋白的水平。所有这些控制了起始蛋白与起始序列的结合。</p><p>因为大肠杆菌甲基化GATC基因序列，所以基因合成导致半甲基化序列。这种半甲基化的DNA被SeqA蛋白识别，SEqa蛋白结合并隔离原始序列；此外，DnaA(启动复制所需的)与半甲基化DNA结合较差。因此，新复制的起点被阻止立即开始另一轮的DNA复制。<sup><a href="https://web.archive.org/web/20221025122658/https://baike.sogou.com/kexue/d10681.htm#quote_28" class="kx_ref">[28]</a></sup> </p><p>当细胞处于丰富的培养基中时，三磷酸腺苷就会积累起来，一旦细胞达到特定的大小，就会触发DNA的复制。三磷酸腺苷与腺苷二磷酸竞争结合DnaA，DnaA-三磷酸腺苷复合体能够启动复制。DNA复制也需要一定数量的DnaA——每次复制起点，DnaA的结合位点数量就会翻倍，需要合成更多的DnaA才能再次启动复制。</p><p>在快速生长的细菌中，如大肠杆菌，染色体复制比分裂细胞需要更多的时间。细菌通过在前一轮复制被终止之前启动新一轮复制来解决这个问题。<sup><a href="https://web.archive.org/web/20221025122658/https://baike.sogou.com/kexue/d10681.htm#quote_29" class="kx_ref">[29]</a></sup> 新一轮复制将形成分裂细胞后两代新生的细胞染色体。这种机制会产生复制周期的重叠。</p></div></div><div id="par_14995458665480721"><h2 class="title">5 聚合酶链反应<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>研究人员通常使用聚合酶链反应（PCR）在体外复制DNA。聚合酶链反应使用一对引物跨越模板DNA中的特定区域，然后使用热稳定的DNA聚合酶从这些引物向各个方向聚合配对链。通过多次循环重复这一过程可以扩增目标DNA区域。在每个循环开始时，加热模板和引物的混合物，分离新合成的分子和模板。然后，随着混合物冷却，这两者都成为新引物退火的模板，聚合酶从这些模板延伸出来。因此，目标区域的拷贝数每轮翻一倍，呈指数级增长。<sup><a href="https://web.archive.org/web/20221025122658/https://baike.sogou.com/kexue/d10681.htm#quote_30" class="kx_ref">[30]</a></sup> </p></div></div><div id="par_14995458665480722"><h2 class="title">6 笔记<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"></div></div></div></div><div id="references"><h2 class="title" id="par_references">参考文献</h2><ul class="references"><li id="quote_1"><span class="references-num">[1]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Pray, Leslie A. "Semi-Conservative DNA Replication; Meselson and Stahl"..</span></p></li><li id="quote_2"><span class="references-num">[2]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Imperfect DNA replication results in mutations. Berg JM, Tymoczko JL, Stryer L, Clarke ND (2002). "Chapter 27: DNA Replication, Recombination, and Repair". Biochemistry. W.H. Freeman and Company. ISBN 0-7167-3051-0.[失效连结].</span></p></li><li id="quote_3"><span class="references-num">[3]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Alberts, B., et al., Molecular Biology of the Cell, Garland Science, 4th ed., 2002, pp. 238–240 ISBN 0-8153-3218-1.</span></p></li><li id="quote_4"><span class="references-num">[4]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Berg JM, Tymoczko JL, Stryer L, Clarke ND (2002). Biochemistry. W.H. Freeman and Company. ISBN 0-7167-3051-0. Chapter 27, Section 2: DNA Polymerases Require a Template and a Primer.</span></p></li><li id="quote_5"><span class="references-num">[5]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">McCulloch, Scott D; Kunkel, Thomas A (January 2008). "The fidelity of DNA synthesis by eukaryotic replicative and translesion synthesis polymerases". Cell Research. 18 (1): 148–161. doi:10.1038/cr.2008.4. PMC 3639319. PMID 18166979..</span></p></li><li id="quote_6"><span class="references-num">[6]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">McCarthy, David; Minner, Charles; Bernstein, Harris; Bernstein, Carol (October 1976). "DNA elongation rates and growing point distributions of wild-type phage T4 and a DNA-delay amber mutant". Journal of Molecular Biology. 106 (4): 963–981. doi:10.1016/0022-2836(76)90346-6. PMID 789903..</span></p></li><li id="quote_7"><span class="references-num">[7]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Drake JW (1970) The Molecular Basis of Mutation. Holden-Day, San Francisco ISBN 0816224501 ISBN 978-0816224500.</span></p></li><li id="quote_8"><span class="references-num">[8]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Alberts B, Johnson A, Lewis J, Raff M, Roberts K, Walter P (2002). Molecular Biology of the Cell. Garland Science. ISBN 0-8153-3218-1. Chapter 5: DNA Replication Mechanisms.</span></p></li><li id="quote_9"><span class="references-num">[9]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Berg JM, Tymoczko JL, Stryer L, Clarke ND (2002). "Chapter 27, Section 4: DNA Replication of Both Strands Proceeds Rapidly from Specific Start Sites". Biochemistry. W.H. Freeman and Company. ISBN 0-7167-3051-0.[失效连结].</span></p></li><li id="quote_10"><span class="references-num">[10]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Lodish H, Berk A, Zipursky LS, Matsudaira P, Baltimore D, Darnell J (2000). Molecular Cell Biology. W. H. Freeman and Company. ISBN 0-7167-3136-3.12.1. General Features of Chromosomal Replication: Three Common Features of Replication Origins.</span></p></li><li id="quote_11"><span class="references-num">[11]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Aravind, L.; Leipe, D. D.; Koonin, E. V. (1998). "Toprim—a conserved catalytic domain in type IA and II topoisomerases, DnaG-type primases, OLD family nucleases and RecR proteins". Nucleic Acids Research. 26 (18): 4205–4213. doi:10.1093/nar/26.18.4205. PMC 147817. PMID 9722641..</span></p></li><li id="quote_12"><span class="references-num">[12]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Frick, David; Richardson, Charles (July 2001). "DNA Primases". Annual Review of Biochemistry. 70: 39–80. doi:10.1146/annurev.biochem.70.1.39. PMID 11395402..</span></p></li><li id="quote_13"><span class="references-num">[13]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Barry, Elizabeth R.; Bell, Stephen D. (8 December 2006). "DNA Replication in the Archaea". Microbiology and Molecular Biology Reviews. 70 (4): 876–887. doi:10.1128/MMBR.00029-06. PMC 1698513. PMID 17158702..</span></p></li><li id="quote_14"><span class="references-num">[14]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Stillman, Bruce (July 2015). "Reconsidering DNA Polymerases at the Replication Fork in Eukaryotes". Molecular Cell. 59 (2): 139–141. doi:10.1016/j.molcel.2015.07.004. PMC 4636199. PMID 26186286..</span></p></li><li id="quote_15"><span class="references-num">[15]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Distinguishing the pathways of primer removal during Eukaryotic Okazaki fragment maturation Contributor Author Rossi, Marie Louise. Date Accessioned: 2009-02-23T17:05:09Z. Date Available: 2009-02-23T17:05:09Z. Date Issued: 2009-02-23T17:05:09Z. Identifier Uri: http://hdl.handle.net/1802/6537. Description: Dr. Robert A. Bambara, Faculty Advisor. Thesis (PhD) – School of Medicine and Dentistry, University of Rochester. UR only until January 2010. UR only until January 2010..</span></p></li><li id="quote_16"><span class="references-num">[16]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Huberman, Joel A.; Riggs, Arthur D. (March 1968). "On the mechanism of DNA replication in mammalian chromosomes". Journal of Molecular Biology. 32 (2): 327–341. doi:10.1016/0022-2836(68)90013-2. PMID 5689363..</span></p></li><li id="quote_17"><span class="references-num">[17]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Gao, Yang; et al. (22 February 2019). "Structures and operating principles of the replisome". Science. 363 (6429): 835. doi:10.1126/science.aav7003..</span></p></li><li id="quote_18"><span class="references-num">[18]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Alberts B, Johnson A, Lewis J, Raff M, Roberts K, Walter P (2002). Molecular Biology of the Cell. Garland Science. ISBN 0-8153-3218-1. DNA Replication Mechanisms: DNA Topoisomerases Prevent DNA Tangling During Replication.</span></p></li><li id="quote_19"><span class="references-num">[19]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Reece, Richard J.; Maxwell, Anthony; Wang, James C. (26 September 2008). "DNA Gyrase: Structure and Function". Critical Reviews in Biochemistry and Molecular Biology. 26 (3–4): 335–375. doi:10.3109/10409239109114072. PMID 1657531..</span></p></li><li id="quote_20"><span class="references-num">[20]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Alberts B, Johnson A, Lewis J, Raff M, Roberts K, Walter P (2002). Molecular Biology of the Cell. Garland Science. ISBN 0-8153-3218-1. DNA Replication Mechanisms: Special Proteins Help to Open Up the DNA Double Helix in Front of the Replication Fork.</span></p></li><li id="quote_21"><span class="references-num">[21]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Alberts B, Johnson A, Lewis J, Raff M, Roberts K, Walter P (2002). "5DNA Replication, Repair, and Recombination". Molecular Biology of the Cell. Garland Science. ISBN 0-8153-3218-1.[失效连结].</span></p></li><li id="quote_22"><span class="references-num">[22]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Will the Hayflick limit keep us from living forever?". Howstuffworks. Retrieved January 20, 2015..</span></p></li><li id="quote_23"><span class="references-num">[23]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">James D. Watson et al. (2008), "Molecular Biology of the gene", Pearson Education: 237.</span></p></li><li id="quote_24"><span class="references-num">[24]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">TA Brown (2002). Genomes. BIOS Scientific Publishers. ISBN 1-85996-228-9.13.2.3. Termination of replication.</span></p></li><li id="quote_25"><span class="references-num">[25]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Alberts B, Johnson A, Lewis J, Raff M, Roberts K, Walter P (2002). Molecular Biology of the Cell. Garland Science. ISBN 0-8153-3218-1. Intracellular Control of Cell-Cycle Events: S-Phase Cyclin-Cdk Complexes (S-Cdks) Initiate DNA Replication Once Per Cycle.</span></p></li><li id="quote_26"><span class="references-num">[26]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Brown, TA (2002). "13". Genomes (2nd ed.). Oxford: Wiley-Liss..</span></p></li><li id="quote_27"><span class="references-num">[27]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Peter Meister, Angela Taddei1, Susan M. Gasser(June 2006), "In and out of the Replication Factory", Cell 125 (7): 1233–1235.</span></p></li><li id="quote_28"><span class="references-num">[28]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Slater, Steven; Wold, Sture; Lu, Min; Boye, Erik; Skarstad, Kirsten; Kleckner, Nancy (September 1995). "E. coli SeqA protein binds oriC in two different methyl-modulated reactions appropriate to its roles in DNA replication initiation and origin sequestration". Cell. 82 (6): 927–936. doi:10.1016/0092-8674(95)90272-4. PMID 7553853..</span></p></li><li id="quote_29"><span class="references-num">[29]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Cooper, Stephen; Helmstetter, Charles E. (February 1968). "Chromosome replication and the division cycle of Escherichia coli". Journal of Molecular Biology. 31 (3): 519–540. doi:10.1016/0022-2836(68)90425-7..</span></p></li><li id="quote_30"><span class="references-num">[30]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Saiki, Randall; Gelfand, David H.; Stoffel, Susanne; Scharf, Stephen J.; Higuchi, Russell; Horn, Glenn T.; Mullis, Kary B.; Erlich, Henry A. (29 January 1988). "Primer-directed enzymatic amplification of DNA with a thermostable DNA polymerase". Science. 239 (4839): 487–491. Bibcode:1988Sci...239..487S. doi:10.1126/science.2448875. PMID 2448875. Retrieved 7 April 2016..</span></p></li></ul></div><div class="read-num">阅读 <!-- -->626</div></div><div class="right-side" id="rightSide"><div class="side" id="lemma-side"><div class="side-title">版本记录</div><ul class="side-lst"><li><p class="side-lst-txt">暂无</p></li></ul><div class="user-card userCard"></div></div><div class="side"><div class="side-event"></div></div></div></div><div class="footer-box"><div id="footer"><div class="footer-logo-wrap"><div class="footer-logo"></div><div class="footer-logo-text">知识·传播·科普</div></div><div class="footer-info">本网站内容采用<a target="_blank" href="https://web.archive.org/web/20221025122658/https://creativecommons.org/licenses/by-sa/3.0/deed.zh?tdsourcetag=s_pctim_aiomsg">CC-BY-SA 3.0</a>授权</div><div class="footer-btn-wrap"><a target="_blank" href="https://web.archive.org/web/20221025122658/https://baike.sogou.com/help/#user_protocol">用户协议</a><a target="_blank" href="https://web.archive.org/web/20221025122658/http://www.sogou.com/docs/terms.htm?v=1">免责声明</a><a target="_blank" href="https://web.archive.org/web/20221025122658/http://corp.sogou.com/private.html">隐私政策</a><a target="_blank" href="https://web.archive.org/web/20221025122658/https://baike.sogou.com/kexue/intro.htm">关于我们</a></div></div></div><script>window.lemmaInfo ={"lemmaId":"10681","versionId":"54554453000380681","title":"基因复制","subtitle":"","abstracts":{"paragraphId":"54554453033935109","title":"摘要","versionId":"54554453000380682","lemmaId":10681,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82815118,"name":"iscience","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599812987,"comment":null,"dependVersionId":0,"contentType":2,"content":"<p>在分子生物学中，DNA复制是从一个原始DNA分子中产生两个完全相同的子代DNA的生物学过程。DNA复制发生在所有生物体内，作为生物遗传的基础。细胞具有独特的分裂特性，这使得DNA复制过程不是必需的。</p>\n<p>DNA由两条互补的双螺旋链组成。在复制过程中，这些链被分离。然后，原始DNA分子的每条链都可作为模板，用于产生对应的DNA分子，这一过程被称为半保守复制。作为半保守复制的结果，新螺旋将由原始的DNA链和新合成的链组成。<sup><a href=\"#quote_1\" class=\"kx_ref\">[1]</a></sup> 细胞校对和错误检查机制确保了DNA复制高保真性。<sup><a href=\"#quote_2\" class=\"kx_ref\">[2]</a></sup> </p>\n<p>在细胞中，DNA复制始于基因组中的特定位置或复制起点。 在复制起点解开DNA从而合成子链，由一种被称为解旋酶的酶调节，导致复制叉从起点双向生长。许多蛋白质与复制叉相关联，有助于启动和延伸DNA合成。最显著的是，DNA聚合酶通过添加核苷酸来补充每条(模板)链，从而合成新的链。DNA复制发生在分裂间期的S期。</p>\n<p>DNA复制(DNA扩增)也可以在体外进行(人工地，在细胞外)。从细胞中分离的DNA聚合酶和人工DNA引物可用于在模板DNA分子中以已知序列开始DNA合成。例如，聚合酶链式反应、连接酶链式反应(LCR)和转录介导的扩增。</p>","pics":[{"originalUrl":"https://web.archive.org/web/20221025122658/https://img02.sogoucdn.com/app/a/200698/sogou_science_13741?width=231&height=174&titlename=DNA%E5%A4%8D%E5%88%B6%EF%BC%9A%E5%8F%8C%E8%9E%BA%E6%97%8B%E6%98%AF%E6%9C%AA%E5%8E%8B%E7%BC%A9%E4%B8%94%E6%9C%AA%E5%8D%B7%E7%BB%95%E7%9A%84%EF%BC%8C%E9%9A%8F%E5%90%8E%E6%AF%8F%E6%9D%A1%E5%88%86%E7%A6%BB%E7%9A%84%E9%93%BE%EF%BC%88%E9%9D%92%E7%BB%BF%E8%89%B2%EF%BC%89%E9%83%BD%E4%BD%9C%E4%B8%BA%E6%A8%A1%E6%9D%BF%EF%BC%8C%E5%A4%8D%E5%88%B6%E4%B8%80%E6%9D%A1%E6%96%B0%E7%9A%84%E4%BC%99%E4%BC%B4%E9%93%BE%EF%BC%88%E7%BB%BF%E8%89%B2%EF%BC%89%E3%80%82%E6%A0%B8%E8%8B%B7%E9%85%B8%EF%BC%88%E7%A2%B1%E5%9F%BA%EF%BC%89%E5%8C%B9%E9%85%8D%E4%BB%A5%E5%90%88%E6%88%90%E6%96%B0%E7%9A%84%E4%BC%99%E4%BC%B4%E9%93%BE%EF%BC%8C%E5%BD%A2%E6%88%90%E4%B8%A4%E4%B8%AA%E6%96%B0%E7%9A%84%E5%8F%8C%E8%9E%BA%E6%97%8B%E3%80%82&w=200&h=403","url":"https://web.archive.org/web/20221025122658/https://img02.sogoucdn.com/app/a/200698/sogou_science_13741","rw":200,"rh":403,"title":"DNA复制：双螺旋是未压缩且未卷绕的，随后每条分离的链（青绿色）都作为模板，复制一条新的伙伴链（绿色）。核苷酸（碱基）匹配以合成新的伙伴链，形成两个新的双螺旋。","alt":null,"width":231,"height":174}],"card":null,"references":[],"versionCount":0},"card":{"paragraphId":"0","title":"","versionId":"0","lemmaId":0,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":0,"comment":null,"dependVersionId":0,"contentType":0,"content":"","pics":null,"card":null,"references":null,"versionCount":0},"categories":[{"id":4,"name":"生物学","parents":[]}],"creator":{"uid":76032140,"name":"Ivan","pic":"https://web.archive.org/web/20221025122658/https://cache.soso.com/qlogo/g?b=oidb&k=R4Tp0e1bbEohYdh6HMicSVw&s=100&t=1557019253","introduction":null,"educations":[{"schoolName":"山东大学","major":"","degree":"硕士","universityId":70,"universityLogo":"https://web.archive.org/web/20221025122658/https://img01.sogoucdn.com/app/a/200943/7cf4032e-6012-11e9-aa89-fc4dd4f70029","majorLevel1":"理学","majorLevel2":"生物学","majorLevel3":"生物化学与分子生物学","majorLevel1Id":1,"majorLevel2Id":189,"majorLevel3Id":218,"state":null,"lab":null,"researchField":null}],"jobs":null,"works":null,"educationBrief":"山东大学","jobBrief":"","role":0,"roleName":null,"title":"山东大学 · 生物学硕士","professionalTitle":null,"phoneNo":null,"editable":true,"partnerId":37,"partnerIdCreateTime":1594286501,"partnerIdPoped":false},"createTime":1568626287,"editor":{"uid":76032140,"name":"Ivan","pic":"https://web.archive.org/web/20221025122658/https://cache.soso.com/qlogo/g?b=oidb&k=R4Tp0e1bbEohYdh6HMicSVw&s=100&t=1557019253","introduction":null,"educations":[{"schoolName":"山东大学","major":"","degree":"硕士","universityId":70,"universityLogo":"https://web.archive.org/web/20221025122658/https://img01.sogoucdn.com/app/a/200943/7cf4032e-6012-11e9-aa89-fc4dd4f70029","majorLevel1":"理学","majorLevel2":"生物学","majorLevel3":"生物化学与分子生物学","majorLevel1Id":1,"majorLevel2Id":189,"majorLevel3Id":218,"state":null,"lab":null,"researchField":null}],"jobs":null,"works":null,"educationBrief":"山东大学","jobBrief":"","role":0,"roleName":null,"title":"山东大学 · 生物学硕士","professionalTitle":null,"phoneNo":null,"editable":true,"partnerId":37,"partnerIdCreateTime":1594286501,"partnerIdPoped":false},"editTime":1576233989,"state":1,"versionCount":1,"upNum":2,"downNum":0,"pics":[{"originalUrl":"https://web.archive.org/web/20221025122658/https://img02.sogoucdn.com/app/a/200698/sogou_science_13741?width=231&height=174&titlename=DNA%E5%A4%8D%E5%88%B6%EF%BC%9A%E5%8F%8C%E8%9E%BA%E6%97%8B%E6%98%AF%E6%9C%AA%E5%8E%8B%E7%BC%A9%E4%B8%94%E6%9C%AA%E5%8D%B7%E7%BB%95%E7%9A%84%EF%BC%8C%E9%9A%8F%E5%90%8E%E6%AF%8F%E6%9D%A1%E5%88%86%E7%A6%BB%E7%9A%84%E9%93%BE%EF%BC%88%E9%9D%92%E7%BB%BF%E8%89%B2%EF%BC%89%E9%83%BD%E4%BD%9C%E4%B8%BA%E6%A8%A1%E6%9D%BF%EF%BC%8C%E5%A4%8D%E5%88%B6%E4%B8%80%E6%9D%A1%E6%96%B0%E7%9A%84%E4%BC%99%E4%BC%B4%E9%93%BE%EF%BC%88%E7%BB%BF%E8%89%B2%EF%BC%89%E3%80%82%E6%A0%B8%E8%8B%B7%E9%85%B8%EF%BC%88%E7%A2%B1%E5%9F%BA%EF%BC%89%E5%8C%B9%E9%85%8D%E4%BB%A5%E5%90%88%E6%88%90%E6%96%B0%E7%9A%84%E4%BC%99%E4%BC%B4%E9%93%BE%EF%BC%8C%E5%BD%A2%E6%88%90%E4%B8%A4%E4%B8%AA%E6%96%B0%E7%9A%84%E5%8F%8C%E8%9E%BA%E6%97%8B%E3%80%82&w=200&h=403","url":"https://web.archive.org/web/20221025122658/https://img02.sogoucdn.com/app/a/200698/sogou_science_13741","rw":200,"rh":403,"title":"DNA复制：双螺旋是未压缩且未卷绕的，随后每条分离的链（青绿色）都作为模板，复制一条新的伙伴链（绿色）。核苷酸（碱基）匹配以合成新的伙伴链，形成两个新的双螺旋。","alt":null,"width":231,"height":174},{"originalUrl":"https://web.archive.org/web/20221025122658/https://img02.sogoucdn.com/app/a/200698/sogou_science_13743?titlename=DNA%E8%81%9A%E5%90%88%E9%85%B6%E5%B0%86%E6%A0%B8%E8%8B%B7%E9%85%B8%E6%B7%BB%E5%8A%A0%E5%88%B0DNA%E9%93%BE%E7%9A%843%20'%E7%AB%AF%E3%80%82%5B1%5D%20%E5%A6%82%E6%9E%9C%E4%B8%8D%E5%B0%8F%E5%BF%83%E5%BC%95%E5%85%A5%E9%94%99%E9%85%8D%EF%BC%8C%E8%81%9A%E5%90%88%E9%85%B6%E5%B0%B1%E4%BC%9A%E5%8F%97%E5%88%B0%E6%8A%91%E5%88%B6%EF%BC%8C%E6%97%A0%E6%B3%95%E8%BF%9B%E4%B8%80%E6%AD%A5%E6%89%A9%E5%B1%95%E3%80%82%E6%A0%A1%E5%AF%B9%E5%88%A0%E9%99%A4%E4%B8%8D%E5%8C%B9%E9%85%8D%E7%9A%84%E6%A0%B8%E8%8B%B7%E9%85%B8%EF%BC%8C%E6%89%A9%E5%B1%95%E5%B0%B1%E4%BC%9A%E7%BB%A7%E7%BB%AD%E8%BF%9B%E8%A1%8C%E3%80%82&w=250&h=500","url":"https://web.archive.org/web/20221025122658/https://img02.sogoucdn.com/app/a/200698/sogou_science_13743","rw":250,"rh":500,"title":"DNA聚合酶将核苷酸添加到DNA链的3 '端。[1] 如果不小心引入错配，聚合酶就会受到抑制，无法进一步扩展。校对删除不匹配的核苷酸，扩展就会继续进行。","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221025122658/https://img03.sogoucdn.com/app/a/200698/sogou_science_13745?w=200&h=312&titlename=%E5%8F%91%E8%B5%B7DNA%E5%A4%8D%E5%88%B6%E7%9A%84%E5%BC%95%E5%8F%91%E5%89%82%E7%9A%84%E4%BD%9C%E7%94%A8%E3%80%82","url":"https://web.archive.org/web/20221025122658/https://img03.sogoucdn.com/app/a/200698/sogou_science_13745","rw":200,"rh":312,"title":"发起DNA复制的引发剂的作用。","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221025122658/https://img02.sogoucdn.com/app/a/200698/sogou_science_13746?w=200&h=117&titlename=%E5%A4%8D%E5%88%B6%E5%89%8D%E5%A4%8D%E5%90%88%E7%89%A9%E7%9A%84%E5%BD%A2%E6%88%90%E3%80%82","url":"https://web.archive.org/web/20221025122658/https://img02.sogoucdn.com/app/a/200698/sogou_science_13746","rw":200,"rh":117,"title":"复制前复合物的形成。","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221025122658/https://img04.sogoucdn.com/app/a/200698/sogou_science_13748?w=300&h=178&titlename=%E5%A4%8D%E5%88%B6%E5%8F%89%E6%96%B9%E6%A1%88%E3%80%82%20a%EF%BC%9A%20%E6%A8%A1%E6%9D%BF%EF%BC%8Cb%EF%BC%9A%20%E5%89%8D%E5%AF%BC%E9%93%BE%EF%BC%8Cc%EF%BC%9A%20%E5%90%8E%E9%9A%8F%E9%93%BE%EF%BC%8Cd%EF%BC%9A%20%E5%A4%8D%E5%88%B6%E5%8F%89%EF%BC%8Ce%EF%BC%9A%20%E5%BC%95%E7%89%A9%EF%BC%8Cf%EF%BC%9A%20%E5%86%88%E5%B4%8E%E7%89%87%E6%AE%B5","url":"https://web.archive.org/web/20221025122658/https://img04.sogoucdn.com/app/a/200698/sogou_science_13748","rw":300,"rh":178,"title":"复制叉方案。 a： 模板，b： 前导链，c： 后随链，d： 复制叉，e： 引物，f： 冈崎片段","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221025122658/https://img04.sogoucdn.com/app/a/200698/sogou_science_13750?w=200&h=216&titlename=%E7%BB%84%E8%A3%85%E7%9A%84%E4%BA%BA%E7%B1%BBDNA%E9%92%B3%EF%BC%8C%E5%AE%83%E6%98%AFPCNA%E8%9B%8B%E7%99%BD%E7%9A%84%E4%B8%89%E8%81%9A%E4%BD%93%E3%80%82","url":"https://web.archive.org/web/20221025122658/https://img04.sogoucdn.com/app/a/200698/sogou_science_13750","rw":200,"rh":216,"title":"组装的人类DNA钳，它是PCNA蛋白的三聚体。","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221025122658/https://img01.sogoucdn.com/app/a/200698/sogou_science_13751?w=150&h=156&titlename=%E7%9C%9F%E6%A0%B8%E7%BB%86%E8%83%9E%E7%9A%84%E7%BB%86%E8%83%9E%E5%91%A8%E6%9C%9F%E3%80%82","url":"https://web.archive.org/web/20221025122658/https://img01.sogoucdn.com/app/a/200698/sogou_science_13751","rw":150,"rh":156,"title":"真核细胞的细胞周期。","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221025122658/https://img02.sogoucdn.com/app/a/200698/sogou_science_13752?w=200&h=359&titlename=%E5%A4%8D%E5%88%B6%E5%90%8E%E7%9A%84GATC%E4%BD%8D%E7%82%B9%E7%9A%84Dam%E7%94%B2%E5%9F%BA%E5%8C%96%E8%85%BA%E5%98%8C%E5%91%A4%E3%80%82","url":"https://web.archive.org/web/20221025122658/https://img02.sogoucdn.com/app/a/200698/sogou_science_13752","rw":200,"rh":359,"title":"复制后的GATC位点的Dam甲基化腺嘌呤。","alt":null,"width":0,"height":0}],"catalogs":[{"level":1,"title":"DNA结构","paragraphId":"14995458648703489","subCatalogs":null},{"level":1,"title":"DNA聚合酶","paragraphId":"14995458648703490","subCatalogs":null},{"level":1,"title":"复制过程","paragraphId":"14995458665480719","subCatalogs":[{"level":2,"title":"开始","paragraphId":"14995458665480719","subCatalogs":null},{"level":2,"title":"延长","paragraphId":"14995458665480719","subCatalogs":null},{"level":2,"title":"复制叉","paragraphId":"14995458665480719","subCatalogs":null},{"level":2,"title":"DNA复制蛋白","paragraphId":"14995458665480719","subCatalogs":null},{"level":2,"title":"复制机械","paragraphId":"14995458665480719","subCatalogs":null},{"level":2,"title":"终止","paragraphId":"14995458665480719","subCatalogs":null}]},{"level":1,"title":"管理","paragraphId":"14995458665480720","subCatalogs":[{"level":2,"title":"真核生物","paragraphId":"14995458665480720","subCatalogs":null},{"level":2,"title":"细菌","paragraphId":"14995458665480720","subCatalogs":null}]},{"level":1,"title":"聚合酶链反应","paragraphId":"14995458665480721","subCatalogs":null},{"level":1,"title":"笔记","paragraphId":"14995458665480722","subCatalogs":null},{"level":1,"title":"参考文献","paragraphId":"-1","subCatalogs":null}],"paragraphs":[{"paragraphId":"14995458648703489","title":"DNA结构","versionId":"54554453000380683","lemmaId":10681,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82815118,"name":"iscience","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599812987,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>DNA以双链结构存在，两条链盘绕在一起形成特有的双螺旋结构。每一股DNA都是由四种核苷酸组成的链。DNA中的核苷酸含有脱氧核糖、磷酸和碱基。这四种类型的核苷酸对应于四种核酸碱基：腺嘌呤、胞嘧啶、鸟嘌呤和胸腺嘧啶，通常缩写为A、C、G和T。腺嘌呤和鸟嘌呤是嘌呤碱基，而胞嘧啶和胸腺嘧啶是嘧啶。这些核苷酸形成磷酸二酯键，形成核酸双螺旋的磷酸-脱氧核糖骨架，核酸碱基指向内部(即，朝向相反的链)。碱基通过氢键在链之间互补配对形成碱基对。腺嘌呤与胸腺嘧啶通过两个氢键互补配对，鸟嘌呤与胞嘧啶通过三个氢键互补配对。</p><p>DNA链具有方向性，单链的不同末端被称为“3′末端”和“5′末端”。按照惯例，如果给出单条DNA链的碱基序列，序列的左端是5’端，而右端是3’端。双螺旋的链是反向平行的，一条是5’到3’，另一条是3’到5’。这些术语指脱氧核糖中的碳原子，链中的下一个磷酸连接到该碳原子上。方向性对DNA合成有影响，因为DNA聚合酶只能从DNA链的5’到3’添加核苷酸从而合成DNA。</p><p>DNA中互补碱基的配对(通过氢键)意味着每条链中包含的信息是多余的。磷酸二酯(链内)键比氢(链间)键更强，这允许双链之间彼此分离。因此，单链上的核苷酸可用于在新合成的配对链上重建核苷酸。<sup><a href=\"#quote_3\" class=\"kx_ref\">[3]</a></sup> </p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14995458648703490","title":"DNA聚合酶","versionId":"54554453000380684","lemmaId":10681,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82815118,"name":"iscience","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599812987,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p><img alt=\"\" class=\"fileimage kx_img ed_imgfloat_right\" img_height=\"500\" img_width=\"250\" titlename=\"DNA聚合酶将核苷酸添加到DNA链的3 '端。[1] 如果不小心引入错配，聚合酶就会受到抑制，无法进一步扩展。校对删除不匹配的核苷酸，扩展就会继续进行。\" data-src=\"https://img02.sogoucdn.com/app/a/200698/sogou_science_13743\"> </p><p>DNA聚合酶是一个进行各种形式的DNA复制的酶家族。<sup><a href=\"#quote_4\" class=\"kx_ref\">[4]</a></sup> 一般来说，DNA聚合酶不能引发新链的合成，但只能延伸与模板链配对的现有DNA或核糖核酸链。要开始合成，必须合成一个短的核糖核酸片段，称为引物，并与模板DNA链配对。</p><p>DNA聚合酶通过延伸现有核苷酸链的3’端来添加新的DNA链，通过磷酸二酯键的产生一次添加一个与模板链匹配的新核苷酸。这个DNA聚合过程的能量来自于附着在每个未掺入碱基上的三个磷酸盐之间的高能磷酸盐(磷酸酐)键的水解。带有磷酸基团的游离碱基称为核苷酸；特别是，带有三个磷酸基团的碱基被称为核苷三磷酸。当核苷酸被添加到生长的DNA链中时，核苷酸的近端磷酸酯与生长链之间磷酸二酯键的形成伴随着高能磷酸酯键的水解以及两个远端磷酸基团作为焦磷酸盐的释放。将所得焦磷酸酶解成无机磷酸盐消耗第二个高能磷酸键，并使反应有效地不可逆。</p><p>首先测量活细胞中的基因复制速率，即噬菌体T4DNA在噬菌体感染的大肠杆菌中的延伸速率。<sup><a href=\"#quote_5\" class=\"kx_ref\">[5]</a></sup> 此外，一些DNA聚合酶也具有校对能力；他们可以去除生长链末端的核苷酸，以纠正错配的碱基。最后，复制后错配修复机制监测DNA的错误，能够区分新合成的DNA链和模板链序列中的错配。总之，这三个鉴别步骤使得每添加109个核苷酸，复制保真度不到一个错误。<sup><a href=\"#quote_5\" class=\"kx_ref\">[5]</a></sup> </p><p>在活细胞中的基因复制速率首先被测量为噬菌体T4基因在噬菌体感染中的延伸速率 <i>E.大肠杆菌</i>。<sup><a href=\"#quote_6\" class=\"kx_ref\">[6]</a></sup> 在37℃下，DNA呈指数增长期间，速率为每秒749个核苷酸。噬菌体T4DNA合成过程中每复制一个碱基对的突变率为1.7/108。<sup><a href=\"#quote_7\" class=\"kx_ref\">[7]</a></sup> </p>","pics":[{"originalUrl":"https://web.archive.org/web/20221025122658/https://img02.sogoucdn.com/app/a/200698/sogou_science_13743?titlename=DNA%E8%81%9A%E5%90%88%E9%85%B6%E5%B0%86%E6%A0%B8%E8%8B%B7%E9%85%B8%E6%B7%BB%E5%8A%A0%E5%88%B0DNA%E9%93%BE%E7%9A%843%20'%E7%AB%AF%E3%80%82%5B1%5D%20%E5%A6%82%E6%9E%9C%E4%B8%8D%E5%B0%8F%E5%BF%83%E5%BC%95%E5%85%A5%E9%94%99%E9%85%8D%EF%BC%8C%E8%81%9A%E5%90%88%E9%85%B6%E5%B0%B1%E4%BC%9A%E5%8F%97%E5%88%B0%E6%8A%91%E5%88%B6%EF%BC%8C%E6%97%A0%E6%B3%95%E8%BF%9B%E4%B8%80%E6%AD%A5%E6%89%A9%E5%B1%95%E3%80%82%E6%A0%A1%E5%AF%B9%E5%88%A0%E9%99%A4%E4%B8%8D%E5%8C%B9%E9%85%8D%E7%9A%84%E6%A0%B8%E8%8B%B7%E9%85%B8%EF%BC%8C%E6%89%A9%E5%B1%95%E5%B0%B1%E4%BC%9A%E7%BB%A7%E7%BB%AD%E8%BF%9B%E8%A1%8C%E3%80%82&w=250&h=500","url":"https://web.archive.org/web/20221025122658/https://img02.sogoucdn.com/app/a/200698/sogou_science_13743","rw":250,"rh":500,"title":"DNA聚合酶将核苷酸添加到DNA链的3 '端。[1] 如果不小心引入错配，聚合酶就会受到抑制，无法进一步扩展。校对删除不匹配的核苷酸，扩展就会继续进行。","alt":null,"width":0,"height":0}],"card":null,"references":[],"versionCount":0},{"paragraphId":"14995458665480719","title":"复制过程","versionId":"54554453000380685","lemmaId":10681,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82815118,"name":"iscience","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599812987,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>像所有生物聚合过程一样，DNA复制在三个酶催化和协调的步骤中进行:起始、延伸和终止。</p><p></p><h3>开始</h3><p></p><p><img alt=\"\" class=\"fileimage kx_img ed_imgfloat_right\" img_height=\"312\" img_width=\"200\" titlename=\"发起DNA复制的引发剂的作用。\" data-src=\"https://img03.sogoucdn.com/app/a/200698/sogou_science_13745\"> </p><p><img alt=\"\" class=\"fileimage kx_img ed_imgfloat_right\" img_height=\"117\" img_width=\"200\" titlename=\"复制前复合物的形成。\" data-src=\"https://img02.sogoucdn.com/app/a/200698/sogou_science_13746\"> </p><p>细胞要分裂，首先必须复制它的DNA。<sup><a href=\"#quote_8\" class=\"kx_ref\">[8]</a></sup> 这一过程是在被称为“起源”的DNA中的特定点开始的，这些点与起始蛋白靶向结合。<sup><a href=\"#quote_9\" class=\"kx_ref\">[9]</a></sup> 在大肠杆菌中，这种蛋白质是DnaA；在酵母中，这是起源识别复合体。<sup><a href=\"#quote_9\" class=\"kx_ref\">[9]</a></sup> 引发蛋白结合的DNA序列往往“富含腺苷酸和胸腺嘧啶碱基”，因为A-T有两个氢键(而不是三个形成的C-G)，因此双链更容易分离。<sup><a href=\"#quote_10\" class=\"kx_ref\">[10]</a></sup> 一旦找到了起点，这些起始蛋白就会招募其他蛋白质，形成复制前复合体，解开双链DNA。</p><p></p><h3>延长</h3><p></p><p>DNA聚合酶具有5′–3′活性。在开始合成之前，所有已知的DNA复制系统都需要一个游离的3’羟基(注意:DNA模板是在3’到5’方向上读取的，而新的链是在5’到3’方向上合成的——（这一点经常会被混淆）。存在四种不同的DNA合成机制:</p><p>\n </p><ol>\n  <li>所有细胞生命形式和许多DNA病毒、噬菌体和质粒都使用一种引物酶来合成一种带有游离3’羟基的短RNA引物，该引物随后被DNA聚合酶延长。</li>\n  <li>反转录因子(包括反转录病毒)采用tRNA，通过提供一种被反转录酶延长的游离3’羟基来启动DNA复制。</li>\n  <li>在腺病毒和φ29噬菌体家族中，3’羟基由基因组连接蛋白(末端蛋白)的氨基酸侧链提供，核苷酸通过DNA聚合酶添加到该蛋白上形成新的链。</li>\n  <li>在单链DNA病毒——包括圆环病毒、双生病毒、细小病毒和其他病毒——以及许多使用滚环复制(RCR)机制的噬菌体和质粒中，RCR核酸内切酶在基因组链(单链病毒)或其中一条DNA链(质粒)上产生缺口。缺口链的5’端转移到核酸酶上的酪氨酸残基，然后游离的3’羟基被DNA聚合酶用来合成新链。</li>\n </ol><p></p><p>第一种是这些机制中最广为人知的，被细胞生物体所使用。在这种机制中，一旦两条链被分离，引物酶将RNA引物添加到模板链中。前导链接收一个RNA引物，而滞后链接收多个RNA引物。前导链通过高加工性的DNA聚合酶从引物中连续延伸，而滞后链从形成冈崎片段的每个引物中不连续延伸。核糖核酸酶去除引物核糖核酸片段，一个不同于复制聚合酶的低加工性DNA聚合酶进入以填补缺口。当该过程完成后，可以在前导链上找到一个缺口，在滞后链上找到几个缺口。连接酶可以填补这些缺口，从而完成新复制的DNA分子。</p><p>细菌和古细菌/真核生物在这一过程中使用的引物酶有显著差异。细菌使用一种属于DnaG蛋白超家族的引物酶，它含有TOPRIM折叠型催化结构域。<sup><a href=\"#quote_11\" class=\"kx_ref\">[11]</a></sup> TOPRIM折叠型催化结构域包含一个α/β核，在Rossmann样拓扑中有四个保守链。这种结构也存在于拓扑异构酶Ia、拓扑异构酶II、OLD家族核酸酶和与RecR蛋白相关的DNA修复蛋白的催化结构域中。</p><p>相比之下，古细菌和真核生物使用的引物酶含有高度衍生的RNA识别基序(RRM)。这种引物酶在结构上类似于许多病毒核糖核酸依赖性核糖核酸聚合酶、逆转录酶、环核苷酸生成环化酶和参与DNA复制和修复的A/B/Y家族DNA聚合酶。在真核复制中，引物酶与Pol α形成复合物。<sup><a href=\"#quote_12\" class=\"kx_ref\">[12]</a></sup> </p><p>多种DNA聚合酶在DNA复制过程中扮演不同的角色。在大肠杆菌中，DNAPol III是主要负责DNA复制的聚合酶。它在复制叉处组装成一个复制复合体，表现出极高的可加工性，在整个复制周期中保持不变。相反，DNAPol 1是负责用DNA取代核糖核酸引物的酶。DNAPol除了聚合酶活性外，还具有5’至3’的核酸外切酶活性，并利用其核酸外切酶活性降解其前面的核糖核酸引物，同时延伸其后面的DNA链，这一过程称为缺口翻译。Pol I比Pol III的加工性差得多，因为它在DNA复制中的主要功能是创造许多短的DNA区域，而不是几个很长的区域。</p><p>在真核生物中，低加工性酶Pol α有助于启动复制，因为它与引物酶形成复合物。<sup><a href=\"#quote_13\" class=\"kx_ref\">[13]</a></sup>在真核生物中，主导链被认为是由Pol ε指导合成；然而，这一观点最近受到了挑战，表明了Pol δ的作用。<sup><a href=\"#quote_14\" class=\"kx_ref\">[14]</a></sup> 引物去除完成Pol δ<sup><a href=\"#quote_15\" class=\"kx_ref\">[15]</a></sup>，而复制过程中的DNA修复由Pol ε完成。</p><p>随着DNA合成的继续，原始DNA链继续在气泡的每一侧展开，形成一个有两个叉的复制叉。在圆形染色体上有单一复制起点的细菌中，这一过程产生了“θ结构”(类似希腊字母θ)。相比之下，真核生物具有更长的线性染色体，并在这些染色体中的多个起点开始复制。<sup><a href=\"#quote_16\" class=\"kx_ref\">[16]</a></sup> </p><p></p><h3>复制叉</h3><p></p><p><img alt=\"\" class=\"fileimage kx_img ed_imgfloat_right\" img_height=\"178\" img_width=\"300\" titlename=\"复制叉方案。 a： 模板，b： 前导链，c： 后随链，d： 复制叉，e： 引物，f： 冈崎片段\" data-src=\"https://img04.sogoucdn.com/app/a/200698/sogou_science_13748\"> </p><p>复制叉是在DNA复制过程中在长螺旋DNA内形成的一种结构。它是由解旋酶产生的，解旋酶打破了将两条DNA链连接在一起的氢键。由此产生的结构有两个分支“叉”，每个叉由一股DNA组成。这两条链作为前导链和滞后链的模板，当DNA聚合酶将互补核苷酸与模板匹配时，就会产生前导链和滞后链；模板可以恰当地称为前导链模板和滞后链模板。</p><p>DNA总是在5’到3’方向合成。由于前导链和滞后链模板在复制叉上方向相反，一个主要问题是如何实现新生(新)滞后链DNA的合成，其合成方向与生长的复制叉的方向相反。</p><p><strong>前导链</strong></p><p>主导链是新生的DNA链，其合成方向与复制叉延伸方向相同。这种基因复制是连续的。。</p><p><strong>后随链</strong></p><p>滞后链是新生的DNA链，其合成方向与复制叉的延伸方向相反。由于滞后链的定向，其复制过程比前导链的复制更复杂。因此，这条链上的DNA聚合酶被认为“落后”于另一条链。</p><p>滞后链是在短的、分离的片段中合成的。在滞后链模板上，引物酶“读取”模板的DNA，并开始合成一个短的互补核糖核酸引物。DNA聚合酶延伸引物片段，形成冈崎片段。然后移除核糖核酸引物并用DNA替换，DNA片段通过DNA连接酶连接在一起。</p><p><strong>复制叉处的动态变化</strong></p><p><img alt=\"\" class=\"fileimage kx_img ed_imgfloat_right\" img_height=\"216\" img_width=\"200\" titlename=\"组装的人类DNA钳，它是PCNA蛋白的三聚体。\" data-src=\"https://img04.sogoucdn.com/app/a/200698/sogou_science_13750\"> </p><p>在所有情况下，解旋酶都是由六种多肽组成的，它们只包绕一条被复制的DNA链。这两种聚合酶结合到解旋酶六聚体上。在真核生物中，螺旋酶缠绕在前导链上，而在原核生物中，解旋酶缠绕在滞后链上。<sup><a href=\"#quote_17\" class=\"kx_ref\">[17]</a></sup> </p><p>当解旋酶在复制叉上解开DNA时，前方的DNA被迫旋转。这个过程导致前方的DNA扭曲堆积。<sup><a href=\"#quote_18\" class=\"kx_ref\">[18]</a></sup>这种积聚形成了扭转阻力，最终会阻止复制叉的前进。拓扑异构酶是暂时断裂DNA链的酶，缓解了由于解开DNA螺旋的两条链而引起的张力；拓扑异构酶(包括DNA旋转酶)通过在DNA螺旋中添加负的超螺旋来实现这一点。<sup><a href=\"#quote_19\" class=\"kx_ref\">[19]</a></sup> </p><p>裸露的单链DNA往往会折回，形成二级结构；这些结构会干扰DNA聚合酶的运动。为了防止这种情况，单链结合蛋白与DNA结合，直到合成第二链，防止二级结构形成。<sup><a href=\"#quote_20\" class=\"kx_ref\">[20]</a></sup> </p><p>钳蛋白质在DNA周围形成滑动夹钳，帮助DNA聚合酶与其模板保持接触，从而有助于加工。夹子的内表面使DNA能够穿过它。一旦聚合酶到达模板末端或检测到双链DNA，滑动夹就会发生构象变化，释放DNA聚合酶。夹钳装载蛋白用于最初装载夹钳，识别模板和核糖核酸引物之间的连接。<sup><a href=\"#quote_21\" class=\"kx_ref\">[21]</a></sup><sup>：274-5</sup> </p><p></p><h3>DNA复制蛋白</h3><p></p><p>在复制叉上，许多复制酶在DNA上组装成一个复杂的分子机器，称为复制体。以下是参与复制子体的主要DNA复制酶的列表:<sup><a href=\"#quote_21\" class=\"kx_ref\">[21]</a></sup> </p><p>\n </p><table class=\"wikitable\">\n  <tbody>\n   <tr>\n    <th>酶</th>\n    <th>在DNA复制中的作用</th>\n   </tr>\n   <tr>\n    <td>DNA解旋酶</td>\n    <td>也被称为螺旋去稳定酶。螺旋酶在拓扑异构酶后面的复制叉上分离出两条DNA链。</td>\n   </tr>\n   <tr>\n    <td>DNA聚合酶</td>\n    <td>在DNA复制过程中，负责在5’到3’方向催化核苷酸底物添加到DNA的酶。还执行校对和纠错。存在许多不同类型的DNA聚合酶，每种聚合酶在不同类型的细胞中执行不同的功能。</td>\n   </tr>\n   <tr>\n    <td>DNA钳</td>\n    <td>一种防止延伸的DNA聚合酶从DNA母链上解离的蛋白质。</td>\n   </tr>\n   <tr>\n    <td>单链结合蛋白</td>\n    <td>与ssDNA结合，防止DNA双螺旋在DNA解旋后重新退火，从而保持链分离，促进新生链的合成。</td>\n   </tr>\n   <tr>\n    <td>拓扑异构酶</td>\n    <td>放松DNA的超螺旋性质。</td>\n   </tr>\n   <tr>\n    <td>DNA旋转酶</td>\n    <td>通过解旋酶释放解旋张力；这是一种特殊类型的拓扑异构酶</td>\n   </tr>\n   <tr>\n    <td>DNA连接酶</td>\n    <td>重新退火半保守链，并加入滞后链的冈崎片段。</td>\n   </tr>\n   <tr>\n    <td>引物酶</td>\n    <td>为DNA聚合酶开始合成新的DNA链提供了核糖核酸(或DNA)的起点。</td>\n   </tr>\n   <tr>\n    <td>端粒酶</td>\n    <td>通过在真核生物染色体末端添加重复的核苷酸序列来延长端粒。这允许生殖细胞和干细胞避免Hayflick界限对细胞分裂的影响。<sup><a href=\"#quote_22\" class=\"kx_ref\">[22]</a></sup> </td>\n   </tr>\n  </tbody>\n </table><p></p><p></p><h3>复制机械</h3><p></p><p>复制机制由参与DNA复制并出现在模板ssDNA上的因素组成。复制机制包括原体是复制酶；DNA聚合酶、DNA螺旋酶、DNA钳和DNA拓扑异构酶以及复制蛋白；例如单链DNA结合蛋白。在复制机制中，这些组件相互协调。在大多数细菌中，所有参与复制的因素都位于复制叉上，复合物在复制过程中停留在叉上。这些复制机制被称为复制体或DNA复制系统。这些术语是位于复制叉上的蛋白质的通用术语。在真核细胞和一些细菌细胞中，不形成复制体。</p><p>由于复制机制不会相对于模板DNA（如工厂）移动，因此它们被称为复制工厂。<sup><a href=\"#quote_23\" class=\"kx_ref\">[23]</a></sup> 在另一个数字中，DNA工厂类似于放映机，DNA就像电影胶片一样不断进入放映机。在复制工厂模型中，在前导链和滞后链的DNA解旋酶都装载到模板DNA上之后，解旋酶沿着DNA链相互运行。解旋酶与复制过程的剩余部分保持关联。彼得·梅斯特(Peter Meister)等人通过监测绿色荧光蛋白标记的DNA聚合酶α，直接观察出芽酵母中的复制位点。他们检测到与复制起点对称隔开的标记基因座对的DNA复制，并发现成对基因座之间的距离随着时间的推移显著减小。 这一发现表明，基因复制的机制与基因工厂有关。也就是说，复制工厂加载在复制源和彼此关联的工厂上。此外，模板DNA进入工厂，导致模板ssDNA和新生DNA的挤压。梅斯特的发现首次证明了复制工厂模式。随后的研究表明，在许多真核细胞中，DNA解旋酶形成二聚体，细菌复制机制在DNA合成过程中停留在单个核内位置。<sup><a href=\"#quote_23\" class=\"kx_ref\">[23]</a></sup> </p><p>复制工厂完成解开姐妹染色单体。姐妹染色单体的解开对于在DNA复制后将染色质分布到子细胞中是必不可少的。因为在DNA复制后姐妹染色单体通过黏连蛋白环相互结合，所以在DNA复制中只有一个解开缠结的机会。将复制机器固定为复制工厂可以提高DNA复制的成功率。如果复制叉在染色体中自由移动，细胞核的连锁会加剧并阻碍有丝分裂分离。 </p><p></p><h3>终止</h3><p></p><p>真核生物在染色体的多个点启动DNA复制，因此复制叉在染色体的许多点相遇并终止。因为真核生物有线性染色体，所以基因复制无法到达染色体的最末端。由于这个问题，在从染色体末端开始的每个复制周期中，DNA都会丢失。端粒是靠近末端的重复DNA区域，有助于防止基因因这种缩短而丢失。端粒缩短是体细胞的正常过程。这缩短了子代DNA染色体的端粒。因此，细胞DNA在失去阻止进一步分裂之前，只能分裂一定次数。（这就是所谓的Hayflick界限。）在将DNA传递给下一代的生殖细胞系中，端粒酶延伸端粒区域的重复序列以防止降解。端粒酶可能在体细胞中异常活跃，有时会导致癌症的形成。端粒酶活性增加是癌症的标志之一。</p><p>终止要求DNA复制叉的进程必须停止或被阻止。当在特定位点发生终止时，涉及两个组分之间的相互作用:(1)在DNA中的终止位点序列，和(2)结合到该序列上以物理停止DNA复制的蛋白质。在各种细菌中，这被称为DNA复制末端位点结合蛋白，或称Ter蛋白。</p><p>因为细菌有环状染色体，当两个复制叉在亲代染色体的另一端相遇时，此时复制终止。大肠杆菌通过使用终止序列来调节这一过程，当终止序列与Tus蛋白结合时，只允许复制叉的一个方向通过。因此，复制叉被限制为总是在染色体的终止区域内相遇。<sup><a href=\"#quote_24\" class=\"kx_ref\">[24]</a></sup> </p>","pics":[{"originalUrl":"https://web.archive.org/web/20221025122658/https://img03.sogoucdn.com/app/a/200698/sogou_science_13745?w=200&h=312&titlename=%E5%8F%91%E8%B5%B7DNA%E5%A4%8D%E5%88%B6%E7%9A%84%E5%BC%95%E5%8F%91%E5%89%82%E7%9A%84%E4%BD%9C%E7%94%A8%E3%80%82","url":"https://web.archive.org/web/20221025122658/https://img03.sogoucdn.com/app/a/200698/sogou_science_13745","rw":200,"rh":312,"title":"发起DNA复制的引发剂的作用。","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221025122658/https://img02.sogoucdn.com/app/a/200698/sogou_science_13746?w=200&h=117&titlename=%E5%A4%8D%E5%88%B6%E5%89%8D%E5%A4%8D%E5%90%88%E7%89%A9%E7%9A%84%E5%BD%A2%E6%88%90%E3%80%82","url":"https://web.archive.org/web/20221025122658/https://img02.sogoucdn.com/app/a/200698/sogou_science_13746","rw":200,"rh":117,"title":"复制前复合物的形成。","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221025122658/https://img04.sogoucdn.com/app/a/200698/sogou_science_13748?w=300&h=178&titlename=%E5%A4%8D%E5%88%B6%E5%8F%89%E6%96%B9%E6%A1%88%E3%80%82%20a%EF%BC%9A%20%E6%A8%A1%E6%9D%BF%EF%BC%8Cb%EF%BC%9A%20%E5%89%8D%E5%AF%BC%E9%93%BE%EF%BC%8Cc%EF%BC%9A%20%E5%90%8E%E9%9A%8F%E9%93%BE%EF%BC%8Cd%EF%BC%9A%20%E5%A4%8D%E5%88%B6%E5%8F%89%EF%BC%8Ce%EF%BC%9A%20%E5%BC%95%E7%89%A9%EF%BC%8Cf%EF%BC%9A%20%E5%86%88%E5%B4%8E%E7%89%87%E6%AE%B5","url":"https://web.archive.org/web/20221025122658/https://img04.sogoucdn.com/app/a/200698/sogou_science_13748","rw":300,"rh":178,"title":"复制叉方案。 a： 模板，b： 前导链，c： 后随链，d： 复制叉，e： 引物，f： 冈崎片段","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221025122658/https://img04.sogoucdn.com/app/a/200698/sogou_science_13750?w=200&h=216&titlename=%E7%BB%84%E8%A3%85%E7%9A%84%E4%BA%BA%E7%B1%BBDNA%E9%92%B3%EF%BC%8C%E5%AE%83%E6%98%AFPCNA%E8%9B%8B%E7%99%BD%E7%9A%84%E4%B8%89%E8%81%9A%E4%BD%93%E3%80%82","url":"https://web.archive.org/web/20221025122658/https://img04.sogoucdn.com/app/a/200698/sogou_science_13750","rw":200,"rh":216,"title":"组装的人类DNA钳，它是PCNA蛋白的三聚体。","alt":null,"width":0,"height":0}],"card":null,"references":[],"versionCount":0},{"paragraphId":"14995458665480720","title":"管理","versionId":"54554453000380686","lemmaId":10681,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82815118,"name":"iscience","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599812987,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p><img alt=\"\" class=\"ed_imgfloat_embedmiddle\" img_height=\"156\" img_width=\"150\" titlename=\"真核细胞的细胞周期。\" data-align=\"embedmiddle\" width=\"150\" data-src=\"https://img01.sogoucdn.com/app/a/200698/sogou_science_13751\"> </p><p></p><h3>真核生物</h3><p></p><p>在真核生物中，DNA的复制受细胞周期调控。随着细胞的生长和分裂，它在细胞周期的各个阶段进行；DNA复制发生在S期（合成期）。真核细胞整个周期进程由细胞周期检查点来控制。通过检查点的进展是通过各种蛋白质之间复杂的相互作用来控制的，包括细胞周期蛋白和细胞周期蛋白依赖性激酶。<sup><a href=\"#quote_25\" class=\"kx_ref\">[25]</a></sup> 与细菌不同，真核生物的DNA在细胞核内复制。<sup><a href=\"#quote_26\" class=\"kx_ref\">[26]</a></sup> </p><p>G1/S期检查点(或限制性检查点)调节真核细胞是否进入DNA复制和随后分裂的过程。没有通过这个检查点的细胞仍然处于G0期，并且不复制它们的DNA。</p><p>叶绿体和线粒体基因组的复制通过D环复制过程独立于细胞周期发生。</p><p><strong>复制焦点</strong></p><p>在脊椎动物细胞中，复制位点集中在被称为复制焦点的位置。<sup><a href=\"#quote_27\" class=\"kx_ref\">[27]</a></sup> 复制位点可以通过子链免疫染色和复制酶以及监测绿色荧光蛋白标记的复制因子来检测。通过这些方法发现，不同大小和位置的复制焦点出现在细胞分裂的S期，每个细胞核的复制焦点数目远小于基因组复制叉的数目。</p><p>P.赫恩等人(2001年)跟踪了芽殖酵母细胞中绿色荧光蛋白标记的复制焦点，并发现了复制起点在G1期和S期不断移动，在S期活力显著下降。<sup><a href=\"#quote_27\" class=\"kx_ref\">[27]</a></sup> 传统上，复制位点由核基质或核纤层蛋白固定在染色体的空间结构上。赫恩的结果否定了传统概念，出芽酵母不含核纤层蛋白，并支持复制起源自组装并形成复制焦点。</p><p>通过激发复制起点，在空间和时间上进行控制，调控复制焦点的形成。杰克逊等人(1998年)揭示了哺乳动物细胞中相邻的起源同时启动。<sup><a href=\"#quote_27\" class=\"kx_ref\">[27]</a></sup> 复制站点的空间并列带来了复制分叉的群集。群集确实挽救了停滞的复制叉，并有利于复制叉的正常进展。复制分叉的进展受到许多因素的抑制；与蛋白质或与强结合于DNA的复合物的碰撞、dNTPs的缺乏、模板DNA上的缺口等。如果复制叉停止，并且停止的叉中的剩余序列没有被复制，子链将获得未复制的位点。母链上的未复制位点将另一条链连接在一起，而不是子链。因此，产生的姐妹染色单体不能彼此分离，也不能分裂成两个子细胞。当相邻原点启动并且一个原点的复制叉停止时，另一个原点的复制叉从停止复制叉的相反方向进入并复制未复制的位点。作为另一种拯救机制，有休眠复制起源的应用，即多余的起点不会在正常的DNA复制中触发。</p><p></p><h3>细菌</h3><p></p><p><img alt=\"\" class=\"fileimage kx_img ed_imgfloat_right currentPic\" img_height=\"359\" img_width=\"200\" titlename=\"复制后的GATC位点的Dam甲基化腺嘌呤。\" data-src=\"https://img02.sogoucdn.com/app/a/200698/sogou_science_13752\"> </p><p>大多数细菌并不经历一个明确的细胞周期，而是不断复制它们的DNA；在快速增长期间，这可能会导致多轮复制同时发生。<sup><a href=\"#quote_27\" class=\"kx_ref\">[27]</a></sup> 在大肠杆菌这种最具特征的细菌中，DNA的复制是通过几种机制来调节的，包括:起始序列的半甲基化和隔离，三磷酸腺苷与二磷酸腺苷的比例，以及DnaA蛋白的水平。所有这些控制了起始蛋白与起始序列的结合。</p><p>因为大肠杆菌甲基化GATC基因序列，所以基因合成导致半甲基化序列。这种半甲基化的DNA被SeqA蛋白识别，SEqa蛋白结合并隔离原始序列；此外，DnaA(启动复制所需的)与半甲基化DNA结合较差。因此，新复制的起点被阻止立即开始另一轮的DNA复制。<sup><a href=\"#quote_28\" class=\"kx_ref\">[28]</a></sup> </p><p>当细胞处于丰富的培养基中时，三磷酸腺苷就会积累起来，一旦细胞达到特定的大小，就会触发DNA的复制。三磷酸腺苷与腺苷二磷酸竞争结合DnaA，DnaA-三磷酸腺苷复合体能够启动复制。DNA复制也需要一定数量的DnaA——每次复制起点，DnaA的结合位点数量就会翻倍，需要合成更多的DnaA才能再次启动复制。</p><p>在快速生长的细菌中，如大肠杆菌，染色体复制比分裂细胞需要更多的时间。细菌通过在前一轮复制被终止之前启动新一轮复制来解决这个问题。<sup><a href=\"#quote_29\" class=\"kx_ref\">[29]</a></sup> 新一轮复制将形成分裂细胞后两代新生的细胞染色体。这种机制会产生复制周期的重叠。</p>","pics":[{"originalUrl":"https://web.archive.org/web/20221025122658/https://img01.sogoucdn.com/app/a/200698/sogou_science_13751?w=150&h=156&titlename=%E7%9C%9F%E6%A0%B8%E7%BB%86%E8%83%9E%E7%9A%84%E7%BB%86%E8%83%9E%E5%91%A8%E6%9C%9F%E3%80%82","url":"https://web.archive.org/web/20221025122658/https://img01.sogoucdn.com/app/a/200698/sogou_science_13751","rw":150,"rh":156,"title":"真核细胞的细胞周期。","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221025122658/https://img02.sogoucdn.com/app/a/200698/sogou_science_13752?w=200&h=359&titlename=%E5%A4%8D%E5%88%B6%E5%90%8E%E7%9A%84GATC%E4%BD%8D%E7%82%B9%E7%9A%84Dam%E7%94%B2%E5%9F%BA%E5%8C%96%E8%85%BA%E5%98%8C%E5%91%A4%E3%80%82","url":"https://web.archive.org/web/20221025122658/https://img02.sogoucdn.com/app/a/200698/sogou_science_13752","rw":200,"rh":359,"title":"复制后的GATC位点的Dam甲基化腺嘌呤。","alt":null,"width":0,"height":0}],"card":null,"references":[],"versionCount":0},{"paragraphId":"14995458665480721","title":"聚合酶链反应","versionId":"54554453000380687","lemmaId":10681,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82815118,"name":"iscience","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599812987,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>研究人员通常使用聚合酶链反应（PCR）在体外复制DNA。聚合酶链反应使用一对引物跨越模板DNA中的特定区域，然后使用热稳定的DNA聚合酶从这些引物向各个方向聚合配对链。通过多次循环重复这一过程可以扩增目标DNA区域。在每个循环开始时，加热模板和引物的混合物，分离新合成的分子和模板。然后，随着混合物冷却，这两者都成为新引物退火的模板，聚合酶从这些模板延伸出来。因此，目标区域的拷贝数每轮翻一倍，呈指数级增长。<sup><a href=\"#quote_30\" class=\"kx_ref\">[30]</a></sup> </p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14995458665480722","title":"笔记","versionId":"54554453000380688","lemmaId":10681,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":82815118,"name":"iscience","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1599812987,"comment":null,"dependVersionId":0,"contentType":1,"content":"","pics":null,"card":null,"references":[],"versionCount":0}],"references":[{"id":1,"type":"book","title":"Pray, Leslie A. \"Semi-Conservative DNA Replication; Meselson and Stahl\".","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":2,"type":"book","title":"Imperfect DNA replication results in mutations. Berg JM, Tymoczko JL, Stryer L, Clarke ND (2002). \"Chapter 27: DNA Replication, Recombination, and Repair\". Biochemistry. W.H. Freeman and Company. ISBN 0-7167-3051-0.[失效连结]","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":3,"type":"book","title":"Alberts, B., et al., Molecular Biology of the Cell, Garland Science, 4th ed., 2002, pp. 238–240 ISBN 0-8153-3218-1","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":4,"type":"book","title":"Berg JM, Tymoczko JL, Stryer L, Clarke ND (2002). Biochemistry. W.H. Freeman and Company. ISBN 0-7167-3051-0. Chapter 27, Section 2: DNA Polymerases Require a Template and a Primer","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":5,"type":"book","title":"McCulloch, Scott D; Kunkel, Thomas A (January 2008). \"The fidelity of DNA synthesis by eukaryotic replicative and translesion synthesis polymerases\". Cell Research. 18 (1): 148–161. doi:10.1038/cr.2008.4. PMC 3639319. PMID 18166979.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":6,"type":"book","title":"McCarthy, David; Minner, Charles; Bernstein, Harris; Bernstein, Carol (October 1976). \"DNA elongation rates and growing point distributions of wild-type phage T4 and a DNA-delay amber mutant\". Journal of Molecular Biology. 106 (4): 963–981. doi:10.1016/0022-2836(76)90346-6. PMID 789903.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":7,"type":"book","title":"Drake JW (1970) The Molecular Basis of Mutation. Holden-Day, San Francisco ISBN 0816224501 ISBN 978-0816224500","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":8,"type":"book","title":"Alberts B, Johnson A, Lewis J, Raff M, Roberts K, Walter P (2002). Molecular Biology of the Cell. Garland Science. ISBN 0-8153-3218-1. Chapter 5: DNA Replication Mechanisms","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":9,"type":"book","title":"Berg JM, Tymoczko JL, Stryer L, Clarke ND (2002). \"Chapter 27, Section 4: DNA Replication of Both Strands Proceeds Rapidly from Specific Start Sites\". Biochemistry. W.H. Freeman and Company. ISBN 0-7167-3051-0.[失效连结]","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":10,"type":"book","title":"Lodish H, Berk A, Zipursky LS, Matsudaira P, Baltimore D, Darnell J (2000). Molecular Cell Biology. W. H. Freeman and Company. ISBN 0-7167-3136-3.12.1. General Features of Chromosomal Replication: Three Common Features of Replication Origins","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":11,"type":"book","title":"Aravind, L.; Leipe, D. D.; Koonin, E. V. (1998). \"Toprim—a conserved catalytic domain in type IA and II topoisomerases, DnaG-type primases, OLD family nucleases and RecR proteins\". Nucleic Acids Research. 26 (18): 4205–4213. doi:10.1093/nar/26.18.4205. PMC 147817. PMID 9722641.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":12,"type":"book","title":"Frick, David; Richardson, Charles (July 2001). \"DNA Primases\". Annual Review of Biochemistry. 70: 39–80. doi:10.1146/annurev.biochem.70.1.39. PMID 11395402.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":13,"type":"book","title":"Barry, Elizabeth R.; Bell, Stephen D. (8 December 2006). \"DNA Replication in the Archaea\". Microbiology and Molecular Biology Reviews. 70 (4): 876–887. doi:10.1128/MMBR.00029-06. PMC 1698513. PMID 17158702.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":14,"type":"book","title":"Stillman, Bruce (July 2015). \"Reconsidering DNA Polymerases at the Replication Fork in Eukaryotes\". Molecular Cell. 59 (2): 139–141. doi:10.1016/j.molcel.2015.07.004. PMC 4636199. PMID 26186286.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":15,"type":"book","title":"Distinguishing the pathways of primer removal during Eukaryotic Okazaki fragment maturation Contributor Author Rossi, Marie Louise. Date Accessioned: 2009-02-23T17:05:09Z. Date Available: 2009-02-23T17:05:09Z. Date Issued: 2009-02-23T17:05:09Z. Identifier Uri: http://hdl.handle.net/1802/6537. Description: Dr. Robert A. Bambara, Faculty Advisor. Thesis (PhD) – School of Medicine and Dentistry, University of Rochester. UR only until January 2010. UR only until January 2010.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":16,"type":"book","title":"Huberman, Joel A.; Riggs, Arthur D. (March 1968). \"On the mechanism of DNA replication in mammalian chromosomes\". Journal of Molecular Biology. 32 (2): 327–341. doi:10.1016/0022-2836(68)90013-2. PMID 5689363.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":17,"type":"book","title":"Gao, Yang; et al. (22 February 2019). \"Structures and operating principles of the replisome\". Science. 363 (6429): 835. doi:10.1126/science.aav7003.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":18,"type":"book","title":"Alberts B, Johnson A, Lewis J, Raff M, Roberts K, Walter P (2002). Molecular Biology of the Cell. Garland Science. ISBN 0-8153-3218-1. DNA Replication Mechanisms: DNA Topoisomerases Prevent DNA Tangling During Replication","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":19,"type":"book","title":"Reece, Richard J.; Maxwell, Anthony; Wang, James C. (26 September 2008). \"DNA Gyrase: Structure and Function\". Critical Reviews in Biochemistry and Molecular Biology. 26 (3–4): 335–375. doi:10.3109/10409239109114072. PMID 1657531.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":20,"type":"book","title":"Alberts B, Johnson A, Lewis J, Raff M, Roberts K, Walter P (2002). Molecular Biology of the Cell. Garland Science. ISBN 0-8153-3218-1. DNA Replication Mechanisms: Special Proteins Help to Open Up the DNA Double Helix in Front of the Replication Fork","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":21,"type":"book","title":"Alberts B, Johnson A, Lewis J, Raff M, Roberts K, Walter P (2002). \"5DNA Replication, Repair, and Recombination\". Molecular Biology of the Cell. Garland Science. ISBN 0-8153-3218-1.[失效连结]","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":22,"type":"book","title":"\"Will the Hayflick limit keep us from living forever?\". Howstuffworks. Retrieved January 20, 2015.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":23,"type":"book","title":"James D. Watson et al. (2008), \"Molecular Biology of the gene\", Pearson Education: 237","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":24,"type":"book","title":"TA Brown (2002). Genomes. BIOS Scientific Publishers. ISBN 1-85996-228-9.13.2.3. Termination of replication","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":25,"type":"book","title":"Alberts B, Johnson A, Lewis J, Raff M, Roberts K, Walter P (2002). Molecular Biology of the Cell. Garland Science. ISBN 0-8153-3218-1. Intracellular Control of Cell-Cycle Events: S-Phase Cyclin-Cdk Complexes (S-Cdks) Initiate DNA Replication Once Per Cycle","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":26,"type":"book","title":"Brown, TA (2002). \"13\". Genomes (2nd ed.). Oxford: Wiley-Liss.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":27,"type":"book","title":"Peter Meister, Angela Taddei1, Susan M. Gasser(June 2006), \"In and out of the Replication Factory\", Cell 125 (7): 1233–1235","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":28,"type":"book","title":"Slater, Steven; Wold, Sture; Lu, Min; Boye, Erik; Skarstad, Kirsten; Kleckner, Nancy (September 1995). \"E. coli SeqA protein binds oriC in two different methyl-modulated reactions appropriate to its roles in DNA replication initiation and origin sequestration\". Cell. 82 (6): 927–936. doi:10.1016/0092-8674(95)90272-4. PMID 7553853.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":29,"type":"book","title":"Cooper, Stephen; Helmstetter, Charles E. (February 1968). \"Chromosome replication and the division cycle of Escherichia coli\". Journal of Molecular Biology. 31 (3): 519–540. doi:10.1016/0022-2836(68)90425-7.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":30,"type":"book","title":"Saiki, Randall; Gelfand, David H.; Stoffel, Susanne; Scharf, Stephen J.; Higuchi, Russell; Horn, Glenn T.; Mullis, Kary B.; Erlich, Henry A. (29 January 1988). \"Primer-directed enzymatic amplification of DNA with a thermostable DNA polymerase\". Science. 239 (4839): 487–491. Bibcode:1988Sci...239..487S. doi:10.1126/science.2448875. PMID 2448875. Retrieved 7 April 2016.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false}],"recommendReferences":null,"auditState":2,"lemmaLevel":1,"origin":0,"originEnTitle":null,"originZhTitle":null,"pv":626,"auditType":0,"synonyms":null,"showEditTime":"2019.12.13 18:46","auditors":[{"uid":0,"name":"Ki.κe","pic":"https://web.archive.org/web/20221025122658/https://wx.qlogo.cn/mmopen/vi_32/y67kfr32Doib4wg71Jiau7jVWvharic3nRKgdRRQSl6koeQJCo0GQs2Krw0vwdFRsOWnHIQOwAZsSg5lIkIrFCOcQ/132","introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":"","jobBrief":"","role":0,"roleName":null,"title":"","professionalTitle":null,"phoneNo":null,"editable":true,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false}],"hasZhishiNav":false,"auditInfos":{},"isHistory":false};</script><script crossorigin="anonymous" src="./681.基因复制 - 搜狗科学百科_files/aegis.min.js.download"></script><script crossorigin="anonymous" src="./681.基因复制 - 搜狗科学百科_files/main_2020092401.js.download"></script><script crossorigin="anonymous" src="./681.基因复制 - 搜狗科学百科_files/react.production.min.js.download"></script><script crossorigin="anonymous" src="./681.基因复制 - 搜狗科学百科_files/react-dom.production.min.js.download"></script><script crossorigin="anonymous" src="./681.基因复制 - 搜狗科学百科_files/jquery-1.11.1.min.js.download"></script><script crossorigin="anonymous" src="./681.基因复制 - 搜狗科学百科_files/main_2022062701.js.download"></script><script crossorigin="anonymous" src="./681.基因复制 - 搜狗科学百科_files/main_66bbe21.js.download"></script><script crossorigin="anonymous" src="./681.基因复制 - 搜狗科学百科_files/react.production.min.js.download"></script><script crossorigin="anonymous" src="./681.基因复制 - 搜狗科学百科_files/react-dom.production.min.js.download"></script><script crossorigin="anonymous" src="./681.基因复制 - 搜狗科学百科_files/main_edf0f08.js.download"></script>
</body></html>