<!DOCTYPE html>
<!-- saved from url=(0083)https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm -->
<html class="" data-reactroot=""><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><script src="./696.Rosetta@home - 搜狗科学百科_files/analytics.js.download" type="text/javascript"></script>
<script type="text/javascript">window.addEventListener('DOMContentLoaded',function(){var v=archive_analytics.values;v.service='wb';v.server_name='wwwb-app28.us.archive.org';v.server_ms=280;archive_analytics.send_pageview({});});</script>
<script type="text/javascript" src="./696.Rosetta@home - 搜狗科学百科_files/bundle-playback.js.download" charset="utf-8"></script>
<script type="text/javascript" src="./696.Rosetta@home - 搜狗科学百科_files/wombat.js.download" charset="utf-8"></script>
<script type="text/javascript">
  __wm.init("https://web.archive.org/web");
  __wm.wombat("https://baike.sogou.com/kexue/d10696.htm","20221025114123","https://web.archive.org/","web","/_static/",
	      "1666698083");
</script>
<link rel="stylesheet" type="text/css" href="./696.Rosetta@home - 搜狗科学百科_files/banner-styles.css">
<link rel="stylesheet" type="text/css" href="./696.Rosetta@home - 搜狗科学百科_files/iconochive.css">
<!-- End Wayback Rewrite JS Include -->
<meta name="save" content="history"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="baidu-site-verification" content="VWGb6TyYx8"><meta content="Rosetta@home - 搜狗科学百科" name="keywords"><meta content="搜狗科学百科是一部有着平等、协作、分享、自由理念的网络科学全书，为每一个互联网用户创造一个涵盖所有领域知识、服务的中文知识性平台。" name="description"><meta http-equiv="x-dns-prefetch-control" content="on"><meta name="server" baike="235" ip="210" env="online"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025114123/https://cache.soso.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025114123/https://hhy.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025114123/https://pic.baike.soso.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025114123/https://ugc.qpic.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025114123/https://xui.ptlogin2.qq.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025114123/https://q1.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025114123/https://q2.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025114123/https://q3.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025114123/https://q4.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025114123/https://q.qlogo.cn/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025114123/https://img01.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025114123/https://img02.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025114123/https://img03.sogoucdn.com/"><link rel="dns-prefetch" href="https://web.archive.org/web/20221025114123/https://img04.sogoucdn.com/"><link rel="Shortcut Icon" href="https://web.archive.org/web/20221025114123im_/https://www.sogou.com/images/logo/new/favicon.ico?v=4"><link rel="Bookmark" href="https://www.sogou.com/images/logo/new/favicon.ico?v=4"><link href="./696.Rosetta@home - 搜狗科学百科_files/base_b849887.css" rel="stylesheet"><link href="./696.Rosetta@home - 搜狗科学百科_files/detail_378aed5.css" rel="stylesheet"><link href="./696.Rosetta@home - 搜狗科学百科_files/inviteAudit_7894507.css" rel="stylesheet"><link rel="stylesheet" href="./696.Rosetta@home - 搜狗科学百科_files/highlight.min.css"><title>Rosetta@home - 搜狗科学百科</title><style>.onekey-close {
	position: absolute;
	top: 16px;
	right: 16px;
	width: 24px;
	height: 24px;
	background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	text-indent: -999em;
	background-size: 84px;
	background-position: -63px 0;
}

.onekey-login {
	position: absolute;
	top: 16.4%;
	left: 0;
	right: 0;
	width: 100%;
}

/* .onekey-login-img {
    width: 75px;
    height: 75px;
    background: url("https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/images/sprite_wap_baike.png") no-repeat;
    background-size: 100px 91px;
    background-position: 0 0;
    background-repeat: no-repeat;
    margin: 0 auto;
} */

.onekey-login-title {
	text-align: center;
	padding-bottom: 3px;
	font-size: 21px;
	font-weight: bold;
	line-height: 30px;
	color: #000;
}

.onekey-login-txt {
	text-align: center;
	font-family: PingFangSC;
	font-size: 14px;
	line-height: 20px;
	color: #8f8f8f;
}

.onekey-login-qq,
.onekey-login-wx,
.onekey-login-phone {
	display: block;
	width: 245px;
	height: 54px;
	border-radius: 45px;
	text-align: center;

	margin: 0 auto;
	font-size: 17px;
	line-height: 24px;
	color: #000;
	/* padding: 16px 77px; */
	border-radius: 12px;
	border: solid 1px #e0e0e0;
}
.onekey-qq-content,
.onekey-vx-content,
.onekey-phone-content {
	display: inline-block;
	margin-top: 16px;
}
.onekey-qq-content {
	padding: 0 5px;
}

.onekey-login-qq {
	margin-top: 48px;
	margin-bottom: 24px;
}

.onekey-login-qq:before {
	display: inline-block;
	content: "";
	width: 20px;
	height: 20px;
	background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	background-size: 80px;
	background-position: -20px 0;
	vertical-align: top;
	margin: 17px 8px 0 0;
}

.onekey-login-wx {
	margin-bottom: 24px;
}

.onekey-login-wx:before {
	display: inline-block;
	content: "";
	width: 21px;
	height: 21px;
	background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	background-size: 84px;
	background-position: 0 0;
	vertical-align: top;
	margin: 17px 10px 0 0;
}

.onekey-login-phone {
}

.onekey-login-phone:before {
	display: inline-block;
	content: "";
	width: 21px;
	height: 21px;
	background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/sprite_wap_baike_37443f3.png) no-repeat;
	background-size: 84px;
	background-position: -42px 0;
	vertical-align: top;
	margin: 17px 10px 0 0;
}

.onekey-fixed {
	z-index: 100;
	position: fixed;
	top: 0;
	bottom: 0;
	left: 0;
	right: 0;
	background: #fff;
	width: 100%;
	height: 100%;
}

.onekey-fixed.forbid {
	z-index: 100;
	position: fixed;
	top: auto;
	bottom: 68px;
	left: 9%;
	right: 0;
	background: rgba(0, 0, 0, 0.7);
	width: 82%;
	height: 43px;
	border-radius: 25px;
	color: #ffffff;
}
.onekey-login-title.forbid {
	text-align: center;
	padding-bottom: 3px;
	font-size: 14px;
	font-weight: normal;
	line-height: 30px;
	color: white;
}
</style><style>#login_mask {
  background: #000;
  opacity: 0.5;
  filter: alpha(opacity=50);
  position: fixed;
  /*fixed好像在哪个IE上有BUG，先用用*/
  left: 0;
  top: 0;
  z-index: 999;
  height: 100%;
}

#login_iframe_container {
  position: fixed;
  width: 550px;
  height: 360px;
  z-index: 1020;
  background-color: #ffffff;
}

@media screen and (max-width: 828px) {
  #login_iframe_container {
    top: 50% !important;
    left: 50% !important;
    transform: translate(-50%, -50%);
  }
}

#login_iframe_container.new-login {
  width: 550px;
  height: 360px;
  background-image: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/background_2a4a8a6.png);
}

#login_iframe_container.new-login.no-bg {
  background: #fff;
}

#login_iframe_container.new-login .login-title {
  width: 100%;
  height: 42px;
  line-height: 42px;
  text-align: center;
  font-size: 30px;
  letter-spacing: 0.19px;
  color: #ffffff;
  margin-top: 62px;
}
#login_iframe_container.new-login .forbid-title {
  width: 100%;
  height: 42px;
  line-height: 42px;
  text-align: center;
  font-size: 24px;
  letter-spacing: 0.19px;
  color: #333333;
  margin-top: 150px;
}

#login_iframe_container.new-login.no-bg .login-title {
  color: #333333;
}

#login_iframe_container.new-login .login-subtitle {
  width: 100%;
  height: 18px;
  line-height: 18px;
  font-size: 13px;
  letter-spacing: 0.08px;
  color: #ffffff;
  text-align: center;
  margin-top: 9px;
  margin-bottom: 43px;
}

#login_iframe_container.new-login.no-bg .login-subtitle {
  color: #999999;
}

#login_iframe_container.new-login .login-subtitle::before {
  content: '';
  display: inline-block;
  width: 10px;
  height: 1px;
  background-color: #ffffff;
  position: relative;
  top: -4px;
  left: -5px;
}

#login_iframe_container.new-login .login-subtitle::after {
  content: '';
  display: inline-block;
  width: 10px;
  height: 1px;
  background-color: #ffffff;
  position: relative;
  top: -4px;
  left: 5px;
}

#login_iframe_container.new-login.no-bg .login-subtitle::before {
  background-color: #999999;
}

#login_iframe_container.new-login.no-bg .login-subtitle::after {
  background-color: #999999;
}

#login_iframe_container.new-login .close-btn {
  position: absolute;
  top: 20px;
  right: 20px;
  width: 12px;
  height: 12px;
  background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/login-sprites_e3853e5.png) -59px -10px;
  background-size: 81px 91px;
  cursor: pointer;
}

#login_iframe_container.new-login .login-btn {
  width: 220px;
  height: 47px;
  border-radius: 24px;
  border: solid 1px #dddddd;
  background-color: #ffffff;
  margin: 0 auto;
  margin-top: 28px;
  position: relative;
  display: block;
}

#login_iframe_container.new-login .login-btn .login-icon {
  position: absolute;
}

#login_iframe_container.new-login .login-btn .login-text {
  width: 61px;
  height: 47px;
  line-height: 47px;
  vertical-align: middle;
  font-size: 15px;
  letter-spacing: 0.1px;
  color: #666666;
  position: absolute;
  right: 62px;
}

#login_iframe_container.new-login .login-btn.qq-btn .login-icon {
  width: 22px;
  height: 27px;
  top: 10px;
  left: 67px;
  background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/login-sprites_e3853e5.png) -10px -54px;
  background-size: 81px 91px;
}

#login_iframe_container.new-login .login-btn.qq-btn .login-text {
  right: 59px;
}

#login_iframe_container.new-login .login-btn.wechat-btn .login-icon {
  width: 29px;
  height: 24px;
  top: 12px;
  left: 62px;
  background: url(//web.archive.org/web/20221025113757/https://hhy.sogoucdn.com/js/common/hhy/login-sprites_e3853e5.png) -10px -10px;
  background-size: 81px 91px;
}</style><style>/* -- container -- */
.rodal,
.rodal-mask {
    top: 0;
    left: 0;
    width: 100%;
    height: 100%;
    z-index: 100;
}

.rodal {
    position: fixed;
}

/* -- mask -- */
.rodal-mask {
    position: fixed;
    background: rgba(0, 0, 0, .5);
}

/* -- dialog -- */
.rodal-dialog {
    position: absolute;
    z-index: 101;
    background: #fff;
    border-radius: 3px;
    box-shadow: 0 1px 3px rgba(0, 0, 0, .2);
}

.rodal-center {
    top: 50%;
    transform: translateY(-50%);
    left: 0;
    right: 0;
    margin: 0 auto;
}

.rodal-bottom {
    left: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

.rodal-top {
    left: 0;
    right: 0;
    top: 0;
    margin: auto;
}

.rodal-left {
    top: 0;
    left: 0;
    bottom: 0;
    margin: auto;
}

.rodal-right {
    top: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

/* -- close button -- */
.rodal-close {
    position: absolute;
    cursor: pointer;
    top: 16px;
    right: 16px;
    width: 16px;
    height: 16px;
}

.rodal-close:before,
.rodal-close:after {
    position: absolute;
    content: '';
    height: 2px;
    width: 100%;
    top: 50%;
    left: 0;
    margin-top: -1px;
    background: #999;
    border-radius: 100%;
    -webkit-transition: background .2s;
    transition: background .2s;
}

.rodal-close:before {
    -webkit-transform: rotate(45deg);
    transform: rotate(45deg);
}

.rodal-close:after {
    -webkit-transform: rotate(-45deg);
    transform: rotate(-45deg);
}

.rodal-close:hover:before,
.rodal-close:hover:after {
    background: #333;
}

/* -- fade -- */
/* @-webkit-keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

@keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

.rodal-fade-enter {
    -webkit-animation: rodal-fade-enter both ease-in;
    animation: rodal-fade-enter both ease-in;
} */

@-webkit-keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

@keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

.rodal-fade-leave {
    -webkit-animation: rodal-fade-leave both ease-out;
    animation: rodal-fade-leave both ease-out;
}

/* -- zoom -- */
@-webkit-keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-enter {
    -webkit-animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-leave {
    -webkit-animation: rodal-zoom-leave both;
    animation: rodal-zoom-leave both;
}

/* -- slideDown -- */
@-webkit-keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-enter {
    -webkit-animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-leave {
    -webkit-animation: rodal-slideDown-leave both;
    animation: rodal-slideDown-leave both;
}

/* -- slideLeft -- */
@-webkit-keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-enter {
    -webkit-animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-leave {
    -webkit-animation: rodal-slideLeft-leave both;
    animation: rodal-slideLeft-leave both;
}

/* -- slideRight -- */
@-webkit-keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-enter {
    -webkit-animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-leave {
    -webkit-animation: rodal-slideRight-leave both;
    animation: rodal-slideRight-leave both;
}

/* -- slideUp -- */
@-webkit-keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-enter {
    -webkit-animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
    animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
}

@-webkit-keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-leave {
    -webkit-animation: rodal-slideUp-leave both;
    animation: rodal-slideUp-leave both;
}

/* -- flip -- */
@-webkit-keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

@keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

.rodal-flip-enter {
    -webkit-animation: rodal-flip-enter both ease-in;
    animation: rodal-flip-enter both ease-in;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

@-webkit-keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

@keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

.rodal-flip-leave {
    -webkit-animation: rodal-flip-leave both;
    animation: rodal-flip-leave both;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

/* -- rotate -- */
@-webkit-keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-enter {
    -webkit-animation: rodal-rotate-enter both;
    animation: rodal-rotate-enter both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

@-webkit-keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-leave {
    -webkit-animation: rodal-rotate-leave both;
    animation: rodal-rotate-leave both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

/* -- door -- */
@-webkit-keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

@keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

.rodal-door-enter {
    -webkit-animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

@keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

.rodal-door-leave {
    -webkit-animation: rodal-door-leave both;
    animation: rodal-door-leave both;
}</style><style>/* -- container -- */
.rodal,
.rodal-mask {
    top: 0;
    left: 0;
    width: 100%;
    height: 100%;
    z-index: 100;
}

.rodal {
    position: fixed;
}

/* -- mask -- */
.rodal-mask {
    position: fixed;
    background: rgba(0, 0, 0, .5);
}

/* -- dialog -- */
.rodal-dialog {
    position: absolute;
    z-index: 101;
    background: #fff;
    border-radius: 3px;
    box-shadow: 0 1px 3px rgba(0, 0, 0, .2);
}

.rodal-center {
    top: 50%;
    transform: translateY(-50%);
    left: 0;
    right: 0;
    margin: 0 auto;
}

.rodal-bottom {
    left: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

.rodal-top {
    left: 0;
    right: 0;
    top: 0;
    margin: auto;
}

.rodal-left {
    top: 0;
    left: 0;
    bottom: 0;
    margin: auto;
}

.rodal-right {
    top: 0;
    right: 0;
    bottom: 0;
    margin: auto;
}

/* -- close button -- */
.rodal-close {
    position: absolute;
    cursor: pointer;
    top: 16px;
    right: 16px;
    width: 16px;
    height: 16px;
}

.rodal-close:before,
.rodal-close:after {
    position: absolute;
    content: '';
    height: 2px;
    width: 100%;
    top: 50%;
    left: 0;
    margin-top: -1px;
    background: #999;
    border-radius: 100%;
    -webkit-transition: background .2s;
    transition: background .2s;
}

.rodal-close:before {
    -webkit-transform: rotate(45deg);
    transform: rotate(45deg);
}

.rodal-close:after {
    -webkit-transform: rotate(-45deg);
    transform: rotate(-45deg);
}

.rodal-close:hover:before,
.rodal-close:hover:after {
    background: #333;
}

/* -- fade -- */
/* @-webkit-keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

@keyframes rodal-fade-enter {
    from {
        opacity: 0;
    }
}

.rodal-fade-enter {
    -webkit-animation: rodal-fade-enter both ease-in;
    animation: rodal-fade-enter both ease-in;
} */

@-webkit-keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

@keyframes rodal-fade-leave {
    to {
        opacity: 0
    }
}

.rodal-fade-leave {
    -webkit-animation: rodal-fade-leave both ease-out;
    animation: rodal-fade-leave both ease-out;
}

/* -- zoom -- */
@-webkit-keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-enter {
    from {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-enter {
    -webkit-animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-zoom-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

@keyframes rodal-zoom-leave {
    to {
        -webkit-transform: scale3d(.3, .3, .3);
        transform: scale3d(.3, .3, .3);
    }
}

.rodal-zoom-leave {
    -webkit-animation: rodal-zoom-leave both;
    animation: rodal-zoom-leave both;
}

/* -- slideDown -- */
@-webkit-keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-enter {
    from {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-enter {
    -webkit-animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideDown-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

@keyframes rodal-slideDown-leave {
    to {
        -webkit-transform: translate3d(0, -100px, 0);
        transform: translate3d(0, -100px, 0);
    }
}

.rodal-slideDown-leave {
    -webkit-animation: rodal-slideDown-leave both;
    animation: rodal-slideDown-leave both;
}

/* -- slideLeft -- */
@-webkit-keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-enter {
    from {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-enter {
    -webkit-animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideLeft-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

@keyframes rodal-slideLeft-leave {
    to {
        -webkit-transform: translate3d(-150px, 0, 0);
        transform: translate3d(-150px, 0, 0);
    }
}

.rodal-slideLeft-leave {
    -webkit-animation: rodal-slideLeft-leave both;
    animation: rodal-slideLeft-leave both;
}

/* -- slideRight -- */
@-webkit-keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-enter {
    from {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-enter {
    -webkit-animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-slideRight-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

@keyframes rodal-slideRight-leave {
    to {
        -webkit-transform: translate3d(150px, 0, 0);
        transform: translate3d(150px, 0, 0);
    }
}

.rodal-slideRight-leave {
    -webkit-animation: rodal-slideRight-leave both;
    animation: rodal-slideRight-leave both;
}

/* -- slideUp -- */
@-webkit-keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-enter {
    from {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-enter {
    -webkit-animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
    animation: rodal-slideUp-enter both cubic-bezier(0.23, 1, 0.32, 1);
}

@-webkit-keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

@keyframes rodal-slideUp-leave {
    to {
        -webkit-transform: translate3d(0, 100px, 0);
        transform: translate3d(0, 100px, 0);
    }
}

.rodal-slideUp-leave {
    -webkit-animation: rodal-slideUp-leave both;
    animation: rodal-slideUp-leave both;
}

/* -- flip -- */
@-webkit-keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

@keyframes rodal-flip-enter {
    from {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 60deg);
    }

    70% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }
}

.rodal-flip-enter {
    -webkit-animation: rodal-flip-enter both ease-in;
    animation: rodal-flip-enter both ease-in;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

@-webkit-keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

@keyframes rodal-flip-leave {
    from {
        -webkit-transform: perspective(400px);
        transform: perspective(400px);
    }

    30% {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
        transform: perspective(400px) rotate3d(1, 0, 0, -15deg);
    }

    to {
        -webkit-transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
        transform: perspective(400px) rotate3d(1, 0, 0, 45deg);
    }
}

.rodal-flip-leave {
    -webkit-animation: rodal-flip-leave both;
    animation: rodal-flip-leave both;
    -webkit-backface-visibility: visible !important;
    backface-visibility: visible !important;
}

/* -- rotate -- */
@-webkit-keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-enter {
    from {
        -webkit-transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, -180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-enter {
    -webkit-animation: rodal-rotate-enter both;
    animation: rodal-rotate-enter both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

@-webkit-keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

@keyframes rodal-rotate-leave {
    to {
        -webkit-transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
        transform: rotate3d(0, 0, 1, 180deg) scale3d(.3, .3, .3);
    }
}

.rodal-rotate-leave {
    -webkit-animation: rodal-rotate-leave both;
    animation: rodal-rotate-leave both;
    -webkit-transform-origin: center;
    transform-origin: center;
}

/* -- door -- */
@-webkit-keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

@keyframes rodal-door-enter {
    from {
        -webkit-transform: scale3d(0, 1, 1);
        transform: scale3d(0, 1, 1);
    }
}

.rodal-door-enter {
    -webkit-animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
    animation: rodal-door-enter both cubic-bezier(0.4, 0, 0, 1.5);
}

@-webkit-keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

@keyframes rodal-door-leave {
    60% {
        -webkit-transform: scale3d(.01, 1, 1);
        transform: scale3d(.01, 1, 1);
    }

    to {
        -webkit-transform: scale3d(0, 1, .1);
        transform: scale3d(0, 1, .1);
    }
}

.rodal-door-leave {
    -webkit-animation: rodal-door-leave both;
    animation: rodal-door-leave both;
}</style>
<!--百度统计-->
<script>
   var _hmt = _hmt || [];
   (function() {
      var hm = document.createElement("script");
      hm.src = "https://hm.baidu.com/hm.js?3c7614be3026469d5a60f41ab30b5082";
      var s = document.getElementsByTagName("script")[0]; 
      s.parentNode.insertBefore(hm, s);
      })();
</script>
</head>
<body class=""><!-- BEGIN WAYBACK TOOLBAR INSERT -->
<style type="text/css">
body {
  margin-top:0 !important;
  padding-top:0 !important;
  /*min-width:800px !important;*/
}
</style>
<script>__wm.rw(0);</script>
<div id="wm-ipp-base" lang="en" style="display: block; direction: ltr;">
</div><div id="wm-ipp-print">The Wayback Machine - https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm</div>
<script type="text/javascript">//<![CDATA[
__wm.bt(675,27,25,2,"web","https://baike.sogou.com/kexue/d10696.htm","20221025114123",1996,"/_static/",["/_static/css/banner-styles.css?v=S1zqJCYt","/_static/css/iconochive.css?v=qtvMKcIJ"], false);
  __wm.rw(1);
//]]></script>
<!-- END WAYBACK TOOLBAR INSERT --><script>window._gtag=window._gtag||{};window._gtag.shouldGrayed = false;if ('ceea99ec91694d7fb11ee89b72b92c61') window._gtag.traceId = 'ceea99ec91694d7fb11ee89b72b92c61';if ({"illegality":true}) window.userInfo = {"illegality":true};</script><div class="topnavbox"><ul class="topnav"><li><a href="https://web.archive.org/web/20221025114123/https://www.sogou.com/web?query=">网页</a></li><li><a href="https://web.archive.org/web/20221025114123/https://weixin.sogou.com/weixin?p=75351201">微信</a></li><li><a href="https://web.archive.org/web/20221025114123/https://zhihu.sogou.com/zhihu?p=75351218">知乎</a></li><li><a href="https://web.archive.org/web/20221025114123/https://pic.sogou.com/pics?query=">图片</a></li><li><a href="https://web.archive.org/web/20221025114123/https://v.sogou.com/v?query=">视频</a></li><li><a href="https://web.archive.org/web/20221025114123/https://mingyi.sogou.com/">医疗</a></li><li class="cur"><strong>科学</strong></li><li><a href="https://web.archive.org/web/20221025114123/https://hanyu.sogou.com/">汉语</a></li><li><a href="https://web.archive.org/web/20221025114123/https://wenwen.sogou.com/">问问</a></li><li><a href="https://web.archive.org/web/20221025114123/https://www.sogou.com/docs/more.htm">更多<span class="topraquo">»</span></a></li></ul></div><div id="header"><div class="header-wrap"><a class="header-logo" href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue"></a><div class="header-search"><div class="querybox" id="suggBox"><form><input id="searchInput" class="query" type="text" placeholder="搜科学领域专业百科词条" name="query" autocomplete="off" value=""><a href="javascript:;" class="query-search"></a></form></div></div><div class="header-rgt"><span class="btn-header-rgt btn-edit" id="editLemma">创建</span><div class="header-user no-login"></div></div></div></div><div class="fixed-placeholder" style="visibility:none"></div><div id="container" class=""><div class="content lemma-level1"><div class="detail-title" id="abstract-title"><h1>Rosetta@home</h1><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#!" class="detail-edit">编辑</a></div><div class="section_content" data-id="14995465208594957"><div><p>Rosetta@home是一个分布式计算项目，用于在伯克利开放式网络计算基础设施(BOINC)平台上进行蛋白质结构预测，由华盛顿大学贝克实验室运行。Rosetta Rosetta@home旨在预测蛋白质-蛋白质对接，并借助平均超过210万亿次（截至2016年7月29日）的约6万台主动志愿计算机来设计新蛋白质<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_1" class="kx_ref">[1]</a></sup>。 Foldit是一款Rosetta@Home视频游戏，旨在通过众包方式实现这些目标。尽管该项目的大部分面向基础研究，以提高蛋白质组学方法的准确性和稳健性，RosettaRosetta@home也在疟疾、阿尔茨海默病和其他病理学方面进行应用研究<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_2" class="kx_ref">[2]</a></sup>。 </p>
<p>像所有BOINC项目一样，Rosetta@home使用志愿者计算机中空闲的计算机处理资源来对单个工作单元进行计算。完成的结果被发送到中央项目服务器，在那里它们被验证并被吸收到项目数据库中。该项目是跨平台的，运行在多种硬件配置上。用户可以在Rosetta@home屏幕保护程序上查看他们个人蛋白质结构预测的进展。 </p>
<p>除了与疾病相关的研究之外，Rosetta Rosetta@home网络还是结构生物信息学新方法的测试框架。在Rosetta Rosetta@home的大型多样的志愿者电脑上充分开发并证明稳定后，这些方法被用于其他基于RosettaRosetta的应用，如Rosetta多克和人类蛋白质组折叠项目。Rosetta Rosetta@home开发的新方法的两个特别重要的测试是蛋白质结构预测技术的关键评估(CASP)和相互作用预测的关键评估(CAPRI)实验，这两个实验分别评估蛋白质结构预测和蛋白质-蛋白质对接预测的最新技术。Rosetta@home始终是最重要的停靠预测因子之一，也是现有的最佳三级结构预测因子之一<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_3" class="kx_ref">[3]</a></sup>。 </p></div></div><div id="catalog"><h2 class="title2">目录<a href="javascript:" class="detail-edit">编辑</a></h2><div class="catalog_wrap" style=""><ul class="catalog_list col3"><li><span class="order">1</span><a href="javascript:" data-level="1" data-id="14995465225372160">计算平台</a></li><li><span class="order">2</span><a href="javascript:" data-level="1" data-id="14995465225372161">项目意义</a></li><li><span class="order">3</span><a href="javascript:" data-level="1" data-id="14995465225372162">疾病相关研究</a></li><li class="secondary_catalog"><span>3.1 </span><a href="javascript:" data-id="14995465225372162">阿尔次海默病</a></li><li class="secondary_catalog"><span>3.2 </span><a href="javascript:" data-id="14995465225372162">炭疽</a></li><li class="secondary_catalog"><span>3.3 </span><a href="javascript:" data-id="14995465225372162">单纯疱疹病毒1</a></li><li class="secondary_catalog"><span>3.4 </span><a href="javascript:" data-id="14995465225372162">艾滋病毒</a></li></ul><ul class="catalog_list col3"><li class="secondary_catalog"><span>3.5 </span><a href="javascript:" data-id="14995465225372162">疟疾</a></li><li class="secondary_catalog"><span>3.6 </span><a href="javascript:" data-id="14995465225372162">其他疾病</a></li><li><span class="order">4</span><a href="javascript:" data-level="1" data-id="14995465225372163">发展历史和分支</a></li><li class="secondary_catalog"><span>4.1 </span><a href="javascript:" data-id="14995465225372163">Rosetta设计</a></li><li class="secondary_catalog"><span>4.2 </span><a href="javascript:" data-id="14995465225372163">Rosetta多克</a></li><li class="secondary_catalog"><span>4.3 </span><a href="javascript:" data-id="14995465225372163">罗伯塔（Robetta）</a></li><li class="secondary_catalog"><span>4.4 </span><a href="javascript:" data-id="14995465225372163">Foldit</a></li></ul><ul class="catalog_list col3"><li><span class="order">5</span><a href="javascript:" data-level="1" data-id="14995465242149376">与类似分布式计算项目的比较</a></li><li class="secondary_catalog"><span>5.1 </span><a href="javascript:" data-id="14995465242149376">Foldit@home</a></li><li class="secondary_catalog"><span>5.2 </span><a href="javascript:" data-id="14995465242149376">世界社区网格</a></li><li class="secondary_catalog"><span>5.3 </span><a href="javascript:" data-id="14995465242149376">Predictor@home</a></li><li><span class="order">6</span><a href="javascript:" data-level="1" data-id="14995465242149377">志愿者贡献</a></li><li><span class="order">7</span><a href="javascript:" data-level="1" data-id="references">参考文献</a></li></ul></div></div><div id="paragraphs"><div><div id="par_14995465225372160"><h2 class="title">1 计算平台<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>Rosetta@home应用程序和BOINC分布式计算平台可用于Windows操作系统、Linux和macOS ，BOINC还在其他几个平台上运行，例如FreeBSD<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_4" class="kx_ref">[4]</a></sup>。参与Rosetta Rosetta@home需要一个时钟速度至少为500兆赫的中央处理器、200兆字节的可用磁盘空间、512兆字节的物理内存和互联网连接<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_5" class="kx_ref">[5]</a></sup>。截至2016年7月20日，Rosetta迷你应用程序的当前版本为3.73<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_6" class="kx_ref">[6]</a></sup>。目前推荐的BOINC程序版本是7.6.22<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_4" class="kx_ref">[4]</a></sup>。标准超文本传输协议(HTTP)(端口80)用于用户的BOINC客户端和华盛顿大学的Rosetta Rosetta@home服务器之间的通信；密码交换时使用HTTPS(端口443)。BOINC客户端的远程和本地控制使用端口31416和端口1043，如果它们在防火墙后，可能需要特别解锁<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_7" class="kx_ref">[7]</a></sup>。包含单个蛋白质数据的工作单元被从位于华盛顿大学贝克实验室的服务器分配到志愿者的计算机上，然后志愿者的计算机计算指定蛋白质的结构预测。为了避免对给定蛋白质的重复结构预测，每个工作单元都用随机的种子号初始化。这给了每个预测一个沿着蛋白质能量景观的独特下降轨迹<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_8" class="kx_ref">[8]</a></sup>。Rosetta Rosetta@home的蛋白质结构预测是给定蛋白质能量格局中全球最小值的近似值。该全局最小值代表蛋白质最有利的构象，即其天然状态。 </p> 
<p></p><p></p><div class="text_img ed_imgfloat_right">
            <a class="ed_image_link lazyLoad" data-src="https://img04.sogoucdn.com/app/a/200698/sogou_science_13808" data-bigsrc="" title="点击查看大图" href="javascript:" data-observer="true"></a>
            <div class="text_img_title">Rosetta@home屏幕保护程序，显示了一个合成泛素蛋白质（PDB ID: 1ogw）的结构预测进行情况。</div>   
        </div> <p></p><p></p> 
<p>Rosetta@home图形用户界面(图形用户界面)的一个主要功能是屏幕保护程序，显示当前工作单元在模拟蛋白质折叠过程中的进度。在当前屏幕保护程序的左上角，目标蛋白质在寻找最低能量结构时采用不同的形状(构象)。右边是最近被接受的结构。右上角显示了当前诱饵的最低能量构象；下面是蛋白质的真实或天然结构，如果已经确定的话。屏幕保护程序中包含三个图形。在中间附近，显示了一个可接受模型的热力学自由能图，该图随着可接受模型的变化而波动。被接受的模型的均方根偏差(RMSD)图显示在最右边，该图衡量了被接受的模型与本地模型在结构上的相似程度。在可接受的能量图的右侧和RMSD图的下方，随着模型的逐步完善，这两个函数的结果被用于产生能量与RMSD图<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_9" class="kx_ref">[9]</a></sup>。 </p>
<p>像所有BOINC项目一样，Rosetta@home在用户计算机的后台运行，在登录主机操作系统的帐户时或之前使用空闲的计算机资源。当其他应用程序需要资源时，该程序从中央处理器中释放资源，这样就不会影响正常的计算机使用。许多程序设置可以通过用户帐户首选项来指定，包括:程序可以使用的最大CPU资源百分比(控制以持续容量运行的计算机的功耗或热量产生)、程序可以运行的时间等等。 </p>
<p>运行在Rosetta@home网络上的软件Rosetta用C++重写，以允许比用Fortran编写的原始版本更容易的开发。这个新版本是面向对象的，于2008年2月8日发布<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_6" class="kx_ref">[6]</a></sup><sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_10" class="kx_ref">[10]</a></sup> 。RosettaRosetta代码的开发是由RosettaRosetta公共组织完成的<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_11" class="kx_ref">[11]</a></sup>。该软件免费授权给学术界，制药公司付费使用<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_11" class="kx_ref">[11]</a></sup>。 </p></div></div><div id="par_14995465225372161"><h2 class="title">2 项目意义<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>随着基因组测序项目的增多，科学家可以推断出许多在细胞内发挥功能的蛋白质的氨基酸序列或一级结构。为了更好地理解蛋白质的功能并帮助合理的药物设计，科学家需要知道蛋白质的三维三级结构。 </p> 
<p></p><p></p><div class="text_img ed_imgfloat_right">
            <a class="ed_image_link lazyLoad" data-src="https://img04.sogoucdn.com/app/a/200698/sogou_science_13809" data-bigsrc="" title="点击查看大图" href="javascript:" data-observer="true"></a>
            <div class="text_img_title">CASP6的目标蛋白质T0281：首个接近原子级精度的ab initio蛋白质结构预测。Rosetta生成了T0281（洋红色）的一个模型，与实验测定的晶体结构（蓝色）之间的RMSD为1.5埃。</div>   
        </div> <p></p><p></p> 
<p>蛋白质三维结构目前是通过x光结晶学或核磁共振波谱实验确定的。这个过程很慢(可能需要几周甚至几个月才能首次弄清楚如何结晶蛋白质)，而且成本很高(大约每种蛋白质10万美元)<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_12" class="kx_ref">[12]</a></sup>。不幸的是，新序列的发现速度远远超过了结构确定的速度——在国家生物技术信息中心(NCBI)非冗余蛋白质数据库中现有的740万个蛋白质序列中，只有不到52000个蛋白质的三维结构被解决并储存在蛋白质数据库中，蛋白质数据库是蛋白质结构信息的主要储存库<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_13" class="kx_ref">[13]</a></sup>。 RosettaRosetta@home的主要目标之一是以与现有方法相同的准确度预测蛋白质结构，但需要的时间和金钱要少得多。RosettaRosetta@home还开发了一些方法来确定膜蛋白(如G蛋白偶联受体<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_14" class="kx_ref">[14]</a></sup>的结构和对接，这些方法很难用传统技术如x光结晶学和核磁共振波谱进行分析，但却代表了现代药物的大多数靶标。 </p>
<p>蛋白质结构预测的进展在半年一次的蛋白质结构预测技术关键评估(CASP)实验中进行评估，来自世界各地的研究人员试图从蛋白质的氨基酸序列中推导出蛋白质的结构。在这个有时具有竞争性的实验中，高分群体被认为是蛋白质结构预测的事实上的标准承担者。Rosetta是Rosetta@home的基础程序，自2002年CASP5以来一直在使用。在2004年CASP6实验中，RosettaRosetta在提交的CASP靶T0281模型中首次提出了接近原子水平的分辨率、从头计算蛋白质结构预测，创造了历史<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_15" class="kx_ref">[15]</a></sup>。从头建模被认为是蛋白质结构预测的一个特别困难的类别，因为它不使用来自结构同源性的信息，并且必须依赖来自序列同源性的信息和建模蛋白质内的物理相互作用。Rosetta@home自2006年以来一直用于CASP，在CASP7中，它是每一类结构预测的顶级预测因子之一<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_16" class="kx_ref">[16]</a></sup><sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_17" class="kx_ref">[17]</a></sup><sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_18" class="kx_ref">[18]</a></sup>。这些高质量的预测是由RosettaRosetta@Home志愿者提供的计算能力实现的<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_19" class="kx_ref">[19]</a></sup>。计算能力的提高使得Rosetta Rosetta@home能够对构象空间的更多区域(蛋白质可能呈现的形状)进行采样，根据莱文塔尔悖论，预计构象空间将随着蛋白质长度呈指数级增长。 </p>
<p>Rosetta@home也用于蛋白质-蛋白质对接预测，它决定了多种复合蛋白质的结构或四级结构。这种蛋白质相互作用影响许多细胞功能，包括抗原抗体和酶抑制剂结合以及细胞进出口。确定这些相互作用对药物设计至关重要。RosettaRosetta用于相互作用预测的临界评估(CAPRI)实验，该实验评估蛋白质对接区的状态，类似于CASP如何测量蛋白质结构预测的进展。Rosetta Rosetta@home项目志愿者提供的计算能力被认为是RosettaRosetta在CAPRI的表现的一个主要因素，它的对接预测是最准确和完整的<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_20" class="kx_ref">[20]</a></sup>。 </p>
<p>2008年初，RosettaRosetta被用于计算设计一种蛋白质，其功能在自然界中从未被观察到<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_21" class="kx_ref">[21]</a></sup>。 这部分是受2004年一篇高调论文的撤回启发，该论文最初描述了一种蛋白质的计算设计，该蛋白质相对于其天然形式具有更高的酶活性<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_22" class="kx_ref">[22]</a></sup>。大卫·贝克小组2008年的研究论文描述了蛋白质是如何制造的，该论文引用了Rosetta Rosetta@home提供的计算资源，代表了这种蛋白质设计方法概念的重要证明<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_21" class="kx_ref">[21]</a></sup>。 这种蛋白质设计可能在药物发现、绿色化学和生物修复方面有未来的应用<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_21" class="kx_ref">[21]</a></sup>。 </p></div></div><div id="par_14995465225372162"><h2 class="title">3 疾病相关研究<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>除了预测蛋白质结构、对接和设计的基础研究之外，Rosetta@home还用于直接疾病相关的研究<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_23" class="kx_ref">[23]</a></sup>。大卫·贝克的《Rosetta Rosetta@Home杂志》描述了许多小型研究项目<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_24" class="kx_ref">[24]</a></sup>。截至2014年2月，论坛正在更新关于最近出版物的信息和对工作的简短描述<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_25" class="kx_ref">[25]</a></sup>。论坛帖子自2016年起不再使用，研究的新闻可以在项目的一般新闻部分找到<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_26" class="kx_ref">[26]</a></sup>。 </p> 
<h3>3.1 <span>阿尔次海默病</span></h3> 
<p>RosettaRosetta软件套件的一个组件，RosettaRosetta设计，用于准确预测淀粉样蛋白的哪些区域最有可能形成淀粉样纤维<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_27" class="kx_ref">[27]</a></sup>。通过提取感兴趣蛋白质的六肽(六个氨基酸长的片段)，并选择与已知形成六肽的原纤维的结构相似的最低能量匹配，Rosetta设计能够识别出形成原纤维的可能性是随机蛋白质的两倍的肽<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_28" class="kx_ref">[28]</a></sup>。Rosetta Rosetta@home在同一项研究中被用来预测淀粉样β蛋白的结构，淀粉样β蛋白是一种纤维形成蛋白，被认为会导致阿尔茨海默病<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_29" class="kx_ref">[29]</a></sup>。RosettaRosetta设计的蛋白质已经产生了初步但尚未公布的结果，这些蛋白质可以防止原纤维的形成，尽管还不知道它是否能预防这种疾病<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_30" class="kx_ref">[30]</a></sup>。 </p> 
<h3>3.2 <span>炭疽</span></h3> 
<p>Rosetta的另一个组成部分，Rosetta多克<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_31" class="kx_ref">[31]</a></sup><sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_32" class="kx_ref">[32]</a></sup><sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_33" class="kx_ref">[33]</a></sup> ，被用于结合实验方法来模拟构成炭疽毒素的三种蛋白质之间的相互作用——致死因子(LF)、水肿因子(EF)和保护性抗原(PA)。计算机模型准确预测了低频和磷酸之间的对接，有助于确定各自蛋白质的哪些结构域参与低频-磷酸复合物。这一见解最终被用于改进炭疽疫苗的研究<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_34" class="kx_ref">[34]</a></sup><sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_35" class="kx_ref">[35]</a></sup>。 </p> 
<h3>3.3 <span>单纯疱疹病毒1</span></h3> 
<p>罗赛多克用于模拟抗体(免疫球蛋白G)和由感冒疮病毒，单纯疱疹病毒1 (HSV-1)表达的表面蛋白之间的对接，后者用于降解抗病毒抗体。Rosetta多克预测的蛋白质复合物与特别难以获得的实验模型非常一致，导致研究人员得出结论，对接方法有可能解决x光结晶学在蛋白质-蛋白质界面建模方面的一些问题<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_36" class="kx_ref">[36]</a></sup>。 </p> 
<h3>3.4 <span>艾滋病毒</span></h3> 
<p>作为比尔及梅林达盖茨基金会1940万美元赠款资助的研究的一部分<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_37" class="kx_ref">[37]</a></sup>，Rosetta@home已被用于设计多种可能的人类免疫缺陷病毒疫苗<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_38" class="kx_ref">[38]</a></sup><sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_39" class="kx_ref">[39]</a></sup>。 </p> 
<h3>3.5 <span>疟疾</span></h3> 
<p>在与全球健康大挑战倡议有关的研究中<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_40" class="kx_ref">[40]</a></sup>，Rosetta被用于计算设计新的归巢核酸内切酶蛋白，这种蛋白可以消灭冈比亚按蚊或使蚊子无法传播疟疾<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_41" class="kx_ref">[41]</a></sup>。 像归巢核酸内切酶一样，能够专门模拟和改变蛋白质-脱氧核糖核酸的相互作用，这使得像Rosetta这样的计算蛋白质设计方法在基因治疗(包括可能的癌症治疗)中发挥了重要作用<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_23" class="kx_ref">[23]</a></sup><sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_42" class="kx_ref">[42]</a></sup>。 </p> 
<h3>3.6 <span>其他疾病</span></h3> 
<p>Rosetta@home研究人员设计了一种称为新白细胞介素-2/15的白细胞介素-2受体激动剂，它不与受体的α亚单位相互作用。这种免疫信号分子可用于癌症治疗。虽然天然白细胞介素-2因与α亚单位相互作用而中毒，但设计的蛋白质更安全，至少在动物模型中是如此<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_43" class="kx_ref">[43]</a></sup>。 </p></div></div><div id="par_14995465225372163"><h2 class="title">4 发展历史和分支<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>最初由贝克实验室在1998年引入，作为结构预测的一种从头开始的方法<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_44" class="kx_ref">[44]</a></sup> ，Rosetta后来已经扩展到几个开发流程和不同的服务。Rosetta平台得名于Rosetta石碑，因为它试图破译蛋白质氨基酸序列的结构“意义”<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_45" class="kx_ref">[45]</a></sup>。Rosetta首次亮相七年多后，Rosetta@home项目于2005年10月6日发布(即宣布不再测试)<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_6" class="kx_ref">[6]</a></sup>。参与Rosetta项目初期开发的许多研究生和其他研究人员后来搬到了其他大学和研究机构，并随后加强了Rosetta项目的不同部分。 </p> 
<h3>4.1 <span>Rosetta设计</span></h3> 
<p></p><p></p><div class="text_img ed_imgfloat_right">
            <a class="ed_image_link lazyLoad" data-src="https://img01.sogoucdn.com/app/a/200698/sogou_science_13810" data-bigsrc="" title="点击查看大图" href="javascript:" data-observer="true"></a>
            <div class="text_img_title">Rosetta设计的TOP7模型（红色）与其X射线晶体结构（蓝色，PDB ID: 1QYS）的叠合。</div>   
        </div> <p></p><p></p> 
<p>Rosetta设计(RosettaDesign)，一种基于Rosetta的蛋白质设计计算方法，始于2000年的一项重新设计蛋白质折叠途径的研究<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_46" class="kx_ref">[46]</a></sup>。2002年，Rosetta设计被用于设计Top7，一种93个氨基酸长的α/β蛋白质，其整体折叠在自然界中从未有过。Rosetta预测这种新的构象在由x光晶体学确定的结构的1.2 RMSD以内，代表了异常精确的结构预测<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_47" class="kx_ref">[47]</a></sup>。 Rosetta和Rosetta设计获得了广泛的认可，因为它们是第一个设计和准确预测这种长度的新蛋白质结构的人，正如2002年在《科学》<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_48" class="kx_ref">[48]</a></sup><sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_49" class="kx_ref">[49]</a></sup> 杂志上发表的描述双重方法的论文所反映的那样，被超过240篇其他科学文章引用<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_50" class="kx_ref">[50]</a></sup>。 该项研究的有形产品Top7在2006年10月被评为RCSB·PDB的“月度分子”<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_51" class="kx_ref">[51]</a></sup>； Rosetta@home标志展示了其预测晶体结构和x光晶体结构各自核心(残基60–79)的叠加<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_15" class="kx_ref">[15]</a></sup>。 </p>
<p>布莱恩·库尔曼(Brian Kuhlman)，大卫·贝克实验室前博士后研究员，现为[教堂山北卡罗来纳大学副教授<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_52" class="kx_ref">[52]</a></sup>， 提供Rosetta设计在线服务<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_53" class="kx_ref">[53]</a></sup>。 </p> 
<h3>4.2 <span>Rosetta多克</span></h3> 
<p>Rosetta多克是在2002年第一次CAPRI实验中作为贝克实验室的蛋白质-蛋白质对接预测算法添加到Rosetta软件套件中的<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_54" class="kx_ref">[54]</a></sup>。在该实验中，Rosetta多克对链球菌化脓性外毒素A和T细胞受体β链之间的对接进行了高精度预测，并对猪α-淀粉酶和骆驼抗体之间的复合物进行了中等精度预测。虽然Rosetta多克方法在七种可能的预测中仅做出了两种可以接受的准确预测，但这足以在第一次CAPRI评估中将其列为十九种预测方法中的第七种<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_54" class="kx_ref">[54]</a></sup>。 </p>
<p>Rosetta多克的开发在随后的CAPRI回合中分成了两个分支，杰弗里·格雷在华盛顿大学为Rosetta多克打下了基础，他在约翰·霍普金斯大学的新职位上继续研究这种方法。贝克实验室的成员在格雷不在时进一步开发了Rosetta多克。这两个版本在侧链建模、诱饵选择和其他方面略有不同<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_33" class="kx_ref">[33]</a></sup><sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_55" class="kx_ref">[55]</a></sup>。尽管存在这些差异，贝克和格雷方法在第二次CAPRI评估中表现良好，分别在30个预测组中排名第五和第七<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_56" class="kx_ref">[56]</a></sup>。杰弗里·格雷(Jeffrey Gray)的Rosetta多克服务器是一项免费对接预测服务，供非商业用途<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_57" class="kx_ref">[57]</a></sup>。 </p>
<p>2006年10月，Rosetta多克被整合到Rosetta@home。该方法仅使用蛋白质骨架，使用快速、粗略的对接模型阶段。随后是缓慢的全原子精细化阶段，其中两种相互作用蛋白质相对于彼此的取向以及蛋白质-蛋白质界面的侧链相互作用被同时优化，以找到最低能量构象<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_58" class="kx_ref">[58]</a></sup>。Rosetta@家庭网络提供的计算能力大幅提高，加上主干灵活性和环路建模的修正折叠树表示法，使Rosetta多克在第三次CAPRI评估的63个预测组中排名第六<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_3" class="kx_ref">[3]</a></sup><sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_20" class="kx_ref">[20]</a></sup>。 </p> 
<h3>4.3 <span>罗伯塔（Robetta）</span></h3> 
<p>Rosetta(Rosetta Beta)服务器是贝克实验室为非商业从头计算和比较建模提供的自动化蛋白质结构预测服务<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_59" class="kx_ref">[59]</a></sup>。自2002年CASP5以来，它作为自动预测服务器参加了半年一次的CASP实验，在自动服务器预测类别中表现最佳<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_60" class="kx_ref">[60]</a></sup>。 Robetta此后参加了CASP6和7的比赛，在这两个项目中，自动化服务器组和人工预测组的表现均优于平均水平<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_18" class="kx_ref">[18]</a></sup><sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_61" class="kx_ref">[61]</a></sup><sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_62" class="kx_ref">[62]</a></sup>。它还参与CAMEO3D连续评估。 </p>
<p>在对CASP6的蛋白质结构建模时，Robetta首先使用BLAST、PSI-BLAST和3D-Jury搜索结构同源物，然后通过将序列与Pfam数据库中的结构家族相匹配，将目标序列解析为其单个结构域，或独立折叠的蛋白质单位。然后，具有结构同源物的结构域遵循“基于模板的模型”(即同源建模)方案。在这里，贝克实验室的内部比对程序，K*sync，产生一组序列同源物，每一个序列同源物都用Rosetta从头方法建模，以产生诱饵(可能的结构)。通过采用由低分辨率Rosetta能量函数确定的最低能量模型来选择最终的结构预测。对于没有检测到结构同源物的区域，遵循从头协议，其中从一组生成的诱饵中选择最低能量模型作为最终预测。然后将这些结构域预测连接在一起，研究蛋白质内的结构域间、三级相互作用。最后，使用蒙特卡罗构象搜索协议对侧链贡献进行建模<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_63" class="kx_ref">[63]</a></sup>。 </p>
<p>在CAPS8中，Rosetta被扩充以使用Rosetta的高分辨率全原子精细化方法<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_64" class="kx_ref">[64]</a></sup>， 缺少这种方法被认为是Rosetta不如CAPS7中Rosetta@Home网络精确的主要原因<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_19" class="kx_ref">[19]</a></sup>。 在CASP11中，增加了一种通过相关蛋白质残基的共同进化来预测蛋白质接触图的方法，称为GREMLIN，这使得更多的新的成功成为可能<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_65" class="kx_ref">[65]</a></sup>。 </p> 
<h3>4.4 <span>Foldit</span></h3> 
<p>2008年5月9日，在Rosetta@home用户提出分布式计算程序的交互式版本后，贝克实验室公开发布了Foldit，这是一款基于Rosetta平台的在线蛋白质结构预测游戏<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_66" class="kx_ref">[66]</a></sup>。截至2008年9月25日，Foldit拥有超过59，000名注册用户<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_67" class="kx_ref">[67]</a></sup>。该游戏给用户提供了一套控制(例如摇动、摆动、重建)来操纵目标蛋白的主链和氨基酸侧链，使其形成更具能量优势的构象。用户可以作为独唱者单独工作，也可以作为进化者集体工作，当他们改进他们的结构预测时，在两个类别下累积点数<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_68" class="kx_ref">[68]</a></sup>。用户也可以通过决斗功能单独与其他用户竞争，在决斗功能中，20次移动后能量结构最低的玩家获胜。 </p></div></div><div id="par_14995465242149376"><h2 class="title">5 与类似分布式计算项目的比较<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>有几个分布式计算项目的研究领域与Rosetta@home类似，但研究方法不同: </p> 
<h3>5.1 <span>Foldit@home</span></h3> 
<p>在所有涉及蛋白质研究的主要分布式计算项目中，Folding@home是唯一一个没有使用BOINC平台的项目<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_69" class="kx_ref">[69]</a></sup><sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_70" class="kx_ref">[70]</a></sup><sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_71" class="kx_ref">[71]</a></sup>。Rosetta“家”和折叠“家”都研究蛋白质错误折叠疾病，如阿尔茨海默氏病，但折叠“家”做得更多<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_72" class="kx_ref">[72]</a></sup><sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_73" class="kx_ref">[73]</a></sup>。Foldit@home几乎完全使用全原子分子动力学模型来理解蛋白质折叠(或潜在错误折叠，随后聚集导致疾病)的方式和原因<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_74" class="kx_ref">[74]</a></sup><sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_75" class="kx_ref">[75]</a></sup>。换句话说，Folding@home的强项是模拟蛋白质折叠的过程，而Rosetta@home的强项是计算蛋白质设计和预测蛋白质结构和对接。 </p>
<p>Rosetta“家”的一些结果被用作一些Foldit@Home项目的基础。Rosetta提供了最有可能的结构，但是还不确定这是不是分子的形式或者它是否可行。Foldit@home然后可以用来验证Rosetta@home的结果，并且可以提供附加的原子级信息，以及分子如何改变形状的细节<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_75" class="kx_ref">[75]</a></sup><sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_76" class="kx_ref">[76]</a></sup>。 </p>
<p>这两个项目在计算能力和主机多样性方面也有显著差异<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_77" class="kx_ref">[77]</a></sup>，Foldit@Home的计算能力几乎是Rosetta“家庭”的108倍<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_1" class="kx_ref">[1]</a></sup>。 </p> 
<h3>5.2 <span>世界社区网格</span></h3> 
<p>人类蛋白质组折叠项目(HPF)的第一阶段和第二阶段(世界社区网格的一个子项目)都使用Rosetta程序对各种基因组进行结构和功能注释<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_78" class="kx_ref">[78]</a></sup><sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_79" class="kx_ref">[79]</a></sup>。尽管他现在用它为生物学家创建数据库，但人类蛋白质组折叠项目的首席科学家理查德·博诺(Richard Bonneau)在获得博士学位的同时，积极参与了大卫·贝克实验室Rosetta的最初开发<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_80" class="kx_ref">[80]</a></sup>。关于HPF1、HPF2和Rosetta@home之间关系的更多信息，请访问理查德·博诺的网站<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_81" class="kx_ref">[81]</a></sup>。 </p> 
<h3>5.3 <span>Predictor@home</span></h3> 
<p>像Rosetta“家”一样，Predictor@Home专门从事蛋白质结构预测<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_82" class="kx_ref">[82]</a></sup>。当Rosetta@home使用Rosetta程序进行结构预测时，Predictor@home使用数据断言方法<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_83" class="kx_ref">[83]</a></sup>。2009年，Predictor@home关闭。 </p>
<p>BOINC上其他与蛋白质相关的分布式计算项目包括：QMC@home, Docking@home, POEM@home、SIMAP和TANPAKU。RALPH@home，Rosetta@home alpha项目，在新的应用程序版本、工作单元和更新进入Rosetta@home之前进行测试，也在BOINC上运行<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_84" class="kx_ref">[84]</a></sup>。 </p></div></div><div id="par_14995465242149377"><h2 class="title">6 志愿者贡献<a href="javascript:" class="detail-edit">编辑</a></h2><div class="section_content"><p>Rosetta@home的研究依赖于各个项目成员捐赠的计算能力。截至2014年4月18日，来自150个国家的约26，900名用户是Rosetta@home的活跃成员，他们共同贡献了约66，000台计算机的空闲处理器时间，平均性能超过83万亿次<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_1" class="kx_ref">[1]</a></sup>。 </p> 
<p></p><p></p><div class="text_img ed_imgfloat_right">
            <a class="ed_image_link lazyLoad" data-src="https://img02.sogoucdn.com/app/a/200698/sogou_science_13811" data-bigsrc="" title="点击查看大图" href="javascript:" data-observer="true"></a>
            <div class="text_img_title">Rosetta@home在CASP8实验期间的用户贡献柱状统计图</div>   
        </div> <p></p><p></p> 
<p>用户被授予BOINC学分作为他们贡献的衡量标准。给予每个工作单位的信用是为该工作单位生产的诱饵数量乘以所有计算机主机为该工作单位提交的诱饵的平均索赔信用。该定制系统旨在解决标准BOINC客户端和优化BOINC客户端授予用户的信用之间的显著差异，以及在视窗和Linux操作系统上运行Rosetta@home的用户之间的信用差异<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_85" class="kx_ref">[85]</a></sup>。Rosetta@home的每秒中央处理器工作的信贷额度低于大多数其他BOINC项目<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_86" class="kx_ref">[86]</a></sup>。就信贷总额而言，Rosetta@home在40多个投资理事会项目中排名第十三<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_87" class="kx_ref">[87]</a></sup>。 </p>
<p>Rosetta@home用户预测提交给CASP实验的蛋白质结构，其结果在科学出版物中得到认可<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_19" class="kx_ref">[19]</a></sup>。在Rosetta@主页上，预测给定工作单位最低能量结构的用户以及他们所属的任何团队都被列为当天的预测者<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_88" class="kx_ref">[88]</a></sup>。每天都从创建Rosetta@home档案的用户中随机选择一名每日用户，并将其放在主页上<sup><a href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/d10696.htm#quote_89" class="kx_ref">[89]</a></sup>。 </p></div></div></div></div><div id="references"><h2 class="title" id="par_references">参考文献</h2><ul class="references"><li id="quote_1"><span class="references-num">[1]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">de Zutter W. "Rosetta@home: Credit overview". boincstats.com. Archived from the original on October 7, 2011. Retrieved October 18, 2011..</span></p></li><li id="quote_2"><span class="references-num">[2]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"What is Rosetta@home?". Rosetta@home forums. University of Washington. Archived from the original on September 13, 2008. Retrieved September 7, 2008..</span></p></li><li id="quote_3"><span class="references-num">[3]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Lensink MF, Méndez R, Wodak SJ (December 2007). "Docking and scoring protein complexes: CAPRI 3rd Edition". Proteins. 69 (4): 704–18. doi:10.1002/prot.21804. PMID 17918726..</span></p></li><li id="quote_4"><span class="references-num">[4]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Download BOINC client software". BOINC. University of California. 2008. Retrieved December 1, 2008..</span></p></li><li id="quote_5"><span class="references-num">[5]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Rosetta@home: Recommended System Requirements". Rosetta@home. University of Washington. 2008. Archived from the original on September 25, 2008. Retrieved October 7, 2008..</span></p></li><li id="quote_6"><span class="references-num">[6]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Rosetta@home: News archive". Rosetta@home. University of Washington. 2016. Retrieved July 20, 2016..</span></p></li><li id="quote_7"><span class="references-num">[7]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Rosetta@home: FAQ (work in progress) (message 10910)". Rosetta@home forums. University of Washington. 2006. Retrieved October 7, 2008..</span></p></li><li id="quote_8"><span class="references-num">[8]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Kim DE (2005). "Rosetta@home: Random Seed (message 3155)". Rosetta@home forums. University of Washington. Retrieved October 7, 2008..</span></p></li><li id="quote_9"><span class="references-num">[9]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Rosetta@home: Quick guide to Rosetta and its graphics". Rosetta@home. University of Washington. 2007. Archived from the original on September 24, 2008. Retrieved October 7, 2008..</span></p></li><li id="quote_10"><span class="references-num">[10]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Kim DE (2008). "Rosetta@home: Problems with minirosetta version 1.+ (Message 51199)". Rosetta@home forums. University of Washington. Retrieved September 7, 2008..</span></p></li><li id="quote_11"><span class="references-num">[11]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Rosetta Commons". RosettaCommons.org. 2008. Archived from the original on September 15, 2008. Retrieved October 7, 2008..</span></p></li><li id="quote_12"><span class="references-num">[12]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Bourne PE, Helge W, eds. (2003). Structural Bioinformatics. Hoboken, NJ: Wiley-Liss. ISBN 978-0-471-20199-1. OCLC 50199108..</span></p></li><li id="quote_13"><span class="references-num">[13]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Yearly Growth of Protein Structures". RCSB Protein Data Bank. 2008. Retrieved November 30, 2008..</span></p></li><li id="quote_14"><span class="references-num">[14]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Baker D (2008). "Rosetta@home: David Baker's Rosetta@home journal (message 55893)". Rosetta@home forums. University of Washington. Retrieved October 7, 2008..</span></p></li><li id="quote_15"><span class="references-num">[15]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Rosetta@home: Research Overview". Rosetta@home. University of Washington. 2007. Archived from the original on September 25, 2008. Retrieved October 7, 2008..</span></p></li><li id="quote_16"><span class="references-num">[16]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Kopp J, Bordoli L, Battey JN, Kiefer F, Schwede T (2007). "Assessment of CASP7 predictions for template-based modeling targets". Proteins. 69 Suppl 8: 38–56. doi:10.1002/prot.21753. PMID 17894352..</span></p></li><li id="quote_17"><span class="references-num">[17]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Read RJ, Chavali G (2007). "Assessment of CASP7 predictions in the high accuracy template-based modeling category". Proteins. 69 Suppl 8: 27–37. doi:10.1002/prot.21662. PMID 17894351..</span></p></li><li id="quote_18"><span class="references-num">[18]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Jauch R, Yeo HC, Kolatkar PR, Clarke ND (2007). "Assessment of CASP7 structure predictions for template free targets". Proteins. 69 Suppl 8: 57–67. doi:10.1002/prot.21771. PMID 17894330..</span></p></li><li id="quote_19"><span class="references-num">[19]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Das R, Qian B, Raman S, et al. (2007). "Structure prediction for CASP7 targets using extensive all-atom refinement with Rosetta@home". Proteins. 69 Suppl 8: 118–28. doi:10.1002/prot.21636. PMID 17894356..</span></p></li><li id="quote_20"><span class="references-num">[20]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Wang C, Schueler-Furman O, Andre I, et al. (December 2007). "RosettaDock in CAPRI rounds 6–12". Proteins. 69 (4): 758–63. doi:10.1002/prot.21684. PMID 17671979..</span></p></li><li id="quote_21"><span class="references-num">[21]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Jiang L, Althoff EA, Clemente FR, et al. (March 2008). "De novo computational design of retro-aldol enzymes". Science. 319 (5868): 1387–91. Bibcode:2008Sci...319.1387J. doi:10.1126/science.1152692. PMC 3431203. PMID 18323453..</span></p></li><li id="quote_22"><span class="references-num">[22]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Hayden EC (February 13, 2008). "Protein prize up for grabs after retraction". Nature. doi:10.1038/news.2008.569..</span></p></li><li id="quote_23"><span class="references-num">[23]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Disease Related Research". Rosetta@home. University of Washington. 2008. Archived from the original on September 23, 2008. Retrieved October 8, 2008..</span></p></li><li id="quote_24"><span class="references-num">[24]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Baker D (2008). "Rosetta@home: David Baker's Rosetta@home journal". Rosetta@home forums. University of Washington. Retrieved September 7, 2008..</span></p></li><li id="quote_25"><span class="references-num">[25]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Rosetta@home Research Updates". Boinc.bakerlab.org. Retrieved 2014-04-18..</span></p></li><li id="quote_26"><span class="references-num">[26]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"News archive". Rosetta@home. Retrieved 10 May 2019..</span></p></li><li id="quote_27"><span class="references-num">[27]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Kuhlman B, Baker D (September 2000). "Native protein sequences are close to optimal for their structures". Proceedings of the National Academy of Sciences of the United States of America. 97 (19): 10383–88. Bibcode:2000PNAS...9710383K. doi:10.1073/pnas.97.19.10383. PMC 27033. PMID 10984534..</span></p></li><li id="quote_28"><span class="references-num">[28]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Thompson MJ, Sievers SA, Karanicolas J, Ivanova MI, Baker D, Eisenberg D (March 2006). "The 3D profile method for identifying fibril-forming segments of proteins". Proceedings of the National Academy of Sciences of the United States of America. 103 (11): 4074–78. Bibcode:2006PNAS..103.4074T. doi:10.1073/pnas.0511295103. PMC 1449648. PMID 16537487..</span></p></li><li id="quote_29"><span class="references-num">[29]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Bradley P. "Rosetta@home forum: Amyloid fibril structure prediction". Rosetta@home forums. University of Washington. Retrieved September 7, 2008..</span></p></li><li id="quote_30"><span class="references-num">[30]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Baker D. "Rosetta@home forum: Publications on R@H's Alzheimer's work? (message 54681)". Rosetta@home forums. University of Washington. Retrieved October 8, 2008..</span></p></li><li id="quote_31"><span class="references-num">[31]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Wang C, Schueler-Furman O, Baker D (May 2005). "Improved side-chain modeling for protein–protein docking". Protein Science. 14 (5): 1328–39. doi:10.1110/ps.041222905. PMC 2253276. PMID 15802647..</span></p></li><li id="quote_32"><span class="references-num">[32]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Gray JJ, Moughon S, Wang C, et al. (August 2003). "Protein–protein docking with simultaneous optimization of rigid-body displacement and side-chain conformations". Journal of Molecular Biology. 331 (1): 281–99. doi:10.1016/S0022-2836(03)00670-3. PMID 12875852..</span></p></li><li id="quote_33"><span class="references-num">[33]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Schueler-Furman O, Wang C, Baker D (August 2005). "Progress in protein–protein docking: atomic resolution predictions in the CAPRI experiment using RosettaDock with an improved treatment of side-chain flexibility". Proteins. 60 (2): 187–94. doi:10.1002/prot.20556. PMID 15981249..</span></p></li><li id="quote_34"><span class="references-num">[34]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Lacy DB, Lin HC, Melnyk RA, et al. (November 2005). "A model of anthrax toxin lethal factor bound to protective antigen". Proceedings of the National Academy of Sciences of the United States of America. 102 (45): 16409–14. Bibcode:2005PNAS..10216409L. doi:10.1073/pnas.0508259102. PMC 1283467. PMID 16251269..</span></p></li><li id="quote_35"><span class="references-num">[35]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Albrecht MT, Li H, Williamson ED, et al. (November 2007). "Human monoclonal antibodies against anthrax lethal factor and protective antigen act independently to protect against Bacillus anthracis infection and enhance endogenous immunity to anthrax". Infection and Immunity. 75 (11): 5425–33. doi:10.1128/IAI.00261-07. PMC 2168292. PMID 17646360..</span></p></li><li id="quote_36"><span class="references-num">[36]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Sprague ER, Wang C, Baker D, Bjorkman PJ (June 2006). "Crystal structure of the HSV-1 Fc receptor bound to Fc reveals a mechanism for antibody bipolar bridging". PLoS Biology. 4 (6): e148. doi:10.1371/journal.pbio.0040148. PMC 1450327. PMID 16646632..</span></p></li><li id="quote_37"><span class="references-num">[37]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Paulson, Tom (July 19, 2006). "Gates Foundation awards $287 million for HIV vaccine research". Seattle Post-Intelligencer. Retrieved September 7, 2008..</span></p></li><li id="quote_38"><span class="references-num">[38]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Liu Y, et al. (2007). "Development of IgG1 b12 scaffolds and HIV-1 env-based outer domain immunogens capable of eliciting and detecting IgG1 b12-like antibodies" (PDF). Global HIV Vaccine Enterprise. Archived from the original (PDF) on February 25, 2009. Retrieved September 28, 2008..</span></p></li><li id="quote_39"><span class="references-num">[39]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Baker D. "David Baker's Rosetta@home journal archives (message 40756)". Rosetta@home forums. University of Washington. Retrieved September 7, 2008..</span></p></li><li id="quote_40"><span class="references-num">[40]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Homing Endonuclease Genes: New Tools for Mosquito Population Engineering and Control". Grand Challenges in Global Health. Retrieved September 7, 2008..</span></p></li><li id="quote_41"><span class="references-num">[41]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Windbichler N, Papathanos PA, Catteruccia F, Ranson H, Burt A, Crisanti A (2007). "Homing endonuclease mediated gene targeting in Anopheles gambiae cells and embryos". Nucleic Acids Research. 35 (17): 5922–33. doi:10.1093/nar/gkm632. PMC 2034484. PMID 17726053..</span></p></li><li id="quote_42"><span class="references-num">[42]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Ashworth J, Havranek JJ, Duarte CM, et al. (June 2006). "Computational redesign of endonuclease DNA binding and cleavage specificity". Nature. 441 (7093): 656–59. Bibcode:2006Natur.441..656A. doi:10.1038/nature04818. PMC 2999987. PMID 16738662..</span></p></li><li id="quote_43"><span class="references-num">[43]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Silva DA, Yu S, Ulge UY, Spangler JB, Jude KM, Labão-Almeida C, Ali LR, Quijano-Rubio A, Ruterbusch M, Leung I, Biary T, Crowley SJ, Marcos E, Walkey CD, Weitzner BD, Pardo-Avila F, Castellanos J, Carter L, Stewart L, Riddell SR, Pepper M, Bernardes GJ, Dougan M, Garcia KC, Baker D (January 2019). "De novo design of potent and selective mimics of IL-2 and IL-15". Nature. 565 (7738): 186–191. doi:10.1038/s41586-018-0830-7. PMID 30626941..</span></p></li><li id="quote_44"><span class="references-num">[44]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Simons KT, Bonneau R, Ruczinski I, Baker D (1999). "Ab initio protein structure prediction of CASP III targets using Rosetta". Proteins. Suppl 3: 171–76. doi:10.1002/(SICI)1097-0134(1999)37:3+&lt;171::AID-PROT21&gt;3.0.CO;2-Z. PMID 10526365..</span></p></li><li id="quote_45"><span class="references-num">[45]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Interview with David Baker". Team Picard Distributed Computing. 2006. Archived from the original on February 18, 2009. Retrieved December 23, 2008..</span></p></li><li id="quote_46"><span class="references-num">[46]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Nauli S, Kuhlman B, Baker D (July 2001). "Computer-based redesign of a protein folding pathway". Nature Structural Biology. 8 (7): 602–05. doi:10.1038/89638. PMID 11427890..</span></p></li><li id="quote_47"><span class="references-num">[47]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Kuhlman B, Dantas G, Ireton GC, Varani G, Stoddard BL, Baker D (November 2003). "Design of a novel globular protein fold with atomic-level accuracy". Science. 302 (5649): 1364–68. Bibcode:2003Sci...302.1364K. doi:10.1126/science.1089427. PMID 14631033..</span></p></li><li id="quote_48"><span class="references-num">[48]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Jones DT (November 2003). "Structural biology. Learning to speak the language of proteins". Science. 302 (5649): 1347–48. doi:10.1126/science.1092492. PMID 14631028..</span></p></li><li id="quote_49"><span class="references-num">[49]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">von Grotthuss M, Wyrwicz LS, Pas J, Rychlewski L (June 2004). "Predicting protein structures accurately". Science. 304 (5677): 1597–99, author reply 1597–99. doi:10.1126/science.304.5677.1597b. PMID 15192202..</span></p></li><li id="quote_50"><span class="references-num">[50]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Articles citing: Kuhlman et al. (2003) 'Design of a novel globular protein fold with atomic-level accuracy'". ISI Web of Science. Retrieved July 10, 2008..</span></p></li><li id="quote_51"><span class="references-num">[51]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"October 2005 molecule of the month: Designer proteins". RCSB Protein Data Bank. Retrieved September 7, 2008..</span></p></li><li id="quote_52"><span class="references-num">[52]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Kuhlman laboratory homepage". Kuhlman Laboratory. University of North Carolina. Retrieved September 7, 2008..</span></p></li><li id="quote_53"><span class="references-num">[53]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"RosettaDesign web server". Kuhlman Laboratory. University of North Carolina. Retrieved September 7, 2008..</span></p></li><li id="quote_54"><span class="references-num">[54]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Gray JJ, Moughon SE, Kortemme T, et al. (July 2003). "Protein–protein docking predictions for the CAPRI experiment". Proteins. 52 (1): 118–22. CiteSeerX 10.1.1.80.9354. doi:10.1002/prot.10384. PMID 12784377..</span></p></li><li id="quote_55"><span class="references-num">[55]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Daily MD, Masica D, Sivasubramanian A, Somarouthu S, Gray JJ (2005). "CAPRI rounds 3–5 reveal promising successes and future challenges for RosettaDock". Proteins. 60 (2): 181–86. CiteSeerX 10.1.1.521.9981. doi:10.1002/prot.20555. PMID 15981262. Archived from the original on 2012-06-30..</span></p></li><li id="quote_56"><span class="references-num">[56]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Méndez R, Leplae R, Lensink MF, Wodak SJ (2005). "Assessment of CAPRI predictions in rounds 3–5 shows progress in docking procedures". Proteins. 60 (2): 150–69. doi:10.1002/prot.20551. PMID 15981261. Archived from the original on 2012-06-30..</span></p></li><li id="quote_57"><span class="references-num">[57]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"RosettaDock server". Gray laboratory. Johns Hopkins University. Retrieved September 7, 2008..</span></p></li><li id="quote_58"><span class="references-num">[58]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Protein–protein docking at Rosetta@home". Rosetta@home forums. University of Washington. Retrieved September 7, 2008..</span></p></li><li id="quote_59"><span class="references-num">[59]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Robetta web server". Baker laboratory. University of Washington. Retrieved May 7, 2019..</span></p></li><li id="quote_60"><span class="references-num">[60]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Aloy P, Stark A, Hadley C, Russell RB (2003). "Predictions without templates: new folds, secondary structure, and contacts in CASP5". Proteins. 53 Suppl 6: 436–56. doi:10.1002/prot.10546. PMID 14579333..</span></p></li><li id="quote_61"><span class="references-num">[61]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Tress M, Ezkurdia I, Graña O, López G, Valencia A (2005). "Assessment of predictions submitted for the CASP6 comparative modeling category". Proteins. 61 Suppl 7: 27–45. doi:10.1002/prot.20720. PMID 16187345..</span></p></li><li id="quote_62"><span class="references-num">[62]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Battey JN, Kopp J, Bordoli L, Read RJ, Clarke ND, Schwede T (2007). "Automated server predictions in CASP7". Proteins. 69 Suppl 8: 68–82. doi:10.1002/prot.21761. PMID 17894354..</span></p></li><li id="quote_63"><span class="references-num">[63]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Chivian D, Kim DE, Malmström L, Schonbrun J, Rohl CA, Baker D (2005). "Prediction of CASP6 structures using automated Robetta protocols". Proteins. 61 Suppl 7: 157–66. doi:10.1002/prot.20733. PMID 16187358..</span></p></li><li id="quote_64"><span class="references-num">[64]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Baker D. "David Baker's Rosetta@home journal, message 52902". Rosetta@home forums. University of Washington. Retrieved September 7, 2008..</span></p></li><li id="quote_65"><span class="references-num">[65]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Ovchinnikov, S; Kim, DE; Wang, RY; Liu, Y; DiMaio, F; Baker, D (September 2016). "Improved de novo structure prediction in CASP11 by incorporating coevolution information into Rosetta". Proteins. 84 Suppl 1: 67–75. doi:10.1002/prot.24974. PMID 26677056..</span></p></li><li id="quote_66"><span class="references-num">[66]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Baker D. "David Baker's Rosetta@home journal (message 52963)". Rosetta@home forums. University of Washington. Retrieved September 16, 2008..</span></p></li><li id="quote_67"><span class="references-num">[67]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Foldit forums: How many users does Foldit have? Etc. (message 2)". University of Washington. Retrieved September 27, 2008..</span></p></li><li id="quote_68"><span class="references-num">[68]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Foldit: Frequently Asked Questions". fold.it. University of Washington. Retrieved September 19, 2008..</span></p></li><li id="quote_69"><span class="references-num">[69]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Project list – BOINC". University of California. Retrieved September 8, 2008..</span></p></li><li id="quote_70"><span class="references-num">[70]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Pande Group (2010). "High Performance FAQ". Stanford University. Archived from the original (FAQ) on September 21, 2012. Retrieved September 19, 2011..</span></p></li><li id="quote_71"><span class="references-num">[71]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">7im (April 2, 2010). "Re: Answers to: Reasons for not using F@H". Retrieved 2011-09-19..</span></p></li><li id="quote_72"><span class="references-num">[72]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Vijay Pande (August 5, 2011). "Results page updated – new key result published in our work in Alzheimer's Disease". Retrieved 2011-09-19..</span></p></li><li id="quote_73"><span class="references-num">[73]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Pande Group. "Folding@home Diseases Studied FAQ". Stanford University. Archived from the original (FAQ) on October 11, 2007. Retrieved September 12, 2011..</span></p></li><li id="quote_74"><span class="references-num">[74]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Vijay Pande (September 26, 2007). "How FAH works: Molecular dynamics". Retrieved 2011-09-10..</span></p></li><li id="quote_75"><span class="references-num">[75]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">tjlane (June 9, 2011). "Re: Course grained Protein folding in under 10 minutes". Retrieved 2011-09-19..</span></p></li><li id="quote_76"><span class="references-num">[76]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">jmn (July 29, 2011). "Rosetta@home and Folding@home: additional projects". Retrieved 2011-09-19..</span></p></li><li id="quote_77"><span class="references-num">[77]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Pande Group. "Client Statistics by OS". Stanford University. Retrieved 2011-10-18..</span></p></li><li id="quote_78"><span class="references-num">[78]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Malmström L, Riffle M, Strauss CE, et al. (April 2007). "Superfamily assignments for the yeast proteome through integration of structure prediction with the gene ontology". PLoS Biology. 5 (4): e76. doi:10.1371/journal.pbio.0050076. PMC 1828141. PMID 17373854..</span></p></li><li id="quote_79"><span class="references-num">[79]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Bonneau R (2006). "World Community Grid Message Board Posts: HPF -&gt; HPF2 transition". Bonneau Lab, New York University. Retrieved September 7, 2008..</span></p></li><li id="quote_80"><span class="references-num">[80]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"List of Richard Bonneau's publications". Bonneau Lab, New York University. Archived from the original on July 7, 2008. Retrieved September 7, 2008..</span></p></li><li id="quote_81"><span class="references-num">[81]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Bonneau R. "World Community Grid Message Board Posts". Bonneau Lab, New York University. Archived from the original on July 4, 2008. Retrieved September 7, 2008..</span></p></li><li id="quote_82"><span class="references-num">[82]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Predictor@home: Developing new application areas for P@H". The Brooks Research Group. Retrieved September 7, 2008.[失效连结].</span></p></li><li id="quote_83"><span class="references-num">[83]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">Carrillo-Tripp M (2007). "dTASSER". The Scripps Research Institute. Archived from the original on July 6, 2007. Retrieved September 7, 2008..</span></p></li><li id="quote_84"><span class="references-num">[84]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"RALPH@home website". RALPH@home forums. University of Washington. Retrieved September 7, 2008..</span></p></li><li id="quote_85"><span class="references-num">[85]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Rosetta@home: The new credit system explained". Rosetta@home forums. University of Washington. 2006. Retrieved October 8, 2008..</span></p></li><li id="quote_86"><span class="references-num">[86]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"BOINCstats: Project Credit Comparison". boincstats.com. 2008. Archived from the original on September 13, 2008. Retrieved October 8, 2008..</span></p></li><li id="quote_87"><span class="references-num">[87]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Credit divided over projects". boincstats.com. Retrieved February 19, 2015..</span></p></li><li id="quote_88"><span class="references-num">[88]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Rosetta@home: Predictor of the day archive". Rosetta@home. University of Washington. 2008. Archived from the original on September 24, 2008. Retrieved October 8, 2008..</span></p></li><li id="quote_89"><span class="references-num">[89]</span><p><a class="ref-back-btn">^</a><span data-url="" class="">"Rosetta@home: Protein Folding, Design, and Docking". Rosetta@home. University of Washington. 2008. Retrieved October 8, 2008..</span></p></li></ul></div><div class="read-num">阅读 <!-- -->129</div></div><div class="right-side" id="rightSide"><div class="side" id="lemma-side"><div class="side-title">版本记录</div><ul class="side-lst"><li><p class="side-lst-txt">暂无</p></li></ul><div class="user-card userCard"></div></div><div class="side"><div class="side-event"></div></div></div></div><div class="footer-box"><div id="footer"><div class="footer-logo-wrap"><div class="footer-logo"></div><div class="footer-logo-text">知识·传播·科普</div></div><div class="footer-info">本网站内容采用<a target="_blank" href="https://web.archive.org/web/20221025114123/https://creativecommons.org/licenses/by-sa/3.0/deed.zh?tdsourcetag=s_pctim_aiomsg">CC-BY-SA 3.0</a>授权</div><div class="footer-btn-wrap"><a target="_blank" href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/help/#user_protocol">用户协议</a><a target="_blank" href="https://web.archive.org/web/20221025114123/http://www.sogou.com/docs/terms.htm?v=1">免责声明</a><a target="_blank" href="https://web.archive.org/web/20221025114123/http://corp.sogou.com/private.html">隐私政策</a><a target="_blank" href="https://web.archive.org/web/20221025114123/https://baike.sogou.com/kexue/intro.htm">关于我们</a></div></div></div><script>window.lemmaInfo ={"lemmaId":"10696","versionId":"14995465208594948","title":"Rosetta@home","subtitle":"","abstracts":{"paragraphId":"14995465208594957","title":"简介","versionId":"14995465208594949","lemmaId":10696,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":76040209,"name":"杨华坤","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233993,"comment":null,"dependVersionId":0,"contentType":2,"content":"<p>Rosetta@home是一个分布式计算项目，用于在伯克利开放式网络计算基础设施(BOINC)平台上进行蛋白质结构预测，由华盛顿大学贝克实验室运行。Rosetta Rosetta@home旨在预测蛋白质-蛋白质对接，并借助平均超过210万亿次（截至2016年7月29日）的约6万台主动志愿计算机来设计新蛋白质<sup><a href=\"#quote_1\" class=\"kx_ref\">[1]</a></sup>。 Foldit是一款Rosetta@Home视频游戏，旨在通过众包方式实现这些目标。尽管该项目的大部分面向基础研究，以提高蛋白质组学方法的准确性和稳健性，RosettaRosetta@home也在疟疾、阿尔茨海默病和其他病理学方面进行应用研究<sup><a href=\"#quote_2\" class=\"kx_ref\">[2]</a></sup>。 </p>\n<p>像所有BOINC项目一样，Rosetta@home使用志愿者计算机中空闲的计算机处理资源来对单个工作单元进行计算。完成的结果被发送到中央项目服务器，在那里它们被验证并被吸收到项目数据库中。该项目是跨平台的，运行在多种硬件配置上。用户可以在Rosetta@home屏幕保护程序上查看他们个人蛋白质结构预测的进展。 </p>\n<p>除了与疾病相关的研究之外，Rosetta Rosetta@home网络还是结构生物信息学新方法的测试框架。在Rosetta Rosetta@home的大型多样的志愿者电脑上充分开发并证明稳定后，这些方法被用于其他基于RosettaRosetta的应用，如Rosetta多克和人类蛋白质组折叠项目。Rosetta Rosetta@home开发的新方法的两个特别重要的测试是蛋白质结构预测技术的关键评估(CASP)和相互作用预测的关键评估(CAPRI)实验，这两个实验分别评估蛋白质结构预测和蛋白质-蛋白质对接预测的最新技术。Rosetta@home始终是最重要的停靠预测因子之一，也是现有的最佳三级结构预测因子之一<sup><a href=\"#quote_3\" class=\"kx_ref\">[3]</a></sup>。 </p>","pics":null,"card":null,"references":[],"versionCount":0},"card":{"paragraphId":"14995465208594958","title":"基本信息","versionId":"14995465208594950","lemmaId":0,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":76040209,"name":"杨华坤","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233993,"comment":null,"dependVersionId":0,"contentType":3,"content":"","pics":null,"card":{"cardItems":[{"key":"开发商","value":"华盛顿大学贝克实验室；Rosetta Commons"},{"key":"初始发布","value":"2005年10月6日，​14 years ago"},{"key":"Stable release","value":"Rosetta: 4.07 / 2018年2月27日，​21 months ago <p>Rosetta Mini: 3.78 / 2017年10月3日，​2 years ago </p> Rosetta for Android: 4.10 / 2018年4月27日，​19 months ago"},{"key":"发展状况","value":"Active"},{"key":"操作系统","value":"Windows, macOS, Linux, Android"},{"key":"平台","value":"BOINC"},{"key":"License","value":"学术使用和非营利使用免费,<sup class=\"reference\"></sup> 商业使用须签署商业协议<sup class=\"reference\"></sup>"},{"key":"平均性能","value":"152.419 TFLOPS<sup class=\"reference\"></sup>"},{"key":"活跃用户","value":"26,620"},{"key":"用户总数","value":"1,287,513"},{"key":"活动主机","value":"71,731"},{"key":"主机总数","value":"88,878"},{"key":"网站","value":"boinc.bakerlab.org/rosetta"}]},"references":[],"versionCount":0},"categories":[{"id":1,"name":"计算机","parents":[]}],"creator":{"uid":76040209,"name":"杨华坤","pic":"https://web.archive.org/web/20221025114123/https://cache.soso.com/qlogo/g?b=oidb&k=D55ZCXF8ibibj2XefibTwiaaKg&s=100&t=1483330692","introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":"","jobBrief":"","role":0,"roleName":null,"title":"","professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":145,"partnerIdCreateTime":1595844881,"partnerIdPoped":false},"createTime":1568626360,"editor":{"uid":76040209,"name":"杨华坤","pic":"https://web.archive.org/web/20221025114123/https://cache.soso.com/qlogo/g?b=oidb&k=D55ZCXF8ibibj2XefibTwiaaKg&s=100&t=1483330692","introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":"","jobBrief":"","role":0,"roleName":null,"title":"","professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":145,"partnerIdCreateTime":1595844881,"partnerIdPoped":false},"editTime":1576233993,"state":1,"versionCount":1,"upNum":2,"downNum":0,"pics":[{"originalUrl":"https://web.archive.org/web/20221025114123/https://img04.sogoucdn.com/app/a/200698/sogou_science_13808?w=384&h=259&titlename=Rosetta@home%E5%B1%8F%E5%B9%95%E4%BF%9D%E6%8A%A4%E7%A8%8B%E5%BA%8F%EF%BC%8C%E6%98%BE%E7%A4%BA%E4%BA%86%E4%B8%80%E4%B8%AA%E5%90%88%E6%88%90%E6%B3%9B%E7%B4%A0%E8%9B%8B%E7%99%BD%E8%B4%A8%EF%BC%88PDB%20ID:%201ogw%EF%BC%89%E7%9A%84%E7%BB%93%E6%9E%84%E9%A2%84%E6%B5%8B%E8%BF%9B%E8%A1%8C%E6%83%85%E5%86%B5%E3%80%82","url":"https://web.archive.org/web/20221025114123/https://img04.sogoucdn.com/app/a/200698/sogou_science_13808","rw":384,"rh":259,"title":"Rosetta@home屏幕保护程序，显示了一个合成泛素蛋白质（PDB ID: 1ogw）的结构预测进行情况。","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221025114123/https://img04.sogoucdn.com/app/a/200698/sogou_science_13809?w=300&h=319&titlename=CASP6%E7%9A%84%E7%9B%AE%E6%A0%87%E8%9B%8B%E7%99%BD%E8%B4%A8T0281%EF%BC%9A%E9%A6%96%E4%B8%AA%E6%8E%A5%E8%BF%91%E5%8E%9F%E5%AD%90%E7%BA%A7%E7%B2%BE%E5%BA%A6%E7%9A%84ab%20initio%E8%9B%8B%E7%99%BD%E8%B4%A8%E7%BB%93%E6%9E%84%E9%A2%84%E6%B5%8B%E3%80%82Rosetta%E7%94%9F%E6%88%90%E4%BA%86T0281%EF%BC%88%E6%B4%8B%E7%BA%A2%E8%89%B2%EF%BC%89%E7%9A%84%E4%B8%80%E4%B8%AA%E6%A8%A1%E5%9E%8B%EF%BC%8C%E4%B8%8E%E5%AE%9E%E9%AA%8C%E6%B5%8B%E5%AE%9A%E7%9A%84%E6%99%B6%E4%BD%93%E7%BB%93%E6%9E%84%EF%BC%88%E8%93%9D%E8%89%B2%EF%BC%89%E4%B9%8B%E9%97%B4%E7%9A%84RMSD%E4%B8%BA1.5%E5%9F%83%E3%80%82","url":"https://web.archive.org/web/20221025114123/https://img04.sogoucdn.com/app/a/200698/sogou_science_13809","rw":300,"rh":319,"title":"CASP6的目标蛋白质T0281：首个接近原子级精度的ab initio蛋白质结构预测。Rosetta生成了T0281（洋红色）的一个模型，与实验测定的晶体结构（蓝色）之间的RMSD为1.5埃。","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221025114123/https://img01.sogoucdn.com/app/a/200698/sogou_science_13810?w=300&h=283&titlename=Rosetta%E8%AE%BE%E8%AE%A1%E7%9A%84TOP7%E6%A8%A1%E5%9E%8B%EF%BC%88%E7%BA%A2%E8%89%B2%EF%BC%89%E4%B8%8E%E5%85%B6X%E5%B0%84%E7%BA%BF%E6%99%B6%E4%BD%93%E7%BB%93%E6%9E%84%EF%BC%88%E8%93%9D%E8%89%B2%EF%BC%8CPDB%20ID:%201QYS%EF%BC%89%E7%9A%84%E5%8F%A0%E5%90%88%E3%80%82","url":"https://web.archive.org/web/20221025114123/https://img01.sogoucdn.com/app/a/200698/sogou_science_13810","rw":300,"rh":283,"title":"Rosetta设计的TOP7模型（红色）与其X射线晶体结构（蓝色，PDB ID: 1QYS）的叠合。","alt":null,"width":0,"height":0},{"originalUrl":"https://web.archive.org/web/20221025114123/https://img02.sogoucdn.com/app/a/200698/sogou_science_13811?w=300&h=138&titlename=Rosetta@home%E5%9C%A8CASP8%E5%AE%9E%E9%AA%8C%E6%9C%9F%E9%97%B4%E7%9A%84%E7%94%A8%E6%88%B7%E8%B4%A1%E7%8C%AE%E6%9F%B1%E7%8A%B6%E7%BB%9F%E8%AE%A1%E5%9B%BE","url":"https://web.archive.org/web/20221025114123/https://img02.sogoucdn.com/app/a/200698/sogou_science_13811","rw":300,"rh":138,"title":"Rosetta@home在CASP8实验期间的用户贡献柱状统计图","alt":null,"width":0,"height":0}],"catalogs":[{"level":1,"title":"计算平台","paragraphId":"14995465225372160","subCatalogs":null},{"level":1,"title":"项目意义","paragraphId":"14995465225372161","subCatalogs":null},{"level":1,"title":"疾病相关研究","paragraphId":"14995465225372162","subCatalogs":[{"level":2,"title":"阿尔次海默病","paragraphId":"14995465225372162","subCatalogs":null},{"level":2,"title":"炭疽","paragraphId":"14995465225372162","subCatalogs":null},{"level":2,"title":"单纯疱疹病毒1","paragraphId":"14995465225372162","subCatalogs":null},{"level":2,"title":"艾滋病毒","paragraphId":"14995465225372162","subCatalogs":null},{"level":2,"title":"疟疾","paragraphId":"14995465225372162","subCatalogs":null},{"level":2,"title":"其他疾病","paragraphId":"14995465225372162","subCatalogs":null}]},{"level":1,"title":"发展历史和分支","paragraphId":"14995465225372163","subCatalogs":[{"level":2,"title":"Rosetta设计","paragraphId":"14995465225372163","subCatalogs":null},{"level":2,"title":"Rosetta多克","paragraphId":"14995465225372163","subCatalogs":null},{"level":2,"title":"罗伯塔（Robetta）","paragraphId":"14995465225372163","subCatalogs":null},{"level":2,"title":"Foldit","paragraphId":"14995465225372163","subCatalogs":null}]},{"level":1,"title":"与类似分布式计算项目的比较","paragraphId":"14995465242149376","subCatalogs":[{"level":2,"title":"Foldit@home","paragraphId":"14995465242149376","subCatalogs":null},{"level":2,"title":"世界社区网格","paragraphId":"14995465242149376","subCatalogs":null},{"level":2,"title":"Predictor@home","paragraphId":"14995465242149376","subCatalogs":null}]},{"level":1,"title":"志愿者贡献","paragraphId":"14995465242149377","subCatalogs":null},{"level":1,"title":"参考文献","paragraphId":"-1","subCatalogs":null}],"paragraphs":[{"paragraphId":"14995465225372160","title":"计算平台","versionId":"14995465208594951","lemmaId":10696,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":76040209,"name":"杨华坤","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233993,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>Rosetta@home应用程序和BOINC分布式计算平台可用于Windows操作系统、Linux和macOS ，BOINC还在其他几个平台上运行，例如FreeBSD<sup><a href=\"#quote_4\" class=\"kx_ref\">[4]</a></sup>。参与Rosetta Rosetta@home需要一个时钟速度至少为500兆赫的中央处理器、200兆字节的可用磁盘空间、512兆字节的物理内存和互联网连接<sup><a href=\"#quote_5\" class=\"kx_ref\">[5]</a></sup>。截至2016年7月20日，Rosetta迷你应用程序的当前版本为3.73<sup><a href=\"#quote_6\" class=\"kx_ref\">[6]</a></sup>。目前推荐的BOINC程序版本是7.6.22<sup><a href=\"#quote_4\" class=\"kx_ref\">[4]</a></sup>。标准超文本传输协议(HTTP)(端口80)用于用户的BOINC客户端和华盛顿大学的Rosetta Rosetta@home服务器之间的通信；密码交换时使用HTTPS(端口443)。BOINC客户端的远程和本地控制使用端口31416和端口1043，如果它们在防火墙后，可能需要特别解锁<sup><a href=\"#quote_7\" class=\"kx_ref\">[7]</a></sup>。包含单个蛋白质数据的工作单元被从位于华盛顿大学贝克实验室的服务器分配到志愿者的计算机上，然后志愿者的计算机计算指定蛋白质的结构预测。为了避免对给定蛋白质的重复结构预测，每个工作单元都用随机的种子号初始化。这给了每个预测一个沿着蛋白质能量景观的独特下降轨迹<sup><a href=\"#quote_8\" class=\"kx_ref\">[8]</a></sup>。Rosetta Rosetta@home的蛋白质结构预测是给定蛋白质能量格局中全球最小值的近似值。该全局最小值代表蛋白质最有利的构象，即其天然状态。 </p> \n<p></p><p><img alt=\"\" class=\"fileimage kx_img ed_imgfloat_right\" img_height=\"259\" img_width=\"384\" titlename=\"Rosetta@home屏幕保护程序，显示了一个合成泛素蛋白质（PDB ID: 1ogw）的结构预测进行情况。\" data-src=\"https://img04.sogoucdn.com/app/a/200698/sogou_science_13808\"> </p><p></p> \n<p>Rosetta@home图形用户界面(图形用户界面)的一个主要功能是屏幕保护程序，显示当前工作单元在模拟蛋白质折叠过程中的进度。在当前屏幕保护程序的左上角，目标蛋白质在寻找最低能量结构时采用不同的形状(构象)。右边是最近被接受的结构。右上角显示了当前诱饵的最低能量构象；下面是蛋白质的真实或天然结构，如果已经确定的话。屏幕保护程序中包含三个图形。在中间附近，显示了一个可接受模型的热力学自由能图，该图随着可接受模型的变化而波动。被接受的模型的均方根偏差(RMSD)图显示在最右边，该图衡量了被接受的模型与本地模型在结构上的相似程度。在可接受的能量图的右侧和RMSD图的下方，随着模型的逐步完善，这两个函数的结果被用于产生能量与RMSD图<sup><a href=\"#quote_9\" class=\"kx_ref\">[9]</a></sup>。 </p>\n<p>像所有BOINC项目一样，Rosetta@home在用户计算机的后台运行，在登录主机操作系统的帐户时或之前使用空闲的计算机资源。当其他应用程序需要资源时，该程序从中央处理器中释放资源，这样就不会影响正常的计算机使用。许多程序设置可以通过用户帐户首选项来指定，包括:程序可以使用的最大CPU资源百分比(控制以持续容量运行的计算机的功耗或热量产生)、程序可以运行的时间等等。 </p>\n<p>运行在Rosetta@home网络上的软件Rosetta用C++重写，以允许比用Fortran编写的原始版本更容易的开发。这个新版本是面向对象的，于2008年2月8日发布<sup><a href=\"#quote_6\" class=\"kx_ref\">[6]</a></sup><sup><a href=\"#quote_10\" class=\"kx_ref\">[10]</a></sup> 。RosettaRosetta代码的开发是由RosettaRosetta公共组织完成的<sup><a href=\"#quote_11\" class=\"kx_ref\">[11]</a></sup>。该软件免费授权给学术界，制药公司付费使用<sup><a href=\"#quote_11\" class=\"kx_ref\">[11]</a></sup>。 </p>","pics":[{"originalUrl":"https://web.archive.org/web/20221025114123/https://img04.sogoucdn.com/app/a/200698/sogou_science_13808?w=384&h=259&titlename=Rosetta@home%E5%B1%8F%E5%B9%95%E4%BF%9D%E6%8A%A4%E7%A8%8B%E5%BA%8F%EF%BC%8C%E6%98%BE%E7%A4%BA%E4%BA%86%E4%B8%80%E4%B8%AA%E5%90%88%E6%88%90%E6%B3%9B%E7%B4%A0%E8%9B%8B%E7%99%BD%E8%B4%A8%EF%BC%88PDB%20ID:%201ogw%EF%BC%89%E7%9A%84%E7%BB%93%E6%9E%84%E9%A2%84%E6%B5%8B%E8%BF%9B%E8%A1%8C%E6%83%85%E5%86%B5%E3%80%82","url":"https://web.archive.org/web/20221025114123/https://img04.sogoucdn.com/app/a/200698/sogou_science_13808","rw":384,"rh":259,"title":"Rosetta@home屏幕保护程序，显示了一个合成泛素蛋白质（PDB ID: 1ogw）的结构预测进行情况。","alt":null,"width":0,"height":0}],"card":null,"references":[],"versionCount":0},{"paragraphId":"14995465225372161","title":"项目意义","versionId":"14995465208594952","lemmaId":10696,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":76040209,"name":"杨华坤","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233993,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>随着基因组测序项目的增多，科学家可以推断出许多在细胞内发挥功能的蛋白质的氨基酸序列或一级结构。为了更好地理解蛋白质的功能并帮助合理的药物设计，科学家需要知道蛋白质的三维三级结构。 </p> \n<p></p><p><img alt=\"\" class=\"fileimage kx_img ed_imgfloat_right\" img_height=\"319\" img_width=\"300\" titlename=\"CASP6的目标蛋白质T0281：首个接近原子级精度的ab initio蛋白质结构预测。Rosetta生成了T0281（洋红色）的一个模型，与实验测定的晶体结构（蓝色）之间的RMSD为1.5埃。\" data-src=\"https://img04.sogoucdn.com/app/a/200698/sogou_science_13809\"> </p><p></p> \n<p>蛋白质三维结构目前是通过x光结晶学或核磁共振波谱实验确定的。这个过程很慢(可能需要几周甚至几个月才能首次弄清楚如何结晶蛋白质)，而且成本很高(大约每种蛋白质10万美元)<sup><a href=\"#quote_12\" class=\"kx_ref\">[12]</a></sup>。不幸的是，新序列的发现速度远远超过了结构确定的速度——在国家生物技术信息中心(NCBI)非冗余蛋白质数据库中现有的740万个蛋白质序列中，只有不到52000个蛋白质的三维结构被解决并储存在蛋白质数据库中，蛋白质数据库是蛋白质结构信息的主要储存库<sup><a href=\"#quote_13\" class=\"kx_ref\">[13]</a></sup>。 RosettaRosetta@home的主要目标之一是以与现有方法相同的准确度预测蛋白质结构，但需要的时间和金钱要少得多。RosettaRosetta@home还开发了一些方法来确定膜蛋白(如G蛋白偶联受体<sup><a href=\"#quote_14\" class=\"kx_ref\">[14]</a></sup>的结构和对接，这些方法很难用传统技术如x光结晶学和核磁共振波谱进行分析，但却代表了现代药物的大多数靶标。 </p>\n<p>蛋白质结构预测的进展在半年一次的蛋白质结构预测技术关键评估(CASP)实验中进行评估，来自世界各地的研究人员试图从蛋白质的氨基酸序列中推导出蛋白质的结构。在这个有时具有竞争性的实验中，高分群体被认为是蛋白质结构预测的事实上的标准承担者。Rosetta是Rosetta@home的基础程序，自2002年CASP5以来一直在使用。在2004年CASP6实验中，RosettaRosetta在提交的CASP靶T0281模型中首次提出了接近原子水平的分辨率、从头计算蛋白质结构预测，创造了历史<sup><a href=\"#quote_15\" class=\"kx_ref\">[15]</a></sup>。从头建模被认为是蛋白质结构预测的一个特别困难的类别，因为它不使用来自结构同源性的信息，并且必须依赖来自序列同源性的信息和建模蛋白质内的物理相互作用。Rosetta@home自2006年以来一直用于CASP，在CASP7中，它是每一类结构预测的顶级预测因子之一<sup><a href=\"#quote_16\" class=\"kx_ref\">[16]</a></sup><sup><a href=\"#quote_17\" class=\"kx_ref\">[17]</a></sup><sup><a href=\"#quote_18\" class=\"kx_ref\">[18]</a></sup>。这些高质量的预测是由RosettaRosetta@Home志愿者提供的计算能力实现的<sup><a href=\"#quote_19\" class=\"kx_ref\">[19]</a></sup>。计算能力的提高使得Rosetta Rosetta@home能够对构象空间的更多区域(蛋白质可能呈现的形状)进行采样，根据莱文塔尔悖论，预计构象空间将随着蛋白质长度呈指数级增长。 </p>\n<p>Rosetta@home也用于蛋白质-蛋白质对接预测，它决定了多种复合蛋白质的结构或四级结构。这种蛋白质相互作用影响许多细胞功能，包括抗原抗体和酶抑制剂结合以及细胞进出口。确定这些相互作用对药物设计至关重要。RosettaRosetta用于相互作用预测的临界评估(CAPRI)实验，该实验评估蛋白质对接区的状态，类似于CASP如何测量蛋白质结构预测的进展。Rosetta Rosetta@home项目志愿者提供的计算能力被认为是RosettaRosetta在CAPRI的表现的一个主要因素，它的对接预测是最准确和完整的<sup><a href=\"#quote_20\" class=\"kx_ref\">[20]</a></sup>。 </p>\n<p>2008年初，RosettaRosetta被用于计算设计一种蛋白质，其功能在自然界中从未被观察到<sup><a href=\"#quote_21\" class=\"kx_ref\">[21]</a></sup>。 这部分是受2004年一篇高调论文的撤回启发，该论文最初描述了一种蛋白质的计算设计，该蛋白质相对于其天然形式具有更高的酶活性<sup><a href=\"#quote_22\" class=\"kx_ref\">[22]</a></sup>。大卫·贝克小组2008年的研究论文描述了蛋白质是如何制造的，该论文引用了Rosetta Rosetta@home提供的计算资源，代表了这种蛋白质设计方法概念的重要证明<sup><a href=\"#quote_21\" class=\"kx_ref\">[21]</a></sup>。 这种蛋白质设计可能在药物发现、绿色化学和生物修复方面有未来的应用<sup><a href=\"#quote_21\" class=\"kx_ref\">[21]</a></sup>。 </p>","pics":[{"originalUrl":"https://web.archive.org/web/20221025114123/https://img04.sogoucdn.com/app/a/200698/sogou_science_13809?w=300&h=319&titlename=CASP6%E7%9A%84%E7%9B%AE%E6%A0%87%E8%9B%8B%E7%99%BD%E8%B4%A8T0281%EF%BC%9A%E9%A6%96%E4%B8%AA%E6%8E%A5%E8%BF%91%E5%8E%9F%E5%AD%90%E7%BA%A7%E7%B2%BE%E5%BA%A6%E7%9A%84ab%20initio%E8%9B%8B%E7%99%BD%E8%B4%A8%E7%BB%93%E6%9E%84%E9%A2%84%E6%B5%8B%E3%80%82Rosetta%E7%94%9F%E6%88%90%E4%BA%86T0281%EF%BC%88%E6%B4%8B%E7%BA%A2%E8%89%B2%EF%BC%89%E7%9A%84%E4%B8%80%E4%B8%AA%E6%A8%A1%E5%9E%8B%EF%BC%8C%E4%B8%8E%E5%AE%9E%E9%AA%8C%E6%B5%8B%E5%AE%9A%E7%9A%84%E6%99%B6%E4%BD%93%E7%BB%93%E6%9E%84%EF%BC%88%E8%93%9D%E8%89%B2%EF%BC%89%E4%B9%8B%E9%97%B4%E7%9A%84RMSD%E4%B8%BA1.5%E5%9F%83%E3%80%82","url":"https://web.archive.org/web/20221025114123/https://img04.sogoucdn.com/app/a/200698/sogou_science_13809","rw":300,"rh":319,"title":"CASP6的目标蛋白质T0281：首个接近原子级精度的ab initio蛋白质结构预测。Rosetta生成了T0281（洋红色）的一个模型，与实验测定的晶体结构（蓝色）之间的RMSD为1.5埃。","alt":null,"width":0,"height":0}],"card":null,"references":[],"versionCount":0},{"paragraphId":"14995465225372162","title":"疾病相关研究","versionId":"14995465208594953","lemmaId":10696,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":76040209,"name":"杨华坤","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233993,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>除了预测蛋白质结构、对接和设计的基础研究之外，Rosetta@home还用于直接疾病相关的研究<sup><a href=\"#quote_23\" class=\"kx_ref\">[23]</a></sup>。大卫·贝克的《Rosetta Rosetta@Home杂志》描述了许多小型研究项目<sup><a href=\"#quote_24\" class=\"kx_ref\">[24]</a></sup>。截至2014年2月，论坛正在更新关于最近出版物的信息和对工作的简短描述<sup><a href=\"#quote_25\" class=\"kx_ref\">[25]</a></sup>。论坛帖子自2016年起不再使用，研究的新闻可以在项目的一般新闻部分找到<sup><a href=\"#quote_26\" class=\"kx_ref\">[26]</a></sup>。 </p> \n<h3>阿尔次海默病</h3> \n<p>RosettaRosetta软件套件的一个组件，RosettaRosetta设计，用于准确预测淀粉样蛋白的哪些区域最有可能形成淀粉样纤维<sup><a href=\"#quote_27\" class=\"kx_ref\">[27]</a></sup>。通过提取感兴趣蛋白质的六肽(六个氨基酸长的片段)，并选择与已知形成六肽的原纤维的结构相似的最低能量匹配，Rosetta设计能够识别出形成原纤维的可能性是随机蛋白质的两倍的肽<sup><a href=\"#quote_28\" class=\"kx_ref\">[28]</a></sup>。Rosetta Rosetta@home在同一项研究中被用来预测淀粉样β蛋白的结构，淀粉样β蛋白是一种纤维形成蛋白，被认为会导致阿尔茨海默病<sup><a href=\"#quote_29\" class=\"kx_ref\">[29]</a></sup>。RosettaRosetta设计的蛋白质已经产生了初步但尚未公布的结果，这些蛋白质可以防止原纤维的形成，尽管还不知道它是否能预防这种疾病<sup><a href=\"#quote_30\" class=\"kx_ref\">[30]</a></sup>。 </p> \n<h3>炭疽</h3> \n<p>Rosetta的另一个组成部分，Rosetta多克<sup><a href=\"#quote_31\" class=\"kx_ref\">[31]</a></sup><sup><a href=\"#quote_32\" class=\"kx_ref\">[32]</a></sup><sup><a href=\"#quote_33\" class=\"kx_ref\">[33]</a></sup> ，被用于结合实验方法来模拟构成炭疽毒素的三种蛋白质之间的相互作用——致死因子(LF)、水肿因子(EF)和保护性抗原(PA)。计算机模型准确预测了低频和磷酸之间的对接，有助于确定各自蛋白质的哪些结构域参与低频-磷酸复合物。这一见解最终被用于改进炭疽疫苗的研究<sup><a href=\"#quote_34\" class=\"kx_ref\">[34]</a></sup><sup><a href=\"#quote_35\" class=\"kx_ref\">[35]</a></sup>。 </p> \n<h3>单纯疱疹病毒1</h3> \n<p>罗赛多克用于模拟抗体(免疫球蛋白G)和由感冒疮病毒，单纯疱疹病毒1 (HSV-1)表达的表面蛋白之间的对接，后者用于降解抗病毒抗体。Rosetta多克预测的蛋白质复合物与特别难以获得的实验模型非常一致，导致研究人员得出结论，对接方法有可能解决x光结晶学在蛋白质-蛋白质界面建模方面的一些问题<sup><a href=\"#quote_36\" class=\"kx_ref\">[36]</a></sup>。 </p> \n<h3>艾滋病毒</h3> \n<p>作为比尔及梅林达盖茨基金会1940万美元赠款资助的研究的一部分<sup><a href=\"#quote_37\" class=\"kx_ref\">[37]</a></sup>，Rosetta@home已被用于设计多种可能的人类免疫缺陷病毒疫苗<sup><a href=\"#quote_38\" class=\"kx_ref\">[38]</a></sup><sup><a href=\"#quote_39\" class=\"kx_ref\">[39]</a></sup>。 </p> \n<h3>疟疾</h3> \n<p>在与全球健康大挑战倡议有关的研究中<sup><a href=\"#quote_40\" class=\"kx_ref\">[40]</a></sup>，Rosetta被用于计算设计新的归巢核酸内切酶蛋白，这种蛋白可以消灭冈比亚按蚊或使蚊子无法传播疟疾<sup><a href=\"#quote_41\" class=\"kx_ref\">[41]</a></sup>。 像归巢核酸内切酶一样，能够专门模拟和改变蛋白质-脱氧核糖核酸的相互作用，这使得像Rosetta这样的计算蛋白质设计方法在基因治疗(包括可能的癌症治疗)中发挥了重要作用<sup><a href=\"#quote_23\" class=\"kx_ref\">[23]</a></sup><sup><a href=\"#quote_42\" class=\"kx_ref\">[42]</a></sup>。 </p> \n<h3>其他疾病</h3> \n<p>Rosetta@home研究人员设计了一种称为新白细胞介素-2/15的白细胞介素-2受体激动剂，它不与受体的α亚单位相互作用。这种免疫信号分子可用于癌症治疗。虽然天然白细胞介素-2因与α亚单位相互作用而中毒，但设计的蛋白质更安全，至少在动物模型中是如此<sup><a href=\"#quote_43\" class=\"kx_ref\">[43]</a></sup>。 </p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14995465225372163","title":"发展历史和分支","versionId":"14995465208594954","lemmaId":10696,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":76040209,"name":"杨华坤","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233993,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>最初由贝克实验室在1998年引入，作为结构预测的一种从头开始的方法<sup><a href=\"#quote_44\" class=\"kx_ref\">[44]</a></sup> ，Rosetta后来已经扩展到几个开发流程和不同的服务。Rosetta平台得名于Rosetta石碑，因为它试图破译蛋白质氨基酸序列的结构“意义”<sup><a href=\"#quote_45\" class=\"kx_ref\">[45]</a></sup>。Rosetta首次亮相七年多后，Rosetta@home项目于2005年10月6日发布(即宣布不再测试)<sup><a href=\"#quote_6\" class=\"kx_ref\">[6]</a></sup>。参与Rosetta项目初期开发的许多研究生和其他研究人员后来搬到了其他大学和研究机构，并随后加强了Rosetta项目的不同部分。 </p> \n<h3>Rosetta设计</h3> \n<p></p><p><img alt=\"\" class=\"fileimage kx_img ed_imgfloat_right\" img_height=\"283\" img_width=\"300\" titlename=\"Rosetta设计的TOP7模型（红色）与其X射线晶体结构（蓝色，PDB ID: 1QYS）的叠合。\" data-src=\"https://img01.sogoucdn.com/app/a/200698/sogou_science_13810\"> </p><p></p> \n<p>Rosetta设计(RosettaDesign)，一种基于Rosetta的蛋白质设计计算方法，始于2000年的一项重新设计蛋白质折叠途径的研究<sup><a href=\"#quote_46\" class=\"kx_ref\">[46]</a></sup>。2002年，Rosetta设计被用于设计Top7，一种93个氨基酸长的α/β蛋白质，其整体折叠在自然界中从未有过。Rosetta预测这种新的构象在由x光晶体学确定的结构的1.2 RMSD以内，代表了异常精确的结构预测<sup><a href=\"#quote_47\" class=\"kx_ref\">[47]</a></sup>。 Rosetta和Rosetta设计获得了广泛的认可，因为它们是第一个设计和准确预测这种长度的新蛋白质结构的人，正如2002年在《科学》<sup><a href=\"#quote_48\" class=\"kx_ref\">[48]</a></sup><sup><a href=\"#quote_49\" class=\"kx_ref\">[49]</a></sup> 杂志上发表的描述双重方法的论文所反映的那样，被超过240篇其他科学文章引用<sup><a href=\"#quote_50\" class=\"kx_ref\">[50]</a></sup>。 该项研究的有形产品Top7在2006年10月被评为RCSB·PDB的“月度分子”<sup><a href=\"#quote_51\" class=\"kx_ref\">[51]</a></sup>； Rosetta@home标志展示了其预测晶体结构和x光晶体结构各自核心(残基60–79)的叠加<sup><a href=\"#quote_15\" class=\"kx_ref\">[15]</a></sup>。 </p>\n<p>布莱恩·库尔曼(Brian Kuhlman)，大卫·贝克实验室前博士后研究员，现为[教堂山北卡罗来纳大学副教授<sup><a href=\"#quote_52\" class=\"kx_ref\">[52]</a></sup>， 提供Rosetta设计在线服务<sup><a href=\"#quote_53\" class=\"kx_ref\">[53]</a></sup>。 </p> \n<h3>Rosetta多克</h3> \n<p>Rosetta多克是在2002年第一次CAPRI实验中作为贝克实验室的蛋白质-蛋白质对接预测算法添加到Rosetta软件套件中的<sup><a href=\"#quote_54\" class=\"kx_ref\">[54]</a></sup>。在该实验中，Rosetta多克对链球菌化脓性外毒素A和T细胞受体β链之间的对接进行了高精度预测，并对猪α-淀粉酶和骆驼抗体之间的复合物进行了中等精度预测。虽然Rosetta多克方法在七种可能的预测中仅做出了两种可以接受的准确预测，但这足以在第一次CAPRI评估中将其列为十九种预测方法中的第七种<sup><a href=\"#quote_54\" class=\"kx_ref\">[54]</a></sup>。 </p>\n<p>Rosetta多克的开发在随后的CAPRI回合中分成了两个分支，杰弗里·格雷在华盛顿大学为Rosetta多克打下了基础，他在约翰·霍普金斯大学的新职位上继续研究这种方法。贝克实验室的成员在格雷不在时进一步开发了Rosetta多克。这两个版本在侧链建模、诱饵选择和其他方面略有不同<sup><a href=\"#quote_33\" class=\"kx_ref\">[33]</a></sup><sup><a href=\"#quote_55\" class=\"kx_ref\">[55]</a></sup>。尽管存在这些差异，贝克和格雷方法在第二次CAPRI评估中表现良好，分别在30个预测组中排名第五和第七<sup><a href=\"#quote_56\" class=\"kx_ref\">[56]</a></sup>。杰弗里·格雷(Jeffrey Gray)的Rosetta多克服务器是一项免费对接预测服务，供非商业用途<sup><a href=\"#quote_57\" class=\"kx_ref\">[57]</a></sup>。 </p>\n<p>2006年10月，Rosetta多克被整合到Rosetta@home。该方法仅使用蛋白质骨架，使用快速、粗略的对接模型阶段。随后是缓慢的全原子精细化阶段，其中两种相互作用蛋白质相对于彼此的取向以及蛋白质-蛋白质界面的侧链相互作用被同时优化，以找到最低能量构象<sup><a href=\"#quote_58\" class=\"kx_ref\">[58]</a></sup>。Rosetta@家庭网络提供的计算能力大幅提高，加上主干灵活性和环路建模的修正折叠树表示法，使Rosetta多克在第三次CAPRI评估的63个预测组中排名第六<sup><a href=\"#quote_3\" class=\"kx_ref\">[3]</a></sup><sup><a href=\"#quote_20\" class=\"kx_ref\">[20]</a></sup>。 </p> \n<h3>罗伯塔（Robetta）</h3> \n<p>Rosetta(Rosetta Beta)服务器是贝克实验室为非商业从头计算和比较建模提供的自动化蛋白质结构预测服务<sup><a href=\"#quote_59\" class=\"kx_ref\">[59]</a></sup>。自2002年CASP5以来，它作为自动预测服务器参加了半年一次的CASP实验，在自动服务器预测类别中表现最佳<sup><a href=\"#quote_60\" class=\"kx_ref\">[60]</a></sup>。 Robetta此后参加了CASP6和7的比赛，在这两个项目中，自动化服务器组和人工预测组的表现均优于平均水平<sup><a href=\"#quote_18\" class=\"kx_ref\">[18]</a></sup><sup><a href=\"#quote_61\" class=\"kx_ref\">[61]</a></sup><sup><a href=\"#quote_62\" class=\"kx_ref\">[62]</a></sup>。它还参与CAMEO3D连续评估。 </p>\n<p>在对CASP6的蛋白质结构建模时，Robetta首先使用BLAST、PSI-BLAST和3D-Jury搜索结构同源物，然后通过将序列与Pfam数据库中的结构家族相匹配，将目标序列解析为其单个结构域，或独立折叠的蛋白质单位。然后，具有结构同源物的结构域遵循“基于模板的模型”(即同源建模)方案。在这里，贝克实验室的内部比对程序，K*sync，产生一组序列同源物，每一个序列同源物都用Rosetta从头方法建模，以产生诱饵(可能的结构)。通过采用由低分辨率Rosetta能量函数确定的最低能量模型来选择最终的结构预测。对于没有检测到结构同源物的区域，遵循从头协议，其中从一组生成的诱饵中选择最低能量模型作为最终预测。然后将这些结构域预测连接在一起，研究蛋白质内的结构域间、三级相互作用。最后，使用蒙特卡罗构象搜索协议对侧链贡献进行建模<sup><a href=\"#quote_63\" class=\"kx_ref\">[63]</a></sup>。 </p>\n<p>在CAPS8中，Rosetta被扩充以使用Rosetta的高分辨率全原子精细化方法<sup><a href=\"#quote_64\" class=\"kx_ref\">[64]</a></sup>， 缺少这种方法被认为是Rosetta不如CAPS7中Rosetta@Home网络精确的主要原因<sup><a href=\"#quote_19\" class=\"kx_ref\">[19]</a></sup>。 在CASP11中，增加了一种通过相关蛋白质残基的共同进化来预测蛋白质接触图的方法，称为GREMLIN，这使得更多的新的成功成为可能<sup><a href=\"#quote_65\" class=\"kx_ref\">[65]</a></sup>。 </p> \n<h3>Foldit</h3> \n<p>2008年5月9日，在Rosetta@home用户提出分布式计算程序的交互式版本后，贝克实验室公开发布了Foldit，这是一款基于Rosetta平台的在线蛋白质结构预测游戏<sup><a href=\"#quote_66\" class=\"kx_ref\">[66]</a></sup>。截至2008年9月25日，Foldit拥有超过59，000名注册用户<sup><a href=\"#quote_67\" class=\"kx_ref\">[67]</a></sup>。该游戏给用户提供了一套控制(例如摇动、摆动、重建)来操纵目标蛋白的主链和氨基酸侧链，使其形成更具能量优势的构象。用户可以作为独唱者单独工作，也可以作为进化者集体工作，当他们改进他们的结构预测时，在两个类别下累积点数<sup><a href=\"#quote_68\" class=\"kx_ref\">[68]</a></sup>。用户也可以通过决斗功能单独与其他用户竞争，在决斗功能中，20次移动后能量结构最低的玩家获胜。 </p>","pics":[{"originalUrl":"https://web.archive.org/web/20221025114123/https://img01.sogoucdn.com/app/a/200698/sogou_science_13810?w=300&h=283&titlename=Rosetta%E8%AE%BE%E8%AE%A1%E7%9A%84TOP7%E6%A8%A1%E5%9E%8B%EF%BC%88%E7%BA%A2%E8%89%B2%EF%BC%89%E4%B8%8E%E5%85%B6X%E5%B0%84%E7%BA%BF%E6%99%B6%E4%BD%93%E7%BB%93%E6%9E%84%EF%BC%88%E8%93%9D%E8%89%B2%EF%BC%8CPDB%20ID:%201QYS%EF%BC%89%E7%9A%84%E5%8F%A0%E5%90%88%E3%80%82","url":"https://web.archive.org/web/20221025114123/https://img01.sogoucdn.com/app/a/200698/sogou_science_13810","rw":300,"rh":283,"title":"Rosetta设计的TOP7模型（红色）与其X射线晶体结构（蓝色，PDB ID: 1QYS）的叠合。","alt":null,"width":0,"height":0}],"card":null,"references":[],"versionCount":0},{"paragraphId":"14995465242149376","title":"与类似分布式计算项目的比较","versionId":"14995465208594955","lemmaId":10696,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":76040209,"name":"杨华坤","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233993,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>有几个分布式计算项目的研究领域与Rosetta@home类似，但研究方法不同: </p> \n<h3>Foldit@home</h3> \n<p>在所有涉及蛋白质研究的主要分布式计算项目中，Folding@home是唯一一个没有使用BOINC平台的项目<sup><a href=\"#quote_69\" class=\"kx_ref\">[69]</a></sup><sup><a href=\"#quote_70\" class=\"kx_ref\">[70]</a></sup><sup><a href=\"#quote_71\" class=\"kx_ref\">[71]</a></sup>。Rosetta“家”和折叠“家”都研究蛋白质错误折叠疾病，如阿尔茨海默氏病，但折叠“家”做得更多<sup><a href=\"#quote_72\" class=\"kx_ref\">[72]</a></sup><sup><a href=\"#quote_73\" class=\"kx_ref\">[73]</a></sup>。Foldit@home几乎完全使用全原子分子动力学模型来理解蛋白质折叠(或潜在错误折叠，随后聚集导致疾病)的方式和原因<sup><a href=\"#quote_74\" class=\"kx_ref\">[74]</a></sup><sup><a href=\"#quote_75\" class=\"kx_ref\">[75]</a></sup>。换句话说，Folding@home的强项是模拟蛋白质折叠的过程，而Rosetta@home的强项是计算蛋白质设计和预测蛋白质结构和对接。 </p>\n<p>Rosetta“家”的一些结果被用作一些Foldit@Home项目的基础。Rosetta提供了最有可能的结构，但是还不确定这是不是分子的形式或者它是否可行。Foldit@home然后可以用来验证Rosetta@home的结果，并且可以提供附加的原子级信息，以及分子如何改变形状的细节<sup><a href=\"#quote_75\" class=\"kx_ref\">[75]</a></sup><sup><a href=\"#quote_76\" class=\"kx_ref\">[76]</a></sup>。 </p>\n<p>这两个项目在计算能力和主机多样性方面也有显著差异<sup><a href=\"#quote_77\" class=\"kx_ref\">[77]</a></sup>，Foldit@Home的计算能力几乎是Rosetta“家庭”的108倍<sup><a href=\"#quote_1\" class=\"kx_ref\">[1]</a></sup>。 </p> \n<h3>世界社区网格</h3> \n<p>人类蛋白质组折叠项目(HPF)的第一阶段和第二阶段(世界社区网格的一个子项目)都使用Rosetta程序对各种基因组进行结构和功能注释<sup><a href=\"#quote_78\" class=\"kx_ref\">[78]</a></sup><sup><a href=\"#quote_79\" class=\"kx_ref\">[79]</a></sup>。尽管他现在用它为生物学家创建数据库，但人类蛋白质组折叠项目的首席科学家理查德·博诺(Richard Bonneau)在获得博士学位的同时，积极参与了大卫·贝克实验室Rosetta的最初开发<sup><a href=\"#quote_80\" class=\"kx_ref\">[80]</a></sup>。关于HPF1、HPF2和Rosetta@home之间关系的更多信息，请访问理查德·博诺的网站<sup><a href=\"#quote_81\" class=\"kx_ref\">[81]</a></sup>。 </p> \n<h3>Predictor@home</h3> \n<p>像Rosetta“家”一样，Predictor@Home专门从事蛋白质结构预测<sup><a href=\"#quote_82\" class=\"kx_ref\">[82]</a></sup>。当Rosetta@home使用Rosetta程序进行结构预测时，Predictor@home使用数据断言方法<sup><a href=\"#quote_83\" class=\"kx_ref\">[83]</a></sup>。2009年，Predictor@home关闭。 </p>\n<p>BOINC上其他与蛋白质相关的分布式计算项目包括：QMC@home, Docking@home, POEM@home、SIMAP和TANPAKU。RALPH@home，Rosetta@home alpha项目，在新的应用程序版本、工作单元和更新进入Rosetta@home之前进行测试，也在BOINC上运行<sup><a href=\"#quote_84\" class=\"kx_ref\">[84]</a></sup>。 </p>","pics":null,"card":null,"references":[],"versionCount":0},{"paragraphId":"14995465242149377","title":"志愿者贡献","versionId":"14995465208594956","lemmaId":10696,"createType":0,"creator":{"uid":0,"name":null,"pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"createTime":0,"versionEditor":{"uid":76040209,"name":"杨华坤","pic":null,"introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":null,"jobBrief":null,"role":0,"roleName":null,"title":null,"professionalTitle":null,"phoneNo":null,"editable":false,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false},"editTime":1576233993,"comment":null,"dependVersionId":0,"contentType":1,"content":"<p>Rosetta@home的研究依赖于各个项目成员捐赠的计算能力。截至2014年4月18日，来自150个国家的约26，900名用户是Rosetta@home的活跃成员，他们共同贡献了约66，000台计算机的空闲处理器时间，平均性能超过83万亿次<sup><a href=\"#quote_1\" class=\"kx_ref\">[1]</a></sup>。 </p> \n<p></p><p><img alt=\"\" class=\"fileimage kx_img ed_imgfloat_right\" img_height=\"138\" img_width=\"300\" titlename=\"Rosetta@home在CASP8实验期间的用户贡献柱状统计图\" data-src=\"https://img02.sogoucdn.com/app/a/200698/sogou_science_13811\"> </p><p></p> \n<p>用户被授予BOINC学分作为他们贡献的衡量标准。给予每个工作单位的信用是为该工作单位生产的诱饵数量乘以所有计算机主机为该工作单位提交的诱饵的平均索赔信用。该定制系统旨在解决标准BOINC客户端和优化BOINC客户端授予用户的信用之间的显著差异，以及在视窗和Linux操作系统上运行Rosetta@home的用户之间的信用差异<sup><a href=\"#quote_85\" class=\"kx_ref\">[85]</a></sup>。Rosetta@home的每秒中央处理器工作的信贷额度低于大多数其他BOINC项目<sup><a href=\"#quote_86\" class=\"kx_ref\">[86]</a></sup>。就信贷总额而言，Rosetta@home在40多个投资理事会项目中排名第十三<sup><a href=\"#quote_87\" class=\"kx_ref\">[87]</a></sup>。 </p>\n<p>Rosetta@home用户预测提交给CASP实验的蛋白质结构，其结果在科学出版物中得到认可<sup><a href=\"#quote_19\" class=\"kx_ref\">[19]</a></sup>。在Rosetta@主页上，预测给定工作单位最低能量结构的用户以及他们所属的任何团队都被列为当天的预测者<sup><a href=\"#quote_88\" class=\"kx_ref\">[88]</a></sup>。每天都从创建Rosetta@home档案的用户中随机选择一名每日用户，并将其放在主页上<sup><a href=\"#quote_89\" class=\"kx_ref\">[89]</a></sup>。 </p>","pics":[{"originalUrl":"https://web.archive.org/web/20221025114123/https://img02.sogoucdn.com/app/a/200698/sogou_science_13811?w=300&h=138&titlename=Rosetta@home%E5%9C%A8CASP8%E5%AE%9E%E9%AA%8C%E6%9C%9F%E9%97%B4%E7%9A%84%E7%94%A8%E6%88%B7%E8%B4%A1%E7%8C%AE%E6%9F%B1%E7%8A%B6%E7%BB%9F%E8%AE%A1%E5%9B%BE","url":"https://web.archive.org/web/20221025114123/https://img02.sogoucdn.com/app/a/200698/sogou_science_13811","rw":300,"rh":138,"title":"Rosetta@home在CASP8实验期间的用户贡献柱状统计图","alt":null,"width":0,"height":0}],"card":null,"references":[],"versionCount":0}],"references":[{"id":1,"type":"book","title":"de Zutter W. \"Rosetta@home: Credit overview\". boincstats.com. Archived from the original on October 7, 2011. Retrieved October 18, 2011.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":2,"type":"book","title":"\"What is Rosetta@home?\". Rosetta@home forums. University of Washington. Archived from the original on September 13, 2008. Retrieved September 7, 2008.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":3,"type":"book","title":"Lensink MF, Méndez R, Wodak SJ (December 2007). \"Docking and scoring protein complexes: CAPRI 3rd Edition\". Proteins. 69 (4): 704–18. doi:10.1002/prot.21804. PMID 17918726.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":4,"type":"book","title":"\"Download BOINC client software\". BOINC. University of California. 2008. Retrieved December 1, 2008.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":5,"type":"book","title":"\"Rosetta@home: Recommended System Requirements\". Rosetta@home. University of Washington. 2008. Archived from the original on September 25, 2008. Retrieved October 7, 2008.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":6,"type":"book","title":"\"Rosetta@home: News archive\". Rosetta@home. University of Washington. 2016. Retrieved July 20, 2016.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":7,"type":"book","title":"\"Rosetta@home: FAQ (work in progress) (message 10910)\". Rosetta@home forums. University of Washington. 2006. Retrieved October 7, 2008.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":8,"type":"book","title":"Kim DE (2005). \"Rosetta@home: Random Seed (message 3155)\". Rosetta@home forums. University of Washington. Retrieved October 7, 2008.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":9,"type":"book","title":"\"Rosetta@home: Quick guide to Rosetta and its graphics\". Rosetta@home. University of Washington. 2007. Archived from the original on September 24, 2008. Retrieved October 7, 2008.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":10,"type":"book","title":"Kim DE (2008). \"Rosetta@home: Problems with minirosetta version 1.+ (Message 51199)\". Rosetta@home forums. University of Washington. Retrieved September 7, 2008.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":11,"type":"book","title":"\"Rosetta Commons\". RosettaCommons.org. 2008. Archived from the original on September 15, 2008. Retrieved October 7, 2008.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":12,"type":"book","title":"Bourne PE, Helge W, eds. (2003). Structural Bioinformatics. Hoboken, NJ: Wiley-Liss. ISBN 978-0-471-20199-1. OCLC 50199108.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":13,"type":"book","title":"\"Yearly Growth of Protein Structures\". RCSB Protein Data Bank. 2008. Retrieved November 30, 2008.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":14,"type":"book","title":"Baker D (2008). \"Rosetta@home: David Baker's Rosetta@home journal (message 55893)\". Rosetta@home forums. University of Washington. Retrieved October 7, 2008.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":15,"type":"book","title":"\"Rosetta@home: Research Overview\". Rosetta@home. University of Washington. 2007. Archived from the original on September 25, 2008. Retrieved October 7, 2008.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":16,"type":"book","title":"Kopp J, Bordoli L, Battey JN, Kiefer F, Schwede T (2007). \"Assessment of CASP7 predictions for template-based modeling targets\". Proteins. 69 Suppl 8: 38–56. doi:10.1002/prot.21753. PMID 17894352.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":17,"type":"book","title":"Read RJ, Chavali G (2007). \"Assessment of CASP7 predictions in the high accuracy template-based modeling category\". Proteins. 69 Suppl 8: 27–37. doi:10.1002/prot.21662. PMID 17894351.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":18,"type":"book","title":"Jauch R, Yeo HC, Kolatkar PR, Clarke ND (2007). \"Assessment of CASP7 structure predictions for template free targets\". Proteins. 69 Suppl 8: 57–67. doi:10.1002/prot.21771. PMID 17894330.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":19,"type":"book","title":"Das R, Qian B, Raman S, et al. (2007). \"Structure prediction for CASP7 targets using extensive all-atom refinement with Rosetta@home\". Proteins. 69 Suppl 8: 118–28. doi:10.1002/prot.21636. PMID 17894356.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":20,"type":"book","title":"Wang C, Schueler-Furman O, Andre I, et al. (December 2007). \"RosettaDock in CAPRI rounds 6–12\". Proteins. 69 (4): 758–63. doi:10.1002/prot.21684. PMID 17671979.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":21,"type":"book","title":"Jiang L, Althoff EA, Clemente FR, et al. (March 2008). \"De novo computational design of retro-aldol enzymes\". Science. 319 (5868): 1387–91. Bibcode:2008Sci...319.1387J. doi:10.1126/science.1152692. PMC 3431203. PMID 18323453.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":22,"type":"book","title":"Hayden EC (February 13, 2008). \"Protein prize up for grabs after retraction\". Nature. doi:10.1038/news.2008.569.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":23,"type":"book","title":"\"Disease Related Research\". Rosetta@home. University of Washington. 2008. Archived from the original on September 23, 2008. Retrieved October 8, 2008.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":24,"type":"book","title":"Baker D (2008). \"Rosetta@home: David Baker's Rosetta@home journal\". Rosetta@home forums. University of Washington. Retrieved September 7, 2008.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":25,"type":"book","title":"\"Rosetta@home Research Updates\". Boinc.bakerlab.org. Retrieved 2014-04-18.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":26,"type":"book","title":"\"News archive\". Rosetta@home. Retrieved 10 May 2019.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":27,"type":"book","title":"Kuhlman B, Baker D (September 2000). \"Native protein sequences are close to optimal for their structures\". Proceedings of the National Academy of Sciences of the United States of America. 97 (19): 10383–88. Bibcode:2000PNAS...9710383K. doi:10.1073/pnas.97.19.10383. PMC 27033. PMID 10984534.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":28,"type":"book","title":"Thompson MJ, Sievers SA, Karanicolas J, Ivanova MI, Baker D, Eisenberg D (March 2006). \"The 3D profile method for identifying fibril-forming segments of proteins\". Proceedings of the National Academy of Sciences of the United States of America. 103 (11): 4074–78. Bibcode:2006PNAS..103.4074T. doi:10.1073/pnas.0511295103. PMC 1449648. PMID 16537487.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":29,"type":"book","title":"Bradley P. \"Rosetta@home forum: Amyloid fibril structure prediction\". Rosetta@home forums. University of Washington. Retrieved September 7, 2008.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":30,"type":"book","title":"Baker D. \"Rosetta@home forum: Publications on R@H's Alzheimer's work? (message 54681)\". Rosetta@home forums. University of Washington. Retrieved October 8, 2008.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":31,"type":"book","title":"Wang C, Schueler-Furman O, Baker D (May 2005). \"Improved side-chain modeling for protein–protein docking\". Protein Science. 14 (5): 1328–39. doi:10.1110/ps.041222905. PMC 2253276. PMID 15802647.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":32,"type":"book","title":"Gray JJ, Moughon S, Wang C, et al. (August 2003). \"Protein–protein docking with simultaneous optimization of rigid-body displacement and side-chain conformations\". Journal of Molecular Biology. 331 (1): 281–99. doi:10.1016/S0022-2836(03)00670-3. PMID 12875852.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":33,"type":"book","title":"Schueler-Furman O, Wang C, Baker D (August 2005). \"Progress in protein–protein docking: atomic resolution predictions in the CAPRI experiment using RosettaDock with an improved treatment of side-chain flexibility\". Proteins. 60 (2): 187–94. doi:10.1002/prot.20556. PMID 15981249.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":34,"type":"book","title":"Lacy DB, Lin HC, Melnyk RA, et al. (November 2005). \"A model of anthrax toxin lethal factor bound to protective antigen\". Proceedings of the National Academy of Sciences of the United States of America. 102 (45): 16409–14. Bibcode:2005PNAS..10216409L. doi:10.1073/pnas.0508259102. PMC 1283467. PMID 16251269.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":35,"type":"book","title":"Albrecht MT, Li H, Williamson ED, et al. (November 2007). \"Human monoclonal antibodies against anthrax lethal factor and protective antigen act independently to protect against Bacillus anthracis infection and enhance endogenous immunity to anthrax\". Infection and Immunity. 75 (11): 5425–33. doi:10.1128/IAI.00261-07. PMC 2168292. PMID 17646360.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":36,"type":"book","title":"Sprague ER, Wang C, Baker D, Bjorkman PJ (June 2006). \"Crystal structure of the HSV-1 Fc receptor bound to Fc reveals a mechanism for antibody bipolar bridging\". PLoS Biology. 4 (6): e148. doi:10.1371/journal.pbio.0040148. PMC 1450327. PMID 16646632.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":37,"type":"book","title":"Paulson, Tom (July 19, 2006). \"Gates Foundation awards $287 million for HIV vaccine research\". Seattle Post-Intelligencer. Retrieved September 7, 2008.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":38,"type":"book","title":"Liu Y, et al. (2007). \"Development of IgG1 b12 scaffolds and HIV-1 env-based outer domain immunogens capable of eliciting and detecting IgG1 b12-like antibodies\" (PDF). Global HIV Vaccine Enterprise. Archived from the original (PDF) on February 25, 2009. Retrieved September 28, 2008.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":39,"type":"book","title":"Baker D. \"David Baker's Rosetta@home journal archives (message 40756)\". Rosetta@home forums. University of Washington. Retrieved September 7, 2008.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":40,"type":"book","title":"\"Homing Endonuclease Genes: New Tools for Mosquito Population Engineering and Control\". Grand Challenges in Global Health. Retrieved September 7, 2008.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":41,"type":"book","title":"Windbichler N, Papathanos PA, Catteruccia F, Ranson H, Burt A, Crisanti A (2007). \"Homing endonuclease mediated gene targeting in Anopheles gambiae cells and embryos\". Nucleic Acids Research. 35 (17): 5922–33. doi:10.1093/nar/gkm632. PMC 2034484. PMID 17726053.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":42,"type":"book","title":"Ashworth J, Havranek JJ, Duarte CM, et al. (June 2006). \"Computational redesign of endonuclease DNA binding and cleavage specificity\". Nature. 441 (7093): 656–59. Bibcode:2006Natur.441..656A. doi:10.1038/nature04818. PMC 2999987. PMID 16738662.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":43,"type":"book","title":"Silva DA, Yu S, Ulge UY, Spangler JB, Jude KM, Labão-Almeida C, Ali LR, Quijano-Rubio A, Ruterbusch M, Leung I, Biary T, Crowley SJ, Marcos E, Walkey CD, Weitzner BD, Pardo-Avila F, Castellanos J, Carter L, Stewart L, Riddell SR, Pepper M, Bernardes GJ, Dougan M, Garcia KC, Baker D (January 2019). \"De novo design of potent and selective mimics of IL-2 and IL-15\". Nature. 565 (7738): 186–191. doi:10.1038/s41586-018-0830-7. PMID 30626941.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":44,"type":"book","title":"Simons KT, Bonneau R, Ruczinski I, Baker D (1999). \"Ab initio protein structure prediction of CASP III targets using Rosetta\". Proteins. Suppl 3: 171–76. doi:10.1002/(SICI)1097-0134(1999)37:3+<171::AID-PROT21>3.0.CO;2-Z. PMID 10526365.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":45,"type":"book","title":"\"Interview with David Baker\". Team Picard Distributed Computing. 2006. Archived from the original on February 18, 2009. Retrieved December 23, 2008.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":46,"type":"book","title":"Nauli S, Kuhlman B, Baker D (July 2001). \"Computer-based redesign of a protein folding pathway\". Nature Structural Biology. 8 (7): 602–05. doi:10.1038/89638. PMID 11427890.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":47,"type":"book","title":"Kuhlman B, Dantas G, Ireton GC, Varani G, Stoddard BL, Baker D (November 2003). \"Design of a novel globular protein fold with atomic-level accuracy\". Science. 302 (5649): 1364–68. Bibcode:2003Sci...302.1364K. doi:10.1126/science.1089427. PMID 14631033.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":48,"type":"book","title":"Jones DT (November 2003). \"Structural biology. Learning to speak the language of proteins\". Science. 302 (5649): 1347–48. doi:10.1126/science.1092492. PMID 14631028.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":49,"type":"book","title":"von Grotthuss M, Wyrwicz LS, Pas J, Rychlewski L (June 2004). \"Predicting protein structures accurately\". Science. 304 (5677): 1597–99, author reply 1597–99. doi:10.1126/science.304.5677.1597b. PMID 15192202.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":50,"type":"book","title":"\"Articles citing: Kuhlman et al. (2003) 'Design of a novel globular protein fold with atomic-level accuracy'\". ISI Web of Science. Retrieved July 10, 2008.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":51,"type":"book","title":"\"October 2005 molecule of the month: Designer proteins\". RCSB Protein Data Bank. Retrieved September 7, 2008.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":52,"type":"book","title":"\"Kuhlman laboratory homepage\". Kuhlman Laboratory. University of North Carolina. Retrieved September 7, 2008.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":53,"type":"book","title":"\"RosettaDesign web server\". Kuhlman Laboratory. University of North Carolina. Retrieved September 7, 2008.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":54,"type":"book","title":"Gray JJ, Moughon SE, Kortemme T, et al. (July 2003). \"Protein–protein docking predictions for the CAPRI experiment\". Proteins. 52 (1): 118–22. CiteSeerX 10.1.1.80.9354. doi:10.1002/prot.10384. PMID 12784377.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":55,"type":"book","title":"Daily MD, Masica D, Sivasubramanian A, Somarouthu S, Gray JJ (2005). \"CAPRI rounds 3–5 reveal promising successes and future challenges for RosettaDock\". Proteins. 60 (2): 181–86. CiteSeerX 10.1.1.521.9981. doi:10.1002/prot.20555. PMID 15981262. Archived from the original on 2012-06-30.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":56,"type":"book","title":"Méndez R, Leplae R, Lensink MF, Wodak SJ (2005). \"Assessment of CAPRI predictions in rounds 3–5 shows progress in docking procedures\". Proteins. 60 (2): 150–69. doi:10.1002/prot.20551. PMID 15981261. Archived from the original on 2012-06-30.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":57,"type":"book","title":"\"RosettaDock server\". Gray laboratory. Johns Hopkins University. Retrieved September 7, 2008.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":58,"type":"book","title":"\"Protein–protein docking at Rosetta@home\". Rosetta@home forums. University of Washington. Retrieved September 7, 2008.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":59,"type":"book","title":"\"Robetta web server\". Baker laboratory. University of Washington. Retrieved May 7, 2019.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":60,"type":"book","title":"Aloy P, Stark A, Hadley C, Russell RB (2003). \"Predictions without templates: new folds, secondary structure, and contacts in CASP5\". Proteins. 53 Suppl 6: 436–56. doi:10.1002/prot.10546. PMID 14579333.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":61,"type":"book","title":"Tress M, Ezkurdia I, Graña O, López G, Valencia A (2005). \"Assessment of predictions submitted for the CASP6 comparative modeling category\". Proteins. 61 Suppl 7: 27–45. doi:10.1002/prot.20720. PMID 16187345.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":62,"type":"book","title":"Battey JN, Kopp J, Bordoli L, Read RJ, Clarke ND, Schwede T (2007). \"Automated server predictions in CASP7\". Proteins. 69 Suppl 8: 68–82. doi:10.1002/prot.21761. PMID 17894354.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":63,"type":"book","title":"Chivian D, Kim DE, Malmström L, Schonbrun J, Rohl CA, Baker D (2005). \"Prediction of CASP6 structures using automated Robetta protocols\". Proteins. 61 Suppl 7: 157–66. doi:10.1002/prot.20733. PMID 16187358.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":64,"type":"book","title":"Baker D. \"David Baker's Rosetta@home journal, message 52902\". Rosetta@home forums. University of Washington. Retrieved September 7, 2008.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":65,"type":"book","title":"Ovchinnikov, S; Kim, DE; Wang, RY; Liu, Y; DiMaio, F; Baker, D (September 2016). \"Improved de novo structure prediction in CASP11 by incorporating coevolution information into Rosetta\". Proteins. 84 Suppl 1: 67–75. doi:10.1002/prot.24974. PMID 26677056.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":66,"type":"book","title":"Baker D. \"David Baker's Rosetta@home journal (message 52963)\". Rosetta@home forums. University of Washington. Retrieved September 16, 2008.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":67,"type":"book","title":"\"Foldit forums: How many users does Foldit have? Etc. (message 2)\". University of Washington. Retrieved September 27, 2008.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":68,"type":"book","title":"\"Foldit: Frequently Asked Questions\". fold.it. University of Washington. Retrieved September 19, 2008.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":69,"type":"book","title":"\"Project list – BOINC\". University of California. Retrieved September 8, 2008.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":70,"type":"book","title":"Pande Group (2010). \"High Performance FAQ\". Stanford University. Archived from the original (FAQ) on September 21, 2012. Retrieved September 19, 2011.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":71,"type":"book","title":"7im (April 2, 2010). \"Re: Answers to: Reasons for not using F@H\". Retrieved 2011-09-19.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":72,"type":"book","title":"Vijay Pande (August 5, 2011). \"Results page updated – new key result published in our work in Alzheimer's Disease\". Retrieved 2011-09-19.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":73,"type":"book","title":"Pande Group. \"Folding@home Diseases Studied FAQ\". Stanford University. Archived from the original (FAQ) on October 11, 2007. Retrieved September 12, 2011.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":74,"type":"book","title":"Vijay Pande (September 26, 2007). \"How FAH works: Molecular dynamics\". Retrieved 2011-09-10.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":75,"type":"book","title":"tjlane (June 9, 2011). \"Re: Course grained Protein folding in under 10 minutes\". Retrieved 2011-09-19.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":76,"type":"book","title":"jmn (July 29, 2011). \"Rosetta@home and Folding@home: additional projects\". Retrieved 2011-09-19.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":77,"type":"book","title":"Pande Group. \"Client Statistics by OS\". Stanford University. Retrieved 2011-10-18.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":78,"type":"book","title":"Malmström L, Riffle M, Strauss CE, et al. (April 2007). \"Superfamily assignments for the yeast proteome through integration of structure prediction with the gene ontology\". PLoS Biology. 5 (4): e76. doi:10.1371/journal.pbio.0050076. PMC 1828141. PMID 17373854.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":79,"type":"book","title":"Bonneau R (2006). \"World Community Grid Message Board Posts: HPF -> HPF2 transition\". Bonneau Lab, New York University. Retrieved September 7, 2008.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":80,"type":"book","title":"\"List of Richard Bonneau's publications\". Bonneau Lab, New York University. Archived from the original on July 7, 2008. Retrieved September 7, 2008.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":81,"type":"book","title":"Bonneau R. \"World Community Grid Message Board Posts\". Bonneau Lab, New York University. Archived from the original on July 4, 2008. Retrieved September 7, 2008.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":82,"type":"book","title":"\"Predictor@home: Developing new application areas for P@H\". The Brooks Research Group. Retrieved September 7, 2008.[失效连结]","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":83,"type":"book","title":"Carrillo-Tripp M (2007). \"dTASSER\". The Scripps Research Institute. Archived from the original on July 6, 2007. Retrieved September 7, 2008.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":84,"type":"book","title":"\"RALPH@home website\". RALPH@home forums. University of Washington. Retrieved September 7, 2008.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":85,"type":"book","title":"\"Rosetta@home: The new credit system explained\". Rosetta@home forums. University of Washington. 2006. Retrieved October 8, 2008.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":86,"type":"book","title":"\"BOINCstats: Project Credit Comparison\". boincstats.com. 2008. Archived from the original on September 13, 2008. Retrieved October 8, 2008.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":87,"type":"book","title":"\"Credit divided over projects\". boincstats.com. Retrieved February 19, 2015.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":88,"type":"book","title":"\"Rosetta@home: Predictor of the day archive\". Rosetta@home. University of Washington. 2008. Archived from the original on September 24, 2008. Retrieved October 8, 2008.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false},{"id":89,"type":"book","title":"\"Rosetta@home: Protein Folding, Design, and Docking\". Rosetta@home. University of Washington. 2008. Retrieved October 8, 2008.","site":"","url":"","journalName":null,"author":"","press":"","publishYear":"","publishTime":null,"publishPlace":"","page":"","volume":"","quoteTime":null,"quoted":false}],"recommendReferences":null,"auditState":2,"lemmaLevel":1,"origin":0,"originEnTitle":null,"originZhTitle":null,"pv":129,"auditType":0,"synonyms":null,"showEditTime":"2019.12.13 18:46","auditors":[{"uid":0,"name":"Ki.κe","pic":"https://web.archive.org/web/20221025114123/https://wx.qlogo.cn/mmopen/vi_32/y67kfr32Doib4wg71Jiau7jVWvharic3nRKgdRRQSl6koeQJCo0GQs2Krw0vwdFRsOWnHIQOwAZsSg5lIkIrFCOcQ/132","introduction":null,"educations":null,"jobs":null,"works":null,"educationBrief":"","jobBrief":"","role":0,"roleName":null,"title":"","professionalTitle":null,"phoneNo":null,"editable":true,"partnerId":0,"partnerIdCreateTime":0,"partnerIdPoped":false}],"hasZhishiNav":false,"auditInfos":{},"isHistory":false};</script><script crossorigin="anonymous" src="./696.Rosetta@home - 搜狗科学百科_files/aegis.min.js.download"></script><script crossorigin="anonymous" src="./696.Rosetta@home - 搜狗科学百科_files/main_2020092401.js.download"></script><script crossorigin="anonymous" src="./696.Rosetta@home - 搜狗科学百科_files/react.production.min.js.download"></script><script crossorigin="anonymous" src="./696.Rosetta@home - 搜狗科学百科_files/react-dom.production.min.js.download"></script><script crossorigin="anonymous" src="./696.Rosetta@home - 搜狗科学百科_files/jquery-1.11.1.min.js.download"></script><script crossorigin="anonymous" src="./696.Rosetta@home - 搜狗科学百科_files/main_2022062701.js.download"></script><script crossorigin="anonymous" src="./696.Rosetta@home - 搜狗科学百科_files/main_66bbe21.js.download"></script><script crossorigin="anonymous" src="./696.Rosetta@home - 搜狗科学百科_files/react.production.min.js.download"></script><script crossorigin="anonymous" src="./696.Rosetta@home - 搜狗科学百科_files/react-dom.production.min.js.download"></script><script crossorigin="anonymous" src="./696.Rosetta@home - 搜狗科学百科_files/main_edf0f08.js.download"></script>
</body></html>